{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[32m\u001b[1m   Updating\u001b[22m\u001b[39m registry at `C:\\Users\\Chandrapal Panwar\\.julia\\registries\\General`\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[?25l"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[32m\u001b[1m   Updating\u001b[22m\u001b[39m git-repo `https://github.com/JuliaRegistries/General.git`\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "    \u001b[36m\u001b[1mFetching:\u001b[22m\u001b[39m [>                                        ]  0.0 %\r",
      "    \u001b[36m\u001b[1mFetching:\u001b[22m\u001b[39m [=======>                                 ]  16.7 %\r",
      "    \u001b[36m\u001b[1mFetching:\u001b[22m\u001b[39m [==============>                          ]  33.3 %\r",
      "    \u001b[36m\u001b[1mFetching:\u001b[22m\u001b[39m [====================>                    ]  50.0 %\r",
      "    \u001b[36m\u001b[1mFetching:\u001b[22m\u001b[39m [===========================>             ]  66.7 %\r",
      "    \u001b[36m\u001b[1mFetching:\u001b[22m\u001b[39m [==================================>      ]  83.3 %\r",
      "    \u001b[36m\u001b[1mFetching:\u001b[22m\u001b[39m [========================================>]  100.0 %\r",
      "\u001b[2K\u001b[?25h"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[32m\u001b[1m  Resolving\u001b[22m\u001b[39m package versions...\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m StrTables ──────────────────── v1.0.1\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m Missings ───────────────────── v0.4.3\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m DiffRules ──────────────────── v1.0.1\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m TextAnalysis ───────────────── v0.6.0\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m DataStructures ─────────────── v0.17.15\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m IRTools ────────────────────── v0.3.2\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m ForwardDiff ────────────────── v0.10.10\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m CommonSubexpressions ───────── v0.2.0\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m CEnum ──────────────────────── v0.2.0\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m ZipFile ────────────────────── v0.9.1\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m DataAPI ────────────────────── v1.3.0\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m OrderedCollections ─────────── v1.1.0\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m CuArrays ───────────────────── v2.1.0\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m Reexport ───────────────────── v0.2.0\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m TranscodingStreams ─────────── v0.9.5\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m FixedPointNumbers ──────────── v0.8.0\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m StaticArrays ───────────────── v0.12.3\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m WordTokenizers ─────────────── v0.5.4\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m CodecZlib ──────────────────── v0.7.0\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m ZygoteRules ────────────────── v0.2.0\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m AbstractFFTs ───────────────── v0.5.0\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m IteratorInterfaceExtensions ── v1.0.0\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m Tables ─────────────────────── v1.0.4\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m LLVM ───────────────────────── v1.3.4\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m TimerOutputs ───────────────── v0.5.5\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m SpecialFunctions ───────────── v0.10.0\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m HTML_Entities ──────────────── v1.0.0\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m NNlib ──────────────────────── v0.6.6\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m InvertedIndices ────────────── v1.0.0\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m Flux ───────────────────────── v0.10.4\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m CUDAdrv ────────────────────── v6.2.3\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m ColorTypes ─────────────────── v0.10.2\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m Adapt ──────────────────────── v1.0.1\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m AbstractTrees ──────────────── v0.3.3\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m BinaryProvider ─────────────── v0.5.9\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m TableTraits ────────────────── v1.0.0\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m CompilerSupportLibraries_jll ─ v0.3.3+0\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m SortingAlgorithms ──────────── v0.3.1\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m ArrayLayouts ───────────────── v0.2.5\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m PooledArrays ───────────────── v0.5.3\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m DiffResults ────────────────── v1.0.2\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m FillArrays ─────────────────── v0.8.8\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m OpenSpecFun_jll ────────────── v0.5.3+3\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m Zlib_jll ───────────────────── v1.2.11+9\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m ExprTools ──────────────────── v0.1.1\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m CodeTracking ───────────────── v0.5.11\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m Languages ──────────────────── v0.4.3\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m StatsBase ──────────────────── v0.33.0\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m NaNMath ────────────────────── v0.3.3\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m Zygote ─────────────────────── v0.4.19\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m Requires ───────────────────── v1.0.1\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m CategoricalArrays ──────────── v0.7.7\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m MacroTools ─────────────────── v0.5.5\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m Juno ───────────────────────── v0.8.1\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m Media ──────────────────────── v0.5.0\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m DataValueInterfaces ────────── v1.0.0\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m DataFrames ─────────────────── v0.20.2\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m CUDAnative ─────────────────── v3.0.4\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m CUDAapi ────────────────────── v4.0.0\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m Compat ─────────────────────── v3.9.0\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m BSON ───────────────────────── v0.2.6\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m GPUArrays ──────────────────── v3.2.0\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m Cthulhu ────────────────────── v1.0.2\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m Colors ─────────────────────── v0.12.0\n",
      "\u001b[32m\u001b[1m   Updating\u001b[22m\u001b[39m `C:\\Users\\Chandrapal Panwar\\.julia\\environments\\v1.4\\Project.toml`\n",
      " \u001b[90m [a2db99b7]\u001b[39m\u001b[92m + TextAnalysis v0.6.0\u001b[39m\n",
      "\u001b[32m\u001b[1m   Updating\u001b[22m\u001b[39m `C:\\Users\\Chandrapal Panwar\\.julia\\environments\\v1.4\\Manifest.toml`\n",
      " \u001b[90m [621f4979]\u001b[39m\u001b[92m + AbstractFFTs v0.5.0\u001b[39m\n",
      " \u001b[90m [1520ce14]\u001b[39m\u001b[92m + AbstractTrees v0.3.3\u001b[39m\n",
      " \u001b[90m [79e6a3ab]\u001b[39m\u001b[92m + Adapt v1.0.1\u001b[39m\n",
      " \u001b[90m [4c555306]\u001b[39m\u001b[92m + ArrayLayouts v0.2.5\u001b[39m\n",
      " \u001b[90m [fbb218c0]\u001b[39m\u001b[92m + BSON v0.2.6\u001b[39m\n",
      " \u001b[90m [b99e7846]\u001b[39m\u001b[92m + BinaryProvider v0.5.9\u001b[39m\n",
      " \u001b[90m [fa961155]\u001b[39m\u001b[92m + CEnum v0.2.0\u001b[39m\n",
      " \u001b[90m [3895d2a7]\u001b[39m\u001b[92m + CUDAapi v4.0.0\u001b[39m\n",
      " \u001b[90m [c5f51814]\u001b[39m\u001b[92m + CUDAdrv v6.2.3\u001b[39m\n",
      " \u001b[90m [be33ccc6]\u001b[39m\u001b[92m + CUDAnative v3.0.4\u001b[39m\n",
      " \u001b[90m [324d7699]\u001b[39m\u001b[92m + CategoricalArrays v0.7.7\u001b[39m\n",
      " \u001b[90m [da1fd8a2]\u001b[39m\u001b[92m + CodeTracking v0.5.11\u001b[39m\n",
      " \u001b[90m [944b1d66]\u001b[39m\u001b[92m + CodecZlib v0.7.0\u001b[39m\n",
      " \u001b[90m [3da002f7]\u001b[39m\u001b[92m + ColorTypes v0.10.2\u001b[39m\n",
      " \u001b[90m [5ae59095]\u001b[39m\u001b[92m + Colors v0.12.0\u001b[39m\n",
      " \u001b[90m [bbf7d656]\u001b[39m\u001b[92m + CommonSubexpressions v0.2.0\u001b[39m\n",
      " \u001b[90m [34da2185]\u001b[39m\u001b[92m + Compat v3.9.0\u001b[39m\n",
      " \u001b[90m [e66e0078]\u001b[39m\u001b[92m + CompilerSupportLibraries_jll v0.3.3+0\u001b[39m\n",
      " \u001b[90m [f68482b8]\u001b[39m\u001b[92m + Cthulhu v1.0.2\u001b[39m\n",
      " \u001b[90m [3a865a2d]\u001b[39m\u001b[92m + CuArrays v2.1.0\u001b[39m\n",
      " \u001b[90m [9a962f9c]\u001b[39m\u001b[92m + DataAPI v1.3.0\u001b[39m\n",
      " \u001b[90m [a93c6f00]\u001b[39m\u001b[92m + DataFrames v0.20.2\u001b[39m\n",
      " \u001b[90m [864edb3b]\u001b[39m\u001b[92m + DataStructures v0.17.15\u001b[39m\n",
      " \u001b[90m [e2d170a0]\u001b[39m\u001b[92m + DataValueInterfaces v1.0.0\u001b[39m\n",
      " \u001b[90m [163ba53b]\u001b[39m\u001b[92m + DiffResults v1.0.2\u001b[39m\n",
      " \u001b[90m [b552c78f]\u001b[39m\u001b[92m + DiffRules v1.0.1\u001b[39m\n",
      " \u001b[90m [e2ba6199]\u001b[39m\u001b[92m + ExprTools v0.1.1\u001b[39m\n",
      " \u001b[90m [1a297f60]\u001b[39m\u001b[92m + FillArrays v0.8.8\u001b[39m\n",
      " \u001b[90m [53c48c17]\u001b[39m\u001b[92m + FixedPointNumbers v0.8.0\u001b[39m\n",
      " \u001b[90m [587475ba]\u001b[39m\u001b[92m + Flux v0.10.4\u001b[39m\n",
      " \u001b[90m [f6369f11]\u001b[39m\u001b[92m + ForwardDiff v0.10.10\u001b[39m\n",
      " \u001b[90m [0c68f7d7]\u001b[39m\u001b[92m + GPUArrays v3.2.0\u001b[39m\n",
      " \u001b[90m [7693890a]\u001b[39m\u001b[92m + HTML_Entities v1.0.0\u001b[39m\n",
      " \u001b[90m [7869d1d1]\u001b[39m\u001b[92m + IRTools v0.3.2\u001b[39m\n",
      " \u001b[90m [41ab1584]\u001b[39m\u001b[92m + InvertedIndices v1.0.0\u001b[39m\n",
      " \u001b[90m [82899510]\u001b[39m\u001b[92m + IteratorInterfaceExtensions v1.0.0\u001b[39m\n",
      " \u001b[90m [e5e0dc1b]\u001b[39m\u001b[92m + Juno v0.8.1\u001b[39m\n",
      " \u001b[90m [929cbde3]\u001b[39m\u001b[92m + LLVM v1.3.4\u001b[39m\n",
      " \u001b[90m [8ef0a80b]\u001b[39m\u001b[92m + Languages v0.4.3\u001b[39m\n",
      " \u001b[90m [1914dd2f]\u001b[39m\u001b[92m + MacroTools v0.5.5\u001b[39m\n",
      " \u001b[90m [e89f7d12]\u001b[39m\u001b[92m + Media v0.5.0\u001b[39m\n",
      " \u001b[90m [e1d29d7a]\u001b[39m\u001b[92m + Missings v0.4.3\u001b[39m\n",
      " \u001b[90m [872c559c]\u001b[39m\u001b[92m + NNlib v0.6.6\u001b[39m\n",
      " \u001b[90m [77ba4419]\u001b[39m\u001b[92m + NaNMath v0.3.3\u001b[39m\n",
      " \u001b[90m [efe28fd5]\u001b[39m\u001b[92m + OpenSpecFun_jll v0.5.3+3\u001b[39m\n",
      " \u001b[90m [bac558e1]\u001b[39m\u001b[92m + OrderedCollections v1.1.0\u001b[39m\n",
      " \u001b[90m [2dfb63ee]\u001b[39m\u001b[92m + PooledArrays v0.5.3\u001b[39m\n",
      " \u001b[90m [189a3867]\u001b[39m\u001b[92m + Reexport v0.2.0\u001b[39m\n",
      " \u001b[90m [ae029012]\u001b[39m\u001b[92m + Requires v1.0.1\u001b[39m\n",
      " \u001b[90m [a2af1166]\u001b[39m\u001b[92m + SortingAlgorithms v0.3.1\u001b[39m\n",
      " \u001b[90m [276daf66]\u001b[39m\u001b[92m + SpecialFunctions v0.10.0\u001b[39m\n",
      " \u001b[90m [90137ffa]\u001b[39m\u001b[92m + StaticArrays v0.12.3\u001b[39m\n",
      " \u001b[90m [2913bbd2]\u001b[39m\u001b[92m + StatsBase v0.33.0\u001b[39m\n",
      " \u001b[90m [9700d1a9]\u001b[39m\u001b[92m + StrTables v1.0.1\u001b[39m\n",
      " \u001b[90m [3783bdb8]\u001b[39m\u001b[92m + TableTraits v1.0.0\u001b[39m\n",
      " \u001b[90m [bd369af6]\u001b[39m\u001b[92m + Tables v1.0.4\u001b[39m\n",
      " \u001b[90m [a2db99b7]\u001b[39m\u001b[92m + TextAnalysis v0.6.0\u001b[39m\n",
      " \u001b[90m [a759f4b9]\u001b[39m\u001b[92m + TimerOutputs v0.5.5\u001b[39m\n",
      " \u001b[90m [3bb67fe8]\u001b[39m\u001b[92m + TranscodingStreams v0.9.5\u001b[39m\n",
      " \u001b[90m [796a5d58]\u001b[39m\u001b[92m + WordTokenizers v0.5.4\u001b[39m\n",
      " \u001b[90m [a5390f91]\u001b[39m\u001b[92m + ZipFile v0.9.1\u001b[39m\n",
      " \u001b[90m [83775a58]\u001b[39m\u001b[92m + Zlib_jll v1.2.11+9\u001b[39m\n",
      " \u001b[90m [e88e6eb3]\u001b[39m\u001b[92m + Zygote v0.4.19\u001b[39m\n",
      " \u001b[90m [700de1a5]\u001b[39m\u001b[92m + ZygoteRules v0.2.0\u001b[39m\n",
      " \u001b[90m [8bb1440f]\u001b[39m\u001b[92m + DelimitedFiles \u001b[39m\n",
      " \u001b[90m [9fa8497b]\u001b[39m\u001b[92m + Future \u001b[39m\n",
      " \u001b[90m [37e2e46d]\u001b[39m\u001b[92m + LinearAlgebra \u001b[39m\n",
      " \u001b[90m [9abbd945]\u001b[39m\u001b[92m + Profile \u001b[39m\n",
      " \u001b[90m [1a1011a3]\u001b[39m\u001b[92m + SharedArrays \u001b[39m\n",
      " \u001b[90m [2f01184e]\u001b[39m\u001b[92m + SparseArrays \u001b[39m\n",
      " \u001b[90m [10745b16]\u001b[39m\u001b[92m + Statistics \u001b[39m\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[32m\u001b[1m   Building\u001b[22m\u001b[39m HTML_Entities → `C:\\Users\\Chandrapal Panwar\\.julia\\packages\\HTML_Entities\\g4t7p\\deps\\build.log`\n",
      "\u001b[32m\u001b[1m   Building\u001b[22m\u001b[39m NNlib ────────→ `C:\\Users\\Chandrapal Panwar\\.julia\\packages\\NNlib\\FAI3o\\deps\\build.log`\n",
      "\u001b[32m\u001b[1m   Building\u001b[22m\u001b[39m TextAnalysis ─→ `C:\\Users\\Chandrapal Panwar\\.julia\\packages\\TextAnalysis\\pcFQf\\deps\\build.log`\n"
     ]
    }
   ],
   "source": [
    "using Pkg\n",
    "Pkg.add(\"TextAnalysis\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "ename": "UndefVarError",
     "evalue": "UndefVarError: clone not defined",
     "output_type": "error",
     "traceback": [
      "UndefVarError: clone not defined",
      "",
      "Stacktrace:",
      " [1] getproperty(::Module, ::Symbol) at .\\Base.jl:26",
      " [2] top-level scope at In[7]:1"
     ]
    }
   ],
   "source": [
    "\"WordTokenizers\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[32m\u001b[1m  Resolving\u001b[22m\u001b[39m package versions...\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m PyCall ─ v1.91.4\n",
      "\u001b[32m\u001b[1m   Updating\u001b[22m\u001b[39m `C:\\Users\\Chandrapal Panwar\\.julia\\environments\\v1.4\\Project.toml`\n",
      " \u001b[90m [438e738f]\u001b[39m\u001b[92m + PyCall v1.91.4\u001b[39m\n",
      "\u001b[32m\u001b[1m   Updating\u001b[22m\u001b[39m `C:\\Users\\Chandrapal Panwar\\.julia\\environments\\v1.4\\Manifest.toml`\n",
      " \u001b[90m [438e738f]\u001b[39m\u001b[92m + PyCall v1.91.4\u001b[39m\n",
      "\u001b[32m\u001b[1m   Building\u001b[22m\u001b[39m PyCall → `C:\\Users\\Chandrapal Panwar\\.julia\\packages\\PyCall\\zqDXB\\deps\\build.log`\n"
     ]
    }
   ],
   "source": [
    "Pkg.add(\"PyCall\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "using WordTokenizers"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[32m\u001b[1m  Resolving\u001b[22m\u001b[39m package versions...\n",
      "\u001b[32m\u001b[1m   Updating\u001b[22m\u001b[39m `C:\\Users\\Chandrapal Panwar\\.julia\\environments\\v1.4\\Project.toml`\n",
      "\u001b[90m [no changes]\u001b[39m\n",
      "\u001b[32m\u001b[1m   Updating\u001b[22m\u001b[39m `C:\\Users\\Chandrapal Panwar\\.julia\\environments\\v1.4\\Manifest.toml`\n",
      "\u001b[90m [no changes]\u001b[39m\n"
     ]
    }
   ],
   "source": [
    "Pkg.add(\"WordTokenizers\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 81,
   "metadata": {},
   "outputs": [],
   "source": [
    "using TextAnalysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 82,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "\"The best error message is the one that never shows up.\\nYou Learn More From Failure Than From Success. \\nThe purpose of software engineering is to control complexity, not to create it\""
      ]
     },
     "execution_count": 82,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "mystr = \"\"\"The best error message is the one that never shows up.\n",
    "You Learn More From Failure Than From Success. \n",
    "The purpose of software engineering is to control complexity, not to create it\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "StringDocument{String}(\"The best error message is the one that never shows up.\\nYou Learn More From Failure Than From Success. \\nThe purpose of software engineering is to control complexity, not to create it\", TextAnalysis.DocumentMetadata(Languages.English(), \"Untitled Document\", \"Unknown Author\", \"Unknown Time\"))"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Basic Way\n",
    "sd1 = Document(mystr)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "StringDocument{String}(\"The best error message is the one that never shows up.\\nYou Learn More From Failure Than From Success. \\nThe purpose of software engineering is to control complexity, not to create it\", TextAnalysis.DocumentMetadata(Languages.English(), \"Untitled Document\", \"Unknown Author\", \"Unknown Time\"))"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Best Way\n",
    "sd2 = StringDocument(mystr)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "\"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_0.txt\""
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "filepath = \"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_0.txt\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "FileDocument(\"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\", TextAnalysis.DocumentMetadata(Languages.English(), \"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\", \"Unknown Author\", \"Unknown Time\"))"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "filedoc = Document(\"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "FileDocument(\"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\", TextAnalysis.DocumentMetadata(Languages.English(), \"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\", \"Unknown Author\", \"Unknown Time\"))"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "fd = FileDocument(\"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "\"Odor perception, and emotional and behavioral responses to odors strongly depend on experience, and learned odor-context associations often last for the lifetime of an animal (Mouly and Sullivan, 2010) . The cellular and neural circuit mechanisms underlying olfactory learning and memory, however, remain poorly understood.\\r\\n Recent studies on episodic and contextual learning in hippocampal neural networks have suggested that memories are encoded in the activity of distributed ensembles of neurons, often referred to as a 'memory trace' (Mayford and Reijmers, 2015; Poo et al., 2016; Tonegawa et al., 2015) . The neurons constituting such a memory trace are thought to encode information about the environmental context and associated emotions of past experiences, and their activity is necessary and sufficient for memory retrieval (Liu et al., 2012; Reijmers et al., 2007; Tanaka et al., 2014) .\\r\\n Here, we investigate the organization of odor memory traces in the olfactory (piriform) cortex of mice. The piriform cortex, a trilaminar paleocortical structure, is the largest cortical area receiving direct afferent inputs from the olfactory bulb, which, in turn, receives topographically organized inputs from olfactory sensory neurons in the nose. Individual piriform neurons respond to combinatorial inputs from multiple Piriform cortex has long been hypothesized to support auto-associative network functions that can retrieve previously learned information from partial or degraded sensory inputs (Haberly, 2001; Wilson and Sullivan, 2011) . Piriform pyramidal cells form a large recurrent network, which is reciprocally connected with adjacent highorder associative areas including the prefrontal, entorhinal and perirhinal cortex and the amygdala (Johnson et al., 2000; Sadrian and Wilson, 2015) . Storage of information is made possible by NMDA-dependent, associative plasticity of connections (Johenning et al., 2009; Kanter and Haberly, 1990; Quinlan et al., 2004) .\\r\\n Furthermore, changes in piriform network activity (Chapuis and Wilson, 2011; Chen et al., 2011; Li et al., 2008; Sevelinges et al., 2004) and stabilization of piriform odor representations (Shakhawat et al., 2015) have been observed after associative olfactory learning. Finally, excitotoxic lesions of the posterior piriform cortex in rats perturb odor fear memories (Sacco and Sacchetti, 2010) , and optogenetic stimulation of artificial piriform ensembles is sufficient to drive learned behaviors (Choi et al., 2011) . Taken together, these studies have led to the hypothesis that piriform neural ensembles encode olfactory memory traces.\\r\\n To test this hypothesis, we have developed an intersectional genetic strategy in mice to target piriform neurons that are activated by olfactory experience. We employed\\r\\n To selectively label and manipulate piriform neurons that were activated during odor exposure, we used cFos-tTA transgenic mice (Reijmers et al., 2007) , in which the activity-dependent cFos promoter drives expression of the tTA transcription factor, and we stereotaxically injected in their piriform cortex Adeno-Associated Viruses (AAVs) expressing Designer Receptors Exclusively Activated by Designer Drugs (DREADDs) under the control of the tTA-responsive promoter tetO ( Figure 1A) .\\r\\n DREADDs (Alexander et al., 2009; Ferguson et al., 2011) increase (hM3Dq:mCherry) or decrease (HA:hM4Di-IRES-mCitrine) the excitability of neurons upon activation by their ligand clozapine N-oxide (CNO). Temporal control of DREADD expression is provided by doxycycline, administered through the diet, which interferes with the binding of tTA to tetO and thus suppresses transgene expression (Gossen et al., 1995) . Mice maintained in their home cage on a doxycycline-containing diet showed low basal DREADD expression, detected by anti-HA immunohistochemistry and native mCherry fluorescence (Figure 1B and C) . To induce DREADD expression, mice were taken off doxycycline five days before odor presentation. Exposure to odor followed by foot shock (see Experimental Procedures) resulted in the Fos-tagging of sparse neural ensembles (median (interquartile range): 1.6 (0.3) % of piriform neurons, n=2 for hM4Di; 2.2 (0.5) % of piriform neurons, n=3 for hM3Dq), which were broadly dispersed throughout the anterior and posterior piriform cortex ( Figure 1B and C), consistent with previous observations of endogenous cFos expression (Datiche et al., 2001; Illig and Haberly, 2003) .\\r\\n To test whether CNO-mediated activation of DREADDs in Fos-tagged piriform neurons alters their excitability, we performed whole-cell recordings in acute brain author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint slices containing piriform cortex. DREADD-expressing neurons were identified based on mCitrine (HA:hM4Di-IRES-mCitrine) or mCherry (hM3Dq:mCherry) fluorescence. Figure 1) . In the absence of the DREADD ligand CNO, the resting membrane potential of DREADD-expressing piriform neurons was indistinguishable from DREADD-negative cells (-71.70 (4.37) mV in n=24, (4.32) mV in hM4Di-expressing cells, n=14, -72.56 (3.92) mV in hM3Dq-expressing cells, n=15, p=0.631, Kruskal-Wallis ANOVA, Figure 1D ). After bath application of CNO (5 µM), the membrane properties selectively changed in DREADD-expressing cells: hM4Di-expressing cells hyperpolarized (-74.51 (5.17 ) mV, n=12, p=0.007, Wilcoxon Signed Ranks Test), and hM3Dq-expressing cells depolarized (-66.95 (4.12) mV, n=13, p=0.001, Wilcoxon Signed Ranks Test), whereas the resting membrane potential of DREADD-negative cells remained unchanged (-70.28 (3.81) mV, n=15, p=0.890, Wilcoxon Signed Ranks Test, Figure 1D ). The relative changes were 1.00 (0.03) in DREADD-negative cells (n=15, p=0.890, One Sample Wilcoxon Signed Rank Test), 1.04 (0.03) in hM4Di-expressing cells, (n=12, p<0.001, One Sample Wilcoxon Signed Rank Test) and 0.91 (0.09) in hM3Dq-expressing cells, (n=13, p=0.001, One Sample Wilcoxon Signed Rank Test), Figure 1E ). To determine the impact of CNO treatment on neuronal excitability, we delivered depolarizing current steps and compared the number of action potentials triggered by 60 pA step current above action potential threshold (see Experimental Procedures). We found that in hM4Di-expressing cells, the number of evoked action potentials significantly decreased after CNO application (3.75 (1.59) vs. 0.30 (3.48), n=10, p=0.010, Wilcoxon Signed Ranks Test), whereas in hM3Dq-expressing cells, the number of author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint action potentials significantly increased after CNO application (8.67 (9.10) vs. 19.0 (27.6) mV, n=11, p=0.006, Wilcoxon Signed Ranks Test). In contrast, DREADDnegative cells did not exhibit a significant change in the number of evoked action potentials upon CNO application (4.33 (1.67) vs. 4.67 (3.00), n=9, p=0.672, Wilcoxon Signed Ranks Test, Figure 1F ). The relative changes were 1.00 (0.41) in DREADDnegative cells, (n=9, p=0.641, One Sample Wilcoxon Signed Rank Tests), 0.09 (0.81) in hM4Di-expressing cells (n=10, p=0.010, One Sample Wilcoxon Signed Rank Tests) and 1.81±1.09 in hM3Dq-expressing cells (n=11, p=0.004, One Sample Wilcoxon Signed Rank Tests, Figure 1G ). Taken together, these experiments show that Fos-tagging during olfactory fear conditioning marks a sparse and dispersed subpopulation of piriform neurons, and that CNO-mediated activation of hM4Di or hM3Dq expressed in Fos-tagged neurons selectively decreases or increases their excitability.\\r\\n If Fos-tagged piriform neurons that were activated during olfactory learning constitute an essential component of an olfactory memory trace, then perturbing the activity of these ensembles should interfere with memory recall. To test this prediction, we Fostagged piriform neurons during olfactory fear conditioning, and then chemogenetically silenced Fos-tagged ensembles during odor fear memory recall. cFos-tTA transgenic mice (\\\"experimental group\\\", n=8) were bilaterally injected in piriform cortex with the AAV-tetO-hM4Di vector. Wild-type mice injected with the same virus served as a control (\\\"control group\\\", n=8). During training, the conditioned odor (CS+) was presented at one of the ends of the conditioning box and paired with a foot shock, applied to the side of the box where the odor was delivered (see Experimental author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint Procedures and Figure 2A ). Mice thus learned to escape from the conditioned stimulus (CS+) by running towards the opposite side of the box. Two control odors, whose presentation was not paired with foot shock (CS-), were used to entrain odorspecific as opposed to generalized fear responses (Chen et al., 2011) . Doxycycline was removed from the diet of mice prior to training, to permit the induction of hM4Di expression in active Fos-expressing piriform neurons of mice of the experimental group. Mice were returned to doxycycline-containing diet immediately after odor fear conditioning, and memory recall of the odor-foot shock association was tested three days later by presenting the odors alone in a new box ( Figure 2B and C). All mice were intraperitoneally (i.p.) injected with CNO before testing, to exclude potential offtarget effects by CNO (Gomez et al., 2017) . For all mice, we verified viral gene expression by post hoc histological examination. Previous studies have suggested that the posterior piriform cortex is important for associative memory encoding (Sacco et al., 2010; Calu et al., 2008; Haberly 2001) . We therefore excluded from our analysis mice in which parts of the posterior piriform cortex were spared from viral infection (Supplementary Figure 2) . We combined two parameters to quantify the learned escape behavior: the position of the mouse in the box, and the increase in its maximum velocity after odor presentation ( Figure 2B ). These parameters were chosen because they described well the behavioral response of mice to the odor-foot shock pairing during training (Supplementary Figure 3) .\\r\\n As expected, mice in the control group exhibited robust escape behavior upon CS+ presentation. The median position of the mice in the box at the end of stimulus delivery was 0.74 (with 0 defined as the site of odor presentation and 1 as the opposite end of the box), and mice exhibited a 1.9-fold increase of their maximum velocity ( Figure 2E, G and H) . In contrast, escape behavior was significantly reduced author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint in mice in which Fos-tagged neurons were silenced. Their median position in the box at the end of stimulus delivery, (0.42), and their 1.1-fold increase in maximum velocity were significantly lower than in the control group (Mann-Whitney test, U=7, p=0.007 for both position and velocity, Figure 2E , G and H). The difference in the position of mice after CS+ and CS-presentation was also significantly different between the control and experimental groups (Mann-Whitney test, U=13, p=0.0499, Figure 2F ). Differences in behavioral responses of cFos-tTA transgenic mice were specifically due to hM4Di activation, as escape behavior after CNO injection was unaffected in cFos-tTA transgenic mice injected with AAVs expressing the calcium indicator GCaMP (n=9, Supplementary Figure 4 ).\\r\\n All mice were similarly close to the odor port at the onset of CS+ odor presentation (Mann-Whitney test, U=28, p=0.7209, Figure 2D ), and had similar maximum velocities during the 7 s time period that preceded odor presentation (Mann-Whitney test, U=24, p=0.4418, Figure 2G ). These observations exclude any inherent differences between groups due to how the CS+ was presented. Furthermore, escape behavior in control mice was specific to the CS+, as none of the mice ran away from the CS-( Figure 2D , E and H). Finally, we did not observe any differences in the overall mobility of mice throughout the entire testing session. The median speed of mice of all groups (0.68 (0.28) cm/s and 0.46 (0.14) cm/s for the control and experimental groups respectively) was not significantly different (Mann-Whitney test, U=22, p=0.3282), nor did mice show a bias for the left or right side of the testing box (time spent on the left side divided by time spent on the right side: 1.36 (0.72) and\\r\\n 1.51 (0.52) for the control and experimental groups respectively, Mann-Whitney test, U=27, p=0.6454). Together, these data suggest that Fos-tagged piriform ensembles that were activated during olfactory fear conditioning are necessary for odor fear author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint memory recall.\\r\\n Even though cFos-tTA transgenic mice expressing hM4Di failed to robustly escape from the CS+, it should be noted that they behaved differently after CS+ and CSpresentation. They were further away from the odor port after CS+ presentation than after CS-presentation (Wilcoxon matched-pairs signed rank test, W=-36, p=0.0078, Figure 2E ), and their maximum velocity ratio was significantly higher (Wilcoxon matched-pairs signed rank test, W=-36, p=0.0078, Figure 2H ). This could indicate a partial memory of the learned association, possibly due to incomplete silencing of Fos-tagged neurons in piriform cortex, or the existence of parallel neural pathways that partially compensate for the loss of piriform functions. Finally, silencing of the piriform cortex in only one hemisphere did not abolish the learned escape behavior to the conditioned odor (Supplementary Figure 5) .\\r\\n It is possible that the chemogenetic silencing of piriform ensembles that were Fostagged during olfactory fear conditioning disrupts odor detection and discrimination rather than selectively affecting odor fear memory recall. To test this possibility, we monitored sniffing behavior of mice during an olfactory habituation/dishabituation assay ( Figure 3A) , a well established test for odor detection and discrimination (Coronas-Samano et al., 2016; Verhagen et al., 2007; Wesson et al., 2008) . Fostagging of piriform neurons during olfactory fear conditioning was performed as described above. Three days later, changes in sniffing behavior in response to odor exposure were tested in a plethysmograph, while hM4Di-expressing piriform neurons author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint in cFos-tTA transgenic mice were silenced using i.p. injection of CNO (see Experimental Procedures). We observed that mice of both experimental (n=8) and control (n=6) groups increased their sniff frequency upon presentation of a novel, neutral odor (1.6 fold increase; Wilcoxon matched-pairs signed rank test, W=21, p=0.0312 and W=36, p=0.0078 for the control and experimental groups, respectively; Figure 3B and C). Repeated exposure to the same odor (neutral odor or CS-) resulted in a decrease in the sniff frequency, reflecting habituation after 4 consecutive exposures (Figure 3C and D) . Finally, presentation of the CS+ increased sniff frequency (1.4 fold increase; Wilcoxon matched-pairs signed rank test, W=21, p=0.0312 and W=34, p=0.0156 for the control and experimental groups, respectively, Figure 3D ). Such changes in odor sampling behavior have been shown to report the detection and discrimination of different odor stimuli (Coronas-Samano et al., 2016; Verhagen et al., 2007; Wesson et al., 2008) . Therefore, these data suggest that basic behaviors characteristic of odor sampling, detection and discrimination were unaffected by the silencing of piriform ensembles that were Fostagged during olfactory fear conditioning.\\r\\n Odors activate unique but overlapping ensembles of piriform neurons (Bolding and Franks, 2017; Poo and Isaacson, 2009; Roland et al., 2017; Stettler and Axel, 2009 ). Therefore, silencing neurons that respond to one odor could partially interfere with the retrieval of information associated with other odors. To test this prediction we modified the behavioral protocol to separate Fos-tagging from olfactory fear conditioning. cFos-tTA transgenic and wild-type control mice were injected with AAV-tetO-hM4Di, and expression of hM4Di in piriform neurons was induced while mice author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint were exposed to odor (eugenol or beta-citronellol) in a neutral environment, without subsequent foot shock. Fos-tagging resulted in the labeling of sparse neural ensembles, similar to those labeled during odor-foot shock association (1.1 (0.3) % of piriform neurons, n=3). Mice were returned to doxycycline-containing diet and trained two days later to associate a different odor (ethyl-acetate, CS+) with foot shock.\\r\\n Behavioral testing of the learned escape behavior in response to the CS+ was performed one day later, after i.p. injection of CNO (see Experimental Procedures and Figure 4A ). We found that learned escape behavior of cFos-tTA transgenic mice expressing hM4Di (experimental group, n=12) was similar but somewhat attenuated compared to wild-type controls (n=10). Both groups exhibited an escape behavior after CS+ presentation, indicated by the significant increase in the maximum velocity (Wilcoxon matched-pairs signed rank test, W=55, p=0.0020 and W=56, p=0.0269 for the control and experimental groups respectively; no difference in the maximum velocity ratio after/before between groups, Mann-Whitney test, U=35, p=0.1071, Figure 4E and F). The difference in the position of mice after CS+ and CSpresentation was also similar between the control and experimental groups (Mann-Whitney test, U=37, p=0.1402, Figure 4D ). However, we observed that silencing neutral odor representations somewhat dampened the behavioral response to the conditioned stimulus, as behavioral responses were less robust compared to controls when quantifying the position of mice in the box after CS+ presentation (Mann-Whitney test, U=22, p=0.0112, Figure 4B and C). The attenuated behavioral responses we observe likely reflect the partial degradation of odor information as a result of the silencing of piriform neurons responsive to both the CS+ and the neutral odor.\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint\\r\\n If piriform neural ensembles that were activated during olfactory fear conditioning encode an odor fear memory trace, then reactivation of these neurons may be sufficient to trigger memory recall. To test this prediction, we devised a modified behavioral task to assess memory recall. We monitored exploratory behavior in an open field during ambient odor exposure or artificial reactivation of Fos-tagged neurons, a test that does not rely on an escape behavior from an odor originating from a spatially defined odor source ( Figure 5A ). We first trained mice to associate ethyl-acetate, the CS+, with foot shock. Upon ambient exposure to the CS+, diffused above the open field, exploratory behavior of mice was significantly attenuated compared to baseline exploratory behavior measured one day earlier (Wilcoxon matched-pairs signed rank test, n=6, W=-21, p=0.0312). Ethyl-acetate exposure did not affect exploratory behavior of mice that had previously been exposed to ethylacetate without foot shock (Wilcoxon matched-pairs signed rank test, n=5, W=1, p>0.9999). These data suggest that re-exposure of mice to the CS+ causes an increase in anxiety-like behavior, expressed as a decrease in exploratory behavior in the open field.\\r\\n We next repeated the experiment in cFos-tTA transgenic mice, in which the piriform cortex had been infected with the activating DREADD AAV-tetO-hM3Dq (Supplementary Figure 6) . Instead of exposure to the CS+, we i.p. injected the DREADD ligand CNO to reactivate hM3Dq-expressing piriform neurons that were Fos-tagged during odor-foot shock association (see Experimental Procedures and Figure 5A ). We found that CNO-mediated reactivation, similar to CS+ exposure, caused a significant decrease in exploratory behavior of mice that had undergone author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint olfactory fear conditioning (Wilcoxon matched-pairs signed rank test, n=8, W=-34, p=0.0156, Figure 5B and C). CNO-mediated reactivation of piriform neurons that were Fos-tagged during ethyl-acetate exposure without foot shock did not affect exploratory behavior of mice (Wilcoxon matched-pairs signed rank test, n=7, W=-10, p=0.4688, Figure 5B and C). Taken together, these data suggest that chemogenetic reactivation of piriform neurons that were active during odor-foot shock exposure is sufficient to trigger fear memory recall.\\r\\n We next asked whether artificial memory recall depends on the specificity of the Fostagged neural ensemble: is piriform reactivation sufficient to trigger fear memory recall, as long as it includes the CS+-tagged ensemble? For this purpose, we generated a synthetic Fos-tagged ensemble, by sequentially exposing mice to ethylacetate paired with foot-shock (CS+) and neutral odors (CS-, eugenol and betacitronellol) ( Figure 5D ). While exposure to the CS+ resulted in anxiety-like behavior similar to what was observed in the previous experiment (Wilcoxon matched-pairs signed rank test, n=6, W=-21, p=0.0312, Figure 5E and F), artificial reactivation of synthetic Fos-tagged ensemble did not result in changes in exploratory behavior that would indicate memory recall (Wilcoxon matched-pairs signed rank test, n=8, W=-4, p=0.8438, Figure 5E and F). These data suggest that during artificial reactivation, piriform cortex cannot extract meaningful information from the synthetic representation generated by the sequential presentation of CS+ and CS-odors. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint\\r\\n The piriform cortex has long been thought to provide the substrate for storing associative olfactory memories. Several studies have shown that piriform cortex cells can encode information that carries behavioral significance (Calu et al., 2008 , Mandairon et al., 2014 , Sacco et al., 2010 ), yet their functional relevance has never been assessed by direct experimental manipulation. Using a cFos-dependent, intersectional genetic approach to visualize and manipulate piriform neurons activated during olfactory fear conditioning (Figure 1 ), we found that chemogenetic silencing of Fos-tagged piriform ensembles abolished a learned odor escape behavior (Figure 2 ). Silencing these Fos-tagged ensembles did not alter basic odor detection and discrimination (Figure 3) . Furthermore, chemogenetic silencing of neurons responsive to a neutral odor only moderately attenuated odor fear memory recall, suggesting that Fos-tagged piriform ensembles are largely odor-specific ( Figure 4) . Finally, specific chemogenetic reactivation of piriform ensembles that were active during learning resulted in reduced exploratory behavior in an open field assay, an effect that was similarly observed when exposing mice to the conditioned odor stimulus (Figure 5) . Together, our experiments identify piriform neurons that were active during learning as necessary and sufficient to trigger odor fear memory recall. This indicates the formation of functional connections between Fos-tagged piriform cells and other associative areas and firmly establish piriform ensembles as an essential component of odor fear memory traces.\\r\\n Artificial reactivation of an olfactory memory trace author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint Reactivation of neurons by CNO-mediated activation of DREADD receptors does not recapitulate the temporal characteristics of piriform odor responses and their modulation by active sampling (Bolding and Franks, 2017; Miura et al., 2012) .\\r\\n Despite this, reactivation of Fos-tagged piriform ensembles activated during odor-foot shock exposure was sufficient to elicit a behavioral response. How piriform neural circuits and downstream target regions process such artificial activity patterns remains to be determined. One possibility is that despite the temporal limitations of memory trace reactivation, piriform network mechanisms can retrieve the perception of the reinforced conditioned stimulus. Consistent with this model, recent studies have shown that spatial patterns of odor-evoked activity, in the absence of precise temporal information, are sufficient to decode odor identity in piriform cortex (Bolding and Franks, 2017; Miura et al., 2012; Roland et al., 2017) . Alternatively, it is possible that reactivation generates a state of stress and anxiety, but without evoking the perception of the odor. Candidate target areas for the processing of odor-fear associations include the basolateral amygdala and the medial prefrontal cortex (Dejean et al., 2016; Gore et al., 2015) , however, the relevant neural circuit components remain to be identified. Finally, reactivating a synthetic ensemble of neurons activated during exposure to the reinforced stimulus (CS+) and the nonreinforced stimuli (CS-) did not elicit a measurable behavioral response. This observation suggests that behaviorally relevant information cannot be extracted from a neural ensemble representing conflicting (aversive versus neutral) information. This result is consistent with the finding that the reactivation of an artificial contextual memory in the hippocampus competes with the retrieval of a learned context-shock association (Ramirez et al., 2013) . author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint\\r\\n A general limitation of the cFos-tTA system is that the tagging of active neurons is transient. The duration of the expression of tTA-dependent proteins is determined by the time course of induction and the stability of cFos-tTA-dependent transcripts and proteins in tagged neurons. Previous studies have shown that TetTag-dependent expression of regulators of neural activity lasts for at least five days but becomes undetectable by thirty days (Cowansage et al., 2014; Liu et al., 2012; Zhang et al., 2015) . We therefore restricted our manipulations of neural activity and behavioral analyses to a three-day time window after Fos-tagging of neurons during olfactory conditioning. Another constraint of the Fos-tagging system is its slow temporal dynamics. As a consequence, neurons responding to the reinforced conditioned stimulus (CS+), the odorant paired with foot shock, and neurons responding to the non-reinforced conditioned stimuli (CS-), the odorants not associated with foot shock, were tagged in our silencing experiments. The use of a CS-compromises the specificity of the Fos-tagged neural ensembles. Nevertheless, the CS-provides an important experimental readout for the odor selectivity of the behavioral response.\\r\\n Interestingly, the number of Fos-tagged piriform neurons we observe is significantly lower than the number of odor-responsive neurons detected in electrophysiological recordings in awake mice (Bolding and Franks, 2017; Miura et al., 2012; Zhan and Luo, 2010) . However, in these experiments, a large fraction of cells exhibited low firing rates. Therefore, the Fos-tagged population could represent a subpopulation of neurons that is strongly activated by odor (Schoenenberger et al., 2009) . Fos-tagging could also mark plastic changes supporting memory formation (Cole et al., 1989; Minatohara et al., 2015) . Indeed, cFos mRNA levels decrease significantly after injection of an antagonist of the NMDA receptor, a key player in the induction of author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint synaptic plasticity (Tayler et al., 2011) , and long-term memory and synaptic plasticity are impaired when cFos production is perturbed in the central nervous system (Fleischmann et al., 2003; Seoane et al., 2012) . Future studies will allow to directly test these hypotheses.\\r\\n The hippocampus has been studied extensively for its role in spatial and contextual memory (Basu and Siegelbaum, 2015) . Recently, Fos-tagging of hippocampal neurons during contextual fear conditioning has provided important insight into the cellular and neural circuit mechanisms of learning and memory (Cai et al., 2016; Garner et al., 2012; Liu et al., 2012; Ramirez et al., 2013; Roy et al., 2017; Ryan et al., 2015) . However, if principles of memory formation and storage in hippocampusrelated neural networks apply to other cortical structures remains largely unknown.\\r\\n Interestingly, pirform cortex and hippocampus share a similar circuit organization and both have been modeled as auto-associative networks (Haberly, 2001) . Consistent with this model, our findings reveal striking similarities between olfactory memory traces in the piriform cortex and memory traces of contextual fear in the hippocampus. In both piriform cortex and hippocampus, learning activates sparse and distributed ensembles of neurons that appear to lack topographic organization. Furthermore, hippocampal neurons tagged during contextual fear conditioning, and piriform neurons tagged during olfactory fear conditioning were both necessary and sufficient for memory retrieval (Liu et al., 2012; Tanaka et al., 2014) , The olfactory system is particularly well-suited for studying learning and memory. Indeed, odor memories and olfactory-driven behaviors are robust in animals, including in mice, providing highly reliable and quantitative behavioral readouts. Furthermore, the author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint olfactory cortex is only two synapses away from the peripheral olfactory sensory neurons in the olfactory epithelium, and the neural inputs that drive piriform cortex activity have been well characterized (Banerjee et al., 2015; Dhawale et al., 2010; Economo et al., 2016; Fukunaga et al., 2012; Kato et al., 2012; Roland et al., 2016; Yamada et al., 2017) . Knowing the properties of neural input patterns will provide the basis for understanding neural circuit input-output computations and their modification by learning and experience.\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint\\r\\n Mice were housed at 24°C with a 12-hour light/12-hour dark cycle with standard food and water provided ad libitum. Mice were group-housed with littermates until the beginning of surgery and then single-housed in ventilated cages throughout the duration of the experiment. cFos-tTA mice (Reijmers et al., 2007) were fed a diet containing 1g/kg doxycycline for a minimum of 4 days before surgery. Wild-type control animals were siblings that did not carry the cFos-tTA transgene. The age of mice at the time of behavioral testing ranged from 10 to 15 weeks. Male mice were used for the hM4Di-mediated neural silencing, and females for the hM3Dq-mediated neural activation experiments. Experiments were carried out according to European and French national institutional animal care guidelines (protocol APAFIS#2016012909576100).\\r\\n The Adeno-associated viruses (AAVs) were generated at Penn Vector Core, University of Pennsylvania (serotype 8, 10 13 genome copies/mL, 1:2 dilution with sterile PBS on the day of injection).\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint\\r\\n Mice were anaesthetized with ketamine/xylazine (100 mg/kg/10 mg/kg, Sigma-Aldrich) and AAV vectors were injected stereotaxically into the piriform cortex Sections were washed 2 times in PBST and 1 time in PBS, mounted on slides and author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint coverslipped with Vectashield mounting medium (Vectorlabs). Images were acquired as Z-stacks (70 to 140 µm in total thickness, step size 7 µm) with a Leica SP5 confocal microscope or as single plane sections with a Zeiss Axio Zoom microscope and processed in Fiji.\\r\\n Counting Z-stacks (7 µm step) were acquired with a Leica SP5 confocal microscope with a 20X objective and a resolution of 512x512 or 1024x1024 pixels (1 pixel: 0:72µm). First, the piriform cortex was delineated on a maximum projection of each stack using a custom-written ImageJ macro. For each mouse, 2 to 4 sections (median volume 9.3e-2 (9.4e-3) mm 3 ) were analyzed, from one or both hemispheres, at Y=-0.8 and Y=-1.6mm relative to bregma.\\r\\n HA-stained neurons (hM4Di) were counted by hand. hM3Dq-mCherry positive neurons were counted using a custom-written ImageJ plugin. After pre-processing the image (Subtract Background, Remove Outliers and Median Filter), the z-stack is thresholded using the RenyiEntropy algorithm. On each slice of the stack, objects with an area smaller than 40µm 2 are removed using the Analyze Particle command in ImageJ. A mask of each slice from the stack is created containing the filled outline of the measured particles. The number of cells is the number of 3D objects detected in the stack. Automated cell detection can be prone to error, thus, the detection efficiency was checked on a few pictures by two people blind to the experimental conditions.\\r\\n The numbers of cells were calculated as numbers of cells per cubic millimeter. An estimate of the total number of neurons (counterstained with NeuroTrace) per cubic millimeter (7.1e4 mm -3 ) was obtained by counting neurons in representative volumes author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint containing the three layers of piriform cortex. Numbers are consistent with those obtained by (Srinivasan and Stevens, 2017) .\\r\\n Parasagittal or coronal slices (300 µm thick) of piriform cortex were prepared from 6-8 week-old cFos-tTA mice injected with AAV-tetO-hM4Di-IRES-mCitrine or AAV-tetO-hM3Dq-mCherry. Animals were anaesthetized with ketamine and xylazine (100 mg/kg/10 mg/kg, Sigma-Aldrich), perfused with ice-cold ACSF (125 mM NaCl, 2.5 mM KCl, 25 mM glucose 25mM NaHCO 3 , 1.25 mM NaH 2 PO 4 , 2 mM CaCl 2 , 1 mM MgCl 2 , 1 mM pyruvic acid, bubbled with 95% O 2 and 5% CO 2 and adjusted to 295±5 mOsm osmolarity), and decapitated. The brain was cooled with ice-cold ACSF solution and then sliced using a 7000SM2 vibrating microtome (Campden Instruments, UK). Slices were incubated in the same solution at 34°C for one hour and then continuously perfused with ACSF solution (2 mL/min) at 34°C in the recording chamber. DREADD expression was detected with two-photon excitation (830nm, ChameleonMRU-X1, Coherent, UK) under a Scientifica TriM Scope II microscope (LaVision, Germany), with a 60x water-immersion objective. Whole-cell recording pipettes with 5-7 MΩ resistance were filled with the following solution (in mM): 122 Kgluconate, 13 KCl, 10 HEPES, 10 phosphocreatine, 4 Mg-ATP, 0.3 Na-GTP, 0.3 EGTA (adjusted to pH 7.35 with KOH). The recording solution also contained the morphological tracer Alexa Fluor 594 (5 µM, red channel) for nonexpressing and hM4Di-positive cells or the Ca 2+ -sensitive dye Fura-2 (300 µM, replacing EGTA in the recording solution, green channel) for hM3Dq-positive cells, to enable identification of the patched cell and to visualize its morphology. The excitability of cells was measured in current-clamp mode by 500 ms steps of current author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint injections from -300 to +500 pA with steps of 20 pA. We compared the number of action potentials triggered by equivalent depolarization at each step 60 pA above the action potential threshold before and after CNO (5 µM) application. The series resistance was usually <20 MΩ, and data were discarded if it changed by more than 20% during the recording. Signals were amplified using EPC10-2 amplifiers (HEKA Elektronik, Lambrecht, Germany). Voltage-clamp recordings were filtered at 5 kHz and sampled at 10 kHz, and current-clamp recordings were filtered at 10 kHz and sampled at 20 kHz, with the Patchmaster v2x32 program (HEKA Elektronik).\\r\\n A training box was used to train mice to escape from an odor. The box was rectangular (L 57 cm, W 17 cm, H 64 cm), with a grid floor made of 72 stainless-steel rods (diameter=6 mm, space between rods=2 mm). Current was delivered by an aversive stimulator (MedAssociates, 115V, 60 Hz). A custom-made switcher allowed an electric foot shock (0.6 mA, 0.6 ms) to be applied independently to either half of the box. A testing box was used to test memory retrieval. Its dimensions were similar to the training box. Materials for the walls and floor were different from the training box to create a different context. The open field box consisted of a white Plexiglas (6 mm thickness) container (50 cm x 50 cm x 38 height).\\r\\n Sniffing behavior was monitored in freely moving mice using a plethysmograph (Emka Technologies). Electrical signals were amplified using RHD 2132 amplifier boards (Intan Technologies) and band-pass filtered (1-30 Hz).\\r\\n Odors were delivered using an 8 channels olfactometer (Automate Scientific).\\r\\n Continuous air flowed through a bottle of mineral oil (Sigma-Aldrich) to avoid pressure changes when presenting the odors. The following odors (Sigma-Aldrich), author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint at a concentration of 1% (vol/vol) in mineral oil were used: ethyl-acetate, betacitronellol, eugenol, limonene, and pinene.\\r\\n AAVs were stereotaxically injected in both hemispheres of piriform cortex of cFos-tTA transgenic mice and littermate wild-type controls. Mice were given 10 days to recover, before being habituated on day 11 to the training and testing/open field/plethysmograph boxes (40 and 10 min each). 24 hours after habituation, doxycycline was removed and replaced by a normal diet. Five days later, mice were fear conditioned to ethyl-acetate or exposed to an odor (odor \\\"C\\\") in a neutral environment. Immediately after fear conditioning or odor presentation, mice were put back on a diet containing doxycycline. During fear conditioning, ethyl-acetate (the CS+) was presented at one extremity of the training box, and the corresponding halfside of the box' floor was electrified for 0.6 s (0.6 mA). The foot shock was delivered 4 seconds after CS+ presentation, and the CS+ lasted for an additional 2 seconds.\\r\\n Mice learned to escape to the opposite side when the CS+ was presented. The CS+ was presented 4 times, 2 times on each side of the box. When fear conditioning included presentation of two non-reinforced conditioned stimuli (CS-1 and CS-2), the choice between CS+ and CS-as well as the side of presentation was pseudorandomized. Training always started with one presentation of each CS-followed by the CS+. The two CS-odors were presented during 7 seconds, 3 times each, with a total presentation of 3 on the right side and 3 on the left side. During testing, the CS+ was presented 4 times (2 times on each side), and each CS-was presented once on each side, for 7 s each. As for the training session, the order and side of odor presentations was pseudo-randomized: testing always started with one presentation author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint of each CS-, followed by one CS+ presentation.\\r\\n Clozapine-N-oxide (C0832, Sigma-Aldrich) was first dissolved in dimethylsulfoxide (DMSO, D2438, Sigma, final concentration of 1% vol/vol), then further diluted in 0.9% sterile saline solution to a final concentration of 0.2 mg/mL. Aliquots were stored for up to two months at -20°C and equilibrated to room temperature before injection. The solution was administered intraperitoneally (3 mg/kg). After injection, mice were left undisturbed for 25 minutes (Figure 2 and 4) or 5 minutes (Figure 3 and 5) in their home cage before the start of the experiment.\\r\\n Electrophysiological data were analyzed with custom-made software in Python 3.0 and averaged in MS Excel (Microsoft, USA). Behavioral sessions were video recorded and analyzed using custom-written MatLab programs (available upon request). Parameters extracted from multiple CS+ (CS-) presentation were averaged for each mouse.\\r\\n Statistical analysis was performed with Prism (GraphPad). Non-parametric tests were used: Mann-Whitney test for between group comparison, and Wilcoxon matchedpairs signed rank test for within group comparison. Values are represented as median (interquartile range) unless otherwise stated. The n represents number of mice, or number of recorded neurons for the electrophysiology experiment. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint before and after CNO application. After CNO application, the number of APs in DREADD-negative cells (Ctrl) does not change, significantly decreases in hM4Diexpressing cells and significantly increases in hM3Dq-expressing cells.\\r\\n (G) The relative change in the number of APs after/before CNO in DREADD-negative cells (Ctrl) does not change, significantly decreases in hM4Di-expressing cells and significantly increases in hM3Dq-expressing cells.\\r\\n Data are shown as individual data points, median and interquartile range, ** p<0.01. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint A) During fear conditioning, mice are trained to associate ethyl-acetate (CS+) with foot shock. The CS+ is presented at one end of the training box, and the corresponding half-side of the box is electrified. Training includes the presentation of two other odors that are not paired with foot shock (CS-, eugenol and beta-citronellol).\\r\\n The CS+ is presented 4 times (2 times on each side) and each CS-three times. The order and side of odor presentations is pseudo-randomized. B) During testing, the CS+ is presented 4 times (2 times on each side), and each CSis presented once on each side, for 7 s each. The testing session is video-recorded and the position of the mouse's centroid in the box along the x-axis is extracted (top).\\r\\n The length of the box is normalized to 1. Since the odor is presented randomly on the left or right side of the testing box, 0 is defined as the extremity where the odor is presented. For velocity measurements, a 7 s time window is defined both before and after stimulus onset (bottom). C) Experimental design: cFos-tTA transgenic mice (experimental group, n=8) or wildtype mice (control group, n=8) are injected with AAV-tetO-hM4Di in both hemispheres of piriform cortex. Ten days later, mice are habituated to the training and testing boxes. Upon doxycycline (DOX) removal, mice are fear-conditioned, and hM4Di expression (green circles) is induced in Fos-tagged neurons (filled in blue and yellow). Mice are returned to DOX-containing diet to avoid further expression of hM4Di. Three days later, all mice are i.p. injected with CNO, which silences hM4Diexpressing neurons (black crosses), and memory retrieval is tested.\\r\\n (D to H) Olfactory memory retrieval after CNO injection, assessed by the position of mice in the testing box and their maximum velocity upon odor exposure. Memory author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint retrieval was significantly impaired in cFos-tTA transgenic mice in which piriform neurons that were active during learning were chemogenetically silenced during memory recall (experimental group).\\r\\n (D) Position of mice in the testing box as a function of time after CS+ (black) and CS-(blue) presentation. (E) Position in the testing box 7 s after CS+ or CS-presentation.\\r\\n (F) Position 7 s after CS-presentation subtracted from the position 7 s after CS+ presentation.\\r\\n (G) Maximum velocity during a 7 s time window before and after CS+ presentation.\\r\\n (H) Maximum velocity ratio after/before CS+ (left) or CS-(right) presentation.\\r\\n Data are shown as median and interquartile range, * p<0.05; ** p<0.01; n.s. not significant.\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint (A) (Top) Experimental design: Fos-tagging during olfactory fear conditioning is performed as described in Figure 2 . Three days after fear conditioning, sniffing behavior is quantified in a plethysmograph. Mice are i.p. injected with CNO, habituated to the plethysmograph for 15 min, and each odor is then presented for 8 s with a 2.5 min inter-trial interval. pin: pinene, CS-: beta-citronellol, CS+: ethyl-acetate.\\r\\n (Bottom) Representative sniff recordings from one mouse when exposed to pinene or the CS+ (ethyl-acetate). Data are shown as median and interquartile range, * p<0.05; ** p<0.01; n.s.: not significant author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint Data are shown as median and interquartile range, * p<0.05; ** p<0.01; n.s. not significant.\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint (B) As a measure for exploratory activity, the distance travelled is computed from minute 15 to 18 (corresponding to odor exposure during test 2). For artificial memory recall, the distance travelled is computed over the entire testing session. Ethylacetate exposure and CNO injection lead to a decrease in exploratory behavior in mice previously fear conditioned to ethyl-acetate (n=6 and n=8 respectively).\\r\\n Previous exposure to ethyl-acetate without foot shock does not alter exploratory behavior upon re-exposure to ethyl-acetate or re-activation of Fos-tagged piriform ensembles (n=5 and n=7 respectively).\\r\\n (C) Distance traveled ratio test 2/test 1. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint \\r\\n Training Testing The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint baseline author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint. SARS-CoV-2, also known as 2019-nCoV, is a novel coronavirus (CoVs) isolated from patients with pneumonia in China 2019. SARS-CoV-2 has a similar incubation period (median, 3.0 days) and a relatively lower fatality rate than SARS-CoV or MERS-CoV (1), but it is estimated that the reproductive number of SARS-CoV-2 is higher than that of SARS-CoV (2) .\\r\\n What's more, some laboratory-confirmed symptomatic cases are absent of apparent cough, fever or radiologic manifestations, making it difficult to timely and accurately find out all infected patients (3, 4) . As of February 28, 2020, patients infected by SARS-CoV-2 have been diagnosed in more than 30 countries, and more than 78,000 confirmed cases and 2,700 deaths associate with SARS-CoV-2 infection are reported in China alone.\\r\\n The genetic information of a virus is essential for not only its classification and traceability, but also its pathogenicity. At the whole genome level, the sequence identify of SARS-CoV-2 was 50% to MERS-CoV, 79% to SARS-CoV, 88% to two bat-derived SARS-like coronaviruses, Bat-SL-CoVZC45 and Bat-SL-CoVZXC21 (collected in 2018 in Zhoushan, China) , and 96% to Bat-SARSr-CoV RaTG13 (collected in 2013 in Yunnan, China) (5) . Each genome of all SARS-CoV-2 strains now submitted online contains nearly 29,900 nucleotides (nt), which are predicted with at least 14 open reading frames (ORFs) (5′ to 3′), such as ORF1ab (P, 21,291 nt), spike (S, 3,822 nt), ORF3a (8,28 nt), envelope (E, 228 nt), membrane (M, 669nt), ORF8 (366 nt), and nucleocapsid (N,1,260 nt) (6) . Among them, the spike gene encoded a glycoprotein that is crucial for determining host tropism and transmission capacity, and highly divergent when compared with that of bat-SARSr-CoV RaTG13 (93.1% nucleotide identity) (6, 7) .\\r\\n Generally, the rates of nucleotide substitution of RNA viruses are faster than their hosts, and . CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint this rapid evolution is mainly shaped by natural selection (mostly purifying selection) (8) . Gene mutations such as nucleotide substitutions, deletions and insertions have been frequently reported when comparing SARS-CoV-2 with other viruses (5) (6) (7) . In this work, we investigated the mutation pattern of SARS-CoV-2 by comprehensive comparative genomic analysis of the nonsynonymous/ synonymous substitution, relative synonymous codon usage (RSCU) and selective pressure in order to explore their potential in evolution and function.\\r\\n The SARS-CoV-2 reference genome Wuhan-Hu-1 (NC_045512) and WIV04 (MN996528) was downloaded from the GenBank database. Other newly sequenced 2019-nCoV genomes were downloaded from the Global Initiative on Sharing Avian Influenza Data (GISAID) database (https://www.gisaid.org/). Twenty-one closely related coronavirus complete genome sequences and their coding sequences (CDS) were downloaded from GenBank database (Supplementary   Table S1 ).\\r\\n Genome sequences were aligned using MUSCLE v3.8.31 (9), followed by manual adjustment using BioEdit v7.2.5. Phylogenetic analyses of complete genome were performed using maximum-likelihood method and general time-reversible model of nucleotide substitution with gamma-distributed rates among sites (GTR+G) in RAxML v8.1.21 (10) . Support for the inferred relationships was evaluated by a bootstrap analysis with 1000 replicates and trees were rooted with the alpha-coronavirus lineage as an outgroup. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint\\r\\n The coding sequences were translated and aligned using the MEGA-X program (11) , and then codon-based sequence alignment was used for further analysis. Phylogenetic analyses of coding sequences were performed using MEGA-X software. The changes of amino acids or nucleotides for each CDS sequences were analyzed using in-house Perl script.\\r\\n The number of synonymous substitutions per synonymous site (dS), and the number of non-synonymous substitutions per non-synonymous site (dN), for each coding region were calculated using the Nei-Gojobori method (Jukes-Cantor) in PAML package (12) .\\r\\n The adaptive evolution server (http://www.datamonkey.org/) was used to identify eventual sites of positive selection. For this purpose, the following test has been used: mixed-effects model of evolution (MEME), which allows the distribution of dN/dS (ω) to vary from site to site and from branch to branch at a site (13) . This test allowed us to infer episodic and pervasive positive selection at individual sites.\\r\\n To investigate the potential relative synonymous codon usage (RSCU) bias of the spike protein from SARS-CoV-2 and its closely related coronaviruses, the coding sequence (CDS) of spike protein in these coronaviruses were calculated with CodonW 1.4.4 (http://codonw.sourceforge.net/). The RSCU of human genes was retrieved from the Codon Usage Database (http://www.kazusa.or.jp/codon/). The potential relationships among these sequences were calculated with a squared Euclidean distance ( u h u ). Besides RSCU, the effective number of codons (ENc) was used as a simple metrics to verify codon bias and to explore the source of virus.\\r\\n . CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint\\r\\n Statistical analyses were performed using the R statistical package (version 3.\\r\\n Chi-squared test was used to compare any two data sets, and data were considered significantly different if the two-tailed p value was less than 0.05.\\r\\n The newly identified SARS-CoV-2 strain WIV04 genome sequence was closely related with Bat-SARSr-CoV RaTG13 and Bat-SL-CoVZC45, which was collected from host Rhinolophus affinis (5) . Compared with RaTG13 genome, many nucleotide substitutions are observed, but there are only five small inserts and deletions (indels) mutations, and the largest insert segment in WIV04 genome was \\\" CGGCGGGCACGT \\\" sequence, which is located near the boundary of S1 and S2 regions of spike protein. Interestingly, only synonymous mutation are observed near this insertion sequence (Fig. 1 , panel C). Compared with Bat-SL-CoVZC45 genome, this insert segment is detected as well, but non-synonymous mutations are also observed around it. Then, we further compared the proportion of synonymous mutations in spike gene between WIV04 and RaTG13 or Bat-SL-CoVZC45. The ratio of nucleotide substitutions (263 NT) to amino acid substitutions (29 AA) was 9.07 from WIV04 to RaTG13, which was significantly higher than the ratio (3.91, 864/221) from WIV04 to Bat-SL-CoVZC45 (p < 0.05) (Fig. 2) . Moreover, we also observed that the synonymous mutations increased in whole genome level between WIV04 and RaTG13 than that between WIV04 and Bat-SL-CoVZC45.\\r\\n Next, the detailed nucleotide changes of each comparison group were analyzed, and we found that the T-to-C (T:C) transition mutation was enriched in the nucleotide substitutions between WIV04 and RaTG13 (Fig. 1B) . It's reported that the CoVs lacking a 3'-to-5' exoribonuclease (ExoN) accumulated 15 to 20 fold more A:G and U:C transition (14) . What's more, the RNA mutagen 5-fluorouracil (5-FU) treatment will also increase the U:C and A:G transitions. Consequently, this increased T:C mutation implies the ExoN of SARS-CoV-2 may be deactivated compared with that of RaTG13. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint Furthermore, we checked whether the ratio of synonymous substitution to missense substitution is increased within SARS-CoV-2 strains. From December 2019 to February 2020, the genome sequences of 108 strains of SARS-CoV-2 virus have been submitted to GISAID database worldwide. Compared with the standard SARS-CoV-2 strain WIV04, total 98 point mutations were detected at 93 nucleotide sites of all SARS-CoV-2 strains with genome sequence available on Feb. 25 2020 (Fig. 1A) . However, only 58 of these nucleotide mutations caused changes in amino acids. Among them, 15 nucleotide substitutions at 14 sites caused changes in 7 amino acids of the spike protein were observed. Consequently, the proportion of synonymous mutations (~40%) among all current reported SARS-CoV-2 strains is similar to that between WIV04 and Bat-SL-CoVZC45 (39.1%), but is dramatically less than that between WIV04 and RaTG13 (90.7%).\\r\\n According to random drift hypothesis (15) ， these nucleotide mutations among different SARS-CoV-2 strains now available are still determined by neutral evolution. In short, there has no powerful factor to force SARS-CoV-2 to evolve in a certain direction by far. However, we should take strict precautions against the strong factors that may cause directional variation of SARS-CoV-2 both in natural environment and infection treatment.\\r\\n To investigate whether the mutation pattern between SARS-CoV-2 and RaTG13 is unique across all coronavirus species, we further compared their alterations of nucleotides and amino acids in comparison to other representative coronaviruses. Phylogenetic analysis of SARS-CoV-2 and its 20 closely related coronaviruses formed four well supported clades (Fig. 2) . The two SARS-CoV-2 strain WIV04 (from Wuhan) and SNU01 (from Korea patient) were clustered with SARS-CoV-related strains to form clade 1, belonging to beta-coronavirus lineage B. The Compared with Bat-SARSr-CoV RaTG13, there were 293 nucleotideswere altered while only 29 amino acids were altered in SARS-CoV-2; their ratio was as high as 9.07. However, this ratio detected in the comparisons of other human coronaviruses with their similar animal coronaviruses was less than 5.0. For example, the ratio of human SARS-CoV Tor2 to bat SARS-like CoVs LYRa11 was 4.81; that ratio of human MERS-CoV EMC2012 to bat SARS-like CoVs RSA2011 was 2.80; that ratio of human coronavirus 229E to bat coronavirus HKU2 was 4.55. These results indicate that the relative level of synonymous substitutions between human SARS-CoV-2 and its possible animal origin (RaTG13) is much higher than that between other human coronaviruses and their potential animal sources.\\r\\n Different organisms, even different protein coding genes of the same species, have different frequency of codon usage (16) . The RSCU bias will reveal the difference of the host source. We calculated the distance of RSCU between spike genes of 20 representative coronaviruses and\\r\\n and Bat-SARSr-CoV RaTG13 was 1.17, which was the lowest except for SNU01 (another strain of SARS-CoV-2), indicating that their codon preference was almost the same; the second lowest codon usage difference was 2.41 that was detected between SARS-CoV-2 WIV04 and Bat-SL-CoVZC45 (Fig. 2) . The coronaviruses in same beta-CoV lineage B have a relatively close \\r\\n To infer whether the retention of synonymous mutations is supported or hindered by natural selection, we further studied non-synonymous substitution rate (dN) and synonymous substitution rate (dS) in spike gene (Table 1) . Generally, positive (Darwinian) selection increases, but negative (purifying) selection decreases the ratios of non-synonymous to synonymous substitution rates (dN/dS). The results showed that both dN and dS of S gene of SARS-CoV-2 WIV04 versus Bat-SARr-CoV RaTG13 were the lowest among all typical coronaviruses, while those of SARS-CoV Tor2 versus bat SARS-like coronavirus WIV1 were the second lowest. When the ratio of dN to dS of spike gene is compared, all the tested dN/dS are less than 1, indicating that these non-synonymous mutations are harmful, and negative selection will reduce their retention speed. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint Among them, dN/dS of spike gene of SARS-CoV-2 WIV04 versus that of Bat-SARr-CoV RaTG13 was 0.04, which was the lowest among all comparisons, reconfirming that the rate of synonymous mutation was increased between WIV04 and RaTG13 strains. Moreover, dN/dS rate of polyprotein (ORF1ab) and nucleocapsid (N) genes were similar with that of spike gene (Table   S2 ).\\r\\n The spike (S) protein undergoes several drastic changes during virus infection. For instance, its large parts are cleaved during infection by cellular proteases and expose the receptors to activate viral attachment to the host (17) . As the receptor binding domain (RBD) of spike protein was involved in interacting with human angiotensin-converting enzyme 2 (ACE2) protein, the RBD region was thought to be preferential targets of natural selection (18) . Consistent with this hypothesis, our results showed that both dN and dS of RBD region are increased in comparison with whole spike gene region across all the virus pairs used in this study (Table 1) . Notably, the dN/dS ratio of RBD region in SARS-CoV-2 WIV04 was dramatically increased in about 3 fold in comparison with full spike region. Consequently, these mutations may be subjected to Darwin's choice and will become evidence of adaptive protein evolution.\\r\\n To detect positive selection on the spike gene, the MEME analysis was performed.\\r\\n Significant (p < 0.05) pervasive episodic selection was detected in three sites (48th, 254th, 330th position using the reference sequence of WIV04). On the 254th position of the spike amino acid sequence, there is a histidine residue instead of a phenylalanine residue, whereas on the 330th amino acidic position in WIV04 sequence, there is a glutamine residue instead of an valine residue.\\r\\n The results described above strongly supporting the action of positive selection in spike gene during the recent evolution of SARS-CoV-2 and RaTG13. However, this result should be . CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint addressed with careful explain, because the RBD region of spike gene from SARS-CoV-2 was quit divergent with that of RaTG13 (Fig. S1) , suggesting it might have originated from homologous recombination between RaTG13 and one yet-unknown coronavirus (19) .\\r\\n Through comprehensive comparative analysis between SARS-CoV-2 and other coronaviruses, we found the synonymous mutations is dramatically elevated between SARS-CoV-2 and RaTG13 than that of other coronavirus strains, and the nucleotide mutations were enriched in T:C transition.\\r\\n The specific mutation pattern may caused by the loss function of RNA 3'-to-5' exoribonuclease (ExoN). Moreover, as the SARS-CoV-2 was supposed to be originated from Bat-SARSr-CoV RaTG13, the increased synonymous substitution between SARS-CoV-2 and RaTG13 strain suggested the SARS-CoV-2 genome was under stronger negative (purifying) selection. We also detected some sites in spike protein was experiencing positive selection. These observations suggest that adaptive evolution might contribute to its host shifts. However, as the RNA mutagen (e.g. 5-FU) could induce the same mutation pattern, the mechanism of the mutation pattern observed between SARS-CoV-2 and RaTG13 should be further investigated in future work. The number of amino acid changes. RSCUdist: the RSCU distance of spike protein; ENc: effective number of codons; S ident(%): the percent identity of S protein sequence between different coronaviruses. The NT/AA ratio between WIV04/RaTG13 was statistically significant higher (Chi-squared test, P<0.01) than that of other strain pairs.\\r\\n . CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint *The whole genome sequence identity between these paired strains was larger than 95%.\\r\\n . CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint. The COVID-19 outbreak began on December 2019 in Wuhan, China (1), and the causative virus, SARS-CoV-2, has rapidly spread into 43 countries as of February 27, 2020. The number of infected patients exceeds 81,000, and the death toll exceeds 2,700. Several drugs, including lopinavir, remdesivir, and chloroquine, have been reported to be presumably effective in treating this disease (2, 3) .\\r\\n Systemic treatment with corticosteroids is contraindicated for the severe pneumonia caused by viruses such as MERS-CoV and SARS-CoV, as steroids suppress the innate immune system, resulting in increased viral replication. In fact, for SARS in 2003 and MERS in 2013, treatment with corticosteroids was associated with increased mortality (4, 5). In the present study, we reconsider the use of inhaled corticosteroids, which have been excluded from the treatment of pneumonia caused by coronavirus.\\r\\n Steroid compounds, which are expected to have dual functions in blocking both coronavirus replication and host inflammation, were screened from a chemical library. The cytopathic effect caused by MERS-CoV infection was measured to evaluate viral replication. Four steroid compounds, ciclesonide, mometasone furoate, mifepristone, and algestone acetophenide conferred a greater than 95% cell survival rate (Fig. S1 ).\\r\\n Next, concentration-dependent viral growth suppression and drug cytotoxicity were assessed. Ciclesonide exhibited low cytotoxicity and potent suppression of viral growth (Fig.   1a ). Cortisone and prednisolone, which are commonly used for systemic steroid treatment, dexamethasone, which has strong immunosuppressant effects, and fluticasone, a commonly used inhaled steroid, did not suppress viral growth (Fig. 1a) .\\r\\n The antiviral effects of steroids against various species of viruses were tested by quantifying propagated virus in culture medium. Ciclesonide and mometasone also suppressed replication of other coronaviruses, HCoV-229E and SARS-CoV, but not replication of RS virus or influenza virus (Fig. S2 ). In addition, ciclesonide slightly but significantly inhibited rubella virus (having positive strand RNA genome) replication (Fig. S2 ), suggesting that ciclesonide interacted with the replication site of positive-strand RNA virus intracellularly.\\r\\n To identify the drug target of virus replication, we conducted 11 consecutive MERS-CoV passages in the presence of 40 μ M ciclesonide or 40 μ M mometasone. A mutant virus that developed resistance ciclesonide was generated (Fig. 1b) , but no resistant virus to author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.11.987016 doi: bioRxiv preprint mometasone was generated. Next-generation sequencing identified that an amino acid substitution at A25V (C19647T in the reference sequence NC_019843.3) in nonstructural protein 15 (NSP15), coronavirus endoribonuclease (6-8), was the predicted mechanism for viral resistance to ciclesonide. A recombinant virus carrying the A25V amino acid substitution (Re-Nsp15-A25V) was generated from the parental MERS-CoV/EMC strain (Re-EMC/MERS) with a BAC reverse genetics system (9), which overcame the antiviral effect of ciclesonide (Fig. 1c) . Interestingly, the mutant virus was inhibited by mometasone, suggesting that the antiviral target of mometasone was different from that of ciclesonide.\\r\\n Next, the effect of ciclesonide on SARS-CoV-2 infection was evaluated. VeroE6/TMPRSS2 cells(10) were infected with authentic SARS-CoV-2 in the presence of steroids or other inhibitors. At 6 h post-infection, cellular RNA was isolated, and real-time PCR was conducted to quantify the amount of viral RNA (11). Ciclesonide and mometasone suppressed viral replication with a similar efficacy to lopinavir (Fig. 2a) . For comparison, viral cell entry inhibitors were tested. E64d substantially reduced viral RNA levels, but nafamostat and camostat had only modest antiviral effects, suggesting that SARS-CoV-2 primarily utilized the cathepsin/endosomal pathway of cell entry rather than the TMPRSS2/cell surface pathway to enter TMPRSS2-expressing cells, consistent with a recent study (12). author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https: //doi.org/10.1101 //doi.org/10. /2020 The copyright holder for this preprint (which was not peer-reviewed) is th . https://doi.org/10.1101/2020.03.11.987016 doi: bioRxiv preprint a RNA copy (log 10 ) SARS-CoV-2 RNA in cell (6 hpi Cell viability (%) author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is th . https://doi.org/10.1101/2020.03.11.987016 doi: bioRxiv preprint. Seventeen years have passed since the outbreak of severe acute respiratory syndrome (SARS) in 2003, but there is yet no approved treatment for infections with the SARS coronavirus (SARS-CoV). 1 One of the reasons is that despite the devastating consequences of SARS for the affected patients, the development of an antiviral drug against this virus would not be commercially viable in view of the fact that the virus has been rapidly contained and did not reappear since 2004. As a result, we were empty-handed when the Middle-East respiratory syndrome coronavirus (MERS-CoV), a close relative of SARS-CoV, emerged in 2012. 2 MERS is characterized by severe respiratory disease, quite similar to SARS, but in addition frequently causes renal failure 3 . Although the number of registered MERS cases is low (2494 as of November 30, 2019; www.who.int), the threat MERS-CoV poses to global public health may be even more serious than that presented by SARS-CoV. This is related to the high casefatality rate (about 35%, compared to 10% for SARS), and to the fact that MERS cases are still accumulating seven years after the discovery of the virus, whereas the SARS outbreak was essentially contained within 6 months. The potential for human-to-human transmission of MERS-CoV has been impressively demonstrated by the 2015 outbreak in South Korea, where 186 cases could be traced back to a single infected traveller returning from the Middle East. 4 SARS-like coronaviruses are still circulating in bats in China, [5] [6] [7] [8] from where they may spill over into the human population; this is probably what caused the current outbreak of atypical pneumonia in Wuhan, which is linked to a seafood and animal market. The RNA genome (Gen-Bank accession code: MN908947.2; http://virological.org/t/initialgenome-release-of-novel-coronavirus/319, last accessed on January 11, 2020) of the new betacoronavirus features around 82% identity to that of SARS-CoV.\\r\\n In spite of the considerable threat posed by SARS-CoV and related viruses, as well as by MERS-CoV, it is obvious that the number of cases so far does not warrant the commercial development of an antiviral drug targeting MERS-and SARS-CoV even if a projected steady growth of the number of MERS cases is taken into account. A possible solution to the problem could be the development of broadspectrum antiviral drugs that are directed against the major viral protease, a target that is shared by all coronavirus genera as well as, in a related form, by members of the large genus Enterovirus in the picornavirus family. Among the members of the genus Alphacoronavirus are the human coronaviruses (HCoV) NL63 (ref. 9) and 229E 10 that usually cause only mild respiratory symptoms in otherwise healthy individuals, but are much more widespread than SARS-CoV or MERS-CoV. Therapeutic intervention against alphacoronaviruses is indicated in cases of accompanying disease such as cystic fibrosis 11 or leukemia, 12 or certain other underlying medical conditions. 13 The enteroviruses include pathogens such as EV-D68, the causative agent of the 2014 outbreak of the \\\"summer flu\\\" in the US, 14 EV-A71 and Coxsackievirus A16 (CVA16), the etiological agents of Hand, Foot, and Mouth Disease (HFMD), 15 Coxsackievirus B3 (CVB3), which can cause myocardic inflammation, 16 and human rhinoviruses (HRV), notoriously known to lead to the common cold but also capable of causing exacerbations of asthma and COPD. 17 Infection with some of these viruses can lead to serious outcome; thus, EV-D68 can cause polio-like disease, 18 and EV-A71 infection can proceed to aseptic meningitis, encephalitis, pulmonary edema, viral myocarditis, and acute flaccid paralysis. 15, [19] [20] Enteroviruses cause clinical disease much more frequently than coronaviruses, so that an antiviral drug targeting both virus families should be commercially viable.\\r\\n However, enteroviruses are very different from coronaviruses. While both of them have a single-stranded RNA genome of positive polarity, that of enteroviruses is very small (just 7 -9 kb) whereas coronaviruses feature the largest RNA genome known to date (27 -34 kb) . Enteroviruses are small, naked particles, whereas coronaviruses are much larger and enveloped. Nevertheless, a related feature shared by these two groups of viruses is their type of major protease, 21 which in the enteroviruses is encoded by the 3C region of the genome (hence the protease is designated 3C pro ). In coronaviruses, non-structural protein 5 (Nsp5) is the main protease (M pro ). Similar to the enteroviral 3C pro , it is a cysteine protease in the vast majority of cases and has therefore also been called \\\"3C-like protease\\\" (3CL pro ). The first crystal structure of a CoV M pro or 3CL pro (ref. 22) revealed that two of the three domains of the enzyme together resemble the chymotrypsin-like fold of the enteroviral 3C pro , but there is an additional a-helical domain that is involved in the dimerization of the protease (Fig. 1A) . This dimerization is essential for the catalytic activity of the CoV M pro , whereas the enteroviral 3C pro (Fig. 1B) functions as a monomer. Further, the enteroviral 3C pro features a classical Cys...His...Glu/Asp catalytic triad, whereas the CoV M pro only has a Cys...His dyad. 22 Yet, there are a number of common features shared between the two types of proteases, in particular their almost absolute requirement for Gln in the P1 position of the substrate and space for only small amino-acid residues such as Gly, Ala, or Ser in the P1' position, encouraging us to explore the coronaviral M pro and the enteroviral 3C pro as a common target for the design of broad-spectrum antiviral compounds. The fact that there is no known human protease with a specificity for Gln at the cleavage site of the substrate increases the attractivity of this viral target, as there is hope that the inhibitors to be developed will not show toxicity versus the host cell. Indeed, neither the enterovirus 3C pro inhibitor rupintrivir, which was developed as a treatment of the common cold caused by HRV, nor the peptide aldehyde inhibitor of the coronavirus M pro that was recently demonstrated to lead to complete recovery of cats from the normally fatal infection with Feline Infectious Peritonitis Virus (FIPV), showed any toxic effects on humans or cats, respectively. 23, 24 Figure 1: Crystal structures of SARS-CoV main protease (M pro , ref. 26 ; PDB entry 2BX4) and Coxsackivirus B3 3C protease (3C pro ; Tan et al., unpublished; PDB entry 3ZYD). Catalytic residues are indicated by spheres (yellow, Cys; blue, His; red: Glu). A. The coronavirus M pro is a homodimer, with each monomer comprising three domains. B. The structure of the monomeric CVB3 3C pro resembles the Nterminal two domains of the SARS-CoV M pro . Structure is on the same scale as image A. C. Superimpostion of residues from the two structures involved in ligand binding. Superimposition was carried out by aligning the catalytic Cys-His pair of each protease. Residues of the SARS-CoV M pro are shown with carbon atoms in cyan, CVB3 3C pro residues have orange carbons and are labeled with an asterisk (*).\\r\\n As the proteases targeted in our study all specifically cleave the peptide bond following a P1-glutamine residue (HCoV-NL63 M pro uniquely also accepts P1 = His at the Nsp13/Nsp14 cleavage site 31 ), we decided to use a 5-membered ring (g-lactam) derivative of glutamine (henceforth called GlnLactam) as the P1 residue in all our aketoamides (see Scheme 1). This moiety has been found to be a good mimic of glutamine and enhance the power of the inhibitors by up to 10-fold, most probably because compared to the flexible glutamine side-chain, the more rigid lactam leads to a reduction of the loss of entropy upon binding to the target protease. 29, 32 Our synthetic efforts therefore aimed at optimizing the substituents at the P1', P2, and P3 positions of the a-ketoamides.\\r\\n Synthesis (Scheme 1) started with the dianionic alkylation of N-Boc glutamic acid dimethyl ester with bromoacetonitrile. As expected, this alkylation occurred in a highly stereoselective manner, giving 1 as the exclusive product. In the following step, the cyano group of 1 was subjected to hydrogenation. The in-situ cyclization of the resulting intermediate afforded the lactam 2. The lactam derivative 3 was generated by removal of the protecting group of 2. On the other hand, the amidation of acyl chloride and a-amino acid methyl ester afforded the intermediates 4, which gave rise to the acids 5 via alkaline hydrolysis. The key intermediates 6 were obtained via the condensation of the lactam derivative 3 and the N-capped amino acids 5. The ester group of compounds 6 was then reduced to the corresponding alcohol. Oxidation of the alcohol products 7 by Dess-Martin periodinane generated the aldehydes 8, which followed by nucleophilic addition with isocyanides gave rise to compounds 9 under acidic conditions. Then, the a-hydroxyamides 10 were prepared by removing the acetyl group of compounds 9. In the final step, the oxidation of the exposed alcohol group in compounds 10 generated our target a-ketoamides 11.\\r\\n The inhibitory potencies of candidate a-ketoamides were evaluated against purified recombinant SARS-CoV M pro , HCoV-NL63 M pro , CVB3 3C pro , and EV-A71 3C pro . The most potent compounds were further tested against viral replicons and against SARS-CoV, MERS-CoV, or a whole range of enteroviruses in cell culture-based assays (Tables 1 -3 and Supplementary Table 1 \\r\\n To enable the rapid and biosafe screening of antivirals against corona-and enteroviruses, a non-infectious, but replication-competent SARS-CoV replicon was used 33 along with subgenomic replicons of CVB3 34 and EV-A71 (kind gift of B. Zhang, Wuhan, China). The easily detectable reporter activity (firefly or Renilla luciferase) of these replicons has previously been shown to reflect viral RNA synthesis. [33] [34] [35] In-vitro RNA transcripts of the enteroviral replicons were also used for transfection. For the SARS-CoV replicon containing the CMV promoter, only the plasmid DNA was used for transfection.\\r\\n The initial compound to be designed and synthesized was 11a, which carries a cinnamoyl N-cap in the P3 position, a benzyl group in P2, the glutamine lactam (GlnLactam) in P1, and benzyl in P1' (Table  1) . This compound showed good to mediocre activities against recombinant SARS-CoV M pro (IC50 = 1.95 µM; for all compounds, see Tables 1 -3 for standard deviations), CVB3 3C pro (IC50 = 6.6 µM), and EV-A71 3C pro (IC50 = 1.2 µM), but was surprisingly completely inactive (IC50 > 50 µM) against HCoV-NL63 M pro . These values were mirrored in the SARS-CoV and in the enterovirus replicons (Table 2) . In virus-infected cell cultures, the results obtained were also good to mediocre (Table 3) : SARS-CoV (EC50 = 5.8 µM in Vero E6 cells), MERS-CoV (EC50 = 0.0047 µM in Huh7 cells), HCoV 229E (EC50 = 11.8 µM in Huh7 cells), or a host of enteroviruses (EC50 = 9.8 µM against EV-A71 in RD cells; EC50 = 0.48 µM against EV-D68 in HeLa Rh cells; EC50 = 5.6 µM against HRV2 in HeLa Rh cells). In all cell types tested, the compound generally proved to be non-toxic, with selectivity indices (CC50/EC50) usually >10 (Table 3) .\\r\\n Crystal structures of compound 11a in complex with SARS-CoV M pro , HCoV-NL63 M pro , and CVB3 3C pro demonstrated that the aketo-carbon is covalently linked to the active-site Cys (no. 145, 144, and 147, resp.) of the protease (Fig. 2, 3a-c) . The resulting thiohemiketal is in the R configuration in the SARS-CoV and HCoV-NL63 M pro but in the S configuration in the CVB3 3C pro complex. The reason for this difference is that the oxygen atom of the thiohemiketal accepts a hydrogen bond from the catalytic His40 in the CVB3 protease, rather than from the main-chain amides of the oxyanion hole as in the SARS-CoV and HCoV-NL63 enzymes (Fig. 3a ,b,c insets). It is remarkable that we succeeded in obtaining a crystal structure of compound 11a in complex with the HCoV-NL63 M pro , even though it has no inhibitory effect on the activity of the enzyme (IC50 > 50 µM), (Fig. 2c) . Apparently, the compound is able to bind to this M pro in the absence of peptide substrate, but cannot compete with substrate for the binding site due to low affinity. A similar observation has been made in one of our previous studies, where we were able to determine the crystal structure of a complex between the inactive Michaelacceptor compound SG74 and the EV-D68 3C pro (ref. 29 ; PDB entry 3ZV9). Figure 2 : Fit of compound 11a (pink carbon atoms) to the target proteases (wheat surfaces) as revealed by X-ray crystallography of the complexes. A. Fo -Fc difference density (contoured at 3s) for 11a in the substrate-binding site of the SARS-CoV M pro (transparent surface). Selected side-chains of the protease are shown with green carbon atoms. B. Another view of 11a in the substrate binding site of the SARS-CoV M pro . Note the \\\"lid\\\" formed by residue Met49 and its neighbors above the S2 pocket. C. 11a in the substrate-binding site of HCoV-NL63 M pro . Because of the restricted size of the S2 pocket, the P2 benzyl group of the compound cannot enter deeply into this site. Note that the S2 pocket is also covered by a \\\"lid\\\" centred around Thr47. D. 11a in the substrate-binding site of the CVB3 3C pro . The S2 site is large and not covered by a \\\"lid\\\".\\r\\n The crystal structures indicated that the fits of the P1' benzyl group of 11a in the S1' pocket and of the P3 cinnamoyl cap in the S3 subsite might be improved (see Fig. 3a -c). Compounds 11b -11e and 11g -11l were synthesized in an attempt to do so; however, none of them showed better inhibitory activity against the majority of the recombinant proteases, compared to the parent compound, 11a (see Supplementary Results). To investigate whether the P3 residue of the inhibitor is dispensible, we synthesized compound 11f, which only comprises P2 = Boc, P1 = GlnLactam, and P1' = benzyl. 11f was inactive against all purified proteases and in all replicons tested, but showed some activity against HRV2 in HeLa Rh cells (EC50 = 9.0 µM). A crystal structure of 11f bound to HCoV-NL63 M pro demonstrated that the P2-Boc group entered the S2 pocket (Fig. 3d ). In conclusion, although there is probably room for further improvement, we decided to maintain the original design with P1' = benzyl and P3 = cinnamoyl, and focussed on improving the P2 substituen.\\r\\n The crystal structures of SARS-CoV M pro , HCoV-NL63 M pro , and CVB3 3C pro in complex with 11a revealed a fundamental difference between the S2 pockets of the coronavirus proteases and the enterovirus proteases: The cavities are covered by a \\\"lid\\\" in the former but are open to one side in the latter (Fig. 2,b-d) . In SARS-CoV M pro , the lid is formed by the 310 helix 46 -51 and in HCoV-NL63 M pro by the loop 43-48. Residues from the lid, in particular Met49 in the case of SARS-CoV M pro , can thus make hydrophobic interactions with the P2 substitutent of the inhibitor, whereas such interaction is missing in the enterovirus 3C pro s. In addition to the lid, the S2 pocket is lined by the \\\"back-wall\\\" (main-chain atoms of residues 186 and 188 and Cb atom of Asp187), the side-walls (Gln189, His41), as well as the \\\"floor\\\" (Met165) in SARS-CoV M pro . In HCoV-NL63 M pro , the corresponding structural elements are main-chain atoms of residues 187 and 188 as well as the Cb atom of Asp187 (back-wall), Pro189 and His41 (side-walls), and Ile165 (floor). Finally, in CVB3 3C pro , Arg39, Asn69, and Glu71 form the back-wall, residues 127-132 and His40 form the side-walls, and Val162 constitutes the floor.\\r\\n In addition, the S2 pocket is of different size in the various proteases. The SARS-CoV enzyme features the largest S2 pocket, with a volume of 252 Å 3 embraced by the residues (Gln189, His41) defining the side-walls of the pocket in the ligand-free enzyme, as calculated by using Chimera, 36 followed by the CVB3 3C pro S2 pocket with about 180 Å 3 (space between Thr130 and His40). The HCoV-NL63 M pro has by far the smallest S2 pocket of the three enzymes, with a free space of only 45 Å 3 between Pro189 and His41, according to Chimera.\\r\\n In agreement with these observations, a good fit is observed between the P2 benzyl group of 11a and the S2 subsite of the SARS-CoV M pro as well as that of the CVB3 3C pro (Fig. 3a,c) . In contrast, the crystal structure of the complex between 11a and HCoV-NL63 M pro , against which the compound is inactive, demonstrates that the P2-benzyl group cannot fully enter the S2 pocket of the enzyme because of the restricted size of this site (Fig. 3b) .\\r\\n Thus, the properties of our target proteases with respect to the S2 pocket were defined at this point as \\\"small\\\" and \\\"covered by a lid\\\" for HCoV-NL63 M pro , \\\"large\\\" and \\\"covered\\\" for SARS-CoV M pro , and \\\"large\\\" and \\\"open\\\" for CVB3 3C pro . Through comparison with crystal structures of other proteases of the same virus genus (HCoV-229E M pro for alphacoronaviruses 28 (PDB entry 1P9S); HKU4 M pro for betacoronaviruses (Ma, Xiao et al., unpublished; PDB entry 2YNA); and EV-A71 3C pro for enteroviruses 30 (PDB entry 3SGK), we ensured that our conclusions drawn from the template structures were valid for other family members as well.\\r\\n To explore the sensitivity of the S2 pocket towards a polar substituent in the para position of the benzyl group, we synthesized compound 11m carrying a 4-fluorobenzyl group in P2. This substitution abolished almost all activity against the SARS-CoV M pro (IC50 > 50 µM), and the compound proved inactive against HCoV-NL63 M pro as well, whereas IC50 values were 2.3 µM against the EV-A71 3C pro and 8.7 µM against CVB3 3C pro . From this, we concluded that the introduction of the polar fluorine atom is not compatible with the geometry of the S2 pocket of SARS-CoV M pro , whereas the fluorine can accept a hydrogen bond from Arg39 in EV-A71 3C pro (ref. 30 ) and probably also CVB3 3C pro . In SARS-CoV M pro , however, the carbonyl groups of residues 186 and 188 might lead to a repulsion of the fluorinated benzyl group. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.10.936898 doi: bioRxiv preprint Table 2 : a-ketoamide-induced inhibition of subgenomic RNA synthesis using replicons in a cell-based assay (EC50, μM) 1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.10.936898 doi: bioRxiv preprint Table 3 . The thiohemiketal is in the R configuration, with its oxygen accepting two hydrogen bonds from the oxyanion-hole amides of Gly143 and Cys145. The amide oxygen accepts an H-bond from His41. The side-chains of Ser144 and Arg188 have been omitted for clarity. B. The P2-benzyl substituent of 11a cannot fully enter the S2 pocket of the HCoV-NL63 M pro , which is much smaller and has less plasticity than the corresponding pocket of SARS-CoV M pro (cf. A). The benzyl therefore binds above the pocket in the view shown here; this is probably the reason for the total inactivity (IC50 > 50 µM) of compound 11a against HCoV-NL63 M pro . The small size of the pocket is due to the replacement of the flexible Gln189 of the SARS-CoV M pro by the more rigid Pro189 in this enzyme. The stereochemistry of the thiohemiketal is R. The sidechains of Ala143 and Gln188 have been omitted for clarity. C. Binding of 11a to the CVB3 3C pro . The stereochemistry of the thiohemiketal is S, as the group accepts a hydrogen bond from His41, whereas the amide keto group accepts three H-bonds from the oxyanion hole (residues 145 -147). The side-chain of Gln146 has been omitted for clarity. D. The crystal structure of 11f in complex with HCoV-NL63 M pro shows that this short (inactive) compound lacking a P3 residue has its P2-Boc group inserted into the S2 pocket of the protease. The stereochemistry of the thiohemiketal is S. The sidechains of Ala143 and Gln188 have been omitted for clarity. E. In contrast to P2 = benzyl in 11a, the isobutyl group of 11n is small and flexible enough to enter into the narrow S2 pocket of the HCoV-NL63 M pro . The thiohemiketal is in the R configuration. The side-chains of Ala143 and Gln188 have been omitted for clarity. F. In spite of its small size, the cyclopropylmethyl side-chain in the P2 position of 11s can tightly bind to the S2 subsite of the SARS-CoV M pro , as this pocket exhibits pronounced plasticity due to the conformational flexibility of Gln189 (see also Fig. 4 ). The stereochemistry of the thiohemiketal is S. The side-chains of Ser144 and Arg188 have been omitted for clarity.\\r\\n As the P2-benzyl group of 11a was apparently too large to fit into the S2 pocket of the HCoV-NL63 M pro , we replaced it by isobutyl in 11n. This resulted in improved activities against SARS-CoV M pro (IC50 = 0.33 µM) and in a very good activity against HCoV-NL63 M pro (IC50 = 1.08 µM, compare with the inactive 11a). For EV-A71 3C pro , however, the activity decreased to IC50 = 13.8 µM, different from CVB3 3C pro , where IC50 was 3.8 µM. Our interpretation of this result is that the smaller P2-isobutyl substituent of 11n can still interact with the \\\"lid\\\" (in particular, Met49) of the SARS-CoV M pro S2 site, but is unable to reach the \\\"back-wall\\\" of the EV-A71 3C pro pocket and thus, in the absence of a \\\"lid\\\", cannot generate sufficient enthalpy of binding. We will see from examples to follow that this trend persists among all inhibitors with a smaller P2 substituent: Even though the SARS-CoV M pro S2 pocket has a larger volume than that of the enterovirus 3C pro , the enzyme can be efficiently inhibited by compounds carrying a small P2 residue that makes hydrophobic interactions with the lid (Met49) and floor (Met165) residues.\\r\\n The EC50 of 11n was >10 µM against the EV-A71 and CVB3 replicons, and even in the SARS-CoV replicon, the activity of 11n was relatively weak (EC50 = 7.0 µM; Table 2 ). In agreement with the replicon data, 11n proved inactive against EV-A71 in RD cells and showed limited activity against HRV2 or HRV14 in HeLa Rh cells (Table 3) . Only the comparatively good activity (EC50 = 4.4 µM) against EV-D68 in HeLa Rh cells was unexpected. The activity of 11n against HCoV 229E in Huh7 cells was good (EC50 = 0.6 µM), and against MERS-CoV in Huh7 cells, it was excellent, with EC50 = 0.0048 µM, while in Vero cells, the EC50 against MERS-CoV was as high as 9.2 µM. Similarly, the EC50 against SARS-CoV in Vero cells was 14.2 µM (Table 3) .\\r\\n We managed to obtain crystals of 11n in complex with the M pro of HCoV NL63 and found the P2 isobutyl group to be well embedded in the S2 pocket (Fig. 3e) . This is not only a consequence of the smaller size of the isobutyl group compared to the benzyl group, but also of its larger conformational flexibility, which allows a better fit to the binding site.\\r\\n When we replaced the P2-isobutyl residue of 11n by n-butyl in 11o, the activities were as follows: IC50 = 8.5 µM for SARS-CoV M pro , totally inactive (IC50 > 50 µM) against HCoV-NL63 M pro , IC50 = 3.2 µM for EV-A71 3C pro , and 5.2 µM for CVB3 3C pro . The decreased activity in case of SARS-CoV M pro and the total inactivity against HCoV-NL63 M pro indicate that the n-butyl chain is too long for the S2 pocket of these proteases, whereas the slight improvement against EV-A71 3C pro and CVB3 3C pro is probably a consequence of the extra space that is available to long and flexible substituents because of the lack of a lid covering the enterovirus 3C pro pocket.\\r\\n As the n-butyl substituent in P2 of 11o was obviously too long, we next synthesized a derivative with the shorter propargyl (ethinylmethyl) as the P2 residue (compound 11p). This led to very mediocre activities against all tested proteases. Using cyclopropyl as the P2 residue (compound 11q), the IC50 values were even higher against most of the proteases tested. Obviously, the P2 side-chain requires a methylene group in the b-position in order to provide the necessary flexibility for the substituent to be embedded in the S2 pocket.\\r\\n Having realized that in addition to size, flexibility of the P2 substituent may be an important factor influencing inhibitory activity, we introduced flexibility into the phenyl ring of 11a by reducing it. The cyclohexylmethyl derivative 11r exhibited IC50 = 0.7 µM against SARS-CoV M pro , 12.3 µM against HCoV-NL63 M pro , 1.7 µM against EV-A71 3C pro , and 0.9 µM against CVB3 3C pro . Thus, the replace- 1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.10.936898 doi: bioRxiv preprint ment of the phenyl group by the cyclohexyl group led to a significant improvement of the inhibitory activity against the recombinant SARS-CoV M pro and to a dramatic improvement in case of CVB3 3C pro . Even for the HCoV-NL63 M pro , against which 11a was completely inactive, greatly improved albeit still weak activity was observed (Table 1 ). In the viral replicons, 11r performed very well, with EC50 = 0.8 -0.9 µM for the EV-A71 replicon, 0.45 µM for CVB3, and 1.4 µM for SARS-CoV (Table 2 ). In the virus-infected cell culture assays (Table 3) , 11r exhibited EC50 = 3.7 µM against EV-A71 in RD cells and EC50 = 0.48 -0.7 µM against EV-D68, HRV2, and HRV14 in HeLa cells. Against HCoV 229E in Huh7 cells, the EC50 was surprisingly low (1.8 µM). Interestingly, the compound proved extremely potent against MERS-CoV in Huh7 cells, with EC50 = 0.0004 µM (400 picomolar!). Even in Vero cells, EC50 against MERS-CoV was 5 µM, and the EC50 against SARS-CoV in Vero E6 cells was 1.8 -2.1 µM, i.e. the best activity we have seen for an M pro inhibitor against SARS-CoV in this type of cells. The therapeutic index (CC50/EC50) of 11r against EV-D68, HRV2, and HRV14 was >15 in HeLa Rh cells as well as against CVB3 in Huh-T7 cells, but only ~5 for EV-A71 in RD cells.\\r\\n At this point, we decided to systematically vary the size of the ring system in P2. The next substituent to be tried was cyclopropylmethyl (compound 11s, which showed good activities against SARS-CoV M pro (IC50 = 0.24 µM) and HCoV-NL63 M pro (1.4 µM), but poor values against EV-A71 3C pro (IC50 = 18.5 µM) and CVB3 3C pro (IC50 = 4.3 µM) (Table 1) . 11s was shown to inhibit the SARS-CoV replicon with an EC50 of about 2 µM, whereas activity against the EV-A71 and CVB3 replicons was poor (EC50 values > 20 µM) ( Table 2 ). The replicon results were mirrored by the antiviral activity of 11s in enterovirus-infected cells (Table 3) , which was weak or very weak. By contrast, the compound inhibited HCoV 229E and MERS-CoV in Huh7 cells with EC50 of 1.3 and 0.08 µM, respectively. The activity against the latter virus in Vero cells was poor (EC50 ~11 µM), and so was the anti-SARS-CoV activity in Vero E6 cells (Table 3) .\\r\\n We next analyzed the crystal structure of the complex between SARS-CoV M pro and compound 11s (Fig. 3f) . The cyclopropylmethyl substituent was found to be incorporated deeply into the S2 pocket, making hydrophobic interactions with Met49 (the lid), Met165 (the floor), and the Cb of Asp187 (the back-wall). In spite of the small size of the P2 substituent, this is possible because the S2 pocket of SARS-CoV M pro is flexible enough to contract and enclose the P2 moiety tightly. This plasticity is expressed in a conformational change of residue Gln189, both in the main chain and in the side-chain. The main-chain conformational change is connected with a flip of the peptide between Gln189 and Thr190. The c1 torsion angle of the Gln189 side-chain changes from roughly antiperiplanar (ap) to (-)synclinal (-sc) (Fig. 4) . The conformational variability of Gln189 has been noted before, both in Molecular Dynamics simulations 26 and in other crystal structures. 37 As a consequence of these changes, the sidechain oxygen of Gln189 can accept a 2.54-Å hydrogen bond from the main-chain NH of the P2 residue in the 11s complex (see Fig. 4 ). The affinity of 11s for the S2 pocket of HCoV-NL63 M pro is good because of an almost ideal match of size and not requiring conformational changes, which this enzyme would not be able to undergo because of the replacement of the flexible Gln189 by the more rigid Pro. On the other hand, docking of the same compound into the crystal structure of the CVB3 3C pro revealed that the cyclopropylmethyl moiety was probably unable to generate sufficient Free Energy of binding because of the missing lid and the large size of the S2 pocket in the enterovirus 3C pro , thereby explaining the poor inhibitory activity of 11s against these targets. (Table 1) . Experiments with the viral replicons confirmed this trend, although the EC50 value for SARS-CoV (6.8 µM) was surprisingly high (Table 2 ). In Huh7 cells infected with MERS-CoV, this compound exhibited EC50 = 0.1 µM (but 9.8 µM in Vero cells), whereas EC50 was 7.0 µM against SARS-CoV in Vero E6 cells. The compound was largely inactive against EV-A71 in RD cells and inhibited the replication of the two HRV subtypes tested (in HeLa Rh cells) with EC50 values of ~4 µM. The CC50 of 11t in HeLa cells was 65 µM, i.e. the therapeutic index was well above 15 (Table 3) .\\r\\n Obviously, this substituent was still a bit too small for the enterovirus proteases, so as the next step, we tested P2 = cyclopentylmethyl (compound 11u). This turned out to be the one compound with acceptable IC50 values against all tested enzymes: 1.3 µM against SARS-CoV M pro , 5.4 µM against HCoV-NL63 M pro , 4.7 µM against EV-A71 3C pro , and 1.9 µM against CVB3 3C pro ( Table 1 ). The activity against the replicons was between 3.6 and 4.9 µM ( Table 2 ). In Huh7 cells infected with HCoV 229E or MERS-CoV, 11u showed EC50 = 2.5 or 0.03 µM (11.1 µM for MERS-CoV in Vero cells), while EC50 was 4.9 µM against SARS-CoV in Vero E6 cells (Table 3) .\\r\\n 11u appeared so far the best compromise compound, yet for each of the individual viral enzymes, the following compounds proved superior: P2 = cyclopropylmethyl (compound 11s) for SARS-CoV M pro , P2 = isobutyl (compound 11n) and P2 = cyclopropylmethyl (11s) for HCoV-NL63 M pro , P2 = benzyl (11a) or cyclohexylmethyl (11r) for EV-A71 3C pro , and 11r for CVB3 3C pro . In other words, the nearly equipotent 11u is indeed a compromise. Therefore, in view of the surprisingly good antiviral activity of 11r against HCoV 229E in Huh7 cells, we relaxed the condition that the universal inhibitor should show good activity against the recombinant HCoV-NL63 M pro , and selected 11r (P2 = cyclohexylmethyl) as the lead compound for further development. This compound exhibited submicromolar IC50 values against CVB3 3C pro and SARS-CoV M pro , and IC50 = 1.7 µM against EV-A71 3C pro (Table 1) , as well as similarly low EC50 values in the replicons of these viruses (Table 2 ). In Huh7 cells infected with MERS-CoV, the performance of this compound was excellent, with EC50 = 0.0004 µM, and even against HCoV 229E in Huh7 cells and SARS-CoV in Vero E6 cells, EC50 values of 1.8 and 2.1 µM, respectively, were observed (Table 3) . Also in enterovirus-infected cell culture, the compound performed well, with EC50 values of 0.7 µM or below against HRV2, HRV14, and EV-D68 in HeLa (Rh) cells and selectivity values >15. The only concern is the activity of the compound against EV-A71 in RD cells, for which the EC50 value was 3.7 µM, resulting in too low a therapeutic index. On the other hand, only weak toxicity was detected for 11r in Vero or Huh-T7 cells. Preliminary pharmacokinetics tests with the compound in mice did not indicate a toxicity problem (to be published elsewhere).\\r\\n We describe here the structure-based design, the synthesis, and the assessment of capped dipeptide a-ketoamides that target the main protease of alpha-or betacoronaviruses as well as the 3C protease of enteroviruses. Through crystallographic analyses of a total of six inhibitor complexes of three different proteases in this study, we found the a-ketoamide warhead (-CO-CO-NH-) to be sterically more versatile than other warheads such as Michael acceptors (-CH=CH-CO-) and aldehydes (-CH=O), because it features two acceptors for hydrogen bonds from the protein, namely the a-keto oxygen and the amide oxygen, whereas the other warheads have only one such acceptor. In the various complexes, the hydroxy group (or oxyanion) of the thiohemiketal that is formed by the nucleophilic attack of the activesite cysteine residue onto the a-keto carbon, can accept one or two hydrogen bonds from the main-chain amides of the oxyanion hole. In addition, the amide oxygen of the inhibitor accepts a hydrogen bond from the catalytic His side-chain. Alternatively, the thiohemiketal can interact with the catalytic His residue and the amide oxygen with the main-chain amides of the oxyanion hole. Depending on the exact interaction, the stereochemistry at the thiohemiketal C atom would be different. We have previously observed a similar difference in case of aldehyde inhibitors, where the single interaction point -the oxyanion of the thiohemiacetal -can accept a hydrogen bond either from the oxyanion hole or from the catalytic His side-chain, 37 resulting in different stereochemistry of the thiohemiacetal carbon. Both aketoamides and aldehydes react reversibly with the catalytic nucleophile of proteases, whereas Michael acceptors form irreversible adducts.\\r\\n In addition to better matching the H-bonding donor/acceptor properties of the catalytic center through offering two hydrogen-bond acceptors instead of one, a-ketoamides have another big advantage over aldehydes and a,b-unsaturated esters (Michael acceptors) in that they allow easy extension of the inhibitors to probe the primed specificity subsites beyond S1', although this has so far rarely been explored (e.g., ref. 38 in case of calpain).\\r\\n The most prominent a-ketoamide drugs are probably telaprivir and boceprivir, peptidomimetic inhibitors of the HCV NS3/4A protease, 39, 40 which have helped revolutionize the treatment of chronic HCV infections. For viral cysteine proteases, a-ketoamides have only occasionally been described as inhibitors and few systematic studies have been carried out.\\r\\n A number of capped dipeptidyl a-ketoamides have been described as inhibitors of the norovirus 3C-like protease. 41 These were optimized with respect to their P1' substituent, whereas P2 was isobutyl in most cases and occasionally benzyl. The former displayed IC50 values one order of magnitude lower than the latter, indicating that the S2 pocket of the norovirus 3CL protease is fairly small. Although we did not include the norovirus 3CL pro in our study, expanding the target range of our inhibitors to norovirus is probably a realistic undertaking. While our study was underway, Zeng et al. 42 published a series of a-ketoamides as inhibitors of the EV-A71 3C pro . These authors mainly studied the structure-activity relationships of the P1' residue and found small alkyl substituents to be superior to larger ones. Interestingly, they also reported that a six-membered d-lactam in the P1 position led to 2 -3 times higher activities, compared to the fivemembered g-lactam. At the same time, Kim et al. 43 described a series of five a-ketoamides with P1' = cyclopropyl that showed submicromolar activity against EV-D68 and two HRV strains.\\r\\n Occasionally, individual a-ketoamides have been reported in the literature as inhibitors of both the enterovirus 3C protease and the coronavirus main protease. A single capped dipeptidyl a-ketoamide, Cbz-Leu-GlnLactam-CO-CO-NH-iPr, was described that inhibited the recombinant transmissible gastroenteritis virus (TGEV) and SARS-CoV M pro s as well as human rhinovirus and poliovirus 3C pro s in the one-digit micromolar range. 44 Coded GC-375, this compound showed poor activity in cell culture against EV-A71 though (EC50 = 15.2 µM), probably because P2 was isobutyl. As we have shown here, an isobutyl side-chain in the P2 position of the inhibitors is too small to completely fill the S2 pocket of the EV-A71 3C pro and the CVB3 3C pro .\\r\\n Among a series of aldehydes, Prior et al. 45 described the capped tripeptidyl a-ketoamide Cbz-1-naphthylalanine-Leu-GlnLactam-CO-CO-NH-iPr, which showed IC50 values in the 3-digit nanomolar range against HRV 3C pro and SARS-CoV M pro , as well as EC50 values of 0.03 µM against HRV18 and 0.5 µM against HCoV 229E in cell culture. No optimization of this compound was performed and no toxicity data have been reported.\\r\\n For compounds with warheads other than a-ketoamides, in-vitro activity against both corona-and enteroviruses has also occasionally been reported. Lee et al. 46 described three peptidyl Michael acceptors that displayed inhibitory activity against the M pro s of SARS-CoV and HCoV 229E as well as against the 3C pro of CVB3. These inhibitors had an IC50 10 -20 times higher for the CVB3 enzyme, compared to SARS-CoV M pro . P2 was invariably isobutyl (leucine) in these compounds, suggesting that further improvement might be possible.\\r\\n In addition to Michael acceptors, peptide aldehydes have also been used to explore the inhibition of coronavirus M pro s as well as enterovirus 3C pro s. Kim et al. 44 reported a dipeptidyl aldehyde and its bisulfite adduct, both of which exhibited good inhibitory activities against the isolated 3C proteases of human rhinovirus and poliovirus as well as against the 3C-like proteases of a number of coronaviruses, but antiviral activities in cell culture against EV-A71 were poor (EC50 >10 µM), again most probably due to P2 being isobutyl (leucine).\\r\\n In our series of compounds, we used P1 = GlnLactam (g-lactam) throughout, because this substituent has proven to be an excellent surrogate for glutamine. 29, 32 While we made some efforts to optimize the P1' residue of the compounds as well as the N-cap (P3), we mainly focussed on optimization of the P2 substituent. In nearly all studies aiming at discovering peptidomimetic inhibitors of coronavirus M pro s, P2 is invariably isobutyl (leucine), and this residue has also been used in the efforts to design compounds that would inhibit enterovirus 3C pro s as well (see above). From crystal structures of our early lead compound, 11a (cinnamoyl-Phe-GlnLactam-CO-CO-NH-Bz), in complex with the M pro s of HCoV NL63 (as representative of the alphacoronavirus proteases) and SARS-CoV (beta-CoV) as well as the 3C pro of Coxsackievirus B3 (enterovirus proteases), we found that the S2 pocket has fundamentally different shapes in these enzymes. In the SARS-CoV M pro , the S2 subsite is a deep hydrophobic pocket that is truly three-dimensional in shape: the \\\"walls\\\" of the groove are formed by the polypeptide main chain around residues 186 -188 as well as by the side-chains of His41 (of the catalytic dyad) and Gln189, whereas the \\\"floor\\\" is formed by Met165 and the \\\"lid\\\" by residues 45 -51, in particular Met49. The two methionines provide important interaction points for the P2 substituents of inhibitors; while these interactions are mostly hydrophobic in character, we have previously described the surprising observation of the carboxylate of an aspartic residue in P2 that made polar interactions with the sulfur atoms of these methionines. 37 Because the pocket offers so many opportunities for interaction and features a pronounced plasticity, P2 substituents such as isobutyl (from Leu), which are too small to fill the pocket entirely, can still generate sufficient binding enthalpy. Accordingly, the S2 pocket of SARS-CoV M pro is the most tolerant among the three enzymes investigated here, in terms of versatility of the P2-substituents accepted.\\r\\n In the S2 pocket of the HCoV-NL63 M pro , Gln189 is replaced by proline and this change is accompanied by a significant loss of flexibility; whereas the side-chain of Gln189 of SARS-CoV M pro is found to accommodate its conformation according to the steric requirements of the P2 substituent, the proline is less flexible, leading to a much smaller space at the entrance to the pocket. As a consequence, a P2benzyl substituent is hindered from penetrating deeply into the pocket, whereas the smaller and more flexible isobutyl group of P2-Leu is not.\\r\\n Finally, in the 3C pro s of EV-A71 and CVB3, the S2 pocket lacks a lid, i.e. it is open to one side. As a consequence, it offers less interaction points for P2 substituents of inhibitors, so that such substituents must reach the \\\"back-wall\\\" of the pocket (formed by Arg39, Asn69, Glu71) in order to create sufficient binding energy. Hence, large aromatic substituents such as benzyl are favored by the enterovirus 3C pro s.\\r\\n When we introduced a fluoro substituent in the para position of the P2-benzyl group of our lead compound, 11a, we observed good activity against the enterovirus 3C pro s but complete inactivity against the coronavirus M pro s (see Table 1 , compound 11m). This is easily explained on the basis of the crystal structures: In the enterovirus 3C pro s, the fluorine can accept a hydrogen bond from Arg39 (ref. 30) , whereas in the coronavirus M pro s, there would be electrostatic repulsion from the main-chain carbonyls of residues 186 and 188. In agreement with this, rupintrivir (which has P2 = p-fluorobenzyl) is a good inhibitor of the enteroviral 3C pro s, 46 but not of the coronaviral main proteases, as we predicted earlier. 28 In this structure-based inhibitor optimization study, we achieved major improvements over our original lead compound, 11a, by systematically varying the size and the flexibility of the P2-substituent. The compound presenting so far the best compromise between the different requirements of the S2 pockets (SARS-CoV M pro : large and covered, HCoV-NL63 M pro : small and covered, and CVB3 3C pro : large and open) is 11u (P2 = cyclopentylmethyl), which has satisfactory broad-spectrum activity against all proteases tested. However, with regard to its antiviral activities in cell cultures, it is inferior to 11r (P2 = cyclohexylmethyl). The latter compound exhibits very good inhibitory activity against the SARS-CoV M pro as well as the enterovirus 3C pro , and its performance in the SARS-CoV and enterovirus replicons is convincing. Being in the low micromolar range (EV-A71, CVB3), the data for the antiviral activity in cell culture for 11r correlate well with the inhibitory power of the compound against the recombinant proteases as well as in the replicon-based assays. This is not true, though, for the surprisingly good in-cellulo activity of 11r against HCoV 229E in Huh7 cells. Also, the correlation does not seem to hold for LLC-MK2 and CaCo2 cells. We tested the antiviral activity of many of our compounds against HCoV NL63 in these two cell types and found that all of them had low-or submicromolar EC50 values against this virus in LLC-MK2 cells but were largely inactive in CaCo2 cells (not shown). Furthermore, 11r and all other compounds that we synthesized are inactive (EC50 > 87 µM) against CVB3 in Vero cells (not shown), but exhibit good to excellent activities against the same virus in Huh-T7 cells. We have previously observed similar poor antiviral activities in Vero cells not only for aketoamides, but also for Michael acceptors (Zhu et al., unpublished work) . A similar cell-type dependence is seen for the antiviral activity of 11r against MERS-CoV and SARS-CoV. Whereas the inhibitor exhibits excellent activity against MERS-CoV when Huh7 cells are the host cells (400 pM), the inhibitory activity is weaker by a factor of up to 12,500 when Vero cells are used (EC50 = 5 µM). On the other hand, 11r exhibits excellent anti-MERS-CoV activity in human Calu3 lung cells, i.e. in the primary target cells where the compound will have to act in a therapeutic setting (A. Kupke, personal communication). As we tested antiviral activity against SARS-CoV exclusively in Vero cells, the EC50 values determined for our compounds against this virus are in the one-digit micromolar range or higher; the best is again compound 11r with EC50 = 2.1 µM. Interestingly, the relatively weaker activity (or even inactivity) of our inhibitors against RNA viruses in Vero cells was observed independently in the virology laboratories in Leuven and in Leiden. It is thus unlikely that the lack of activity in Vero cells is related to problems with the experimental set-up. In preliminary experiments, we replaced the P3 cinnamoyl group of 11r by the fluorophor coumaryl and found by fluorescence microscopy that much more inhibitor appeared to accumulate in Huh7 cells compared to Vero cells (D.L., R.H. & Irina Majoul, unpublished).\\r\\n Regardless of which cell system is the most suitable one for the testing of peptidomimetic antiviral compounds, we next plan to test 11r in small-animal models for MERS and for Coxsackievirusinduced pancreatitis. In parallel, we aim to refine the experiments to quantify the accumulation of peptidomimetic protease inhibitors in different host-cell types, in the hope to find an explanation for the observed cell-type dependencies.\\r\\n This work demonstrates the power of structure-based approaches in the design of broad-spectrum antiviral compounds with roughly equipotent activity against coronaviruses and enteroviruses. We observed a good correlation between the inhibitory activity of the designed compounds against the isolated proteases, in viral replicons, and in virus-infected Huh7 cells. One of the compounds (11r) exhibits excellent anti-MERS-CoV activity in virus-infected Huh7 cells. Because of the high similarity between the main proteases of SARS-CoV and the novel BetaCoV/Wuhan/2019, we expect 11r to exhibit good antiviral activity against the new coronavirus as well.\\r\\n Crystallization. The recombinant production and purification of SARS-CoV M pro with authentic N and C termini was described in detail previously. 48, 49 Using an Amicon YM10 membrane (EMD Millipore), the purified SARS-CoV M pro was concentrated to 21 mg·mL -1 in buffer A (20 mM Tris-HCl, 150 mM NaCl, 1 mM DTT, 1 mM EDTA, pH 7.5). Crystallization was performed by equilibrating 1 µL protein (mixed with 1 µL precipitant solution) against 500 µl reservoir containing 6 -8% polyethylene glycol (PEG) 6,000, 0.1 M MES (pH 6.0), at 20℃ using the vapor diffusion sitting-drop method. Compounds 11a and 11s were dissolved in 100% DMSO at 50 mM and 200 mM stock concentrations, respectively. A crystal of the free enzyme was soaked in cryo-protectant buffer containing 20% MPD, 6% PEG 6,000, 0.1 M MES, 7.5 mM 11a, pH 6.0, for 2 h at 20℃. Another set of free enzyme crystals was soaked in another cryoprotectant buffer with 6% PEG 6,000, 5% MPD, 0.1 M MES, 15% glycerol, 10 mM 11s, pH 6.0, for 2 h. Subsequently, crystals were fished and flash-cooled in liquid nitrogen prior to data collection.\\r\\n Crystals of HCoV-NL63 M pro with 11a were obtained using cocrystallization. The concentrated HCoV-NL63 M pro (45 mg·mL -1 ) was incubated with 5 mM 11a for 4 h at 20℃, followed by setting up crystallization using the vapor diffusion sitting-drop method at 20℃ with equilibration of 1 µL protein (mixed with 1 µL mother liquor) against 500 µL reservoir composed of 0.1 M lithium sulfate monohydrate, 0.1 M sodium citrate tribasic dihydrate, 25% PEG 1,000, pH 6.0. The crystals were protected by a cryo-buffer containing 0.1 M lithium sulfate monohydrate, 0.1 M sodium citrate tribasic dihydrate, 25% PEG 1,000, 15% glycerol, 2 mM 11a, pH 6.0 and flash-cooled in liquid nitrogen.\\r\\n Crystals of HCoV-NL63 M pro with 11n or 11f were generated by using the soaking method. Several free-enzyme crystals were soaked in cryo-protectant buffer containing 0.1 M lithium sulfate monohydrate, 0.1 M sodium citrate tribasic dihydrate, 25% PEG 1,000, 15% glycerol, 5 mM 11n (or 11f), pH 6.0. Subsequently, the soaked crystals were flash-cooled in liquid nitrogen.\\r\\n Freshly prepared CVB3 3C pro at a concentration of 21.8 mg·mL -1 was incubated with 5 mM 11a pre-dissolved in 100% DMSO at room tempature for 1 h. Some white precipitate appeared in the mixture. Afterwards, the sample was centrifuged at 13,000 x g for 20 min at 4 ℃. The supernatant was subjected to crystallization trials using the following, commercially available kits: Sigma TM (Sigma-Aldrich), Index TM , and PEG Rx TM (Hampton Research). Single rod-like crystals were detected both from the Index TM screen, under the condition of 0.1 M MgCl2 hexahydrate, 0.1 M Bis-Tris, 25% PEG 3,350, pH 5.5, and from the Sigma TM screen at 0.2 M Li2SO4, 0.1 M Tris-HCl, 30% PEG 4,000, pH 8.5. Crystal optimization was performed by using the vapor-diffusion sitting-drop method, with 1 μL CVB3 3C pro -inhibitor complex mixed with 1 μL precipitant solution, and equilibration against 500 μL reservoir containing 0.1 M Tris-HCl, 0.2 M MgCl2, pH 8.5, and PEG 3,350 varied from 22% to 27%. Another optimization screen was also performed against a different reservoir, 0.1 M Tris-HCl, 0.2 M MgCl2, pH range from 7.5 to 8.5, and PEG 4,000 varied from 24% to 34%. Crystals were fished from different drops and protected by cryo-protectant solution consisting of the mother liquor and 10% glycerol. Subsequently, the crystals were flash-cooled with liquid nitrogen.\\r\\n Diffraction data collection, structure elucidation and refinement. Diffraction data from the crystal of the SARS-CoV M pro in complex with 11a were collected at 100 K at synchrotron beamline PXI-X06SA (PSI, Villigen, Switzerland) using a Pilatus 6M detector (DECTRIS). A diffraction data set from the SARS-CoV M pro crystal with compound 11s was collected at 100 K at beamline P11 of PETRA III (DESY, Hamburg, Germany), using the same type of detector. All diffraction data sets of HCoV-NL63 M pro complex structures and of the complex of CVB3 3C pro with 11a were collected at synchrotron beamline BL14.2 of BESSY (Berlin, Germany), using an MX225 CCD detector (Rayonics). All data sets were processed by the program XDSAPP and scaled by SCALA from the CCP4 suite. [50] [51] [52] The structure of SARS-CoV M pro with 11a was determined by molecular replacement with the structure of the complex between SARS-CoV M pro and SG85 (PDB entry 3TNT; Zhu et al., unpublished) as search model, employing the MOLREP program (also from the CCP4 suite). 52, 53 The complex structures of HCoV-NL63 M pro with 11a, 11f, and 11n were also determined with MOLREP, using as a search model the structure of the free enzyme determined by us (LZ et al., unpublished) . The complex structure between CVB3 3C pro and 11a was determined based on the search model of the free-enzyme structure (PDB entry 3ZYD; Tan et al., unpublished) . Geometric restraints for the compounds 11a, 11f, 11n, and 11s were generated by using JLIGAND 52, 54 and built into Fo-Fc difference density using the COOT software. 55 Refinement of the structures was performed with REFMAC version 5.8.0131 (ref. 52, 56, 57) .\\r\\n A buffer containing 20 mM Tris-HCl, 100 mM NaCl, 1 mM EDTA, 1 mM DTT, pH 7.3, was used for all the enzymatic assays. Two substrates with the cleavage sites of M pro and 3C pro , respectively (indicated by the arrow, ↓), Dabcyl-KTSAVLQ¯SGFRKM-E(Edans)-NH2 and Dabcyl-KEALFQ¯GPPQF-E(Edans)-NH2 (95% purity; Biosyntan), were employed in the fluorescence resonance energy transfer (FRET)based cleavage assay, using a 96-well microtiter plate. The dequenching of the Edans fluorescence due to the cleavage of the substrate as catalyzed by the proteases was monitored at 460 nm with excitation at 360 nm, using a Flx800 fluorescence spectrophotometer (BioTek). Curves of relative fluorescence units (RFU) against substrate concentration were linear for all substrates up to beyond 50 µM, indicating a minimal influence of the inner-filter effect. Stock solutions of the compounds were prepared by dissolving them in 100% DMSO. The UV absorption of 11a was found to be negligible at l = 360 nm, so that no interference with the FRET signal through the inner-filter effect was to be expected. For the determination of the IC50, different proteases at a specified final concentration (0.5 µM SARS-CoV or HCoV-NL63 M pro , 2 µM CVB3 3C pro , 3 µM EV-A71 3C pro ) were separately incubated with the inhibitor at various concentrations (0 to 100 μM) in reaction buffer at 37℃ for 10 min. Afterwards, the reaction was initiated by adding FRET peptide substrate at 20 μM final concentration (final volume: 50 μl). The IC50 value was determined by using the GraphPad Prism 6.0 software (GraphPad). Measurements of enzymatic activity were performed in triplicate and are presented as the mean ± standard deviations (SD).\\r\\n Assessment of inhibitory activity of a-ketoamides using viral replicons and virus-infected cells Cells and viruses. Hepatocellular carcinoma cells (Huh7; ref. 58) and their derivative constitutively expressing T7 RNA polymerase (Huh-T7; ref. 59) were grown in Dulbecco's modified minimal essential medium (DMEM) supplemented with 2 mM glutamine, 100 U·mL -1 penicillin, 100 µg·mL -1 streptomycin sulfate, and fetal calf serum (10% in growth medium and 2% in maintenance medium). Huh-T7 cells were additionally supplemented with geneticin (G-418 sulfate, 400 µg·mL -1 ). Huh-T7 cells were used for the enteroviral replicons as well as for infection experiments with CVB3 strain Nancy.\\r\\n For enterovirus (except CVB3) infection experiments, human rhabdomyosarcoma cells (RD; for EV-A71; BRCR strain) and HeLa Rh cells (for EV-D68 and human rhinoviruses) were grown in MEM Rega 3 medium supplemented with 1% sodium bicarbonate, 1% Lglutamine, and fetal calf serum (10% in growth medium and 2% in maintenance medium). For HCoV-229E (a kind gift from Volker Thiel (Bern, Switzerland)), culture and infection experiments were carried out as described. 60 For MERS-CoV or SARS-CoV infection experiments, Vero, Vero E6, and Huh7 cells were cultured as described previously. 61, 62 Infection of Vero and Huh7 cells with MERS-CoV (strain EMC/2012) and SARS-CoV infection of Vero E6 cells (strain Frankfurt-1) at low multiplicity of infection (MOI) were done as described before. 61, 63 All work with live MERS-CoV and SARS-CoV was performed inside biosafety cabinets in biosafety level-3 facilities at Leiden University Medical Center, The Netherlands. Viral replicons. The DNA-launched SARS-CoV replicon harbouring Renilla luciferase as reporter directly downstream of the SARS-CoV replicase polyprotein-coding sequence (pp1a, pp1ab, Urbani strain, acc. AY278741), in the context of a bacterial artificial chromosome (BAC) under the control of the CMV promoter, has been described previously (pBAC-REP-RLuc). 33 Apart from the replicase polyprotein, the replicon encodes the following features: the 5'-and 3'-non-translated regions (NTR), a ribozyme (Rz), the bovine growth hormone sequence, and structural protein N.\\r\\n Subgenomic replicons of CVB3 (pT7-CVB3-FLuc 34 ) and EV-A71 (pT7-EV71-RLuc) harbouring T7-controlled complete viral genomes, in which the P1 capsid-coding sequence was replaced by the Firefly (Photinus pyralis) or Renilla (Renilla renifor) luciferase gene, were generously provided by F. van Kuppeveld and B. Zhang, respectively. To prepare CVB3 and EV-A71 replicon RNA transcripts, plasmid DNAs were linearized by digestion with SalI or HindIII (New England Biolabs), respectively. Copy RNA transcripts were synthesized in vitro using linearized DNA templates, T7 RNA polymerase, and the T7 RiboMax™ Large-Scale RNA Production System (Promega) according to the manufacturer's recommendations.\\r\\n Transfection. Huh-T7 cells grown in 12-well plates to a confluency of 80% -90% (2 -3 x 10 5 cells/well) were washed with 1 mL OptiMEM (Invitrogen) and transfected with 0.25 µg of the replication-competent replicon and Lipofectamin2000 or X-tremeGENE9 in 300 µl OptiMEM (final volume) as recommended by the manufacturer (Invitrogen or Roche, respectively). The transfection mixtures were incubated at 37°C for 4 to 5 h (Lipofectamin2000) or overnight (X-tremeGENE9), prior to being replaced with growth medium containing the compound under investigation. For RNA-launched transfection of enteroviral replicons, DMRIE-C was used as transfection reagent according to the manufacturer's recommendations (Invitrogen). All experiments were done in triplicate or quadruplicate and the results are presented as mean values ± SD.\\r\\n Testing for inhibitory activity of candidate compounds. Initially, we performed a quick assessment of the inhibitory activity of the candidate compounds towards the enteroviral and coronaviral replicons at a concentration of 40 µM in Huh-T7 cells. Compounds that were relatively powerful and non-toxic at this concentration, were assayed in a dose-dependent manner to estimate their half-maximal effective concentration (EC50) as well as their cytotoxicity (CC50), as described. 29 In brief, different concentrations of a-ketoamides (40 µM in screening experiments or increasing concentration (0, 1.25, 2.5, 5, 10, 20, 40 µM) when determining the EC50) were added to growth medium of replicon-transfected Huh-T7 cells. Twenty-four hours later, the cells were washed with 1 mL phosphate-buffered saline (PBS or OPTIMEM, Invitrogen) and lysed in 0.15 mL Passive lysis buffer (Promega) at room temperature (RT) for 10 min. After freezing (-80 o C) and thawing (RT), the cell debris was removed by centrifugation (16,000 x g, 1 min) and the supernatant (10 or 20 µl) was assayed for Firefly or Renilla luciferase activity (Promega or Biotrend Chemikalien) using an Anthos Lucy-3 luminescence plate reader (Anthos Microsystem).\\r\\n The antiviral activity of the compounds was evaluated in a cytopathic effect (CPE) readout assay using MTS [3-(4,5-dimethylthiazol-2-yl)-5-(3carboxymethoxyphenol)-2-(4-sulfophenyl)-2H-tetrazolium, innersalt]-based assay. Briefly, 24 h prior to infection, cells were seeded in 96-well plates at a density of 2.5 x 10 4 (RD cells) or of 1.7 x 10 4 (HeLa Rh) per well in medium supplemented with 2% FCS. For HRV2 and HRV14 infection, the medium contained 30 mM MgCl2. The next day, serial dilutions of the compounds and virus inoculum were added. The read-out was performed 3 days post infection as follows: The medium was removed and 100 μl of 5% MTS in Phenol Red-free MEM was added to each well. Plates were incubated for 1 h at 37°C, then the optical density at 498 nm (OD498) of each well was measured by microtiter plate reader (Saffire 2 , Tecan). The OD values were converted to percentage of controls and the EC50 was calculated by logarithmic interpolation as the concentration of compound that results in a 50% protective effect against virus-induced CPE. For each condition, cell morphology was also evaluated microscopically.\\r\\n Assays with MERS-CoV and SARS-CoV were performed as previously described. 61, 63 In brief, Huh7, Vero, or Vero E6 cells were seeded in 96-well plates at a density of 1 × 10 4 (Huh7 and Vero E6) or 2 × 10 4 cells (Vero) per well. After overnight growth, cells were treated with the indicated compound concentrations or DMSO (solvent control) and infected with an MOI of 0.005 (final volume 150 µl/well in Eagle's minimal essential medium (EMEM) containing 2% FCS, 2 mM L-glutamine, and antibiotics). Huh7 cells were incubated for two days and Vero/VeroE6 cells for three days, and differences in cell viability caused by virus-induced CPE or by compound-specific side effects were analyzed using the CellTiter 96 AQueous Non-Radioactive Cell Proliferation Assay (Promega), according to the manufacturer's instructions. Absorbance at 490 nm (A490) was measured using a Berthold Mithras LB 940 96-well plate reader (Berthold). Cytotoxic effects caused by compound treatment alone were monitored in parallel plates containing mock-infected cells. HCoV-229E-Rev: 5′-ggTCGTTTAGTTGAGAAAAGT -3′, and 229E-ZNA probe: 5'-6-Fam-AGA (pdC)TT(pdU)G(pdU)GT(pdC)TA(pdC)T-ZNA-3-BHQ-1 -3' (Metabion). Standard curves were prepared using serial dilutions of RNA isolated from virus stock. Data were analysed using GraphPad Prism 5.0; EC50 values were calculated based on a 4parameter logistic statistics equation. In parallel to the qPCR assays with inhibitors, cell viability assays were performed using Alamar-Blue™ Cell Viability Reagent (ThermoFisher) according to the manufacturer's instruction. CC50 values were calculated using inhibitor versus normalized response statistics equation by including proper controls (no inhibitor and 1% Triton-X-100-treated cells).\\r\\n Determination of the cell toxicity of candidate compounds. The CellTiter 96Aqueous One Solution Cell Proliferation Assay (MTS test, Promega), the CellTiter Glo assay kit (Promega), the Non-Destructive Cytotoxicity Bio-Assay (ToxiLight (measuring the release of adenylate kinase from damaged cells), Lonza Rockland), or the AlamarBlue™ Cell Viability Reagent (ThermoFisher) were used to determine the cytotoxic effect of compounds towards host cells according to the manufacturers' recommendations. 29, 65 Chemical synthesis of a-ketoamides General procedure. Reagents were purchased from commercial sources and used without purification. HSGF 254 (0.15 -0.2 mm thickness) was used for analytical thin-layer chromatography (TLC). All products were characterized by their NMR and MS spectra. 1 H NMR spectra were recorded on 300-MHz, 400-MHz, or 500-MHz instruments. Chemical shifts were reported in parts per million (ppm, δ) down-field from tetramethylsilane. Proton coupling patterns were described as singlet (s), doublet (d), triplet (t), quartet (q), multiplet (m), and broad (br). Mass spectra were recorded using a Bruker ESI ion-trap HCT Ultra. HPLC spectra were recorded by LC20A or LC10A (Shimadzu Corporation) with Shim-pack GIST C18 (5 µm, 4.6x150mm) with three solvent systems (methanol/water, methanol/ 0.1% HCOOH in water or methanol/0.1% ammonia in water). Purity was determined by reversed-phase HPLC and was ≥95% for all compounds tested biologically.\\r\\n Synthesis of (2S,4R)-dimethyl 2-(tert-butoxycarbonylamino)-4-(cyanomethyl) pentanedioate (1).\\r\\n To a solution of N-Boc-L-glutamic acid dimethyl ester (6.0 g, 21.8 mmol) in THF (60 mL) was added dropwise a solution of lithium bis(trimethylsilyl)amide (LHMDS) in THF (47 mL, 1 M) at −78°C under nitrogen. The resulting dark mixture was stirred at −78°C. Meanwhile, bromoacetonitrile (1.62 mL, 23.3 mmol) was added dropwise to the dianion solution over a period of 1 h while keeping the temperature below −70°C. The reaction mixture was stirred at −78°C for additional 2 h. After the consumption of the reactant was confirmed by TLC analysis, the reaction was quenched by methanol (3 mL), and acetic acid (3 mL) in precooled THF (20 mL) was added.\\r\\n After stirring for 30 min, the cooling bath was removed. The reaction mixture was allowed to warm up to room temperature and then poured into brine (40 mL). The organic layer was concentrated and purified by flash column chromatography (petroleum ether/ethyl acetate = 4/1) to give product 1 (4.92 g, 72%) as colorless oil. 1 H NMR (CDCl3, 400 MHz) δ 5.23 (1H, d, J = 9.0 Hz), 4.43-4.36 (1H, m), 3.77(1H, s), 3.76 (1H, s), 2.89-2.69 (3H, m), 2.20-2.14 (2H, m), 1.45 (9H, s). ESI-MS (m/z): 315 (M + H) + .\\r\\n Synthesis of (S)-methyl 2-(tert-butoxycarbonylamino)-3-((S)-2oxopyrrolidin-3-yl)propanoate (2) .\\r\\n In a hydrogenation flask was placed compound 1 (4.0 g, 12.7 mmol), 5 mL of chloroform and 60 mL of methanol before the addition of PtO2. The resulting mixture was stirred under hydrogen at 20 o C for 12 h. Then the mixture was filtered over Celite to remove the catalyst. NaOAc (6.77 g, 25.5 mmol) was added to the filtrate before the resulting mixture was stirred at 60 o C for 12 h. The reaction was quenched with water (30 mL). The suspension was extracted with ethyl acetate. The organic layers were combined, dried (MgSO4), and filtered. The light-brown filtrate was concentrated and purified by silica gel column chromatography (petroleum ether/ethyl acetate = 4/1) to give the product 2 (2.20 g, 61%) as white solid. 1 H NMR (CDCl3) δ 6.02 (1H, br), 5. Synthesis of (S)-methyl 2-amino-3-((S)-2-oxopyrrolidin-3yl)propanoate (3).\\r\\n Compound 2 (1.0 g, 3.5 mmol) was dissolved in 10 mL dichloromethane (DCM), then 10 mL trifluoroacetic acid (TFA) was added. The reaction mixture was stirred at 20 o C for 0.5 h, and concentrated in vacuo to get a colorless oil, which could be used for the following step without purification.\\r\\n ESI-MS (m/z): 187 (M + H) + .\\r\\n Synthesis of methyl N-substituted amino-acid esters 4 General procedure. The methyl amino-acid ester hydrochloride (6.0 mmol) was dissolved in 20 mL CH2Cl2, and then acyl chloride (6.0 mmol), triethylamine (1.69 mL, 12.0 mmol) were added, before the reaction was stirred for 2 h at 20 o C. The reaction mixture was diluted with 20 mL CH2Cl2, washed with 50 mL of saturated brine (2 × 25 mL), and dried over Na2SO4. The solvent was evaporated and the product 4 was obtained as white solid (70-95% yield), which could be used for the next step without further purification.\\r\\n (S)-methyl 2-cinnamamido-3-phenylpropanoate (4a). The methyl L-phenylalaninate hydrochloride (1.30 g, 6.0 mmol) was dissolved in 20 mL CH2Cl2, and then cinnamoyl chloride (1.00 g, 6.0 mmol), triethylamine (1.69 mL, 12.0 mmol) were added, before the reaction was stirred for 2 h at room temperature. The reaction mixture was diluted with 20 mL CH2Cl2, washed with 50 mL of saturated brine (2×25 mL), and dried over Na2SO4. The solvent was evaporated and the product 4a was obtained as white solid (1.75 g, 95%), which could be used for the next step without further purification.\\r\\n Synthesis of N-substituted amino acids 5 (general procedure). 1 M NaOH (5 mL) was added to a solution of compound 4 (3.0 mmol) in methanol (5 mL). The reaction was stirred for 20 min at 20 o C. Then 1 M HCl was added to the reaction solution until pH = 1. Then the reaction mixture was extracted with 100 mL of CH2Cl2 (2 × 50 mL) and the organic layer was washed with 50 mL of brine and dried over Na2SO4. The solvent was evaporated and the crude material purified on silica, eluted with mixtures of CH2Cl2/MeOH (20/1) to afford the product 5 (90-96% yield) as a white solid.\\r\\n Synthesis of compounds 6 (General procedure). Compound 5 (2.7 mmol) was dissolved in 10 mL of dry CH2Cl2. To this solution, 1.5 equiv (1.54 g) of 1-[bis(dimethylamino)methylene]-1H-1,2,3triazolo[4,5-b]pyridinium 3-oxide hexafluorophosphate (HATU) was added, and the reaction was stirred for 0.5 h at 20 o C. Then compound 3 (500 mg, 2.7 mmol) and TEA (0.70 mL, 5.42 mmol) was added to the reaction. The reaction was stirred for another 6 h. The reaction mixture was poured into 10 mL water. The aqueous solution was extracted with 50 mL of CH2Cl2 (2 × 25 mL) and washed with 50 mL of saturated brine (2 × 25 mL) and dried over Na2SO4. The solvent was evaporated and the crude material purified on silica, eluted with a mixture of CH2Cl2/MeOH (40/1) to give the product 6 (62-84% yield).\\r\\n Synthesis of alcohols 7 (general procedure). Compound 6 (1.1 mmol) was dissolved in methanol (40 mL), then NaBH4 (0.34 g, 8.8 mmol) was added under ambient conditions. The reaction mixture was stirred at 20 o C for 2 h. Then the reaction was quenched with water (30 mL). The suspension was extracted with ethyl acetate. The organic layers were combined, dried, and filtered. The filtrate was evaporated to dryness and could be used for the next step without further purification (46-85% yield).\\r\\n Synthesis of aldehydes 8 (general procedure). Compound 7 (0.75 mmol) was dissolved in CH2Cl2, then Dess-Martin periodinane (337 mg, 0.79 mmol) and NaHCO3 (66 mg, 0.79 mmol) were added. The resulting mixture was stirred at 20 o C for 1 h. The mixture was concentrated and purified by column chromatography on silica gel (CH2Cl2/MeOH = 20/1) to give the product 8 as white solid (88-95% yield).\\r\\n Synthesis of compounds 9 (general procedure). Compound 8 (0.40 mmol) was dissolved in CH2Cl2, and then acetic acid (0.028 g, 0.47 mmol) and isocyanide (0.43 mmol) were added successively to the solution. The reaction was stirred at 20 o C for 24 h. Then the solvent was evaporated and the crude material purified on silica, eluted with a mixture of CH2Cl2/MeOH (20/1) to give the product 9 (46-84%).\\r\\n Synthesis of α-hydroxyamides 10 (general procedure). 1 M NaOH (0.5 mL) was added to a solution of compound 9 (0.164 mmol) in methanol (5 mL). The reaction was stirred at 20 o C for 0.5 h until the consumption of compound 9 was confirmed by TLC analysis. Then, 1 M HCl was added to the reaction solution until pH = 7. Following this, the solvent was evaporated to generate the product 10 as white solid, which could be used directly in the next step.\\r\\n Synthesis of α-ketoamides 11 (general procedure). Compound 10 was dissolved in CH2Cl2, then Dess-Martin periodinane (74 mg, 0.176 mmol) and NaHCO3 (30 mg, 0.176 mmol) were added. The resulting mixture was stirred at 20 o C for 1 h. The mixture was concentrated and purified by column chromatography on silica gel (CH2Cl2/MeOH = 20/1) to give the a-ketoamides 11 as light yellow solid (52-79% in two steps). \\r\\n Detailed results of the variation of the P1' and P3 substituents Synthesis of a-ketoamides 11b -11e and 11g -11l Supplemental Table 1 : inhibitory activities (IC50 (μM)) of α-ketoamides with P1' and P3 modifications against viral proteases Supplemental Table 2 : crystallographic data for complexes between viral proteases and α-ketoamides Molecular formula strings and biological data (CSV)\\r\\n Authors will release the atomic coordinates and experimental data upon article publication. Atomic coordinates include SARS-CoV M pro in complex with compounds 11a (5N19), 11s (5N5O), HCoV-NL63 M pro in complex with 11a (6FV2), 11n (6FV1), 11f (5NH0), and CVB3 3C pro in complex with (5NFS). 1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.10.936898 doi: bioRxiv preprint  1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.10.936898 doi: bioRxiv preprint. Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) or 2019-Novel Coronavirus (2019-nCoV) is the pathogen of Corona Virus Disease 2019 [1] . Nucleic acid detection of 2019-nCoV is one of the key indicators for clinical diagnosis. However, the false negative rate of 2019-nCoV nucleic acid in clinical practice is very high, and the positive rate is only 30-50% [2] . False negative detection means missed detection, which will lead to not only the delayed diagnosis of suspected patients, but also the 2019-nCoV carriers becoming the potential sources of virus infection. Therefore, it is very urgent to improve the detection rate of 2019-nCoV nucleic acid clinically.\\r\\n Currently, fluorescent quantitative RT-PCR (qRT-PCR) technology is mainly used to detect the nucleic acid of 2019-nCoV. The quality of nucleic acid template is admittedly one of the key factors affecting the detection efficiency. According to \\\"The Laboratory Technical Guidelines for Detection 2019-nCoV (Fourth Edition)\\\" issued by National Health and Commission of China [3] and \\\"The Experts' Consensus on Nucleic Acid Detection of 2019-nCoV\\\" released by Chinese Society of Laboratory Medicine [4] , the human samples must be placed in 56 o C for at least 45 minutes or higher for shorter time to inactivate the viruses in order to keep the inspectors from virus infection before the nucleic acids were isolated.\\r\\n However, 2019-nCoV is a single stranded RNA virus. Theoretically, the progress of killing the viruses with high temperature will damage the integrity of their RNA genomes in the samples, thus improperly reducing the amount of target viral templates, and eventually leading to high false negative rate of the virus detection. In this study, we employed porcine epidemic diarrhea virus (PEDV), a kind of coronavirus, as a model virus to test the above hypothesis and our results demonstrated that the virus inactivation progress by high temperature damaged the detectable template of the coronavirus seriously. Biological. Company, Ltd.) or 1 ng λDNA (3010, Takara, Japan) were suspended and mixed together using 1.5 mL of sample preservation medium (R503, Vamzye, China) or 1.5 mL Hank's solution.\\r\\n Nucleic acid (DNA/RNA) was extracted using a commercial kit (RC311-C1, Vazyme) according to the manufacturer's protocol. Briefly, 500 µL samples preserved in R503 was first added into a 1.5 ml EP tube containing 200 µL of absolute ethanol and 200 µL samples preserved in Hank's solution was added into a 1.5 ml EP tube containing 500 µL lysis buffer.\\r\\n The mixture was vortexed thoroughly and then transferred into a column placed onto a 2 mL EP collection tube. After centrifuge for 1 min at 12,000×g, the column was placed onto a fresh 2 mL EP collection tube, followed by adding 600 µL washing buffer and then centrifuge for 30 sec at 12,000×g. Washing step was repeated once. The column was placed onto a new 2 mL collection. After centrifuge for 2 min at 12,000×g, the column was then placed onto a 1.5 ml collection tube. 100 µL and 40 µL elution buffer was added into the center of the membrane of column collected the sample prepared by R503 and Hank's solution respectively and then centrifuged for 1 min at 12,000 × g to collect the nucleic acid. The nucleic acid was stored at -30 ~ -15 o C for usage within short time and at -70 o C for long storage 2.3 Agarose gel electrophoresis 5 µL of the extracted nucleic acids, 1 µL of 10 × DNA loading buffer (P022, Vazyme) and 4 µL of nuclease-free H 2 O were mixed and then vortexed briefly. The mixture was heated in PCR instrument to denature the nucleic acid for 2 min and then subjected to 1.2% agarose gel electrophoresis separation on ice for 7-10 min together with the standard DNA marker (MD103, Vazyme).\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint\\r\\n Quantitative Real-Time PCR (qRT-PCR) was performed to detect the PEDV RNA and λ phage DNA using HiScript II One Step qRT-PCR Probe Kit (Q222-CN, Vazyme) according to the manufacturer's protocol. The reaction (25 µL) was assembled as follows: 12.5 µL of 2 × One Step U+ Mix, 1.25 µL of One Step U+ Enzyme Mix, 0.5 µL of 50 × ROX Reference Dye 1, 0.5 µL of Gene Specific Primer Forward (10 µM), 0.5 µL of Gene Specific Primer Reverse (10 µM), 0.25 µL of TaqMan Probe (10 µM), and 4.5 µL nuclease free H2O.\\r\\n The reaction was carried out on ABI StepOnePlusTM using following conditions: \\r\\n Statistical significance was determined using the paired two-tailed t test. A value of p < 0.05 was considered statistically significant, and p < 0.01 was considered statistically very significant. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint 6 / 16 into five groups. Three groups were incubated at 56 o C for 30, 45 and 60 minutes, respectively. The fourth was incubated at 92 o C for 5 minutes, and the fifth group was stored at 4 o C as no inactivation control. After the high-temperature inactivation was completed, nucleic acids (including DNA and RNA) were extracted by conventional methods, and subjected to run 1.2% agarose gel electrophoresis. \\r\\n To understand the effect of the virus inactivation progress with the temperature higher than 56 o C on the integrity of viral nucleic acid, we quantitated the amount of Table S1 ). The results suggest that only 50.11% of the detectable viral templates left after the inactivation progress of incubation at 56 o C for 30 minutes and only 3.36% of the detectable viral templates left after the inactivation progress of incubation at 92 o C for 5 minutes.\\r\\n To investigate whether any solutions are able to protect viral RNA integrity from author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint 7 / 16 disrupting during the inactivation progress with high temperature, R503 (Vazyme, China) was used to prepare the samples instead of Hank's solution. We then performed the same viral inactivation experiments as using Hank's solution.\\r\\n Compared with the control group that the sample was stored at 4 o C, both DNA and RNA in the samples incubated at 56 o C for 30 -60 minutes also displayed obvious degradation, among which the 28 S and 18 S bands of total RNA of human cells were obviously smeared, and the large bands of genomic DNA became weaker while there were almost no visible genomic DNA and 28 S RNA bands in the sample incubated at 92 o C for 5 minutes (Fig 1D) . However, the degradation of nucleic acid during the inactivation progress was not as serious as that of the samples prepared by Hank's solution. The results suggest that R503 should have a significant protective effect of both total RNA and genomic DNA from degradation.\\r\\n To determine the effect of the inactivation progress with temperature higher than 56 o C on the integrity of viral nucleic acid in the samples prepared with R503, we examined the amount of porcine PEDV coronavirus by qRT-PCR as above.\\r\\n Compared with the CT (31.64 ± 0.10) of the control group that the sample was stored at 4 o C, the mean value of CT (31.31 ± 0.17) of the group incubated at 56 o C for 30 minutes had no significant change. The difference of CTs between the two groups was less than 0.5 ( Fig. 1E ; Figure S1B ; Table S1 ). Moreover, the mean value of CT (31.43 ± 0.08) in the group incubated at 92 o C for 5 minutes was also not changed significantly. The difference of CTs between the two groups was less than 0.5 either (Fig. 1F; Fig. S1B ; Table S1 ). These results showed that the inactivation progress with the temperature higher than 56 o C did not significantly affect the detectable template of viral nucleic acid when it was prepared with R503, suggesting that the fragmented viral RNA could still be good for using as a template for qRT-PCR even if the viral nucleic acid was degraded.\\r\\n In order to understand the advantage of R503 over common isotonic solution in author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint\\r\\n protecting the detectable amount of coronavirus RNA from disrupting, we first compared the difference of the detectable amount of coronavirus RNA between the two samples that are prepared with R503 and Hank's solution and stored at 4 o C. The results showed that the detectable amount of the viral RNA in the two prepared solutions was similar (Fig. 1G, Table S1 ), and the difference between the mean values of their CT was less than 0.5 (31.64 ± 0.10 vs 31.95 ± 0.10). The results suggest that there is no significant difference in the amount of detectable viral RNA templates between the samples prepared by either R503 or Hank's solutions that were stored at Table S1 ). The results suggest that the detectable templates of viruses in the samples prepared by Hank's solution was only 32.30% of R503 solution after the samples were inactivated at 56 o C for 30 minutes to kill the virus. Furthermore, after inactivation at 92 o C for 5 minutes, the CT of detected PEDV virus in the two preservation solutions were 31.43 ± 0.08 (R503) and 36.84 ± 0.37 (Hank's), respectively. The mean value of CT of detected virus in the samples prepared by Hank's solution was 5.4125 higher than that prepared by R503 (P < 0.0001; Figure 1I , Table S1 ). The results reveal the detectable amount of virus in the samples prepared by Hank's solution was only 2.35% of R503 solution after the samples were inactivated at 92 o C for 5 minutes to kill the virus.\\r\\n In addition to the disruptive effect of the virus inactivation with high temperature on the detection of RNA virus, the detectable amount of the viral DNA (λ phage) in the samples was also significantly affected. qPCR results (Fig. S1C, Table S1) showed that mean value of CT (23.55 ± 0.20) of detectable virus in the samples author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint 9 / 16 incubated at 56 o C for 30 minutes was increased by 0.5230 (P < 0.01) compared with the CT (23.02 ± 0.15) of the samples stored at 4 o C. The mean value of CT (25.58 ± 0.21) was increased by 2.5572 (P < 0.0001) in the sample incubated at 92 o C for 5 minutes.\\r\\n Furthermore, we examined the amount of phage DNA in the sample prepared with R503 after high temperature treatment by qPCR. The results (Fig. S1D, Table S1) showed that the mean value of CT (21.71 ± 0.04) in the sample prepared by R503 and incubated at 56 o C for 30 minutes had no significant change compared with that in the control group (21.61 ± 0.06) (the difference between the two was less than 0.5). The mean value of CT (22.90 ± 0.26) was increased by 1.2880 in the sample that was inactivated at 92 o C for 5 minutes, suggesting that 92 o C treatment significantly reduced the detectable amount of DNA viruses in sample even prepared by R503.\\r\\n In conclusion, our results show that virus inactivation with temperature higher than 56 o C result in the degradation of viral nucleic acids seriously, which will lead to the artificial shortage of detectable templates of viral nucleic acids in samples, and finally lead to false negative of clinical detection in some samples. Preserved with the optimized solution such as R503, the detectable templates of viral nucleic acids can be kept unchanged after the samples were incubated at 56 o C or higher for killing the viruses although the integrity of cellular nucleic acids were obviously disrupted. The results suggest that if the samples were prepared with an optimal solution, the detectable numbers of viral templates would be unchanged because virus inactivation progress with high temperature does not erase all the RNA but leave small fragments of nucleic acids that are good enough for using as the template for qRT-PCR detection.\\r\\n If it is true for 2019-nCoV, preparing the human samples with the optimal solution can not only keep the clinical inspectors from virus infection by inactivating the virus using 56 o C or higher as normal, but also improve the detection sensitivity of viral nucleic acid in the sample, leading to avoiding or at least reducing false negative author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint 10 / 16 results of detection.\\r\\n Recently, Chen reported that the inactivation by incubating the sample with 56 o C for 30 min to kill the virus had no significant effect on the detection of 2019-nCoV by qRT-PCR [5] . However, combining the limitations such as only two samples tested and higher amount of virus in the two samples as the authors discussed in their researches, and our experimental evidences that the virus inactivation by high temperature did reduce the detectable amount of the porcine coronavirus template seriously, it is highly recommended to carry out systematic investigation on the impact of high temperature inactivation on the integrity of 2019-nCoV nucleic acids and develop a sample preservation solution to protect the detectable templates of 2019-nCoV nucleic acids from high temperature inactivation damage. Table S1 . Figure S1 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint 15 / 16 The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint. HCoV-HKU1 replication in the presence of both inhibitors (Fig. 2B) . All inhibitors were used 151 at non-toxic concentrations (Fig. 2C) . 152 The experiments conducted so far suggested that KLK13 is required for virus infection. 153 However, one may question the specificity of the KLK13 protease inhibitors. To ensure that 154 KLK13 is indeed the priming enzyme during HCoV-HKU1 infection, we developed HAE 155 cultures by transforming cells with lentiviral vectors encoding shRNAs targeting KLK13 156 mRNA. We then confirmed that the expression of the protease was silenced (HAE_shKLK13). (Fig. 3A) . Importantly, HAE_shKLK13 cells continued to differentiate and formed 162 pseudostratified cultures (Fig. 3B) . Next, we infected HAE_ctrl, HAE_GFP, HAE_vector, and 163 HAE_shKLK13 with HCoV-HKU1 (10 6 RNA copies per ml) and incubated them for 2 h at 164 32°C with the viral stock solution. Cultures were maintained at 32°C for 5 days at an air-liquid 165 interface. Apical washes were collected, and virus yield was determined by RT-qPCR. We 166 found that, in contrast to that in control cultures, replication of virus in HAE_shKLK13 was 167 abolished (Fig. 3C) . Overall, these data suggest that silencing the KLK13 gene in HAE inhibits 168 virus infection, indicating that KLK13 is necessary for HCoV-HKU1 infection. 171 We determined that KLK13 is essential for efficient HCoV-HKU1 infection in HAE The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint proteases my render RD cells permissive, we generated RD cells expressing human KLK13 or 176 TMPRSS2 proteases. RD cells were transduced with lentiviral vectors harboring the KLK13 177 gene (RD_KLK13), control vector (RD_ctrl), or TMPRSS2 (RD_TMPRSS2). Due to the lack 178 of KLK13 specific antibodies, we verified its presence based on RT-PCR (Fig. 4A) . The 179 presence of TMPRSS2 in RD_TMPRSS2 cells was confirmed using Western blot. The 180 TMPRSS2 band in RD cells was observed at 25 kDa, which corresponds to one of the naturally 181 occurring splicing variants (Fig. 4B) . Subsequently, we transduced RD_ctrl, RD_KLK13 and pseudoviruses, compared to ΔEnv, which was completely abolished in the presence of KLK13 193 inhibitor (Fig 4D) . Overall, these data demonstrated that KLK13 activity drives HCoV-HKU1 194 entry into cells. 195 196 KLK13 enables the replication of HCoV-HKU1 in RD cells 197 Obtained results showed that KLK13 expression on RD cells was sufficient for HCoV-198 HKU1 pseudovirus entry. Here, we aimed to test whether KLK13 presence renders RD cells 199 permissive for HCoV-HKU1 infection. For this, we infected RD_ctrl and RD_KLK13 cells 200 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint with HCoV-HKU1 (10 8 RNA copies per ml) and incubated the culture for 7 days at 32°C in the 201 presence or absence of a KLK13 inhibitor (10 µM) or DMSO. Next, cellular RNA was isolated 202 and the presence of HCoV-HKU1 N subgenomic mRNA (N sg mRNA), which is considered 203 to be a hallmark of coronaviral infection, was assessed (14). sg mRNA appeared in RD_KLK13 204 cells, while no signal was detected in cultures supplemented with the KLK13 inhibitor nor in 205 RD_ctrl cells (Fig. 5A) . 206 To further confirm the role of KLK13 during HCoV-HKU1 infection RD HCoV-HKU1 infection was analyzed by means of N sg mRNA detection. Again, we found that 212 N sg mRNA was produced only in the presence of KLK13 (Fig. 5B) . Further, we passaged in the presence of KLK13 (Fig. 5B) . However, we observed no cytopathic effects (CPEs), and 218 replication levels were very low (no significant increase over control levels on RT-qPCR; data 219 not shown). To further test the effect of KLK13 on replication of HCoV-HKU1 in RD cells, the 220 virus stock was incubated with purified KLK13 (200 nM) and incubated in the presence or 221 absence of a KLK13 inhibitor (10 µM) or DMSO. After 2 h at 32°C pre-treated virus stock was 222 diluted in media as described above and overlaid on RD cells. After 7 days at 32°C we evaluated 223 the presence of the HCoV-HKU1 N sg mRNA. The virus replicated only after treatment with 224 KLK13, and supplementation with the inhibitor blocked this effect infection (Fig. 5C) . 225 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint KLK13 primes the HCoV-HKU1 S protein 227 Expression of KLK13 by cells previously resistant to HCoV-HKU1 renders them 228 susceptible; therefore, we asked whether this is due to proteolytic activation of the S protein. 229 We tested this using the CleavEx method, in which a peptide of interest is exposed in the KLK13. Subsequently, proteins were resolved by SDS-PAGE and detected by western blotting 238 with antibodies specific for His-tagged proteins. The analysis showed that, in the presence of 239 500 nM KLK13, the CleavEx protein harboring the S1/S2 cleavage site was degraded; however, 240 the CleavEx protein harboring the S2/S2' site remained intact. Because KLKs are produced as 241 pro-forms that undergo self-activation, an additional band of His-tagged purified pro-KLK13 242 (HisTag-pro-KLK13) was observed after treatment with 500 nM KLK13 (Fig. 6A) . The product 243 of the S1/S2 cleavage was further sequenced by N-terminal Edman degradation showing the 244 following sequence: R↓SISA, which corresponds to the S1/S2 site. This result shows clearly 245 that the S1/S2 region of the HCoV-HKU1 S protein is prone to KLK13-mediated cleavage. 246 Furthermore, we aimed to confirm the cleavage using a full-length Spike protein of HCoV-HKU1 (HKU1-S). For this, we expressed the HKU1-S in 293T cells, purified the protein 248 using 6 × His tag, and incubated for 3 h at 37°C with increasing concentrations of purified 249 KLK13. Subsequently, HKU-S or mock proteins were resolved by SDS-PAGE and detected by 250 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint western blotting with antibodies specific to the tag. The analysis showed that in the presence of 251 1 µM KLK13 the HKU1-S was degraded ( Figure 6B ). The S protein was observed at 252 ~150 kDa, which is consistent with the migration speed reported for these highly glycosylated 253 proteins (1). 254 255 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint\\r\\n Receptor recognition is the first, essential step of the virus infection process. The The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint concentrations(63). Broad spectrum protease inhibitors are now used widely for virus research, 281 although their non-specific activity makes the results equivocal. For example, Matsuyama et al. 282 showed recently that the furin inhibitor dec-RVKR-CMK interferes with the activity of several 283 proteases, and that its previously described inhibitory activity during MERS-CoV infection is 284 not specific to furin; instead, its activity is due to non-specific inhibition of cathepsin L and 285 TMPRSS2 (81). We tried to use specific KLK inhibitors developed in our laboratory (61).\\r\\n Considering the small arsenal of tools available to researchers studying KLKs, only three 287 compounds were readily available. Treating HAE cultures with these inhibitors revealed that 288 only compounds designed to inhibit KLK13 hampered HCoV-HKU1 replication. However, the 289 great similarity between different KLKs makes one doubt the specificity of these inhibitors, 290 despite their performance in biochemical assays. Therefore, we decided to silence KLK13 in 291 HAE cultures. This abolished virus replication ex vivo, thereby confirming the importance of 292 KLK13 during infection. KLK13 is thought to be secreted and membrane-bound(82, 83). 293 This study showed that HCoV-HKU1 infection in HAE modulates the expression of 294 different KLKs, including KLK13. In our study, KLKs expression was tested using semi-295 quantitative PCR, and for that reason, we were unable to show the level of KLKs modulation 296 after HCoV-HKU1. However, the pattern of virus-induced expression of several KLKs could 297 be observed. KLK mechanism of activation is a complex process and until now it has only 298 been proven that most KLK genes are regulated by steroids and other hormones (84). It is also 299 important to remember, that KLK expression is regulated in a similar manner, and the induction 300 of a single gene usually results in overexpression of the whole cluster (85, 86). While one may 301 assume that the virus stimulates KLK13 production to promote the infection, this up-regulation 302 is likely a natural response of the damaged tissue, as KLKs were previously reported to take 303 part also in tissue regeneration (87-89). Further, increased expression of KLKs may be the 304 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint response to the inflammatory process, as Seliga et al demonstrated that KLK-kinin system is a 305 potent modulator of innate immune responses (90).\\r\\n The experiments performed herein show the importance of KLK13 for virus entry into 307 susceptible cells; therefore, we speculated that scattered distribution of different KLKs in 308 different tissues may be one of the determinants of the HCoV-HKU1 tropism (53, 91). We 309 tested the purified enzyme expressed in the eukaryotic cells; however, we also developed a cell 310 line constitutively expressing the enzyme. As an in vitro model for our studies, we used RD 311 cells previously reported to carry attachment receptors for the virus (26). Here, using 312 pseudoviruses decorated with S-HKU1 proteins we showed that KLK13 presence on RD cells 313 is sufficient for virus entry and renders these cells permissive. Our experiments also showed 314 that in contrast to previous reports, TMPRSS2 is not involved in this process(51). Furthermore, 315 we observed that RD cells supported the replication of the virus in the presence of KLK13 and 316 that this effect was reversed in the presence of the specific KLK13 inhibitor. We were, however, 317 not able to culture the virus to high yields. HCoV-HKU1 replication in KLK13-expressing RD 318 cells remained inefficient and RTqPCR assessment did not reveal significant increases in the 319 amounts of viral RNA. For that reason, we are only able to detect viral sg mRNAs, which are 320 considered to be the hallmark of coronaviral replication.We believe that this may be due to 321 non-optimal infection conditions, which may include inappropriate KLK13 concentrations or 322 low density of the entry receptor. Also, it is possible that RD cells may not support efficient 323 replication of the virus due to factors unrelated to the entry process. Nonetheless, our results\\r\\n show that the HCoV-HKU1 entry receptor is present on RD cells, and we were able to trigger 325 virus entry and replication; these findings warrant further exploration.\\r\\n Most coronaviral S proteins are processed into S1 and S2 subunits by host proteases, The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint HCoV-HKU1 S protein has two regions that are prone to proteolytic activation: the S1/S2 furin 330 cleavage site and a secondary cleavage site termed S2', which is adjacent to a potential fusion 331 peptide (38). While the S1/S2 site is believed to be processed by furin during protein 332 biosynthesis, the S2/S2' site is expected to be cleaved during virus entry. As we already knew being tested. We found that the S1/S2 site was efficiently cleaved by KLK13, whereas the 338 S2/S2' region remained intact. As CleavEx technique is a convenient surrogate system allowing 339 for precise mapping of the cleavage site, it has some limitations. To ensure the reliability of 340 results, purified full-length HCoV-HKU1 S protein was subjected to the proteolytic cleavage.\\r\\n Also here we observed efficient cleavage of the HCoV-HKU1 S protein by KLK13.\\r\\n While the results presented here show that KLK13 is able to process the HCoV-HKU1 The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint to cross the species barrier and cause severe diseases in humans, further research on the role of 355 different proteases in coronaviral infections is necessary. 356 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint\\r\\n Plasmid constructs 358 KLK13 and TMPRSS2 genes were amplified by PCR using cDNA obtained from HAE 359 cells. Each PCR product was cloned into pWPI plasmid for lentivirus production and sequence Table 2 . (Table 3 ; 500 nM each). β-actin was used as a household gene reference.\\r\\n The reaction was carried out according to the scheme: 5 min at 95°C, followed by 35 cycles of 465 30 s at 95°C, 20 s at 59°C and 20 s at 72°C, followed by 10 min at 72°C. containing proKLK13 or proKLK14 were concentrated and the buffer was changed to 50mM 528 Tris pH 7.5, 150 mM NaCl. After purification and self-activation at 37°C for 24 h, activity of 529 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint proteases was assessed by serine protease inhibitor Kazal-type 6 (SPINK6) titration, as 530 described previously (63). Protein expression was performed in E. coli BL21 and was induced by the addition of 553 0.5 mM IPTG to the bacterial culture (OD600 0.5-0.6), followed by shaking for 3h at 37°C. 554 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint Then, the bacteria were spun down and the pellet was suspended in buffer A (10 mM sodium 555 phosphate, 500 mM NaCl and 5 mM imidazole, pH 7.4). The pellet suspension was then 556 sonicated and spun down. Soluble proteins were purified using HisTrap TM Excel (GE The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint. 4 hepatoma cells was largely specific to CoV. Among 10 genetically divergent viruses tested, only hepatitis C virus was also restricted (Extended Data Fig. 1f ).\\r\\n LY6E is a member of the LY6/uPAR family of GPI-anchored proteins, and is implicated in diverse cellular processes 14 . To evaluate the specificity of LY6E for inhibiting CoV, we ectopically expressed select LY6/uPAR proteins 9 and infected with HCoV-229E. Only LY6E inhibited CoV infection (Extended Data Fig. 1g) . We previously showed that a conserved residue (L36) is required for LY6Emediated enhancement of influenza A virus 9 . L36 was also required to restrict HCoV-229E infection (Extended Data Fig. 1h) . These data suggest a conserved functional domain that is responsible for both the virus-enhancing and -restricting phenotypes.\\r\\n Next, we investigated the effect of LY6E on specific steps of the virus replication cycle. First, we tested whether LY6E restricts HCoV-229E attachment to the cell surface and observed no effect (Extended Data Fig. 2a) . In accordance, LY6E had no effect on surface expression of CD13, the specific receptor for HCoV-229E (Extended Data Fig. 2b) . To test whether LY6E restricts viral entry, we used a vesicular stomatitis virus (VSV) pseudoparticle (pp) system bearing CoV spike (S) proteins.\\r\\n Strikingly, LY6E significantly inhibited VSVpp entry mediated by S proteins from HCoV-229E (Fig.   2a) , MERS-CoV (Fig. 2b) , SARS-CoV (Fig. 2c) , and SARS-CoV-2 (Fig. 2d) . Entry mediated by VSV glycoprotein G was only marginally impacted by LY6E (Extended Data Fig. 2c, d) . To test whether LY6E interferes with the fusion of viral and cellular membranes, we performed a syncytia formation assay using propagation competent VSV pseudoviruses co-expressing CoV S protein and a GFP reporter (VSV*ΔG(CoV S)) (Extended Data Fig. 3a) . LY6E potently inhibited CoV S protein-mediated syncytia formation (Fig. 2e-f ). To determine whether LY6E impairs S protein expression or maturation, we also performed a heterologous syncytia formation assay. CoV-resistant hamster cells infected with trans-complemented GFP reporter VSV*ΔG(CoV) were co-cultured with susceptible target cells that ectopically expressed LY6E. LY6E again significantly blocked syncytia formation, demonstrating that LY6E inhibits S protein-mediated fusion and not S protein expression or maturation (Extended Data Fig. 3b-c) . As membrane fusion can occur without syncytia formation, we also performed a quantitative fusion assay in which mixing of cell contents results in complementation of split luciferase (Fig. 2g) .\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint Cells co-transfected with plasmids encoding CoV S proteins and split luciferase 1-7 were mixed with susceptible target cells co-expressing LY6E or vector control and split luciferase 8-11. LY6E reduced CoV S-mediated cell-cell fusion, indicating that LY6E blocks fusion of viral and cellular membranes ( Fig. 2h) . Time-course experiments revealed that LY6E had no effect on HCoV-229E translation, replication, assembly, or release (Extended Data Fig. 2e-i) . Collectively, our results demonstrate that LY6E specifically inhibits CoV S protein-mediated membrane fusion, which is required for viral entry.\\r\\n We next addressed whether LY6E modulates proteolytic activation of the S protein. Upon S proteinmediated binding to the respective receptor, host proteases cleave the S protein and make it fusion competent for \\\"early entry\\\" at the cell surface or \\\"late entry\\\" in endosomes 15, 16 . Pharmacological inhibition of cell surface and endosomal proteases had no effect on LY6E-mediated restriction of CoV infection (Extended Data Fig. 4a-c) . Next, we addressed whether LY6E directly interferes with the proteolytic activation of S protein. Mutation of S1/S2 and S2′ in the S protein prevents activation (Extended Data Fig. 4d ) 16, 17 . Ectopic LY6E expression had no effect on MERS-CoV S cleavage or infection by cleavage-resistant MERS-CoV pp (Extended Data Fig. 4e-g) . Collectively, our data indicate that restriction of viral fusion is independent of S protein activation.\\r\\n Next, we aimed to evaluate LY6E-mediated CoV restriction in vivo. Systemic genetic ablation of Ly6e in mice is embryonic lethal 18 . Therefore, we generated transgenic Ly6e fl/fl mice, which are functionally 'wild-type (WT)' for Ly6e expression, and crossed them with Vav1-iCre mice to ablate Ly6e in cells that originate from a hematopoietic stem cell (HSC) progenitor (Extended Data Fig. 5a-b ). This genetic model was chosen since immune cells are critical for protection from CoV in vivo 5,19-23 . Bonemarrow derived macrophages (BMDM) from Ly6e ΔHSC mice had reduced Ly6e mRNA levels (Extended Data Fig. 5c ). The natural mouse pathogen MHV is a well-studied model CoV that causes hepatitis and encephalomyelitis in mice 24 . Ly6e ΔHSC BMDM were more susceptible to MHV infection than Ly6e fl/fl BMDM, demonstrating that murine Ly6e is a restriction factor for murine CoV (Extended Data Fig.   5d ).\\r\\n To determine whether Ly6e is important for controlling CoV infection in vivo, Ly6e fl/fl and Ly6e ΔHSC mice were infected with a high dose (5 x 10 4 PFU) of MHV that causes sub-lethal hepatitis in WT author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint C57BL/6J mice 25 . Since MHV pathogenesis is influenced by sex 26 , we performed independent experiments in male and female mice. Ly6e ΔHSC mice of both sexes rapidly succumbed to MHV infection by day 6 ( Fig. 3a-b, Extended Data Fig. 6a-b) . To examine viral pathogenesis and immunopathology in infected mice, Ly6e fl/fl and Ly6e ΔHSC mice were infected with the same high dose of MHV and euthanized 3-or 5-days post-infection. At both time points, female Ly6e ΔHSC mice exhibited significantly higher levels of liver damage as measured by serum ALT while no difference was observed in male mice (Fig. 3c,i, Extended Data Fig. 6c,i) . However, hepatic viral burden did not differ in either sex ( Fig. 3d,j, Extended Data Fig. 6d,j) , possibly due to tissue saturation at this high dose. Indeed, female but not male Ly6e ΔHSC mice exhibited increased liver damage and viral burden at a low dose (5 PFU) of MHV (Extended Data Fig. 7a -b, f-g). Spleen viral burden was elevated in female and male Ly6e ΔHSC mice at both time points and doses of MHV (Fig. 3e,k, Extended Data Fig. 6e ,k, Extended Data Fig.   7c ,h). At 3 days post-infection, there was no difference in liver necrosis, but inflammation was moderately reduced in female but not male Ly6e ΔHSC mice ( Fig. 3f-h, Extended Data Fig. 6f-h) . By 5 days post-infection and at both low and high doses of MHV, female Ly6e ΔHSC mice had significantly higher levels of liver necrosis and a reduced presence of mononuclear immune cells ( Fig. 3l- Fig. 7k-n) . These data demonstrate that Ly6e in hematopoietic cells is important for controlling murine CoV infection.\\r\\n We next used a transcriptomic approach to evaluate the effect of Ly6e ablation on global gene expression in liver and spleen of female Ly6e fl/fl and Ly6e ΔHSC mice injected with PBS or the high dose of MHV at 3-and 5-days post-infection. In infected tissues, loss of Ly6e in hematopoietic cells correlated with differential gene expression in pathways associated with tissue damage, such as liverspecific metabolic genes, angiogenesis, wound healing, and immune response to viruses (Fig. 4a-c) . We also observed a striking loss of genes associated with the type I IFN response, inflammation, antigen presentation, and B cells in infected Ly6e ΔHSC mice ( Fig. 4d-e, Extended Data Fig. 8) . 7 The histopathology (Fig. 3h,n) and transcriptome data (Fig 4a-e, Extended Data Fig. 8 Fig. 7k-n, Extended Data Fig. 9 ). Multiple cell types were depleted in livers and spleens of infected Ly6e ΔHSC mice, irrespective of sex (Figure 4f g, Extended Data Fig. 10a-b) . B cells were dramatically reduced in livers from infected Ly6e ΔHSC mice.\\r\\n Hepatic CD4 + T cells, NK cells, dendritic cells (DC), macrophages, and neutrophils were also depleted in infected Ly6e ΔHSC mice, whereas CD8 + T cells were unchanged (Fig. 4f, Extended Data Fig. 10a ).\\r\\n Infected Ly6e ΔHSC mice had a reduction in all splenic immune cell subsets except for CD4 + and CD8 + T cells (Fig. 4g, Extended Data Fig. 10b) . Depletion of immune cell populations in Ly6e ΔHSC organs was MHV-dependent, as cell numbers were not altered in PBS-injected mice.\\r\\n To determine whether loss of immune cells in MHV-infected Ly6e ΔHSC mice correlates with increased permissiveness to infection, we assessed MHV infection in splenocytes cultured from Ly6e ΔHSC , Ly6e fl/fl , and Ifnar -/mice. In Ly6e fl/fl splenocyte cultures, MHV infected macrophages, neutrophils, DC, and B cells, but not CD4 + or CD8 + T cells, as previously published (Fig. 4h, Extended Data Fig. 10c Together, our data demonstrate an important role for Ly6e in immune cell-mediated control of CoV infection. Higher viral burden and lower liver inflammation suggests that liver damage in MHV-infected female Ly6e ΔHSC mice may be due to viral pathogenesis that exceeds the ability of the immune system to control the infection. Notably, we observed MHV-induced loss of three classes of cells: 1) cells that are not permissive to the virus (CD4 + T cells), 2) cells that are permissive to MHV but do not demonstrate Ly6e KO-dependent infection (NK cells, macrophages, and neutrophils), and 3) cells that are permissive to MHV in a Ly6e KO-dependent manner (B cells and DC). This latter group is capable of antigen-presentation to CD4 + T cells 27,28 . Thus, in the absence of Ly6e, unrestricted MHV infection author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint of B cells or DC may impair T cell priming, thereby negatively impacting recruitment and activation of NK cells, macrophages, and neutrophils that contribute to control of viral disease.\\r\\n In conclusion, we identified LY6E as a pan CoV restriction factor that limits CoV entry and protects the host from severe viral disease. Our findings are striking given that LY6E has been primarily associated with an enhancing phenotype in which LY6E promotes entry of multiple viruses 29 . However, our data clearly expand the role of LY6E during coronaviral infection and establishes a novel function in protecting the host immune cell compartment. Determining the precise molecular mechanism underlying how LY6E inhibits CoV S protein-mediated membrane fusion will advance our understanding of cellular antiviral defenses against these important human pathogens. Antiviral membrane fusion inhibitors have been successfully implemented for treatment of HIV-1 infection 30 . A therapeutic approach mimicking the mechanism of action of LY6E could provide a first line of defense against novel emerging CoV infections. Furthermore, delineating which Ly6e-expressing immune cells protect mice from MHV will provide new insight into how individual antiviral effectors in distinct cellular compartments modulate viral pathogenesis. author/funder. All rights reserved. No reuse allowed without permission. 34-41 (1996) . Welch's correction (c-d, f-g), one-tailed Mann-Whitney U test (e), ratio paired student's t-test (g), 2way\\r\\n ANOVA followed by Sidak's or Dunnett's multiple comparison test (j,l). Error bars: SD. P values: c, ** p=0.0088; d, *** p=0.0001; e, * p=0.0143; f, * p=0.0286; g, * p=0.0101; h, *** p=0.0010; j, ** p=0.0073, p=0.0033; l, ns=0.0740, ** p=0.0013, *** p=0.0001. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint p=<0.0001; c, **p=0.0066; d, **** p=<0.0001; f, **** p=<0.0001; h, ns p=0.9975, ** p=0.0054, ns p=0.1520, 0.9892. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint scale bars are 500 µM (4x) and 100 µM (20x). Statistical significance was determined by Mantel-Cox test (a), two-tailed unpaired student's t-test with Welch's correction (c-e, i-k), two-tailed Mann-Whitney U test (f-g, l-m). Error bars: SEM (b), SD (c-g, i-m). P values: a, *** p=0.0002; c, ** p=0.0031; d, ns p=0.2136; e, **** p=5.759 x 10 -7 ; f, ns p=0.3698; g, ** p=0.0054; i, * p=0.0430; j, ns p=0.1321; k, ** p=0.0094; l, ** p=0.0072; m, **** p=8.741 x 10 -5 . author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint maps displaying significant changes (mean RPKM > 0.5; fold change > 2; FDR ≤ 0.05) in liver (a) and spleen (b). Dendrograms are normalized read data (row z-score) clustered with complete linkage method employing Spearman Rank correlation distance measurement. c, Pathway analysis of KO versus WT.\\r\\n Up-(red) or down-regulated (blue) pathways indicated by activation z-score. Numbers show significantly dysregulated genes as percentage of total gene number included in pathway. d-e, Expression of select genes in liver (d) and spleen (e). f-g, Immune cell counts from liver (f) and spleen \\r\\n HCoV-229E: 1.5 x 10 5 stable LY6E expressing or control cells were seeded in a 12-well plate and infected with HCoV-229E in a serial dilution. Cells were incubated for 2 hours at 33°C, then viral supernatant was removed, and an overlay of 1:1 Avicel (2.4%) (Avicel RC-581NF) and 2x DMEM, author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint supplemented with 10% FBS and 1x penicillin/streptomycin (p-s), was added. Cells were incubated for 3 days at 33°C before medium was removed, cells were washed, and stained with crystal violet (Sigma-Aldrich). Plaque forming units (PFU) were calculated.\\r\\n HCoV-OC43: 5.0 x 10 4 stable LY6E expressing or control cells were seeded in a 24-well plate and infected at 33°C for 1 hour with HCoV-OC43 (MOI=1). Virus was aspirated and 10% FBS/1x NEAA/1x p-s/RPMI (cRPMI) was added back to cells. 24 hours post-infection, cells were dissociated using Accumax (Sigma-Aldrich), fixed in 1% PFA, permeabilized per manufacturer's protocol (BD Cytofix/Cytoperm), and stained for nucleoprotein (1:500) and a goat anti-mouse AlexaFluor488- The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint anti-mouse secondary antibody. Images were acquired with a fluorescence microscope and analyzed using ImageXpress Micro XLS (Molecular Devices, Sunnyvale, CA).\\r\\n For infection assays, 1 x 10 5 target cells were seeded in a 24-well plate one day prior infection. Cells were infected with HCoV-229E-Rluc (MOI 0.1) in OptiMEM (Gibco) for 2 hours at 33°C. Cells were washed 1x with PBS and 10% FBS/1x NEAA/1x p-s/DMEM (cDMEM) was added back. Cells were incubated at 33°C for 24 hours, then washed with PBS and lysed using the Renilla Luciferase Assay System kit (Promega). Rluc activity was measured using a plate luminometer (EnSpire 2300 Multilabel reader by Perkin Elmer). For the reconstitution with CRISPR resistant LY6E (CR LY6E), 2.5 x 10 4 cells were seeded in a 24-well plate. One day post-seeding, cells were either left untransduced or transduced with a lentiviral vector encoding for CR LY6E or the empty control. 48 hours post-transduction, cell lysates were either infected with HCoV-229E-Rluc (MOI=0.1) for 24 hours as described above or harvested for Western blot.\\r\\n Pseudotyping of VSV*ΔG(Fluc) was performed as previously described 50 with two silent mutation (T2568G, T3327C)), have been described previously [51] [52] [53] . The expression plasmid for SARS-CoV-2 was generated as follows: the coding sequence of a synthetic, codonoptimized (for human cells) SARS-CoV-2 DNA (GeneArt Gene Synthesis, Thermo Fisher Scientific) author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint based on the publicly available protein sequence in the National Center for Biotechnology Information database (NCBI Reference Sequence: YP_009724390.1) was PCR-amplified and cloned into the pCG1 expression vector between BamHI and XbaI restriction sites. The MERS-CoV S cleavage mutants have been described before 17 \\r\\n The split luciferase fusion assay has been described before, with slight modifications 54 . Briefly, 6 x 10 5 293LTV cells were transfected with plasmids encoding CoV S proteins (pCAGGS-HCoV-229E, pCAGGS-MERS-CoV) together with a split-luciferase construct (Rluc8155-156 DSP1-7). The empty expression plasmid (pCAGGS-mCherry) as well as a construct encoding for VSV G served as negative and positive control, respectively. 2 x 10 5 stable LY6E expressing or empty control Huh7 cells were transfected using Lipofectamine 2000 (Thermo Fisher Scientific) with a plasmid encoding the second half of the split-luciferase protein (Rluc8155-156 DSP8-11). Approximately 30 hours post-transfection, both cell populations were dissociated (TryPLE Express, Gibco), counted, and equal cell numbers (2x10 4 cells each) were co-cultured for 16 -20 hours at 37°C. Supernatant was harvested and cells lysed using ice-cold passive lysis buffer (Promega). Cells were immediately transferred to -80°C for at least author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint 1 hour, before Rluc activity was determined using the Renilla Luciferase Assay System (Promega). Cells were kept on ice during all times post-cell lysis to eliminate DSP post-lysis complementation.\\r\\n Following extraction of total RNA from MERS-CoV (strain EMC) infected VeroE6 cells, the cDNA encoding the MERS-CoV S protein was generated by reverse transcription (RevertAid Premium Reverse Transkriptase, ThermoScientific). Three overlapping cDNA fragments were amplified by PCR (Phusion Hot Start II High Fidelity Polymerase, Thermo Scientific) and subsequently assembled by overlapping PCR. The cDNAs encoding the S proteins of either MERS-CoV (strain EMC) or HCoV-229E (truncated variant lacking the 10 amino acids at the C terminus, original plasmid pCAGGS-229E S) were amplified by PCR and inserted into the pVSV* plasmid 55 between MluI and BstEII restriction sites. The resulting plasmids were used to generate the recombinant viruses VSV*ΔG(MERS S) and VSV*ΔG(229E S) according to a published procedure 56 . All viruses were propagated on BHK-G43 cells 57 , resulting in viruses predominantly containing the homotypic VSV glycoprotein G in the envelope.\\r\\n Cells expressing LY6E or empty control were infected with VSV*ΔG(MERS S) or VSV*ΔG(229E S) at a MOI of 0.01. At 20 hours post-infection, the cells were fixed with 3% PFA/PBS and the nuclei stained with 4',6-diamidine-2'-phenylindole (DAPI, Sigma). An inverted fluorescence microscope (Zeiss) was used to determine area percentage area covered by syncytia.\\r\\n For heterologous cell-cell fusion assay, BHK-21 cells were infected with VSV*ΔG(CoV S) at a MOI of 1. After 2 hours, the cells were treated with trypsin/EDTA (Life Technologies, Zug, Switzerland) and resuspended in DMEM supplemented with 5% FBS (2x10 4 cells/mL). The infected BHK-21 cell suspension (500 µL) was seeded into 24-well cell culture plates along with LY6E-or empty controlexpressing Huh7 (1 x 10 4 cells). Cell co-cultures were incubated for 7 hours at 37°C, fixed with 3.6% author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint formaldehyde diluted in PBS, (Grogg-Chemie AG, Stettlen, Switzerland) and stained with DAPI. The percentage area covered by syncytia was calculated as above.\\r\\n Ly6e tm1a ES cells were obtained from the EUCOMM consortium 58 \\r\\n Six to twelve-week-old female and male mice were injected intraperitoneally with MHV-A59 diluted in PBS to the indicated titers or PBS for a mock infection control. All infected mice were monitored daily for weight and mortality. Animals that lost more than 20% of their original body weight were euthanized per IACUC guidelines.\\r\\n Viral titers in liver and spleen were determined from frozen organs after weighing, homogenization, and plaque assay on L929 cells. Alanine aminotransferase (ALT) was measured in fresh, unfrozen serum using VITROS MicroSlide Technology by the UTSW Mouse Metabolic Core. Livers were fixed in 10% author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint neutral buffer formalin, embedded in paraffin, sectioned at 5 µM, and stained with hematoxylin and eosin (H&E). Slides were analyzed by an independent pathologist (UTSW Animal Resource Center) who was blinded to experimental conditions. A numerical score was assigned for degree of inflammation and degree of necrosis for each liver. Inflammatory cell infiltration was scored using the following criteria: 0 = none, 1 = minimal, 2 = mild, 3 = moderate, and 4 = marked. The necrosis score was defined by the percent necrosis. 0 = none, 1 = <2%, 2 = 2-20%, 3 = 21-40%, and 4 = >40%.\\r\\n All primary cells were maintained in 10% FBS/10 mM HEPES/1 mM sodium pyruvate/2 mM Lglutamine/1x p-s/RPMI (cRPMI2) unless otherwise indicated.\\r\\n Primary bone marrow derived macrophages (BMDM) were prepared as described previously 60 . Viable cells were quantified using trypan blue exclusion. BMDM were plated at 2.5 x 10 4 cells per well in non- The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint separated from IHIC by centrifugation at 850 x g for 30 minutes, 22°C, with no brake. Red blood cells were removed from the resulting pellet with RBC lysis buffer (Tonbo Biosciences). Viable cells were quantified using trypan blue exclusion. IHIC and splenocytes (4 x 10 5 ) from infected or mock-infected mice were stained with a fluorescent viability dye (Ghostdye Violet 450, Tonbo Biosciences), treated with anti-CD16/CD32 (Tonbo Biosciences), stained with antibodies that recognize surface lineage markers, and then fixed in 1% PFA/PBS. Fixed volumes of cell suspensions were analyzed by flow cytometry the next day. Absolute cell counts were determined by multiplying final number of lineage marker-positive cells by dilution factor used to normalize cell counts for antibody staining.\\r\\n All primary cell samples were resuspended in 3% FBS/PBS and analyzed using a S1000 Flow Cytometer with a A600 96-well plate high throughput extension and compensated using CellCapture software (Stratedigm). Data was analyzed with FlowJo Software (Treestar). The flow gating strategy for liver and spleen immune cells is included in Extended Data Figure 9 .\\r\\n Differences in data were tested for significance using GraphPad Prism v8.3.1 for Windows (GraphPad).\\r\\n For details regarding the statistical tests applied, please refer to the figure legends. P values < 0.05 were considered significant.\\r\\n The authors declare that the data supporting the findings of this study are available within the article and its Supplementary Information files or are available on request. The RNAseq data discussed in this publication have been deposited in the Gene Expression Omnibus (GEO) database, https://www.ncbi.nlm.nih.gov/geo (GSE146074).\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint\\r\\n We would like to thank the following people for their generous contribution of reagents and infrastructure which greatly enhanced this study, especially the sequencing facility in Bern, the diagnostic facility of IVI, and the Animal Resource Center at UTSW. Furthermore, we are grateful to \\r\\n The authors declare no competing interests.\\r\\n Supplementary Information is available for this paper.\\r\\n Correspondence and requests for materials should be addressed to Volker Thiel, John W. Schoggins, and Charles M. Rice.\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint was harvested and extracellular viral RNA was extracted and viral replication detected via qRT-PCR (f). Cell lysates were harvested and intracellular Renilla luciferase activity was detected upon cell lysis (g). Cells were subjected to 3 rounds of freeze/thaw cycles. Cell debris was removed and the supernatant titrated on naïve Huh7 cells. Intracellular infectivity was determined (h). Supernatant was harvested and The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint ANOVA followed by Sidak's multiple comparisons test (c). Error bars: SD. P values: c, **** p=<0.0001 <0.0001. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint two pooled experiments. For i-m, n=8 (Ly6e fl/fl ,), n=6 (Ly6e ΔHSC ), or n=3 (PBS) from two pooled experiments. In h,n, scale bars are 500 µM (4x) and 100 µM (20x). Significance for was tested by\\r\\n Mann-Whitney U test (f-g, l-m). Error bars: SEM (b), SD (c-g, i-m). P value: a, **** p=3.946 x 10 -6 ; c, ns p=1144; d, ns p=0.1407; e, ** p=0.0026; f, ns p=0.1708; g, ns p=0.0779; i, ns p=0.6191; j, ns p=0.6842; k, * p=0.0119; l, ns p>0.9999; m, ns p=0.0779. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint -c, f-h) , two-tailed Mann-Whitney U test (d-e, i-j). Error bars: SD. P value: a, * p=0.0234; b, * p=0.0428; c, *** p=0.0004; d, * p=0.0210; e, ** p=0.0054; f, ns p=0.1544; g, ns p=0.5322; h, ns p=0.0662; i, ns p=0.0592; j, ns p=0.9064. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint DDX3X -DDX3, VIPERIN -RSAD2, SERPINE1 -PAI 1, IL28RA -CRF2, VEGFC -VRP, SP110 -IFI41, IRF2 -PRIC285, ZNF313 -RNF114, DNAPTP6 -LOC26010, LOC400759 -PSEUDOGENE,   PSCD1 -CYTH1, IFITM1 -IFI17, SIGLEC1 -FRAG, CCL5 -SYCA5, CXCL9 -MIG, IFIH1 Huh7 cells expressing TMPRSS2 were generated as follows. In order to generate TMPRSS2-encoding retroviral vectors, the open reading frame of human TMPRRS2 was first PCR amplified with primers adding an N-terminal cMYC epitope to the TMPRSS2 coding sequence. The resulting sequence was inserted into a modified version of the pQCXIP plasmid that contains a blasticidin resistance cassette instead of the usual puromycin resistance cassette 17 . Huh7 cells stably expressing human TMPRSS2 were generated by retroviral transduction and selection with the antibiotic blasticidin (50 µg/ml).\\r\\n Following selection, cells were maintained in culture medium (cDMEM) supplemented with 10 µg/ml blasticidin. To generate STAT1 -/-_CEACAM1 cells, human STAT1 −/− fibroblasts were transduced with lentiviruses encoding for the murine CoV receptor CEACAM1 (kind gift from David Wentworth, CDC, Atlanta, USA 59 and subsequently selected with 1 µg/mL puromycin. Stable LY6E expressing cell lines were generated upon lentiviral transduction with SCRPSY LY6E or as control SCRPSY Empty in DMEM containing 4 µg/ml polybrene (Millipore) and 20 mM HEPES buffer solution (Gibco). Cells author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint were selected using 2-4 µg/mL puromycin and were passaged until all cells in control wells without lentivirus were killed. The puromycin selected cells were further passaged and frozen down for subsequent experiments. The stable cell lines harboring LY6E orthologues have been described before, with the exception of C. dromedarius 9 . For this, a gBlock was ordered (XM_031439745.1) and Gateway cloning performed as described previously to generate pENTR and pSCRPSY plasmids 9 . Constructs encoding for Ly6/uPAR family members and LY6E ASM mutants have been described before 9 .\\r\\n To generate clonal LY6E KO cells and CRISPR-resistant LY6E, A549 cells were transduced with lentivirus containing a LY6E-specific sgRNA and Cas9 as described previously 9 . To generate a clonal cell line, the bulk transduced population was plated at single cell dilutions. Candidate clones were screened by Western blot for LY6E expression and Sanger sequencing. Silent mutations in the region targeted by the LY6E-specific sgRNA were introduced into HA-tagged LY6E to generate CRISPRresistant LY6E (CR LY6E). LY6E KO A549 cells were reconstituted with CR-LY6E by lentiviral transduction and expression confirmed by Western blot.\\r\\n All cell lines were regularly tested to check they were free of mycoplasma contamination using a commercially available system (PCR Mycoplasma test kit I/RT Variant C, PromKine).\\r\\n The ISG screen was performed as described previously with slight modifications 7,8 . Briefly, 5 x 10 3 Huh7 cells were seeded, transduced with individual lentiviruses, and 48 hours post-transduction infected with HCoV-229E at 33°C. Infection was stopped 24 hours (MOI=0.1) or 48 hours (MOI=0.01) postinfection and plates were immuno-stained as described previously 61 with an anti-HCoV-229E N protein antibody and a AlexaFluor488-conjugated donkey anti-mouse secondary antibody (Extended methods;\\r\\n Antibodies for immunofluorescence and flow cytometry). For high-content high-throughput imaging analysis ImageXpress Micro XLS (Molecular Devices, Sunnyvale, CA) was used as previously described 8 . Hits were normalized to cells expressing the empty vector. Depicted are genes that were author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint expressed in two independent screens with a transduction efficiency of at least 1 % (cut off). ISGs which were cytotoxic for the cells were excluded from the analysis. Normalized data can be found in Extended Data \\r\\n Viral RNA was extracted from cell lysates (NucleoMag-96RNA kit, Macherey Nagel) or supernatant (NucleoMag-Vet kit, Macherey Nagel) using the KingFisher Flex robot according to the manufacturer's recommendations. The commercially available TaqManTM Fast Virus 1-Step Master Mix (Applied Biosystems) was used for RT-qPCR with 2 µl of RNA input added to 8 µl of prepared mastermix per author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint sample. Viral RNA was detected using HCoV-229E specific primers normalized to a qRT-PCR standard for HCoV-229E (contains the M gene of HCoV-229E 63 ). Intracellular viral RNA was normalized to total RNA (determined via the housekeeping gene Beta-2-Microglobulin (B2M)). Cellular RNA was extracted from cell lysates of Huh7 cells using the Nucleospin RNA kit (Machery Nagel) according to the manufacturer's recommendations, quantified via NanoDrop, and used to generate a standard curve.\\r\\n PCR conditions are available upon request. \\r\\n Stable LY6E expressing or control Huh7 cells were seeded in a 24-well plate (4 x 10 4 cells) and mock infected or inoculated with HCoV-229E (MOI=5) in OptiMEM (Gibco) on ice for 1 hour. Cells were washed at least 3x with PBS and harvested immediately (t=0 h) or incubated at 33°C for 24 hours (t=24 h), before cell lysis and extraction of viral RNA as described above. Viral RNA was detected via RT-qPCR normalized to the housekeeping gene B2M as described above.\\r\\n Time course experiment 8 x 10 4 cells stable LY6E expressing or control Huh7 cells were seeded in 12-well plates and mock infected or infected with 229E-CoV-Rluc (MOI=0.1) for 2 hours at 33°C in OptiMEM. Cells were washed 3x with PBS and samples harvested at the indicated time points. To determine intracellular replication, cell lysates were collected and viral RNA extracted (NucleoMag-96RNA kit, Macherey author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint Nagel), followed by RT-qPCR as described before, or cells were lysed using the Renilla Luciferase Assay System kit (Promega) according to the manufacturer's recommendations and Rluc activity determined. To determine extracellular viral replication, viral supernatant was harvested, and viral RNA extracted (NucleoMag-Vet kit, Macherey Nagel), followed by RT-qPCR as described before. To determine intracellular infectivity, cells were subjected to 3 rounds of freeze/thaw cycles, centrifuged to remove debris (4,000 × g for 10 min), and the supernatant titrated on naïve Huh7 cells. To determine extracellular infectivity, supernatant was harvested at the indicated time points and titrated on naïve \\r\\n To test various protease inhibitors, 2 x 10 4 naïve or TMPRSS2-expressing, control or LY6E cells were seeded in a 96-well plate. One day post seeding, cells were pre-treated with the following compounds for 1 hour in OptiMEM, at 37°C: DMSO (1:500, Sigma-Aldrich), E64 D (10 µM, Sigma-Aldrich), and/or Camostat (100 µM, Sigma-Aldrich). Cells were mock infected or infected with HCoV-229E-Rluc (MOI 0.1) in the presence of the inhibitors for 2 hours at 33°C. Medium was changed to DMEM and cells incubated for 24 hours at 33°C. Cells were lysed and Rluc activity detected using the Renilla Luciferase Assay System kit (Promega) according to the manufacturer's recommendations.\\r\\n LY6E expressing or control Huh7 (2 x 10 5 cells) were seeded in 6-well cell culture plates. Cells were transfected using Lipofectamine 2000 (Invitrogen) with an expression plasmid encoding for MERS-CoV S (pCAGGS-MERS S). Cell lysates were harvested 48 hours post-transfection and subjected to Western blot analysis as described.\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint\\r\\n Multiple sequence alignment of LY6E amino acid sequences of different species (NCBI accession numbers provided in the figure) was performed using the EBI web based Clustal Omega tool (doi:\\r\\n 10.1093/nar/gkz268). The phylogenetic tree was built with the \\\"one click\\\" mode for phylogenetic analysis (http://www.phylogeny.fr/) with standard settings (doi: 10.1093/nar/gkn180; doi:\\r\\n 10.1186/1471-2148-10-8).\\r\\n Sections of liver and spleen were preserved in RNAlater Stabilization Solution (Thermo Fisher Scientific) and frozen. Thawed samples were transferred to PBS and homogenized. One-eighth of the homogenate was mixed with TRIzol Reagent (Thermo Fisher Scientific). BMDM were directly lysed in TRIzol Reagent and frozen. Total RNA was isolated according to the manufacturer's protocol. Liver and spleen RNA were subject to DNase treatment (TURBO DNA-Free kit, Thermo Fisher Scientific) per the manufacturer's protocol prior to RNA-Seq analysis. BMDM RNA was analyzed by one-step qRT-PCR using QuantiFast SYBR Green RT-PCR Kit (Qiagen) using Applied Biosciences 7500 Fast Real-Time PCR System.\\r\\n The quantity and quality of the extracted RNA was assessed using a Thermo Fisher Scientific qubit 2. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint Data generated from individual samples (>30 million read per sample, single read 50-mers) were mapped separately against the GRCm38 murine reference genome. Gene expression was calculated for individual transcripts as reads per kilobase per million bases mapped (RPKM). All transcriptomic analyses were performed using CLC Genomics Workbench 20 (Qiagen, Aarhaus).\\r\\n Differentially expressed genes (DEGs) were identified by calculating fold changes in expression, pvalues were corrected by taking false discovery rate (FDR) for multiple comparison into account.\\r\\n Mus Musculus EBI Gene Ontology Annotation Database was used to execute Gene ontology (GO) Enrichment Analyses for biological processes. Gene identifiers for DEGs absolute FC > 5, RPKM > 2) were used as input and identification of significantly enriched GO categories. P-values for specific GO categories were generated after Bonferroni correction for multiple testing. Z-scores, used as indicator for activation of biological processes (positive value) or inactivation (negative value), were calculated based on the expression fold change as follows: The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint gctggcatcgggaacgtggtggactttggctac accctgaacaagggctgctccccgatctgtccc ggcccgagcgtcaatcttggagtggcgtccgt gggcacccactgctgccagagcttcctgtgca acatcagtgcagccgacggcgggctgcgggc cagcacccacgtgctgggcctcgggctcctgc tcagcctgctgtccgccctgctgcggcttggcc cctgaacccagctttcttgtacaaagtggtcccc author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint. The rapid evolution of RNA viruses represents a significant challenge for preventing, treating, 57 and eradicating RNA viral diseases. High mutation rates in RNA viruses generate extensive 58 opportunities to overcome evolutionary hurdles, such as antiviral drugs, host immunity, or 59 engineered attenuating changes (1). The evolutionary pathways traversed by RNA viruses are 60 shaped by natural selection, which will favor some evolutionary trajectories more than others 61 based on whether mutations are beneficial, deleterious, or neutral (2). Predicting the likely results 62 of RNA virus evolution is an important step for anticipating viral emergence and for developing 63 escape-resistant antiviral drugs and vaccines (3, 4). 64 65 Coronaviruses (CoVs) are a family of positive-sense RNA viruses that cause human illnesses 66 ranging from the common cold to severe and lethal respiratory disease (5). All CoVs encode a 67 proofreading exoribonuclease within nonstructural protein 14 (nsp14-ExoN) that is critical for 68 replication, fidelity, fitness, and virulence, and ExoN-inactivation has been proposed as a 69 strategy for live-attenuated vaccine development (6-15). As members of the DEDDh superfamily 70 of exonucleases, CoV ExoNs hydrolyze nucleotides using four metal-coordinating amino acids 71 The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint ExoN active site mutants in alphacoronaviruses, including transmissible gastroenteritis virus and 77 hCoV-229E, have yet to be recovered and are proposed to be lethal for replication (19, 20) . infections and prolonged passage in tissue culture and following treatment with multiple 87 nucleoside analogues (6-11, 13, 14). SARS-CoV-ExoN-AA also is stable during acute and 88 persistent animal infections in immunocompetent and immune-compromised mice (12) . The lack 89 of primary reversion is not due simply to reduced adaptive capacity, as both SARS-CoV-and 90 MHV-ExoN-AA can adapt for increased replication (7, 14). Most strikingly, long-term passage 91 of MHV-ExoN-AA (250 passages, P250) yielded a highly fit population that had directly 92 compensated for defective proofreading through evolution of a likely high-fidelity RdRp (7 requires mutations to all four sites ( Figure 1A ). Viral mutation rates in the absence of 113 proofreading range from 10 -4 to 10 -6 mutations per nucleotide per round of replication (µ) (1). 114\\r\\n Assuming an ExoN-AA mutation rate of 10 -4 µ and accounting for codon degeneracy, the 115 probability of restoring the native amino acid sequence in a single round of replication is 10 -18 . 116\\r\\n Only rarely do ExoN-AA titers exceed 10 6 PFU/mL, so it is unlikely that ExoN-AA could 117 navigate this genetic barrier in a single infectious cycle. Thus, we hypothesized that ExoN-AA 118 reversion, if possible, would proceed incrementally. To identify potential pathways towards 119\\r\\n ExoN-AA reversion, we examined the possible single-nucleotide substitutions surrounding A89 120 and A91 ( Figure 1B ). Three mutations are synonymous, and five mutations yield amino acids 121 unlikely to coordinate with the positively-charged metals required for ExoN catalysis (glycine, 122 valine, proline, threonine, and serine) (16, 19, 21, 22). One mutation per site can restore the 123 acidic charge (i.e. AA-to-ED) but not the native amino acid. These variants have not been tested 124 in a CoV ExoN, but biochemical studies of E. coli DNA polymerase I ExoN mutants suggest that 125 these conservative substitutions would not restore WT-like ExoN activity (23) . We predicted 126 stepwise pathways to ExoN-AAàDE reversion based on restoration of acidic charge followed 127 by reversion to native amino acids ( Figure 1C ). We engineered and recovered variants in ExoN-128 AA requiring three mutations (3nt; ExoN-AD, ExoN-EA), two mutations (2nt; ExoN-DA, ExoN-129 ED, ExoN-AE), or one mutation (1nt; ExoN-DD, ExoN-EE) for reversion to WT-ExoN-DE 130 (Table 1) . We will hereafter refer to these mutants as intermediate revertants. All intermediate 131 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint revertants generated viable progeny during recovery, demonstrating that reversion to WT-ExoN-132 DE along these pathways is theoretically possible. The 3nt and 2nt mutants were genetically 133 stable during recovery, as confirmed by dideoxy sequencing. However, both 1nt mutants (ExoN-134 DD and ExoN-EE) reverted to WT-ExoN-DE during three independent recovery attempts, 135\\r\\n suggesting that these two variants are less fit than WT-ExoN-DE and demonstrating that 136 reversion by 1nt mutation is readily accessible. To test whether the 3nt or 2nt mutants would 137 revert more rapidly than ExoN-AA (4nt), we passaged three lineages of each mutant 10 times at (Table 1 ). In contrast, the 2nt ExoN-AE contained WT-revertants by P2 145 in all lineages at MOI = 0.5 PFU/cell and by P8 in one lineage at MOI = 0.01 PFU/cell. Once 146 observed, WT-revertants dominated the ExoN-AE population for the remaining passages. These 147 data indicate that at least one 2nt mutation pathway can lead to full reversion in tissue culture. 148\\r\\n The probability of ExoN-AE arising during a single infectious cycle of ExoN-AA is low but 149 theoretically achievable (~10 -9 ), so ExoN-AA could conceivably revert within just two infectious 150 cycles. However, complete reversion has never been observed even during prolonged passage or 151 persistent infections, suggesting that additional barriers to the replication, fitness, or maintenance 152 of intermediate revertants exist. 153 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint \\r\\n GAT GAA\\r\\n GCA GCT\\r\\n ExoN Motif I C All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. \\r\\n All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint\\r\\n the intermediate revertants are viable as recombinants but are not found in ExoN-AA 159 populations, we hypothesized that they confer no selective advantage over ExoN-AA (8, 9, 13). 160\\r\\n To test this hypothesis, we first analyzed replication of the 3nt and 2nt intermediate revertants 161 ( The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. nsp12/14-P250) were significantly more sensitive to 5-FU ( Figure 4B ). Finally, the nsp12-P250 224 and nsp14-P250 mutations significantly decreased fitness relative to WT-ExoN-DE ( Figure 4C ). 225\\r\\n We detected no statistical differences between the specific infectivity of WT-ExoN-DE and any 226 of the nsp12-P250 and nsp14-P250 variants in isolated infections ( Figure 4D ). Thus, mutations 227 in nsp12 and nsp14 that arose in the ExoN-AA background were detrimental to replication, 228 mutagen sensitivity, and competitive fitness in the presence of a fully-reverted ExoN-DE. These 229 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint Our results also extend existing studies of CoV ExoN motif I. Motif I AAàDE mutations in the 249 SARS-CoV nsp14-ExoN dramatically reduce nuclease activity in biochemical assays, but no 250 study has examined the contributions of each residue independently (16, 18) . Intermediate 251 revertants of ExoN-AA did not display consistent or statistical differences in replication, 5-FU 252 sensitivity, or competitive fitness relative to ExoN-AA, suggesting that they remain 253 phenotypically ExoN(-) during infection and supporting previous studies that motif I DE is 254 essential for WT ExoN function. Given these results, we were surprised to observe repeated 255 reversion of the ExoN-AE but not the other two 2nt variants, ExoN-DA and ExoN-ED. One 256 potential explanation is that the specific mutational bias of ExoN-AE makes the revertant 257 mutations more accessible than in ExoN-DA or ExoN-ED. Alternatively, if ExoN-AE has 258 profound replication or fitness defects, selection could drive primary reversion more quickly 259 away from this genotype. Consistent with this hypothesis, ExoN-AE reverted more quickly at a 260 higher MOI, where natural selection acts more efficiently on a larger population size (Table 1) Consistent with this hypothesis, the fitness effects of mutations in nsp12-P250 and nsp14-P250 269 differ based on the motif I genotype; they are beneficial in ExoN-AA but detrimental in the WT-270 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint ExoN-DE background. In previous studies, it has been difficult to determine whether the fitness 271 defects in ExoN(-) CoVs are directly linked to low-fidelity replication or through some other 272 mechanism. Our data suggest that the proofreading function of nsp14-ExoN can be uncoupled 273 from its more general role in replication (Figure 4) The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint Cloning and recovery of recombinant viruses. Site-directed mutagenesis in MHV genome 306 fragments was performed using \\\"round the horn\\\" PCR (originally described in (33)). Briefly, 307 adjacent primers containing the mutation of interest were 5¢-phosphorylated using T4 308 polynucleotide kinase (NEB, M0201S) using the buffer from the T4 DNA ligase, which contains 309 ATP (M0202S). PCR was performed on a plasmid template using the Q5 High-fidelity 2x previously (9). Briefly, genomic RNA was detected with a 5' 6-carboxyfluorescein (FAM) and 341 3' black hole quencher 1 (BHQ-1) labeled probe targeting nsp2 (Biosearch Technologies,  342 Petaluma, CA), and RNA copy number was calculated by reference to an RNA standard derived 343 from the MHV A fragment. Samples were plated in technical duplicate to minimize well-to-well 344 variation. Titers were determined by plaque assay in DBT-9 cells, and specific infectivity was 345 calculated as PFU per supernatant genomic RNA copy. 346 347 5-fluorouracil sensitivity assays. Stock solutions of 5-fluorouracil (Sigma F6627) were 348 prepared in dimethyl sulfoxide (DMSO). Sensitivity assays were performed in 24-well plates at 349 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint MOI = 0.01 PFU/cell, as previously described (7) were averaged, and values were excluded from subsequent analysis if the duplicate wells differed 371 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint by > 0.5 Ct. The relative abundance of competitor and reference were determined by subtracting 372\\r\\n Ct thresholds (DCtcompetitor = Ctcompetitor -Ctreference) and converted to reflect the fold-change in 373 ratio (Dratio = 2 -DCt competitor ). The log10Dratio was plotted against passage number, and the change 374 in log10Dratio (i.e. slope of linear regression) is the relative fitness. Note that regressions were fit 375 only through P1-P4, as slight deviations in 1:1 ratio in the input (P0) can skew the slope. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint. 9 also deleted ORF 4a, which is dispensable for MHV replication in cell culture (52). The C-150 terminal region of ns2 within MHV-ExoN(-) P250 was truncated and fused to HE with a -1 151 frameshift. Ns2 is a phosphodiesterase (PDE) that protects viral RNA by degrading 2¢\\uf0a2à\\uf0e05¢\\uf0a2 152 oligoadenylate, the activating factor for cellular RNase L (53-55). The portion of ns2 deleted in The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint similar for all four viruses (Fig. 4B) . These data suggest that the increased replication of P250 173 viruses relative to WT-MHV is not fully accounted for by increased RNA synthesis. In addition 174 to RNA synthesis defects, ExoN(-) CoVs have up-to 20-fold increased mutation frequencies and 175 profoundly increased sensitivity to nucleoside and base analogs relative to WT CoVs (13, 14, 16, 176 17, 38). To determine whether nucleoside analog sensitivity of MHV-ExoN(-) was altered by 177 long-term passage, we treated cells infected with parental and passaged viruses with the base 178 analog, 5-fluorouracil (5-FU). 5-FU is converted intracellularly into a nucleoside analog that 179 incorporates into growing RNA strands and causes A:G and U:C mutations. For simplicity, we 180 hereafter refer to 5-FU as a nucleoside analog. Incorporation of 5-FU is increased in the absence 181 of ExoN activity (16). All viruses displayed a concentration-dependent decrease in viral titer but 182 differed greatly in their susceptibility to 5-FU (Fig. 4C ). At 120µM, WT-MHV P3 titers were 183 reduced by ~1 log 10 , while MHV-ExoN(-) P3 titers were undetectable (> 5 log 10 -fold reduction). 184\\r\\n WT-MHV 5-FU sensitivity was not altered by passage. MHV-ExoN(-) P250 was less susceptible 185 than MHV-ExoN(-) P3 to 5-FU treatment, with only a ~1.5 log 10 decrease in titer at 120 µM. replication cycles in which 5-FU can be incorporated (59) . MHV-ExoN(-) P250 had multiple 193 mutations in the spike glycoprotein, including one in the spike furin cleavage site that reduced 194 syncytia formation. To test whether the spike mutations manifested in resistance to 5-FU, we 195 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. However, M288T is not predicted to catalyze nucleotidylation. Within nsp14, 4 NS mutations 215\\r\\n were identified in the ExoN domain, and 2 NS mutations were in the C-terminal N7-216 methyltransferase domain (Fig. 6B ). We next modeled the structure of MHV nsp14 using Phyre 2 217 software (63) , resulting in highest-probability similarity to the SARS-CoV nsp14-nsp10 complex 218 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint between the structures) of 100% for residues 3-519 of MHV-nsp14. The model predicts that five 220 mutations are located close to surface of the protein (Fig. 6B ). All three modeled zinc finger 221 domains contain one NS mutation (F216Y, Y248H, L473I). Two mutations, D128E and F216Y, 222 are located near the interface between nsp10 and nsp14, though neither site has previously been 223 implicated in nsp10-nsp14 interaction (15, 64, 65). One NS mutation resulted in a D272E 224 substitution in ExoN motif III, a metal-coordinating active site residue. We previously reported 225 that alanine substitution of D272 results in an ExoN(-) phenotype (14), but the viability or 226 phenotype of a D272E substitution was not tested in that study. These data suggest that a 227 network of residues evolved to regulate nsp12 and nsp14 activity or stability in the ExoN(-) 228 background. 229 230 Fixed mutations in nsp12 and nsp14 in MHV-ExoN(-) P250 directly correlate with 231 increased resistance to multiple nucleoside analogs. To determine approximately when the 232 mutations in nsp12 and nsp14 arose, we performed di-deoxy sequencing across these protein-233 coding regions roughly every 20 passages (P10, 31, 50, 72, 90, 100, 120, 140, 160, 180, 200, 220, 234 240) . By this method, we detected consensus NS mutations at P10, P50, and P160 for nsp12, and 235 at P50 and P160 for nsp14 (Fig. 6) . Both nsp12 and nsp14 carried their full complement of P250 236 consensus mutations by P160, except for a minority variant (D913E) in nsp12 maintained at 237 <50% of the population between P200 and P250. These passage levels correlated with increased 238 replication of MHV-ExoN(-) (Fig. 2B ) and with decreasing sensitivity to 5-FU (Fig. 7A ). Neither 239 replication nor 5-FU sensitivity of MHV-ExoN(-) changed substantially between P160 and P250. 240\\r\\n To determine whether MHV-ExoN(-) evolved increased resistance to multiple nucleoside 241 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. implicating a complex evolutionary interaction between these two proteins. The measured fitness 284 correlated with the patterns of nucleoside analog resistance and RNA synthesis associated with 285 mutations in nsp12 and nsp14, suggesting a link between the evolution of these virus phenotypes. 286 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint\\r\\n The result also confirms that nsp12 and nsp14 are important but not sufficient to account for the 287 significantly increased fitness of MHV-ExoN(-) P250 relative to MHV-ExoN(-) P3. 288 289 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. Mutations conferring increased replication fidelity to RNA viruses have most frequently been 334 mapped to RdRps (24, 25, 30, 72) . Three findings suggest that mutations within nsp12-P250 335 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. with nsp12, acting as a primase/processivity factor (74, 75) and a helicase/NTPase, respectively 358 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. \\r\\n Cell culture. DBT-9 (delayed brain tumor, murine astrocytoma clone 9) cells were maintained 409 as described previously (91) The virus titer of each stock was determined by plaque assay using DBT-9 cells as described 425 previously (14, 91). For plaque assays of viruses containing the spike protein from MHV-ExoN(-426 ) P250, which does not form syncytia, plaques were visualized with neutral red (Sigma #N6264, 427 diluted 1:10 in PBS containing calcium and magnesium). Neutral red was added 24 hours after 428 plating and incubated for an additional 3-8 hours before formaldehyde fixation. Plaque 429 purification was performed by infecting DBT cells with serial dilutions of virus and overlaying 430 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint the cultures with agar. Single plaques were isolated, resuspended in PBS containing calcium and 431 magnesium, and inoculated onto fresh DBTs. This process was completed 3 times before 432 experimental stocks were generated, as above. and RNA copy number was calculated by reference to an RNA standard derived from the MHV 475 A fragment. Samples were plated in technical duplicate to minimize well-to-well variation. Titers 476 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint were determined by plaque assay in DBT-9 cells, and specific infectivity was calculated as PFU 477 per supernatant genomic RNA copy. 478\\r\\n Nucleoside and base analog sensitivity assays. 5-azacytidine (AZC), 5-fluorouracil (5-FU), and 480 ribavirin (RBV) were purchased from Sigma (product numbers A2385, F6627, and R9644, 481 respectively). Stock solutions of 5-FU and RBV were prepared in dimethyl sulfoxide (DMSO). 482\\r\\n 2'-C-methyladenosine (CMeA) was received from Gilead Sciences, Inc (Foster City, CA). 483\\r\\n Sensitivity assays were performed as described previously (16) viruses were detected with the same probe used in specific infectivity analyses (14). Reference 516 viruses were detected by a probe targeting the same region but with 10 silent mutations (5'-517 TCCGAACTACTGCAACCCCAAGTG-3') and labeled with 5' Quasar 670 and 3' black hole 518 quencher 2 (BHQ-2) (Biosearch Technologies, Petaluma, CA). RNA copy number was 519 calculated by reference to an RNA standard generated by in vitro transcription of the 520 corresponding MHV A fragment, and relative RNA abundance was calculated as the ratio of 521 competitor genomes to reference genomes. 522 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint and synonymous mutations (purple) are below. 564 The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint statistical significance was determined using one-way ANOVA. For Panel C, statistical 587 significance for change in titer of MHV-ExoN(-) spike-P250 relative to MHV-ExoN(-) P3 was 588 determined using Mann-Whitney test (*P<0.05, **<0.01, ***P<0.001, ns = not significant). 589 The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint tuberculosis DNA polymerase III αβ2ε replicase promotes polymerization and reduces 855 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. Virus titer (log 10 PFU/ml) All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint. \""
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "text(fd)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Languages.English()"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "language(fd)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "45119-element Array{String,1}:\n",
       " \"Odor\"\n",
       " \"perception\"\n",
       " \",\"\n",
       " \"and\"\n",
       " \"emotional\"\n",
       " \"and\"\n",
       " \"behavioral\"\n",
       " \"responses\"\n",
       " \"to\"\n",
       " \"odors\"\n",
       " \"strongly\"\n",
       " \"depend\"\n",
       " \"on\"\n",
       " ⋮\n",
       " \"the\"\n",
       " \"author/funder.\"\n",
       " \".\"\n",
       " \"https://doi.org/\"\n",
       " \"10.1101\"\n",
       " \"/\"\n",
       " \"175562\"\n",
       " \"doi\"\n",
       " \":\"\n",
       " \"bioRxiv\"\n",
       " \"preprint\"\n",
       " \".\""
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "tokens(fd)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "using WordTokenizers"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "FileDocument(\"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\", TextAnalysis.DocumentMetadata(Languages.English(), \"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\", \"Unknown Author\", \"Unknown Time\"))"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "fd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "45119-element Array{String,1}:\n",
       " \"Odor\"\n",
       " \"perception\"\n",
       " \",\"\n",
       " \"and\"\n",
       " \"emotional\"\n",
       " \"and\"\n",
       " \"behavioral\"\n",
       " \"responses\"\n",
       " \"to\"\n",
       " \"odors\"\n",
       " \"strongly\"\n",
       " \"depend\"\n",
       " \"on\"\n",
       " ⋮\n",
       " \"the\"\n",
       " \"author/funder.\"\n",
       " \".\"\n",
       " \"https://doi.org/\"\n",
       " \"10.1101\"\n",
       " \"/\"\n",
       " \"175562\"\n",
       " \"doi\"\n",
       " \":\"\n",
       " \"bioRxiv\"\n",
       " \"preprint\"\n",
       " \".\""
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "tokenize(text(fd))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "FileDocument(\"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\", TextAnalysis.DocumentMetadata(Languages.English(), \"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\", \"Unknown Author\", \"Unknown Time\"))"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sent_files = FileDocument(\"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "\"Odor perception, and emotional and behavioral responses to odors strongly depend on experience, and learned odor-context associations often last for the lifetime of an animal (Mouly and Sullivan, 2010) . The cellular and neural circuit mechanisms underlying olfactory learning and memory, however, remain poorly understood.\\r\\n Recent studies on episodic and contextual learning in hippocampal neural networks have suggested that memories are encoded in the activity of distributed ensembles of neurons, often referred to as a 'memory trace' (Mayford and Reijmers, 2015; Poo et al., 2016; Tonegawa et al., 2015) . The neurons constituting such a memory trace are thought to encode information about the environmental context and associated emotions of past experiences, and their activity is necessary and sufficient for memory retrieval (Liu et al., 2012; Reijmers et al., 2007; Tanaka et al., 2014) .\\r\\n Here, we investigate the organization of odor memory traces in the olfactory (piriform) cortex of mice. The piriform cortex, a trilaminar paleocortical structure, is the largest cortical area receiving direct afferent inputs from the olfactory bulb, which, in turn, receives topographically organized inputs from olfactory sensory neurons in the nose. Individual piriform neurons respond to combinatorial inputs from multiple Piriform cortex has long been hypothesized to support auto-associative network functions that can retrieve previously learned information from partial or degraded sensory inputs (Haberly, 2001; Wilson and Sullivan, 2011) . Piriform pyramidal cells form a large recurrent network, which is reciprocally connected with adjacent highorder associative areas including the prefrontal, entorhinal and perirhinal cortex and the amygdala (Johnson et al., 2000; Sadrian and Wilson, 2015) . Storage of information is made possible by NMDA-dependent, associative plasticity of connections (Johenning et al., 2009; Kanter and Haberly, 1990; Quinlan et al., 2004) .\\r\\n Furthermore, changes in piriform network activity (Chapuis and Wilson, 2011; Chen et al., 2011; Li et al., 2008; Sevelinges et al., 2004) and stabilization of piriform odor representations (Shakhawat et al., 2015) have been observed after associative olfactory learning. Finally, excitotoxic lesions of the posterior piriform cortex in rats perturb odor fear memories (Sacco and Sacchetti, 2010) , and optogenetic stimulation of artificial piriform ensembles is sufficient to drive learned behaviors (Choi et al., 2011) . Taken together, these studies have led to the hypothesis that piriform neural ensembles encode olfactory memory traces.\\r\\n To test this hypothesis, we have developed an intersectional genetic strategy in mice to target piriform neurons that are activated by olfactory experience. We employed\\r\\n To selectively label and manipulate piriform neurons that were activated during odor exposure, we used cFos-tTA transgenic mice (Reijmers et al., 2007) , in which the activity-dependent cFos promoter drives expression of the tTA transcription factor, and we stereotaxically injected in their piriform cortex Adeno-Associated Viruses (AAVs) expressing Designer Receptors Exclusively Activated by Designer Drugs (DREADDs) under the control of the tTA-responsive promoter tetO ( Figure 1A) .\\r\\n DREADDs (Alexander et al., 2009; Ferguson et al., 2011) increase (hM3Dq:mCherry) or decrease (HA:hM4Di-IRES-mCitrine) the excitability of neurons upon activation by their ligand clozapine N-oxide (CNO). Temporal control of DREADD expression is provided by doxycycline, administered through the diet, which interferes with the binding of tTA to tetO and thus suppresses transgene expression (Gossen et al., 1995) . Mice maintained in their home cage on a doxycycline-containing diet showed low basal DREADD expression, detected by anti-HA immunohistochemistry and native mCherry fluorescence (Figure 1B and C) . To induce DREADD expression, mice were taken off doxycycline five days before odor presentation. Exposure to odor followed by foot shock (see Experimental Procedures) resulted in the Fos-tagging of sparse neural ensembles (median (interquartile range): 1.6 (0.3) % of piriform neurons, n=2 for hM4Di; 2.2 (0.5) % of piriform neurons, n=3 for hM3Dq), which were broadly dispersed throughout the anterior and posterior piriform cortex ( Figure 1B and C), consistent with previous observations of endogenous cFos expression (Datiche et al., 2001; Illig and Haberly, 2003) .\\r\\n To test whether CNO-mediated activation of DREADDs in Fos-tagged piriform neurons alters their excitability, we performed whole-cell recordings in acute brain author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint slices containing piriform cortex. DREADD-expressing neurons were identified based on mCitrine (HA:hM4Di-IRES-mCitrine) or mCherry (hM3Dq:mCherry) fluorescence. Figure 1) . In the absence of the DREADD ligand CNO, the resting membrane potential of DREADD-expressing piriform neurons was indistinguishable from DREADD-negative cells (-71.70 (4.37) mV in n=24, (4.32) mV in hM4Di-expressing cells, n=14, -72.56 (3.92) mV in hM3Dq-expressing cells, n=15, p=0.631, Kruskal-Wallis ANOVA, Figure 1D ). After bath application of CNO (5 µM), the membrane properties selectively changed in DREADD-expressing cells: hM4Di-expressing cells hyperpolarized (-74.51 (5.17 ) mV, n=12, p=0.007, Wilcoxon Signed Ranks Test), and hM3Dq-expressing cells depolarized (-66.95 (4.12) mV, n=13, p=0.001, Wilcoxon Signed Ranks Test), whereas the resting membrane potential of DREADD-negative cells remained unchanged (-70.28 (3.81) mV, n=15, p=0.890, Wilcoxon Signed Ranks Test, Figure 1D ). The relative changes were 1.00 (0.03) in DREADD-negative cells (n=15, p=0.890, One Sample Wilcoxon Signed Rank Test), 1.04 (0.03) in hM4Di-expressing cells, (n=12, p<0.001, One Sample Wilcoxon Signed Rank Test) and 0.91 (0.09) in hM3Dq-expressing cells, (n=13, p=0.001, One Sample Wilcoxon Signed Rank Test), Figure 1E ). To determine the impact of CNO treatment on neuronal excitability, we delivered depolarizing current steps and compared the number of action potentials triggered by 60 pA step current above action potential threshold (see Experimental Procedures). We found that in hM4Di-expressing cells, the number of evoked action potentials significantly decreased after CNO application (3.75 (1.59) vs. 0.30 (3.48), n=10, p=0.010, Wilcoxon Signed Ranks Test), whereas in hM3Dq-expressing cells, the number of author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint action potentials significantly increased after CNO application (8.67 (9.10) vs. 19.0 (27.6) mV, n=11, p=0.006, Wilcoxon Signed Ranks Test). In contrast, DREADDnegative cells did not exhibit a significant change in the number of evoked action potentials upon CNO application (4.33 (1.67) vs. 4.67 (3.00), n=9, p=0.672, Wilcoxon Signed Ranks Test, Figure 1F ). The relative changes were 1.00 (0.41) in DREADDnegative cells, (n=9, p=0.641, One Sample Wilcoxon Signed Rank Tests), 0.09 (0.81) in hM4Di-expressing cells (n=10, p=0.010, One Sample Wilcoxon Signed Rank Tests) and 1.81±1.09 in hM3Dq-expressing cells (n=11, p=0.004, One Sample Wilcoxon Signed Rank Tests, Figure 1G ). Taken together, these experiments show that Fos-tagging during olfactory fear conditioning marks a sparse and dispersed subpopulation of piriform neurons, and that CNO-mediated activation of hM4Di or hM3Dq expressed in Fos-tagged neurons selectively decreases or increases their excitability.\\r\\n If Fos-tagged piriform neurons that were activated during olfactory learning constitute an essential component of an olfactory memory trace, then perturbing the activity of these ensembles should interfere with memory recall. To test this prediction, we Fostagged piriform neurons during olfactory fear conditioning, and then chemogenetically silenced Fos-tagged ensembles during odor fear memory recall. cFos-tTA transgenic mice (\\\"experimental group\\\", n=8) were bilaterally injected in piriform cortex with the AAV-tetO-hM4Di vector. Wild-type mice injected with the same virus served as a control (\\\"control group\\\", n=8). During training, the conditioned odor (CS+) was presented at one of the ends of the conditioning box and paired with a foot shock, applied to the side of the box where the odor was delivered (see Experimental author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint Procedures and Figure 2A ). Mice thus learned to escape from the conditioned stimulus (CS+) by running towards the opposite side of the box. Two control odors, whose presentation was not paired with foot shock (CS-), were used to entrain odorspecific as opposed to generalized fear responses (Chen et al., 2011) . Doxycycline was removed from the diet of mice prior to training, to permit the induction of hM4Di expression in active Fos-expressing piriform neurons of mice of the experimental group. Mice were returned to doxycycline-containing diet immediately after odor fear conditioning, and memory recall of the odor-foot shock association was tested three days later by presenting the odors alone in a new box ( Figure 2B and C). All mice were intraperitoneally (i.p.) injected with CNO before testing, to exclude potential offtarget effects by CNO (Gomez et al., 2017) . For all mice, we verified viral gene expression by post hoc histological examination. Previous studies have suggested that the posterior piriform cortex is important for associative memory encoding (Sacco et al., 2010; Calu et al., 2008; Haberly 2001) . We therefore excluded from our analysis mice in which parts of the posterior piriform cortex were spared from viral infection (Supplementary Figure 2) . We combined two parameters to quantify the learned escape behavior: the position of the mouse in the box, and the increase in its maximum velocity after odor presentation ( Figure 2B ). These parameters were chosen because they described well the behavioral response of mice to the odor-foot shock pairing during training (Supplementary Figure 3) .\\r\\n As expected, mice in the control group exhibited robust escape behavior upon CS+ presentation. The median position of the mice in the box at the end of stimulus delivery was 0.74 (with 0 defined as the site of odor presentation and 1 as the opposite end of the box), and mice exhibited a 1.9-fold increase of their maximum velocity ( Figure 2E, G and H) . In contrast, escape behavior was significantly reduced author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint in mice in which Fos-tagged neurons were silenced. Their median position in the box at the end of stimulus delivery, (0.42), and their 1.1-fold increase in maximum velocity were significantly lower than in the control group (Mann-Whitney test, U=7, p=0.007 for both position and velocity, Figure 2E , G and H). The difference in the position of mice after CS+ and CS-presentation was also significantly different between the control and experimental groups (Mann-Whitney test, U=13, p=0.0499, Figure 2F ). Differences in behavioral responses of cFos-tTA transgenic mice were specifically due to hM4Di activation, as escape behavior after CNO injection was unaffected in cFos-tTA transgenic mice injected with AAVs expressing the calcium indicator GCaMP (n=9, Supplementary Figure 4 ).\\r\\n All mice were similarly close to the odor port at the onset of CS+ odor presentation (Mann-Whitney test, U=28, p=0.7209, Figure 2D ), and had similar maximum velocities during the 7 s time period that preceded odor presentation (Mann-Whitney test, U=24, p=0.4418, Figure 2G ). These observations exclude any inherent differences between groups due to how the CS+ was presented. Furthermore, escape behavior in control mice was specific to the CS+, as none of the mice ran away from the CS-( Figure 2D , E and H). Finally, we did not observe any differences in the overall mobility of mice throughout the entire testing session. The median speed of mice of all groups (0.68 (0.28) cm/s and 0.46 (0.14) cm/s for the control and experimental groups respectively) was not significantly different (Mann-Whitney test, U=22, p=0.3282), nor did mice show a bias for the left or right side of the testing box (time spent on the left side divided by time spent on the right side: 1.36 (0.72) and\\r\\n 1.51 (0.52) for the control and experimental groups respectively, Mann-Whitney test, U=27, p=0.6454). Together, these data suggest that Fos-tagged piriform ensembles that were activated during olfactory fear conditioning are necessary for odor fear author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint memory recall.\\r\\n Even though cFos-tTA transgenic mice expressing hM4Di failed to robustly escape from the CS+, it should be noted that they behaved differently after CS+ and CSpresentation. They were further away from the odor port after CS+ presentation than after CS-presentation (Wilcoxon matched-pairs signed rank test, W=-36, p=0.0078, Figure 2E ), and their maximum velocity ratio was significantly higher (Wilcoxon matched-pairs signed rank test, W=-36, p=0.0078, Figure 2H ). This could indicate a partial memory of the learned association, possibly due to incomplete silencing of Fos-tagged neurons in piriform cortex, or the existence of parallel neural pathways that partially compensate for the loss of piriform functions. Finally, silencing of the piriform cortex in only one hemisphere did not abolish the learned escape behavior to the conditioned odor (Supplementary Figure 5) .\\r\\n It is possible that the chemogenetic silencing of piriform ensembles that were Fostagged during olfactory fear conditioning disrupts odor detection and discrimination rather than selectively affecting odor fear memory recall. To test this possibility, we monitored sniffing behavior of mice during an olfactory habituation/dishabituation assay ( Figure 3A) , a well established test for odor detection and discrimination (Coronas-Samano et al., 2016; Verhagen et al., 2007; Wesson et al., 2008) . Fostagging of piriform neurons during olfactory fear conditioning was performed as described above. Three days later, changes in sniffing behavior in response to odor exposure were tested in a plethysmograph, while hM4Di-expressing piriform neurons author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint in cFos-tTA transgenic mice were silenced using i.p. injection of CNO (see Experimental Procedures). We observed that mice of both experimental (n=8) and control (n=6) groups increased their sniff frequency upon presentation of a novel, neutral odor (1.6 fold increase; Wilcoxon matched-pairs signed rank test, W=21, p=0.0312 and W=36, p=0.0078 for the control and experimental groups, respectively; Figure 3B and C). Repeated exposure to the same odor (neutral odor or CS-) resulted in a decrease in the sniff frequency, reflecting habituation after 4 consecutive exposures (Figure 3C and D) . Finally, presentation of the CS+ increased sniff frequency (1.4 fold increase; Wilcoxon matched-pairs signed rank test, W=21, p=0.0312 and W=34, p=0.0156 for the control and experimental groups, respectively, Figure 3D ). Such changes in odor sampling behavior have been shown to report the detection and discrimination of different odor stimuli (Coronas-Samano et al., 2016; Verhagen et al., 2007; Wesson et al., 2008) . Therefore, these data suggest that basic behaviors characteristic of odor sampling, detection and discrimination were unaffected by the silencing of piriform ensembles that were Fostagged during olfactory fear conditioning.\\r\\n Odors activate unique but overlapping ensembles of piriform neurons (Bolding and Franks, 2017; Poo and Isaacson, 2009; Roland et al., 2017; Stettler and Axel, 2009 ). Therefore, silencing neurons that respond to one odor could partially interfere with the retrieval of information associated with other odors. To test this prediction we modified the behavioral protocol to separate Fos-tagging from olfactory fear conditioning. cFos-tTA transgenic and wild-type control mice were injected with AAV-tetO-hM4Di, and expression of hM4Di in piriform neurons was induced while mice author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint were exposed to odor (eugenol or beta-citronellol) in a neutral environment, without subsequent foot shock. Fos-tagging resulted in the labeling of sparse neural ensembles, similar to those labeled during odor-foot shock association (1.1 (0.3) % of piriform neurons, n=3). Mice were returned to doxycycline-containing diet and trained two days later to associate a different odor (ethyl-acetate, CS+) with foot shock.\\r\\n Behavioral testing of the learned escape behavior in response to the CS+ was performed one day later, after i.p. injection of CNO (see Experimental Procedures and Figure 4A ). We found that learned escape behavior of cFos-tTA transgenic mice expressing hM4Di (experimental group, n=12) was similar but somewhat attenuated compared to wild-type controls (n=10). Both groups exhibited an escape behavior after CS+ presentation, indicated by the significant increase in the maximum velocity (Wilcoxon matched-pairs signed rank test, W=55, p=0.0020 and W=56, p=0.0269 for the control and experimental groups respectively; no difference in the maximum velocity ratio after/before between groups, Mann-Whitney test, U=35, p=0.1071, Figure 4E and F). The difference in the position of mice after CS+ and CSpresentation was also similar between the control and experimental groups (Mann-Whitney test, U=37, p=0.1402, Figure 4D ). However, we observed that silencing neutral odor representations somewhat dampened the behavioral response to the conditioned stimulus, as behavioral responses were less robust compared to controls when quantifying the position of mice in the box after CS+ presentation (Mann-Whitney test, U=22, p=0.0112, Figure 4B and C). The attenuated behavioral responses we observe likely reflect the partial degradation of odor information as a result of the silencing of piriform neurons responsive to both the CS+ and the neutral odor.\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint\\r\\n If piriform neural ensembles that were activated during olfactory fear conditioning encode an odor fear memory trace, then reactivation of these neurons may be sufficient to trigger memory recall. To test this prediction, we devised a modified behavioral task to assess memory recall. We monitored exploratory behavior in an open field during ambient odor exposure or artificial reactivation of Fos-tagged neurons, a test that does not rely on an escape behavior from an odor originating from a spatially defined odor source ( Figure 5A ). We first trained mice to associate ethyl-acetate, the CS+, with foot shock. Upon ambient exposure to the CS+, diffused above the open field, exploratory behavior of mice was significantly attenuated compared to baseline exploratory behavior measured one day earlier (Wilcoxon matched-pairs signed rank test, n=6, W=-21, p=0.0312). Ethyl-acetate exposure did not affect exploratory behavior of mice that had previously been exposed to ethylacetate without foot shock (Wilcoxon matched-pairs signed rank test, n=5, W=1, p>0.9999). These data suggest that re-exposure of mice to the CS+ causes an increase in anxiety-like behavior, expressed as a decrease in exploratory behavior in the open field.\\r\\n We next repeated the experiment in cFos-tTA transgenic mice, in which the piriform cortex had been infected with the activating DREADD AAV-tetO-hM3Dq (Supplementary Figure 6) . Instead of exposure to the CS+, we i.p. injected the DREADD ligand CNO to reactivate hM3Dq-expressing piriform neurons that were Fos-tagged during odor-foot shock association (see Experimental Procedures and Figure 5A ). We found that CNO-mediated reactivation, similar to CS+ exposure, caused a significant decrease in exploratory behavior of mice that had undergone author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint olfactory fear conditioning (Wilcoxon matched-pairs signed rank test, n=8, W=-34, p=0.0156, Figure 5B and C). CNO-mediated reactivation of piriform neurons that were Fos-tagged during ethyl-acetate exposure without foot shock did not affect exploratory behavior of mice (Wilcoxon matched-pairs signed rank test, n=7, W=-10, p=0.4688, Figure 5B and C). Taken together, these data suggest that chemogenetic reactivation of piriform neurons that were active during odor-foot shock exposure is sufficient to trigger fear memory recall.\\r\\n We next asked whether artificial memory recall depends on the specificity of the Fostagged neural ensemble: is piriform reactivation sufficient to trigger fear memory recall, as long as it includes the CS+-tagged ensemble? For this purpose, we generated a synthetic Fos-tagged ensemble, by sequentially exposing mice to ethylacetate paired with foot-shock (CS+) and neutral odors (CS-, eugenol and betacitronellol) ( Figure 5D ). While exposure to the CS+ resulted in anxiety-like behavior similar to what was observed in the previous experiment (Wilcoxon matched-pairs signed rank test, n=6, W=-21, p=0.0312, Figure 5E and F), artificial reactivation of synthetic Fos-tagged ensemble did not result in changes in exploratory behavior that would indicate memory recall (Wilcoxon matched-pairs signed rank test, n=8, W=-4, p=0.8438, Figure 5E and F). These data suggest that during artificial reactivation, piriform cortex cannot extract meaningful information from the synthetic representation generated by the sequential presentation of CS+ and CS-odors. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint\\r\\n The piriform cortex has long been thought to provide the substrate for storing associative olfactory memories. Several studies have shown that piriform cortex cells can encode information that carries behavioral significance (Calu et al., 2008 , Mandairon et al., 2014 , Sacco et al., 2010 ), yet their functional relevance has never been assessed by direct experimental manipulation. Using a cFos-dependent, intersectional genetic approach to visualize and manipulate piriform neurons activated during olfactory fear conditioning (Figure 1 ), we found that chemogenetic silencing of Fos-tagged piriform ensembles abolished a learned odor escape behavior (Figure 2 ). Silencing these Fos-tagged ensembles did not alter basic odor detection and discrimination (Figure 3) . Furthermore, chemogenetic silencing of neurons responsive to a neutral odor only moderately attenuated odor fear memory recall, suggesting that Fos-tagged piriform ensembles are largely odor-specific ( Figure 4) . Finally, specific chemogenetic reactivation of piriform ensembles that were active during learning resulted in reduced exploratory behavior in an open field assay, an effect that was similarly observed when exposing mice to the conditioned odor stimulus (Figure 5) . Together, our experiments identify piriform neurons that were active during learning as necessary and sufficient to trigger odor fear memory recall. This indicates the formation of functional connections between Fos-tagged piriform cells and other associative areas and firmly establish piriform ensembles as an essential component of odor fear memory traces.\\r\\n Artificial reactivation of an olfactory memory trace author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint Reactivation of neurons by CNO-mediated activation of DREADD receptors does not recapitulate the temporal characteristics of piriform odor responses and their modulation by active sampling (Bolding and Franks, 2017; Miura et al., 2012) .\\r\\n Despite this, reactivation of Fos-tagged piriform ensembles activated during odor-foot shock exposure was sufficient to elicit a behavioral response. How piriform neural circuits and downstream target regions process such artificial activity patterns remains to be determined. One possibility is that despite the temporal limitations of memory trace reactivation, piriform network mechanisms can retrieve the perception of the reinforced conditioned stimulus. Consistent with this model, recent studies have shown that spatial patterns of odor-evoked activity, in the absence of precise temporal information, are sufficient to decode odor identity in piriform cortex (Bolding and Franks, 2017; Miura et al., 2012; Roland et al., 2017) . Alternatively, it is possible that reactivation generates a state of stress and anxiety, but without evoking the perception of the odor. Candidate target areas for the processing of odor-fear associations include the basolateral amygdala and the medial prefrontal cortex (Dejean et al., 2016; Gore et al., 2015) , however, the relevant neural circuit components remain to be identified. Finally, reactivating a synthetic ensemble of neurons activated during exposure to the reinforced stimulus (CS+) and the nonreinforced stimuli (CS-) did not elicit a measurable behavioral response. This observation suggests that behaviorally relevant information cannot be extracted from a neural ensemble representing conflicting (aversive versus neutral) information. This result is consistent with the finding that the reactivation of an artificial contextual memory in the hippocampus competes with the retrieval of a learned context-shock association (Ramirez et al., 2013) . author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint\\r\\n A general limitation of the cFos-tTA system is that the tagging of active neurons is transient. The duration of the expression of tTA-dependent proteins is determined by the time course of induction and the stability of cFos-tTA-dependent transcripts and proteins in tagged neurons. Previous studies have shown that TetTag-dependent expression of regulators of neural activity lasts for at least five days but becomes undetectable by thirty days (Cowansage et al., 2014; Liu et al., 2012; Zhang et al., 2015) . We therefore restricted our manipulations of neural activity and behavioral analyses to a three-day time window after Fos-tagging of neurons during olfactory conditioning. Another constraint of the Fos-tagging system is its slow temporal dynamics. As a consequence, neurons responding to the reinforced conditioned stimulus (CS+), the odorant paired with foot shock, and neurons responding to the non-reinforced conditioned stimuli (CS-), the odorants not associated with foot shock, were tagged in our silencing experiments. The use of a CS-compromises the specificity of the Fos-tagged neural ensembles. Nevertheless, the CS-provides an important experimental readout for the odor selectivity of the behavioral response.\\r\\n Interestingly, the number of Fos-tagged piriform neurons we observe is significantly lower than the number of odor-responsive neurons detected in electrophysiological recordings in awake mice (Bolding and Franks, 2017; Miura et al., 2012; Zhan and Luo, 2010) . However, in these experiments, a large fraction of cells exhibited low firing rates. Therefore, the Fos-tagged population could represent a subpopulation of neurons that is strongly activated by odor (Schoenenberger et al., 2009) . Fos-tagging could also mark plastic changes supporting memory formation (Cole et al., 1989; Minatohara et al., 2015) . Indeed, cFos mRNA levels decrease significantly after injection of an antagonist of the NMDA receptor, a key player in the induction of author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint synaptic plasticity (Tayler et al., 2011) , and long-term memory and synaptic plasticity are impaired when cFos production is perturbed in the central nervous system (Fleischmann et al., 2003; Seoane et al., 2012) . Future studies will allow to directly test these hypotheses.\\r\\n The hippocampus has been studied extensively for its role in spatial and contextual memory (Basu and Siegelbaum, 2015) . Recently, Fos-tagging of hippocampal neurons during contextual fear conditioning has provided important insight into the cellular and neural circuit mechanisms of learning and memory (Cai et al., 2016; Garner et al., 2012; Liu et al., 2012; Ramirez et al., 2013; Roy et al., 2017; Ryan et al., 2015) . However, if principles of memory formation and storage in hippocampusrelated neural networks apply to other cortical structures remains largely unknown.\\r\\n Interestingly, pirform cortex and hippocampus share a similar circuit organization and both have been modeled as auto-associative networks (Haberly, 2001) . Consistent with this model, our findings reveal striking similarities between olfactory memory traces in the piriform cortex and memory traces of contextual fear in the hippocampus. In both piriform cortex and hippocampus, learning activates sparse and distributed ensembles of neurons that appear to lack topographic organization. Furthermore, hippocampal neurons tagged during contextual fear conditioning, and piriform neurons tagged during olfactory fear conditioning were both necessary and sufficient for memory retrieval (Liu et al., 2012; Tanaka et al., 2014) , The olfactory system is particularly well-suited for studying learning and memory. Indeed, odor memories and olfactory-driven behaviors are robust in animals, including in mice, providing highly reliable and quantitative behavioral readouts. Furthermore, the author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint olfactory cortex is only two synapses away from the peripheral olfactory sensory neurons in the olfactory epithelium, and the neural inputs that drive piriform cortex activity have been well characterized (Banerjee et al., 2015; Dhawale et al., 2010; Economo et al., 2016; Fukunaga et al., 2012; Kato et al., 2012; Roland et al., 2016; Yamada et al., 2017) . Knowing the properties of neural input patterns will provide the basis for understanding neural circuit input-output computations and their modification by learning and experience.\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint\\r\\n Mice were housed at 24°C with a 12-hour light/12-hour dark cycle with standard food and water provided ad libitum. Mice were group-housed with littermates until the beginning of surgery and then single-housed in ventilated cages throughout the duration of the experiment. cFos-tTA mice (Reijmers et al., 2007) were fed a diet containing 1g/kg doxycycline for a minimum of 4 days before surgery. Wild-type control animals were siblings that did not carry the cFos-tTA transgene. The age of mice at the time of behavioral testing ranged from 10 to 15 weeks. Male mice were used for the hM4Di-mediated neural silencing, and females for the hM3Dq-mediated neural activation experiments. Experiments were carried out according to European and French national institutional animal care guidelines (protocol APAFIS#2016012909576100).\\r\\n The Adeno-associated viruses (AAVs) were generated at Penn Vector Core, University of Pennsylvania (serotype 8, 10 13 genome copies/mL, 1:2 dilution with sterile PBS on the day of injection).\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint\\r\\n Mice were anaesthetized with ketamine/xylazine (100 mg/kg/10 mg/kg, Sigma-Aldrich) and AAV vectors were injected stereotaxically into the piriform cortex Sections were washed 2 times in PBST and 1 time in PBS, mounted on slides and author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint coverslipped with Vectashield mounting medium (Vectorlabs). Images were acquired as Z-stacks (70 to 140 µm in total thickness, step size 7 µm) with a Leica SP5 confocal microscope or as single plane sections with a Zeiss Axio Zoom microscope and processed in Fiji.\\r\\n Counting Z-stacks (7 µm step) were acquired with a Leica SP5 confocal microscope with a 20X objective and a resolution of 512x512 or 1024x1024 pixels (1 pixel: 0:72µm). First, the piriform cortex was delineated on a maximum projection of each stack using a custom-written ImageJ macro. For each mouse, 2 to 4 sections (median volume 9.3e-2 (9.4e-3) mm 3 ) were analyzed, from one or both hemispheres, at Y=-0.8 and Y=-1.6mm relative to bregma.\\r\\n HA-stained neurons (hM4Di) were counted by hand. hM3Dq-mCherry positive neurons were counted using a custom-written ImageJ plugin. After pre-processing the image (Subtract Background, Remove Outliers and Median Filter), the z-stack is thresholded using the RenyiEntropy algorithm. On each slice of the stack, objects with an area smaller than 40µm 2 are removed using the Analyze Particle command in ImageJ. A mask of each slice from the stack is created containing the filled outline of the measured particles. The number of cells is the number of 3D objects detected in the stack. Automated cell detection can be prone to error, thus, the detection efficiency was checked on a few pictures by two people blind to the experimental conditions.\\r\\n The numbers of cells were calculated as numbers of cells per cubic millimeter. An estimate of the total number of neurons (counterstained with NeuroTrace) per cubic millimeter (7.1e4 mm -3 ) was obtained by counting neurons in representative volumes author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint containing the three layers of piriform cortex. Numbers are consistent with those obtained by (Srinivasan and Stevens, 2017) .\\r\\n Parasagittal or coronal slices (300 µm thick) of piriform cortex were prepared from 6-8 week-old cFos-tTA mice injected with AAV-tetO-hM4Di-IRES-mCitrine or AAV-tetO-hM3Dq-mCherry. Animals were anaesthetized with ketamine and xylazine (100 mg/kg/10 mg/kg, Sigma-Aldrich), perfused with ice-cold ACSF (125 mM NaCl, 2.5 mM KCl, 25 mM glucose 25mM NaHCO 3 , 1.25 mM NaH 2 PO 4 , 2 mM CaCl 2 , 1 mM MgCl 2 , 1 mM pyruvic acid, bubbled with 95% O 2 and 5% CO 2 and adjusted to 295±5 mOsm osmolarity), and decapitated. The brain was cooled with ice-cold ACSF solution and then sliced using a 7000SM2 vibrating microtome (Campden Instruments, UK). Slices were incubated in the same solution at 34°C for one hour and then continuously perfused with ACSF solution (2 mL/min) at 34°C in the recording chamber. DREADD expression was detected with two-photon excitation (830nm, ChameleonMRU-X1, Coherent, UK) under a Scientifica TriM Scope II microscope (LaVision, Germany), with a 60x water-immersion objective. Whole-cell recording pipettes with 5-7 MΩ resistance were filled with the following solution (in mM): 122 Kgluconate, 13 KCl, 10 HEPES, 10 phosphocreatine, 4 Mg-ATP, 0.3 Na-GTP, 0.3 EGTA (adjusted to pH 7.35 with KOH). The recording solution also contained the morphological tracer Alexa Fluor 594 (5 µM, red channel) for nonexpressing and hM4Di-positive cells or the Ca 2+ -sensitive dye Fura-2 (300 µM, replacing EGTA in the recording solution, green channel) for hM3Dq-positive cells, to enable identification of the patched cell and to visualize its morphology. The excitability of cells was measured in current-clamp mode by 500 ms steps of current author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint injections from -300 to +500 pA with steps of 20 pA. We compared the number of action potentials triggered by equivalent depolarization at each step 60 pA above the action potential threshold before and after CNO (5 µM) application. The series resistance was usually <20 MΩ, and data were discarded if it changed by more than 20% during the recording. Signals were amplified using EPC10-2 amplifiers (HEKA Elektronik, Lambrecht, Germany). Voltage-clamp recordings were filtered at 5 kHz and sampled at 10 kHz, and current-clamp recordings were filtered at 10 kHz and sampled at 20 kHz, with the Patchmaster v2x32 program (HEKA Elektronik).\\r\\n A training box was used to train mice to escape from an odor. The box was rectangular (L 57 cm, W 17 cm, H 64 cm), with a grid floor made of 72 stainless-steel rods (diameter=6 mm, space between rods=2 mm). Current was delivered by an aversive stimulator (MedAssociates, 115V, 60 Hz). A custom-made switcher allowed an electric foot shock (0.6 mA, 0.6 ms) to be applied independently to either half of the box. A testing box was used to test memory retrieval. Its dimensions were similar to the training box. Materials for the walls and floor were different from the training box to create a different context. The open field box consisted of a white Plexiglas (6 mm thickness) container (50 cm x 50 cm x 38 height).\\r\\n Sniffing behavior was monitored in freely moving mice using a plethysmograph (Emka Technologies). Electrical signals were amplified using RHD 2132 amplifier boards (Intan Technologies) and band-pass filtered (1-30 Hz).\\r\\n Odors were delivered using an 8 channels olfactometer (Automate Scientific).\\r\\n Continuous air flowed through a bottle of mineral oil (Sigma-Aldrich) to avoid pressure changes when presenting the odors. The following odors (Sigma-Aldrich), author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint at a concentration of 1% (vol/vol) in mineral oil were used: ethyl-acetate, betacitronellol, eugenol, limonene, and pinene.\\r\\n AAVs were stereotaxically injected in both hemispheres of piriform cortex of cFos-tTA transgenic mice and littermate wild-type controls. Mice were given 10 days to recover, before being habituated on day 11 to the training and testing/open field/plethysmograph boxes (40 and 10 min each). 24 hours after habituation, doxycycline was removed and replaced by a normal diet. Five days later, mice were fear conditioned to ethyl-acetate or exposed to an odor (odor \\\"C\\\") in a neutral environment. Immediately after fear conditioning or odor presentation, mice were put back on a diet containing doxycycline. During fear conditioning, ethyl-acetate (the CS+) was presented at one extremity of the training box, and the corresponding halfside of the box' floor was electrified for 0.6 s (0.6 mA). The foot shock was delivered 4 seconds after CS+ presentation, and the CS+ lasted for an additional 2 seconds.\\r\\n Mice learned to escape to the opposite side when the CS+ was presented. The CS+ was presented 4 times, 2 times on each side of the box. When fear conditioning included presentation of two non-reinforced conditioned stimuli (CS-1 and CS-2), the choice between CS+ and CS-as well as the side of presentation was pseudorandomized. Training always started with one presentation of each CS-followed by the CS+. The two CS-odors were presented during 7 seconds, 3 times each, with a total presentation of 3 on the right side and 3 on the left side. During testing, the CS+ was presented 4 times (2 times on each side), and each CS-was presented once on each side, for 7 s each. As for the training session, the order and side of odor presentations was pseudo-randomized: testing always started with one presentation author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint of each CS-, followed by one CS+ presentation.\\r\\n Clozapine-N-oxide (C0832, Sigma-Aldrich) was first dissolved in dimethylsulfoxide (DMSO, D2438, Sigma, final concentration of 1% vol/vol), then further diluted in 0.9% sterile saline solution to a final concentration of 0.2 mg/mL. Aliquots were stored for up to two months at -20°C and equilibrated to room temperature before injection. The solution was administered intraperitoneally (3 mg/kg). After injection, mice were left undisturbed for 25 minutes (Figure 2 and 4) or 5 minutes (Figure 3 and 5) in their home cage before the start of the experiment.\\r\\n Electrophysiological data were analyzed with custom-made software in Python 3.0 and averaged in MS Excel (Microsoft, USA). Behavioral sessions were video recorded and analyzed using custom-written MatLab programs (available upon request). Parameters extracted from multiple CS+ (CS-) presentation were averaged for each mouse.\\r\\n Statistical analysis was performed with Prism (GraphPad). Non-parametric tests were used: Mann-Whitney test for between group comparison, and Wilcoxon matchedpairs signed rank test for within group comparison. Values are represented as median (interquartile range) unless otherwise stated. The n represents number of mice, or number of recorded neurons for the electrophysiology experiment. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint before and after CNO application. After CNO application, the number of APs in DREADD-negative cells (Ctrl) does not change, significantly decreases in hM4Diexpressing cells and significantly increases in hM3Dq-expressing cells.\\r\\n (G) The relative change in the number of APs after/before CNO in DREADD-negative cells (Ctrl) does not change, significantly decreases in hM4Di-expressing cells and significantly increases in hM3Dq-expressing cells.\\r\\n Data are shown as individual data points, median and interquartile range, ** p<0.01. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint A) During fear conditioning, mice are trained to associate ethyl-acetate (CS+) with foot shock. The CS+ is presented at one end of the training box, and the corresponding half-side of the box is electrified. Training includes the presentation of two other odors that are not paired with foot shock (CS-, eugenol and beta-citronellol).\\r\\n The CS+ is presented 4 times (2 times on each side) and each CS-three times. The order and side of odor presentations is pseudo-randomized. B) During testing, the CS+ is presented 4 times (2 times on each side), and each CSis presented once on each side, for 7 s each. The testing session is video-recorded and the position of the mouse's centroid in the box along the x-axis is extracted (top).\\r\\n The length of the box is normalized to 1. Since the odor is presented randomly on the left or right side of the testing box, 0 is defined as the extremity where the odor is presented. For velocity measurements, a 7 s time window is defined both before and after stimulus onset (bottom). C) Experimental design: cFos-tTA transgenic mice (experimental group, n=8) or wildtype mice (control group, n=8) are injected with AAV-tetO-hM4Di in both hemispheres of piriform cortex. Ten days later, mice are habituated to the training and testing boxes. Upon doxycycline (DOX) removal, mice are fear-conditioned, and hM4Di expression (green circles) is induced in Fos-tagged neurons (filled in blue and yellow). Mice are returned to DOX-containing diet to avoid further expression of hM4Di. Three days later, all mice are i.p. injected with CNO, which silences hM4Diexpressing neurons (black crosses), and memory retrieval is tested.\\r\\n (D to H) Olfactory memory retrieval after CNO injection, assessed by the position of mice in the testing box and their maximum velocity upon odor exposure. Memory author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint retrieval was significantly impaired in cFos-tTA transgenic mice in which piriform neurons that were active during learning were chemogenetically silenced during memory recall (experimental group).\\r\\n (D) Position of mice in the testing box as a function of time after CS+ (black) and CS-(blue) presentation. (E) Position in the testing box 7 s after CS+ or CS-presentation.\\r\\n (F) Position 7 s after CS-presentation subtracted from the position 7 s after CS+ presentation.\\r\\n (G) Maximum velocity during a 7 s time window before and after CS+ presentation.\\r\\n (H) Maximum velocity ratio after/before CS+ (left) or CS-(right) presentation.\\r\\n Data are shown as median and interquartile range, * p<0.05; ** p<0.01; n.s. not significant.\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint (A) (Top) Experimental design: Fos-tagging during olfactory fear conditioning is performed as described in Figure 2 . Three days after fear conditioning, sniffing behavior is quantified in a plethysmograph. Mice are i.p. injected with CNO, habituated to the plethysmograph for 15 min, and each odor is then presented for 8 s with a 2.5 min inter-trial interval. pin: pinene, CS-: beta-citronellol, CS+: ethyl-acetate.\\r\\n (Bottom) Representative sniff recordings from one mouse when exposed to pinene or the CS+ (ethyl-acetate). Data are shown as median and interquartile range, * p<0.05; ** p<0.01; n.s.: not significant author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint Data are shown as median and interquartile range, * p<0.05; ** p<0.01; n.s. not significant.\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint (B) As a measure for exploratory activity, the distance travelled is computed from minute 15 to 18 (corresponding to odor exposure during test 2). For artificial memory recall, the distance travelled is computed over the entire testing session. Ethylacetate exposure and CNO injection lead to a decrease in exploratory behavior in mice previously fear conditioned to ethyl-acetate (n=6 and n=8 respectively).\\r\\n Previous exposure to ethyl-acetate without foot shock does not alter exploratory behavior upon re-exposure to ethyl-acetate or re-activation of Fos-tagged piriform ensembles (n=5 and n=7 respectively).\\r\\n (C) Distance traveled ratio test 2/test 1. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint \\r\\n Training Testing The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint baseline author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint. SARS-CoV-2, also known as 2019-nCoV, is a novel coronavirus (CoVs) isolated from patients with pneumonia in China 2019. SARS-CoV-2 has a similar incubation period (median, 3.0 days) and a relatively lower fatality rate than SARS-CoV or MERS-CoV (1), but it is estimated that the reproductive number of SARS-CoV-2 is higher than that of SARS-CoV (2) .\\r\\n What's more, some laboratory-confirmed symptomatic cases are absent of apparent cough, fever or radiologic manifestations, making it difficult to timely and accurately find out all infected patients (3, 4) . As of February 28, 2020, patients infected by SARS-CoV-2 have been diagnosed in more than 30 countries, and more than 78,000 confirmed cases and 2,700 deaths associate with SARS-CoV-2 infection are reported in China alone.\\r\\n The genetic information of a virus is essential for not only its classification and traceability, but also its pathogenicity. At the whole genome level, the sequence identify of SARS-CoV-2 was 50% to MERS-CoV, 79% to SARS-CoV, 88% to two bat-derived SARS-like coronaviruses, Bat-SL-CoVZC45 and Bat-SL-CoVZXC21 (collected in 2018 in Zhoushan, China) , and 96% to Bat-SARSr-CoV RaTG13 (collected in 2013 in Yunnan, China) (5) . Each genome of all SARS-CoV-2 strains now submitted online contains nearly 29,900 nucleotides (nt), which are predicted with at least 14 open reading frames (ORFs) (5′ to 3′), such as ORF1ab (P, 21,291 nt), spike (S, 3,822 nt), ORF3a (8,28 nt), envelope (E, 228 nt), membrane (M, 669nt), ORF8 (366 nt), and nucleocapsid (N,1,260 nt) (6) . Among them, the spike gene encoded a glycoprotein that is crucial for determining host tropism and transmission capacity, and highly divergent when compared with that of bat-SARSr-CoV RaTG13 (93.1% nucleotide identity) (6, 7) .\\r\\n Generally, the rates of nucleotide substitution of RNA viruses are faster than their hosts, and . CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint this rapid evolution is mainly shaped by natural selection (mostly purifying selection) (8) . Gene mutations such as nucleotide substitutions, deletions and insertions have been frequently reported when comparing SARS-CoV-2 with other viruses (5) (6) (7) . In this work, we investigated the mutation pattern of SARS-CoV-2 by comprehensive comparative genomic analysis of the nonsynonymous/ synonymous substitution, relative synonymous codon usage (RSCU) and selective pressure in order to explore their potential in evolution and function.\\r\\n The SARS-CoV-2 reference genome Wuhan-Hu-1 (NC_045512) and WIV04 (MN996528) was downloaded from the GenBank database. Other newly sequenced 2019-nCoV genomes were downloaded from the Global Initiative on Sharing Avian Influenza Data (GISAID) database (https://www.gisaid.org/). Twenty-one closely related coronavirus complete genome sequences and their coding sequences (CDS) were downloaded from GenBank database (Supplementary   Table S1 ).\\r\\n Genome sequences were aligned using MUSCLE v3.8.31 (9), followed by manual adjustment using BioEdit v7.2.5. Phylogenetic analyses of complete genome were performed using maximum-likelihood method and general time-reversible model of nucleotide substitution with gamma-distributed rates among sites (GTR+G) in RAxML v8.1.21 (10) . Support for the inferred relationships was evaluated by a bootstrap analysis with 1000 replicates and trees were rooted with the alpha-coronavirus lineage as an outgroup. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint\\r\\n The coding sequences were translated and aligned using the MEGA-X program (11) , and then codon-based sequence alignment was used for further analysis. Phylogenetic analyses of coding sequences were performed using MEGA-X software. The changes of amino acids or nucleotides for each CDS sequences were analyzed using in-house Perl script.\\r\\n The number of synonymous substitutions per synonymous site (dS), and the number of non-synonymous substitutions per non-synonymous site (dN), for each coding region were calculated using the Nei-Gojobori method (Jukes-Cantor) in PAML package (12) .\\r\\n The adaptive evolution server (http://www.datamonkey.org/) was used to identify eventual sites of positive selection. For this purpose, the following test has been used: mixed-effects model of evolution (MEME), which allows the distribution of dN/dS (ω) to vary from site to site and from branch to branch at a site (13) . This test allowed us to infer episodic and pervasive positive selection at individual sites.\\r\\n To investigate the potential relative synonymous codon usage (RSCU) bias of the spike protein from SARS-CoV-2 and its closely related coronaviruses, the coding sequence (CDS) of spike protein in these coronaviruses were calculated with CodonW 1.4.4 (http://codonw.sourceforge.net/). The RSCU of human genes was retrieved from the Codon Usage Database (http://www.kazusa.or.jp/codon/). The potential relationships among these sequences were calculated with a squared Euclidean distance ( u h u ). Besides RSCU, the effective number of codons (ENc) was used as a simple metrics to verify codon bias and to explore the source of virus.\\r\\n . CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint\\r\\n Statistical analyses were performed using the R statistical package (version 3.\\r\\n Chi-squared test was used to compare any two data sets, and data were considered significantly different if the two-tailed p value was less than 0.05.\\r\\n The newly identified SARS-CoV-2 strain WIV04 genome sequence was closely related with Bat-SARSr-CoV RaTG13 and Bat-SL-CoVZC45, which was collected from host Rhinolophus affinis (5) . Compared with RaTG13 genome, many nucleotide substitutions are observed, but there are only five small inserts and deletions (indels) mutations, and the largest insert segment in WIV04 genome was \\\" CGGCGGGCACGT \\\" sequence, which is located near the boundary of S1 and S2 regions of spike protein. Interestingly, only synonymous mutation are observed near this insertion sequence (Fig. 1 , panel C). Compared with Bat-SL-CoVZC45 genome, this insert segment is detected as well, but non-synonymous mutations are also observed around it. Then, we further compared the proportion of synonymous mutations in spike gene between WIV04 and RaTG13 or Bat-SL-CoVZC45. The ratio of nucleotide substitutions (263 NT) to amino acid substitutions (29 AA) was 9.07 from WIV04 to RaTG13, which was significantly higher than the ratio (3.91, 864/221) from WIV04 to Bat-SL-CoVZC45 (p < 0.05) (Fig. 2) . Moreover, we also observed that the synonymous mutations increased in whole genome level between WIV04 and RaTG13 than that between WIV04 and Bat-SL-CoVZC45.\\r\\n Next, the detailed nucleotide changes of each comparison group were analyzed, and we found that the T-to-C (T:C) transition mutation was enriched in the nucleotide substitutions between WIV04 and RaTG13 (Fig. 1B) . It's reported that the CoVs lacking a 3'-to-5' exoribonuclease (ExoN) accumulated 15 to 20 fold more A:G and U:C transition (14) . What's more, the RNA mutagen 5-fluorouracil (5-FU) treatment will also increase the U:C and A:G transitions. Consequently, this increased T:C mutation implies the ExoN of SARS-CoV-2 may be deactivated compared with that of RaTG13. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint Furthermore, we checked whether the ratio of synonymous substitution to missense substitution is increased within SARS-CoV-2 strains. From December 2019 to February 2020, the genome sequences of 108 strains of SARS-CoV-2 virus have been submitted to GISAID database worldwide. Compared with the standard SARS-CoV-2 strain WIV04, total 98 point mutations were detected at 93 nucleotide sites of all SARS-CoV-2 strains with genome sequence available on Feb. 25 2020 (Fig. 1A) . However, only 58 of these nucleotide mutations caused changes in amino acids. Among them, 15 nucleotide substitutions at 14 sites caused changes in 7 amino acids of the spike protein were observed. Consequently, the proportion of synonymous mutations (~40%) among all current reported SARS-CoV-2 strains is similar to that between WIV04 and Bat-SL-CoVZC45 (39.1%), but is dramatically less than that between WIV04 and RaTG13 (90.7%).\\r\\n According to random drift hypothesis (15) ， these nucleotide mutations among different SARS-CoV-2 strains now available are still determined by neutral evolution. In short, there has no powerful factor to force SARS-CoV-2 to evolve in a certain direction by far. However, we should take strict precautions against the strong factors that may cause directional variation of SARS-CoV-2 both in natural environment and infection treatment.\\r\\n To investigate whether the mutation pattern between SARS-CoV-2 and RaTG13 is unique across all coronavirus species, we further compared their alterations of nucleotides and amino acids in comparison to other representative coronaviruses. Phylogenetic analysis of SARS-CoV-2 and its 20 closely related coronaviruses formed four well supported clades (Fig. 2) . The two SARS-CoV-2 strain WIV04 (from Wuhan) and SNU01 (from Korea patient) were clustered with SARS-CoV-related strains to form clade 1, belonging to beta-coronavirus lineage B. The Compared with Bat-SARSr-CoV RaTG13, there were 293 nucleotideswere altered while only 29 amino acids were altered in SARS-CoV-2; their ratio was as high as 9.07. However, this ratio detected in the comparisons of other human coronaviruses with their similar animal coronaviruses was less than 5.0. For example, the ratio of human SARS-CoV Tor2 to bat SARS-like CoVs LYRa11 was 4.81; that ratio of human MERS-CoV EMC2012 to bat SARS-like CoVs RSA2011 was 2.80; that ratio of human coronavirus 229E to bat coronavirus HKU2 was 4.55. These results indicate that the relative level of synonymous substitutions between human SARS-CoV-2 and its possible animal origin (RaTG13) is much higher than that between other human coronaviruses and their potential animal sources.\\r\\n Different organisms, even different protein coding genes of the same species, have different frequency of codon usage (16) . The RSCU bias will reveal the difference of the host source. We calculated the distance of RSCU between spike genes of 20 representative coronaviruses and\\r\\n and Bat-SARSr-CoV RaTG13 was 1.17, which was the lowest except for SNU01 (another strain of SARS-CoV-2), indicating that their codon preference was almost the same; the second lowest codon usage difference was 2.41 that was detected between SARS-CoV-2 WIV04 and Bat-SL-CoVZC45 (Fig. 2) . The coronaviruses in same beta-CoV lineage B have a relatively close \\r\\n To infer whether the retention of synonymous mutations is supported or hindered by natural selection, we further studied non-synonymous substitution rate (dN) and synonymous substitution rate (dS) in spike gene (Table 1) . Generally, positive (Darwinian) selection increases, but negative (purifying) selection decreases the ratios of non-synonymous to synonymous substitution rates (dN/dS). The results showed that both dN and dS of S gene of SARS-CoV-2 WIV04 versus Bat-SARr-CoV RaTG13 were the lowest among all typical coronaviruses, while those of SARS-CoV Tor2 versus bat SARS-like coronavirus WIV1 were the second lowest. When the ratio of dN to dS of spike gene is compared, all the tested dN/dS are less than 1, indicating that these non-synonymous mutations are harmful, and negative selection will reduce their retention speed. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint Among them, dN/dS of spike gene of SARS-CoV-2 WIV04 versus that of Bat-SARr-CoV RaTG13 was 0.04, which was the lowest among all comparisons, reconfirming that the rate of synonymous mutation was increased between WIV04 and RaTG13 strains. Moreover, dN/dS rate of polyprotein (ORF1ab) and nucleocapsid (N) genes were similar with that of spike gene (Table   S2 ).\\r\\n The spike (S) protein undergoes several drastic changes during virus infection. For instance, its large parts are cleaved during infection by cellular proteases and expose the receptors to activate viral attachment to the host (17) . As the receptor binding domain (RBD) of spike protein was involved in interacting with human angiotensin-converting enzyme 2 (ACE2) protein, the RBD region was thought to be preferential targets of natural selection (18) . Consistent with this hypothesis, our results showed that both dN and dS of RBD region are increased in comparison with whole spike gene region across all the virus pairs used in this study (Table 1) . Notably, the dN/dS ratio of RBD region in SARS-CoV-2 WIV04 was dramatically increased in about 3 fold in comparison with full spike region. Consequently, these mutations may be subjected to Darwin's choice and will become evidence of adaptive protein evolution.\\r\\n To detect positive selection on the spike gene, the MEME analysis was performed.\\r\\n Significant (p < 0.05) pervasive episodic selection was detected in three sites (48th, 254th, 330th position using the reference sequence of WIV04). On the 254th position of the spike amino acid sequence, there is a histidine residue instead of a phenylalanine residue, whereas on the 330th amino acidic position in WIV04 sequence, there is a glutamine residue instead of an valine residue.\\r\\n The results described above strongly supporting the action of positive selection in spike gene during the recent evolution of SARS-CoV-2 and RaTG13. However, this result should be . CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint addressed with careful explain, because the RBD region of spike gene from SARS-CoV-2 was quit divergent with that of RaTG13 (Fig. S1) , suggesting it might have originated from homologous recombination between RaTG13 and one yet-unknown coronavirus (19) .\\r\\n Through comprehensive comparative analysis between SARS-CoV-2 and other coronaviruses, we found the synonymous mutations is dramatically elevated between SARS-CoV-2 and RaTG13 than that of other coronavirus strains, and the nucleotide mutations were enriched in T:C transition.\\r\\n The specific mutation pattern may caused by the loss function of RNA 3'-to-5' exoribonuclease (ExoN). Moreover, as the SARS-CoV-2 was supposed to be originated from Bat-SARSr-CoV RaTG13, the increased synonymous substitution between SARS-CoV-2 and RaTG13 strain suggested the SARS-CoV-2 genome was under stronger negative (purifying) selection. We also detected some sites in spike protein was experiencing positive selection. These observations suggest that adaptive evolution might contribute to its host shifts. However, as the RNA mutagen (e.g. 5-FU) could induce the same mutation pattern, the mechanism of the mutation pattern observed between SARS-CoV-2 and RaTG13 should be further investigated in future work. The number of amino acid changes. RSCUdist: the RSCU distance of spike protein; ENc: effective number of codons; S ident(%): the percent identity of S protein sequence between different coronaviruses. The NT/AA ratio between WIV04/RaTG13 was statistically significant higher (Chi-squared test, P<0.01) than that of other strain pairs.\\r\\n . CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint *The whole genome sequence identity between these paired strains was larger than 95%.\\r\\n . CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint. The COVID-19 outbreak began on December 2019 in Wuhan, China (1), and the causative virus, SARS-CoV-2, has rapidly spread into 43 countries as of February 27, 2020. The number of infected patients exceeds 81,000, and the death toll exceeds 2,700. Several drugs, including lopinavir, remdesivir, and chloroquine, have been reported to be presumably effective in treating this disease (2, 3) .\\r\\n Systemic treatment with corticosteroids is contraindicated for the severe pneumonia caused by viruses such as MERS-CoV and SARS-CoV, as steroids suppress the innate immune system, resulting in increased viral replication. In fact, for SARS in 2003 and MERS in 2013, treatment with corticosteroids was associated with increased mortality (4, 5). In the present study, we reconsider the use of inhaled corticosteroids, which have been excluded from the treatment of pneumonia caused by coronavirus.\\r\\n Steroid compounds, which are expected to have dual functions in blocking both coronavirus replication and host inflammation, were screened from a chemical library. The cytopathic effect caused by MERS-CoV infection was measured to evaluate viral replication. Four steroid compounds, ciclesonide, mometasone furoate, mifepristone, and algestone acetophenide conferred a greater than 95% cell survival rate (Fig. S1 ).\\r\\n Next, concentration-dependent viral growth suppression and drug cytotoxicity were assessed. Ciclesonide exhibited low cytotoxicity and potent suppression of viral growth (Fig.   1a ). Cortisone and prednisolone, which are commonly used for systemic steroid treatment, dexamethasone, which has strong immunosuppressant effects, and fluticasone, a commonly used inhaled steroid, did not suppress viral growth (Fig. 1a) .\\r\\n The antiviral effects of steroids against various species of viruses were tested by quantifying propagated virus in culture medium. Ciclesonide and mometasone also suppressed replication of other coronaviruses, HCoV-229E and SARS-CoV, but not replication of RS virus or influenza virus (Fig. S2 ). In addition, ciclesonide slightly but significantly inhibited rubella virus (having positive strand RNA genome) replication (Fig. S2 ), suggesting that ciclesonide interacted with the replication site of positive-strand RNA virus intracellularly.\\r\\n To identify the drug target of virus replication, we conducted 11 consecutive MERS-CoV passages in the presence of 40 μ M ciclesonide or 40 μ M mometasone. A mutant virus that developed resistance ciclesonide was generated (Fig. 1b) , but no resistant virus to author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.11.987016 doi: bioRxiv preprint mometasone was generated. Next-generation sequencing identified that an amino acid substitution at A25V (C19647T in the reference sequence NC_019843.3) in nonstructural protein 15 (NSP15), coronavirus endoribonuclease (6-8), was the predicted mechanism for viral resistance to ciclesonide. A recombinant virus carrying the A25V amino acid substitution (Re-Nsp15-A25V) was generated from the parental MERS-CoV/EMC strain (Re-EMC/MERS) with a BAC reverse genetics system (9), which overcame the antiviral effect of ciclesonide (Fig. 1c) . Interestingly, the mutant virus was inhibited by mometasone, suggesting that the antiviral target of mometasone was different from that of ciclesonide.\\r\\n Next, the effect of ciclesonide on SARS-CoV-2 infection was evaluated. VeroE6/TMPRSS2 cells(10) were infected with authentic SARS-CoV-2 in the presence of steroids or other inhibitors. At 6 h post-infection, cellular RNA was isolated, and real-time PCR was conducted to quantify the amount of viral RNA (11). Ciclesonide and mometasone suppressed viral replication with a similar efficacy to lopinavir (Fig. 2a) . For comparison, viral cell entry inhibitors were tested. E64d substantially reduced viral RNA levels, but nafamostat and camostat had only modest antiviral effects, suggesting that SARS-CoV-2 primarily utilized the cathepsin/endosomal pathway of cell entry rather than the TMPRSS2/cell surface pathway to enter TMPRSS2-expressing cells, consistent with a recent study (12). author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https: //doi.org/10.1101 //doi.org/10. /2020 The copyright holder for this preprint (which was not peer-reviewed) is th . https://doi.org/10.1101/2020.03.11.987016 doi: bioRxiv preprint a RNA copy (log 10 ) SARS-CoV-2 RNA in cell (6 hpi Cell viability (%) author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is th . https://doi.org/10.1101/2020.03.11.987016 doi: bioRxiv preprint. Seventeen years have passed since the outbreak of severe acute respiratory syndrome (SARS) in 2003, but there is yet no approved treatment for infections with the SARS coronavirus (SARS-CoV). 1 One of the reasons is that despite the devastating consequences of SARS for the affected patients, the development of an antiviral drug against this virus would not be commercially viable in view of the fact that the virus has been rapidly contained and did not reappear since 2004. As a result, we were empty-handed when the Middle-East respiratory syndrome coronavirus (MERS-CoV), a close relative of SARS-CoV, emerged in 2012. 2 MERS is characterized by severe respiratory disease, quite similar to SARS, but in addition frequently causes renal failure 3 . Although the number of registered MERS cases is low (2494 as of November 30, 2019; www.who.int), the threat MERS-CoV poses to global public health may be even more serious than that presented by SARS-CoV. This is related to the high casefatality rate (about 35%, compared to 10% for SARS), and to the fact that MERS cases are still accumulating seven years after the discovery of the virus, whereas the SARS outbreak was essentially contained within 6 months. The potential for human-to-human transmission of MERS-CoV has been impressively demonstrated by the 2015 outbreak in South Korea, where 186 cases could be traced back to a single infected traveller returning from the Middle East. 4 SARS-like coronaviruses are still circulating in bats in China, [5] [6] [7] [8] from where they may spill over into the human population; this is probably what caused the current outbreak of atypical pneumonia in Wuhan, which is linked to a seafood and animal market. The RNA genome (Gen-Bank accession code: MN908947.2; http://virological.org/t/initialgenome-release-of-novel-coronavirus/319, last accessed on January 11, 2020) of the new betacoronavirus features around 82% identity to that of SARS-CoV.\\r\\n In spite of the considerable threat posed by SARS-CoV and related viruses, as well as by MERS-CoV, it is obvious that the number of cases so far does not warrant the commercial development of an antiviral drug targeting MERS-and SARS-CoV even if a projected steady growth of the number of MERS cases is taken into account. A possible solution to the problem could be the development of broadspectrum antiviral drugs that are directed against the major viral protease, a target that is shared by all coronavirus genera as well as, in a related form, by members of the large genus Enterovirus in the picornavirus family. Among the members of the genus Alphacoronavirus are the human coronaviruses (HCoV) NL63 (ref. 9) and 229E 10 that usually cause only mild respiratory symptoms in otherwise healthy individuals, but are much more widespread than SARS-CoV or MERS-CoV. Therapeutic intervention against alphacoronaviruses is indicated in cases of accompanying disease such as cystic fibrosis 11 or leukemia, 12 or certain other underlying medical conditions. 13 The enteroviruses include pathogens such as EV-D68, the causative agent of the 2014 outbreak of the \\\"summer flu\\\" in the US, 14 EV-A71 and Coxsackievirus A16 (CVA16), the etiological agents of Hand, Foot, and Mouth Disease (HFMD), 15 Coxsackievirus B3 (CVB3), which can cause myocardic inflammation, 16 and human rhinoviruses (HRV), notoriously known to lead to the common cold but also capable of causing exacerbations of asthma and COPD. 17 Infection with some of these viruses can lead to serious outcome; thus, EV-D68 can cause polio-like disease, 18 and EV-A71 infection can proceed to aseptic meningitis, encephalitis, pulmonary edema, viral myocarditis, and acute flaccid paralysis. 15, [19] [20] Enteroviruses cause clinical disease much more frequently than coronaviruses, so that an antiviral drug targeting both virus families should be commercially viable.\\r\\n However, enteroviruses are very different from coronaviruses. While both of them have a single-stranded RNA genome of positive polarity, that of enteroviruses is very small (just 7 -9 kb) whereas coronaviruses feature the largest RNA genome known to date (27 -34 kb) . Enteroviruses are small, naked particles, whereas coronaviruses are much larger and enveloped. Nevertheless, a related feature shared by these two groups of viruses is their type of major protease, 21 which in the enteroviruses is encoded by the 3C region of the genome (hence the protease is designated 3C pro ). In coronaviruses, non-structural protein 5 (Nsp5) is the main protease (M pro ). Similar to the enteroviral 3C pro , it is a cysteine protease in the vast majority of cases and has therefore also been called \\\"3C-like protease\\\" (3CL pro ). The first crystal structure of a CoV M pro or 3CL pro (ref. 22) revealed that two of the three domains of the enzyme together resemble the chymotrypsin-like fold of the enteroviral 3C pro , but there is an additional a-helical domain that is involved in the dimerization of the protease (Fig. 1A) . This dimerization is essential for the catalytic activity of the CoV M pro , whereas the enteroviral 3C pro (Fig. 1B) functions as a monomer. Further, the enteroviral 3C pro features a classical Cys...His...Glu/Asp catalytic triad, whereas the CoV M pro only has a Cys...His dyad. 22 Yet, there are a number of common features shared between the two types of proteases, in particular their almost absolute requirement for Gln in the P1 position of the substrate and space for only small amino-acid residues such as Gly, Ala, or Ser in the P1' position, encouraging us to explore the coronaviral M pro and the enteroviral 3C pro as a common target for the design of broad-spectrum antiviral compounds. The fact that there is no known human protease with a specificity for Gln at the cleavage site of the substrate increases the attractivity of this viral target, as there is hope that the inhibitors to be developed will not show toxicity versus the host cell. Indeed, neither the enterovirus 3C pro inhibitor rupintrivir, which was developed as a treatment of the common cold caused by HRV, nor the peptide aldehyde inhibitor of the coronavirus M pro that was recently demonstrated to lead to complete recovery of cats from the normally fatal infection with Feline Infectious Peritonitis Virus (FIPV), showed any toxic effects on humans or cats, respectively. 23, 24 Figure 1: Crystal structures of SARS-CoV main protease (M pro , ref. 26 ; PDB entry 2BX4) and Coxsackivirus B3 3C protease (3C pro ; Tan et al., unpublished; PDB entry 3ZYD). Catalytic residues are indicated by spheres (yellow, Cys; blue, His; red: Glu). A. The coronavirus M pro is a homodimer, with each monomer comprising three domains. B. The structure of the monomeric CVB3 3C pro resembles the Nterminal two domains of the SARS-CoV M pro . Structure is on the same scale as image A. C. Superimpostion of residues from the two structures involved in ligand binding. Superimposition was carried out by aligning the catalytic Cys-His pair of each protease. Residues of the SARS-CoV M pro are shown with carbon atoms in cyan, CVB3 3C pro residues have orange carbons and are labeled with an asterisk (*).\\r\\n As the proteases targeted in our study all specifically cleave the peptide bond following a P1-glutamine residue (HCoV-NL63 M pro uniquely also accepts P1 = His at the Nsp13/Nsp14 cleavage site 31 ), we decided to use a 5-membered ring (g-lactam) derivative of glutamine (henceforth called GlnLactam) as the P1 residue in all our aketoamides (see Scheme 1). This moiety has been found to be a good mimic of glutamine and enhance the power of the inhibitors by up to 10-fold, most probably because compared to the flexible glutamine side-chain, the more rigid lactam leads to a reduction of the loss of entropy upon binding to the target protease. 29, 32 Our synthetic efforts therefore aimed at optimizing the substituents at the P1', P2, and P3 positions of the a-ketoamides.\\r\\n Synthesis (Scheme 1) started with the dianionic alkylation of N-Boc glutamic acid dimethyl ester with bromoacetonitrile. As expected, this alkylation occurred in a highly stereoselective manner, giving 1 as the exclusive product. In the following step, the cyano group of 1 was subjected to hydrogenation. The in-situ cyclization of the resulting intermediate afforded the lactam 2. The lactam derivative 3 was generated by removal of the protecting group of 2. On the other hand, the amidation of acyl chloride and a-amino acid methyl ester afforded the intermediates 4, which gave rise to the acids 5 via alkaline hydrolysis. The key intermediates 6 were obtained via the condensation of the lactam derivative 3 and the N-capped amino acids 5. The ester group of compounds 6 was then reduced to the corresponding alcohol. Oxidation of the alcohol products 7 by Dess-Martin periodinane generated the aldehydes 8, which followed by nucleophilic addition with isocyanides gave rise to compounds 9 under acidic conditions. Then, the a-hydroxyamides 10 were prepared by removing the acetyl group of compounds 9. In the final step, the oxidation of the exposed alcohol group in compounds 10 generated our target a-ketoamides 11.\\r\\n The inhibitory potencies of candidate a-ketoamides were evaluated against purified recombinant SARS-CoV M pro , HCoV-NL63 M pro , CVB3 3C pro , and EV-A71 3C pro . The most potent compounds were further tested against viral replicons and against SARS-CoV, MERS-CoV, or a whole range of enteroviruses in cell culture-based assays (Tables 1 -3 and Supplementary Table 1 \\r\\n To enable the rapid and biosafe screening of antivirals against corona-and enteroviruses, a non-infectious, but replication-competent SARS-CoV replicon was used 33 along with subgenomic replicons of CVB3 34 and EV-A71 (kind gift of B. Zhang, Wuhan, China). The easily detectable reporter activity (firefly or Renilla luciferase) of these replicons has previously been shown to reflect viral RNA synthesis. [33] [34] [35] In-vitro RNA transcripts of the enteroviral replicons were also used for transfection. For the SARS-CoV replicon containing the CMV promoter, only the plasmid DNA was used for transfection.\\r\\n The initial compound to be designed and synthesized was 11a, which carries a cinnamoyl N-cap in the P3 position, a benzyl group in P2, the glutamine lactam (GlnLactam) in P1, and benzyl in P1' (Table  1) . This compound showed good to mediocre activities against recombinant SARS-CoV M pro (IC50 = 1.95 µM; for all compounds, see Tables 1 -3 for standard deviations), CVB3 3C pro (IC50 = 6.6 µM), and EV-A71 3C pro (IC50 = 1.2 µM), but was surprisingly completely inactive (IC50 > 50 µM) against HCoV-NL63 M pro . These values were mirrored in the SARS-CoV and in the enterovirus replicons (Table 2) . In virus-infected cell cultures, the results obtained were also good to mediocre (Table 3) : SARS-CoV (EC50 = 5.8 µM in Vero E6 cells), MERS-CoV (EC50 = 0.0047 µM in Huh7 cells), HCoV 229E (EC50 = 11.8 µM in Huh7 cells), or a host of enteroviruses (EC50 = 9.8 µM against EV-A71 in RD cells; EC50 = 0.48 µM against EV-D68 in HeLa Rh cells; EC50 = 5.6 µM against HRV2 in HeLa Rh cells). In all cell types tested, the compound generally proved to be non-toxic, with selectivity indices (CC50/EC50) usually >10 (Table 3) .\\r\\n Crystal structures of compound 11a in complex with SARS-CoV M pro , HCoV-NL63 M pro , and CVB3 3C pro demonstrated that the aketo-carbon is covalently linked to the active-site Cys (no. 145, 144, and 147, resp.) of the protease (Fig. 2, 3a-c) . The resulting thiohemiketal is in the R configuration in the SARS-CoV and HCoV-NL63 M pro but in the S configuration in the CVB3 3C pro complex. The reason for this difference is that the oxygen atom of the thiohemiketal accepts a hydrogen bond from the catalytic His40 in the CVB3 protease, rather than from the main-chain amides of the oxyanion hole as in the SARS-CoV and HCoV-NL63 enzymes (Fig. 3a ,b,c insets). It is remarkable that we succeeded in obtaining a crystal structure of compound 11a in complex with the HCoV-NL63 M pro , even though it has no inhibitory effect on the activity of the enzyme (IC50 > 50 µM), (Fig. 2c) . Apparently, the compound is able to bind to this M pro in the absence of peptide substrate, but cannot compete with substrate for the binding site due to low affinity. A similar observation has been made in one of our previous studies, where we were able to determine the crystal structure of a complex between the inactive Michaelacceptor compound SG74 and the EV-D68 3C pro (ref. 29 ; PDB entry 3ZV9). Figure 2 : Fit of compound 11a (pink carbon atoms) to the target proteases (wheat surfaces) as revealed by X-ray crystallography of the complexes. A. Fo -Fc difference density (contoured at 3s) for 11a in the substrate-binding site of the SARS-CoV M pro (transparent surface). Selected side-chains of the protease are shown with green carbon atoms. B. Another view of 11a in the substrate binding site of the SARS-CoV M pro . Note the \\\"lid\\\" formed by residue Met49 and its neighbors above the S2 pocket. C. 11a in the substrate-binding site of HCoV-NL63 M pro . Because of the restricted size of the S2 pocket, the P2 benzyl group of the compound cannot enter deeply into this site. Note that the S2 pocket is also covered by a \\\"lid\\\" centred around Thr47. D. 11a in the substrate-binding site of the CVB3 3C pro . The S2 site is large and not covered by a \\\"lid\\\".\\r\\n The crystal structures indicated that the fits of the P1' benzyl group of 11a in the S1' pocket and of the P3 cinnamoyl cap in the S3 subsite might be improved (see Fig. 3a -c). Compounds 11b -11e and 11g -11l were synthesized in an attempt to do so; however, none of them showed better inhibitory activity against the majority of the recombinant proteases, compared to the parent compound, 11a (see Supplementary Results). To investigate whether the P3 residue of the inhibitor is dispensible, we synthesized compound 11f, which only comprises P2 = Boc, P1 = GlnLactam, and P1' = benzyl. 11f was inactive against all purified proteases and in all replicons tested, but showed some activity against HRV2 in HeLa Rh cells (EC50 = 9.0 µM). A crystal structure of 11f bound to HCoV-NL63 M pro demonstrated that the P2-Boc group entered the S2 pocket (Fig. 3d ). In conclusion, although there is probably room for further improvement, we decided to maintain the original design with P1' = benzyl and P3 = cinnamoyl, and focussed on improving the P2 substituen.\\r\\n The crystal structures of SARS-CoV M pro , HCoV-NL63 M pro , and CVB3 3C pro in complex with 11a revealed a fundamental difference between the S2 pockets of the coronavirus proteases and the enterovirus proteases: The cavities are covered by a \\\"lid\\\" in the former but are open to one side in the latter (Fig. 2,b-d) . In SARS-CoV M pro , the lid is formed by the 310 helix 46 -51 and in HCoV-NL63 M pro by the loop 43-48. Residues from the lid, in particular Met49 in the case of SARS-CoV M pro , can thus make hydrophobic interactions with the P2 substitutent of the inhibitor, whereas such interaction is missing in the enterovirus 3C pro s. In addition to the lid, the S2 pocket is lined by the \\\"back-wall\\\" (main-chain atoms of residues 186 and 188 and Cb atom of Asp187), the side-walls (Gln189, His41), as well as the \\\"floor\\\" (Met165) in SARS-CoV M pro . In HCoV-NL63 M pro , the corresponding structural elements are main-chain atoms of residues 187 and 188 as well as the Cb atom of Asp187 (back-wall), Pro189 and His41 (side-walls), and Ile165 (floor). Finally, in CVB3 3C pro , Arg39, Asn69, and Glu71 form the back-wall, residues 127-132 and His40 form the side-walls, and Val162 constitutes the floor.\\r\\n In addition, the S2 pocket is of different size in the various proteases. The SARS-CoV enzyme features the largest S2 pocket, with a volume of 252 Å 3 embraced by the residues (Gln189, His41) defining the side-walls of the pocket in the ligand-free enzyme, as calculated by using Chimera, 36 followed by the CVB3 3C pro S2 pocket with about 180 Å 3 (space between Thr130 and His40). The HCoV-NL63 M pro has by far the smallest S2 pocket of the three enzymes, with a free space of only 45 Å 3 between Pro189 and His41, according to Chimera.\\r\\n In agreement with these observations, a good fit is observed between the P2 benzyl group of 11a and the S2 subsite of the SARS-CoV M pro as well as that of the CVB3 3C pro (Fig. 3a,c) . In contrast, the crystal structure of the complex between 11a and HCoV-NL63 M pro , against which the compound is inactive, demonstrates that the P2-benzyl group cannot fully enter the S2 pocket of the enzyme because of the restricted size of this site (Fig. 3b) .\\r\\n Thus, the properties of our target proteases with respect to the S2 pocket were defined at this point as \\\"small\\\" and \\\"covered by a lid\\\" for HCoV-NL63 M pro , \\\"large\\\" and \\\"covered\\\" for SARS-CoV M pro , and \\\"large\\\" and \\\"open\\\" for CVB3 3C pro . Through comparison with crystal structures of other proteases of the same virus genus (HCoV-229E M pro for alphacoronaviruses 28 (PDB entry 1P9S); HKU4 M pro for betacoronaviruses (Ma, Xiao et al., unpublished; PDB entry 2YNA); and EV-A71 3C pro for enteroviruses 30 (PDB entry 3SGK), we ensured that our conclusions drawn from the template structures were valid for other family members as well.\\r\\n To explore the sensitivity of the S2 pocket towards a polar substituent in the para position of the benzyl group, we synthesized compound 11m carrying a 4-fluorobenzyl group in P2. This substitution abolished almost all activity against the SARS-CoV M pro (IC50 > 50 µM), and the compound proved inactive against HCoV-NL63 M pro as well, whereas IC50 values were 2.3 µM against the EV-A71 3C pro and 8.7 µM against CVB3 3C pro . From this, we concluded that the introduction of the polar fluorine atom is not compatible with the geometry of the S2 pocket of SARS-CoV M pro , whereas the fluorine can accept a hydrogen bond from Arg39 in EV-A71 3C pro (ref. 30 ) and probably also CVB3 3C pro . In SARS-CoV M pro , however, the carbonyl groups of residues 186 and 188 might lead to a repulsion of the fluorinated benzyl group. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.10.936898 doi: bioRxiv preprint Table 2 : a-ketoamide-induced inhibition of subgenomic RNA synthesis using replicons in a cell-based assay (EC50, μM) 1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.10.936898 doi: bioRxiv preprint Table 3 . The thiohemiketal is in the R configuration, with its oxygen accepting two hydrogen bonds from the oxyanion-hole amides of Gly143 and Cys145. The amide oxygen accepts an H-bond from His41. The side-chains of Ser144 and Arg188 have been omitted for clarity. B. The P2-benzyl substituent of 11a cannot fully enter the S2 pocket of the HCoV-NL63 M pro , which is much smaller and has less plasticity than the corresponding pocket of SARS-CoV M pro (cf. A). The benzyl therefore binds above the pocket in the view shown here; this is probably the reason for the total inactivity (IC50 > 50 µM) of compound 11a against HCoV-NL63 M pro . The small size of the pocket is due to the replacement of the flexible Gln189 of the SARS-CoV M pro by the more rigid Pro189 in this enzyme. The stereochemistry of the thiohemiketal is R. The sidechains of Ala143 and Gln188 have been omitted for clarity. C. Binding of 11a to the CVB3 3C pro . The stereochemistry of the thiohemiketal is S, as the group accepts a hydrogen bond from His41, whereas the amide keto group accepts three H-bonds from the oxyanion hole (residues 145 -147). The side-chain of Gln146 has been omitted for clarity. D. The crystal structure of 11f in complex with HCoV-NL63 M pro shows that this short (inactive) compound lacking a P3 residue has its P2-Boc group inserted into the S2 pocket of the protease. The stereochemistry of the thiohemiketal is S. The sidechains of Ala143 and Gln188 have been omitted for clarity. E. In contrast to P2 = benzyl in 11a, the isobutyl group of 11n is small and flexible enough to enter into the narrow S2 pocket of the HCoV-NL63 M pro . The thiohemiketal is in the R configuration. The side-chains of Ala143 and Gln188 have been omitted for clarity. F. In spite of its small size, the cyclopropylmethyl side-chain in the P2 position of 11s can tightly bind to the S2 subsite of the SARS-CoV M pro , as this pocket exhibits pronounced plasticity due to the conformational flexibility of Gln189 (see also Fig. 4 ). The stereochemistry of the thiohemiketal is S. The side-chains of Ser144 and Arg188 have been omitted for clarity.\\r\\n As the P2-benzyl group of 11a was apparently too large to fit into the S2 pocket of the HCoV-NL63 M pro , we replaced it by isobutyl in 11n. This resulted in improved activities against SARS-CoV M pro (IC50 = 0.33 µM) and in a very good activity against HCoV-NL63 M pro (IC50 = 1.08 µM, compare with the inactive 11a). For EV-A71 3C pro , however, the activity decreased to IC50 = 13.8 µM, different from CVB3 3C pro , where IC50 was 3.8 µM. Our interpretation of this result is that the smaller P2-isobutyl substituent of 11n can still interact with the \\\"lid\\\" (in particular, Met49) of the SARS-CoV M pro S2 site, but is unable to reach the \\\"back-wall\\\" of the EV-A71 3C pro pocket and thus, in the absence of a \\\"lid\\\", cannot generate sufficient enthalpy of binding. We will see from examples to follow that this trend persists among all inhibitors with a smaller P2 substituent: Even though the SARS-CoV M pro S2 pocket has a larger volume than that of the enterovirus 3C pro , the enzyme can be efficiently inhibited by compounds carrying a small P2 residue that makes hydrophobic interactions with the lid (Met49) and floor (Met165) residues.\\r\\n The EC50 of 11n was >10 µM against the EV-A71 and CVB3 replicons, and even in the SARS-CoV replicon, the activity of 11n was relatively weak (EC50 = 7.0 µM; Table 2 ). In agreement with the replicon data, 11n proved inactive against EV-A71 in RD cells and showed limited activity against HRV2 or HRV14 in HeLa Rh cells (Table 3) . Only the comparatively good activity (EC50 = 4.4 µM) against EV-D68 in HeLa Rh cells was unexpected. The activity of 11n against HCoV 229E in Huh7 cells was good (EC50 = 0.6 µM), and against MERS-CoV in Huh7 cells, it was excellent, with EC50 = 0.0048 µM, while in Vero cells, the EC50 against MERS-CoV was as high as 9.2 µM. Similarly, the EC50 against SARS-CoV in Vero cells was 14.2 µM (Table 3) .\\r\\n We managed to obtain crystals of 11n in complex with the M pro of HCoV NL63 and found the P2 isobutyl group to be well embedded in the S2 pocket (Fig. 3e) . This is not only a consequence of the smaller size of the isobutyl group compared to the benzyl group, but also of its larger conformational flexibility, which allows a better fit to the binding site.\\r\\n When we replaced the P2-isobutyl residue of 11n by n-butyl in 11o, the activities were as follows: IC50 = 8.5 µM for SARS-CoV M pro , totally inactive (IC50 > 50 µM) against HCoV-NL63 M pro , IC50 = 3.2 µM for EV-A71 3C pro , and 5.2 µM for CVB3 3C pro . The decreased activity in case of SARS-CoV M pro and the total inactivity against HCoV-NL63 M pro indicate that the n-butyl chain is too long for the S2 pocket of these proteases, whereas the slight improvement against EV-A71 3C pro and CVB3 3C pro is probably a consequence of the extra space that is available to long and flexible substituents because of the lack of a lid covering the enterovirus 3C pro pocket.\\r\\n As the n-butyl substituent in P2 of 11o was obviously too long, we next synthesized a derivative with the shorter propargyl (ethinylmethyl) as the P2 residue (compound 11p). This led to very mediocre activities against all tested proteases. Using cyclopropyl as the P2 residue (compound 11q), the IC50 values were even higher against most of the proteases tested. Obviously, the P2 side-chain requires a methylene group in the b-position in order to provide the necessary flexibility for the substituent to be embedded in the S2 pocket.\\r\\n Having realized that in addition to size, flexibility of the P2 substituent may be an important factor influencing inhibitory activity, we introduced flexibility into the phenyl ring of 11a by reducing it. The cyclohexylmethyl derivative 11r exhibited IC50 = 0.7 µM against SARS-CoV M pro , 12.3 µM against HCoV-NL63 M pro , 1.7 µM against EV-A71 3C pro , and 0.9 µM against CVB3 3C pro . Thus, the replace- 1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.10.936898 doi: bioRxiv preprint ment of the phenyl group by the cyclohexyl group led to a significant improvement of the inhibitory activity against the recombinant SARS-CoV M pro and to a dramatic improvement in case of CVB3 3C pro . Even for the HCoV-NL63 M pro , against which 11a was completely inactive, greatly improved albeit still weak activity was observed (Table 1 ). In the viral replicons, 11r performed very well, with EC50 = 0.8 -0.9 µM for the EV-A71 replicon, 0.45 µM for CVB3, and 1.4 µM for SARS-CoV (Table 2 ). In the virus-infected cell culture assays (Table 3) , 11r exhibited EC50 = 3.7 µM against EV-A71 in RD cells and EC50 = 0.48 -0.7 µM against EV-D68, HRV2, and HRV14 in HeLa cells. Against HCoV 229E in Huh7 cells, the EC50 was surprisingly low (1.8 µM). Interestingly, the compound proved extremely potent against MERS-CoV in Huh7 cells, with EC50 = 0.0004 µM (400 picomolar!). Even in Vero cells, EC50 against MERS-CoV was 5 µM, and the EC50 against SARS-CoV in Vero E6 cells was 1.8 -2.1 µM, i.e. the best activity we have seen for an M pro inhibitor against SARS-CoV in this type of cells. The therapeutic index (CC50/EC50) of 11r against EV-D68, HRV2, and HRV14 was >15 in HeLa Rh cells as well as against CVB3 in Huh-T7 cells, but only ~5 for EV-A71 in RD cells.\\r\\n At this point, we decided to systematically vary the size of the ring system in P2. The next substituent to be tried was cyclopropylmethyl (compound 11s, which showed good activities against SARS-CoV M pro (IC50 = 0.24 µM) and HCoV-NL63 M pro (1.4 µM), but poor values against EV-A71 3C pro (IC50 = 18.5 µM) and CVB3 3C pro (IC50 = 4.3 µM) (Table 1) . 11s was shown to inhibit the SARS-CoV replicon with an EC50 of about 2 µM, whereas activity against the EV-A71 and CVB3 replicons was poor (EC50 values > 20 µM) ( Table 2 ). The replicon results were mirrored by the antiviral activity of 11s in enterovirus-infected cells (Table 3) , which was weak or very weak. By contrast, the compound inhibited HCoV 229E and MERS-CoV in Huh7 cells with EC50 of 1.3 and 0.08 µM, respectively. The activity against the latter virus in Vero cells was poor (EC50 ~11 µM), and so was the anti-SARS-CoV activity in Vero E6 cells (Table 3) .\\r\\n We next analyzed the crystal structure of the complex between SARS-CoV M pro and compound 11s (Fig. 3f) . The cyclopropylmethyl substituent was found to be incorporated deeply into the S2 pocket, making hydrophobic interactions with Met49 (the lid), Met165 (the floor), and the Cb of Asp187 (the back-wall). In spite of the small size of the P2 substituent, this is possible because the S2 pocket of SARS-CoV M pro is flexible enough to contract and enclose the P2 moiety tightly. This plasticity is expressed in a conformational change of residue Gln189, both in the main chain and in the side-chain. The main-chain conformational change is connected with a flip of the peptide between Gln189 and Thr190. The c1 torsion angle of the Gln189 side-chain changes from roughly antiperiplanar (ap) to (-)synclinal (-sc) (Fig. 4) . The conformational variability of Gln189 has been noted before, both in Molecular Dynamics simulations 26 and in other crystal structures. 37 As a consequence of these changes, the sidechain oxygen of Gln189 can accept a 2.54-Å hydrogen bond from the main-chain NH of the P2 residue in the 11s complex (see Fig. 4 ). The affinity of 11s for the S2 pocket of HCoV-NL63 M pro is good because of an almost ideal match of size and not requiring conformational changes, which this enzyme would not be able to undergo because of the replacement of the flexible Gln189 by the more rigid Pro. On the other hand, docking of the same compound into the crystal structure of the CVB3 3C pro revealed that the cyclopropylmethyl moiety was probably unable to generate sufficient Free Energy of binding because of the missing lid and the large size of the S2 pocket in the enterovirus 3C pro , thereby explaining the poor inhibitory activity of 11s against these targets. (Table 1) . Experiments with the viral replicons confirmed this trend, although the EC50 value for SARS-CoV (6.8 µM) was surprisingly high (Table 2 ). In Huh7 cells infected with MERS-CoV, this compound exhibited EC50 = 0.1 µM (but 9.8 µM in Vero cells), whereas EC50 was 7.0 µM against SARS-CoV in Vero E6 cells. The compound was largely inactive against EV-A71 in RD cells and inhibited the replication of the two HRV subtypes tested (in HeLa Rh cells) with EC50 values of ~4 µM. The CC50 of 11t in HeLa cells was 65 µM, i.e. the therapeutic index was well above 15 (Table 3) .\\r\\n Obviously, this substituent was still a bit too small for the enterovirus proteases, so as the next step, we tested P2 = cyclopentylmethyl (compound 11u). This turned out to be the one compound with acceptable IC50 values against all tested enzymes: 1.3 µM against SARS-CoV M pro , 5.4 µM against HCoV-NL63 M pro , 4.7 µM against EV-A71 3C pro , and 1.9 µM against CVB3 3C pro ( Table 1 ). The activity against the replicons was between 3.6 and 4.9 µM ( Table 2 ). In Huh7 cells infected with HCoV 229E or MERS-CoV, 11u showed EC50 = 2.5 or 0.03 µM (11.1 µM for MERS-CoV in Vero cells), while EC50 was 4.9 µM against SARS-CoV in Vero E6 cells (Table 3) .\\r\\n 11u appeared so far the best compromise compound, yet for each of the individual viral enzymes, the following compounds proved superior: P2 = cyclopropylmethyl (compound 11s) for SARS-CoV M pro , P2 = isobutyl (compound 11n) and P2 = cyclopropylmethyl (11s) for HCoV-NL63 M pro , P2 = benzyl (11a) or cyclohexylmethyl (11r) for EV-A71 3C pro , and 11r for CVB3 3C pro . In other words, the nearly equipotent 11u is indeed a compromise. Therefore, in view of the surprisingly good antiviral activity of 11r against HCoV 229E in Huh7 cells, we relaxed the condition that the universal inhibitor should show good activity against the recombinant HCoV-NL63 M pro , and selected 11r (P2 = cyclohexylmethyl) as the lead compound for further development. This compound exhibited submicromolar IC50 values against CVB3 3C pro and SARS-CoV M pro , and IC50 = 1.7 µM against EV-A71 3C pro (Table 1) , as well as similarly low EC50 values in the replicons of these viruses (Table 2 ). In Huh7 cells infected with MERS-CoV, the performance of this compound was excellent, with EC50 = 0.0004 µM, and even against HCoV 229E in Huh7 cells and SARS-CoV in Vero E6 cells, EC50 values of 1.8 and 2.1 µM, respectively, were observed (Table 3) . Also in enterovirus-infected cell culture, the compound performed well, with EC50 values of 0.7 µM or below against HRV2, HRV14, and EV-D68 in HeLa (Rh) cells and selectivity values >15. The only concern is the activity of the compound against EV-A71 in RD cells, for which the EC50 value was 3.7 µM, resulting in too low a therapeutic index. On the other hand, only weak toxicity was detected for 11r in Vero or Huh-T7 cells. Preliminary pharmacokinetics tests with the compound in mice did not indicate a toxicity problem (to be published elsewhere).\\r\\n We describe here the structure-based design, the synthesis, and the assessment of capped dipeptide a-ketoamides that target the main protease of alpha-or betacoronaviruses as well as the 3C protease of enteroviruses. Through crystallographic analyses of a total of six inhibitor complexes of three different proteases in this study, we found the a-ketoamide warhead (-CO-CO-NH-) to be sterically more versatile than other warheads such as Michael acceptors (-CH=CH-CO-) and aldehydes (-CH=O), because it features two acceptors for hydrogen bonds from the protein, namely the a-keto oxygen and the amide oxygen, whereas the other warheads have only one such acceptor. In the various complexes, the hydroxy group (or oxyanion) of the thiohemiketal that is formed by the nucleophilic attack of the activesite cysteine residue onto the a-keto carbon, can accept one or two hydrogen bonds from the main-chain amides of the oxyanion hole. In addition, the amide oxygen of the inhibitor accepts a hydrogen bond from the catalytic His side-chain. Alternatively, the thiohemiketal can interact with the catalytic His residue and the amide oxygen with the main-chain amides of the oxyanion hole. Depending on the exact interaction, the stereochemistry at the thiohemiketal C atom would be different. We have previously observed a similar difference in case of aldehyde inhibitors, where the single interaction point -the oxyanion of the thiohemiacetal -can accept a hydrogen bond either from the oxyanion hole or from the catalytic His side-chain, 37 resulting in different stereochemistry of the thiohemiacetal carbon. Both aketoamides and aldehydes react reversibly with the catalytic nucleophile of proteases, whereas Michael acceptors form irreversible adducts.\\r\\n In addition to better matching the H-bonding donor/acceptor properties of the catalytic center through offering two hydrogen-bond acceptors instead of one, a-ketoamides have another big advantage over aldehydes and a,b-unsaturated esters (Michael acceptors) in that they allow easy extension of the inhibitors to probe the primed specificity subsites beyond S1', although this has so far rarely been explored (e.g., ref. 38 in case of calpain).\\r\\n The most prominent a-ketoamide drugs are probably telaprivir and boceprivir, peptidomimetic inhibitors of the HCV NS3/4A protease, 39, 40 which have helped revolutionize the treatment of chronic HCV infections. For viral cysteine proteases, a-ketoamides have only occasionally been described as inhibitors and few systematic studies have been carried out.\\r\\n A number of capped dipeptidyl a-ketoamides have been described as inhibitors of the norovirus 3C-like protease. 41 These were optimized with respect to their P1' substituent, whereas P2 was isobutyl in most cases and occasionally benzyl. The former displayed IC50 values one order of magnitude lower than the latter, indicating that the S2 pocket of the norovirus 3CL protease is fairly small. Although we did not include the norovirus 3CL pro in our study, expanding the target range of our inhibitors to norovirus is probably a realistic undertaking. While our study was underway, Zeng et al. 42 published a series of a-ketoamides as inhibitors of the EV-A71 3C pro . These authors mainly studied the structure-activity relationships of the P1' residue and found small alkyl substituents to be superior to larger ones. Interestingly, they also reported that a six-membered d-lactam in the P1 position led to 2 -3 times higher activities, compared to the fivemembered g-lactam. At the same time, Kim et al. 43 described a series of five a-ketoamides with P1' = cyclopropyl that showed submicromolar activity against EV-D68 and two HRV strains.\\r\\n Occasionally, individual a-ketoamides have been reported in the literature as inhibitors of both the enterovirus 3C protease and the coronavirus main protease. A single capped dipeptidyl a-ketoamide, Cbz-Leu-GlnLactam-CO-CO-NH-iPr, was described that inhibited the recombinant transmissible gastroenteritis virus (TGEV) and SARS-CoV M pro s as well as human rhinovirus and poliovirus 3C pro s in the one-digit micromolar range. 44 Coded GC-375, this compound showed poor activity in cell culture against EV-A71 though (EC50 = 15.2 µM), probably because P2 was isobutyl. As we have shown here, an isobutyl side-chain in the P2 position of the inhibitors is too small to completely fill the S2 pocket of the EV-A71 3C pro and the CVB3 3C pro .\\r\\n Among a series of aldehydes, Prior et al. 45 described the capped tripeptidyl a-ketoamide Cbz-1-naphthylalanine-Leu-GlnLactam-CO-CO-NH-iPr, which showed IC50 values in the 3-digit nanomolar range against HRV 3C pro and SARS-CoV M pro , as well as EC50 values of 0.03 µM against HRV18 and 0.5 µM against HCoV 229E in cell culture. No optimization of this compound was performed and no toxicity data have been reported.\\r\\n For compounds with warheads other than a-ketoamides, in-vitro activity against both corona-and enteroviruses has also occasionally been reported. Lee et al. 46 described three peptidyl Michael acceptors that displayed inhibitory activity against the M pro s of SARS-CoV and HCoV 229E as well as against the 3C pro of CVB3. These inhibitors had an IC50 10 -20 times higher for the CVB3 enzyme, compared to SARS-CoV M pro . P2 was invariably isobutyl (leucine) in these compounds, suggesting that further improvement might be possible.\\r\\n In addition to Michael acceptors, peptide aldehydes have also been used to explore the inhibition of coronavirus M pro s as well as enterovirus 3C pro s. Kim et al. 44 reported a dipeptidyl aldehyde and its bisulfite adduct, both of which exhibited good inhibitory activities against the isolated 3C proteases of human rhinovirus and poliovirus as well as against the 3C-like proteases of a number of coronaviruses, but antiviral activities in cell culture against EV-A71 were poor (EC50 >10 µM), again most probably due to P2 being isobutyl (leucine).\\r\\n In our series of compounds, we used P1 = GlnLactam (g-lactam) throughout, because this substituent has proven to be an excellent surrogate for glutamine. 29, 32 While we made some efforts to optimize the P1' residue of the compounds as well as the N-cap (P3), we mainly focussed on optimization of the P2 substituent. In nearly all studies aiming at discovering peptidomimetic inhibitors of coronavirus M pro s, P2 is invariably isobutyl (leucine), and this residue has also been used in the efforts to design compounds that would inhibit enterovirus 3C pro s as well (see above). From crystal structures of our early lead compound, 11a (cinnamoyl-Phe-GlnLactam-CO-CO-NH-Bz), in complex with the M pro s of HCoV NL63 (as representative of the alphacoronavirus proteases) and SARS-CoV (beta-CoV) as well as the 3C pro of Coxsackievirus B3 (enterovirus proteases), we found that the S2 pocket has fundamentally different shapes in these enzymes. In the SARS-CoV M pro , the S2 subsite is a deep hydrophobic pocket that is truly three-dimensional in shape: the \\\"walls\\\" of the groove are formed by the polypeptide main chain around residues 186 -188 as well as by the side-chains of His41 (of the catalytic dyad) and Gln189, whereas the \\\"floor\\\" is formed by Met165 and the \\\"lid\\\" by residues 45 -51, in particular Met49. The two methionines provide important interaction points for the P2 substituents of inhibitors; while these interactions are mostly hydrophobic in character, we have previously described the surprising observation of the carboxylate of an aspartic residue in P2 that made polar interactions with the sulfur atoms of these methionines. 37 Because the pocket offers so many opportunities for interaction and features a pronounced plasticity, P2 substituents such as isobutyl (from Leu), which are too small to fill the pocket entirely, can still generate sufficient binding enthalpy. Accordingly, the S2 pocket of SARS-CoV M pro is the most tolerant among the three enzymes investigated here, in terms of versatility of the P2-substituents accepted.\\r\\n In the S2 pocket of the HCoV-NL63 M pro , Gln189 is replaced by proline and this change is accompanied by a significant loss of flexibility; whereas the side-chain of Gln189 of SARS-CoV M pro is found to accommodate its conformation according to the steric requirements of the P2 substituent, the proline is less flexible, leading to a much smaller space at the entrance to the pocket. As a consequence, a P2benzyl substituent is hindered from penetrating deeply into the pocket, whereas the smaller and more flexible isobutyl group of P2-Leu is not.\\r\\n Finally, in the 3C pro s of EV-A71 and CVB3, the S2 pocket lacks a lid, i.e. it is open to one side. As a consequence, it offers less interaction points for P2 substituents of inhibitors, so that such substituents must reach the \\\"back-wall\\\" of the pocket (formed by Arg39, Asn69, Glu71) in order to create sufficient binding energy. Hence, large aromatic substituents such as benzyl are favored by the enterovirus 3C pro s.\\r\\n When we introduced a fluoro substituent in the para position of the P2-benzyl group of our lead compound, 11a, we observed good activity against the enterovirus 3C pro s but complete inactivity against the coronavirus M pro s (see Table 1 , compound 11m). This is easily explained on the basis of the crystal structures: In the enterovirus 3C pro s, the fluorine can accept a hydrogen bond from Arg39 (ref. 30) , whereas in the coronavirus M pro s, there would be electrostatic repulsion from the main-chain carbonyls of residues 186 and 188. In agreement with this, rupintrivir (which has P2 = p-fluorobenzyl) is a good inhibitor of the enteroviral 3C pro s, 46 but not of the coronaviral main proteases, as we predicted earlier. 28 In this structure-based inhibitor optimization study, we achieved major improvements over our original lead compound, 11a, by systematically varying the size and the flexibility of the P2-substituent. The compound presenting so far the best compromise between the different requirements of the S2 pockets (SARS-CoV M pro : large and covered, HCoV-NL63 M pro : small and covered, and CVB3 3C pro : large and open) is 11u (P2 = cyclopentylmethyl), which has satisfactory broad-spectrum activity against all proteases tested. However, with regard to its antiviral activities in cell cultures, it is inferior to 11r (P2 = cyclohexylmethyl). The latter compound exhibits very good inhibitory activity against the SARS-CoV M pro as well as the enterovirus 3C pro , and its performance in the SARS-CoV and enterovirus replicons is convincing. Being in the low micromolar range (EV-A71, CVB3), the data for the antiviral activity in cell culture for 11r correlate well with the inhibitory power of the compound against the recombinant proteases as well as in the replicon-based assays. This is not true, though, for the surprisingly good in-cellulo activity of 11r against HCoV 229E in Huh7 cells. Also, the correlation does not seem to hold for LLC-MK2 and CaCo2 cells. We tested the antiviral activity of many of our compounds against HCoV NL63 in these two cell types and found that all of them had low-or submicromolar EC50 values against this virus in LLC-MK2 cells but were largely inactive in CaCo2 cells (not shown). Furthermore, 11r and all other compounds that we synthesized are inactive (EC50 > 87 µM) against CVB3 in Vero cells (not shown), but exhibit good to excellent activities against the same virus in Huh-T7 cells. We have previously observed similar poor antiviral activities in Vero cells not only for aketoamides, but also for Michael acceptors (Zhu et al., unpublished work) . A similar cell-type dependence is seen for the antiviral activity of 11r against MERS-CoV and SARS-CoV. Whereas the inhibitor exhibits excellent activity against MERS-CoV when Huh7 cells are the host cells (400 pM), the inhibitory activity is weaker by a factor of up to 12,500 when Vero cells are used (EC50 = 5 µM). On the other hand, 11r exhibits excellent anti-MERS-CoV activity in human Calu3 lung cells, i.e. in the primary target cells where the compound will have to act in a therapeutic setting (A. Kupke, personal communication). As we tested antiviral activity against SARS-CoV exclusively in Vero cells, the EC50 values determined for our compounds against this virus are in the one-digit micromolar range or higher; the best is again compound 11r with EC50 = 2.1 µM. Interestingly, the relatively weaker activity (or even inactivity) of our inhibitors against RNA viruses in Vero cells was observed independently in the virology laboratories in Leuven and in Leiden. It is thus unlikely that the lack of activity in Vero cells is related to problems with the experimental set-up. In preliminary experiments, we replaced the P3 cinnamoyl group of 11r by the fluorophor coumaryl and found by fluorescence microscopy that much more inhibitor appeared to accumulate in Huh7 cells compared to Vero cells (D.L., R.H. & Irina Majoul, unpublished).\\r\\n Regardless of which cell system is the most suitable one for the testing of peptidomimetic antiviral compounds, we next plan to test 11r in small-animal models for MERS and for Coxsackievirusinduced pancreatitis. In parallel, we aim to refine the experiments to quantify the accumulation of peptidomimetic protease inhibitors in different host-cell types, in the hope to find an explanation for the observed cell-type dependencies.\\r\\n This work demonstrates the power of structure-based approaches in the design of broad-spectrum antiviral compounds with roughly equipotent activity against coronaviruses and enteroviruses. We observed a good correlation between the inhibitory activity of the designed compounds against the isolated proteases, in viral replicons, and in virus-infected Huh7 cells. One of the compounds (11r) exhibits excellent anti-MERS-CoV activity in virus-infected Huh7 cells. Because of the high similarity between the main proteases of SARS-CoV and the novel BetaCoV/Wuhan/2019, we expect 11r to exhibit good antiviral activity against the new coronavirus as well.\\r\\n Crystallization. The recombinant production and purification of SARS-CoV M pro with authentic N and C termini was described in detail previously. 48, 49 Using an Amicon YM10 membrane (EMD Millipore), the purified SARS-CoV M pro was concentrated to 21 mg·mL -1 in buffer A (20 mM Tris-HCl, 150 mM NaCl, 1 mM DTT, 1 mM EDTA, pH 7.5). Crystallization was performed by equilibrating 1 µL protein (mixed with 1 µL precipitant solution) against 500 µl reservoir containing 6 -8% polyethylene glycol (PEG) 6,000, 0.1 M MES (pH 6.0), at 20℃ using the vapor diffusion sitting-drop method. Compounds 11a and 11s were dissolved in 100% DMSO at 50 mM and 200 mM stock concentrations, respectively. A crystal of the free enzyme was soaked in cryo-protectant buffer containing 20% MPD, 6% PEG 6,000, 0.1 M MES, 7.5 mM 11a, pH 6.0, for 2 h at 20℃. Another set of free enzyme crystals was soaked in another cryoprotectant buffer with 6% PEG 6,000, 5% MPD, 0.1 M MES, 15% glycerol, 10 mM 11s, pH 6.0, for 2 h. Subsequently, crystals were fished and flash-cooled in liquid nitrogen prior to data collection.\\r\\n Crystals of HCoV-NL63 M pro with 11a were obtained using cocrystallization. The concentrated HCoV-NL63 M pro (45 mg·mL -1 ) was incubated with 5 mM 11a for 4 h at 20℃, followed by setting up crystallization using the vapor diffusion sitting-drop method at 20℃ with equilibration of 1 µL protein (mixed with 1 µL mother liquor) against 500 µL reservoir composed of 0.1 M lithium sulfate monohydrate, 0.1 M sodium citrate tribasic dihydrate, 25% PEG 1,000, pH 6.0. The crystals were protected by a cryo-buffer containing 0.1 M lithium sulfate monohydrate, 0.1 M sodium citrate tribasic dihydrate, 25% PEG 1,000, 15% glycerol, 2 mM 11a, pH 6.0 and flash-cooled in liquid nitrogen.\\r\\n Crystals of HCoV-NL63 M pro with 11n or 11f were generated by using the soaking method. Several free-enzyme crystals were soaked in cryo-protectant buffer containing 0.1 M lithium sulfate monohydrate, 0.1 M sodium citrate tribasic dihydrate, 25% PEG 1,000, 15% glycerol, 5 mM 11n (or 11f), pH 6.0. Subsequently, the soaked crystals were flash-cooled in liquid nitrogen.\\r\\n Freshly prepared CVB3 3C pro at a concentration of 21.8 mg·mL -1 was incubated with 5 mM 11a pre-dissolved in 100% DMSO at room tempature for 1 h. Some white precipitate appeared in the mixture. Afterwards, the sample was centrifuged at 13,000 x g for 20 min at 4 ℃. The supernatant was subjected to crystallization trials using the following, commercially available kits: Sigma TM (Sigma-Aldrich), Index TM , and PEG Rx TM (Hampton Research). Single rod-like crystals were detected both from the Index TM screen, under the condition of 0.1 M MgCl2 hexahydrate, 0.1 M Bis-Tris, 25% PEG 3,350, pH 5.5, and from the Sigma TM screen at 0.2 M Li2SO4, 0.1 M Tris-HCl, 30% PEG 4,000, pH 8.5. Crystal optimization was performed by using the vapor-diffusion sitting-drop method, with 1 μL CVB3 3C pro -inhibitor complex mixed with 1 μL precipitant solution, and equilibration against 500 μL reservoir containing 0.1 M Tris-HCl, 0.2 M MgCl2, pH 8.5, and PEG 3,350 varied from 22% to 27%. Another optimization screen was also performed against a different reservoir, 0.1 M Tris-HCl, 0.2 M MgCl2, pH range from 7.5 to 8.5, and PEG 4,000 varied from 24% to 34%. Crystals were fished from different drops and protected by cryo-protectant solution consisting of the mother liquor and 10% glycerol. Subsequently, the crystals were flash-cooled with liquid nitrogen.\\r\\n Diffraction data collection, structure elucidation and refinement. Diffraction data from the crystal of the SARS-CoV M pro in complex with 11a were collected at 100 K at synchrotron beamline PXI-X06SA (PSI, Villigen, Switzerland) using a Pilatus 6M detector (DECTRIS). A diffraction data set from the SARS-CoV M pro crystal with compound 11s was collected at 100 K at beamline P11 of PETRA III (DESY, Hamburg, Germany), using the same type of detector. All diffraction data sets of HCoV-NL63 M pro complex structures and of the complex of CVB3 3C pro with 11a were collected at synchrotron beamline BL14.2 of BESSY (Berlin, Germany), using an MX225 CCD detector (Rayonics). All data sets were processed by the program XDSAPP and scaled by SCALA from the CCP4 suite. [50] [51] [52] The structure of SARS-CoV M pro with 11a was determined by molecular replacement with the structure of the complex between SARS-CoV M pro and SG85 (PDB entry 3TNT; Zhu et al., unpublished) as search model, employing the MOLREP program (also from the CCP4 suite). 52, 53 The complex structures of HCoV-NL63 M pro with 11a, 11f, and 11n were also determined with MOLREP, using as a search model the structure of the free enzyme determined by us (LZ et al., unpublished) . The complex structure between CVB3 3C pro and 11a was determined based on the search model of the free-enzyme structure (PDB entry 3ZYD; Tan et al., unpublished) . Geometric restraints for the compounds 11a, 11f, 11n, and 11s were generated by using JLIGAND 52, 54 and built into Fo-Fc difference density using the COOT software. 55 Refinement of the structures was performed with REFMAC version 5.8.0131 (ref. 52, 56, 57) .\\r\\n A buffer containing 20 mM Tris-HCl, 100 mM NaCl, 1 mM EDTA, 1 mM DTT, pH 7.3, was used for all the enzymatic assays. Two substrates with the cleavage sites of M pro and 3C pro , respectively (indicated by the arrow, ↓), Dabcyl-KTSAVLQ¯SGFRKM-E(Edans)-NH2 and Dabcyl-KEALFQ¯GPPQF-E(Edans)-NH2 (95% purity; Biosyntan), were employed in the fluorescence resonance energy transfer (FRET)based cleavage assay, using a 96-well microtiter plate. The dequenching of the Edans fluorescence due to the cleavage of the substrate as catalyzed by the proteases was monitored at 460 nm with excitation at 360 nm, using a Flx800 fluorescence spectrophotometer (BioTek). Curves of relative fluorescence units (RFU) against substrate concentration were linear for all substrates up to beyond 50 µM, indicating a minimal influence of the inner-filter effect. Stock solutions of the compounds were prepared by dissolving them in 100% DMSO. The UV absorption of 11a was found to be negligible at l = 360 nm, so that no interference with the FRET signal through the inner-filter effect was to be expected. For the determination of the IC50, different proteases at a specified final concentration (0.5 µM SARS-CoV or HCoV-NL63 M pro , 2 µM CVB3 3C pro , 3 µM EV-A71 3C pro ) were separately incubated with the inhibitor at various concentrations (0 to 100 μM) in reaction buffer at 37℃ for 10 min. Afterwards, the reaction was initiated by adding FRET peptide substrate at 20 μM final concentration (final volume: 50 μl). The IC50 value was determined by using the GraphPad Prism 6.0 software (GraphPad). Measurements of enzymatic activity were performed in triplicate and are presented as the mean ± standard deviations (SD).\\r\\n Assessment of inhibitory activity of a-ketoamides using viral replicons and virus-infected cells Cells and viruses. Hepatocellular carcinoma cells (Huh7; ref. 58) and their derivative constitutively expressing T7 RNA polymerase (Huh-T7; ref. 59) were grown in Dulbecco's modified minimal essential medium (DMEM) supplemented with 2 mM glutamine, 100 U·mL -1 penicillin, 100 µg·mL -1 streptomycin sulfate, and fetal calf serum (10% in growth medium and 2% in maintenance medium). Huh-T7 cells were additionally supplemented with geneticin (G-418 sulfate, 400 µg·mL -1 ). Huh-T7 cells were used for the enteroviral replicons as well as for infection experiments with CVB3 strain Nancy.\\r\\n For enterovirus (except CVB3) infection experiments, human rhabdomyosarcoma cells (RD; for EV-A71; BRCR strain) and HeLa Rh cells (for EV-D68 and human rhinoviruses) were grown in MEM Rega 3 medium supplemented with 1% sodium bicarbonate, 1% Lglutamine, and fetal calf serum (10% in growth medium and 2% in maintenance medium). For HCoV-229E (a kind gift from Volker Thiel (Bern, Switzerland)), culture and infection experiments were carried out as described. 60 For MERS-CoV or SARS-CoV infection experiments, Vero, Vero E6, and Huh7 cells were cultured as described previously. 61, 62 Infection of Vero and Huh7 cells with MERS-CoV (strain EMC/2012) and SARS-CoV infection of Vero E6 cells (strain Frankfurt-1) at low multiplicity of infection (MOI) were done as described before. 61, 63 All work with live MERS-CoV and SARS-CoV was performed inside biosafety cabinets in biosafety level-3 facilities at Leiden University Medical Center, The Netherlands. Viral replicons. The DNA-launched SARS-CoV replicon harbouring Renilla luciferase as reporter directly downstream of the SARS-CoV replicase polyprotein-coding sequence (pp1a, pp1ab, Urbani strain, acc. AY278741), in the context of a bacterial artificial chromosome (BAC) under the control of the CMV promoter, has been described previously (pBAC-REP-RLuc). 33 Apart from the replicase polyprotein, the replicon encodes the following features: the 5'-and 3'-non-translated regions (NTR), a ribozyme (Rz), the bovine growth hormone sequence, and structural protein N.\\r\\n Subgenomic replicons of CVB3 (pT7-CVB3-FLuc 34 ) and EV-A71 (pT7-EV71-RLuc) harbouring T7-controlled complete viral genomes, in which the P1 capsid-coding sequence was replaced by the Firefly (Photinus pyralis) or Renilla (Renilla renifor) luciferase gene, were generously provided by F. van Kuppeveld and B. Zhang, respectively. To prepare CVB3 and EV-A71 replicon RNA transcripts, plasmid DNAs were linearized by digestion with SalI or HindIII (New England Biolabs), respectively. Copy RNA transcripts were synthesized in vitro using linearized DNA templates, T7 RNA polymerase, and the T7 RiboMax™ Large-Scale RNA Production System (Promega) according to the manufacturer's recommendations.\\r\\n Transfection. Huh-T7 cells grown in 12-well plates to a confluency of 80% -90% (2 -3 x 10 5 cells/well) were washed with 1 mL OptiMEM (Invitrogen) and transfected with 0.25 µg of the replication-competent replicon and Lipofectamin2000 or X-tremeGENE9 in 300 µl OptiMEM (final volume) as recommended by the manufacturer (Invitrogen or Roche, respectively). The transfection mixtures were incubated at 37°C for 4 to 5 h (Lipofectamin2000) or overnight (X-tremeGENE9), prior to being replaced with growth medium containing the compound under investigation. For RNA-launched transfection of enteroviral replicons, DMRIE-C was used as transfection reagent according to the manufacturer's recommendations (Invitrogen). All experiments were done in triplicate or quadruplicate and the results are presented as mean values ± SD.\\r\\n Testing for inhibitory activity of candidate compounds. Initially, we performed a quick assessment of the inhibitory activity of the candidate compounds towards the enteroviral and coronaviral replicons at a concentration of 40 µM in Huh-T7 cells. Compounds that were relatively powerful and non-toxic at this concentration, were assayed in a dose-dependent manner to estimate their half-maximal effective concentration (EC50) as well as their cytotoxicity (CC50), as described. 29 In brief, different concentrations of a-ketoamides (40 µM in screening experiments or increasing concentration (0, 1.25, 2.5, 5, 10, 20, 40 µM) when determining the EC50) were added to growth medium of replicon-transfected Huh-T7 cells. Twenty-four hours later, the cells were washed with 1 mL phosphate-buffered saline (PBS or OPTIMEM, Invitrogen) and lysed in 0.15 mL Passive lysis buffer (Promega) at room temperature (RT) for 10 min. After freezing (-80 o C) and thawing (RT), the cell debris was removed by centrifugation (16,000 x g, 1 min) and the supernatant (10 or 20 µl) was assayed for Firefly or Renilla luciferase activity (Promega or Biotrend Chemikalien) using an Anthos Lucy-3 luminescence plate reader (Anthos Microsystem).\\r\\n The antiviral activity of the compounds was evaluated in a cytopathic effect (CPE) readout assay using MTS [3-(4,5-dimethylthiazol-2-yl)-5-(3carboxymethoxyphenol)-2-(4-sulfophenyl)-2H-tetrazolium, innersalt]-based assay. Briefly, 24 h prior to infection, cells were seeded in 96-well plates at a density of 2.5 x 10 4 (RD cells) or of 1.7 x 10 4 (HeLa Rh) per well in medium supplemented with 2% FCS. For HRV2 and HRV14 infection, the medium contained 30 mM MgCl2. The next day, serial dilutions of the compounds and virus inoculum were added. The read-out was performed 3 days post infection as follows: The medium was removed and 100 μl of 5% MTS in Phenol Red-free MEM was added to each well. Plates were incubated for 1 h at 37°C, then the optical density at 498 nm (OD498) of each well was measured by microtiter plate reader (Saffire 2 , Tecan). The OD values were converted to percentage of controls and the EC50 was calculated by logarithmic interpolation as the concentration of compound that results in a 50% protective effect against virus-induced CPE. For each condition, cell morphology was also evaluated microscopically.\\r\\n Assays with MERS-CoV and SARS-CoV were performed as previously described. 61, 63 In brief, Huh7, Vero, or Vero E6 cells were seeded in 96-well plates at a density of 1 × 10 4 (Huh7 and Vero E6) or 2 × 10 4 cells (Vero) per well. After overnight growth, cells were treated with the indicated compound concentrations or DMSO (solvent control) and infected with an MOI of 0.005 (final volume 150 µl/well in Eagle's minimal essential medium (EMEM) containing 2% FCS, 2 mM L-glutamine, and antibiotics). Huh7 cells were incubated for two days and Vero/VeroE6 cells for three days, and differences in cell viability caused by virus-induced CPE or by compound-specific side effects were analyzed using the CellTiter 96 AQueous Non-Radioactive Cell Proliferation Assay (Promega), according to the manufacturer's instructions. Absorbance at 490 nm (A490) was measured using a Berthold Mithras LB 940 96-well plate reader (Berthold). Cytotoxic effects caused by compound treatment alone were monitored in parallel plates containing mock-infected cells. HCoV-229E-Rev: 5′-ggTCGTTTAGTTGAGAAAAGT -3′, and 229E-ZNA probe: 5'-6-Fam-AGA (pdC)TT(pdU)G(pdU)GT(pdC)TA(pdC)T-ZNA-3-BHQ-1 -3' (Metabion). Standard curves were prepared using serial dilutions of RNA isolated from virus stock. Data were analysed using GraphPad Prism 5.0; EC50 values were calculated based on a 4parameter logistic statistics equation. In parallel to the qPCR assays with inhibitors, cell viability assays were performed using Alamar-Blue™ Cell Viability Reagent (ThermoFisher) according to the manufacturer's instruction. CC50 values were calculated using inhibitor versus normalized response statistics equation by including proper controls (no inhibitor and 1% Triton-X-100-treated cells).\\r\\n Determination of the cell toxicity of candidate compounds. The CellTiter 96Aqueous One Solution Cell Proliferation Assay (MTS test, Promega), the CellTiter Glo assay kit (Promega), the Non-Destructive Cytotoxicity Bio-Assay (ToxiLight (measuring the release of adenylate kinase from damaged cells), Lonza Rockland), or the AlamarBlue™ Cell Viability Reagent (ThermoFisher) were used to determine the cytotoxic effect of compounds towards host cells according to the manufacturers' recommendations. 29, 65 Chemical synthesis of a-ketoamides General procedure. Reagents were purchased from commercial sources and used without purification. HSGF 254 (0.15 -0.2 mm thickness) was used for analytical thin-layer chromatography (TLC). All products were characterized by their NMR and MS spectra. 1 H NMR spectra were recorded on 300-MHz, 400-MHz, or 500-MHz instruments. Chemical shifts were reported in parts per million (ppm, δ) down-field from tetramethylsilane. Proton coupling patterns were described as singlet (s), doublet (d), triplet (t), quartet (q), multiplet (m), and broad (br). Mass spectra were recorded using a Bruker ESI ion-trap HCT Ultra. HPLC spectra were recorded by LC20A or LC10A (Shimadzu Corporation) with Shim-pack GIST C18 (5 µm, 4.6x150mm) with three solvent systems (methanol/water, methanol/ 0.1% HCOOH in water or methanol/0.1% ammonia in water). Purity was determined by reversed-phase HPLC and was ≥95% for all compounds tested biologically.\\r\\n Synthesis of (2S,4R)-dimethyl 2-(tert-butoxycarbonylamino)-4-(cyanomethyl) pentanedioate (1).\\r\\n To a solution of N-Boc-L-glutamic acid dimethyl ester (6.0 g, 21.8 mmol) in THF (60 mL) was added dropwise a solution of lithium bis(trimethylsilyl)amide (LHMDS) in THF (47 mL, 1 M) at −78°C under nitrogen. The resulting dark mixture was stirred at −78°C. Meanwhile, bromoacetonitrile (1.62 mL, 23.3 mmol) was added dropwise to the dianion solution over a period of 1 h while keeping the temperature below −70°C. The reaction mixture was stirred at −78°C for additional 2 h. After the consumption of the reactant was confirmed by TLC analysis, the reaction was quenched by methanol (3 mL), and acetic acid (3 mL) in precooled THF (20 mL) was added.\\r\\n After stirring for 30 min, the cooling bath was removed. The reaction mixture was allowed to warm up to room temperature and then poured into brine (40 mL). The organic layer was concentrated and purified by flash column chromatography (petroleum ether/ethyl acetate = 4/1) to give product 1 (4.92 g, 72%) as colorless oil. 1 H NMR (CDCl3, 400 MHz) δ 5.23 (1H, d, J = 9.0 Hz), 4.43-4.36 (1H, m), 3.77(1H, s), 3.76 (1H, s), 2.89-2.69 (3H, m), 2.20-2.14 (2H, m), 1.45 (9H, s). ESI-MS (m/z): 315 (M + H) + .\\r\\n Synthesis of (S)-methyl 2-(tert-butoxycarbonylamino)-3-((S)-2oxopyrrolidin-3-yl)propanoate (2) .\\r\\n In a hydrogenation flask was placed compound 1 (4.0 g, 12.7 mmol), 5 mL of chloroform and 60 mL of methanol before the addition of PtO2. The resulting mixture was stirred under hydrogen at 20 o C for 12 h. Then the mixture was filtered over Celite to remove the catalyst. NaOAc (6.77 g, 25.5 mmol) was added to the filtrate before the resulting mixture was stirred at 60 o C for 12 h. The reaction was quenched with water (30 mL). The suspension was extracted with ethyl acetate. The organic layers were combined, dried (MgSO4), and filtered. The light-brown filtrate was concentrated and purified by silica gel column chromatography (petroleum ether/ethyl acetate = 4/1) to give the product 2 (2.20 g, 61%) as white solid. 1 H NMR (CDCl3) δ 6.02 (1H, br), 5. Synthesis of (S)-methyl 2-amino-3-((S)-2-oxopyrrolidin-3yl)propanoate (3).\\r\\n Compound 2 (1.0 g, 3.5 mmol) was dissolved in 10 mL dichloromethane (DCM), then 10 mL trifluoroacetic acid (TFA) was added. The reaction mixture was stirred at 20 o C for 0.5 h, and concentrated in vacuo to get a colorless oil, which could be used for the following step without purification.\\r\\n ESI-MS (m/z): 187 (M + H) + .\\r\\n Synthesis of methyl N-substituted amino-acid esters 4 General procedure. The methyl amino-acid ester hydrochloride (6.0 mmol) was dissolved in 20 mL CH2Cl2, and then acyl chloride (6.0 mmol), triethylamine (1.69 mL, 12.0 mmol) were added, before the reaction was stirred for 2 h at 20 o C. The reaction mixture was diluted with 20 mL CH2Cl2, washed with 50 mL of saturated brine (2 × 25 mL), and dried over Na2SO4. The solvent was evaporated and the product 4 was obtained as white solid (70-95% yield), which could be used for the next step without further purification.\\r\\n (S)-methyl 2-cinnamamido-3-phenylpropanoate (4a). The methyl L-phenylalaninate hydrochloride (1.30 g, 6.0 mmol) was dissolved in 20 mL CH2Cl2, and then cinnamoyl chloride (1.00 g, 6.0 mmol), triethylamine (1.69 mL, 12.0 mmol) were added, before the reaction was stirred for 2 h at room temperature. The reaction mixture was diluted with 20 mL CH2Cl2, washed with 50 mL of saturated brine (2×25 mL), and dried over Na2SO4. The solvent was evaporated and the product 4a was obtained as white solid (1.75 g, 95%), which could be used for the next step without further purification.\\r\\n Synthesis of N-substituted amino acids 5 (general procedure). 1 M NaOH (5 mL) was added to a solution of compound 4 (3.0 mmol) in methanol (5 mL). The reaction was stirred for 20 min at 20 o C. Then 1 M HCl was added to the reaction solution until pH = 1. Then the reaction mixture was extracted with 100 mL of CH2Cl2 (2 × 50 mL) and the organic layer was washed with 50 mL of brine and dried over Na2SO4. The solvent was evaporated and the crude material purified on silica, eluted with mixtures of CH2Cl2/MeOH (20/1) to afford the product 5 (90-96% yield) as a white solid.\\r\\n Synthesis of compounds 6 (General procedure). Compound 5 (2.7 mmol) was dissolved in 10 mL of dry CH2Cl2. To this solution, 1.5 equiv (1.54 g) of 1-[bis(dimethylamino)methylene]-1H-1,2,3triazolo[4,5-b]pyridinium 3-oxide hexafluorophosphate (HATU) was added, and the reaction was stirred for 0.5 h at 20 o C. Then compound 3 (500 mg, 2.7 mmol) and TEA (0.70 mL, 5.42 mmol) was added to the reaction. The reaction was stirred for another 6 h. The reaction mixture was poured into 10 mL water. The aqueous solution was extracted with 50 mL of CH2Cl2 (2 × 25 mL) and washed with 50 mL of saturated brine (2 × 25 mL) and dried over Na2SO4. The solvent was evaporated and the crude material purified on silica, eluted with a mixture of CH2Cl2/MeOH (40/1) to give the product 6 (62-84% yield).\\r\\n Synthesis of alcohols 7 (general procedure). Compound 6 (1.1 mmol) was dissolved in methanol (40 mL), then NaBH4 (0.34 g, 8.8 mmol) was added under ambient conditions. The reaction mixture was stirred at 20 o C for 2 h. Then the reaction was quenched with water (30 mL). The suspension was extracted with ethyl acetate. The organic layers were combined, dried, and filtered. The filtrate was evaporated to dryness and could be used for the next step without further purification (46-85% yield).\\r\\n Synthesis of aldehydes 8 (general procedure). Compound 7 (0.75 mmol) was dissolved in CH2Cl2, then Dess-Martin periodinane (337 mg, 0.79 mmol) and NaHCO3 (66 mg, 0.79 mmol) were added. The resulting mixture was stirred at 20 o C for 1 h. The mixture was concentrated and purified by column chromatography on silica gel (CH2Cl2/MeOH = 20/1) to give the product 8 as white solid (88-95% yield).\\r\\n Synthesis of compounds 9 (general procedure). Compound 8 (0.40 mmol) was dissolved in CH2Cl2, and then acetic acid (0.028 g, 0.47 mmol) and isocyanide (0.43 mmol) were added successively to the solution. The reaction was stirred at 20 o C for 24 h. Then the solvent was evaporated and the crude material purified on silica, eluted with a mixture of CH2Cl2/MeOH (20/1) to give the product 9 (46-84%).\\r\\n Synthesis of α-hydroxyamides 10 (general procedure). 1 M NaOH (0.5 mL) was added to a solution of compound 9 (0.164 mmol) in methanol (5 mL). The reaction was stirred at 20 o C for 0.5 h until the consumption of compound 9 was confirmed by TLC analysis. Then, 1 M HCl was added to the reaction solution until pH = 7. Following this, the solvent was evaporated to generate the product 10 as white solid, which could be used directly in the next step.\\r\\n Synthesis of α-ketoamides 11 (general procedure). Compound 10 was dissolved in CH2Cl2, then Dess-Martin periodinane (74 mg, 0.176 mmol) and NaHCO3 (30 mg, 0.176 mmol) were added. The resulting mixture was stirred at 20 o C for 1 h. The mixture was concentrated and purified by column chromatography on silica gel (CH2Cl2/MeOH = 20/1) to give the a-ketoamides 11 as light yellow solid (52-79% in two steps). \\r\\n Detailed results of the variation of the P1' and P3 substituents Synthesis of a-ketoamides 11b -11e and 11g -11l Supplemental Table 1 : inhibitory activities (IC50 (μM)) of α-ketoamides with P1' and P3 modifications against viral proteases Supplemental Table 2 : crystallographic data for complexes between viral proteases and α-ketoamides Molecular formula strings and biological data (CSV)\\r\\n Authors will release the atomic coordinates and experimental data upon article publication. Atomic coordinates include SARS-CoV M pro in complex with compounds 11a (5N19), 11s (5N5O), HCoV-NL63 M pro in complex with 11a (6FV2), 11n (6FV1), 11f (5NH0), and CVB3 3C pro in complex with (5NFS). 1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.10.936898 doi: bioRxiv preprint  1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.10.936898 doi: bioRxiv preprint. Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) or 2019-Novel Coronavirus (2019-nCoV) is the pathogen of Corona Virus Disease 2019 [1] . Nucleic acid detection of 2019-nCoV is one of the key indicators for clinical diagnosis. However, the false negative rate of 2019-nCoV nucleic acid in clinical practice is very high, and the positive rate is only 30-50% [2] . False negative detection means missed detection, which will lead to not only the delayed diagnosis of suspected patients, but also the 2019-nCoV carriers becoming the potential sources of virus infection. Therefore, it is very urgent to improve the detection rate of 2019-nCoV nucleic acid clinically.\\r\\n Currently, fluorescent quantitative RT-PCR (qRT-PCR) technology is mainly used to detect the nucleic acid of 2019-nCoV. The quality of nucleic acid template is admittedly one of the key factors affecting the detection efficiency. According to \\\"The Laboratory Technical Guidelines for Detection 2019-nCoV (Fourth Edition)\\\" issued by National Health and Commission of China [3] and \\\"The Experts' Consensus on Nucleic Acid Detection of 2019-nCoV\\\" released by Chinese Society of Laboratory Medicine [4] , the human samples must be placed in 56 o C for at least 45 minutes or higher for shorter time to inactivate the viruses in order to keep the inspectors from virus infection before the nucleic acids were isolated.\\r\\n However, 2019-nCoV is a single stranded RNA virus. Theoretically, the progress of killing the viruses with high temperature will damage the integrity of their RNA genomes in the samples, thus improperly reducing the amount of target viral templates, and eventually leading to high false negative rate of the virus detection. In this study, we employed porcine epidemic diarrhea virus (PEDV), a kind of coronavirus, as a model virus to test the above hypothesis and our results demonstrated that the virus inactivation progress by high temperature damaged the detectable template of the coronavirus seriously. Biological. Company, Ltd.) or 1 ng λDNA (3010, Takara, Japan) were suspended and mixed together using 1.5 mL of sample preservation medium (R503, Vamzye, China) or 1.5 mL Hank's solution.\\r\\n Nucleic acid (DNA/RNA) was extracted using a commercial kit (RC311-C1, Vazyme) according to the manufacturer's protocol. Briefly, 500 µL samples preserved in R503 was first added into a 1.5 ml EP tube containing 200 µL of absolute ethanol and 200 µL samples preserved in Hank's solution was added into a 1.5 ml EP tube containing 500 µL lysis buffer.\\r\\n The mixture was vortexed thoroughly and then transferred into a column placed onto a 2 mL EP collection tube. After centrifuge for 1 min at 12,000×g, the column was placed onto a fresh 2 mL EP collection tube, followed by adding 600 µL washing buffer and then centrifuge for 30 sec at 12,000×g. Washing step was repeated once. The column was placed onto a new 2 mL collection. After centrifuge for 2 min at 12,000×g, the column was then placed onto a 1.5 ml collection tube. 100 µL and 40 µL elution buffer was added into the center of the membrane of column collected the sample prepared by R503 and Hank's solution respectively and then centrifuged for 1 min at 12,000 × g to collect the nucleic acid. The nucleic acid was stored at -30 ~ -15 o C for usage within short time and at -70 o C for long storage 2.3 Agarose gel electrophoresis 5 µL of the extracted nucleic acids, 1 µL of 10 × DNA loading buffer (P022, Vazyme) and 4 µL of nuclease-free H 2 O were mixed and then vortexed briefly. The mixture was heated in PCR instrument to denature the nucleic acid for 2 min and then subjected to 1.2% agarose gel electrophoresis separation on ice for 7-10 min together with the standard DNA marker (MD103, Vazyme).\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint\\r\\n Quantitative Real-Time PCR (qRT-PCR) was performed to detect the PEDV RNA and λ phage DNA using HiScript II One Step qRT-PCR Probe Kit (Q222-CN, Vazyme) according to the manufacturer's protocol. The reaction (25 µL) was assembled as follows: 12.5 µL of 2 × One Step U+ Mix, 1.25 µL of One Step U+ Enzyme Mix, 0.5 µL of 50 × ROX Reference Dye 1, 0.5 µL of Gene Specific Primer Forward (10 µM), 0.5 µL of Gene Specific Primer Reverse (10 µM), 0.25 µL of TaqMan Probe (10 µM), and 4.5 µL nuclease free H2O.\\r\\n The reaction was carried out on ABI StepOnePlusTM using following conditions: \\r\\n Statistical significance was determined using the paired two-tailed t test. A value of p < 0.05 was considered statistically significant, and p < 0.01 was considered statistically very significant. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint 6 / 16 into five groups. Three groups were incubated at 56 o C for 30, 45 and 60 minutes, respectively. The fourth was incubated at 92 o C for 5 minutes, and the fifth group was stored at 4 o C as no inactivation control. After the high-temperature inactivation was completed, nucleic acids (including DNA and RNA) were extracted by conventional methods, and subjected to run 1.2% agarose gel electrophoresis. \\r\\n To understand the effect of the virus inactivation progress with the temperature higher than 56 o C on the integrity of viral nucleic acid, we quantitated the amount of Table S1 ). The results suggest that only 50.11% of the detectable viral templates left after the inactivation progress of incubation at 56 o C for 30 minutes and only 3.36% of the detectable viral templates left after the inactivation progress of incubation at 92 o C for 5 minutes.\\r\\n To investigate whether any solutions are able to protect viral RNA integrity from author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint 7 / 16 disrupting during the inactivation progress with high temperature, R503 (Vazyme, China) was used to prepare the samples instead of Hank's solution. We then performed the same viral inactivation experiments as using Hank's solution.\\r\\n Compared with the control group that the sample was stored at 4 o C, both DNA and RNA in the samples incubated at 56 o C for 30 -60 minutes also displayed obvious degradation, among which the 28 S and 18 S bands of total RNA of human cells were obviously smeared, and the large bands of genomic DNA became weaker while there were almost no visible genomic DNA and 28 S RNA bands in the sample incubated at 92 o C for 5 minutes (Fig 1D) . However, the degradation of nucleic acid during the inactivation progress was not as serious as that of the samples prepared by Hank's solution. The results suggest that R503 should have a significant protective effect of both total RNA and genomic DNA from degradation.\\r\\n To determine the effect of the inactivation progress with temperature higher than 56 o C on the integrity of viral nucleic acid in the samples prepared with R503, we examined the amount of porcine PEDV coronavirus by qRT-PCR as above.\\r\\n Compared with the CT (31.64 ± 0.10) of the control group that the sample was stored at 4 o C, the mean value of CT (31.31 ± 0.17) of the group incubated at 56 o C for 30 minutes had no significant change. The difference of CTs between the two groups was less than 0.5 ( Fig. 1E ; Figure S1B ; Table S1 ). Moreover, the mean value of CT (31.43 ± 0.08) in the group incubated at 92 o C for 5 minutes was also not changed significantly. The difference of CTs between the two groups was less than 0.5 either (Fig. 1F; Fig. S1B ; Table S1 ). These results showed that the inactivation progress with the temperature higher than 56 o C did not significantly affect the detectable template of viral nucleic acid when it was prepared with R503, suggesting that the fragmented viral RNA could still be good for using as a template for qRT-PCR even if the viral nucleic acid was degraded.\\r\\n In order to understand the advantage of R503 over common isotonic solution in author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint\\r\\n protecting the detectable amount of coronavirus RNA from disrupting, we first compared the difference of the detectable amount of coronavirus RNA between the two samples that are prepared with R503 and Hank's solution and stored at 4 o C. The results showed that the detectable amount of the viral RNA in the two prepared solutions was similar (Fig. 1G, Table S1 ), and the difference between the mean values of their CT was less than 0.5 (31.64 ± 0.10 vs 31.95 ± 0.10). The results suggest that there is no significant difference in the amount of detectable viral RNA templates between the samples prepared by either R503 or Hank's solutions that were stored at Table S1 ). The results suggest that the detectable templates of viruses in the samples prepared by Hank's solution was only 32.30% of R503 solution after the samples were inactivated at 56 o C for 30 minutes to kill the virus. Furthermore, after inactivation at 92 o C for 5 minutes, the CT of detected PEDV virus in the two preservation solutions were 31.43 ± 0.08 (R503) and 36.84 ± 0.37 (Hank's), respectively. The mean value of CT of detected virus in the samples prepared by Hank's solution was 5.4125 higher than that prepared by R503 (P < 0.0001; Figure 1I , Table S1 ). The results reveal the detectable amount of virus in the samples prepared by Hank's solution was only 2.35% of R503 solution after the samples were inactivated at 92 o C for 5 minutes to kill the virus.\\r\\n In addition to the disruptive effect of the virus inactivation with high temperature on the detection of RNA virus, the detectable amount of the viral DNA (λ phage) in the samples was also significantly affected. qPCR results (Fig. S1C, Table S1) showed that mean value of CT (23.55 ± 0.20) of detectable virus in the samples author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint 9 / 16 incubated at 56 o C for 30 minutes was increased by 0.5230 (P < 0.01) compared with the CT (23.02 ± 0.15) of the samples stored at 4 o C. The mean value of CT (25.58 ± 0.21) was increased by 2.5572 (P < 0.0001) in the sample incubated at 92 o C for 5 minutes.\\r\\n Furthermore, we examined the amount of phage DNA in the sample prepared with R503 after high temperature treatment by qPCR. The results (Fig. S1D, Table S1) showed that the mean value of CT (21.71 ± 0.04) in the sample prepared by R503 and incubated at 56 o C for 30 minutes had no significant change compared with that in the control group (21.61 ± 0.06) (the difference between the two was less than 0.5). The mean value of CT (22.90 ± 0.26) was increased by 1.2880 in the sample that was inactivated at 92 o C for 5 minutes, suggesting that 92 o C treatment significantly reduced the detectable amount of DNA viruses in sample even prepared by R503.\\r\\n In conclusion, our results show that virus inactivation with temperature higher than 56 o C result in the degradation of viral nucleic acids seriously, which will lead to the artificial shortage of detectable templates of viral nucleic acids in samples, and finally lead to false negative of clinical detection in some samples. Preserved with the optimized solution such as R503, the detectable templates of viral nucleic acids can be kept unchanged after the samples were incubated at 56 o C or higher for killing the viruses although the integrity of cellular nucleic acids were obviously disrupted. The results suggest that if the samples were prepared with an optimal solution, the detectable numbers of viral templates would be unchanged because virus inactivation progress with high temperature does not erase all the RNA but leave small fragments of nucleic acids that are good enough for using as the template for qRT-PCR detection.\\r\\n If it is true for 2019-nCoV, preparing the human samples with the optimal solution can not only keep the clinical inspectors from virus infection by inactivating the virus using 56 o C or higher as normal, but also improve the detection sensitivity of viral nucleic acid in the sample, leading to avoiding or at least reducing false negative author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint 10 / 16 results of detection.\\r\\n Recently, Chen reported that the inactivation by incubating the sample with 56 o C for 30 min to kill the virus had no significant effect on the detection of 2019-nCoV by qRT-PCR [5] . However, combining the limitations such as only two samples tested and higher amount of virus in the two samples as the authors discussed in their researches, and our experimental evidences that the virus inactivation by high temperature did reduce the detectable amount of the porcine coronavirus template seriously, it is highly recommended to carry out systematic investigation on the impact of high temperature inactivation on the integrity of 2019-nCoV nucleic acids and develop a sample preservation solution to protect the detectable templates of 2019-nCoV nucleic acids from high temperature inactivation damage. Table S1 . Figure S1 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint 15 / 16 The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint. HCoV-HKU1 replication in the presence of both inhibitors (Fig. 2B) . All inhibitors were used 151 at non-toxic concentrations (Fig. 2C) . 152 The experiments conducted so far suggested that KLK13 is required for virus infection. 153 However, one may question the specificity of the KLK13 protease inhibitors. To ensure that 154 KLK13 is indeed the priming enzyme during HCoV-HKU1 infection, we developed HAE 155 cultures by transforming cells with lentiviral vectors encoding shRNAs targeting KLK13 156 mRNA. We then confirmed that the expression of the protease was silenced (HAE_shKLK13). (Fig. 3A) . Importantly, HAE_shKLK13 cells continued to differentiate and formed 162 pseudostratified cultures (Fig. 3B) . Next, we infected HAE_ctrl, HAE_GFP, HAE_vector, and 163 HAE_shKLK13 with HCoV-HKU1 (10 6 RNA copies per ml) and incubated them for 2 h at 164 32°C with the viral stock solution. Cultures were maintained at 32°C for 5 days at an air-liquid 165 interface. Apical washes were collected, and virus yield was determined by RT-qPCR. We 166 found that, in contrast to that in control cultures, replication of virus in HAE_shKLK13 was 167 abolished (Fig. 3C) . Overall, these data suggest that silencing the KLK13 gene in HAE inhibits 168 virus infection, indicating that KLK13 is necessary for HCoV-HKU1 infection. 171 We determined that KLK13 is essential for efficient HCoV-HKU1 infection in HAE The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint proteases my render RD cells permissive, we generated RD cells expressing human KLK13 or 176 TMPRSS2 proteases. RD cells were transduced with lentiviral vectors harboring the KLK13 177 gene (RD_KLK13), control vector (RD_ctrl), or TMPRSS2 (RD_TMPRSS2). Due to the lack 178 of KLK13 specific antibodies, we verified its presence based on RT-PCR (Fig. 4A) . The 179 presence of TMPRSS2 in RD_TMPRSS2 cells was confirmed using Western blot. The 180 TMPRSS2 band in RD cells was observed at 25 kDa, which corresponds to one of the naturally 181 occurring splicing variants (Fig. 4B) . Subsequently, we transduced RD_ctrl, RD_KLK13 and pseudoviruses, compared to ΔEnv, which was completely abolished in the presence of KLK13 193 inhibitor (Fig 4D) . Overall, these data demonstrated that KLK13 activity drives HCoV-HKU1 194 entry into cells. 195 196 KLK13 enables the replication of HCoV-HKU1 in RD cells 197 Obtained results showed that KLK13 expression on RD cells was sufficient for HCoV-198 HKU1 pseudovirus entry. Here, we aimed to test whether KLK13 presence renders RD cells 199 permissive for HCoV-HKU1 infection. For this, we infected RD_ctrl and RD_KLK13 cells 200 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint with HCoV-HKU1 (10 8 RNA copies per ml) and incubated the culture for 7 days at 32°C in the 201 presence or absence of a KLK13 inhibitor (10 µM) or DMSO. Next, cellular RNA was isolated 202 and the presence of HCoV-HKU1 N subgenomic mRNA (N sg mRNA), which is considered 203 to be a hallmark of coronaviral infection, was assessed (14). sg mRNA appeared in RD_KLK13 204 cells, while no signal was detected in cultures supplemented with the KLK13 inhibitor nor in 205 RD_ctrl cells (Fig. 5A) . 206 To further confirm the role of KLK13 during HCoV-HKU1 infection RD HCoV-HKU1 infection was analyzed by means of N sg mRNA detection. Again, we found that 212 N sg mRNA was produced only in the presence of KLK13 (Fig. 5B) . Further, we passaged in the presence of KLK13 (Fig. 5B) . However, we observed no cytopathic effects (CPEs), and 218 replication levels were very low (no significant increase over control levels on RT-qPCR; data 219 not shown). To further test the effect of KLK13 on replication of HCoV-HKU1 in RD cells, the 220 virus stock was incubated with purified KLK13 (200 nM) and incubated in the presence or 221 absence of a KLK13 inhibitor (10 µM) or DMSO. After 2 h at 32°C pre-treated virus stock was 222 diluted in media as described above and overlaid on RD cells. After 7 days at 32°C we evaluated 223 the presence of the HCoV-HKU1 N sg mRNA. The virus replicated only after treatment with 224 KLK13, and supplementation with the inhibitor blocked this effect infection (Fig. 5C) . 225 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint KLK13 primes the HCoV-HKU1 S protein 227 Expression of KLK13 by cells previously resistant to HCoV-HKU1 renders them 228 susceptible; therefore, we asked whether this is due to proteolytic activation of the S protein. 229 We tested this using the CleavEx method, in which a peptide of interest is exposed in the KLK13. Subsequently, proteins were resolved by SDS-PAGE and detected by western blotting 238 with antibodies specific for His-tagged proteins. The analysis showed that, in the presence of 239 500 nM KLK13, the CleavEx protein harboring the S1/S2 cleavage site was degraded; however, 240 the CleavEx protein harboring the S2/S2' site remained intact. Because KLKs are produced as 241 pro-forms that undergo self-activation, an additional band of His-tagged purified pro-KLK13 242 (HisTag-pro-KLK13) was observed after treatment with 500 nM KLK13 (Fig. 6A) . The product 243 of the S1/S2 cleavage was further sequenced by N-terminal Edman degradation showing the 244 following sequence: R↓SISA, which corresponds to the S1/S2 site. This result shows clearly 245 that the S1/S2 region of the HCoV-HKU1 S protein is prone to KLK13-mediated cleavage. 246 Furthermore, we aimed to confirm the cleavage using a full-length Spike protein of HCoV-HKU1 (HKU1-S). For this, we expressed the HKU1-S in 293T cells, purified the protein 248 using 6 × His tag, and incubated for 3 h at 37°C with increasing concentrations of purified 249 KLK13. Subsequently, HKU-S or mock proteins were resolved by SDS-PAGE and detected by 250 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint western blotting with antibodies specific to the tag. The analysis showed that in the presence of 251 1 µM KLK13 the HKU1-S was degraded ( Figure 6B ). The S protein was observed at 252 ~150 kDa, which is consistent with the migration speed reported for these highly glycosylated 253 proteins (1). 254 255 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint\\r\\n Receptor recognition is the first, essential step of the virus infection process. The The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint concentrations(63). Broad spectrum protease inhibitors are now used widely for virus research, 281 although their non-specific activity makes the results equivocal. For example, Matsuyama et al. 282 showed recently that the furin inhibitor dec-RVKR-CMK interferes with the activity of several 283 proteases, and that its previously described inhibitory activity during MERS-CoV infection is 284 not specific to furin; instead, its activity is due to non-specific inhibition of cathepsin L and 285 TMPRSS2 (81). We tried to use specific KLK inhibitors developed in our laboratory (61).\\r\\n Considering the small arsenal of tools available to researchers studying KLKs, only three 287 compounds were readily available. Treating HAE cultures with these inhibitors revealed that 288 only compounds designed to inhibit KLK13 hampered HCoV-HKU1 replication. However, the 289 great similarity between different KLKs makes one doubt the specificity of these inhibitors, 290 despite their performance in biochemical assays. Therefore, we decided to silence KLK13 in 291 HAE cultures. This abolished virus replication ex vivo, thereby confirming the importance of 292 KLK13 during infection. KLK13 is thought to be secreted and membrane-bound(82, 83). 293 This study showed that HCoV-HKU1 infection in HAE modulates the expression of 294 different KLKs, including KLK13. In our study, KLKs expression was tested using semi-295 quantitative PCR, and for that reason, we were unable to show the level of KLKs modulation 296 after HCoV-HKU1. However, the pattern of virus-induced expression of several KLKs could 297 be observed. KLK mechanism of activation is a complex process and until now it has only 298 been proven that most KLK genes are regulated by steroids and other hormones (84). It is also 299 important to remember, that KLK expression is regulated in a similar manner, and the induction 300 of a single gene usually results in overexpression of the whole cluster (85, 86). While one may 301 assume that the virus stimulates KLK13 production to promote the infection, this up-regulation 302 is likely a natural response of the damaged tissue, as KLKs were previously reported to take 303 part also in tissue regeneration (87-89). Further, increased expression of KLKs may be the 304 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint response to the inflammatory process, as Seliga et al demonstrated that KLK-kinin system is a 305 potent modulator of innate immune responses (90).\\r\\n The experiments performed herein show the importance of KLK13 for virus entry into 307 susceptible cells; therefore, we speculated that scattered distribution of different KLKs in 308 different tissues may be one of the determinants of the HCoV-HKU1 tropism (53, 91). We 309 tested the purified enzyme expressed in the eukaryotic cells; however, we also developed a cell 310 line constitutively expressing the enzyme. As an in vitro model for our studies, we used RD 311 cells previously reported to carry attachment receptors for the virus (26). Here, using 312 pseudoviruses decorated with S-HKU1 proteins we showed that KLK13 presence on RD cells 313 is sufficient for virus entry and renders these cells permissive. Our experiments also showed 314 that in contrast to previous reports, TMPRSS2 is not involved in this process(51). Furthermore, 315 we observed that RD cells supported the replication of the virus in the presence of KLK13 and 316 that this effect was reversed in the presence of the specific KLK13 inhibitor. We were, however, 317 not able to culture the virus to high yields. HCoV-HKU1 replication in KLK13-expressing RD 318 cells remained inefficient and RTqPCR assessment did not reveal significant increases in the 319 amounts of viral RNA. For that reason, we are only able to detect viral sg mRNAs, which are 320 considered to be the hallmark of coronaviral replication.We believe that this may be due to 321 non-optimal infection conditions, which may include inappropriate KLK13 concentrations or 322 low density of the entry receptor. Also, it is possible that RD cells may not support efficient 323 replication of the virus due to factors unrelated to the entry process. Nonetheless, our results\\r\\n show that the HCoV-HKU1 entry receptor is present on RD cells, and we were able to trigger 325 virus entry and replication; these findings warrant further exploration.\\r\\n Most coronaviral S proteins are processed into S1 and S2 subunits by host proteases, The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint HCoV-HKU1 S protein has two regions that are prone to proteolytic activation: the S1/S2 furin 330 cleavage site and a secondary cleavage site termed S2', which is adjacent to a potential fusion 331 peptide (38). While the S1/S2 site is believed to be processed by furin during protein 332 biosynthesis, the S2/S2' site is expected to be cleaved during virus entry. As we already knew being tested. We found that the S1/S2 site was efficiently cleaved by KLK13, whereas the 338 S2/S2' region remained intact. As CleavEx technique is a convenient surrogate system allowing 339 for precise mapping of the cleavage site, it has some limitations. To ensure the reliability of 340 results, purified full-length HCoV-HKU1 S protein was subjected to the proteolytic cleavage.\\r\\n Also here we observed efficient cleavage of the HCoV-HKU1 S protein by KLK13.\\r\\n While the results presented here show that KLK13 is able to process the HCoV-HKU1 The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint to cross the species barrier and cause severe diseases in humans, further research on the role of 355 different proteases in coronaviral infections is necessary. 356 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint\\r\\n Plasmid constructs 358 KLK13 and TMPRSS2 genes were amplified by PCR using cDNA obtained from HAE 359 cells. Each PCR product was cloned into pWPI plasmid for lentivirus production and sequence Table 2 . (Table 3 ; 500 nM each). β-actin was used as a household gene reference.\\r\\n The reaction was carried out according to the scheme: 5 min at 95°C, followed by 35 cycles of 465 30 s at 95°C, 20 s at 59°C and 20 s at 72°C, followed by 10 min at 72°C. containing proKLK13 or proKLK14 were concentrated and the buffer was changed to 50mM 528 Tris pH 7.5, 150 mM NaCl. After purification and self-activation at 37°C for 24 h, activity of 529 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint proteases was assessed by serine protease inhibitor Kazal-type 6 (SPINK6) titration, as 530 described previously (63). Protein expression was performed in E. coli BL21 and was induced by the addition of 553 0.5 mM IPTG to the bacterial culture (OD600 0.5-0.6), followed by shaking for 3h at 37°C. 554 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint Then, the bacteria were spun down and the pellet was suspended in buffer A (10 mM sodium 555 phosphate, 500 mM NaCl and 5 mM imidazole, pH 7.4). The pellet suspension was then 556 sonicated and spun down. Soluble proteins were purified using HisTrap TM Excel (GE The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint. 4 hepatoma cells was largely specific to CoV. Among 10 genetically divergent viruses tested, only hepatitis C virus was also restricted (Extended Data Fig. 1f ).\\r\\n LY6E is a member of the LY6/uPAR family of GPI-anchored proteins, and is implicated in diverse cellular processes 14 . To evaluate the specificity of LY6E for inhibiting CoV, we ectopically expressed select LY6/uPAR proteins 9 and infected with HCoV-229E. Only LY6E inhibited CoV infection (Extended Data Fig. 1g) . We previously showed that a conserved residue (L36) is required for LY6Emediated enhancement of influenza A virus 9 . L36 was also required to restrict HCoV-229E infection (Extended Data Fig. 1h) . These data suggest a conserved functional domain that is responsible for both the virus-enhancing and -restricting phenotypes.\\r\\n Next, we investigated the effect of LY6E on specific steps of the virus replication cycle. First, we tested whether LY6E restricts HCoV-229E attachment to the cell surface and observed no effect (Extended Data Fig. 2a) . In accordance, LY6E had no effect on surface expression of CD13, the specific receptor for HCoV-229E (Extended Data Fig. 2b) . To test whether LY6E restricts viral entry, we used a vesicular stomatitis virus (VSV) pseudoparticle (pp) system bearing CoV spike (S) proteins.\\r\\n Strikingly, LY6E significantly inhibited VSVpp entry mediated by S proteins from HCoV-229E (Fig.   2a) , MERS-CoV (Fig. 2b) , SARS-CoV (Fig. 2c) , and SARS-CoV-2 (Fig. 2d) . Entry mediated by VSV glycoprotein G was only marginally impacted by LY6E (Extended Data Fig. 2c, d) . To test whether LY6E interferes with the fusion of viral and cellular membranes, we performed a syncytia formation assay using propagation competent VSV pseudoviruses co-expressing CoV S protein and a GFP reporter (VSV*ΔG(CoV S)) (Extended Data Fig. 3a) . LY6E potently inhibited CoV S protein-mediated syncytia formation (Fig. 2e-f ). To determine whether LY6E impairs S protein expression or maturation, we also performed a heterologous syncytia formation assay. CoV-resistant hamster cells infected with trans-complemented GFP reporter VSV*ΔG(CoV) were co-cultured with susceptible target cells that ectopically expressed LY6E. LY6E again significantly blocked syncytia formation, demonstrating that LY6E inhibits S protein-mediated fusion and not S protein expression or maturation (Extended Data Fig. 3b-c) . As membrane fusion can occur without syncytia formation, we also performed a quantitative fusion assay in which mixing of cell contents results in complementation of split luciferase (Fig. 2g) .\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint Cells co-transfected with plasmids encoding CoV S proteins and split luciferase 1-7 were mixed with susceptible target cells co-expressing LY6E or vector control and split luciferase 8-11. LY6E reduced CoV S-mediated cell-cell fusion, indicating that LY6E blocks fusion of viral and cellular membranes ( Fig. 2h) . Time-course experiments revealed that LY6E had no effect on HCoV-229E translation, replication, assembly, or release (Extended Data Fig. 2e-i) . Collectively, our results demonstrate that LY6E specifically inhibits CoV S protein-mediated membrane fusion, which is required for viral entry.\\r\\n We next addressed whether LY6E modulates proteolytic activation of the S protein. Upon S proteinmediated binding to the respective receptor, host proteases cleave the S protein and make it fusion competent for \\\"early entry\\\" at the cell surface or \\\"late entry\\\" in endosomes 15, 16 . Pharmacological inhibition of cell surface and endosomal proteases had no effect on LY6E-mediated restriction of CoV infection (Extended Data Fig. 4a-c) . Next, we addressed whether LY6E directly interferes with the proteolytic activation of S protein. Mutation of S1/S2 and S2′ in the S protein prevents activation (Extended Data Fig. 4d ) 16, 17 . Ectopic LY6E expression had no effect on MERS-CoV S cleavage or infection by cleavage-resistant MERS-CoV pp (Extended Data Fig. 4e-g) . Collectively, our data indicate that restriction of viral fusion is independent of S protein activation.\\r\\n Next, we aimed to evaluate LY6E-mediated CoV restriction in vivo. Systemic genetic ablation of Ly6e in mice is embryonic lethal 18 . Therefore, we generated transgenic Ly6e fl/fl mice, which are functionally 'wild-type (WT)' for Ly6e expression, and crossed them with Vav1-iCre mice to ablate Ly6e in cells that originate from a hematopoietic stem cell (HSC) progenitor (Extended Data Fig. 5a-b ). This genetic model was chosen since immune cells are critical for protection from CoV in vivo 5,19-23 . Bonemarrow derived macrophages (BMDM) from Ly6e ΔHSC mice had reduced Ly6e mRNA levels (Extended Data Fig. 5c ). The natural mouse pathogen MHV is a well-studied model CoV that causes hepatitis and encephalomyelitis in mice 24 . Ly6e ΔHSC BMDM were more susceptible to MHV infection than Ly6e fl/fl BMDM, demonstrating that murine Ly6e is a restriction factor for murine CoV (Extended Data Fig.   5d ).\\r\\n To determine whether Ly6e is important for controlling CoV infection in vivo, Ly6e fl/fl and Ly6e ΔHSC mice were infected with a high dose (5 x 10 4 PFU) of MHV that causes sub-lethal hepatitis in WT author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint C57BL/6J mice 25 . Since MHV pathogenesis is influenced by sex 26 , we performed independent experiments in male and female mice. Ly6e ΔHSC mice of both sexes rapidly succumbed to MHV infection by day 6 ( Fig. 3a-b, Extended Data Fig. 6a-b) . To examine viral pathogenesis and immunopathology in infected mice, Ly6e fl/fl and Ly6e ΔHSC mice were infected with the same high dose of MHV and euthanized 3-or 5-days post-infection. At both time points, female Ly6e ΔHSC mice exhibited significantly higher levels of liver damage as measured by serum ALT while no difference was observed in male mice (Fig. 3c,i, Extended Data Fig. 6c,i) . However, hepatic viral burden did not differ in either sex ( Fig. 3d,j, Extended Data Fig. 6d,j) , possibly due to tissue saturation at this high dose. Indeed, female but not male Ly6e ΔHSC mice exhibited increased liver damage and viral burden at a low dose (5 PFU) of MHV (Extended Data Fig. 7a -b, f-g). Spleen viral burden was elevated in female and male Ly6e ΔHSC mice at both time points and doses of MHV (Fig. 3e,k, Extended Data Fig. 6e ,k, Extended Data Fig.   7c ,h). At 3 days post-infection, there was no difference in liver necrosis, but inflammation was moderately reduced in female but not male Ly6e ΔHSC mice ( Fig. 3f-h, Extended Data Fig. 6f-h) . By 5 days post-infection and at both low and high doses of MHV, female Ly6e ΔHSC mice had significantly higher levels of liver necrosis and a reduced presence of mononuclear immune cells ( Fig. 3l- Fig. 7k-n) . These data demonstrate that Ly6e in hematopoietic cells is important for controlling murine CoV infection.\\r\\n We next used a transcriptomic approach to evaluate the effect of Ly6e ablation on global gene expression in liver and spleen of female Ly6e fl/fl and Ly6e ΔHSC mice injected with PBS or the high dose of MHV at 3-and 5-days post-infection. In infected tissues, loss of Ly6e in hematopoietic cells correlated with differential gene expression in pathways associated with tissue damage, such as liverspecific metabolic genes, angiogenesis, wound healing, and immune response to viruses (Fig. 4a-c) . We also observed a striking loss of genes associated with the type I IFN response, inflammation, antigen presentation, and B cells in infected Ly6e ΔHSC mice ( Fig. 4d-e, Extended Data Fig. 8) . 7 The histopathology (Fig. 3h,n) and transcriptome data (Fig 4a-e, Extended Data Fig. 8 Fig. 7k-n, Extended Data Fig. 9 ). Multiple cell types were depleted in livers and spleens of infected Ly6e ΔHSC mice, irrespective of sex (Figure 4f g, Extended Data Fig. 10a-b) . B cells were dramatically reduced in livers from infected Ly6e ΔHSC mice.\\r\\n Hepatic CD4 + T cells, NK cells, dendritic cells (DC), macrophages, and neutrophils were also depleted in infected Ly6e ΔHSC mice, whereas CD8 + T cells were unchanged (Fig. 4f, Extended Data Fig. 10a ).\\r\\n Infected Ly6e ΔHSC mice had a reduction in all splenic immune cell subsets except for CD4 + and CD8 + T cells (Fig. 4g, Extended Data Fig. 10b) . Depletion of immune cell populations in Ly6e ΔHSC organs was MHV-dependent, as cell numbers were not altered in PBS-injected mice.\\r\\n To determine whether loss of immune cells in MHV-infected Ly6e ΔHSC mice correlates with increased permissiveness to infection, we assessed MHV infection in splenocytes cultured from Ly6e ΔHSC , Ly6e fl/fl , and Ifnar -/mice. In Ly6e fl/fl splenocyte cultures, MHV infected macrophages, neutrophils, DC, and B cells, but not CD4 + or CD8 + T cells, as previously published (Fig. 4h, Extended Data Fig. 10c Together, our data demonstrate an important role for Ly6e in immune cell-mediated control of CoV infection. Higher viral burden and lower liver inflammation suggests that liver damage in MHV-infected female Ly6e ΔHSC mice may be due to viral pathogenesis that exceeds the ability of the immune system to control the infection. Notably, we observed MHV-induced loss of three classes of cells: 1) cells that are not permissive to the virus (CD4 + T cells), 2) cells that are permissive to MHV but do not demonstrate Ly6e KO-dependent infection (NK cells, macrophages, and neutrophils), and 3) cells that are permissive to MHV in a Ly6e KO-dependent manner (B cells and DC). This latter group is capable of antigen-presentation to CD4 + T cells 27,28 . Thus, in the absence of Ly6e, unrestricted MHV infection author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint of B cells or DC may impair T cell priming, thereby negatively impacting recruitment and activation of NK cells, macrophages, and neutrophils that contribute to control of viral disease.\\r\\n In conclusion, we identified LY6E as a pan CoV restriction factor that limits CoV entry and protects the host from severe viral disease. Our findings are striking given that LY6E has been primarily associated with an enhancing phenotype in which LY6E promotes entry of multiple viruses 29 . However, our data clearly expand the role of LY6E during coronaviral infection and establishes a novel function in protecting the host immune cell compartment. Determining the precise molecular mechanism underlying how LY6E inhibits CoV S protein-mediated membrane fusion will advance our understanding of cellular antiviral defenses against these important human pathogens. Antiviral membrane fusion inhibitors have been successfully implemented for treatment of HIV-1 infection 30 . A therapeutic approach mimicking the mechanism of action of LY6E could provide a first line of defense against novel emerging CoV infections. Furthermore, delineating which Ly6e-expressing immune cells protect mice from MHV will provide new insight into how individual antiviral effectors in distinct cellular compartments modulate viral pathogenesis. author/funder. All rights reserved. No reuse allowed without permission. 34-41 (1996) . Welch's correction (c-d, f-g), one-tailed Mann-Whitney U test (e), ratio paired student's t-test (g), 2way\\r\\n ANOVA followed by Sidak's or Dunnett's multiple comparison test (j,l). Error bars: SD. P values: c, ** p=0.0088; d, *** p=0.0001; e, * p=0.0143; f, * p=0.0286; g, * p=0.0101; h, *** p=0.0010; j, ** p=0.0073, p=0.0033; l, ns=0.0740, ** p=0.0013, *** p=0.0001. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint p=<0.0001; c, **p=0.0066; d, **** p=<0.0001; f, **** p=<0.0001; h, ns p=0.9975, ** p=0.0054, ns p=0.1520, 0.9892. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint scale bars are 500 µM (4x) and 100 µM (20x). Statistical significance was determined by Mantel-Cox test (a), two-tailed unpaired student's t-test with Welch's correction (c-e, i-k), two-tailed Mann-Whitney U test (f-g, l-m). Error bars: SEM (b), SD (c-g, i-m). P values: a, *** p=0.0002; c, ** p=0.0031; d, ns p=0.2136; e, **** p=5.759 x 10 -7 ; f, ns p=0.3698; g, ** p=0.0054; i, * p=0.0430; j, ns p=0.1321; k, ** p=0.0094; l, ** p=0.0072; m, **** p=8.741 x 10 -5 . author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint maps displaying significant changes (mean RPKM > 0.5; fold change > 2; FDR ≤ 0.05) in liver (a) and spleen (b). Dendrograms are normalized read data (row z-score) clustered with complete linkage method employing Spearman Rank correlation distance measurement. c, Pathway analysis of KO versus WT.\\r\\n Up-(red) or down-regulated (blue) pathways indicated by activation z-score. Numbers show significantly dysregulated genes as percentage of total gene number included in pathway. d-e, Expression of select genes in liver (d) and spleen (e). f-g, Immune cell counts from liver (f) and spleen \\r\\n HCoV-229E: 1.5 x 10 5 stable LY6E expressing or control cells were seeded in a 12-well plate and infected with HCoV-229E in a serial dilution. Cells were incubated for 2 hours at 33°C, then viral supernatant was removed, and an overlay of 1:1 Avicel (2.4%) (Avicel RC-581NF) and 2x DMEM, author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint supplemented with 10% FBS and 1x penicillin/streptomycin (p-s), was added. Cells were incubated for 3 days at 33°C before medium was removed, cells were washed, and stained with crystal violet (Sigma-Aldrich). Plaque forming units (PFU) were calculated.\\r\\n HCoV-OC43: 5.0 x 10 4 stable LY6E expressing or control cells were seeded in a 24-well plate and infected at 33°C for 1 hour with HCoV-OC43 (MOI=1). Virus was aspirated and 10% FBS/1x NEAA/1x p-s/RPMI (cRPMI) was added back to cells. 24 hours post-infection, cells were dissociated using Accumax (Sigma-Aldrich), fixed in 1% PFA, permeabilized per manufacturer's protocol (BD Cytofix/Cytoperm), and stained for nucleoprotein (1:500) and a goat anti-mouse AlexaFluor488- The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint anti-mouse secondary antibody. Images were acquired with a fluorescence microscope and analyzed using ImageXpress Micro XLS (Molecular Devices, Sunnyvale, CA).\\r\\n For infection assays, 1 x 10 5 target cells were seeded in a 24-well plate one day prior infection. Cells were infected with HCoV-229E-Rluc (MOI 0.1) in OptiMEM (Gibco) for 2 hours at 33°C. Cells were washed 1x with PBS and 10% FBS/1x NEAA/1x p-s/DMEM (cDMEM) was added back. Cells were incubated at 33°C for 24 hours, then washed with PBS and lysed using the Renilla Luciferase Assay System kit (Promega). Rluc activity was measured using a plate luminometer (EnSpire 2300 Multilabel reader by Perkin Elmer). For the reconstitution with CRISPR resistant LY6E (CR LY6E), 2.5 x 10 4 cells were seeded in a 24-well plate. One day post-seeding, cells were either left untransduced or transduced with a lentiviral vector encoding for CR LY6E or the empty control. 48 hours post-transduction, cell lysates were either infected with HCoV-229E-Rluc (MOI=0.1) for 24 hours as described above or harvested for Western blot.\\r\\n Pseudotyping of VSV*ΔG(Fluc) was performed as previously described 50 with two silent mutation (T2568G, T3327C)), have been described previously [51] [52] [53] . The expression plasmid for SARS-CoV-2 was generated as follows: the coding sequence of a synthetic, codonoptimized (for human cells) SARS-CoV-2 DNA (GeneArt Gene Synthesis, Thermo Fisher Scientific) author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint based on the publicly available protein sequence in the National Center for Biotechnology Information database (NCBI Reference Sequence: YP_009724390.1) was PCR-amplified and cloned into the pCG1 expression vector between BamHI and XbaI restriction sites. The MERS-CoV S cleavage mutants have been described before 17 \\r\\n The split luciferase fusion assay has been described before, with slight modifications 54 . Briefly, 6 x 10 5 293LTV cells were transfected with plasmids encoding CoV S proteins (pCAGGS-HCoV-229E, pCAGGS-MERS-CoV) together with a split-luciferase construct (Rluc8155-156 DSP1-7). The empty expression plasmid (pCAGGS-mCherry) as well as a construct encoding for VSV G served as negative and positive control, respectively. 2 x 10 5 stable LY6E expressing or empty control Huh7 cells were transfected using Lipofectamine 2000 (Thermo Fisher Scientific) with a plasmid encoding the second half of the split-luciferase protein (Rluc8155-156 DSP8-11). Approximately 30 hours post-transfection, both cell populations were dissociated (TryPLE Express, Gibco), counted, and equal cell numbers (2x10 4 cells each) were co-cultured for 16 -20 hours at 37°C. Supernatant was harvested and cells lysed using ice-cold passive lysis buffer (Promega). Cells were immediately transferred to -80°C for at least author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint 1 hour, before Rluc activity was determined using the Renilla Luciferase Assay System (Promega). Cells were kept on ice during all times post-cell lysis to eliminate DSP post-lysis complementation.\\r\\n Following extraction of total RNA from MERS-CoV (strain EMC) infected VeroE6 cells, the cDNA encoding the MERS-CoV S protein was generated by reverse transcription (RevertAid Premium Reverse Transkriptase, ThermoScientific). Three overlapping cDNA fragments were amplified by PCR (Phusion Hot Start II High Fidelity Polymerase, Thermo Scientific) and subsequently assembled by overlapping PCR. The cDNAs encoding the S proteins of either MERS-CoV (strain EMC) or HCoV-229E (truncated variant lacking the 10 amino acids at the C terminus, original plasmid pCAGGS-229E S) were amplified by PCR and inserted into the pVSV* plasmid 55 between MluI and BstEII restriction sites. The resulting plasmids were used to generate the recombinant viruses VSV*ΔG(MERS S) and VSV*ΔG(229E S) according to a published procedure 56 . All viruses were propagated on BHK-G43 cells 57 , resulting in viruses predominantly containing the homotypic VSV glycoprotein G in the envelope.\\r\\n Cells expressing LY6E or empty control were infected with VSV*ΔG(MERS S) or VSV*ΔG(229E S) at a MOI of 0.01. At 20 hours post-infection, the cells were fixed with 3% PFA/PBS and the nuclei stained with 4',6-diamidine-2'-phenylindole (DAPI, Sigma). An inverted fluorescence microscope (Zeiss) was used to determine area percentage area covered by syncytia.\\r\\n For heterologous cell-cell fusion assay, BHK-21 cells were infected with VSV*ΔG(CoV S) at a MOI of 1. After 2 hours, the cells were treated with trypsin/EDTA (Life Technologies, Zug, Switzerland) and resuspended in DMEM supplemented with 5% FBS (2x10 4 cells/mL). The infected BHK-21 cell suspension (500 µL) was seeded into 24-well cell culture plates along with LY6E-or empty controlexpressing Huh7 (1 x 10 4 cells). Cell co-cultures were incubated for 7 hours at 37°C, fixed with 3.6% author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint formaldehyde diluted in PBS, (Grogg-Chemie AG, Stettlen, Switzerland) and stained with DAPI. The percentage area covered by syncytia was calculated as above.\\r\\n Ly6e tm1a ES cells were obtained from the EUCOMM consortium 58 \\r\\n Six to twelve-week-old female and male mice were injected intraperitoneally with MHV-A59 diluted in PBS to the indicated titers or PBS for a mock infection control. All infected mice were monitored daily for weight and mortality. Animals that lost more than 20% of their original body weight were euthanized per IACUC guidelines.\\r\\n Viral titers in liver and spleen were determined from frozen organs after weighing, homogenization, and plaque assay on L929 cells. Alanine aminotransferase (ALT) was measured in fresh, unfrozen serum using VITROS MicroSlide Technology by the UTSW Mouse Metabolic Core. Livers were fixed in 10% author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint neutral buffer formalin, embedded in paraffin, sectioned at 5 µM, and stained with hematoxylin and eosin (H&E). Slides were analyzed by an independent pathologist (UTSW Animal Resource Center) who was blinded to experimental conditions. A numerical score was assigned for degree of inflammation and degree of necrosis for each liver. Inflammatory cell infiltration was scored using the following criteria: 0 = none, 1 = minimal, 2 = mild, 3 = moderate, and 4 = marked. The necrosis score was defined by the percent necrosis. 0 = none, 1 = <2%, 2 = 2-20%, 3 = 21-40%, and 4 = >40%.\\r\\n All primary cells were maintained in 10% FBS/10 mM HEPES/1 mM sodium pyruvate/2 mM Lglutamine/1x p-s/RPMI (cRPMI2) unless otherwise indicated.\\r\\n Primary bone marrow derived macrophages (BMDM) were prepared as described previously 60 . Viable cells were quantified using trypan blue exclusion. BMDM were plated at 2.5 x 10 4 cells per well in non- The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint separated from IHIC by centrifugation at 850 x g for 30 minutes, 22°C, with no brake. Red blood cells were removed from the resulting pellet with RBC lysis buffer (Tonbo Biosciences). Viable cells were quantified using trypan blue exclusion. IHIC and splenocytes (4 x 10 5 ) from infected or mock-infected mice were stained with a fluorescent viability dye (Ghostdye Violet 450, Tonbo Biosciences), treated with anti-CD16/CD32 (Tonbo Biosciences), stained with antibodies that recognize surface lineage markers, and then fixed in 1% PFA/PBS. Fixed volumes of cell suspensions were analyzed by flow cytometry the next day. Absolute cell counts were determined by multiplying final number of lineage marker-positive cells by dilution factor used to normalize cell counts for antibody staining.\\r\\n All primary cell samples were resuspended in 3% FBS/PBS and analyzed using a S1000 Flow Cytometer with a A600 96-well plate high throughput extension and compensated using CellCapture software (Stratedigm). Data was analyzed with FlowJo Software (Treestar). The flow gating strategy for liver and spleen immune cells is included in Extended Data Figure 9 .\\r\\n Differences in data were tested for significance using GraphPad Prism v8.3.1 for Windows (GraphPad).\\r\\n For details regarding the statistical tests applied, please refer to the figure legends. P values < 0.05 were considered significant.\\r\\n The authors declare that the data supporting the findings of this study are available within the article and its Supplementary Information files or are available on request. The RNAseq data discussed in this publication have been deposited in the Gene Expression Omnibus (GEO) database, https://www.ncbi.nlm.nih.gov/geo (GSE146074).\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint\\r\\n We would like to thank the following people for their generous contribution of reagents and infrastructure which greatly enhanced this study, especially the sequencing facility in Bern, the diagnostic facility of IVI, and the Animal Resource Center at UTSW. Furthermore, we are grateful to \\r\\n The authors declare no competing interests.\\r\\n Supplementary Information is available for this paper.\\r\\n Correspondence and requests for materials should be addressed to Volker Thiel, John W. Schoggins, and Charles M. Rice.\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint was harvested and extracellular viral RNA was extracted and viral replication detected via qRT-PCR (f). Cell lysates were harvested and intracellular Renilla luciferase activity was detected upon cell lysis (g). Cells were subjected to 3 rounds of freeze/thaw cycles. Cell debris was removed and the supernatant titrated on naïve Huh7 cells. Intracellular infectivity was determined (h). Supernatant was harvested and The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint ANOVA followed by Sidak's multiple comparisons test (c). Error bars: SD. P values: c, **** p=<0.0001 <0.0001. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint two pooled experiments. For i-m, n=8 (Ly6e fl/fl ,), n=6 (Ly6e ΔHSC ), or n=3 (PBS) from two pooled experiments. In h,n, scale bars are 500 µM (4x) and 100 µM (20x). Significance for was tested by\\r\\n Mann-Whitney U test (f-g, l-m). Error bars: SEM (b), SD (c-g, i-m). P value: a, **** p=3.946 x 10 -6 ; c, ns p=1144; d, ns p=0.1407; e, ** p=0.0026; f, ns p=0.1708; g, ns p=0.0779; i, ns p=0.6191; j, ns p=0.6842; k, * p=0.0119; l, ns p>0.9999; m, ns p=0.0779. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint -c, f-h) , two-tailed Mann-Whitney U test (d-e, i-j). Error bars: SD. P value: a, * p=0.0234; b, * p=0.0428; c, *** p=0.0004; d, * p=0.0210; e, ** p=0.0054; f, ns p=0.1544; g, ns p=0.5322; h, ns p=0.0662; i, ns p=0.0592; j, ns p=0.9064. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint DDX3X -DDX3, VIPERIN -RSAD2, SERPINE1 -PAI 1, IL28RA -CRF2, VEGFC -VRP, SP110 -IFI41, IRF2 -PRIC285, ZNF313 -RNF114, DNAPTP6 -LOC26010, LOC400759 -PSEUDOGENE,   PSCD1 -CYTH1, IFITM1 -IFI17, SIGLEC1 -FRAG, CCL5 -SYCA5, CXCL9 -MIG, IFIH1 Huh7 cells expressing TMPRSS2 were generated as follows. In order to generate TMPRSS2-encoding retroviral vectors, the open reading frame of human TMPRRS2 was first PCR amplified with primers adding an N-terminal cMYC epitope to the TMPRSS2 coding sequence. The resulting sequence was inserted into a modified version of the pQCXIP plasmid that contains a blasticidin resistance cassette instead of the usual puromycin resistance cassette 17 . Huh7 cells stably expressing human TMPRSS2 were generated by retroviral transduction and selection with the antibiotic blasticidin (50 µg/ml).\\r\\n Following selection, cells were maintained in culture medium (cDMEM) supplemented with 10 µg/ml blasticidin. To generate STAT1 -/-_CEACAM1 cells, human STAT1 −/− fibroblasts were transduced with lentiviruses encoding for the murine CoV receptor CEACAM1 (kind gift from David Wentworth, CDC, Atlanta, USA 59 and subsequently selected with 1 µg/mL puromycin. Stable LY6E expressing cell lines were generated upon lentiviral transduction with SCRPSY LY6E or as control SCRPSY Empty in DMEM containing 4 µg/ml polybrene (Millipore) and 20 mM HEPES buffer solution (Gibco). Cells author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint were selected using 2-4 µg/mL puromycin and were passaged until all cells in control wells without lentivirus were killed. The puromycin selected cells were further passaged and frozen down for subsequent experiments. The stable cell lines harboring LY6E orthologues have been described before, with the exception of C. dromedarius 9 . For this, a gBlock was ordered (XM_031439745.1) and Gateway cloning performed as described previously to generate pENTR and pSCRPSY plasmids 9 . Constructs encoding for Ly6/uPAR family members and LY6E ASM mutants have been described before 9 .\\r\\n To generate clonal LY6E KO cells and CRISPR-resistant LY6E, A549 cells were transduced with lentivirus containing a LY6E-specific sgRNA and Cas9 as described previously 9 . To generate a clonal cell line, the bulk transduced population was plated at single cell dilutions. Candidate clones were screened by Western blot for LY6E expression and Sanger sequencing. Silent mutations in the region targeted by the LY6E-specific sgRNA were introduced into HA-tagged LY6E to generate CRISPRresistant LY6E (CR LY6E). LY6E KO A549 cells were reconstituted with CR-LY6E by lentiviral transduction and expression confirmed by Western blot.\\r\\n All cell lines were regularly tested to check they were free of mycoplasma contamination using a commercially available system (PCR Mycoplasma test kit I/RT Variant C, PromKine).\\r\\n The ISG screen was performed as described previously with slight modifications 7,8 . Briefly, 5 x 10 3 Huh7 cells were seeded, transduced with individual lentiviruses, and 48 hours post-transduction infected with HCoV-229E at 33°C. Infection was stopped 24 hours (MOI=0.1) or 48 hours (MOI=0.01) postinfection and plates were immuno-stained as described previously 61 with an anti-HCoV-229E N protein antibody and a AlexaFluor488-conjugated donkey anti-mouse secondary antibody (Extended methods;\\r\\n Antibodies for immunofluorescence and flow cytometry). For high-content high-throughput imaging analysis ImageXpress Micro XLS (Molecular Devices, Sunnyvale, CA) was used as previously described 8 . Hits were normalized to cells expressing the empty vector. Depicted are genes that were author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint expressed in two independent screens with a transduction efficiency of at least 1 % (cut off). ISGs which were cytotoxic for the cells were excluded from the analysis. Normalized data can be found in Extended Data \\r\\n Viral RNA was extracted from cell lysates (NucleoMag-96RNA kit, Macherey Nagel) or supernatant (NucleoMag-Vet kit, Macherey Nagel) using the KingFisher Flex robot according to the manufacturer's recommendations. The commercially available TaqManTM Fast Virus 1-Step Master Mix (Applied Biosystems) was used for RT-qPCR with 2 µl of RNA input added to 8 µl of prepared mastermix per author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint sample. Viral RNA was detected using HCoV-229E specific primers normalized to a qRT-PCR standard for HCoV-229E (contains the M gene of HCoV-229E 63 ). Intracellular viral RNA was normalized to total RNA (determined via the housekeeping gene Beta-2-Microglobulin (B2M)). Cellular RNA was extracted from cell lysates of Huh7 cells using the Nucleospin RNA kit (Machery Nagel) according to the manufacturer's recommendations, quantified via NanoDrop, and used to generate a standard curve.\\r\\n PCR conditions are available upon request. \\r\\n Stable LY6E expressing or control Huh7 cells were seeded in a 24-well plate (4 x 10 4 cells) and mock infected or inoculated with HCoV-229E (MOI=5) in OptiMEM (Gibco) on ice for 1 hour. Cells were washed at least 3x with PBS and harvested immediately (t=0 h) or incubated at 33°C for 24 hours (t=24 h), before cell lysis and extraction of viral RNA as described above. Viral RNA was detected via RT-qPCR normalized to the housekeeping gene B2M as described above.\\r\\n Time course experiment 8 x 10 4 cells stable LY6E expressing or control Huh7 cells were seeded in 12-well plates and mock infected or infected with 229E-CoV-Rluc (MOI=0.1) for 2 hours at 33°C in OptiMEM. Cells were washed 3x with PBS and samples harvested at the indicated time points. To determine intracellular replication, cell lysates were collected and viral RNA extracted (NucleoMag-96RNA kit, Macherey author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint Nagel), followed by RT-qPCR as described before, or cells were lysed using the Renilla Luciferase Assay System kit (Promega) according to the manufacturer's recommendations and Rluc activity determined. To determine extracellular viral replication, viral supernatant was harvested, and viral RNA extracted (NucleoMag-Vet kit, Macherey Nagel), followed by RT-qPCR as described before. To determine intracellular infectivity, cells were subjected to 3 rounds of freeze/thaw cycles, centrifuged to remove debris (4,000 × g for 10 min), and the supernatant titrated on naïve Huh7 cells. To determine extracellular infectivity, supernatant was harvested at the indicated time points and titrated on naïve \\r\\n To test various protease inhibitors, 2 x 10 4 naïve or TMPRSS2-expressing, control or LY6E cells were seeded in a 96-well plate. One day post seeding, cells were pre-treated with the following compounds for 1 hour in OptiMEM, at 37°C: DMSO (1:500, Sigma-Aldrich), E64 D (10 µM, Sigma-Aldrich), and/or Camostat (100 µM, Sigma-Aldrich). Cells were mock infected or infected with HCoV-229E-Rluc (MOI 0.1) in the presence of the inhibitors for 2 hours at 33°C. Medium was changed to DMEM and cells incubated for 24 hours at 33°C. Cells were lysed and Rluc activity detected using the Renilla Luciferase Assay System kit (Promega) according to the manufacturer's recommendations.\\r\\n LY6E expressing or control Huh7 (2 x 10 5 cells) were seeded in 6-well cell culture plates. Cells were transfected using Lipofectamine 2000 (Invitrogen) with an expression plasmid encoding for MERS-CoV S (pCAGGS-MERS S). Cell lysates were harvested 48 hours post-transfection and subjected to Western blot analysis as described.\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint\\r\\n Multiple sequence alignment of LY6E amino acid sequences of different species (NCBI accession numbers provided in the figure) was performed using the EBI web based Clustal Omega tool (doi:\\r\\n 10.1093/nar/gkz268). The phylogenetic tree was built with the \\\"one click\\\" mode for phylogenetic analysis (http://www.phylogeny.fr/) with standard settings (doi: 10.1093/nar/gkn180; doi:\\r\\n 10.1186/1471-2148-10-8).\\r\\n Sections of liver and spleen were preserved in RNAlater Stabilization Solution (Thermo Fisher Scientific) and frozen. Thawed samples were transferred to PBS and homogenized. One-eighth of the homogenate was mixed with TRIzol Reagent (Thermo Fisher Scientific). BMDM were directly lysed in TRIzol Reagent and frozen. Total RNA was isolated according to the manufacturer's protocol. Liver and spleen RNA were subject to DNase treatment (TURBO DNA-Free kit, Thermo Fisher Scientific) per the manufacturer's protocol prior to RNA-Seq analysis. BMDM RNA was analyzed by one-step qRT-PCR using QuantiFast SYBR Green RT-PCR Kit (Qiagen) using Applied Biosciences 7500 Fast Real-Time PCR System.\\r\\n The quantity and quality of the extracted RNA was assessed using a Thermo Fisher Scientific qubit 2. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint Data generated from individual samples (>30 million read per sample, single read 50-mers) were mapped separately against the GRCm38 murine reference genome. Gene expression was calculated for individual transcripts as reads per kilobase per million bases mapped (RPKM). All transcriptomic analyses were performed using CLC Genomics Workbench 20 (Qiagen, Aarhaus).\\r\\n Differentially expressed genes (DEGs) were identified by calculating fold changes in expression, pvalues were corrected by taking false discovery rate (FDR) for multiple comparison into account.\\r\\n Mus Musculus EBI Gene Ontology Annotation Database was used to execute Gene ontology (GO) Enrichment Analyses for biological processes. Gene identifiers for DEGs absolute FC > 5, RPKM > 2) were used as input and identification of significantly enriched GO categories. P-values for specific GO categories were generated after Bonferroni correction for multiple testing. Z-scores, used as indicator for activation of biological processes (positive value) or inactivation (negative value), were calculated based on the expression fold change as follows: The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint gctggcatcgggaacgtggtggactttggctac accctgaacaagggctgctccccgatctgtccc ggcccgagcgtcaatcttggagtggcgtccgt gggcacccactgctgccagagcttcctgtgca acatcagtgcagccgacggcgggctgcgggc cagcacccacgtgctgggcctcgggctcctgc tcagcctgctgtccgccctgctgcggcttggcc cctgaacccagctttcttgtacaaagtggtcccc author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint. The rapid evolution of RNA viruses represents a significant challenge for preventing, treating, 57 and eradicating RNA viral diseases. High mutation rates in RNA viruses generate extensive 58 opportunities to overcome evolutionary hurdles, such as antiviral drugs, host immunity, or 59 engineered attenuating changes (1). The evolutionary pathways traversed by RNA viruses are 60 shaped by natural selection, which will favor some evolutionary trajectories more than others 61 based on whether mutations are beneficial, deleterious, or neutral (2). Predicting the likely results 62 of RNA virus evolution is an important step for anticipating viral emergence and for developing 63 escape-resistant antiviral drugs and vaccines (3, 4). 64 65 Coronaviruses (CoVs) are a family of positive-sense RNA viruses that cause human illnesses 66 ranging from the common cold to severe and lethal respiratory disease (5). All CoVs encode a 67 proofreading exoribonuclease within nonstructural protein 14 (nsp14-ExoN) that is critical for 68 replication, fidelity, fitness, and virulence, and ExoN-inactivation has been proposed as a 69 strategy for live-attenuated vaccine development (6-15). As members of the DEDDh superfamily 70 of exonucleases, CoV ExoNs hydrolyze nucleotides using four metal-coordinating amino acids 71 The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint ExoN active site mutants in alphacoronaviruses, including transmissible gastroenteritis virus and 77 hCoV-229E, have yet to be recovered and are proposed to be lethal for replication (19, 20) . infections and prolonged passage in tissue culture and following treatment with multiple 87 nucleoside analogues (6-11, 13, 14). SARS-CoV-ExoN-AA also is stable during acute and 88 persistent animal infections in immunocompetent and immune-compromised mice (12) . The lack 89 of primary reversion is not due simply to reduced adaptive capacity, as both SARS-CoV-and 90 MHV-ExoN-AA can adapt for increased replication (7, 14). Most strikingly, long-term passage 91 of MHV-ExoN-AA (250 passages, P250) yielded a highly fit population that had directly 92 compensated for defective proofreading through evolution of a likely high-fidelity RdRp (7 requires mutations to all four sites ( Figure 1A ). Viral mutation rates in the absence of 113 proofreading range from 10 -4 to 10 -6 mutations per nucleotide per round of replication (µ) (1). 114\\r\\n Assuming an ExoN-AA mutation rate of 10 -4 µ and accounting for codon degeneracy, the 115 probability of restoring the native amino acid sequence in a single round of replication is 10 -18 . 116\\r\\n Only rarely do ExoN-AA titers exceed 10 6 PFU/mL, so it is unlikely that ExoN-AA could 117 navigate this genetic barrier in a single infectious cycle. Thus, we hypothesized that ExoN-AA 118 reversion, if possible, would proceed incrementally. To identify potential pathways towards 119\\r\\n ExoN-AA reversion, we examined the possible single-nucleotide substitutions surrounding A89 120 and A91 ( Figure 1B ). Three mutations are synonymous, and five mutations yield amino acids 121 unlikely to coordinate with the positively-charged metals required for ExoN catalysis (glycine, 122 valine, proline, threonine, and serine) (16, 19, 21, 22). One mutation per site can restore the 123 acidic charge (i.e. AA-to-ED) but not the native amino acid. These variants have not been tested 124 in a CoV ExoN, but biochemical studies of E. coli DNA polymerase I ExoN mutants suggest that 125 these conservative substitutions would not restore WT-like ExoN activity (23) . We predicted 126 stepwise pathways to ExoN-AAàDE reversion based on restoration of acidic charge followed 127 by reversion to native amino acids ( Figure 1C ). We engineered and recovered variants in ExoN-128 AA requiring three mutations (3nt; ExoN-AD, ExoN-EA), two mutations (2nt; ExoN-DA, ExoN-129 ED, ExoN-AE), or one mutation (1nt; ExoN-DD, ExoN-EE) for reversion to WT-ExoN-DE 130 (Table 1) . We will hereafter refer to these mutants as intermediate revertants. All intermediate 131 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint revertants generated viable progeny during recovery, demonstrating that reversion to WT-ExoN-132 DE along these pathways is theoretically possible. The 3nt and 2nt mutants were genetically 133 stable during recovery, as confirmed by dideoxy sequencing. However, both 1nt mutants (ExoN-134 DD and ExoN-EE) reverted to WT-ExoN-DE during three independent recovery attempts, 135\\r\\n suggesting that these two variants are less fit than WT-ExoN-DE and demonstrating that 136 reversion by 1nt mutation is readily accessible. To test whether the 3nt or 2nt mutants would 137 revert more rapidly than ExoN-AA (4nt), we passaged three lineages of each mutant 10 times at (Table 1 ). In contrast, the 2nt ExoN-AE contained WT-revertants by P2 145 in all lineages at MOI = 0.5 PFU/cell and by P8 in one lineage at MOI = 0.01 PFU/cell. Once 146 observed, WT-revertants dominated the ExoN-AE population for the remaining passages. These 147 data indicate that at least one 2nt mutation pathway can lead to full reversion in tissue culture. 148\\r\\n The probability of ExoN-AE arising during a single infectious cycle of ExoN-AA is low but 149 theoretically achievable (~10 -9 ), so ExoN-AA could conceivably revert within just two infectious 150 cycles. However, complete reversion has never been observed even during prolonged passage or 151 persistent infections, suggesting that additional barriers to the replication, fitness, or maintenance 152 of intermediate revertants exist. 153 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint \\r\\n GAT GAA\\r\\n GCA GCT\\r\\n ExoN Motif I C All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. \\r\\n All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint\\r\\n the intermediate revertants are viable as recombinants but are not found in ExoN-AA 159 populations, we hypothesized that they confer no selective advantage over ExoN-AA (8, 9, 13). 160\\r\\n To test this hypothesis, we first analyzed replication of the 3nt and 2nt intermediate revertants 161 ( The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. nsp12/14-P250) were significantly more sensitive to 5-FU ( Figure 4B ). Finally, the nsp12-P250 224 and nsp14-P250 mutations significantly decreased fitness relative to WT-ExoN-DE ( Figure 4C ). 225\\r\\n We detected no statistical differences between the specific infectivity of WT-ExoN-DE and any 226 of the nsp12-P250 and nsp14-P250 variants in isolated infections ( Figure 4D ). Thus, mutations 227 in nsp12 and nsp14 that arose in the ExoN-AA background were detrimental to replication, 228 mutagen sensitivity, and competitive fitness in the presence of a fully-reverted ExoN-DE. These 229 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint Our results also extend existing studies of CoV ExoN motif I. Motif I AAàDE mutations in the 249 SARS-CoV nsp14-ExoN dramatically reduce nuclease activity in biochemical assays, but no 250 study has examined the contributions of each residue independently (16, 18) . Intermediate 251 revertants of ExoN-AA did not display consistent or statistical differences in replication, 5-FU 252 sensitivity, or competitive fitness relative to ExoN-AA, suggesting that they remain 253 phenotypically ExoN(-) during infection and supporting previous studies that motif I DE is 254 essential for WT ExoN function. Given these results, we were surprised to observe repeated 255 reversion of the ExoN-AE but not the other two 2nt variants, ExoN-DA and ExoN-ED. One 256 potential explanation is that the specific mutational bias of ExoN-AE makes the revertant 257 mutations more accessible than in ExoN-DA or ExoN-ED. Alternatively, if ExoN-AE has 258 profound replication or fitness defects, selection could drive primary reversion more quickly 259 away from this genotype. Consistent with this hypothesis, ExoN-AE reverted more quickly at a 260 higher MOI, where natural selection acts more efficiently on a larger population size (Table 1) Consistent with this hypothesis, the fitness effects of mutations in nsp12-P250 and nsp14-P250 269 differ based on the motif I genotype; they are beneficial in ExoN-AA but detrimental in the WT-270 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint ExoN-DE background. In previous studies, it has been difficult to determine whether the fitness 271 defects in ExoN(-) CoVs are directly linked to low-fidelity replication or through some other 272 mechanism. Our data suggest that the proofreading function of nsp14-ExoN can be uncoupled 273 from its more general role in replication (Figure 4) The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint Cloning and recovery of recombinant viruses. Site-directed mutagenesis in MHV genome 306 fragments was performed using \\\"round the horn\\\" PCR (originally described in (33)). Briefly, 307 adjacent primers containing the mutation of interest were 5¢-phosphorylated using T4 308 polynucleotide kinase (NEB, M0201S) using the buffer from the T4 DNA ligase, which contains 309 ATP (M0202S). PCR was performed on a plasmid template using the Q5 High-fidelity 2x previously (9). Briefly, genomic RNA was detected with a 5' 6-carboxyfluorescein (FAM) and 341 3' black hole quencher 1 (BHQ-1) labeled probe targeting nsp2 (Biosearch Technologies,  342 Petaluma, CA), and RNA copy number was calculated by reference to an RNA standard derived 343 from the MHV A fragment. Samples were plated in technical duplicate to minimize well-to-well 344 variation. Titers were determined by plaque assay in DBT-9 cells, and specific infectivity was 345 calculated as PFU per supernatant genomic RNA copy. 346 347 5-fluorouracil sensitivity assays. Stock solutions of 5-fluorouracil (Sigma F6627) were 348 prepared in dimethyl sulfoxide (DMSO). Sensitivity assays were performed in 24-well plates at 349 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint MOI = 0.01 PFU/cell, as previously described (7) were averaged, and values were excluded from subsequent analysis if the duplicate wells differed 371 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint by > 0.5 Ct. The relative abundance of competitor and reference were determined by subtracting 372\\r\\n Ct thresholds (DCtcompetitor = Ctcompetitor -Ctreference) and converted to reflect the fold-change in 373 ratio (Dratio = 2 -DCt competitor ). The log10Dratio was plotted against passage number, and the change 374 in log10Dratio (i.e. slope of linear regression) is the relative fitness. Note that regressions were fit 375 only through P1-P4, as slight deviations in 1:1 ratio in the input (P0) can skew the slope. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint. 9 also deleted ORF 4a, which is dispensable for MHV replication in cell culture (52). The C-150 terminal region of ns2 within MHV-ExoN(-) P250 was truncated and fused to HE with a -1 151 frameshift. Ns2 is a phosphodiesterase (PDE) that protects viral RNA by degrading 2¢\\uf0a2à\\uf0e05¢\\uf0a2 152 oligoadenylate, the activating factor for cellular RNase L (53-55). The portion of ns2 deleted in The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint similar for all four viruses (Fig. 4B) . These data suggest that the increased replication of P250 173 viruses relative to WT-MHV is not fully accounted for by increased RNA synthesis. In addition 174 to RNA synthesis defects, ExoN(-) CoVs have up-to 20-fold increased mutation frequencies and 175 profoundly increased sensitivity to nucleoside and base analogs relative to WT CoVs (13, 14, 16, 176 17, 38). To determine whether nucleoside analog sensitivity of MHV-ExoN(-) was altered by 177 long-term passage, we treated cells infected with parental and passaged viruses with the base 178 analog, 5-fluorouracil (5-FU). 5-FU is converted intracellularly into a nucleoside analog that 179 incorporates into growing RNA strands and causes A:G and U:C mutations. For simplicity, we 180 hereafter refer to 5-FU as a nucleoside analog. Incorporation of 5-FU is increased in the absence 181 of ExoN activity (16). All viruses displayed a concentration-dependent decrease in viral titer but 182 differed greatly in their susceptibility to 5-FU (Fig. 4C ). At 120µM, WT-MHV P3 titers were 183 reduced by ~1 log 10 , while MHV-ExoN(-) P3 titers were undetectable (> 5 log 10 -fold reduction). 184\\r\\n WT-MHV 5-FU sensitivity was not altered by passage. MHV-ExoN(-) P250 was less susceptible 185 than MHV-ExoN(-) P3 to 5-FU treatment, with only a ~1.5 log 10 decrease in titer at 120 µM. replication cycles in which 5-FU can be incorporated (59) . MHV-ExoN(-) P250 had multiple 193 mutations in the spike glycoprotein, including one in the spike furin cleavage site that reduced 194 syncytia formation. To test whether the spike mutations manifested in resistance to 5-FU, we 195 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. However, M288T is not predicted to catalyze nucleotidylation. Within nsp14, 4 NS mutations 215\\r\\n were identified in the ExoN domain, and 2 NS mutations were in the C-terminal N7-216 methyltransferase domain (Fig. 6B ). We next modeled the structure of MHV nsp14 using Phyre 2 217 software (63) , resulting in highest-probability similarity to the SARS-CoV nsp14-nsp10 complex 218 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint between the structures) of 100% for residues 3-519 of MHV-nsp14. The model predicts that five 220 mutations are located close to surface of the protein (Fig. 6B ). All three modeled zinc finger 221 domains contain one NS mutation (F216Y, Y248H, L473I). Two mutations, D128E and F216Y, 222 are located near the interface between nsp10 and nsp14, though neither site has previously been 223 implicated in nsp10-nsp14 interaction (15, 64, 65). One NS mutation resulted in a D272E 224 substitution in ExoN motif III, a metal-coordinating active site residue. We previously reported 225 that alanine substitution of D272 results in an ExoN(-) phenotype (14), but the viability or 226 phenotype of a D272E substitution was not tested in that study. These data suggest that a 227 network of residues evolved to regulate nsp12 and nsp14 activity or stability in the ExoN(-) 228 background. 229 230 Fixed mutations in nsp12 and nsp14 in MHV-ExoN(-) P250 directly correlate with 231 increased resistance to multiple nucleoside analogs. To determine approximately when the 232 mutations in nsp12 and nsp14 arose, we performed di-deoxy sequencing across these protein-233 coding regions roughly every 20 passages (P10, 31, 50, 72, 90, 100, 120, 140, 160, 180, 200, 220, 234 240) . By this method, we detected consensus NS mutations at P10, P50, and P160 for nsp12, and 235 at P50 and P160 for nsp14 (Fig. 6) . Both nsp12 and nsp14 carried their full complement of P250 236 consensus mutations by P160, except for a minority variant (D913E) in nsp12 maintained at 237 <50% of the population between P200 and P250. These passage levels correlated with increased 238 replication of MHV-ExoN(-) (Fig. 2B ) and with decreasing sensitivity to 5-FU (Fig. 7A ). Neither 239 replication nor 5-FU sensitivity of MHV-ExoN(-) changed substantially between P160 and P250. 240\\r\\n To determine whether MHV-ExoN(-) evolved increased resistance to multiple nucleoside 241 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. implicating a complex evolutionary interaction between these two proteins. The measured fitness 284 correlated with the patterns of nucleoside analog resistance and RNA synthesis associated with 285 mutations in nsp12 and nsp14, suggesting a link between the evolution of these virus phenotypes. 286 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint\\r\\n The result also confirms that nsp12 and nsp14 are important but not sufficient to account for the 287 significantly increased fitness of MHV-ExoN(-) P250 relative to MHV-ExoN(-) P3. 288 289 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. Mutations conferring increased replication fidelity to RNA viruses have most frequently been 334 mapped to RdRps (24, 25, 30, 72) . Three findings suggest that mutations within nsp12-P250 335 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. with nsp12, acting as a primase/processivity factor (74, 75) and a helicase/NTPase, respectively 358 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. \\r\\n Cell culture. DBT-9 (delayed brain tumor, murine astrocytoma clone 9) cells were maintained 409 as described previously (91) The virus titer of each stock was determined by plaque assay using DBT-9 cells as described 425 previously (14, 91). For plaque assays of viruses containing the spike protein from MHV-ExoN(-426 ) P250, which does not form syncytia, plaques were visualized with neutral red (Sigma #N6264, 427 diluted 1:10 in PBS containing calcium and magnesium). Neutral red was added 24 hours after 428 plating and incubated for an additional 3-8 hours before formaldehyde fixation. Plaque 429 purification was performed by infecting DBT cells with serial dilutions of virus and overlaying 430 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint the cultures with agar. Single plaques were isolated, resuspended in PBS containing calcium and 431 magnesium, and inoculated onto fresh DBTs. This process was completed 3 times before 432 experimental stocks were generated, as above. and RNA copy number was calculated by reference to an RNA standard derived from the MHV 475 A fragment. Samples were plated in technical duplicate to minimize well-to-well variation. Titers 476 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint were determined by plaque assay in DBT-9 cells, and specific infectivity was calculated as PFU 477 per supernatant genomic RNA copy. 478\\r\\n Nucleoside and base analog sensitivity assays. 5-azacytidine (AZC), 5-fluorouracil (5-FU), and 480 ribavirin (RBV) were purchased from Sigma (product numbers A2385, F6627, and R9644, 481 respectively). Stock solutions of 5-FU and RBV were prepared in dimethyl sulfoxide (DMSO). 482\\r\\n 2'-C-methyladenosine (CMeA) was received from Gilead Sciences, Inc (Foster City, CA). 483\\r\\n Sensitivity assays were performed as described previously (16) viruses were detected with the same probe used in specific infectivity analyses (14). Reference 516 viruses were detected by a probe targeting the same region but with 10 silent mutations (5'-517 TCCGAACTACTGCAACCCCAAGTG-3') and labeled with 5' Quasar 670 and 3' black hole 518 quencher 2 (BHQ-2) (Biosearch Technologies, Petaluma, CA). RNA copy number was 519 calculated by reference to an RNA standard generated by in vitro transcription of the 520 corresponding MHV A fragment, and relative RNA abundance was calculated as the ratio of 521 competitor genomes to reference genomes. 522 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint and synonymous mutations (purple) are below. 564 The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint statistical significance was determined using one-way ANOVA. For Panel C, statistical 587 significance for change in titer of MHV-ExoN(-) spike-P250 relative to MHV-ExoN(-) P3 was 588 determined using Mann-Whitney test (*P<0.05, **<0.01, ***P<0.001, ns = not significant). 589 The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint tuberculosis DNA polymerase III αβ2ε replicase promotes polymerization and reduces 855 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. Virus titer (log 10 PFU/ml) All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint. \""
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "text(sent_files)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "1952-element Array{SubString{String},1}:\n",
       " \"Odor perception, and emotional and behavioral responses to odors strongly depend on experience, and learned odor-context associations often last for the lifetime of an animal (Mouly and Sullivan, 2010) .\"\n",
       " \"The cellular and neural circuit mechanisms underlying olfactory learning and memory, however, remain poorly understood.\"\n",
       " \"\"\n",
       " \" Recent studies on episodic and contextual learning in hippocampal neural networks have suggested that memories are encoded in the activity of distributed ensembles of neurons, often referred to as a 'memory trace' (Mayford and Reijmers, 2015; Poo et al., 2016; Tonegawa et al., 2015) .\"\n",
       " \"The neurons constituting such a memory trace are thought to encode information about the environmental context and associated emotions of past experiences, and their activity is necessary and sufficient for memory retrieval (Liu et al., 2012; Reijmers et al., 2007; Tanaka et al., 2014) .\"\n",
       " \"\"\n",
       " \" Here, we investigate the organization of odor memory traces in the olfactory (piriform) cortex of mice.\"\n",
       " \"The piriform cortex, a trilaminar paleocortical structure, is the largest cortical area receiving direct afferent inputs from the olfactory bulb, which, in turn, receives topographically organized inputs from olfactory sensory neurons in the nose.\"\n",
       " \"Individual piriform neurons respond to combinatorial inputs from multiple Piriform cortex has long been hypothesized to support auto-associative network functions that can retrieve previously learned information from partial or degraded sensory inputs (Haberly, 2001; Wilson and Sullivan, 2011) .\"\n",
       " \"Piriform pyramidal cells form a large recurrent network, which is reciprocally connected with adjacent highorder associative areas including the prefrontal, entorhinal and perirhinal cortex and the amygdala (Johnson et al., 2000; Sadrian and Wilson, 2015) .\"\n",
       " \"Storage of information is made possible by NMDA-dependent, associative plasticity of connections (Johenning et al., 2009; Kanter and Haberly, 1990; Quinlan et al., 2004) .\"\n",
       " \"\"\n",
       " \" Furthermore, changes in piriform network activity (Chapuis and Wilson, 2011; Chen et al., 2011; Li et al., 2008; Sevelinges et al., 2004) and stabilization of piriform odor representations (Shakhawat et al., 2015) have been observed after associative olfactory learning.\"\n",
       " ⋮\n",
       " \"All rights reserved.\"\n",
       " \"No reuse allowed without permission.\"\n",
       " \"\"\n",
       " \" The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.\"\n",
       " \"Virus titer (log 10 PFU/ml) All rights reserved.\"\n",
       " \"No reuse allowed without permission.\"\n",
       " \"\"\n",
       " \" The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/175562 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/175562 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/175562 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.\"\n",
       " \"The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.\"\n",
       " \"The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.\"\n",
       " \"The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/175562 doi: bioRxiv preprint.\"\n",
       " \"\""
      ]
     },
     "execution_count": 29,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "split_sentences(text(sent_files))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Odor perception, and emotional and behavioral responses to odors strongly depend on experience, and learned odor-context associations often last for the lifetime of an animal (Mouly and Sullivan, 2010) .\n",
      "The cellular and neural circuit mechanisms underlying olfactory learning and memory, however, remain poorly understood.\n",
      "\n",
      " Recent studies on episodic and contextual learning in hippocampal neural networks have suggested that memories are encoded in the activity of distributed ensembles of neurons, often referred to as a 'memory trace' (Mayford and Reijmers, 2015; Poo et al., 2016; Tonegawa et al., 2015) .\n",
      "The neurons constituting such a memory trace are thought to encode information about the environmental context and associated emotions of past experiences, and their activity is necessary and sufficient for memory retrieval (Liu et al., 2012; Reijmers et al., 2007; Tanaka et al., 2014) .\n",
      "\n",
      " Here, we investigate the organization of odor memory traces in the olfactory (piriform) cortex of mice.\n",
      "The piriform cortex, a trilaminar paleocortical structure, is the largest cortical area receiving direct afferent inputs from the olfactory bulb, which, in turn, receives topographically organized inputs from olfactory sensory neurons in the nose.\n",
      "Individual piriform neurons respond to combinatorial inputs from multiple Piriform cortex has long been hypothesized to support auto-associative network functions that can retrieve previously learned information from partial or degraded sensory inputs (Haberly, 2001; Wilson and Sullivan, 2011) .\n",
      "Piriform pyramidal cells form a large recurrent network, which is reciprocally connected with adjacent highorder associative areas including the prefrontal, entorhinal and perirhinal cortex and the amygdala (Johnson et al., 2000; Sadrian and Wilson, 2015) .\n",
      "Storage of information is made possible by NMDA-dependent, associative plasticity of connections (Johenning et al., 2009; Kanter and Haberly, 1990; Quinlan et al., 2004) .\n",
      "\n",
      " Furthermore, changes in piriform network activity (Chapuis and Wilson, 2011; Chen et al., 2011; Li et al., 2008; Sevelinges et al., 2004) and stabilization of piriform odor representations (Shakhawat et al., 2015) have been observed after associative olfactory learning.\n",
      "Finally, excitotoxic lesions of the posterior piriform cortex in rats perturb odor fear memories (Sacco and Sacchetti, 2010) , and optogenetic stimulation of artificial piriform ensembles is sufficient to drive learned behaviors (Choi et al., 2011) .\n",
      "Taken together, these studies have led to the hypothesis that piriform neural ensembles encode olfactory memory traces.\n",
      "\n",
      " To test this hypothesis, we have developed an intersectional genetic strategy in mice to target piriform neurons that are activated by olfactory experience.\n",
      "We employed\n",
      " To selectively label and manipulate piriform neurons that were activated during odor exposure, we used cFos-tTA transgenic mice (Reijmers et al., 2007) , in which the activity-dependent cFos promoter drives expression of the tTA transcription factor, and we stereotaxically injected in their piriform cortex Adeno-Associated Viruses (AAVs) expressing Designer Receptors Exclusively Activated by Designer Drugs (DREADDs) under the control of the tTA-responsive promoter tetO ( Figure 1A) .\n",
      "\n",
      " DREADDs (Alexander et al., 2009; Ferguson et al., 2011) increase (hM3Dq:mCherry) or decrease (HA:hM4Di-IRES-mCitrine) the excitability of neurons upon activation by their ligand clozapine N-oxide (CNO).\n",
      "Temporal control of DREADD expression is provided by doxycycline, administered through the diet, which interferes with the binding of tTA to tetO and thus suppresses transgene expression (Gossen et al., 1995) .\n",
      "Mice maintained in their home cage on a doxycycline-containing diet showed low basal DREADD expression, detected by anti-HA immunohistochemistry and native mCherry fluorescence (Figure 1B and C) .\n",
      "To induce DREADD expression, mice were taken off doxycycline five days before odor presentation.\n",
      "Exposure to odor followed by foot shock (see Experimental Procedures) resulted in the Fos-tagging of sparse neural ensembles (median (interquartile range): 1.6 (0.3) % of piriform neurons, n=2 for hM4Di; 2.2 (0.5) % of piriform neurons, n=3 for hM3Dq), which were broadly dispersed throughout the anterior and posterior piriform cortex ( Figure 1B and C), consistent with previous observations of endogenous cFos expression (Datiche et al., 2001; Illig and Haberly, 2003) .\n",
      "\n",
      " To test whether CNO-mediated activation of DREADDs in Fos-tagged piriform neurons alters their excitability, we performed whole-cell recordings in acute brain author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint slices containing piriform cortex.\n",
      "DREADD-expressing neurons were identified based on mCitrine (HA:hM4Di-IRES-mCitrine) or mCherry (hM3Dq:mCherry) fluorescence.\n",
      "Figure 1) .\n",
      "In the absence of the DREADD ligand CNO, the resting membrane potential of DREADD-expressing piriform neurons was indistinguishable from DREADD-negative cells (-71.70 (4.37) mV in n=24, (4.32) mV in hM4Di-expressing cells, n=14, -72.56 (3.92) mV in hM3Dq-expressing cells, n=15, p=0.631, Kruskal-Wallis ANOVA, Figure 1D ).\n",
      "After bath application of CNO (5 µM), the membrane properties selectively changed in DREADD-expressing cells: hM4Di-expressing cells hyperpolarized (-74.51 (5.17 ) mV, n=12, p=0.007, Wilcoxon Signed Ranks Test), and hM3Dq-expressing cells depolarized (-66.95 (4.12) mV, n=13, p=0.001, Wilcoxon Signed Ranks Test), whereas the resting membrane potential of DREADD-negative cells remained unchanged (-70.28 (3.81) mV, n=15, p=0.890, Wilcoxon Signed Ranks Test, Figure 1D ).\n",
      "The relative changes were 1.00 (0.03) in DREADD-negative cells (n=15, p=0.890, One Sample Wilcoxon Signed Rank Test), 1.04 (0.03) in hM4Di-expressing cells, (n=12, p<0.001, One Sample Wilcoxon Signed Rank Test) and 0.91 (0.09) in hM3Dq-expressing cells, (n=13, p=0.001, One Sample Wilcoxon Signed Rank Test), Figure 1E ).\n",
      "To determine the impact of CNO treatment on neuronal excitability, we delivered depolarizing current steps and compared the number of action potentials triggered by 60 pA step current above action potential threshold (see Experimental Procedures).\n",
      "We found that in hM4Di-expressing cells, the number of evoked action potentials significantly decreased after CNO application (3.75 (1.59) vs. 0.30 (3.48), n=10, p=0.010, Wilcoxon Signed Ranks Test), whereas in hM3Dq-expressing cells, the number of author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint action potentials significantly increased after CNO application (8.67 (9.10) vs. 19.0 (27.6) mV, n=11, p=0.006, Wilcoxon Signed Ranks Test).\n",
      "In contrast, DREADDnegative cells did not exhibit a significant change in the number of evoked action potentials upon CNO application (4.33 (1.67) vs. 4.67 (3.00), n=9, p=0.672, Wilcoxon Signed Ranks Test, Figure 1F ).\n",
      "The relative changes were 1.00 (0.41) in DREADDnegative cells, (n=9, p=0.641, One Sample Wilcoxon Signed Rank Tests), 0.09 (0.81) in hM4Di-expressing cells (n=10, p=0.010, One Sample Wilcoxon Signed Rank Tests) and 1.81±1.09 in hM3Dq-expressing cells (n=11, p=0.004, One Sample Wilcoxon Signed Rank Tests, Figure 1G ).\n",
      "Taken together, these experiments show that Fos-tagging during olfactory fear conditioning marks a sparse and dispersed subpopulation of piriform neurons, and that CNO-mediated activation of hM4Di or hM3Dq expressed in Fos-tagged neurons selectively decreases or increases their excitability.\n",
      "\n",
      " If Fos-tagged piriform neurons that were activated during olfactory learning constitute an essential component of an olfactory memory trace, then perturbing the activity of these ensembles should interfere with memory recall.\n",
      "To test this prediction, we Fostagged piriform neurons during olfactory fear conditioning, and then chemogenetically silenced Fos-tagged ensembles during odor fear memory recall.\n",
      "cFos-tTA transgenic mice (\"experimental group\", n=8) were bilaterally injected in piriform cortex with the AAV-tetO-hM4Di vector.\n",
      "Wild-type mice injected with the same virus served as a control (\"control group\", n=8).\n",
      "During training, the conditioned odor (CS+) was presented at one of the ends of the conditioning box and paired with a foot shock, applied to the side of the box where the odor was delivered (see Experimental author/funder. All rights reserved. No reuse allowed without permission.   The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint Procedures and Figure 2A ).\n",
      "Mice thus learned to escape from the conditioned stimulus (CS+) by running towards the opposite side of the box.\n",
      "Two control odors, whose presentation was not paired with foot shock (CS-), were used to entrain odorspecific as opposed to generalized fear responses (Chen et al., 2011) .\n",
      "Doxycycline was removed from the diet of mice prior to training, to permit the induction of hM4Di expression in active Fos-expressing piriform neurons of mice of the experimental group.\n",
      "Mice were returned to doxycycline-containing diet immediately after odor fear conditioning, and memory recall of the odor-foot shock association was tested three days later by presenting the odors alone in a new box ( Figure 2B and C).\n",
      "All mice were intraperitoneally (i.p.) injected with CNO before testing, to exclude potential offtarget effects by CNO (Gomez et al., 2017) .\n",
      "For all mice, we verified viral gene expression by post hoc histological examination.\n",
      "Previous studies have suggested that the posterior piriform cortex is important for associative memory encoding (Sacco et al., 2010; Calu et al., 2008; Haberly 2001) .\n",
      "We therefore excluded from our analysis mice in which parts of the posterior piriform cortex were spared from viral infection (Supplementary Figure 2) .\n",
      "We combined two parameters to quantify the learned escape behavior: the position of the mouse in the box, and the increase in its maximum velocity after odor presentation ( Figure 2B ).\n",
      "These parameters were chosen because they described well the behavioral response of mice to the odor-foot shock pairing during training (Supplementary Figure 3) .\n",
      "\n",
      " As expected, mice in the control group exhibited robust escape behavior upon CS+ presentation.\n",
      "The median position of the mice in the box at the end of stimulus delivery was 0.74 (with 0 defined as the site of odor presentation and 1 as the opposite end of the box), and mice exhibited a 1.9-fold increase of their maximum velocity ( Figure 2E, G and H) .\n",
      "In contrast, escape behavior was significantly reduced author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint in mice in which Fos-tagged neurons were silenced.\n",
      "Their median position in the box at the end of stimulus delivery, (0.42), and their 1.1-fold increase in maximum velocity were significantly lower than in the control group (Mann-Whitney test, U=7, p=0.007 for both position and velocity, Figure 2E , G and H).\n",
      "The difference in the position of mice after CS+ and CS-presentation was also significantly different between the control and experimental groups (Mann-Whitney test, U=13, p=0.0499, Figure 2F ).\n",
      "Differences in behavioral responses of cFos-tTA transgenic mice were specifically due to hM4Di activation, as escape behavior after CNO injection was unaffected in cFos-tTA transgenic mice injected with AAVs expressing the calcium indicator GCaMP (n=9, Supplementary Figure 4 ).\n",
      "\n",
      " All mice were similarly close to the odor port at the onset of CS+ odor presentation (Mann-Whitney test, U=28, p=0.7209, Figure 2D ), and had similar maximum velocities during the 7 s time period that preceded odor presentation (Mann-Whitney test, U=24, p=0.4418, Figure 2G ).\n",
      "These observations exclude any inherent differences between groups due to how the CS+ was presented.\n",
      "Furthermore, escape behavior in control mice was specific to the CS+, as none of the mice ran away from the CS-( Figure 2D , E and H).\n",
      "Finally, we did not observe any differences in the overall mobility of mice throughout the entire testing session.\n",
      "The median speed of mice of all groups (0.68 (0.28) cm/s and 0.46 (0.14) cm/s for the control and experimental groups respectively) was not significantly different (Mann-Whitney test, U=22, p=0.3282), nor did mice show a bias for the left or right side of the testing box (time spent on the left side divided by time spent on the right side: 1.36 (0.72) and  1.51 (0.52) for the control and experimental groups respectively, Mann-Whitney test, U=27, p=0.6454).\n",
      "Together, these data suggest that Fos-tagged piriform ensembles that were activated during olfactory fear conditioning are necessary for odor fear author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint memory recall.\n",
      "\n",
      " Even though cFos-tTA transgenic mice expressing hM4Di failed to robustly escape from the CS+, it should be noted that they behaved differently after CS+ and CSpresentation.\n",
      "They were further away from the odor port after CS+ presentation than after CS-presentation (Wilcoxon matched-pairs signed rank test, W=-36, p=0.0078, Figure 2E ), and their maximum velocity ratio was significantly higher (Wilcoxon matched-pairs signed rank test, W=-36, p=0.0078, Figure 2H ).\n",
      "This could indicate a partial memory of the learned association, possibly due to incomplete silencing of Fos-tagged neurons in piriform cortex, or the existence of parallel neural pathways that partially compensate for the loss of piriform functions.\n",
      "Finally, silencing of the piriform cortex in only one hemisphere did not abolish the learned escape behavior to the conditioned odor (Supplementary Figure 5) .\n",
      "\n",
      " It is possible that the chemogenetic silencing of piriform ensembles that were Fostagged during olfactory fear conditioning disrupts odor detection and discrimination rather than selectively affecting odor fear memory recall.\n",
      "To test this possibility, we monitored sniffing behavior of mice during an olfactory habituation/dishabituation assay ( Figure 3A) , a well established test for odor detection and discrimination (Coronas-Samano et al., 2016; Verhagen et al., 2007; Wesson et al., 2008) .\n",
      "Fostagging of piriform neurons during olfactory fear conditioning was performed as described above.\n",
      "Three days later, changes in sniffing behavior in response to odor exposure were tested in a plethysmograph, while hM4Di-expressing piriform neurons author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint in cFos-tTA transgenic mice were silenced using i.p. injection of CNO (see Experimental Procedures).\n",
      "We observed that mice of both experimental (n=8) and control (n=6) groups increased their sniff frequency upon presentation of a novel, neutral odor (1.6 fold increase; Wilcoxon matched-pairs signed rank test, W=21, p=0.0312 and W=36, p=0.0078 for the control and experimental groups, respectively; Figure 3B and C).\n",
      "Repeated exposure to the same odor (neutral odor or CS-) resulted in a decrease in the sniff frequency, reflecting habituation after 4 consecutive exposures (Figure 3C and D) .\n",
      "Finally, presentation of the CS+ increased sniff frequency (1.4 fold increase; Wilcoxon matched-pairs signed rank test, W=21, p=0.0312 and W=34, p=0.0156 for the control and experimental groups, respectively, Figure 3D ).\n",
      "Such changes in odor sampling behavior have been shown to report the detection and discrimination of different odor stimuli (Coronas-Samano et al., 2016; Verhagen et al., 2007; Wesson et al., 2008) .\n",
      "Therefore, these data suggest that basic behaviors characteristic of odor sampling, detection and discrimination were unaffected by the silencing of piriform ensembles that were Fostagged during olfactory fear conditioning.\n",
      "\n",
      " Odors activate unique but overlapping ensembles of piriform neurons (Bolding and Franks, 2017; Poo and Isaacson, 2009; Roland et al., 2017; Stettler and Axel, 2009 ).\n",
      "Therefore, silencing neurons that respond to one odor could partially interfere with the retrieval of information associated with other odors.\n",
      "To test this prediction we modified the behavioral protocol to separate Fos-tagging from olfactory fear conditioning.\n",
      "cFos-tTA transgenic and wild-type control mice were injected with AAV-tetO-hM4Di, and expression of hM4Di in piriform neurons was induced while mice author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint were exposed to odor (eugenol or beta-citronellol) in a neutral environment, without subsequent foot shock.\n",
      "Fos-tagging resulted in the labeling of sparse neural ensembles, similar to those labeled during odor-foot shock association (1.1 (0.3) % of piriform neurons, n=3).\n",
      "Mice were returned to doxycycline-containing diet and trained two days later to associate a different odor (ethyl-acetate, CS+) with foot shock.\n",
      "\n",
      " Behavioral testing of the learned escape behavior in response to the CS+ was performed one day later, after i.p. injection of CNO (see Experimental Procedures and Figure 4A ).\n",
      "We found that learned escape behavior of cFos-tTA transgenic mice expressing hM4Di (experimental group, n=12) was similar but somewhat attenuated compared to wild-type controls (n=10).\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Both groups exhibited an escape behavior after CS+ presentation, indicated by the significant increase in the maximum velocity (Wilcoxon matched-pairs signed rank test, W=55, p=0.0020 and W=56, p=0.0269 for the control and experimental groups respectively; no difference in the maximum velocity ratio after/before between groups, Mann-Whitney test, U=35, p=0.1071, Figure 4E and F).\n",
      "The difference in the position of mice after CS+ and CSpresentation was also similar between the control and experimental groups (Mann-Whitney test, U=37, p=0.1402, Figure 4D ).\n",
      "However, we observed that silencing neutral odor representations somewhat dampened the behavioral response to the conditioned stimulus, as behavioral responses were less robust compared to controls when quantifying the position of mice in the box after CS+ presentation (Mann-Whitney test, U=22, p=0.0112, Figure 4B and C).\n",
      "The attenuated behavioral responses we observe likely reflect the partial degradation of odor information as a result of the silencing of piriform neurons responsive to both the CS+ and the neutral odor.\n",
      "\n",
      " author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint\n",
      " If piriform neural ensembles that were activated during olfactory fear conditioning encode an odor fear memory trace, then reactivation of these neurons may be sufficient to trigger memory recall.\n",
      "To test this prediction, we devised a modified behavioral task to assess memory recall.\n",
      "We monitored exploratory behavior in an open field during ambient odor exposure or artificial reactivation of Fos-tagged neurons, a test that does not rely on an escape behavior from an odor originating from a spatially defined odor source ( Figure 5A ).\n",
      "We first trained mice to associate ethyl-acetate, the CS+, with foot shock.\n",
      "Upon ambient exposure to the CS+, diffused above the open field, exploratory behavior of mice was significantly attenuated compared to baseline exploratory behavior measured one day earlier (Wilcoxon matched-pairs signed rank test, n=6, W=-21, p=0.0312).\n",
      "Ethyl-acetate exposure did not affect exploratory behavior of mice that had previously been exposed to ethylacetate without foot shock (Wilcoxon matched-pairs signed rank test, n=5, W=1, p>0.9999).\n",
      "These data suggest that re-exposure of mice to the CS+ causes an increase in anxiety-like behavior, expressed as a decrease in exploratory behavior in the open field.\n",
      "\n",
      " We next repeated the experiment in cFos-tTA transgenic mice, in which the piriform cortex had been infected with the activating DREADD AAV-tetO-hM3Dq (Supplementary Figure 6) .\n",
      "Instead of exposure to the CS+, we i.p. injected the DREADD ligand CNO to reactivate hM3Dq-expressing piriform neurons that were Fos-tagged during odor-foot shock association (see Experimental Procedures and Figure 5A ).\n",
      "We found that CNO-mediated reactivation, similar to CS+ exposure, caused a significant decrease in exploratory behavior of mice that had undergone author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint olfactory fear conditioning (Wilcoxon matched-pairs signed rank test, n=8, W=-34, p=0.0156, Figure 5B and C).\n",
      "CNO-mediated reactivation of piriform neurons that were Fos-tagged during ethyl-acetate exposure without foot shock did not affect exploratory behavior of mice (Wilcoxon matched-pairs signed rank test, n=7, W=-10, p=0.4688, Figure 5B and C).\n",
      "Taken together, these data suggest that chemogenetic reactivation of piriform neurons that were active during odor-foot shock exposure is sufficient to trigger fear memory recall.\n",
      "\n",
      " We next asked whether artificial memory recall depends on the specificity of the Fostagged neural ensemble: is piriform reactivation sufficient to trigger fear memory recall, as long as it includes the CS+-tagged ensemble?\n",
      "For this purpose, we generated a synthetic Fos-tagged ensemble, by sequentially exposing mice to ethylacetate paired with foot-shock (CS+) and neutral odors (CS-, eugenol and betacitronellol) ( Figure 5D ).\n",
      "While exposure to the CS+ resulted in anxiety-like behavior similar to what was observed in the previous experiment (Wilcoxon matched-pairs signed rank test, n=6, W=-21, p=0.0312, Figure 5E and F), artificial reactivation of synthetic Fos-tagged ensemble did not result in changes in exploratory behavior that would indicate memory recall (Wilcoxon matched-pairs signed rank test, n=8, W=-4, p=0.8438, Figure 5E and F).\n",
      "These data suggest that during artificial reactivation, piriform cortex cannot extract meaningful information from the synthetic representation generated by the sequential presentation of CS+ and CS-odors.\n",
      "author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint\n",
      " The piriform cortex has long been thought to provide the substrate for storing associative olfactory memories.\n",
      "Several studies have shown that piriform cortex cells can encode information that carries behavioral significance (Calu et al., 2008 , Mandairon et al., 2014 , Sacco et al., 2010 ), yet their functional relevance has never been assessed by direct experimental manipulation.\n",
      "Using a cFos-dependent, intersectional genetic approach to visualize and manipulate piriform neurons activated during olfactory fear conditioning (Figure 1 ), we found that chemogenetic silencing of Fos-tagged piriform ensembles abolished a learned odor escape behavior (Figure 2 ).\n",
      "Silencing these Fos-tagged ensembles did not alter basic odor detection and discrimination (Figure 3) .\n",
      "Furthermore, chemogenetic silencing of neurons responsive to a neutral odor only moderately attenuated odor fear memory recall, suggesting that Fos-tagged piriform ensembles are largely odor-specific ( Figure 4) .\n",
      "Finally, specific chemogenetic reactivation of piriform ensembles that were active during learning resulted in reduced exploratory behavior in an open field assay, an effect that was similarly observed when exposing mice to the conditioned odor stimulus (Figure 5) .\n",
      "Together, our experiments identify piriform neurons that were active during learning as necessary and sufficient to trigger odor fear memory recall.\n",
      "This indicates the formation of functional connections between Fos-tagged piriform cells and other associative areas and firmly establish piriform ensembles as an essential component of odor fear memory traces.\n",
      "\n",
      " Artificial reactivation of an olfactory memory trace author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint Reactivation of neurons by CNO-mediated activation of DREADD receptors does not recapitulate the temporal characteristics of piriform odor responses and their modulation by active sampling (Bolding and Franks, 2017; Miura et al., 2012) .\n",
      "\n",
      " Despite this, reactivation of Fos-tagged piriform ensembles activated during odor-foot shock exposure was sufficient to elicit a behavioral response.\n",
      "How piriform neural circuits and downstream target regions process such artificial activity patterns remains to be determined.\n",
      "One possibility is that despite the temporal limitations of memory trace reactivation, piriform network mechanisms can retrieve the perception of the reinforced conditioned stimulus.\n",
      "Consistent with this model, recent studies have shown that spatial patterns of odor-evoked activity, in the absence of precise temporal information, are sufficient to decode odor identity in piriform cortex (Bolding and Franks, 2017; Miura et al., 2012; Roland et al., 2017) .\n",
      "Alternatively, it is possible that reactivation generates a state of stress and anxiety, but without evoking the perception of the odor.\n",
      "Candidate target areas for the processing of odor-fear associations include the basolateral amygdala and the medial prefrontal cortex (Dejean et al., 2016; Gore et al., 2015) , however, the relevant neural circuit components remain to be identified.\n",
      "Finally, reactivating a synthetic ensemble of neurons activated during exposure to the reinforced stimulus (CS+) and the nonreinforced stimuli (CS-) did not elicit a measurable behavioral response.\n",
      "This observation suggests that behaviorally relevant information cannot be extracted from a neural ensemble representing conflicting (aversive versus neutral) information.\n",
      "This result is consistent with the finding that the reactivation of an artificial contextual memory in the hippocampus competes with the retrieval of a learned context-shock association (Ramirez et al., 2013) .\n",
      "author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint\n",
      " A general limitation of the cFos-tTA system is that the tagging of active neurons is transient.\n",
      "The duration of the expression of tTA-dependent proteins is determined by the time course of induction and the stability of cFos-tTA-dependent transcripts and proteins in tagged neurons.\n",
      "Previous studies have shown that TetTag-dependent expression of regulators of neural activity lasts for at least five days but becomes undetectable by thirty days (Cowansage et al., 2014; Liu et al., 2012; Zhang et al., 2015) .\n",
      "We therefore restricted our manipulations of neural activity and behavioral analyses to a three-day time window after Fos-tagging of neurons during olfactory conditioning.\n",
      "Another constraint of the Fos-tagging system is its slow temporal dynamics.\n",
      "As a consequence, neurons responding to the reinforced conditioned stimulus (CS+), the odorant paired with foot shock, and neurons responding to the non-reinforced conditioned stimuli (CS-), the odorants not associated with foot shock, were tagged in our silencing experiments.\n",
      "The use of a CS-compromises the specificity of the Fos-tagged neural ensembles.\n",
      "Nevertheless, the CS-provides an important experimental readout for the odor selectivity of the behavioral response.\n",
      "\n",
      " Interestingly, the number of Fos-tagged piriform neurons we observe is significantly lower than the number of odor-responsive neurons detected in electrophysiological recordings in awake mice (Bolding and Franks, 2017; Miura et al., 2012; Zhan and Luo, 2010) .\n",
      "However, in these experiments, a large fraction of cells exhibited low firing rates.\n",
      "Therefore, the Fos-tagged population could represent a subpopulation of neurons that is strongly activated by odor (Schoenenberger et al., 2009) .\n",
      "Fos-tagging could also mark plastic changes supporting memory formation (Cole et al., 1989; Minatohara et al., 2015) .\n",
      "Indeed, cFos mRNA levels decrease significantly after injection of an antagonist of the NMDA receptor, a key player in the induction of author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint synaptic plasticity (Tayler et al., 2011) , and long-term memory and synaptic plasticity are impaired when cFos production is perturbed in the central nervous system (Fleischmann et al., 2003; Seoane et al., 2012) .\n",
      "Future studies will allow to directly test these hypotheses.\n",
      "\n",
      " The hippocampus has been studied extensively for its role in spatial and contextual memory (Basu and Siegelbaum, 2015) .\n",
      "Recently, Fos-tagging of hippocampal neurons during contextual fear conditioning has provided important insight into the cellular and neural circuit mechanisms of learning and memory (Cai et al., 2016; Garner et al., 2012; Liu et al., 2012; Ramirez et al., 2013; Roy et al., 2017; Ryan et al., 2015) .\n",
      "However, if principles of memory formation and storage in hippocampusrelated neural networks apply to other cortical structures remains largely unknown.\n",
      "\n",
      " Interestingly, pirform cortex and hippocampus share a similar circuit organization and both have been modeled as auto-associative networks (Haberly, 2001) .\n",
      "Consistent with this model, our findings reveal striking similarities between olfactory memory traces in the piriform cortex and memory traces of contextual fear in the hippocampus.\n",
      "In both piriform cortex and hippocampus, learning activates sparse and distributed ensembles of neurons that appear to lack topographic organization.\n",
      "Furthermore, hippocampal neurons tagged during contextual fear conditioning, and piriform neurons tagged during olfactory fear conditioning were both necessary and sufficient for memory retrieval (Liu et al., 2012; Tanaka et al., 2014) , The olfactory system is particularly well-suited for studying learning and memory.\n",
      "Indeed, odor memories and olfactory-driven behaviors are robust in animals, including in mice, providing highly reliable and quantitative behavioral readouts.\n",
      "Furthermore, the author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint olfactory cortex is only two synapses away from the peripheral olfactory sensory neurons in the olfactory epithelium, and the neural inputs that drive piriform cortex activity have been well characterized (Banerjee et al., 2015; Dhawale et al., 2010; Economo et al., 2016; Fukunaga et al., 2012; Kato et al., 2012; Roland et al., 2016; Yamada et al., 2017) .\n",
      "Knowing the properties of neural input patterns will provide the basis for understanding neural circuit input-output computations and their modification by learning and experience.\n",
      "\n",
      " author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint\n",
      " Mice were housed at 24°C with a 12-hour light/12-hour dark cycle with standard food and water provided ad libitum.\n",
      "Mice were group-housed with littermates until the beginning of surgery and then single-housed in ventilated cages throughout the duration of the experiment.\n",
      "cFos-tTA mice (Reijmers et al., 2007) were fed a diet containing 1g/kg doxycycline for a minimum of 4 days before surgery.\n",
      "Wild-type control animals were siblings that did not carry the cFos-tTA transgene.\n",
      "The age of mice at the time of behavioral testing ranged from 10 to 15 weeks.\n",
      "Male mice were used for the hM4Di-mediated neural silencing, and females for the hM3Dq-mediated neural activation experiments.\n",
      "Experiments were carried out according to European and French national institutional animal care guidelines (protocol APAFIS#2016012909576100).\n",
      "\n",
      " The Adeno-associated viruses (AAVs) were generated at Penn Vector Core, University of Pennsylvania (serotype 8, 10 13 genome copies/mL, 1:2 dilution with sterile PBS on the day of injection).\n",
      "\n",
      " author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint\n",
      " Mice were anaesthetized with ketamine/xylazine (100 mg/kg/10 mg/kg, Sigma-Aldrich) and AAV vectors were injected stereotaxically into the piriform cortex Sections were washed 2 times in PBST and 1 time in PBS, mounted on slides and author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint coverslipped with Vectashield mounting medium (Vectorlabs).\n",
      "Images were acquired as Z-stacks (70 to 140 µm in total thickness, step size 7 µm) with a Leica SP5 confocal microscope or as single plane sections with a Zeiss Axio Zoom microscope and processed in Fiji.\n",
      "\n",
      " Counting Z-stacks (7 µm step) were acquired with a Leica SP5 confocal microscope with a 20X objective and a resolution of 512x512 or 1024x1024 pixels (1 pixel: 0:72µm).\n",
      "First, the piriform cortex was delineated on a maximum projection of each stack using a custom-written ImageJ macro.\n",
      "For each mouse, 2 to 4 sections (median volume 9.3e-2 (9.4e-3) mm 3 ) were analyzed, from one or both hemispheres, at Y=-0.8 and Y=-1.6mm relative to bregma.\n",
      "\n",
      " HA-stained neurons (hM4Di) were counted by hand.\n",
      "hM3Dq-mCherry positive neurons were counted using a custom-written ImageJ plugin.\n",
      "After pre-processing the image (Subtract Background, Remove Outliers and Median Filter), the z-stack is thresholded using the RenyiEntropy algorithm.\n",
      "On each slice of the stack, objects with an area smaller than 40µm 2 are removed using the Analyze Particle command in ImageJ.\n",
      "A mask of each slice from the stack is created containing the filled outline of the measured particles.\n",
      "The number of cells is the number of 3D objects detected in the stack.\n",
      "Automated cell detection can be prone to error, thus, the detection efficiency was checked on a few pictures by two people blind to the experimental conditions.\n",
      "\n",
      " The numbers of cells were calculated as numbers of cells per cubic millimeter.\n",
      "An estimate of the total number of neurons (counterstained with NeuroTrace) per cubic millimeter (7.1e4 mm -3 ) was obtained by counting neurons in representative volumes author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint containing the three layers of piriform cortex.\n",
      "Numbers are consistent with those obtained by (Srinivasan and Stevens, 2017) .\n",
      "\n",
      " Parasagittal or coronal slices (300 µm thick) of piriform cortex were prepared from 6-8 week-old cFos-tTA mice injected with AAV-tetO-hM4Di-IRES-mCitrine or AAV-tetO-hM3Dq-mCherry.\n",
      "Animals were anaesthetized with ketamine and xylazine (100 mg/kg/10 mg/kg, Sigma-Aldrich), perfused with ice-cold ACSF (125 mM NaCl, 2.5 mM KCl, 25 mM glucose 25mM NaHCO 3 , 1.25 mM NaH 2 PO 4 , 2 mM CaCl 2 , 1 mM MgCl 2 , 1 mM pyruvic acid, bubbled with 95% O 2 and 5% CO 2 and adjusted to 295±5 mOsm osmolarity), and decapitated.\n",
      "The brain was cooled with ice-cold ACSF solution and then sliced using a 7000SM2 vibrating microtome (Campden Instruments, UK).\n",
      "Slices were incubated in the same solution at 34°C for one hour and then continuously perfused with ACSF solution (2 mL/min) at 34°C in the recording chamber.\n",
      "DREADD expression was detected with two-photon excitation (830nm, ChameleonMRU-X1, Coherent, UK) under a Scientifica TriM Scope II microscope (LaVision, Germany), with a 60x water-immersion objective.\n",
      "Whole-cell recording pipettes with 5-7 MΩ resistance were filled with the following solution (in mM): 122 Kgluconate, 13 KCl, 10 HEPES, 10 phosphocreatine, 4 Mg-ATP, 0.3 Na-GTP, 0.3 EGTA (adjusted to pH 7.35 with KOH).\n",
      "The recording solution also contained the morphological tracer Alexa Fluor 594 (5 µM, red channel) for nonexpressing and hM4Di-positive cells or the Ca 2+ -sensitive dye Fura-2 (300 µM, replacing EGTA in the recording solution, green channel) for hM3Dq-positive cells, to enable identification of the patched cell and to visualize its morphology.\n",
      "The excitability of cells was measured in current-clamp mode by 500 ms steps of current author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint injections from -300 to +500 pA with steps of 20 pA.\n",
      "We compared the number of action potentials triggered by equivalent depolarization at each step 60 pA above the action potential threshold before and after CNO (5 µM) application.\n",
      "The series resistance was usually <20 MΩ, and data were discarded if it changed by more than 20% during the recording.\n",
      "Signals were amplified using EPC10-2 amplifiers (HEKA Elektronik, Lambrecht, Germany).\n",
      "Voltage-clamp recordings were filtered at 5 kHz and sampled at 10 kHz, and current-clamp recordings were filtered at 10 kHz and sampled at 20 kHz, with the Patchmaster v2x32 program (HEKA Elektronik).\n",
      "\n",
      " A training box was used to train mice to escape from an odor.\n",
      "The box was rectangular (L 57 cm, W 17 cm, H 64 cm), with a grid floor made of 72 stainless-steel rods (diameter=6 mm, space between rods=2 mm).\n",
      "Current was delivered by an aversive stimulator (MedAssociates, 115V, 60 Hz).\n",
      "A custom-made switcher allowed an electric foot shock (0.6 mA, 0.6 ms) to be applied independently to either half of the box.\n",
      "A testing box was used to test memory retrieval.\n",
      "Its dimensions were similar to the training box.\n",
      "Materials for the walls and floor were different from the training box to create a different context.\n",
      "The open field box consisted of a white Plexiglas (6 mm thickness) container (50 cm x 50 cm x 38 height).\n",
      "\n",
      " Sniffing behavior was monitored in freely moving mice using a plethysmograph (Emka Technologies).\n",
      "Electrical signals were amplified using RHD 2132 amplifier boards (Intan Technologies) and band-pass filtered (1-30 Hz).\n",
      "\n",
      " Odors were delivered using an 8 channels olfactometer (Automate Scientific).\n",
      "\n",
      " Continuous air flowed through a bottle of mineral oil (Sigma-Aldrich) to avoid pressure changes when presenting the odors.\n",
      "The following odors (Sigma-Aldrich), author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint at a concentration of 1% (vol/vol) in mineral oil were used: ethyl-acetate, betacitronellol, eugenol, limonene, and pinene.\n",
      "\n",
      " AAVs were stereotaxically injected in both hemispheres of piriform cortex of cFos-tTA transgenic mice and littermate wild-type controls.\n",
      "Mice were given 10 days to recover, before being habituated on day 11 to the training and testing/open field/plethysmograph boxes (40 and 10 min each).\n",
      "24 hours after habituation, doxycycline was removed and replaced by a normal diet.\n",
      "Five days later, mice were fear conditioned to ethyl-acetate or exposed to an odor (odor \"C\") in a neutral environment.\n",
      "Immediately after fear conditioning or odor presentation, mice were put back on a diet containing doxycycline.\n",
      "During fear conditioning, ethyl-acetate (the CS+) was presented at one extremity of the training box, and the corresponding halfside of the box' floor was electrified for 0.6 s (0.6 mA).\n",
      "The foot shock was delivered 4 seconds after CS+ presentation, and the CS+ lasted for an additional 2 seconds.\n",
      "\n",
      " Mice learned to escape to the opposite side when the CS+ was presented.\n",
      "The CS+ was presented 4 times, 2 times on each side of the box.\n",
      "When fear conditioning included presentation of two non-reinforced conditioned stimuli (CS-1 and CS-2), the choice between CS+ and CS-as well as the side of presentation was pseudorandomized.\n",
      "Training always started with one presentation of each CS-followed by the CS+.\n",
      "The two CS-odors were presented during 7 seconds, 3 times each, with a total presentation of 3 on the right side and 3 on the left side.\n",
      "During testing, the CS+ was presented 4 times (2 times on each side), and each CS-was presented once on each side, for 7 s each.\n",
      "As for the training session, the order and side of odor presentations was pseudo-randomized: testing always started with one presentation author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint of each CS-, followed by one CS+ presentation.\n",
      "\n",
      " Clozapine-N-oxide (C0832, Sigma-Aldrich) was first dissolved in dimethylsulfoxide (DMSO, D2438, Sigma, final concentration of 1% vol/vol), then further diluted in 0.9% sterile saline solution to a final concentration of 0.2 mg/mL.\n",
      "Aliquots were stored for up to two months at -20°C and equilibrated to room temperature before injection.\n",
      "The solution was administered intraperitoneally (3 mg/kg).\n",
      "After injection, mice were left undisturbed for 25 minutes (Figure 2 and 4) or 5 minutes (Figure 3 and 5) in their home cage before the start of the experiment.\n",
      "\n",
      " Electrophysiological data were analyzed with custom-made software in Python 3.0 and averaged in MS Excel (Microsoft, USA).\n",
      "Behavioral sessions were video recorded and analyzed using custom-written MatLab programs (available upon request).\n",
      "Parameters extracted from multiple CS+ (CS-) presentation were averaged for each mouse.\n",
      "\n",
      " Statistical analysis was performed with Prism (GraphPad).\n",
      "Non-parametric tests were used: Mann-Whitney test for between group comparison, and Wilcoxon matchedpairs signed rank test for within group comparison.\n",
      "Values are represented as median (interquartile range) unless otherwise stated.\n",
      "The n represents number of mice, or number of recorded neurons for the electrophysiology experiment.\n",
      "author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint before and after CNO application.\n",
      "After CNO application, the number of APs in DREADD-negative cells (Ctrl) does not change, significantly decreases in hM4Diexpressing cells and significantly increases in hM3Dq-expressing cells.\n",
      "\n",
      " (G) The relative change in the number of APs after/before CNO in DREADD-negative cells (Ctrl) does not change, significantly decreases in hM4Di-expressing cells and significantly increases in hM3Dq-expressing cells.\n",
      "\n",
      " Data are shown as individual data points, median and interquartile range, ** p<0.01.\n",
      "author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint A) During fear conditioning, mice are trained to associate ethyl-acetate (CS+) with foot shock.\n",
      "The CS+ is presented at one end of the training box, and the corresponding half-side of the box is electrified.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Training includes the presentation of two other odors that are not paired with foot shock (CS-, eugenol and beta-citronellol).\n",
      "\n",
      " The CS+ is presented 4 times (2 times on each side) and each CS-three times.\n",
      "The order and side of odor presentations is pseudo-randomized.\n",
      "B) During testing, the CS+ is presented 4 times (2 times on each side), and each CSis presented once on each side, for 7 s each.\n",
      "The testing session is video-recorded and the position of the mouse's centroid in the box along the x-axis is extracted (top).\n",
      "\n",
      " The length of the box is normalized to 1.\n",
      "Since the odor is presented randomly on the left or right side of the testing box, 0 is defined as the extremity where the odor is presented.\n",
      "For velocity measurements, a 7 s time window is defined both before and after stimulus onset (bottom).\n",
      "C) Experimental design: cFos-tTA transgenic mice (experimental group, n=8) or wildtype mice (control group, n=8) are injected with AAV-tetO-hM4Di in both hemispheres of piriform cortex.\n",
      "Ten days later, mice are habituated to the training and testing boxes.\n",
      "Upon doxycycline (DOX) removal, mice are fear-conditioned, and hM4Di expression (green circles) is induced in Fos-tagged neurons (filled in blue and yellow).\n",
      "Mice are returned to DOX-containing diet to avoid further expression of hM4Di.\n",
      "Three days later, all mice are i.p. injected with CNO, which silences hM4Diexpressing neurons (black crosses), and memory retrieval is tested.\n",
      "\n",
      " (D to H) Olfactory memory retrieval after CNO injection, assessed by the position of mice in the testing box and their maximum velocity upon odor exposure.\n",
      "Memory author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint retrieval was significantly impaired in cFos-tTA transgenic mice in which piriform neurons that were active during learning were chemogenetically silenced during memory recall (experimental group).\n",
      "\n",
      " (D) Position of mice in the testing box as a function of time after CS+ (black) and CS-(blue) presentation.\n",
      "(E) Position in the testing box 7 s after CS+ or CS-presentation.\n",
      "\n",
      " (F) Position 7 s after CS-presentation subtracted from the position 7 s after CS+ presentation.\n",
      "\n",
      " (G) Maximum velocity during a 7 s time window before and after CS+ presentation.\n",
      "\n",
      " (H) Maximum velocity ratio after/before CS+ (left) or CS-(right) presentation.\n",
      "\n",
      " Data are shown as median and interquartile range, * p<0.05; ** p<0.01; n.s. not significant.\n",
      "\n",
      " author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint (A) (Top) Experimental design: Fos-tagging during olfactory fear conditioning is performed as described in Figure 2 .\n",
      "Three days after fear conditioning, sniffing behavior is quantified in a plethysmograph.\n",
      "Mice are i.p. injected with CNO, habituated to the plethysmograph for 15 min, and each odor is then presented for 8 s with a 2.5 min inter-trial interval. pin: pinene, CS-: beta-citronellol, CS+: ethyl-acetate.\n",
      "\n",
      " (Bottom) Representative sniff recordings from one mouse when exposed to pinene or the CS+ (ethyl-acetate).\n",
      "Data are shown as median and interquartile range, * p<0.05; ** p<0.01; n.s.: not significant author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint Data are shown as median and interquartile range, * p<0.05; ** p<0.01; n.s. not significant.\n",
      "\n",
      " author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint (B) As a measure for exploratory activity, the distance travelled is computed from minute 15 to 18 (corresponding to odor exposure during test 2).\n",
      "For artificial memory recall, the distance travelled is computed over the entire testing session.\n",
      "Ethylacetate exposure and CNO injection lead to a decrease in exploratory behavior in mice previously fear conditioned to ethyl-acetate (n=6 and n=8 respectively).\n",
      "\n",
      " Previous exposure to ethyl-acetate without foot shock does not alter exploratory behavior upon re-exposure to ethyl-acetate or re-activation of Fos-tagged piriform ensembles (n=5 and n=7 respectively).\n",
      "\n",
      " (C) Distance traveled ratio test 2/test 1.\n",
      "author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint \n",
      " Training Testing The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint baseline author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint.\n",
      "SARS-CoV-2, also known as 2019-nCoV, is a novel coronavirus (CoVs) isolated from patients with pneumonia in China 2019.\n",
      "SARS-CoV-2 has a similar incubation period (median, 3.0 days) and a relatively lower fatality rate than SARS-CoV or MERS-CoV (1), but it is estimated that the reproductive number of SARS-CoV-2 is higher than that of SARS-CoV (2) .\n",
      "\n",
      " What's more, some laboratory-confirmed symptomatic cases are absent of apparent cough, fever or radiologic manifestations, making it difficult to timely and accurately find out all infected patients (3, 4) .\n",
      "As of February 28, 2020, patients infected by SARS-CoV-2 have been diagnosed in more than 30 countries, and more than 78,000 confirmed cases and 2,700 deaths associate with SARS-CoV-2 infection are reported in China alone.\n",
      "\n",
      " The genetic information of a virus is essential for not only its classification and traceability, but also its pathogenicity.\n",
      "At the whole genome level, the sequence identify of SARS-CoV-2 was 50% to MERS-CoV, 79% to SARS-CoV, 88% to two bat-derived SARS-like coronaviruses, Bat-SL-CoVZC45 and Bat-SL-CoVZXC21 (collected in 2018 in Zhoushan, China) , and 96% to Bat-SARSr-CoV RaTG13 (collected in 2013 in Yunnan, China) (5) .\n",
      "Each genome of all SARS-CoV-2 strains now submitted online contains nearly 29,900 nucleotides (nt), which are predicted with at least 14 open reading frames (ORFs) (5′ to 3′), such as ORF1ab (P, 21,291 nt), spike (S, 3,822 nt), ORF3a (8,28 nt), envelope (E, 228 nt), membrane (M, 669nt), ORF8 (366 nt), and nucleocapsid (N,1,260 nt) (6) .\n",
      "Among them, the spike gene encoded a glycoprotein that is crucial for determining host tropism and transmission capacity, and highly divergent when compared with that of bat-SARSr-CoV RaTG13 (93.1% nucleotide identity) (6, 7) .\n",
      "\n",
      " Generally, the rates of nucleotide substitution of RNA viruses are faster than their hosts, and .\n",
      "CC-BY-ND 4.0 International license author/funder.\n",
      "It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint this rapid evolution is mainly shaped by natural selection (mostly purifying selection) (8) .\n",
      "Gene mutations such as nucleotide substitutions, deletions and insertions have been frequently reported when comparing SARS-CoV-2 with other viruses (5) (6) (7) .\n",
      "In this work, we investigated the mutation pattern of SARS-CoV-2 by comprehensive comparative genomic analysis of the nonsynonymous/ synonymous substitution, relative synonymous codon usage (RSCU) and selective pressure in order to explore their potential in evolution and function.\n",
      "\n",
      " The SARS-CoV-2 reference genome Wuhan-Hu-1 (NC_045512) and WIV04 (MN996528) was downloaded from the GenBank database.\n",
      "Other newly sequenced 2019-nCoV genomes were downloaded from the Global Initiative on Sharing Avian Influenza Data (GISAID) database (https://www.gisaid.org/).\n",
      "Twenty-one closely related coronavirus complete genome sequences and their coding sequences (CDS) were downloaded from GenBank database (Supplementary   Table S1 ).\n",
      "\n",
      " Genome sequences were aligned using MUSCLE v3.8.31 (9), followed by manual adjustment using BioEdit v7.2.5.\n",
      "Phylogenetic analyses of complete genome were performed using maximum-likelihood method and general time-reversible model of nucleotide substitution with gamma-distributed rates among sites (GTR+G) in RAxML v8.1.21 (10) .\n",
      "Support for the inferred relationships was evaluated by a bootstrap analysis with 1000 replicates and trees were rooted with the alpha-coronavirus lineage as an outgroup.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint\n",
      " The coding sequences were translated and aligned using the MEGA-X program (11) , and then codon-based sequence alignment was used for further analysis.\n",
      "Phylogenetic analyses of coding sequences were performed using MEGA-X software.\n",
      "The changes of amino acids or nucleotides for each CDS sequences were analyzed using in-house Perl script.\n",
      "\n",
      " The number of synonymous substitutions per synonymous site (dS), and the number of non-synonymous substitutions per non-synonymous site (dN), for each coding region were calculated using the Nei-Gojobori method (Jukes-Cantor) in PAML package (12) .\n",
      "\n",
      " The adaptive evolution server (http://www.datamonkey.org/) was used to identify eventual sites of positive selection.\n",
      "For this purpose, the following test has been used: mixed-effects model of evolution (MEME), which allows the distribution of dN/dS (ω) to vary from site to site and from branch to branch at a site (13) .\n",
      "This test allowed us to infer episodic and pervasive positive selection at individual sites.\n",
      "\n",
      " To investigate the potential relative synonymous codon usage (RSCU) bias of the spike protein from SARS-CoV-2 and its closely related coronaviruses, the coding sequence (CDS) of spike protein in these coronaviruses were calculated with CodonW 1.4.4 (http://codonw.sourceforge.net/).\n",
      "The RSCU of human genes was retrieved from the Codon Usage Database (http://www.kazusa.or.jp/codon/).\n",
      "The potential relationships among these sequences were calculated with a squared Euclidean distance ( u h u ).\n",
      "Besides RSCU, the effective number of codons (ENc) was used as a simple metrics to verify codon bias and to explore the source of virus.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      " .\n",
      "CC-BY-ND 4.0 International license author/funder.\n",
      "It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint\n",
      " Statistical analyses were performed using the R statistical package (version 3.\n",
      "\n",
      " Chi-squared test was used to compare any two data sets, and data were considered significantly different if the two-tailed p value was less than 0.05.\n",
      "\n",
      " The newly identified SARS-CoV-2 strain WIV04 genome sequence was closely related with Bat-SARSr-CoV RaTG13 and Bat-SL-CoVZC45, which was collected from host Rhinolophus affinis (5) .\n",
      "Compared with RaTG13 genome, many nucleotide substitutions are observed, but there are only five small inserts and deletions (indels) mutations, and the largest insert segment in WIV04 genome was \" CGGCGGGCACGT \" sequence, which is located near the boundary of S1 and S2 regions of spike protein.\n",
      "Interestingly, only synonymous mutation are observed near this insertion sequence (Fig. 1 , panel C).\n",
      "Compared with Bat-SL-CoVZC45 genome, this insert segment is detected as well, but non-synonymous mutations are also observed around it.\n",
      "Then, we further compared the proportion of synonymous mutations in spike gene between WIV04 and RaTG13 or Bat-SL-CoVZC45.\n",
      "The ratio of nucleotide substitutions (263 NT) to amino acid substitutions (29 AA) was 9.07 from WIV04 to RaTG13, which was significantly higher than the ratio (3.91, 864/221) from WIV04 to Bat-SL-CoVZC45 (p < 0.05) (Fig. 2) .\n",
      "Moreover, we also observed that the synonymous mutations increased in whole genome level between WIV04 and RaTG13 than that between WIV04 and Bat-SL-CoVZC45.\n",
      "\n",
      " Next, the detailed nucleotide changes of each comparison group were analyzed, and we found that the T-to-C (T:C) transition mutation was enriched in the nucleotide substitutions between WIV04 and RaTG13 (Fig. 1B) .\n",
      "It's reported that the CoVs lacking a 3'-to-5' exoribonuclease (ExoN) accumulated 15 to 20 fold more A:G and U:C transition (14) .\n",
      "What's more, the RNA mutagen 5-fluorouracil (5-FU) treatment will also increase the U:C and A:G transitions.\n",
      "Consequently, this increased T:C mutation implies the ExoN of SARS-CoV-2 may be deactivated compared with that of RaTG13.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint Furthermore, we checked whether the ratio of synonymous substitution to missense substitution is increased within SARS-CoV-2 strains.\n",
      "From December 2019 to February 2020, the genome sequences of 108 strains of SARS-CoV-2 virus have been submitted to GISAID database worldwide.\n",
      "Compared with the standard SARS-CoV-2 strain WIV04, total 98 point mutations were detected at 93 nucleotide sites of all SARS-CoV-2 strains with genome sequence available on Feb.\n",
      "25 2020 (Fig. 1A) .\n",
      "However, only 58 of these nucleotide mutations caused changes in amino acids.\n",
      "Among them, 15 nucleotide substitutions at 14 sites caused changes in 7 amino acids of the spike protein were observed.\n",
      "Consequently, the proportion of synonymous mutations (~40%) among all current reported SARS-CoV-2 strains is similar to that between WIV04 and Bat-SL-CoVZC45 (39.1%), but is dramatically less than that between WIV04 and RaTG13 (90.7%).\n",
      "\n",
      " According to random drift hypothesis (15) ， these nucleotide mutations among different SARS-CoV-2 strains now available are still determined by neutral evolution.\n",
      "In short, there has no powerful factor to force SARS-CoV-2 to evolve in a certain direction by far.\n",
      "However, we should take strict precautions against the strong factors that may cause directional variation of SARS-CoV-2 both in natural environment and infection treatment.\n",
      "\n",
      " To investigate whether the mutation pattern between SARS-CoV-2 and RaTG13 is unique across all coronavirus species, we further compared their alterations of nucleotides and amino acids in comparison to other representative coronaviruses.\n",
      "Phylogenetic analysis of SARS-CoV-2 and its 20 closely related coronaviruses formed four well supported clades (Fig. 2) .\n",
      "The two SARS-CoV-2 strain WIV04 (from Wuhan) and SNU01 (from Korea patient) were clustered with SARS-CoV-related strains to form clade 1, belonging to beta-coronavirus lineage B. The Compared with Bat-SARSr-CoV RaTG13, there were 293 nucleotideswere altered while only 29 amino acids were altered in SARS-CoV-2; their ratio was as high as 9.07.\n",
      "However, this ratio detected in the comparisons of other human coronaviruses with their similar animal coronaviruses was less than 5.0.\n",
      "For example, the ratio of human SARS-CoV Tor2 to bat SARS-like CoVs LYRa11 was 4.81; that ratio of human MERS-CoV EMC2012 to bat SARS-like CoVs RSA2011 was 2.80; that ratio of human coronavirus 229E to bat coronavirus HKU2 was 4.55.\n",
      "These results indicate that the relative level of synonymous substitutions between human SARS-CoV-2 and its possible animal origin (RaTG13) is much higher than that between other human coronaviruses and their potential animal sources.\n",
      "\n",
      " Different organisms, even different protein coding genes of the same species, have different frequency of codon usage (16) .\n",
      "The RSCU bias will reveal the difference of the host source.\n",
      "We calculated the distance of RSCU between spike genes of 20 representative coronaviruses and\n",
      " and Bat-SARSr-CoV RaTG13 was 1.17, which was the lowest except for SNU01 (another strain of SARS-CoV-2), indicating that their codon preference was almost the same; the second lowest codon usage difference was 2.41 that was detected between SARS-CoV-2 WIV04 and Bat-SL-CoVZC45 (Fig. 2) .\n",
      "The coronaviruses in same beta-CoV lineage B have a relatively close \n",
      " To infer whether the retention of synonymous mutations is supported or hindered by natural selection, we further studied non-synonymous substitution rate (dN) and synonymous substitution rate (dS) in spike gene (Table 1) .\n",
      "Generally, positive (Darwinian) selection increases, but negative (purifying) selection decreases the ratios of non-synonymous to synonymous substitution rates (dN/dS).\n",
      "The results showed that both dN and dS of S gene of SARS-CoV-2 WIV04 versus Bat-SARr-CoV RaTG13 were the lowest among all typical coronaviruses, while those of SARS-CoV Tor2 versus bat SARS-like coronavirus WIV1 were the second lowest.\n",
      "When the ratio of dN to dS of spike gene is compared, all the tested dN/dS are less than 1, indicating that these non-synonymous mutations are harmful, and negative selection will reduce their retention speed.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint Among them, dN/dS of spike gene of SARS-CoV-2 WIV04 versus that of Bat-SARr-CoV RaTG13 was 0.04, which was the lowest among all comparisons, reconfirming that the rate of synonymous mutation was increased between WIV04 and RaTG13 strains.\n",
      "Moreover, dN/dS rate of polyprotein (ORF1ab) and nucleocapsid (N) genes were similar with that of spike gene (Table   S2 ).\n",
      "\n",
      " The spike (S) protein undergoes several drastic changes during virus infection.\n",
      "For instance, its large parts are cleaved during infection by cellular proteases and expose the receptors to activate viral attachment to the host (17) .\n",
      "As the receptor binding domain (RBD) of spike protein was involved in interacting with human angiotensin-converting enzyme 2 (ACE2) protein, the RBD region was thought to be preferential targets of natural selection (18) .\n",
      "Consistent with this hypothesis, our results showed that both dN and dS of RBD region are increased in comparison with whole spike gene region across all the virus pairs used in this study (Table 1) .\n",
      "Notably, the dN/dS ratio of RBD region in SARS-CoV-2 WIV04 was dramatically increased in about 3 fold in comparison with full spike region.\n",
      "Consequently, these mutations may be subjected to Darwin's choice and will become evidence of adaptive protein evolution.\n",
      "\n",
      " To detect positive selection on the spike gene, the MEME analysis was performed.\n",
      "\n",
      " Significant (p < 0.05) pervasive episodic selection was detected in three sites (48th, 254th, 330th position using the reference sequence of WIV04).\n",
      "On the 254th position of the spike amino acid sequence, there is a histidine residue instead of a phenylalanine residue, whereas on the 330th amino acidic position in WIV04 sequence, there is a glutamine residue instead of an valine residue.\n",
      "\n",
      " The results described above strongly supporting the action of positive selection in spike gene during the recent evolution of SARS-CoV-2 and RaTG13.\n",
      "However, this result should be .\n",
      "CC-BY-ND 4.0 International license author/funder.\n",
      "It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint addressed with careful explain, because the RBD region of spike gene from SARS-CoV-2 was quit divergent with that of RaTG13 (Fig. S1) , suggesting it might have originated from homologous recombination between RaTG13 and one yet-unknown coronavirus (19) .\n",
      "\n",
      " Through comprehensive comparative analysis between SARS-CoV-2 and other coronaviruses, we found the synonymous mutations is dramatically elevated between SARS-CoV-2 and RaTG13 than that of other coronavirus strains, and the nucleotide mutations were enriched in T:C transition.\n",
      "\n",
      " The specific mutation pattern may caused by the loss function of RNA 3'-to-5' exoribonuclease (ExoN).\n",
      "Moreover, as the SARS-CoV-2 was supposed to be originated from Bat-SARSr-CoV RaTG13, the increased synonymous substitution between SARS-CoV-2 and RaTG13 strain suggested the SARS-CoV-2 genome was under stronger negative (purifying) selection.\n",
      "We also detected some sites in spike protein was experiencing positive selection.\n",
      "These observations suggest that adaptive evolution might contribute to its host shifts.\n",
      "However, as the RNA mutagen (e.g. 5-FU) could induce the same mutation pattern, the mechanism of the mutation pattern observed between SARS-CoV-2 and RaTG13 should be further investigated in future work.\n",
      "The number of amino acid changes.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RSCUdist: the RSCU distance of spike protein; ENc: effective number of codons; S ident(%): the percent identity of S protein sequence between different coronaviruses.\n",
      "The NT/AA ratio between WIV04/RaTG13 was statistically significant higher (Chi-squared test, P<0.01) than that of other strain pairs.\n",
      "\n",
      " .\n",
      "CC-BY-ND 4.0 International license author/funder.\n",
      "It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint *The whole genome sequence identity between these paired strains was larger than 95%.\n",
      "\n",
      " .\n",
      "CC-BY-ND 4.0 International license author/funder.\n",
      "It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint.\n",
      "The COVID-19 outbreak began on December 2019 in Wuhan, China (1), and the causative virus, SARS-CoV-2, has rapidly spread into 43 countries as of February 27, 2020.\n",
      "The number of infected patients exceeds 81,000, and the death toll exceeds 2,700.\n",
      "Several drugs, including lopinavir, remdesivir, and chloroquine, have been reported to be presumably effective in treating this disease (2, 3) .\n",
      "\n",
      " Systemic treatment with corticosteroids is contraindicated for the severe pneumonia caused by viruses such as MERS-CoV and SARS-CoV, as steroids suppress the innate immune system, resulting in increased viral replication.\n",
      "In fact, for SARS in 2003 and MERS in 2013, treatment with corticosteroids was associated with increased mortality (4, 5).\n",
      "In the present study, we reconsider the use of inhaled corticosteroids, which have been excluded from the treatment of pneumonia caused by coronavirus.\n",
      "\n",
      " Steroid compounds, which are expected to have dual functions in blocking both coronavirus replication and host inflammation, were screened from a chemical library.\n",
      "The cytopathic effect caused by MERS-CoV infection was measured to evaluate viral replication.\n",
      "Four steroid compounds, ciclesonide, mometasone furoate, mifepristone, and algestone acetophenide conferred a greater than 95% cell survival rate (Fig. S1 ).\n",
      "\n",
      " Next, concentration-dependent viral growth suppression and drug cytotoxicity were assessed.\n",
      "Ciclesonide exhibited low cytotoxicity and potent suppression of viral growth (Fig.   1a ).\n",
      "Cortisone and prednisolone, which are commonly used for systemic steroid treatment, dexamethasone, which has strong immunosuppressant effects, and fluticasone, a commonly used inhaled steroid, did not suppress viral growth (Fig. 1a) .\n",
      "\n",
      " The antiviral effects of steroids against various species of viruses were tested by quantifying propagated virus in culture medium.\n",
      "Ciclesonide and mometasone also suppressed replication of other coronaviruses, HCoV-229E and SARS-CoV, but not replication of RS virus or influenza virus (Fig. S2 ).\n",
      "In addition, ciclesonide slightly but significantly inhibited rubella virus (having positive strand RNA genome) replication (Fig. S2 ), suggesting that ciclesonide interacted with the replication site of positive-strand RNA virus intracellularly.\n",
      "\n",
      " To identify the drug target of virus replication, we conducted 11 consecutive MERS-CoV passages in the presence of 40 μ M ciclesonide or 40 μ M mometasone.\n",
      "A mutant virus that developed resistance ciclesonide was generated (Fig. 1b) , but no resistant virus to author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.11.987016 doi: bioRxiv preprint mometasone was generated.\n",
      "Next-generation sequencing identified that an amino acid substitution at A25V (C19647T in the reference sequence NC_019843.3) in nonstructural protein 15 (NSP15), coronavirus endoribonuclease (6-8), was the predicted mechanism for viral resistance to ciclesonide.\n",
      "A recombinant virus carrying the A25V amino acid substitution (Re-Nsp15-A25V) was generated from the parental MERS-CoV/EMC strain (Re-EMC/MERS) with a BAC reverse genetics system (9), which overcame the antiviral effect of ciclesonide (Fig. 1c) .\n",
      "Interestingly, the mutant virus was inhibited by mometasone, suggesting that the antiviral target of mometasone was different from that of ciclesonide.\n",
      "\n",
      " Next, the effect of ciclesonide on SARS-CoV-2 infection was evaluated.\n",
      "VeroE6/TMPRSS2 cells(10) were infected with authentic SARS-CoV-2 in the presence of steroids or other inhibitors.\n",
      "At 6 h post-infection, cellular RNA was isolated, and real-time PCR was conducted to quantify the amount of viral RNA (11).\n",
      "Ciclesonide and mometasone suppressed viral replication with a similar efficacy to lopinavir (Fig. 2a) .\n",
      "For comparison, viral cell entry inhibitors were tested.\n",
      "E64d substantially reduced viral RNA levels, but nafamostat and camostat had only modest antiviral effects, suggesting that SARS-CoV-2 primarily utilized the cathepsin/endosomal pathway of cell entry rather than the TMPRSS2/cell surface pathway to enter TMPRSS2-expressing cells, consistent with a recent study (12).\n",
      "author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https: //doi.org/10.1101 //doi.org/10.\n",
      "/2020 The copyright holder for this preprint (which was not peer-reviewed) is th . https://doi.org/10.1101/2020.03.11.987016 doi: bioRxiv preprint a RNA copy (log 10 ) SARS-CoV-2 RNA in cell (6 hpi Cell viability (%) author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is th . https://doi.org/10.1101/2020.03.11.987016 doi: bioRxiv preprint.\n",
      "Seventeen years have passed since the outbreak of severe acute respiratory syndrome (SARS) in 2003, but there is yet no approved treatment for infections with the SARS coronavirus (SARS-CoV).\n",
      "1 One of the reasons is that despite the devastating consequences of SARS for the affected patients, the development of an antiviral drug against this virus would not be commercially viable in view of the fact that the virus has been rapidly contained and did not reappear since 2004.\n",
      "As a result, we were empty-handed when the Middle-East respiratory syndrome coronavirus (MERS-CoV), a close relative of SARS-CoV, emerged in 2012.\n",
      "2 MERS is characterized by severe respiratory disease, quite similar to SARS, but in addition frequently causes renal failure 3 .\n",
      "Although the number of registered MERS cases is low (2494 as of November 30, 2019; www.who.int), the threat MERS-CoV poses to global public health may be even more serious than that presented by SARS-CoV.\n",
      "This is related to the high casefatality rate (about 35%, compared to 10% for SARS), and to the fact that MERS cases are still accumulating seven years after the discovery of the virus, whereas the SARS outbreak was essentially contained within 6 months.\n",
      "The potential for human-to-human transmission of MERS-CoV has been impressively demonstrated by the 2015 outbreak in South Korea, where 186 cases could be traced back to a single infected traveller returning from the Middle East.\n",
      "4 SARS-like coronaviruses are still circulating in bats in China, [5] [6] [7] [8] from where they may spill over into the human population; this is probably what caused the current outbreak of atypical pneumonia in Wuhan, which is linked to a seafood and animal market.\n",
      "The RNA genome (Gen-Bank accession code: MN908947.2; http://virological.org/t/initialgenome-release-of-novel-coronavirus/319, last accessed on January 11, 2020) of the new betacoronavirus features around 82% identity to that of SARS-CoV.\n",
      "\n",
      " In spite of the considerable threat posed by SARS-CoV and related viruses, as well as by MERS-CoV, it is obvious that the number of cases so far does not warrant the commercial development of an antiviral drug targeting MERS-and SARS-CoV even if a projected steady growth of the number of MERS cases is taken into account.\n",
      "A possible solution to the problem could be the development of broadspectrum antiviral drugs that are directed against the major viral protease, a target that is shared by all coronavirus genera as well as, in a related form, by members of the large genus Enterovirus in the picornavirus family.\n",
      "Among the members of the genus Alphacoronavirus are the human coronaviruses (HCoV) NL63 (ref. 9) and 229E 10 that usually cause only mild respiratory symptoms in otherwise healthy individuals, but are much more widespread than SARS-CoV or MERS-CoV.\n",
      "Therapeutic intervention against alphacoronaviruses is indicated in cases of accompanying disease such as cystic fibrosis 11 or leukemia, 12 or certain other underlying medical conditions.\n",
      "13 The enteroviruses include pathogens such as EV-D68, the causative agent of the 2014 outbreak of the \"summer flu\" in the US, 14 EV-A71 and Coxsackievirus A16 (CVA16), the etiological agents of Hand, Foot, and Mouth Disease (HFMD), 15 Coxsackievirus B3 (CVB3), which can cause myocardic inflammation, 16 and human rhinoviruses (HRV), notoriously known to lead to the common cold but also capable of causing exacerbations of asthma and COPD.\n",
      "17 Infection with some of these viruses can lead to serious outcome; thus, EV-D68 can cause polio-like disease, 18 and EV-A71 infection can proceed to aseptic meningitis, encephalitis, pulmonary edema, viral myocarditis, and acute flaccid paralysis.\n",
      "15, [19] [20] Enteroviruses cause clinical disease much more frequently than coronaviruses, so that an antiviral drug targeting both virus families should be commercially viable.\n",
      "\n",
      " However, enteroviruses are very different from coronaviruses.\n",
      "While both of them have a single-stranded RNA genome of positive polarity, that of enteroviruses is very small (just 7 -9 kb) whereas coronaviruses feature the largest RNA genome known to date (27 -34 kb) .\n",
      "Enteroviruses are small, naked particles, whereas coronaviruses are much larger and enveloped.\n",
      "Nevertheless, a related feature shared by these two groups of viruses is their type of major protease, 21 which in the enteroviruses is encoded by the 3C region of the genome (hence the protease is designated 3C pro ).\n",
      "In coronaviruses, non-structural protein 5 (Nsp5) is the main protease (M pro ).\n",
      "Similar to the enteroviral 3C pro , it is a cysteine protease in the vast majority of cases and has therefore also been called \"3C-like protease\" (3CL pro ).\n",
      "The first crystal structure of a CoV M pro or 3CL pro (ref. 22) revealed that two of the three domains of the enzyme together resemble the chymotrypsin-like fold of the enteroviral 3C pro , but there is an additional a-helical domain that is involved in the dimerization of the protease (Fig. 1A) .\n",
      "This dimerization is essential for the catalytic activity of the CoV M pro , whereas the enteroviral 3C pro (Fig. 1B) functions as a monomer.\n",
      "Further, the enteroviral 3C pro features a classical Cys...His...Glu/Asp catalytic triad, whereas the CoV M pro only has a Cys...His dyad.\n",
      "22 Yet, there are a number of common features shared between the two types of proteases, in particular their almost absolute requirement for Gln in the P1 position of the substrate and space for only small amino-acid residues such as Gly, Ala, or Ser in the P1' position, encouraging us to explore the coronaviral M pro and the enteroviral 3C pro as a common target for the design of broad-spectrum antiviral compounds.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The fact that there is no known human protease with a specificity for Gln at the cleavage site of the substrate increases the attractivity of this viral target, as there is hope that the inhibitors to be developed will not show toxicity versus the host cell.\n",
      "Indeed, neither the enterovirus 3C pro inhibitor rupintrivir, which was developed as a treatment of the common cold caused by HRV, nor the peptide aldehyde inhibitor of the coronavirus M pro that was recently demonstrated to lead to complete recovery of cats from the normally fatal infection with Feline Infectious Peritonitis Virus (FIPV), showed any toxic effects on humans or cats, respectively.\n",
      "23, 24 Figure 1: Crystal structures of SARS-CoV main protease (M pro , ref. 26 ; PDB entry 2BX4) and Coxsackivirus B3 3C protease (3C pro ; Tan et al., unpublished; PDB entry 3ZYD).\n",
      "Catalytic residues are indicated by spheres (yellow, Cys; blue, His; red: Glu).\n",
      "A. The coronavirus M pro is a homodimer, with each monomer comprising three domains.\n",
      "B. The structure of the monomeric CVB3 3C pro resembles the Nterminal two domains of the SARS-CoV M pro .\n",
      "Structure is on the same scale as image A. C. Superimpostion of residues from the two structures involved in ligand binding.\n",
      "Superimposition was carried out by aligning the catalytic Cys-His pair of each protease.\n",
      "Residues of the SARS-CoV M pro are shown with carbon atoms in cyan, CVB3 3C pro residues have orange carbons and are labeled with an asterisk (*).\n",
      "\n",
      " As the proteases targeted in our study all specifically cleave the peptide bond following a P1-glutamine residue (HCoV-NL63 M pro uniquely also accepts P1 = His at the Nsp13/Nsp14 cleavage site 31 ), we decided to use a 5-membered ring (g-lactam) derivative of glutamine (henceforth called GlnLactam) as the P1 residue in all our aketoamides (see Scheme 1).\n",
      "This moiety has been found to be a good mimic of glutamine and enhance the power of the inhibitors by up to 10-fold, most probably because compared to the flexible glutamine side-chain, the more rigid lactam leads to a reduction of the loss of entropy upon binding to the target protease.\n",
      "29, 32 Our synthetic efforts therefore aimed at optimizing the substituents at the P1', P2, and P3 positions of the a-ketoamides.\n",
      "\n",
      " Synthesis (Scheme 1) started with the dianionic alkylation of N-Boc glutamic acid dimethyl ester with bromoacetonitrile.\n",
      "As expected, this alkylation occurred in a highly stereoselective manner, giving 1 as the exclusive product.\n",
      "In the following step, the cyano group of 1 was subjected to hydrogenation.\n",
      "The in-situ cyclization of the resulting intermediate afforded the lactam 2.\n",
      "The lactam derivative 3 was generated by removal of the protecting group of 2.\n",
      "On the other hand, the amidation of acyl chloride and a-amino acid methyl ester afforded the intermediates 4, which gave rise to the acids 5 via alkaline hydrolysis.\n",
      "The key intermediates 6 were obtained via the condensation of the lactam derivative 3 and the N-capped amino acids 5.\n",
      "The ester group of compounds 6 was then reduced to the corresponding alcohol.\n",
      "Oxidation of the alcohol products 7 by Dess-Martin periodinane generated the aldehydes 8, which followed by nucleophilic addition with isocyanides gave rise to compounds 9 under acidic conditions.\n",
      "Then, the a-hydroxyamides 10 were prepared by removing the acetyl group of compounds 9.\n",
      "In the final step, the oxidation of the exposed alcohol group in compounds 10 generated our target a-ketoamides 11.\n",
      "\n",
      " The inhibitory potencies of candidate a-ketoamides were evaluated against purified recombinant SARS-CoV M pro , HCoV-NL63 M pro , CVB3 3C pro , and EV-A71 3C pro .\n",
      "The most potent compounds were further tested against viral replicons and against SARS-CoV, MERS-CoV, or a whole range of enteroviruses in cell culture-based assays (Tables 1 -3 and Supplementary Table 1 \n",
      " To enable the rapid and biosafe screening of antivirals against corona-and enteroviruses, a non-infectious, but replication-competent SARS-CoV replicon was used 33 along with subgenomic replicons of CVB3 34 and EV-A71 (kind gift of B. Zhang, Wuhan, China).\n",
      "The easily detectable reporter activity (firefly or Renilla luciferase) of these replicons has previously been shown to reflect viral RNA synthesis.\n",
      "[33] [34] [35] In-vitro RNA transcripts of the enteroviral replicons were also used for transfection.\n",
      "For the SARS-CoV replicon containing the CMV promoter, only the plasmid DNA was used for transfection.\n",
      "\n",
      " The initial compound to be designed and synthesized was 11a, which carries a cinnamoyl N-cap in the P3 position, a benzyl group in P2, the glutamine lactam (GlnLactam) in P1, and benzyl in P1' (Table  1) .\n",
      "This compound showed good to mediocre activities against recombinant SARS-CoV M pro (IC50 = 1.95 µM; for all compounds, see Tables 1 -3 for standard deviations), CVB3 3C pro (IC50 = 6.6 µM), and EV-A71 3C pro (IC50 = 1.2 µM), but was surprisingly completely inactive (IC50 > 50 µM) against HCoV-NL63 M pro .\n",
      "These values were mirrored in the SARS-CoV and in the enterovirus replicons (Table 2) .\n",
      "In virus-infected cell cultures, the results obtained were also good to mediocre (Table 3) : SARS-CoV (EC50 = 5.8 µM in Vero E6 cells), MERS-CoV (EC50 = 0.0047 µM in Huh7 cells), HCoV 229E (EC50 = 11.8 µM in Huh7 cells), or a host of enteroviruses (EC50 = 9.8 µM against EV-A71 in RD cells; EC50 = 0.48 µM against EV-D68 in HeLa Rh cells; EC50 = 5.6 µM against HRV2 in HeLa Rh cells).\n",
      "In all cell types tested, the compound generally proved to be non-toxic, with selectivity indices (CC50/EC50) usually >10 (Table 3) .\n",
      "\n",
      " Crystal structures of compound 11a in complex with SARS-CoV M pro , HCoV-NL63 M pro , and CVB3 3C pro demonstrated that the aketo-carbon is covalently linked to the active-site Cys (no. 145, 144, and 147, resp.) of the protease (Fig. 2, 3a-c) .\n",
      "The resulting thiohemiketal is in the R configuration in the SARS-CoV and HCoV-NL63 M pro but in the S configuration in the CVB3 3C pro complex.\n",
      "The reason for this difference is that the oxygen atom of the thiohemiketal accepts a hydrogen bond from the catalytic His40 in the CVB3 protease, rather than from the main-chain amides of the oxyanion hole as in the SARS-CoV and HCoV-NL63 enzymes (Fig. 3a ,b,c insets).\n",
      "It is remarkable that we succeeded in obtaining a crystal structure of compound 11a in complex with the HCoV-NL63 M pro , even though it has no inhibitory effect on the activity of the enzyme (IC50 > 50 µM), (Fig. 2c) .\n",
      "Apparently, the compound is able to bind to this M pro in the absence of peptide substrate, but cannot compete with substrate for the binding site due to low affinity.\n",
      "A similar observation has been made in one of our previous studies, where we were able to determine the crystal structure of a complex between the inactive Michaelacceptor compound SG74 and the EV-D68 3C pro (ref. 29 ; PDB entry 3ZV9).\n",
      "Figure 2 : Fit of compound 11a (pink carbon atoms) to the target proteases (wheat surfaces) as revealed by X-ray crystallography of the complexes.\n",
      "A. Fo -Fc difference density (contoured at 3s) for 11a in the substrate-binding site of the SARS-CoV M pro (transparent surface).\n",
      "Selected side-chains of the protease are shown with green carbon atoms.\n",
      "B. Another view of 11a in the substrate binding site of the SARS-CoV M pro .\n",
      "Note the \"lid\" formed by residue Met49 and its neighbors above the S2 pocket.\n",
      "C. 11a in the substrate-binding site of HCoV-NL63 M pro .\n",
      "Because of the restricted size of the S2 pocket, the P2 benzyl group of the compound cannot enter deeply into this site.\n",
      "Note that the S2 pocket is also covered by a \"lid\" centred around Thr47.\n",
      "D. 11a in the substrate-binding site of the CVB3 3C pro .\n",
      "The S2 site is large and not covered by a \"lid\".\n",
      "\n",
      " The crystal structures indicated that the fits of the P1' benzyl group of 11a in the S1' pocket and of the P3 cinnamoyl cap in the S3 subsite might be improved (see Fig. 3a -c).\n",
      "Compounds 11b -11e and 11g -11l were synthesized in an attempt to do so; however, none of them showed better inhibitory activity against the majority of the recombinant proteases, compared to the parent compound, 11a (see Supplementary Results).\n",
      "To investigate whether the P3 residue of the inhibitor is dispensible, we synthesized compound 11f, which only comprises P2 = Boc, P1 = GlnLactam, and P1' = benzyl.\n",
      "11f was inactive against all purified proteases and in all replicons tested, but showed some activity against HRV2 in HeLa Rh cells (EC50 = 9.0 µM).\n",
      "A crystal structure of 11f bound to HCoV-NL63 M pro demonstrated that the P2-Boc group entered the S2 pocket (Fig. 3d ).\n",
      "In conclusion, although there is probably room for further improvement, we decided to maintain the original design with P1' = benzyl and P3 = cinnamoyl, and focussed on improving the P2 substituen.\n",
      "\n",
      " The crystal structures of SARS-CoV M pro , HCoV-NL63 M pro , and CVB3 3C pro in complex with 11a revealed a fundamental difference between the S2 pockets of the coronavirus proteases and the enterovirus proteases: The cavities are covered by a \"lid\" in the former but are open to one side in the latter (Fig. 2,b-d) .\n",
      "In SARS-CoV M pro , the lid is formed by the 310 helix 46 -51 and in HCoV-NL63 M pro by the loop 43-48.\n",
      "Residues from the lid, in particular Met49 in the case of SARS-CoV M pro , can thus make hydrophobic interactions with the P2 substitutent of the inhibitor, whereas such interaction is missing in the enterovirus 3C pro s.\n",
      "In addition to the lid, the S2 pocket is lined by the \"back-wall\" (main-chain atoms of residues 186 and 188 and Cb atom of Asp187), the side-walls (Gln189, His41), as well as the \"floor\" (Met165) in SARS-CoV M pro .\n",
      "In HCoV-NL63 M pro , the corresponding structural elements are main-chain atoms of residues 187 and 188 as well as the Cb atom of Asp187 (back-wall), Pro189 and His41 (side-walls), and Ile165 (floor).\n",
      "Finally, in CVB3 3C pro , Arg39, Asn69, and Glu71 form the back-wall, residues 127-132 and His40 form the side-walls, and Val162 constitutes the floor.\n",
      "\n",
      " In addition, the S2 pocket is of different size in the various proteases.\n",
      "The SARS-CoV enzyme features the largest S2 pocket, with a volume of 252 Å 3 embraced by the residues (Gln189, His41) defining the side-walls of the pocket in the ligand-free enzyme, as calculated by using Chimera, 36 followed by the CVB3 3C pro S2 pocket with about 180 Å 3 (space between Thr130 and His40).\n",
      "The HCoV-NL63 M pro has by far the smallest S2 pocket of the three enzymes, with a free space of only 45 Å 3 between Pro189 and His41, according to Chimera.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      " In agreement with these observations, a good fit is observed between the P2 benzyl group of 11a and the S2 subsite of the SARS-CoV M pro as well as that of the CVB3 3C pro (Fig. 3a,c) .\n",
      "In contrast, the crystal structure of the complex between 11a and HCoV-NL63 M pro , against which the compound is inactive, demonstrates that the P2-benzyl group cannot fully enter the S2 pocket of the enzyme because of the restricted size of this site (Fig. 3b) .\n",
      "\n",
      " Thus, the properties of our target proteases with respect to the S2 pocket were defined at this point as \"small\" and \"covered by a lid\" for HCoV-NL63 M pro , \"large\" and \"covered\" for SARS-CoV M pro , and \"large\" and \"open\" for CVB3 3C pro .\n",
      "Through comparison with crystal structures of other proteases of the same virus genus (HCoV-229E M pro for alphacoronaviruses 28 (PDB entry 1P9S); HKU4 M pro for betacoronaviruses (Ma, Xiao et al., unpublished; PDB entry 2YNA); and EV-A71 3C pro for enteroviruses 30 (PDB entry 3SGK), we ensured that our conclusions drawn from the template structures were valid for other family members as well.\n",
      "\n",
      " To explore the sensitivity of the S2 pocket towards a polar substituent in the para position of the benzyl group, we synthesized compound 11m carrying a 4-fluorobenzyl group in P2.\n",
      "This substitution abolished almost all activity against the SARS-CoV M pro (IC50 > 50 µM), and the compound proved inactive against HCoV-NL63 M pro as well, whereas IC50 values were 2.3 µM against the EV-A71 3C pro and 8.7 µM against CVB3 3C pro .\n",
      "From this, we concluded that the introduction of the polar fluorine atom is not compatible with the geometry of the S2 pocket of SARS-CoV M pro , whereas the fluorine can accept a hydrogen bond from Arg39 in EV-A71 3C pro (ref. 30 ) and probably also CVB3 3C pro .\n",
      "In SARS-CoV M pro , however, the carbonyl groups of residues 186 and 188 might lead to a repulsion of the fluorinated benzyl group.\n",
      "author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.10.936898 doi: bioRxiv preprint Table 2 : a-ketoamide-induced inhibition of subgenomic RNA synthesis using replicons in a cell-based assay (EC50, μM) 1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60 author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.10.936898 doi: bioRxiv preprint Table 3 .\n",
      "The thiohemiketal is in the R configuration, with its oxygen accepting two hydrogen bonds from the oxyanion-hole amides of Gly143 and Cys145.\n",
      "The amide oxygen accepts an H-bond from His41.\n",
      "The side-chains of Ser144 and Arg188 have been omitted for clarity.\n",
      "B. The P2-benzyl substituent of 11a cannot fully enter the S2 pocket of the HCoV-NL63 M pro , which is much smaller and has less plasticity than the corresponding pocket of SARS-CoV M pro (cf. A).\n",
      "The benzyl therefore binds above the pocket in the view shown here; this is probably the reason for the total inactivity (IC50 > 50 µM) of compound 11a against HCoV-NL63 M pro .\n",
      "The small size of the pocket is due to the replacement of the flexible Gln189 of the SARS-CoV M pro by the more rigid Pro189 in this enzyme.\n",
      "The stereochemistry of the thiohemiketal is R. The sidechains of Ala143 and Gln188 have been omitted for clarity.\n",
      "C. Binding of 11a to the CVB3 3C pro .\n",
      "The stereochemistry of the thiohemiketal is S, as the group accepts a hydrogen bond from His41, whereas the amide keto group accepts three H-bonds from the oxyanion hole (residues 145 -147).\n",
      "The side-chain of Gln146 has been omitted for clarity.\n",
      "D. The crystal structure of 11f in complex with HCoV-NL63 M pro shows that this short (inactive) compound lacking a P3 residue has its P2-Boc group inserted into the S2 pocket of the protease.\n",
      "The stereochemistry of the thiohemiketal is S. The sidechains of Ala143 and Gln188 have been omitted for clarity.\n",
      "E. In contrast to P2 = benzyl in 11a, the isobutyl group of 11n is small and flexible enough to enter into the narrow S2 pocket of the HCoV-NL63 M pro .\n",
      "The thiohemiketal is in the R configuration.\n",
      "The side-chains of Ala143 and Gln188 have been omitted for clarity.\n",
      "F. In spite of its small size, the cyclopropylmethyl side-chain in the P2 position of 11s can tightly bind to the S2 subsite of the SARS-CoV M pro , as this pocket exhibits pronounced plasticity due to the conformational flexibility of Gln189 (see also Fig. 4 ).\n",
      "The stereochemistry of the thiohemiketal is S. The side-chains of Ser144 and Arg188 have been omitted for clarity.\n",
      "\n",
      " As the P2-benzyl group of 11a was apparently too large to fit into the S2 pocket of the HCoV-NL63 M pro , we replaced it by isobutyl in 11n.\n",
      "This resulted in improved activities against SARS-CoV M pro (IC50 = 0.33 µM) and in a very good activity against HCoV-NL63 M pro (IC50 = 1.08 µM, compare with the inactive 11a).\n",
      "For EV-A71 3C pro , however, the activity decreased to IC50 = 13.8 µM, different from CVB3 3C pro , where IC50 was 3.8 µM.\n",
      "Our interpretation of this result is that the smaller P2-isobutyl substituent of 11n can still interact with the \"lid\" (in particular, Met49) of the SARS-CoV M pro S2 site, but is unable to reach the \"back-wall\" of the EV-A71 3C pro pocket and thus, in the absence of a \"lid\", cannot generate sufficient enthalpy of binding.\n",
      "We will see from examples to follow that this trend persists among all inhibitors with a smaller P2 substituent: Even though the SARS-CoV M pro S2 pocket has a larger volume than that of the enterovirus 3C pro , the enzyme can be efficiently inhibited by compounds carrying a small P2 residue that makes hydrophobic interactions with the lid (Met49) and floor (Met165) residues.\n",
      "\n",
      " The EC50 of 11n was >10 µM against the EV-A71 and CVB3 replicons, and even in the SARS-CoV replicon, the activity of 11n was relatively weak (EC50 = 7.0 µM; Table 2 ).\n",
      "In agreement with the replicon data, 11n proved inactive against EV-A71 in RD cells and showed limited activity against HRV2 or HRV14 in HeLa Rh cells (Table 3) .\n",
      "Only the comparatively good activity (EC50 = 4.4 µM) against EV-D68 in HeLa Rh cells was unexpected.\n",
      "The activity of 11n against HCoV 229E in Huh7 cells was good (EC50 = 0.6 µM), and against MERS-CoV in Huh7 cells, it was excellent, with EC50 = 0.0048 µM, while in Vero cells, the EC50 against MERS-CoV was as high as 9.2 µM.\n",
      "Similarly, the EC50 against SARS-CoV in Vero cells was 14.2 µM (Table 3) .\n",
      "\n",
      " We managed to obtain crystals of 11n in complex with the M pro of HCoV NL63 and found the P2 isobutyl group to be well embedded in the S2 pocket (Fig. 3e) .\n",
      "This is not only a consequence of the smaller size of the isobutyl group compared to the benzyl group, but also of its larger conformational flexibility, which allows a better fit to the binding site.\n",
      "\n",
      " When we replaced the P2-isobutyl residue of 11n by n-butyl in 11o, the activities were as follows: IC50 = 8.5 µM for SARS-CoV M pro , totally inactive (IC50 > 50 µM) against HCoV-NL63 M pro , IC50 = 3.2 µM for EV-A71 3C pro , and 5.2 µM for CVB3 3C pro .\n",
      "The decreased activity in case of SARS-CoV M pro and the total inactivity against HCoV-NL63 M pro indicate that the n-butyl chain is too long for the S2 pocket of these proteases, whereas the slight improvement against EV-A71 3C pro and CVB3 3C pro is probably a consequence of the extra space that is available to long and flexible substituents because of the lack of a lid covering the enterovirus 3C pro pocket.\n",
      "\n",
      " As the n-butyl substituent in P2 of 11o was obviously too long, we next synthesized a derivative with the shorter propargyl (ethinylmethyl) as the P2 residue (compound 11p).\n",
      "This led to very mediocre activities against all tested proteases.\n",
      "Using cyclopropyl as the P2 residue (compound 11q), the IC50 values were even higher against most of the proteases tested.\n",
      "Obviously, the P2 side-chain requires a methylene group in the b-position in order to provide the necessary flexibility for the substituent to be embedded in the S2 pocket.\n",
      "\n",
      " Having realized that in addition to size, flexibility of the P2 substituent may be an important factor influencing inhibitory activity, we introduced flexibility into the phenyl ring of 11a by reducing it.\n",
      "The cyclohexylmethyl derivative 11r exhibited IC50 = 0.7 µM against SARS-CoV M pro , 12.3 µM against HCoV-NL63 M pro , 1.7 µM against EV-A71 3C pro , and 0.9 µM against CVB3 3C pro .\n",
      "Thus, the replace- 1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60 author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.10.936898 doi: bioRxiv preprint ment of the phenyl group by the cyclohexyl group led to a significant improvement of the inhibitory activity against the recombinant SARS-CoV M pro and to a dramatic improvement in case of CVB3 3C pro .\n",
      "Even for the HCoV-NL63 M pro , against which 11a was completely inactive, greatly improved albeit still weak activity was observed (Table 1 ).\n",
      "In the viral replicons, 11r performed very well, with EC50 = 0.8 -0.9 µM for the EV-A71 replicon, 0.45 µM for CVB3, and 1.4 µM for SARS-CoV (Table 2 ).\n",
      "In the virus-infected cell culture assays (Table 3) , 11r exhibited EC50 = 3.7 µM against EV-A71 in RD cells and EC50 = 0.48 -0.7 µM against EV-D68, HRV2, and HRV14 in HeLa cells.\n",
      "Against HCoV 229E in Huh7 cells, the EC50 was surprisingly low (1.8 µM).\n",
      "Interestingly, the compound proved extremely potent against MERS-CoV in Huh7 cells, with EC50 = 0.0004 µM (400 picomolar!).\n",
      "Even in Vero cells, EC50 against MERS-CoV was 5 µM, and the EC50 against SARS-CoV in Vero E6 cells was 1.8 -2.1 µM, i.e. the best activity we have seen for an M pro inhibitor against SARS-CoV in this type of cells.\n",
      "The therapeutic index (CC50/EC50) of 11r against EV-D68, HRV2, and HRV14 was >15 in HeLa Rh cells as well as against CVB3 in Huh-T7 cells, but only ~5 for EV-A71 in RD cells.\n",
      "\n",
      " At this point, we decided to systematically vary the size of the ring system in P2.\n",
      "The next substituent to be tried was cyclopropylmethyl (compound 11s, which showed good activities against SARS-CoV M pro (IC50 = 0.24 µM) and HCoV-NL63 M pro (1.4 µM), but poor values against EV-A71 3C pro (IC50 = 18.5 µM) and CVB3 3C pro (IC50 = 4.3 µM) (Table 1) .\n",
      "11s was shown to inhibit the SARS-CoV replicon with an EC50 of about 2 µM, whereas activity against the EV-A71 and CVB3 replicons was poor (EC50 values > 20 µM) ( Table 2 ).\n",
      "The replicon results were mirrored by the antiviral activity of 11s in enterovirus-infected cells (Table 3) , which was weak or very weak.\n",
      "By contrast, the compound inhibited HCoV 229E and MERS-CoV in Huh7 cells with EC50 of 1.3 and 0.08 µM, respectively.\n",
      "The activity against the latter virus in Vero cells was poor (EC50 ~11 µM), and so was the anti-SARS-CoV activity in Vero E6 cells (Table 3) .\n",
      "\n",
      " We next analyzed the crystal structure of the complex between SARS-CoV M pro and compound 11s (Fig. 3f) .\n",
      "The cyclopropylmethyl substituent was found to be incorporated deeply into the S2 pocket, making hydrophobic interactions with Met49 (the lid), Met165 (the floor), and the Cb of Asp187 (the back-wall).\n",
      "In spite of the small size of the P2 substituent, this is possible because the S2 pocket of SARS-CoV M pro is flexible enough to contract and enclose the P2 moiety tightly.\n",
      "This plasticity is expressed in a conformational change of residue Gln189, both in the main chain and in the side-chain.\n",
      "The main-chain conformational change is connected with a flip of the peptide between Gln189 and Thr190.\n",
      "The c1 torsion angle of the Gln189 side-chain changes from roughly antiperiplanar (ap) to (-)synclinal (-sc) (Fig. 4) .\n",
      "The conformational variability of Gln189 has been noted before, both in Molecular Dynamics simulations 26 and in other crystal structures.\n",
      "37 As a consequence of these changes, the sidechain oxygen of Gln189 can accept a 2.54-Å hydrogen bond from the main-chain NH of the P2 residue in the 11s complex (see Fig. 4 ).\n",
      "The affinity of 11s for the S2 pocket of HCoV-NL63 M pro is good because of an almost ideal match of size and not requiring conformational changes, which this enzyme would not be able to undergo because of the replacement of the flexible Gln189 by the more rigid Pro.\n",
      "On the other hand, docking of the same compound into the crystal structure of the CVB3 3C pro revealed that the cyclopropylmethyl moiety was probably unable to generate sufficient Free Energy of binding because of the missing lid and the large size of the S2 pocket in the enterovirus 3C pro , thereby explaining the poor inhibitory activity of 11s against these targets.\n",
      "(Table 1) .\n",
      "Experiments with the viral replicons confirmed this trend, although the EC50 value for SARS-CoV (6.8 µM) was surprisingly high (Table 2 ).\n",
      "In Huh7 cells infected with MERS-CoV, this compound exhibited EC50 = 0.1 µM (but 9.8 µM in Vero cells), whereas EC50 was 7.0 µM against SARS-CoV in Vero E6 cells.\n",
      "The compound was largely inactive against EV-A71 in RD cells and inhibited the replication of the two HRV subtypes tested (in HeLa Rh cells) with EC50 values of ~4 µM.\n",
      "The CC50 of 11t in HeLa cells was 65 µM, i.e. the therapeutic index was well above 15 (Table 3) .\n",
      "\n",
      " Obviously, this substituent was still a bit too small for the enterovirus proteases, so as the next step, we tested P2 = cyclopentylmethyl (compound 11u).\n",
      "This turned out to be the one compound with acceptable IC50 values against all tested enzymes: 1.3 µM against SARS-CoV M pro , 5.4 µM against HCoV-NL63 M pro , 4.7 µM against EV-A71 3C pro , and 1.9 µM against CVB3 3C pro ( Table 1 ).\n",
      "The activity against the replicons was between 3.6 and 4.9 µM ( Table 2 ).\n",
      "In Huh7 cells infected with HCoV 229E or MERS-CoV, 11u showed EC50 = 2.5 or 0.03 µM (11.1 µM for MERS-CoV in Vero cells), while EC50 was 4.9 µM against SARS-CoV in Vero E6 cells (Table 3) .\n",
      "\n",
      " 11u appeared so far the best compromise compound, yet for each of the individual viral enzymes, the following compounds proved superior: P2 = cyclopropylmethyl (compound 11s) for SARS-CoV M pro , P2 = isobutyl (compound 11n) and P2 = cyclopropylmethyl (11s) for HCoV-NL63 M pro , P2 = benzyl (11a) or cyclohexylmethyl (11r) for EV-A71 3C pro , and 11r for CVB3 3C pro .\n",
      "In other words, the nearly equipotent 11u is indeed a compromise.\n",
      "Therefore, in view of the surprisingly good antiviral activity of 11r against HCoV 229E in Huh7 cells, we relaxed the condition that the universal inhibitor should show good activity against the recombinant HCoV-NL63 M pro , and selected 11r (P2 = cyclohexylmethyl) as the lead compound for further development.\n",
      "This compound exhibited submicromolar IC50 values against CVB3 3C pro and SARS-CoV M pro , and IC50 = 1.7 µM against EV-A71 3C pro (Table 1) , as well as similarly low EC50 values in the replicons of these viruses (Table 2 ).\n",
      "In Huh7 cells infected with MERS-CoV, the performance of this compound was excellent, with EC50 = 0.0004 µM, and even against HCoV 229E in Huh7 cells and SARS-CoV in Vero E6 cells, EC50 values of 1.8 and 2.1 µM, respectively, were observed (Table 3) .\n",
      "Also in enterovirus-infected cell culture, the compound performed well, with EC50 values of 0.7 µM or below against HRV2, HRV14, and EV-D68 in HeLa (Rh) cells and selectivity values >15.\n",
      "The only concern is the activity of the compound against EV-A71 in RD cells, for which the EC50 value was 3.7 µM, resulting in too low a therapeutic index.\n",
      "On the other hand, only weak toxicity was detected for 11r in Vero or Huh-T7 cells.\n",
      "Preliminary pharmacokinetics tests with the compound in mice did not indicate a toxicity problem (to be published elsewhere).\n",
      "\n",
      " We describe here the structure-based design, the synthesis, and the assessment of capped dipeptide a-ketoamides that target the main protease of alpha-or betacoronaviruses as well as the 3C protease of enteroviruses.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Through crystallographic analyses of a total of six inhibitor complexes of three different proteases in this study, we found the a-ketoamide warhead (-CO-CO-NH-) to be sterically more versatile than other warheads such as Michael acceptors (-CH=CH-CO-) and aldehydes (-CH=O), because it features two acceptors for hydrogen bonds from the protein, namely the a-keto oxygen and the amide oxygen, whereas the other warheads have only one such acceptor.\n",
      "In the various complexes, the hydroxy group (or oxyanion) of the thiohemiketal that is formed by the nucleophilic attack of the activesite cysteine residue onto the a-keto carbon, can accept one or two hydrogen bonds from the main-chain amides of the oxyanion hole.\n",
      "In addition, the amide oxygen of the inhibitor accepts a hydrogen bond from the catalytic His side-chain.\n",
      "Alternatively, the thiohemiketal can interact with the catalytic His residue and the amide oxygen with the main-chain amides of the oxyanion hole.\n",
      "Depending on the exact interaction, the stereochemistry at the thiohemiketal C atom would be different.\n",
      "We have previously observed a similar difference in case of aldehyde inhibitors, where the single interaction point -the oxyanion of the thiohemiacetal -can accept a hydrogen bond either from the oxyanion hole or from the catalytic His side-chain, 37 resulting in different stereochemistry of the thiohemiacetal carbon.\n",
      "Both aketoamides and aldehydes react reversibly with the catalytic nucleophile of proteases, whereas Michael acceptors form irreversible adducts.\n",
      "\n",
      " In addition to better matching the H-bonding donor/acceptor properties of the catalytic center through offering two hydrogen-bond acceptors instead of one, a-ketoamides have another big advantage over aldehydes and a,b-unsaturated esters (Michael acceptors) in that they allow easy extension of the inhibitors to probe the primed specificity subsites beyond S1', although this has so far rarely been explored (e.g., ref. 38 in case of calpain).\n",
      "\n",
      " The most prominent a-ketoamide drugs are probably telaprivir and boceprivir, peptidomimetic inhibitors of the HCV NS3/4A protease, 39, 40 which have helped revolutionize the treatment of chronic HCV infections.\n",
      "For viral cysteine proteases, a-ketoamides have only occasionally been described as inhibitors and few systematic studies have been carried out.\n",
      "\n",
      " A number of capped dipeptidyl a-ketoamides have been described as inhibitors of the norovirus 3C-like protease.\n",
      "41 These were optimized with respect to their P1' substituent, whereas P2 was isobutyl in most cases and occasionally benzyl.\n",
      "The former displayed IC50 values one order of magnitude lower than the latter, indicating that the S2 pocket of the norovirus 3CL protease is fairly small.\n",
      "Although we did not include the norovirus 3CL pro in our study, expanding the target range of our inhibitors to norovirus is probably a realistic undertaking.\n",
      "While our study was underway, Zeng et al.\n",
      "42 published a series of a-ketoamides as inhibitors of the EV-A71 3C pro .\n",
      "These authors mainly studied the structure-activity relationships of the P1' residue and found small alkyl substituents to be superior to larger ones.\n",
      "Interestingly, they also reported that a six-membered d-lactam in the P1 position led to 2 -3 times higher activities, compared to the fivemembered g-lactam.\n",
      "At the same time, Kim et al.\n",
      "43 described a series of five a-ketoamides with P1' = cyclopropyl that showed submicromolar activity against EV-D68 and two HRV strains.\n",
      "\n",
      " Occasionally, individual a-ketoamides have been reported in the literature as inhibitors of both the enterovirus 3C protease and the coronavirus main protease.\n",
      "A single capped dipeptidyl a-ketoamide, Cbz-Leu-GlnLactam-CO-CO-NH-iPr, was described that inhibited the recombinant transmissible gastroenteritis virus (TGEV) and SARS-CoV M pro s as well as human rhinovirus and poliovirus 3C pro s in the one-digit micromolar range.\n",
      "44 Coded GC-375, this compound showed poor activity in cell culture against EV-A71 though (EC50 = 15.2 µM), probably because P2 was isobutyl.\n",
      "As we have shown here, an isobutyl side-chain in the P2 position of the inhibitors is too small to completely fill the S2 pocket of the EV-A71 3C pro and the CVB3 3C pro .\n",
      "\n",
      " Among a series of aldehydes, Prior et al.\n",
      "45 described the capped tripeptidyl a-ketoamide Cbz-1-naphthylalanine-Leu-GlnLactam-CO-CO-NH-iPr, which showed IC50 values in the 3-digit nanomolar range against HRV 3C pro and SARS-CoV M pro , as well as EC50 values of 0.03 µM against HRV18 and 0.5 µM against HCoV 229E in cell culture.\n",
      "No optimization of this compound was performed and no toxicity data have been reported.\n",
      "\n",
      " For compounds with warheads other than a-ketoamides, in-vitro activity against both corona-and enteroviruses has also occasionally been reported.\n",
      "Lee et al.\n",
      "46 described three peptidyl Michael acceptors that displayed inhibitory activity against the M pro s of SARS-CoV and HCoV 229E as well as against the 3C pro of CVB3.\n",
      "These inhibitors had an IC50 10 -20 times higher for the CVB3 enzyme, compared to SARS-CoV M pro .\n",
      "P2 was invariably isobutyl (leucine) in these compounds, suggesting that further improvement might be possible.\n",
      "\n",
      " In addition to Michael acceptors, peptide aldehydes have also been used to explore the inhibition of coronavirus M pro s as well as enterovirus 3C pro s.\n",
      "Kim et al.\n",
      "44 reported a dipeptidyl aldehyde and its bisulfite adduct, both of which exhibited good inhibitory activities against the isolated 3C proteases of human rhinovirus and poliovirus as well as against the 3C-like proteases of a number of coronaviruses, but antiviral activities in cell culture against EV-A71 were poor (EC50 >10 µM), again most probably due to P2 being isobutyl (leucine).\n",
      "\n",
      " In our series of compounds, we used P1 = GlnLactam (g-lactam) throughout, because this substituent has proven to be an excellent surrogate for glutamine.\n",
      "29, 32 While we made some efforts to optimize the P1' residue of the compounds as well as the N-cap (P3), we mainly focussed on optimization of the P2 substituent.\n",
      "In nearly all studies aiming at discovering peptidomimetic inhibitors of coronavirus M pro s, P2 is invariably isobutyl (leucine), and this residue has also been used in the efforts to design compounds that would inhibit enterovirus 3C pro s as well (see above).\n",
      "From crystal structures of our early lead compound, 11a (cinnamoyl-Phe-GlnLactam-CO-CO-NH-Bz), in complex with the M pro s of HCoV NL63 (as representative of the alphacoronavirus proteases) and SARS-CoV (beta-CoV) as well as the 3C pro of Coxsackievirus B3 (enterovirus proteases), we found that the S2 pocket has fundamentally different shapes in these enzymes.\n",
      "In the SARS-CoV M pro , the S2 subsite is a deep hydrophobic pocket that is truly three-dimensional in shape: the \"walls\" of the groove are formed by the polypeptide main chain around residues 186 -188 as well as by the side-chains of His41 (of the catalytic dyad) and Gln189, whereas the \"floor\" is formed by Met165 and the \"lid\" by residues 45 -51, in particular Met49.\n",
      "The two methionines provide important interaction points for the P2 substituents of inhibitors; while these interactions are mostly hydrophobic in character, we have previously described the surprising observation of the carboxylate of an aspartic residue in P2 that made polar interactions with the sulfur atoms of these methionines.\n",
      "37 Because the pocket offers so many opportunities for interaction and features a pronounced plasticity, P2 substituents such as isobutyl (from Leu), which are too small to fill the pocket entirely, can still generate sufficient binding enthalpy.\n",
      "Accordingly, the S2 pocket of SARS-CoV M pro is the most tolerant among the three enzymes investigated here, in terms of versatility of the P2-substituents accepted.\n",
      "\n",
      " In the S2 pocket of the HCoV-NL63 M pro , Gln189 is replaced by proline and this change is accompanied by a significant loss of flexibility; whereas the side-chain of Gln189 of SARS-CoV M pro is found to accommodate its conformation according to the steric requirements of the P2 substituent, the proline is less flexible, leading to a much smaller space at the entrance to the pocket.\n",
      "As a consequence, a P2benzyl substituent is hindered from penetrating deeply into the pocket, whereas the smaller and more flexible isobutyl group of P2-Leu is not.\n",
      "\n",
      " Finally, in the 3C pro s of EV-A71 and CVB3, the S2 pocket lacks a lid, i.e. it is open to one side.\n",
      "As a consequence, it offers less interaction points for P2 substituents of inhibitors, so that such substituents must reach the \"back-wall\" of the pocket (formed by Arg39, Asn69, Glu71) in order to create sufficient binding energy.\n",
      "Hence, large aromatic substituents such as benzyl are favored by the enterovirus 3C pro s.\n",
      "\n",
      " When we introduced a fluoro substituent in the para position of the P2-benzyl group of our lead compound, 11a, we observed good activity against the enterovirus 3C pro s but complete inactivity against the coronavirus M pro s (see Table 1 , compound 11m).\n",
      "This is easily explained on the basis of the crystal structures: In the enterovirus 3C pro s, the fluorine can accept a hydrogen bond from Arg39 (ref. 30) , whereas in the coronavirus M pro s, there would be electrostatic repulsion from the main-chain carbonyls of residues 186 and 188.\n",
      "In agreement with this, rupintrivir (which has P2 = p-fluorobenzyl) is a good inhibitor of the enteroviral 3C pro s, 46 but not of the coronaviral main proteases, as we predicted earlier.\n",
      "28 In this structure-based inhibitor optimization study, we achieved major improvements over our original lead compound, 11a, by systematically varying the size and the flexibility of the P2-substituent.\n",
      "The compound presenting so far the best compromise between the different requirements of the S2 pockets (SARS-CoV M pro : large and covered, HCoV-NL63 M pro : small and covered, and CVB3 3C pro : large and open) is 11u (P2 = cyclopentylmethyl), which has satisfactory broad-spectrum activity against all proteases tested.\n",
      "However, with regard to its antiviral activities in cell cultures, it is inferior to 11r (P2 = cyclohexylmethyl).\n",
      "The latter compound exhibits very good inhibitory activity against the SARS-CoV M pro as well as the enterovirus 3C pro , and its performance in the SARS-CoV and enterovirus replicons is convincing.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Being in the low micromolar range (EV-A71, CVB3), the data for the antiviral activity in cell culture for 11r correlate well with the inhibitory power of the compound against the recombinant proteases as well as in the replicon-based assays.\n",
      "This is not true, though, for the surprisingly good in-cellulo activity of 11r against HCoV 229E in Huh7 cells.\n",
      "Also, the correlation does not seem to hold for LLC-MK2 and CaCo2 cells.\n",
      "We tested the antiviral activity of many of our compounds against HCoV NL63 in these two cell types and found that all of them had low-or submicromolar EC50 values against this virus in LLC-MK2 cells but were largely inactive in CaCo2 cells (not shown).\n",
      "Furthermore, 11r and all other compounds that we synthesized are inactive (EC50 > 87 µM) against CVB3 in Vero cells (not shown), but exhibit good to excellent activities against the same virus in Huh-T7 cells.\n",
      "We have previously observed similar poor antiviral activities in Vero cells not only for aketoamides, but also for Michael acceptors (Zhu et al., unpublished work) .\n",
      "A similar cell-type dependence is seen for the antiviral activity of 11r against MERS-CoV and SARS-CoV.\n",
      "Whereas the inhibitor exhibits excellent activity against MERS-CoV when Huh7 cells are the host cells (400 pM), the inhibitory activity is weaker by a factor of up to 12,500 when Vero cells are used (EC50 = 5 µM).\n",
      "On the other hand, 11r exhibits excellent anti-MERS-CoV activity in human Calu3 lung cells, i.e. in the primary target cells where the compound will have to act in a therapeutic setting (A. Kupke, personal communication).\n",
      "As we tested antiviral activity against SARS-CoV exclusively in Vero cells, the EC50 values determined for our compounds against this virus are in the one-digit micromolar range or higher; the best is again compound 11r with EC50 = 2.1 µM.\n",
      "Interestingly, the relatively weaker activity (or even inactivity) of our inhibitors against RNA viruses in Vero cells was observed independently in the virology laboratories in Leuven and in Leiden.\n",
      "It is thus unlikely that the lack of activity in Vero cells is related to problems with the experimental set-up.\n",
      "In preliminary experiments, we replaced the P3 cinnamoyl group of 11r by the fluorophor coumaryl and found by fluorescence microscopy that much more inhibitor appeared to accumulate in Huh7 cells compared to Vero cells (D.L., R.H. & Irina Majoul, unpublished).\n",
      "\n",
      " Regardless of which cell system is the most suitable one for the testing of peptidomimetic antiviral compounds, we next plan to test 11r in small-animal models for MERS and for Coxsackievirusinduced pancreatitis.\n",
      "In parallel, we aim to refine the experiments to quantify the accumulation of peptidomimetic protease inhibitors in different host-cell types, in the hope to find an explanation for the observed cell-type dependencies.\n",
      "\n",
      " This work demonstrates the power of structure-based approaches in the design of broad-spectrum antiviral compounds with roughly equipotent activity against coronaviruses and enteroviruses.\n",
      "We observed a good correlation between the inhibitory activity of the designed compounds against the isolated proteases, in viral replicons, and in virus-infected Huh7 cells.\n",
      "One of the compounds (11r) exhibits excellent anti-MERS-CoV activity in virus-infected Huh7 cells.\n",
      "Because of the high similarity between the main proteases of SARS-CoV and the novel BetaCoV/Wuhan/2019, we expect 11r to exhibit good antiviral activity against the new coronavirus as well.\n",
      "\n",
      " Crystallization.\n",
      "The recombinant production and purification of SARS-CoV M pro with authentic N and C termini was described in detail previously.\n",
      "48, 49 Using an Amicon YM10 membrane (EMD Millipore), the purified SARS-CoV M pro was concentrated to 21 mg·mL -1 in buffer A (20 mM Tris-HCl, 150 mM NaCl, 1 mM DTT, 1 mM EDTA, pH 7.5).\n",
      "Crystallization was performed by equilibrating 1 µL protein (mixed with 1 µL precipitant solution) against 500 µl reservoir containing 6 -8% polyethylene glycol (PEG) 6,000, 0.1 M MES (pH 6.0), at 20℃ using the vapor diffusion sitting-drop method.\n",
      "Compounds 11a and 11s were dissolved in 100% DMSO at 50 mM and 200 mM stock concentrations, respectively.\n",
      "A crystal of the free enzyme was soaked in cryo-protectant buffer containing 20% MPD, 6% PEG 6,000, 0.1 M MES, 7.5 mM 11a, pH 6.0, for 2 h at 20℃.\n",
      "Another set of free enzyme crystals was soaked in another cryoprotectant buffer with 6% PEG 6,000, 5% MPD, 0.1 M MES, 15% glycerol, 10 mM 11s, pH 6.0, for 2 h.\n",
      "Subsequently, crystals were fished and flash-cooled in liquid nitrogen prior to data collection.\n",
      "\n",
      " Crystals of HCoV-NL63 M pro with 11a were obtained using cocrystallization.\n",
      "The concentrated HCoV-NL63 M pro (45 mg·mL -1 ) was incubated with 5 mM 11a for 4 h at 20℃, followed by setting up crystallization using the vapor diffusion sitting-drop method at 20℃ with equilibration of 1 µL protein (mixed with 1 µL mother liquor) against 500 µL reservoir composed of 0.1 M lithium sulfate monohydrate, 0.1 M sodium citrate tribasic dihydrate, 25% PEG 1,000, pH 6.0.\n",
      "The crystals were protected by a cryo-buffer containing 0.1 M lithium sulfate monohydrate, 0.1 M sodium citrate tribasic dihydrate, 25% PEG 1,000, 15% glycerol, 2 mM 11a, pH 6.0 and flash-cooled in liquid nitrogen.\n",
      "\n",
      " Crystals of HCoV-NL63 M pro with 11n or 11f were generated by using the soaking method.\n",
      "Several free-enzyme crystals were soaked in cryo-protectant buffer containing 0.1 M lithium sulfate monohydrate, 0.1 M sodium citrate tribasic dihydrate, 25% PEG 1,000, 15% glycerol, 5 mM 11n (or 11f), pH 6.0.\n",
      "Subsequently, the soaked crystals were flash-cooled in liquid nitrogen.\n",
      "\n",
      " Freshly prepared CVB3 3C pro at a concentration of 21.8 mg·mL -1 was incubated with 5 mM 11a pre-dissolved in 100% DMSO at room tempature for 1 h.\n",
      "Some white precipitate appeared in the mixture.\n",
      "Afterwards, the sample was centrifuged at 13,000 x g for 20 min at 4 ℃.\n",
      "The supernatant was subjected to crystallization trials using the following, commercially available kits: Sigma TM (Sigma-Aldrich), Index TM , and PEG Rx TM (Hampton Research).\n",
      "Single rod-like crystals were detected both from the Index TM screen, under the condition of 0.1 M MgCl2 hexahydrate, 0.1 M Bis-Tris, 25% PEG 3,350, pH 5.5, and from the Sigma TM screen at 0.2 M Li2SO4, 0.1 M Tris-HCl, 30% PEG 4,000, pH 8.5.\n",
      "Crystal optimization was performed by using the vapor-diffusion sitting-drop method, with 1 μL CVB3 3C pro -inhibitor complex mixed with 1 μL precipitant solution, and equilibration against 500 μL reservoir containing 0.1 M Tris-HCl, 0.2 M MgCl2, pH 8.5, and PEG 3,350 varied from 22% to 27%.\n",
      "Another optimization screen was also performed against a different reservoir, 0.1 M Tris-HCl, 0.2 M MgCl2, pH range from 7.5 to 8.5, and PEG 4,000 varied from 24% to 34%.\n",
      "Crystals were fished from different drops and protected by cryo-protectant solution consisting of the mother liquor and 10% glycerol.\n",
      "Subsequently, the crystals were flash-cooled with liquid nitrogen.\n",
      "\n",
      " Diffraction data collection, structure elucidation and refinement.\n",
      "Diffraction data from the crystal of the SARS-CoV M pro in complex with 11a were collected at 100 K at synchrotron beamline PXI-X06SA (PSI, Villigen, Switzerland) using a Pilatus 6M detector (DECTRIS).\n",
      "A diffraction data set from the SARS-CoV M pro crystal with compound 11s was collected at 100 K at beamline P11 of PETRA III (DESY, Hamburg, Germany), using the same type of detector.\n",
      "All diffraction data sets of HCoV-NL63 M pro complex structures and of the complex of CVB3 3C pro with 11a were collected at synchrotron beamline BL14.2 of BESSY (Berlin, Germany), using an MX225 CCD detector (Rayonics).\n",
      "All data sets were processed by the program XDSAPP and scaled by SCALA from the CCP4 suite.\n",
      "[50] [51] [52] The structure of SARS-CoV M pro with 11a was determined by molecular replacement with the structure of the complex between SARS-CoV M pro and SG85 (PDB entry 3TNT; Zhu et al., unpublished) as search model, employing the MOLREP program (also from the CCP4 suite).\n",
      "52, 53 The complex structures of HCoV-NL63 M pro with 11a, 11f, and 11n were also determined with MOLREP, using as a search model the structure of the free enzyme determined by us (LZ et al., unpublished) .\n",
      "The complex structure between CVB3 3C pro and 11a was determined based on the search model of the free-enzyme structure (PDB entry 3ZYD; Tan et al., unpublished) .\n",
      "Geometric restraints for the compounds 11a, 11f, 11n, and 11s were generated by using JLIGAND 52, 54 and built into Fo-Fc difference density using the COOT software.\n",
      "55 Refinement of the structures was performed with REFMAC version 5.8.0131 (ref. 52, 56, 57) .\n",
      "\n",
      " A buffer containing 20 mM Tris-HCl, 100 mM NaCl, 1 mM EDTA, 1 mM DTT, pH 7.3, was used for all the enzymatic assays.\n",
      "Two substrates with the cleavage sites of M pro and 3C pro , respectively (indicated by the arrow, ↓), Dabcyl-KTSAVLQ¯SGFRKM-E(Edans)-NH2 and Dabcyl-KEALFQ¯GPPQF-E(Edans)-NH2 (95% purity; Biosyntan), were employed in the fluorescence resonance energy transfer (FRET)based cleavage assay, using a 96-well microtiter plate.\n",
      "The dequenching of the Edans fluorescence due to the cleavage of the substrate as catalyzed by the proteases was monitored at 460 nm with excitation at 360 nm, using a Flx800 fluorescence spectrophotometer (BioTek).\n",
      "Curves of relative fluorescence units (RFU) against substrate concentration were linear for all substrates up to beyond 50 µM, indicating a minimal influence of the inner-filter effect.\n",
      "Stock solutions of the compounds were prepared by dissolving them in 100% DMSO.\n",
      "The UV absorption of 11a was found to be negligible at l = 360 nm, so that no interference with the FRET signal through the inner-filter effect was to be expected.\n",
      "For the determination of the IC50, different proteases at a specified final concentration (0.5 µM SARS-CoV or HCoV-NL63 M pro , 2 µM CVB3 3C pro , 3 µM EV-A71 3C pro ) were separately incubated with the inhibitor at various concentrations (0 to 100 μM) in reaction buffer at 37℃ for 10 min.\n",
      "Afterwards, the reaction was initiated by adding FRET peptide substrate at 20 μM final concentration (final volume: 50 μl).\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The IC50 value was determined by using the GraphPad Prism 6.0 software (GraphPad).\n",
      "Measurements of enzymatic activity were performed in triplicate and are presented as the mean ± standard deviations (SD).\n",
      "\n",
      " Assessment of inhibitory activity of a-ketoamides using viral replicons and virus-infected cells Cells and viruses.\n",
      "Hepatocellular carcinoma cells (Huh7; ref. 58) and their derivative constitutively expressing T7 RNA polymerase (Huh-T7; ref. 59) were grown in Dulbecco's modified minimal essential medium (DMEM) supplemented with 2 mM glutamine, 100 U·mL -1 penicillin, 100 µg·mL -1 streptomycin sulfate, and fetal calf serum (10% in growth medium and 2% in maintenance medium).\n",
      "Huh-T7 cells were additionally supplemented with geneticin (G-418 sulfate, 400 µg·mL -1 ).\n",
      "Huh-T7 cells were used for the enteroviral replicons as well as for infection experiments with CVB3 strain Nancy.\n",
      "\n",
      " For enterovirus (except CVB3) infection experiments, human rhabdomyosarcoma cells (RD; for EV-A71; BRCR strain) and HeLa Rh cells (for EV-D68 and human rhinoviruses) were grown in MEM Rega 3 medium supplemented with 1% sodium bicarbonate, 1% Lglutamine, and fetal calf serum (10% in growth medium and 2% in maintenance medium).\n",
      "For HCoV-229E (a kind gift from Volker Thiel (Bern, Switzerland)), culture and infection experiments were carried out as described.\n",
      "60 For MERS-CoV or SARS-CoV infection experiments, Vero, Vero E6, and Huh7 cells were cultured as described previously.\n",
      "61, 62 Infection of Vero and Huh7 cells with MERS-CoV (strain EMC/2012) and SARS-CoV infection of Vero E6 cells (strain Frankfurt-1) at low multiplicity of infection (MOI) were done as described before.\n",
      "61, 63 All work with live MERS-CoV and SARS-CoV was performed inside biosafety cabinets in biosafety level-3 facilities at Leiden University Medical Center, The Netherlands.\n",
      "Viral replicons.\n",
      "The DNA-launched SARS-CoV replicon harbouring Renilla luciferase as reporter directly downstream of the SARS-CoV replicase polyprotein-coding sequence (pp1a, pp1ab, Urbani strain, acc. AY278741), in the context of a bacterial artificial chromosome (BAC) under the control of the CMV promoter, has been described previously (pBAC-REP-RLuc).\n",
      "33 Apart from the replicase polyprotein, the replicon encodes the following features: the 5'-and 3'-non-translated regions (NTR), a ribozyme (Rz), the bovine growth hormone sequence, and structural protein N. \n",
      " Subgenomic replicons of CVB3 (pT7-CVB3-FLuc 34 ) and EV-A71 (pT7-EV71-RLuc) harbouring T7-controlled complete viral genomes, in which the P1 capsid-coding sequence was replaced by the Firefly (Photinus pyralis) or Renilla (Renilla renifor) luciferase gene, were generously provided by F. van Kuppeveld and B. Zhang, respectively.\n",
      "To prepare CVB3 and EV-A71 replicon RNA transcripts, plasmid DNAs were linearized by digestion with SalI or HindIII (New England Biolabs), respectively.\n",
      "Copy RNA transcripts were synthesized in vitro using linearized DNA templates, T7 RNA polymerase, and the T7 RiboMax™ Large-Scale RNA Production System (Promega) according to the manufacturer's recommendations.\n",
      "\n",
      " Transfection.\n",
      "Huh-T7 cells grown in 12-well plates to a confluency of 80% -90% (2 -3 x 10 5 cells/well) were washed with 1 mL OptiMEM (Invitrogen) and transfected with 0.25 µg of the replication-competent replicon and Lipofectamin2000 or X-tremeGENE9 in 300 µl OptiMEM (final volume) as recommended by the manufacturer (Invitrogen or Roche, respectively).\n",
      "The transfection mixtures were incubated at 37°C for 4 to 5 h (Lipofectamin2000) or overnight (X-tremeGENE9), prior to being replaced with growth medium containing the compound under investigation.\n",
      "For RNA-launched transfection of enteroviral replicons, DMRIE-C was used as transfection reagent according to the manufacturer's recommendations (Invitrogen).\n",
      "All experiments were done in triplicate or quadruplicate and the results are presented as mean values ± SD.\n",
      "\n",
      " Testing for inhibitory activity of candidate compounds.\n",
      "Initially, we performed a quick assessment of the inhibitory activity of the candidate compounds towards the enteroviral and coronaviral replicons at a concentration of 40 µM in Huh-T7 cells.\n",
      "Compounds that were relatively powerful and non-toxic at this concentration, were assayed in a dose-dependent manner to estimate their half-maximal effective concentration (EC50) as well as their cytotoxicity (CC50), as described.\n",
      "29 In brief, different concentrations of a-ketoamides (40 µM in screening experiments or increasing concentration (0, 1.25, 2.5, 5, 10, 20, 40 µM) when determining the EC50) were added to growth medium of replicon-transfected Huh-T7 cells.\n",
      "Twenty-four hours later, the cells were washed with 1 mL phosphate-buffered saline (PBS or OPTIMEM, Invitrogen) and lysed in 0.15 mL Passive lysis buffer (Promega) at room temperature (RT) for 10 min.\n",
      "After freezing (-80 o C) and thawing (RT), the cell debris was removed by centrifugation (16,000 x g, 1 min) and the supernatant (10 or 20 µl) was assayed for Firefly or Renilla luciferase activity (Promega or Biotrend Chemikalien) using an Anthos Lucy-3 luminescence plate reader (Anthos Microsystem).\n",
      "\n",
      " The antiviral activity of the compounds was evaluated in a cytopathic effect (CPE) readout assay using MTS [3-(4,5-dimethylthiazol-2-yl)-5-(3carboxymethoxyphenol)-2-(4-sulfophenyl)-2H-tetrazolium, innersalt]-based assay.\n",
      "Briefly, 24 h prior to infection, cells were seeded in 96-well plates at a density of 2.5 x 10 4 (RD cells) or of 1.7 x 10 4 (HeLa Rh) per well in medium supplemented with 2% FCS.\n",
      "For HRV2 and HRV14 infection, the medium contained 30 mM MgCl2.\n",
      "The next day, serial dilutions of the compounds and virus inoculum were added.\n",
      "The read-out was performed 3 days post infection as follows: The medium was removed and 100 μl of 5% MTS in Phenol Red-free MEM was added to each well.\n",
      "Plates were incubated for 1 h at 37°C, then the optical density at 498 nm (OD498) of each well was measured by microtiter plate reader (Saffire 2 , Tecan).\n",
      "The OD values were converted to percentage of controls and the EC50 was calculated by logarithmic interpolation as the concentration of compound that results in a 50% protective effect against virus-induced CPE.\n",
      "For each condition, cell morphology was also evaluated microscopically.\n",
      "\n",
      " Assays with MERS-CoV and SARS-CoV were performed as previously described.\n",
      "61, 63 In brief, Huh7, Vero, or Vero E6 cells were seeded in 96-well plates at a density of 1 × 10 4 (Huh7 and Vero E6) or 2 × 10 4 cells (Vero) per well.\n",
      "After overnight growth, cells were treated with the indicated compound concentrations or DMSO (solvent control) and infected with an MOI of 0.005 (final volume 150 µl/well in Eagle's minimal essential medium (EMEM) containing 2% FCS, 2 mM L-glutamine, and antibiotics).\n",
      "Huh7 cells were incubated for two days and Vero/VeroE6 cells for three days, and differences in cell viability caused by virus-induced CPE or by compound-specific side effects were analyzed using the CellTiter 96 AQueous Non-Radioactive Cell Proliferation Assay (Promega), according to the manufacturer's instructions.\n",
      "Absorbance at 490 nm (A490) was measured using a Berthold Mithras LB 940 96-well plate reader (Berthold).\n",
      "Cytotoxic effects caused by compound treatment alone were monitored in parallel plates containing mock-infected cells.\n",
      "HCoV-229E-Rev: 5′-ggTCGTTTAGTTGAGAAAAGT -3′, and 229E-ZNA probe: 5'-6-Fam-AGA (pdC)TT(pdU)G(pdU)GT(pdC)TA(pdC)T-ZNA-3-BHQ-1 -3' (Metabion).\n",
      "Standard curves were prepared using serial dilutions of RNA isolated from virus stock.\n",
      "Data were analysed using GraphPad Prism 5.0; EC50 values were calculated based on a 4parameter logistic statistics equation.\n",
      "In parallel to the qPCR assays with inhibitors, cell viability assays were performed using Alamar-Blue™ Cell Viability Reagent (ThermoFisher) according to the manufacturer's instruction.\n",
      "CC50 values were calculated using inhibitor versus normalized response statistics equation by including proper controls (no inhibitor and 1% Triton-X-100-treated cells).\n",
      "\n",
      " Determination of the cell toxicity of candidate compounds.\n",
      "The CellTiter 96Aqueous One Solution Cell Proliferation Assay (MTS test, Promega), the CellTiter Glo assay kit (Promega), the Non-Destructive Cytotoxicity Bio-Assay (ToxiLight (measuring the release of adenylate kinase from damaged cells), Lonza Rockland), or the AlamarBlue™ Cell Viability Reagent (ThermoFisher) were used to determine the cytotoxic effect of compounds towards host cells according to the manufacturers' recommendations.\n",
      "29, 65 Chemical synthesis of a-ketoamides General procedure.\n",
      "Reagents were purchased from commercial sources and used without purification.\n",
      "HSGF 254 (0.15 -0.2 mm thickness) was used for analytical thin-layer chromatography (TLC).\n",
      "All products were characterized by their NMR and MS spectra.\n",
      "1 H NMR spectra were recorded on 300-MHz, 400-MHz, or 500-MHz instruments.\n",
      "Chemical shifts were reported in parts per million (ppm, δ) down-field from tetramethylsilane.\n",
      "Proton coupling patterns were described as singlet (s), doublet (d), triplet (t), quartet (q), multiplet (m), and broad (br).\n",
      "Mass spectra were recorded using a Bruker ESI ion-trap HCT Ultra.\n",
      "HPLC spectra were recorded by LC20A or LC10A (Shimadzu Corporation) with Shim-pack GIST C18 (5 µm, 4.6x150mm) with three solvent systems (methanol/water, methanol/ 0.1% HCOOH in water or methanol/0.1% ammonia in water).\n",
      "Purity was determined by reversed-phase HPLC and was ≥95% for all compounds tested biologically.\n",
      "\n",
      " Synthesis of (2S,4R)-dimethyl 2-(tert-butoxycarbonylamino)-4-(cyanomethyl) pentanedioate (1).\n",
      "\n",
      " To a solution of N-Boc-L-glutamic acid dimethyl ester (6.0 g, 21.8 mmol) in THF (60 mL) was added dropwise a solution of lithium bis(trimethylsilyl)amide (LHMDS) in THF (47 mL, 1 M) at −78°C under nitrogen.\n",
      "The resulting dark mixture was stirred at −78°C. Meanwhile, bromoacetonitrile (1.62 mL, 23.3 mmol) was added dropwise to the dianion solution over a period of 1 h while keeping the temperature below −70°C. The reaction mixture was stirred at −78°C for additional 2 h.\n",
      "After the consumption of the reactant was confirmed by TLC analysis, the reaction was quenched by methanol (3 mL), and acetic acid (3 mL) in precooled THF (20 mL) was added.\n",
      "\n",
      " After stirring for 30 min, the cooling bath was removed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The reaction mixture was allowed to warm up to room temperature and then poured into brine (40 mL).\n",
      "The organic layer was concentrated and purified by flash column chromatography (petroleum ether/ethyl acetate = 4/1) to give product 1 (4.92 g, 72%) as colorless oil.\n",
      "1 H NMR (CDCl3, 400 MHz) δ 5.23 (1H, d, J = 9.0 Hz), 4.43-4.36 (1H, m), 3.77(1H, s), 3.76 (1H, s), 2.89-2.69 (3H, m), 2.20-2.14 (2H, m), 1.45 (9H, s).\n",
      "ESI-MS (m/z): 315 (M + H) + .\n",
      "\n",
      " Synthesis of (S)-methyl 2-(tert-butoxycarbonylamino)-3-((S)-2oxopyrrolidin-3-yl)propanoate (2) .\n",
      "\n",
      " In a hydrogenation flask was placed compound 1 (4.0 g, 12.7 mmol), 5 mL of chloroform and 60 mL of methanol before the addition of PtO2.\n",
      "The resulting mixture was stirred under hydrogen at 20 o C for 12 h.\n",
      "Then the mixture was filtered over Celite to remove the catalyst.\n",
      "NaOAc (6.77 g, 25.5 mmol) was added to the filtrate before the resulting mixture was stirred at 60 o C for 12 h.\n",
      "The reaction was quenched with water (30 mL).\n",
      "The suspension was extracted with ethyl acetate.\n",
      "The organic layers were combined, dried (MgSO4), and filtered.\n",
      "The light-brown filtrate was concentrated and purified by silica gel column chromatography (petroleum ether/ethyl acetate = 4/1) to give the product 2 (2.20 g, 61%) as white solid.\n",
      "1 H NMR (CDCl3) δ 6.02 (1H, br), 5.\n",
      "Synthesis of (S)-methyl 2-amino-3-((S)-2-oxopyrrolidin-3yl)propanoate (3).\n",
      "\n",
      " Compound 2 (1.0 g, 3.5 mmol) was dissolved in 10 mL dichloromethane (DCM), then 10 mL trifluoroacetic acid (TFA) was added.\n",
      "The reaction mixture was stirred at 20 o C for 0.5 h, and concentrated in vacuo to get a colorless oil, which could be used for the following step without purification.\n",
      "\n",
      " ESI-MS (m/z): 187 (M + H) + .\n",
      "\n",
      " Synthesis of methyl N-substituted amino-acid esters 4 General procedure.\n",
      "The methyl amino-acid ester hydrochloride (6.0 mmol) was dissolved in 20 mL CH2Cl2, and then acyl chloride (6.0 mmol), triethylamine (1.69 mL, 12.0 mmol) were added, before the reaction was stirred for 2 h at 20 o C. The reaction mixture was diluted with 20 mL CH2Cl2, washed with 50 mL of saturated brine (2 × 25 mL), and dried over Na2SO4.\n",
      "The solvent was evaporated and the product 4 was obtained as white solid (70-95% yield), which could be used for the next step without further purification.\n",
      "\n",
      " (S)-methyl 2-cinnamamido-3-phenylpropanoate (4a).\n",
      "The methyl L-phenylalaninate hydrochloride (1.30 g, 6.0 mmol) was dissolved in 20 mL CH2Cl2, and then cinnamoyl chloride (1.00 g, 6.0 mmol), triethylamine (1.69 mL, 12.0 mmol) were added, before the reaction was stirred for 2 h at room temperature.\n",
      "The reaction mixture was diluted with 20 mL CH2Cl2, washed with 50 mL of saturated brine (2×25 mL), and dried over Na2SO4.\n",
      "The solvent was evaporated and the product 4a was obtained as white solid (1.75 g, 95%), which could be used for the next step without further purification.\n",
      "\n",
      " Synthesis of N-substituted amino acids 5 (general procedure).\n",
      "1 M NaOH (5 mL) was added to a solution of compound 4 (3.0 mmol) in methanol (5 mL).\n",
      "The reaction was stirred for 20 min at 20 o C. Then 1 M HCl was added to the reaction solution until pH = 1.\n",
      "Then the reaction mixture was extracted with 100 mL of CH2Cl2 (2 × 50 mL) and the organic layer was washed with 50 mL of brine and dried over Na2SO4.\n",
      "The solvent was evaporated and the crude material purified on silica, eluted with mixtures of CH2Cl2/MeOH (20/1) to afford the product 5 (90-96% yield) as a white solid.\n",
      "\n",
      " Synthesis of compounds 6 (General procedure).\n",
      "Compound 5 (2.7 mmol) was dissolved in 10 mL of dry CH2Cl2.\n",
      "To this solution, 1.5 equiv (1.54 g) of 1-[bis(dimethylamino)methylene]-1H-1,2,3triazolo[4,5-b]pyridinium 3-oxide hexafluorophosphate (HATU) was added, and the reaction was stirred for 0.5 h at 20 o C. Then compound 3 (500 mg, 2.7 mmol) and TEA (0.70 mL, 5.42 mmol) was added to the reaction.\n",
      "The reaction was stirred for another 6 h.\n",
      "The reaction mixture was poured into 10 mL water.\n",
      "The aqueous solution was extracted with 50 mL of CH2Cl2 (2 × 25 mL) and washed with 50 mL of saturated brine (2 × 25 mL) and dried over Na2SO4.\n",
      "The solvent was evaporated and the crude material purified on silica, eluted with a mixture of CH2Cl2/MeOH (40/1) to give the product 6 (62-84% yield).\n",
      "\n",
      " Synthesis of alcohols 7 (general procedure).\n",
      "Compound 6 (1.1 mmol) was dissolved in methanol (40 mL), then NaBH4 (0.34 g, 8.8 mmol) was added under ambient conditions.\n",
      "The reaction mixture was stirred at 20 o C for 2 h.\n",
      "Then the reaction was quenched with water (30 mL).\n",
      "The suspension was extracted with ethyl acetate.\n",
      "The organic layers were combined, dried, and filtered.\n",
      "The filtrate was evaporated to dryness and could be used for the next step without further purification (46-85% yield).\n",
      "\n",
      " Synthesis of aldehydes 8 (general procedure).\n",
      "Compound 7 (0.75 mmol) was dissolved in CH2Cl2, then Dess-Martin periodinane (337 mg, 0.79 mmol) and NaHCO3 (66 mg, 0.79 mmol) were added.\n",
      "The resulting mixture was stirred at 20 o C for 1 h.\n",
      "The mixture was concentrated and purified by column chromatography on silica gel (CH2Cl2/MeOH = 20/1) to give the product 8 as white solid (88-95% yield).\n",
      "\n",
      " Synthesis of compounds 9 (general procedure).\n",
      "Compound 8 (0.40 mmol) was dissolved in CH2Cl2, and then acetic acid (0.028 g, 0.47 mmol) and isocyanide (0.43 mmol) were added successively to the solution.\n",
      "The reaction was stirred at 20 o C for 24 h.\n",
      "Then the solvent was evaporated and the crude material purified on silica, eluted with a mixture of CH2Cl2/MeOH (20/1) to give the product 9 (46-84%).\n",
      "\n",
      " Synthesis of α-hydroxyamides 10 (general procedure).\n",
      "1 M NaOH (0.5 mL) was added to a solution of compound 9 (0.164 mmol) in methanol (5 mL).\n",
      "The reaction was stirred at 20 o C for 0.5 h until the consumption of compound 9 was confirmed by TLC analysis.\n",
      "Then, 1 M HCl was added to the reaction solution until pH = 7.\n",
      "Following this, the solvent was evaporated to generate the product 10 as white solid, which could be used directly in the next step.\n",
      "\n",
      " Synthesis of α-ketoamides 11 (general procedure).\n",
      "Compound 10 was dissolved in CH2Cl2, then Dess-Martin periodinane (74 mg, 0.176 mmol) and NaHCO3 (30 mg, 0.176 mmol) were added.\n",
      "The resulting mixture was stirred at 20 o C for 1 h.\n",
      "The mixture was concentrated and purified by column chromatography on silica gel (CH2Cl2/MeOH = 20/1) to give the a-ketoamides 11 as light yellow solid (52-79% in two steps).\n",
      "\n",
      " Detailed results of the variation of the P1' and P3 substituents Synthesis of a-ketoamides 11b -11e and 11g -11l Supplemental Table 1 : inhibitory activities (IC50 (μM)) of α-ketoamides with P1' and P3 modifications against viral proteases Supplemental Table 2 : crystallographic data for complexes between viral proteases and α-ketoamides Molecular formula strings and biological data (CSV)\n",
      " Authors will release the atomic coordinates and experimental data upon article publication.\n",
      "Atomic coordinates include SARS-CoV M pro in complex with compounds 11a (5N19), 11s (5N5O), HCoV-NL63 M pro in complex with 11a (6FV2), 11n (6FV1), 11f (5NH0), and CVB3 3C pro in complex with (5NFS).\n",
      "1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60 author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.10.936898 doi: bioRxiv preprint  1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60 author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.10.936898 doi: bioRxiv preprint.\n",
      "Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) or 2019-Novel Coronavirus (2019-nCoV) is the pathogen of Corona Virus Disease 2019 [1] .\n",
      "Nucleic acid detection of 2019-nCoV is one of the key indicators for clinical diagnosis.\n",
      "However, the false negative rate of 2019-nCoV nucleic acid in clinical practice is very high, and the positive rate is only 30-50% [2] .\n",
      "False negative detection means missed detection, which will lead to not only the delayed diagnosis of suspected patients, but also the 2019-nCoV carriers becoming the potential sources of virus infection.\n",
      "Therefore, it is very urgent to improve the detection rate of 2019-nCoV nucleic acid clinically.\n",
      "\n",
      " Currently, fluorescent quantitative RT-PCR (qRT-PCR) technology is mainly used to detect the nucleic acid of 2019-nCoV.\n",
      "The quality of nucleic acid template is admittedly one of the key factors affecting the detection efficiency.\n",
      "According to \"The Laboratory Technical Guidelines for Detection 2019-nCoV (Fourth Edition)\" issued by National Health and Commission of China [3] and \"The Experts' Consensus on Nucleic Acid Detection of 2019-nCoV\" released by Chinese Society of Laboratory Medicine [4] , the human samples must be placed in 56 o C for at least 45 minutes or higher for shorter time to inactivate the viruses in order to keep the inspectors from virus infection before the nucleic acids were isolated.\n",
      "\n",
      " However, 2019-nCoV is a single stranded RNA virus.\n",
      "Theoretically, the progress of killing the viruses with high temperature will damage the integrity of their RNA genomes in the samples, thus improperly reducing the amount of target viral templates, and eventually leading to high false negative rate of the virus detection.\n",
      "In this study, we employed porcine epidemic diarrhea virus (PEDV), a kind of coronavirus, as a model virus to test the above hypothesis and our results demonstrated that the virus inactivation progress by high temperature damaged the detectable template of the coronavirus seriously.\n",
      "Biological.\n",
      "Company, Ltd.) or 1 ng λDNA (3010, Takara, Japan) were suspended and mixed together using 1.5 mL of sample preservation medium (R503, Vamzye, China) or 1.5 mL Hank's solution.\n",
      "\n",
      " Nucleic acid (DNA/RNA) was extracted using a commercial kit (RC311-C1, Vazyme) according to the manufacturer's protocol.\n",
      "Briefly, 500 µL samples preserved in R503 was first added into a 1.5 ml EP tube containing 200 µL of absolute ethanol and 200 µL samples preserved in Hank's solution was added into a 1.5 ml EP tube containing 500 µL lysis buffer.\n",
      "\n",
      " The mixture was vortexed thoroughly and then transferred into a column placed onto a 2 mL EP collection tube.\n",
      "After centrifuge for 1 min at 12,000×g, the column was placed onto a fresh 2 mL EP collection tube, followed by adding 600 µL washing buffer and then centrifuge for 30 sec at 12,000×g.\n",
      "Washing step was repeated once.\n",
      "The column was placed onto a new 2 mL collection.\n",
      "After centrifuge for 2 min at 12,000×g, the column was then placed onto a 1.5 ml collection tube.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "100 µL and 40 µL elution buffer was added into the center of the membrane of column collected the sample prepared by R503 and Hank's solution respectively and then centrifuged for 1 min at 12,000 × g to collect the nucleic acid.\n",
      "The nucleic acid was stored at -30 ~ -15 o C for usage within short time and at -70 o C for long storage 2.3 Agarose gel electrophoresis 5 µL of the extracted nucleic acids, 1 µL of 10 × DNA loading buffer (P022, Vazyme) and 4 µL of nuclease-free H 2 O were mixed and then vortexed briefly.\n",
      "The mixture was heated in PCR instrument to denature the nucleic acid for 2 min and then subjected to 1.2% agarose gel electrophoresis separation on ice for 7-10 min together with the standard DNA marker (MD103, Vazyme).\n",
      "\n",
      " author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint\n",
      " Quantitative Real-Time PCR (qRT-PCR) was performed to detect the PEDV RNA and λ phage DNA using HiScript II One Step qRT-PCR Probe Kit (Q222-CN, Vazyme) according to the manufacturer's protocol.\n",
      "The reaction (25 µL) was assembled as follows: 12.5 µL of 2 × One Step U+ Mix, 1.25 µL of One Step U+ Enzyme Mix, 0.5 µL of 50 × ROX Reference Dye 1, 0.5 µL of Gene Specific Primer Forward (10 µM), 0.5 µL of Gene Specific Primer Reverse (10 µM), 0.25 µL of TaqMan Probe (10 µM), and 4.5 µL nuclease free H2O.\n",
      "\n",
      " The reaction was carried out on ABI StepOnePlusTM using following conditions: \n",
      " Statistical significance was determined using the paired two-tailed t test.\n",
      "A value of p < 0.05 was considered statistically significant, and p < 0.01 was considered statistically very significant.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint 6 / 16 into five groups.\n",
      "Three groups were incubated at 56 o C for 30, 45 and 60 minutes, respectively.\n",
      "The fourth was incubated at 92 o C for 5 minutes, and the fifth group was stored at 4 o C as no inactivation control.\n",
      "After the high-temperature inactivation was completed, nucleic acids (including DNA and RNA) were extracted by conventional methods, and subjected to run 1.2% agarose gel electrophoresis.\n",
      "\n",
      " To understand the effect of the virus inactivation progress with the temperature higher than 56 o C on the integrity of viral nucleic acid, we quantitated the amount of Table S1 ).\n",
      "The results suggest that only 50.11% of the detectable viral templates left after the inactivation progress of incubation at 56 o C for 30 minutes and only 3.36% of the detectable viral templates left after the inactivation progress of incubation at 92 o C for 5 minutes.\n",
      "\n",
      " To investigate whether any solutions are able to protect viral RNA integrity from author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint 7 / 16 disrupting during the inactivation progress with high temperature, R503 (Vazyme, China) was used to prepare the samples instead of Hank's solution.\n",
      "We then performed the same viral inactivation experiments as using Hank's solution.\n",
      "\n",
      " Compared with the control group that the sample was stored at 4 o C, both DNA and RNA in the samples incubated at 56 o C for 30 -60 minutes also displayed obvious degradation, among which the 28 S and 18 S bands of total RNA of human cells were obviously smeared, and the large bands of genomic DNA became weaker while there were almost no visible genomic DNA and 28 S RNA bands in the sample incubated at 92 o C for 5 minutes (Fig 1D) .\n",
      "However, the degradation of nucleic acid during the inactivation progress was not as serious as that of the samples prepared by Hank's solution.\n",
      "The results suggest that R503 should have a significant protective effect of both total RNA and genomic DNA from degradation.\n",
      "\n",
      " To determine the effect of the inactivation progress with temperature higher than 56 o C on the integrity of viral nucleic acid in the samples prepared with R503, we examined the amount of porcine PEDV coronavirus by qRT-PCR as above.\n",
      "\n",
      " Compared with the CT (31.64 ± 0.10) of the control group that the sample was stored at 4 o C, the mean value of CT (31.31 ± 0.17) of the group incubated at 56 o C for 30 minutes had no significant change.\n",
      "The difference of CTs between the two groups was less than 0.5 ( Fig. 1E ; Figure S1B ; Table S1 ).\n",
      "Moreover, the mean value of CT (31.43 ± 0.08) in the group incubated at 92 o C for 5 minutes was also not changed significantly.\n",
      "The difference of CTs between the two groups was less than 0.5 either (Fig. 1F; Fig. S1B ; Table S1 ).\n",
      "These results showed that the inactivation progress with the temperature higher than 56 o C did not significantly affect the detectable template of viral nucleic acid when it was prepared with R503, suggesting that the fragmented viral RNA could still be good for using as a template for qRT-PCR even if the viral nucleic acid was degraded.\n",
      "\n",
      " In order to understand the advantage of R503 over common isotonic solution in author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint\n",
      " protecting the detectable amount of coronavirus RNA from disrupting, we first compared the difference of the detectable amount of coronavirus RNA between the two samples that are prepared with R503 and Hank's solution and stored at 4 o C. The results showed that the detectable amount of the viral RNA in the two prepared solutions was similar (Fig. 1G, Table S1 ), and the difference between the mean values of their CT was less than 0.5 (31.64 ± 0.10 vs 31.95 ± 0.10).\n",
      "The results suggest that there is no significant difference in the amount of detectable viral RNA templates between the samples prepared by either R503 or Hank's solutions that were stored at Table S1 ).\n",
      "The results suggest that the detectable templates of viruses in the samples prepared by Hank's solution was only 32.30% of R503 solution after the samples were inactivated at 56 o C for 30 minutes to kill the virus.\n",
      "Furthermore, after inactivation at 92 o C for 5 minutes, the CT of detected PEDV virus in the two preservation solutions were 31.43 ± 0.08 (R503) and 36.84 ± 0.37 (Hank's), respectively.\n",
      "The mean value of CT of detected virus in the samples prepared by Hank's solution was 5.4125 higher than that prepared by R503 (P < 0.0001; Figure 1I , Table S1 ).\n",
      "The results reveal the detectable amount of virus in the samples prepared by Hank's solution was only 2.35% of R503 solution after the samples were inactivated at 92 o C for 5 minutes to kill the virus.\n",
      "\n",
      " In addition to the disruptive effect of the virus inactivation with high temperature on the detection of RNA virus, the detectable amount of the viral DNA (λ phage) in the samples was also significantly affected.\n",
      "qPCR results (Fig. S1C, Table S1) showed that mean value of CT (23.55 ± 0.20) of detectable virus in the samples author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint 9 / 16 incubated at 56 o C for 30 minutes was increased by 0.5230 (P < 0.01) compared with the CT (23.02 ± 0.15) of the samples stored at 4 o C. The mean value of CT (25.58 ± 0.21) was increased by 2.5572 (P < 0.0001) in the sample incubated at 92 o C for 5 minutes.\n",
      "\n",
      " Furthermore, we examined the amount of phage DNA in the sample prepared with R503 after high temperature treatment by qPCR.\n",
      "The results (Fig. S1D, Table S1) showed that the mean value of CT (21.71 ± 0.04) in the sample prepared by R503 and incubated at 56 o C for 30 minutes had no significant change compared with that in the control group (21.61 ± 0.06) (the difference between the two was less than 0.5).\n",
      "The mean value of CT (22.90 ± 0.26) was increased by 1.2880 in the sample that was inactivated at 92 o C for 5 minutes, suggesting that 92 o C treatment significantly reduced the detectable amount of DNA viruses in sample even prepared by R503.\n",
      "\n",
      " In conclusion, our results show that virus inactivation with temperature higher than 56 o C result in the degradation of viral nucleic acids seriously, which will lead to the artificial shortage of detectable templates of viral nucleic acids in samples, and finally lead to false negative of clinical detection in some samples.\n",
      "Preserved with the optimized solution such as R503, the detectable templates of viral nucleic acids can be kept unchanged after the samples were incubated at 56 o C or higher for killing the viruses although the integrity of cellular nucleic acids were obviously disrupted.\n",
      "The results suggest that if the samples were prepared with an optimal solution, the detectable numbers of viral templates would be unchanged because virus inactivation progress with high temperature does not erase all the RNA but leave small fragments of nucleic acids that are good enough for using as the template for qRT-PCR detection.\n",
      "\n",
      " If it is true for 2019-nCoV, preparing the human samples with the optimal solution can not only keep the clinical inspectors from virus infection by inactivating the virus using 56 o C or higher as normal, but also improve the detection sensitivity of viral nucleic acid in the sample, leading to avoiding or at least reducing false negative author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint 10 / 16 results of detection.\n",
      "\n",
      " Recently, Chen reported that the inactivation by incubating the sample with 56 o C for 30 min to kill the virus had no significant effect on the detection of 2019-nCoV by qRT-PCR [5] .\n",
      "However, combining the limitations such as only two samples tested and higher amount of virus in the two samples as the authors discussed in their researches, and our experimental evidences that the virus inactivation by high temperature did reduce the detectable amount of the porcine coronavirus template seriously, it is highly recommended to carry out systematic investigation on the impact of high temperature inactivation on the integrity of 2019-nCoV nucleic acids and develop a sample preservation solution to protect the detectable templates of 2019-nCoV nucleic acids from high temperature inactivation damage.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Table S1 .\n",
      "Figure S1 author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint 15 / 16 The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint.\n",
      "HCoV-HKU1 replication in the presence of both inhibitors (Fig. 2B) .\n",
      "All inhibitors were used 151 at non-toxic concentrations (Fig. 2C) .\n",
      "152 The experiments conducted so far suggested that KLK13 is required for virus infection.\n",
      "153 However, one may question the specificity of the KLK13 protease inhibitors.\n",
      "To ensure that 154 KLK13 is indeed the priming enzyme during HCoV-HKU1 infection, we developed HAE 155 cultures by transforming cells with lentiviral vectors encoding shRNAs targeting KLK13 156 mRNA.\n",
      "We then confirmed that the expression of the protease was silenced (HAE_shKLK13).\n",
      "(Fig. 3A) .\n",
      "Importantly, HAE_shKLK13 cells continued to differentiate and formed 162 pseudostratified cultures (Fig. 3B) .\n",
      "Next, we infected HAE_ctrl, HAE_GFP, HAE_vector, and 163 HAE_shKLK13 with HCoV-HKU1 (10 6 RNA copies per ml) and incubated them for 2 h at 164 32°C with the viral stock solution.\n",
      "Cultures were maintained at 32°C for 5 days at an air-liquid 165 interface.\n",
      "Apical washes were collected, and virus yield was determined by RT-qPCR.\n",
      "We 166 found that, in contrast to that in control cultures, replication of virus in HAE_shKLK13 was 167 abolished (Fig. 3C) .\n",
      "Overall, these data suggest that silencing the KLK13 gene in HAE inhibits 168 virus infection, indicating that KLK13 is necessary for HCoV-HKU1 infection.\n",
      "171 We determined that KLK13 is essential for efficient HCoV-HKU1 infection in HAE The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint proteases my render RD cells permissive, we generated RD cells expressing human KLK13 or 176 TMPRSS2 proteases.\n",
      "RD cells were transduced with lentiviral vectors harboring the KLK13 177 gene (RD_KLK13), control vector (RD_ctrl), or TMPRSS2 (RD_TMPRSS2).\n",
      "Due to the lack 178 of KLK13 specific antibodies, we verified its presence based on RT-PCR (Fig. 4A) .\n",
      "The 179 presence of TMPRSS2 in RD_TMPRSS2 cells was confirmed using Western blot.\n",
      "The 180 TMPRSS2 band in RD cells was observed at 25 kDa, which corresponds to one of the naturally 181 occurring splicing variants (Fig. 4B) .\n",
      "Subsequently, we transduced RD_ctrl, RD_KLK13 and pseudoviruses, compared to ΔEnv, which was completely abolished in the presence of KLK13 193 inhibitor (Fig 4D) .\n",
      "Overall, these data demonstrated that KLK13 activity drives HCoV-HKU1 194 entry into cells.\n",
      "195 196 KLK13 enables the replication of HCoV-HKU1 in RD cells 197 Obtained results showed that KLK13 expression on RD cells was sufficient for HCoV-198 HKU1 pseudovirus entry.\n",
      "Here, we aimed to test whether KLK13 presence renders RD cells 199 permissive for HCoV-HKU1 infection.\n",
      "For this, we infected RD_ctrl and RD_KLK13 cells 200 author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint with HCoV-HKU1 (10 8 RNA copies per ml) and incubated the culture for 7 days at 32°C in the 201 presence or absence of a KLK13 inhibitor (10 µM) or DMSO.\n",
      "Next, cellular RNA was isolated 202 and the presence of HCoV-HKU1 N subgenomic mRNA (N sg mRNA), which is considered 203 to be a hallmark of coronaviral infection, was assessed (14).\n",
      "sg mRNA appeared in RD_KLK13 204 cells, while no signal was detected in cultures supplemented with the KLK13 inhibitor nor in 205 RD_ctrl cells (Fig. 5A) .\n",
      "206 To further confirm the role of KLK13 during HCoV-HKU1 infection RD HCoV-HKU1 infection was analyzed by means of N sg mRNA detection.\n",
      "Again, we found that 212 N sg mRNA was produced only in the presence of KLK13 (Fig. 5B) .\n",
      "Further, we passaged in the presence of KLK13 (Fig. 5B) .\n",
      "However, we observed no cytopathic effects (CPEs), and 218 replication levels were very low (no significant increase over control levels on RT-qPCR; data 219 not shown).\n",
      "To further test the effect of KLK13 on replication of HCoV-HKU1 in RD cells, the 220 virus stock was incubated with purified KLK13 (200 nM) and incubated in the presence or 221 absence of a KLK13 inhibitor (10 µM) or DMSO.\n",
      "After 2 h at 32°C pre-treated virus stock was 222 diluted in media as described above and overlaid on RD cells.\n",
      "After 7 days at 32°C we evaluated 223 the presence of the HCoV-HKU1 N sg mRNA.\n",
      "The virus replicated only after treatment with 224 KLK13, and supplementation with the inhibitor blocked this effect infection (Fig. 5C) .\n",
      "225 author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint KLK13 primes the HCoV-HKU1 S protein 227 Expression of KLK13 by cells previously resistant to HCoV-HKU1 renders them 228 susceptible; therefore, we asked whether this is due to proteolytic activation of the S protein.\n",
      "229 We tested this using the CleavEx method, in which a peptide of interest is exposed in the KLK13.\n",
      "Subsequently, proteins were resolved by SDS-PAGE and detected by western blotting 238 with antibodies specific for His-tagged proteins.\n",
      "The analysis showed that, in the presence of 239 500 nM KLK13, the CleavEx protein harboring the S1/S2 cleavage site was degraded; however, 240 the CleavEx protein harboring the S2/S2' site remained intact.\n",
      "Because KLKs are produced as 241 pro-forms that undergo self-activation, an additional band of His-tagged purified pro-KLK13 242 (HisTag-pro-KLK13) was observed after treatment with 500 nM KLK13 (Fig. 6A) .\n",
      "The product 243 of the S1/S2 cleavage was further sequenced by N-terminal Edman degradation showing the 244 following sequence: R↓SISA, which corresponds to the S1/S2 site.\n",
      "This result shows clearly 245 that the S1/S2 region of the HCoV-HKU1 S protein is prone to KLK13-mediated cleavage.\n",
      "246 Furthermore, we aimed to confirm the cleavage using a full-length Spike protein of HCoV-HKU1 (HKU1-S).\n",
      "For this, we expressed the HKU1-S in 293T cells, purified the protein 248 using 6 × His tag, and incubated for 3 h at 37°C with increasing concentrations of purified 249 KLK13.\n",
      "Subsequently, HKU-S or mock proteins were resolved by SDS-PAGE and detected by 250 author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint western blotting with antibodies specific to the tag.\n",
      "The analysis showed that in the presence of 251 1 µM KLK13 the HKU1-S was degraded ( Figure 6B ).\n",
      "The S protein was observed at 252 ~150 kDa, which is consistent with the migration speed reported for these highly glycosylated 253 proteins (1).\n",
      "254 255 author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint\n",
      " Receptor recognition is the first, essential step of the virus infection process.\n",
      "The The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint concentrations(63).\n",
      "Broad spectrum protease inhibitors are now used widely for virus research, 281 although their non-specific activity makes the results equivocal.\n",
      "For example, Matsuyama et al.\n",
      "282 showed recently that the furin inhibitor dec-RVKR-CMK interferes with the activity of several 283 proteases, and that its previously described inhibitory activity during MERS-CoV infection is 284 not specific to furin; instead, its activity is due to non-specific inhibition of cathepsin L and 285 TMPRSS2 (81).\n",
      "We tried to use specific KLK inhibitors developed in our laboratory (61).\n",
      "\n",
      " Considering the small arsenal of tools available to researchers studying KLKs, only three 287 compounds were readily available.\n",
      "Treating HAE cultures with these inhibitors revealed that 288 only compounds designed to inhibit KLK13 hampered HCoV-HKU1 replication.\n",
      "However, the 289 great similarity between different KLKs makes one doubt the specificity of these inhibitors, 290 despite their performance in biochemical assays.\n",
      "Therefore, we decided to silence KLK13 in 291 HAE cultures.\n",
      "This abolished virus replication ex vivo, thereby confirming the importance of 292 KLK13 during infection.\n",
      "KLK13 is thought to be secreted and membrane-bound(82, 83).\n",
      "293 This study showed that HCoV-HKU1 infection in HAE modulates the expression of 294 different KLKs, including KLK13.\n",
      "In our study, KLKs expression was tested using semi-295 quantitative PCR, and for that reason, we were unable to show the level of KLKs modulation 296 after HCoV-HKU1.\n",
      "However, the pattern of virus-induced expression of several KLKs could 297 be observed.\n",
      "KLK mechanism of activation is a complex process and until now it has only 298 been proven that most KLK genes are regulated by steroids and other hormones (84).\n",
      "It is also 299 important to remember, that KLK expression is regulated in a similar manner, and the induction 300 of a single gene usually results in overexpression of the whole cluster (85, 86).\n",
      "While one may 301 assume that the virus stimulates KLK13 production to promote the infection, this up-regulation 302 is likely a natural response of the damaged tissue, as KLKs were previously reported to take 303 part also in tissue regeneration (87-89).\n",
      "Further, increased expression of KLKs may be the 304 author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint response to the inflammatory process, as Seliga et al demonstrated that KLK-kinin system is a 305 potent modulator of innate immune responses (90).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " The experiments performed herein show the importance of KLK13 for virus entry into 307 susceptible cells; therefore, we speculated that scattered distribution of different KLKs in 308 different tissues may be one of the determinants of the HCoV-HKU1 tropism (53, 91).\n",
      "We 309 tested the purified enzyme expressed in the eukaryotic cells; however, we also developed a cell 310 line constitutively expressing the enzyme.\n",
      "As an in vitro model for our studies, we used RD 311 cells previously reported to carry attachment receptors for the virus (26).\n",
      "Here, using 312 pseudoviruses decorated with S-HKU1 proteins we showed that KLK13 presence on RD cells 313 is sufficient for virus entry and renders these cells permissive.\n",
      "Our experiments also showed 314 that in contrast to previous reports, TMPRSS2 is not involved in this process(51).\n",
      "Furthermore, 315 we observed that RD cells supported the replication of the virus in the presence of KLK13 and 316 that this effect was reversed in the presence of the specific KLK13 inhibitor.\n",
      "We were, however, 317 not able to culture the virus to high yields.\n",
      "HCoV-HKU1 replication in KLK13-expressing RD 318 cells remained inefficient and RTqPCR assessment did not reveal significant increases in the 319 amounts of viral RNA.\n",
      "For that reason, we are only able to detect viral sg mRNAs, which are 320 considered to be the hallmark of coronaviral replication.We believe that this may be due to 321 non-optimal infection conditions, which may include inappropriate KLK13 concentrations or 322 low density of the entry receptor.\n",
      "Also, it is possible that RD cells may not support efficient 323 replication of the virus due to factors unrelated to the entry process.\n",
      "Nonetheless, our results\n",
      " show that the HCoV-HKU1 entry receptor is present on RD cells, and we were able to trigger 325 virus entry and replication; these findings warrant further exploration.\n",
      "\n",
      " Most coronaviral S proteins are processed into S1 and S2 subunits by host proteases, The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint HCoV-HKU1 S protein has two regions that are prone to proteolytic activation: the S1/S2 furin 330 cleavage site and a secondary cleavage site termed S2', which is adjacent to a potential fusion 331 peptide (38).\n",
      "While the S1/S2 site is believed to be processed by furin during protein 332 biosynthesis, the S2/S2' site is expected to be cleaved during virus entry.\n",
      "As we already knew being tested.\n",
      "We found that the S1/S2 site was efficiently cleaved by KLK13, whereas the 338 S2/S2' region remained intact.\n",
      "As CleavEx technique is a convenient surrogate system allowing 339 for precise mapping of the cleavage site, it has some limitations.\n",
      "To ensure the reliability of 340 results, purified full-length HCoV-HKU1 S protein was subjected to the proteolytic cleavage.\n",
      "\n",
      " Also here we observed efficient cleavage of the HCoV-HKU1 S protein by KLK13.\n",
      "\n",
      " While the results presented here show that KLK13 is able to process the HCoV-HKU1 The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint to cross the species barrier and cause severe diseases in humans, further research on the role of 355 different proteases in coronaviral infections is necessary.\n",
      "356 author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint\n",
      " Plasmid constructs 358 KLK13 and TMPRSS2 genes were amplified by PCR using cDNA obtained from HAE 359 cells.\n",
      "Each PCR product was cloned into pWPI plasmid for lentivirus production and sequence Table 2 .\n",
      "(Table 3 ; 500 nM each).\n",
      "β-actin was used as a household gene reference.\n",
      "\n",
      " The reaction was carried out according to the scheme: 5 min at 95°C, followed by 35 cycles of 465 30 s at 95°C, 20 s at 59°C and 20 s at 72°C, followed by 10 min at 72°C. containing proKLK13 or proKLK14 were concentrated and the buffer was changed to 50mM 528 Tris pH 7.5, 150 mM NaCl.\n",
      "After purification and self-activation at 37°C for 24 h, activity of 529 author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint proteases was assessed by serine protease inhibitor Kazal-type 6 (SPINK6) titration, as 530 described previously (63).\n",
      "Protein expression was performed in E. coli BL21 and was induced by the addition of 553 0.5 mM IPTG to the bacterial culture (OD600 0.5-0.6), followed by shaking for 3h at 37°C. 554 author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint Then, the bacteria were spun down and the pellet was suspended in buffer A (10 mM sodium 555 phosphate, 500 mM NaCl and 5 mM imidazole, pH 7.4).\n",
      "The pellet suspension was then 556 sonicated and spun down.\n",
      "Soluble proteins were purified using HisTrap TM Excel (GE The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint.\n",
      "4 hepatoma cells was largely specific to CoV.\n",
      "Among 10 genetically divergent viruses tested, only hepatitis C virus was also restricted (Extended Data Fig. 1f ).\n",
      "\n",
      " LY6E is a member of the LY6/uPAR family of GPI-anchored proteins, and is implicated in diverse cellular processes 14 .\n",
      "To evaluate the specificity of LY6E for inhibiting CoV, we ectopically expressed select LY6/uPAR proteins 9 and infected with HCoV-229E.\n",
      "Only LY6E inhibited CoV infection (Extended Data Fig. 1g) .\n",
      "We previously showed that a conserved residue (L36) is required for LY6Emediated enhancement of influenza A virus 9 .\n",
      "L36 was also required to restrict HCoV-229E infection (Extended Data Fig. 1h) .\n",
      "These data suggest a conserved functional domain that is responsible for both the virus-enhancing and -restricting phenotypes.\n",
      "\n",
      " Next, we investigated the effect of LY6E on specific steps of the virus replication cycle.\n",
      "First, we tested whether LY6E restricts HCoV-229E attachment to the cell surface and observed no effect (Extended Data Fig. 2a) .\n",
      "In accordance, LY6E had no effect on surface expression of CD13, the specific receptor for HCoV-229E (Extended Data Fig. 2b) .\n",
      "To test whether LY6E restricts viral entry, we used a vesicular stomatitis virus (VSV) pseudoparticle (pp) system bearing CoV spike (S) proteins.\n",
      "\n",
      " Strikingly, LY6E significantly inhibited VSVpp entry mediated by S proteins from HCoV-229E (Fig.   2a) , MERS-CoV (Fig. 2b) , SARS-CoV (Fig. 2c) , and SARS-CoV-2 (Fig. 2d) .\n",
      "Entry mediated by VSV glycoprotein G was only marginally impacted by LY6E (Extended Data Fig. 2c, d) .\n",
      "To test whether LY6E interferes with the fusion of viral and cellular membranes, we performed a syncytia formation assay using propagation competent VSV pseudoviruses co-expressing CoV S protein and a GFP reporter (VSV*ΔG(CoV S)) (Extended Data Fig. 3a) .\n",
      "LY6E potently inhibited CoV S protein-mediated syncytia formation (Fig. 2e-f ).\n",
      "To determine whether LY6E impairs S protein expression or maturation, we also performed a heterologous syncytia formation assay.\n",
      "CoV-resistant hamster cells infected with trans-complemented GFP reporter VSV*ΔG(CoV) were co-cultured with susceptible target cells that ectopically expressed LY6E.\n",
      "LY6E again significantly blocked syncytia formation, demonstrating that LY6E inhibits S protein-mediated fusion and not S protein expression or maturation (Extended Data Fig. 3b-c) .\n",
      "As membrane fusion can occur without syncytia formation, we also performed a quantitative fusion assay in which mixing of cell contents results in complementation of split luciferase (Fig. 2g) .\n",
      "\n",
      " author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint Cells co-transfected with plasmids encoding CoV S proteins and split luciferase 1-7 were mixed with susceptible target cells co-expressing LY6E or vector control and split luciferase 8-11.\n",
      "LY6E reduced CoV S-mediated cell-cell fusion, indicating that LY6E blocks fusion of viral and cellular membranes ( Fig. 2h) .\n",
      "Time-course experiments revealed that LY6E had no effect on HCoV-229E translation, replication, assembly, or release (Extended Data Fig. 2e-i) .\n",
      "Collectively, our results demonstrate that LY6E specifically inhibits CoV S protein-mediated membrane fusion, which is required for viral entry.\n",
      "\n",
      " We next addressed whether LY6E modulates proteolytic activation of the S protein.\n",
      "Upon S proteinmediated binding to the respective receptor, host proteases cleave the S protein and make it fusion competent for \"early entry\" at the cell surface or \"late entry\" in endosomes 15, 16 .\n",
      "Pharmacological inhibition of cell surface and endosomal proteases had no effect on LY6E-mediated restriction of CoV infection (Extended Data Fig. 4a-c) .\n",
      "Next, we addressed whether LY6E directly interferes with the proteolytic activation of S protein.\n",
      "Mutation of S1/S2 and S2′ in the S protein prevents activation (Extended Data Fig. 4d ) 16, 17 .\n",
      "Ectopic LY6E expression had no effect on MERS-CoV S cleavage or infection by cleavage-resistant MERS-CoV pp (Extended Data Fig. 4e-g) .\n",
      "Collectively, our data indicate that restriction of viral fusion is independent of S protein activation.\n",
      "\n",
      " Next, we aimed to evaluate LY6E-mediated CoV restriction in vivo.\n",
      "Systemic genetic ablation of Ly6e in mice is embryonic lethal 18 .\n",
      "Therefore, we generated transgenic Ly6e fl/fl mice, which are functionally 'wild-type (WT)' for Ly6e expression, and crossed them with Vav1-iCre mice to ablate Ly6e in cells that originate from a hematopoietic stem cell (HSC) progenitor (Extended Data Fig. 5a-b ).\n",
      "This genetic model was chosen since immune cells are critical for protection from CoV in vivo 5,19-23 .\n",
      "Bonemarrow derived macrophages (BMDM) from Ly6e ΔHSC mice had reduced Ly6e mRNA levels (Extended Data Fig. 5c ).\n",
      "The natural mouse pathogen MHV is a well-studied model CoV that causes hepatitis and encephalomyelitis in mice 24 .\n",
      "Ly6e ΔHSC BMDM were more susceptible to MHV infection than Ly6e fl/fl BMDM, demonstrating that murine Ly6e is a restriction factor for murine CoV (Extended Data Fig.   5d ).\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      " To determine whether Ly6e is important for controlling CoV infection in vivo, Ly6e fl/fl and Ly6e ΔHSC mice were infected with a high dose (5 x 10 4 PFU) of MHV that causes sub-lethal hepatitis in WT author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint C57BL/6J mice 25 .\n",
      "Since MHV pathogenesis is influenced by sex 26 , we performed independent experiments in male and female mice.\n",
      "Ly6e ΔHSC mice of both sexes rapidly succumbed to MHV infection by day 6 ( Fig. 3a-b, Extended Data Fig. 6a-b) .\n",
      "To examine viral pathogenesis and immunopathology in infected mice, Ly6e fl/fl and Ly6e ΔHSC mice were infected with the same high dose of MHV and euthanized 3-or 5-days post-infection.\n",
      "At both time points, female Ly6e ΔHSC mice exhibited significantly higher levels of liver damage as measured by serum ALT while no difference was observed in male mice (Fig. 3c,i, Extended Data Fig. 6c,i) .\n",
      "However, hepatic viral burden did not differ in either sex ( Fig. 3d,j, Extended Data Fig. 6d,j) , possibly due to tissue saturation at this high dose.\n",
      "Indeed, female but not male Ly6e ΔHSC mice exhibited increased liver damage and viral burden at a low dose (5 PFU) of MHV (Extended Data Fig. 7a -b, f-g).\n",
      "Spleen viral burden was elevated in female and male Ly6e ΔHSC mice at both time points and doses of MHV (Fig. 3e,k, Extended Data Fig. 6e ,k, Extended Data Fig.   7c ,h).\n",
      "At 3 days post-infection, there was no difference in liver necrosis, but inflammation was moderately reduced in female but not male Ly6e ΔHSC mice ( Fig. 3f-h, Extended Data Fig. 6f-h) .\n",
      "By 5 days post-infection and at both low and high doses of MHV, female Ly6e ΔHSC mice had significantly higher levels of liver necrosis and a reduced presence of mononuclear immune cells ( Fig. 3l- Fig. 7k-n) .\n",
      "These data demonstrate that Ly6e in hematopoietic cells is important for controlling murine CoV infection.\n",
      "\n",
      " We next used a transcriptomic approach to evaluate the effect of Ly6e ablation on global gene expression in liver and spleen of female Ly6e fl/fl and Ly6e ΔHSC mice injected with PBS or the high dose of MHV at 3-and 5-days post-infection.\n",
      "In infected tissues, loss of Ly6e in hematopoietic cells correlated with differential gene expression in pathways associated with tissue damage, such as liverspecific metabolic genes, angiogenesis, wound healing, and immune response to viruses (Fig. 4a-c) .\n",
      "We also observed a striking loss of genes associated with the type I IFN response, inflammation, antigen presentation, and B cells in infected Ly6e ΔHSC mice ( Fig. 4d-e, Extended Data Fig. 8) .\n",
      "7 The histopathology (Fig. 3h,n) and transcriptome data (Fig 4a-e, Extended Data Fig. 8 Fig. 7k-n, Extended Data Fig. 9 ).\n",
      "Multiple cell types were depleted in livers and spleens of infected Ly6e ΔHSC mice, irrespective of sex (Figure 4f g, Extended Data Fig. 10a-b) .\n",
      "B cells were dramatically reduced in livers from infected Ly6e ΔHSC mice.\n",
      "\n",
      " Hepatic CD4 + T cells, NK cells, dendritic cells (DC), macrophages, and neutrophils were also depleted in infected Ly6e ΔHSC mice, whereas CD8 + T cells were unchanged (Fig. 4f, Extended Data Fig. 10a ).\n",
      "\n",
      " Infected Ly6e ΔHSC mice had a reduction in all splenic immune cell subsets except for CD4 + and CD8 + T cells (Fig. 4g, Extended Data Fig. 10b) .\n",
      "Depletion of immune cell populations in Ly6e ΔHSC organs was MHV-dependent, as cell numbers were not altered in PBS-injected mice.\n",
      "\n",
      " To determine whether loss of immune cells in MHV-infected Ly6e ΔHSC mice correlates with increased permissiveness to infection, we assessed MHV infection in splenocytes cultured from Ly6e ΔHSC , Ly6e fl/fl , and Ifnar -/mice.\n",
      "In Ly6e fl/fl splenocyte cultures, MHV infected macrophages, neutrophils, DC, and B cells, but not CD4 + or CD8 + T cells, as previously published (Fig. 4h, Extended Data Fig. 10c Together, our data demonstrate an important role for Ly6e in immune cell-mediated control of CoV infection. Higher viral burden and lower liver inflammation suggests that liver damage in MHV-infected female Ly6e ΔHSC mice may be due to viral pathogenesis that exceeds the ability of the immune system to control the infection. Notably, we observed MHV-induced loss of three classes of cells: 1) cells that are not permissive to the virus (CD4 + T cells), 2) cells that are permissive to MHV but do not demonstrate Ly6e KO-dependent infection (NK cells, macrophages, and neutrophils), and 3) cells that are permissive to MHV in a Ly6e KO-dependent manner (B cells and DC).\n",
      "This latter group is capable of antigen-presentation to CD4 + T cells 27,28 .\n",
      "Thus, in the absence of Ly6e, unrestricted MHV infection author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint of B cells or DC may impair T cell priming, thereby negatively impacting recruitment and activation of NK cells, macrophages, and neutrophils that contribute to control of viral disease.\n",
      "\n",
      " In conclusion, we identified LY6E as a pan CoV restriction factor that limits CoV entry and protects the host from severe viral disease.\n",
      "Our findings are striking given that LY6E has been primarily associated with an enhancing phenotype in which LY6E promotes entry of multiple viruses 29 .\n",
      "However, our data clearly expand the role of LY6E during coronaviral infection and establishes a novel function in protecting the host immune cell compartment.\n",
      "Determining the precise molecular mechanism underlying how LY6E inhibits CoV S protein-mediated membrane fusion will advance our understanding of cellular antiviral defenses against these important human pathogens.\n",
      "Antiviral membrane fusion inhibitors have been successfully implemented for treatment of HIV-1 infection 30 .\n",
      "A therapeutic approach mimicking the mechanism of action of LY6E could provide a first line of defense against novel emerging CoV infections.\n",
      "Furthermore, delineating which Ly6e-expressing immune cells protect mice from MHV will provide new insight into how individual antiviral effectors in distinct cellular compartments modulate viral pathogenesis.\n",
      "author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "34-41 (1996) .\n",
      "Welch's correction (c-d, f-g), one-tailed Mann-Whitney U test (e), ratio paired student's t-test (g), 2way\n",
      " ANOVA followed by Sidak's or Dunnett's multiple comparison test (j,l).\n",
      "Error bars: SD.\n",
      "P values: c, ** p=0.0088; d, *** p=0.0001; e, * p=0.0143; f, * p=0.0286; g, * p=0.0101; h, *** p=0.0010; j, ** p=0.0073, p=0.0033; l, ns=0.0740, ** p=0.0013, *** p=0.0001.\n",
      "author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint p=<0.0001; c, **p=0.0066; d, **** p=<0.0001; f, **** p=<0.0001; h, ns p=0.9975, ** p=0.0054, ns p=0.1520, 0.9892.\n",
      "author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint scale bars are 500 µM (4x) and 100 µM (20x).\n",
      "Statistical significance was determined by Mantel-Cox test (a), two-tailed unpaired student's t-test with Welch's correction (c-e, i-k), two-tailed Mann-Whitney U test (f-g, l-m).\n",
      "Error bars: SEM (b), SD (c-g, i-m).\n",
      "P values: a, *** p=0.0002; c, ** p=0.0031; d, ns p=0.2136; e, **** p=5.759 x 10 -7 ; f, ns p=0.3698; g, ** p=0.0054; i, * p=0.0430; j, ns p=0.1321; k, ** p=0.0094; l, ** p=0.0072; m, **** p=8.741 x 10 -5 .\n",
      "author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint maps displaying significant changes (mean RPKM > 0.5; fold change > 2; FDR ≤ 0.05) in liver (a) and spleen (b).\n",
      "Dendrograms are normalized read data (row z-score) clustered with complete linkage method employing Spearman Rank correlation distance measurement.\n",
      "c, Pathway analysis of KO versus WT.\n",
      "\n",
      " Up-(red) or down-regulated (blue) pathways indicated by activation z-score.\n",
      "Numbers show significantly dysregulated genes as percentage of total gene number included in pathway.\n",
      "d-e, Expression of select genes in liver (d) and spleen (e).\n",
      "f-g, Immune cell counts from liver (f) and spleen \n",
      " HCoV-229E: 1.5 x 10 5 stable LY6E expressing or control cells were seeded in a 12-well plate and infected with HCoV-229E in a serial dilution.\n",
      "Cells were incubated for 2 hours at 33°C, then viral supernatant was removed, and an overlay of 1:1 Avicel (2.4%) (Avicel RC-581NF) and 2x DMEM, author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint supplemented with 10% FBS and 1x penicillin/streptomycin (p-s), was added.\n",
      "Cells were incubated for 3 days at 33°C before medium was removed, cells were washed, and stained with crystal violet (Sigma-Aldrich).\n",
      "Plaque forming units (PFU) were calculated.\n",
      "\n",
      " HCoV-OC43: 5.0 x 10 4 stable LY6E expressing or control cells were seeded in a 24-well plate and infected at 33°C for 1 hour with HCoV-OC43 (MOI=1).\n",
      "Virus was aspirated and 10% FBS/1x NEAA/1x p-s/RPMI (cRPMI) was added back to cells.\n",
      "24 hours post-infection, cells were dissociated using Accumax (Sigma-Aldrich), fixed in 1% PFA, permeabilized per manufacturer's protocol (BD Cytofix/Cytoperm), and stained for nucleoprotein (1:500) and a goat anti-mouse AlexaFluor488- The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint anti-mouse secondary antibody.\n",
      "Images were acquired with a fluorescence microscope and analyzed using ImageXpress Micro XLS (Molecular Devices, Sunnyvale, CA).\n",
      "\n",
      " For infection assays, 1 x 10 5 target cells were seeded in a 24-well plate one day prior infection.\n",
      "Cells were infected with HCoV-229E-Rluc (MOI 0.1) in OptiMEM (Gibco) for 2 hours at 33°C. Cells were washed 1x with PBS and 10% FBS/1x NEAA/1x p-s/DMEM (cDMEM) was added back.\n",
      "Cells were incubated at 33°C for 24 hours, then washed with PBS and lysed using the Renilla Luciferase Assay System kit (Promega).\n",
      "Rluc activity was measured using a plate luminometer (EnSpire 2300 Multilabel reader by Perkin Elmer).\n",
      "For the reconstitution with CRISPR resistant LY6E (CR LY6E), 2.5 x 10 4 cells were seeded in a 24-well plate.\n",
      "One day post-seeding, cells were either left untransduced or transduced with a lentiviral vector encoding for CR LY6E or the empty control.\n",
      "48 hours post-transduction, cell lysates were either infected with HCoV-229E-Rluc (MOI=0.1) for 24 hours as described above or harvested for Western blot.\n",
      "\n",
      " Pseudotyping of VSV*ΔG(Fluc) was performed as previously described 50 with two silent mutation (T2568G, T3327C)), have been described previously [51] [52] [53] .\n",
      "The expression plasmid for SARS-CoV-2 was generated as follows: the coding sequence of a synthetic, codonoptimized (for human cells) SARS-CoV-2 DNA (GeneArt Gene Synthesis, Thermo Fisher Scientific) author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint based on the publicly available protein sequence in the National Center for Biotechnology Information database (NCBI Reference Sequence: YP_009724390.1) was PCR-amplified and cloned into the pCG1 expression vector between BamHI and XbaI restriction sites.\n",
      "The MERS-CoV S cleavage mutants have been described before 17 \n",
      " The split luciferase fusion assay has been described before, with slight modifications 54 .\n",
      "Briefly, 6 x 10 5 293LTV cells were transfected with plasmids encoding CoV S proteins (pCAGGS-HCoV-229E, pCAGGS-MERS-CoV) together with a split-luciferase construct (Rluc8155-156 DSP1-7).\n",
      "The empty expression plasmid (pCAGGS-mCherry) as well as a construct encoding for VSV G served as negative and positive control, respectively.\n",
      "2 x 10 5 stable LY6E expressing or empty control Huh7 cells were transfected using Lipofectamine 2000 (Thermo Fisher Scientific) with a plasmid encoding the second half of the split-luciferase protein (Rluc8155-156 DSP8-11).\n",
      "Approximately 30 hours post-transfection, both cell populations were dissociated (TryPLE Express, Gibco), counted, and equal cell numbers (2x10 4 cells each) were co-cultured for 16 -20 hours at 37°C. Supernatant was harvested and cells lysed using ice-cold passive lysis buffer (Promega).\n",
      "Cells were immediately transferred to -80°C for at least author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint 1 hour, before Rluc activity was determined using the Renilla Luciferase Assay System (Promega).\n",
      "Cells were kept on ice during all times post-cell lysis to eliminate DSP post-lysis complementation.\n",
      "\n",
      " Following extraction of total RNA from MERS-CoV (strain EMC) infected VeroE6 cells, the cDNA encoding the MERS-CoV S protein was generated by reverse transcription (RevertAid Premium Reverse Transkriptase, ThermoScientific).\n",
      "Three overlapping cDNA fragments were amplified by PCR (Phusion Hot Start II High Fidelity Polymerase, Thermo Scientific) and subsequently assembled by overlapping PCR.\n",
      "The cDNAs encoding the S proteins of either MERS-CoV (strain EMC) or HCoV-229E (truncated variant lacking the 10 amino acids at the C terminus, original plasmid pCAGGS-229E S) were amplified by PCR and inserted into the pVSV* plasmid 55 between MluI and BstEII restriction sites.\n",
      "The resulting plasmids were used to generate the recombinant viruses VSV*ΔG(MERS S) and VSV*ΔG(229E S) according to a published procedure 56 .\n",
      "All viruses were propagated on BHK-G43 cells 57 , resulting in viruses predominantly containing the homotypic VSV glycoprotein G in the envelope.\n",
      "\n",
      " Cells expressing LY6E or empty control were infected with VSV*ΔG(MERS S) or VSV*ΔG(229E S) at a MOI of 0.01.\n",
      "At 20 hours post-infection, the cells were fixed with 3% PFA/PBS and the nuclei stained with 4',6-diamidine-2'-phenylindole (DAPI, Sigma).\n",
      "An inverted fluorescence microscope (Zeiss) was used to determine area percentage area covered by syncytia.\n",
      "\n",
      " For heterologous cell-cell fusion assay, BHK-21 cells were infected with VSV*ΔG(CoV S) at a MOI of 1.\n",
      "After 2 hours, the cells were treated with trypsin/EDTA (Life Technologies, Zug, Switzerland) and resuspended in DMEM supplemented with 5% FBS (2x10 4 cells/mL).\n",
      "The infected BHK-21 cell suspension (500 µL) was seeded into 24-well cell culture plates along with LY6E-or empty controlexpressing Huh7 (1 x 10 4 cells).\n",
      "Cell co-cultures were incubated for 7 hours at 37°C, fixed with 3.6% author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint formaldehyde diluted in PBS, (Grogg-Chemie AG, Stettlen, Switzerland) and stained with DAPI.\n",
      "The percentage area covered by syncytia was calculated as above.\n",
      "\n",
      " Ly6e tm1a ES cells were obtained from the EUCOMM consortium 58 \n",
      " Six to twelve-week-old female and male mice were injected intraperitoneally with MHV-A59 diluted in PBS to the indicated titers or PBS for a mock infection control.\n",
      "All infected mice were monitored daily for weight and mortality.\n",
      "Animals that lost more than 20% of their original body weight were euthanized per IACUC guidelines.\n",
      "\n",
      " Viral titers in liver and spleen were determined from frozen organs after weighing, homogenization, and plaque assay on L929 cells.\n",
      "Alanine aminotransferase (ALT) was measured in fresh, unfrozen serum using VITROS MicroSlide Technology by the UTSW Mouse Metabolic Core.\n",
      "Livers were fixed in 10% author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint neutral buffer formalin, embedded in paraffin, sectioned at 5 µM, and stained with hematoxylin and eosin (H&E).\n",
      "Slides were analyzed by an independent pathologist (UTSW Animal Resource Center) who was blinded to experimental conditions.\n",
      "A numerical score was assigned for degree of inflammation and degree of necrosis for each liver.\n",
      "Inflammatory cell infiltration was scored using the following criteria: 0 = none, 1 = minimal, 2 = mild, 3 = moderate, and 4 = marked.\n",
      "The necrosis score was defined by the percent necrosis.\n",
      "0 = none, 1 = <2%, 2 = 2-20%, 3 = 21-40%, and 4 = >40%.\n",
      "\n",
      " All primary cells were maintained in 10% FBS/10 mM HEPES/1 mM sodium pyruvate/2 mM Lglutamine/1x p-s/RPMI (cRPMI2) unless otherwise indicated.\n",
      "\n",
      " Primary bone marrow derived macrophages (BMDM) were prepared as described previously 60 .\n",
      "Viable cells were quantified using trypan blue exclusion.\n",
      "BMDM were plated at 2.5 x 10 4 cells per well in non- The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint separated from IHIC by centrifugation at 850 x g for 30 minutes, 22°C, with no brake.\n",
      "Red blood cells were removed from the resulting pellet with RBC lysis buffer (Tonbo Biosciences).\n",
      "Viable cells were quantified using trypan blue exclusion.\n",
      "IHIC and splenocytes (4 x 10 5 ) from infected or mock-infected mice were stained with a fluorescent viability dye (Ghostdye Violet 450, Tonbo Biosciences), treated with anti-CD16/CD32 (Tonbo Biosciences), stained with antibodies that recognize surface lineage markers, and then fixed in 1% PFA/PBS.\n",
      "Fixed volumes of cell suspensions were analyzed by flow cytometry the next day.\n",
      "Absolute cell counts were determined by multiplying final number of lineage marker-positive cells by dilution factor used to normalize cell counts for antibody staining.\n",
      "\n",
      " All primary cell samples were resuspended in 3% FBS/PBS and analyzed using a S1000 Flow Cytometer with a A600 96-well plate high throughput extension and compensated using CellCapture software (Stratedigm).\n",
      "Data was analyzed with FlowJo Software (Treestar).\n",
      "The flow gating strategy for liver and spleen immune cells is included in Extended Data Figure 9 .\n",
      "\n",
      " Differences in data were tested for significance using GraphPad Prism v8.3.1 for Windows (GraphPad).\n",
      "\n",
      " For details regarding the statistical tests applied, please refer to the figure legends.\n",
      "P values < 0.05 were considered significant.\n",
      "\n",
      " The authors declare that the data supporting the findings of this study are available within the article and its Supplementary Information files or are available on request.\n",
      "The RNAseq data discussed in this publication have been deposited in the Gene Expression Omnibus (GEO) database, https://www.ncbi.nlm.nih.gov/geo (GSE146074).\n",
      "\n",
      " author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint\n",
      " We would like to thank the following people for their generous contribution of reagents and infrastructure which greatly enhanced this study, especially the sequencing facility in Bern, the diagnostic facility of IVI, and the Animal Resource Center at UTSW.\n",
      "Furthermore, we are grateful to \n",
      " The authors declare no competing interests.\n",
      "\n",
      " Supplementary Information is available for this paper.\n",
      "\n",
      " Correspondence and requests for materials should be addressed to Volker Thiel, John W. Schoggins, and Charles M. Rice.\n",
      "\n",
      " author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint was harvested and extracellular viral RNA was extracted and viral replication detected via qRT-PCR (f).\n",
      "Cell lysates were harvested and intracellular Renilla luciferase activity was detected upon cell lysis (g).\n",
      "Cells were subjected to 3 rounds of freeze/thaw cycles.\n",
      "Cell debris was removed and the supernatant titrated on naïve Huh7 cells.\n",
      "Intracellular infectivity was determined (h).\n",
      "Supernatant was harvested and The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint ANOVA followed by Sidak's multiple comparisons test (c).\n",
      "Error bars: SD.\n",
      "P values: c, **** p=<0.0001 <0.0001.\n",
      "author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint two pooled experiments.\n",
      "For i-m, n=8 (Ly6e fl/fl ,), n=6 (Ly6e ΔHSC ), or n=3 (PBS) from two pooled experiments.\n",
      "In h,n, scale bars are 500 µM (4x) and 100 µM (20x).\n",
      "Significance for was tested by\n",
      " Mann-Whitney U test (f-g, l-m).\n",
      "Error bars: SEM (b), SD (c-g, i-m).\n",
      "P value: a, **** p=3.946 x 10 -6 ; c, ns p=1144; d, ns p=0.1407; e, ** p=0.0026; f, ns p=0.1708; g, ns p=0.0779; i, ns p=0.6191; j, ns p=0.6842; k, * p=0.0119; l, ns p>0.9999; m, ns p=0.0779.\n",
      "author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint -c, f-h) , two-tailed Mann-Whitney U test (d-e, i-j).\n",
      "Error bars: SD.\n",
      "P value: a, * p=0.0234; b, * p=0.0428; c, *** p=0.0004; d, * p=0.0210; e, ** p=0.0054; f, ns p=0.1544; g, ns p=0.5322; h, ns p=0.0662; i, ns p=0.0592; j, ns p=0.9064.\n",
      "author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint DDX3X -DDX3, VIPERIN -RSAD2, SERPINE1 -PAI 1, IL28RA -CRF2, VEGFC -VRP, SP110 -IFI41, IRF2 -PRIC285, ZNF313 -RNF114, DNAPTP6 -LOC26010, LOC400759 -PSEUDOGENE,   PSCD1 -CYTH1, IFITM1 -IFI17, SIGLEC1 -FRAG, CCL5 -SYCA5, CXCL9 -MIG, IFIH1 Huh7 cells expressing TMPRSS2 were generated as follows.\n",
      "In order to generate TMPRSS2-encoding retroviral vectors, the open reading frame of human TMPRRS2 was first PCR amplified with primers adding an N-terminal cMYC epitope to the TMPRSS2 coding sequence.\n",
      "The resulting sequence was inserted into a modified version of the pQCXIP plasmid that contains a blasticidin resistance cassette instead of the usual puromycin resistance cassette 17 .\n",
      "Huh7 cells stably expressing human TMPRSS2 were generated by retroviral transduction and selection with the antibiotic blasticidin (50 µg/ml).\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      " Following selection, cells were maintained in culture medium (cDMEM) supplemented with 10 µg/ml blasticidin.\n",
      "To generate STAT1 -/-_CEACAM1 cells, human STAT1 −/− fibroblasts were transduced with lentiviruses encoding for the murine CoV receptor CEACAM1 (kind gift from David Wentworth, CDC, Atlanta, USA 59 and subsequently selected with 1 µg/mL puromycin.\n",
      "Stable LY6E expressing cell lines were generated upon lentiviral transduction with SCRPSY LY6E or as control SCRPSY Empty in DMEM containing 4 µg/ml polybrene (Millipore) and 20 mM HEPES buffer solution (Gibco).\n",
      "Cells author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint were selected using 2-4 µg/mL puromycin and were passaged until all cells in control wells without lentivirus were killed.\n",
      "The puromycin selected cells were further passaged and frozen down for subsequent experiments.\n",
      "The stable cell lines harboring LY6E orthologues have been described before, with the exception of C. dromedarius 9 .\n",
      "For this, a gBlock was ordered (XM_031439745.1) and Gateway cloning performed as described previously to generate pENTR and pSCRPSY plasmids 9 .\n",
      "Constructs encoding for Ly6/uPAR family members and LY6E ASM mutants have been described before 9 .\n",
      "\n",
      " To generate clonal LY6E KO cells and CRISPR-resistant LY6E, A549 cells were transduced with lentivirus containing a LY6E-specific sgRNA and Cas9 as described previously 9 .\n",
      "To generate a clonal cell line, the bulk transduced population was plated at single cell dilutions.\n",
      "Candidate clones were screened by Western blot for LY6E expression and Sanger sequencing.\n",
      "Silent mutations in the region targeted by the LY6E-specific sgRNA were introduced into HA-tagged LY6E to generate CRISPRresistant LY6E (CR LY6E).\n",
      "LY6E KO A549 cells were reconstituted with CR-LY6E by lentiviral transduction and expression confirmed by Western blot.\n",
      "\n",
      " All cell lines were regularly tested to check they were free of mycoplasma contamination using a commercially available system (PCR Mycoplasma test kit I/RT Variant C, PromKine).\n",
      "\n",
      " The ISG screen was performed as described previously with slight modifications 7,8 .\n",
      "Briefly, 5 x 10 3 Huh7 cells were seeded, transduced with individual lentiviruses, and 48 hours post-transduction infected with HCoV-229E at 33°C. Infection was stopped 24 hours (MOI=0.1) or 48 hours (MOI=0.01) postinfection and plates were immuno-stained as described previously 61 with an anti-HCoV-229E N protein antibody and a AlexaFluor488-conjugated donkey anti-mouse secondary antibody (Extended methods;  Antibodies for immunofluorescence and flow cytometry).\n",
      "For high-content high-throughput imaging analysis ImageXpress Micro XLS (Molecular Devices, Sunnyvale, CA) was used as previously described 8 .\n",
      "Hits were normalized to cells expressing the empty vector.\n",
      "Depicted are genes that were author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint expressed in two independent screens with a transduction efficiency of at least 1 % (cut off).\n",
      "ISGs which were cytotoxic for the cells were excluded from the analysis.\n",
      "Normalized data can be found in Extended Data \n",
      " Viral RNA was extracted from cell lysates (NucleoMag-96RNA kit, Macherey Nagel) or supernatant (NucleoMag-Vet kit, Macherey Nagel) using the KingFisher Flex robot according to the manufacturer's recommendations.\n",
      "The commercially available TaqManTM Fast Virus 1-Step Master Mix (Applied Biosystems) was used for RT-qPCR with 2 µl of RNA input added to 8 µl of prepared mastermix per author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint sample.\n",
      "Viral RNA was detected using HCoV-229E specific primers normalized to a qRT-PCR standard for HCoV-229E (contains the M gene of HCoV-229E 63 ).\n",
      "Intracellular viral RNA was normalized to total RNA (determined via the housekeeping gene Beta-2-Microglobulin (B2M)).\n",
      "Cellular RNA was extracted from cell lysates of Huh7 cells using the Nucleospin RNA kit (Machery Nagel) according to the manufacturer's recommendations, quantified via NanoDrop, and used to generate a standard curve.\n",
      "\n",
      " PCR conditions are available upon request.\n",
      "\n",
      " Stable LY6E expressing or control Huh7 cells were seeded in a 24-well plate (4 x 10 4 cells) and mock infected or inoculated with HCoV-229E (MOI=5) in OptiMEM (Gibco) on ice for 1 hour.\n",
      "Cells were washed at least 3x with PBS and harvested immediately (t=0 h) or incubated at 33°C for 24 hours (t=24 h), before cell lysis and extraction of viral RNA as described above.\n",
      "Viral RNA was detected via RT-qPCR normalized to the housekeeping gene B2M as described above.\n",
      "\n",
      " Time course experiment 8 x 10 4 cells stable LY6E expressing or control Huh7 cells were seeded in 12-well plates and mock infected or infected with 229E-CoV-Rluc (MOI=0.1) for 2 hours at 33°C in OptiMEM.\n",
      "Cells were washed 3x with PBS and samples harvested at the indicated time points.\n",
      "To determine intracellular replication, cell lysates were collected and viral RNA extracted (NucleoMag-96RNA kit, Macherey author/funder. All rights reserved. No reuse allowed without permission.   The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint Nagel), followed by RT-qPCR as described before, or cells were lysed using the Renilla Luciferase Assay System kit (Promega) according to the manufacturer's recommendations and Rluc activity determined.\n",
      "To determine extracellular viral replication, viral supernatant was harvested, and viral RNA extracted (NucleoMag-Vet kit, Macherey Nagel), followed by RT-qPCR as described before.\n",
      "To determine intracellular infectivity, cells were subjected to 3 rounds of freeze/thaw cycles, centrifuged to remove debris (4,000 × g for 10 min), and the supernatant titrated on naïve Huh7 cells.\n",
      "To determine extracellular infectivity, supernatant was harvested at the indicated time points and titrated on naïve \n",
      " To test various protease inhibitors, 2 x 10 4 naïve or TMPRSS2-expressing, control or LY6E cells were seeded in a 96-well plate.\n",
      "One day post seeding, cells were pre-treated with the following compounds for 1 hour in OptiMEM, at 37°C: DMSO (1:500, Sigma-Aldrich), E64 D (10 µM, Sigma-Aldrich), and/or Camostat (100 µM, Sigma-Aldrich).\n",
      "Cells were mock infected or infected with HCoV-229E-Rluc (MOI 0.1) in the presence of the inhibitors for 2 hours at 33°C. Medium was changed to DMEM and cells incubated for 24 hours at 33°C. Cells were lysed and Rluc activity detected using the Renilla Luciferase Assay System kit (Promega) according to the manufacturer's recommendations.\n",
      "\n",
      " LY6E expressing or control Huh7 (2 x 10 5 cells) were seeded in 6-well cell culture plates.\n",
      "Cells were transfected using Lipofectamine 2000 (Invitrogen) with an expression plasmid encoding for MERS-CoV S (pCAGGS-MERS S).\n",
      "Cell lysates were harvested 48 hours post-transfection and subjected to Western blot analysis as described.\n",
      "\n",
      " author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint\n",
      " Multiple sequence alignment of LY6E amino acid sequences of different species (NCBI accession numbers provided in the figure) was performed using the EBI web based Clustal Omega tool (doi:  10.1093/nar/gkz268).\n",
      "The phylogenetic tree was built with the \"one click\" mode for phylogenetic analysis (http://www.phylogeny.fr/) with standard settings (doi: 10.1093/nar/gkn180; doi:  10.1186/1471-2148-10-8).\n",
      "\n",
      " Sections of liver and spleen were preserved in RNAlater Stabilization Solution (Thermo Fisher Scientific) and frozen.\n",
      "Thawed samples were transferred to PBS and homogenized.\n",
      "One-eighth of the homogenate was mixed with TRIzol Reagent (Thermo Fisher Scientific).\n",
      "BMDM were directly lysed in TRIzol Reagent and frozen.\n",
      "Total RNA was isolated according to the manufacturer's protocol.\n",
      "Liver and spleen RNA were subject to DNase treatment (TURBO DNA-Free kit, Thermo Fisher Scientific) per the manufacturer's protocol prior to RNA-Seq analysis.\n",
      "BMDM RNA was analyzed by one-step qRT-PCR using QuantiFast SYBR Green RT-PCR Kit (Qiagen) using Applied Biosciences 7500 Fast Real-Time PCR System.\n",
      "\n",
      " The quantity and quality of the extracted RNA was assessed using a Thermo Fisher Scientific qubit 2.\n",
      "author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint Data generated from individual samples (>30 million read per sample, single read 50-mers) were mapped separately against the GRCm38 murine reference genome.\n",
      "Gene expression was calculated for individual transcripts as reads per kilobase per million bases mapped (RPKM).\n",
      "All transcriptomic analyses were performed using CLC Genomics Workbench 20 (Qiagen, Aarhaus).\n",
      "\n",
      " Differentially expressed genes (DEGs) were identified by calculating fold changes in expression, pvalues were corrected by taking false discovery rate (FDR) for multiple comparison into account.\n",
      "\n",
      " Mus Musculus EBI Gene Ontology Annotation Database was used to execute Gene ontology (GO) Enrichment Analyses for biological processes.\n",
      "Gene identifiers for DEGs absolute FC > 5, RPKM > 2) were used as input and identification of significantly enriched GO categories.\n",
      "P-values for specific GO categories were generated after Bonferroni correction for multiple testing.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Z-scores, used as indicator for activation of biological processes (positive value) or inactivation (negative value), were calculated based on the expression fold change as follows: The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint gctggcatcgggaacgtggtggactttggctac accctgaacaagggctgctccccgatctgtccc ggcccgagcgtcaatcttggagtggcgtccgt gggcacccactgctgccagagcttcctgtgca acatcagtgcagccgacggcgggctgcgggc cagcacccacgtgctgggcctcgggctcctgc tcagcctgctgtccgccctgctgcggcttggcc cctgaacccagctttcttgtacaaagtggtcccc author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint.\n",
      "The rapid evolution of RNA viruses represents a significant challenge for preventing, treating, 57 and eradicating RNA viral diseases.\n",
      "High mutation rates in RNA viruses generate extensive 58 opportunities to overcome evolutionary hurdles, such as antiviral drugs, host immunity, or 59 engineered attenuating changes (1).\n",
      "The evolutionary pathways traversed by RNA viruses are 60 shaped by natural selection, which will favor some evolutionary trajectories more than others 61 based on whether mutations are beneficial, deleterious, or neutral (2).\n",
      "Predicting the likely results 62 of RNA virus evolution is an important step for anticipating viral emergence and for developing 63 escape-resistant antiviral drugs and vaccines (3, 4).\n",
      "64 65 Coronaviruses (CoVs) are a family of positive-sense RNA viruses that cause human illnesses 66 ranging from the common cold to severe and lethal respiratory disease (5).\n",
      "All CoVs encode a 67 proofreading exoribonuclease within nonstructural protein 14 (nsp14-ExoN) that is critical for 68 replication, fidelity, fitness, and virulence, and ExoN-inactivation has been proposed as a 69 strategy for live-attenuated vaccine development (6-15).\n",
      "As members of the DEDDh superfamily 70 of exonucleases, CoV ExoNs hydrolyze nucleotides using four metal-coordinating amino acids 71 The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/618249 doi: bioRxiv preprint ExoN active site mutants in alphacoronaviruses, including transmissible gastroenteritis virus and 77 hCoV-229E, have yet to be recovered and are proposed to be lethal for replication (19, 20) . infections and prolonged passage in tissue culture and following treatment with multiple 87 nucleoside analogues (6-11, 13, 14).\n",
      "SARS-CoV-ExoN-AA also is stable during acute and 88 persistent animal infections in immunocompetent and immune-compromised mice (12) .\n",
      "The lack 89 of primary reversion is not due simply to reduced adaptive capacity, as both SARS-CoV-and 90 MHV-ExoN-AA can adapt for increased replication (7, 14).\n",
      "Most strikingly, long-term passage 91 of MHV-ExoN-AA (250 passages, P250) yielded a highly fit population that had directly 92 compensated for defective proofreading through evolution of a likely high-fidelity RdRp (7 requires mutations to all four sites ( Figure 1A ).\n",
      "Viral mutation rates in the absence of 113 proofreading range from 10 -4 to 10 -6 mutations per nucleotide per round of replication (µ) (1).\n",
      "114\n",
      " Assuming an ExoN-AA mutation rate of 10 -4 µ and accounting for codon degeneracy, the 115 probability of restoring the native amino acid sequence in a single round of replication is 10 -18 .\n",
      "116\n",
      " Only rarely do ExoN-AA titers exceed 10 6 PFU/mL, so it is unlikely that ExoN-AA could 117 navigate this genetic barrier in a single infectious cycle.\n",
      "Thus, we hypothesized that ExoN-AA 118 reversion, if possible, would proceed incrementally.\n",
      "To identify potential pathways towards 119\n",
      " ExoN-AA reversion, we examined the possible single-nucleotide substitutions surrounding A89 120 and A91 ( Figure 1B ).\n",
      "Three mutations are synonymous, and five mutations yield amino acids 121 unlikely to coordinate with the positively-charged metals required for ExoN catalysis (glycine, 122 valine, proline, threonine, and serine) (16, 19, 21, 22).\n",
      "One mutation per site can restore the 123 acidic charge (i.e. AA-to-ED) but not the native amino acid.\n",
      "These variants have not been tested 124 in a CoV ExoN, but biochemical studies of E. coli DNA polymerase I ExoN mutants suggest that 125 these conservative substitutions would not restore WT-like ExoN activity (23) .\n",
      "We predicted 126 stepwise pathways to ExoN-AAàDE reversion based on restoration of acidic charge followed 127 by reversion to native amino acids ( Figure 1C ).\n",
      "We engineered and recovered variants in ExoN-128 AA requiring three mutations (3nt; ExoN-AD, ExoN-EA), two mutations (2nt; ExoN-DA, ExoN-129 ED, ExoN-AE), or one mutation (1nt; ExoN-DD, ExoN-EE) for reversion to WT-ExoN-DE 130 (Table 1) .\n",
      "We will hereafter refer to these mutants as intermediate revertants.\n",
      "All intermediate 131 All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/618249 doi: bioRxiv preprint revertants generated viable progeny during recovery, demonstrating that reversion to WT-ExoN-132 DE along these pathways is theoretically possible.\n",
      "The 3nt and 2nt mutants were genetically 133 stable during recovery, as confirmed by dideoxy sequencing.\n",
      "However, both 1nt mutants (ExoN-134 DD and ExoN-EE) reverted to WT-ExoN-DE during three independent recovery attempts, 135\n",
      " suggesting that these two variants are less fit than WT-ExoN-DE and demonstrating that 136 reversion by 1nt mutation is readily accessible.\n",
      "To test whether the 3nt or 2nt mutants would 137 revert more rapidly than ExoN-AA (4nt), we passaged three lineages of each mutant 10 times at (Table 1 ).\n",
      "In contrast, the 2nt ExoN-AE contained WT-revertants by P2 145 in all lineages at MOI = 0.5 PFU/cell and by P8 in one lineage at MOI = 0.01 PFU/cell.\n",
      "Once 146 observed, WT-revertants dominated the ExoN-AE population for the remaining passages.\n",
      "These 147 data indicate that at least one 2nt mutation pathway can lead to full reversion in tissue culture.\n",
      "148\n",
      " The probability of ExoN-AE arising during a single infectious cycle of ExoN-AA is low but 149 theoretically achievable (~10 -9 ), so ExoN-AA could conceivably revert within just two infectious 150 cycles.\n",
      "However, complete reversion has never been observed even during prolonged passage or 151 persistent infections, suggesting that additional barriers to the replication, fitness, or maintenance 152 of intermediate revertants exist.\n",
      "153 All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/618249 doi: bioRxiv preprint \n",
      " GAT GAA\n",
      " GCA GCT\n",
      " ExoN Motif I C All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.\n",
      "\n",
      " All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/618249 doi: bioRxiv preprint\n",
      " the intermediate revertants are viable as recombinants but are not found in ExoN-AA 159 populations, we hypothesized that they confer no selective advantage over ExoN-AA (8, 9, 13).\n",
      "160\n",
      " To test this hypothesis, we first analyzed replication of the 3nt and 2nt intermediate revertants 161 ( The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. nsp12/14-P250) were significantly more sensitive to 5-FU ( Figure 4B ).\n",
      "Finally, the nsp12-P250 224 and nsp14-P250 mutations significantly decreased fitness relative to WT-ExoN-DE ( Figure 4C ).\n",
      "225\n",
      " We detected no statistical differences between the specific infectivity of WT-ExoN-DE and any 226 of the nsp12-P250 and nsp14-P250 variants in isolated infections ( Figure 4D ).\n",
      "Thus, mutations 227 in nsp12 and nsp14 that arose in the ExoN-AA background were detrimental to replication, 228 mutagen sensitivity, and competitive fitness in the presence of a fully-reverted ExoN-DE.\n",
      "These 229 All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/618249 doi: bioRxiv preprint Our results also extend existing studies of CoV ExoN motif I.\n",
      "Motif I AAàDE mutations in the 249 SARS-CoV nsp14-ExoN dramatically reduce nuclease activity in biochemical assays, but no 250 study has examined the contributions of each residue independently (16, 18) .\n",
      "Intermediate 251 revertants of ExoN-AA did not display consistent or statistical differences in replication, 5-FU 252 sensitivity, or competitive fitness relative to ExoN-AA, suggesting that they remain 253 phenotypically ExoN(-) during infection and supporting previous studies that motif I DE is 254 essential for WT ExoN function.\n",
      "Given these results, we were surprised to observe repeated 255 reversion of the ExoN-AE but not the other two 2nt variants, ExoN-DA and ExoN-ED.\n",
      "One 256 potential explanation is that the specific mutational bias of ExoN-AE makes the revertant 257 mutations more accessible than in ExoN-DA or ExoN-ED.\n",
      "Alternatively, if ExoN-AE has 258 profound replication or fitness defects, selection could drive primary reversion more quickly 259 away from this genotype.\n",
      "Consistent with this hypothesis, ExoN-AE reverted more quickly at a 260 higher MOI, where natural selection acts more efficiently on a larger population size (Table 1) Consistent with this hypothesis, the fitness effects of mutations in nsp12-P250 and nsp14-P250 269 differ based on the motif I genotype; they are beneficial in ExoN-AA but detrimental in the WT-270 All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/618249 doi: bioRxiv preprint ExoN-DE background.\n",
      "In previous studies, it has been difficult to determine whether the fitness 271 defects in ExoN(-) CoVs are directly linked to low-fidelity replication or through some other 272 mechanism.\n",
      "Our data suggest that the proofreading function of nsp14-ExoN can be uncoupled 273 from its more general role in replication (Figure 4) The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/618249 doi: bioRxiv preprint Cloning and recovery of recombinant viruses.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Site-directed mutagenesis in MHV genome 306 fragments was performed using \"round the horn\" PCR (originally described in (33)).\n",
      "Briefly, 307 adjacent primers containing the mutation of interest were 5¢-phosphorylated using T4 308 polynucleotide kinase (NEB, M0201S) using the buffer from the T4 DNA ligase, which contains 309 ATP (M0202S).\n",
      "PCR was performed on a plasmid template using the Q5 High-fidelity 2x previously (9).\n",
      "Briefly, genomic RNA was detected with a 5' 6-carboxyfluorescein (FAM) and 341 3' black hole quencher 1 (BHQ-1) labeled probe targeting nsp2 (Biosearch Technologies,  342 Petaluma, CA), and RNA copy number was calculated by reference to an RNA standard derived 343 from the MHV A fragment.\n",
      "Samples were plated in technical duplicate to minimize well-to-well 344 variation.\n",
      "Titers were determined by plaque assay in DBT-9 cells, and specific infectivity was 345 calculated as PFU per supernatant genomic RNA copy.\n",
      "346 347 5-fluorouracil sensitivity assays.\n",
      "Stock solutions of 5-fluorouracil (Sigma F6627) were 348 prepared in dimethyl sulfoxide (DMSO).\n",
      "Sensitivity assays were performed in 24-well plates at 349 All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/618249 doi: bioRxiv preprint MOI = 0.01 PFU/cell, as previously described (7) were averaged, and values were excluded from subsequent analysis if the duplicate wells differed 371 All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/618249 doi: bioRxiv preprint by > 0.5 Ct.\n",
      "The relative abundance of competitor and reference were determined by subtracting 372\n",
      " Ct thresholds (DCtcompetitor = Ctcompetitor -Ctreference) and converted to reflect the fold-change in 373 ratio (Dratio = 2 -DCt competitor ).\n",
      "The log10Dratio was plotted against passage number, and the change 374 in log10Dratio (i.e. slope of linear regression) is the relative fitness.\n",
      "Note that regressions were fit 375 only through P1-P4, as slight deviations in 1:1 ratio in the input (P0) can skew the slope.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/618249 doi: bioRxiv preprint.\n",
      "9 also deleted ORF 4a, which is dispensable for MHV replication in cell culture (52).\n",
      "The C-150 terminal region of ns2 within MHV-ExoN(-) P250 was truncated and fused to HE with a -1 151 frameshift.\n",
      "Ns2 is a phosphodiesterase (PDE) that protects viral RNA by degrading 2¢à5¢ 152 oligoadenylate, the activating factor for cellular RNase L (53-55).\n",
      "The portion of ns2 deleted in The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/175562 doi: bioRxiv preprint similar for all four viruses (Fig. 4B) .\n",
      "These data suggest that the increased replication of P250 173 viruses relative to WT-MHV is not fully accounted for by increased RNA synthesis.\n",
      "In addition 174 to RNA synthesis defects, ExoN(-) CoVs have up-to 20-fold increased mutation frequencies and 175 profoundly increased sensitivity to nucleoside and base analogs relative to WT CoVs (13, 14, 16, 176 17, 38).\n",
      "To determine whether nucleoside analog sensitivity of MHV-ExoN(-) was altered by 177 long-term passage, we treated cells infected with parental and passaged viruses with the base 178 analog, 5-fluorouracil (5-FU).\n",
      "5-FU is converted intracellularly into a nucleoside analog that 179 incorporates into growing RNA strands and causes A:G and U:C mutations.\n",
      "For simplicity, we 180 hereafter refer to 5-FU as a nucleoside analog.\n",
      "Incorporation of 5-FU is increased in the absence 181 of ExoN activity (16).\n",
      "All viruses displayed a concentration-dependent decrease in viral titer but 182 differed greatly in their susceptibility to 5-FU (Fig. 4C ).\n",
      "At 120µM, WT-MHV P3 titers were 183 reduced by ~1 log 10 , while MHV-ExoN(-) P3 titers were undetectable (> 5 log 10 -fold reduction).\n",
      "184\n",
      " WT-MHV 5-FU sensitivity was not altered by passage.\n",
      "MHV-ExoN(-) P250 was less susceptible 185 than MHV-ExoN(-) P3 to 5-FU treatment, with only a ~1.5 log 10 decrease in titer at 120 µM. replication cycles in which 5-FU can be incorporated (59) .\n",
      "MHV-ExoN(-) P250 had multiple 193 mutations in the spike glycoprotein, including one in the spike furin cleavage site that reduced 194 syncytia formation.\n",
      "To test whether the spike mutations manifested in resistance to 5-FU, we 195 All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.\n",
      "However, M288T is not predicted to catalyze nucleotidylation.\n",
      "Within nsp14, 4 NS mutations 215\n",
      " were identified in the ExoN domain, and 2 NS mutations were in the C-terminal N7-216 methyltransferase domain (Fig. 6B ).\n",
      "We next modeled the structure of MHV nsp14 using Phyre 2 217 software (63) , resulting in highest-probability similarity to the SARS-CoV nsp14-nsp10 complex 218 All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/175562 doi: bioRxiv preprint between the structures) of 100% for residues 3-519 of MHV-nsp14.\n",
      "The model predicts that five 220 mutations are located close to surface of the protein (Fig. 6B ).\n",
      "All three modeled zinc finger 221 domains contain one NS mutation (F216Y, Y248H, L473I).\n",
      "Two mutations, D128E and F216Y, 222 are located near the interface between nsp10 and nsp14, though neither site has previously been 223 implicated in nsp10-nsp14 interaction (15, 64, 65).\n",
      "One NS mutation resulted in a D272E 224 substitution in ExoN motif III, a metal-coordinating active site residue.\n",
      "We previously reported 225 that alanine substitution of D272 results in an ExoN(-) phenotype (14), but the viability or 226 phenotype of a D272E substitution was not tested in that study.\n",
      "These data suggest that a 227 network of residues evolved to regulate nsp12 and nsp14 activity or stability in the ExoN(-) 228 background.\n",
      "229 230 Fixed mutations in nsp12 and nsp14 in MHV-ExoN(-) P250 directly correlate with 231 increased resistance to multiple nucleoside analogs.\n",
      "To determine approximately when the 232 mutations in nsp12 and nsp14 arose, we performed di-deoxy sequencing across these protein-233 coding regions roughly every 20 passages (P10, 31, 50, 72, 90, 100, 120, 140, 160, 180, 200, 220, 234 240) .\n",
      "By this method, we detected consensus NS mutations at P10, P50, and P160 for nsp12, and 235 at P50 and P160 for nsp14 (Fig. 6) .\n",
      "Both nsp12 and nsp14 carried their full complement of P250 236 consensus mutations by P160, except for a minority variant (D913E) in nsp12 maintained at 237 <50% of the population between P200 and P250.\n",
      "These passage levels correlated with increased 238 replication of MHV-ExoN(-) (Fig. 2B ) and with decreasing sensitivity to 5-FU (Fig. 7A ).\n",
      "Neither 239 replication nor 5-FU sensitivity of MHV-ExoN(-) changed substantially between P160 and P250.\n",
      "240\n",
      " To determine whether MHV-ExoN(-) evolved increased resistance to multiple nucleoside 241 All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. implicating a complex evolutionary interaction between these two proteins.\n",
      "The measured fitness 284 correlated with the patterns of nucleoside analog resistance and RNA synthesis associated with 285 mutations in nsp12 and nsp14, suggesting a link between the evolution of these virus phenotypes.\n",
      "286 All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/175562 doi: bioRxiv preprint\n",
      " The result also confirms that nsp12 and nsp14 are important but not sufficient to account for the 287 significantly increased fitness of MHV-ExoN(-) P250 relative to MHV-ExoN(-) P3.\n",
      "288 289 All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.\n",
      "Mutations conferring increased replication fidelity to RNA viruses have most frequently been 334 mapped to RdRps (24, 25, 30, 72) .\n",
      "Three findings suggest that mutations within nsp12-P250 335 All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. with nsp12, acting as a primase/processivity factor (74, 75) and a helicase/NTPase, respectively 358 All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.\n",
      "\n",
      " Cell culture.\n",
      "DBT-9 (delayed brain tumor, murine astrocytoma clone 9) cells were maintained 409 as described previously (91) The virus titer of each stock was determined by plaque assay using DBT-9 cells as described 425 previously (14, 91).\n",
      "For plaque assays of viruses containing the spike protein from MHV-ExoN(-426 ) P250, which does not form syncytia, plaques were visualized with neutral red (Sigma #N6264, 427 diluted 1:10 in PBS containing calcium and magnesium).\n",
      "Neutral red was added 24 hours after 428 plating and incubated for an additional 3-8 hours before formaldehyde fixation.\n",
      "Plaque 429 purification was performed by infecting DBT cells with serial dilutions of virus and overlaying 430 All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/175562 doi: bioRxiv preprint the cultures with agar.\n",
      "Single plaques were isolated, resuspended in PBS containing calcium and 431 magnesium, and inoculated onto fresh DBTs.\n",
      "This process was completed 3 times before 432 experimental stocks were generated, as above. and RNA copy number was calculated by reference to an RNA standard derived from the MHV 475 A fragment.\n",
      "Samples were plated in technical duplicate to minimize well-to-well variation.\n",
      "Titers 476 All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/175562 doi: bioRxiv preprint were determined by plaque assay in DBT-9 cells, and specific infectivity was calculated as PFU 477 per supernatant genomic RNA copy.\n",
      "478\n",
      " Nucleoside and base analog sensitivity assays.\n",
      "5-azacytidine (AZC), 5-fluorouracil (5-FU), and 480 ribavirin (RBV) were purchased from Sigma (product numbers A2385, F6627, and R9644, 481 respectively).\n",
      "Stock solutions of 5-FU and RBV were prepared in dimethyl sulfoxide (DMSO).\n",
      "482\n",
      " 2'-C-methyladenosine (CMeA) was received from Gilead Sciences, Inc (Foster City, CA).\n",
      "483\n",
      " Sensitivity assays were performed as described previously (16) viruses were detected with the same probe used in specific infectivity analyses (14).\n",
      "Reference 516 viruses were detected by a probe targeting the same region but with 10 silent mutations (5'-517 TCCGAACTACTGCAACCCCAAGTG-3') and labeled with 5' Quasar 670 and 3' black hole 518 quencher 2 (BHQ-2) (Biosearch Technologies, Petaluma, CA).\n",
      "RNA copy number was 519 calculated by reference to an RNA standard generated by in vitro transcription of the 520 corresponding MHV A fragment, and relative RNA abundance was calculated as the ratio of 521 competitor genomes to reference genomes.\n",
      "522 All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/175562 doi: bioRxiv preprint and synonymous mutations (purple) are below.\n",
      "564 The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/175562 doi: bioRxiv preprint statistical significance was determined using one-way ANOVA.\n",
      "For Panel C, statistical 587 significance for change in titer of MHV-ExoN(-) spike-P250 relative to MHV-ExoN(-) P3 was 588 determined using Mann-Whitney test (*P<0.05, **<0.01, ***P<0.001, ns = not significant).\n",
      "589 The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/175562 doi: bioRxiv preprint All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/175562 doi: bioRxiv preprint All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/175562 doi: bioRxiv preprint tuberculosis DNA polymerase III αβ2ε replicase promotes polymerization and reduces 855 All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/175562 doi: bioRxiv preprint All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.\n",
      "All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.\n",
      "Virus titer (log 10 PFU/ml) All rights reserved.\n",
      "No reuse allowed without permission.\n",
      "\n",
      " The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/175562 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/175562 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/175562 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/175562 doi: bioRxiv preprint.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "for sentence in split_sentences(text(sent_files))\n",
    "    println(sentence)\n",
    "end"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"Odor\", \"perception\", \",\", \"and\", \"emotional\", \"and\", \"behavioral\", \"responses\", \"to\", \"odors\", \"strongly\", \"depend\", \"on\", \"experience\", \",\", \"and\", \"learned\", \"odor-context\", \"associations\", \"often\", \"last\", \"for\", \"the\", \"lifetime\", \"of\", \"an\", \"animal\", \"(\", \"Mouly\", \"and\", \"Sullivan\", \",\", \"2010\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"cellular\", \"and\", \"neural\", \"circuit\", \"mechanisms\", \"underlying\", \"olfactory\", \"learning\", \"and\", \"memory\", \",\", \"however\", \",\", \"remain\", \"poorly\", \"understood\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Recent\", \"studies\", \"on\", \"episodic\", \"and\", \"contextual\", \"learning\", \"in\", \"hippocampal\", \"neural\", \"networks\", \"have\", \"suggested\", \"that\", \"memories\", \"are\", \"encoded\", \"in\", \"the\", \"activity\", \"of\", \"distributed\", \"ensembles\", \"of\", \"neurons\", \",\", \"often\", \"referred\", \"to\", \"as\", \"a\", \"'\", \"memory\", \"trace\", \"'\", \"(\", \"Mayford\", \"and\", \"Reijmers\", \",\", \"2015\", \";\", \"Poo\", \"et\", \"al.\", \",\", \"2016\", \";\", \"Tonegawa\", \"et\", \"al.\", \",\", \"2015\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"neurons\", \"constituting\", \"such\", \"a\", \"memory\", \"trace\", \"are\", \"thought\", \"to\", \"encode\", \"information\", \"about\", \"the\", \"environmental\", \"context\", \"and\", \"associated\", \"emotions\", \"of\", \"past\", \"experiences\", \",\", \"and\", \"their\", \"activity\", \"is\", \"necessary\", \"and\", \"sufficient\", \"for\", \"memory\", \"retrieval\", \"(\", \"Liu\", \"et\", \"al.\", \",\", \"2012\", \";\", \"Reijmers\", \"et\", \"al.\", \",\", \"2007\", \";\", \"Tanaka\", \"et\", \"al.\", \",\", \"2014\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Here\", \",\", \"we\", \"investigate\", \"the\", \"organization\", \"of\", \"odor\", \"memory\", \"traces\", \"in\", \"the\", \"olfactory\", \"(\", \"piriform\", \")\", \"cortex\", \"of\", \"mice\", \".\"]\n",
      "Word token=> [\"The\", \"piriform\", \"cortex\", \",\", \"a\", \"trilaminar\", \"paleocortical\", \"structure\", \",\", \"is\", \"the\", \"largest\", \"cortical\", \"area\", \"receiving\", \"direct\", \"afferent\", \"inputs\", \"from\", \"the\", \"olfactory\", \"bulb\", \",\", \"which\", \",\", \"in\", \"turn\", \",\", \"receives\", \"topographically\", \"organized\", \"inputs\", \"from\", \"olfactory\", \"sensory\", \"neurons\", \"in\", \"the\", \"nose\", \".\"]\n",
      "Word token=> [\"Individual\", \"piriform\", \"neurons\", \"respond\", \"to\", \"combinatorial\", \"inputs\", \"from\", \"multiple\", \"Piriform\", \"cortex\", \"has\", \"long\", \"been\", \"hypothesized\", \"to\", \"support\", \"auto-associative\", \"network\", \"functions\", \"that\", \"can\", \"retrieve\", \"previously\", \"learned\", \"information\", \"from\", \"partial\", \"or\", \"degraded\", \"sensory\", \"inputs\", \"(\", \"Haberly\", \",\", \"2001\", \";\", \"Wilson\", \"and\", \"Sullivan\", \",\", \"2011\", \")\", \".\"]\n",
      "Word token=> [\"Piriform\", \"pyramidal\", \"cells\", \"form\", \"a\", \"large\", \"recurrent\", \"network\", \",\", \"which\", \"is\", \"reciprocally\", \"connected\", \"with\", \"adjacent\", \"highorder\", \"associative\", \"areas\", \"including\", \"the\", \"prefrontal\", \",\", \"entorhinal\", \"and\", \"perirhinal\", \"cortex\", \"and\", \"the\", \"amygdala\", \"(\", \"Johnson\", \"et\", \"al.\", \",\", \"2000\", \";\", \"Sadrian\", \"and\", \"Wilson\", \",\", \"2015\", \")\", \".\"]\n",
      "Word token=> [\"Storage\", \"of\", \"information\", \"is\", \"made\", \"possible\", \"by\", \"NMDA-dependent\", \",\", \"associative\", \"plasticity\", \"of\", \"connections\", \"(\", \"Johenning\", \"et\", \"al.\", \",\", \"2009\", \";\", \"Kanter\", \"and\", \"Haberly\", \",\", \"1990\", \";\", \"Quinlan\", \"et\", \"al.\", \",\", \"2004\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Furthermore\", \",\", \"changes\", \"in\", \"piriform\", \"network\", \"activity\", \"(\", \"Chapuis\", \"and\", \"Wilson\", \",\", \"2011\", \";\", \"Chen\", \"et\", \"al.\", \",\", \"2011\", \";\", \"Li\", \"et\", \"al.\", \",\", \"2008\", \";\", \"Sevelinges\", \"et\", \"al.\", \",\", \"2004\", \")\", \"and\", \"stabilization\", \"of\", \"piriform\", \"odor\", \"representations\", \"(\", \"Shakhawat\", \"et\", \"al.\", \",\", \"2015\", \")\", \"have\", \"been\", \"observed\", \"after\", \"associative\", \"olfactory\", \"learning\", \".\"]\n",
      "Word token=> [\"Finally\", \",\", \"excitotoxic\", \"lesions\", \"of\", \"the\", \"posterior\", \"piriform\", \"cortex\", \"in\", \"rats\", \"perturb\", \"odor\", \"fear\", \"memories\", \"(\", \"Sacco\", \"and\", \"Sacchetti\", \",\", \"2010\", \")\", \",\", \"and\", \"optogenetic\", \"stimulation\", \"of\", \"artificial\", \"piriform\", \"ensembles\", \"is\", \"sufficient\", \"to\", \"drive\", \"learned\", \"behaviors\", \"(\", \"Choi\", \"et\", \"al.\", \",\", \"2011\", \")\", \".\"]\n",
      "Word token=> [\"Taken\", \"together\", \",\", \"these\", \"studies\", \"have\", \"led\", \"to\", \"the\", \"hypothesis\", \"that\", \"piriform\", \"neural\", \"ensembles\", \"encode\", \"olfactory\", \"memory\", \"traces\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"To\", \"test\", \"this\", \"hypothesis\", \",\", \"we\", \"have\", \"developed\", \"an\", \"intersectional\", \"genetic\", \"strategy\", \"in\", \"mice\", \"to\", \"target\", \"piriform\", \"neurons\", \"that\", \"are\", \"activated\", \"by\", \"olfactory\", \"experience\", \".\"]\n",
      "Word token=> [\"We\", \"employed\"]\n",
      "Word token=> [\"To\", \"selectively\", \"label\", \"and\", \"manipulate\", \"piriform\", \"neurons\", \"that\", \"were\", \"activated\", \"during\", \"odor\", \"exposure\", \",\", \"we\", \"used\", \"cFos-tTA\", \"transgenic\", \"mice\", \"(\", \"Reijmers\", \"et\", \"al.\", \",\", \"2007\", \")\", \",\", \"in\", \"which\", \"the\", \"activity-dependent\", \"cFos\", \"promoter\", \"drives\", \"expression\", \"of\", \"the\", \"tTA\", \"transcription\", \"factor\", \",\", \"and\", \"we\", \"stereotaxically\", \"injected\", \"in\", \"their\", \"piriform\", \"cortex\", \"Adeno-Associated\", \"Viruses\", \"(\", \"AAVs\", \")\", \"expressing\", \"Designer\", \"Receptors\", \"Exclusively\", \"Activated\", \"by\", \"Designer\", \"Drugs\", \"(\", \"DREADDs\", \")\", \"under\", \"the\", \"control\", \"of\", \"the\", \"tTA-responsive\", \"promoter\", \"tetO\", \"(\", \"Figure\", \"1\", \"A\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"DREADDs\", \"(\", \"Alexander\", \"et\", \"al.\", \",\", \"2009\", \";\", \"Ferguson\", \"et\", \"al.\", \",\", \"2011\", \")\", \"increase\", \"(\", \"hM\", \"3\", \"Dq\", \":\", \"mCherry\", \")\", \"or\", \"decrease\", \"(\", \"HA\", \":\", \"hM\", \"4\", \"Di-IRES-mCitrine\", \")\", \"the\", \"excitability\", \"of\", \"neurons\", \"upon\", \"activation\", \"by\", \"their\", \"ligand\", \"clozapine\", \"N-oxide\", \"(\", \"CNO\", \")\", \".\"]\n",
      "Word token=> [\"Temporal\", \"control\", \"of\", \"DREADD\", \"expression\", \"is\", \"provided\", \"by\", \"doxycycline\", \",\", \"administered\", \"through\", \"the\", \"diet\", \",\", \"which\", \"interferes\", \"with\", \"the\", \"binding\", \"of\", \"tTA\", \"to\", \"tetO\", \"and\", \"thus\", \"suppresses\", \"transgene\", \"expression\", \"(\", \"Gossen\", \"et\", \"al.\", \",\", \"1995\", \")\", \".\"]\n",
      "Word token=> [\"Mice\", \"maintained\", \"in\", \"their\", \"home\", \"cage\", \"on\", \"a\", \"doxycycline-containing\", \"diet\", \"showed\", \"low\", \"basal\", \"DREADD\", \"expression\", \",\", \"detected\", \"by\", \"anti-HA\", \"immunohistochemistry\", \"and\", \"native\", \"mCherry\", \"fluorescence\", \"(\", \"Figure\", \"1\", \"B\", \"and\", \"C\", \")\", \".\"]\n",
      "Word token=> [\"To\", \"induce\", \"DREADD\", \"expression\", \",\", \"mice\", \"were\", \"taken\", \"off\", \"doxycycline\", \"five\", \"days\", \"before\", \"odor\", \"presentation\", \".\"]\n",
      "Word token=> [\"Exposure\", \"to\", \"odor\", \"followed\", \"by\", \"foot\", \"shock\", \"(\", \"see\", \"Experimental\", \"Procedures\", \")\", \"resulted\", \"in\", \"the\", \"Fos-tagging\", \"of\", \"sparse\", \"neural\", \"ensembles\", \"(\", \"median\", \"(\", \"interquartile\", \"range\", \")\", \":\", \"1.6\", \"(\", \"0.3\", \")\", \"%\", \"of\", \"piriform\", \"neurons\", \",\", \"n=\", \"2\", \"for\", \"hM\", \"4\", \"Di\", \";\", \"2.2\", \"(\", \"0.5\", \")\", \"%\", \"of\", \"piriform\", \"neurons\", \",\", \"n=\", \"3\", \"for\", \"hM\", \"3\", \"Dq\", \")\", \",\", \"which\", \"were\", \"broadly\", \"dispersed\", \"throughout\", \"the\", \"anterior\", \"and\", \"posterior\", \"piriform\", \"cortex\", \"(\", \"Figure\", \"1\", \"B\", \"and\", \"C\", \")\", \",\", \"consistent\", \"with\", \"previous\", \"observations\", \"of\", \"endogenous\", \"cFos\", \"expression\", \"(\", \"Datiche\", \"et\", \"al.\", \",\", \"2001\", \";\", \"Illig\", \"and\", \"Haberly\", \",\", \"2003\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"To\", \"test\", \"whether\", \"CNO-mediated\", \"activation\", \"of\", \"DREADDs\", \"in\", \"Fos-tagged\", \"piriform\", \"neurons\", \"alters\", \"their\", \"excitability\", \",\", \"we\", \"performed\", \"whole-cell\", \"recordings\", \"in\", \"acute\", \"brain\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"slices\", \"containing\", \"piriform\", \"cortex\", \".\"]\n",
      "Word token=> [\"DREADD-expressing\", \"neurons\", \"were\", \"identified\", \"based\", \"on\", \"mCitrine\", \"(\", \"HA\", \":\", \"hM\", \"4\", \"Di-IRES-mCitrine\", \")\", \"or\", \"mCherry\", \"(\", \"hM\", \"3\", \"Dq\", \":\", \"mCherry\", \")\", \"fluorescence\", \".\"]\n",
      "Word token=> [\"Figure\", \"1\", \")\", \".\"]\n",
      "Word token=> [\"In\", \"the\", \"absence\", \"of\", \"the\", \"DREADD\", \"ligand\", \"CNO\", \",\", \"the\", \"resting\", \"membrane\", \"potential\", \"of\", \"DREADD-expressing\", \"piriform\", \"neurons\", \"was\", \"indistinguishable\", \"from\", \"DREADD-negative\", \"cells\", \"(\", \"-\", \"71.70\", \"(\", \"4.37\", \")\", \"mV\", \"in\", \"n=\", \"24\", \",\", \"(\", \"4.32\", \")\", \"mV\", \"in\", \"hM\", \"4\", \"Di-expressing\", \"cells\", \",\", \"n=\", \"14\", \",\", \"-\", \"72.56\", \"(\", \"3.92\", \")\", \"mV\", \"in\", \"hM\", \"3\", \"Dq-expressing\", \"cells\", \",\", \"n=\", \"15\", \",\", \"p=\", \"0.631\", \",\", \"Kruskal-Wallis\", \"ANOVA\", \",\", \"Figure\", \"1\", \"D\", \")\", \".\"]\n",
      "Word token=> [\"After\", \"bath\", \"application\", \"of\", \"CNO\", \"(\", \"5\", \"µM\", \")\", \",\", \"the\", \"membrane\", \"properties\", \"selectively\", \"changed\", \"in\", \"DREADD-expressing\", \"cells\", \":\", \"hM\", \"4\", \"Di-expressing\", \"cells\", \"hyperpolarized\", \"(\", \"-\", \"74.51\", \"(\", \"5.17\", \")\", \"mV\", \",\", \"n=\", \"12\", \",\", \"p=\", \"0.007\", \",\", \"Wilcoxon\", \"Signed\", \"Ranks\", \"Test\", \")\", \",\", \"and\", \"hM\", \"3\", \"Dq-expressing\", \"cells\", \"depolarized\", \"(\", \"-\", \"66.95\", \"(\", \"4.12\", \")\", \"mV\", \",\", \"n=\", \"13\", \",\", \"p=\", \"0.001\", \",\", \"Wilcoxon\", \"Signed\", \"Ranks\", \"Test\", \")\", \",\", \"whereas\", \"the\", \"resting\", \"membrane\", \"potential\", \"of\", \"DREADD-negative\", \"cells\", \"remained\", \"unchanged\", \"(\", \"-\", \"70.28\", \"(\", \"3.81\", \")\", \"mV\", \",\", \"n=\", \"15\", \",\", \"p=\", \"0.890\", \",\", \"Wilcoxon\", \"Signed\", \"Ranks\", \"Test\", \",\", \"Figure\", \"1\", \"D\", \")\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"The\", \"relative\", \"changes\", \"were\", \"1.00\", \"(\", \"0.03\", \")\", \"in\", \"DREADD-negative\", \"cells\", \"(\", \"n=\", \"15\", \",\", \"p=\", \"0.890\", \",\", \"One\", \"Sample\", \"Wilcoxon\", \"Signed\", \"Rank\", \"Test\", \")\", \",\", \"1.04\", \"(\", \"0.03\", \")\", \"in\", \"hM\", \"4\", \"Di-expressing\", \"cells\", \",\", \"(\", \"n=\", \"12\", \",\", \"p<\", \"0.001\", \",\", \"One\", \"Sample\", \"Wilcoxon\", \"Signed\", \"Rank\", \"Test\", \")\", \"and\", \"0.91\", \"(\", \"0.09\", \")\", \"in\", \"hM\", \"3\", \"Dq-expressing\", \"cells\", \",\", \"(\", \"n=\", \"13\", \",\", \"p=\", \"0.001\", \",\", \"One\", \"Sample\", \"Wilcoxon\", \"Signed\", \"Rank\", \"Test\", \")\", \",\", \"Figure\", \"1\", \"E\", \")\", \".\"]\n",
      "Word token=> [\"To\", \"determine\", \"the\", \"impact\", \"of\", \"CNO\", \"treatment\", \"on\", \"neuronal\", \"excitability\", \",\", \"we\", \"delivered\", \"depolarizing\", \"current\", \"steps\", \"and\", \"compared\", \"the\", \"number\", \"of\", \"action\", \"potentials\", \"triggered\", \"by\", \"60\", \"pA\", \"step\", \"current\", \"above\", \"action\", \"potential\", \"threshold\", \"(\", \"see\", \"Experimental\", \"Procedures\", \")\", \".\"]\n",
      "Word token=> [\"We\", \"found\", \"that\", \"in\", \"hM\", \"4\", \"Di-expressing\", \"cells\", \",\", \"the\", \"number\", \"of\", \"evoked\", \"action\", \"potentials\", \"significantly\", \"decreased\", \"after\", \"CNO\", \"application\", \"(\", \"3.75\", \"(\", \"1.59\", \")\", \"vs.\", \"0.30\", \"(\", \"3.48\", \")\", \",\", \"n=\", \"10\", \",\", \"p=\", \"0.010\", \",\", \"Wilcoxon\", \"Signed\", \"Ranks\", \"Test\", \")\", \",\", \"whereas\", \"in\", \"hM\", \"3\", \"Dq-expressing\", \"cells\", \",\", \"the\", \"number\", \"of\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"action\", \"potentials\", \"significantly\", \"increased\", \"after\", \"CNO\", \"application\", \"(\", \"8.67\", \"(\", \"9.10\", \")\", \"vs.\", \"19.0\", \"(\", \"27.6\", \")\", \"mV\", \",\", \"n=\", \"11\", \",\", \"p=\", \"0.006\", \",\", \"Wilcoxon\", \"Signed\", \"Ranks\", \"Test\", \")\", \".\"]\n",
      "Word token=> [\"In\", \"contrast\", \",\", \"DREADDnegative\", \"cells\", \"did\", \"not\", \"exhibit\", \"a\", \"significant\", \"change\", \"in\", \"the\", \"number\", \"of\", \"evoked\", \"action\", \"potentials\", \"upon\", \"CNO\", \"application\", \"(\", \"4.33\", \"(\", \"1.67\", \")\", \"vs.\", \"4.67\", \"(\", \"3.00\", \")\", \",\", \"n=\", \"9\", \",\", \"p=\", \"0.672\", \",\", \"Wilcoxon\", \"Signed\", \"Ranks\", \"Test\", \",\", \"Figure\", \"1\", \"F\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"relative\", \"changes\", \"were\", \"1.00\", \"(\", \"0.41\", \")\", \"in\", \"DREADDnegative\", \"cells\", \",\", \"(\", \"n=\", \"9\", \",\", \"p=\", \"0.641\", \",\", \"One\", \"Sample\", \"Wilcoxon\", \"Signed\", \"Rank\", \"Tests\", \")\", \",\", \"0.09\", \"(\", \"0.81\", \")\", \"in\", \"hM\", \"4\", \"Di-expressing\", \"cells\", \"(\", \"n=\", \"10\", \",\", \"p=\", \"0.010\", \",\", \"One\", \"Sample\", \"Wilcoxon\", \"Signed\", \"Rank\", \"Tests\", \")\", \"and\", \"1.81\", \"±\", \"1.09\", \"in\", \"hM\", \"3\", \"Dq-expressing\", \"cells\", \"(\", \"n=\", \"11\", \",\", \"p=\", \"0.004\", \",\", \"One\", \"Sample\", \"Wilcoxon\", \"Signed\", \"Rank\", \"Tests\", \",\", \"Figure\", \"1\", \"G\", \")\", \".\"]\n",
      "Word token=> [\"Taken\", \"together\", \",\", \"these\", \"experiments\", \"show\", \"that\", \"Fos-tagging\", \"during\", \"olfactory\", \"fear\", \"conditioning\", \"marks\", \"a\", \"sparse\", \"and\", \"dispersed\", \"subpopulation\", \"of\", \"piriform\", \"neurons\", \",\", \"and\", \"that\", \"CNO-mediated\", \"activation\", \"of\", \"hM\", \"4\", \"Di\", \"or\", \"hM\", \"3\", \"Dq\", \"expressed\", \"in\", \"Fos-tagged\", \"neurons\", \"selectively\", \"decreases\", \"or\", \"increases\", \"their\", \"excitability\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"If\", \"Fos-tagged\", \"piriform\", \"neurons\", \"that\", \"were\", \"activated\", \"during\", \"olfactory\", \"learning\", \"constitute\", \"an\", \"essential\", \"component\", \"of\", \"an\", \"olfactory\", \"memory\", \"trace\", \",\", \"then\", \"perturbing\", \"the\", \"activity\", \"of\", \"these\", \"ensembles\", \"should\", \"interfere\", \"with\", \"memory\", \"recall\", \".\"]\n",
      "Word token=> [\"To\", \"test\", \"this\", \"prediction\", \",\", \"we\", \"Fostagged\", \"piriform\", \"neurons\", \"during\", \"olfactory\", \"fear\", \"conditioning\", \",\", \"and\", \"then\", \"chemogenetically\", \"silenced\", \"Fos-tagged\", \"ensembles\", \"during\", \"odor\", \"fear\", \"memory\", \"recall\", \".\"]\n",
      "Word token=> [\"cFos-tTA\", \"transgenic\", \"mice\", \"(\", \"\\\"\", \"experimental\", \"group\", \"\\\"\", \",\", \"n=\", \"8\", \")\", \"were\", \"bilaterally\", \"injected\", \"in\", \"piriform\", \"cortex\", \"with\", \"the\", \"AAV-tetO-hM\", \"4\", \"Di\", \"vector\", \".\"]\n",
      "Word token=> [\"Wild-type\", \"mice\", \"injected\", \"with\", \"the\", \"same\", \"virus\", \"served\", \"as\", \"a\", \"control\", \"(\", \"\\\"\", \"control\", \"group\", \"\\\"\", \",\", \"n=\", \"8\", \")\", \".\"]\n",
      "Word token=> [\"During\", \"training\", \",\", \"the\", \"conditioned\", \"odor\", \"(\", \"CS+\", \")\", \"was\", \"presented\", \"at\", \"one\", \"of\", \"the\", \"ends\", \"of\", \"the\", \"conditioning\", \"box\", \"and\", \"paired\", \"with\", \"a\", \"foot\", \"shock\", \",\", \"applied\", \"to\", \"the\", \"side\", \"of\", \"the\", \"box\", \"where\", \"the\", \"odor\", \"was\", \"delivered\", \"(\", \"see\", \"Experimental\", \"author/funder.\", \"All\", \"rights\", \"reserved.\", \"No\", \"reuse\", \"allowed\", \"without\", \"permission.\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"Procedures\", \"and\", \"Figure\", \"2\", \"A\", \")\", \".\"]\n",
      "Word token=> [\"Mice\", \"thus\", \"learned\", \"to\", \"escape\", \"from\", \"the\", \"conditioned\", \"stimulus\", \"(\", \"CS+\", \")\", \"by\", \"running\", \"towards\", \"the\", \"opposite\", \"side\", \"of\", \"the\", \"box\", \".\"]\n",
      "Word token=> [\"Two\", \"control\", \"odors\", \",\", \"whose\", \"presentation\", \"was\", \"not\", \"paired\", \"with\", \"foot\", \"shock\", \"(\", \"CS-\", \")\", \",\", \"were\", \"used\", \"to\", \"entrain\", \"odorspecific\", \"as\", \"opposed\", \"to\", \"generalized\", \"fear\", \"responses\", \"(\", \"Chen\", \"et\", \"al.\", \",\", \"2011\", \")\", \".\"]\n",
      "Word token=> [\"Doxycycline\", \"was\", \"removed\", \"from\", \"the\", \"diet\", \"of\", \"mice\", \"prior\", \"to\", \"training\", \",\", \"to\", \"permit\", \"the\", \"induction\", \"of\", \"hM\", \"4\", \"Di\", \"expression\", \"in\", \"active\", \"Fos-expressing\", \"piriform\", \"neurons\", \"of\", \"mice\", \"of\", \"the\", \"experimental\", \"group\", \".\"]\n",
      "Word token=> [\"Mice\", \"were\", \"returned\", \"to\", \"doxycycline-containing\", \"diet\", \"immediately\", \"after\", \"odor\", \"fear\", \"conditioning\", \",\", \"and\", \"memory\", \"recall\", \"of\", \"the\", \"odor-foot\", \"shock\", \"association\", \"was\", \"tested\", \"three\", \"days\", \"later\", \"by\", \"presenting\", \"the\", \"odors\", \"alone\", \"in\", \"a\", \"new\", \"box\", \"(\", \"Figure\", \"2\", \"B\", \"and\", \"C\", \")\", \".\"]\n",
      "Word token=> [\"All\", \"mice\", \"were\", \"intraperitoneally\", \"(\", \"i.p.\", \")\", \"injected\", \"with\", \"CNO\", \"before\", \"testing\", \",\", \"to\", \"exclude\", \"potential\", \"offtarget\", \"effects\", \"by\", \"CNO\", \"(\", \"Gomez\", \"et\", \"al.\", \",\", \"2017\", \")\", \".\"]\n",
      "Word token=> [\"For\", \"all\", \"mice\", \",\", \"we\", \"verified\", \"viral\", \"gene\", \"expression\", \"by\", \"post\", \"hoc\", \"histological\", \"examination\", \".\"]\n",
      "Word token=> [\"Previous\", \"studies\", \"have\", \"suggested\", \"that\", \"the\", \"posterior\", \"piriform\", \"cortex\", \"is\", \"important\", \"for\", \"associative\", \"memory\", \"encoding\", \"(\", \"Sacco\", \"et\", \"al.\", \",\", \"2010\", \";\", \"Calu\", \"et\", \"al.\", \",\", \"2008\", \";\", \"Haberly\", \"2001\", \")\", \".\"]\n",
      "Word token=> [\"We\", \"therefore\", \"excluded\", \"from\", \"our\", \"analysis\", \"mice\", \"in\", \"which\", \"parts\", \"of\", \"the\", \"posterior\", \"piriform\", \"cortex\", \"were\", \"spared\", \"from\", \"viral\", \"infection\", \"(\", \"Supplementary\", \"Figure\", \"2\", \")\", \".\"]\n",
      "Word token=> [\"We\", \"combined\", \"two\", \"parameters\", \"to\", \"quantify\", \"the\", \"learned\", \"escape\", \"behavior\", \":\", \"the\", \"position\", \"of\", \"the\", \"mouse\", \"in\", \"the\", \"box\", \",\", \"and\", \"the\", \"increase\", \"in\", \"its\", \"maximum\", \"velocity\", \"after\", \"odor\", \"presentation\", \"(\", \"Figure\", \"2\", \"B\", \")\", \".\"]\n",
      "Word token=> [\"These\", \"parameters\", \"were\", \"chosen\", \"because\", \"they\", \"described\", \"well\", \"the\", \"behavioral\", \"response\", \"of\", \"mice\", \"to\", \"the\", \"odor-foot\", \"shock\", \"pairing\", \"during\", \"training\", \"(\", \"Supplementary\", \"Figure\", \"3\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"As\", \"expected\", \",\", \"mice\", \"in\", \"the\", \"control\", \"group\", \"exhibited\", \"robust\", \"escape\", \"behavior\", \"upon\", \"CS+\", \"presentation\", \".\"]\n",
      "Word token=> [\"The\", \"median\", \"position\", \"of\", \"the\", \"mice\", \"in\", \"the\", \"box\", \"at\", \"the\", \"end\", \"of\", \"stimulus\", \"delivery\", \"was\", \"0.74\", \"(\", \"with\", \"0\", \"defined\", \"as\", \"the\", \"site\", \"of\", \"odor\", \"presentation\", \"and\", \"1\", \"as\", \"the\", \"opposite\", \"end\", \"of\", \"the\", \"box\", \")\", \",\", \"and\", \"mice\", \"exhibited\", \"a\", \"1.9\", \"-fold\", \"increase\", \"of\", \"their\", \"maximum\", \"velocity\", \"(\", \"Figure\", \"2\", \"E\", \",\", \"G\", \"and\", \"H\", \")\", \".\"]\n",
      "Word token=> [\"In\", \"contrast\", \",\", \"escape\", \"behavior\", \"was\", \"significantly\", \"reduced\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"in\", \"mice\", \"in\", \"which\", \"Fos-tagged\", \"neurons\", \"were\", \"silenced\", \".\"]\n",
      "Word token=> [\"Their\", \"median\", \"position\", \"in\", \"the\", \"box\", \"at\", \"the\", \"end\", \"of\", \"stimulus\", \"delivery\", \",\", \"(\", \"0.42\", \")\", \",\", \"and\", \"their\", \"1.1\", \"-fold\", \"increase\", \"in\", \"maximum\", \"velocity\", \"were\", \"significantly\", \"lower\", \"than\", \"in\", \"the\", \"control\", \"group\", \"(\", \"Mann-Whitney\", \"test\", \",\", \"U=\", \"7\", \",\", \"p=\", \"0.007\", \"for\", \"both\", \"position\", \"and\", \"velocity\", \",\", \"Figure\", \"2\", \"E\", \",\", \"G\", \"and\", \"H\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"difference\", \"in\", \"the\", \"position\", \"of\", \"mice\", \"after\", \"CS+\", \"and\", \"CS-presentation\", \"was\", \"also\", \"significantly\", \"different\", \"between\", \"the\", \"control\", \"and\", \"experimental\", \"groups\", \"(\", \"Mann-Whitney\", \"test\", \",\", \"U=\", \"13\", \",\", \"p=\", \"0.0499\", \",\", \"Figure\", \"2\", \"F\", \")\", \".\"]\n",
      "Word token=> [\"Differences\", \"in\", \"behavioral\", \"responses\", \"of\", \"cFos-tTA\", \"transgenic\", \"mice\", \"were\", \"specifically\", \"due\", \"to\", \"hM\", \"4\", \"Di\", \"activation\", \",\", \"as\", \"escape\", \"behavior\", \"after\", \"CNO\", \"injection\", \"was\", \"unaffected\", \"in\", \"cFos-tTA\", \"transgenic\", \"mice\", \"injected\", \"with\", \"AAVs\", \"expressing\", \"the\", \"calcium\", \"indicator\", \"GCaMP\", \"(\", \"n=\", \"9\", \",\", \"Supplementary\", \"Figure\", \"4\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"All\", \"mice\", \"were\", \"similarly\", \"close\", \"to\", \"the\", \"odor\", \"port\", \"at\", \"the\", \"onset\", \"of\", \"CS+\", \"odor\", \"presentation\", \"(\", \"Mann-Whitney\", \"test\", \",\", \"U=\", \"28\", \",\", \"p=\", \"0.7209\", \",\", \"Figure\", \"2\", \"D\", \")\", \",\", \"and\", \"had\", \"similar\", \"maximum\", \"velocities\", \"during\", \"the\", \"7\", \"s\", \"time\", \"period\", \"that\", \"preceded\", \"odor\", \"presentation\", \"(\", \"Mann-Whitney\", \"test\", \",\", \"U=\", \"24\", \",\", \"p=\", \"0.4418\", \",\", \"Figure\", \"2\", \"G\", \")\", \".\"]\n",
      "Word token=> [\"These\", \"observations\", \"exclude\", \"any\", \"inherent\", \"differences\", \"between\", \"groups\", \"due\", \"to\", \"how\", \"the\", \"CS+\", \"was\", \"presented\", \".\"]\n",
      "Word token=> [\"Furthermore\", \",\", \"escape\", \"behavior\", \"in\", \"control\", \"mice\", \"was\", \"specific\", \"to\", \"the\", \"CS+\", \",\", \"as\", \"none\", \"of\", \"the\", \"mice\", \"ran\", \"away\", \"from\", \"the\", \"CS-\", \"(\", \"Figure\", \"2\", \"D\", \",\", \"E\", \"and\", \"H\", \")\", \".\"]\n",
      "Word token=> [\"Finally\", \",\", \"we\", \"did\", \"not\", \"observe\", \"any\", \"differences\", \"in\", \"the\", \"overall\", \"mobility\", \"of\", \"mice\", \"throughout\", \"the\", \"entire\", \"testing\", \"session\", \".\"]\n",
      "Word token=> [\"The\", \"median\", \"speed\", \"of\", \"mice\", \"of\", \"all\", \"groups\", \"(\", \"0.68\", \"(\", \"0.28\", \")\", \"cm/s\", \"and\", \"0.46\", \"(\", \"0.14\", \")\", \"cm/s\", \"for\", \"the\", \"control\", \"and\", \"experimental\", \"groups\", \"respectively\", \")\", \"was\", \"not\", \"significantly\", \"different\", \"(\", \"Mann-Whitney\", \"test\", \",\", \"U=\", \"22\", \",\", \"p=\", \"0.3282\", \")\", \",\", \"nor\", \"did\", \"mice\", \"show\", \"a\", \"bias\", \"for\", \"the\", \"left\", \"or\", \"right\", \"side\", \"of\", \"the\", \"testing\", \"box\", \"(\", \"time\", \"spent\", \"on\", \"the\", \"left\", \"side\", \"divided\", \"by\", \"time\", \"spent\", \"on\", \"the\", \"right\", \"side\", \":\", \"1.36\", \"(\", \"0.72\", \")\", \"and\", \"1.51\", \"(\", \"0.52\", \")\", \"for\", \"the\", \"control\", \"and\", \"experimental\", \"groups\", \"respectively\", \",\", \"Mann-Whitney\", \"test\", \",\", \"U=\", \"27\", \",\", \"p=\", \"0.6454\", \")\", \".\"]\n",
      "Word token=> [\"Together\", \",\", \"these\", \"data\", \"suggest\", \"that\", \"Fos-tagged\", \"piriform\", \"ensembles\", \"that\", \"were\", \"activated\", \"during\", \"olfactory\", \"fear\", \"conditioning\", \"are\", \"necessary\", \"for\", \"odor\", \"fear\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"memory\", \"recall\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Even\", \"though\", \"cFos-tTA\", \"transgenic\", \"mice\", \"expressing\", \"hM\", \"4\", \"Di\", \"failed\", \"to\", \"robustly\", \"escape\", \"from\", \"the\", \"CS+\", \",\", \"it\", \"should\", \"be\", \"noted\", \"that\", \"they\", \"behaved\", \"differently\", \"after\", \"CS+\", \"and\", \"CSpresentation\", \".\"]\n",
      "Word token=> [\"They\", \"were\", \"further\", \"away\", \"from\", \"the\", \"odor\", \"port\", \"after\", \"CS+\", \"presentation\", \"than\", \"after\", \"CS-presentation\", \"(\", \"Wilcoxon\", \"matched-pairs\", \"signed\", \"rank\", \"test\", \",\", \"W=-\", \"36\", \",\", \"p=\", \"0.0078\", \",\", \"Figure\", \"2\", \"E\", \")\", \",\", \"and\", \"their\", \"maximum\", \"velocity\", \"ratio\", \"was\", \"significantly\", \"higher\", \"(\", \"Wilcoxon\", \"matched-pairs\", \"signed\", \"rank\", \"test\", \",\", \"W=-\", \"36\", \",\", \"p=\", \"0.0078\", \",\", \"Figure\", \"2\", \"H\", \")\", \".\"]\n",
      "Word token=> [\"This\", \"could\", \"indicate\", \"a\", \"partial\", \"memory\", \"of\", \"the\", \"learned\", \"association\", \",\", \"possibly\", \"due\", \"to\", \"incomplete\", \"silencing\", \"of\", \"Fos-tagged\", \"neurons\", \"in\", \"piriform\", \"cortex\", \",\", \"or\", \"the\", \"existence\", \"of\", \"parallel\", \"neural\", \"pathways\", \"that\", \"partially\", \"compensate\", \"for\", \"the\", \"loss\", \"of\", \"piriform\", \"functions\", \".\"]\n",
      "Word token=> [\"Finally\", \",\", \"silencing\", \"of\", \"the\", \"piriform\", \"cortex\", \"in\", \"only\", \"one\", \"hemisphere\", \"did\", \"not\", \"abolish\", \"the\", \"learned\", \"escape\", \"behavior\", \"to\", \"the\", \"conditioned\", \"odor\", \"(\", \"Supplementary\", \"Figure\", \"5\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"It\", \"is\", \"possible\", \"that\", \"the\", \"chemogenetic\", \"silencing\", \"of\", \"piriform\", \"ensembles\", \"that\", \"were\", \"Fostagged\", \"during\", \"olfactory\", \"fear\", \"conditioning\", \"disrupts\", \"odor\", \"detection\", \"and\", \"discrimination\", \"rather\", \"than\", \"selectively\", \"affecting\", \"odor\", \"fear\", \"memory\", \"recall\", \".\"]\n",
      "Word token=> [\"To\", \"test\", \"this\", \"possibility\", \",\", \"we\", \"monitored\", \"sniffing\", \"behavior\", \"of\", \"mice\", \"during\", \"an\", \"olfactory\", \"habituation/dishabituation\", \"assay\", \"(\", \"Figure\", \"3\", \"A\", \")\", \",\", \"a\", \"well\", \"established\", \"test\", \"for\", \"odor\", \"detection\", \"and\", \"discrimination\", \"(\", \"Coronas-Samano\", \"et\", \"al.\", \",\", \"2016\", \";\", \"Verhagen\", \"et\", \"al.\", \",\", \"2007\", \";\", \"Wesson\", \"et\", \"al.\", \",\", \"2008\", \")\", \".\"]\n",
      "Word token=> [\"Fostagging\", \"of\", \"piriform\", \"neurons\", \"during\", \"olfactory\", \"fear\", \"conditioning\", \"was\", \"performed\", \"as\", \"described\", \"above\", \".\"]\n",
      "Word token=> [\"Three\", \"days\", \"later\", \",\", \"changes\", \"in\", \"sniffing\", \"behavior\", \"in\", \"response\", \"to\", \"odor\", \"exposure\", \"were\", \"tested\", \"in\", \"a\", \"plethysmograph\", \",\", \"while\", \"hM\", \"4\", \"Di-expressing\", \"piriform\", \"neurons\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"in\", \"cFos-tTA\", \"transgenic\", \"mice\", \"were\", \"silenced\", \"using\", \"i.p.\", \"injection\", \"of\", \"CNO\", \"(\", \"see\", \"Experimental\", \"Procedures\", \")\", \".\"]\n",
      "Word token=> [\"We\", \"observed\", \"that\", \"mice\", \"of\", \"both\", \"experimental\", \"(\", \"n=\", \"8\", \")\", \"and\", \"control\", \"(\", \"n=\", \"6\", \")\", \"groups\", \"increased\", \"their\", \"sniff\", \"frequency\", \"upon\", \"presentation\", \"of\", \"a\", \"novel\", \",\", \"neutral\", \"odor\", \"(\", \"1.6\", \"fold\", \"increase\", \";\", \"Wilcoxon\", \"matched-pairs\", \"signed\", \"rank\", \"test\", \",\", \"W=\", \"21\", \",\", \"p=\", \"0.0312\", \"and\", \"W=\", \"36\", \",\", \"p=\", \"0.0078\", \"for\", \"the\", \"control\", \"and\", \"experimental\", \"groups\", \",\", \"respectively\", \";\", \"Figure\", \"3\", \"B\", \"and\", \"C\", \")\", \".\"]\n",
      "Word token=> [\"Repeated\", \"exposure\", \"to\", \"the\", \"same\", \"odor\", \"(\", \"neutral\", \"odor\", \"or\", \"CS-\", \")\", \"resulted\", \"in\", \"a\", \"decrease\", \"in\", \"the\", \"sniff\", \"frequency\", \",\", \"reflecting\", \"habituation\", \"after\", \"4\", \"consecutive\", \"exposures\", \"(\", \"Figure\", \"3\", \"C\", \"and\", \"D\", \")\", \".\"]\n",
      "Word token=> [\"Finally\", \",\", \"presentation\", \"of\", \"the\", \"CS+\", \"increased\", \"sniff\", \"frequency\", \"(\", \"1.4\", \"fold\", \"increase\", \";\", \"Wilcoxon\", \"matched-pairs\", \"signed\", \"rank\", \"test\", \",\", \"W=\", \"21\", \",\", \"p=\", \"0.0312\", \"and\", \"W=\", \"34\", \",\", \"p=\", \"0.0156\", \"for\", \"the\", \"control\", \"and\", \"experimental\", \"groups\", \",\", \"respectively\", \",\", \"Figure\", \"3\", \"D\", \")\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"Such\", \"changes\", \"in\", \"odor\", \"sampling\", \"behavior\", \"have\", \"been\", \"shown\", \"to\", \"report\", \"the\", \"detection\", \"and\", \"discrimination\", \"of\", \"different\", \"odor\", \"stimuli\", \"(\", \"Coronas-Samano\", \"et\", \"al.\", \",\", \"2016\", \";\", \"Verhagen\", \"et\", \"al.\", \",\", \"2007\", \";\", \"Wesson\", \"et\", \"al.\", \",\", \"2008\", \")\", \".\"]\n",
      "Word token=> [\"Therefore\", \",\", \"these\", \"data\", \"suggest\", \"that\", \"basic\", \"behaviors\", \"characteristic\", \"of\", \"odor\", \"sampling\", \",\", \"detection\", \"and\", \"discrimination\", \"were\", \"unaffected\", \"by\", \"the\", \"silencing\", \"of\", \"piriform\", \"ensembles\", \"that\", \"were\", \"Fostagged\", \"during\", \"olfactory\", \"fear\", \"conditioning\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Odors\", \"activate\", \"unique\", \"but\", \"overlapping\", \"ensembles\", \"of\", \"piriform\", \"neurons\", \"(\", \"Bolding\", \"and\", \"Franks\", \",\", \"2017\", \";\", \"Poo\", \"and\", \"Isaacson\", \",\", \"2009\", \";\", \"Roland\", \"et\", \"al.\", \",\", \"2017\", \";\", \"Stettler\", \"and\", \"Axel\", \",\", \"2009\", \")\", \".\"]\n",
      "Word token=> [\"Therefore\", \",\", \"silencing\", \"neurons\", \"that\", \"respond\", \"to\", \"one\", \"odor\", \"could\", \"partially\", \"interfere\", \"with\", \"the\", \"retrieval\", \"of\", \"information\", \"associated\", \"with\", \"other\", \"odors\", \".\"]\n",
      "Word token=> [\"To\", \"test\", \"this\", \"prediction\", \"we\", \"modified\", \"the\", \"behavioral\", \"protocol\", \"to\", \"separate\", \"Fos-tagging\", \"from\", \"olfactory\", \"fear\", \"conditioning\", \".\"]\n",
      "Word token=> [\"cFos-tTA\", \"transgenic\", \"and\", \"wild-type\", \"control\", \"mice\", \"were\", \"injected\", \"with\", \"AAV-tetO-hM\", \"4\", \"Di\", \",\", \"and\", \"expression\", \"of\", \"hM\", \"4\", \"Di\", \"in\", \"piriform\", \"neurons\", \"was\", \"induced\", \"while\", \"mice\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"were\", \"exposed\", \"to\", \"odor\", \"(\", \"eugenol\", \"or\", \"beta-citronellol\", \")\", \"in\", \"a\", \"neutral\", \"environment\", \",\", \"without\", \"subsequent\", \"foot\", \"shock\", \".\"]\n",
      "Word token=> [\"Fos-tagging\", \"resulted\", \"in\", \"the\", \"labeling\", \"of\", \"sparse\", \"neural\", \"ensembles\", \",\", \"similar\", \"to\", \"those\", \"labeled\", \"during\", \"odor-foot\", \"shock\", \"association\", \"(\", \"1.1\", \"(\", \"0.3\", \")\", \"%\", \"of\", \"piriform\", \"neurons\", \",\", \"n=\", \"3\", \")\", \".\"]\n",
      "Word token=> [\"Mice\", \"were\", \"returned\", \"to\", \"doxycycline-containing\", \"diet\", \"and\", \"trained\", \"two\", \"days\", \"later\", \"to\", \"associate\", \"a\", \"different\", \"odor\", \"(\", \"ethyl-acetate\", \",\", \"CS+\", \")\", \"with\", \"foot\", \"shock\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Behavioral\", \"testing\", \"of\", \"the\", \"learned\", \"escape\", \"behavior\", \"in\", \"response\", \"to\", \"the\", \"CS+\", \"was\", \"performed\", \"one\", \"day\", \"later\", \",\", \"after\", \"i.p.\", \"injection\", \"of\", \"CNO\", \"(\", \"see\", \"Experimental\", \"Procedures\", \"and\", \"Figure\", \"4\", \"A\", \")\", \".\"]\n",
      "Word token=> [\"We\", \"found\", \"that\", \"learned\", \"escape\", \"behavior\", \"of\", \"cFos-tTA\", \"transgenic\", \"mice\", \"expressing\", \"hM\", \"4\", \"Di\", \"(\", \"experimental\", \"group\", \",\", \"n=\", \"12\", \")\", \"was\", \"similar\", \"but\", \"somewhat\", \"attenuated\", \"compared\", \"to\", \"wild-type\", \"controls\", \"(\", \"n=\", \"10\", \")\", \".\"]\n",
      "Word token=> [\"Both\", \"groups\", \"exhibited\", \"an\", \"escape\", \"behavior\", \"after\", \"CS+\", \"presentation\", \",\", \"indicated\", \"by\", \"the\", \"significant\", \"increase\", \"in\", \"the\", \"maximum\", \"velocity\", \"(\", \"Wilcoxon\", \"matched-pairs\", \"signed\", \"rank\", \"test\", \",\", \"W=\", \"55\", \",\", \"p=\", \"0.0020\", \"and\", \"W=\", \"56\", \",\", \"p=\", \"0.0269\", \"for\", \"the\", \"control\", \"and\", \"experimental\", \"groups\", \"respectively\", \";\", \"no\", \"difference\", \"in\", \"the\", \"maximum\", \"velocity\", \"ratio\", \"after/before\", \"between\", \"groups\", \",\", \"Mann-Whitney\", \"test\", \",\", \"U=\", \"35\", \",\", \"p=\", \"0.1071\", \",\", \"Figure\", \"4\", \"E\", \"and\", \"F\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"difference\", \"in\", \"the\", \"position\", \"of\", \"mice\", \"after\", \"CS+\", \"and\", \"CSpresentation\", \"was\", \"also\", \"similar\", \"between\", \"the\", \"control\", \"and\", \"experimental\", \"groups\", \"(\", \"Mann-Whitney\", \"test\", \",\", \"U=\", \"37\", \",\", \"p=\", \"0.1402\", \",\", \"Figure\", \"4\", \"D\", \")\", \".\"]\n",
      "Word token=> [\"However\", \",\", \"we\", \"observed\", \"that\", \"silencing\", \"neutral\", \"odor\", \"representations\", \"somewhat\", \"dampened\", \"the\", \"behavioral\", \"response\", \"to\", \"the\", \"conditioned\", \"stimulus\", \",\", \"as\", \"behavioral\", \"responses\", \"were\", \"less\", \"robust\", \"compared\", \"to\", \"controls\", \"when\", \"quantifying\", \"the\", \"position\", \"of\", \"mice\", \"in\", \"the\", \"box\", \"after\", \"CS+\", \"presentation\", \"(\", \"Mann-Whitney\", \"test\", \",\", \"U=\", \"22\", \",\", \"p=\", \"0.0112\", \",\", \"Figure\", \"4\", \"B\", \"and\", \"C\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"attenuated\", \"behavioral\", \"responses\", \"we\", \"observe\", \"likely\", \"reflect\", \"the\", \"partial\", \"degradation\", \"of\", \"odor\", \"information\", \"as\", \"a\", \"result\", \"of\", \"the\", \"silencing\", \"of\", \"piriform\", \"neurons\", \"responsive\", \"to\", \"both\", \"the\", \"CS+\", \"and\", \"the\", \"neutral\", \"odor\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\"]\n",
      "Word token=> [\"If\", \"piriform\", \"neural\", \"ensembles\", \"that\", \"were\", \"activated\", \"during\", \"olfactory\", \"fear\", \"conditioning\", \"encode\", \"an\", \"odor\", \"fear\", \"memory\", \"trace\", \",\", \"then\", \"reactivation\", \"of\", \"these\", \"neurons\", \"may\", \"be\", \"sufficient\", \"to\", \"trigger\", \"memory\", \"recall\", \".\"]\n",
      "Word token=> [\"To\", \"test\", \"this\", \"prediction\", \",\", \"we\", \"devised\", \"a\", \"modified\", \"behavioral\", \"task\", \"to\", \"assess\", \"memory\", \"recall\", \".\"]\n",
      "Word token=> [\"We\", \"monitored\", \"exploratory\", \"behavior\", \"in\", \"an\", \"open\", \"field\", \"during\", \"ambient\", \"odor\", \"exposure\", \"or\", \"artificial\", \"reactivation\", \"of\", \"Fos-tagged\", \"neurons\", \",\", \"a\", \"test\", \"that\", \"does\", \"not\", \"rely\", \"on\", \"an\", \"escape\", \"behavior\", \"from\", \"an\", \"odor\", \"originating\", \"from\", \"a\", \"spatially\", \"defined\", \"odor\", \"source\", \"(\", \"Figure\", \"5\", \"A\", \")\", \".\"]\n",
      "Word token=> [\"We\", \"first\", \"trained\", \"mice\", \"to\", \"associate\", \"ethyl-acetate\", \",\", \"the\", \"CS+\", \",\", \"with\", \"foot\", \"shock\", \".\"]\n",
      "Word token=> [\"Upon\", \"ambient\", \"exposure\", \"to\", \"the\", \"CS+\", \",\", \"diffused\", \"above\", \"the\", \"open\", \"field\", \",\", \"exploratory\", \"behavior\", \"of\", \"mice\", \"was\", \"significantly\", \"attenuated\", \"compared\", \"to\", \"baseline\", \"exploratory\", \"behavior\", \"measured\", \"one\", \"day\", \"earlier\", \"(\", \"Wilcoxon\", \"matched-pairs\", \"signed\", \"rank\", \"test\", \",\", \"n=\", \"6\", \",\", \"W=-\", \"21\", \",\", \"p=\", \"0.0312\", \")\", \".\"]\n",
      "Word token=> [\"Ethyl-acetate\", \"exposure\", \"did\", \"not\", \"affect\", \"exploratory\", \"behavior\", \"of\", \"mice\", \"that\", \"had\", \"previously\", \"been\", \"exposed\", \"to\", \"ethylacetate\", \"without\", \"foot\", \"shock\", \"(\", \"Wilcoxon\", \"matched-pairs\", \"signed\", \"rank\", \"test\", \",\", \"n=\", \"5\", \",\", \"W=\", \"1\", \",\", \"p>\", \"0.9999\", \")\", \".\"]\n",
      "Word token=> [\"These\", \"data\", \"suggest\", \"that\", \"re-exposure\", \"of\", \"mice\", \"to\", \"the\", \"CS+\", \"causes\", \"an\", \"increase\", \"in\", \"anxiety-like\", \"behavior\", \",\", \"expressed\", \"as\", \"a\", \"decrease\", \"in\", \"exploratory\", \"behavior\", \"in\", \"the\", \"open\", \"field\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"We\", \"next\", \"repeated\", \"the\", \"experiment\", \"in\", \"cFos-tTA\", \"transgenic\", \"mice\", \",\", \"in\", \"which\", \"the\", \"piriform\", \"cortex\", \"had\", \"been\", \"infected\", \"with\", \"the\", \"activating\", \"DREADD\", \"AAV-tetO-hM\", \"3\", \"Dq\", \"(\", \"Supplementary\", \"Figure\", \"6\", \")\", \".\"]\n",
      "Word token=> [\"Instead\", \"of\", \"exposure\", \"to\", \"the\", \"CS+\", \",\", \"we\", \"i.p.\", \"injected\", \"the\", \"DREADD\", \"ligand\", \"CNO\", \"to\", \"reactivate\", \"hM\", \"3\", \"Dq-expressing\", \"piriform\", \"neurons\", \"that\", \"were\", \"Fos-tagged\", \"during\", \"odor-foot\", \"shock\", \"association\", \"(\", \"see\", \"Experimental\", \"Procedures\", \"and\", \"Figure\", \"5\", \"A\", \")\", \".\"]\n",
      "Word token=> [\"We\", \"found\", \"that\", \"CNO-mediated\", \"reactivation\", \",\", \"similar\", \"to\", \"CS+\", \"exposure\", \",\", \"caused\", \"a\", \"significant\", \"decrease\", \"in\", \"exploratory\", \"behavior\", \"of\", \"mice\", \"that\", \"had\", \"undergone\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"olfactory\", \"fear\", \"conditioning\", \"(\", \"Wilcoxon\", \"matched-pairs\", \"signed\", \"rank\", \"test\", \",\", \"n=\", \"8\", \",\", \"W=-\", \"34\", \",\", \"p=\", \"0.0156\", \",\", \"Figure\", \"5\", \"B\", \"and\", \"C\", \")\", \".\"]\n",
      "Word token=> [\"CNO-mediated\", \"reactivation\", \"of\", \"piriform\", \"neurons\", \"that\", \"were\", \"Fos-tagged\", \"during\", \"ethyl-acetate\", \"exposure\", \"without\", \"foot\", \"shock\", \"did\", \"not\", \"affect\", \"exploratory\", \"behavior\", \"of\", \"mice\", \"(\", \"Wilcoxon\", \"matched-pairs\", \"signed\", \"rank\", \"test\", \",\", \"n=\", \"7\", \",\", \"W=-\", \"10\", \",\", \"p=\", \"0.4688\", \",\", \"Figure\", \"5\", \"B\", \"and\", \"C\", \")\", \".\"]\n",
      "Word token=> [\"Taken\", \"together\", \",\", \"these\", \"data\", \"suggest\", \"that\", \"chemogenetic\", \"reactivation\", \"of\", \"piriform\", \"neurons\", \"that\", \"were\", \"active\", \"during\", \"odor-foot\", \"shock\", \"exposure\", \"is\", \"sufficient\", \"to\", \"trigger\", \"fear\", \"memory\", \"recall\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"We\", \"next\", \"asked\", \"whether\", \"artificial\", \"memory\", \"recall\", \"depends\", \"on\", \"the\", \"specificity\", \"of\", \"the\", \"Fostagged\", \"neural\", \"ensemble\", \":\", \"is\", \"piriform\", \"reactivation\", \"sufficient\", \"to\", \"trigger\", \"fear\", \"memory\", \"recall\", \",\", \"as\", \"long\", \"as\", \"it\", \"includes\", \"the\", \"CS+-tagged\", \"ensemble\", \"?\"]\n",
      "Word token=> [\"For\", \"this\", \"purpose\", \",\", \"we\", \"generated\", \"a\", \"synthetic\", \"Fos-tagged\", \"ensemble\", \",\", \"by\", \"sequentially\", \"exposing\", \"mice\", \"to\", \"ethylacetate\", \"paired\", \"with\", \"foot-shock\", \"(\", \"CS+\", \")\", \"and\", \"neutral\", \"odors\", \"(\", \"CS-\", \",\", \"eugenol\", \"and\", \"betacitronellol\", \")\", \"(\", \"Figure\", \"5\", \"D\", \")\", \".\"]\n",
      "Word token=> [\"While\", \"exposure\", \"to\", \"the\", \"CS+\", \"resulted\", \"in\", \"anxiety-like\", \"behavior\", \"similar\", \"to\", \"what\", \"was\", \"observed\", \"in\", \"the\", \"previous\", \"experiment\", \"(\", \"Wilcoxon\", \"matched-pairs\", \"signed\", \"rank\", \"test\", \",\", \"n=\", \"6\", \",\", \"W=-\", \"21\", \",\", \"p=\", \"0.0312\", \",\", \"Figure\", \"5\", \"E\", \"and\", \"F\", \")\", \",\", \"artificial\", \"reactivation\", \"of\", \"synthetic\", \"Fos-tagged\", \"ensemble\", \"did\", \"not\", \"result\", \"in\", \"changes\", \"in\", \"exploratory\", \"behavior\", \"that\", \"would\", \"indicate\", \"memory\", \"recall\", \"(\", \"Wilcoxon\", \"matched-pairs\", \"signed\", \"rank\", \"test\", \",\", \"n=\", \"8\", \",\", \"W=-\", \"4\", \",\", \"p=\", \"0.8438\", \",\", \"Figure\", \"5\", \"E\", \"and\", \"F\", \")\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"These\", \"data\", \"suggest\", \"that\", \"during\", \"artificial\", \"reactivation\", \",\", \"piriform\", \"cortex\", \"cannot\", \"extract\", \"meaningful\", \"information\", \"from\", \"the\", \"synthetic\", \"representation\", \"generated\", \"by\", \"the\", \"sequential\", \"presentation\", \"of\", \"CS+\", \"and\", \"CS-odors\", \".\"]\n",
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\"]\n",
      "Word token=> [\"The\", \"piriform\", \"cortex\", \"has\", \"long\", \"been\", \"thought\", \"to\", \"provide\", \"the\", \"substrate\", \"for\", \"storing\", \"associative\", \"olfactory\", \"memories\", \".\"]\n",
      "Word token=> [\"Several\", \"studies\", \"have\", \"shown\", \"that\", \"piriform\", \"cortex\", \"cells\", \"can\", \"encode\", \"information\", \"that\", \"carries\", \"behavioral\", \"significance\", \"(\", \"Calu\", \"et\", \"al.\", \",\", \"2008\", \",\", \"Mandairon\", \"et\", \"al.\", \",\", \"2014\", \",\", \"Sacco\", \"et\", \"al.\", \",\", \"2010\", \")\", \",\", \"yet\", \"their\", \"functional\", \"relevance\", \"has\", \"never\", \"been\", \"assessed\", \"by\", \"direct\", \"experimental\", \"manipulation\", \".\"]\n",
      "Word token=> [\"Using\", \"a\", \"cFos-dependent\", \",\", \"intersectional\", \"genetic\", \"approach\", \"to\", \"visualize\", \"and\", \"manipulate\", \"piriform\", \"neurons\", \"activated\", \"during\", \"olfactory\", \"fear\", \"conditioning\", \"(\", \"Figure\", \"1\", \")\", \",\", \"we\", \"found\", \"that\", \"chemogenetic\", \"silencing\", \"of\", \"Fos-tagged\", \"piriform\", \"ensembles\", \"abolished\", \"a\", \"learned\", \"odor\", \"escape\", \"behavior\", \"(\", \"Figure\", \"2\", \")\", \".\"]\n",
      "Word token=> [\"Silencing\", \"these\", \"Fos-tagged\", \"ensembles\", \"did\", \"not\", \"alter\", \"basic\", \"odor\", \"detection\", \"and\", \"discrimination\", \"(\", \"Figure\", \"3\", \")\", \".\"]\n",
      "Word token=> [\"Furthermore\", \",\", \"chemogenetic\", \"silencing\", \"of\", \"neurons\", \"responsive\", \"to\", \"a\", \"neutral\", \"odor\", \"only\", \"moderately\", \"attenuated\", \"odor\", \"fear\", \"memory\", \"recall\", \",\", \"suggesting\", \"that\", \"Fos-tagged\", \"piriform\", \"ensembles\", \"are\", \"largely\", \"odor-specific\", \"(\", \"Figure\", \"4\", \")\", \".\"]\n",
      "Word token=> [\"Finally\", \",\", \"specific\", \"chemogenetic\", \"reactivation\", \"of\", \"piriform\", \"ensembles\", \"that\", \"were\", \"active\", \"during\", \"learning\", \"resulted\", \"in\", \"reduced\", \"exploratory\", \"behavior\", \"in\", \"an\", \"open\", \"field\", \"assay\", \",\", \"an\", \"effect\", \"that\", \"was\", \"similarly\", \"observed\", \"when\", \"exposing\", \"mice\", \"to\", \"the\", \"conditioned\", \"odor\", \"stimulus\", \"(\", \"Figure\", \"5\", \")\", \".\"]\n",
      "Word token=> [\"Together\", \",\", \"our\", \"experiments\", \"identify\", \"piriform\", \"neurons\", \"that\", \"were\", \"active\", \"during\", \"learning\", \"as\", \"necessary\", \"and\", \"sufficient\", \"to\", \"trigger\", \"odor\", \"fear\", \"memory\", \"recall\", \".\"]\n",
      "Word token=> [\"This\", \"indicates\", \"the\", \"formation\", \"of\", \"functional\", \"connections\", \"between\", \"Fos-tagged\", \"piriform\", \"cells\", \"and\", \"other\", \"associative\", \"areas\", \"and\", \"firmly\", \"establish\", \"piriform\", \"ensembles\", \"as\", \"an\", \"essential\", \"component\", \"of\", \"odor\", \"fear\", \"memory\", \"traces\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Artificial\", \"reactivation\", \"of\", \"an\", \"olfactory\", \"memory\", \"trace\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"Reactivation\", \"of\", \"neurons\", \"by\", \"CNO-mediated\", \"activation\", \"of\", \"DREADD\", \"receptors\", \"does\", \"not\", \"recapitulate\", \"the\", \"temporal\", \"characteristics\", \"of\", \"piriform\", \"odor\", \"responses\", \"and\", \"their\", \"modulation\", \"by\", \"active\", \"sampling\", \"(\", \"Bolding\", \"and\", \"Franks\", \",\", \"2017\", \";\", \"Miura\", \"et\", \"al.\", \",\", \"2012\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Despite\", \"this\", \",\", \"reactivation\", \"of\", \"Fos-tagged\", \"piriform\", \"ensembles\", \"activated\", \"during\", \"odor-foot\", \"shock\", \"exposure\", \"was\", \"sufficient\", \"to\", \"elicit\", \"a\", \"behavioral\", \"response\", \".\"]\n",
      "Word token=> [\"How\", \"piriform\", \"neural\", \"circuits\", \"and\", \"downstream\", \"target\", \"regions\", \"process\", \"such\", \"artificial\", \"activity\", \"patterns\", \"remains\", \"to\", \"be\", \"determined\", \".\"]\n",
      "Word token=> [\"One\", \"possibility\", \"is\", \"that\", \"despite\", \"the\", \"temporal\", \"limitations\", \"of\", \"memory\", \"trace\", \"reactivation\", \",\", \"piriform\", \"network\", \"mechanisms\", \"can\", \"retrieve\", \"the\", \"perception\", \"of\", \"the\", \"reinforced\", \"conditioned\", \"stimulus\", \".\"]\n",
      "Word token=> [\"Consistent\", \"with\", \"this\", \"model\", \",\", \"recent\", \"studies\", \"have\", \"shown\", \"that\", \"spatial\", \"patterns\", \"of\", \"odor-evoked\", \"activity\", \",\", \"in\", \"the\", \"absence\", \"of\", \"precise\", \"temporal\", \"information\", \",\", \"are\", \"sufficient\", \"to\", \"decode\", \"odor\", \"identity\", \"in\", \"piriform\", \"cortex\", \"(\", \"Bolding\", \"and\", \"Franks\", \",\", \"2017\", \";\", \"Miura\", \"et\", \"al.\", \",\", \"2012\", \";\", \"Roland\", \"et\", \"al.\", \",\", \"2017\", \")\", \".\"]\n",
      "Word token=> [\"Alternatively\", \",\", \"it\", \"is\", \"possible\", \"that\", \"reactivation\", \"generates\", \"a\", \"state\", \"of\", \"stress\", \"and\", \"anxiety\", \",\", \"but\", \"without\", \"evoking\", \"the\", \"perception\", \"of\", \"the\", \"odor\", \".\"]\n",
      "Word token=> [\"Candidate\", \"target\", \"areas\", \"for\", \"the\", \"processing\", \"of\", \"odor-fear\", \"associations\", \"include\", \"the\", \"basolateral\", \"amygdala\", \"and\", \"the\", \"medial\", \"prefrontal\", \"cortex\", \"(\", \"Dejean\", \"et\", \"al.\", \",\", \"2016\", \";\", \"Gore\", \"et\", \"al.\", \",\", \"2015\", \")\", \",\", \"however\", \",\", \"the\", \"relevant\", \"neural\", \"circuit\", \"components\", \"remain\", \"to\", \"be\", \"identified\", \".\"]\n",
      "Word token=> [\"Finally\", \",\", \"reactivating\", \"a\", \"synthetic\", \"ensemble\", \"of\", \"neurons\", \"activated\", \"during\", \"exposure\", \"to\", \"the\", \"reinforced\", \"stimulus\", \"(\", \"CS+\", \")\", \"and\", \"the\", \"nonreinforced\", \"stimuli\", \"(\", \"CS-\", \")\", \"did\", \"not\", \"elicit\", \"a\", \"measurable\", \"behavioral\", \"response\", \".\"]\n",
      "Word token=> [\"This\", \"observation\", \"suggests\", \"that\", \"behaviorally\", \"relevant\", \"information\", \"cannot\", \"be\", \"extracted\", \"from\", \"a\", \"neural\", \"ensemble\", \"representing\", \"conflicting\", \"(\", \"aversive\", \"versus\", \"neutral\", \")\", \"information\", \".\"]\n",
      "Word token=> [\"This\", \"result\", \"is\", \"consistent\", \"with\", \"the\", \"finding\", \"that\", \"the\", \"reactivation\", \"of\", \"an\", \"artificial\", \"contextual\", \"memory\", \"in\", \"the\", \"hippocampus\", \"competes\", \"with\", \"the\", \"retrieval\", \"of\", \"a\", \"learned\", \"context-shock\", \"association\", \"(\", \"Ramirez\", \"et\", \"al.\", \",\", \"2013\", \")\", \".\"]\n",
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\"]\n",
      "Word token=> [\"A\", \"general\", \"limitation\", \"of\", \"the\", \"cFos-tTA\", \"system\", \"is\", \"that\", \"the\", \"tagging\", \"of\", \"active\", \"neurons\", \"is\", \"transient\", \".\"]\n",
      "Word token=> [\"The\", \"duration\", \"of\", \"the\", \"expression\", \"of\", \"tTA-dependent\", \"proteins\", \"is\", \"determined\", \"by\", \"the\", \"time\", \"course\", \"of\", \"induction\", \"and\", \"the\", \"stability\", \"of\", \"cFos-tTA-dependent\", \"transcripts\", \"and\", \"proteins\", \"in\", \"tagged\", \"neurons\", \".\"]\n",
      "Word token=> [\"Previous\", \"studies\", \"have\", \"shown\", \"that\", \"TetTag-dependent\", \"expression\", \"of\", \"regulators\", \"of\", \"neural\", \"activity\", \"lasts\", \"for\", \"at\", \"least\", \"five\", \"days\", \"but\", \"becomes\", \"undetectable\", \"by\", \"thirty\", \"days\", \"(\", \"Cowansage\", \"et\", \"al.\", \",\", \"2014\", \";\", \"Liu\", \"et\", \"al.\", \",\", \"2012\", \";\", \"Zhang\", \"et\", \"al.\", \",\", \"2015\", \")\", \".\"]\n",
      "Word token=> [\"We\", \"therefore\", \"restricted\", \"our\", \"manipulations\", \"of\", \"neural\", \"activity\", \"and\", \"behavioral\", \"analyses\", \"to\", \"a\", \"three-day\", \"time\", \"window\", \"after\", \"Fos-tagging\", \"of\", \"neurons\", \"during\", \"olfactory\", \"conditioning\", \".\"]\n",
      "Word token=> [\"Another\", \"constraint\", \"of\", \"the\", \"Fos-tagging\", \"system\", \"is\", \"its\", \"slow\", \"temporal\", \"dynamics\", \".\"]\n",
      "Word token=> [\"As\", \"a\", \"consequence\", \",\", \"neurons\", \"responding\", \"to\", \"the\", \"reinforced\", \"conditioned\", \"stimulus\", \"(\", \"CS+\", \")\", \",\", \"the\", \"odorant\", \"paired\", \"with\", \"foot\", \"shock\", \",\", \"and\", \"neurons\", \"responding\", \"to\", \"the\", \"non-reinforced\", \"conditioned\", \"stimuli\", \"(\", \"CS-\", \")\", \",\", \"the\", \"odorants\", \"not\", \"associated\", \"with\", \"foot\", \"shock\", \",\", \"were\", \"tagged\", \"in\", \"our\", \"silencing\", \"experiments\", \".\"]\n",
      "Word token=> [\"The\", \"use\", \"of\", \"a\", \"CS-compromises\", \"the\", \"specificity\", \"of\", \"the\", \"Fos-tagged\", \"neural\", \"ensembles\", \".\"]\n",
      "Word token=> [\"Nevertheless\", \",\", \"the\", \"CS-provides\", \"an\", \"important\", \"experimental\", \"readout\", \"for\", \"the\", \"odor\", \"selectivity\", \"of\", \"the\", \"behavioral\", \"response\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Interestingly\", \",\", \"the\", \"number\", \"of\", \"Fos-tagged\", \"piriform\", \"neurons\", \"we\", \"observe\", \"is\", \"significantly\", \"lower\", \"than\", \"the\", \"number\", \"of\", \"odor-responsive\", \"neurons\", \"detected\", \"in\", \"electrophysiological\", \"recordings\", \"in\", \"awake\", \"mice\", \"(\", \"Bolding\", \"and\", \"Franks\", \",\", \"2017\", \";\", \"Miura\", \"et\", \"al.\", \",\", \"2012\", \";\", \"Zhan\", \"and\", \"Luo\", \",\", \"2010\", \")\", \".\"]\n",
      "Word token=> [\"However\", \",\", \"in\", \"these\", \"experiments\", \",\", \"a\", \"large\", \"fraction\", \"of\", \"cells\", \"exhibited\", \"low\", \"firing\", \"rates\", \".\"]\n",
      "Word token=> [\"Therefore\", \",\", \"the\", \"Fos-tagged\", \"population\", \"could\", \"represent\", \"a\", \"subpopulation\", \"of\", \"neurons\", \"that\", \"is\", \"strongly\", \"activated\", \"by\", \"odor\", \"(\", \"Schoenenberger\", \"et\", \"al.\", \",\", \"2009\", \")\", \".\"]\n",
      "Word token=> [\"Fos-tagging\", \"could\", \"also\", \"mark\", \"plastic\", \"changes\", \"supporting\", \"memory\", \"formation\", \"(\", \"Cole\", \"et\", \"al.\", \",\", \"1989\", \";\", \"Minatohara\", \"et\", \"al.\", \",\", \"2015\", \")\", \".\"]\n",
      "Word token=> [\"Indeed\", \",\", \"cFos\", \"mRNA\", \"levels\", \"decrease\", \"significantly\", \"after\", \"injection\", \"of\", \"an\", \"antagonist\", \"of\", \"the\", \"NMDA\", \"receptor\", \",\", \"a\", \"key\", \"player\", \"in\", \"the\", \"induction\", \"of\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"synaptic\", \"plasticity\", \"(\", \"Tayler\", \"et\", \"al.\", \",\", \"2011\", \")\", \",\", \"and\", \"long-term\", \"memory\", \"and\", \"synaptic\", \"plasticity\", \"are\", \"impaired\", \"when\", \"cFos\", \"production\", \"is\", \"perturbed\", \"in\", \"the\", \"central\", \"nervous\", \"system\", \"(\", \"Fleischmann\", \"et\", \"al.\", \",\", \"2003\", \";\", \"Seoane\", \"et\", \"al.\", \",\", \"2012\", \")\", \".\"]\n",
      "Word token=> [\"Future\", \"studies\", \"will\", \"allow\", \"to\", \"directly\", \"test\", \"these\", \"hypotheses\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"hippocampus\", \"has\", \"been\", \"studied\", \"extensively\", \"for\", \"its\", \"role\", \"in\", \"spatial\", \"and\", \"contextual\", \"memory\", \"(\", \"Basu\", \"and\", \"Siegelbaum\", \",\", \"2015\", \")\", \".\"]\n",
      "Word token=> [\"Recently\", \",\", \"Fos-tagging\", \"of\", \"hippocampal\", \"neurons\", \"during\", \"contextual\", \"fear\", \"conditioning\", \"has\", \"provided\", \"important\", \"insight\", \"into\", \"the\", \"cellular\", \"and\", \"neural\", \"circuit\", \"mechanisms\", \"of\", \"learning\", \"and\", \"memory\", \"(\", \"Cai\", \"et\", \"al.\", \",\", \"2016\", \";\", \"Garner\", \"et\", \"al.\", \",\", \"2012\", \";\", \"Liu\", \"et\", \"al.\", \",\", \"2012\", \";\", \"Ramirez\", \"et\", \"al.\", \",\", \"2013\", \";\", \"Roy\", \"et\", \"al.\", \",\", \"2017\", \";\", \"Ryan\", \"et\", \"al.\", \",\", \"2015\", \")\", \".\"]\n",
      "Word token=> [\"However\", \",\", \"if\", \"principles\", \"of\", \"memory\", \"formation\", \"and\", \"storage\", \"in\", \"hippocampusrelated\", \"neural\", \"networks\", \"apply\", \"to\", \"other\", \"cortical\", \"structures\", \"remains\", \"largely\", \"unknown\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Interestingly\", \",\", \"pirform\", \"cortex\", \"and\", \"hippocampus\", \"share\", \"a\", \"similar\", \"circuit\", \"organization\", \"and\", \"both\", \"have\", \"been\", \"modeled\", \"as\", \"auto-associative\", \"networks\", \"(\", \"Haberly\", \",\", \"2001\", \")\", \".\"]\n",
      "Word token=> [\"Consistent\", \"with\", \"this\", \"model\", \",\", \"our\", \"findings\", \"reveal\", \"striking\", \"similarities\", \"between\", \"olfactory\", \"memory\", \"traces\", \"in\", \"the\", \"piriform\", \"cortex\", \"and\", \"memory\", \"traces\", \"of\", \"contextual\", \"fear\", \"in\", \"the\", \"hippocampus\", \".\"]\n",
      "Word token=> [\"In\", \"both\", \"piriform\", \"cortex\", \"and\", \"hippocampus\", \",\", \"learning\", \"activates\", \"sparse\", \"and\", \"distributed\", \"ensembles\", \"of\", \"neurons\", \"that\", \"appear\", \"to\", \"lack\", \"topographic\", \"organization\", \".\"]\n",
      "Word token=> [\"Furthermore\", \",\", \"hippocampal\", \"neurons\", \"tagged\", \"during\", \"contextual\", \"fear\", \"conditioning\", \",\", \"and\", \"piriform\", \"neurons\", \"tagged\", \"during\", \"olfactory\", \"fear\", \"conditioning\", \"were\", \"both\", \"necessary\", \"and\", \"sufficient\", \"for\", \"memory\", \"retrieval\", \"(\", \"Liu\", \"et\", \"al.\", \",\", \"2012\", \";\", \"Tanaka\", \"et\", \"al.\", \",\", \"2014\", \")\", \",\", \"The\", \"olfactory\", \"system\", \"is\", \"particularly\", \"well-suited\", \"for\", \"studying\", \"learning\", \"and\", \"memory\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"Indeed\", \",\", \"odor\", \"memories\", \"and\", \"olfactory-driven\", \"behaviors\", \"are\", \"robust\", \"in\", \"animals\", \",\", \"including\", \"in\", \"mice\", \",\", \"providing\", \"highly\", \"reliable\", \"and\", \"quantitative\", \"behavioral\", \"readouts\", \".\"]\n",
      "Word token=> [\"Furthermore\", \",\", \"the\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"olfactory\", \"cortex\", \"is\", \"only\", \"two\", \"synapses\", \"away\", \"from\", \"the\", \"peripheral\", \"olfactory\", \"sensory\", \"neurons\", \"in\", \"the\", \"olfactory\", \"epithelium\", \",\", \"and\", \"the\", \"neural\", \"inputs\", \"that\", \"drive\", \"piriform\", \"cortex\", \"activity\", \"have\", \"been\", \"well\", \"characterized\", \"(\", \"Banerjee\", \"et\", \"al.\", \",\", \"2015\", \";\", \"Dhawale\", \"et\", \"al.\", \",\", \"2010\", \";\", \"Economo\", \"et\", \"al.\", \",\", \"2016\", \";\", \"Fukunaga\", \"et\", \"al.\", \",\", \"2012\", \";\", \"Kato\", \"et\", \"al.\", \",\", \"2012\", \";\", \"Roland\", \"et\", \"al.\", \",\", \"2016\", \";\", \"Yamada\", \"et\", \"al.\", \",\", \"2017\", \")\", \".\"]\n",
      "Word token=> [\"Knowing\", \"the\", \"properties\", \"of\", \"neural\", \"input\", \"patterns\", \"will\", \"provide\", \"the\", \"basis\", \"for\", \"understanding\", \"neural\", \"circuit\", \"input-output\", \"computations\", \"and\", \"their\", \"modification\", \"by\", \"learning\", \"and\", \"experience\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\"]\n",
      "Word token=> [\"Mice\", \"were\", \"housed\", \"at\", \"24\", \"°\", \"C\", \"with\", \"a\", \"12\", \"-hour\", \"light/\", \"12\", \"-hour\", \"dark\", \"cycle\", \"with\", \"standard\", \"food\", \"and\", \"water\", \"provided\", \"ad\", \"libitum\", \".\"]\n",
      "Word token=> [\"Mice\", \"were\", \"group-housed\", \"with\", \"littermates\", \"until\", \"the\", \"beginning\", \"of\", \"surgery\", \"and\", \"then\", \"single-housed\", \"in\", \"ventilated\", \"cages\", \"throughout\", \"the\", \"duration\", \"of\", \"the\", \"experiment\", \".\"]\n",
      "Word token=> [\"cFos-tTA\", \"mice\", \"(\", \"Reijmers\", \"et\", \"al.\", \",\", \"2007\", \")\", \"were\", \"fed\", \"a\", \"diet\", \"containing\", \"1\", \"g/kg\", \"doxycycline\", \"for\", \"a\", \"minimum\", \"of\", \"4\", \"days\", \"before\", \"surgery\", \".\"]\n",
      "Word token=> [\"Wild-type\", \"control\", \"animals\", \"were\", \"siblings\", \"that\", \"did\", \"not\", \"carry\", \"the\", \"cFos-tTA\", \"transgene\", \".\"]\n",
      "Word token=> [\"The\", \"age\", \"of\", \"mice\", \"at\", \"the\", \"time\", \"of\", \"behavioral\", \"testing\", \"ranged\", \"from\", \"10\", \"to\", \"15\", \"weeks\", \".\"]\n",
      "Word token=> [\"Male\", \"mice\", \"were\", \"used\", \"for\", \"the\", \"hM\", \"4\", \"Di-mediated\", \"neural\", \"silencing\", \",\", \"and\", \"females\", \"for\", \"the\", \"hM\", \"3\", \"Dq-mediated\", \"neural\", \"activation\", \"experiments\", \".\"]\n",
      "Word token=> [\"Experiments\", \"were\", \"carried\", \"out\", \"according\", \"to\", \"European\", \"and\", \"French\", \"national\", \"institutional\", \"animal\", \"care\", \"guidelines\", \"(\", \"protocol\", \"APAFIS\", \"#\", \"2016012909576100\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"Adeno-associated\", \"viruses\", \"(\", \"AAVs\", \")\", \"were\", \"generated\", \"at\", \"Penn\", \"Vector\", \"Core\", \",\", \"University\", \"of\", \"Pennsylvania\", \"(\", \"serotype\", \"8\", \",\", \"10\", \"13\", \"genome\", \"copies/mL\", \",\", \"1:2\", \"dilution\", \"with\", \"sterile\", \"PBS\", \"on\", \"the\", \"day\", \"of\", \"injection\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\"]\n",
      "Word token=> [\"Mice\", \"were\", \"anaesthetized\", \"with\", \"ketamine/xylazine\", \"(\", \"100\", \"mg/kg/\", \"10\", \"mg/kg\", \",\", \"Sigma-Aldrich\", \")\", \"and\", \"AAV\", \"vectors\", \"were\", \"injected\", \"stereotaxically\", \"into\", \"the\", \"piriform\", \"cortex\", \"Sections\", \"were\", \"washed\", \"2\", \"times\", \"in\", \"PBST\", \"and\", \"1\", \"time\", \"in\", \"PBS\", \",\", \"mounted\", \"on\", \"slides\", \"and\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"coverslipped\", \"with\", \"Vectashield\", \"mounting\", \"medium\", \"(\", \"Vectorlabs\", \")\", \".\"]\n",
      "Word token=> [\"Images\", \"were\", \"acquired\", \"as\", \"Z-stacks\", \"(\", \"70\", \"to\", \"140\", \"µm\", \"in\", \"total\", \"thickness\", \",\", \"step\", \"size\", \"7\", \"µm\", \")\", \"with\", \"a\", \"Leica\", \"SP\", \"5\", \"confocal\", \"microscope\", \"or\", \"as\", \"single\", \"plane\", \"sections\", \"with\", \"a\", \"Zeiss\", \"Axio\", \"Zoom\", \"microscope\", \"and\", \"processed\", \"in\", \"Fiji\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Counting\", \"Z-stacks\", \"(\", \"7\", \"µm\", \"step\", \")\", \"were\", \"acquired\", \"with\", \"a\", \"Leica\", \"SP\", \"5\", \"confocal\", \"microscope\", \"with\", \"a\", \"20\", \"X\", \"objective\", \"and\", \"a\", \"resolution\", \"of\", \"512\", \"x\", \"512\", \"or\", \"1024\", \"x\", \"1024\", \"pixels\", \"(\", \"1\", \"pixel\", \":\", \"0:72\", \"µm\", \")\", \".\"]\n",
      "Word token=> [\"First\", \",\", \"the\", \"piriform\", \"cortex\", \"was\", \"delineated\", \"on\", \"a\", \"maximum\", \"projection\", \"of\", \"each\", \"stack\", \"using\", \"a\", \"custom-written\", \"ImageJ\", \"macro\", \".\"]\n",
      "Word token=> [\"For\", \"each\", \"mouse\", \",\", \"2\", \"to\", \"4\", \"sections\", \"(\", \"median\", \"volume\", \"9.3\", \"e-\", \"2\", \"(\", \"9.4\", \"e-\", \"3\", \")\", \"mm\", \"3\", \")\", \"were\", \"analyzed\", \",\", \"from\", \"one\", \"or\", \"both\", \"hemispheres\", \",\", \"at\", \"Y=-\", \"0.8\", \"and\", \"Y=-\", \"1.6\", \"mm\", \"relative\", \"to\", \"bregma\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"HA-stained\", \"neurons\", \"(\", \"hM\", \"4\", \"Di\", \")\", \"were\", \"counted\", \"by\", \"hand\", \".\"]\n",
      "Word token=> [\"hM\", \"3\", \"Dq-mCherry\", \"positive\", \"neurons\", \"were\", \"counted\", \"using\", \"a\", \"custom-written\", \"ImageJ\", \"plugin\", \".\"]\n",
      "Word token=> [\"After\", \"pre-processing\", \"the\", \"image\", \"(\", \"Subtract\", \"Background\", \",\", \"Remove\", \"Outliers\", \"and\", \"Median\", \"Filter\", \")\", \",\", \"the\", \"z-stack\", \"is\", \"thresholded\", \"using\", \"the\", \"RenyiEntropy\", \"algorithm\", \".\"]\n",
      "Word token=> [\"On\", \"each\", \"slice\", \"of\", \"the\", \"stack\", \",\", \"objects\", \"with\", \"an\", \"area\", \"smaller\", \"than\", \"40\", \"µm\", \"2\", \"are\", \"removed\", \"using\", \"the\", \"Analyze\", \"Particle\", \"command\", \"in\", \"ImageJ\", \".\"]\n",
      "Word token=> [\"A\", \"mask\", \"of\", \"each\", \"slice\", \"from\", \"the\", \"stack\", \"is\", \"created\", \"containing\", \"the\", \"filled\", \"outline\", \"of\", \"the\", \"measured\", \"particles\", \".\"]\n",
      "Word token=> [\"The\", \"number\", \"of\", \"cells\", \"is\", \"the\", \"number\", \"of\", \"3\", \"D\", \"objects\", \"detected\", \"in\", \"the\", \"stack\", \".\"]\n",
      "Word token=> [\"Automated\", \"cell\", \"detection\", \"can\", \"be\", \"prone\", \"to\", \"error\", \",\", \"thus\", \",\", \"the\", \"detection\", \"efficiency\", \"was\", \"checked\", \"on\", \"a\", \"few\", \"pictures\", \"by\", \"two\", \"people\", \"blind\", \"to\", \"the\", \"experimental\", \"conditions\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"numbers\", \"of\", \"cells\", \"were\", \"calculated\", \"as\", \"numbers\", \"of\", \"cells\", \"per\", \"cubic\", \"millimeter\", \".\"]\n",
      "Word token=> [\"An\", \"estimate\", \"of\", \"the\", \"total\", \"number\", \"of\", \"neurons\", \"(\", \"counterstained\", \"with\", \"NeuroTrace\", \")\", \"per\", \"cubic\", \"millimeter\", \"(\", \"7.1\", \"e\", \"4\", \"mm\", \"-\", \"3\", \")\", \"was\", \"obtained\", \"by\", \"counting\", \"neurons\", \"in\", \"representative\", \"volumes\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"containing\", \"the\", \"three\", \"layers\", \"of\", \"piriform\", \"cortex\", \".\"]\n",
      "Word token=> [\"Numbers\", \"are\", \"consistent\", \"with\", \"those\", \"obtained\", \"by\", \"(\", \"Srinivasan\", \"and\", \"Stevens\", \",\", \"2017\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Parasagittal\", \"or\", \"coronal\", \"slices\", \"(\", \"300\", \"µm\", \"thick\", \")\", \"of\", \"piriform\", \"cortex\", \"were\", \"prepared\", \"from\", \"6\", \"-\", \"8\", \"week-old\", \"cFos-tTA\", \"mice\", \"injected\", \"with\", \"AAV-tetO-hM\", \"4\", \"Di-IRES-mCitrine\", \"or\", \"AAV-tetO-hM\", \"3\", \"Dq-mCherry\", \".\"]\n",
      "Word token=> [\"Animals\", \"were\", \"anaesthetized\", \"with\", \"ketamine\", \"and\", \"xylazine\", \"(\", \"100\", \"mg/kg/\", \"10\", \"mg/kg\", \",\", \"Sigma-Aldrich\", \")\", \",\", \"perfused\", \"with\", \"ice-cold\", \"ACSF\", \"(\", \"125\", \"mM\", \"NaCl\", \",\", \"2.5\", \"mM\", \"KCl\", \",\", \"25\", \"mM\", \"glucose\", \"25\", \"mM\", \"NaHCO\", \"3\", \",\", \"1.25\", \"mM\", \"NaH\", \"2\", \"PO\", \"4\", \",\", \"2\", \"mM\", \"CaCl\", \"2\", \",\", \"1\", \"mM\", \"MgCl\", \"2\", \",\", \"1\", \"mM\", \"pyruvic\", \"acid\", \",\", \"bubbled\", \"with\", \"95\", \"%\", \"O\", \"2\", \"and\", \"5\", \"%\", \"CO\", \"2\", \"and\", \"adjusted\", \"to\", \"295\", \"±\", \"5\", \"mOsm\", \"osmolarity\", \")\", \",\", \"and\", \"decapitated\", \".\"]\n",
      "Word token=> [\"The\", \"brain\", \"was\", \"cooled\", \"with\", \"ice-cold\", \"ACSF\", \"solution\", \"and\", \"then\", \"sliced\", \"using\", \"a\", \"7000\", \"SM\", \"2\", \"vibrating\", \"microtome\", \"(\", \"Campden\", \"Instruments\", \",\", \"UK\", \")\", \".\"]\n",
      "Word token=> [\"Slices\", \"were\", \"incubated\", \"in\", \"the\", \"same\", \"solution\", \"at\", \"34\", \"°\", \"C\", \"for\", \"one\", \"hour\", \"and\", \"then\", \"continuously\", \"perfused\", \"with\", \"ACSF\", \"solution\", \"(\", \"2\", \"mL/min\", \")\", \"at\", \"34\", \"°\", \"C\", \"in\", \"the\", \"recording\", \"chamber\", \".\"]\n",
      "Word token=> [\"DREADD\", \"expression\", \"was\", \"detected\", \"with\", \"two-photon\", \"excitation\", \"(\", \"830\", \"nm\", \",\", \"ChameleonMRU-X\", \"1\", \",\", \"Coherent\", \",\", \"UK\", \")\", \"under\", \"a\", \"Scientifica\", \"TriM\", \"Scope\", \"II\", \"microscope\", \"(\", \"LaVision\", \",\", \"Germany\", \")\", \",\", \"with\", \"a\", \"60\", \"x\", \"water-immersion\", \"objective\", \".\"]\n",
      "Word token=> [\"Whole-cell\", \"recording\", \"pipettes\", \"with\", \"5\", \"-\", \"7\", \"MΩ\", \"resistance\", \"were\", \"filled\", \"with\", \"the\", \"following\", \"solution\", \"(\", \"in\", \"mM\", \")\", \":\", \"122\", \"Kgluconate\", \",\", \"13\", \"KCl\", \",\", \"10\", \"HEPES\", \",\", \"10\", \"phosphocreatine\", \",\", \"4\", \"Mg-ATP\", \",\", \"0.3\", \"Na-GTP\", \",\", \"0.3\", \"EGTA\", \"(\", \"adjusted\", \"to\", \"pH\", \"7.35\", \"with\", \"KOH\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"recording\", \"solution\", \"also\", \"contained\", \"the\", \"morphological\", \"tracer\", \"Alexa\", \"Fluor\", \"594\", \"(\", \"5\", \"µM\", \",\", \"red\", \"channel\", \")\", \"for\", \"nonexpressing\", \"and\", \"hM\", \"4\", \"Di-positive\", \"cells\", \"or\", \"the\", \"Ca\", \"2\", \"+\", \"-sensitive\", \"dye\", \"Fura-\", \"2\", \"(\", \"300\", \"µM\", \",\", \"replacing\", \"EGTA\", \"in\", \"the\", \"recording\", \"solution\", \",\", \"green\", \"channel\", \")\", \"for\", \"hM\", \"3\", \"Dq-positive\", \"cells\", \",\", \"to\", \"enable\", \"identification\", \"of\", \"the\", \"patched\", \"cell\", \"and\", \"to\", \"visualize\", \"its\", \"morphology\", \".\"]\n",
      "Word token=> [\"The\", \"excitability\", \"of\", \"cells\", \"was\", \"measured\", \"in\", \"current-clamp\", \"mode\", \"by\", \"500\", \"ms\", \"steps\", \"of\", \"current\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"injections\", \"from\", \"-\", \"300\", \"to\", \"+\", \"500\", \"pA\", \"with\", \"steps\", \"of\", \"20\", \"pA\", \".\"]\n",
      "Word token=> [\"We\", \"compared\", \"the\", \"number\", \"of\", \"action\", \"potentials\", \"triggered\", \"by\", \"equivalent\", \"depolarization\", \"at\", \"each\", \"step\", \"60\", \"pA\", \"above\", \"the\", \"action\", \"potential\", \"threshold\", \"before\", \"and\", \"after\", \"CNO\", \"(\", \"5\", \"µM\", \")\", \"application\", \".\"]\n",
      "Word token=> [\"The\", \"series\", \"resistance\", \"was\", \"usually\", \"<\", \"20\", \"MΩ\", \",\", \"and\", \"data\", \"were\", \"discarded\", \"if\", \"it\", \"changed\", \"by\", \"more\", \"than\", \"20\", \"%\", \"during\", \"the\", \"recording\", \".\"]\n",
      "Word token=> [\"Signals\", \"were\", \"amplified\", \"using\", \"EPC\", \"10\", \"-\", \"2\", \"amplifiers\", \"(\", \"HEKA\", \"Elektronik\", \",\", \"Lambrecht\", \",\", \"Germany\", \")\", \".\"]\n",
      "Word token=> [\"Voltage-clamp\", \"recordings\", \"were\", \"filtered\", \"at\", \"5\", \"kHz\", \"and\", \"sampled\", \"at\", \"10\", \"kHz\", \",\", \"and\", \"current-clamp\", \"recordings\", \"were\", \"filtered\", \"at\", \"10\", \"kHz\", \"and\", \"sampled\", \"at\", \"20\", \"kHz\", \",\", \"with\", \"the\", \"Patchmaster\", \"v\", \"2\", \"x\", \"32\", \"program\", \"(\", \"HEKA\", \"Elektronik\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"A\", \"training\", \"box\", \"was\", \"used\", \"to\", \"train\", \"mice\", \"to\", \"escape\", \"from\", \"an\", \"odor\", \".\"]\n",
      "Word token=> [\"The\", \"box\", \"was\", \"rectangular\", \"(\", \"L\", \"57\", \"cm\", \",\", \"W\", \"17\", \"cm\", \",\", \"H\", \"64\", \"cm\", \")\", \",\", \"with\", \"a\", \"grid\", \"floor\", \"made\", \"of\", \"72\", \"stainless-steel\", \"rods\", \"(\", \"diameter=\", \"6\", \"mm\", \",\", \"space\", \"between\", \"rods=\", \"2\", \"mm\", \")\", \".\"]\n",
      "Word token=> [\"Current\", \"was\", \"delivered\", \"by\", \"an\", \"aversive\", \"stimulator\", \"(\", \"MedAssociates\", \",\", \"115\", \"V\", \",\", \"60\", \"Hz\", \")\", \".\"]\n",
      "Word token=> [\"A\", \"custom-made\", \"switcher\", \"allowed\", \"an\", \"electric\", \"foot\", \"shock\", \"(\", \"0.6\", \"mA\", \",\", \"0.6\", \"ms\", \")\", \"to\", \"be\", \"applied\", \"independently\", \"to\", \"either\", \"half\", \"of\", \"the\", \"box\", \".\"]\n",
      "Word token=> [\"A\", \"testing\", \"box\", \"was\", \"used\", \"to\", \"test\", \"memory\", \"retrieval\", \".\"]\n",
      "Word token=> [\"Its\", \"dimensions\", \"were\", \"similar\", \"to\", \"the\", \"training\", \"box\", \".\"]\n",
      "Word token=> [\"Materials\", \"for\", \"the\", \"walls\", \"and\", \"floor\", \"were\", \"different\", \"from\", \"the\", \"training\", \"box\", \"to\", \"create\", \"a\", \"different\", \"context\", \".\"]\n",
      "Word token=> [\"The\", \"open\", \"field\", \"box\", \"consisted\", \"of\", \"a\", \"white\", \"Plexiglas\", \"(\", \"6\", \"mm\", \"thickness\", \")\", \"container\", \"(\", \"50\", \"cm\", \"x\", \"50\", \"cm\", \"x\", \"38\", \"height\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Sniffing\", \"behavior\", \"was\", \"monitored\", \"in\", \"freely\", \"moving\", \"mice\", \"using\", \"a\", \"plethysmograph\", \"(\", \"Emka\", \"Technologies\", \")\", \".\"]\n",
      "Word token=> [\"Electrical\", \"signals\", \"were\", \"amplified\", \"using\", \"RHD\", \"2132\", \"amplifier\", \"boards\", \"(\", \"Intan\", \"Technologies\", \")\", \"and\", \"band-pass\", \"filtered\", \"(\", \"1\", \"-\", \"30\", \"Hz\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Odors\", \"were\", \"delivered\", \"using\", \"an\", \"8\", \"channels\", \"olfactometer\", \"(\", \"Automate\", \"Scientific\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Continuous\", \"air\", \"flowed\", \"through\", \"a\", \"bottle\", \"of\", \"mineral\", \"oil\", \"(\", \"Sigma-Aldrich\", \")\", \"to\", \"avoid\", \"pressure\", \"changes\", \"when\", \"presenting\", \"the\", \"odors\", \".\"]\n",
      "Word token=> [\"The\", \"following\", \"odors\", \"(\", \"Sigma-Aldrich\", \")\", \",\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"at\", \"a\", \"concentration\", \"of\", \"1\", \"%\", \"(\", \"vol/vol\", \")\", \"in\", \"mineral\", \"oil\", \"were\", \"used\", \":\", \"ethyl-acetate\", \",\", \"betacitronellol\", \",\", \"eugenol\", \",\", \"limonene\", \",\", \"and\", \"pinene\", \".\"]\n",
      "Word token=> String[]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"AAVs\", \"were\", \"stereotaxically\", \"injected\", \"in\", \"both\", \"hemispheres\", \"of\", \"piriform\", \"cortex\", \"of\", \"cFos-tTA\", \"transgenic\", \"mice\", \"and\", \"littermate\", \"wild-type\", \"controls\", \".\"]\n",
      "Word token=> [\"Mice\", \"were\", \"given\", \"10\", \"days\", \"to\", \"recover\", \",\", \"before\", \"being\", \"habituated\", \"on\", \"day\", \"11\", \"to\", \"the\", \"training\", \"and\", \"testing/open\", \"field/plethysmograph\", \"boxes\", \"(\", \"40\", \"and\", \"10\", \"min\", \"each\", \")\", \".\"]\n",
      "Word token=> [\"24\", \"hours\", \"after\", \"habituation\", \",\", \"doxycycline\", \"was\", \"removed\", \"and\", \"replaced\", \"by\", \"a\", \"normal\", \"diet\", \".\"]\n",
      "Word token=> [\"Five\", \"days\", \"later\", \",\", \"mice\", \"were\", \"fear\", \"conditioned\", \"to\", \"ethyl-acetate\", \"or\", \"exposed\", \"to\", \"an\", \"odor\", \"(\", \"odor\", \"\\\"\", \"C\", \"\\\"\", \")\", \"in\", \"a\", \"neutral\", \"environment\", \".\"]\n",
      "Word token=> [\"Immediately\", \"after\", \"fear\", \"conditioning\", \"or\", \"odor\", \"presentation\", \",\", \"mice\", \"were\", \"put\", \"back\", \"on\", \"a\", \"diet\", \"containing\", \"doxycycline\", \".\"]\n",
      "Word token=> [\"During\", \"fear\", \"conditioning\", \",\", \"ethyl-acetate\", \"(\", \"the\", \"CS+\", \")\", \"was\", \"presented\", \"at\", \"one\", \"extremity\", \"of\", \"the\", \"training\", \"box\", \",\", \"and\", \"the\", \"corresponding\", \"halfside\", \"of\", \"the\", \"box\", \"'\", \"floor\", \"was\", \"electrified\", \"for\", \"0.6\", \"s\", \"(\", \"0.6\", \"mA\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"foot\", \"shock\", \"was\", \"delivered\", \"4\", \"seconds\", \"after\", \"CS+\", \"presentation\", \",\", \"and\", \"the\", \"CS+\", \"lasted\", \"for\", \"an\", \"additional\", \"2\", \"seconds\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Mice\", \"learned\", \"to\", \"escape\", \"to\", \"the\", \"opposite\", \"side\", \"when\", \"the\", \"CS+\", \"was\", \"presented\", \".\"]\n",
      "Word token=> [\"The\", \"CS+\", \"was\", \"presented\", \"4\", \"times\", \",\", \"2\", \"times\", \"on\", \"each\", \"side\", \"of\", \"the\", \"box\", \".\"]\n",
      "Word token=> [\"When\", \"fear\", \"conditioning\", \"included\", \"presentation\", \"of\", \"two\", \"non-reinforced\", \"conditioned\", \"stimuli\", \"(\", \"CS-\", \"1\", \"and\", \"CS-\", \"2\", \")\", \",\", \"the\", \"choice\", \"between\", \"CS+\", \"and\", \"CS-as\", \"well\", \"as\", \"the\", \"side\", \"of\", \"presentation\", \"was\", \"pseudorandomized\", \".\"]\n",
      "Word token=> [\"Training\", \"always\", \"started\", \"with\", \"one\", \"presentation\", \"of\", \"each\", \"CS-followed\", \"by\", \"the\", \"CS+\", \".\"]\n",
      "Word token=> [\"The\", \"two\", \"CS-odors\", \"were\", \"presented\", \"during\", \"7\", \"seconds\", \",\", \"3\", \"times\", \"each\", \",\", \"with\", \"a\", \"total\", \"presentation\", \"of\", \"3\", \"on\", \"the\", \"right\", \"side\", \"and\", \"3\", \"on\", \"the\", \"left\", \"side\", \".\"]\n",
      "Word token=> [\"During\", \"testing\", \",\", \"the\", \"CS+\", \"was\", \"presented\", \"4\", \"times\", \"(\", \"2\", \"times\", \"on\", \"each\", \"side\", \")\", \",\", \"and\", \"each\", \"CS-was\", \"presented\", \"once\", \"on\", \"each\", \"side\", \",\", \"for\", \"7\", \"s\", \"each\", \".\"]\n",
      "Word token=> [\"As\", \"for\", \"the\", \"training\", \"session\", \",\", \"the\", \"order\", \"and\", \"side\", \"of\", \"odor\", \"presentations\", \"was\", \"pseudo-randomized\", \":\", \"testing\", \"always\", \"started\", \"with\", \"one\", \"presentation\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"of\", \"each\", \"CS-\", \",\", \"followed\", \"by\", \"one\", \"CS+\", \"presentation\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Clozapine-N-oxide\", \"(\", \"C\", \"0832\", \",\", \"Sigma-Aldrich\", \")\", \"was\", \"first\", \"dissolved\", \"in\", \"dimethylsulfoxide\", \"(\", \"DMSO\", \",\", \"D\", \"2438\", \",\", \"Sigma\", \",\", \"final\", \"concentration\", \"of\", \"1\", \"%\", \"vol/vol\", \")\", \",\", \"then\", \"further\", \"diluted\", \"in\", \"0.9\", \"%\", \"sterile\", \"saline\", \"solution\", \"to\", \"a\", \"final\", \"concentration\", \"of\", \"0.2\", \"mg/mL\", \".\"]\n",
      "Word token=> [\"Aliquots\", \"were\", \"stored\", \"for\", \"up\", \"to\", \"two\", \"months\", \"at\", \"-\", \"20\", \"°\", \"C\", \"and\", \"equilibrated\", \"to\", \"room\", \"temperature\", \"before\", \"injection\", \".\"]\n",
      "Word token=> [\"The\", \"solution\", \"was\", \"administered\", \"intraperitoneally\", \"(\", \"3\", \"mg/kg\", \")\", \".\"]\n",
      "Word token=> [\"After\", \"injection\", \",\", \"mice\", \"were\", \"left\", \"undisturbed\", \"for\", \"25\", \"minutes\", \"(\", \"Figure\", \"2\", \"and\", \"4\", \")\", \"or\", \"5\", \"minutes\", \"(\", \"Figure\", \"3\", \"and\", \"5\", \")\", \"in\", \"their\", \"home\", \"cage\", \"before\", \"the\", \"start\", \"of\", \"the\", \"experiment\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Electrophysiological\", \"data\", \"were\", \"analyzed\", \"with\", \"custom-made\", \"software\", \"in\", \"Python\", \"3.0\", \"and\", \"averaged\", \"in\", \"MS\", \"Excel\", \"(\", \"Microsoft\", \",\", \"USA\", \")\", \".\"]\n",
      "Word token=> [\"Behavioral\", \"sessions\", \"were\", \"video\", \"recorded\", \"and\", \"analyzed\", \"using\", \"custom-written\", \"MatLab\", \"programs\", \"(\", \"available\", \"upon\", \"request\", \")\", \".\"]\n",
      "Word token=> [\"Parameters\", \"extracted\", \"from\", \"multiple\", \"CS+\", \"(\", \"CS-\", \")\", \"presentation\", \"were\", \"averaged\", \"for\", \"each\", \"mouse\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Statistical\", \"analysis\", \"was\", \"performed\", \"with\", \"Prism\", \"(\", \"GraphPad\", \")\", \".\"]\n",
      "Word token=> [\"Non-parametric\", \"tests\", \"were\", \"used\", \":\", \"Mann-Whitney\", \"test\", \"for\", \"between\", \"group\", \"comparison\", \",\", \"and\", \"Wilcoxon\", \"matchedpairs\", \"signed\", \"rank\", \"test\", \"for\", \"within\", \"group\", \"comparison\", \".\"]\n",
      "Word token=> [\"Values\", \"are\", \"represented\", \"as\", \"median\", \"(\", \"interquartile\", \"range\", \")\", \"unless\", \"otherwise\", \"stated\", \".\"]\n",
      "Word token=> [\"The\", \"n\", \"represents\", \"number\", \"of\", \"mice\", \",\", \"or\", \"number\", \"of\", \"recorded\", \"neurons\", \"for\", \"the\", \"electrophysiology\", \"experiment\", \".\"]\n",
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"before\", \"and\", \"after\", \"CNO\", \"application\", \".\"]\n",
      "Word token=> [\"After\", \"CNO\", \"application\", \",\", \"the\", \"number\", \"of\", \"APs\", \"in\", \"DREADD-negative\", \"cells\", \"(\", \"Ctrl\", \")\", \"does\", \"not\", \"change\", \",\", \"significantly\", \"decreases\", \"in\", \"hM\", \"4\", \"Diexpressing\", \"cells\", \"and\", \"significantly\", \"increases\", \"in\", \"hM\", \"3\", \"Dq-expressing\", \"cells\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"(\", \"G\", \")\", \"The\", \"relative\", \"change\", \"in\", \"the\", \"number\", \"of\", \"APs\", \"after/before\", \"CNO\", \"in\", \"DREADD-negative\", \"cells\", \"(\", \"Ctrl\", \")\", \"does\", \"not\", \"change\", \",\", \"significantly\", \"decreases\", \"in\", \"hM\", \"4\", \"Di-expressing\", \"cells\", \"and\", \"significantly\", \"increases\", \"in\", \"hM\", \"3\", \"Dq-expressing\", \"cells\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Data\", \"are\", \"shown\", \"as\", \"individual\", \"data\", \"points\", \",\", \"median\", \"and\", \"interquartile\", \"range\", \",\", \"*\", \"*\", \"p<\", \"0.01\", \".\"]\n",
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"A\", \")\", \"During\", \"fear\", \"conditioning\", \",\", \"mice\", \"are\", \"trained\", \"to\", \"associate\", \"ethyl-acetate\", \"(\", \"CS+\", \")\", \"with\", \"foot\", \"shock\", \".\"]\n",
      "Word token=> [\"The\", \"CS+\", \"is\", \"presented\", \"at\", \"one\", \"end\", \"of\", \"the\", \"training\", \"box\", \",\", \"and\", \"the\", \"corresponding\", \"half-side\", \"of\", \"the\", \"box\", \"is\", \"electrified\", \".\"]\n",
      "Word token=> [\"Training\", \"includes\", \"the\", \"presentation\", \"of\", \"two\", \"other\", \"odors\", \"that\", \"are\", \"not\", \"paired\", \"with\", \"foot\", \"shock\", \"(\", \"CS-\", \",\", \"eugenol\", \"and\", \"beta-citronellol\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"CS+\", \"is\", \"presented\", \"4\", \"times\", \"(\", \"2\", \"times\", \"on\", \"each\", \"side\", \")\", \"and\", \"each\", \"CS-three\", \"times\", \".\"]\n",
      "Word token=> [\"The\", \"order\", \"and\", \"side\", \"of\", \"odor\", \"presentations\", \"is\", \"pseudo-randomized\", \".\"]\n",
      "Word token=> [\"B\", \")\", \"During\", \"testing\", \",\", \"the\", \"CS+\", \"is\", \"presented\", \"4\", \"times\", \"(\", \"2\", \"times\", \"on\", \"each\", \"side\", \")\", \",\", \"and\", \"each\", \"CSis\", \"presented\", \"once\", \"on\", \"each\", \"side\", \",\", \"for\", \"7\", \"s\", \"each\", \".\"]\n",
      "Word token=> [\"The\", \"testing\", \"session\", \"is\", \"video-recorded\", \"and\", \"the\", \"position\", \"of\", \"the\", \"mouse\", \"'\", \"s\", \"centroid\", \"in\", \"the\", \"box\", \"along\", \"the\", \"x-axis\", \"is\", \"extracted\", \"(\", \"top\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"length\", \"of\", \"the\", \"box\", \"is\", \"normalized\", \"to\", \"1\", \".\"]\n",
      "Word token=> [\"Since\", \"the\", \"odor\", \"is\", \"presented\", \"randomly\", \"on\", \"the\", \"left\", \"or\", \"right\", \"side\", \"of\", \"the\", \"testing\", \"box\", \",\", \"0\", \"is\", \"defined\", \"as\", \"the\", \"extremity\", \"where\", \"the\", \"odor\", \"is\", \"presented\", \".\"]\n",
      "Word token=> [\"For\", \"velocity\", \"measurements\", \",\", \"a\", \"7\", \"s\", \"time\", \"window\", \"is\", \"defined\", \"both\", \"before\", \"and\", \"after\", \"stimulus\", \"onset\", \"(\", \"bottom\", \")\", \".\"]\n",
      "Word token=> [\"C\", \")\", \"Experimental\", \"design\", \":\", \"cFos-tTA\", \"transgenic\", \"mice\", \"(\", \"experimental\", \"group\", \",\", \"n=\", \"8\", \")\", \"or\", \"wildtype\", \"mice\", \"(\", \"control\", \"group\", \",\", \"n=\", \"8\", \")\", \"are\", \"injected\", \"with\", \"AAV-tetO-hM\", \"4\", \"Di\", \"in\", \"both\", \"hemispheres\", \"of\", \"piriform\", \"cortex\", \".\"]\n",
      "Word token=> [\"Ten\", \"days\", \"later\", \",\", \"mice\", \"are\", \"habituated\", \"to\", \"the\", \"training\", \"and\", \"testing\", \"boxes\", \".\"]\n",
      "Word token=> [\"Upon\", \"doxycycline\", \"(\", \"DOX\", \")\", \"removal\", \",\", \"mice\", \"are\", \"fear-conditioned\", \",\", \"and\", \"hM\", \"4\", \"Di\", \"expression\", \"(\", \"green\", \"circles\", \")\", \"is\", \"induced\", \"in\", \"Fos-tagged\", \"neurons\", \"(\", \"filled\", \"in\", \"blue\", \"and\", \"yellow\", \")\", \".\"]\n",
      "Word token=> [\"Mice\", \"are\", \"returned\", \"to\", \"DOX-containing\", \"diet\", \"to\", \"avoid\", \"further\", \"expression\", \"of\", \"hM\", \"4\", \"Di\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"Three\", \"days\", \"later\", \",\", \"all\", \"mice\", \"are\", \"i.p.\", \"injected\", \"with\", \"CNO\", \",\", \"which\", \"silences\", \"hM\", \"4\", \"Diexpressing\", \"neurons\", \"(\", \"black\", \"crosses\", \")\", \",\", \"and\", \"memory\", \"retrieval\", \"is\", \"tested\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"(\", \"D\", \"to\", \"H\", \")\", \"Olfactory\", \"memory\", \"retrieval\", \"after\", \"CNO\", \"injection\", \",\", \"assessed\", \"by\", \"the\", \"position\", \"of\", \"mice\", \"in\", \"the\", \"testing\", \"box\", \"and\", \"their\", \"maximum\", \"velocity\", \"upon\", \"odor\", \"exposure\", \".\"]\n",
      "Word token=> [\"Memory\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"retrieval\", \"was\", \"significantly\", \"impaired\", \"in\", \"cFos-tTA\", \"transgenic\", \"mice\", \"in\", \"which\", \"piriform\", \"neurons\", \"that\", \"were\", \"active\", \"during\", \"learning\", \"were\", \"chemogenetically\", \"silenced\", \"during\", \"memory\", \"recall\", \"(\", \"experimental\", \"group\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"(\", \"D\", \")\", \"Position\", \"of\", \"mice\", \"in\", \"the\", \"testing\", \"box\", \"as\", \"a\", \"function\", \"of\", \"time\", \"after\", \"CS+\", \"(\", \"black\", \")\", \"and\", \"CS-\", \"(\", \"blue\", \")\", \"presentation\", \".\"]\n",
      "Word token=> [\"(\", \"E\", \")\", \"Position\", \"in\", \"the\", \"testing\", \"box\", \"7\", \"s\", \"after\", \"CS+\", \"or\", \"CS-presentation\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"(\", \"F\", \")\", \"Position\", \"7\", \"s\", \"after\", \"CS-presentation\", \"subtracted\", \"from\", \"the\", \"position\", \"7\", \"s\", \"after\", \"CS+\", \"presentation\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"(\", \"G\", \")\", \"Maximum\", \"velocity\", \"during\", \"a\", \"7\", \"s\", \"time\", \"window\", \"before\", \"and\", \"after\", \"CS+\", \"presentation\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"(\", \"H\", \")\", \"Maximum\", \"velocity\", \"ratio\", \"after/before\", \"CS+\", \"(\", \"left\", \")\", \"or\", \"CS-\", \"(\", \"right\", \")\", \"presentation\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Data\", \"are\", \"shown\", \"as\", \"median\", \"and\", \"interquartile\", \"range\", \",\", \"*\", \"p<\", \"0.05\", \";\", \"*\", \"*\", \"p<\", \"0.01\", \";\", \"n.s.\", \"not\", \"significant\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"(\", \"A\", \")\", \"(\", \"Top\", \")\", \"Experimental\", \"design\", \":\", \"Fos-tagging\", \"during\", \"olfactory\", \"fear\", \"conditioning\", \"is\", \"performed\", \"as\", \"described\", \"in\", \"Figure\", \"2\", \".\"]\n",
      "Word token=> [\"Three\", \"days\", \"after\", \"fear\", \"conditioning\", \",\", \"sniffing\", \"behavior\", \"is\", \"quantified\", \"in\", \"a\", \"plethysmograph\", \".\"]\n",
      "Word token=> [\"Mice\", \"are\", \"i.p.\", \"injected\", \"with\", \"CNO\", \",\", \"habituated\", \"to\", \"the\", \"plethysmograph\", \"for\", \"15\", \"min\", \",\", \"and\", \"each\", \"odor\", \"is\", \"then\", \"presented\", \"for\", \"8\", \"s\", \"with\", \"a\", \"2.5\", \"min\", \"inter-trial\", \"interval.\", \"pin\", \":\", \"pinene\", \",\", \"CS-\", \":\", \"beta-citronellol\", \",\", \"CS+\", \":\", \"ethyl-acetate\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"(\", \"Bottom\", \")\", \"Representative\", \"sniff\", \"recordings\", \"from\", \"one\", \"mouse\", \"when\", \"exposed\", \"to\", \"pinene\", \"or\", \"the\", \"CS+\", \"(\", \"ethyl-acetate\", \")\", \".\"]\n",
      "Word token=> [\"Data\", \"are\", \"shown\", \"as\", \"median\", \"and\", \"interquartile\", \"range\", \",\", \"*\", \"p<\", \"0.05\", \";\", \"*\", \"*\", \"p<\", \"0.01\", \";\", \"n.s.\", \":\", \"not\", \"significant\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"Data\", \"are\", \"shown\", \"as\", \"median\", \"and\", \"interquartile\", \"range\", \",\", \"*\", \"p<\", \"0.05\", \";\", \"*\", \"*\", \"p<\", \"0.01\", \";\", \"n.s.\", \"not\", \"significant\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"(\", \"B\", \")\", \"As\", \"a\", \"measure\", \"for\", \"exploratory\", \"activity\", \",\", \"the\", \"distance\", \"travelled\", \"is\", \"computed\", \"from\", \"minute\", \"15\", \"to\", \"18\", \"(\", \"corresponding\", \"to\", \"odor\", \"exposure\", \"during\", \"test\", \"2\", \")\", \".\"]\n",
      "Word token=> [\"For\", \"artificial\", \"memory\", \"recall\", \",\", \"the\", \"distance\", \"travelled\", \"is\", \"computed\", \"over\", \"the\", \"entire\", \"testing\", \"session\", \".\"]\n",
      "Word token=> [\"Ethylacetate\", \"exposure\", \"and\", \"CNO\", \"injection\", \"lead\", \"to\", \"a\", \"decrease\", \"in\", \"exploratory\", \"behavior\", \"in\", \"mice\", \"previously\", \"fear\", \"conditioned\", \"to\", \"ethyl-acetate\", \"(\", \"n=\", \"6\", \"and\", \"n=\", \"8\", \"respectively\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Previous\", \"exposure\", \"to\", \"ethyl-acetate\", \"without\", \"foot\", \"shock\", \"does\", \"not\", \"alter\", \"exploratory\", \"behavior\", \"upon\", \"re-exposure\", \"to\", \"ethyl-acetate\", \"or\", \"re-activation\", \"of\", \"Fos-tagged\", \"piriform\", \"ensembles\", \"(\", \"n=\", \"5\", \"and\", \"n=\", \"7\", \"respectively\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"(\", \"C\", \")\", \"Distance\", \"traveled\", \"ratio\", \"test\", \"2\", \"/test\", \"1\", \".\"]\n",
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\"]\n",
      "Word token=> [\"Training\", \"Testing\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"baseline\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"297226\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \".\"]\n",
      "Word token=> [\"SARS-CoV-\", \"2\", \",\", \"also\", \"known\", \"as\", \"2019\", \"-nCoV\", \",\", \"is\", \"a\", \"novel\", \"coronavirus\", \"(\", \"CoVs\", \")\", \"isolated\", \"from\", \"patients\", \"with\", \"pneumonia\", \"in\", \"China\", \"2019\", \".\"]\n",
      "Word token=> [\"SARS-CoV-\", \"2\", \"has\", \"a\", \"similar\", \"incubation\", \"period\", \"(\", \"median\", \",\", \"3.0\", \"days\", \")\", \"and\", \"a\", \"relatively\", \"lower\", \"fatality\", \"rate\", \"than\", \"SARS-CoV\", \"or\", \"MERS-CoV\", \"(\", \"1\", \")\", \",\", \"but\", \"it\", \"is\", \"estimated\", \"that\", \"the\", \"reproductive\", \"number\", \"of\", \"SARS-CoV-\", \"2\", \"is\", \"higher\", \"than\", \"that\", \"of\", \"SARS-CoV\", \"(\", \"2\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"What\", \"'\", \"s\", \"more\", \",\", \"some\", \"laboratory-confirmed\", \"symptomatic\", \"cases\", \"are\", \"absent\", \"of\", \"apparent\", \"cough\", \",\", \"fever\", \"or\", \"radiologic\", \"manifestations\", \",\", \"making\", \"it\", \"difficult\", \"to\", \"timely\", \"and\", \"accurately\", \"find\", \"out\", \"all\", \"infected\", \"patients\", \"(\", \"3\", \",\", \"4\", \")\", \".\"]\n",
      "Word token=> [\"As\", \"of\", \"February\", \"28\", \",\", \"2020\", \",\", \"patients\", \"infected\", \"by\", \"SARS-CoV-\", \"2\", \"have\", \"been\", \"diagnosed\", \"in\", \"more\", \"than\", \"30\", \"countries\", \",\", \"and\", \"more\", \"than\", \"78,000\", \"confirmed\", \"cases\", \"and\", \"2,700\", \"deaths\", \"associate\", \"with\", \"SARS-CoV-\", \"2\", \"infection\", \"are\", \"reported\", \"in\", \"China\", \"alone\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"genetic\", \"information\", \"of\", \"a\", \"virus\", \"is\", \"essential\", \"for\", \"not\", \"only\", \"its\", \"classification\", \"and\", \"traceability\", \",\", \"but\", \"also\", \"its\", \"pathogenicity\", \".\"]\n",
      "Word token=> [\"At\", \"the\", \"whole\", \"genome\", \"level\", \",\", \"the\", \"sequence\", \"identify\", \"of\", \"SARS-CoV-\", \"2\", \"was\", \"50\", \"%\", \"to\", \"MERS-CoV\", \",\", \"79\", \"%\", \"to\", \"SARS-CoV\", \",\", \"88\", \"%\", \"to\", \"two\", \"bat-derived\", \"SARS-like\", \"coronaviruses\", \",\", \"Bat-SL-CoVZC\", \"45\", \"and\", \"Bat-SL-CoVZXC\", \"21\", \"(\", \"collected\", \"in\", \"2018\", \"in\", \"Zhoushan\", \",\", \"China\", \")\", \",\", \"and\", \"96\", \"%\", \"to\", \"Bat-SARSr-CoV\", \"RaTG\", \"13\", \"(\", \"collected\", \"in\", \"2013\", \"in\", \"Yunnan\", \",\", \"China\", \")\", \"(\", \"5\", \")\", \".\"]\n",
      "Word token=> [\"Each\", \"genome\", \"of\", \"all\", \"SARS-CoV-\", \"2\", \"strains\", \"now\", \"submitted\", \"online\", \"contains\", \"nearly\", \"29,900\", \"nucleotides\", \"(\", \"nt\", \")\", \",\", \"which\", \"are\", \"predicted\", \"with\", \"at\", \"least\", \"14\", \"open\", \"reading\", \"frames\", \"(\", \"ORFs\", \")\", \"(\", \"5\", \"′\", \"to\", \"3\", \"′\", \")\", \",\", \"such\", \"as\", \"ORF\", \"1\", \"ab\", \"(\", \"P\", \",\", \"21,291\", \"nt\", \")\", \",\", \"spike\", \"(\", \"S\", \",\", \"3,822\", \"nt\", \")\", \",\", \"ORF\", \"3\", \"a\", \"(\", \"8,28\", \"nt\", \")\", \",\", \"envelope\", \"(\", \"E\", \",\", \"228\", \"nt\", \")\", \",\", \"membrane\", \"(\", \"M\", \",\", \"669\", \"nt\", \")\", \",\", \"ORF\", \"8\", \"(\", \"366\", \"nt\", \")\", \",\", \"and\", \"nucleocapsid\", \"(\", \"N\", \",\", \"1,260\", \"nt\", \")\", \"(\", \"6\", \")\", \".\"]\n",
      "Word token=> [\"Among\", \"them\", \",\", \"the\", \"spike\", \"gene\", \"encoded\", \"a\", \"glycoprotein\", \"that\", \"is\", \"crucial\", \"for\", \"determining\", \"host\", \"tropism\", \"and\", \"transmission\", \"capacity\", \",\", \"and\", \"highly\", \"divergent\", \"when\", \"compared\", \"with\", \"that\", \"of\", \"bat-SARSr-CoV\", \"RaTG\", \"13\", \"(\", \"93.1\", \"%\", \"nucleotide\", \"identity\", \")\", \"(\", \"6\", \",\", \"7\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Generally\", \",\", \"the\", \"rates\", \"of\", \"nucleotide\", \"substitution\", \"of\", \"RNA\", \"viruses\", \"are\", \"faster\", \"than\", \"their\", \"hosts\", \",\", \"and\", \".\"]\n",
      "Word token=> [\"CC-BY-ND\", \"4.0\", \"International\", \"license\", \"author/funder\", \".\"]\n",
      "Word token=> [\"It\", \"is\", \"made\", \"available\", \"under\", \"a\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.02.27.969006\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"this\", \"rapid\", \"evolution\", \"is\", \"mainly\", \"shaped\", \"by\", \"natural\", \"selection\", \"(\", \"mostly\", \"purifying\", \"selection\", \")\", \"(\", \"8\", \")\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"Gene\", \"mutations\", \"such\", \"as\", \"nucleotide\", \"substitutions\", \",\", \"deletions\", \"and\", \"insertions\", \"have\", \"been\", \"frequently\", \"reported\", \"when\", \"comparing\", \"SARS-CoV-\", \"2\", \"with\", \"other\", \"viruses\", \"(\", \"5\", \")\", \"(\", \"6\", \")\", \"(\", \"7\", \")\", \".\"]\n",
      "Word token=> [\"In\", \"this\", \"work\", \",\", \"we\", \"investigated\", \"the\", \"mutation\", \"pattern\", \"of\", \"SARS-CoV-\", \"2\", \"by\", \"comprehensive\", \"comparative\", \"genomic\", \"analysis\", \"of\", \"the\", \"nonsynonymous/\", \"synonymous\", \"substitution\", \",\", \"relative\", \"synonymous\", \"codon\", \"usage\", \"(\", \"RSCU\", \")\", \"and\", \"selective\", \"pressure\", \"in\", \"order\", \"to\", \"explore\", \"their\", \"potential\", \"in\", \"evolution\", \"and\", \"function\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"SARS-CoV-\", \"2\", \"reference\", \"genome\", \"Wuhan-Hu-\", \"1\", \"(\", \"NC\", \"_\", \"045512\", \")\", \"and\", \"WIV\", \"04\", \"(\", \"MN\", \"996528\", \")\", \"was\", \"downloaded\", \"from\", \"the\", \"GenBank\", \"database\", \".\"]\n",
      "Word token=> [\"Other\", \"newly\", \"sequenced\", \"2019\", \"-nCoV\", \"genomes\", \"were\", \"downloaded\", \"from\", \"the\", \"Global\", \"Initiative\", \"on\", \"Sharing\", \"Avian\", \"Influenza\", \"Data\", \"(\", \"GISAID\", \")\", \"database\", \"(\", \"https://www.gisaid.org/\", \")\", \".\"]\n",
      "Word token=> [\"Twenty-one\", \"closely\", \"related\", \"coronavirus\", \"complete\", \"genome\", \"sequences\", \"and\", \"their\", \"coding\", \"sequences\", \"(\", \"CDS\", \")\", \"were\", \"downloaded\", \"from\", \"GenBank\", \"database\", \"(\", \"Supplementary\", \"Table\", \"S\", \"1\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Genome\", \"sequences\", \"were\", \"aligned\", \"using\", \"MUSCLE\", \"v\", \"3.8.31\", \"(\", \"9\", \")\", \",\", \"followed\", \"by\", \"manual\", \"adjustment\", \"using\", \"BioEdit\", \"v\", \"7.2.5\", \".\"]\n",
      "Word token=> [\"Phylogenetic\", \"analyses\", \"of\", \"complete\", \"genome\", \"were\", \"performed\", \"using\", \"maximum-likelihood\", \"method\", \"and\", \"general\", \"time-reversible\", \"model\", \"of\", \"nucleotide\", \"substitution\", \"with\", \"gamma-distributed\", \"rates\", \"among\", \"sites\", \"(\", \"GTR+G\", \")\", \"in\", \"RAxML\", \"v\", \"8.1.21\", \"(\", \"10\", \")\", \".\"]\n",
      "Word token=> [\"Support\", \"for\", \"the\", \"inferred\", \"relationships\", \"was\", \"evaluated\", \"by\", \"a\", \"bootstrap\", \"analysis\", \"with\", \"1000\", \"replicates\", \"and\", \"trees\", \"were\", \"rooted\", \"with\", \"the\", \"alpha-coronavirus\", \"lineage\", \"as\", \"an\", \"outgroup\", \".\"]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.02.27.969006\", \"doi\", \":\", \"bioRxiv\", \"preprint\"]\n",
      "Word token=> [\"The\", \"coding\", \"sequences\", \"were\", \"translated\", \"and\", \"aligned\", \"using\", \"the\", \"MEGA-X\", \"program\", \"(\", \"11\", \")\", \",\", \"and\", \"then\", \"codon-based\", \"sequence\", \"alignment\", \"was\", \"used\", \"for\", \"further\", \"analysis\", \".\"]\n",
      "Word token=> [\"Phylogenetic\", \"analyses\", \"of\", \"coding\", \"sequences\", \"were\", \"performed\", \"using\", \"MEGA-X\", \"software\", \".\"]\n",
      "Word token=> [\"The\", \"changes\", \"of\", \"amino\", \"acids\", \"or\", \"nucleotides\", \"for\", \"each\", \"CDS\", \"sequences\", \"were\", \"analyzed\", \"using\", \"in-house\", \"Perl\", \"script\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"number\", \"of\", \"synonymous\", \"substitutions\", \"per\", \"synonymous\", \"site\", \"(\", \"dS\", \")\", \",\", \"and\", \"the\", \"number\", \"of\", \"non-synonymous\", \"substitutions\", \"per\", \"non-synonymous\", \"site\", \"(\", \"dN\", \")\", \",\", \"for\", \"each\", \"coding\", \"region\", \"were\", \"calculated\", \"using\", \"the\", \"Nei-Gojobori\", \"method\", \"(\", \"Jukes-Cantor\", \")\", \"in\", \"PAML\", \"package\", \"(\", \"12\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"adaptive\", \"evolution\", \"server\", \"(\", \"http://www.datamonkey.org/\", \")\", \"was\", \"used\", \"to\", \"identify\", \"eventual\", \"sites\", \"of\", \"positive\", \"selection\", \".\"]\n",
      "Word token=> [\"For\", \"this\", \"purpose\", \",\", \"the\", \"following\", \"test\", \"has\", \"been\", \"used\", \":\", \"mixed-effects\", \"model\", \"of\", \"evolution\", \"(\", \"MEME\", \")\", \",\", \"which\", \"allows\", \"the\", \"distribution\", \"of\", \"dN/dS\", \"(\", \"ω\", \")\", \"to\", \"vary\", \"from\", \"site\", \"to\", \"site\", \"and\", \"from\", \"branch\", \"to\", \"branch\", \"at\", \"a\", \"site\", \"(\", \"13\", \")\", \".\"]\n",
      "Word token=> [\"This\", \"test\", \"allowed\", \"us\", \"to\", \"infer\", \"episodic\", \"and\", \"pervasive\", \"positive\", \"selection\", \"at\", \"individual\", \"sites\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"To\", \"investigate\", \"the\", \"potential\", \"relative\", \"synonymous\", \"codon\", \"usage\", \"(\", \"RSCU\", \")\", \"bias\", \"of\", \"the\", \"spike\", \"protein\", \"from\", \"SARS-CoV-\", \"2\", \"and\", \"its\", \"closely\", \"related\", \"coronaviruses\", \",\", \"the\", \"coding\", \"sequence\", \"(\", \"CDS\", \")\", \"of\", \"spike\", \"protein\", \"in\", \"these\", \"coronaviruses\", \"were\", \"calculated\", \"with\", \"CodonW\", \"1.4.4\", \"(\", \"http://codonw.sourceforge.net/\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"RSCU\", \"of\", \"human\", \"genes\", \"was\", \"retrieved\", \"from\", \"the\", \"Codon\", \"Usage\", \"Database\", \"(\", \"http://www.kazusa.or.jp/codon/\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"potential\", \"relationships\", \"among\", \"these\", \"sequences\", \"were\", \"calculated\", \"with\", \"a\", \"squared\", \"Euclidean\", \"distance\", \"(\", \"u\", \"h\", \"u\", \")\", \".\"]\n",
      "Word token=> [\"Besides\", \"RSCU\", \",\", \"the\", \"effective\", \"number\", \"of\", \"codons\", \"(\", \"ENc\", \")\", \"was\", \"used\", \"as\", \"a\", \"simple\", \"metrics\", \"to\", \"verify\", \"codon\", \"bias\", \"and\", \"to\", \"explore\", \"the\", \"source\", \"of\", \"virus\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\".\"]\n",
      "Word token=> [\"CC-BY-ND\", \"4.0\", \"International\", \"license\", \"author/funder\", \".\"]\n",
      "Word token=> [\"It\", \"is\", \"made\", \"available\", \"under\", \"a\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.02.27.969006\", \"doi\", \":\", \"bioRxiv\", \"preprint\"]\n",
      "Word token=> [\"Statistical\", \"analyses\", \"were\", \"performed\", \"using\", \"the\", \"R\", \"statistical\", \"package\", \"(\", \"version\", \"3\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Chi-squared\", \"test\", \"was\", \"used\", \"to\", \"compare\", \"any\", \"two\", \"data\", \"sets\", \",\", \"and\", \"data\", \"were\", \"considered\", \"significantly\", \"different\", \"if\", \"the\", \"two-tailed\", \"p\", \"value\", \"was\", \"less\", \"than\", \"0.05\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"newly\", \"identified\", \"SARS-CoV-\", \"2\", \"strain\", \"WIV\", \"04\", \"genome\", \"sequence\", \"was\", \"closely\", \"related\", \"with\", \"Bat-SARSr-CoV\", \"RaTG\", \"13\", \"and\", \"Bat-SL-CoVZC\", \"45\", \",\", \"which\", \"was\", \"collected\", \"from\", \"host\", \"Rhinolophus\", \"affinis\", \"(\", \"5\", \")\", \".\"]\n",
      "Word token=> [\"Compared\", \"with\", \"RaTG\", \"13\", \"genome\", \",\", \"many\", \"nucleotide\", \"substitutions\", \"are\", \"observed\", \",\", \"but\", \"there\", \"are\", \"only\", \"five\", \"small\", \"inserts\", \"and\", \"deletions\", \"(\", \"indels\", \")\", \"mutations\", \",\", \"and\", \"the\", \"largest\", \"insert\", \"segment\", \"in\", \"WIV\", \"04\", \"genome\", \"was\", \"\\\"\", \"CGGCGGGCACGT\", \"\\\"\", \"sequence\", \",\", \"which\", \"is\", \"located\", \"near\", \"the\", \"boundary\", \"of\", \"S\", \"1\", \"and\", \"S\", \"2\", \"regions\", \"of\", \"spike\", \"protein\", \".\"]\n",
      "Word token=> [\"Interestingly\", \",\", \"only\", \"synonymous\", \"mutation\", \"are\", \"observed\", \"near\", \"this\", \"insertion\", \"sequence\", \"(\", \"Fig.\", \"1\", \",\", \"panel\", \"C\", \")\", \".\"]\n",
      "Word token=> [\"Compared\", \"with\", \"Bat-SL-CoVZC\", \"45\", \"genome\", \",\", \"this\", \"insert\", \"segment\", \"is\", \"detected\", \"as\", \"well\", \",\", \"but\", \"non-synonymous\", \"mutations\", \"are\", \"also\", \"observed\", \"around\", \"it\", \".\"]\n",
      "Word token=> [\"Then\", \",\", \"we\", \"further\", \"compared\", \"the\", \"proportion\", \"of\", \"synonymous\", \"mutations\", \"in\", \"spike\", \"gene\", \"between\", \"WIV\", \"04\", \"and\", \"RaTG\", \"13\", \"or\", \"Bat-SL-CoVZC\", \"45\", \".\"]\n",
      "Word token=> [\"The\", \"ratio\", \"of\", \"nucleotide\", \"substitutions\", \"(\", \"263\", \"NT\", \")\", \"to\", \"amino\", \"acid\", \"substitutions\", \"(\", \"29\", \"AA\", \")\", \"was\", \"9.07\", \"from\", \"WIV\", \"04\", \"to\", \"RaTG\", \"13\", \",\", \"which\", \"was\", \"significantly\", \"higher\", \"than\", \"the\", \"ratio\", \"(\", \"3.91\", \",\", \"864\", \"/\", \"221\", \")\", \"from\", \"WIV\", \"04\", \"to\", \"Bat-SL-CoVZC\", \"45\", \"(\", \"p\", \"<\", \"0.05\", \")\", \"(\", \"Fig.\", \"2\", \")\", \".\"]\n",
      "Word token=> [\"Moreover\", \",\", \"we\", \"also\", \"observed\", \"that\", \"the\", \"synonymous\", \"mutations\", \"increased\", \"in\", \"whole\", \"genome\", \"level\", \"between\", \"WIV\", \"04\", \"and\", \"RaTG\", \"13\", \"than\", \"that\", \"between\", \"WIV\", \"04\", \"and\", \"Bat-SL-CoVZC\", \"45\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Next\", \",\", \"the\", \"detailed\", \"nucleotide\", \"changes\", \"of\", \"each\", \"comparison\", \"group\", \"were\", \"analyzed\", \",\", \"and\", \"we\", \"found\", \"that\", \"the\", \"T-to-C\", \"(\", \"T\", \":\", \"C\", \")\", \"transition\", \"mutation\", \"was\", \"enriched\", \"in\", \"the\", \"nucleotide\", \"substitutions\", \"between\", \"WIV\", \"04\", \"and\", \"RaTG\", \"13\", \"(\", \"Fig.\", \"1\", \"B\", \")\", \".\"]\n",
      "Word token=> [\"It\", \"'\", \"s\", \"reported\", \"that\", \"the\", \"CoVs\", \"lacking\", \"a\", \"3\", \"'\", \"-to-\", \"5\", \"'\", \"exoribonuclease\", \"(\", \"ExoN\", \")\", \"accumulated\", \"15\", \"to\", \"20\", \"fold\", \"more\", \"A\", \":\", \"G\", \"and\", \"U\", \":\", \"C\", \"transition\", \"(\", \"14\", \")\", \".\"]\n",
      "Word token=> [\"What\", \"'\", \"s\", \"more\", \",\", \"the\", \"RNA\", \"mutagen\", \"5\", \"-fluorouracil\", \"(\", \"5\", \"-FU\", \")\", \"treatment\", \"will\", \"also\", \"increase\", \"the\", \"U\", \":\", \"C\", \"and\", \"A\", \":\", \"G\", \"transitions\", \".\"]\n",
      "Word token=> [\"Consequently\", \",\", \"this\", \"increased\", \"T\", \":\", \"C\", \"mutation\", \"implies\", \"the\", \"ExoN\", \"of\", \"SARS-CoV-\", \"2\", \"may\", \"be\", \"deactivated\", \"compared\", \"with\", \"that\", \"of\", \"RaTG\", \"13\", \".\"]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.02.27.969006\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"Furthermore\", \",\", \"we\", \"checked\", \"whether\", \"the\", \"ratio\", \"of\", \"synonymous\", \"substitution\", \"to\", \"missense\", \"substitution\", \"is\", \"increased\", \"within\", \"SARS-CoV-\", \"2\", \"strains\", \".\"]\n",
      "Word token=> [\"From\", \"December\", \"2019\", \"to\", \"February\", \"2020\", \",\", \"the\", \"genome\", \"sequences\", \"of\", \"108\", \"strains\", \"of\", \"SARS-CoV-\", \"2\", \"virus\", \"have\", \"been\", \"submitted\", \"to\", \"GISAID\", \"database\", \"worldwide\", \".\"]\n",
      "Word token=> [\"Compared\", \"with\", \"the\", \"standard\", \"SARS-CoV-\", \"2\", \"strain\", \"WIV\", \"04\", \",\", \"total\", \"98\", \"point\", \"mutations\", \"were\", \"detected\", \"at\", \"93\", \"nucleotide\", \"sites\", \"of\", \"all\", \"SARS-CoV-\", \"2\", \"strains\", \"with\", \"genome\", \"sequence\", \"available\", \"on\", \"Feb\", \".\"]\n",
      "Word token=> [\"25\", \"2020\", \"(\", \"Fig.\", \"1\", \"A\", \")\", \".\"]\n",
      "Word token=> [\"However\", \",\", \"only\", \"58\", \"of\", \"these\", \"nucleotide\", \"mutations\", \"caused\", \"changes\", \"in\", \"amino\", \"acids\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"Among\", \"them\", \",\", \"15\", \"nucleotide\", \"substitutions\", \"at\", \"14\", \"sites\", \"caused\", \"changes\", \"in\", \"7\", \"amino\", \"acids\", \"of\", \"the\", \"spike\", \"protein\", \"were\", \"observed\", \".\"]\n",
      "Word token=> [\"Consequently\", \",\", \"the\", \"proportion\", \"of\", \"synonymous\", \"mutations\", \"(\", \"~\", \"40\", \"%\", \")\", \"among\", \"all\", \"current\", \"reported\", \"SARS-CoV-\", \"2\", \"strains\", \"is\", \"similar\", \"to\", \"that\", \"between\", \"WIV\", \"04\", \"and\", \"Bat-SL-CoVZC\", \"45\", \"(\", \"39.1\", \"%\", \")\", \",\", \"but\", \"is\", \"dramatically\", \"less\", \"than\", \"that\", \"between\", \"WIV\", \"04\", \"and\", \"RaTG\", \"13\", \"(\", \"90.7\", \"%\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"According\", \"to\", \"random\", \"drift\", \"hypothesis\", \"(\", \"15\", \")\", \"，\", \"these\", \"nucleotide\", \"mutations\", \"among\", \"different\", \"SARS-CoV-\", \"2\", \"strains\", \"now\", \"available\", \"are\", \"still\", \"determined\", \"by\", \"neutral\", \"evolution\", \".\"]\n",
      "Word token=> [\"In\", \"short\", \",\", \"there\", \"has\", \"no\", \"powerful\", \"factor\", \"to\", \"force\", \"SARS-CoV-\", \"2\", \"to\", \"evolve\", \"in\", \"a\", \"certain\", \"direction\", \"by\", \"far\", \".\"]\n",
      "Word token=> [\"However\", \",\", \"we\", \"should\", \"take\", \"strict\", \"precautions\", \"against\", \"the\", \"strong\", \"factors\", \"that\", \"may\", \"cause\", \"directional\", \"variation\", \"of\", \"SARS-CoV-\", \"2\", \"both\", \"in\", \"natural\", \"environment\", \"and\", \"infection\", \"treatment\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"To\", \"investigate\", \"whether\", \"the\", \"mutation\", \"pattern\", \"between\", \"SARS-CoV-\", \"2\", \"and\", \"RaTG\", \"13\", \"is\", \"unique\", \"across\", \"all\", \"coronavirus\", \"species\", \",\", \"we\", \"further\", \"compared\", \"their\", \"alterations\", \"of\", \"nucleotides\", \"and\", \"amino\", \"acids\", \"in\", \"comparison\", \"to\", \"other\", \"representative\", \"coronaviruses\", \".\"]\n",
      "Word token=> [\"Phylogenetic\", \"analysis\", \"of\", \"SARS-CoV-\", \"2\", \"and\", \"its\", \"20\", \"closely\", \"related\", \"coronaviruses\", \"formed\", \"four\", \"well\", \"supported\", \"clades\", \"(\", \"Fig.\", \"2\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"two\", \"SARS-CoV-\", \"2\", \"strain\", \"WIV\", \"04\", \"(\", \"from\", \"Wuhan\", \")\", \"and\", \"SNU\", \"01\", \"(\", \"from\", \"Korea\", \"patient\", \")\", \"were\", \"clustered\", \"with\", \"SARS-CoV-related\", \"strains\", \"to\", \"form\", \"clade\", \"1\", \",\", \"belonging\", \"to\", \"beta-coronavirus\", \"lineage\", \"B.\", \"The\", \"Compared\", \"with\", \"Bat-SARSr-CoV\", \"RaTG\", \"13\", \",\", \"there\", \"were\", \"293\", \"nucleotideswere\", \"altered\", \"while\", \"only\", \"29\", \"amino\", \"acids\", \"were\", \"altered\", \"in\", \"SARS-CoV-\", \"2\", \";\", \"their\", \"ratio\", \"was\", \"as\", \"high\", \"as\", \"9.07\", \".\"]\n",
      "Word token=> [\"However\", \",\", \"this\", \"ratio\", \"detected\", \"in\", \"the\", \"comparisons\", \"of\", \"other\", \"human\", \"coronaviruses\", \"with\", \"their\", \"similar\", \"animal\", \"coronaviruses\", \"was\", \"less\", \"than\", \"5.0\", \".\"]\n",
      "Word token=> [\"For\", \"example\", \",\", \"the\", \"ratio\", \"of\", \"human\", \"SARS-CoV\", \"Tor\", \"2\", \"to\", \"bat\", \"SARS-like\", \"CoVs\", \"LYRa\", \"11\", \"was\", \"4.81\", \";\", \"that\", \"ratio\", \"of\", \"human\", \"MERS-CoV\", \"EMC\", \"2012\", \"to\", \"bat\", \"SARS-like\", \"CoVs\", \"RSA\", \"2011\", \"was\", \"2.80\", \";\", \"that\", \"ratio\", \"of\", \"human\", \"coronavirus\", \"229\", \"E\", \"to\", \"bat\", \"coronavirus\", \"HKU\", \"2\", \"was\", \"4.55\", \".\"]\n",
      "Word token=> [\"These\", \"results\", \"indicate\", \"that\", \"the\", \"relative\", \"level\", \"of\", \"synonymous\", \"substitutions\", \"between\", \"human\", \"SARS-CoV-\", \"2\", \"and\", \"its\", \"possible\", \"animal\", \"origin\", \"(\", \"RaTG\", \"13\", \")\", \"is\", \"much\", \"higher\", \"than\", \"that\", \"between\", \"other\", \"human\", \"coronaviruses\", \"and\", \"their\", \"potential\", \"animal\", \"sources\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Different\", \"organisms\", \",\", \"even\", \"different\", \"protein\", \"coding\", \"genes\", \"of\", \"the\", \"same\", \"species\", \",\", \"have\", \"different\", \"frequency\", \"of\", \"codon\", \"usage\", \"(\", \"16\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"RSCU\", \"bias\", \"will\", \"reveal\", \"the\", \"difference\", \"of\", \"the\", \"host\", \"source\", \".\"]\n",
      "Word token=> [\"We\", \"calculated\", \"the\", \"distance\", \"of\", \"RSCU\", \"between\", \"spike\", \"genes\", \"of\", \"20\", \"representative\", \"coronaviruses\", \"and\"]\n",
      "Word token=> [\"and\", \"Bat-SARSr-CoV\", \"RaTG\", \"13\", \"was\", \"1.17\", \",\", \"which\", \"was\", \"the\", \"lowest\", \"except\", \"for\", \"SNU\", \"01\", \"(\", \"another\", \"strain\", \"of\", \"SARS-CoV-\", \"2\", \")\", \",\", \"indicating\", \"that\", \"their\", \"codon\", \"preference\", \"was\", \"almost\", \"the\", \"same\", \";\", \"the\", \"second\", \"lowest\", \"codon\", \"usage\", \"difference\", \"was\", \"2.41\", \"that\", \"was\", \"detected\", \"between\", \"SARS-CoV-\", \"2\", \"WIV\", \"04\", \"and\", \"Bat-SL-CoVZC\", \"45\", \"(\", \"Fig.\", \"2\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"coronaviruses\", \"in\", \"same\", \"beta-CoV\", \"lineage\", \"B\", \"have\", \"a\", \"relatively\", \"close\"]\n",
      "Word token=> [\"To\", \"infer\", \"whether\", \"the\", \"retention\", \"of\", \"synonymous\", \"mutations\", \"is\", \"supported\", \"or\", \"hindered\", \"by\", \"natural\", \"selection\", \",\", \"we\", \"further\", \"studied\", \"non-synonymous\", \"substitution\", \"rate\", \"(\", \"dN\", \")\", \"and\", \"synonymous\", \"substitution\", \"rate\", \"(\", \"dS\", \")\", \"in\", \"spike\", \"gene\", \"(\", \"Table\", \"1\", \")\", \".\"]\n",
      "Word token=> [\"Generally\", \",\", \"positive\", \"(\", \"Darwinian\", \")\", \"selection\", \"increases\", \",\", \"but\", \"negative\", \"(\", \"purifying\", \")\", \"selection\", \"decreases\", \"the\", \"ratios\", \"of\", \"non-synonymous\", \"to\", \"synonymous\", \"substitution\", \"rates\", \"(\", \"dN/dS\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"results\", \"showed\", \"that\", \"both\", \"dN\", \"and\", \"dS\", \"of\", \"S\", \"gene\", \"of\", \"SARS-CoV-\", \"2\", \"WIV\", \"04\", \"versus\", \"Bat-SARr-CoV\", \"RaTG\", \"13\", \"were\", \"the\", \"lowest\", \"among\", \"all\", \"typical\", \"coronaviruses\", \",\", \"while\", \"those\", \"of\", \"SARS-CoV\", \"Tor\", \"2\", \"versus\", \"bat\", \"SARS-like\", \"coronavirus\", \"WIV\", \"1\", \"were\", \"the\", \"second\", \"lowest\", \".\"]\n",
      "Word token=> [\"When\", \"the\", \"ratio\", \"of\", \"dN\", \"to\", \"dS\", \"of\", \"spike\", \"gene\", \"is\", \"compared\", \",\", \"all\", \"the\", \"tested\", \"dN/dS\", \"are\", \"less\", \"than\", \"1\", \",\", \"indicating\", \"that\", \"these\", \"non-synonymous\", \"mutations\", \"are\", \"harmful\", \",\", \"and\", \"negative\", \"selection\", \"will\", \"reduce\", \"their\", \"retention\", \"speed\", \".\"]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.02.27.969006\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"Among\", \"them\", \",\", \"dN/dS\", \"of\", \"spike\", \"gene\", \"of\", \"SARS-CoV-\", \"2\", \"WIV\", \"04\", \"versus\", \"that\", \"of\", \"Bat-SARr-CoV\", \"RaTG\", \"13\", \"was\", \"0.04\", \",\", \"which\", \"was\", \"the\", \"lowest\", \"among\", \"all\", \"comparisons\", \",\", \"reconfirming\", \"that\", \"the\", \"rate\", \"of\", \"synonymous\", \"mutation\", \"was\", \"increased\", \"between\", \"WIV\", \"04\", \"and\", \"RaTG\", \"13\", \"strains\", \".\"]\n",
      "Word token=> [\"Moreover\", \",\", \"dN/dS\", \"rate\", \"of\", \"polyprotein\", \"(\", \"ORF\", \"1\", \"ab\", \")\", \"and\", \"nucleocapsid\", \"(\", \"N\", \")\", \"genes\", \"were\", \"similar\", \"with\", \"that\", \"of\", \"spike\", \"gene\", \"(\", \"Table\", \"S\", \"2\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"spike\", \"(\", \"S\", \")\", \"protein\", \"undergoes\", \"several\", \"drastic\", \"changes\", \"during\", \"virus\", \"infection\", \".\"]\n",
      "Word token=> [\"For\", \"instance\", \",\", \"its\", \"large\", \"parts\", \"are\", \"cleaved\", \"during\", \"infection\", \"by\", \"cellular\", \"proteases\", \"and\", \"expose\", \"the\", \"receptors\", \"to\", \"activate\", \"viral\", \"attachment\", \"to\", \"the\", \"host\", \"(\", \"17\", \")\", \".\"]\n",
      "Word token=> [\"As\", \"the\", \"receptor\", \"binding\", \"domain\", \"(\", \"RBD\", \")\", \"of\", \"spike\", \"protein\", \"was\", \"involved\", \"in\", \"interacting\", \"with\", \"human\", \"angiotensin-converting\", \"enzyme\", \"2\", \"(\", \"ACE\", \"2\", \")\", \"protein\", \",\", \"the\", \"RBD\", \"region\", \"was\", \"thought\", \"to\", \"be\", \"preferential\", \"targets\", \"of\", \"natural\", \"selection\", \"(\", \"18\", \")\", \".\"]\n",
      "Word token=> [\"Consistent\", \"with\", \"this\", \"hypothesis\", \",\", \"our\", \"results\", \"showed\", \"that\", \"both\", \"dN\", \"and\", \"dS\", \"of\", \"RBD\", \"region\", \"are\", \"increased\", \"in\", \"comparison\", \"with\", \"whole\", \"spike\", \"gene\", \"region\", \"across\", \"all\", \"the\", \"virus\", \"pairs\", \"used\", \"in\", \"this\", \"study\", \"(\", \"Table\", \"1\", \")\", \".\"]\n",
      "Word token=> [\"Notably\", \",\", \"the\", \"dN/dS\", \"ratio\", \"of\", \"RBD\", \"region\", \"in\", \"SARS-CoV-\", \"2\", \"WIV\", \"04\", \"was\", \"dramatically\", \"increased\", \"in\", \"about\", \"3\", \"fold\", \"in\", \"comparison\", \"with\", \"full\", \"spike\", \"region\", \".\"]\n",
      "Word token=> [\"Consequently\", \",\", \"these\", \"mutations\", \"may\", \"be\", \"subjected\", \"to\", \"Darwin\", \"'\", \"s\", \"choice\", \"and\", \"will\", \"become\", \"evidence\", \"of\", \"adaptive\", \"protein\", \"evolution\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"To\", \"detect\", \"positive\", \"selection\", \"on\", \"the\", \"spike\", \"gene\", \",\", \"the\", \"MEME\", \"analysis\", \"was\", \"performed\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Significant\", \"(\", \"p\", \"<\", \"0.05\", \")\", \"pervasive\", \"episodic\", \"selection\", \"was\", \"detected\", \"in\", \"three\", \"sites\", \"(\", \"48\", \"th\", \",\", \"254\", \"th\", \",\", \"330\", \"th\", \"position\", \"using\", \"the\", \"reference\", \"sequence\", \"of\", \"WIV\", \"04\", \")\", \".\"]\n",
      "Word token=> [\"On\", \"the\", \"254\", \"th\", \"position\", \"of\", \"the\", \"spike\", \"amino\", \"acid\", \"sequence\", \",\", \"there\", \"is\", \"a\", \"histidine\", \"residue\", \"instead\", \"of\", \"a\", \"phenylalanine\", \"residue\", \",\", \"whereas\", \"on\", \"the\", \"330\", \"th\", \"amino\", \"acidic\", \"position\", \"in\", \"WIV\", \"04\", \"sequence\", \",\", \"there\", \"is\", \"a\", \"glutamine\", \"residue\", \"instead\", \"of\", \"an\", \"valine\", \"residue\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"results\", \"described\", \"above\", \"strongly\", \"supporting\", \"the\", \"action\", \"of\", \"positive\", \"selection\", \"in\", \"spike\", \"gene\", \"during\", \"the\", \"recent\", \"evolution\", \"of\", \"SARS-CoV-\", \"2\", \"and\", \"RaTG\", \"13\", \".\"]\n",
      "Word token=> [\"However\", \",\", \"this\", \"result\", \"should\", \"be\", \".\"]\n",
      "Word token=> [\"CC-BY-ND\", \"4.0\", \"International\", \"license\", \"author/funder\", \".\"]\n",
      "Word token=> [\"It\", \"is\", \"made\", \"available\", \"under\", \"a\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.02.27.969006\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"addressed\", \"with\", \"careful\", \"explain\", \",\", \"because\", \"the\", \"RBD\", \"region\", \"of\", \"spike\", \"gene\", \"from\", \"SARS-CoV-\", \"2\", \"was\", \"quit\", \"divergent\", \"with\", \"that\", \"of\", \"RaTG\", \"13\", \"(\", \"Fig.\", \"S\", \"1\", \")\", \",\", \"suggesting\", \"it\", \"might\", \"have\", \"originated\", \"from\", \"homologous\", \"recombination\", \"between\", \"RaTG\", \"13\", \"and\", \"one\", \"yet-unknown\", \"coronavirus\", \"(\", \"19\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Through\", \"comprehensive\", \"comparative\", \"analysis\", \"between\", \"SARS-CoV-\", \"2\", \"and\", \"other\", \"coronaviruses\", \",\", \"we\", \"found\", \"the\", \"synonymous\", \"mutations\", \"is\", \"dramatically\", \"elevated\", \"between\", \"SARS-CoV-\", \"2\", \"and\", \"RaTG\", \"13\", \"than\", \"that\", \"of\", \"other\", \"coronavirus\", \"strains\", \",\", \"and\", \"the\", \"nucleotide\", \"mutations\", \"were\", \"enriched\", \"in\", \"T\", \":\", \"C\", \"transition\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"specific\", \"mutation\", \"pattern\", \"may\", \"caused\", \"by\", \"the\", \"loss\", \"function\", \"of\", \"RNA\", \"3\", \"'\", \"-to-\", \"5\", \"'\", \"exoribonuclease\", \"(\", \"ExoN\", \")\", \".\"]\n",
      "Word token=> [\"Moreover\", \",\", \"as\", \"the\", \"SARS-CoV-\", \"2\", \"was\", \"supposed\", \"to\", \"be\", \"originated\", \"from\", \"Bat-SARSr-CoV\", \"RaTG\", \"13\", \",\", \"the\", \"increased\", \"synonymous\", \"substitution\", \"between\", \"SARS-CoV-\", \"2\", \"and\", \"RaTG\", \"13\", \"strain\", \"suggested\", \"the\", \"SARS-CoV-\", \"2\", \"genome\", \"was\", \"under\", \"stronger\", \"negative\", \"(\", \"purifying\", \")\", \"selection\", \".\"]\n",
      "Word token=> [\"We\", \"also\", \"detected\", \"some\", \"sites\", \"in\", \"spike\", \"protein\", \"was\", \"experiencing\", \"positive\", \"selection\", \".\"]\n",
      "Word token=> [\"These\", \"observations\", \"suggest\", \"that\", \"adaptive\", \"evolution\", \"might\", \"contribute\", \"to\", \"its\", \"host\", \"shifts\", \".\"]\n",
      "Word token=> [\"However\", \",\", \"as\", \"the\", \"RNA\", \"mutagen\", \"(\", \"e.g.\", \"5\", \"-FU\", \")\", \"could\", \"induce\", \"the\", \"same\", \"mutation\", \"pattern\", \",\", \"the\", \"mechanism\", \"of\", \"the\", \"mutation\", \"pattern\", \"observed\", \"between\", \"SARS-CoV-\", \"2\", \"and\", \"RaTG\", \"13\", \"should\", \"be\", \"further\", \"investigated\", \"in\", \"future\", \"work\", \".\"]\n",
      "Word token=> [\"The\", \"number\", \"of\", \"amino\", \"acid\", \"changes\", \".\"]\n",
      "Word token=> [\"RSCUdist\", \":\", \"the\", \"RSCU\", \"distance\", \"of\", \"spike\", \"protein\", \";\", \"ENc\", \":\", \"effective\", \"number\", \"of\", \"codons\", \";\", \"S\", \"ident\", \"(\", \"%\", \")\", \":\", \"the\", \"percent\", \"identity\", \"of\", \"S\", \"protein\", \"sequence\", \"between\", \"different\", \"coronaviruses\", \".\"]\n",
      "Word token=> [\"The\", \"NT/AA\", \"ratio\", \"between\", \"WIV\", \"04\", \"/RaTG\", \"13\", \"was\", \"statistically\", \"significant\", \"higher\", \"(\", \"Chi-squared\", \"test\", \",\", \"P<\", \"0.01\", \")\", \"than\", \"that\", \"of\", \"other\", \"strain\", \"pairs\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\".\"]\n",
      "Word token=> [\"CC-BY-ND\", \"4.0\", \"International\", \"license\", \"author/funder\", \".\"]\n",
      "Word token=> [\"It\", \"is\", \"made\", \"available\", \"under\", \"a\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.02.27.969006\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"*\", \"The\", \"whole\", \"genome\", \"sequence\", \"identity\", \"between\", \"these\", \"paired\", \"strains\", \"was\", \"larger\", \"than\", \"95\", \"%\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\".\"]\n",
      "Word token=> [\"CC-BY-ND\", \"4.0\", \"International\", \"license\", \"author/funder\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"It\", \"is\", \"made\", \"available\", \"under\", \"a\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.02.27.969006\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \".\"]\n",
      "Word token=> [\"The\", \"COVID-\", \"19\", \"outbreak\", \"began\", \"on\", \"December\", \"2019\", \"in\", \"Wuhan\", \",\", \"China\", \"(\", \"1\", \")\", \",\", \"and\", \"the\", \"causative\", \"virus\", \",\", \"SARS-CoV-\", \"2\", \",\", \"has\", \"rapidly\", \"spread\", \"into\", \"43\", \"countries\", \"as\", \"of\", \"February\", \"27\", \",\", \"2020\", \".\"]\n",
      "Word token=> [\"The\", \"number\", \"of\", \"infected\", \"patients\", \"exceeds\", \"81,000\", \",\", \"and\", \"the\", \"death\", \"toll\", \"exceeds\", \"2,700\", \".\"]\n",
      "Word token=> [\"Several\", \"drugs\", \",\", \"including\", \"lopinavir\", \",\", \"remdesivir\", \",\", \"and\", \"chloroquine\", \",\", \"have\", \"been\", \"reported\", \"to\", \"be\", \"presumably\", \"effective\", \"in\", \"treating\", \"this\", \"disease\", \"(\", \"2\", \",\", \"3\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Systemic\", \"treatment\", \"with\", \"corticosteroids\", \"is\", \"contraindicated\", \"for\", \"the\", \"severe\", \"pneumonia\", \"caused\", \"by\", \"viruses\", \"such\", \"as\", \"MERS-CoV\", \"and\", \"SARS-CoV\", \",\", \"as\", \"steroids\", \"suppress\", \"the\", \"innate\", \"immune\", \"system\", \",\", \"resulting\", \"in\", \"increased\", \"viral\", \"replication\", \".\"]\n",
      "Word token=> [\"In\", \"fact\", \",\", \"for\", \"SARS\", \"in\", \"2003\", \"and\", \"MERS\", \"in\", \"2013\", \",\", \"treatment\", \"with\", \"corticosteroids\", \"was\", \"associated\", \"with\", \"increased\", \"mortality\", \"(\", \"4\", \",\", \"5\", \")\", \".\"]\n",
      "Word token=> [\"In\", \"the\", \"present\", \"study\", \",\", \"we\", \"reconsider\", \"the\", \"use\", \"of\", \"inhaled\", \"corticosteroids\", \",\", \"which\", \"have\", \"been\", \"excluded\", \"from\", \"the\", \"treatment\", \"of\", \"pneumonia\", \"caused\", \"by\", \"coronavirus\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Steroid\", \"compounds\", \",\", \"which\", \"are\", \"expected\", \"to\", \"have\", \"dual\", \"functions\", \"in\", \"blocking\", \"both\", \"coronavirus\", \"replication\", \"and\", \"host\", \"inflammation\", \",\", \"were\", \"screened\", \"from\", \"a\", \"chemical\", \"library\", \".\"]\n",
      "Word token=> [\"The\", \"cytopathic\", \"effect\", \"caused\", \"by\", \"MERS-CoV\", \"infection\", \"was\", \"measured\", \"to\", \"evaluate\", \"viral\", \"replication\", \".\"]\n",
      "Word token=> [\"Four\", \"steroid\", \"compounds\", \",\", \"ciclesonide\", \",\", \"mometasone\", \"furoate\", \",\", \"mifepristone\", \",\", \"and\", \"algestone\", \"acetophenide\", \"conferred\", \"a\", \"greater\", \"than\", \"95\", \"%\", \"cell\", \"survival\", \"rate\", \"(\", \"Fig.\", \"S\", \"1\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Next\", \",\", \"concentration-dependent\", \"viral\", \"growth\", \"suppression\", \"and\", \"drug\", \"cytotoxicity\", \"were\", \"assessed\", \".\"]\n",
      "Word token=> [\"Ciclesonide\", \"exhibited\", \"low\", \"cytotoxicity\", \"and\", \"potent\", \"suppression\", \"of\", \"viral\", \"growth\", \"(\", \"Fig.\", \"1\", \"a\", \")\", \".\"]\n",
      "Word token=> [\"Cortisone\", \"and\", \"prednisolone\", \",\", \"which\", \"are\", \"commonly\", \"used\", \"for\", \"systemic\", \"steroid\", \"treatment\", \",\", \"dexamethasone\", \",\", \"which\", \"has\", \"strong\", \"immunosuppressant\", \"effects\", \",\", \"and\", \"fluticasone\", \",\", \"a\", \"commonly\", \"used\", \"inhaled\", \"steroid\", \",\", \"did\", \"not\", \"suppress\", \"viral\", \"growth\", \"(\", \"Fig.\", \"1\", \"a\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"antiviral\", \"effects\", \"of\", \"steroids\", \"against\", \"various\", \"species\", \"of\", \"viruses\", \"were\", \"tested\", \"by\", \"quantifying\", \"propagated\", \"virus\", \"in\", \"culture\", \"medium\", \".\"]\n",
      "Word token=> [\"Ciclesonide\", \"and\", \"mometasone\", \"also\", \"suppressed\", \"replication\", \"of\", \"other\", \"coronaviruses\", \",\", \"HCoV-\", \"229\", \"E\", \"and\", \"SARS-CoV\", \",\", \"but\", \"not\", \"replication\", \"of\", \"RS\", \"virus\", \"or\", \"influenza\", \"virus\", \"(\", \"Fig.\", \"S\", \"2\", \")\", \".\"]\n",
      "Word token=> [\"In\", \"addition\", \",\", \"ciclesonide\", \"slightly\", \"but\", \"significantly\", \"inhibited\", \"rubella\", \"virus\", \"(\", \"having\", \"positive\", \"strand\", \"RNA\", \"genome\", \")\", \"replication\", \"(\", \"Fig.\", \"S\", \"2\", \")\", \",\", \"suggesting\", \"that\", \"ciclesonide\", \"interacted\", \"with\", \"the\", \"replication\", \"site\", \"of\", \"positive-strand\", \"RNA\", \"virus\", \"intracellularly\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"To\", \"identify\", \"the\", \"drug\", \"target\", \"of\", \"virus\", \"replication\", \",\", \"we\", \"conducted\", \"11\", \"consecutive\", \"MERS-CoV\", \"passages\", \"in\", \"the\", \"presence\", \"of\", \"40\", \"μ\", \"M\", \"ciclesonide\", \"or\", \"40\", \"μ\", \"M\", \"mometasone\", \".\"]\n",
      "Word token=> [\"A\", \"mutant\", \"virus\", \"that\", \"developed\", \"resistance\", \"ciclesonide\", \"was\", \"generated\", \"(\", \"Fig.\", \"1\", \"b\", \")\", \",\", \"but\", \"no\", \"resistant\", \"virus\", \"to\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.11.987016\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"mometasone\", \"was\", \"generated\", \".\"]\n",
      "Word token=> [\"Next-generation\", \"sequencing\", \"identified\", \"that\", \"an\", \"amino\", \"acid\", \"substitution\", \"at\", \"A\", \"25\", \"V\", \"(\", \"C\", \"19647\", \"T\", \"in\", \"the\", \"reference\", \"sequence\", \"NC\", \"_\", \"019843.3\", \")\", \"in\", \"nonstructural\", \"protein\", \"15\", \"(\", \"NSP\", \"15\", \")\", \",\", \"coronavirus\", \"endoribonuclease\", \"(\", \"6\", \"-\", \"8\", \")\", \",\", \"was\", \"the\", \"predicted\", \"mechanism\", \"for\", \"viral\", \"resistance\", \"to\", \"ciclesonide\", \".\"]\n",
      "Word token=> [\"A\", \"recombinant\", \"virus\", \"carrying\", \"the\", \"A\", \"25\", \"V\", \"amino\", \"acid\", \"substitution\", \"(\", \"Re-Nsp\", \"15\", \"-A\", \"25\", \"V\", \")\", \"was\", \"generated\", \"from\", \"the\", \"parental\", \"MERS-CoV/EMC\", \"strain\", \"(\", \"Re-EMC/MERS\", \")\", \"with\", \"a\", \"BAC\", \"reverse\", \"genetics\", \"system\", \"(\", \"9\", \")\", \",\", \"which\", \"overcame\", \"the\", \"antiviral\", \"effect\", \"of\", \"ciclesonide\", \"(\", \"Fig.\", \"1\", \"c\", \")\", \".\"]\n",
      "Word token=> [\"Interestingly\", \",\", \"the\", \"mutant\", \"virus\", \"was\", \"inhibited\", \"by\", \"mometasone\", \",\", \"suggesting\", \"that\", \"the\", \"antiviral\", \"target\", \"of\", \"mometasone\", \"was\", \"different\", \"from\", \"that\", \"of\", \"ciclesonide\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Next\", \",\", \"the\", \"effect\", \"of\", \"ciclesonide\", \"on\", \"SARS-CoV-\", \"2\", \"infection\", \"was\", \"evaluated\", \".\"]\n",
      "Word token=> [\"VeroE\", \"6\", \"/TMPRSS\", \"2\", \"cells\", \"(\", \"10\", \")\", \"were\", \"infected\", \"with\", \"authentic\", \"SARS-CoV-\", \"2\", \"in\", \"the\", \"presence\", \"of\", \"steroids\", \"or\", \"other\", \"inhibitors\", \".\"]\n",
      "Word token=> [\"At\", \"6\", \"h\", \"post-infection\", \",\", \"cellular\", \"RNA\", \"was\", \"isolated\", \",\", \"and\", \"real-time\", \"PCR\", \"was\", \"conducted\", \"to\", \"quantify\", \"the\", \"amount\", \"of\", \"viral\", \"RNA\", \"(\", \"11\", \")\", \".\"]\n",
      "Word token=> [\"Ciclesonide\", \"and\", \"mometasone\", \"suppressed\", \"viral\", \"replication\", \"with\", \"a\", \"similar\", \"efficacy\", \"to\", \"lopinavir\", \"(\", \"Fig.\", \"2\", \"a\", \")\", \".\"]\n",
      "Word token=> [\"For\", \"comparison\", \",\", \"viral\", \"cell\", \"entry\", \"inhibitors\", \"were\", \"tested\", \".\"]\n",
      "Word token=> [\"E\", \"64\", \"d\", \"substantially\", \"reduced\", \"viral\", \"RNA\", \"levels\", \",\", \"but\", \"nafamostat\", \"and\", \"camostat\", \"had\", \"only\", \"modest\", \"antiviral\", \"effects\", \",\", \"suggesting\", \"that\", \"SARS-CoV-\", \"2\", \"primarily\", \"utilized\", \"the\", \"cathepsin/endosomal\", \"pathway\", \"of\", \"cell\", \"entry\", \"rather\", \"than\", \"the\", \"TMPRSS\", \"2\", \"/cell\", \"surface\", \"pathway\", \"to\", \"enter\", \"TMPRSS\", \"2\", \"-expressing\", \"cells\", \",\", \"consistent\", \"with\", \"a\", \"recent\", \"study\", \"(\", \"12\", \")\", \".\"]\n",
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https\", \":\", \"/\", \"/doi.org/\", \"10.1101\", \"/\", \"/doi.org/\", \"10\", \".\"]\n",
      "Word token=> [\"/\", \"2020\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"th\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.11.987016\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"a\", \"RNA\", \"copy\", \"(\", \"log\", \"10\", \")\", \"SARS-CoV-\", \"2\", \"RNA\", \"in\", \"cell\", \"(\", \"6\", \"hpi\", \"Cell\", \"viability\", \"(\", \"%\", \")\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"th\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.11.987016\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \".\"]\n",
      "Word token=> [\"Seventeen\", \"years\", \"have\", \"passed\", \"since\", \"the\", \"outbreak\", \"of\", \"severe\", \"acute\", \"respiratory\", \"syndrome\", \"(\", \"SARS\", \")\", \"in\", \"2003\", \",\", \"but\", \"there\", \"is\", \"yet\", \"no\", \"approved\", \"treatment\", \"for\", \"infections\", \"with\", \"the\", \"SARS\", \"coronavirus\", \"(\", \"SARS-CoV\", \")\", \".\"]\n",
      "Word token=> [\"1\", \"One\", \"of\", \"the\", \"reasons\", \"is\", \"that\", \"despite\", \"the\", \"devastating\", \"consequences\", \"of\", \"SARS\", \"for\", \"the\", \"affected\", \"patients\", \",\", \"the\", \"development\", \"of\", \"an\", \"antiviral\", \"drug\", \"against\", \"this\", \"virus\", \"would\", \"not\", \"be\", \"commercially\", \"viable\", \"in\", \"view\", \"of\", \"the\", \"fact\", \"that\", \"the\", \"virus\", \"has\", \"been\", \"rapidly\", \"contained\", \"and\", \"did\", \"not\", \"reappear\", \"since\", \"2004\", \".\"]\n",
      "Word token=> [\"As\", \"a\", \"result\", \",\", \"we\", \"were\", \"empty-handed\", \"when\", \"the\", \"Middle-East\", \"respiratory\", \"syndrome\", \"coronavirus\", \"(\", \"MERS-CoV\", \")\", \",\", \"a\", \"close\", \"relative\", \"of\", \"SARS-CoV\", \",\", \"emerged\", \"in\", \"2012\", \".\"]\n",
      "Word token=> [\"2\", \"MERS\", \"is\", \"characterized\", \"by\", \"severe\", \"respiratory\", \"disease\", \",\", \"quite\", \"similar\", \"to\", \"SARS\", \",\", \"but\", \"in\", \"addition\", \"frequently\", \"causes\", \"renal\", \"failure\", \"3\", \".\"]\n",
      "Word token=> [\"Although\", \"the\", \"number\", \"of\", \"registered\", \"MERS\", \"cases\", \"is\", \"low\", \"(\", \"2494\", \"as\", \"of\", \"November\", \"30\", \",\", \"2019\", \";\", \"www.who.int\", \")\", \",\", \"the\", \"threat\", \"MERS-CoV\", \"poses\", \"to\", \"global\", \"public\", \"health\", \"may\", \"be\", \"even\", \"more\", \"serious\", \"than\", \"that\", \"presented\", \"by\", \"SARS-CoV\", \".\"]\n",
      "Word token=> [\"This\", \"is\", \"related\", \"to\", \"the\", \"high\", \"casefatality\", \"rate\", \"(\", \"about\", \"35\", \"%\", \",\", \"compared\", \"to\", \"10\", \"%\", \"for\", \"SARS\", \")\", \",\", \"and\", \"to\", \"the\", \"fact\", \"that\", \"MERS\", \"cases\", \"are\", \"still\", \"accumulating\", \"seven\", \"years\", \"after\", \"the\", \"discovery\", \"of\", \"the\", \"virus\", \",\", \"whereas\", \"the\", \"SARS\", \"outbreak\", \"was\", \"essentially\", \"contained\", \"within\", \"6\", \"months\", \".\"]\n",
      "Word token=> [\"The\", \"potential\", \"for\", \"human-to-human\", \"transmission\", \"of\", \"MERS-CoV\", \"has\", \"been\", \"impressively\", \"demonstrated\", \"by\", \"the\", \"2015\", \"outbreak\", \"in\", \"South\", \"Korea\", \",\", \"where\", \"186\", \"cases\", \"could\", \"be\", \"traced\", \"back\", \"to\", \"a\", \"single\", \"infected\", \"traveller\", \"returning\", \"from\", \"the\", \"Middle\", \"East\", \".\"]\n",
      "Word token=> [\"4\", \"SARS-like\", \"coronaviruses\", \"are\", \"still\", \"circulating\", \"in\", \"bats\", \"in\", \"China\", \",\", \"[\", \"5\", \"]\", \"[\", \"6\", \"]\", \"[\", \"7\", \"]\", \"[\", \"8\", \"]\", \"from\", \"where\", \"they\", \"may\", \"spill\", \"over\", \"into\", \"the\", \"human\", \"population\", \";\", \"this\", \"is\", \"probably\", \"what\", \"caused\", \"the\", \"current\", \"outbreak\", \"of\", \"atypical\", \"pneumonia\", \"in\", \"Wuhan\", \",\", \"which\", \"is\", \"linked\", \"to\", \"a\", \"seafood\", \"and\", \"animal\", \"market\", \".\"]\n",
      "Word token=> [\"The\", \"RNA\", \"genome\", \"(\", \"Gen-Bank\", \"accession\", \"code\", \":\", \"MN\", \"908947.2\", \";\", \"http://virological.org/t/initialgenome-release-of-novel-coronavirus/\", \"319\", \",\", \"last\", \"accessed\", \"on\", \"January\", \"11\", \",\", \"2020\", \")\", \"of\", \"the\", \"new\", \"betacoronavirus\", \"features\", \"around\", \"82\", \"%\", \"identity\", \"to\", \"that\", \"of\", \"SARS-CoV\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"In\", \"spite\", \"of\", \"the\", \"considerable\", \"threat\", \"posed\", \"by\", \"SARS-CoV\", \"and\", \"related\", \"viruses\", \",\", \"as\", \"well\", \"as\", \"by\", \"MERS-CoV\", \",\", \"it\", \"is\", \"obvious\", \"that\", \"the\", \"number\", \"of\", \"cases\", \"so\", \"far\", \"does\", \"not\", \"warrant\", \"the\", \"commercial\", \"development\", \"of\", \"an\", \"antiviral\", \"drug\", \"targeting\", \"MERS-and\", \"SARS-CoV\", \"even\", \"if\", \"a\", \"projected\", \"steady\", \"growth\", \"of\", \"the\", \"number\", \"of\", \"MERS\", \"cases\", \"is\", \"taken\", \"into\", \"account\", \".\"]\n",
      "Word token=> [\"A\", \"possible\", \"solution\", \"to\", \"the\", \"problem\", \"could\", \"be\", \"the\", \"development\", \"of\", \"broadspectrum\", \"antiviral\", \"drugs\", \"that\", \"are\", \"directed\", \"against\", \"the\", \"major\", \"viral\", \"protease\", \",\", \"a\", \"target\", \"that\", \"is\", \"shared\", \"by\", \"all\", \"coronavirus\", \"genera\", \"as\", \"well\", \"as\", \",\", \"in\", \"a\", \"related\", \"form\", \",\", \"by\", \"members\", \"of\", \"the\", \"large\", \"genus\", \"Enterovirus\", \"in\", \"the\", \"picornavirus\", \"family\", \".\"]\n",
      "Word token=> [\"Among\", \"the\", \"members\", \"of\", \"the\", \"genus\", \"Alphacoronavirus\", \"are\", \"the\", \"human\", \"coronaviruses\", \"(\", \"HCoV\", \")\", \"NL\", \"63\", \"(\", \"ref.\", \"9\", \")\", \"and\", \"229\", \"E\", \"10\", \"that\", \"usually\", \"cause\", \"only\", \"mild\", \"respiratory\", \"symptoms\", \"in\", \"otherwise\", \"healthy\", \"individuals\", \",\", \"but\", \"are\", \"much\", \"more\", \"widespread\", \"than\", \"SARS-CoV\", \"or\", \"MERS-CoV\", \".\"]\n",
      "Word token=> [\"Therapeutic\", \"intervention\", \"against\", \"alphacoronaviruses\", \"is\", \"indicated\", \"in\", \"cases\", \"of\", \"accompanying\", \"disease\", \"such\", \"as\", \"cystic\", \"fibrosis\", \"11\", \"or\", \"leukemia\", \",\", \"12\", \"or\", \"certain\", \"other\", \"underlying\", \"medical\", \"conditions\", \".\"]\n",
      "Word token=> [\"13\", \"The\", \"enteroviruses\", \"include\", \"pathogens\", \"such\", \"as\", \"EV-D\", \"68\", \",\", \"the\", \"causative\", \"agent\", \"of\", \"the\", \"2014\", \"outbreak\", \"of\", \"the\", \"\\\"\", \"summer\", \"flu\", \"\\\"\", \"in\", \"the\", \"US\", \",\", \"14\", \"EV-A\", \"71\", \"and\", \"Coxsackievirus\", \"A\", \"16\", \"(\", \"CVA\", \"16\", \")\", \",\", \"the\", \"etiological\", \"agents\", \"of\", \"Hand\", \",\", \"Foot\", \",\", \"and\", \"Mouth\", \"Disease\", \"(\", \"HFMD\", \")\", \",\", \"15\", \"Coxsackievirus\", \"B\", \"3\", \"(\", \"CVB\", \"3\", \")\", \",\", \"which\", \"can\", \"cause\", \"myocardic\", \"inflammation\", \",\", \"16\", \"and\", \"human\", \"rhinoviruses\", \"(\", \"HRV\", \")\", \",\", \"notoriously\", \"known\", \"to\", \"lead\", \"to\", \"the\", \"common\", \"cold\", \"but\", \"also\", \"capable\", \"of\", \"causing\", \"exacerbations\", \"of\", \"asthma\", \"and\", \"COPD\", \".\"]\n",
      "Word token=> [\"17\", \"Infection\", \"with\", \"some\", \"of\", \"these\", \"viruses\", \"can\", \"lead\", \"to\", \"serious\", \"outcome\", \";\", \"thus\", \",\", \"EV-D\", \"68\", \"can\", \"cause\", \"polio-like\", \"disease\", \",\", \"18\", \"and\", \"EV-A\", \"71\", \"infection\", \"can\", \"proceed\", \"to\", \"aseptic\", \"meningitis\", \",\", \"encephalitis\", \",\", \"pulmonary\", \"edema\", \",\", \"viral\", \"myocarditis\", \",\", \"and\", \"acute\", \"flaccid\", \"paralysis\", \".\"]\n",
      "Word token=> [\"15\", \",\", \"[\", \"19\", \"]\", \"[\", \"20\", \"]\", \"Enteroviruses\", \"cause\", \"clinical\", \"disease\", \"much\", \"more\", \"frequently\", \"than\", \"coronaviruses\", \",\", \"so\", \"that\", \"an\", \"antiviral\", \"drug\", \"targeting\", \"both\", \"virus\", \"families\", \"should\", \"be\", \"commercially\", \"viable\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"However\", \",\", \"enteroviruses\", \"are\", \"very\", \"different\", \"from\", \"coronaviruses\", \".\"]\n",
      "Word token=> [\"While\", \"both\", \"of\", \"them\", \"have\", \"a\", \"single-stranded\", \"RNA\", \"genome\", \"of\", \"positive\", \"polarity\", \",\", \"that\", \"of\", \"enteroviruses\", \"is\", \"very\", \"small\", \"(\", \"just\", \"7\", \"-\", \"9\", \"kb\", \")\", \"whereas\", \"coronaviruses\", \"feature\", \"the\", \"largest\", \"RNA\", \"genome\", \"known\", \"to\", \"date\", \"(\", \"27\", \"-\", \"34\", \"kb\", \")\", \".\"]\n",
      "Word token=> [\"Enteroviruses\", \"are\", \"small\", \",\", \"naked\", \"particles\", \",\", \"whereas\", \"coronaviruses\", \"are\", \"much\", \"larger\", \"and\", \"enveloped\", \".\"]\n",
      "Word token=> [\"Nevertheless\", \",\", \"a\", \"related\", \"feature\", \"shared\", \"by\", \"these\", \"two\", \"groups\", \"of\", \"viruses\", \"is\", \"their\", \"type\", \"of\", \"major\", \"protease\", \",\", \"21\", \"which\", \"in\", \"the\", \"enteroviruses\", \"is\", \"encoded\", \"by\", \"the\", \"3\", \"C\", \"region\", \"of\", \"the\", \"genome\", \"(\", \"hence\", \"the\", \"protease\", \"is\", \"designated\", \"3\", \"C\", \"pro\", \")\", \".\"]\n",
      "Word token=> [\"In\", \"coronaviruses\", \",\", \"non-structural\", \"protein\", \"5\", \"(\", \"Nsp\", \"5\", \")\", \"is\", \"the\", \"main\", \"protease\", \"(\", \"M\", \"pro\", \")\", \".\"]\n",
      "Word token=> [\"Similar\", \"to\", \"the\", \"enteroviral\", \"3\", \"C\", \"pro\", \",\", \"it\", \"is\", \"a\", \"cysteine\", \"protease\", \"in\", \"the\", \"vast\", \"majority\", \"of\", \"cases\", \"and\", \"has\", \"therefore\", \"also\", \"been\", \"called\", \"\\\"\", \"3\", \"C-like\", \"protease\", \"\\\"\", \"(\", \"3\", \"CL\", \"pro\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"first\", \"crystal\", \"structure\", \"of\", \"a\", \"CoV\", \"M\", \"pro\", \"or\", \"3\", \"CL\", \"pro\", \"(\", \"ref.\", \"22\", \")\", \"revealed\", \"that\", \"two\", \"of\", \"the\", \"three\", \"domains\", \"of\", \"the\", \"enzyme\", \"together\", \"resemble\", \"the\", \"chymotrypsin-like\", \"fold\", \"of\", \"the\", \"enteroviral\", \"3\", \"C\", \"pro\", \",\", \"but\", \"there\", \"is\", \"an\", \"additional\", \"a-helical\", \"domain\", \"that\", \"is\", \"involved\", \"in\", \"the\", \"dimerization\", \"of\", \"the\", \"protease\", \"(\", \"Fig.\", \"1\", \"A\", \")\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"This\", \"dimerization\", \"is\", \"essential\", \"for\", \"the\", \"catalytic\", \"activity\", \"of\", \"the\", \"CoV\", \"M\", \"pro\", \",\", \"whereas\", \"the\", \"enteroviral\", \"3\", \"C\", \"pro\", \"(\", \"Fig.\", \"1\", \"B\", \")\", \"functions\", \"as\", \"a\", \"monomer\", \".\"]\n",
      "Word token=> [\"Further\", \",\", \"the\", \"enteroviral\", \"3\", \"C\", \"pro\", \"features\", \"a\", \"classical\", \"CysHisGlu/Asp\", \"catalytic\", \"triad\", \",\", \"whereas\", \"the\", \"CoV\", \"M\", \"pro\", \"only\", \"has\", \"a\", \"CysHis\", \"dyad\", \".\"]\n",
      "Word token=> [\"22\", \"Yet\", \",\", \"there\", \"are\", \"a\", \"number\", \"of\", \"common\", \"features\", \"shared\", \"between\", \"the\", \"two\", \"types\", \"of\", \"proteases\", \",\", \"in\", \"particular\", \"their\", \"almost\", \"absolute\", \"requirement\", \"for\", \"Gln\", \"in\", \"the\", \"P\", \"1\", \"position\", \"of\", \"the\", \"substrate\", \"and\", \"space\", \"for\", \"only\", \"small\", \"amino-acid\", \"residues\", \"such\", \"as\", \"Gly\", \",\", \"Ala\", \",\", \"or\", \"Ser\", \"in\", \"the\", \"P\", \"1\", \"'\", \"position\", \",\", \"encouraging\", \"us\", \"to\", \"explore\", \"the\", \"coronaviral\", \"M\", \"pro\", \"and\", \"the\", \"enteroviral\", \"3\", \"C\", \"pro\", \"as\", \"a\", \"common\", \"target\", \"for\", \"the\", \"design\", \"of\", \"broad-spectrum\", \"antiviral\", \"compounds\", \".\"]\n",
      "Word token=> [\"The\", \"fact\", \"that\", \"there\", \"is\", \"no\", \"known\", \"human\", \"protease\", \"with\", \"a\", \"specificity\", \"for\", \"Gln\", \"at\", \"the\", \"cleavage\", \"site\", \"of\", \"the\", \"substrate\", \"increases\", \"the\", \"attractivity\", \"of\", \"this\", \"viral\", \"target\", \",\", \"as\", \"there\", \"is\", \"hope\", \"that\", \"the\", \"inhibitors\", \"to\", \"be\", \"developed\", \"will\", \"not\", \"show\", \"toxicity\", \"versus\", \"the\", \"host\", \"cell\", \".\"]\n",
      "Word token=> [\"Indeed\", \",\", \"neither\", \"the\", \"enterovirus\", \"3\", \"C\", \"pro\", \"inhibitor\", \"rupintrivir\", \",\", \"which\", \"was\", \"developed\", \"as\", \"a\", \"treatment\", \"of\", \"the\", \"common\", \"cold\", \"caused\", \"by\", \"HRV\", \",\", \"nor\", \"the\", \"peptide\", \"aldehyde\", \"inhibitor\", \"of\", \"the\", \"coronavirus\", \"M\", \"pro\", \"that\", \"was\", \"recently\", \"demonstrated\", \"to\", \"lead\", \"to\", \"complete\", \"recovery\", \"of\", \"cats\", \"from\", \"the\", \"normally\", \"fatal\", \"infection\", \"with\", \"Feline\", \"Infectious\", \"Peritonitis\", \"Virus\", \"(\", \"FIPV\", \")\", \",\", \"showed\", \"any\", \"toxic\", \"effects\", \"on\", \"humans\", \"or\", \"cats\", \",\", \"respectively\", \".\"]\n",
      "Word token=> [\"23\", \",\", \"24\", \"Figure\", \"1\", \":\", \"Crystal\", \"structures\", \"of\", \"SARS-CoV\", \"main\", \"protease\", \"(\", \"M\", \"pro\", \",\", \"ref.\", \"26\", \";\", \"PDB\", \"entry\", \"2\", \"BX\", \"4\", \")\", \"and\", \"Coxsackivirus\", \"B\", \"3\", \"3\", \"C\", \"protease\", \"(\", \"3\", \"C\", \"pro\", \";\", \"Tan\", \"et\", \"al.\", \",\", \"unpublished\", \";\", \"PDB\", \"entry\", \"3\", \"ZYD\", \")\", \".\"]\n",
      "Word token=> [\"Catalytic\", \"residues\", \"are\", \"indicated\", \"by\", \"spheres\", \"(\", \"yellow\", \",\", \"Cys\", \";\", \"blue\", \",\", \"His\", \";\", \"red\", \":\", \"Glu\", \")\", \".\"]\n",
      "Word token=> [\"A.\", \"The\", \"coronavirus\", \"M\", \"pro\", \"is\", \"a\", \"homodimer\", \",\", \"with\", \"each\", \"monomer\", \"comprising\", \"three\", \"domains\", \".\"]\n",
      "Word token=> [\"B.\", \"The\", \"structure\", \"of\", \"the\", \"monomeric\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \"resembles\", \"the\", \"Nterminal\", \"two\", \"domains\", \"of\", \"the\", \"SARS-CoV\", \"M\", \"pro\", \".\"]\n",
      "Word token=> [\"Structure\", \"is\", \"on\", \"the\", \"same\", \"scale\", \"as\", \"image\", \"A.\", \"C.\", \"Superimpostion\", \"of\", \"residues\", \"from\", \"the\", \"two\", \"structures\", \"involved\", \"in\", \"ligand\", \"binding\", \".\"]\n",
      "Word token=> [\"Superimposition\", \"was\", \"carried\", \"out\", \"by\", \"aligning\", \"the\", \"catalytic\", \"Cys-His\", \"pair\", \"of\", \"each\", \"protease\", \".\"]\n",
      "Word token=> [\"Residues\", \"of\", \"the\", \"SARS-CoV\", \"M\", \"pro\", \"are\", \"shown\", \"with\", \"carbon\", \"atoms\", \"in\", \"cyan\", \",\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \"residues\", \"have\", \"orange\", \"carbons\", \"and\", \"are\", \"labeled\", \"with\", \"an\", \"asterisk\", \"(\", \"*\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"As\", \"the\", \"proteases\", \"targeted\", \"in\", \"our\", \"study\", \"all\", \"specifically\", \"cleave\", \"the\", \"peptide\", \"bond\", \"following\", \"a\", \"P\", \"1\", \"-glutamine\", \"residue\", \"(\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \"uniquely\", \"also\", \"accepts\", \"P\", \"1\", \"=\", \"His\", \"at\", \"the\", \"Nsp\", \"13\", \"/Nsp\", \"14\", \"cleavage\", \"site\", \"31\", \")\", \",\", \"we\", \"decided\", \"to\", \"use\", \"a\", \"5\", \"-membered\", \"ring\", \"(\", \"g-lactam\", \")\", \"derivative\", \"of\", \"glutamine\", \"(\", \"henceforth\", \"called\", \"GlnLactam\", \")\", \"as\", \"the\", \"P\", \"1\", \"residue\", \"in\", \"all\", \"our\", \"aketoamides\", \"(\", \"see\", \"Scheme\", \"1\", \")\", \".\"]\n",
      "Word token=> [\"This\", \"moiety\", \"has\", \"been\", \"found\", \"to\", \"be\", \"a\", \"good\", \"mimic\", \"of\", \"glutamine\", \"and\", \"enhance\", \"the\", \"power\", \"of\", \"the\", \"inhibitors\", \"by\", \"up\", \"to\", \"10\", \"-fold\", \",\", \"most\", \"probably\", \"because\", \"compared\", \"to\", \"the\", \"flexible\", \"glutamine\", \"side-chain\", \",\", \"the\", \"more\", \"rigid\", \"lactam\", \"leads\", \"to\", \"a\", \"reduction\", \"of\", \"the\", \"loss\", \"of\", \"entropy\", \"upon\", \"binding\", \"to\", \"the\", \"target\", \"protease\", \".\"]\n",
      "Word token=> [\"29\", \",\", \"32\", \"Our\", \"synthetic\", \"efforts\", \"therefore\", \"aimed\", \"at\", \"optimizing\", \"the\", \"substituents\", \"at\", \"the\", \"P\", \"1\", \"'\", \",\", \"P\", \"2\", \",\", \"and\", \"P\", \"3\", \"positions\", \"of\", \"the\", \"a-ketoamides\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Synthesis\", \"(\", \"Scheme\", \"1\", \")\", \"started\", \"with\", \"the\", \"dianionic\", \"alkylation\", \"of\", \"N-Boc\", \"glutamic\", \"acid\", \"dimethyl\", \"ester\", \"with\", \"bromoacetonitrile\", \".\"]\n",
      "Word token=> [\"As\", \"expected\", \",\", \"this\", \"alkylation\", \"occurred\", \"in\", \"a\", \"highly\", \"stereoselective\", \"manner\", \",\", \"giving\", \"1\", \"as\", \"the\", \"exclusive\", \"product\", \".\"]\n",
      "Word token=> [\"In\", \"the\", \"following\", \"step\", \",\", \"the\", \"cyano\", \"group\", \"of\", \"1\", \"was\", \"subjected\", \"to\", \"hydrogenation\", \".\"]\n",
      "Word token=> [\"The\", \"in-situ\", \"cyclization\", \"of\", \"the\", \"resulting\", \"intermediate\", \"afforded\", \"the\", \"lactam\", \"2\", \".\"]\n",
      "Word token=> [\"The\", \"lactam\", \"derivative\", \"3\", \"was\", \"generated\", \"by\", \"removal\", \"of\", \"the\", \"protecting\", \"group\", \"of\", \"2\", \".\"]\n",
      "Word token=> [\"On\", \"the\", \"other\", \"hand\", \",\", \"the\", \"amidation\", \"of\", \"acyl\", \"chloride\", \"and\", \"a-amino\", \"acid\", \"methyl\", \"ester\", \"afforded\", \"the\", \"intermediates\", \"4\", \",\", \"which\", \"gave\", \"rise\", \"to\", \"the\", \"acids\", \"5\", \"via\", \"alkaline\", \"hydrolysis\", \".\"]\n",
      "Word token=> [\"The\", \"key\", \"intermediates\", \"6\", \"were\", \"obtained\", \"via\", \"the\", \"condensation\", \"of\", \"the\", \"lactam\", \"derivative\", \"3\", \"and\", \"the\", \"N-capped\", \"amino\", \"acids\", \"5\", \".\"]\n",
      "Word token=> [\"The\", \"ester\", \"group\", \"of\", \"compounds\", \"6\", \"was\", \"then\", \"reduced\", \"to\", \"the\", \"corresponding\", \"alcohol\", \".\"]\n",
      "Word token=> [\"Oxidation\", \"of\", \"the\", \"alcohol\", \"products\", \"7\", \"by\", \"Dess-Martin\", \"periodinane\", \"generated\", \"the\", \"aldehydes\", \"8\", \",\", \"which\", \"followed\", \"by\", \"nucleophilic\", \"addition\", \"with\", \"isocyanides\", \"gave\", \"rise\", \"to\", \"compounds\", \"9\", \"under\", \"acidic\", \"conditions\", \".\"]\n",
      "Word token=> [\"Then\", \",\", \"the\", \"a-hydroxyamides\", \"10\", \"were\", \"prepared\", \"by\", \"removing\", \"the\", \"acetyl\", \"group\", \"of\", \"compounds\", \"9\", \".\"]\n",
      "Word token=> [\"In\", \"the\", \"final\", \"step\", \",\", \"the\", \"oxidation\", \"of\", \"the\", \"exposed\", \"alcohol\", \"group\", \"in\", \"compounds\", \"10\", \"generated\", \"our\", \"target\", \"a-ketoamides\", \"11\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"inhibitory\", \"potencies\", \"of\", \"candidate\", \"a-ketoamides\", \"were\", \"evaluated\", \"against\", \"purified\", \"recombinant\", \"SARS-CoV\", \"M\", \"pro\", \",\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \",\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \",\", \"and\", \"EV-A\", \"71\", \"3\", \"C\", \"pro\", \".\"]\n",
      "Word token=> [\"The\", \"most\", \"potent\", \"compounds\", \"were\", \"further\", \"tested\", \"against\", \"viral\", \"replicons\", \"and\", \"against\", \"SARS-CoV\", \",\", \"MERS-CoV\", \",\", \"or\", \"a\", \"whole\", \"range\", \"of\", \"enteroviruses\", \"in\", \"cell\", \"culture-based\", \"assays\", \"(\", \"Tables\", \"1\", \"-\", \"3\", \"and\", \"Supplementary\", \"Table\", \"1\"]\n",
      "Word token=> [\"To\", \"enable\", \"the\", \"rapid\", \"and\", \"biosafe\", \"screening\", \"of\", \"antivirals\", \"against\", \"corona-and\", \"enteroviruses\", \",\", \"a\", \"non-infectious\", \",\", \"but\", \"replication-competent\", \"SARS-CoV\", \"replicon\", \"was\", \"used\", \"33\", \"along\", \"with\", \"subgenomic\", \"replicons\", \"of\", \"CVB\", \"3\", \"34\", \"and\", \"EV-A\", \"71\", \"(\", \"kind\", \"gift\", \"of\", \"B.\", \"Zhang\", \",\", \"Wuhan\", \",\", \"China\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"easily\", \"detectable\", \"reporter\", \"activity\", \"(\", \"firefly\", \"or\", \"Renilla\", \"luciferase\", \")\", \"of\", \"these\", \"replicons\", \"has\", \"previously\", \"been\", \"shown\", \"to\", \"reflect\", \"viral\", \"RNA\", \"synthesis\", \".\"]\n",
      "Word token=> [\"[\", \"33\", \"]\", \"[\", \"34\", \"]\", \"[\", \"35\", \"]\", \"In-vitro\", \"RNA\", \"transcripts\", \"of\", \"the\", \"enteroviral\", \"replicons\", \"were\", \"also\", \"used\", \"for\", \"transfection\", \".\"]\n",
      "Word token=> [\"For\", \"the\", \"SARS-CoV\", \"replicon\", \"containing\", \"the\", \"CMV\", \"promoter\", \",\", \"only\", \"the\", \"plasmid\", \"DNA\", \"was\", \"used\", \"for\", \"transfection\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"initial\", \"compound\", \"to\", \"be\", \"designed\", \"and\", \"synthesized\", \"was\", \"11\", \"a\", \",\", \"which\", \"carries\", \"a\", \"cinnamoyl\", \"N-cap\", \"in\", \"the\", \"P\", \"3\", \"position\", \",\", \"a\", \"benzyl\", \"group\", \"in\", \"P\", \"2\", \",\", \"the\", \"glutamine\", \"lactam\", \"(\", \"GlnLactam\", \")\", \"in\", \"P\", \"1\", \",\", \"and\", \"benzyl\", \"in\", \"P\", \"1\", \"'\", \"(\", \"Table\", \"1\", \")\", \".\"]\n",
      "Word token=> [\"This\", \"compound\", \"showed\", \"good\", \"to\", \"mediocre\", \"activities\", \"against\", \"recombinant\", \"SARS-CoV\", \"M\", \"pro\", \"(\", \"IC\", \"50\", \"=\", \"1.95\", \"µM\", \";\", \"for\", \"all\", \"compounds\", \",\", \"see\", \"Tables\", \"1\", \"-\", \"3\", \"for\", \"standard\", \"deviations\", \")\", \",\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \"(\", \"IC\", \"50\", \"=\", \"6.6\", \"µM\", \")\", \",\", \"and\", \"EV-A\", \"71\", \"3\", \"C\", \"pro\", \"(\", \"IC\", \"50\", \"=\", \"1.2\", \"µM\", \")\", \",\", \"but\", \"was\", \"surprisingly\", \"completely\", \"inactive\", \"(\", \"IC\", \"50\", \">\", \"50\", \"µM\", \")\", \"against\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \".\"]\n",
      "Word token=> [\"These\", \"values\", \"were\", \"mirrored\", \"in\", \"the\", \"SARS-CoV\", \"and\", \"in\", \"the\", \"enterovirus\", \"replicons\", \"(\", \"Table\", \"2\", \")\", \".\"]\n",
      "Word token=> [\"In\", \"virus-infected\", \"cell\", \"cultures\", \",\", \"the\", \"results\", \"obtained\", \"were\", \"also\", \"good\", \"to\", \"mediocre\", \"(\", \"Table\", \"3\", \")\", \":\", \"SARS-CoV\", \"(\", \"EC\", \"50\", \"=\", \"5.8\", \"µM\", \"in\", \"Vero\", \"E\", \"6\", \"cells\", \")\", \",\", \"MERS-CoV\", \"(\", \"EC\", \"50\", \"=\", \"0.0047\", \"µM\", \"in\", \"Huh\", \"7\", \"cells\", \")\", \",\", \"HCoV\", \"229\", \"E\", \"(\", \"EC\", \"50\", \"=\", \"11.8\", \"µM\", \"in\", \"Huh\", \"7\", \"cells\", \")\", \",\", \"or\", \"a\", \"host\", \"of\", \"enteroviruses\", \"(\", \"EC\", \"50\", \"=\", \"9.8\", \"µM\", \"against\", \"EV-A\", \"71\", \"in\", \"RD\", \"cells\", \";\", \"EC\", \"50\", \"=\", \"0.48\", \"µM\", \"against\", \"EV-D\", \"68\", \"in\", \"HeLa\", \"Rh\", \"cells\", \";\", \"EC\", \"50\", \"=\", \"5.6\", \"µM\", \"against\", \"HRV\", \"2\", \"in\", \"HeLa\", \"Rh\", \"cells\", \")\", \".\"]\n",
      "Word token=> [\"In\", \"all\", \"cell\", \"types\", \"tested\", \",\", \"the\", \"compound\", \"generally\", \"proved\", \"to\", \"be\", \"non-toxic\", \",\", \"with\", \"selectivity\", \"indices\", \"(\", \"CC\", \"50\", \"/EC\", \"50\", \")\", \"usually\", \">\", \"10\", \"(\", \"Table\", \"3\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Crystal\", \"structures\", \"of\", \"compound\", \"11\", \"a\", \"in\", \"complex\", \"with\", \"SARS-CoV\", \"M\", \"pro\", \",\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \",\", \"and\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \"demonstrated\", \"that\", \"the\", \"aketo-carbon\", \"is\", \"covalently\", \"linked\", \"to\", \"the\", \"active-site\", \"Cys\", \"(\", \"no.\", \"145\", \",\", \"144\", \",\", \"and\", \"147\", \",\", \"resp.\", \")\", \"of\", \"the\", \"protease\", \"(\", \"Fig.\", \"2\", \",\", \"3\", \"a-c\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"resulting\", \"thiohemiketal\", \"is\", \"in\", \"the\", \"R\", \"configuration\", \"in\", \"the\", \"SARS-CoV\", \"and\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \"but\", \"in\", \"the\", \"S\", \"configuration\", \"in\", \"the\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \"complex\", \".\"]\n",
      "Word token=> [\"The\", \"reason\", \"for\", \"this\", \"difference\", \"is\", \"that\", \"the\", \"oxygen\", \"atom\", \"of\", \"the\", \"thiohemiketal\", \"accepts\", \"a\", \"hydrogen\", \"bond\", \"from\", \"the\", \"catalytic\", \"His\", \"40\", \"in\", \"the\", \"CVB\", \"3\", \"protease\", \",\", \"rather\", \"than\", \"from\", \"the\", \"main-chain\", \"amides\", \"of\", \"the\", \"oxyanion\", \"hole\", \"as\", \"in\", \"the\", \"SARS-CoV\", \"and\", \"HCoV-NL\", \"63\", \"enzymes\", \"(\", \"Fig.\", \"3\", \"a\", \",\", \"b\", \",\", \"c\", \"insets\", \")\", \".\"]\n",
      "Word token=> [\"It\", \"is\", \"remarkable\", \"that\", \"we\", \"succeeded\", \"in\", \"obtaining\", \"a\", \"crystal\", \"structure\", \"of\", \"compound\", \"11\", \"a\", \"in\", \"complex\", \"with\", \"the\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \",\", \"even\", \"though\", \"it\", \"has\", \"no\", \"inhibitory\", \"effect\", \"on\", \"the\", \"activity\", \"of\", \"the\", \"enzyme\", \"(\", \"IC\", \"50\", \">\", \"50\", \"µM\", \")\", \",\", \"(\", \"Fig.\", \"2\", \"c\", \")\", \".\"]\n",
      "Word token=> [\"Apparently\", \",\", \"the\", \"compound\", \"is\", \"able\", \"to\", \"bind\", \"to\", \"this\", \"M\", \"pro\", \"in\", \"the\", \"absence\", \"of\", \"peptide\", \"substrate\", \",\", \"but\", \"cannot\", \"compete\", \"with\", \"substrate\", \"for\", \"the\", \"binding\", \"site\", \"due\", \"to\", \"low\", \"affinity\", \".\"]\n",
      "Word token=> [\"A\", \"similar\", \"observation\", \"has\", \"been\", \"made\", \"in\", \"one\", \"of\", \"our\", \"previous\", \"studies\", \",\", \"where\", \"we\", \"were\", \"able\", \"to\", \"determine\", \"the\", \"crystal\", \"structure\", \"of\", \"a\", \"complex\", \"between\", \"the\", \"inactive\", \"Michaelacceptor\", \"compound\", \"SG\", \"74\", \"and\", \"the\", \"EV-D\", \"68\", \"3\", \"C\", \"pro\", \"(\", \"ref.\", \"29\", \";\", \"PDB\", \"entry\", \"3\", \"ZV\", \"9\", \")\", \".\"]\n",
      "Word token=> [\"Figure\", \"2\", \":\", \"Fit\", \"of\", \"compound\", \"11\", \"a\", \"(\", \"pink\", \"carbon\", \"atoms\", \")\", \"to\", \"the\", \"target\", \"proteases\", \"(\", \"wheat\", \"surfaces\", \")\", \"as\", \"revealed\", \"by\", \"X-ray\", \"crystallography\", \"of\", \"the\", \"complexes\", \".\"]\n",
      "Word token=> [\"A.\", \"Fo\", \"-Fc\", \"difference\", \"density\", \"(\", \"contoured\", \"at\", \"3\", \"s\", \")\", \"for\", \"11\", \"a\", \"in\", \"the\", \"substrate-binding\", \"site\", \"of\", \"the\", \"SARS-CoV\", \"M\", \"pro\", \"(\", \"transparent\", \"surface\", \")\", \".\"]\n",
      "Word token=> [\"Selected\", \"side-chains\", \"of\", \"the\", \"protease\", \"are\", \"shown\", \"with\", \"green\", \"carbon\", \"atoms\", \".\"]\n",
      "Word token=> [\"B.\", \"Another\", \"view\", \"of\", \"11\", \"a\", \"in\", \"the\", \"substrate\", \"binding\", \"site\", \"of\", \"the\", \"SARS-CoV\", \"M\", \"pro\", \".\"]\n",
      "Word token=> [\"Note\", \"the\", \"\\\"\", \"lid\", \"\\\"\", \"formed\", \"by\", \"residue\", \"Met\", \"49\", \"and\", \"its\", \"neighbors\", \"above\", \"the\", \"S\", \"2\", \"pocket\", \".\"]\n",
      "Word token=> [\"C.\", \"11\", \"a\", \"in\", \"the\", \"substrate-binding\", \"site\", \"of\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \".\"]\n",
      "Word token=> [\"Because\", \"of\", \"the\", \"restricted\", \"size\", \"of\", \"the\", \"S\", \"2\", \"pocket\", \",\", \"the\", \"P\", \"2\", \"benzyl\", \"group\", \"of\", \"the\", \"compound\", \"cannot\", \"enter\", \"deeply\", \"into\", \"this\", \"site\", \".\"]\n",
      "Word token=> [\"Note\", \"that\", \"the\", \"S\", \"2\", \"pocket\", \"is\", \"also\", \"covered\", \"by\", \"a\", \"\\\"\", \"lid\", \"\\\"\", \"centred\", \"around\", \"Thr\", \"47\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"D.\", \"11\", \"a\", \"in\", \"the\", \"substrate-binding\", \"site\", \"of\", \"the\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \".\"]\n",
      "Word token=> [\"The\", \"S\", \"2\", \"site\", \"is\", \"large\", \"and\", \"not\", \"covered\", \"by\", \"a\", \"\\\"\", \"lid\", \"\\\"\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"crystal\", \"structures\", \"indicated\", \"that\", \"the\", \"fits\", \"of\", \"the\", \"P\", \"1\", \"'\", \"benzyl\", \"group\", \"of\", \"11\", \"a\", \"in\", \"the\", \"S\", \"1\", \"'\", \"pocket\", \"and\", \"of\", \"the\", \"P\", \"3\", \"cinnamoyl\", \"cap\", \"in\", \"the\", \"S\", \"3\", \"subsite\", \"might\", \"be\", \"improved\", \"(\", \"see\", \"Fig.\", \"3\", \"a\", \"-c\", \")\", \".\"]\n",
      "Word token=> [\"Compounds\", \"11\", \"b\", \"-\", \"11\", \"e\", \"and\", \"11\", \"g\", \"-\", \"11\", \"l\", \"were\", \"synthesized\", \"in\", \"an\", \"attempt\", \"to\", \"do\", \"so\", \";\", \"however\", \",\", \"none\", \"of\", \"them\", \"showed\", \"better\", \"inhibitory\", \"activity\", \"against\", \"the\", \"majority\", \"of\", \"the\", \"recombinant\", \"proteases\", \",\", \"compared\", \"to\", \"the\", \"parent\", \"compound\", \",\", \"11\", \"a\", \"(\", \"see\", \"Supplementary\", \"Results\", \")\", \".\"]\n",
      "Word token=> [\"To\", \"investigate\", \"whether\", \"the\", \"P\", \"3\", \"residue\", \"of\", \"the\", \"inhibitor\", \"is\", \"dispensible\", \",\", \"we\", \"synthesized\", \"compound\", \"11\", \"f\", \",\", \"which\", \"only\", \"comprises\", \"P\", \"2\", \"=\", \"Boc\", \",\", \"P\", \"1\", \"=\", \"GlnLactam\", \",\", \"and\", \"P\", \"1\", \"'\", \"=\", \"benzyl\", \".\"]\n",
      "Word token=> [\"11\", \"f\", \"was\", \"inactive\", \"against\", \"all\", \"purified\", \"proteases\", \"and\", \"in\", \"all\", \"replicons\", \"tested\", \",\", \"but\", \"showed\", \"some\", \"activity\", \"against\", \"HRV\", \"2\", \"in\", \"HeLa\", \"Rh\", \"cells\", \"(\", \"EC\", \"50\", \"=\", \"9.0\", \"µM\", \")\", \".\"]\n",
      "Word token=> [\"A\", \"crystal\", \"structure\", \"of\", \"11\", \"f\", \"bound\", \"to\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \"demonstrated\", \"that\", \"the\", \"P\", \"2\", \"-Boc\", \"group\", \"entered\", \"the\", \"S\", \"2\", \"pocket\", \"(\", \"Fig.\", \"3\", \"d\", \")\", \".\"]\n",
      "Word token=> [\"In\", \"conclusion\", \",\", \"although\", \"there\", \"is\", \"probably\", \"room\", \"for\", \"further\", \"improvement\", \",\", \"we\", \"decided\", \"to\", \"maintain\", \"the\", \"original\", \"design\", \"with\", \"P\", \"1\", \"'\", \"=\", \"benzyl\", \"and\", \"P\", \"3\", \"=\", \"cinnamoyl\", \",\", \"and\", \"focussed\", \"on\", \"improving\", \"the\", \"P\", \"2\", \"substituen\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"crystal\", \"structures\", \"of\", \"SARS-CoV\", \"M\", \"pro\", \",\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \",\", \"and\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \"in\", \"complex\", \"with\", \"11\", \"a\", \"revealed\", \"a\", \"fundamental\", \"difference\", \"between\", \"the\", \"S\", \"2\", \"pockets\", \"of\", \"the\", \"coronavirus\", \"proteases\", \"and\", \"the\", \"enterovirus\", \"proteases\", \":\", \"The\", \"cavities\", \"are\", \"covered\", \"by\", \"a\", \"\\\"\", \"lid\", \"\\\"\", \"in\", \"the\", \"former\", \"but\", \"are\", \"open\", \"to\", \"one\", \"side\", \"in\", \"the\", \"latter\", \"(\", \"Fig.\", \"2\", \",\", \"b-d\", \")\", \".\"]\n",
      "Word token=> [\"In\", \"SARS-CoV\", \"M\", \"pro\", \",\", \"the\", \"lid\", \"is\", \"formed\", \"by\", \"the\", \"310\", \"helix\", \"46\", \"-\", \"51\", \"and\", \"in\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \"by\", \"the\", \"loop\", \"43\", \"-\", \"48\", \".\"]\n",
      "Word token=> [\"Residues\", \"from\", \"the\", \"lid\", \",\", \"in\", \"particular\", \"Met\", \"49\", \"in\", \"the\", \"case\", \"of\", \"SARS-CoV\", \"M\", \"pro\", \",\", \"can\", \"thus\", \"make\", \"hydrophobic\", \"interactions\", \"with\", \"the\", \"P\", \"2\", \"substitutent\", \"of\", \"the\", \"inhibitor\", \",\", \"whereas\", \"such\", \"interaction\", \"is\", \"missing\", \"in\", \"the\", \"enterovirus\", \"3\", \"C\", \"pro\", \"s\", \".\"]\n",
      "Word token=> [\"In\", \"addition\", \"to\", \"the\", \"lid\", \",\", \"the\", \"S\", \"2\", \"pocket\", \"is\", \"lined\", \"by\", \"the\", \"\\\"\", \"back-wall\", \"\\\"\", \"(\", \"main-chain\", \"atoms\", \"of\", \"residues\", \"186\", \"and\", \"188\", \"and\", \"Cb\", \"atom\", \"of\", \"Asp\", \"187\", \")\", \",\", \"the\", \"side-walls\", \"(\", \"Gln\", \"189\", \",\", \"His\", \"41\", \")\", \",\", \"as\", \"well\", \"as\", \"the\", \"\\\"\", \"floor\", \"\\\"\", \"(\", \"Met\", \"165\", \")\", \"in\", \"SARS-CoV\", \"M\", \"pro\", \".\"]\n",
      "Word token=> [\"In\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \",\", \"the\", \"corresponding\", \"structural\", \"elements\", \"are\", \"main-chain\", \"atoms\", \"of\", \"residues\", \"187\", \"and\", \"188\", \"as\", \"well\", \"as\", \"the\", \"Cb\", \"atom\", \"of\", \"Asp\", \"187\", \"(\", \"back-wall\", \")\", \",\", \"Pro\", \"189\", \"and\", \"His\", \"41\", \"(\", \"side-walls\", \")\", \",\", \"and\", \"Ile\", \"165\", \"(\", \"floor\", \")\", \".\"]\n",
      "Word token=> [\"Finally\", \",\", \"in\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \",\", \"Arg\", \"39\", \",\", \"Asn\", \"69\", \",\", \"and\", \"Glu\", \"71\", \"form\", \"the\", \"back-wall\", \",\", \"residues\", \"127\", \"-\", \"132\", \"and\", \"His\", \"40\", \"form\", \"the\", \"side-walls\", \",\", \"and\", \"Val\", \"162\", \"constitutes\", \"the\", \"floor\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"In\", \"addition\", \",\", \"the\", \"S\", \"2\", \"pocket\", \"is\", \"of\", \"different\", \"size\", \"in\", \"the\", \"various\", \"proteases\", \".\"]\n",
      "Word token=> [\"The\", \"SARS-CoV\", \"enzyme\", \"features\", \"the\", \"largest\", \"S\", \"2\", \"pocket\", \",\", \"with\", \"a\", \"volume\", \"of\", \"252\", \"Å\", \"3\", \"embraced\", \"by\", \"the\", \"residues\", \"(\", \"Gln\", \"189\", \",\", \"His\", \"41\", \")\", \"defining\", \"the\", \"side-walls\", \"of\", \"the\", \"pocket\", \"in\", \"the\", \"ligand-free\", \"enzyme\", \",\", \"as\", \"calculated\", \"by\", \"using\", \"Chimera\", \",\", \"36\", \"followed\", \"by\", \"the\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \"S\", \"2\", \"pocket\", \"with\", \"about\", \"180\", \"Å\", \"3\", \"(\", \"space\", \"between\", \"Thr\", \"130\", \"and\", \"His\", \"40\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \"has\", \"by\", \"far\", \"the\", \"smallest\", \"S\", \"2\", \"pocket\", \"of\", \"the\", \"three\", \"enzymes\", \",\", \"with\", \"a\", \"free\", \"space\", \"of\", \"only\", \"45\", \"Å\", \"3\", \"between\", \"Pro\", \"189\", \"and\", \"His\", \"41\", \",\", \"according\", \"to\", \"Chimera\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"In\", \"agreement\", \"with\", \"these\", \"observations\", \",\", \"a\", \"good\", \"fit\", \"is\", \"observed\", \"between\", \"the\", \"P\", \"2\", \"benzyl\", \"group\", \"of\", \"11\", \"a\", \"and\", \"the\", \"S\", \"2\", \"subsite\", \"of\", \"the\", \"SARS-CoV\", \"M\", \"pro\", \"as\", \"well\", \"as\", \"that\", \"of\", \"the\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \"(\", \"Fig.\", \"3\", \"a\", \",\", \"c\", \")\", \".\"]\n",
      "Word token=> [\"In\", \"contrast\", \",\", \"the\", \"crystal\", \"structure\", \"of\", \"the\", \"complex\", \"between\", \"11\", \"a\", \"and\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \",\", \"against\", \"which\", \"the\", \"compound\", \"is\", \"inactive\", \",\", \"demonstrates\", \"that\", \"the\", \"P\", \"2\", \"-benzyl\", \"group\", \"cannot\", \"fully\", \"enter\", \"the\", \"S\", \"2\", \"pocket\", \"of\", \"the\", \"enzyme\", \"because\", \"of\", \"the\", \"restricted\", \"size\", \"of\", \"this\", \"site\", \"(\", \"Fig.\", \"3\", \"b\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Thus\", \",\", \"the\", \"properties\", \"of\", \"our\", \"target\", \"proteases\", \"with\", \"respect\", \"to\", \"the\", \"S\", \"2\", \"pocket\", \"were\", \"defined\", \"at\", \"this\", \"point\", \"as\", \"\\\"\", \"small\", \"\\\"\", \"and\", \"\\\"\", \"covered\", \"by\", \"a\", \"lid\", \"\\\"\", \"for\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \",\", \"\\\"\", \"large\", \"\\\"\", \"and\", \"\\\"\", \"covered\", \"\\\"\", \"for\", \"SARS-CoV\", \"M\", \"pro\", \",\", \"and\", \"\\\"\", \"large\", \"\\\"\", \"and\", \"\\\"\", \"open\", \"\\\"\", \"for\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \".\"]\n",
      "Word token=> [\"Through\", \"comparison\", \"with\", \"crystal\", \"structures\", \"of\", \"other\", \"proteases\", \"of\", \"the\", \"same\", \"virus\", \"genus\", \"(\", \"HCoV-\", \"229\", \"E\", \"M\", \"pro\", \"for\", \"alphacoronaviruses\", \"28\", \"(\", \"PDB\", \"entry\", \"1\", \"P\", \"9\", \"S\", \")\", \";\", \"HKU\", \"4\", \"M\", \"pro\", \"for\", \"betacoronaviruses\", \"(\", \"Ma\", \",\", \"Xiao\", \"et\", \"al.\", \",\", \"unpublished\", \";\", \"PDB\", \"entry\", \"2\", \"YNA\", \")\", \";\", \"and\", \"EV-A\", \"71\", \"3\", \"C\", \"pro\", \"for\", \"enteroviruses\", \"30\", \"(\", \"PDB\", \"entry\", \"3\", \"SGK\", \")\", \",\", \"we\", \"ensured\", \"that\", \"our\", \"conclusions\", \"drawn\", \"from\", \"the\", \"template\", \"structures\", \"were\", \"valid\", \"for\", \"other\", \"family\", \"members\", \"as\", \"well\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"To\", \"explore\", \"the\", \"sensitivity\", \"of\", \"the\", \"S\", \"2\", \"pocket\", \"towards\", \"a\", \"polar\", \"substituent\", \"in\", \"the\", \"para\", \"position\", \"of\", \"the\", \"benzyl\", \"group\", \",\", \"we\", \"synthesized\", \"compound\", \"11\", \"m\", \"carrying\", \"a\", \"4\", \"-fluorobenzyl\", \"group\", \"in\", \"P\", \"2\", \".\"]\n",
      "Word token=> [\"This\", \"substitution\", \"abolished\", \"almost\", \"all\", \"activity\", \"against\", \"the\", \"SARS-CoV\", \"M\", \"pro\", \"(\", \"IC\", \"50\", \">\", \"50\", \"µM\", \")\", \",\", \"and\", \"the\", \"compound\", \"proved\", \"inactive\", \"against\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \"as\", \"well\", \",\", \"whereas\", \"IC\", \"50\", \"values\", \"were\", \"2.3\", \"µM\", \"against\", \"the\", \"EV-A\", \"71\", \"3\", \"C\", \"pro\", \"and\", \"8.7\", \"µM\", \"against\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \".\"]\n",
      "Word token=> [\"From\", \"this\", \",\", \"we\", \"concluded\", \"that\", \"the\", \"introduction\", \"of\", \"the\", \"polar\", \"fluorine\", \"atom\", \"is\", \"not\", \"compatible\", \"with\", \"the\", \"geometry\", \"of\", \"the\", \"S\", \"2\", \"pocket\", \"of\", \"SARS-CoV\", \"M\", \"pro\", \",\", \"whereas\", \"the\", \"fluorine\", \"can\", \"accept\", \"a\", \"hydrogen\", \"bond\", \"from\", \"Arg\", \"39\", \"in\", \"EV-A\", \"71\", \"3\", \"C\", \"pro\", \"(\", \"ref.\", \"30\", \")\", \"and\", \"probably\", \"also\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \".\"]\n",
      "Word token=> [\"In\", \"SARS-CoV\", \"M\", \"pro\", \",\", \"however\", \",\", \"the\", \"carbonyl\", \"groups\", \"of\", \"residues\", \"186\", \"and\", \"188\", \"might\", \"lead\", \"to\", \"a\", \"repulsion\", \"of\", \"the\", \"fluorinated\", \"benzyl\", \"group\", \".\"]\n",
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.02.10.936898\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"Table\", \"2\", \":\", \"a-ketoamide-induced\", \"inhibition\", \"of\", \"subgenomic\", \"RNA\", \"synthesis\", \"using\", \"replicons\", \"in\", \"a\", \"cell-based\", \"assay\", \"(\", \"EC\", \"50\", \",\", \"μM\", \")\", \"1\", \"2\", \"3\", \"4\", \"5\", \"6\", \"7\", \"8\", \"9\", \"10\", \"11\", \"12\", \"13\", \"14\", \"15\", \"16\", \"17\", \"18\", \"19\", \"20\", \"21\", \"22\", \"23\", \"24\", \"25\", \"26\", \"27\", \"28\", \"29\", \"30\", \"31\", \"32\", \"33\", \"34\", \"35\", \"36\", \"37\", \"38\", \"39\", \"40\", \"41\", \"42\", \"43\", \"44\", \"45\", \"46\", \"47\", \"48\", \"49\", \"50\", \"51\", \"52\", \"53\", \"54\", \"55\", \"56\", \"57\", \"58\", \"59\", \"60\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.02.10.936898\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"Table\", \"3\", \".\"]\n",
      "Word token=> [\"The\", \"thiohemiketal\", \"is\", \"in\", \"the\", \"R\", \"configuration\", \",\", \"with\", \"its\", \"oxygen\", \"accepting\", \"two\", \"hydrogen\", \"bonds\", \"from\", \"the\", \"oxyanion-hole\", \"amides\", \"of\", \"Gly\", \"143\", \"and\", \"Cys\", \"145\", \".\"]\n",
      "Word token=> [\"The\", \"amide\", \"oxygen\", \"accepts\", \"an\", \"H-bond\", \"from\", \"His\", \"41\", \".\"]\n",
      "Word token=> [\"The\", \"side-chains\", \"of\", \"Ser\", \"144\", \"and\", \"Arg\", \"188\", \"have\", \"been\", \"omitted\", \"for\", \"clarity\", \".\"]\n",
      "Word token=> [\"B.\", \"The\", \"P\", \"2\", \"-benzyl\", \"substituent\", \"of\", \"11\", \"a\", \"cannot\", \"fully\", \"enter\", \"the\", \"S\", \"2\", \"pocket\", \"of\", \"the\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \",\", \"which\", \"is\", \"much\", \"smaller\", \"and\", \"has\", \"less\", \"plasticity\", \"than\", \"the\", \"corresponding\", \"pocket\", \"of\", \"SARS-CoV\", \"M\", \"pro\", \"(\", \"cf.\", \"A\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"benzyl\", \"therefore\", \"binds\", \"above\", \"the\", \"pocket\", \"in\", \"the\", \"view\", \"shown\", \"here\", \";\", \"this\", \"is\", \"probably\", \"the\", \"reason\", \"for\", \"the\", \"total\", \"inactivity\", \"(\", \"IC\", \"50\", \">\", \"50\", \"µM\", \")\", \"of\", \"compound\", \"11\", \"a\", \"against\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \".\"]\n",
      "Word token=> [\"The\", \"small\", \"size\", \"of\", \"the\", \"pocket\", \"is\", \"due\", \"to\", \"the\", \"replacement\", \"of\", \"the\", \"flexible\", \"Gln\", \"189\", \"of\", \"the\", \"SARS-CoV\", \"M\", \"pro\", \"by\", \"the\", \"more\", \"rigid\", \"Pro\", \"189\", \"in\", \"this\", \"enzyme\", \".\"]\n",
      "Word token=> [\"The\", \"stereochemistry\", \"of\", \"the\", \"thiohemiketal\", \"is\", \"R.\", \"The\", \"sidechains\", \"of\", \"Ala\", \"143\", \"and\", \"Gln\", \"188\", \"have\", \"been\", \"omitted\", \"for\", \"clarity\", \".\"]\n",
      "Word token=> [\"C.\", \"Binding\", \"of\", \"11\", \"a\", \"to\", \"the\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \".\"]\n",
      "Word token=> [\"The\", \"stereochemistry\", \"of\", \"the\", \"thiohemiketal\", \"is\", \"S\", \",\", \"as\", \"the\", \"group\", \"accepts\", \"a\", \"hydrogen\", \"bond\", \"from\", \"His\", \"41\", \",\", \"whereas\", \"the\", \"amide\", \"keto\", \"group\", \"accepts\", \"three\", \"H-bonds\", \"from\", \"the\", \"oxyanion\", \"hole\", \"(\", \"residues\", \"145\", \"-\", \"147\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"side-chain\", \"of\", \"Gln\", \"146\", \"has\", \"been\", \"omitted\", \"for\", \"clarity\", \".\"]\n",
      "Word token=> [\"D.\", \"The\", \"crystal\", \"structure\", \"of\", \"11\", \"f\", \"in\", \"complex\", \"with\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \"shows\", \"that\", \"this\", \"short\", \"(\", \"inactive\", \")\", \"compound\", \"lacking\", \"a\", \"P\", \"3\", \"residue\", \"has\", \"its\", \"P\", \"2\", \"-Boc\", \"group\", \"inserted\", \"into\", \"the\", \"S\", \"2\", \"pocket\", \"of\", \"the\", \"protease\", \".\"]\n",
      "Word token=> [\"The\", \"stereochemistry\", \"of\", \"the\", \"thiohemiketal\", \"is\", \"S.\", \"The\", \"sidechains\", \"of\", \"Ala\", \"143\", \"and\", \"Gln\", \"188\", \"have\", \"been\", \"omitted\", \"for\", \"clarity\", \".\"]\n",
      "Word token=> [\"E.\", \"In\", \"contrast\", \"to\", \"P\", \"2\", \"=\", \"benzyl\", \"in\", \"11\", \"a\", \",\", \"the\", \"isobutyl\", \"group\", \"of\", \"11\", \"n\", \"is\", \"small\", \"and\", \"flexible\", \"enough\", \"to\", \"enter\", \"into\", \"the\", \"narrow\", \"S\", \"2\", \"pocket\", \"of\", \"the\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \".\"]\n",
      "Word token=> [\"The\", \"thiohemiketal\", \"is\", \"in\", \"the\", \"R\", \"configuration\", \".\"]\n",
      "Word token=> [\"The\", \"side-chains\", \"of\", \"Ala\", \"143\", \"and\", \"Gln\", \"188\", \"have\", \"been\", \"omitted\", \"for\", \"clarity\", \".\"]\n",
      "Word token=> [\"F.\", \"In\", \"spite\", \"of\", \"its\", \"small\", \"size\", \",\", \"the\", \"cyclopropylmethyl\", \"side-chain\", \"in\", \"the\", \"P\", \"2\", \"position\", \"of\", \"11\", \"s\", \"can\", \"tightly\", \"bind\", \"to\", \"the\", \"S\", \"2\", \"subsite\", \"of\", \"the\", \"SARS-CoV\", \"M\", \"pro\", \",\", \"as\", \"this\", \"pocket\", \"exhibits\", \"pronounced\", \"plasticity\", \"due\", \"to\", \"the\", \"conformational\", \"flexibility\", \"of\", \"Gln\", \"189\", \"(\", \"see\", \"also\", \"Fig.\", \"4\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"stereochemistry\", \"of\", \"the\", \"thiohemiketal\", \"is\", \"S.\", \"The\", \"side-chains\", \"of\", \"Ser\", \"144\", \"and\", \"Arg\", \"188\", \"have\", \"been\", \"omitted\", \"for\", \"clarity\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"As\", \"the\", \"P\", \"2\", \"-benzyl\", \"group\", \"of\", \"11\", \"a\", \"was\", \"apparently\", \"too\", \"large\", \"to\", \"fit\", \"into\", \"the\", \"S\", \"2\", \"pocket\", \"of\", \"the\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \",\", \"we\", \"replaced\", \"it\", \"by\", \"isobutyl\", \"in\", \"11\", \"n\", \".\"]\n",
      "Word token=> [\"This\", \"resulted\", \"in\", \"improved\", \"activities\", \"against\", \"SARS-CoV\", \"M\", \"pro\", \"(\", \"IC\", \"50\", \"=\", \"0.33\", \"µM\", \")\", \"and\", \"in\", \"a\", \"very\", \"good\", \"activity\", \"against\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \"(\", \"IC\", \"50\", \"=\", \"1.08\", \"µM\", \",\", \"compare\", \"with\", \"the\", \"inactive\", \"11\", \"a\", \")\", \".\"]\n",
      "Word token=> [\"For\", \"EV-A\", \"71\", \"3\", \"C\", \"pro\", \",\", \"however\", \",\", \"the\", \"activity\", \"decreased\", \"to\", \"IC\", \"50\", \"=\", \"13.8\", \"µM\", \",\", \"different\", \"from\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \",\", \"where\", \"IC\", \"50\", \"was\", \"3.8\", \"µM\", \".\"]\n",
      "Word token=> [\"Our\", \"interpretation\", \"of\", \"this\", \"result\", \"is\", \"that\", \"the\", \"smaller\", \"P\", \"2\", \"-isobutyl\", \"substituent\", \"of\", \"11\", \"n\", \"can\", \"still\", \"interact\", \"with\", \"the\", \"\\\"\", \"lid\", \"\\\"\", \"(\", \"in\", \"particular\", \",\", \"Met\", \"49\", \")\", \"of\", \"the\", \"SARS-CoV\", \"M\", \"pro\", \"S\", \"2\", \"site\", \",\", \"but\", \"is\", \"unable\", \"to\", \"reach\", \"the\", \"\\\"\", \"back-wall\", \"\\\"\", \"of\", \"the\", \"EV-A\", \"71\", \"3\", \"C\", \"pro\", \"pocket\", \"and\", \"thus\", \",\", \"in\", \"the\", \"absence\", \"of\", \"a\", \"\\\"\", \"lid\", \"\\\"\", \",\", \"cannot\", \"generate\", \"sufficient\", \"enthalpy\", \"of\", \"binding\", \".\"]\n",
      "Word token=> [\"We\", \"will\", \"see\", \"from\", \"examples\", \"to\", \"follow\", \"that\", \"this\", \"trend\", \"persists\", \"among\", \"all\", \"inhibitors\", \"with\", \"a\", \"smaller\", \"P\", \"2\", \"substituent\", \":\", \"Even\", \"though\", \"the\", \"SARS-CoV\", \"M\", \"pro\", \"S\", \"2\", \"pocket\", \"has\", \"a\", \"larger\", \"volume\", \"than\", \"that\", \"of\", \"the\", \"enterovirus\", \"3\", \"C\", \"pro\", \",\", \"the\", \"enzyme\", \"can\", \"be\", \"efficiently\", \"inhibited\", \"by\", \"compounds\", \"carrying\", \"a\", \"small\", \"P\", \"2\", \"residue\", \"that\", \"makes\", \"hydrophobic\", \"interactions\", \"with\", \"the\", \"lid\", \"(\", \"Met\", \"49\", \")\", \"and\", \"floor\", \"(\", \"Met\", \"165\", \")\", \"residues\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"EC\", \"50\", \"of\", \"11\", \"n\", \"was\", \">\", \"10\", \"µM\", \"against\", \"the\", \"EV-A\", \"71\", \"and\", \"CVB\", \"3\", \"replicons\", \",\", \"and\", \"even\", \"in\", \"the\", \"SARS-CoV\", \"replicon\", \",\", \"the\", \"activity\", \"of\", \"11\", \"n\", \"was\", \"relatively\", \"weak\", \"(\", \"EC\", \"50\", \"=\", \"7.0\", \"µM\", \";\", \"Table\", \"2\", \")\", \".\"]\n",
      "Word token=> [\"In\", \"agreement\", \"with\", \"the\", \"replicon\", \"data\", \",\", \"11\", \"n\", \"proved\", \"inactive\", \"against\", \"EV-A\", \"71\", \"in\", \"RD\", \"cells\", \"and\", \"showed\", \"limited\", \"activity\", \"against\", \"HRV\", \"2\", \"or\", \"HRV\", \"14\", \"in\", \"HeLa\", \"Rh\", \"cells\", \"(\", \"Table\", \"3\", \")\", \".\"]\n",
      "Word token=> [\"Only\", \"the\", \"comparatively\", \"good\", \"activity\", \"(\", \"EC\", \"50\", \"=\", \"4.4\", \"µM\", \")\", \"against\", \"EV-D\", \"68\", \"in\", \"HeLa\", \"Rh\", \"cells\", \"was\", \"unexpected\", \".\"]\n",
      "Word token=> [\"The\", \"activity\", \"of\", \"11\", \"n\", \"against\", \"HCoV\", \"229\", \"E\", \"in\", \"Huh\", \"7\", \"cells\", \"was\", \"good\", \"(\", \"EC\", \"50\", \"=\", \"0.6\", \"µM\", \")\", \",\", \"and\", \"against\", \"MERS-CoV\", \"in\", \"Huh\", \"7\", \"cells\", \",\", \"it\", \"was\", \"excellent\", \",\", \"with\", \"EC\", \"50\", \"=\", \"0.0048\", \"µM\", \",\", \"while\", \"in\", \"Vero\", \"cells\", \",\", \"the\", \"EC\", \"50\", \"against\", \"MERS-CoV\", \"was\", \"as\", \"high\", \"as\", \"9.2\", \"µM\", \".\"]\n",
      "Word token=> [\"Similarly\", \",\", \"the\", \"EC\", \"50\", \"against\", \"SARS-CoV\", \"in\", \"Vero\", \"cells\", \"was\", \"14.2\", \"µM\", \"(\", \"Table\", \"3\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"We\", \"managed\", \"to\", \"obtain\", \"crystals\", \"of\", \"11\", \"n\", \"in\", \"complex\", \"with\", \"the\", \"M\", \"pro\", \"of\", \"HCoV\", \"NL\", \"63\", \"and\", \"found\", \"the\", \"P\", \"2\", \"isobutyl\", \"group\", \"to\", \"be\", \"well\", \"embedded\", \"in\", \"the\", \"S\", \"2\", \"pocket\", \"(\", \"Fig.\", \"3\", \"e\", \")\", \".\"]\n",
      "Word token=> [\"This\", \"is\", \"not\", \"only\", \"a\", \"consequence\", \"of\", \"the\", \"smaller\", \"size\", \"of\", \"the\", \"isobutyl\", \"group\", \"compared\", \"to\", \"the\", \"benzyl\", \"group\", \",\", \"but\", \"also\", \"of\", \"its\", \"larger\", \"conformational\", \"flexibility\", \",\", \"which\", \"allows\", \"a\", \"better\", \"fit\", \"to\", \"the\", \"binding\", \"site\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"When\", \"we\", \"replaced\", \"the\", \"P\", \"2\", \"-isobutyl\", \"residue\", \"of\", \"11\", \"n\", \"by\", \"n-butyl\", \"in\", \"11\", \"o\", \",\", \"the\", \"activities\", \"were\", \"as\", \"follows\", \":\", \"IC\", \"50\", \"=\", \"8.5\", \"µM\", \"for\", \"SARS-CoV\", \"M\", \"pro\", \",\", \"totally\", \"inactive\", \"(\", \"IC\", \"50\", \">\", \"50\", \"µM\", \")\", \"against\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \",\", \"IC\", \"50\", \"=\", \"3.2\", \"µM\", \"for\", \"EV-A\", \"71\", \"3\", \"C\", \"pro\", \",\", \"and\", \"5.2\", \"µM\", \"for\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"The\", \"decreased\", \"activity\", \"in\", \"case\", \"of\", \"SARS-CoV\", \"M\", \"pro\", \"and\", \"the\", \"total\", \"inactivity\", \"against\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \"indicate\", \"that\", \"the\", \"n-butyl\", \"chain\", \"is\", \"too\", \"long\", \"for\", \"the\", \"S\", \"2\", \"pocket\", \"of\", \"these\", \"proteases\", \",\", \"whereas\", \"the\", \"slight\", \"improvement\", \"against\", \"EV-A\", \"71\", \"3\", \"C\", \"pro\", \"and\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \"is\", \"probably\", \"a\", \"consequence\", \"of\", \"the\", \"extra\", \"space\", \"that\", \"is\", \"available\", \"to\", \"long\", \"and\", \"flexible\", \"substituents\", \"because\", \"of\", \"the\", \"lack\", \"of\", \"a\", \"lid\", \"covering\", \"the\", \"enterovirus\", \"3\", \"C\", \"pro\", \"pocket\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"As\", \"the\", \"n-butyl\", \"substituent\", \"in\", \"P\", \"2\", \"of\", \"11\", \"o\", \"was\", \"obviously\", \"too\", \"long\", \",\", \"we\", \"next\", \"synthesized\", \"a\", \"derivative\", \"with\", \"the\", \"shorter\", \"propargyl\", \"(\", \"ethinylmethyl\", \")\", \"as\", \"the\", \"P\", \"2\", \"residue\", \"(\", \"compound\", \"11\", \"p\", \")\", \".\"]\n",
      "Word token=> [\"This\", \"led\", \"to\", \"very\", \"mediocre\", \"activities\", \"against\", \"all\", \"tested\", \"proteases\", \".\"]\n",
      "Word token=> [\"Using\", \"cyclopropyl\", \"as\", \"the\", \"P\", \"2\", \"residue\", \"(\", \"compound\", \"11\", \"q\", \")\", \",\", \"the\", \"IC\", \"50\", \"values\", \"were\", \"even\", \"higher\", \"against\", \"most\", \"of\", \"the\", \"proteases\", \"tested\", \".\"]\n",
      "Word token=> [\"Obviously\", \",\", \"the\", \"P\", \"2\", \"side-chain\", \"requires\", \"a\", \"methylene\", \"group\", \"in\", \"the\", \"b-position\", \"in\", \"order\", \"to\", \"provide\", \"the\", \"necessary\", \"flexibility\", \"for\", \"the\", \"substituent\", \"to\", \"be\", \"embedded\", \"in\", \"the\", \"S\", \"2\", \"pocket\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Having\", \"realized\", \"that\", \"in\", \"addition\", \"to\", \"size\", \",\", \"flexibility\", \"of\", \"the\", \"P\", \"2\", \"substituent\", \"may\", \"be\", \"an\", \"important\", \"factor\", \"influencing\", \"inhibitory\", \"activity\", \",\", \"we\", \"introduced\", \"flexibility\", \"into\", \"the\", \"phenyl\", \"ring\", \"of\", \"11\", \"a\", \"by\", \"reducing\", \"it\", \".\"]\n",
      "Word token=> [\"The\", \"cyclohexylmethyl\", \"derivative\", \"11\", \"r\", \"exhibited\", \"IC\", \"50\", \"=\", \"0.7\", \"µM\", \"against\", \"SARS-CoV\", \"M\", \"pro\", \",\", \"12.3\", \"µM\", \"against\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \",\", \"1.7\", \"µM\", \"against\", \"EV-A\", \"71\", \"3\", \"C\", \"pro\", \",\", \"and\", \"0.9\", \"µM\", \"against\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \".\"]\n",
      "Word token=> [\"Thus\", \",\", \"the\", \"replace-\", \"1\", \"2\", \"3\", \"4\", \"5\", \"6\", \"7\", \"8\", \"9\", \"10\", \"11\", \"12\", \"13\", \"14\", \"15\", \"16\", \"17\", \"18\", \"19\", \"20\", \"21\", \"22\", \"23\", \"24\", \"25\", \"26\", \"27\", \"28\", \"29\", \"30\", \"31\", \"32\", \"33\", \"34\", \"35\", \"36\", \"37\", \"38\", \"39\", \"40\", \"41\", \"42\", \"43\", \"44\", \"45\", \"46\", \"47\", \"48\", \"49\", \"50\", \"51\", \"52\", \"53\", \"54\", \"55\", \"56\", \"57\", \"58\", \"59\", \"60\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.02.10.936898\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"ment\", \"of\", \"the\", \"phenyl\", \"group\", \"by\", \"the\", \"cyclohexyl\", \"group\", \"led\", \"to\", \"a\", \"significant\", \"improvement\", \"of\", \"the\", \"inhibitory\", \"activity\", \"against\", \"the\", \"recombinant\", \"SARS-CoV\", \"M\", \"pro\", \"and\", \"to\", \"a\", \"dramatic\", \"improvement\", \"in\", \"case\", \"of\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \".\"]\n",
      "Word token=> [\"Even\", \"for\", \"the\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \",\", \"against\", \"which\", \"11\", \"a\", \"was\", \"completely\", \"inactive\", \",\", \"greatly\", \"improved\", \"albeit\", \"still\", \"weak\", \"activity\", \"was\", \"observed\", \"(\", \"Table\", \"1\", \")\", \".\"]\n",
      "Word token=> [\"In\", \"the\", \"viral\", \"replicons\", \",\", \"11\", \"r\", \"performed\", \"very\", \"well\", \",\", \"with\", \"EC\", \"50\", \"=\", \"0.8\", \"-\", \"0.9\", \"µM\", \"for\", \"the\", \"EV-A\", \"71\", \"replicon\", \",\", \"0.45\", \"µM\", \"for\", \"CVB\", \"3\", \",\", \"and\", \"1.4\", \"µM\", \"for\", \"SARS-CoV\", \"(\", \"Table\", \"2\", \")\", \".\"]\n",
      "Word token=> [\"In\", \"the\", \"virus-infected\", \"cell\", \"culture\", \"assays\", \"(\", \"Table\", \"3\", \")\", \",\", \"11\", \"r\", \"exhibited\", \"EC\", \"50\", \"=\", \"3.7\", \"µM\", \"against\", \"EV-A\", \"71\", \"in\", \"RD\", \"cells\", \"and\", \"EC\", \"50\", \"=\", \"0.48\", \"-\", \"0.7\", \"µM\", \"against\", \"EV-D\", \"68\", \",\", \"HRV\", \"2\", \",\", \"and\", \"HRV\", \"14\", \"in\", \"HeLa\", \"cells\", \".\"]\n",
      "Word token=> [\"Against\", \"HCoV\", \"229\", \"E\", \"in\", \"Huh\", \"7\", \"cells\", \",\", \"the\", \"EC\", \"50\", \"was\", \"surprisingly\", \"low\", \"(\", \"1.8\", \"µM\", \")\", \".\"]\n",
      "Word token=> [\"Interestingly\", \",\", \"the\", \"compound\", \"proved\", \"extremely\", \"potent\", \"against\", \"MERS-CoV\", \"in\", \"Huh\", \"7\", \"cells\", \",\", \"with\", \"EC\", \"50\", \"=\", \"0.0004\", \"µM\", \"(\", \"400\", \"picomolar\", \"!\", \")\", \".\"]\n",
      "Word token=> [\"Even\", \"in\", \"Vero\", \"cells\", \",\", \"EC\", \"50\", \"against\", \"MERS-CoV\", \"was\", \"5\", \"µM\", \",\", \"and\", \"the\", \"EC\", \"50\", \"against\", \"SARS-CoV\", \"in\", \"Vero\", \"E\", \"6\", \"cells\", \"was\", \"1.8\", \"-\", \"2.1\", \"µM\", \",\", \"i.e.\", \"the\", \"best\", \"activity\", \"we\", \"have\", \"seen\", \"for\", \"an\", \"M\", \"pro\", \"inhibitor\", \"against\", \"SARS-CoV\", \"in\", \"this\", \"type\", \"of\", \"cells\", \".\"]\n",
      "Word token=> [\"The\", \"therapeutic\", \"index\", \"(\", \"CC\", \"50\", \"/EC\", \"50\", \")\", \"of\", \"11\", \"r\", \"against\", \"EV-D\", \"68\", \",\", \"HRV\", \"2\", \",\", \"and\", \"HRV\", \"14\", \"was\", \">\", \"15\", \"in\", \"HeLa\", \"Rh\", \"cells\", \"as\", \"well\", \"as\", \"against\", \"CVB\", \"3\", \"in\", \"Huh-T\", \"7\", \"cells\", \",\", \"but\", \"only\", \"~\", \"5\", \"for\", \"EV-A\", \"71\", \"in\", \"RD\", \"cells\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"At\", \"this\", \"point\", \",\", \"we\", \"decided\", \"to\", \"systematically\", \"vary\", \"the\", \"size\", \"of\", \"the\", \"ring\", \"system\", \"in\", \"P\", \"2\", \".\"]\n",
      "Word token=> [\"The\", \"next\", \"substituent\", \"to\", \"be\", \"tried\", \"was\", \"cyclopropylmethyl\", \"(\", \"compound\", \"11\", \"s\", \",\", \"which\", \"showed\", \"good\", \"activities\", \"against\", \"SARS-CoV\", \"M\", \"pro\", \"(\", \"IC\", \"50\", \"=\", \"0.24\", \"µM\", \")\", \"and\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \"(\", \"1.4\", \"µM\", \")\", \",\", \"but\", \"poor\", \"values\", \"against\", \"EV-A\", \"71\", \"3\", \"C\", \"pro\", \"(\", \"IC\", \"50\", \"=\", \"18.5\", \"µM\", \")\", \"and\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \"(\", \"IC\", \"50\", \"=\", \"4.3\", \"µM\", \")\", \"(\", \"Table\", \"1\", \")\", \".\"]\n",
      "Word token=> [\"11\", \"s\", \"was\", \"shown\", \"to\", \"inhibit\", \"the\", \"SARS-CoV\", \"replicon\", \"with\", \"an\", \"EC\", \"50\", \"of\", \"about\", \"2\", \"µM\", \",\", \"whereas\", \"activity\", \"against\", \"the\", \"EV-A\", \"71\", \"and\", \"CVB\", \"3\", \"replicons\", \"was\", \"poor\", \"(\", \"EC\", \"50\", \"values\", \">\", \"20\", \"µM\", \")\", \"(\", \"Table\", \"2\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"replicon\", \"results\", \"were\", \"mirrored\", \"by\", \"the\", \"antiviral\", \"activity\", \"of\", \"11\", \"s\", \"in\", \"enterovirus-infected\", \"cells\", \"(\", \"Table\", \"3\", \")\", \",\", \"which\", \"was\", \"weak\", \"or\", \"very\", \"weak\", \".\"]\n",
      "Word token=> [\"By\", \"contrast\", \",\", \"the\", \"compound\", \"inhibited\", \"HCoV\", \"229\", \"E\", \"and\", \"MERS-CoV\", \"in\", \"Huh\", \"7\", \"cells\", \"with\", \"EC\", \"50\", \"of\", \"1.3\", \"and\", \"0.08\", \"µM\", \",\", \"respectively\", \".\"]\n",
      "Word token=> [\"The\", \"activity\", \"against\", \"the\", \"latter\", \"virus\", \"in\", \"Vero\", \"cells\", \"was\", \"poor\", \"(\", \"EC\", \"50\", \"~\", \"11\", \"µM\", \")\", \",\", \"and\", \"so\", \"was\", \"the\", \"anti-SARS-CoV\", \"activity\", \"in\", \"Vero\", \"E\", \"6\", \"cells\", \"(\", \"Table\", \"3\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"We\", \"next\", \"analyzed\", \"the\", \"crystal\", \"structure\", \"of\", \"the\", \"complex\", \"between\", \"SARS-CoV\", \"M\", \"pro\", \"and\", \"compound\", \"11\", \"s\", \"(\", \"Fig.\", \"3\", \"f\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"cyclopropylmethyl\", \"substituent\", \"was\", \"found\", \"to\", \"be\", \"incorporated\", \"deeply\", \"into\", \"the\", \"S\", \"2\", \"pocket\", \",\", \"making\", \"hydrophobic\", \"interactions\", \"with\", \"Met\", \"49\", \"(\", \"the\", \"lid\", \")\", \",\", \"Met\", \"165\", \"(\", \"the\", \"floor\", \")\", \",\", \"and\", \"the\", \"Cb\", \"of\", \"Asp\", \"187\", \"(\", \"the\", \"back-wall\", \")\", \".\"]\n",
      "Word token=> [\"In\", \"spite\", \"of\", \"the\", \"small\", \"size\", \"of\", \"the\", \"P\", \"2\", \"substituent\", \",\", \"this\", \"is\", \"possible\", \"because\", \"the\", \"S\", \"2\", \"pocket\", \"of\", \"SARS-CoV\", \"M\", \"pro\", \"is\", \"flexible\", \"enough\", \"to\", \"contract\", \"and\", \"enclose\", \"the\", \"P\", \"2\", \"moiety\", \"tightly\", \".\"]\n",
      "Word token=> [\"This\", \"plasticity\", \"is\", \"expressed\", \"in\", \"a\", \"conformational\", \"change\", \"of\", \"residue\", \"Gln\", \"189\", \",\", \"both\", \"in\", \"the\", \"main\", \"chain\", \"and\", \"in\", \"the\", \"side-chain\", \".\"]\n",
      "Word token=> [\"The\", \"main-chain\", \"conformational\", \"change\", \"is\", \"connected\", \"with\", \"a\", \"flip\", \"of\", \"the\", \"peptide\", \"between\", \"Gln\", \"189\", \"and\", \"Thr\", \"190\", \".\"]\n",
      "Word token=> [\"The\", \"c\", \"1\", \"torsion\", \"angle\", \"of\", \"the\", \"Gln\", \"189\", \"side-chain\", \"changes\", \"from\", \"roughly\", \"antiperiplanar\", \"(\", \"ap\", \")\", \"to\", \"(\", \"-\", \")\", \"synclinal\", \"(\", \"-sc\", \")\", \"(\", \"Fig.\", \"4\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"conformational\", \"variability\", \"of\", \"Gln\", \"189\", \"has\", \"been\", \"noted\", \"before\", \",\", \"both\", \"in\", \"Molecular\", \"Dynamics\", \"simulations\", \"26\", \"and\", \"in\", \"other\", \"crystal\", \"structures\", \".\"]\n",
      "Word token=> [\"37\", \"As\", \"a\", \"consequence\", \"of\", \"these\", \"changes\", \",\", \"the\", \"sidechain\", \"oxygen\", \"of\", \"Gln\", \"189\", \"can\", \"accept\", \"a\", \"2.54\", \"-Å\", \"hydrogen\", \"bond\", \"from\", \"the\", \"main-chain\", \"NH\", \"of\", \"the\", \"P\", \"2\", \"residue\", \"in\", \"the\", \"11\", \"s\", \"complex\", \"(\", \"see\", \"Fig.\", \"4\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"affinity\", \"of\", \"11\", \"s\", \"for\", \"the\", \"S\", \"2\", \"pocket\", \"of\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \"is\", \"good\", \"because\", \"of\", \"an\", \"almost\", \"ideal\", \"match\", \"of\", \"size\", \"and\", \"not\", \"requiring\", \"conformational\", \"changes\", \",\", \"which\", \"this\", \"enzyme\", \"would\", \"not\", \"be\", \"able\", \"to\", \"undergo\", \"because\", \"of\", \"the\", \"replacement\", \"of\", \"the\", \"flexible\", \"Gln\", \"189\", \"by\", \"the\", \"more\", \"rigid\", \"Pro\", \".\"]\n",
      "Word token=> [\"On\", \"the\", \"other\", \"hand\", \",\", \"docking\", \"of\", \"the\", \"same\", \"compound\", \"into\", \"the\", \"crystal\", \"structure\", \"of\", \"the\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \"revealed\", \"that\", \"the\", \"cyclopropylmethyl\", \"moiety\", \"was\", \"probably\", \"unable\", \"to\", \"generate\", \"sufficient\", \"Free\", \"Energy\", \"of\", \"binding\", \"because\", \"of\", \"the\", \"missing\", \"lid\", \"and\", \"the\", \"large\", \"size\", \"of\", \"the\", \"S\", \"2\", \"pocket\", \"in\", \"the\", \"enterovirus\", \"3\", \"C\", \"pro\", \",\", \"thereby\", \"explaining\", \"the\", \"poor\", \"inhibitory\", \"activity\", \"of\", \"11\", \"s\", \"against\", \"these\", \"targets\", \".\"]\n",
      "Word token=> [\"(\", \"Table\", \"1\", \")\", \".\"]\n",
      "Word token=> [\"Experiments\", \"with\", \"the\", \"viral\", \"replicons\", \"confirmed\", \"this\", \"trend\", \",\", \"although\", \"the\", \"EC\", \"50\", \"value\", \"for\", \"SARS-CoV\", \"(\", \"6.8\", \"µM\", \")\", \"was\", \"surprisingly\", \"high\", \"(\", \"Table\", \"2\", \")\", \".\"]\n",
      "Word token=> [\"In\", \"Huh\", \"7\", \"cells\", \"infected\", \"with\", \"MERS-CoV\", \",\", \"this\", \"compound\", \"exhibited\", \"EC\", \"50\", \"=\", \"0.1\", \"µM\", \"(\", \"but\", \"9.8\", \"µM\", \"in\", \"Vero\", \"cells\", \")\", \",\", \"whereas\", \"EC\", \"50\", \"was\", \"7.0\", \"µM\", \"against\", \"SARS-CoV\", \"in\", \"Vero\", \"E\", \"6\", \"cells\", \".\"]\n",
      "Word token=> [\"The\", \"compound\", \"was\", \"largely\", \"inactive\", \"against\", \"EV-A\", \"71\", \"in\", \"RD\", \"cells\", \"and\", \"inhibited\", \"the\", \"replication\", \"of\", \"the\", \"two\", \"HRV\", \"subtypes\", \"tested\", \"(\", \"in\", \"HeLa\", \"Rh\", \"cells\", \")\", \"with\", \"EC\", \"50\", \"values\", \"of\", \"~\", \"4\", \"µM\", \".\"]\n",
      "Word token=> [\"The\", \"CC\", \"50\", \"of\", \"11\", \"t\", \"in\", \"HeLa\", \"cells\", \"was\", \"65\", \"µM\", \",\", \"i.e.\", \"the\", \"therapeutic\", \"index\", \"was\", \"well\", \"above\", \"15\", \"(\", \"Table\", \"3\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Obviously\", \",\", \"this\", \"substituent\", \"was\", \"still\", \"a\", \"bit\", \"too\", \"small\", \"for\", \"the\", \"enterovirus\", \"proteases\", \",\", \"so\", \"as\", \"the\", \"next\", \"step\", \",\", \"we\", \"tested\", \"P\", \"2\", \"=\", \"cyclopentylmethyl\", \"(\", \"compound\", \"11\", \"u\", \")\", \".\"]\n",
      "Word token=> [\"This\", \"turned\", \"out\", \"to\", \"be\", \"the\", \"one\", \"compound\", \"with\", \"acceptable\", \"IC\", \"50\", \"values\", \"against\", \"all\", \"tested\", \"enzymes\", \":\", \"1.3\", \"µM\", \"against\", \"SARS-CoV\", \"M\", \"pro\", \",\", \"5.4\", \"µM\", \"against\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \",\", \"4.7\", \"µM\", \"against\", \"EV-A\", \"71\", \"3\", \"C\", \"pro\", \",\", \"and\", \"1.9\", \"µM\", \"against\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \"(\", \"Table\", \"1\", \")\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"The\", \"activity\", \"against\", \"the\", \"replicons\", \"was\", \"between\", \"3.6\", \"and\", \"4.9\", \"µM\", \"(\", \"Table\", \"2\", \")\", \".\"]\n",
      "Word token=> [\"In\", \"Huh\", \"7\", \"cells\", \"infected\", \"with\", \"HCoV\", \"229\", \"E\", \"or\", \"MERS-CoV\", \",\", \"11\", \"u\", \"showed\", \"EC\", \"50\", \"=\", \"2.5\", \"or\", \"0.03\", \"µM\", \"(\", \"11.1\", \"µM\", \"for\", \"MERS-CoV\", \"in\", \"Vero\", \"cells\", \")\", \",\", \"while\", \"EC\", \"50\", \"was\", \"4.9\", \"µM\", \"against\", \"SARS-CoV\", \"in\", \"Vero\", \"E\", \"6\", \"cells\", \"(\", \"Table\", \"3\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"11\", \"u\", \"appeared\", \"so\", \"far\", \"the\", \"best\", \"compromise\", \"compound\", \",\", \"yet\", \"for\", \"each\", \"of\", \"the\", \"individual\", \"viral\", \"enzymes\", \",\", \"the\", \"following\", \"compounds\", \"proved\", \"superior\", \":\", \"P\", \"2\", \"=\", \"cyclopropylmethyl\", \"(\", \"compound\", \"11\", \"s\", \")\", \"for\", \"SARS-CoV\", \"M\", \"pro\", \",\", \"P\", \"2\", \"=\", \"isobutyl\", \"(\", \"compound\", \"11\", \"n\", \")\", \"and\", \"P\", \"2\", \"=\", \"cyclopropylmethyl\", \"(\", \"11\", \"s\", \")\", \"for\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \",\", \"P\", \"2\", \"=\", \"benzyl\", \"(\", \"11\", \"a\", \")\", \"or\", \"cyclohexylmethyl\", \"(\", \"11\", \"r\", \")\", \"for\", \"EV-A\", \"71\", \"3\", \"C\", \"pro\", \",\", \"and\", \"11\", \"r\", \"for\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \".\"]\n",
      "Word token=> [\"In\", \"other\", \"words\", \",\", \"the\", \"nearly\", \"equipotent\", \"11\", \"u\", \"is\", \"indeed\", \"a\", \"compromise\", \".\"]\n",
      "Word token=> [\"Therefore\", \",\", \"in\", \"view\", \"of\", \"the\", \"surprisingly\", \"good\", \"antiviral\", \"activity\", \"of\", \"11\", \"r\", \"against\", \"HCoV\", \"229\", \"E\", \"in\", \"Huh\", \"7\", \"cells\", \",\", \"we\", \"relaxed\", \"the\", \"condition\", \"that\", \"the\", \"universal\", \"inhibitor\", \"should\", \"show\", \"good\", \"activity\", \"against\", \"the\", \"recombinant\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \",\", \"and\", \"selected\", \"11\", \"r\", \"(\", \"P\", \"2\", \"=\", \"cyclohexylmethyl\", \")\", \"as\", \"the\", \"lead\", \"compound\", \"for\", \"further\", \"development\", \".\"]\n",
      "Word token=> [\"This\", \"compound\", \"exhibited\", \"submicromolar\", \"IC\", \"50\", \"values\", \"against\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \"and\", \"SARS-CoV\", \"M\", \"pro\", \",\", \"and\", \"IC\", \"50\", \"=\", \"1.7\", \"µM\", \"against\", \"EV-A\", \"71\", \"3\", \"C\", \"pro\", \"(\", \"Table\", \"1\", \")\", \",\", \"as\", \"well\", \"as\", \"similarly\", \"low\", \"EC\", \"50\", \"values\", \"in\", \"the\", \"replicons\", \"of\", \"these\", \"viruses\", \"(\", \"Table\", \"2\", \")\", \".\"]\n",
      "Word token=> [\"In\", \"Huh\", \"7\", \"cells\", \"infected\", \"with\", \"MERS-CoV\", \",\", \"the\", \"performance\", \"of\", \"this\", \"compound\", \"was\", \"excellent\", \",\", \"with\", \"EC\", \"50\", \"=\", \"0.0004\", \"µM\", \",\", \"and\", \"even\", \"against\", \"HCoV\", \"229\", \"E\", \"in\", \"Huh\", \"7\", \"cells\", \"and\", \"SARS-CoV\", \"in\", \"Vero\", \"E\", \"6\", \"cells\", \",\", \"EC\", \"50\", \"values\", \"of\", \"1.8\", \"and\", \"2.1\", \"µM\", \",\", \"respectively\", \",\", \"were\", \"observed\", \"(\", \"Table\", \"3\", \")\", \".\"]\n",
      "Word token=> [\"Also\", \"in\", \"enterovirus-infected\", \"cell\", \"culture\", \",\", \"the\", \"compound\", \"performed\", \"well\", \",\", \"with\", \"EC\", \"50\", \"values\", \"of\", \"0.7\", \"µM\", \"or\", \"below\", \"against\", \"HRV\", \"2\", \",\", \"HRV\", \"14\", \",\", \"and\", \"EV-D\", \"68\", \"in\", \"HeLa\", \"(\", \"Rh\", \")\", \"cells\", \"and\", \"selectivity\", \"values\", \">\", \"15\", \".\"]\n",
      "Word token=> [\"The\", \"only\", \"concern\", \"is\", \"the\", \"activity\", \"of\", \"the\", \"compound\", \"against\", \"EV-A\", \"71\", \"in\", \"RD\", \"cells\", \",\", \"for\", \"which\", \"the\", \"EC\", \"50\", \"value\", \"was\", \"3.7\", \"µM\", \",\", \"resulting\", \"in\", \"too\", \"low\", \"a\", \"therapeutic\", \"index\", \".\"]\n",
      "Word token=> [\"On\", \"the\", \"other\", \"hand\", \",\", \"only\", \"weak\", \"toxicity\", \"was\", \"detected\", \"for\", \"11\", \"r\", \"in\", \"Vero\", \"or\", \"Huh-T\", \"7\", \"cells\", \".\"]\n",
      "Word token=> [\"Preliminary\", \"pharmacokinetics\", \"tests\", \"with\", \"the\", \"compound\", \"in\", \"mice\", \"did\", \"not\", \"indicate\", \"a\", \"toxicity\", \"problem\", \"(\", \"to\", \"be\", \"published\", \"elsewhere\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"We\", \"describe\", \"here\", \"the\", \"structure-based\", \"design\", \",\", \"the\", \"synthesis\", \",\", \"and\", \"the\", \"assessment\", \"of\", \"capped\", \"dipeptide\", \"a-ketoamides\", \"that\", \"target\", \"the\", \"main\", \"protease\", \"of\", \"alpha-or\", \"betacoronaviruses\", \"as\", \"well\", \"as\", \"the\", \"3\", \"C\", \"protease\", \"of\", \"enteroviruses\", \".\"]\n",
      "Word token=> [\"Through\", \"crystallographic\", \"analyses\", \"of\", \"a\", \"total\", \"of\", \"six\", \"inhibitor\", \"complexes\", \"of\", \"three\", \"different\", \"proteases\", \"in\", \"this\", \"study\", \",\", \"we\", \"found\", \"the\", \"a-ketoamide\", \"warhead\", \"(\", \"-CO-CO-NH-\", \")\", \"to\", \"be\", \"sterically\", \"more\", \"versatile\", \"than\", \"other\", \"warheads\", \"such\", \"as\", \"Michael\", \"acceptors\", \"(\", \"-CH=CH-CO-\", \")\", \"and\", \"aldehydes\", \"(\", \"-CH=O\", \")\", \",\", \"because\", \"it\", \"features\", \"two\", \"acceptors\", \"for\", \"hydrogen\", \"bonds\", \"from\", \"the\", \"protein\", \",\", \"namely\", \"the\", \"a-keto\", \"oxygen\", \"and\", \"the\", \"amide\", \"oxygen\", \",\", \"whereas\", \"the\", \"other\", \"warheads\", \"have\", \"only\", \"one\", \"such\", \"acceptor\", \".\"]\n",
      "Word token=> [\"In\", \"the\", \"various\", \"complexes\", \",\", \"the\", \"hydroxy\", \"group\", \"(\", \"or\", \"oxyanion\", \")\", \"of\", \"the\", \"thiohemiketal\", \"that\", \"is\", \"formed\", \"by\", \"the\", \"nucleophilic\", \"attack\", \"of\", \"the\", \"activesite\", \"cysteine\", \"residue\", \"onto\", \"the\", \"a-keto\", \"carbon\", \",\", \"can\", \"accept\", \"one\", \"or\", \"two\", \"hydrogen\", \"bonds\", \"from\", \"the\", \"main-chain\", \"amides\", \"of\", \"the\", \"oxyanion\", \"hole\", \".\"]\n",
      "Word token=> [\"In\", \"addition\", \",\", \"the\", \"amide\", \"oxygen\", \"of\", \"the\", \"inhibitor\", \"accepts\", \"a\", \"hydrogen\", \"bond\", \"from\", \"the\", \"catalytic\", \"His\", \"side-chain\", \".\"]\n",
      "Word token=> [\"Alternatively\", \",\", \"the\", \"thiohemiketal\", \"can\", \"interact\", \"with\", \"the\", \"catalytic\", \"His\", \"residue\", \"and\", \"the\", \"amide\", \"oxygen\", \"with\", \"the\", \"main-chain\", \"amides\", \"of\", \"the\", \"oxyanion\", \"hole\", \".\"]\n",
      "Word token=> [\"Depending\", \"on\", \"the\", \"exact\", \"interaction\", \",\", \"the\", \"stereochemistry\", \"at\", \"the\", \"thiohemiketal\", \"C\", \"atom\", \"would\", \"be\", \"different\", \".\"]\n",
      "Word token=> [\"We\", \"have\", \"previously\", \"observed\", \"a\", \"similar\", \"difference\", \"in\", \"case\", \"of\", \"aldehyde\", \"inhibitors\", \",\", \"where\", \"the\", \"single\", \"interaction\", \"point\", \"-the\", \"oxyanion\", \"of\", \"the\", \"thiohemiacetal\", \"-can\", \"accept\", \"a\", \"hydrogen\", \"bond\", \"either\", \"from\", \"the\", \"oxyanion\", \"hole\", \"or\", \"from\", \"the\", \"catalytic\", \"His\", \"side-chain\", \",\", \"37\", \"resulting\", \"in\", \"different\", \"stereochemistry\", \"of\", \"the\", \"thiohemiacetal\", \"carbon\", \".\"]\n",
      "Word token=> [\"Both\", \"aketoamides\", \"and\", \"aldehydes\", \"react\", \"reversibly\", \"with\", \"the\", \"catalytic\", \"nucleophile\", \"of\", \"proteases\", \",\", \"whereas\", \"Michael\", \"acceptors\", \"form\", \"irreversible\", \"adducts\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"In\", \"addition\", \"to\", \"better\", \"matching\", \"the\", \"H-bonding\", \"donor/acceptor\", \"properties\", \"of\", \"the\", \"catalytic\", \"center\", \"through\", \"offering\", \"two\", \"hydrogen-bond\", \"acceptors\", \"instead\", \"of\", \"one\", \",\", \"a-ketoamides\", \"have\", \"another\", \"big\", \"advantage\", \"over\", \"aldehydes\", \"and\", \"a\", \",\", \"b-unsaturated\", \"esters\", \"(\", \"Michael\", \"acceptors\", \")\", \"in\", \"that\", \"they\", \"allow\", \"easy\", \"extension\", \"of\", \"the\", \"inhibitors\", \"to\", \"probe\", \"the\", \"primed\", \"specificity\", \"subsites\", \"beyond\", \"S\", \"1\", \"'\", \",\", \"although\", \"this\", \"has\", \"so\", \"far\", \"rarely\", \"been\", \"explored\", \"(\", \"e.g.\", \",\", \"ref.\", \"38\", \"in\", \"case\", \"of\", \"calpain\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"most\", \"prominent\", \"a-ketoamide\", \"drugs\", \"are\", \"probably\", \"telaprivir\", \"and\", \"boceprivir\", \",\", \"peptidomimetic\", \"inhibitors\", \"of\", \"the\", \"HCV\", \"NS\", \"3\", \"/\", \"4\", \"A\", \"protease\", \",\", \"39\", \",\", \"40\", \"which\", \"have\", \"helped\", \"revolutionize\", \"the\", \"treatment\", \"of\", \"chronic\", \"HCV\", \"infections\", \".\"]\n",
      "Word token=> [\"For\", \"viral\", \"cysteine\", \"proteases\", \",\", \"a-ketoamides\", \"have\", \"only\", \"occasionally\", \"been\", \"described\", \"as\", \"inhibitors\", \"and\", \"few\", \"systematic\", \"studies\", \"have\", \"been\", \"carried\", \"out\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"A\", \"number\", \"of\", \"capped\", \"dipeptidyl\", \"a-ketoamides\", \"have\", \"been\", \"described\", \"as\", \"inhibitors\", \"of\", \"the\", \"norovirus\", \"3\", \"C-like\", \"protease\", \".\"]\n",
      "Word token=> [\"41\", \"These\", \"were\", \"optimized\", \"with\", \"respect\", \"to\", \"their\", \"P\", \"1\", \"'\", \"substituent\", \",\", \"whereas\", \"P\", \"2\", \"was\", \"isobutyl\", \"in\", \"most\", \"cases\", \"and\", \"occasionally\", \"benzyl\", \".\"]\n",
      "Word token=> [\"The\", \"former\", \"displayed\", \"IC\", \"50\", \"values\", \"one\", \"order\", \"of\", \"magnitude\", \"lower\", \"than\", \"the\", \"latter\", \",\", \"indicating\", \"that\", \"the\", \"S\", \"2\", \"pocket\", \"of\", \"the\", \"norovirus\", \"3\", \"CL\", \"protease\", \"is\", \"fairly\", \"small\", \".\"]\n",
      "Word token=> [\"Although\", \"we\", \"did\", \"not\", \"include\", \"the\", \"norovirus\", \"3\", \"CL\", \"pro\", \"in\", \"our\", \"study\", \",\", \"expanding\", \"the\", \"target\", \"range\", \"of\", \"our\", \"inhibitors\", \"to\", \"norovirus\", \"is\", \"probably\", \"a\", \"realistic\", \"undertaking\", \".\"]\n",
      "Word token=> [\"While\", \"our\", \"study\", \"was\", \"underway\", \",\", \"Zeng\", \"et\", \"al\", \".\"]\n",
      "Word token=> [\"42\", \"published\", \"a\", \"series\", \"of\", \"a-ketoamides\", \"as\", \"inhibitors\", \"of\", \"the\", \"EV-A\", \"71\", \"3\", \"C\", \"pro\", \".\"]\n",
      "Word token=> [\"These\", \"authors\", \"mainly\", \"studied\", \"the\", \"structure-activity\", \"relationships\", \"of\", \"the\", \"P\", \"1\", \"'\", \"residue\", \"and\", \"found\", \"small\", \"alkyl\", \"substituents\", \"to\", \"be\", \"superior\", \"to\", \"larger\", \"ones\", \".\"]\n",
      "Word token=> [\"Interestingly\", \",\", \"they\", \"also\", \"reported\", \"that\", \"a\", \"six-membered\", \"d-lactam\", \"in\", \"the\", \"P\", \"1\", \"position\", \"led\", \"to\", \"2\", \"-\", \"3\", \"times\", \"higher\", \"activities\", \",\", \"compared\", \"to\", \"the\", \"fivemembered\", \"g-lactam\", \".\"]\n",
      "Word token=> [\"At\", \"the\", \"same\", \"time\", \",\", \"Kim\", \"et\", \"al\", \".\"]\n",
      "Word token=> [\"43\", \"described\", \"a\", \"series\", \"of\", \"five\", \"a-ketoamides\", \"with\", \"P\", \"1\", \"'\", \"=\", \"cyclopropyl\", \"that\", \"showed\", \"submicromolar\", \"activity\", \"against\", \"EV-D\", \"68\", \"and\", \"two\", \"HRV\", \"strains\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Occasionally\", \",\", \"individual\", \"a-ketoamides\", \"have\", \"been\", \"reported\", \"in\", \"the\", \"literature\", \"as\", \"inhibitors\", \"of\", \"both\", \"the\", \"enterovirus\", \"3\", \"C\", \"protease\", \"and\", \"the\", \"coronavirus\", \"main\", \"protease\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"A\", \"single\", \"capped\", \"dipeptidyl\", \"a-ketoamide\", \",\", \"Cbz-Leu-GlnLactam-CO-CO-NH-iPr\", \",\", \"was\", \"described\", \"that\", \"inhibited\", \"the\", \"recombinant\", \"transmissible\", \"gastroenteritis\", \"virus\", \"(\", \"TGEV\", \")\", \"and\", \"SARS-CoV\", \"M\", \"pro\", \"s\", \"as\", \"well\", \"as\", \"human\", \"rhinovirus\", \"and\", \"poliovirus\", \"3\", \"C\", \"pro\", \"s\", \"in\", \"the\", \"one-digit\", \"micromolar\", \"range\", \".\"]\n",
      "Word token=> [\"44\", \"Coded\", \"GC-\", \"375\", \",\", \"this\", \"compound\", \"showed\", \"poor\", \"activity\", \"in\", \"cell\", \"culture\", \"against\", \"EV-A\", \"71\", \"though\", \"(\", \"EC\", \"50\", \"=\", \"15.2\", \"µM\", \")\", \",\", \"probably\", \"because\", \"P\", \"2\", \"was\", \"isobutyl\", \".\"]\n",
      "Word token=> [\"As\", \"we\", \"have\", \"shown\", \"here\", \",\", \"an\", \"isobutyl\", \"side-chain\", \"in\", \"the\", \"P\", \"2\", \"position\", \"of\", \"the\", \"inhibitors\", \"is\", \"too\", \"small\", \"to\", \"completely\", \"fill\", \"the\", \"S\", \"2\", \"pocket\", \"of\", \"the\", \"EV-A\", \"71\", \"3\", \"C\", \"pro\", \"and\", \"the\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Among\", \"a\", \"series\", \"of\", \"aldehydes\", \",\", \"Prior\", \"et\", \"al\", \".\"]\n",
      "Word token=> [\"45\", \"described\", \"the\", \"capped\", \"tripeptidyl\", \"a-ketoamide\", \"Cbz-\", \"1\", \"-naphthylalanine-Leu-GlnLactam-CO-CO-NH-iPr\", \",\", \"which\", \"showed\", \"IC\", \"50\", \"values\", \"in\", \"the\", \"3\", \"-digit\", \"nanomolar\", \"range\", \"against\", \"HRV\", \"3\", \"C\", \"pro\", \"and\", \"SARS-CoV\", \"M\", \"pro\", \",\", \"as\", \"well\", \"as\", \"EC\", \"50\", \"values\", \"of\", \"0.03\", \"µM\", \"against\", \"HRV\", \"18\", \"and\", \"0.5\", \"µM\", \"against\", \"HCoV\", \"229\", \"E\", \"in\", \"cell\", \"culture\", \".\"]\n",
      "Word token=> [\"No\", \"optimization\", \"of\", \"this\", \"compound\", \"was\", \"performed\", \"and\", \"no\", \"toxicity\", \"data\", \"have\", \"been\", \"reported\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"For\", \"compounds\", \"with\", \"warheads\", \"other\", \"than\", \"a-ketoamides\", \",\", \"in-vitro\", \"activity\", \"against\", \"both\", \"corona-and\", \"enteroviruses\", \"has\", \"also\", \"occasionally\", \"been\", \"reported\", \".\"]\n",
      "Word token=> [\"Lee\", \"et\", \"al\", \".\"]\n",
      "Word token=> [\"46\", \"described\", \"three\", \"peptidyl\", \"Michael\", \"acceptors\", \"that\", \"displayed\", \"inhibitory\", \"activity\", \"against\", \"the\", \"M\", \"pro\", \"s\", \"of\", \"SARS-CoV\", \"and\", \"HCoV\", \"229\", \"E\", \"as\", \"well\", \"as\", \"against\", \"the\", \"3\", \"C\", \"pro\", \"of\", \"CVB\", \"3\", \".\"]\n",
      "Word token=> [\"These\", \"inhibitors\", \"had\", \"an\", \"IC\", \"50\", \"10\", \"-\", \"20\", \"times\", \"higher\", \"for\", \"the\", \"CVB\", \"3\", \"enzyme\", \",\", \"compared\", \"to\", \"SARS-CoV\", \"M\", \"pro\", \".\"]\n",
      "Word token=> [\"P\", \"2\", \"was\", \"invariably\", \"isobutyl\", \"(\", \"leucine\", \")\", \"in\", \"these\", \"compounds\", \",\", \"suggesting\", \"that\", \"further\", \"improvement\", \"might\", \"be\", \"possible\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"In\", \"addition\", \"to\", \"Michael\", \"acceptors\", \",\", \"peptide\", \"aldehydes\", \"have\", \"also\", \"been\", \"used\", \"to\", \"explore\", \"the\", \"inhibition\", \"of\", \"coronavirus\", \"M\", \"pro\", \"s\", \"as\", \"well\", \"as\", \"enterovirus\", \"3\", \"C\", \"pro\", \"s\", \".\"]\n",
      "Word token=> [\"Kim\", \"et\", \"al\", \".\"]\n",
      "Word token=> [\"44\", \"reported\", \"a\", \"dipeptidyl\", \"aldehyde\", \"and\", \"its\", \"bisulfite\", \"adduct\", \",\", \"both\", \"of\", \"which\", \"exhibited\", \"good\", \"inhibitory\", \"activities\", \"against\", \"the\", \"isolated\", \"3\", \"C\", \"proteases\", \"of\", \"human\", \"rhinovirus\", \"and\", \"poliovirus\", \"as\", \"well\", \"as\", \"against\", \"the\", \"3\", \"C-like\", \"proteases\", \"of\", \"a\", \"number\", \"of\", \"coronaviruses\", \",\", \"but\", \"antiviral\", \"activities\", \"in\", \"cell\", \"culture\", \"against\", \"EV-A\", \"71\", \"were\", \"poor\", \"(\", \"EC\", \"50\", \">\", \"10\", \"µM\", \")\", \",\", \"again\", \"most\", \"probably\", \"due\", \"to\", \"P\", \"2\", \"being\", \"isobutyl\", \"(\", \"leucine\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"In\", \"our\", \"series\", \"of\", \"compounds\", \",\", \"we\", \"used\", \"P\", \"1\", \"=\", \"GlnLactam\", \"(\", \"g-lactam\", \")\", \"throughout\", \",\", \"because\", \"this\", \"substituent\", \"has\", \"proven\", \"to\", \"be\", \"an\", \"excellent\", \"surrogate\", \"for\", \"glutamine\", \".\"]\n",
      "Word token=> [\"29\", \",\", \"32\", \"While\", \"we\", \"made\", \"some\", \"efforts\", \"to\", \"optimize\", \"the\", \"P\", \"1\", \"'\", \"residue\", \"of\", \"the\", \"compounds\", \"as\", \"well\", \"as\", \"the\", \"N-cap\", \"(\", \"P\", \"3\", \")\", \",\", \"we\", \"mainly\", \"focussed\", \"on\", \"optimization\", \"of\", \"the\", \"P\", \"2\", \"substituent\", \".\"]\n",
      "Word token=> [\"In\", \"nearly\", \"all\", \"studies\", \"aiming\", \"at\", \"discovering\", \"peptidomimetic\", \"inhibitors\", \"of\", \"coronavirus\", \"M\", \"pro\", \"s\", \",\", \"P\", \"2\", \"is\", \"invariably\", \"isobutyl\", \"(\", \"leucine\", \")\", \",\", \"and\", \"this\", \"residue\", \"has\", \"also\", \"been\", \"used\", \"in\", \"the\", \"efforts\", \"to\", \"design\", \"compounds\", \"that\", \"would\", \"inhibit\", \"enterovirus\", \"3\", \"C\", \"pro\", \"s\", \"as\", \"well\", \"(\", \"see\", \"above\", \")\", \".\"]\n",
      "Word token=> [\"From\", \"crystal\", \"structures\", \"of\", \"our\", \"early\", \"lead\", \"compound\", \",\", \"11\", \"a\", \"(\", \"cinnamoyl-Phe-GlnLactam-CO-CO-NH-Bz\", \")\", \",\", \"in\", \"complex\", \"with\", \"the\", \"M\", \"pro\", \"s\", \"of\", \"HCoV\", \"NL\", \"63\", \"(\", \"as\", \"representative\", \"of\", \"the\", \"alphacoronavirus\", \"proteases\", \")\", \"and\", \"SARS-CoV\", \"(\", \"beta-CoV\", \")\", \"as\", \"well\", \"as\", \"the\", \"3\", \"C\", \"pro\", \"of\", \"Coxsackievirus\", \"B\", \"3\", \"(\", \"enterovirus\", \"proteases\", \")\", \",\", \"we\", \"found\", \"that\", \"the\", \"S\", \"2\", \"pocket\", \"has\", \"fundamentally\", \"different\", \"shapes\", \"in\", \"these\", \"enzymes\", \".\"]\n",
      "Word token=> [\"In\", \"the\", \"SARS-CoV\", \"M\", \"pro\", \",\", \"the\", \"S\", \"2\", \"subsite\", \"is\", \"a\", \"deep\", \"hydrophobic\", \"pocket\", \"that\", \"is\", \"truly\", \"three-dimensional\", \"in\", \"shape\", \":\", \"the\", \"\\\"\", \"walls\", \"\\\"\", \"of\", \"the\", \"groove\", \"are\", \"formed\", \"by\", \"the\", \"polypeptide\", \"main\", \"chain\", \"around\", \"residues\", \"186\", \"-\", \"188\", \"as\", \"well\", \"as\", \"by\", \"the\", \"side-chains\", \"of\", \"His\", \"41\", \"(\", \"of\", \"the\", \"catalytic\", \"dyad\", \")\", \"and\", \"Gln\", \"189\", \",\", \"whereas\", \"the\", \"\\\"\", \"floor\", \"\\\"\", \"is\", \"formed\", \"by\", \"Met\", \"165\", \"and\", \"the\", \"\\\"\", \"lid\", \"\\\"\", \"by\", \"residues\", \"45\", \"-\", \"51\", \",\", \"in\", \"particular\", \"Met\", \"49\", \".\"]\n",
      "Word token=> [\"The\", \"two\", \"methionines\", \"provide\", \"important\", \"interaction\", \"points\", \"for\", \"the\", \"P\", \"2\", \"substituents\", \"of\", \"inhibitors\", \";\", \"while\", \"these\", \"interactions\", \"are\", \"mostly\", \"hydrophobic\", \"in\", \"character\", \",\", \"we\", \"have\", \"previously\", \"described\", \"the\", \"surprising\", \"observation\", \"of\", \"the\", \"carboxylate\", \"of\", \"an\", \"aspartic\", \"residue\", \"in\", \"P\", \"2\", \"that\", \"made\", \"polar\", \"interactions\", \"with\", \"the\", \"sulfur\", \"atoms\", \"of\", \"these\", \"methionines\", \".\"]\n",
      "Word token=> [\"37\", \"Because\", \"the\", \"pocket\", \"offers\", \"so\", \"many\", \"opportunities\", \"for\", \"interaction\", \"and\", \"features\", \"a\", \"pronounced\", \"plasticity\", \",\", \"P\", \"2\", \"substituents\", \"such\", \"as\", \"isobutyl\", \"(\", \"from\", \"Leu\", \")\", \",\", \"which\", \"are\", \"too\", \"small\", \"to\", \"fill\", \"the\", \"pocket\", \"entirely\", \",\", \"can\", \"still\", \"generate\", \"sufficient\", \"binding\", \"enthalpy\", \".\"]\n",
      "Word token=> [\"Accordingly\", \",\", \"the\", \"S\", \"2\", \"pocket\", \"of\", \"SARS-CoV\", \"M\", \"pro\", \"is\", \"the\", \"most\", \"tolerant\", \"among\", \"the\", \"three\", \"enzymes\", \"investigated\", \"here\", \",\", \"in\", \"terms\", \"of\", \"versatility\", \"of\", \"the\", \"P\", \"2\", \"-substituents\", \"accepted\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"In\", \"the\", \"S\", \"2\", \"pocket\", \"of\", \"the\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \",\", \"Gln\", \"189\", \"is\", \"replaced\", \"by\", \"proline\", \"and\", \"this\", \"change\", \"is\", \"accompanied\", \"by\", \"a\", \"significant\", \"loss\", \"of\", \"flexibility\", \";\", \"whereas\", \"the\", \"side-chain\", \"of\", \"Gln\", \"189\", \"of\", \"SARS-CoV\", \"M\", \"pro\", \"is\", \"found\", \"to\", \"accommodate\", \"its\", \"conformation\", \"according\", \"to\", \"the\", \"steric\", \"requirements\", \"of\", \"the\", \"P\", \"2\", \"substituent\", \",\", \"the\", \"proline\", \"is\", \"less\", \"flexible\", \",\", \"leading\", \"to\", \"a\", \"much\", \"smaller\", \"space\", \"at\", \"the\", \"entrance\", \"to\", \"the\", \"pocket\", \".\"]\n",
      "Word token=> [\"As\", \"a\", \"consequence\", \",\", \"a\", \"P\", \"2\", \"benzyl\", \"substituent\", \"is\", \"hindered\", \"from\", \"penetrating\", \"deeply\", \"into\", \"the\", \"pocket\", \",\", \"whereas\", \"the\", \"smaller\", \"and\", \"more\", \"flexible\", \"isobutyl\", \"group\", \"of\", \"P\", \"2\", \"-Leu\", \"is\", \"not\", \".\"]\n",
      "Word token=> String[]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"Finally\", \",\", \"in\", \"the\", \"3\", \"C\", \"pro\", \"s\", \"of\", \"EV-A\", \"71\", \"and\", \"CVB\", \"3\", \",\", \"the\", \"S\", \"2\", \"pocket\", \"lacks\", \"a\", \"lid\", \",\", \"i.e.\", \"it\", \"is\", \"open\", \"to\", \"one\", \"side\", \".\"]\n",
      "Word token=> [\"As\", \"a\", \"consequence\", \",\", \"it\", \"offers\", \"less\", \"interaction\", \"points\", \"for\", \"P\", \"2\", \"substituents\", \"of\", \"inhibitors\", \",\", \"so\", \"that\", \"such\", \"substituents\", \"must\", \"reach\", \"the\", \"\\\"\", \"back-wall\", \"\\\"\", \"of\", \"the\", \"pocket\", \"(\", \"formed\", \"by\", \"Arg\", \"39\", \",\", \"Asn\", \"69\", \",\", \"Glu\", \"71\", \")\", \"in\", \"order\", \"to\", \"create\", \"sufficient\", \"binding\", \"energy\", \".\"]\n",
      "Word token=> [\"Hence\", \",\", \"large\", \"aromatic\", \"substituents\", \"such\", \"as\", \"benzyl\", \"are\", \"favored\", \"by\", \"the\", \"enterovirus\", \"3\", \"C\", \"pro\", \"s\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"When\", \"we\", \"introduced\", \"a\", \"fluoro\", \"substituent\", \"in\", \"the\", \"para\", \"position\", \"of\", \"the\", \"P\", \"2\", \"-benzyl\", \"group\", \"of\", \"our\", \"lead\", \"compound\", \",\", \"11\", \"a\", \",\", \"we\", \"observed\", \"good\", \"activity\", \"against\", \"the\", \"enterovirus\", \"3\", \"C\", \"pro\", \"s\", \"but\", \"complete\", \"inactivity\", \"against\", \"the\", \"coronavirus\", \"M\", \"pro\", \"s\", \"(\", \"see\", \"Table\", \"1\", \",\", \"compound\", \"11\", \"m\", \")\", \".\"]\n",
      "Word token=> [\"This\", \"is\", \"easily\", \"explained\", \"on\", \"the\", \"basis\", \"of\", \"the\", \"crystal\", \"structures\", \":\", \"In\", \"the\", \"enterovirus\", \"3\", \"C\", \"pro\", \"s\", \",\", \"the\", \"fluorine\", \"can\", \"accept\", \"a\", \"hydrogen\", \"bond\", \"from\", \"Arg\", \"39\", \"(\", \"ref.\", \"30\", \")\", \",\", \"whereas\", \"in\", \"the\", \"coronavirus\", \"M\", \"pro\", \"s\", \",\", \"there\", \"would\", \"be\", \"electrostatic\", \"repulsion\", \"from\", \"the\", \"main-chain\", \"carbonyls\", \"of\", \"residues\", \"186\", \"and\", \"188\", \".\"]\n",
      "Word token=> [\"In\", \"agreement\", \"with\", \"this\", \",\", \"rupintrivir\", \"(\", \"which\", \"has\", \"P\", \"2\", \"=\", \"p-fluorobenzyl\", \")\", \"is\", \"a\", \"good\", \"inhibitor\", \"of\", \"the\", \"enteroviral\", \"3\", \"C\", \"pro\", \"s\", \",\", \"46\", \"but\", \"not\", \"of\", \"the\", \"coronaviral\", \"main\", \"proteases\", \",\", \"as\", \"we\", \"predicted\", \"earlier\", \".\"]\n",
      "Word token=> [\"28\", \"In\", \"this\", \"structure-based\", \"inhibitor\", \"optimization\", \"study\", \",\", \"we\", \"achieved\", \"major\", \"improvements\", \"over\", \"our\", \"original\", \"lead\", \"compound\", \",\", \"11\", \"a\", \",\", \"by\", \"systematically\", \"varying\", \"the\", \"size\", \"and\", \"the\", \"flexibility\", \"of\", \"the\", \"P\", \"2\", \"-substituent\", \".\"]\n",
      "Word token=> [\"The\", \"compound\", \"presenting\", \"so\", \"far\", \"the\", \"best\", \"compromise\", \"between\", \"the\", \"different\", \"requirements\", \"of\", \"the\", \"S\", \"2\", \"pockets\", \"(\", \"SARS-CoV\", \"M\", \"pro\", \":\", \"large\", \"and\", \"covered\", \",\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \":\", \"small\", \"and\", \"covered\", \",\", \"and\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \":\", \"large\", \"and\", \"open\", \")\", \"is\", \"11\", \"u\", \"(\", \"P\", \"2\", \"=\", \"cyclopentylmethyl\", \")\", \",\", \"which\", \"has\", \"satisfactory\", \"broad-spectrum\", \"activity\", \"against\", \"all\", \"proteases\", \"tested\", \".\"]\n",
      "Word token=> [\"However\", \",\", \"with\", \"regard\", \"to\", \"its\", \"antiviral\", \"activities\", \"in\", \"cell\", \"cultures\", \",\", \"it\", \"is\", \"inferior\", \"to\", \"11\", \"r\", \"(\", \"P\", \"2\", \"=\", \"cyclohexylmethyl\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"latter\", \"compound\", \"exhibits\", \"very\", \"good\", \"inhibitory\", \"activity\", \"against\", \"the\", \"SARS-CoV\", \"M\", \"pro\", \"as\", \"well\", \"as\", \"the\", \"enterovirus\", \"3\", \"C\", \"pro\", \",\", \"and\", \"its\", \"performance\", \"in\", \"the\", \"SARS-CoV\", \"and\", \"enterovirus\", \"replicons\", \"is\", \"convincing\", \".\"]\n",
      "Word token=> [\"Being\", \"in\", \"the\", \"low\", \"micromolar\", \"range\", \"(\", \"EV-A\", \"71\", \",\", \"CVB\", \"3\", \")\", \",\", \"the\", \"data\", \"for\", \"the\", \"antiviral\", \"activity\", \"in\", \"cell\", \"culture\", \"for\", \"11\", \"r\", \"correlate\", \"well\", \"with\", \"the\", \"inhibitory\", \"power\", \"of\", \"the\", \"compound\", \"against\", \"the\", \"recombinant\", \"proteases\", \"as\", \"well\", \"as\", \"in\", \"the\", \"replicon-based\", \"assays\", \".\"]\n",
      "Word token=> [\"This\", \"is\", \"not\", \"true\", \",\", \"though\", \",\", \"for\", \"the\", \"surprisingly\", \"good\", \"in-cellulo\", \"activity\", \"of\", \"11\", \"r\", \"against\", \"HCoV\", \"229\", \"E\", \"in\", \"Huh\", \"7\", \"cells\", \".\"]\n",
      "Word token=> [\"Also\", \",\", \"the\", \"correlation\", \"does\", \"not\", \"seem\", \"to\", \"hold\", \"for\", \"LLC-MK\", \"2\", \"and\", \"CaCo\", \"2\", \"cells\", \".\"]\n",
      "Word token=> [\"We\", \"tested\", \"the\", \"antiviral\", \"activity\", \"of\", \"many\", \"of\", \"our\", \"compounds\", \"against\", \"HCoV\", \"NL\", \"63\", \"in\", \"these\", \"two\", \"cell\", \"types\", \"and\", \"found\", \"that\", \"all\", \"of\", \"them\", \"had\", \"low-or\", \"submicromolar\", \"EC\", \"50\", \"values\", \"against\", \"this\", \"virus\", \"in\", \"LLC-MK\", \"2\", \"cells\", \"but\", \"were\", \"largely\", \"inactive\", \"in\", \"CaCo\", \"2\", \"cells\", \"(\", \"not\", \"shown\", \")\", \".\"]\n",
      "Word token=> [\"Furthermore\", \",\", \"11\", \"r\", \"and\", \"all\", \"other\", \"compounds\", \"that\", \"we\", \"synthesized\", \"are\", \"inactive\", \"(\", \"EC\", \"50\", \">\", \"87\", \"µM\", \")\", \"against\", \"CVB\", \"3\", \"in\", \"Vero\", \"cells\", \"(\", \"not\", \"shown\", \")\", \",\", \"but\", \"exhibit\", \"good\", \"to\", \"excellent\", \"activities\", \"against\", \"the\", \"same\", \"virus\", \"in\", \"Huh-T\", \"7\", \"cells\", \".\"]\n",
      "Word token=> [\"We\", \"have\", \"previously\", \"observed\", \"similar\", \"poor\", \"antiviral\", \"activities\", \"in\", \"Vero\", \"cells\", \"not\", \"only\", \"for\", \"aketoamides\", \",\", \"but\", \"also\", \"for\", \"Michael\", \"acceptors\", \"(\", \"Zhu\", \"et\", \"al.\", \",\", \"unpublished\", \"work\", \")\", \".\"]\n",
      "Word token=> [\"A\", \"similar\", \"cell-type\", \"dependence\", \"is\", \"seen\", \"for\", \"the\", \"antiviral\", \"activity\", \"of\", \"11\", \"r\", \"against\", \"MERS-CoV\", \"and\", \"SARS-CoV\", \".\"]\n",
      "Word token=> [\"Whereas\", \"the\", \"inhibitor\", \"exhibits\", \"excellent\", \"activity\", \"against\", \"MERS-CoV\", \"when\", \"Huh\", \"7\", \"cells\", \"are\", \"the\", \"host\", \"cells\", \"(\", \"400\", \"pM\", \")\", \",\", \"the\", \"inhibitory\", \"activity\", \"is\", \"weaker\", \"by\", \"a\", \"factor\", \"of\", \"up\", \"to\", \"12,500\", \"when\", \"Vero\", \"cells\", \"are\", \"used\", \"(\", \"EC\", \"50\", \"=\", \"5\", \"µM\", \")\", \".\"]\n",
      "Word token=> [\"On\", \"the\", \"other\", \"hand\", \",\", \"11\", \"r\", \"exhibits\", \"excellent\", \"anti-MERS-CoV\", \"activity\", \"in\", \"human\", \"Calu\", \"3\", \"lung\", \"cells\", \",\", \"i.e.\", \"in\", \"the\", \"primary\", \"target\", \"cells\", \"where\", \"the\", \"compound\", \"will\", \"have\", \"to\", \"act\", \"in\", \"a\", \"therapeutic\", \"setting\", \"(\", \"A.\", \"Kupke\", \",\", \"personal\", \"communication\", \")\", \".\"]\n",
      "Word token=> [\"As\", \"we\", \"tested\", \"antiviral\", \"activity\", \"against\", \"SARS-CoV\", \"exclusively\", \"in\", \"Vero\", \"cells\", \",\", \"the\", \"EC\", \"50\", \"values\", \"determined\", \"for\", \"our\", \"compounds\", \"against\", \"this\", \"virus\", \"are\", \"in\", \"the\", \"one-digit\", \"micromolar\", \"range\", \"or\", \"higher\", \";\", \"the\", \"best\", \"is\", \"again\", \"compound\", \"11\", \"r\", \"with\", \"EC\", \"50\", \"=\", \"2.1\", \"µM\", \".\"]\n",
      "Word token=> [\"Interestingly\", \",\", \"the\", \"relatively\", \"weaker\", \"activity\", \"(\", \"or\", \"even\", \"inactivity\", \")\", \"of\", \"our\", \"inhibitors\", \"against\", \"RNA\", \"viruses\", \"in\", \"Vero\", \"cells\", \"was\", \"observed\", \"independently\", \"in\", \"the\", \"virology\", \"laboratories\", \"in\", \"Leuven\", \"and\", \"in\", \"Leiden\", \".\"]\n",
      "Word token=> [\"It\", \"is\", \"thus\", \"unlikely\", \"that\", \"the\", \"lack\", \"of\", \"activity\", \"in\", \"Vero\", \"cells\", \"is\", \"related\", \"to\", \"problems\", \"with\", \"the\", \"experimental\", \"set-up\", \".\"]\n",
      "Word token=> [\"In\", \"preliminary\", \"experiments\", \",\", \"we\", \"replaced\", \"the\", \"P\", \"3\", \"cinnamoyl\", \"group\", \"of\", \"11\", \"r\", \"by\", \"the\", \"fluorophor\", \"coumaryl\", \"and\", \"found\", \"by\", \"fluorescence\", \"microscopy\", \"that\", \"much\", \"more\", \"inhibitor\", \"appeared\", \"to\", \"accumulate\", \"in\", \"Huh\", \"7\", \"cells\", \"compared\", \"to\", \"Vero\", \"cells\", \"(\", \"D.L.\", \",\", \"R.H.\", \"&\", \"Irina\", \"Majoul\", \",\", \"unpublished\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Regardless\", \"of\", \"which\", \"cell\", \"system\", \"is\", \"the\", \"most\", \"suitable\", \"one\", \"for\", \"the\", \"testing\", \"of\", \"peptidomimetic\", \"antiviral\", \"compounds\", \",\", \"we\", \"next\", \"plan\", \"to\", \"test\", \"11\", \"r\", \"in\", \"small-animal\", \"models\", \"for\", \"MERS\", \"and\", \"for\", \"Coxsackievirusinduced\", \"pancreatitis\", \".\"]\n",
      "Word token=> [\"In\", \"parallel\", \",\", \"we\", \"aim\", \"to\", \"refine\", \"the\", \"experiments\", \"to\", \"quantify\", \"the\", \"accumulation\", \"of\", \"peptidomimetic\", \"protease\", \"inhibitors\", \"in\", \"different\", \"host-cell\", \"types\", \",\", \"in\", \"the\", \"hope\", \"to\", \"find\", \"an\", \"explanation\", \"for\", \"the\", \"observed\", \"cell-type\", \"dependencies\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"This\", \"work\", \"demonstrates\", \"the\", \"power\", \"of\", \"structure-based\", \"approaches\", \"in\", \"the\", \"design\", \"of\", \"broad-spectrum\", \"antiviral\", \"compounds\", \"with\", \"roughly\", \"equipotent\", \"activity\", \"against\", \"coronaviruses\", \"and\", \"enteroviruses\", \".\"]\n",
      "Word token=> [\"We\", \"observed\", \"a\", \"good\", \"correlation\", \"between\", \"the\", \"inhibitory\", \"activity\", \"of\", \"the\", \"designed\", \"compounds\", \"against\", \"the\", \"isolated\", \"proteases\", \",\", \"in\", \"viral\", \"replicons\", \",\", \"and\", \"in\", \"virus-infected\", \"Huh\", \"7\", \"cells\", \".\"]\n",
      "Word token=> [\"One\", \"of\", \"the\", \"compounds\", \"(\", \"11\", \"r\", \")\", \"exhibits\", \"excellent\", \"anti-MERS-CoV\", \"activity\", \"in\", \"virus-infected\", \"Huh\", \"7\", \"cells\", \".\"]\n",
      "Word token=> [\"Because\", \"of\", \"the\", \"high\", \"similarity\", \"between\", \"the\", \"main\", \"proteases\", \"of\", \"SARS-CoV\", \"and\", \"the\", \"novel\", \"BetaCoV/Wuhan/\", \"2019\", \",\", \"we\", \"expect\", \"11\", \"r\", \"to\", \"exhibit\", \"good\", \"antiviral\", \"activity\", \"against\", \"the\", \"new\", \"coronavirus\", \"as\", \"well\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Crystallization\", \".\"]\n",
      "Word token=> [\"The\", \"recombinant\", \"production\", \"and\", \"purification\", \"of\", \"SARS-CoV\", \"M\", \"pro\", \"with\", \"authentic\", \"N\", \"and\", \"C\", \"termini\", \"was\", \"described\", \"in\", \"detail\", \"previously\", \".\"]\n",
      "Word token=> [\"48\", \",\", \"49\", \"Using\", \"an\", \"Amicon\", \"YM\", \"10\", \"membrane\", \"(\", \"EMD\", \"Millipore\", \")\", \",\", \"the\", \"purified\", \"SARS-CoV\", \"M\", \"pro\", \"was\", \"concentrated\", \"to\", \"21\", \"mg\", \"·\", \"mL\", \"-\", \"1\", \"in\", \"buffer\", \"A\", \"(\", \"20\", \"mM\", \"Tris-HCl\", \",\", \"150\", \"mM\", \"NaCl\", \",\", \"1\", \"mM\", \"DTT\", \",\", \"1\", \"mM\", \"EDTA\", \",\", \"pH\", \"7.5\", \")\", \".\"]\n",
      "Word token=> [\"Crystallization\", \"was\", \"performed\", \"by\", \"equilibrating\", \"1\", \"µL\", \"protein\", \"(\", \"mixed\", \"with\", \"1\", \"µL\", \"precipitant\", \"solution\", \")\", \"against\", \"500\", \"µl\", \"reservoir\", \"containing\", \"6\", \"-\", \"8\", \"%\", \"polyethylene\", \"glycol\", \"(\", \"PEG\", \")\", \"6,000\", \",\", \"0.1\", \"M\", \"MES\", \"(\", \"pH\", \"6.0\", \")\", \",\", \"at\", \"20\", \"℃\", \"using\", \"the\", \"vapor\", \"diffusion\", \"sitting-drop\", \"method\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"Compounds\", \"11\", \"a\", \"and\", \"11\", \"s\", \"were\", \"dissolved\", \"in\", \"100\", \"%\", \"DMSO\", \"at\", \"50\", \"mM\", \"and\", \"200\", \"mM\", \"stock\", \"concentrations\", \",\", \"respectively\", \".\"]\n",
      "Word token=> [\"A\", \"crystal\", \"of\", \"the\", \"free\", \"enzyme\", \"was\", \"soaked\", \"in\", \"cryo-protectant\", \"buffer\", \"containing\", \"20\", \"%\", \"MPD\", \",\", \"6\", \"%\", \"PEG\", \"6,000\", \",\", \"0.1\", \"M\", \"MES\", \",\", \"7.5\", \"mM\", \"11\", \"a\", \",\", \"pH\", \"6.0\", \",\", \"for\", \"2\", \"h\", \"at\", \"20\", \"℃\", \".\"]\n",
      "Word token=> [\"Another\", \"set\", \"of\", \"free\", \"enzyme\", \"crystals\", \"was\", \"soaked\", \"in\", \"another\", \"cryoprotectant\", \"buffer\", \"with\", \"6\", \"%\", \"PEG\", \"6,000\", \",\", \"5\", \"%\", \"MPD\", \",\", \"0.1\", \"M\", \"MES\", \",\", \"15\", \"%\", \"glycerol\", \",\", \"10\", \"mM\", \"11\", \"s\", \",\", \"pH\", \"6.0\", \",\", \"for\", \"2\", \"h\", \".\"]\n",
      "Word token=> [\"Subsequently\", \",\", \"crystals\", \"were\", \"fished\", \"and\", \"flash-cooled\", \"in\", \"liquid\", \"nitrogen\", \"prior\", \"to\", \"data\", \"collection\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Crystals\", \"of\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \"with\", \"11\", \"a\", \"were\", \"obtained\", \"using\", \"cocrystallization\", \".\"]\n",
      "Word token=> [\"The\", \"concentrated\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \"(\", \"45\", \"mg\", \"·\", \"mL\", \"-\", \"1\", \")\", \"was\", \"incubated\", \"with\", \"5\", \"mM\", \"11\", \"a\", \"for\", \"4\", \"h\", \"at\", \"20\", \"℃\", \",\", \"followed\", \"by\", \"setting\", \"up\", \"crystallization\", \"using\", \"the\", \"vapor\", \"diffusion\", \"sitting-drop\", \"method\", \"at\", \"20\", \"℃\", \"with\", \"equilibration\", \"of\", \"1\", \"µL\", \"protein\", \"(\", \"mixed\", \"with\", \"1\", \"µL\", \"mother\", \"liquor\", \")\", \"against\", \"500\", \"µL\", \"reservoir\", \"composed\", \"of\", \"0.1\", \"M\", \"lithium\", \"sulfate\", \"monohydrate\", \",\", \"0.1\", \"M\", \"sodium\", \"citrate\", \"tribasic\", \"dihydrate\", \",\", \"25\", \"%\", \"PEG\", \"1,000\", \",\", \"pH\", \"6.0\", \".\"]\n",
      "Word token=> [\"The\", \"crystals\", \"were\", \"protected\", \"by\", \"a\", \"cryo-buffer\", \"containing\", \"0.1\", \"M\", \"lithium\", \"sulfate\", \"monohydrate\", \",\", \"0.1\", \"M\", \"sodium\", \"citrate\", \"tribasic\", \"dihydrate\", \",\", \"25\", \"%\", \"PEG\", \"1,000\", \",\", \"15\", \"%\", \"glycerol\", \",\", \"2\", \"mM\", \"11\", \"a\", \",\", \"pH\", \"6.0\", \"and\", \"flash-cooled\", \"in\", \"liquid\", \"nitrogen\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Crystals\", \"of\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \"with\", \"11\", \"n\", \"or\", \"11\", \"f\", \"were\", \"generated\", \"by\", \"using\", \"the\", \"soaking\", \"method\", \".\"]\n",
      "Word token=> [\"Several\", \"free-enzyme\", \"crystals\", \"were\", \"soaked\", \"in\", \"cryo-protectant\", \"buffer\", \"containing\", \"0.1\", \"M\", \"lithium\", \"sulfate\", \"monohydrate\", \",\", \"0.1\", \"M\", \"sodium\", \"citrate\", \"tribasic\", \"dihydrate\", \",\", \"25\", \"%\", \"PEG\", \"1,000\", \",\", \"15\", \"%\", \"glycerol\", \",\", \"5\", \"mM\", \"11\", \"n\", \"(\", \"or\", \"11\", \"f\", \")\", \",\", \"pH\", \"6.0\", \".\"]\n",
      "Word token=> [\"Subsequently\", \",\", \"the\", \"soaked\", \"crystals\", \"were\", \"flash-cooled\", \"in\", \"liquid\", \"nitrogen\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Freshly\", \"prepared\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \"at\", \"a\", \"concentration\", \"of\", \"21.8\", \"mg\", \"·\", \"mL\", \"-\", \"1\", \"was\", \"incubated\", \"with\", \"5\", \"mM\", \"11\", \"a\", \"pre-dissolved\", \"in\", \"100\", \"%\", \"DMSO\", \"at\", \"room\", \"tempature\", \"for\", \"1\", \"h\", \".\"]\n",
      "Word token=> [\"Some\", \"white\", \"precipitate\", \"appeared\", \"in\", \"the\", \"mixture\", \".\"]\n",
      "Word token=> [\"Afterwards\", \",\", \"the\", \"sample\", \"was\", \"centrifuged\", \"at\", \"13,000\", \"x\", \"g\", \"for\", \"20\", \"min\", \"at\", \"4\", \"℃\", \".\"]\n",
      "Word token=> [\"The\", \"supernatant\", \"was\", \"subjected\", \"to\", \"crystallization\", \"trials\", \"using\", \"the\", \"following\", \",\", \"commercially\", \"available\", \"kits\", \":\", \"Sigma\", \"TM\", \"(\", \"Sigma-Aldrich\", \")\", \",\", \"Index\", \"TM\", \",\", \"and\", \"PEG\", \"Rx\", \"TM\", \"(\", \"Hampton\", \"Research\", \")\", \".\"]\n",
      "Word token=> [\"Single\", \"rod-like\", \"crystals\", \"were\", \"detected\", \"both\", \"from\", \"the\", \"Index\", \"TM\", \"screen\", \",\", \"under\", \"the\", \"condition\", \"of\", \"0.1\", \"M\", \"MgCl\", \"2\", \"hexahydrate\", \",\", \"0.1\", \"M\", \"Bis-Tris\", \",\", \"25\", \"%\", \"PEG\", \"3,350\", \",\", \"pH\", \"5.5\", \",\", \"and\", \"from\", \"the\", \"Sigma\", \"TM\", \"screen\", \"at\", \"0.2\", \"M\", \"Li\", \"2\", \"SO\", \"4\", \",\", \"0.1\", \"M\", \"Tris-HCl\", \",\", \"30\", \"%\", \"PEG\", \"4,000\", \",\", \"pH\", \"8.5\", \".\"]\n",
      "Word token=> [\"Crystal\", \"optimization\", \"was\", \"performed\", \"by\", \"using\", \"the\", \"vapor-diffusion\", \"sitting-drop\", \"method\", \",\", \"with\", \"1\", \"μL\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \"-inhibitor\", \"complex\", \"mixed\", \"with\", \"1\", \"μL\", \"precipitant\", \"solution\", \",\", \"and\", \"equilibration\", \"against\", \"500\", \"μL\", \"reservoir\", \"containing\", \"0.1\", \"M\", \"Tris-HCl\", \",\", \"0.2\", \"M\", \"MgCl\", \"2\", \",\", \"pH\", \"8.5\", \",\", \"and\", \"PEG\", \"3,350\", \"varied\", \"from\", \"22\", \"%\", \"to\", \"27\", \"%\", \".\"]\n",
      "Word token=> [\"Another\", \"optimization\", \"screen\", \"was\", \"also\", \"performed\", \"against\", \"a\", \"different\", \"reservoir\", \",\", \"0.1\", \"M\", \"Tris-HCl\", \",\", \"0.2\", \"M\", \"MgCl\", \"2\", \",\", \"pH\", \"range\", \"from\", \"7.5\", \"to\", \"8.5\", \",\", \"and\", \"PEG\", \"4,000\", \"varied\", \"from\", \"24\", \"%\", \"to\", \"34\", \"%\", \".\"]\n",
      "Word token=> [\"Crystals\", \"were\", \"fished\", \"from\", \"different\", \"drops\", \"and\", \"protected\", \"by\", \"cryo-protectant\", \"solution\", \"consisting\", \"of\", \"the\", \"mother\", \"liquor\", \"and\", \"10\", \"%\", \"glycerol\", \".\"]\n",
      "Word token=> [\"Subsequently\", \",\", \"the\", \"crystals\", \"were\", \"flash-cooled\", \"with\", \"liquid\", \"nitrogen\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Diffraction\", \"data\", \"collection\", \",\", \"structure\", \"elucidation\", \"and\", \"refinement\", \".\"]\n",
      "Word token=> [\"Diffraction\", \"data\", \"from\", \"the\", \"crystal\", \"of\", \"the\", \"SARS-CoV\", \"M\", \"pro\", \"in\", \"complex\", \"with\", \"11\", \"a\", \"were\", \"collected\", \"at\", \"100\", \"K\", \"at\", \"synchrotron\", \"beamline\", \"PXI-X\", \"06\", \"SA\", \"(\", \"PSI\", \",\", \"Villigen\", \",\", \"Switzerland\", \")\", \"using\", \"a\", \"Pilatus\", \"6\", \"M\", \"detector\", \"(\", \"DECTRIS\", \")\", \".\"]\n",
      "Word token=> [\"A\", \"diffraction\", \"data\", \"set\", \"from\", \"the\", \"SARS-CoV\", \"M\", \"pro\", \"crystal\", \"with\", \"compound\", \"11\", \"s\", \"was\", \"collected\", \"at\", \"100\", \"K\", \"at\", \"beamline\", \"P\", \"11\", \"of\", \"PETRA\", \"III\", \"(\", \"DESY\", \",\", \"Hamburg\", \",\", \"Germany\", \")\", \",\", \"using\", \"the\", \"same\", \"type\", \"of\", \"detector\", \".\"]\n",
      "Word token=> [\"All\", \"diffraction\", \"data\", \"sets\", \"of\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \"complex\", \"structures\", \"and\", \"of\", \"the\", \"complex\", \"of\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \"with\", \"11\", \"a\", \"were\", \"collected\", \"at\", \"synchrotron\", \"beamline\", \"BL\", \"14.2\", \"of\", \"BESSY\", \"(\", \"Berlin\", \",\", \"Germany\", \")\", \",\", \"using\", \"an\", \"MX\", \"225\", \"CCD\", \"detector\", \"(\", \"Rayonics\", \")\", \".\"]\n",
      "Word token=> [\"All\", \"data\", \"sets\", \"were\", \"processed\", \"by\", \"the\", \"program\", \"XDSAPP\", \"and\", \"scaled\", \"by\", \"SCALA\", \"from\", \"the\", \"CCP\", \"4\", \"suite\", \".\"]\n",
      "Word token=> [\"[\", \"50\", \"]\", \"[\", \"51\", \"]\", \"[\", \"52\", \"]\", \"The\", \"structure\", \"of\", \"SARS-CoV\", \"M\", \"pro\", \"with\", \"11\", \"a\", \"was\", \"determined\", \"by\", \"molecular\", \"replacement\", \"with\", \"the\", \"structure\", \"of\", \"the\", \"complex\", \"between\", \"SARS-CoV\", \"M\", \"pro\", \"and\", \"SG\", \"85\", \"(\", \"PDB\", \"entry\", \"3\", \"TNT\", \";\", \"Zhu\", \"et\", \"al.\", \",\", \"unpublished\", \")\", \"as\", \"search\", \"model\", \",\", \"employing\", \"the\", \"MOLREP\", \"program\", \"(\", \"also\", \"from\", \"the\", \"CCP\", \"4\", \"suite\", \")\", \".\"]\n",
      "Word token=> [\"52\", \",\", \"53\", \"The\", \"complex\", \"structures\", \"of\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \"with\", \"11\", \"a\", \",\", \"11\", \"f\", \",\", \"and\", \"11\", \"n\", \"were\", \"also\", \"determined\", \"with\", \"MOLREP\", \",\", \"using\", \"as\", \"a\", \"search\", \"model\", \"the\", \"structure\", \"of\", \"the\", \"free\", \"enzyme\", \"determined\", \"by\", \"us\", \"(\", \"LZ\", \"et\", \"al.\", \",\", \"unpublished\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"complex\", \"structure\", \"between\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \"and\", \"11\", \"a\", \"was\", \"determined\", \"based\", \"on\", \"the\", \"search\", \"model\", \"of\", \"the\", \"free-enzyme\", \"structure\", \"(\", \"PDB\", \"entry\", \"3\", \"ZYD\", \";\", \"Tan\", \"et\", \"al.\", \",\", \"unpublished\", \")\", \".\"]\n",
      "Word token=> [\"Geometric\", \"restraints\", \"for\", \"the\", \"compounds\", \"11\", \"a\", \",\", \"11\", \"f\", \",\", \"11\", \"n\", \",\", \"and\", \"11\", \"s\", \"were\", \"generated\", \"by\", \"using\", \"JLIGAND\", \"52\", \",\", \"54\", \"and\", \"built\", \"into\", \"Fo-Fc\", \"difference\", \"density\", \"using\", \"the\", \"COOT\", \"software\", \".\"]\n",
      "Word token=> [\"55\", \"Refinement\", \"of\", \"the\", \"structures\", \"was\", \"performed\", \"with\", \"REFMAC\", \"version\", \"5.8.0131\", \"(\", \"ref.\", \"52\", \",\", \"56\", \",\", \"57\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"A\", \"buffer\", \"containing\", \"20\", \"mM\", \"Tris-HCl\", \",\", \"100\", \"mM\", \"NaCl\", \",\", \"1\", \"mM\", \"EDTA\", \",\", \"1\", \"mM\", \"DTT\", \",\", \"pH\", \"7.3\", \",\", \"was\", \"used\", \"for\", \"all\", \"the\", \"enzymatic\", \"assays\", \".\"]\n",
      "Word token=> [\"Two\", \"substrates\", \"with\", \"the\", \"cleavage\", \"sites\", \"of\", \"M\", \"pro\", \"and\", \"3\", \"C\", \"pro\", \",\", \"respectively\", \"(\", \"indicated\", \"by\", \"the\", \"arrow\", \",\", \"↓\", \")\", \",\", \"Dabcyl-KTSAVLQ¯SGFRKM-E\", \"(\", \"Edans\", \")\", \"-NH\", \"2\", \"and\", \"Dabcyl-KEALFQ¯GPPQF-E\", \"(\", \"Edans\", \")\", \"-NH\", \"2\", \"(\", \"95\", \"%\", \"purity\", \";\", \"Biosyntan\", \")\", \",\", \"were\", \"employed\", \"in\", \"the\", \"fluorescence\", \"resonance\", \"energy\", \"transfer\", \"(\", \"FRET\", \")\", \"based\", \"cleavage\", \"assay\", \",\", \"using\", \"a\", \"96\", \"-well\", \"microtiter\", \"plate\", \".\"]\n",
      "Word token=> [\"The\", \"dequenching\", \"of\", \"the\", \"Edans\", \"fluorescence\", \"due\", \"to\", \"the\", \"cleavage\", \"of\", \"the\", \"substrate\", \"as\", \"catalyzed\", \"by\", \"the\", \"proteases\", \"was\", \"monitored\", \"at\", \"460\", \"nm\", \"with\", \"excitation\", \"at\", \"360\", \"nm\", \",\", \"using\", \"a\", \"Flx\", \"800\", \"fluorescence\", \"spectrophotometer\", \"(\", \"BioTek\", \")\", \".\"]\n",
      "Word token=> [\"Curves\", \"of\", \"relative\", \"fluorescence\", \"units\", \"(\", \"RFU\", \")\", \"against\", \"substrate\", \"concentration\", \"were\", \"linear\", \"for\", \"all\", \"substrates\", \"up\", \"to\", \"beyond\", \"50\", \"µM\", \",\", \"indicating\", \"a\", \"minimal\", \"influence\", \"of\", \"the\", \"inner-filter\", \"effect\", \".\"]\n",
      "Word token=> [\"Stock\", \"solutions\", \"of\", \"the\", \"compounds\", \"were\", \"prepared\", \"by\", \"dissolving\", \"them\", \"in\", \"100\", \"%\", \"DMSO\", \".\"]\n",
      "Word token=> [\"The\", \"UV\", \"absorption\", \"of\", \"11\", \"a\", \"was\", \"found\", \"to\", \"be\", \"negligible\", \"at\", \"l\", \"=\", \"360\", \"nm\", \",\", \"so\", \"that\", \"no\", \"interference\", \"with\", \"the\", \"FRET\", \"signal\", \"through\", \"the\", \"inner-filter\", \"effect\", \"was\", \"to\", \"be\", \"expected\", \".\"]\n",
      "Word token=> [\"For\", \"the\", \"determination\", \"of\", \"the\", \"IC\", \"50\", \",\", \"different\", \"proteases\", \"at\", \"a\", \"specified\", \"final\", \"concentration\", \"(\", \"0.5\", \"µM\", \"SARS-CoV\", \"or\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \",\", \"2\", \"µM\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \",\", \"3\", \"µM\", \"EV-A\", \"71\", \"3\", \"C\", \"pro\", \")\", \"were\", \"separately\", \"incubated\", \"with\", \"the\", \"inhibitor\", \"at\", \"various\", \"concentrations\", \"(\", \"0\", \"to\", \"100\", \"μM\", \")\", \"in\", \"reaction\", \"buffer\", \"at\", \"37\", \"℃\", \"for\", \"10\", \"min\", \".\"]\n",
      "Word token=> [\"Afterwards\", \",\", \"the\", \"reaction\", \"was\", \"initiated\", \"by\", \"adding\", \"FRET\", \"peptide\", \"substrate\", \"at\", \"20\", \"μM\", \"final\", \"concentration\", \"(\", \"final\", \"volume\", \":\", \"50\", \"μl\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"IC\", \"50\", \"value\", \"was\", \"determined\", \"by\", \"using\", \"the\", \"GraphPad\", \"Prism\", \"6.0\", \"software\", \"(\", \"GraphPad\", \")\", \".\"]\n",
      "Word token=> [\"Measurements\", \"of\", \"enzymatic\", \"activity\", \"were\", \"performed\", \"in\", \"triplicate\", \"and\", \"are\", \"presented\", \"as\", \"the\", \"mean\", \"±\", \"standard\", \"deviations\", \"(\", \"SD\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Assessment\", \"of\", \"inhibitory\", \"activity\", \"of\", \"a-ketoamides\", \"using\", \"viral\", \"replicons\", \"and\", \"virus-infected\", \"cells\", \"Cells\", \"and\", \"viruses\", \".\"]\n",
      "Word token=> [\"Hepatocellular\", \"carcinoma\", \"cells\", \"(\", \"Huh\", \"7\", \";\", \"ref.\", \"58\", \")\", \"and\", \"their\", \"derivative\", \"constitutively\", \"expressing\", \"T\", \"7\", \"RNA\", \"polymerase\", \"(\", \"Huh-T\", \"7\", \";\", \"ref.\", \"59\", \")\", \"were\", \"grown\", \"in\", \"Dulbecco\", \"'\", \"s\", \"modified\", \"minimal\", \"essential\", \"medium\", \"(\", \"DMEM\", \")\", \"supplemented\", \"with\", \"2\", \"mM\", \"glutamine\", \",\", \"100\", \"U\", \"·\", \"mL\", \"-\", \"1\", \"penicillin\", \",\", \"100\", \"µg\", \"·\", \"mL\", \"-\", \"1\", \"streptomycin\", \"sulfate\", \",\", \"and\", \"fetal\", \"calf\", \"serum\", \"(\", \"10\", \"%\", \"in\", \"growth\", \"medium\", \"and\", \"2\", \"%\", \"in\", \"maintenance\", \"medium\", \")\", \".\"]\n",
      "Word token=> [\"Huh-T\", \"7\", \"cells\", \"were\", \"additionally\", \"supplemented\", \"with\", \"geneticin\", \"(\", \"G-\", \"418\", \"sulfate\", \",\", \"400\", \"µg\", \"·\", \"mL\", \"-\", \"1\", \")\", \".\"]\n",
      "Word token=> [\"Huh-T\", \"7\", \"cells\", \"were\", \"used\", \"for\", \"the\", \"enteroviral\", \"replicons\", \"as\", \"well\", \"as\", \"for\", \"infection\", \"experiments\", \"with\", \"CVB\", \"3\", \"strain\", \"Nancy\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"For\", \"enterovirus\", \"(\", \"except\", \"CVB\", \"3\", \")\", \"infection\", \"experiments\", \",\", \"human\", \"rhabdomyosarcoma\", \"cells\", \"(\", \"RD\", \";\", \"for\", \"EV-A\", \"71\", \";\", \"BRCR\", \"strain\", \")\", \"and\", \"HeLa\", \"Rh\", \"cells\", \"(\", \"for\", \"EV-D\", \"68\", \"and\", \"human\", \"rhinoviruses\", \")\", \"were\", \"grown\", \"in\", \"MEM\", \"Rega\", \"3\", \"medium\", \"supplemented\", \"with\", \"1\", \"%\", \"sodium\", \"bicarbonate\", \",\", \"1\", \"%\", \"Lglutamine\", \",\", \"and\", \"fetal\", \"calf\", \"serum\", \"(\", \"10\", \"%\", \"in\", \"growth\", \"medium\", \"and\", \"2\", \"%\", \"in\", \"maintenance\", \"medium\", \")\", \".\"]\n",
      "Word token=> [\"For\", \"HCoV-\", \"229\", \"E\", \"(\", \"a\", \"kind\", \"gift\", \"from\", \"Volker\", \"Thiel\", \"(\", \"Bern\", \",\", \"Switzerland\", \")\", \")\", \",\", \"culture\", \"and\", \"infection\", \"experiments\", \"were\", \"carried\", \"out\", \"as\", \"described\", \".\"]\n",
      "Word token=> [\"60\", \"For\", \"MERS-CoV\", \"or\", \"SARS-CoV\", \"infection\", \"experiments\", \",\", \"Vero\", \",\", \"Vero\", \"E\", \"6\", \",\", \"and\", \"Huh\", \"7\", \"cells\", \"were\", \"cultured\", \"as\", \"described\", \"previously\", \".\"]\n",
      "Word token=> [\"61\", \",\", \"62\", \"Infection\", \"of\", \"Vero\", \"and\", \"Huh\", \"7\", \"cells\", \"with\", \"MERS-CoV\", \"(\", \"strain\", \"EMC/\", \"2012\", \")\", \"and\", \"SARS-CoV\", \"infection\", \"of\", \"Vero\", \"E\", \"6\", \"cells\", \"(\", \"strain\", \"Frankfurt-\", \"1\", \")\", \"at\", \"low\", \"multiplicity\", \"of\", \"infection\", \"(\", \"MOI\", \")\", \"were\", \"done\", \"as\", \"described\", \"before\", \".\"]\n",
      "Word token=> [\"61\", \",\", \"63\", \"All\", \"work\", \"with\", \"live\", \"MERS-CoV\", \"and\", \"SARS-CoV\", \"was\", \"performed\", \"inside\", \"biosafety\", \"cabinets\", \"in\", \"biosafety\", \"level-\", \"3\", \"facilities\", \"at\", \"Leiden\", \"University\", \"Medical\", \"Center\", \",\", \"The\", \"Netherlands\", \".\"]\n",
      "Word token=> [\"Viral\", \"replicons\", \".\"]\n",
      "Word token=> [\"The\", \"DNA-launched\", \"SARS-CoV\", \"replicon\", \"harbouring\", \"Renilla\", \"luciferase\", \"as\", \"reporter\", \"directly\", \"downstream\", \"of\", \"the\", \"SARS-CoV\", \"replicase\", \"polyprotein-coding\", \"sequence\", \"(\", \"pp\", \"1\", \"a\", \",\", \"pp\", \"1\", \"ab\", \",\", \"Urbani\", \"strain\", \",\", \"acc.\", \"AY\", \"278741\", \")\", \",\", \"in\", \"the\", \"context\", \"of\", \"a\", \"bacterial\", \"artificial\", \"chromosome\", \"(\", \"BAC\", \")\", \"under\", \"the\", \"control\", \"of\", \"the\", \"CMV\", \"promoter\", \",\", \"has\", \"been\", \"described\", \"previously\", \"(\", \"pBAC-REP-RLuc\", \")\", \".\"]\n",
      "Word token=> [\"33\", \"Apart\", \"from\", \"the\", \"replicase\", \"polyprotein\", \",\", \"the\", \"replicon\", \"encodes\", \"the\", \"following\", \"features\", \":\", \"the\", \"5\", \"'\", \"-and\", \"3\", \"'\", \"-non-translated\", \"regions\", \"(\", \"NTR\", \")\", \",\", \"a\", \"ribozyme\", \"(\", \"Rz\", \")\", \",\", \"the\", \"bovine\", \"growth\", \"hormone\", \"sequence\", \",\", \"and\", \"structural\", \"protein\", \"N\", \".\"]\n",
      "Word token=> [\"Subgenomic\", \"replicons\", \"of\", \"CVB\", \"3\", \"(\", \"pT\", \"7\", \"-CVB\", \"3\", \"-FLuc\", \"34\", \")\", \"and\", \"EV-A\", \"71\", \"(\", \"pT\", \"7\", \"-EV\", \"71\", \"-RLuc\", \")\", \"harbouring\", \"T\", \"7\", \"-controlled\", \"complete\", \"viral\", \"genomes\", \",\", \"in\", \"which\", \"the\", \"P\", \"1\", \"capsid-coding\", \"sequence\", \"was\", \"replaced\", \"by\", \"the\", \"Firefly\", \"(\", \"Photinus\", \"pyralis\", \")\", \"or\", \"Renilla\", \"(\", \"Renilla\", \"renifor\", \")\", \"luciferase\", \"gene\", \",\", \"were\", \"generously\", \"provided\", \"by\", \"F.\", \"van\", \"Kuppeveld\", \"and\", \"B.\", \"Zhang\", \",\", \"respectively\", \".\"]\n",
      "Word token=> [\"To\", \"prepare\", \"CVB\", \"3\", \"and\", \"EV-A\", \"71\", \"replicon\", \"RNA\", \"transcripts\", \",\", \"plasmid\", \"DNAs\", \"were\", \"linearized\", \"by\", \"digestion\", \"with\", \"SalI\", \"or\", \"HindIII\", \"(\", \"New\", \"England\", \"Biolabs\", \")\", \",\", \"respectively\", \".\"]\n",
      "Word token=> [\"Copy\", \"RNA\", \"transcripts\", \"were\", \"synthesized\", \"in\", \"vitro\", \"using\", \"linearized\", \"DNA\", \"templates\", \",\", \"T\", \"7\", \"RNA\", \"polymerase\", \",\", \"and\", \"the\", \"T\", \"7\", \"RiboMax™\", \"Large-Scale\", \"RNA\", \"Production\", \"System\", \"(\", \"Promega\", \")\", \"according\", \"to\", \"the\", \"manufacturer\", \"'\", \"s\", \"recommendations\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Transfection\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"Huh-T\", \"7\", \"cells\", \"grown\", \"in\", \"12\", \"-well\", \"plates\", \"to\", \"a\", \"confluency\", \"of\", \"80\", \"%\", \"-\", \"90\", \"%\", \"(\", \"2\", \"-\", \"3\", \"x\", \"10\", \"5\", \"cells/well\", \")\", \"were\", \"washed\", \"with\", \"1\", \"mL\", \"OptiMEM\", \"(\", \"Invitrogen\", \")\", \"and\", \"transfected\", \"with\", \"0.25\", \"µg\", \"of\", \"the\", \"replication-competent\", \"replicon\", \"and\", \"Lipofectamin\", \"2000\", \"or\", \"X-tremeGENE\", \"9\", \"in\", \"300\", \"µl\", \"OptiMEM\", \"(\", \"final\", \"volume\", \")\", \"as\", \"recommended\", \"by\", \"the\", \"manufacturer\", \"(\", \"Invitrogen\", \"or\", \"Roche\", \",\", \"respectively\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"transfection\", \"mixtures\", \"were\", \"incubated\", \"at\", \"37\", \"°\", \"C\", \"for\", \"4\", \"to\", \"5\", \"h\", \"(\", \"Lipofectamin\", \"2000\", \")\", \"or\", \"overnight\", \"(\", \"X-tremeGENE\", \"9\", \")\", \",\", \"prior\", \"to\", \"being\", \"replaced\", \"with\", \"growth\", \"medium\", \"containing\", \"the\", \"compound\", \"under\", \"investigation\", \".\"]\n",
      "Word token=> [\"For\", \"RNA-launched\", \"transfection\", \"of\", \"enteroviral\", \"replicons\", \",\", \"DMRIE-C\", \"was\", \"used\", \"as\", \"transfection\", \"reagent\", \"according\", \"to\", \"the\", \"manufacturer\", \"'\", \"s\", \"recommendations\", \"(\", \"Invitrogen\", \")\", \".\"]\n",
      "Word token=> [\"All\", \"experiments\", \"were\", \"done\", \"in\", \"triplicate\", \"or\", \"quadruplicate\", \"and\", \"the\", \"results\", \"are\", \"presented\", \"as\", \"mean\", \"values\", \"±\", \"SD\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Testing\", \"for\", \"inhibitory\", \"activity\", \"of\", \"candidate\", \"compounds\", \".\"]\n",
      "Word token=> [\"Initially\", \",\", \"we\", \"performed\", \"a\", \"quick\", \"assessment\", \"of\", \"the\", \"inhibitory\", \"activity\", \"of\", \"the\", \"candidate\", \"compounds\", \"towards\", \"the\", \"enteroviral\", \"and\", \"coronaviral\", \"replicons\", \"at\", \"a\", \"concentration\", \"of\", \"40\", \"µM\", \"in\", \"Huh-T\", \"7\", \"cells\", \".\"]\n",
      "Word token=> [\"Compounds\", \"that\", \"were\", \"relatively\", \"powerful\", \"and\", \"non-toxic\", \"at\", \"this\", \"concentration\", \",\", \"were\", \"assayed\", \"in\", \"a\", \"dose-dependent\", \"manner\", \"to\", \"estimate\", \"their\", \"half-maximal\", \"effective\", \"concentration\", \"(\", \"EC\", \"50\", \")\", \"as\", \"well\", \"as\", \"their\", \"cytotoxicity\", \"(\", \"CC\", \"50\", \")\", \",\", \"as\", \"described\", \".\"]\n",
      "Word token=> [\"29\", \"In\", \"brief\", \",\", \"different\", \"concentrations\", \"of\", \"a-ketoamides\", \"(\", \"40\", \"µM\", \"in\", \"screening\", \"experiments\", \"or\", \"increasing\", \"concentration\", \"(\", \"0\", \",\", \"1.25\", \",\", \"2.5\", \",\", \"5\", \",\", \"10\", \",\", \"20\", \",\", \"40\", \"µM\", \")\", \"when\", \"determining\", \"the\", \"EC\", \"50\", \")\", \"were\", \"added\", \"to\", \"growth\", \"medium\", \"of\", \"replicon-transfected\", \"Huh-T\", \"7\", \"cells\", \".\"]\n",
      "Word token=> [\"Twenty-four\", \"hours\", \"later\", \",\", \"the\", \"cells\", \"were\", \"washed\", \"with\", \"1\", \"mL\", \"phosphate-buffered\", \"saline\", \"(\", \"PBS\", \"or\", \"OPTIMEM\", \",\", \"Invitrogen\", \")\", \"and\", \"lysed\", \"in\", \"0.15\", \"mL\", \"Passive\", \"lysis\", \"buffer\", \"(\", \"Promega\", \")\", \"at\", \"room\", \"temperature\", \"(\", \"RT\", \")\", \"for\", \"10\", \"min\", \".\"]\n",
      "Word token=> [\"After\", \"freezing\", \"(\", \"-\", \"80\", \"o\", \"C\", \")\", \"and\", \"thawing\", \"(\", \"RT\", \")\", \",\", \"the\", \"cell\", \"debris\", \"was\", \"removed\", \"by\", \"centrifugation\", \"(\", \"16,000\", \"x\", \"g\", \",\", \"1\", \"min\", \")\", \"and\", \"the\", \"supernatant\", \"(\", \"10\", \"or\", \"20\", \"µl\", \")\", \"was\", \"assayed\", \"for\", \"Firefly\", \"or\", \"Renilla\", \"luciferase\", \"activity\", \"(\", \"Promega\", \"or\", \"Biotrend\", \"Chemikalien\", \")\", \"using\", \"an\", \"Anthos\", \"Lucy-\", \"3\", \"luminescence\", \"plate\", \"reader\", \"(\", \"Anthos\", \"Microsystem\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"antiviral\", \"activity\", \"of\", \"the\", \"compounds\", \"was\", \"evaluated\", \"in\", \"a\", \"cytopathic\", \"effect\", \"(\", \"CPE\", \")\", \"readout\", \"assay\", \"using\", \"MTS\", \"[\", \"3\", \"-\", \"(\", \"4,5\", \"-dimethylthiazol-\", \"2\", \"-yl\", \")\", \"-\", \"5\", \"-\", \"(\", \"3\", \"carboxymethoxyphenol\", \")\", \"-\", \"2\", \"-\", \"(\", \"4\", \"-sulfophenyl\", \")\", \"-\", \"2\", \"H-tetrazolium\", \",\", \"innersalt\", \"]\", \"-based\", \"assay\", \".\"]\n",
      "Word token=> [\"Briefly\", \",\", \"24\", \"h\", \"prior\", \"to\", \"infection\", \",\", \"cells\", \"were\", \"seeded\", \"in\", \"96\", \"-well\", \"plates\", \"at\", \"a\", \"density\", \"of\", \"2.5\", \"x\", \"10\", \"4\", \"(\", \"RD\", \"cells\", \")\", \"or\", \"of\", \"1.7\", \"x\", \"10\", \"4\", \"(\", \"HeLa\", \"Rh\", \")\", \"per\", \"well\", \"in\", \"medium\", \"supplemented\", \"with\", \"2\", \"%\", \"FCS\", \".\"]\n",
      "Word token=> [\"For\", \"HRV\", \"2\", \"and\", \"HRV\", \"14\", \"infection\", \",\", \"the\", \"medium\", \"contained\", \"30\", \"mM\", \"MgCl\", \"2\", \".\"]\n",
      "Word token=> [\"The\", \"next\", \"day\", \",\", \"serial\", \"dilutions\", \"of\", \"the\", \"compounds\", \"and\", \"virus\", \"inoculum\", \"were\", \"added\", \".\"]\n",
      "Word token=> [\"The\", \"read-out\", \"was\", \"performed\", \"3\", \"days\", \"post\", \"infection\", \"as\", \"follows\", \":\", \"The\", \"medium\", \"was\", \"removed\", \"and\", \"100\", \"μl\", \"of\", \"5\", \"%\", \"MTS\", \"in\", \"Phenol\", \"Red-free\", \"MEM\", \"was\", \"added\", \"to\", \"each\", \"well\", \".\"]\n",
      "Word token=> [\"Plates\", \"were\", \"incubated\", \"for\", \"1\", \"h\", \"at\", \"37\", \"°\", \"C\", \",\", \"then\", \"the\", \"optical\", \"density\", \"at\", \"498\", \"nm\", \"(\", \"OD\", \"498\", \")\", \"of\", \"each\", \"well\", \"was\", \"measured\", \"by\", \"microtiter\", \"plate\", \"reader\", \"(\", \"Saffire\", \"2\", \",\", \"Tecan\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"OD\", \"values\", \"were\", \"converted\", \"to\", \"percentage\", \"of\", \"controls\", \"and\", \"the\", \"EC\", \"50\", \"was\", \"calculated\", \"by\", \"logarithmic\", \"interpolation\", \"as\", \"the\", \"concentration\", \"of\", \"compound\", \"that\", \"results\", \"in\", \"a\", \"50\", \"%\", \"protective\", \"effect\", \"against\", \"virus-induced\", \"CPE\", \".\"]\n",
      "Word token=> [\"For\", \"each\", \"condition\", \",\", \"cell\", \"morphology\", \"was\", \"also\", \"evaluated\", \"microscopically\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Assays\", \"with\", \"MERS-CoV\", \"and\", \"SARS-CoV\", \"were\", \"performed\", \"as\", \"previously\", \"described\", \".\"]\n",
      "Word token=> [\"61\", \",\", \"63\", \"In\", \"brief\", \",\", \"Huh\", \"7\", \",\", \"Vero\", \",\", \"or\", \"Vero\", \"E\", \"6\", \"cells\", \"were\", \"seeded\", \"in\", \"96\", \"-well\", \"plates\", \"at\", \"a\", \"density\", \"of\", \"1\", \"×\", \"10\", \"4\", \"(\", \"Huh\", \"7\", \"and\", \"Vero\", \"E\", \"6\", \")\", \"or\", \"2\", \"×\", \"10\", \"4\", \"cells\", \"(\", \"Vero\", \")\", \"per\", \"well\", \".\"]\n",
      "Word token=> [\"After\", \"overnight\", \"growth\", \",\", \"cells\", \"were\", \"treated\", \"with\", \"the\", \"indicated\", \"compound\", \"concentrations\", \"or\", \"DMSO\", \"(\", \"solvent\", \"control\", \")\", \"and\", \"infected\", \"with\", \"an\", \"MOI\", \"of\", \"0.005\", \"(\", \"final\", \"volume\", \"150\", \"µl/well\", \"in\", \"Eagle\", \"'\", \"s\", \"minimal\", \"essential\", \"medium\", \"(\", \"EMEM\", \")\", \"containing\", \"2\", \"%\", \"FCS\", \",\", \"2\", \"mM\", \"L-glutamine\", \",\", \"and\", \"antibiotics\", \")\", \".\"]\n",
      "Word token=> [\"Huh\", \"7\", \"cells\", \"were\", \"incubated\", \"for\", \"two\", \"days\", \"and\", \"Vero/VeroE\", \"6\", \"cells\", \"for\", \"three\", \"days\", \",\", \"and\", \"differences\", \"in\", \"cell\", \"viability\", \"caused\", \"by\", \"virus-induced\", \"CPE\", \"or\", \"by\", \"compound-specific\", \"side\", \"effects\", \"were\", \"analyzed\", \"using\", \"the\", \"CellTiter\", \"96\", \"AQueous\", \"Non-Radioactive\", \"Cell\", \"Proliferation\", \"Assay\", \"(\", \"Promega\", \")\", \",\", \"according\", \"to\", \"the\", \"manufacturer\", \"'\", \"s\", \"instructions\", \".\"]\n",
      "Word token=> [\"Absorbance\", \"at\", \"490\", \"nm\", \"(\", \"A\", \"490\", \")\", \"was\", \"measured\", \"using\", \"a\", \"Berthold\", \"Mithras\", \"LB\", \"940\", \"96\", \"-well\", \"plate\", \"reader\", \"(\", \"Berthold\", \")\", \".\"]\n",
      "Word token=> [\"Cytotoxic\", \"effects\", \"caused\", \"by\", \"compound\", \"treatment\", \"alone\", \"were\", \"monitored\", \"in\", \"parallel\", \"plates\", \"containing\", \"mock-infected\", \"cells\", \".\"]\n",
      "Word token=> [\"HCoV-\", \"229\", \"E-Rev\", \":\", \"5\", \"′\", \"-ggTCGTTTAGTTGAGAAAAGT\", \"-\", \"3\", \"′\", \",\", \"and\", \"229\", \"E-ZNA\", \"probe\", \":\", \"5\", \"'\", \"-\", \"6\", \"-Fam-AGA\", \"(\", \"pdC\", \")\", \"TT\", \"(\", \"pdU\", \")\", \"G\", \"(\", \"pdU\", \")\", \"GT\", \"(\", \"pdC\", \")\", \"TA\", \"(\", \"pdC\", \")\", \"T-ZNA-\", \"3\", \"-BHQ-\", \"1\", \"-\", \"3\", \"'\", \"(\", \"Metabion\", \")\", \".\"]\n",
      "Word token=> [\"Standard\", \"curves\", \"were\", \"prepared\", \"using\", \"serial\", \"dilutions\", \"of\", \"RNA\", \"isolated\", \"from\", \"virus\", \"stock\", \".\"]\n",
      "Word token=> [\"Data\", \"were\", \"analysed\", \"using\", \"GraphPad\", \"Prism\", \"5.0\", \";\", \"EC\", \"50\", \"values\", \"were\", \"calculated\", \"based\", \"on\", \"a\", \"4\", \"parameter\", \"logistic\", \"statistics\", \"equation\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"In\", \"parallel\", \"to\", \"the\", \"qPCR\", \"assays\", \"with\", \"inhibitors\", \",\", \"cell\", \"viability\", \"assays\", \"were\", \"performed\", \"using\", \"Alamar-Blue™\", \"Cell\", \"Viability\", \"Reagent\", \"(\", \"ThermoFisher\", \")\", \"according\", \"to\", \"the\", \"manufacturer\", \"'\", \"s\", \"instruction\", \".\"]\n",
      "Word token=> [\"CC\", \"50\", \"values\", \"were\", \"calculated\", \"using\", \"inhibitor\", \"versus\", \"normalized\", \"response\", \"statistics\", \"equation\", \"by\", \"including\", \"proper\", \"controls\", \"(\", \"no\", \"inhibitor\", \"and\", \"1\", \"%\", \"Triton-X-\", \"100\", \"-treated\", \"cells\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Determination\", \"of\", \"the\", \"cell\", \"toxicity\", \"of\", \"candidate\", \"compounds\", \".\"]\n",
      "Word token=> [\"The\", \"CellTiter\", \"96\", \"Aqueous\", \"One\", \"Solution\", \"Cell\", \"Proliferation\", \"Assay\", \"(\", \"MTS\", \"test\", \",\", \"Promega\", \")\", \",\", \"the\", \"CellTiter\", \"Glo\", \"assay\", \"kit\", \"(\", \"Promega\", \")\", \",\", \"the\", \"Non-Destructive\", \"Cytotoxicity\", \"Bio-Assay\", \"(\", \"ToxiLight\", \"(\", \"measuring\", \"the\", \"release\", \"of\", \"adenylate\", \"kinase\", \"from\", \"damaged\", \"cells\", \")\", \",\", \"Lonza\", \"Rockland\", \")\", \",\", \"or\", \"the\", \"AlamarBlue™\", \"Cell\", \"Viability\", \"Reagent\", \"(\", \"ThermoFisher\", \")\", \"were\", \"used\", \"to\", \"determine\", \"the\", \"cytotoxic\", \"effect\", \"of\", \"compounds\", \"towards\", \"host\", \"cells\", \"according\", \"to\", \"the\", \"manufacturers\", \"'\", \"recommendations\", \".\"]\n",
      "Word token=> [\"29\", \",\", \"65\", \"Chemical\", \"synthesis\", \"of\", \"a-ketoamides\", \"General\", \"procedure\", \".\"]\n",
      "Word token=> [\"Reagents\", \"were\", \"purchased\", \"from\", \"commercial\", \"sources\", \"and\", \"used\", \"without\", \"purification\", \".\"]\n",
      "Word token=> [\"HSGF\", \"254\", \"(\", \"0.15\", \"-\", \"0.2\", \"mm\", \"thickness\", \")\", \"was\", \"used\", \"for\", \"analytical\", \"thin-layer\", \"chromatography\", \"(\", \"TLC\", \")\", \".\"]\n",
      "Word token=> [\"All\", \"products\", \"were\", \"characterized\", \"by\", \"their\", \"NMR\", \"and\", \"MS\", \"spectra\", \".\"]\n",
      "Word token=> [\"1\", \"H\", \"NMR\", \"spectra\", \"were\", \"recorded\", \"on\", \"300\", \"-MHz\", \",\", \"400\", \"-MHz\", \",\", \"or\", \"500\", \"-MHz\", \"instruments\", \".\"]\n",
      "Word token=> [\"Chemical\", \"shifts\", \"were\", \"reported\", \"in\", \"parts\", \"per\", \"million\", \"(\", \"ppm\", \",\", \"δ\", \")\", \"down-field\", \"from\", \"tetramethylsilane\", \".\"]\n",
      "Word token=> [\"Proton\", \"coupling\", \"patterns\", \"were\", \"described\", \"as\", \"singlet\", \"(\", \"s\", \")\", \",\", \"doublet\", \"(\", \"d\", \")\", \",\", \"triplet\", \"(\", \"t\", \")\", \",\", \"quartet\", \"(\", \"q\", \")\", \",\", \"multiplet\", \"(\", \"m\", \")\", \",\", \"and\", \"broad\", \"(\", \"br\", \")\", \".\"]\n",
      "Word token=> [\"Mass\", \"spectra\", \"were\", \"recorded\", \"using\", \"a\", \"Bruker\", \"ESI\", \"ion-trap\", \"HCT\", \"Ultra\", \".\"]\n",
      "Word token=> [\"HPLC\", \"spectra\", \"were\", \"recorded\", \"by\", \"LC\", \"20\", \"A\", \"or\", \"LC\", \"10\", \"A\", \"(\", \"Shimadzu\", \"Corporation\", \")\", \"with\", \"Shim-pack\", \"GIST\", \"C\", \"18\", \"(\", \"5\", \"µm\", \",\", \"4.6\", \"x\", \"150\", \"mm\", \")\", \"with\", \"three\", \"solvent\", \"systems\", \"(\", \"methanol/water\", \",\", \"methanol/\", \"0.1\", \"%\", \"HCOOH\", \"in\", \"water\", \"or\", \"methanol/\", \"0.1\", \"%\", \"ammonia\", \"in\", \"water\", \")\", \".\"]\n",
      "Word token=> [\"Purity\", \"was\", \"determined\", \"by\", \"reversed-phase\", \"HPLC\", \"and\", \"was\", \"≥\", \"95\", \"%\", \"for\", \"all\", \"compounds\", \"tested\", \"biologically\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Synthesis\", \"of\", \"(\", \"2\", \"S\", \",\", \"4\", \"R\", \")\", \"-dimethyl\", \"2\", \"-\", \"(\", \"tert-butoxycarbonylamino\", \")\", \"-\", \"4\", \"-\", \"(\", \"cyanomethyl\", \")\", \"pentanedioate\", \"(\", \"1\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"To\", \"a\", \"solution\", \"of\", \"N-Boc-L-glutamic\", \"acid\", \"dimethyl\", \"ester\", \"(\", \"6.0\", \"g\", \",\", \"21.8\", \"mmol\", \")\", \"in\", \"THF\", \"(\", \"60\", \"mL\", \")\", \"was\", \"added\", \"dropwise\", \"a\", \"solution\", \"of\", \"lithium\", \"bis\", \"(\", \"trimethylsilyl\", \")\", \"amide\", \"(\", \"LHMDS\", \")\", \"in\", \"THF\", \"(\", \"47\", \"mL\", \",\", \"1\", \"M\", \")\", \"at\", \"−\", \"78\", \"°\", \"C\", \"under\", \"nitrogen\", \".\"]\n",
      "Word token=> [\"The\", \"resulting\", \"dark\", \"mixture\", \"was\", \"stirred\", \"at\", \"−\", \"78\", \"°\", \"C.\", \"Meanwhile\", \",\", \"bromoacetonitrile\", \"(\", \"1.62\", \"mL\", \",\", \"23.3\", \"mmol\", \")\", \"was\", \"added\", \"dropwise\", \"to\", \"the\", \"dianion\", \"solution\", \"over\", \"a\", \"period\", \"of\", \"1\", \"h\", \"while\", \"keeping\", \"the\", \"temperature\", \"below\", \"−\", \"70\", \"°\", \"C.\", \"The\", \"reaction\", \"mixture\", \"was\", \"stirred\", \"at\", \"−\", \"78\", \"°\", \"C\", \"for\", \"additional\", \"2\", \"h\", \".\"]\n",
      "Word token=> [\"After\", \"the\", \"consumption\", \"of\", \"the\", \"reactant\", \"was\", \"confirmed\", \"by\", \"TLC\", \"analysis\", \",\", \"the\", \"reaction\", \"was\", \"quenched\", \"by\", \"methanol\", \"(\", \"3\", \"mL\", \")\", \",\", \"and\", \"acetic\", \"acid\", \"(\", \"3\", \"mL\", \")\", \"in\", \"precooled\", \"THF\", \"(\", \"20\", \"mL\", \")\", \"was\", \"added\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"After\", \"stirring\", \"for\", \"30\", \"min\", \",\", \"the\", \"cooling\", \"bath\", \"was\", \"removed\", \".\"]\n",
      "Word token=> [\"The\", \"reaction\", \"mixture\", \"was\", \"allowed\", \"to\", \"warm\", \"up\", \"to\", \"room\", \"temperature\", \"and\", \"then\", \"poured\", \"into\", \"brine\", \"(\", \"40\", \"mL\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"organic\", \"layer\", \"was\", \"concentrated\", \"and\", \"purified\", \"by\", \"flash\", \"column\", \"chromatography\", \"(\", \"petroleum\", \"ether/ethyl\", \"acetate\", \"=\", \"4\", \"/\", \"1\", \")\", \"to\", \"give\", \"product\", \"1\", \"(\", \"4.92\", \"g\", \",\", \"72\", \"%\", \")\", \"as\", \"colorless\", \"oil\", \".\"]\n",
      "Word token=> [\"1\", \"H\", \"NMR\", \"(\", \"CDCl\", \"3\", \",\", \"400\", \"MHz\", \")\", \"δ\", \"5.23\", \"(\", \"1\", \"H\", \",\", \"d\", \",\", \"J\", \"=\", \"9.0\", \"Hz\", \")\", \",\", \"4.43\", \"-\", \"4.36\", \"(\", \"1\", \"H\", \",\", \"m\", \")\", \",\", \"3.77\", \"(\", \"1\", \"H\", \",\", \"s\", \")\", \",\", \"3.76\", \"(\", \"1\", \"H\", \",\", \"s\", \")\", \",\", \"2.89\", \"-\", \"2.69\", \"(\", \"3\", \"H\", \",\", \"m\", \")\", \",\", \"2.20\", \"-\", \"2.14\", \"(\", \"2\", \"H\", \",\", \"m\", \")\", \",\", \"1.45\", \"(\", \"9\", \"H\", \",\", \"s\", \")\", \".\"]\n",
      "Word token=> [\"ESI-MS\", \"(\", \"m/z\", \")\", \":\", \"315\", \"(\", \"M\", \"+\", \"H\", \")\", \"+\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Synthesis\", \"of\", \"(\", \"S\", \")\", \"-methyl\", \"2\", \"-\", \"(\", \"tert-butoxycarbonylamino\", \")\", \"-\", \"3\", \"-\", \"(\", \"(\", \"S\", \")\", \"-\", \"2\", \"oxopyrrolidin-\", \"3\", \"-yl\", \")\", \"propanoate\", \"(\", \"2\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"In\", \"a\", \"hydrogenation\", \"flask\", \"was\", \"placed\", \"compound\", \"1\", \"(\", \"4.0\", \"g\", \",\", \"12.7\", \"mmol\", \")\", \",\", \"5\", \"mL\", \"of\", \"chloroform\", \"and\", \"60\", \"mL\", \"of\", \"methanol\", \"before\", \"the\", \"addition\", \"of\", \"PtO\", \"2\", \".\"]\n",
      "Word token=> [\"The\", \"resulting\", \"mixture\", \"was\", \"stirred\", \"under\", \"hydrogen\", \"at\", \"20\", \"o\", \"C\", \"for\", \"12\", \"h\", \".\"]\n",
      "Word token=> [\"Then\", \"the\", \"mixture\", \"was\", \"filtered\", \"over\", \"Celite\", \"to\", \"remove\", \"the\", \"catalyst\", \".\"]\n",
      "Word token=> [\"NaOAc\", \"(\", \"6.77\", \"g\", \",\", \"25.5\", \"mmol\", \")\", \"was\", \"added\", \"to\", \"the\", \"filtrate\", \"before\", \"the\", \"resulting\", \"mixture\", \"was\", \"stirred\", \"at\", \"60\", \"o\", \"C\", \"for\", \"12\", \"h\", \".\"]\n",
      "Word token=> [\"The\", \"reaction\", \"was\", \"quenched\", \"with\", \"water\", \"(\", \"30\", \"mL\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"suspension\", \"was\", \"extracted\", \"with\", \"ethyl\", \"acetate\", \".\"]\n",
      "Word token=> [\"The\", \"organic\", \"layers\", \"were\", \"combined\", \",\", \"dried\", \"(\", \"MgSO\", \"4\", \")\", \",\", \"and\", \"filtered\", \".\"]\n",
      "Word token=> [\"The\", \"light-brown\", \"filtrate\", \"was\", \"concentrated\", \"and\", \"purified\", \"by\", \"silica\", \"gel\", \"column\", \"chromatography\", \"(\", \"petroleum\", \"ether/ethyl\", \"acetate\", \"=\", \"4\", \"/\", \"1\", \")\", \"to\", \"give\", \"the\", \"product\", \"2\", \"(\", \"2.20\", \"g\", \",\", \"61\", \"%\", \")\", \"as\", \"white\", \"solid\", \".\"]\n",
      "Word token=> [\"1\", \"H\", \"NMR\", \"(\", \"CDCl\", \"3\", \")\", \"δ\", \"6.02\", \"(\", \"1\", \"H\", \",\", \"br\", \")\", \",\", \"5\", \".\"]\n",
      "Word token=> [\"Synthesis\", \"of\", \"(\", \"S\", \")\", \"-methyl\", \"2\", \"-amino-\", \"3\", \"-\", \"(\", \"(\", \"S\", \")\", \"-\", \"2\", \"-oxopyrrolidin-\", \"3\", \"yl\", \")\", \"propanoate\", \"(\", \"3\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Compound\", \"2\", \"(\", \"1.0\", \"g\", \",\", \"3.5\", \"mmol\", \")\", \"was\", \"dissolved\", \"in\", \"10\", \"mL\", \"dichloromethane\", \"(\", \"DCM\", \")\", \",\", \"then\", \"10\", \"mL\", \"trifluoroacetic\", \"acid\", \"(\", \"TFA\", \")\", \"was\", \"added\", \".\"]\n",
      "Word token=> [\"The\", \"reaction\", \"mixture\", \"was\", \"stirred\", \"at\", \"20\", \"o\", \"C\", \"for\", \"0.5\", \"h\", \",\", \"and\", \"concentrated\", \"in\", \"vacuo\", \"to\", \"get\", \"a\", \"colorless\", \"oil\", \",\", \"which\", \"could\", \"be\", \"used\", \"for\", \"the\", \"following\", \"step\", \"without\", \"purification\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"ESI-MS\", \"(\", \"m/z\", \")\", \":\", \"187\", \"(\", \"M\", \"+\", \"H\", \")\", \"+\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Synthesis\", \"of\", \"methyl\", \"N-substituted\", \"amino-acid\", \"esters\", \"4\", \"General\", \"procedure\", \".\"]\n",
      "Word token=> [\"The\", \"methyl\", \"amino-acid\", \"ester\", \"hydrochloride\", \"(\", \"6.0\", \"mmol\", \")\", \"was\", \"dissolved\", \"in\", \"20\", \"mL\", \"CH\", \"2\", \"Cl\", \"2\", \",\", \"and\", \"then\", \"acyl\", \"chloride\", \"(\", \"6.0\", \"mmol\", \")\", \",\", \"triethylamine\", \"(\", \"1.69\", \"mL\", \",\", \"12.0\", \"mmol\", \")\", \"were\", \"added\", \",\", \"before\", \"the\", \"reaction\", \"was\", \"stirred\", \"for\", \"2\", \"h\", \"at\", \"20\", \"o\", \"C.\", \"The\", \"reaction\", \"mixture\", \"was\", \"diluted\", \"with\", \"20\", \"mL\", \"CH\", \"2\", \"Cl\", \"2\", \",\", \"washed\", \"with\", \"50\", \"mL\", \"of\", \"saturated\", \"brine\", \"(\", \"2\", \"×\", \"25\", \"mL\", \")\", \",\", \"and\", \"dried\", \"over\", \"Na\", \"2\", \"SO\", \"4\", \".\"]\n",
      "Word token=> [\"The\", \"solvent\", \"was\", \"evaporated\", \"and\", \"the\", \"product\", \"4\", \"was\", \"obtained\", \"as\", \"white\", \"solid\", \"(\", \"70\", \"-\", \"95\", \"%\", \"yield\", \")\", \",\", \"which\", \"could\", \"be\", \"used\", \"for\", \"the\", \"next\", \"step\", \"without\", \"further\", \"purification\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"(\", \"S\", \")\", \"-methyl\", \"2\", \"-cinnamamido-\", \"3\", \"-phenylpropanoate\", \"(\", \"4\", \"a\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"methyl\", \"L-phenylalaninate\", \"hydrochloride\", \"(\", \"1.30\", \"g\", \",\", \"6.0\", \"mmol\", \")\", \"was\", \"dissolved\", \"in\", \"20\", \"mL\", \"CH\", \"2\", \"Cl\", \"2\", \",\", \"and\", \"then\", \"cinnamoyl\", \"chloride\", \"(\", \"1.00\", \"g\", \",\", \"6.0\", \"mmol\", \")\", \",\", \"triethylamine\", \"(\", \"1.69\", \"mL\", \",\", \"12.0\", \"mmol\", \")\", \"were\", \"added\", \",\", \"before\", \"the\", \"reaction\", \"was\", \"stirred\", \"for\", \"2\", \"h\", \"at\", \"room\", \"temperature\", \".\"]\n",
      "Word token=> [\"The\", \"reaction\", \"mixture\", \"was\", \"diluted\", \"with\", \"20\", \"mL\", \"CH\", \"2\", \"Cl\", \"2\", \",\", \"washed\", \"with\", \"50\", \"mL\", \"of\", \"saturated\", \"brine\", \"(\", \"2\", \"×\", \"25\", \"mL\", \")\", \",\", \"and\", \"dried\", \"over\", \"Na\", \"2\", \"SO\", \"4\", \".\"]\n",
      "Word token=> [\"The\", \"solvent\", \"was\", \"evaporated\", \"and\", \"the\", \"product\", \"4\", \"a\", \"was\", \"obtained\", \"as\", \"white\", \"solid\", \"(\", \"1.75\", \"g\", \",\", \"95\", \"%\", \")\", \",\", \"which\", \"could\", \"be\", \"used\", \"for\", \"the\", \"next\", \"step\", \"without\", \"further\", \"purification\", \".\"]\n",
      "Word token=> String[]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"Synthesis\", \"of\", \"N-substituted\", \"amino\", \"acids\", \"5\", \"(\", \"general\", \"procedure\", \")\", \".\"]\n",
      "Word token=> [\"1\", \"M\", \"NaOH\", \"(\", \"5\", \"mL\", \")\", \"was\", \"added\", \"to\", \"a\", \"solution\", \"of\", \"compound\", \"4\", \"(\", \"3.0\", \"mmol\", \")\", \"in\", \"methanol\", \"(\", \"5\", \"mL\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"reaction\", \"was\", \"stirred\", \"for\", \"20\", \"min\", \"at\", \"20\", \"o\", \"C.\", \"Then\", \"1\", \"M\", \"HCl\", \"was\", \"added\", \"to\", \"the\", \"reaction\", \"solution\", \"until\", \"pH\", \"=\", \"1\", \".\"]\n",
      "Word token=> [\"Then\", \"the\", \"reaction\", \"mixture\", \"was\", \"extracted\", \"with\", \"100\", \"mL\", \"of\", \"CH\", \"2\", \"Cl\", \"2\", \"(\", \"2\", \"×\", \"50\", \"mL\", \")\", \"and\", \"the\", \"organic\", \"layer\", \"was\", \"washed\", \"with\", \"50\", \"mL\", \"of\", \"brine\", \"and\", \"dried\", \"over\", \"Na\", \"2\", \"SO\", \"4\", \".\"]\n",
      "Word token=> [\"The\", \"solvent\", \"was\", \"evaporated\", \"and\", \"the\", \"crude\", \"material\", \"purified\", \"on\", \"silica\", \",\", \"eluted\", \"with\", \"mixtures\", \"of\", \"CH\", \"2\", \"Cl\", \"2\", \"/MeOH\", \"(\", \"20\", \"/\", \"1\", \")\", \"to\", \"afford\", \"the\", \"product\", \"5\", \"(\", \"90\", \"-\", \"96\", \"%\", \"yield\", \")\", \"as\", \"a\", \"white\", \"solid\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Synthesis\", \"of\", \"compounds\", \"6\", \"(\", \"General\", \"procedure\", \")\", \".\"]\n",
      "Word token=> [\"Compound\", \"5\", \"(\", \"2.7\", \"mmol\", \")\", \"was\", \"dissolved\", \"in\", \"10\", \"mL\", \"of\", \"dry\", \"CH\", \"2\", \"Cl\", \"2\", \".\"]\n",
      "Word token=> [\"To\", \"this\", \"solution\", \",\", \"1.5\", \"equiv\", \"(\", \"1.54\", \"g\", \")\", \"of\", \"1\", \"-\", \"[\", \"bis\", \"(\", \"dimethylamino\", \")\", \"methylene\", \"]\", \"-\", \"1\", \"H-\", \"1,2,3\", \"triazolo\", \"[\", \"4,5\", \"-b\", \"]\", \"pyridinium\", \"3\", \"-oxide\", \"hexafluorophosphate\", \"(\", \"HATU\", \")\", \"was\", \"added\", \",\", \"and\", \"the\", \"reaction\", \"was\", \"stirred\", \"for\", \"0.5\", \"h\", \"at\", \"20\", \"o\", \"C.\", \"Then\", \"compound\", \"3\", \"(\", \"500\", \"mg\", \",\", \"2.7\", \"mmol\", \")\", \"and\", \"TEA\", \"(\", \"0.70\", \"mL\", \",\", \"5.42\", \"mmol\", \")\", \"was\", \"added\", \"to\", \"the\", \"reaction\", \".\"]\n",
      "Word token=> [\"The\", \"reaction\", \"was\", \"stirred\", \"for\", \"another\", \"6\", \"h\", \".\"]\n",
      "Word token=> [\"The\", \"reaction\", \"mixture\", \"was\", \"poured\", \"into\", \"10\", \"mL\", \"water\", \".\"]\n",
      "Word token=> [\"The\", \"aqueous\", \"solution\", \"was\", \"extracted\", \"with\", \"50\", \"mL\", \"of\", \"CH\", \"2\", \"Cl\", \"2\", \"(\", \"2\", \"×\", \"25\", \"mL\", \")\", \"and\", \"washed\", \"with\", \"50\", \"mL\", \"of\", \"saturated\", \"brine\", \"(\", \"2\", \"×\", \"25\", \"mL\", \")\", \"and\", \"dried\", \"over\", \"Na\", \"2\", \"SO\", \"4\", \".\"]\n",
      "Word token=> [\"The\", \"solvent\", \"was\", \"evaporated\", \"and\", \"the\", \"crude\", \"material\", \"purified\", \"on\", \"silica\", \",\", \"eluted\", \"with\", \"a\", \"mixture\", \"of\", \"CH\", \"2\", \"Cl\", \"2\", \"/MeOH\", \"(\", \"40\", \"/\", \"1\", \")\", \"to\", \"give\", \"the\", \"product\", \"6\", \"(\", \"62\", \"-\", \"84\", \"%\", \"yield\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Synthesis\", \"of\", \"alcohols\", \"7\", \"(\", \"general\", \"procedure\", \")\", \".\"]\n",
      "Word token=> [\"Compound\", \"6\", \"(\", \"1.1\", \"mmol\", \")\", \"was\", \"dissolved\", \"in\", \"methanol\", \"(\", \"40\", \"mL\", \")\", \",\", \"then\", \"NaBH\", \"4\", \"(\", \"0.34\", \"g\", \",\", \"8.8\", \"mmol\", \")\", \"was\", \"added\", \"under\", \"ambient\", \"conditions\", \".\"]\n",
      "Word token=> [\"The\", \"reaction\", \"mixture\", \"was\", \"stirred\", \"at\", \"20\", \"o\", \"C\", \"for\", \"2\", \"h\", \".\"]\n",
      "Word token=> [\"Then\", \"the\", \"reaction\", \"was\", \"quenched\", \"with\", \"water\", \"(\", \"30\", \"mL\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"suspension\", \"was\", \"extracted\", \"with\", \"ethyl\", \"acetate\", \".\"]\n",
      "Word token=> [\"The\", \"organic\", \"layers\", \"were\", \"combined\", \",\", \"dried\", \",\", \"and\", \"filtered\", \".\"]\n",
      "Word token=> [\"The\", \"filtrate\", \"was\", \"evaporated\", \"to\", \"dryness\", \"and\", \"could\", \"be\", \"used\", \"for\", \"the\", \"next\", \"step\", \"without\", \"further\", \"purification\", \"(\", \"46\", \"-\", \"85\", \"%\", \"yield\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Synthesis\", \"of\", \"aldehydes\", \"8\", \"(\", \"general\", \"procedure\", \")\", \".\"]\n",
      "Word token=> [\"Compound\", \"7\", \"(\", \"0.75\", \"mmol\", \")\", \"was\", \"dissolved\", \"in\", \"CH\", \"2\", \"Cl\", \"2\", \",\", \"then\", \"Dess-Martin\", \"periodinane\", \"(\", \"337\", \"mg\", \",\", \"0.79\", \"mmol\", \")\", \"and\", \"NaHCO\", \"3\", \"(\", \"66\", \"mg\", \",\", \"0.79\", \"mmol\", \")\", \"were\", \"added\", \".\"]\n",
      "Word token=> [\"The\", \"resulting\", \"mixture\", \"was\", \"stirred\", \"at\", \"20\", \"o\", \"C\", \"for\", \"1\", \"h\", \".\"]\n",
      "Word token=> [\"The\", \"mixture\", \"was\", \"concentrated\", \"and\", \"purified\", \"by\", \"column\", \"chromatography\", \"on\", \"silica\", \"gel\", \"(\", \"CH\", \"2\", \"Cl\", \"2\", \"/MeOH\", \"=\", \"20\", \"/\", \"1\", \")\", \"to\", \"give\", \"the\", \"product\", \"8\", \"as\", \"white\", \"solid\", \"(\", \"88\", \"-\", \"95\", \"%\", \"yield\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Synthesis\", \"of\", \"compounds\", \"9\", \"(\", \"general\", \"procedure\", \")\", \".\"]\n",
      "Word token=> [\"Compound\", \"8\", \"(\", \"0.40\", \"mmol\", \")\", \"was\", \"dissolved\", \"in\", \"CH\", \"2\", \"Cl\", \"2\", \",\", \"and\", \"then\", \"acetic\", \"acid\", \"(\", \"0.028\", \"g\", \",\", \"0.47\", \"mmol\", \")\", \"and\", \"isocyanide\", \"(\", \"0.43\", \"mmol\", \")\", \"were\", \"added\", \"successively\", \"to\", \"the\", \"solution\", \".\"]\n",
      "Word token=> [\"The\", \"reaction\", \"was\", \"stirred\", \"at\", \"20\", \"o\", \"C\", \"for\", \"24\", \"h\", \".\"]\n",
      "Word token=> [\"Then\", \"the\", \"solvent\", \"was\", \"evaporated\", \"and\", \"the\", \"crude\", \"material\", \"purified\", \"on\", \"silica\", \",\", \"eluted\", \"with\", \"a\", \"mixture\", \"of\", \"CH\", \"2\", \"Cl\", \"2\", \"/MeOH\", \"(\", \"20\", \"/\", \"1\", \")\", \"to\", \"give\", \"the\", \"product\", \"9\", \"(\", \"46\", \"-\", \"84\", \"%\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Synthesis\", \"of\", \"α-hydroxyamides\", \"10\", \"(\", \"general\", \"procedure\", \")\", \".\"]\n",
      "Word token=> [\"1\", \"M\", \"NaOH\", \"(\", \"0.5\", \"mL\", \")\", \"was\", \"added\", \"to\", \"a\", \"solution\", \"of\", \"compound\", \"9\", \"(\", \"0.164\", \"mmol\", \")\", \"in\", \"methanol\", \"(\", \"5\", \"mL\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"reaction\", \"was\", \"stirred\", \"at\", \"20\", \"o\", \"C\", \"for\", \"0.5\", \"h\", \"until\", \"the\", \"consumption\", \"of\", \"compound\", \"9\", \"was\", \"confirmed\", \"by\", \"TLC\", \"analysis\", \".\"]\n",
      "Word token=> [\"Then\", \",\", \"1\", \"M\", \"HCl\", \"was\", \"added\", \"to\", \"the\", \"reaction\", \"solution\", \"until\", \"pH\", \"=\", \"7\", \".\"]\n",
      "Word token=> [\"Following\", \"this\", \",\", \"the\", \"solvent\", \"was\", \"evaporated\", \"to\", \"generate\", \"the\", \"product\", \"10\", \"as\", \"white\", \"solid\", \",\", \"which\", \"could\", \"be\", \"used\", \"directly\", \"in\", \"the\", \"next\", \"step\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Synthesis\", \"of\", \"α-ketoamides\", \"11\", \"(\", \"general\", \"procedure\", \")\", \".\"]\n",
      "Word token=> [\"Compound\", \"10\", \"was\", \"dissolved\", \"in\", \"CH\", \"2\", \"Cl\", \"2\", \",\", \"then\", \"Dess-Martin\", \"periodinane\", \"(\", \"74\", \"mg\", \",\", \"0.176\", \"mmol\", \")\", \"and\", \"NaHCO\", \"3\", \"(\", \"30\", \"mg\", \",\", \"0.176\", \"mmol\", \")\", \"were\", \"added\", \".\"]\n",
      "Word token=> [\"The\", \"resulting\", \"mixture\", \"was\", \"stirred\", \"at\", \"20\", \"o\", \"C\", \"for\", \"1\", \"h\", \".\"]\n",
      "Word token=> [\"The\", \"mixture\", \"was\", \"concentrated\", \"and\", \"purified\", \"by\", \"column\", \"chromatography\", \"on\", \"silica\", \"gel\", \"(\", \"CH\", \"2\", \"Cl\", \"2\", \"/MeOH\", \"=\", \"20\", \"/\", \"1\", \")\", \"to\", \"give\", \"the\", \"a-ketoamides\", \"11\", \"as\", \"light\", \"yellow\", \"solid\", \"(\", \"52\", \"-\", \"79\", \"%\", \"in\", \"two\", \"steps\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Detailed\", \"results\", \"of\", \"the\", \"variation\", \"of\", \"the\", \"P\", \"1\", \"'\", \"and\", \"P\", \"3\", \"substituents\", \"Synthesis\", \"of\", \"a-ketoamides\", \"11\", \"b\", \"-\", \"11\", \"e\", \"and\", \"11\", \"g\", \"-\", \"11\", \"l\", \"Supplemental\", \"Table\", \"1\", \":\", \"inhibitory\", \"activities\", \"(\", \"IC\", \"50\", \"(\", \"μM\", \")\", \")\", \"of\", \"α-ketoamides\", \"with\", \"P\", \"1\", \"'\", \"and\", \"P\", \"3\", \"modifications\", \"against\", \"viral\", \"proteases\", \"Supplemental\", \"Table\", \"2\", \":\", \"crystallographic\", \"data\", \"for\", \"complexes\", \"between\", \"viral\", \"proteases\", \"and\", \"α-ketoamides\", \"Molecular\", \"formula\", \"strings\", \"and\", \"biological\", \"data\", \"(\", \"CSV\", \")\"]\n",
      "Word token=> [\"Authors\", \"will\", \"release\", \"the\", \"atomic\", \"coordinates\", \"and\", \"experimental\", \"data\", \"upon\", \"article\", \"publication\", \".\"]\n",
      "Word token=> [\"Atomic\", \"coordinates\", \"include\", \"SARS-CoV\", \"M\", \"pro\", \"in\", \"complex\", \"with\", \"compounds\", \"11\", \"a\", \"(\", \"5\", \"N\", \"19\", \")\", \",\", \"11\", \"s\", \"(\", \"5\", \"N\", \"5\", \"O\", \")\", \",\", \"HCoV-NL\", \"63\", \"M\", \"pro\", \"in\", \"complex\", \"with\", \"11\", \"a\", \"(\", \"6\", \"FV\", \"2\", \")\", \",\", \"11\", \"n\", \"(\", \"6\", \"FV\", \"1\", \")\", \",\", \"11\", \"f\", \"(\", \"5\", \"NH\", \"0\", \")\", \",\", \"and\", \"CVB\", \"3\", \"3\", \"C\", \"pro\", \"in\", \"complex\", \"with\", \"(\", \"5\", \"NFS\", \")\", \".\"]\n",
      "Word token=> [\"1\", \"2\", \"3\", \"4\", \"5\", \"6\", \"7\", \"8\", \"9\", \"10\", \"11\", \"12\", \"13\", \"14\", \"15\", \"16\", \"17\", \"18\", \"19\", \"20\", \"21\", \"22\", \"23\", \"24\", \"25\", \"26\", \"27\", \"28\", \"29\", \"30\", \"31\", \"32\", \"33\", \"34\", \"35\", \"36\", \"37\", \"38\", \"39\", \"40\", \"41\", \"42\", \"43\", \"44\", \"45\", \"46\", \"47\", \"48\", \"49\", \"50\", \"51\", \"52\", \"53\", \"54\", \"55\", \"56\", \"57\", \"58\", \"59\", \"60\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.02.10.936898\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"1\", \"2\", \"3\", \"4\", \"5\", \"6\", \"7\", \"8\", \"9\", \"10\", \"11\", \"12\", \"13\", \"14\", \"15\", \"16\", \"17\", \"18\", \"19\", \"20\", \"21\", \"22\", \"23\", \"24\", \"25\", \"26\", \"27\", \"28\", \"29\", \"30\", \"31\", \"32\", \"33\", \"34\", \"35\", \"36\", \"37\", \"38\", \"39\", \"40\", \"41\", \"42\", \"43\", \"44\", \"45\", \"46\", \"47\", \"48\", \"49\", \"50\", \"51\", \"52\", \"53\", \"54\", \"55\", \"56\", \"57\", \"58\", \"59\", \"60\", \"author/funder\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.02.10.936898\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \".\"]\n",
      "Word token=> [\"Severe\", \"Acute\", \"Respiratory\", \"Syndrome\", \"Coronavirus\", \"2\", \"(\", \"SARS-CoV-\", \"2\", \")\", \"or\", \"2019\", \"-Novel\", \"Coronavirus\", \"(\", \"2019\", \"-nCoV\", \")\", \"is\", \"the\", \"pathogen\", \"of\", \"Corona\", \"Virus\", \"Disease\", \"2019\", \"[\", \"1\", \"]\", \".\"]\n",
      "Word token=> [\"Nucleic\", \"acid\", \"detection\", \"of\", \"2019\", \"-nCoV\", \"is\", \"one\", \"of\", \"the\", \"key\", \"indicators\", \"for\", \"clinical\", \"diagnosis\", \".\"]\n",
      "Word token=> [\"However\", \",\", \"the\", \"false\", \"negative\", \"rate\", \"of\", \"2019\", \"-nCoV\", \"nucleic\", \"acid\", \"in\", \"clinical\", \"practice\", \"is\", \"very\", \"high\", \",\", \"and\", \"the\", \"positive\", \"rate\", \"is\", \"only\", \"30\", \"-\", \"50\", \"%\", \"[\", \"2\", \"]\", \".\"]\n",
      "Word token=> [\"False\", \"negative\", \"detection\", \"means\", \"missed\", \"detection\", \",\", \"which\", \"will\", \"lead\", \"to\", \"not\", \"only\", \"the\", \"delayed\", \"diagnosis\", \"of\", \"suspected\", \"patients\", \",\", \"but\", \"also\", \"the\", \"2019\", \"-nCoV\", \"carriers\", \"becoming\", \"the\", \"potential\", \"sources\", \"of\", \"virus\", \"infection\", \".\"]\n",
      "Word token=> [\"Therefore\", \",\", \"it\", \"is\", \"very\", \"urgent\", \"to\", \"improve\", \"the\", \"detection\", \"rate\", \"of\", \"2019\", \"-nCoV\", \"nucleic\", \"acid\", \"clinically\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Currently\", \",\", \"fluorescent\", \"quantitative\", \"RT-PCR\", \"(\", \"qRT-PCR\", \")\", \"technology\", \"is\", \"mainly\", \"used\", \"to\", \"detect\", \"the\", \"nucleic\", \"acid\", \"of\", \"2019\", \"-nCoV\", \".\"]\n",
      "Word token=> [\"The\", \"quality\", \"of\", \"nucleic\", \"acid\", \"template\", \"is\", \"admittedly\", \"one\", \"of\", \"the\", \"key\", \"factors\", \"affecting\", \"the\", \"detection\", \"efficiency\", \".\"]\n",
      "Word token=> [\"According\", \"to\", \"\\\"\", \"The\", \"Laboratory\", \"Technical\", \"Guidelines\", \"for\", \"Detection\", \"2019\", \"-nCoV\", \"(\", \"Fourth\", \"Edition\", \")\", \"\\\"\", \"issued\", \"by\", \"National\", \"Health\", \"and\", \"Commission\", \"of\", \"China\", \"[\", \"3\", \"]\", \"and\", \"\\\"\", \"The\", \"Experts\", \"'\", \"Consensus\", \"on\", \"Nucleic\", \"Acid\", \"Detection\", \"of\", \"2019\", \"-nCoV\", \"\\\"\", \"released\", \"by\", \"Chinese\", \"Society\", \"of\", \"Laboratory\", \"Medicine\", \"[\", \"4\", \"]\", \",\", \"the\", \"human\", \"samples\", \"must\", \"be\", \"placed\", \"in\", \"56\", \"o\", \"C\", \"for\", \"at\", \"least\", \"45\", \"minutes\", \"or\", \"higher\", \"for\", \"shorter\", \"time\", \"to\", \"inactivate\", \"the\", \"viruses\", \"in\", \"order\", \"to\", \"keep\", \"the\", \"inspectors\", \"from\", \"virus\", \"infection\", \"before\", \"the\", \"nucleic\", \"acids\", \"were\", \"isolated\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"However\", \",\", \"2019\", \"-nCoV\", \"is\", \"a\", \"single\", \"stranded\", \"RNA\", \"virus\", \".\"]\n",
      "Word token=> [\"Theoretically\", \",\", \"the\", \"progress\", \"of\", \"killing\", \"the\", \"viruses\", \"with\", \"high\", \"temperature\", \"will\", \"damage\", \"the\", \"integrity\", \"of\", \"their\", \"RNA\", \"genomes\", \"in\", \"the\", \"samples\", \",\", \"thus\", \"improperly\", \"reducing\", \"the\", \"amount\", \"of\", \"target\", \"viral\", \"templates\", \",\", \"and\", \"eventually\", \"leading\", \"to\", \"high\", \"false\", \"negative\", \"rate\", \"of\", \"the\", \"virus\", \"detection\", \".\"]\n",
      "Word token=> [\"In\", \"this\", \"study\", \",\", \"we\", \"employed\", \"porcine\", \"epidemic\", \"diarrhea\", \"virus\", \"(\", \"PEDV\", \")\", \",\", \"a\", \"kind\", \"of\", \"coronavirus\", \",\", \"as\", \"a\", \"model\", \"virus\", \"to\", \"test\", \"the\", \"above\", \"hypothesis\", \"and\", \"our\", \"results\", \"demonstrated\", \"that\", \"the\", \"virus\", \"inactivation\", \"progress\", \"by\", \"high\", \"temperature\", \"damaged\", \"the\", \"detectable\", \"template\", \"of\", \"the\", \"coronavirus\", \"seriously\", \".\"]\n",
      "Word token=> [\"Biological\", \".\"]\n",
      "Word token=> [\"Company\", \",\", \"Ltd.\", \")\", \"or\", \"1\", \"ng\", \"λDNA\", \"(\", \"3010\", \",\", \"Takara\", \",\", \"Japan\", \")\", \"were\", \"suspended\", \"and\", \"mixed\", \"together\", \"using\", \"1.5\", \"mL\", \"of\", \"sample\", \"preservation\", \"medium\", \"(\", \"R\", \"503\", \",\", \"Vamzye\", \",\", \"China\", \")\", \"or\", \"1.5\", \"mL\", \"Hank\", \"'\", \"s\", \"solution\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Nucleic\", \"acid\", \"(\", \"DNA/RNA\", \")\", \"was\", \"extracted\", \"using\", \"a\", \"commercial\", \"kit\", \"(\", \"RC\", \"311\", \"-C\", \"1\", \",\", \"Vazyme\", \")\", \"according\", \"to\", \"the\", \"manufacturer\", \"'\", \"s\", \"protocol\", \".\"]\n",
      "Word token=> [\"Briefly\", \",\", \"500\", \"µL\", \"samples\", \"preserved\", \"in\", \"R\", \"503\", \"was\", \"first\", \"added\", \"into\", \"a\", \"1.5\", \"ml\", \"EP\", \"tube\", \"containing\", \"200\", \"µL\", \"of\", \"absolute\", \"ethanol\", \"and\", \"200\", \"µL\", \"samples\", \"preserved\", \"in\", \"Hank\", \"'\", \"s\", \"solution\", \"was\", \"added\", \"into\", \"a\", \"1.5\", \"ml\", \"EP\", \"tube\", \"containing\", \"500\", \"µL\", \"lysis\", \"buffer\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"mixture\", \"was\", \"vortexed\", \"thoroughly\", \"and\", \"then\", \"transferred\", \"into\", \"a\", \"column\", \"placed\", \"onto\", \"a\", \"2\", \"mL\", \"EP\", \"collection\", \"tube\", \".\"]\n",
      "Word token=> [\"After\", \"centrifuge\", \"for\", \"1\", \"min\", \"at\", \"12,000\", \"×g\", \",\", \"the\", \"column\", \"was\", \"placed\", \"onto\", \"a\", \"fresh\", \"2\", \"mL\", \"EP\", \"collection\", \"tube\", \",\", \"followed\", \"by\", \"adding\", \"600\", \"µL\", \"washing\", \"buffer\", \"and\", \"then\", \"centrifuge\", \"for\", \"30\", \"sec\", \"at\", \"12,000\", \"×g\", \".\"]\n",
      "Word token=> [\"Washing\", \"step\", \"was\", \"repeated\", \"once\", \".\"]\n",
      "Word token=> [\"The\", \"column\", \"was\", \"placed\", \"onto\", \"a\", \"new\", \"2\", \"mL\", \"collection\", \".\"]\n",
      "Word token=> [\"After\", \"centrifuge\", \"for\", \"2\", \"min\", \"at\", \"12,000\", \"×g\", \",\", \"the\", \"column\", \"was\", \"then\", \"placed\", \"onto\", \"a\", \"1.5\", \"ml\", \"collection\", \"tube\", \".\"]\n",
      "Word token=> [\"100\", \"µL\", \"and\", \"40\", \"µL\", \"elution\", \"buffer\", \"was\", \"added\", \"into\", \"the\", \"center\", \"of\", \"the\", \"membrane\", \"of\", \"column\", \"collected\", \"the\", \"sample\", \"prepared\", \"by\", \"R\", \"503\", \"and\", \"Hank\", \"'\", \"s\", \"solution\", \"respectively\", \"and\", \"then\", \"centrifuged\", \"for\", \"1\", \"min\", \"at\", \"12,000\", \"×\", \"g\", \"to\", \"collect\", \"the\", \"nucleic\", \"acid\", \".\"]\n",
      "Word token=> [\"The\", \"nucleic\", \"acid\", \"was\", \"stored\", \"at\", \"-\", \"30\", \"~\", \"-\", \"15\", \"o\", \"C\", \"for\", \"usage\", \"within\", \"short\", \"time\", \"and\", \"at\", \"-\", \"70\", \"o\", \"C\", \"for\", \"long\", \"storage\", \"2.3\", \"Agarose\", \"gel\", \"electrophoresis\", \"5\", \"µL\", \"of\", \"the\", \"extracted\", \"nucleic\", \"acids\", \",\", \"1\", \"µL\", \"of\", \"10\", \"×\", \"DNA\", \"loading\", \"buffer\", \"(\", \"P\", \"022\", \",\", \"Vazyme\", \")\", \"and\", \"4\", \"µL\", \"of\", \"nuclease-free\", \"H\", \"2\", \"O\", \"were\", \"mixed\", \"and\", \"then\", \"vortexed\", \"briefly\", \".\"]\n",
      "Word token=> [\"The\", \"mixture\", \"was\", \"heated\", \"in\", \"PCR\", \"instrument\", \"to\", \"denature\", \"the\", \"nucleic\", \"acid\", \"for\", \"2\", \"min\", \"and\", \"then\", \"subjected\", \"to\", \"1.2\", \"%\", \"agarose\", \"gel\", \"electrophoresis\", \"separation\", \"on\", \"ice\", \"for\", \"7\", \"-\", \"10\", \"min\", \"together\", \"with\", \"the\", \"standard\", \"DNA\", \"marker\", \"(\", \"MD\", \"103\", \",\", \"Vazyme\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.02.20.958785\", \"doi\", \":\", \"bioRxiv\", \"preprint\"]\n",
      "Word token=> [\"Quantitative\", \"Real-Time\", \"PCR\", \"(\", \"qRT-PCR\", \")\", \"was\", \"performed\", \"to\", \"detect\", \"the\", \"PEDV\", \"RNA\", \"and\", \"λ\", \"phage\", \"DNA\", \"using\", \"HiScript\", \"II\", \"One\", \"Step\", \"qRT-PCR\", \"Probe\", \"Kit\", \"(\", \"Q\", \"222\", \"-CN\", \",\", \"Vazyme\", \")\", \"according\", \"to\", \"the\", \"manufacturer\", \"'\", \"s\", \"protocol\", \".\"]\n",
      "Word token=> [\"The\", \"reaction\", \"(\", \"25\", \"µL\", \")\", \"was\", \"assembled\", \"as\", \"follows\", \":\", \"12.5\", \"µL\", \"of\", \"2\", \"×\", \"One\", \"Step\", \"U+\", \"Mix\", \",\", \"1.25\", \"µL\", \"of\", \"One\", \"Step\", \"U+\", \"Enzyme\", \"Mix\", \",\", \"0.5\", \"µL\", \"of\", \"50\", \"×\", \"ROX\", \"Reference\", \"Dye\", \"1\", \",\", \"0.5\", \"µL\", \"of\", \"Gene\", \"Specific\", \"Primer\", \"Forward\", \"(\", \"10\", \"µM\", \")\", \",\", \"0.5\", \"µL\", \"of\", \"Gene\", \"Specific\", \"Primer\", \"Reverse\", \"(\", \"10\", \"µM\", \")\", \",\", \"0.25\", \"µL\", \"of\", \"TaqMan\", \"Probe\", \"(\", \"10\", \"µM\", \")\", \",\", \"and\", \"4.5\", \"µL\", \"nuclease\", \"free\", \"H\", \"2\", \"O\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"reaction\", \"was\", \"carried\", \"out\", \"on\", \"ABI\", \"StepOnePlusTM\", \"using\", \"following\", \"conditions\", \":\"]\n",
      "Word token=> [\"Statistical\", \"significance\", \"was\", \"determined\", \"using\", \"the\", \"paired\", \"two-tailed\", \"t\", \"test\", \".\"]\n",
      "Word token=> [\"A\", \"value\", \"of\", \"p\", \"<\", \"0.05\", \"was\", \"considered\", \"statistically\", \"significant\", \",\", \"and\", \"p\", \"<\", \"0.01\", \"was\", \"considered\", \"statistically\", \"very\", \"significant\", \".\"]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.02.20.958785\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"6\", \"/\", \"16\", \"into\", \"five\", \"groups\", \".\"]\n",
      "Word token=> [\"Three\", \"groups\", \"were\", \"incubated\", \"at\", \"56\", \"o\", \"C\", \"for\", \"30\", \",\", \"45\", \"and\", \"60\", \"minutes\", \",\", \"respectively\", \".\"]\n",
      "Word token=> [\"The\", \"fourth\", \"was\", \"incubated\", \"at\", \"92\", \"o\", \"C\", \"for\", \"5\", \"minutes\", \",\", \"and\", \"the\", \"fifth\", \"group\", \"was\", \"stored\", \"at\", \"4\", \"o\", \"C\", \"as\", \"no\", \"inactivation\", \"control\", \".\"]\n",
      "Word token=> [\"After\", \"the\", \"high-temperature\", \"inactivation\", \"was\", \"completed\", \",\", \"nucleic\", \"acids\", \"(\", \"including\", \"DNA\", \"and\", \"RNA\", \")\", \"were\", \"extracted\", \"by\", \"conventional\", \"methods\", \",\", \"and\", \"subjected\", \"to\", \"run\", \"1.2\", \"%\", \"agarose\", \"gel\", \"electrophoresis\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"To\", \"understand\", \"the\", \"effect\", \"of\", \"the\", \"virus\", \"inactivation\", \"progress\", \"with\", \"the\", \"temperature\", \"higher\", \"than\", \"56\", \"o\", \"C\", \"on\", \"the\", \"integrity\", \"of\", \"viral\", \"nucleic\", \"acid\", \",\", \"we\", \"quantitated\", \"the\", \"amount\", \"of\", \"Table\", \"S\", \"1\", \")\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"The\", \"results\", \"suggest\", \"that\", \"only\", \"50.11\", \"%\", \"of\", \"the\", \"detectable\", \"viral\", \"templates\", \"left\", \"after\", \"the\", \"inactivation\", \"progress\", \"of\", \"incubation\", \"at\", \"56\", \"o\", \"C\", \"for\", \"30\", \"minutes\", \"and\", \"only\", \"3.36\", \"%\", \"of\", \"the\", \"detectable\", \"viral\", \"templates\", \"left\", \"after\", \"the\", \"inactivation\", \"progress\", \"of\", \"incubation\", \"at\", \"92\", \"o\", \"C\", \"for\", \"5\", \"minutes\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"To\", \"investigate\", \"whether\", \"any\", \"solutions\", \"are\", \"able\", \"to\", \"protect\", \"viral\", \"RNA\", \"integrity\", \"from\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.02.20.958785\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"7\", \"/\", \"16\", \"disrupting\", \"during\", \"the\", \"inactivation\", \"progress\", \"with\", \"high\", \"temperature\", \",\", \"R\", \"503\", \"(\", \"Vazyme\", \",\", \"China\", \")\", \"was\", \"used\", \"to\", \"prepare\", \"the\", \"samples\", \"instead\", \"of\", \"Hank\", \"'\", \"s\", \"solution\", \".\"]\n",
      "Word token=> [\"We\", \"then\", \"performed\", \"the\", \"same\", \"viral\", \"inactivation\", \"experiments\", \"as\", \"using\", \"Hank\", \"'\", \"s\", \"solution\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Compared\", \"with\", \"the\", \"control\", \"group\", \"that\", \"the\", \"sample\", \"was\", \"stored\", \"at\", \"4\", \"o\", \"C\", \",\", \"both\", \"DNA\", \"and\", \"RNA\", \"in\", \"the\", \"samples\", \"incubated\", \"at\", \"56\", \"o\", \"C\", \"for\", \"30\", \"-\", \"60\", \"minutes\", \"also\", \"displayed\", \"obvious\", \"degradation\", \",\", \"among\", \"which\", \"the\", \"28\", \"S\", \"and\", \"18\", \"S\", \"bands\", \"of\", \"total\", \"RNA\", \"of\", \"human\", \"cells\", \"were\", \"obviously\", \"smeared\", \",\", \"and\", \"the\", \"large\", \"bands\", \"of\", \"genomic\", \"DNA\", \"became\", \"weaker\", \"while\", \"there\", \"were\", \"almost\", \"no\", \"visible\", \"genomic\", \"DNA\", \"and\", \"28\", \"S\", \"RNA\", \"bands\", \"in\", \"the\", \"sample\", \"incubated\", \"at\", \"92\", \"o\", \"C\", \"for\", \"5\", \"minutes\", \"(\", \"Fig\", \"1\", \"D\", \")\", \".\"]\n",
      "Word token=> [\"However\", \",\", \"the\", \"degradation\", \"of\", \"nucleic\", \"acid\", \"during\", \"the\", \"inactivation\", \"progress\", \"was\", \"not\", \"as\", \"serious\", \"as\", \"that\", \"of\", \"the\", \"samples\", \"prepared\", \"by\", \"Hank\", \"'\", \"s\", \"solution\", \".\"]\n",
      "Word token=> [\"The\", \"results\", \"suggest\", \"that\", \"R\", \"503\", \"should\", \"have\", \"a\", \"significant\", \"protective\", \"effect\", \"of\", \"both\", \"total\", \"RNA\", \"and\", \"genomic\", \"DNA\", \"from\", \"degradation\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"To\", \"determine\", \"the\", \"effect\", \"of\", \"the\", \"inactivation\", \"progress\", \"with\", \"temperature\", \"higher\", \"than\", \"56\", \"o\", \"C\", \"on\", \"the\", \"integrity\", \"of\", \"viral\", \"nucleic\", \"acid\", \"in\", \"the\", \"samples\", \"prepared\", \"with\", \"R\", \"503\", \",\", \"we\", \"examined\", \"the\", \"amount\", \"of\", \"porcine\", \"PEDV\", \"coronavirus\", \"by\", \"qRT-PCR\", \"as\", \"above\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Compared\", \"with\", \"the\", \"CT\", \"(\", \"31.64\", \"±\", \"0.10\", \")\", \"of\", \"the\", \"control\", \"group\", \"that\", \"the\", \"sample\", \"was\", \"stored\", \"at\", \"4\", \"o\", \"C\", \",\", \"the\", \"mean\", \"value\", \"of\", \"CT\", \"(\", \"31.31\", \"±\", \"0.17\", \")\", \"of\", \"the\", \"group\", \"incubated\", \"at\", \"56\", \"o\", \"C\", \"for\", \"30\", \"minutes\", \"had\", \"no\", \"significant\", \"change\", \".\"]\n",
      "Word token=> [\"The\", \"difference\", \"of\", \"CTs\", \"between\", \"the\", \"two\", \"groups\", \"was\", \"less\", \"than\", \"0.5\", \"(\", \"Fig.\", \"1\", \"E\", \";\", \"Figure\", \"S\", \"1\", \"B\", \";\", \"Table\", \"S\", \"1\", \")\", \".\"]\n",
      "Word token=> [\"Moreover\", \",\", \"the\", \"mean\", \"value\", \"of\", \"CT\", \"(\", \"31.43\", \"±\", \"0.08\", \")\", \"in\", \"the\", \"group\", \"incubated\", \"at\", \"92\", \"o\", \"C\", \"for\", \"5\", \"minutes\", \"was\", \"also\", \"not\", \"changed\", \"significantly\", \".\"]\n",
      "Word token=> [\"The\", \"difference\", \"of\", \"CTs\", \"between\", \"the\", \"two\", \"groups\", \"was\", \"less\", \"than\", \"0.5\", \"either\", \"(\", \"Fig.\", \"1\", \"F\", \";\", \"Fig.\", \"S\", \"1\", \"B\", \";\", \"Table\", \"S\", \"1\", \")\", \".\"]\n",
      "Word token=> [\"These\", \"results\", \"showed\", \"that\", \"the\", \"inactivation\", \"progress\", \"with\", \"the\", \"temperature\", \"higher\", \"than\", \"56\", \"o\", \"C\", \"did\", \"not\", \"significantly\", \"affect\", \"the\", \"detectable\", \"template\", \"of\", \"viral\", \"nucleic\", \"acid\", \"when\", \"it\", \"was\", \"prepared\", \"with\", \"R\", \"503\", \",\", \"suggesting\", \"that\", \"the\", \"fragmented\", \"viral\", \"RNA\", \"could\", \"still\", \"be\", \"good\", \"for\", \"using\", \"as\", \"a\", \"template\", \"for\", \"qRT-PCR\", \"even\", \"if\", \"the\", \"viral\", \"nucleic\", \"acid\", \"was\", \"degraded\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"In\", \"order\", \"to\", \"understand\", \"the\", \"advantage\", \"of\", \"R\", \"503\", \"over\", \"common\", \"isotonic\", \"solution\", \"in\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.02.20.958785\", \"doi\", \":\", \"bioRxiv\", \"preprint\"]\n",
      "Word token=> [\"protecting\", \"the\", \"detectable\", \"amount\", \"of\", \"coronavirus\", \"RNA\", \"from\", \"disrupting\", \",\", \"we\", \"first\", \"compared\", \"the\", \"difference\", \"of\", \"the\", \"detectable\", \"amount\", \"of\", \"coronavirus\", \"RNA\", \"between\", \"the\", \"two\", \"samples\", \"that\", \"are\", \"prepared\", \"with\", \"R\", \"503\", \"and\", \"Hank\", \"'\", \"s\", \"solution\", \"and\", \"stored\", \"at\", \"4\", \"o\", \"C.\", \"The\", \"results\", \"showed\", \"that\", \"the\", \"detectable\", \"amount\", \"of\", \"the\", \"viral\", \"RNA\", \"in\", \"the\", \"two\", \"prepared\", \"solutions\", \"was\", \"similar\", \"(\", \"Fig.\", \"1\", \"G\", \",\", \"Table\", \"S\", \"1\", \")\", \",\", \"and\", \"the\", \"difference\", \"between\", \"the\", \"mean\", \"values\", \"of\", \"their\", \"CT\", \"was\", \"less\", \"than\", \"0.5\", \"(\", \"31.64\", \"±\", \"0.10\", \"vs\", \"31.95\", \"±\", \"0.10\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"results\", \"suggest\", \"that\", \"there\", \"is\", \"no\", \"significant\", \"difference\", \"in\", \"the\", \"amount\", \"of\", \"detectable\", \"viral\", \"RNA\", \"templates\", \"between\", \"the\", \"samples\", \"prepared\", \"by\", \"either\", \"R\", \"503\", \"or\", \"Hank\", \"'\", \"s\", \"solutions\", \"that\", \"were\", \"stored\", \"at\", \"Table\", \"S\", \"1\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"results\", \"suggest\", \"that\", \"the\", \"detectable\", \"templates\", \"of\", \"viruses\", \"in\", \"the\", \"samples\", \"prepared\", \"by\", \"Hank\", \"'\", \"s\", \"solution\", \"was\", \"only\", \"32.30\", \"%\", \"of\", \"R\", \"503\", \"solution\", \"after\", \"the\", \"samples\", \"were\", \"inactivated\", \"at\", \"56\", \"o\", \"C\", \"for\", \"30\", \"minutes\", \"to\", \"kill\", \"the\", \"virus\", \".\"]\n",
      "Word token=> [\"Furthermore\", \",\", \"after\", \"inactivation\", \"at\", \"92\", \"o\", \"C\", \"for\", \"5\", \"minutes\", \",\", \"the\", \"CT\", \"of\", \"detected\", \"PEDV\", \"virus\", \"in\", \"the\", \"two\", \"preservation\", \"solutions\", \"were\", \"31.43\", \"±\", \"0.08\", \"(\", \"R\", \"503\", \")\", \"and\", \"36.84\", \"±\", \"0.37\", \"(\", \"Hank\", \"'\", \"s\", \")\", \",\", \"respectively\", \".\"]\n",
      "Word token=> [\"The\", \"mean\", \"value\", \"of\", \"CT\", \"of\", \"detected\", \"virus\", \"in\", \"the\", \"samples\", \"prepared\", \"by\", \"Hank\", \"'\", \"s\", \"solution\", \"was\", \"5.4125\", \"higher\", \"than\", \"that\", \"prepared\", \"by\", \"R\", \"503\", \"(\", \"P\", \"<\", \"0.0001\", \";\", \"Figure\", \"1\", \"I\", \",\", \"Table\", \"S\", \"1\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"results\", \"reveal\", \"the\", \"detectable\", \"amount\", \"of\", \"virus\", \"in\", \"the\", \"samples\", \"prepared\", \"by\", \"Hank\", \"'\", \"s\", \"solution\", \"was\", \"only\", \"2.35\", \"%\", \"of\", \"R\", \"503\", \"solution\", \"after\", \"the\", \"samples\", \"were\", \"inactivated\", \"at\", \"92\", \"o\", \"C\", \"for\", \"5\", \"minutes\", \"to\", \"kill\", \"the\", \"virus\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"In\", \"addition\", \"to\", \"the\", \"disruptive\", \"effect\", \"of\", \"the\", \"virus\", \"inactivation\", \"with\", \"high\", \"temperature\", \"on\", \"the\", \"detection\", \"of\", \"RNA\", \"virus\", \",\", \"the\", \"detectable\", \"amount\", \"of\", \"the\", \"viral\", \"DNA\", \"(\", \"λ\", \"phage\", \")\", \"in\", \"the\", \"samples\", \"was\", \"also\", \"significantly\", \"affected\", \".\"]\n",
      "Word token=> [\"qPCR\", \"results\", \"(\", \"Fig.\", \"S\", \"1\", \"C\", \",\", \"Table\", \"S\", \"1\", \")\", \"showed\", \"that\", \"mean\", \"value\", \"of\", \"CT\", \"(\", \"23.55\", \"±\", \"0.20\", \")\", \"of\", \"detectable\", \"virus\", \"in\", \"the\", \"samples\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.02.20.958785\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"9\", \"/\", \"16\", \"incubated\", \"at\", \"56\", \"o\", \"C\", \"for\", \"30\", \"minutes\", \"was\", \"increased\", \"by\", \"0.5230\", \"(\", \"P\", \"<\", \"0.01\", \")\", \"compared\", \"with\", \"the\", \"CT\", \"(\", \"23.02\", \"±\", \"0.15\", \")\", \"of\", \"the\", \"samples\", \"stored\", \"at\", \"4\", \"o\", \"C.\", \"The\", \"mean\", \"value\", \"of\", \"CT\", \"(\", \"25.58\", \"±\", \"0.21\", \")\", \"was\", \"increased\", \"by\", \"2.5572\", \"(\", \"P\", \"<\", \"0.0001\", \")\", \"in\", \"the\", \"sample\", \"incubated\", \"at\", \"92\", \"o\", \"C\", \"for\", \"5\", \"minutes\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Furthermore\", \",\", \"we\", \"examined\", \"the\", \"amount\", \"of\", \"phage\", \"DNA\", \"in\", \"the\", \"sample\", \"prepared\", \"with\", \"R\", \"503\", \"after\", \"high\", \"temperature\", \"treatment\", \"by\", \"qPCR\", \".\"]\n",
      "Word token=> [\"The\", \"results\", \"(\", \"Fig.\", \"S\", \"1\", \"D\", \",\", \"Table\", \"S\", \"1\", \")\", \"showed\", \"that\", \"the\", \"mean\", \"value\", \"of\", \"CT\", \"(\", \"21.71\", \"±\", \"0.04\", \")\", \"in\", \"the\", \"sample\", \"prepared\", \"by\", \"R\", \"503\", \"and\", \"incubated\", \"at\", \"56\", \"o\", \"C\", \"for\", \"30\", \"minutes\", \"had\", \"no\", \"significant\", \"change\", \"compared\", \"with\", \"that\", \"in\", \"the\", \"control\", \"group\", \"(\", \"21.61\", \"±\", \"0.06\", \")\", \"(\", \"the\", \"difference\", \"between\", \"the\", \"two\", \"was\", \"less\", \"than\", \"0.5\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"mean\", \"value\", \"of\", \"CT\", \"(\", \"22.90\", \"±\", \"0.26\", \")\", \"was\", \"increased\", \"by\", \"1.2880\", \"in\", \"the\", \"sample\", \"that\", \"was\", \"inactivated\", \"at\", \"92\", \"o\", \"C\", \"for\", \"5\", \"minutes\", \",\", \"suggesting\", \"that\", \"92\", \"o\", \"C\", \"treatment\", \"significantly\", \"reduced\", \"the\", \"detectable\", \"amount\", \"of\", \"DNA\", \"viruses\", \"in\", \"sample\", \"even\", \"prepared\", \"by\", \"R\", \"503\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> String[]\n",
      "Word token=> [\"In\", \"conclusion\", \",\", \"our\", \"results\", \"show\", \"that\", \"virus\", \"inactivation\", \"with\", \"temperature\", \"higher\", \"than\", \"56\", \"o\", \"C\", \"result\", \"in\", \"the\", \"degradation\", \"of\", \"viral\", \"nucleic\", \"acids\", \"seriously\", \",\", \"which\", \"will\", \"lead\", \"to\", \"the\", \"artificial\", \"shortage\", \"of\", \"detectable\", \"templates\", \"of\", \"viral\", \"nucleic\", \"acids\", \"in\", \"samples\", \",\", \"and\", \"finally\", \"lead\", \"to\", \"false\", \"negative\", \"of\", \"clinical\", \"detection\", \"in\", \"some\", \"samples\", \".\"]\n",
      "Word token=> [\"Preserved\", \"with\", \"the\", \"optimized\", \"solution\", \"such\", \"as\", \"R\", \"503\", \",\", \"the\", \"detectable\", \"templates\", \"of\", \"viral\", \"nucleic\", \"acids\", \"can\", \"be\", \"kept\", \"unchanged\", \"after\", \"the\", \"samples\", \"were\", \"incubated\", \"at\", \"56\", \"o\", \"C\", \"or\", \"higher\", \"for\", \"killing\", \"the\", \"viruses\", \"although\", \"the\", \"integrity\", \"of\", \"cellular\", \"nucleic\", \"acids\", \"were\", \"obviously\", \"disrupted\", \".\"]\n",
      "Word token=> [\"The\", \"results\", \"suggest\", \"that\", \"if\", \"the\", \"samples\", \"were\", \"prepared\", \"with\", \"an\", \"optimal\", \"solution\", \",\", \"the\", \"detectable\", \"numbers\", \"of\", \"viral\", \"templates\", \"would\", \"be\", \"unchanged\", \"because\", \"virus\", \"inactivation\", \"progress\", \"with\", \"high\", \"temperature\", \"does\", \"not\", \"erase\", \"all\", \"the\", \"RNA\", \"but\", \"leave\", \"small\", \"fragments\", \"of\", \"nucleic\", \"acids\", \"that\", \"are\", \"good\", \"enough\", \"for\", \"using\", \"as\", \"the\", \"template\", \"for\", \"qRT-PCR\", \"detection\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"If\", \"it\", \"is\", \"true\", \"for\", \"2019\", \"-nCoV\", \",\", \"preparing\", \"the\", \"human\", \"samples\", \"with\", \"the\", \"optimal\", \"solution\", \"can\", \"not\", \"only\", \"keep\", \"the\", \"clinical\", \"inspectors\", \"from\", \"virus\", \"infection\", \"by\", \"inactivating\", \"the\", \"virus\", \"using\", \"56\", \"o\", \"C\", \"or\", \"higher\", \"as\", \"normal\", \",\", \"but\", \"also\", \"improve\", \"the\", \"detection\", \"sensitivity\", \"of\", \"viral\", \"nucleic\", \"acid\", \"in\", \"the\", \"sample\", \",\", \"leading\", \"to\", \"avoiding\", \"or\", \"at\", \"least\", \"reducing\", \"false\", \"negative\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.02.20.958785\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"10\", \"/\", \"16\", \"results\", \"of\", \"detection\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Recently\", \",\", \"Chen\", \"reported\", \"that\", \"the\", \"inactivation\", \"by\", \"incubating\", \"the\", \"sample\", \"with\", \"56\", \"o\", \"C\", \"for\", \"30\", \"min\", \"to\", \"kill\", \"the\", \"virus\", \"had\", \"no\", \"significant\", \"effect\", \"on\", \"the\", \"detection\", \"of\", \"2019\", \"-nCoV\", \"by\", \"qRT-PCR\", \"[\", \"5\", \"]\", \".\"]\n",
      "Word token=> [\"However\", \",\", \"combining\", \"the\", \"limitations\", \"such\", \"as\", \"only\", \"two\", \"samples\", \"tested\", \"and\", \"higher\", \"amount\", \"of\", \"virus\", \"in\", \"the\", \"two\", \"samples\", \"as\", \"the\", \"authors\", \"discussed\", \"in\", \"their\", \"researches\", \",\", \"and\", \"our\", \"experimental\", \"evidences\", \"that\", \"the\", \"virus\", \"inactivation\", \"by\", \"high\", \"temperature\", \"did\", \"reduce\", \"the\", \"detectable\", \"amount\", \"of\", \"the\", \"porcine\", \"coronavirus\", \"template\", \"seriously\", \",\", \"it\", \"is\", \"highly\", \"recommended\", \"to\", \"carry\", \"out\", \"systematic\", \"investigation\", \"on\", \"the\", \"impact\", \"of\", \"high\", \"temperature\", \"inactivation\", \"on\", \"the\", \"integrity\", \"of\", \"2019\", \"-nCoV\", \"nucleic\", \"acids\", \"and\", \"develop\", \"a\", \"sample\", \"preservation\", \"solution\", \"to\", \"protect\", \"the\", \"detectable\", \"templates\", \"of\", \"2019\", \"-nCoV\", \"nucleic\", \"acids\", \"from\", \"high\", \"temperature\", \"inactivation\", \"damage\", \".\"]\n",
      "Word token=> [\"Table\", \"S\", \"1\", \".\"]\n",
      "Word token=> [\"Figure\", \"S\", \"1\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.02.20.958785\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"15\", \"/\", \"16\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.02.20.958785\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \".\"]\n",
      "Word token=> [\"HCoV-HKU\", \"1\", \"replication\", \"in\", \"the\", \"presence\", \"of\", \"both\", \"inhibitors\", \"(\", \"Fig.\", \"2\", \"B\", \")\", \".\"]\n",
      "Word token=> [\"All\", \"inhibitors\", \"were\", \"used\", \"151\", \"at\", \"non-toxic\", \"concentrations\", \"(\", \"Fig.\", \"2\", \"C\", \")\", \".\"]\n",
      "Word token=> [\"152\", \"The\", \"experiments\", \"conducted\", \"so\", \"far\", \"suggested\", \"that\", \"KLK\", \"13\", \"is\", \"required\", \"for\", \"virus\", \"infection\", \".\"]\n",
      "Word token=> [\"153\", \"However\", \",\", \"one\", \"may\", \"question\", \"the\", \"specificity\", \"of\", \"the\", \"KLK\", \"13\", \"protease\", \"inhibitors\", \".\"]\n",
      "Word token=> [\"To\", \"ensure\", \"that\", \"154\", \"KLK\", \"13\", \"is\", \"indeed\", \"the\", \"priming\", \"enzyme\", \"during\", \"HCoV-HKU\", \"1\", \"infection\", \",\", \"we\", \"developed\", \"HAE\", \"155\", \"cultures\", \"by\", \"transforming\", \"cells\", \"with\", \"lentiviral\", \"vectors\", \"encoding\", \"shRNAs\", \"targeting\", \"KLK\", \"13\", \"156\", \"mRNA\", \".\"]\n",
      "Word token=> [\"We\", \"then\", \"confirmed\", \"that\", \"the\", \"expression\", \"of\", \"the\", \"protease\", \"was\", \"silenced\", \"(\", \"HAE\", \"_\", \"shKLK\", \"13\", \")\", \".\"]\n",
      "Word token=> [\"(\", \"Fig.\", \"3\", \"A\", \")\", \".\"]\n",
      "Word token=> [\"Importantly\", \",\", \"HAE\", \"_\", \"shKLK\", \"13\", \"cells\", \"continued\", \"to\", \"differentiate\", \"and\", \"formed\", \"162\", \"pseudostratified\", \"cultures\", \"(\", \"Fig.\", \"3\", \"B\", \")\", \".\"]\n",
      "Word token=> [\"Next\", \",\", \"we\", \"infected\", \"HAE\", \"_\", \"ctrl\", \",\", \"HAE\", \"_\", \"GFP\", \",\", \"HAE\", \"_\", \"vector\", \",\", \"and\", \"163\", \"HAE\", \"_\", \"shKLK\", \"13\", \"with\", \"HCoV-HKU\", \"1\", \"(\", \"10\", \"6\", \"RNA\", \"copies\", \"per\", \"ml\", \")\", \"and\", \"incubated\", \"them\", \"for\", \"2\", \"h\", \"at\", \"164\", \"32\", \"°\", \"C\", \"with\", \"the\", \"viral\", \"stock\", \"solution\", \".\"]\n",
      "Word token=> [\"Cultures\", \"were\", \"maintained\", \"at\", \"32\", \"°\", \"C\", \"for\", \"5\", \"days\", \"at\", \"an\", \"air-liquid\", \"165\", \"interface\", \".\"]\n",
      "Word token=> [\"Apical\", \"washes\", \"were\", \"collected\", \",\", \"and\", \"virus\", \"yield\", \"was\", \"determined\", \"by\", \"RT-qPCR\", \".\"]\n",
      "Word token=> [\"We\", \"166\", \"found\", \"that\", \",\", \"in\", \"contrast\", \"to\", \"that\", \"in\", \"control\", \"cultures\", \",\", \"replication\", \"of\", \"virus\", \"in\", \"HAE\", \"_\", \"shKLK\", \"13\", \"was\", \"167\", \"abolished\", \"(\", \"Fig.\", \"3\", \"C\", \")\", \".\"]\n",
      "Word token=> [\"Overall\", \",\", \"these\", \"data\", \"suggest\", \"that\", \"silencing\", \"the\", \"KLK\", \"13\", \"gene\", \"in\", \"HAE\", \"inhibits\", \"168\", \"virus\", \"infection\", \",\", \"indicating\", \"that\", \"KLK\", \"13\", \"is\", \"necessary\", \"for\", \"HCoV-HKU\", \"1\", \"infection\", \".\"]\n",
      "Word token=> [\"171\", \"We\", \"determined\", \"that\", \"KLK\", \"13\", \"is\", \"essential\", \"for\", \"efficient\", \"HCoV-HKU\", \"1\", \"infection\", \"in\", \"HAE\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.01.971499\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"proteases\", \"my\", \"render\", \"RD\", \"cells\", \"permissive\", \",\", \"we\", \"generated\", \"RD\", \"cells\", \"expressing\", \"human\", \"KLK\", \"13\", \"or\", \"176\", \"TMPRSS\", \"2\", \"proteases\", \".\"]\n",
      "Word token=> [\"RD\", \"cells\", \"were\", \"transduced\", \"with\", \"lentiviral\", \"vectors\", \"harboring\", \"the\", \"KLK\", \"13\", \"177\", \"gene\", \"(\", \"RD\", \"_\", \"KLK\", \"13\", \")\", \",\", \"control\", \"vector\", \"(\", \"RD\", \"_\", \"ctrl\", \")\", \",\", \"or\", \"TMPRSS\", \"2\", \"(\", \"RD\", \"_\", \"TMPRSS\", \"2\", \")\", \".\"]\n",
      "Word token=> [\"Due\", \"to\", \"the\", \"lack\", \"178\", \"of\", \"KLK\", \"13\", \"specific\", \"antibodies\", \",\", \"we\", \"verified\", \"its\", \"presence\", \"based\", \"on\", \"RT-PCR\", \"(\", \"Fig.\", \"4\", \"A\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"179\", \"presence\", \"of\", \"TMPRSS\", \"2\", \"in\", \"RD\", \"_\", \"TMPRSS\", \"2\", \"cells\", \"was\", \"confirmed\", \"using\", \"Western\", \"blot\", \".\"]\n",
      "Word token=> [\"The\", \"180\", \"TMPRSS\", \"2\", \"band\", \"in\", \"RD\", \"cells\", \"was\", \"observed\", \"at\", \"25\", \"kDa\", \",\", \"which\", \"corresponds\", \"to\", \"one\", \"of\", \"the\", \"naturally\", \"181\", \"occurring\", \"splicing\", \"variants\", \"(\", \"Fig.\", \"4\", \"B\", \")\", \".\"]\n",
      "Word token=> [\"Subsequently\", \",\", \"we\", \"transduced\", \"RD\", \"_\", \"ctrl\", \",\", \"RD\", \"_\", \"KLK\", \"13\", \"and\", \"pseudoviruses\", \",\", \"compared\", \"to\", \"ΔEnv\", \",\", \"which\", \"was\", \"completely\", \"abolished\", \"in\", \"the\", \"presence\", \"of\", \"KLK\", \"13\", \"193\", \"inhibitor\", \"(\", \"Fig\", \"4\", \"D\", \")\", \".\"]\n",
      "Word token=> [\"Overall\", \",\", \"these\", \"data\", \"demonstrated\", \"that\", \"KLK\", \"13\", \"activity\", \"drives\", \"HCoV-HKU\", \"1\", \"194\", \"entry\", \"into\", \"cells\", \".\"]\n",
      "Word token=> [\"195\", \"196\", \"KLK\", \"13\", \"enables\", \"the\", \"replication\", \"of\", \"HCoV-HKU\", \"1\", \"in\", \"RD\", \"cells\", \"197\", \"Obtained\", \"results\", \"showed\", \"that\", \"KLK\", \"13\", \"expression\", \"on\", \"RD\", \"cells\", \"was\", \"sufficient\", \"for\", \"HCoV-\", \"198\", \"HKU\", \"1\", \"pseudovirus\", \"entry\", \".\"]\n",
      "Word token=> [\"Here\", \",\", \"we\", \"aimed\", \"to\", \"test\", \"whether\", \"KLK\", \"13\", \"presence\", \"renders\", \"RD\", \"cells\", \"199\", \"permissive\", \"for\", \"HCoV-HKU\", \"1\", \"infection\", \".\"]\n",
      "Word token=> [\"For\", \"this\", \",\", \"we\", \"infected\", \"RD\", \"_\", \"ctrl\", \"and\", \"RD\", \"_\", \"KLK\", \"13\", \"cells\", \"200\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.01.971499\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"with\", \"HCoV-HKU\", \"1\", \"(\", \"10\", \"8\", \"RNA\", \"copies\", \"per\", \"ml\", \")\", \"and\", \"incubated\", \"the\", \"culture\", \"for\", \"7\", \"days\", \"at\", \"32\", \"°\", \"C\", \"in\", \"the\", \"201\", \"presence\", \"or\", \"absence\", \"of\", \"a\", \"KLK\", \"13\", \"inhibitor\", \"(\", \"10\", \"µM\", \")\", \"or\", \"DMSO\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"Next\", \",\", \"cellular\", \"RNA\", \"was\", \"isolated\", \"202\", \"and\", \"the\", \"presence\", \"of\", \"HCoV-HKU\", \"1\", \"N\", \"subgenomic\", \"mRNA\", \"(\", \"N\", \"sg\", \"mRNA\", \")\", \",\", \"which\", \"is\", \"considered\", \"203\", \"to\", \"be\", \"a\", \"hallmark\", \"of\", \"coronaviral\", \"infection\", \",\", \"was\", \"assessed\", \"(\", \"14\", \")\", \".\"]\n",
      "Word token=> [\"sg\", \"mRNA\", \"appeared\", \"in\", \"RD\", \"_\", \"KLK\", \"13\", \"204\", \"cells\", \",\", \"while\", \"no\", \"signal\", \"was\", \"detected\", \"in\", \"cultures\", \"supplemented\", \"with\", \"the\", \"KLK\", \"13\", \"inhibitor\", \"nor\", \"in\", \"205\", \"RD\", \"_\", \"ctrl\", \"cells\", \"(\", \"Fig.\", \"5\", \"A\", \")\", \".\"]\n",
      "Word token=> [\"206\", \"To\", \"further\", \"confirm\", \"the\", \"role\", \"of\", \"KLK\", \"13\", \"during\", \"HCoV-HKU\", \"1\", \"infection\", \"RD\", \"HCoV-HKU\", \"1\", \"infection\", \"was\", \"analyzed\", \"by\", \"means\", \"of\", \"N\", \"sg\", \"mRNA\", \"detection\", \".\"]\n",
      "Word token=> [\"Again\", \",\", \"we\", \"found\", \"that\", \"212\", \"N\", \"sg\", \"mRNA\", \"was\", \"produced\", \"only\", \"in\", \"the\", \"presence\", \"of\", \"KLK\", \"13\", \"(\", \"Fig.\", \"5\", \"B\", \")\", \".\"]\n",
      "Word token=> [\"Further\", \",\", \"we\", \"passaged\", \"in\", \"the\", \"presence\", \"of\", \"KLK\", \"13\", \"(\", \"Fig.\", \"5\", \"B\", \")\", \".\"]\n",
      "Word token=> [\"However\", \",\", \"we\", \"observed\", \"no\", \"cytopathic\", \"effects\", \"(\", \"CPEs\", \")\", \",\", \"and\", \"218\", \"replication\", \"levels\", \"were\", \"very\", \"low\", \"(\", \"no\", \"significant\", \"increase\", \"over\", \"control\", \"levels\", \"on\", \"RT-qPCR\", \";\", \"data\", \"219\", \"not\", \"shown\", \")\", \".\"]\n",
      "Word token=> [\"To\", \"further\", \"test\", \"the\", \"effect\", \"of\", \"KLK\", \"13\", \"on\", \"replication\", \"of\", \"HCoV-HKU\", \"1\", \"in\", \"RD\", \"cells\", \",\", \"the\", \"220\", \"virus\", \"stock\", \"was\", \"incubated\", \"with\", \"purified\", \"KLK\", \"13\", \"(\", \"200\", \"nM\", \")\", \"and\", \"incubated\", \"in\", \"the\", \"presence\", \"or\", \"221\", \"absence\", \"of\", \"a\", \"KLK\", \"13\", \"inhibitor\", \"(\", \"10\", \"µM\", \")\", \"or\", \"DMSO\", \".\"]\n",
      "Word token=> [\"After\", \"2\", \"h\", \"at\", \"32\", \"°\", \"C\", \"pre-treated\", \"virus\", \"stock\", \"was\", \"222\", \"diluted\", \"in\", \"media\", \"as\", \"described\", \"above\", \"and\", \"overlaid\", \"on\", \"RD\", \"cells\", \".\"]\n",
      "Word token=> [\"After\", \"7\", \"days\", \"at\", \"32\", \"°\", \"C\", \"we\", \"evaluated\", \"223\", \"the\", \"presence\", \"of\", \"the\", \"HCoV-HKU\", \"1\", \"N\", \"sg\", \"mRNA\", \".\"]\n",
      "Word token=> [\"The\", \"virus\", \"replicated\", \"only\", \"after\", \"treatment\", \"with\", \"224\", \"KLK\", \"13\", \",\", \"and\", \"supplementation\", \"with\", \"the\", \"inhibitor\", \"blocked\", \"this\", \"effect\", \"infection\", \"(\", \"Fig.\", \"5\", \"C\", \")\", \".\"]\n",
      "Word token=> [\"225\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.01.971499\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"KLK\", \"13\", \"primes\", \"the\", \"HCoV-HKU\", \"1\", \"S\", \"protein\", \"227\", \"Expression\", \"of\", \"KLK\", \"13\", \"by\", \"cells\", \"previously\", \"resistant\", \"to\", \"HCoV-HKU\", \"1\", \"renders\", \"them\", \"228\", \"susceptible\", \";\", \"therefore\", \",\", \"we\", \"asked\", \"whether\", \"this\", \"is\", \"due\", \"to\", \"proteolytic\", \"activation\", \"of\", \"the\", \"S\", \"protein\", \".\"]\n",
      "Word token=> [\"229\", \"We\", \"tested\", \"this\", \"using\", \"the\", \"CleavEx\", \"method\", \",\", \"in\", \"which\", \"a\", \"peptide\", \"of\", \"interest\", \"is\", \"exposed\", \"in\", \"the\", \"KLK\", \"13\", \".\"]\n",
      "Word token=> [\"Subsequently\", \",\", \"proteins\", \"were\", \"resolved\", \"by\", \"SDS-PAGE\", \"and\", \"detected\", \"by\", \"western\", \"blotting\", \"238\", \"with\", \"antibodies\", \"specific\", \"for\", \"His-tagged\", \"proteins\", \".\"]\n",
      "Word token=> [\"The\", \"analysis\", \"showed\", \"that\", \",\", \"in\", \"the\", \"presence\", \"of\", \"239\", \"500\", \"nM\", \"KLK\", \"13\", \",\", \"the\", \"CleavEx\", \"protein\", \"harboring\", \"the\", \"S\", \"1\", \"/S\", \"2\", \"cleavage\", \"site\", \"was\", \"degraded\", \";\", \"however\", \",\", \"240\", \"the\", \"CleavEx\", \"protein\", \"harboring\", \"the\", \"S\", \"2\", \"/S\", \"2\", \"'\", \"site\", \"remained\", \"intact\", \".\"]\n",
      "Word token=> [\"Because\", \"KLKs\", \"are\", \"produced\", \"as\", \"241\", \"pro-forms\", \"that\", \"undergo\", \"self-activation\", \",\", \"an\", \"additional\", \"band\", \"of\", \"His-tagged\", \"purified\", \"pro-KLK\", \"13\", \"242\", \"(\", \"HisTag-pro-KLK\", \"13\", \")\", \"was\", \"observed\", \"after\", \"treatment\", \"with\", \"500\", \"nM\", \"KLK\", \"13\", \"(\", \"Fig.\", \"6\", \"A\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"product\", \"243\", \"of\", \"the\", \"S\", \"1\", \"/S\", \"2\", \"cleavage\", \"was\", \"further\", \"sequenced\", \"by\", \"N-terminal\", \"Edman\", \"degradation\", \"showing\", \"the\", \"244\", \"following\", \"sequence\", \":\", \"R↓SISA\", \",\", \"which\", \"corresponds\", \"to\", \"the\", \"S\", \"1\", \"/S\", \"2\", \"site\", \".\"]\n",
      "Word token=> [\"This\", \"result\", \"shows\", \"clearly\", \"245\", \"that\", \"the\", \"S\", \"1\", \"/S\", \"2\", \"region\", \"of\", \"the\", \"HCoV-HKU\", \"1\", \"S\", \"protein\", \"is\", \"prone\", \"to\", \"KLK\", \"13\", \"-mediated\", \"cleavage\", \".\"]\n",
      "Word token=> [\"246\", \"Furthermore\", \",\", \"we\", \"aimed\", \"to\", \"confirm\", \"the\", \"cleavage\", \"using\", \"a\", \"full-length\", \"Spike\", \"protein\", \"of\", \"HCoV-HKU\", \"1\", \"(\", \"HKU\", \"1\", \"-S\", \")\", \".\"]\n",
      "Word token=> [\"For\", \"this\", \",\", \"we\", \"expressed\", \"the\", \"HKU\", \"1\", \"-S\", \"in\", \"293\", \"T\", \"cells\", \",\", \"purified\", \"the\", \"protein\", \"248\", \"using\", \"6\", \"×\", \"His\", \"tag\", \",\", \"and\", \"incubated\", \"for\", \"3\", \"h\", \"at\", \"37\", \"°\", \"C\", \"with\", \"increasing\", \"concentrations\", \"of\", \"purified\", \"249\", \"KLK\", \"13\", \".\"]\n",
      "Word token=> [\"Subsequently\", \",\", \"HKU-S\", \"or\", \"mock\", \"proteins\", \"were\", \"resolved\", \"by\", \"SDS-PAGE\", \"and\", \"detected\", \"by\", \"250\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.01.971499\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"western\", \"blotting\", \"with\", \"antibodies\", \"specific\", \"to\", \"the\", \"tag\", \".\"]\n",
      "Word token=> [\"The\", \"analysis\", \"showed\", \"that\", \"in\", \"the\", \"presence\", \"of\", \"251\", \"1\", \"µM\", \"KLK\", \"13\", \"the\", \"HKU\", \"1\", \"-S\", \"was\", \"degraded\", \"(\", \"Figure\", \"6\", \"B\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"S\", \"protein\", \"was\", \"observed\", \"at\", \"252\", \"~\", \"150\", \"kDa\", \",\", \"which\", \"is\", \"consistent\", \"with\", \"the\", \"migration\", \"speed\", \"reported\", \"for\", \"these\", \"highly\", \"glycosylated\", \"253\", \"proteins\", \"(\", \"1\", \")\", \".\"]\n",
      "Word token=> [\"254\", \"255\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.01.971499\", \"doi\", \":\", \"bioRxiv\", \"preprint\"]\n",
      "Word token=> [\"Receptor\", \"recognition\", \"is\", \"the\", \"first\", \",\", \"essential\", \"step\", \"of\", \"the\", \"virus\", \"infection\", \"process\", \".\"]\n",
      "Word token=> [\"The\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.01.971499\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"concentrations\", \"(\", \"63\", \")\", \".\"]\n",
      "Word token=> [\"Broad\", \"spectrum\", \"protease\", \"inhibitors\", \"are\", \"now\", \"used\", \"widely\", \"for\", \"virus\", \"research\", \",\", \"281\", \"although\", \"their\", \"non-specific\", \"activity\", \"makes\", \"the\", \"results\", \"equivocal\", \".\"]\n",
      "Word token=> [\"For\", \"example\", \",\", \"Matsuyama\", \"et\", \"al\", \".\"]\n",
      "Word token=> [\"282\", \"showed\", \"recently\", \"that\", \"the\", \"furin\", \"inhibitor\", \"dec-RVKR-CMK\", \"interferes\", \"with\", \"the\", \"activity\", \"of\", \"several\", \"283\", \"proteases\", \",\", \"and\", \"that\", \"its\", \"previously\", \"described\", \"inhibitory\", \"activity\", \"during\", \"MERS-CoV\", \"infection\", \"is\", \"284\", \"not\", \"specific\", \"to\", \"furin\", \";\", \"instead\", \",\", \"its\", \"activity\", \"is\", \"due\", \"to\", \"non-specific\", \"inhibition\", \"of\", \"cathepsin\", \"L\", \"and\", \"285\", \"TMPRSS\", \"2\", \"(\", \"81\", \")\", \".\"]\n",
      "Word token=> [\"We\", \"tried\", \"to\", \"use\", \"specific\", \"KLK\", \"inhibitors\", \"developed\", \"in\", \"our\", \"laboratory\", \"(\", \"61\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Considering\", \"the\", \"small\", \"arsenal\", \"of\", \"tools\", \"available\", \"to\", \"researchers\", \"studying\", \"KLKs\", \",\", \"only\", \"three\", \"287\", \"compounds\", \"were\", \"readily\", \"available\", \".\"]\n",
      "Word token=> [\"Treating\", \"HAE\", \"cultures\", \"with\", \"these\", \"inhibitors\", \"revealed\", \"that\", \"288\", \"only\", \"compounds\", \"designed\", \"to\", \"inhibit\", \"KLK\", \"13\", \"hampered\", \"HCoV-HKU\", \"1\", \"replication\", \".\"]\n",
      "Word token=> [\"However\", \",\", \"the\", \"289\", \"great\", \"similarity\", \"between\", \"different\", \"KLKs\", \"makes\", \"one\", \"doubt\", \"the\", \"specificity\", \"of\", \"these\", \"inhibitors\", \",\", \"290\", \"despite\", \"their\", \"performance\", \"in\", \"biochemical\", \"assays\", \".\"]\n",
      "Word token=> [\"Therefore\", \",\", \"we\", \"decided\", \"to\", \"silence\", \"KLK\", \"13\", \"in\", \"291\", \"HAE\", \"cultures\", \".\"]\n",
      "Word token=> [\"This\", \"abolished\", \"virus\", \"replication\", \"ex\", \"vivo\", \",\", \"thereby\", \"confirming\", \"the\", \"importance\", \"of\", \"292\", \"KLK\", \"13\", \"during\", \"infection\", \".\"]\n",
      "Word token=> [\"KLK\", \"13\", \"is\", \"thought\", \"to\", \"be\", \"secreted\", \"and\", \"membrane-bound\", \"(\", \"82\", \",\", \"83\", \")\", \".\"]\n",
      "Word token=> [\"293\", \"This\", \"study\", \"showed\", \"that\", \"HCoV-HKU\", \"1\", \"infection\", \"in\", \"HAE\", \"modulates\", \"the\", \"expression\", \"of\", \"294\", \"different\", \"KLKs\", \",\", \"including\", \"KLK\", \"13\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"In\", \"our\", \"study\", \",\", \"KLKs\", \"expression\", \"was\", \"tested\", \"using\", \"semi-\", \"295\", \"quantitative\", \"PCR\", \",\", \"and\", \"for\", \"that\", \"reason\", \",\", \"we\", \"were\", \"unable\", \"to\", \"show\", \"the\", \"level\", \"of\", \"KLKs\", \"modulation\", \"296\", \"after\", \"HCoV-HKU\", \"1\", \".\"]\n",
      "Word token=> [\"However\", \",\", \"the\", \"pattern\", \"of\", \"virus-induced\", \"expression\", \"of\", \"several\", \"KLKs\", \"could\", \"297\", \"be\", \"observed\", \".\"]\n",
      "Word token=> [\"KLK\", \"mechanism\", \"of\", \"activation\", \"is\", \"a\", \"complex\", \"process\", \"and\", \"until\", \"now\", \"it\", \"has\", \"only\", \"298\", \"been\", \"proven\", \"that\", \"most\", \"KLK\", \"genes\", \"are\", \"regulated\", \"by\", \"steroids\", \"and\", \"other\", \"hormones\", \"(\", \"84\", \")\", \".\"]\n",
      "Word token=> [\"It\", \"is\", \"also\", \"299\", \"important\", \"to\", \"remember\", \",\", \"that\", \"KLK\", \"expression\", \"is\", \"regulated\", \"in\", \"a\", \"similar\", \"manner\", \",\", \"and\", \"the\", \"induction\", \"300\", \"of\", \"a\", \"single\", \"gene\", \"usually\", \"results\", \"in\", \"overexpression\", \"of\", \"the\", \"whole\", \"cluster\", \"(\", \"85\", \",\", \"86\", \")\", \".\"]\n",
      "Word token=> [\"While\", \"one\", \"may\", \"301\", \"assume\", \"that\", \"the\", \"virus\", \"stimulates\", \"KLK\", \"13\", \"production\", \"to\", \"promote\", \"the\", \"infection\", \",\", \"this\", \"up-regulation\", \"302\", \"is\", \"likely\", \"a\", \"natural\", \"response\", \"of\", \"the\", \"damaged\", \"tissue\", \",\", \"as\", \"KLKs\", \"were\", \"previously\", \"reported\", \"to\", \"take\", \"303\", \"part\", \"also\", \"in\", \"tissue\", \"regeneration\", \"(\", \"87\", \"-\", \"89\", \")\", \".\"]\n",
      "Word token=> [\"Further\", \",\", \"increased\", \"expression\", \"of\", \"KLKs\", \"may\", \"be\", \"the\", \"304\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.01.971499\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"response\", \"to\", \"the\", \"inflammatory\", \"process\", \",\", \"as\", \"Seliga\", \"et\", \"al\", \"demonstrated\", \"that\", \"KLK-kinin\", \"system\", \"is\", \"a\", \"305\", \"potent\", \"modulator\", \"of\", \"innate\", \"immune\", \"responses\", \"(\", \"90\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"experiments\", \"performed\", \"herein\", \"show\", \"the\", \"importance\", \"of\", \"KLK\", \"13\", \"for\", \"virus\", \"entry\", \"into\", \"307\", \"susceptible\", \"cells\", \";\", \"therefore\", \",\", \"we\", \"speculated\", \"that\", \"scattered\", \"distribution\", \"of\", \"different\", \"KLKs\", \"in\", \"308\", \"different\", \"tissues\", \"may\", \"be\", \"one\", \"of\", \"the\", \"determinants\", \"of\", \"the\", \"HCoV-HKU\", \"1\", \"tropism\", \"(\", \"53\", \",\", \"91\", \")\", \".\"]\n",
      "Word token=> [\"We\", \"309\", \"tested\", \"the\", \"purified\", \"enzyme\", \"expressed\", \"in\", \"the\", \"eukaryotic\", \"cells\", \";\", \"however\", \",\", \"we\", \"also\", \"developed\", \"a\", \"cell\", \"310\", \"line\", \"constitutively\", \"expressing\", \"the\", \"enzyme\", \".\"]\n",
      "Word token=> [\"As\", \"an\", \"in\", \"vitro\", \"model\", \"for\", \"our\", \"studies\", \",\", \"we\", \"used\", \"RD\", \"311\", \"cells\", \"previously\", \"reported\", \"to\", \"carry\", \"attachment\", \"receptors\", \"for\", \"the\", \"virus\", \"(\", \"26\", \")\", \".\"]\n",
      "Word token=> [\"Here\", \",\", \"using\", \"312\", \"pseudoviruses\", \"decorated\", \"with\", \"S-HKU\", \"1\", \"proteins\", \"we\", \"showed\", \"that\", \"KLK\", \"13\", \"presence\", \"on\", \"RD\", \"cells\", \"313\", \"is\", \"sufficient\", \"for\", \"virus\", \"entry\", \"and\", \"renders\", \"these\", \"cells\", \"permissive\", \".\"]\n",
      "Word token=> [\"Our\", \"experiments\", \"also\", \"showed\", \"314\", \"that\", \"in\", \"contrast\", \"to\", \"previous\", \"reports\", \",\", \"TMPRSS\", \"2\", \"is\", \"not\", \"involved\", \"in\", \"this\", \"process\", \"(\", \"51\", \")\", \".\"]\n",
      "Word token=> [\"Furthermore\", \",\", \"315\", \"we\", \"observed\", \"that\", \"RD\", \"cells\", \"supported\", \"the\", \"replication\", \"of\", \"the\", \"virus\", \"in\", \"the\", \"presence\", \"of\", \"KLK\", \"13\", \"and\", \"316\", \"that\", \"this\", \"effect\", \"was\", \"reversed\", \"in\", \"the\", \"presence\", \"of\", \"the\", \"specific\", \"KLK\", \"13\", \"inhibitor\", \".\"]\n",
      "Word token=> [\"We\", \"were\", \",\", \"however\", \",\", \"317\", \"not\", \"able\", \"to\", \"culture\", \"the\", \"virus\", \"to\", \"high\", \"yields\", \".\"]\n",
      "Word token=> [\"HCoV-HKU\", \"1\", \"replication\", \"in\", \"KLK\", \"13\", \"-expressing\", \"RD\", \"318\", \"cells\", \"remained\", \"inefficient\", \"and\", \"RTqPCR\", \"assessment\", \"did\", \"not\", \"reveal\", \"significant\", \"increases\", \"in\", \"the\", \"319\", \"amounts\", \"of\", \"viral\", \"RNA\", \".\"]\n",
      "Word token=> [\"For\", \"that\", \"reason\", \",\", \"we\", \"are\", \"only\", \"able\", \"to\", \"detect\", \"viral\", \"sg\", \"mRNAs\", \",\", \"which\", \"are\", \"320\", \"considered\", \"to\", \"be\", \"the\", \"hallmark\", \"of\", \"coronaviral\", \"replication.We\", \"believe\", \"that\", \"this\", \"may\", \"be\", \"due\", \"to\", \"321\", \"non-optimal\", \"infection\", \"conditions\", \",\", \"which\", \"may\", \"include\", \"inappropriate\", \"KLK\", \"13\", \"concentrations\", \"or\", \"322\", \"low\", \"density\", \"of\", \"the\", \"entry\", \"receptor\", \".\"]\n",
      "Word token=> [\"Also\", \",\", \"it\", \"is\", \"possible\", \"that\", \"RD\", \"cells\", \"may\", \"not\", \"support\", \"efficient\", \"323\", \"replication\", \"of\", \"the\", \"virus\", \"due\", \"to\", \"factors\", \"unrelated\", \"to\", \"the\", \"entry\", \"process\", \".\"]\n",
      "Word token=> [\"Nonetheless\", \",\", \"our\", \"results\"]\n",
      "Word token=> [\"show\", \"that\", \"the\", \"HCoV-HKU\", \"1\", \"entry\", \"receptor\", \"is\", \"present\", \"on\", \"RD\", \"cells\", \",\", \"and\", \"we\", \"were\", \"able\", \"to\", \"trigger\", \"325\", \"virus\", \"entry\", \"and\", \"replication\", \";\", \"these\", \"findings\", \"warrant\", \"further\", \"exploration\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Most\", \"coronaviral\", \"S\", \"proteins\", \"are\", \"processed\", \"into\", \"S\", \"1\", \"and\", \"S\", \"2\", \"subunits\", \"by\", \"host\", \"proteases\", \",\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.01.971499\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"HCoV-HKU\", \"1\", \"S\", \"protein\", \"has\", \"two\", \"regions\", \"that\", \"are\", \"prone\", \"to\", \"proteolytic\", \"activation\", \":\", \"the\", \"S\", \"1\", \"/S\", \"2\", \"furin\", \"330\", \"cleavage\", \"site\", \"and\", \"a\", \"secondary\", \"cleavage\", \"site\", \"termed\", \"S\", \"2\", \"'\", \",\", \"which\", \"is\", \"adjacent\", \"to\", \"a\", \"potential\", \"fusion\", \"331\", \"peptide\", \"(\", \"38\", \")\", \".\"]\n",
      "Word token=> [\"While\", \"the\", \"S\", \"1\", \"/S\", \"2\", \"site\", \"is\", \"believed\", \"to\", \"be\", \"processed\", \"by\", \"furin\", \"during\", \"protein\", \"332\", \"biosynthesis\", \",\", \"the\", \"S\", \"2\", \"/S\", \"2\", \"'\", \"site\", \"is\", \"expected\", \"to\", \"be\", \"cleaved\", \"during\", \"virus\", \"entry\", \".\"]\n",
      "Word token=> [\"As\", \"we\", \"already\", \"knew\", \"being\", \"tested\", \".\"]\n",
      "Word token=> [\"We\", \"found\", \"that\", \"the\", \"S\", \"1\", \"/S\", \"2\", \"site\", \"was\", \"efficiently\", \"cleaved\", \"by\", \"KLK\", \"13\", \",\", \"whereas\", \"the\", \"338\", \"S\", \"2\", \"/S\", \"2\", \"'\", \"region\", \"remained\", \"intact\", \".\"]\n",
      "Word token=> [\"As\", \"CleavEx\", \"technique\", \"is\", \"a\", \"convenient\", \"surrogate\", \"system\", \"allowing\", \"339\", \"for\", \"precise\", \"mapping\", \"of\", \"the\", \"cleavage\", \"site\", \",\", \"it\", \"has\", \"some\", \"limitations\", \".\"]\n",
      "Word token=> [\"To\", \"ensure\", \"the\", \"reliability\", \"of\", \"340\", \"results\", \",\", \"purified\", \"full-length\", \"HCoV-HKU\", \"1\", \"S\", \"protein\", \"was\", \"subjected\", \"to\", \"the\", \"proteolytic\", \"cleavage\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Also\", \"here\", \"we\", \"observed\", \"efficient\", \"cleavage\", \"of\", \"the\", \"HCoV-HKU\", \"1\", \"S\", \"protein\", \"by\", \"KLK\", \"13\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"While\", \"the\", \"results\", \"presented\", \"here\", \"show\", \"that\", \"KLK\", \"13\", \"is\", \"able\", \"to\", \"process\", \"the\", \"HCoV-HKU\", \"1\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.01.971499\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"to\", \"cross\", \"the\", \"species\", \"barrier\", \"and\", \"cause\", \"severe\", \"diseases\", \"in\", \"humans\", \",\", \"further\", \"research\", \"on\", \"the\", \"role\", \"of\", \"355\", \"different\", \"proteases\", \"in\", \"coronaviral\", \"infections\", \"is\", \"necessary\", \".\"]\n",
      "Word token=> [\"356\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.01.971499\", \"doi\", \":\", \"bioRxiv\", \"preprint\"]\n",
      "Word token=> [\"Plasmid\", \"constructs\", \"358\", \"KLK\", \"13\", \"and\", \"TMPRSS\", \"2\", \"genes\", \"were\", \"amplified\", \"by\", \"PCR\", \"using\", \"cDNA\", \"obtained\", \"from\", \"HAE\", \"359\", \"cells\", \".\"]\n",
      "Word token=> [\"Each\", \"PCR\", \"product\", \"was\", \"cloned\", \"into\", \"pWPI\", \"plasmid\", \"for\", \"lentivirus\", \"production\", \"and\", \"sequence\", \"Table\", \"2\", \".\"]\n",
      "Word token=> [\"(\", \"Table\", \"3\", \";\", \"500\", \"nM\", \"each\", \")\", \".\"]\n",
      "Word token=> [\"β-actin\", \"was\", \"used\", \"as\", \"a\", \"household\", \"gene\", \"reference\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"reaction\", \"was\", \"carried\", \"out\", \"according\", \"to\", \"the\", \"scheme\", \":\", \"5\", \"min\", \"at\", \"95\", \"°\", \"C\", \",\", \"followed\", \"by\", \"35\", \"cycles\", \"of\", \"465\", \"30\", \"s\", \"at\", \"95\", \"°\", \"C\", \",\", \"20\", \"s\", \"at\", \"59\", \"°\", \"C\", \"and\", \"20\", \"s\", \"at\", \"72\", \"°\", \"C\", \",\", \"followed\", \"by\", \"10\", \"min\", \"at\", \"72\", \"°\", \"C.\", \"containing\", \"proKLK\", \"13\", \"or\", \"proKLK\", \"14\", \"were\", \"concentrated\", \"and\", \"the\", \"buffer\", \"was\", \"changed\", \"to\", \"50\", \"mM\", \"528\", \"Tris\", \"pH\", \"7.5\", \",\", \"150\", \"mM\", \"NaCl\", \".\"]\n",
      "Word token=> [\"After\", \"purification\", \"and\", \"self-activation\", \"at\", \"37\", \"°\", \"C\", \"for\", \"24\", \"h\", \",\", \"activity\", \"of\", \"529\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.01.971499\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"proteases\", \"was\", \"assessed\", \"by\", \"serine\", \"protease\", \"inhibitor\", \"Kazal-type\", \"6\", \"(\", \"SPINK\", \"6\", \")\", \"titration\", \",\", \"as\", \"530\", \"described\", \"previously\", \"(\", \"63\", \")\", \".\"]\n",
      "Word token=> [\"Protein\", \"expression\", \"was\", \"performed\", \"in\", \"E.\", \"coli\", \"BL\", \"21\", \"and\", \"was\", \"induced\", \"by\", \"the\", \"addition\", \"of\", \"553\", \"0.5\", \"mM\", \"IPTG\", \"to\", \"the\", \"bacterial\", \"culture\", \"(\", \"OD\", \"600\", \"0.5\", \"-\", \"0.6\", \")\", \",\", \"followed\", \"by\", \"shaking\", \"for\", \"3\", \"h\", \"at\", \"37\", \"°\", \"C.\", \"554\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.01.971499\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"Then\", \",\", \"the\", \"bacteria\", \"were\", \"spun\", \"down\", \"and\", \"the\", \"pellet\", \"was\", \"suspended\", \"in\", \"buffer\", \"A\", \"(\", \"10\", \"mM\", \"sodium\", \"555\", \"phosphate\", \",\", \"500\", \"mM\", \"NaCl\", \"and\", \"5\", \"mM\", \"imidazole\", \",\", \"pH\", \"7.4\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"pellet\", \"suspension\", \"was\", \"then\", \"556\", \"sonicated\", \"and\", \"spun\", \"down\", \".\"]\n",
      "Word token=> [\"Soluble\", \"proteins\", \"were\", \"purified\", \"using\", \"HisTrap\", \"TM\", \"Excel\", \"(\", \"GE\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.01.971499\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \".\"]\n",
      "Word token=> [\"4\", \"hepatoma\", \"cells\", \"was\", \"largely\", \"specific\", \"to\", \"CoV\", \".\"]\n",
      "Word token=> [\"Among\", \"10\", \"genetically\", \"divergent\", \"viruses\", \"tested\", \",\", \"only\", \"hepatitis\", \"C\", \"virus\", \"was\", \"also\", \"restricted\", \"(\", \"Extended\", \"Data\", \"Fig.\", \"1\", \"f\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"LY\", \"6\", \"E\", \"is\", \"a\", \"member\", \"of\", \"the\", \"LY\", \"6\", \"/uPAR\", \"family\", \"of\", \"GPI-anchored\", \"proteins\", \",\", \"and\", \"is\", \"implicated\", \"in\", \"diverse\", \"cellular\", \"processes\", \"14\", \".\"]\n",
      "Word token=> [\"To\", \"evaluate\", \"the\", \"specificity\", \"of\", \"LY\", \"6\", \"E\", \"for\", \"inhibiting\", \"CoV\", \",\", \"we\", \"ectopically\", \"expressed\", \"select\", \"LY\", \"6\", \"/uPAR\", \"proteins\", \"9\", \"and\", \"infected\", \"with\", \"HCoV-\", \"229\", \"E\", \".\"]\n",
      "Word token=> [\"Only\", \"LY\", \"6\", \"E\", \"inhibited\", \"CoV\", \"infection\", \"(\", \"Extended\", \"Data\", \"Fig.\", \"1\", \"g\", \")\", \".\"]\n",
      "Word token=> [\"We\", \"previously\", \"showed\", \"that\", \"a\", \"conserved\", \"residue\", \"(\", \"L\", \"36\", \")\", \"is\", \"required\", \"for\", \"LY\", \"6\", \"Emediated\", \"enhancement\", \"of\", \"influenza\", \"A\", \"virus\", \"9\", \".\"]\n",
      "Word token=> [\"L\", \"36\", \"was\", \"also\", \"required\", \"to\", \"restrict\", \"HCoV-\", \"229\", \"E\", \"infection\", \"(\", \"Extended\", \"Data\", \"Fig.\", \"1\", \"h\", \")\", \".\"]\n",
      "Word token=> [\"These\", \"data\", \"suggest\", \"a\", \"conserved\", \"functional\", \"domain\", \"that\", \"is\", \"responsible\", \"for\", \"both\", \"the\", \"virus-enhancing\", \"and\", \"-restricting\", \"phenotypes\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Next\", \",\", \"we\", \"investigated\", \"the\", \"effect\", \"of\", \"LY\", \"6\", \"E\", \"on\", \"specific\", \"steps\", \"of\", \"the\", \"virus\", \"replication\", \"cycle\", \".\"]\n",
      "Word token=> [\"First\", \",\", \"we\", \"tested\", \"whether\", \"LY\", \"6\", \"E\", \"restricts\", \"HCoV-\", \"229\", \"E\", \"attachment\", \"to\", \"the\", \"cell\", \"surface\", \"and\", \"observed\", \"no\", \"effect\", \"(\", \"Extended\", \"Data\", \"Fig.\", \"2\", \"a\", \")\", \".\"]\n",
      "Word token=> [\"In\", \"accordance\", \",\", \"LY\", \"6\", \"E\", \"had\", \"no\", \"effect\", \"on\", \"surface\", \"expression\", \"of\", \"CD\", \"13\", \",\", \"the\", \"specific\", \"receptor\", \"for\", \"HCoV-\", \"229\", \"E\", \"(\", \"Extended\", \"Data\", \"Fig.\", \"2\", \"b\", \")\", \".\"]\n",
      "Word token=> [\"To\", \"test\", \"whether\", \"LY\", \"6\", \"E\", \"restricts\", \"viral\", \"entry\", \",\", \"we\", \"used\", \"a\", \"vesicular\", \"stomatitis\", \"virus\", \"(\", \"VSV\", \")\", \"pseudoparticle\", \"(\", \"pp\", \")\", \"system\", \"bearing\", \"CoV\", \"spike\", \"(\", \"S\", \")\", \"proteins\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Strikingly\", \",\", \"LY\", \"6\", \"E\", \"significantly\", \"inhibited\", \"VSVpp\", \"entry\", \"mediated\", \"by\", \"S\", \"proteins\", \"from\", \"HCoV-\", \"229\", \"E\", \"(\", \"Fig.\", \"2\", \"a\", \")\", \",\", \"MERS-CoV\", \"(\", \"Fig.\", \"2\", \"b\", \")\", \",\", \"SARS-CoV\", \"(\", \"Fig.\", \"2\", \"c\", \")\", \",\", \"and\", \"SARS-CoV-\", \"2\", \"(\", \"Fig.\", \"2\", \"d\", \")\", \".\"]\n",
      "Word token=> [\"Entry\", \"mediated\", \"by\", \"VSV\", \"glycoprotein\", \"G\", \"was\", \"only\", \"marginally\", \"impacted\", \"by\", \"LY\", \"6\", \"E\", \"(\", \"Extended\", \"Data\", \"Fig.\", \"2\", \"c\", \",\", \"d\", \")\", \".\"]\n",
      "Word token=> [\"To\", \"test\", \"whether\", \"LY\", \"6\", \"E\", \"interferes\", \"with\", \"the\", \"fusion\", \"of\", \"viral\", \"and\", \"cellular\", \"membranes\", \",\", \"we\", \"performed\", \"a\", \"syncytia\", \"formation\", \"assay\", \"using\", \"propagation\", \"competent\", \"VSV\", \"pseudoviruses\", \"co-expressing\", \"CoV\", \"S\", \"protein\", \"and\", \"a\", \"GFP\", \"reporter\", \"(\", \"VSV\", \"*\", \"ΔG\", \"(\", \"CoV\", \"S\", \")\", \")\", \"(\", \"Extended\", \"Data\", \"Fig.\", \"3\", \"a\", \")\", \".\"]\n",
      "Word token=> [\"LY\", \"6\", \"E\", \"potently\", \"inhibited\", \"CoV\", \"S\", \"protein-mediated\", \"syncytia\", \"formation\", \"(\", \"Fig.\", \"2\", \"e-f\", \")\", \".\"]\n",
      "Word token=> [\"To\", \"determine\", \"whether\", \"LY\", \"6\", \"E\", \"impairs\", \"S\", \"protein\", \"expression\", \"or\", \"maturation\", \",\", \"we\", \"also\", \"performed\", \"a\", \"heterologous\", \"syncytia\", \"formation\", \"assay\", \".\"]\n",
      "Word token=> [\"CoV-resistant\", \"hamster\", \"cells\", \"infected\", \"with\", \"trans-complemented\", \"GFP\", \"reporter\", \"VSV\", \"*\", \"ΔG\", \"(\", \"CoV\", \")\", \"were\", \"co-cultured\", \"with\", \"susceptible\", \"target\", \"cells\", \"that\", \"ectopically\", \"expressed\", \"LY\", \"6\", \"E\", \".\"]\n",
      "Word token=> [\"LY\", \"6\", \"E\", \"again\", \"significantly\", \"blocked\", \"syncytia\", \"formation\", \",\", \"demonstrating\", \"that\", \"LY\", \"6\", \"E\", \"inhibits\", \"S\", \"protein-mediated\", \"fusion\", \"and\", \"not\", \"S\", \"protein\", \"expression\", \"or\", \"maturation\", \"(\", \"Extended\", \"Data\", \"Fig.\", \"3\", \"b-c\", \")\", \".\"]\n",
      "Word token=> [\"As\", \"membrane\", \"fusion\", \"can\", \"occur\", \"without\", \"syncytia\", \"formation\", \",\", \"we\", \"also\", \"performed\", \"a\", \"quantitative\", \"fusion\", \"assay\", \"in\", \"which\", \"mixing\", \"of\", \"cell\", \"contents\", \"results\", \"in\", \"complementation\", \"of\", \"split\", \"luciferase\", \"(\", \"Fig.\", \"2\", \"g\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"Cells\", \"co-transfected\", \"with\", \"plasmids\", \"encoding\", \"CoV\", \"S\", \"proteins\", \"and\", \"split\", \"luciferase\", \"1\", \"-\", \"7\", \"were\", \"mixed\", \"with\", \"susceptible\", \"target\", \"cells\", \"co-expressing\", \"LY\", \"6\", \"E\", \"or\", \"vector\", \"control\", \"and\", \"split\", \"luciferase\", \"8\", \"-\", \"11\", \".\"]\n",
      "Word token=> [\"LY\", \"6\", \"E\", \"reduced\", \"CoV\", \"S-mediated\", \"cell-cell\", \"fusion\", \",\", \"indicating\", \"that\", \"LY\", \"6\", \"E\", \"blocks\", \"fusion\", \"of\", \"viral\", \"and\", \"cellular\", \"membranes\", \"(\", \"Fig.\", \"2\", \"h\", \")\", \".\"]\n",
      "Word token=> [\"Time-course\", \"experiments\", \"revealed\", \"that\", \"LY\", \"6\", \"E\", \"had\", \"no\", \"effect\", \"on\", \"HCoV-\", \"229\", \"E\", \"translation\", \",\", \"replication\", \",\", \"assembly\", \",\", \"or\", \"release\", \"(\", \"Extended\", \"Data\", \"Fig.\", \"2\", \"e-i\", \")\", \".\"]\n",
      "Word token=> [\"Collectively\", \",\", \"our\", \"results\", \"demonstrate\", \"that\", \"LY\", \"6\", \"E\", \"specifically\", \"inhibits\", \"CoV\", \"S\", \"protein-mediated\", \"membrane\", \"fusion\", \",\", \"which\", \"is\", \"required\", \"for\", \"viral\", \"entry\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"We\", \"next\", \"addressed\", \"whether\", \"LY\", \"6\", \"E\", \"modulates\", \"proteolytic\", \"activation\", \"of\", \"the\", \"S\", \"protein\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"Upon\", \"S\", \"proteinmediated\", \"binding\", \"to\", \"the\", \"respective\", \"receptor\", \",\", \"host\", \"proteases\", \"cleave\", \"the\", \"S\", \"protein\", \"and\", \"make\", \"it\", \"fusion\", \"competent\", \"for\", \"\\\"\", \"early\", \"entry\", \"\\\"\", \"at\", \"the\", \"cell\", \"surface\", \"or\", \"\\\"\", \"late\", \"entry\", \"\\\"\", \"in\", \"endosomes\", \"15\", \",\", \"16\", \".\"]\n",
      "Word token=> [\"Pharmacological\", \"inhibition\", \"of\", \"cell\", \"surface\", \"and\", \"endosomal\", \"proteases\", \"had\", \"no\", \"effect\", \"on\", \"LY\", \"6\", \"E-mediated\", \"restriction\", \"of\", \"CoV\", \"infection\", \"(\", \"Extended\", \"Data\", \"Fig.\", \"4\", \"a-c\", \")\", \".\"]\n",
      "Word token=> [\"Next\", \",\", \"we\", \"addressed\", \"whether\", \"LY\", \"6\", \"E\", \"directly\", \"interferes\", \"with\", \"the\", \"proteolytic\", \"activation\", \"of\", \"S\", \"protein\", \".\"]\n",
      "Word token=> [\"Mutation\", \"of\", \"S\", \"1\", \"/S\", \"2\", \"and\", \"S\", \"2\", \"′\", \"in\", \"the\", \"S\", \"protein\", \"prevents\", \"activation\", \"(\", \"Extended\", \"Data\", \"Fig.\", \"4\", \"d\", \")\", \"16\", \",\", \"17\", \".\"]\n",
      "Word token=> [\"Ectopic\", \"LY\", \"6\", \"E\", \"expression\", \"had\", \"no\", \"effect\", \"on\", \"MERS-CoV\", \"S\", \"cleavage\", \"or\", \"infection\", \"by\", \"cleavage-resistant\", \"MERS-CoV\", \"pp\", \"(\", \"Extended\", \"Data\", \"Fig.\", \"4\", \"e-g\", \")\", \".\"]\n",
      "Word token=> [\"Collectively\", \",\", \"our\", \"data\", \"indicate\", \"that\", \"restriction\", \"of\", \"viral\", \"fusion\", \"is\", \"independent\", \"of\", \"S\", \"protein\", \"activation\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Next\", \",\", \"we\", \"aimed\", \"to\", \"evaluate\", \"LY\", \"6\", \"E-mediated\", \"CoV\", \"restriction\", \"in\", \"vivo\", \".\"]\n",
      "Word token=> [\"Systemic\", \"genetic\", \"ablation\", \"of\", \"Ly\", \"6\", \"e\", \"in\", \"mice\", \"is\", \"embryonic\", \"lethal\", \"18\", \".\"]\n",
      "Word token=> [\"Therefore\", \",\", \"we\", \"generated\", \"transgenic\", \"Ly\", \"6\", \"e\", \"fl/fl\", \"mice\", \",\", \"which\", \"are\", \"functionally\", \"'\", \"wild-type\", \"(\", \"WT\", \")\", \"'\", \"for\", \"Ly\", \"6\", \"e\", \"expression\", \",\", \"and\", \"crossed\", \"them\", \"with\", \"Vav\", \"1\", \"-iCre\", \"mice\", \"to\", \"ablate\", \"Ly\", \"6\", \"e\", \"in\", \"cells\", \"that\", \"originate\", \"from\", \"a\", \"hematopoietic\", \"stem\", \"cell\", \"(\", \"HSC\", \")\", \"progenitor\", \"(\", \"Extended\", \"Data\", \"Fig.\", \"5\", \"a-b\", \")\", \".\"]\n",
      "Word token=> [\"This\", \"genetic\", \"model\", \"was\", \"chosen\", \"since\", \"immune\", \"cells\", \"are\", \"critical\", \"for\", \"protection\", \"from\", \"CoV\", \"in\", \"vivo\", \"5,19\", \"-\", \"23\", \".\"]\n",
      "Word token=> [\"Bonemarrow\", \"derived\", \"macrophages\", \"(\", \"BMDM\", \")\", \"from\", \"Ly\", \"6\", \"e\", \"ΔHSC\", \"mice\", \"had\", \"reduced\", \"Ly\", \"6\", \"e\", \"mRNA\", \"levels\", \"(\", \"Extended\", \"Data\", \"Fig.\", \"5\", \"c\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"natural\", \"mouse\", \"pathogen\", \"MHV\", \"is\", \"a\", \"well-studied\", \"model\", \"CoV\", \"that\", \"causes\", \"hepatitis\", \"and\", \"encephalomyelitis\", \"in\", \"mice\", \"24\", \".\"]\n",
      "Word token=> [\"Ly\", \"6\", \"e\", \"ΔHSC\", \"BMDM\", \"were\", \"more\", \"susceptible\", \"to\", \"MHV\", \"infection\", \"than\", \"Ly\", \"6\", \"e\", \"fl/fl\", \"BMDM\", \",\", \"demonstrating\", \"that\", \"murine\", \"Ly\", \"6\", \"e\", \"is\", \"a\", \"restriction\", \"factor\", \"for\", \"murine\", \"CoV\", \"(\", \"Extended\", \"Data\", \"Fig.\", \"5\", \"d\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"To\", \"determine\", \"whether\", \"Ly\", \"6\", \"e\", \"is\", \"important\", \"for\", \"controlling\", \"CoV\", \"infection\", \"in\", \"vivo\", \",\", \"Ly\", \"6\", \"e\", \"fl/fl\", \"and\", \"Ly\", \"6\", \"e\", \"ΔHSC\", \"mice\", \"were\", \"infected\", \"with\", \"a\", \"high\", \"dose\", \"(\", \"5\", \"x\", \"10\", \"4\", \"PFU\", \")\", \"of\", \"MHV\", \"that\", \"causes\", \"sub-lethal\", \"hepatitis\", \"in\", \"WT\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"C\", \"57\", \"BL/\", \"6\", \"J\", \"mice\", \"25\", \".\"]\n",
      "Word token=> [\"Since\", \"MHV\", \"pathogenesis\", \"is\", \"influenced\", \"by\", \"sex\", \"26\", \",\", \"we\", \"performed\", \"independent\", \"experiments\", \"in\", \"male\", \"and\", \"female\", \"mice\", \".\"]\n",
      "Word token=> [\"Ly\", \"6\", \"e\", \"ΔHSC\", \"mice\", \"of\", \"both\", \"sexes\", \"rapidly\", \"succumbed\", \"to\", \"MHV\", \"infection\", \"by\", \"day\", \"6\", \"(\", \"Fig.\", \"3\", \"a-b\", \",\", \"Extended\", \"Data\", \"Fig.\", \"6\", \"a-b\", \")\", \".\"]\n",
      "Word token=> [\"To\", \"examine\", \"viral\", \"pathogenesis\", \"and\", \"immunopathology\", \"in\", \"infected\", \"mice\", \",\", \"Ly\", \"6\", \"e\", \"fl/fl\", \"and\", \"Ly\", \"6\", \"e\", \"ΔHSC\", \"mice\", \"were\", \"infected\", \"with\", \"the\", \"same\", \"high\", \"dose\", \"of\", \"MHV\", \"and\", \"euthanized\", \"3\", \"-or\", \"5\", \"-days\", \"post-infection\", \".\"]\n",
      "Word token=> [\"At\", \"both\", \"time\", \"points\", \",\", \"female\", \"Ly\", \"6\", \"e\", \"ΔHSC\", \"mice\", \"exhibited\", \"significantly\", \"higher\", \"levels\", \"of\", \"liver\", \"damage\", \"as\", \"measured\", \"by\", \"serum\", \"ALT\", \"while\", \"no\", \"difference\", \"was\", \"observed\", \"in\", \"male\", \"mice\", \"(\", \"Fig.\", \"3\", \"c\", \",\", \"i\", \",\", \"Extended\", \"Data\", \"Fig.\", \"6\", \"c\", \",\", \"i\", \")\", \".\"]\n",
      "Word token=> [\"However\", \",\", \"hepatic\", \"viral\", \"burden\", \"did\", \"not\", \"differ\", \"in\", \"either\", \"sex\", \"(\", \"Fig.\", \"3\", \"d\", \",\", \"j\", \",\", \"Extended\", \"Data\", \"Fig.\", \"6\", \"d\", \",\", \"j\", \")\", \",\", \"possibly\", \"due\", \"to\", \"tissue\", \"saturation\", \"at\", \"this\", \"high\", \"dose\", \".\"]\n",
      "Word token=> [\"Indeed\", \",\", \"female\", \"but\", \"not\", \"male\", \"Ly\", \"6\", \"e\", \"ΔHSC\", \"mice\", \"exhibited\", \"increased\", \"liver\", \"damage\", \"and\", \"viral\", \"burden\", \"at\", \"a\", \"low\", \"dose\", \"(\", \"5\", \"PFU\", \")\", \"of\", \"MHV\", \"(\", \"Extended\", \"Data\", \"Fig.\", \"7\", \"a\", \"-b\", \",\", \"f-g\", \")\", \".\"]\n",
      "Word token=> [\"Spleen\", \"viral\", \"burden\", \"was\", \"elevated\", \"in\", \"female\", \"and\", \"male\", \"Ly\", \"6\", \"e\", \"ΔHSC\", \"mice\", \"at\", \"both\", \"time\", \"points\", \"and\", \"doses\", \"of\", \"MHV\", \"(\", \"Fig.\", \"3\", \"e\", \",\", \"k\", \",\", \"Extended\", \"Data\", \"Fig.\", \"6\", \"e\", \",\", \"k\", \",\", \"Extended\", \"Data\", \"Fig.\", \"7\", \"c\", \",\", \"h\", \")\", \".\"]\n",
      "Word token=> [\"At\", \"3\", \"days\", \"post-infection\", \",\", \"there\", \"was\", \"no\", \"difference\", \"in\", \"liver\", \"necrosis\", \",\", \"but\", \"inflammation\", \"was\", \"moderately\", \"reduced\", \"in\", \"female\", \"but\", \"not\", \"male\", \"Ly\", \"6\", \"e\", \"ΔHSC\", \"mice\", \"(\", \"Fig.\", \"3\", \"f-h\", \",\", \"Extended\", \"Data\", \"Fig.\", \"6\", \"f-h\", \")\", \".\"]\n",
      "Word token=> [\"By\", \"5\", \"days\", \"post-infection\", \"and\", \"at\", \"both\", \"low\", \"and\", \"high\", \"doses\", \"of\", \"MHV\", \",\", \"female\", \"Ly\", \"6\", \"e\", \"ΔHSC\", \"mice\", \"had\", \"significantly\", \"higher\", \"levels\", \"of\", \"liver\", \"necrosis\", \"and\", \"a\", \"reduced\", \"presence\", \"of\", \"mononuclear\", \"immune\", \"cells\", \"(\", \"Fig.\", \"3\", \"l-\", \"Fig.\", \"7\", \"k-n\", \")\", \".\"]\n",
      "Word token=> [\"These\", \"data\", \"demonstrate\", \"that\", \"Ly\", \"6\", \"e\", \"in\", \"hematopoietic\", \"cells\", \"is\", \"important\", \"for\", \"controlling\", \"murine\", \"CoV\", \"infection\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"We\", \"next\", \"used\", \"a\", \"transcriptomic\", \"approach\", \"to\", \"evaluate\", \"the\", \"effect\", \"of\", \"Ly\", \"6\", \"e\", \"ablation\", \"on\", \"global\", \"gene\", \"expression\", \"in\", \"liver\", \"and\", \"spleen\", \"of\", \"female\", \"Ly\", \"6\", \"e\", \"fl/fl\", \"and\", \"Ly\", \"6\", \"e\", \"ΔHSC\", \"mice\", \"injected\", \"with\", \"PBS\", \"or\", \"the\", \"high\", \"dose\", \"of\", \"MHV\", \"at\", \"3\", \"-and\", \"5\", \"-days\", \"post-infection\", \".\"]\n",
      "Word token=> [\"In\", \"infected\", \"tissues\", \",\", \"loss\", \"of\", \"Ly\", \"6\", \"e\", \"in\", \"hematopoietic\", \"cells\", \"correlated\", \"with\", \"differential\", \"gene\", \"expression\", \"in\", \"pathways\", \"associated\", \"with\", \"tissue\", \"damage\", \",\", \"such\", \"as\", \"liverspecific\", \"metabolic\", \"genes\", \",\", \"angiogenesis\", \",\", \"wound\", \"healing\", \",\", \"and\", \"immune\", \"response\", \"to\", \"viruses\", \"(\", \"Fig.\", \"4\", \"a-c\", \")\", \".\"]\n",
      "Word token=> [\"We\", \"also\", \"observed\", \"a\", \"striking\", \"loss\", \"of\", \"genes\", \"associated\", \"with\", \"the\", \"type\", \"I\", \"IFN\", \"response\", \",\", \"inflammation\", \",\", \"antigen\", \"presentation\", \",\", \"and\", \"B\", \"cells\", \"in\", \"infected\", \"Ly\", \"6\", \"e\", \"ΔHSC\", \"mice\", \"(\", \"Fig.\", \"4\", \"d-e\", \",\", \"Extended\", \"Data\", \"Fig.\", \"8\", \")\", \".\"]\n",
      "Word token=> [\"7\", \"The\", \"histopathology\", \"(\", \"Fig.\", \"3\", \"h\", \",\", \"n\", \")\", \"and\", \"transcriptome\", \"data\", \"(\", \"Fig\", \"4\", \"a-e\", \",\", \"Extended\", \"Data\", \"Fig.\", \"8\", \"Fig.\", \"7\", \"k-n\", \",\", \"Extended\", \"Data\", \"Fig.\", \"9\", \")\", \".\"]\n",
      "Word token=> [\"Multiple\", \"cell\", \"types\", \"were\", \"depleted\", \"in\", \"livers\", \"and\", \"spleens\", \"of\", \"infected\", \"Ly\", \"6\", \"e\", \"ΔHSC\", \"mice\", \",\", \"irrespective\", \"of\", \"sex\", \"(\", \"Figure\", \"4\", \"f\", \"g\", \",\", \"Extended\", \"Data\", \"Fig.\", \"10\", \"a-b\", \")\", \".\"]\n",
      "Word token=> [\"B\", \"cells\", \"were\", \"dramatically\", \"reduced\", \"in\", \"livers\", \"from\", \"infected\", \"Ly\", \"6\", \"e\", \"ΔHSC\", \"mice\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Hepatic\", \"CD\", \"4\", \"+\", \"T\", \"cells\", \",\", \"NK\", \"cells\", \",\", \"dendritic\", \"cells\", \"(\", \"DC\", \")\", \",\", \"macrophages\", \",\", \"and\", \"neutrophils\", \"were\", \"also\", \"depleted\", \"in\", \"infected\", \"Ly\", \"6\", \"e\", \"ΔHSC\", \"mice\", \",\", \"whereas\", \"CD\", \"8\", \"+\", \"T\", \"cells\", \"were\", \"unchanged\", \"(\", \"Fig.\", \"4\", \"f\", \",\", \"Extended\", \"Data\", \"Fig.\", \"10\", \"a\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Infected\", \"Ly\", \"6\", \"e\", \"ΔHSC\", \"mice\", \"had\", \"a\", \"reduction\", \"in\", \"all\", \"splenic\", \"immune\", \"cell\", \"subsets\", \"except\", \"for\", \"CD\", \"4\", \"+\", \"and\", \"CD\", \"8\", \"+\", \"T\", \"cells\", \"(\", \"Fig.\", \"4\", \"g\", \",\", \"Extended\", \"Data\", \"Fig.\", \"10\", \"b\", \")\", \".\"]\n",
      "Word token=> [\"Depletion\", \"of\", \"immune\", \"cell\", \"populations\", \"in\", \"Ly\", \"6\", \"e\", \"ΔHSC\", \"organs\", \"was\", \"MHV-dependent\", \",\", \"as\", \"cell\", \"numbers\", \"were\", \"not\", \"altered\", \"in\", \"PBS-injected\", \"mice\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"To\", \"determine\", \"whether\", \"loss\", \"of\", \"immune\", \"cells\", \"in\", \"MHV-infected\", \"Ly\", \"6\", \"e\", \"ΔHSC\", \"mice\", \"correlates\", \"with\", \"increased\", \"permissiveness\", \"to\", \"infection\", \",\", \"we\", \"assessed\", \"MHV\", \"infection\", \"in\", \"splenocytes\", \"cultured\", \"from\", \"Ly\", \"6\", \"e\", \"ΔHSC\", \",\", \"Ly\", \"6\", \"e\", \"fl/fl\", \",\", \"and\", \"Ifnar\", \"-/mice\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"In\", \"Ly\", \"6\", \"e\", \"fl/fl\", \"splenocyte\", \"cultures\", \",\", \"MHV\", \"infected\", \"macrophages\", \",\", \"neutrophils\", \",\", \"DC\", \",\", \"and\", \"B\", \"cells\", \",\", \"but\", \"not\", \"CD\", \"4\", \"+\", \"or\", \"CD\", \"8\", \"+\", \"T\", \"cells\", \",\", \"as\", \"previously\", \"published\", \"(\", \"Fig.\", \"4\", \"h\", \",\", \"Extended\", \"Data\", \"Fig.\", \"10\", \"c\", \"Together\", \",\", \"our\", \"data\", \"demonstrate\", \"an\", \"important\", \"role\", \"for\", \"Ly\", \"6\", \"e\", \"in\", \"immune\", \"cell-mediated\", \"control\", \"of\", \"CoV\", \"infection.\", \"Higher\", \"viral\", \"burden\", \"and\", \"lower\", \"liver\", \"inflammation\", \"suggests\", \"that\", \"liver\", \"damage\", \"in\", \"MHV-infected\", \"female\", \"Ly\", \"6\", \"e\", \"ΔHSC\", \"mice\", \"may\", \"be\", \"due\", \"to\", \"viral\", \"pathogenesis\", \"that\", \"exceeds\", \"the\", \"ability\", \"of\", \"the\", \"immune\", \"system\", \"to\", \"control\", \"the\", \"infection.\", \"Notably\", \",\", \"we\", \"observed\", \"MHV-induced\", \"loss\", \"of\", \"three\", \"classes\", \"of\", \"cells\", \":\", \"1\", \")\", \"cells\", \"that\", \"are\", \"not\", \"permissive\", \"to\", \"the\", \"virus\", \"(\", \"CD\", \"4\", \"+\", \"T\", \"cells\", \")\", \",\", \"2\", \")\", \"cells\", \"that\", \"are\", \"permissive\", \"to\", \"MHV\", \"but\", \"do\", \"not\", \"demonstrate\", \"Ly\", \"6\", \"e\", \"KO-dependent\", \"infection\", \"(\", \"NK\", \"cells\", \",\", \"macrophages\", \",\", \"and\", \"neutrophils\", \")\", \",\", \"and\", \"3\", \")\", \"cells\", \"that\", \"are\", \"permissive\", \"to\", \"MHV\", \"in\", \"a\", \"Ly\", \"6\", \"e\", \"KO-dependent\", \"manner\", \"(\", \"B\", \"cells\", \"and\", \"DC\", \")\", \".\"]\n",
      "Word token=> [\"This\", \"latter\", \"group\", \"is\", \"capable\", \"of\", \"antigen-presentation\", \"to\", \"CD\", \"4\", \"+\", \"T\", \"cells\", \"27,28\", \".\"]\n",
      "Word token=> [\"Thus\", \",\", \"in\", \"the\", \"absence\", \"of\", \"Ly\", \"6\", \"e\", \",\", \"unrestricted\", \"MHV\", \"infection\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"of\", \"B\", \"cells\", \"or\", \"DC\", \"may\", \"impair\", \"T\", \"cell\", \"priming\", \",\", \"thereby\", \"negatively\", \"impacting\", \"recruitment\", \"and\", \"activation\", \"of\", \"NK\", \"cells\", \",\", \"macrophages\", \",\", \"and\", \"neutrophils\", \"that\", \"contribute\", \"to\", \"control\", \"of\", \"viral\", \"disease\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"In\", \"conclusion\", \",\", \"we\", \"identified\", \"LY\", \"6\", \"E\", \"as\", \"a\", \"pan\", \"CoV\", \"restriction\", \"factor\", \"that\", \"limits\", \"CoV\", \"entry\", \"and\", \"protects\", \"the\", \"host\", \"from\", \"severe\", \"viral\", \"disease\", \".\"]\n",
      "Word token=> [\"Our\", \"findings\", \"are\", \"striking\", \"given\", \"that\", \"LY\", \"6\", \"E\", \"has\", \"been\", \"primarily\", \"associated\", \"with\", \"an\", \"enhancing\", \"phenotype\", \"in\", \"which\", \"LY\", \"6\", \"E\", \"promotes\", \"entry\", \"of\", \"multiple\", \"viruses\", \"29\", \".\"]\n",
      "Word token=> [\"However\", \",\", \"our\", \"data\", \"clearly\", \"expand\", \"the\", \"role\", \"of\", \"LY\", \"6\", \"E\", \"during\", \"coronaviral\", \"infection\", \"and\", \"establishes\", \"a\", \"novel\", \"function\", \"in\", \"protecting\", \"the\", \"host\", \"immune\", \"cell\", \"compartment\", \".\"]\n",
      "Word token=> [\"Determining\", \"the\", \"precise\", \"molecular\", \"mechanism\", \"underlying\", \"how\", \"LY\", \"6\", \"E\", \"inhibits\", \"CoV\", \"S\", \"protein-mediated\", \"membrane\", \"fusion\", \"will\", \"advance\", \"our\", \"understanding\", \"of\", \"cellular\", \"antiviral\", \"defenses\", \"against\", \"these\", \"important\", \"human\", \"pathogens\", \".\"]\n",
      "Word token=> [\"Antiviral\", \"membrane\", \"fusion\", \"inhibitors\", \"have\", \"been\", \"successfully\", \"implemented\", \"for\", \"treatment\", \"of\", \"HIV-\", \"1\", \"infection\", \"30\", \".\"]\n",
      "Word token=> [\"A\", \"therapeutic\", \"approach\", \"mimicking\", \"the\", \"mechanism\", \"of\", \"action\", \"of\", \"LY\", \"6\", \"E\", \"could\", \"provide\", \"a\", \"first\", \"line\", \"of\", \"defense\", \"against\", \"novel\", \"emerging\", \"CoV\", \"infections\", \".\"]\n",
      "Word token=> [\"Furthermore\", \",\", \"delineating\", \"which\", \"Ly\", \"6\", \"e-expressing\", \"immune\", \"cells\", \"protect\", \"mice\", \"from\", \"MHV\", \"will\", \"provide\", \"new\", \"insight\", \"into\", \"how\", \"individual\", \"antiviral\", \"effectors\", \"in\", \"distinct\", \"cellular\", \"compartments\", \"modulate\", \"viral\", \"pathogenesis\", \".\"]\n",
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> [\"34\", \"-\", \"41\", \"(\", \"1996\", \")\", \".\"]\n",
      "Word token=> [\"Welch\", \"'\", \"s\", \"correction\", \"(\", \"c-d\", \",\", \"f-g\", \")\", \",\", \"one-tailed\", \"Mann-Whitney\", \"U\", \"test\", \"(\", \"e\", \")\", \",\", \"ratio\", \"paired\", \"student\", \"'\", \"s\", \"t-test\", \"(\", \"g\", \")\", \",\", \"2\", \"way\"]\n",
      "Word token=> [\"ANOVA\", \"followed\", \"by\", \"Sidak\", \"'\", \"s\", \"or\", \"Dunnett\", \"'\", \"s\", \"multiple\", \"comparison\", \"test\", \"(\", \"j\", \",\", \"l\", \")\", \".\"]\n",
      "Word token=> [\"Error\", \"bars\", \":\", \"SD\", \".\"]\n",
      "Word token=> [\"P\", \"values\", \":\", \"c\", \",\", \"*\", \"*\", \"p=\", \"0.0088\", \";\", \"d\", \",\", \"*\", \"*\", \"*\", \"p=\", \"0.0001\", \";\", \"e\", \",\", \"*\", \"p=\", \"0.0143\", \";\", \"f\", \",\", \"*\", \"p=\", \"0.0286\", \";\", \"g\", \",\", \"*\", \"p=\", \"0.0101\", \";\", \"h\", \",\", \"*\", \"*\", \"*\", \"p=\", \"0.0010\", \";\", \"j\", \",\", \"*\", \"*\", \"p=\", \"0.0073\", \",\", \"p=\", \"0.0033\", \";\", \"l\", \",\", \"ns=\", \"0.0740\", \",\", \"*\", \"*\", \"p=\", \"0.0013\", \",\", \"*\", \"*\", \"*\", \"p=\", \"0.0001\", \".\"]\n",
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"p=<\", \"0.0001\", \";\", \"c\", \",\", \"*\", \"*\", \"p=\", \"0.0066\", \";\", \"d\", \",\", \"*\", \"*\", \"*\", \"*\", \"p=<\", \"0.0001\", \";\", \"f\", \",\", \"*\", \"*\", \"*\", \"*\", \"p=<\", \"0.0001\", \";\", \"h\", \",\", \"ns\", \"p=\", \"0.9975\", \",\", \"*\", \"*\", \"p=\", \"0.0054\", \",\", \"ns\", \"p=\", \"0.1520\", \",\", \"0.9892\", \".\"]\n",
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"scale\", \"bars\", \"are\", \"500\", \"µM\", \"(\", \"4\", \"x\", \")\", \"and\", \"100\", \"µM\", \"(\", \"20\", \"x\", \")\", \".\"]\n",
      "Word token=> [\"Statistical\", \"significance\", \"was\", \"determined\", \"by\", \"Mantel-Cox\", \"test\", \"(\", \"a\", \")\", \",\", \"two-tailed\", \"unpaired\", \"student\", \"'\", \"s\", \"t-test\", \"with\", \"Welch\", \"'\", \"s\", \"correction\", \"(\", \"c-e\", \",\", \"i-k\", \")\", \",\", \"two-tailed\", \"Mann-Whitney\", \"U\", \"test\", \"(\", \"f-g\", \",\", \"l-m\", \")\", \".\"]\n",
      "Word token=> [\"Error\", \"bars\", \":\", \"SEM\", \"(\", \"b\", \")\", \",\", \"SD\", \"(\", \"c-g\", \",\", \"i-m\", \")\", \".\"]\n",
      "Word token=> [\"P\", \"values\", \":\", \"a\", \",\", \"*\", \"*\", \"*\", \"p=\", \"0.0002\", \";\", \"c\", \",\", \"*\", \"*\", \"p=\", \"0.0031\", \";\", \"d\", \",\", \"ns\", \"p=\", \"0.2136\", \";\", \"e\", \",\", \"*\", \"*\", \"*\", \"*\", \"p=\", \"5.759\", \"x\", \"10\", \"-\", \"7\", \";\", \"f\", \",\", \"ns\", \"p=\", \"0.3698\", \";\", \"g\", \",\", \"*\", \"*\", \"p=\", \"0.0054\", \";\", \"i\", \",\", \"*\", \"p=\", \"0.0430\", \";\", \"j\", \",\", \"ns\", \"p=\", \"0.1321\", \";\", \"k\", \",\", \"*\", \"*\", \"p=\", \"0.0094\", \";\", \"l\", \",\", \"*\", \"*\", \"p=\", \"0.0072\", \";\", \"m\", \",\", \"*\", \"*\", \"*\", \"*\", \"p=\", \"8.741\", \"x\", \"10\", \"-\", \"5\", \".\"]\n",
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"maps\", \"displaying\", \"significant\", \"changes\", \"(\", \"mean\", \"RPKM\", \">\", \"0.5\", \";\", \"fold\", \"change\", \">\", \"2\", \";\", \"FDR\", \"≤\", \"0.05\", \")\", \"in\", \"liver\", \"(\", \"a\", \")\", \"and\", \"spleen\", \"(\", \"b\", \")\", \".\"]\n",
      "Word token=> [\"Dendrograms\", \"are\", \"normalized\", \"read\", \"data\", \"(\", \"row\", \"z-score\", \")\", \"clustered\", \"with\", \"complete\", \"linkage\", \"method\", \"employing\", \"Spearman\", \"Rank\", \"correlation\", \"distance\", \"measurement\", \".\"]\n",
      "Word token=> [\"c\", \",\", \"Pathway\", \"analysis\", \"of\", \"KO\", \"versus\", \"WT\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Up-\", \"(\", \"red\", \")\", \"or\", \"down-regulated\", \"(\", \"blue\", \")\", \"pathways\", \"indicated\", \"by\", \"activation\", \"z-score\", \".\"]\n",
      "Word token=> [\"Numbers\", \"show\", \"significantly\", \"dysregulated\", \"genes\", \"as\", \"percentage\", \"of\", \"total\", \"gene\", \"number\", \"included\", \"in\", \"pathway\", \".\"]\n",
      "Word token=> [\"d-e\", \",\", \"Expression\", \"of\", \"select\", \"genes\", \"in\", \"liver\", \"(\", \"d\", \")\", \"and\", \"spleen\", \"(\", \"e\", \")\", \".\"]\n",
      "Word token=> [\"f-g\", \",\", \"Immune\", \"cell\", \"counts\", \"from\", \"liver\", \"(\", \"f\", \")\", \"and\", \"spleen\"]\n",
      "Word token=> [\"HCoV-\", \"229\", \"E\", \":\", \"1.5\", \"x\", \"10\", \"5\", \"stable\", \"LY\", \"6\", \"E\", \"expressing\", \"or\", \"control\", \"cells\", \"were\", \"seeded\", \"in\", \"a\", \"12\", \"-well\", \"plate\", \"and\", \"infected\", \"with\", \"HCoV-\", \"229\", \"E\", \"in\", \"a\", \"serial\", \"dilution\", \".\"]\n",
      "Word token=> [\"Cells\", \"were\", \"incubated\", \"for\", \"2\", \"hours\", \"at\", \"33\", \"°\", \"C\", \",\", \"then\", \"viral\", \"supernatant\", \"was\", \"removed\", \",\", \"and\", \"an\", \"overlay\", \"of\", \"1:1\", \"Avicel\", \"(\", \"2.4\", \"%\", \")\", \"(\", \"Avicel\", \"RC-\", \"581\", \"NF\", \")\", \"and\", \"2\", \"x\", \"DMEM\", \",\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"supplemented\", \"with\", \"10\", \"%\", \"FBS\", \"and\", \"1\", \"x\", \"penicillin/streptomycin\", \"(\", \"p-s\", \")\", \",\", \"was\", \"added\", \".\"]\n",
      "Word token=> [\"Cells\", \"were\", \"incubated\", \"for\", \"3\", \"days\", \"at\", \"33\", \"°\", \"C\", \"before\", \"medium\", \"was\", \"removed\", \",\", \"cells\", \"were\", \"washed\", \",\", \"and\", \"stained\", \"with\", \"crystal\", \"violet\", \"(\", \"Sigma-Aldrich\", \")\", \".\"]\n",
      "Word token=> [\"Plaque\", \"forming\", \"units\", \"(\", \"PFU\", \")\", \"were\", \"calculated\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"HCoV-OC\", \"43\", \":\", \"5.0\", \"x\", \"10\", \"4\", \"stable\", \"LY\", \"6\", \"E\", \"expressing\", \"or\", \"control\", \"cells\", \"were\", \"seeded\", \"in\", \"a\", \"24\", \"-well\", \"plate\", \"and\", \"infected\", \"at\", \"33\", \"°\", \"C\", \"for\", \"1\", \"hour\", \"with\", \"HCoV-OC\", \"43\", \"(\", \"MOI=\", \"1\", \")\", \".\"]\n",
      "Word token=> [\"Virus\", \"was\", \"aspirated\", \"and\", \"10\", \"%\", \"FBS/\", \"1\", \"x\", \"NEAA/\", \"1\", \"x\", \"p-s/RPMI\", \"(\", \"cRPMI\", \")\", \"was\", \"added\", \"back\", \"to\", \"cells\", \".\"]\n",
      "Word token=> [\"24\", \"hours\", \"post-infection\", \",\", \"cells\", \"were\", \"dissociated\", \"using\", \"Accumax\", \"(\", \"Sigma-Aldrich\", \")\", \",\", \"fixed\", \"in\", \"1\", \"%\", \"PFA\", \",\", \"permeabilized\", \"per\", \"manufacturer\", \"'\", \"s\", \"protocol\", \"(\", \"BD\", \"Cytofix/Cytoperm\", \")\", \",\", \"and\", \"stained\", \"for\", \"nucleoprotein\", \"(\", \"1:500\", \")\", \"and\", \"a\", \"goat\", \"anti-mouse\", \"AlexaFluor\", \"488\", \"-\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"anti-mouse\", \"secondary\", \"antibody\", \".\"]\n",
      "Word token=> [\"Images\", \"were\", \"acquired\", \"with\", \"a\", \"fluorescence\", \"microscope\", \"and\", \"analyzed\", \"using\", \"ImageXpress\", \"Micro\", \"XLS\", \"(\", \"Molecular\", \"Devices\", \",\", \"Sunnyvale\", \",\", \"CA\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"For\", \"infection\", \"assays\", \",\", \"1\", \"x\", \"10\", \"5\", \"target\", \"cells\", \"were\", \"seeded\", \"in\", \"a\", \"24\", \"-well\", \"plate\", \"one\", \"day\", \"prior\", \"infection\", \".\"]\n",
      "Word token=> [\"Cells\", \"were\", \"infected\", \"with\", \"HCoV-\", \"229\", \"E-Rluc\", \"(\", \"MOI\", \"0.1\", \")\", \"in\", \"OptiMEM\", \"(\", \"Gibco\", \")\", \"for\", \"2\", \"hours\", \"at\", \"33\", \"°\", \"C.\", \"Cells\", \"were\", \"washed\", \"1\", \"x\", \"with\", \"PBS\", \"and\", \"10\", \"%\", \"FBS/\", \"1\", \"x\", \"NEAA/\", \"1\", \"x\", \"p-s/DMEM\", \"(\", \"cDMEM\", \")\", \"was\", \"added\", \"back\", \".\"]\n",
      "Word token=> [\"Cells\", \"were\", \"incubated\", \"at\", \"33\", \"°\", \"C\", \"for\", \"24\", \"hours\", \",\", \"then\", \"washed\", \"with\", \"PBS\", \"and\", \"lysed\", \"using\", \"the\", \"Renilla\", \"Luciferase\", \"Assay\", \"System\", \"kit\", \"(\", \"Promega\", \")\", \".\"]\n",
      "Word token=> [\"Rluc\", \"activity\", \"was\", \"measured\", \"using\", \"a\", \"plate\", \"luminometer\", \"(\", \"EnSpire\", \"2300\", \"Multilabel\", \"reader\", \"by\", \"Perkin\", \"Elmer\", \")\", \".\"]\n",
      "Word token=> [\"For\", \"the\", \"reconstitution\", \"with\", \"CRISPR\", \"resistant\", \"LY\", \"6\", \"E\", \"(\", \"CR\", \"LY\", \"6\", \"E\", \")\", \",\", \"2.5\", \"x\", \"10\", \"4\", \"cells\", \"were\", \"seeded\", \"in\", \"a\", \"24\", \"-well\", \"plate\", \".\"]\n",
      "Word token=> [\"One\", \"day\", \"post-seeding\", \",\", \"cells\", \"were\", \"either\", \"left\", \"untransduced\", \"or\", \"transduced\", \"with\", \"a\", \"lentiviral\", \"vector\", \"encoding\", \"for\", \"CR\", \"LY\", \"6\", \"E\", \"or\", \"the\", \"empty\", \"control\", \".\"]\n",
      "Word token=> [\"48\", \"hours\", \"post-transduction\", \",\", \"cell\", \"lysates\", \"were\", \"either\", \"infected\", \"with\", \"HCoV-\", \"229\", \"E-Rluc\", \"(\", \"MOI=\", \"0.1\", \")\", \"for\", \"24\", \"hours\", \"as\", \"described\", \"above\", \"or\", \"harvested\", \"for\", \"Western\", \"blot\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Pseudotyping\", \"of\", \"VSV\", \"*\", \"ΔG\", \"(\", \"Fluc\", \")\", \"was\", \"performed\", \"as\", \"previously\", \"described\", \"50\", \"with\", \"two\", \"silent\", \"mutation\", \"(\", \"T\", \"2568\", \"G\", \",\", \"T\", \"3327\", \"C\", \")\", \")\", \",\", \"have\", \"been\", \"described\", \"previously\", \"[\", \"51\", \"]\", \"[\", \"52\", \"]\", \"[\", \"53\", \"]\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"The\", \"expression\", \"plasmid\", \"for\", \"SARS-CoV-\", \"2\", \"was\", \"generated\", \"as\", \"follows\", \":\", \"the\", \"coding\", \"sequence\", \"of\", \"a\", \"synthetic\", \",\", \"codonoptimized\", \"(\", \"for\", \"human\", \"cells\", \")\", \"SARS-CoV-\", \"2\", \"DNA\", \"(\", \"GeneArt\", \"Gene\", \"Synthesis\", \",\", \"Thermo\", \"Fisher\", \"Scientific\", \")\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"based\", \"on\", \"the\", \"publicly\", \"available\", \"protein\", \"sequence\", \"in\", \"the\", \"National\", \"Center\", \"for\", \"Biotechnology\", \"Information\", \"database\", \"(\", \"NCBI\", \"Reference\", \"Sequence\", \":\", \"YP\", \"_\", \"009724390.1\", \")\", \"was\", \"PCR-amplified\", \"and\", \"cloned\", \"into\", \"the\", \"pCG\", \"1\", \"expression\", \"vector\", \"between\", \"BamHI\", \"and\", \"XbaI\", \"restriction\", \"sites\", \".\"]\n",
      "Word token=> [\"The\", \"MERS-CoV\", \"S\", \"cleavage\", \"mutants\", \"have\", \"been\", \"described\", \"before\", \"17\"]\n",
      "Word token=> [\"The\", \"split\", \"luciferase\", \"fusion\", \"assay\", \"has\", \"been\", \"described\", \"before\", \",\", \"with\", \"slight\", \"modifications\", \"54\", \".\"]\n",
      "Word token=> [\"Briefly\", \",\", \"6\", \"x\", \"10\", \"5\", \"293\", \"LTV\", \"cells\", \"were\", \"transfected\", \"with\", \"plasmids\", \"encoding\", \"CoV\", \"S\", \"proteins\", \"(\", \"pCAGGS-HCoV-\", \"229\", \"E\", \",\", \"pCAGGS-MERS-CoV\", \")\", \"together\", \"with\", \"a\", \"split-luciferase\", \"construct\", \"(\", \"Rluc\", \"8155\", \"-\", \"156\", \"DSP\", \"1\", \"-\", \"7\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"empty\", \"expression\", \"plasmid\", \"(\", \"pCAGGS-mCherry\", \")\", \"as\", \"well\", \"as\", \"a\", \"construct\", \"encoding\", \"for\", \"VSV\", \"G\", \"served\", \"as\", \"negative\", \"and\", \"positive\", \"control\", \",\", \"respectively\", \".\"]\n",
      "Word token=> [\"2\", \"x\", \"10\", \"5\", \"stable\", \"LY\", \"6\", \"E\", \"expressing\", \"or\", \"empty\", \"control\", \"Huh\", \"7\", \"cells\", \"were\", \"transfected\", \"using\", \"Lipofectamine\", \"2000\", \"(\", \"Thermo\", \"Fisher\", \"Scientific\", \")\", \"with\", \"a\", \"plasmid\", \"encoding\", \"the\", \"second\", \"half\", \"of\", \"the\", \"split-luciferase\", \"protein\", \"(\", \"Rluc\", \"8155\", \"-\", \"156\", \"DSP\", \"8\", \"-\", \"11\", \")\", \".\"]\n",
      "Word token=> [\"Approximately\", \"30\", \"hours\", \"post-transfection\", \",\", \"both\", \"cell\", \"populations\", \"were\", \"dissociated\", \"(\", \"TryPLE\", \"Express\", \",\", \"Gibco\", \")\", \",\", \"counted\", \",\", \"and\", \"equal\", \"cell\", \"numbers\", \"(\", \"2\", \"x\", \"10\", \"4\", \"cells\", \"each\", \")\", \"were\", \"co-cultured\", \"for\", \"16\", \"-\", \"20\", \"hours\", \"at\", \"37\", \"°\", \"C.\", \"Supernatant\", \"was\", \"harvested\", \"and\", \"cells\", \"lysed\", \"using\", \"ice-cold\", \"passive\", \"lysis\", \"buffer\", \"(\", \"Promega\", \")\", \".\"]\n",
      "Word token=> [\"Cells\", \"were\", \"immediately\", \"transferred\", \"to\", \"-\", \"80\", \"°\", \"C\", \"for\", \"at\", \"least\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"1\", \"hour\", \",\", \"before\", \"Rluc\", \"activity\", \"was\", \"determined\", \"using\", \"the\", \"Renilla\", \"Luciferase\", \"Assay\", \"System\", \"(\", \"Promega\", \")\", \".\"]\n",
      "Word token=> [\"Cells\", \"were\", \"kept\", \"on\", \"ice\", \"during\", \"all\", \"times\", \"post-cell\", \"lysis\", \"to\", \"eliminate\", \"DSP\", \"post-lysis\", \"complementation\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Following\", \"extraction\", \"of\", \"total\", \"RNA\", \"from\", \"MERS-CoV\", \"(\", \"strain\", \"EMC\", \")\", \"infected\", \"VeroE\", \"6\", \"cells\", \",\", \"the\", \"cDNA\", \"encoding\", \"the\", \"MERS-CoV\", \"S\", \"protein\", \"was\", \"generated\", \"by\", \"reverse\", \"transcription\", \"(\", \"RevertAid\", \"Premium\", \"Reverse\", \"Transkriptase\", \",\", \"ThermoScientific\", \")\", \".\"]\n",
      "Word token=> [\"Three\", \"overlapping\", \"cDNA\", \"fragments\", \"were\", \"amplified\", \"by\", \"PCR\", \"(\", \"Phusion\", \"Hot\", \"Start\", \"II\", \"High\", \"Fidelity\", \"Polymerase\", \",\", \"Thermo\", \"Scientific\", \")\", \"and\", \"subsequently\", \"assembled\", \"by\", \"overlapping\", \"PCR\", \".\"]\n",
      "Word token=> [\"The\", \"cDNAs\", \"encoding\", \"the\", \"S\", \"proteins\", \"of\", \"either\", \"MERS-CoV\", \"(\", \"strain\", \"EMC\", \")\", \"or\", \"HCoV-\", \"229\", \"E\", \"(\", \"truncated\", \"variant\", \"lacking\", \"the\", \"10\", \"amino\", \"acids\", \"at\", \"the\", \"C\", \"terminus\", \",\", \"original\", \"plasmid\", \"pCAGGS-\", \"229\", \"E\", \"S\", \")\", \"were\", \"amplified\", \"by\", \"PCR\", \"and\", \"inserted\", \"into\", \"the\", \"pVSV\", \"*\", \"plasmid\", \"55\", \"between\", \"MluI\", \"and\", \"BstEII\", \"restriction\", \"sites\", \".\"]\n",
      "Word token=> [\"The\", \"resulting\", \"plasmids\", \"were\", \"used\", \"to\", \"generate\", \"the\", \"recombinant\", \"viruses\", \"VSV\", \"*\", \"ΔG\", \"(\", \"MERS\", \"S\", \")\", \"and\", \"VSV\", \"*\", \"ΔG\", \"(\", \"229\", \"E\", \"S\", \")\", \"according\", \"to\", \"a\", \"published\", \"procedure\", \"56\", \".\"]\n",
      "Word token=> [\"All\", \"viruses\", \"were\", \"propagated\", \"on\", \"BHK-G\", \"43\", \"cells\", \"57\", \",\", \"resulting\", \"in\", \"viruses\", \"predominantly\", \"containing\", \"the\", \"homotypic\", \"VSV\", \"glycoprotein\", \"G\", \"in\", \"the\", \"envelope\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Cells\", \"expressing\", \"LY\", \"6\", \"E\", \"or\", \"empty\", \"control\", \"were\", \"infected\", \"with\", \"VSV\", \"*\", \"ΔG\", \"(\", \"MERS\", \"S\", \")\", \"or\", \"VSV\", \"*\", \"ΔG\", \"(\", \"229\", \"E\", \"S\", \")\", \"at\", \"a\", \"MOI\", \"of\", \"0.01\", \".\"]\n",
      "Word token=> [\"At\", \"20\", \"hours\", \"post-infection\", \",\", \"the\", \"cells\", \"were\", \"fixed\", \"with\", \"3\", \"%\", \"PFA/PBS\", \"and\", \"the\", \"nuclei\", \"stained\", \"with\", \"4\", \"'\", \",\", \"6\", \"-diamidine-\", \"2\", \"'\", \"-phenylindole\", \"(\", \"DAPI\", \",\", \"Sigma\", \")\", \".\"]\n",
      "Word token=> [\"An\", \"inverted\", \"fluorescence\", \"microscope\", \"(\", \"Zeiss\", \")\", \"was\", \"used\", \"to\", \"determine\", \"area\", \"percentage\", \"area\", \"covered\", \"by\", \"syncytia\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"For\", \"heterologous\", \"cell-cell\", \"fusion\", \"assay\", \",\", \"BHK-\", \"21\", \"cells\", \"were\", \"infected\", \"with\", \"VSV\", \"*\", \"ΔG\", \"(\", \"CoV\", \"S\", \")\", \"at\", \"a\", \"MOI\", \"of\", \"1\", \".\"]\n",
      "Word token=> [\"After\", \"2\", \"hours\", \",\", \"the\", \"cells\", \"were\", \"treated\", \"with\", \"trypsin/EDTA\", \"(\", \"Life\", \"Technologies\", \",\", \"Zug\", \",\", \"Switzerland\", \")\", \"and\", \"resuspended\", \"in\", \"DMEM\", \"supplemented\", \"with\", \"5\", \"%\", \"FBS\", \"(\", \"2\", \"x\", \"10\", \"4\", \"cells/mL\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"infected\", \"BHK-\", \"21\", \"cell\", \"suspension\", \"(\", \"500\", \"µL\", \")\", \"was\", \"seeded\", \"into\", \"24\", \"-well\", \"cell\", \"culture\", \"plates\", \"along\", \"with\", \"LY\", \"6\", \"E-or\", \"empty\", \"controlexpressing\", \"Huh\", \"7\", \"(\", \"1\", \"x\", \"10\", \"4\", \"cells\", \")\", \".\"]\n",
      "Word token=> [\"Cell\", \"co-cultures\", \"were\", \"incubated\", \"for\", \"7\", \"hours\", \"at\", \"37\", \"°\", \"C\", \",\", \"fixed\", \"with\", \"3.6\", \"%\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"formaldehyde\", \"diluted\", \"in\", \"PBS\", \",\", \"(\", \"Grogg-Chemie\", \"AG\", \",\", \"Stettlen\", \",\", \"Switzerland\", \")\", \"and\", \"stained\", \"with\", \"DAPI\", \".\"]\n",
      "Word token=> [\"The\", \"percentage\", \"area\", \"covered\", \"by\", \"syncytia\", \"was\", \"calculated\", \"as\", \"above\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Ly\", \"6\", \"e\", \"tm\", \"1\", \"a\", \"ES\", \"cells\", \"were\", \"obtained\", \"from\", \"the\", \"EUCOMM\", \"consortium\", \"58\"]\n",
      "Word token=> [\"Six\", \"to\", \"twelve-week-old\", \"female\", \"and\", \"male\", \"mice\", \"were\", \"injected\", \"intraperitoneally\", \"with\", \"MHV-A\", \"59\", \"diluted\", \"in\", \"PBS\", \"to\", \"the\", \"indicated\", \"titers\", \"or\", \"PBS\", \"for\", \"a\", \"mock\", \"infection\", \"control\", \".\"]\n",
      "Word token=> [\"All\", \"infected\", \"mice\", \"were\", \"monitored\", \"daily\", \"for\", \"weight\", \"and\", \"mortality\", \".\"]\n",
      "Word token=> [\"Animals\", \"that\", \"lost\", \"more\", \"than\", \"20\", \"%\", \"of\", \"their\", \"original\", \"body\", \"weight\", \"were\", \"euthanized\", \"per\", \"IACUC\", \"guidelines\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Viral\", \"titers\", \"in\", \"liver\", \"and\", \"spleen\", \"were\", \"determined\", \"from\", \"frozen\", \"organs\", \"after\", \"weighing\", \",\", \"homogenization\", \",\", \"and\", \"plaque\", \"assay\", \"on\", \"L\", \"929\", \"cells\", \".\"]\n",
      "Word token=> [\"Alanine\", \"aminotransferase\", \"(\", \"ALT\", \")\", \"was\", \"measured\", \"in\", \"fresh\", \",\", \"unfrozen\", \"serum\", \"using\", \"VITROS\", \"MicroSlide\", \"Technology\", \"by\", \"the\", \"UTSW\", \"Mouse\", \"Metabolic\", \"Core\", \".\"]\n",
      "Word token=> [\"Livers\", \"were\", \"fixed\", \"in\", \"10\", \"%\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"neutral\", \"buffer\", \"formalin\", \",\", \"embedded\", \"in\", \"paraffin\", \",\", \"sectioned\", \"at\", \"5\", \"µM\", \",\", \"and\", \"stained\", \"with\", \"hematoxylin\", \"and\", \"eosin\", \"(\", \"H\", \"&\", \"E\", \")\", \".\"]\n",
      "Word token=> [\"Slides\", \"were\", \"analyzed\", \"by\", \"an\", \"independent\", \"pathologist\", \"(\", \"UTSW\", \"Animal\", \"Resource\", \"Center\", \")\", \"who\", \"was\", \"blinded\", \"to\", \"experimental\", \"conditions\", \".\"]\n",
      "Word token=> [\"A\", \"numerical\", \"score\", \"was\", \"assigned\", \"for\", \"degree\", \"of\", \"inflammation\", \"and\", \"degree\", \"of\", \"necrosis\", \"for\", \"each\", \"liver\", \".\"]\n",
      "Word token=> [\"Inflammatory\", \"cell\", \"infiltration\", \"was\", \"scored\", \"using\", \"the\", \"following\", \"criteria\", \":\", \"0\", \"=\", \"none\", \",\", \"1\", \"=\", \"minimal\", \",\", \"2\", \"=\", \"mild\", \",\", \"3\", \"=\", \"moderate\", \",\", \"and\", \"4\", \"=\", \"marked\", \".\"]\n",
      "Word token=> [\"The\", \"necrosis\", \"score\", \"was\", \"defined\", \"by\", \"the\", \"percent\", \"necrosis\", \".\"]\n",
      "Word token=> [\"0\", \"=\", \"none\", \",\", \"1\", \"=\", \"<\", \"2\", \"%\", \",\", \"2\", \"=\", \"2\", \"-\", \"20\", \"%\", \",\", \"3\", \"=\", \"21\", \"-\", \"40\", \"%\", \",\", \"and\", \"4\", \"=\", \">\", \"40\", \"%\", \".\"]\n",
      "Word token=> String[]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"All\", \"primary\", \"cells\", \"were\", \"maintained\", \"in\", \"10\", \"%\", \"FBS/\", \"10\", \"mM\", \"HEPES/\", \"1\", \"mM\", \"sodium\", \"pyruvate/\", \"2\", \"mM\", \"Lglutamine/\", \"1\", \"x\", \"p-s/RPMI\", \"(\", \"cRPMI\", \"2\", \")\", \"unless\", \"otherwise\", \"indicated\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Primary\", \"bone\", \"marrow\", \"derived\", \"macrophages\", \"(\", \"BMDM\", \")\", \"were\", \"prepared\", \"as\", \"described\", \"previously\", \"60\", \".\"]\n",
      "Word token=> [\"Viable\", \"cells\", \"were\", \"quantified\", \"using\", \"trypan\", \"blue\", \"exclusion\", \".\"]\n",
      "Word token=> [\"BMDM\", \"were\", \"plated\", \"at\", \"2.5\", \"x\", \"10\", \"4\", \"cells\", \"per\", \"well\", \"in\", \"non-\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"separated\", \"from\", \"IHIC\", \"by\", \"centrifugation\", \"at\", \"850\", \"x\", \"g\", \"for\", \"30\", \"minutes\", \",\", \"22\", \"°\", \"C\", \",\", \"with\", \"no\", \"brake\", \".\"]\n",
      "Word token=> [\"Red\", \"blood\", \"cells\", \"were\", \"removed\", \"from\", \"the\", \"resulting\", \"pellet\", \"with\", \"RBC\", \"lysis\", \"buffer\", \"(\", \"Tonbo\", \"Biosciences\", \")\", \".\"]\n",
      "Word token=> [\"Viable\", \"cells\", \"were\", \"quantified\", \"using\", \"trypan\", \"blue\", \"exclusion\", \".\"]\n",
      "Word token=> [\"IHIC\", \"and\", \"splenocytes\", \"(\", \"4\", \"x\", \"10\", \"5\", \")\", \"from\", \"infected\", \"or\", \"mock-infected\", \"mice\", \"were\", \"stained\", \"with\", \"a\", \"fluorescent\", \"viability\", \"dye\", \"(\", \"Ghostdye\", \"Violet\", \"450\", \",\", \"Tonbo\", \"Biosciences\", \")\", \",\", \"treated\", \"with\", \"anti-CD\", \"16\", \"/CD\", \"32\", \"(\", \"Tonbo\", \"Biosciences\", \")\", \",\", \"stained\", \"with\", \"antibodies\", \"that\", \"recognize\", \"surface\", \"lineage\", \"markers\", \",\", \"and\", \"then\", \"fixed\", \"in\", \"1\", \"%\", \"PFA/PBS\", \".\"]\n",
      "Word token=> [\"Fixed\", \"volumes\", \"of\", \"cell\", \"suspensions\", \"were\", \"analyzed\", \"by\", \"flow\", \"cytometry\", \"the\", \"next\", \"day\", \".\"]\n",
      "Word token=> [\"Absolute\", \"cell\", \"counts\", \"were\", \"determined\", \"by\", \"multiplying\", \"final\", \"number\", \"of\", \"lineage\", \"marker-positive\", \"cells\", \"by\", \"dilution\", \"factor\", \"used\", \"to\", \"normalize\", \"cell\", \"counts\", \"for\", \"antibody\", \"staining\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"All\", \"primary\", \"cell\", \"samples\", \"were\", \"resuspended\", \"in\", \"3\", \"%\", \"FBS/PBS\", \"and\", \"analyzed\", \"using\", \"a\", \"S\", \"1000\", \"Flow\", \"Cytometer\", \"with\", \"a\", \"A\", \"600\", \"96\", \"-well\", \"plate\", \"high\", \"throughput\", \"extension\", \"and\", \"compensated\", \"using\", \"CellCapture\", \"software\", \"(\", \"Stratedigm\", \")\", \".\"]\n",
      "Word token=> [\"Data\", \"was\", \"analyzed\", \"with\", \"FlowJo\", \"Software\", \"(\", \"Treestar\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"flow\", \"gating\", \"strategy\", \"for\", \"liver\", \"and\", \"spleen\", \"immune\", \"cells\", \"is\", \"included\", \"in\", \"Extended\", \"Data\", \"Figure\", \"9\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Differences\", \"in\", \"data\", \"were\", \"tested\", \"for\", \"significance\", \"using\", \"GraphPad\", \"Prism\", \"v\", \"8.3.1\", \"for\", \"Windows\", \"(\", \"GraphPad\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"For\", \"details\", \"regarding\", \"the\", \"statistical\", \"tests\", \"applied\", \",\", \"please\", \"refer\", \"to\", \"the\", \"figure\", \"legends\", \".\"]\n",
      "Word token=> [\"P\", \"values\", \"<\", \"0.05\", \"were\", \"considered\", \"significant\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"authors\", \"declare\", \"that\", \"the\", \"data\", \"supporting\", \"the\", \"findings\", \"of\", \"this\", \"study\", \"are\", \"available\", \"within\", \"the\", \"article\", \"and\", \"its\", \"Supplementary\", \"Information\", \"files\", \"or\", \"are\", \"available\", \"on\", \"request\", \".\"]\n",
      "Word token=> [\"The\", \"RNAseq\", \"data\", \"discussed\", \"in\", \"this\", \"publication\", \"have\", \"been\", \"deposited\", \"in\", \"the\", \"Gene\", \"Expression\", \"Omnibus\", \"(\", \"GEO\", \")\", \"database\", \",\", \"https://www.ncbi.nlm.nih.gov/geo\", \"(\", \"GSE\", \"146074\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\"]\n",
      "Word token=> [\"We\", \"would\", \"like\", \"to\", \"thank\", \"the\", \"following\", \"people\", \"for\", \"their\", \"generous\", \"contribution\", \"of\", \"reagents\", \"and\", \"infrastructure\", \"which\", \"greatly\", \"enhanced\", \"this\", \"study\", \",\", \"especially\", \"the\", \"sequencing\", \"facility\", \"in\", \"Bern\", \",\", \"the\", \"diagnostic\", \"facility\", \"of\", \"IVI\", \",\", \"and\", \"the\", \"Animal\", \"Resource\", \"Center\", \"at\", \"UTSW\", \".\"]\n",
      "Word token=> [\"Furthermore\", \",\", \"we\", \"are\", \"grateful\", \"to\"]\n",
      "Word token=> [\"The\", \"authors\", \"declare\", \"no\", \"competing\", \"interests\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Supplementary\", \"Information\", \"is\", \"available\", \"for\", \"this\", \"paper\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Correspondence\", \"and\", \"requests\", \"for\", \"materials\", \"should\", \"be\", \"addressed\", \"to\", \"Volker\", \"Thiel\", \",\", \"John\", \"W.\", \"Schoggins\", \",\", \"and\", \"Charles\", \"M.\", \"Rice\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"was\", \"harvested\", \"and\", \"extracellular\", \"viral\", \"RNA\", \"was\", \"extracted\", \"and\", \"viral\", \"replication\", \"detected\", \"via\", \"qRT-PCR\", \"(\", \"f\", \")\", \".\"]\n",
      "Word token=> [\"Cell\", \"lysates\", \"were\", \"harvested\", \"and\", \"intracellular\", \"Renilla\", \"luciferase\", \"activity\", \"was\", \"detected\", \"upon\", \"cell\", \"lysis\", \"(\", \"g\", \")\", \".\"]\n",
      "Word token=> [\"Cells\", \"were\", \"subjected\", \"to\", \"3\", \"rounds\", \"of\", \"freeze/thaw\", \"cycles\", \".\"]\n",
      "Word token=> [\"Cell\", \"debris\", \"was\", \"removed\", \"and\", \"the\", \"supernatant\", \"titrated\", \"on\", \"naïve\", \"Huh\", \"7\", \"cells\", \".\"]\n",
      "Word token=> [\"Intracellular\", \"infectivity\", \"was\", \"determined\", \"(\", \"h\", \")\", \".\"]\n",
      "Word token=> [\"Supernatant\", \"was\", \"harvested\", \"and\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"ANOVA\", \"followed\", \"by\", \"Sidak\", \"'\", \"s\", \"multiple\", \"comparisons\", \"test\", \"(\", \"c\", \")\", \".\"]\n",
      "Word token=> [\"Error\", \"bars\", \":\", \"SD\", \".\"]\n",
      "Word token=> [\"P\", \"values\", \":\", \"c\", \",\", \"*\", \"*\", \"*\", \"*\", \"p=<\", \"0.0001\", \"<\", \"0.0001\", \".\"]\n",
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"two\", \"pooled\", \"experiments\", \".\"]\n",
      "Word token=> [\"For\", \"i-m\", \",\", \"n=\", \"8\", \"(\", \"Ly\", \"6\", \"e\", \"fl/fl\", \",\", \")\", \",\", \"n=\", \"6\", \"(\", \"Ly\", \"6\", \"e\", \"ΔHSC\", \")\", \",\", \"or\", \"n=\", \"3\", \"(\", \"PBS\", \")\", \"from\", \"two\", \"pooled\", \"experiments\", \".\"]\n",
      "Word token=> [\"In\", \"h\", \",\", \"n\", \",\", \"scale\", \"bars\", \"are\", \"500\", \"µM\", \"(\", \"4\", \"x\", \")\", \"and\", \"100\", \"µM\", \"(\", \"20\", \"x\", \")\", \".\"]\n",
      "Word token=> [\"Significance\", \"for\", \"was\", \"tested\", \"by\"]\n",
      "Word token=> [\"Mann-Whitney\", \"U\", \"test\", \"(\", \"f-g\", \",\", \"l-m\", \")\", \".\"]\n",
      "Word token=> [\"Error\", \"bars\", \":\", \"SEM\", \"(\", \"b\", \")\", \",\", \"SD\", \"(\", \"c-g\", \",\", \"i-m\", \")\", \".\"]\n",
      "Word token=> [\"P\", \"value\", \":\", \"a\", \",\", \"*\", \"*\", \"*\", \"*\", \"p=\", \"3.946\", \"x\", \"10\", \"-\", \"6\", \";\", \"c\", \",\", \"ns\", \"p=\", \"1144\", \";\", \"d\", \",\", \"ns\", \"p=\", \"0.1407\", \";\", \"e\", \",\", \"*\", \"*\", \"p=\", \"0.0026\", \";\", \"f\", \",\", \"ns\", \"p=\", \"0.1708\", \";\", \"g\", \",\", \"ns\", \"p=\", \"0.0779\", \";\", \"i\", \",\", \"ns\", \"p=\", \"0.6191\", \";\", \"j\", \",\", \"ns\", \"p=\", \"0.6842\", \";\", \"k\", \",\", \"*\", \"p=\", \"0.0119\", \";\", \"l\", \",\", \"ns\", \"p>\", \"0.9999\", \";\", \"m\", \",\", \"ns\", \"p=\", \"0.0779\", \".\"]\n",
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"-c\", \",\", \"f-h\", \")\", \",\", \"two-tailed\", \"Mann-Whitney\", \"U\", \"test\", \"(\", \"d-e\", \",\", \"i-j\", \")\", \".\"]\n",
      "Word token=> [\"Error\", \"bars\", \":\", \"SD\", \".\"]\n",
      "Word token=> [\"P\", \"value\", \":\", \"a\", \",\", \"*\", \"p=\", \"0.0234\", \";\", \"b\", \",\", \"*\", \"p=\", \"0.0428\", \";\", \"c\", \",\", \"*\", \"*\", \"*\", \"p=\", \"0.0004\", \";\", \"d\", \",\", \"*\", \"p=\", \"0.0210\", \";\", \"e\", \",\", \"*\", \"*\", \"p=\", \"0.0054\", \";\", \"f\", \",\", \"ns\", \"p=\", \"0.1544\", \";\", \"g\", \",\", \"ns\", \"p=\", \"0.5322\", \";\", \"h\", \",\", \"ns\", \"p=\", \"0.0662\", \";\", \"i\", \",\", \"ns\", \"p=\", \"0.0592\", \";\", \"j\", \",\", \"ns\", \"p=\", \"0.9064\", \".\"]\n",
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"DDX\", \"3\", \"X\", \"-DDX\", \"3\", \",\", \"VIPERIN\", \"-RSAD\", \"2\", \",\", \"SERPINE\", \"1\", \"-PAI\", \"1\", \",\", \"IL\", \"28\", \"RA\", \"-CRF\", \"2\", \",\", \"VEGFC\", \"-VRP\", \",\", \"SP\", \"110\", \"-IFI\", \"41\", \",\", \"IRF\", \"2\", \"-PRIC\", \"285\", \",\", \"ZNF\", \"313\", \"-RNF\", \"114\", \",\", \"DNAPTP\", \"6\", \"-LOC\", \"26010\", \",\", \"LOC\", \"400759\", \"-PSEUDOGENE\", \",\", \"PSCD\", \"1\", \"-CYTH\", \"1\", \",\", \"IFITM\", \"1\", \"-IFI\", \"17\", \",\", \"SIGLEC\", \"1\", \"-FRAG\", \",\", \"CCL\", \"5\", \"-SYCA\", \"5\", \",\", \"CXCL\", \"9\", \"-MIG\", \",\", \"IFIH\", \"1\", \"Huh\", \"7\", \"cells\", \"expressing\", \"TMPRSS\", \"2\", \"were\", \"generated\", \"as\", \"follows\", \".\"]\n",
      "Word token=> [\"In\", \"order\", \"to\", \"generate\", \"TMPRSS\", \"2\", \"-encoding\", \"retroviral\", \"vectors\", \",\", \"the\", \"open\", \"reading\", \"frame\", \"of\", \"human\", \"TMPRRS\", \"2\", \"was\", \"first\", \"PCR\", \"amplified\", \"with\", \"primers\", \"adding\", \"an\", \"N-terminal\", \"cMYC\", \"epitope\", \"to\", \"the\", \"TMPRSS\", \"2\", \"coding\", \"sequence\", \".\"]\n",
      "Word token=> [\"The\", \"resulting\", \"sequence\", \"was\", \"inserted\", \"into\", \"a\", \"modified\", \"version\", \"of\", \"the\", \"pQCXIP\", \"plasmid\", \"that\", \"contains\", \"a\", \"blasticidin\", \"resistance\", \"cassette\", \"instead\", \"of\", \"the\", \"usual\", \"puromycin\", \"resistance\", \"cassette\", \"17\", \".\"]\n",
      "Word token=> [\"Huh\", \"7\", \"cells\", \"stably\", \"expressing\", \"human\", \"TMPRSS\", \"2\", \"were\", \"generated\", \"by\", \"retroviral\", \"transduction\", \"and\", \"selection\", \"with\", \"the\", \"antibiotic\", \"blasticidin\", \"(\", \"50\", \"µg/ml\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Following\", \"selection\", \",\", \"cells\", \"were\", \"maintained\", \"in\", \"culture\", \"medium\", \"(\", \"cDMEM\", \")\", \"supplemented\", \"with\", \"10\", \"µg/ml\", \"blasticidin\", \".\"]\n",
      "Word token=> [\"To\", \"generate\", \"STAT\", \"1\", \"-/-\", \"_\", \"CEACAM\", \"1\", \"cells\", \",\", \"human\", \"STAT\", \"1\", \"−/−\", \"fibroblasts\", \"were\", \"transduced\", \"with\", \"lentiviruses\", \"encoding\", \"for\", \"the\", \"murine\", \"CoV\", \"receptor\", \"CEACAM\", \"1\", \"(\", \"kind\", \"gift\", \"from\", \"David\", \"Wentworth\", \",\", \"CDC\", \",\", \"Atlanta\", \",\", \"USA\", \"59\", \"and\", \"subsequently\", \"selected\", \"with\", \"1\", \"µg/mL\", \"puromycin\", \".\"]\n",
      "Word token=> [\"Stable\", \"LY\", \"6\", \"E\", \"expressing\", \"cell\", \"lines\", \"were\", \"generated\", \"upon\", \"lentiviral\", \"transduction\", \"with\", \"SCRPSY\", \"LY\", \"6\", \"E\", \"or\", \"as\", \"control\", \"SCRPSY\", \"Empty\", \"in\", \"DMEM\", \"containing\", \"4\", \"µg/ml\", \"polybrene\", \"(\", \"Millipore\", \")\", \"and\", \"20\", \"mM\", \"HEPES\", \"buffer\", \"solution\", \"(\", \"Gibco\", \")\", \".\"]\n",
      "Word token=> [\"Cells\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"were\", \"selected\", \"using\", \"2\", \"-\", \"4\", \"µg/mL\", \"puromycin\", \"and\", \"were\", \"passaged\", \"until\", \"all\", \"cells\", \"in\", \"control\", \"wells\", \"without\", \"lentivirus\", \"were\", \"killed\", \".\"]\n",
      "Word token=> [\"The\", \"puromycin\", \"selected\", \"cells\", \"were\", \"further\", \"passaged\", \"and\", \"frozen\", \"down\", \"for\", \"subsequent\", \"experiments\", \".\"]\n",
      "Word token=> [\"The\", \"stable\", \"cell\", \"lines\", \"harboring\", \"LY\", \"6\", \"E\", \"orthologues\", \"have\", \"been\", \"described\", \"before\", \",\", \"with\", \"the\", \"exception\", \"of\", \"C.\", \"dromedarius\", \"9\", \".\"]\n",
      "Word token=> [\"For\", \"this\", \",\", \"a\", \"gBlock\", \"was\", \"ordered\", \"(\", \"XM\", \"_\", \"031439745.1\", \")\", \"and\", \"Gateway\", \"cloning\", \"performed\", \"as\", \"described\", \"previously\", \"to\", \"generate\", \"pENTR\", \"and\", \"pSCRPSY\", \"plasmids\", \"9\", \".\"]\n",
      "Word token=> [\"Constructs\", \"encoding\", \"for\", \"Ly\", \"6\", \"/uPAR\", \"family\", \"members\", \"and\", \"LY\", \"6\", \"E\", \"ASM\", \"mutants\", \"have\", \"been\", \"described\", \"before\", \"9\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"To\", \"generate\", \"clonal\", \"LY\", \"6\", \"E\", \"KO\", \"cells\", \"and\", \"CRISPR-resistant\", \"LY\", \"6\", \"E\", \",\", \"A\", \"549\", \"cells\", \"were\", \"transduced\", \"with\", \"lentivirus\", \"containing\", \"a\", \"LY\", \"6\", \"E-specific\", \"sgRNA\", \"and\", \"Cas\", \"9\", \"as\", \"described\", \"previously\", \"9\", \".\"]\n",
      "Word token=> [\"To\", \"generate\", \"a\", \"clonal\", \"cell\", \"line\", \",\", \"the\", \"bulk\", \"transduced\", \"population\", \"was\", \"plated\", \"at\", \"single\", \"cell\", \"dilutions\", \".\"]\n",
      "Word token=> [\"Candidate\", \"clones\", \"were\", \"screened\", \"by\", \"Western\", \"blot\", \"for\", \"LY\", \"6\", \"E\", \"expression\", \"and\", \"Sanger\", \"sequencing\", \".\"]\n",
      "Word token=> [\"Silent\", \"mutations\", \"in\", \"the\", \"region\", \"targeted\", \"by\", \"the\", \"LY\", \"6\", \"E-specific\", \"sgRNA\", \"were\", \"introduced\", \"into\", \"HA-tagged\", \"LY\", \"6\", \"E\", \"to\", \"generate\", \"CRISPRresistant\", \"LY\", \"6\", \"E\", \"(\", \"CR\", \"LY\", \"6\", \"E\", \")\", \".\"]\n",
      "Word token=> [\"LY\", \"6\", \"E\", \"KO\", \"A\", \"549\", \"cells\", \"were\", \"reconstituted\", \"with\", \"CR-LY\", \"6\", \"E\", \"by\", \"lentiviral\", \"transduction\", \"and\", \"expression\", \"confirmed\", \"by\", \"Western\", \"blot\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"All\", \"cell\", \"lines\", \"were\", \"regularly\", \"tested\", \"to\", \"check\", \"they\", \"were\", \"free\", \"of\", \"mycoplasma\", \"contamination\", \"using\", \"a\", \"commercially\", \"available\", \"system\", \"(\", \"PCR\", \"Mycoplasma\", \"test\", \"kit\", \"I/RT\", \"Variant\", \"C\", \",\", \"PromKine\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"ISG\", \"screen\", \"was\", \"performed\", \"as\", \"described\", \"previously\", \"with\", \"slight\", \"modifications\", \"7,8\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"Briefly\", \",\", \"5\", \"x\", \"10\", \"3\", \"Huh\", \"7\", \"cells\", \"were\", \"seeded\", \",\", \"transduced\", \"with\", \"individual\", \"lentiviruses\", \",\", \"and\", \"48\", \"hours\", \"post-transduction\", \"infected\", \"with\", \"HCoV-\", \"229\", \"E\", \"at\", \"33\", \"°\", \"C.\", \"Infection\", \"was\", \"stopped\", \"24\", \"hours\", \"(\", \"MOI=\", \"0.1\", \")\", \"or\", \"48\", \"hours\", \"(\", \"MOI=\", \"0.01\", \")\", \"postinfection\", \"and\", \"plates\", \"were\", \"immuno-stained\", \"as\", \"described\", \"previously\", \"61\", \"with\", \"an\", \"anti-HCoV-\", \"229\", \"E\", \"N\", \"protein\", \"antibody\", \"and\", \"a\", \"AlexaFluor\", \"488\", \"-conjugated\", \"donkey\", \"anti-mouse\", \"secondary\", \"antibody\", \"(\", \"Extended\", \"methods\", \";\", \"Antibodies\", \"for\", \"immunofluorescence\", \"and\", \"flow\", \"cytometry\", \")\", \".\"]\n",
      "Word token=> [\"For\", \"high-content\", \"high-throughput\", \"imaging\", \"analysis\", \"ImageXpress\", \"Micro\", \"XLS\", \"(\", \"Molecular\", \"Devices\", \",\", \"Sunnyvale\", \",\", \"CA\", \")\", \"was\", \"used\", \"as\", \"previously\", \"described\", \"8\", \".\"]\n",
      "Word token=> [\"Hits\", \"were\", \"normalized\", \"to\", \"cells\", \"expressing\", \"the\", \"empty\", \"vector\", \".\"]\n",
      "Word token=> [\"Depicted\", \"are\", \"genes\", \"that\", \"were\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"expressed\", \"in\", \"two\", \"independent\", \"screens\", \"with\", \"a\", \"transduction\", \"efficiency\", \"of\", \"at\", \"least\", \"1\", \"%\", \"(\", \"cut\", \"off\", \")\", \".\"]\n",
      "Word token=> [\"ISGs\", \"which\", \"were\", \"cytotoxic\", \"for\", \"the\", \"cells\", \"were\", \"excluded\", \"from\", \"the\", \"analysis\", \".\"]\n",
      "Word token=> [\"Normalized\", \"data\", \"can\", \"be\", \"found\", \"in\", \"Extended\", \"Data\"]\n",
      "Word token=> [\"Viral\", \"RNA\", \"was\", \"extracted\", \"from\", \"cell\", \"lysates\", \"(\", \"NucleoMag-\", \"96\", \"RNA\", \"kit\", \",\", \"Macherey\", \"Nagel\", \")\", \"or\", \"supernatant\", \"(\", \"NucleoMag-Vet\", \"kit\", \",\", \"Macherey\", \"Nagel\", \")\", \"using\", \"the\", \"KingFisher\", \"Flex\", \"robot\", \"according\", \"to\", \"the\", \"manufacturer\", \"'\", \"s\", \"recommendations\", \".\"]\n",
      "Word token=> [\"The\", \"commercially\", \"available\", \"TaqManTM\", \"Fast\", \"Virus\", \"1\", \"-Step\", \"Master\", \"Mix\", \"(\", \"Applied\", \"Biosystems\", \")\", \"was\", \"used\", \"for\", \"RT-qPCR\", \"with\", \"2\", \"µl\", \"of\", \"RNA\", \"input\", \"added\", \"to\", \"8\", \"µl\", \"of\", \"prepared\", \"mastermix\", \"per\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"sample\", \".\"]\n",
      "Word token=> [\"Viral\", \"RNA\", \"was\", \"detected\", \"using\", \"HCoV-\", \"229\", \"E\", \"specific\", \"primers\", \"normalized\", \"to\", \"a\", \"qRT-PCR\", \"standard\", \"for\", \"HCoV-\", \"229\", \"E\", \"(\", \"contains\", \"the\", \"M\", \"gene\", \"of\", \"HCoV-\", \"229\", \"E\", \"63\", \")\", \".\"]\n",
      "Word token=> [\"Intracellular\", \"viral\", \"RNA\", \"was\", \"normalized\", \"to\", \"total\", \"RNA\", \"(\", \"determined\", \"via\", \"the\", \"housekeeping\", \"gene\", \"Beta-\", \"2\", \"-Microglobulin\", \"(\", \"B\", \"2\", \"M\", \")\", \")\", \".\"]\n",
      "Word token=> [\"Cellular\", \"RNA\", \"was\", \"extracted\", \"from\", \"cell\", \"lysates\", \"of\", \"Huh\", \"7\", \"cells\", \"using\", \"the\", \"Nucleospin\", \"RNA\", \"kit\", \"(\", \"Machery\", \"Nagel\", \")\", \"according\", \"to\", \"the\", \"manufacturer\", \"'\", \"s\", \"recommendations\", \",\", \"quantified\", \"via\", \"NanoDrop\", \",\", \"and\", \"used\", \"to\", \"generate\", \"a\", \"standard\", \"curve\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"PCR\", \"conditions\", \"are\", \"available\", \"upon\", \"request\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Stable\", \"LY\", \"6\", \"E\", \"expressing\", \"or\", \"control\", \"Huh\", \"7\", \"cells\", \"were\", \"seeded\", \"in\", \"a\", \"24\", \"-well\", \"plate\", \"(\", \"4\", \"x\", \"10\", \"4\", \"cells\", \")\", \"and\", \"mock\", \"infected\", \"or\", \"inoculated\", \"with\", \"HCoV-\", \"229\", \"E\", \"(\", \"MOI=\", \"5\", \")\", \"in\", \"OptiMEM\", \"(\", \"Gibco\", \")\", \"on\", \"ice\", \"for\", \"1\", \"hour\", \".\"]\n",
      "Word token=> [\"Cells\", \"were\", \"washed\", \"at\", \"least\", \"3\", \"x\", \"with\", \"PBS\", \"and\", \"harvested\", \"immediately\", \"(\", \"t=\", \"0\", \"h\", \")\", \"or\", \"incubated\", \"at\", \"33\", \"°\", \"C\", \"for\", \"24\", \"hours\", \"(\", \"t=\", \"24\", \"h\", \")\", \",\", \"before\", \"cell\", \"lysis\", \"and\", \"extraction\", \"of\", \"viral\", \"RNA\", \"as\", \"described\", \"above\", \".\"]\n",
      "Word token=> [\"Viral\", \"RNA\", \"was\", \"detected\", \"via\", \"RT-qPCR\", \"normalized\", \"to\", \"the\", \"housekeeping\", \"gene\", \"B\", \"2\", \"M\", \"as\", \"described\", \"above\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Time\", \"course\", \"experiment\", \"8\", \"x\", \"10\", \"4\", \"cells\", \"stable\", \"LY\", \"6\", \"E\", \"expressing\", \"or\", \"control\", \"Huh\", \"7\", \"cells\", \"were\", \"seeded\", \"in\", \"12\", \"-well\", \"plates\", \"and\", \"mock\", \"infected\", \"or\", \"infected\", \"with\", \"229\", \"E-CoV-Rluc\", \"(\", \"MOI=\", \"0.1\", \")\", \"for\", \"2\", \"hours\", \"at\", \"33\", \"°\", \"C\", \"in\", \"OptiMEM\", \".\"]\n",
      "Word token=> [\"Cells\", \"were\", \"washed\", \"3\", \"x\", \"with\", \"PBS\", \"and\", \"samples\", \"harvested\", \"at\", \"the\", \"indicated\", \"time\", \"points\", \".\"]\n",
      "Word token=> [\"To\", \"determine\", \"intracellular\", \"replication\", \",\", \"cell\", \"lysates\", \"were\", \"collected\", \"and\", \"viral\", \"RNA\", \"extracted\", \"(\", \"NucleoMag-\", \"96\", \"RNA\", \"kit\", \",\", \"Macherey\", \"author/funder.\", \"All\", \"rights\", \"reserved.\", \"No\", \"reuse\", \"allowed\", \"without\", \"permission.\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"Nagel\", \")\", \",\", \"followed\", \"by\", \"RT-qPCR\", \"as\", \"described\", \"before\", \",\", \"or\", \"cells\", \"were\", \"lysed\", \"using\", \"the\", \"Renilla\", \"Luciferase\", \"Assay\", \"System\", \"kit\", \"(\", \"Promega\", \")\", \"according\", \"to\", \"the\", \"manufacturer\", \"'\", \"s\", \"recommendations\", \"and\", \"Rluc\", \"activity\", \"determined\", \".\"]\n",
      "Word token=> [\"To\", \"determine\", \"extracellular\", \"viral\", \"replication\", \",\", \"viral\", \"supernatant\", \"was\", \"harvested\", \",\", \"and\", \"viral\", \"RNA\", \"extracted\", \"(\", \"NucleoMag-Vet\", \"kit\", \",\", \"Macherey\", \"Nagel\", \")\", \",\", \"followed\", \"by\", \"RT-qPCR\", \"as\", \"described\", \"before\", \".\"]\n",
      "Word token=> [\"To\", \"determine\", \"intracellular\", \"infectivity\", \",\", \"cells\", \"were\", \"subjected\", \"to\", \"3\", \"rounds\", \"of\", \"freeze/thaw\", \"cycles\", \",\", \"centrifuged\", \"to\", \"remove\", \"debris\", \"(\", \"4,000\", \"×\", \"g\", \"for\", \"10\", \"min\", \")\", \",\", \"and\", \"the\", \"supernatant\", \"titrated\", \"on\", \"naïve\", \"Huh\", \"7\", \"cells\", \".\"]\n",
      "Word token=> [\"To\", \"determine\", \"extracellular\", \"infectivity\", \",\", \"supernatant\", \"was\", \"harvested\", \"at\", \"the\", \"indicated\", \"time\", \"points\", \"and\", \"titrated\", \"on\", \"naïve\"]\n",
      "Word token=> [\"To\", \"test\", \"various\", \"protease\", \"inhibitors\", \",\", \"2\", \"x\", \"10\", \"4\", \"naïve\", \"or\", \"TMPRSS\", \"2\", \"-expressing\", \",\", \"control\", \"or\", \"LY\", \"6\", \"E\", \"cells\", \"were\", \"seeded\", \"in\", \"a\", \"96\", \"-well\", \"plate\", \".\"]\n",
      "Word token=> [\"One\", \"day\", \"post\", \"seeding\", \",\", \"cells\", \"were\", \"pre-treated\", \"with\", \"the\", \"following\", \"compounds\", \"for\", \"1\", \"hour\", \"in\", \"OptiMEM\", \",\", \"at\", \"37\", \"°\", \"C\", \":\", \"DMSO\", \"(\", \"1:500\", \",\", \"Sigma-Aldrich\", \")\", \",\", \"E\", \"64\", \"D\", \"(\", \"10\", \"µM\", \",\", \"Sigma-Aldrich\", \")\", \",\", \"and/or\", \"Camostat\", \"(\", \"100\", \"µM\", \",\", \"Sigma-Aldrich\", \")\", \".\"]\n",
      "Word token=> [\"Cells\", \"were\", \"mock\", \"infected\", \"or\", \"infected\", \"with\", \"HCoV-\", \"229\", \"E-Rluc\", \"(\", \"MOI\", \"0.1\", \")\", \"in\", \"the\", \"presence\", \"of\", \"the\", \"inhibitors\", \"for\", \"2\", \"hours\", \"at\", \"33\", \"°\", \"C.\", \"Medium\", \"was\", \"changed\", \"to\", \"DMEM\", \"and\", \"cells\", \"incubated\", \"for\", \"24\", \"hours\", \"at\", \"33\", \"°\", \"C.\", \"Cells\", \"were\", \"lysed\", \"and\", \"Rluc\", \"activity\", \"detected\", \"using\", \"the\", \"Renilla\", \"Luciferase\", \"Assay\", \"System\", \"kit\", \"(\", \"Promega\", \")\", \"according\", \"to\", \"the\", \"manufacturer\", \"'\", \"s\", \"recommendations\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"LY\", \"6\", \"E\", \"expressing\", \"or\", \"control\", \"Huh\", \"7\", \"(\", \"2\", \"x\", \"10\", \"5\", \"cells\", \")\", \"were\", \"seeded\", \"in\", \"6\", \"-well\", \"cell\", \"culture\", \"plates\", \".\"]\n",
      "Word token=> [\"Cells\", \"were\", \"transfected\", \"using\", \"Lipofectamine\", \"2000\", \"(\", \"Invitrogen\", \")\", \"with\", \"an\", \"expression\", \"plasmid\", \"encoding\", \"for\", \"MERS-CoV\", \"S\", \"(\", \"pCAGGS-MERS\", \"S\", \")\", \".\"]\n",
      "Word token=> [\"Cell\", \"lysates\", \"were\", \"harvested\", \"48\", \"hours\", \"post-transfection\", \"and\", \"subjected\", \"to\", \"Western\", \"blot\", \"analysis\", \"as\", \"described\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\"]\n",
      "Word token=> [\"Multiple\", \"sequence\", \"alignment\", \"of\", \"LY\", \"6\", \"E\", \"amino\", \"acid\", \"sequences\", \"of\", \"different\", \"species\", \"(\", \"NCBI\", \"accession\", \"numbers\", \"provided\", \"in\", \"the\", \"figure\", \")\", \"was\", \"performed\", \"using\", \"the\", \"EBI\", \"web\", \"based\", \"Clustal\", \"Omega\", \"tool\", \"(\", \"doi\", \":\", \"10.1093\", \"/nar/gkz\", \"268\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"phylogenetic\", \"tree\", \"was\", \"built\", \"with\", \"the\", \"\\\"\", \"one\", \"click\", \"\\\"\", \"mode\", \"for\", \"phylogenetic\", \"analysis\", \"(\", \"http://www.phylogeny.fr/\", \")\", \"with\", \"standard\", \"settings\", \"(\", \"doi\", \":\", \"10.1093\", \"/nar/gkn\", \"180\", \";\", \"doi\", \":\", \"10.1186\", \"/\", \"1471\", \"-\", \"2148\", \"-\", \"10\", \"-\", \"8\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Sections\", \"of\", \"liver\", \"and\", \"spleen\", \"were\", \"preserved\", \"in\", \"RNAlater\", \"Stabilization\", \"Solution\", \"(\", \"Thermo\", \"Fisher\", \"Scientific\", \")\", \"and\", \"frozen\", \".\"]\n",
      "Word token=> [\"Thawed\", \"samples\", \"were\", \"transferred\", \"to\", \"PBS\", \"and\", \"homogenized\", \".\"]\n",
      "Word token=> [\"One-eighth\", \"of\", \"the\", \"homogenate\", \"was\", \"mixed\", \"with\", \"TRIzol\", \"Reagent\", \"(\", \"Thermo\", \"Fisher\", \"Scientific\", \")\", \".\"]\n",
      "Word token=> [\"BMDM\", \"were\", \"directly\", \"lysed\", \"in\", \"TRIzol\", \"Reagent\", \"and\", \"frozen\", \".\"]\n",
      "Word token=> [\"Total\", \"RNA\", \"was\", \"isolated\", \"according\", \"to\", \"the\", \"manufacturer\", \"'\", \"s\", \"protocol\", \".\"]\n",
      "Word token=> [\"Liver\", \"and\", \"spleen\", \"RNA\", \"were\", \"subject\", \"to\", \"DNase\", \"treatment\", \"(\", \"TURBO\", \"DNA-Free\", \"kit\", \",\", \"Thermo\", \"Fisher\", \"Scientific\", \")\", \"per\", \"the\", \"manufacturer\", \"'\", \"s\", \"protocol\", \"prior\", \"to\", \"RNA-Seq\", \"analysis\", \".\"]\n",
      "Word token=> [\"BMDM\", \"RNA\", \"was\", \"analyzed\", \"by\", \"one-step\", \"qRT-PCR\", \"using\", \"QuantiFast\", \"SYBR\", \"Green\", \"RT-PCR\", \"Kit\", \"(\", \"Qiagen\", \")\", \"using\", \"Applied\", \"Biosciences\", \"7500\", \"Fast\", \"Real-Time\", \"PCR\", \"System\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"quantity\", \"and\", \"quality\", \"of\", \"the\", \"extracted\", \"RNA\", \"was\", \"assessed\", \"using\", \"a\", \"Thermo\", \"Fisher\", \"Scientific\", \"qubit\", \"2\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"Data\", \"generated\", \"from\", \"individual\", \"samples\", \"(\", \">\", \"30\", \"million\", \"read\", \"per\", \"sample\", \",\", \"single\", \"read\", \"50\", \"-mers\", \")\", \"were\", \"mapped\", \"separately\", \"against\", \"the\", \"GRCm\", \"38\", \"murine\", \"reference\", \"genome\", \".\"]\n",
      "Word token=> [\"Gene\", \"expression\", \"was\", \"calculated\", \"for\", \"individual\", \"transcripts\", \"as\", \"reads\", \"per\", \"kilobase\", \"per\", \"million\", \"bases\", \"mapped\", \"(\", \"RPKM\", \")\", \".\"]\n",
      "Word token=> [\"All\", \"transcriptomic\", \"analyses\", \"were\", \"performed\", \"using\", \"CLC\", \"Genomics\", \"Workbench\", \"20\", \"(\", \"Qiagen\", \",\", \"Aarhaus\", \")\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Differentially\", \"expressed\", \"genes\", \"(\", \"DEGs\", \")\", \"were\", \"identified\", \"by\", \"calculating\", \"fold\", \"changes\", \"in\", \"expression\", \",\", \"pvalues\", \"were\", \"corrected\", \"by\", \"taking\", \"false\", \"discovery\", \"rate\", \"(\", \"FDR\", \")\", \"for\", \"multiple\", \"comparison\", \"into\", \"account\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Mus\", \"Musculus\", \"EBI\", \"Gene\", \"Ontology\", \"Annotation\", \"Database\", \"was\", \"used\", \"to\", \"execute\", \"Gene\", \"ontology\", \"(\", \"GO\", \")\", \"Enrichment\", \"Analyses\", \"for\", \"biological\", \"processes\", \".\"]\n",
      "Word token=> [\"Gene\", \"identifiers\", \"for\", \"DEGs\", \"absolute\", \"FC\", \">\", \"5\", \",\", \"RPKM\", \">\", \"2\", \")\", \"were\", \"used\", \"as\", \"input\", \"and\", \"identification\", \"of\", \"significantly\", \"enriched\", \"GO\", \"categories\", \".\"]\n",
      "Word token=> [\"P-values\", \"for\", \"specific\", \"GO\", \"categories\", \"were\", \"generated\", \"after\", \"Bonferroni\", \"correction\", \"for\", \"multiple\", \"testing\", \".\"]\n",
      "Word token=> [\"Z-scores\", \",\", \"used\", \"as\", \"indicator\", \"for\", \"activation\", \"of\", \"biological\", \"processes\", \"(\", \"positive\", \"value\", \")\", \"or\", \"inactivation\", \"(\", \"negative\", \"value\", \")\", \",\", \"were\", \"calculated\", \"based\", \"on\", \"the\", \"expression\", \"fold\", \"change\", \"as\", \"follows\", \":\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"gctggcatcgggaacgtggtggactttggctac\", \"accctgaacaagggctgctccccgatctgtccc\", \"ggcccgagcgtcaatcttggagtggcgtccgt\", \"gggcacccactgctgccagagcttcctgtgca\", \"acatcagtgcagccgacggcgggctgcgggc\", \"cagcacccacgtgctgggcctcgggctcctgc\", \"tcagcctgctgtccgccctgctgcggcttggcc\", \"cctgaacccagctttcttgtacaaagtggtcccc\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \".\", \"https://doi.org/\", \"10.1101\", \"/\", \"2020.03.05.979260\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \".\"]\n",
      "Word token=> [\"The\", \"rapid\", \"evolution\", \"of\", \"RNA\", \"viruses\", \"represents\", \"a\", \"significant\", \"challenge\", \"for\", \"preventing\", \",\", \"treating\", \",\", \"57\", \"and\", \"eradicating\", \"RNA\", \"viral\", \"diseases\", \".\"]\n",
      "Word token=> [\"High\", \"mutation\", \"rates\", \"in\", \"RNA\", \"viruses\", \"generate\", \"extensive\", \"58\", \"opportunities\", \"to\", \"overcome\", \"evolutionary\", \"hurdles\", \",\", \"such\", \"as\", \"antiviral\", \"drugs\", \",\", \"host\", \"immunity\", \",\", \"or\", \"59\", \"engineered\", \"attenuating\", \"changes\", \"(\", \"1\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"evolutionary\", \"pathways\", \"traversed\", \"by\", \"RNA\", \"viruses\", \"are\", \"60\", \"shaped\", \"by\", \"natural\", \"selection\", \",\", \"which\", \"will\", \"favor\", \"some\", \"evolutionary\", \"trajectories\", \"more\", \"than\", \"others\", \"61\", \"based\", \"on\", \"whether\", \"mutations\", \"are\", \"beneficial\", \",\", \"deleterious\", \",\", \"or\", \"neutral\", \"(\", \"2\", \")\", \".\"]\n",
      "Word token=> [\"Predicting\", \"the\", \"likely\", \"results\", \"62\", \"of\", \"RNA\", \"virus\", \"evolution\", \"is\", \"an\", \"important\", \"step\", \"for\", \"anticipating\", \"viral\", \"emergence\", \"and\", \"for\", \"developing\", \"63\", \"escape-resistant\", \"antiviral\", \"drugs\", \"and\", \"vaccines\", \"(\", \"3\", \",\", \"4\", \")\", \".\"]\n",
      "Word token=> [\"64\", \"65\", \"Coronaviruses\", \"(\", \"CoVs\", \")\", \"are\", \"a\", \"family\", \"of\", \"positive-sense\", \"RNA\", \"viruses\", \"that\", \"cause\", \"human\", \"illnesses\", \"66\", \"ranging\", \"from\", \"the\", \"common\", \"cold\", \"to\", \"severe\", \"and\", \"lethal\", \"respiratory\", \"disease\", \"(\", \"5\", \")\", \".\"]\n",
      "Word token=> [\"All\", \"CoVs\", \"encode\", \"a\", \"67\", \"proofreading\", \"exoribonuclease\", \"within\", \"nonstructural\", \"protein\", \"14\", \"(\", \"nsp\", \"14\", \"-ExoN\", \")\", \"that\", \"is\", \"critical\", \"for\", \"68\", \"replication\", \",\", \"fidelity\", \",\", \"fitness\", \",\", \"and\", \"virulence\", \",\", \"and\", \"ExoN-inactivation\", \"has\", \"been\", \"proposed\", \"as\", \"a\", \"69\", \"strategy\", \"for\", \"live-attenuated\", \"vaccine\", \"development\", \"(\", \"6\", \"-\", \"15\", \")\", \".\"]\n",
      "Word token=> [\"As\", \"members\", \"of\", \"the\", \"DEDDh\", \"superfamily\", \"70\", \"of\", \"exonucleases\", \",\", \"CoV\", \"ExoNs\", \"hydrolyze\", \"nucleotides\", \"using\", \"four\", \"metal-coordinating\", \"amino\", \"acids\", \"71\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"618249\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"ExoN\", \"active\", \"site\", \"mutants\", \"in\", \"alphacoronaviruses\", \",\", \"including\", \"transmissible\", \"gastroenteritis\", \"virus\", \"and\", \"77\", \"hCoV-\", \"229\", \"E\", \",\", \"have\", \"yet\", \"to\", \"be\", \"recovered\", \"and\", \"are\", \"proposed\", \"to\", \"be\", \"lethal\", \"for\", \"replication\", \"(\", \"19\", \",\", \"20\", \")\", \".\", \"infections\", \"and\", \"prolonged\", \"passage\", \"in\", \"tissue\", \"culture\", \"and\", \"following\", \"treatment\", \"with\", \"multiple\", \"87\", \"nucleoside\", \"analogues\", \"(\", \"6\", \"-\", \"11\", \",\", \"13\", \",\", \"14\", \")\", \".\"]\n",
      "Word token=> [\"SARS-CoV-ExoN-AA\", \"also\", \"is\", \"stable\", \"during\", \"acute\", \"and\", \"88\", \"persistent\", \"animal\", \"infections\", \"in\", \"immunocompetent\", \"and\", \"immune-compromised\", \"mice\", \"(\", \"12\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"lack\", \"89\", \"of\", \"primary\", \"reversion\", \"is\", \"not\", \"due\", \"simply\", \"to\", \"reduced\", \"adaptive\", \"capacity\", \",\", \"as\", \"both\", \"SARS-CoV-and\", \"90\", \"MHV-ExoN-AA\", \"can\", \"adapt\", \"for\", \"increased\", \"replication\", \"(\", \"7\", \",\", \"14\", \")\", \".\"]\n",
      "Word token=> [\"Most\", \"strikingly\", \",\", \"long-term\", \"passage\", \"91\", \"of\", \"MHV-ExoN-AA\", \"(\", \"250\", \"passages\", \",\", \"P\", \"250\", \")\", \"yielded\", \"a\", \"highly\", \"fit\", \"population\", \"that\", \"had\", \"directly\", \"92\", \"compensated\", \"for\", \"defective\", \"proofreading\", \"through\", \"evolution\", \"of\", \"a\", \"likely\", \"high-fidelity\", \"RdRp\", \"(\", \"7\", \"requires\", \"mutations\", \"to\", \"all\", \"four\", \"sites\", \"(\", \"Figure\", \"1\", \"A\", \")\", \".\"]\n",
      "Word token=> [\"Viral\", \"mutation\", \"rates\", \"in\", \"the\", \"absence\", \"of\", \"113\", \"proofreading\", \"range\", \"from\", \"10\", \"-\", \"4\", \"to\", \"10\", \"-\", \"6\", \"mutations\", \"per\", \"nucleotide\", \"per\", \"round\", \"of\", \"replication\", \"(\", \"µ\", \")\", \"(\", \"1\", \")\", \".\"]\n",
      "Word token=> [\"114\"]\n",
      "Word token=> [\"Assuming\", \"an\", \"ExoN-AA\", \"mutation\", \"rate\", \"of\", \"10\", \"-\", \"4\", \"µ\", \"and\", \"accounting\", \"for\", \"codon\", \"degeneracy\", \",\", \"the\", \"115\", \"probability\", \"of\", \"restoring\", \"the\", \"native\", \"amino\", \"acid\", \"sequence\", \"in\", \"a\", \"single\", \"round\", \"of\", \"replication\", \"is\", \"10\", \"-\", \"18\", \".\"]\n",
      "Word token=> [\"116\"]\n",
      "Word token=> [\"Only\", \"rarely\", \"do\", \"ExoN-AA\", \"titers\", \"exceed\", \"10\", \"6\", \"PFU/mL\", \",\", \"so\", \"it\", \"is\", \"unlikely\", \"that\", \"ExoN-AA\", \"could\", \"117\", \"navigate\", \"this\", \"genetic\", \"barrier\", \"in\", \"a\", \"single\", \"infectious\", \"cycle\", \".\"]\n",
      "Word token=> [\"Thus\", \",\", \"we\", \"hypothesized\", \"that\", \"ExoN-AA\", \"118\", \"reversion\", \",\", \"if\", \"possible\", \",\", \"would\", \"proceed\", \"incrementally\", \".\"]\n",
      "Word token=> [\"To\", \"identify\", \"potential\", \"pathways\", \"towards\", \"119\"]\n",
      "Word token=> [\"ExoN-AA\", \"reversion\", \",\", \"we\", \"examined\", \"the\", \"possible\", \"single-nucleotide\", \"substitutions\", \"surrounding\", \"A\", \"89\", \"120\", \"and\", \"A\", \"91\", \"(\", \"Figure\", \"1\", \"B\", \")\", \".\"]\n",
      "Word token=> [\"Three\", \"mutations\", \"are\", \"synonymous\", \",\", \"and\", \"five\", \"mutations\", \"yield\", \"amino\", \"acids\", \"121\", \"unlikely\", \"to\", \"coordinate\", \"with\", \"the\", \"positively-charged\", \"metals\", \"required\", \"for\", \"ExoN\", \"catalysis\", \"(\", \"glycine\", \",\", \"122\", \"valine\", \",\", \"proline\", \",\", \"threonine\", \",\", \"and\", \"serine\", \")\", \"(\", \"16\", \",\", \"19\", \",\", \"21\", \",\", \"22\", \")\", \".\"]\n",
      "Word token=> [\"One\", \"mutation\", \"per\", \"site\", \"can\", \"restore\", \"the\", \"123\", \"acidic\", \"charge\", \"(\", \"i.e.\", \"AA-to-ED\", \")\", \"but\", \"not\", \"the\", \"native\", \"amino\", \"acid\", \".\"]\n",
      "Word token=> [\"These\", \"variants\", \"have\", \"not\", \"been\", \"tested\", \"124\", \"in\", \"a\", \"CoV\", \"ExoN\", \",\", \"but\", \"biochemical\", \"studies\", \"of\", \"E.\", \"coli\", \"DNA\", \"polymerase\", \"I\", \"ExoN\", \"mutants\", \"suggest\", \"that\", \"125\", \"these\", \"conservative\", \"substitutions\", \"would\", \"not\", \"restore\", \"WT-like\", \"ExoN\", \"activity\", \"(\", \"23\", \")\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"We\", \"predicted\", \"126\", \"stepwise\", \"pathways\", \"to\", \"ExoN-AAàDE\", \"reversion\", \"based\", \"on\", \"restoration\", \"of\", \"acidic\", \"charge\", \"followed\", \"127\", \"by\", \"reversion\", \"to\", \"native\", \"amino\", \"acids\", \"(\", \"Figure\", \"1\", \"C\", \")\", \".\"]\n",
      "Word token=> [\"We\", \"engineered\", \"and\", \"recovered\", \"variants\", \"in\", \"ExoN-\", \"128\", \"AA\", \"requiring\", \"three\", \"mutations\", \"(\", \"3\", \"nt\", \";\", \"ExoN-AD\", \",\", \"ExoN-EA\", \")\", \",\", \"two\", \"mutations\", \"(\", \"2\", \"nt\", \";\", \"ExoN-DA\", \",\", \"ExoN-\", \"129\", \"ED\", \",\", \"ExoN-AE\", \")\", \",\", \"or\", \"one\", \"mutation\", \"(\", \"1\", \"nt\", \";\", \"ExoN-DD\", \",\", \"ExoN-EE\", \")\", \"for\", \"reversion\", \"to\", \"WT-ExoN-DE\", \"130\", \"(\", \"Table\", \"1\", \")\", \".\"]\n",
      "Word token=> [\"We\", \"will\", \"hereafter\", \"refer\", \"to\", \"these\", \"mutants\", \"as\", \"intermediate\", \"revertants\", \".\"]\n",
      "Word token=> [\"All\", \"intermediate\", \"131\", \"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"618249\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"revertants\", \"generated\", \"viable\", \"progeny\", \"during\", \"recovery\", \",\", \"demonstrating\", \"that\", \"reversion\", \"to\", \"WT-ExoN-\", \"132\", \"DE\", \"along\", \"these\", \"pathways\", \"is\", \"theoretically\", \"possible\", \".\"]\n",
      "Word token=> [\"The\", \"3\", \"nt\", \"and\", \"2\", \"nt\", \"mutants\", \"were\", \"genetically\", \"133\", \"stable\", \"during\", \"recovery\", \",\", \"as\", \"confirmed\", \"by\", \"dideoxy\", \"sequencing\", \".\"]\n",
      "Word token=> [\"However\", \",\", \"both\", \"1\", \"nt\", \"mutants\", \"(\", \"ExoN-\", \"134\", \"DD\", \"and\", \"ExoN-EE\", \")\", \"reverted\", \"to\", \"WT-ExoN-DE\", \"during\", \"three\", \"independent\", \"recovery\", \"attempts\", \",\", \"135\"]\n",
      "Word token=> [\"suggesting\", \"that\", \"these\", \"two\", \"variants\", \"are\", \"less\", \"fit\", \"than\", \"WT-ExoN-DE\", \"and\", \"demonstrating\", \"that\", \"136\", \"reversion\", \"by\", \"1\", \"nt\", \"mutation\", \"is\", \"readily\", \"accessible\", \".\"]\n",
      "Word token=> [\"To\", \"test\", \"whether\", \"the\", \"3\", \"nt\", \"or\", \"2\", \"nt\", \"mutants\", \"would\", \"137\", \"revert\", \"more\", \"rapidly\", \"than\", \"ExoN-AA\", \"(\", \"4\", \"nt\", \")\", \",\", \"we\", \"passaged\", \"three\", \"lineages\", \"of\", \"each\", \"mutant\", \"10\", \"times\", \"at\", \"(\", \"Table\", \"1\", \")\", \".\"]\n",
      "Word token=> [\"In\", \"contrast\", \",\", \"the\", \"2\", \"nt\", \"ExoN-AE\", \"contained\", \"WT-revertants\", \"by\", \"P\", \"2\", \"145\", \"in\", \"all\", \"lineages\", \"at\", \"MOI\", \"=\", \"0.5\", \"PFU/cell\", \"and\", \"by\", \"P\", \"8\", \"in\", \"one\", \"lineage\", \"at\", \"MOI\", \"=\", \"0.01\", \"PFU/cell\", \".\"]\n",
      "Word token=> [\"Once\", \"146\", \"observed\", \",\", \"WT-revertants\", \"dominated\", \"the\", \"ExoN-AE\", \"population\", \"for\", \"the\", \"remaining\", \"passages\", \".\"]\n",
      "Word token=> [\"These\", \"147\", \"data\", \"indicate\", \"that\", \"at\", \"least\", \"one\", \"2\", \"nt\", \"mutation\", \"pathway\", \"can\", \"lead\", \"to\", \"full\", \"reversion\", \"in\", \"tissue\", \"culture\", \".\"]\n",
      "Word token=> [\"148\"]\n",
      "Word token=> [\"The\", \"probability\", \"of\", \"ExoN-AE\", \"arising\", \"during\", \"a\", \"single\", \"infectious\", \"cycle\", \"of\", \"ExoN-AA\", \"is\", \"low\", \"but\", \"149\", \"theoretically\", \"achievable\", \"(\", \"~\", \"10\", \"-\", \"9\", \")\", \",\", \"so\", \"ExoN-AA\", \"could\", \"conceivably\", \"revert\", \"within\", \"just\", \"two\", \"infectious\", \"150\", \"cycles\", \".\"]\n",
      "Word token=> [\"However\", \",\", \"complete\", \"reversion\", \"has\", \"never\", \"been\", \"observed\", \"even\", \"during\", \"prolonged\", \"passage\", \"or\", \"151\", \"persistent\", \"infections\", \",\", \"suggesting\", \"that\", \"additional\", \"barriers\", \"to\", \"the\", \"replication\", \",\", \"fitness\", \",\", \"or\", \"maintenance\", \"152\", \"of\", \"intermediate\", \"revertants\", \"exist\", \".\"]\n",
      "Word token=> [\"153\", \"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"618249\", \"doi\", \":\", \"bioRxiv\", \"preprint\"]\n",
      "Word token=> [\"GAT\", \"GAA\"]\n",
      "Word token=> [\"GCA\", \"GCT\"]\n",
      "Word token=> [\"ExoN\", \"Motif\", \"I\", \"C\", \"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"618249\", \"doi\", \":\", \"bioRxiv\", \"preprint\"]\n",
      "Word token=> [\"the\", \"intermediate\", \"revertants\", \"are\", \"viable\", \"as\", \"recombinants\", \"but\", \"are\", \"not\", \"found\", \"in\", \"ExoN-AA\", \"159\", \"populations\", \",\", \"we\", \"hypothesized\", \"that\", \"they\", \"confer\", \"no\", \"selective\", \"advantage\", \"over\", \"ExoN-AA\", \"(\", \"8\", \",\", \"9\", \",\", \"13\", \")\", \".\"]\n",
      "Word token=> [\"160\"]\n",
      "Word token=> [\"To\", \"test\", \"this\", \"hypothesis\", \",\", \"we\", \"first\", \"analyzed\", \"replication\", \"of\", \"the\", \"3\", \"nt\", \"and\", \"2\", \"nt\", \"intermediate\", \"revertants\", \"161\", \"(\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder.\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder.\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder.\", \"nsp\", \"12\", \"/\", \"14\", \"-P\", \"250\", \")\", \"were\", \"significantly\", \"more\", \"sensitive\", \"to\", \"5\", \"-FU\", \"(\", \"Figure\", \"4\", \"B\", \")\", \".\"]\n",
      "Word token=> [\"Finally\", \",\", \"the\", \"nsp\", \"12\", \"-P\", \"250\", \"224\", \"and\", \"nsp\", \"14\", \"-P\", \"250\", \"mutations\", \"significantly\", \"decreased\", \"fitness\", \"relative\", \"to\", \"WT-ExoN-DE\", \"(\", \"Figure\", \"4\", \"C\", \")\", \".\"]\n",
      "Word token=> [\"225\"]\n",
      "Word token=> [\"We\", \"detected\", \"no\", \"statistical\", \"differences\", \"between\", \"the\", \"specific\", \"infectivity\", \"of\", \"WT-ExoN-DE\", \"and\", \"any\", \"226\", \"of\", \"the\", \"nsp\", \"12\", \"-P\", \"250\", \"and\", \"nsp\", \"14\", \"-P\", \"250\", \"variants\", \"in\", \"isolated\", \"infections\", \"(\", \"Figure\", \"4\", \"D\", \")\", \".\"]\n",
      "Word token=> [\"Thus\", \",\", \"mutations\", \"227\", \"in\", \"nsp\", \"12\", \"and\", \"nsp\", \"14\", \"that\", \"arose\", \"in\", \"the\", \"ExoN-AA\", \"background\", \"were\", \"detrimental\", \"to\", \"replication\", \",\", \"228\", \"mutagen\", \"sensitivity\", \",\", \"and\", \"competitive\", \"fitness\", \"in\", \"the\", \"presence\", \"of\", \"a\", \"fully-reverted\", \"ExoN-DE\", \".\"]\n",
      "Word token=> [\"These\", \"229\", \"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \".\"]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"618249\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"Our\", \"results\", \"also\", \"extend\", \"existing\", \"studies\", \"of\", \"CoV\", \"ExoN\", \"motif\", \"I\", \".\"]\n",
      "Word token=> [\"Motif\", \"I\", \"AAàDE\", \"mutations\", \"in\", \"the\", \"249\", \"SARS-CoV\", \"nsp\", \"14\", \"-ExoN\", \"dramatically\", \"reduce\", \"nuclease\", \"activity\", \"in\", \"biochemical\", \"assays\", \",\", \"but\", \"no\", \"250\", \"study\", \"has\", \"examined\", \"the\", \"contributions\", \"of\", \"each\", \"residue\", \"independently\", \"(\", \"16\", \",\", \"18\", \")\", \".\"]\n",
      "Word token=> [\"Intermediate\", \"251\", \"revertants\", \"of\", \"ExoN-AA\", \"did\", \"not\", \"display\", \"consistent\", \"or\", \"statistical\", \"differences\", \"in\", \"replication\", \",\", \"5\", \"-FU\", \"252\", \"sensitivity\", \",\", \"or\", \"competitive\", \"fitness\", \"relative\", \"to\", \"ExoN-AA\", \",\", \"suggesting\", \"that\", \"they\", \"remain\", \"253\", \"phenotypically\", \"ExoN\", \"(\", \"-\", \")\", \"during\", \"infection\", \"and\", \"supporting\", \"previous\", \"studies\", \"that\", \"motif\", \"I\", \"DE\", \"is\", \"254\", \"essential\", \"for\", \"WT\", \"ExoN\", \"function\", \".\"]\n",
      "Word token=> [\"Given\", \"these\", \"results\", \",\", \"we\", \"were\", \"surprised\", \"to\", \"observe\", \"repeated\", \"255\", \"reversion\", \"of\", \"the\", \"ExoN-AE\", \"but\", \"not\", \"the\", \"other\", \"two\", \"2\", \"nt\", \"variants\", \",\", \"ExoN-DA\", \"and\", \"ExoN-ED\", \".\"]\n",
      "Word token=> [\"One\", \"256\", \"potential\", \"explanation\", \"is\", \"that\", \"the\", \"specific\", \"mutational\", \"bias\", \"of\", \"ExoN-AE\", \"makes\", \"the\", \"revertant\", \"257\", \"mutations\", \"more\", \"accessible\", \"than\", \"in\", \"ExoN-DA\", \"or\", \"ExoN-ED\", \".\"]\n",
      "Word token=> [\"Alternatively\", \",\", \"if\", \"ExoN-AE\", \"has\", \"258\", \"profound\", \"replication\", \"or\", \"fitness\", \"defects\", \",\", \"selection\", \"could\", \"drive\", \"primary\", \"reversion\", \"more\", \"quickly\", \"259\", \"away\", \"from\", \"this\", \"genotype\", \".\"]\n",
      "Word token=> [\"Consistent\", \"with\", \"this\", \"hypothesis\", \",\", \"ExoN-AE\", \"reverted\", \"more\", \"quickly\", \"at\", \"a\", \"260\", \"higher\", \"MOI\", \",\", \"where\", \"natural\", \"selection\", \"acts\", \"more\", \"efficiently\", \"on\", \"a\", \"larger\", \"population\", \"size\", \"(\", \"Table\", \"1\", \")\", \"Consistent\", \"with\", \"this\", \"hypothesis\", \",\", \"the\", \"fitness\", \"effects\", \"of\", \"mutations\", \"in\", \"nsp\", \"12\", \"-P\", \"250\", \"and\", \"nsp\", \"14\", \"-P\", \"250\", \"269\", \"differ\", \"based\", \"on\", \"the\", \"motif\", \"I\", \"genotype\", \";\", \"they\", \"are\", \"beneficial\", \"in\", \"ExoN-AA\", \"but\", \"detrimental\", \"in\", \"the\", \"WT-\", \"270\", \"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"618249\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"ExoN-DE\", \"background\", \".\"]\n",
      "Word token=> [\"In\", \"previous\", \"studies\", \",\", \"it\", \"has\", \"been\", \"difficult\", \"to\", \"determine\", \"whether\", \"the\", \"fitness\", \"271\", \"defects\", \"in\", \"ExoN\", \"(\", \"-\", \")\", \"CoVs\", \"are\", \"directly\", \"linked\", \"to\", \"low-fidelity\", \"replication\", \"or\", \"through\", \"some\", \"other\", \"272\", \"mechanism\", \".\"]\n",
      "Word token=> [\"Our\", \"data\", \"suggest\", \"that\", \"the\", \"proofreading\", \"function\", \"of\", \"nsp\", \"14\", \"-ExoN\", \"can\", \"be\", \"uncoupled\", \"273\", \"from\", \"its\", \"more\", \"general\", \"role\", \"in\", \"replication\", \"(\", \"Figure\", \"4\", \")\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"618249\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"Cloning\", \"and\", \"recovery\", \"of\", \"recombinant\", \"viruses\", \".\"]\n",
      "Word token=> [\"Site-directed\", \"mutagenesis\", \"in\", \"MHV\", \"genome\", \"306\", \"fragments\", \"was\", \"performed\", \"using\", \"\\\"\", \"round\", \"the\", \"horn\", \"\\\"\", \"PCR\", \"(\", \"originally\", \"described\", \"in\", \"(\", \"33\", \")\", \")\", \".\"]\n",
      "Word token=> [\"Briefly\", \",\", \"307\", \"adjacent\", \"primers\", \"containing\", \"the\", \"mutation\", \"of\", \"interest\", \"were\", \"5\", \"¢\", \"-phosphorylated\", \"using\", \"T\", \"4\", \"308\", \"polynucleotide\", \"kinase\", \"(\", \"NEB\", \",\", \"M\", \"0201\", \"S\", \")\", \"using\", \"the\", \"buffer\", \"from\", \"the\", \"T\", \"4\", \"DNA\", \"ligase\", \",\", \"which\", \"contains\", \"309\", \"ATP\", \"(\", \"M\", \"0202\", \"S\", \")\", \".\"]\n",
      "Word token=> [\"PCR\", \"was\", \"performed\", \"on\", \"a\", \"plasmid\", \"template\", \"using\", \"the\", \"Q\", \"5\", \"High-fidelity\", \"2\", \"x\", \"previously\", \"(\", \"9\", \")\", \".\"]\n",
      "Word token=> [\"Briefly\", \",\", \"genomic\", \"RNA\", \"was\", \"detected\", \"with\", \"a\", \"5\", \"'\", \"6\", \"-carboxyfluorescein\", \"(\", \"FAM\", \")\", \"and\", \"341\", \"3\", \"'\", \"black\", \"hole\", \"quencher\", \"1\", \"(\", \"BHQ-\", \"1\", \")\", \"labeled\", \"probe\", \"targeting\", \"nsp\", \"2\", \"(\", \"Biosearch\", \"Technologies\", \",\", \"342\", \"Petaluma\", \",\", \"CA\", \")\", \",\", \"and\", \"RNA\", \"copy\", \"number\", \"was\", \"calculated\", \"by\", \"reference\", \"to\", \"an\", \"RNA\", \"standard\", \"derived\", \"343\", \"from\", \"the\", \"MHV\", \"A\", \"fragment\", \".\"]\n",
      "Word token=> [\"Samples\", \"were\", \"plated\", \"in\", \"technical\", \"duplicate\", \"to\", \"minimize\", \"well-to-well\", \"344\", \"variation\", \".\"]\n",
      "Word token=> [\"Titers\", \"were\", \"determined\", \"by\", \"plaque\", \"assay\", \"in\", \"DBT-\", \"9\", \"cells\", \",\", \"and\", \"specific\", \"infectivity\", \"was\", \"345\", \"calculated\", \"as\", \"PFU\", \"per\", \"supernatant\", \"genomic\", \"RNA\", \"copy\", \".\"]\n",
      "Word token=> [\"346\", \"347\", \"5\", \"-fluorouracil\", \"sensitivity\", \"assays\", \".\"]\n",
      "Word token=> [\"Stock\", \"solutions\", \"of\", \"5\", \"-fluorouracil\", \"(\", \"Sigma\", \"F\", \"6627\", \")\", \"were\", \"348\", \"prepared\", \"in\", \"dimethyl\", \"sulfoxide\", \"(\", \"DMSO\", \")\", \".\"]\n",
      "Word token=> [\"Sensitivity\", \"assays\", \"were\", \"performed\", \"in\", \"24\", \"-well\", \"plates\", \"at\", \"349\", \"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"618249\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"MOI\", \"=\", \"0.01\", \"PFU/cell\", \",\", \"as\", \"previously\", \"described\", \"(\", \"7\", \")\", \"were\", \"averaged\", \",\", \"and\", \"values\", \"were\", \"excluded\", \"from\", \"subsequent\", \"analysis\", \"if\", \"the\", \"duplicate\", \"wells\", \"differed\", \"371\", \"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"618249\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"by\", \">\", \"0.5\", \"Ct\", \".\"]\n",
      "Word token=> [\"The\", \"relative\", \"abundance\", \"of\", \"competitor\", \"and\", \"reference\", \"were\", \"determined\", \"by\", \"subtracting\", \"372\"]\n",
      "Word token=> [\"Ct\", \"thresholds\", \"(\", \"DCtcompetitor\", \"=\", \"Ctcompetitor\", \"-Ctreference\", \")\", \"and\", \"converted\", \"to\", \"reflect\", \"the\", \"fold-change\", \"in\", \"373\", \"ratio\", \"(\", \"Dratio\", \"=\", \"2\", \"-DCt\", \"competitor\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"log\", \"10\", \"Dratio\", \"was\", \"plotted\", \"against\", \"passage\", \"number\", \",\", \"and\", \"the\", \"change\", \"374\", \"in\", \"log\", \"10\", \"Dratio\", \"(\", \"i.e.\", \"slope\", \"of\", \"linear\", \"regression\", \")\", \"is\", \"the\", \"relative\", \"fitness\", \".\"]\n",
      "Word token=> [\"Note\", \"that\", \"regressions\", \"were\", \"fit\", \"375\", \"only\", \"through\", \"P\", \"1\", \"-P\", \"4\", \",\", \"as\", \"slight\", \"deviations\", \"in\", \"1:1\", \"ratio\", \"in\", \"the\", \"input\", \"(\", \"P\", \"0\", \")\", \"can\", \"skew\", \"the\", \"slope\", \".\"]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"618249\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \".\"]\n",
      "Word token=> [\"9\", \"also\", \"deleted\", \"ORF\", \"4\", \"a\", \",\", \"which\", \"is\", \"dispensable\", \"for\", \"MHV\", \"replication\", \"in\", \"cell\", \"culture\", \"(\", \"52\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"C-\", \"150\", \"terminal\", \"region\", \"of\", \"ns\", \"2\", \"within\", \"MHV-ExoN\", \"(\", \"-\", \")\", \"P\", \"250\", \"was\", \"truncated\", \"and\", \"fused\", \"to\", \"HE\", \"with\", \"a\", \"-\", \"1\", \"151\", \"frameshift\", \".\"]\n",
      "Word token=> [\"Ns\", \"2\", \"is\", \"a\", \"phosphodiesterase\", \"(\", \"PDE\", \")\", \"that\", \"protects\", \"viral\", \"RNA\", \"by\", \"degrading\", \"2\", \"¢\", \"\\uf0a2à\\uf0e0\", \"5\", \"¢\", \"\\uf0a2\", \"152\", \"oligoadenylate\", \",\", \"the\", \"activating\", \"factor\", \"for\", \"cellular\", \"RNase\", \"L\", \"(\", \"53\", \"-\", \"55\", \")\", \".\"]\n",
      "Word token=> [\"The\", \"portion\", \"of\", \"ns\", \"2\", \"deleted\", \"in\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"175562\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"similar\", \"for\", \"all\", \"four\", \"viruses\", \"(\", \"Fig.\", \"4\", \"B\", \")\", \".\"]\n",
      "Word token=> [\"These\", \"data\", \"suggest\", \"that\", \"the\", \"increased\", \"replication\", \"of\", \"P\", \"250\", \"173\", \"viruses\", \"relative\", \"to\", \"WT-MHV\", \"is\", \"not\", \"fully\", \"accounted\", \"for\", \"by\", \"increased\", \"RNA\", \"synthesis\", \".\"]\n",
      "Word token=> [\"In\", \"addition\", \"174\", \"to\", \"RNA\", \"synthesis\", \"defects\", \",\", \"ExoN\", \"(\", \"-\", \")\", \"CoVs\", \"have\", \"up-to\", \"20\", \"-fold\", \"increased\", \"mutation\", \"frequencies\", \"and\", \"175\", \"profoundly\", \"increased\", \"sensitivity\", \"to\", \"nucleoside\", \"and\", \"base\", \"analogs\", \"relative\", \"to\", \"WT\", \"CoVs\", \"(\", \"13\", \",\", \"14\", \",\", \"16\", \",\", \"176\", \"17\", \",\", \"38\", \")\", \".\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"To\", \"determine\", \"whether\", \"nucleoside\", \"analog\", \"sensitivity\", \"of\", \"MHV-ExoN\", \"(\", \"-\", \")\", \"was\", \"altered\", \"by\", \"177\", \"long-term\", \"passage\", \",\", \"we\", \"treated\", \"cells\", \"infected\", \"with\", \"parental\", \"and\", \"passaged\", \"viruses\", \"with\", \"the\", \"base\", \"178\", \"analog\", \",\", \"5\", \"-fluorouracil\", \"(\", \"5\", \"-FU\", \")\", \".\"]\n",
      "Word token=> [\"5\", \"-FU\", \"is\", \"converted\", \"intracellularly\", \"into\", \"a\", \"nucleoside\", \"analog\", \"that\", \"179\", \"incorporates\", \"into\", \"growing\", \"RNA\", \"strands\", \"and\", \"causes\", \"A\", \":\", \"G\", \"and\", \"U\", \":\", \"C\", \"mutations\", \".\"]\n",
      "Word token=> [\"For\", \"simplicity\", \",\", \"we\", \"180\", \"hereafter\", \"refer\", \"to\", \"5\", \"-FU\", \"as\", \"a\", \"nucleoside\", \"analog\", \".\"]\n",
      "Word token=> [\"Incorporation\", \"of\", \"5\", \"-FU\", \"is\", \"increased\", \"in\", \"the\", \"absence\", \"181\", \"of\", \"ExoN\", \"activity\", \"(\", \"16\", \")\", \".\"]\n",
      "Word token=> [\"All\", \"viruses\", \"displayed\", \"a\", \"concentration-dependent\", \"decrease\", \"in\", \"viral\", \"titer\", \"but\", \"182\", \"differed\", \"greatly\", \"in\", \"their\", \"susceptibility\", \"to\", \"5\", \"-FU\", \"(\", \"Fig.\", \"4\", \"C\", \")\", \".\"]\n",
      "Word token=> [\"At\", \"120\", \"µM\", \",\", \"WT-MHV\", \"P\", \"3\", \"titers\", \"were\", \"183\", \"reduced\", \"by\", \"~\", \"1\", \"log\", \"10\", \",\", \"while\", \"MHV-ExoN\", \"(\", \"-\", \")\", \"P\", \"3\", \"titers\", \"were\", \"undetectable\", \"(\", \">\", \"5\", \"log\", \"10\", \"-fold\", \"reduction\", \")\", \".\"]\n",
      "Word token=> [\"184\"]\n",
      "Word token=> [\"WT-MHV\", \"5\", \"-FU\", \"sensitivity\", \"was\", \"not\", \"altered\", \"by\", \"passage\", \".\"]\n",
      "Word token=> [\"MHV-ExoN\", \"(\", \"-\", \")\", \"P\", \"250\", \"was\", \"less\", \"susceptible\", \"185\", \"than\", \"MHV-ExoN\", \"(\", \"-\", \")\", \"P\", \"3\", \"to\", \"5\", \"-FU\", \"treatment\", \",\", \"with\", \"only\", \"a\", \"~\", \"1.5\", \"log\", \"10\", \"decrease\", \"in\", \"titer\", \"at\", \"120\", \"µM.\", \"replication\", \"cycles\", \"in\", \"which\", \"5\", \"-FU\", \"can\", \"be\", \"incorporated\", \"(\", \"59\", \")\", \".\"]\n",
      "Word token=> [\"MHV-ExoN\", \"(\", \"-\", \")\", \"P\", \"250\", \"had\", \"multiple\", \"193\", \"mutations\", \"in\", \"the\", \"spike\", \"glycoprotein\", \",\", \"including\", \"one\", \"in\", \"the\", \"spike\", \"furin\", \"cleavage\", \"site\", \"that\", \"reduced\", \"194\", \"syncytia\", \"formation\", \".\"]\n",
      "Word token=> [\"To\", \"test\", \"whether\", \"the\", \"spike\", \"mutations\", \"manifested\", \"in\", \"resistance\", \"to\", \"5\", \"-FU\", \",\", \"we\", \"195\", \"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \".\"]\n",
      "Word token=> [\"However\", \",\", \"M\", \"288\", \"T\", \"is\", \"not\", \"predicted\", \"to\", \"catalyze\", \"nucleotidylation\", \".\"]\n",
      "Word token=> [\"Within\", \"nsp\", \"14\", \",\", \"4\", \"NS\", \"mutations\", \"215\"]\n",
      "Word token=> [\"were\", \"identified\", \"in\", \"the\", \"ExoN\", \"domain\", \",\", \"and\", \"2\", \"NS\", \"mutations\", \"were\", \"in\", \"the\", \"C-terminal\", \"N\", \"7\", \"-\", \"216\", \"methyltransferase\", \"domain\", \"(\", \"Fig.\", \"6\", \"B\", \")\", \".\"]\n",
      "Word token=> [\"We\", \"next\", \"modeled\", \"the\", \"structure\", \"of\", \"MHV\", \"nsp\", \"14\", \"using\", \"Phyre\", \"2\", \"217\", \"software\", \"(\", \"63\", \")\", \",\", \"resulting\", \"in\", \"highest-probability\", \"similarity\", \"to\", \"the\", \"SARS-CoV\", \"nsp\", \"14\", \"-nsp\", \"10\", \"complex\", \"218\", \"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"175562\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"between\", \"the\", \"structures\", \")\", \"of\", \"100\", \"%\", \"for\", \"residues\", \"3\", \"-\", \"519\", \"of\", \"MHV-nsp\", \"14\", \".\"]\n",
      "Word token=> [\"The\", \"model\", \"predicts\", \"that\", \"five\", \"220\", \"mutations\", \"are\", \"located\", \"close\", \"to\", \"surface\", \"of\", \"the\", \"protein\", \"(\", \"Fig.\", \"6\", \"B\", \")\", \".\"]\n",
      "Word token=> [\"All\", \"three\", \"modeled\", \"zinc\", \"finger\", \"221\", \"domains\", \"contain\", \"one\", \"NS\", \"mutation\", \"(\", \"F\", \"216\", \"Y\", \",\", \"Y\", \"248\", \"H\", \",\", \"L\", \"473\", \"I\", \")\", \".\"]\n",
      "Word token=> [\"Two\", \"mutations\", \",\", \"D\", \"128\", \"E\", \"and\", \"F\", \"216\", \"Y\", \",\", \"222\", \"are\", \"located\", \"near\", \"the\", \"interface\", \"between\", \"nsp\", \"10\", \"and\", \"nsp\", \"14\", \",\", \"though\", \"neither\", \"site\", \"has\", \"previously\", \"been\", \"223\", \"implicated\", \"in\", \"nsp\", \"10\", \"-nsp\", \"14\", \"interaction\", \"(\", \"15\", \",\", \"64\", \",\", \"65\", \")\", \".\"]\n",
      "Word token=> [\"One\", \"NS\", \"mutation\", \"resulted\", \"in\", \"a\", \"D\", \"272\", \"E\", \"224\", \"substitution\", \"in\", \"ExoN\", \"motif\", \"III\", \",\", \"a\", \"metal-coordinating\", \"active\", \"site\", \"residue\", \".\"]\n",
      "Word token=> [\"We\", \"previously\", \"reported\", \"225\", \"that\", \"alanine\", \"substitution\", \"of\", \"D\", \"272\", \"results\", \"in\", \"an\", \"ExoN\", \"(\", \"-\", \")\", \"phenotype\", \"(\", \"14\", \")\", \",\", \"but\", \"the\", \"viability\", \"or\", \"226\", \"phenotype\", \"of\", \"a\", \"D\", \"272\", \"E\", \"substitution\", \"was\", \"not\", \"tested\", \"in\", \"that\", \"study\", \".\"]\n",
      "Word token=> [\"These\", \"data\", \"suggest\", \"that\", \"a\", \"227\", \"network\", \"of\", \"residues\", \"evolved\", \"to\", \"regulate\", \"nsp\", \"12\", \"and\", \"nsp\", \"14\", \"activity\", \"or\", \"stability\", \"in\", \"the\", \"ExoN\", \"(\", \"-\", \")\", \"228\", \"background\", \".\"]\n",
      "Word token=> [\"229\", \"230\", \"Fixed\", \"mutations\", \"in\", \"nsp\", \"12\", \"and\", \"nsp\", \"14\", \"in\", \"MHV-ExoN\", \"(\", \"-\", \")\", \"P\", \"250\", \"directly\", \"correlate\", \"with\", \"231\", \"increased\", \"resistance\", \"to\", \"multiple\", \"nucleoside\", \"analogs\", \".\"]\n",
      "Word token=> [\"To\", \"determine\", \"approximately\", \"when\", \"the\", \"232\", \"mutations\", \"in\", \"nsp\", \"12\", \"and\", \"nsp\", \"14\", \"arose\", \",\", \"we\", \"performed\", \"di-deoxy\", \"sequencing\", \"across\", \"these\", \"protein-\", \"233\", \"coding\", \"regions\", \"roughly\", \"every\", \"20\", \"passages\", \"(\", \"P\", \"10\", \",\", \"31\", \",\", \"50\", \",\", \"72\", \",\", \"90\", \",\", \"100\", \",\", \"120\", \",\", \"140\", \",\", \"160\", \",\", \"180\", \",\", \"200\", \",\", \"220\", \",\", \"234\", \"240\", \")\", \".\"]\n",
      "Word token=> [\"By\", \"this\", \"method\", \",\", \"we\", \"detected\", \"consensus\", \"NS\", \"mutations\", \"at\", \"P\", \"10\", \",\", \"P\", \"50\", \",\", \"and\", \"P\", \"160\", \"for\", \"nsp\", \"12\", \",\", \"and\", \"235\", \"at\", \"P\", \"50\", \"and\", \"P\", \"160\", \"for\", \"nsp\", \"14\", \"(\", \"Fig.\", \"6\", \")\", \".\"]\n",
      "Word token=> [\"Both\", \"nsp\", \"12\", \"and\", \"nsp\", \"14\", \"carried\", \"their\", \"full\", \"complement\", \"of\", \"P\", \"250\", \"236\", \"consensus\", \"mutations\", \"by\", \"P\", \"160\", \",\", \"except\", \"for\", \"a\", \"minority\", \"variant\", \"(\", \"D\", \"913\", \"E\", \")\", \"in\", \"nsp\", \"12\", \"maintained\", \"at\", \"237\", \"<\", \"50\", \"%\", \"of\", \"the\", \"population\", \"between\", \"P\", \"200\", \"and\", \"P\", \"250\", \".\"]\n",
      "Word token=> [\"These\", \"passage\", \"levels\", \"correlated\", \"with\", \"increased\", \"238\", \"replication\", \"of\", \"MHV-ExoN\", \"(\", \"-\", \")\", \"(\", \"Fig.\", \"2\", \"B\", \")\", \"and\", \"with\", \"decreasing\", \"sensitivity\", \"to\", \"5\", \"-FU\", \"(\", \"Fig.\", \"7\", \"A\", \")\", \".\"]\n",
      "Word token=> [\"Neither\", \"239\", \"replication\", \"nor\", \"5\", \"-FU\", \"sensitivity\", \"of\", \"MHV-ExoN\", \"(\", \"-\", \")\", \"changed\", \"substantially\", \"between\", \"P\", \"160\", \"and\", \"P\", \"250\", \".\"]\n",
      "Word token=> [\"240\"]\n",
      "Word token=> [\"To\", \"determine\", \"whether\", \"MHV-ExoN\", \"(\", \"-\", \")\", \"evolved\", \"increased\", \"resistance\", \"to\", \"multiple\", \"nucleoside\", \"241\", \"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \".\"]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder.\", \"implicating\", \"a\", \"complex\", \"evolutionary\", \"interaction\", \"between\", \"these\", \"two\", \"proteins\", \".\"]\n",
      "Word token=> [\"The\", \"measured\", \"fitness\", \"284\", \"correlated\", \"with\", \"the\", \"patterns\", \"of\", \"nucleoside\", \"analog\", \"resistance\", \"and\", \"RNA\", \"synthesis\", \"associated\", \"with\", \"285\", \"mutations\", \"in\", \"nsp\", \"12\", \"and\", \"nsp\", \"14\", \",\", \"suggesting\", \"a\", \"link\", \"between\", \"the\", \"evolution\", \"of\", \"these\", \"virus\", \"phenotypes\", \".\"]\n",
      "Word token=> [\"286\", \"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"175562\", \"doi\", \":\", \"bioRxiv\", \"preprint\"]\n",
      "Word token=> [\"The\", \"result\", \"also\", \"confirms\", \"that\", \"nsp\", \"12\", \"and\", \"nsp\", \"14\", \"are\", \"important\", \"but\", \"not\", \"sufficient\", \"to\", \"account\", \"for\", \"the\", \"287\", \"significantly\", \"increased\", \"fitness\", \"of\", \"MHV-ExoN\", \"(\", \"-\", \")\", \"P\", \"250\", \"relative\", \"to\", \"MHV-ExoN\", \"(\", \"-\", \")\", \"P\", \"3\", \".\"]\n",
      "Word token=> [\"288\", \"289\", \"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \".\"]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \".\"]\n",
      "Word token=> [\"Mutations\", \"conferring\", \"increased\", \"replication\", \"fidelity\", \"to\", \"RNA\", \"viruses\", \"have\", \"most\", \"frequently\", \"been\", \"334\", \"mapped\", \"to\", \"RdRps\", \"(\", \"24\", \",\", \"25\", \",\", \"30\", \",\", \"72\", \")\", \".\"]\n",
      "Word token=> [\"Three\", \"findings\", \"suggest\", \"that\", \"mutations\", \"within\", \"nsp\", \"12\", \"-P\", \"250\", \"335\", \"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder.\", \"with\", \"nsp\", \"12\", \",\", \"acting\", \"as\", \"a\", \"primase/processivity\", \"factor\", \"(\", \"74\", \",\", \"75\", \")\", \"and\", \"a\", \"helicase/NTPase\", \",\", \"respectively\", \"358\", \"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \".\"]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"Cell\", \"culture\", \".\"]\n",
      "Word token=> [\"DBT-\", \"9\", \"(\", \"delayed\", \"brain\", \"tumor\", \",\", \"murine\", \"astrocytoma\", \"clone\", \"9\", \")\", \"cells\", \"were\", \"maintained\", \"409\", \"as\", \"described\", \"previously\", \"(\", \"91\", \")\", \"The\", \"virus\", \"titer\", \"of\", \"each\", \"stock\", \"was\", \"determined\", \"by\", \"plaque\", \"assay\", \"using\", \"DBT-\", \"9\", \"cells\", \"as\", \"described\", \"425\", \"previously\", \"(\", \"14\", \",\", \"91\", \")\", \".\"]\n",
      "Word token=> [\"For\", \"plaque\", \"assays\", \"of\", \"viruses\", \"containing\", \"the\", \"spike\", \"protein\", \"from\", \"MHV-ExoN\", \"(\", \"-\", \"426\", \")\", \"P\", \"250\", \",\", \"which\", \"does\", \"not\", \"form\", \"syncytia\", \",\", \"plaques\", \"were\", \"visualized\", \"with\", \"neutral\", \"red\", \"(\", \"Sigma\", \"#\", \"N\", \"6264\", \",\", \"427\", \"diluted\", \"1:10\", \"in\", \"PBS\", \"containing\", \"calcium\", \"and\", \"magnesium\", \")\", \".\"]\n",
      "Word token=> [\"Neutral\", \"red\", \"was\", \"added\", \"24\", \"hours\", \"after\", \"428\", \"plating\", \"and\", \"incubated\", \"for\", \"an\", \"additional\", \"3\", \"-\", \"8\", \"hours\", \"before\", \"formaldehyde\", \"fixation\", \".\"]\n",
      "Word token=> [\"Plaque\", \"429\", \"purification\", \"was\", \"performed\", \"by\", \"infecting\", \"DBT\", \"cells\", \"with\", \"serial\", \"dilutions\", \"of\", \"virus\", \"and\", \"overlaying\", \"430\", \"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"175562\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"the\", \"cultures\", \"with\", \"agar\", \".\"]\n",
      "Word token=> [\"Single\", \"plaques\", \"were\", \"isolated\", \",\", \"resuspended\", \"in\", \"PBS\", \"containing\", \"calcium\", \"and\", \"431\", \"magnesium\", \",\", \"and\", \"inoculated\", \"onto\", \"fresh\", \"DBTs\", \".\"]\n",
      "Word token=> [\"This\", \"process\", \"was\", \"completed\", \"3\", \"times\", \"before\", \"432\", \"experimental\", \"stocks\", \"were\", \"generated\", \",\", \"as\", \"above.\", \"and\", \"RNA\", \"copy\", \"number\", \"was\", \"calculated\", \"by\", \"reference\", \"to\", \"an\", \"RNA\", \"standard\", \"derived\", \"from\", \"the\", \"MHV\", \"475\", \"A\", \"fragment\", \".\"]\n",
      "Word token=> [\"Samples\", \"were\", \"plated\", \"in\", \"technical\", \"duplicate\", \"to\", \"minimize\", \"well-to-well\", \"variation\", \".\"]\n",
      "Word token=> [\"Titers\", \"476\", \"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"175562\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"were\", \"determined\", \"by\", \"plaque\", \"assay\", \"in\", \"DBT-\", \"9\", \"cells\", \",\", \"and\", \"specific\", \"infectivity\", \"was\", \"calculated\", \"as\", \"PFU\", \"477\", \"per\", \"supernatant\", \"genomic\", \"RNA\", \"copy\", \".\"]\n",
      "Word token=> [\"478\"]\n",
      "Word token=> [\"Nucleoside\", \"and\", \"base\", \"analog\", \"sensitivity\", \"assays\", \".\"]\n",
      "Word token=> [\"5\", \"-azacytidine\", \"(\", \"AZC\", \")\", \",\", \"5\", \"-fluorouracil\", \"(\", \"5\", \"-FU\", \")\", \",\", \"and\", \"480\", \"ribavirin\", \"(\", \"RBV\", \")\", \"were\", \"purchased\", \"from\", \"Sigma\", \"(\", \"product\", \"numbers\", \"A\", \"2385\", \",\", \"F\", \"6627\", \",\", \"and\", \"R\", \"9644\", \",\", \"481\", \"respectively\", \")\", \".\"]\n",
      "Word token=> [\"Stock\", \"solutions\", \"of\", \"5\", \"-FU\", \"and\", \"RBV\", \"were\", \"prepared\", \"in\", \"dimethyl\", \"sulfoxide\", \"(\", \"DMSO\", \")\", \".\"]\n",
      "Word token=> [\"482\"]\n",
      "Word token=> [\"2\", \"'\", \"-C-methyladenosine\", \"(\", \"CMeA\", \")\", \"was\", \"received\", \"from\", \"Gilead\", \"Sciences\", \",\", \"Inc\", \"(\", \"Foster\", \"City\", \",\", \"CA\", \")\", \".\"]\n",
      "Word token=> [\"483\"]\n",
      "Word token=> [\"Sensitivity\", \"assays\", \"were\", \"performed\", \"as\", \"described\", \"previously\", \"(\", \"16\", \")\", \"viruses\", \"were\", \"detected\", \"with\", \"the\", \"same\", \"probe\", \"used\", \"in\", \"specific\", \"infectivity\", \"analyses\", \"(\", \"14\", \")\", \".\"]\n",
      "Word token=> [\"Reference\", \"516\", \"viruses\", \"were\", \"detected\", \"by\", \"a\", \"probe\", \"targeting\", \"the\", \"same\", \"region\", \"but\", \"with\", \"10\", \"silent\", \"mutations\", \"(\", \"5\", \"'\", \"-\", \"517\", \"TCCGAACTACTGCAACCCCAAGTG-\", \"3\", \"'\", \")\", \"and\", \"labeled\", \"with\", \"5\", \"'\", \"Quasar\", \"670\", \"and\", \"3\", \"'\", \"black\", \"hole\", \"518\", \"quencher\", \"2\", \"(\", \"BHQ-\", \"2\", \")\", \"(\", \"Biosearch\", \"Technologies\", \",\", \"Petaluma\", \",\", \"CA\", \")\", \".\"]\n",
      "Word token=> [\"RNA\", \"copy\", \"number\", \"was\", \"519\", \"calculated\", \"by\", \"reference\", \"to\", \"an\", \"RNA\", \"standard\", \"generated\", \"by\", \"in\", \"vitro\", \"transcription\", \"of\", \"the\", \"520\", \"corresponding\", \"MHV\", \"A\", \"fragment\", \",\", \"and\", \"relative\", \"RNA\", \"abundance\", \"was\", \"calculated\", \"as\", \"the\", \"ratio\", \"of\", \"521\", \"competitor\", \"genomes\", \"to\", \"reference\", \"genomes\", \".\"]\n",
      "Word token=> [\"522\", \"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \".\"]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"175562\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"and\", \"synonymous\", \"mutations\", \"(\", \"purple\", \")\", \"are\", \"below\", \".\"]\n",
      "Word token=> [\"564\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"175562\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"statistical\", \"significance\", \"was\", \"determined\", \"using\", \"one-way\", \"ANOVA\", \".\"]\n",
      "Word token=> [\"For\", \"Panel\", \"C\", \",\", \"statistical\", \"587\", \"significance\", \"for\", \"change\", \"in\", \"titer\", \"of\", \"MHV-ExoN\", \"(\", \"-\", \")\", \"spike-P\", \"250\", \"relative\", \"to\", \"MHV-ExoN\", \"(\", \"-\", \")\", \"P\", \"3\", \"was\", \"588\", \"determined\", \"using\", \"Mann-Whitney\", \"test\", \"(\", \"*\", \"P<\", \"0.05\", \",\", \"*\", \"*\", \"<\", \"0.01\", \",\", \"*\", \"*\", \"*\", \"P<\", \"0.001\", \",\", \"ns\", \"=\", \"not\", \"significant\", \")\", \".\"]\n",
      "Word token=> [\"589\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"175562\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"175562\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"175562\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"tuberculosis\", \"DNA\", \"polymerase\", \"III\", \"αβ\", \"2\", \"ε\", \"replicase\", \"promotes\", \"polymerization\", \"and\", \"reduces\", \"855\", \"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"175562\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \".\"]\n",
      "Word token=> [\"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \".\"]\n",
      "Word token=> [\"Virus\", \"titer\", \"(\", \"log\", \"10\", \"PFU/ml\", \")\", \"All\", \"rights\", \"reserved\", \".\"]\n",
      "Word token=> [\"No\", \"reuse\", \"allowed\", \"without\", \"permission\", \".\"]\n",
      "Word token=> String[]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"175562\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"175562\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"175562\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \".\"]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \".\"]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \".\"]\n",
      "Word token=> [\"The\", \"copyright\", \"holder\", \"for\", \"this\", \"preprint\", \"(\", \"which\", \"was\", \"not\", \"peer-reviewed\", \")\", \"is\", \"the\", \"author/funder\", \"https://doi.org/\", \"10.1101\", \"/\", \"175562\", \"doi\", \":\", \"bioRxiv\", \"preprint\", \".\"]\n",
      "Word token=> String[]\n"
     ]
    }
   ],
   "source": [
    "for sentence in split_sentences(text(sent_files))\n",
    "    wordtokens = tokenize(sentence)\n",
    "    println(\"Word token=> $wordtokens\")\n",
    "end"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "FileDocument(\"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\", TextAnalysis.DocumentMetadata(Languages.English(), \"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\", \"Unknown Author\", \"Unknown Time\"))"
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sd1 = FileDocument(\"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "StringDocument{String}(\"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\", TextAnalysis.DocumentMetadata(Languages.English(), \"Untitled Document\", \"Unknown Author\", \"Unknown Time\"))"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#sd3 = StringDocument(\"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Dict{String,Int64} with 6204 entries:\n",
       "  \"Forward\"           => 1\n",
       "  \"P-values\"          => 1\n",
       "  \"proteases\"         => 36\n",
       "  \"titration\"         => 1\n",
       "  \"whose\"             => 1\n",
       "  \"favor\"             => 1\n",
       "  \"228\"               => 4\n",
       "  \"/Nsp\"              => 1\n",
       "  \"strand\"            => 1\n",
       "  \"Rz\"                => 1\n",
       "  \"plan\"              => 1\n",
       "  \"those\"             => 3\n",
       "  \"concentrations\"    => 8\n",
       "  \"Biosystems\"        => 1\n",
       "  \"deletions\"         => 2\n",
       "  \"months\"            => 1\n",
       "  \"9.2\"               => 1\n",
       "  \"Core\"              => 1\n",
       "  \"rather\"            => 3\n",
       "  \"tube.\"             => 2\n",
       "  \"general\"           => 9\n",
       "  \"lacking\"           => 3\n",
       "  \"2020.03.01.971499\" => 13\n",
       "  \"dye\"               => 2\n",
       "  \"these\"             => 46\n",
       "  ⋮                   => ⋮"
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ngrams(sd1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Dict{String,Int64} with 10 entries:\n",
       "  \":\"                            => 1\n",
       "  \"Cluster\"                      => 1\n",
       "  \"1\"                            => 1\n",
       "  \"C\"                            => 1\n",
       "  \"19\"                           => 1\n",
       "  \"/treat\"                       => 1\n",
       "  \".txt\"                         => 1\n",
       "  \"Panwar/Desktop/python/covid-\" => 1\n",
       "  \"_\"                            => 2\n",
       "  \"/Users/Chandrapal\"            => 1"
      ]
     },
     "execution_count": 39,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#ngrams(sd3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Dict{AbstractString,Int64} with 28297 entries:\n",
       "  \"its slow\"              => 1\n",
       "  \"to +\"                  => 1\n",
       "  \"for aketoamides\"       => 1\n",
       "  \"mouse '\"               => 1\n",
       "  \"from MHV-ExoN\"         => 1\n",
       "  \"u h\"                   => 1\n",
       "  \"proteases\"             => 36\n",
       "  \"obtained by\"           => 2\n",
       "  \"their almost\"          => 1\n",
       "  \"= 0.0048\"              => 1\n",
       "  \"MERS-CoV )\"            => 1\n",
       "  \"( Thermo\"              => 3\n",
       "  \"titration\"             => 1\n",
       "  \"suspension was\"        => 3\n",
       "  \"cortex cells\"          => 1\n",
       "  \"was isobutyl.\"         => 1\n",
       "  \"260 higher\"            => 1\n",
       "  \"228\"                   => 4\n",
       "  \"the region\"            => 1\n",
       "  \"/Nsp\"                  => 1\n",
       "  \"and probably\"          => 1\n",
       "  \"human STAT\"            => 1\n",
       "  \"cells 197\"             => 1\n",
       "  \"( h\"                   => 1\n",
       "  \"the structures\"        => 2\n",
       "  ⋮                       => ⋮"
      ]
     },
     "execution_count": 40,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ngrams(sd1,2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\". The next\", 2)\n",
      "Pair{AbstractString,Int64}(\"staining. All primary\", 1)\n",
      "Pair{AbstractString,Int64}(\"A 16 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"from MHV-ExoN\", 1)\n",
      "Pair{AbstractString,Int64}(\"= 0.0048\", 1)\n",
      "Pair{AbstractString,Int64}(\"site residue. We\", 1)\n",
      "Pair{AbstractString,Int64}(\"to remember ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"titration\", 1)\n",
      "Pair{AbstractString,Int64}(\"found the P\", 1)\n",
      "Pair{AbstractString,Int64}(\"was isobutyl.\", 1)\n",
      "Pair{AbstractString,Int64}(\"the boundary of\", 1)\n",
      "Pair{AbstractString,Int64}(\"CPE. For each\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells. The compound\", 1)\n",
      "Pair{AbstractString,Int64}(\"filtered. The light-brown\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 ) showed\", 2)\n",
      "Pair{AbstractString,Int64}(\", and had\", 1)\n",
      "Pair{AbstractString,Int64}(\"Core\", 1)\n",
      "Pair{AbstractString,Int64}(\"measurement. c\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibitor optimization study\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 (\", 2)\n",
      "Pair{AbstractString,Int64}(\"flow cytometry the\", 1)\n",
      "Pair{AbstractString,Int64}(\"originating from a\", 1)\n",
      "Pair{AbstractString,Int64}(\"methods\", 2)\n",
      "Pair{AbstractString,Int64}(\"the perception of\", 2)\n",
      "Pair{AbstractString,Int64}(\"series resistance\", 1)\n",
      "Pair{AbstractString,Int64}(\"a-ketoamide ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"was elevated in\", 1)\n",
      "Pair{AbstractString,Int64}(\"or odor presentation\", 1)\n",
      "Pair{AbstractString,Int64}(\"_ ctrl cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"to pinene\", 1)\n",
      "Pair{AbstractString,Int64}(\"of coronaviral infection\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 x\", 8)\n",
      "Pair{AbstractString,Int64}(\"24 ° C\", 1)\n",
      "Pair{AbstractString,Int64}(\"[\", 23)\n",
      "Pair{AbstractString,Int64}(\"purpose ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"In\", 64)\n",
      "Pair{AbstractString,Int64}(\"methyltransferase domain (\", 1)\n",
      "Pair{AbstractString,Int64}(\"transduced population\", 1)\n",
      "Pair{AbstractString,Int64}(\"were reconstituted\", 1)\n",
      "Pair{AbstractString,Int64}(\"The fact\", 1)\n",
      "Pair{AbstractString,Int64}(\"28 ( PDB\", 1)\n",
      "Pair{AbstractString,Int64}(\"points and\", 2)\n",
      "Pair{AbstractString,Int64}(\"product 1 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"Two mutations ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"for 20 min\", 2)\n",
      "Pair{AbstractString,Int64}(\"metal-coordinating active\", 1)\n",
      "Pair{AbstractString,Int64}(\"air flowed\", 1)\n",
      "Pair{AbstractString,Int64}(\"existing studies of\", 1)\n",
      "Pair{AbstractString,Int64}(\"312 pseudoviruses decorated\", 1)\n",
      "Pair{AbstractString,Int64}(\"in treating\", 1)\n",
      "Pair{AbstractString,Int64}(\"and compensated\", 1)\n",
      "Pair{AbstractString,Int64}(\"were converted to\", 1)\n",
      "Pair{AbstractString,Int64}(\"groups.\", 1)\n",
      "Pair{AbstractString,Int64}(\"91 ( Figure\", 1)\n",
      "Pair{AbstractString,Int64}(\", Ltd.\", 1)\n",
      "Pair{AbstractString,Int64}(\"observed efficient\", 1)\n",
      "Pair{AbstractString,Int64}(\"6 J\", 1)\n",
      "Pair{AbstractString,Int64}(\"p= 0.0073\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 1\", 15)\n",
      "Pair{AbstractString,Int64}(\"assigned for degree\", 1)\n",
      "Pair{AbstractString,Int64}(\"v 2 x\", 1)\n",
      "Pair{AbstractString,Int64}(\") down-field from\", 1)\n",
      "Pair{AbstractString,Int64}(\"as mean values\", 1)\n",
      "Pair{AbstractString,Int64}(\"tagged in our\", 1)\n",
      "Pair{AbstractString,Int64}(\"the CS- (\", 1)\n",
      "Pair{AbstractString,Int64}(\"several drastic changes\", 1)\n",
      "Pair{AbstractString,Int64}(\"in MEM\", 1)\n",
      "Pair{AbstractString,Int64}(\"recombinant SARS-CoV M\", 3)\n",
      "Pair{AbstractString,Int64}(\"RC-\", 1)\n",
      "Pair{AbstractString,Int64}(\"demonstrating that murine\", 1)\n",
      "Pair{AbstractString,Int64}(\"assay.\", 2)\n",
      "Pair{AbstractString,Int64}(\"have most\", 1)\n",
      "Pair{AbstractString,Int64}(\"CS- , eugenol\", 2)\n",
      "Pair{AbstractString,Int64}(\"low and high\", 1)\n",
      "Pair{AbstractString,Int64}(\"complete genome were\", 1)\n",
      "Pair{AbstractString,Int64}(\"cases and\", 3)\n",
      "Pair{AbstractString,Int64}(\"A similar observation\", 1)\n",
      "Pair{AbstractString,Int64}(\"form syncytia\", 1)\n",
      "Pair{AbstractString,Int64}(\"up-regulation 302\", 1)\n",
      "Pair{AbstractString,Int64}(\", and ExoN-inactivation\", 1)\n",
      "Pair{AbstractString,Int64}(\"or LY 6\", 1)\n",
      "Pair{AbstractString,Int64}(\") +\", 2)\n",
      "Pair{AbstractString,Int64}(\"related feature shared\", 1)\n",
      "Pair{AbstractString,Int64}(\"81,000 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\", a good\", 1)\n",
      "Pair{AbstractString,Int64}(\"modeled the\", 1)\n",
      "Pair{AbstractString,Int64}(\"not warrant\", 1)\n",
      "Pair{AbstractString,Int64}(\"after/before between groups\", 1)\n",
      "Pair{AbstractString,Int64}(\"8 in\", 1)\n",
      "Pair{AbstractString,Int64}(\"replicon , 0.45\", 1)\n",
      "Pair{AbstractString,Int64}(\"µM ,\", 18)\n",
      "Pair{AbstractString,Int64}(\"fresh DBTs. This\", 1)\n",
      "Pair{AbstractString,Int64}(\"inactivation with temperature\", 1)\n",
      "Pair{AbstractString,Int64}(\"formation assay\", 1)\n",
      "Pair{AbstractString,Int64}(\"intracellular Renilla\", 1)\n",
      "Pair{AbstractString,Int64}(\"with serial\", 1)\n",
      "Pair{AbstractString,Int64}(\"of P 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"These authors\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells , dendritic\", 1)\n",
      "Pair{AbstractString,Int64}(\"low basal\", 1)\n",
      "Pair{AbstractString,Int64}(\"mice were silenced\", 1)\n",
      "Pair{AbstractString,Int64}(\"PEG 3,350 varied\", 1)\n",
      "Pair{AbstractString,Int64}(\"matchedpairs signed rank\", 1)\n",
      "Pair{AbstractString,Int64}(\"D.L. ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"every\", 1)\n",
      "Pair{AbstractString,Int64}(\"often referred to\", 1)\n",
      "Pair{AbstractString,Int64}(\"significantly inhibited rubella\", 1)\n",
      "Pair{AbstractString,Int64}(\"observed a similar\", 1)\n",
      "Pair{AbstractString,Int64}(\"authors mainly studied\", 1)\n",
      "Pair{AbstractString,Int64}(\", Figure 1\", 5)\n",
      "Pair{AbstractString,Int64}(\"' s solution.\", 4)\n",
      "Pair{AbstractString,Int64}(\"introduction of\", 1)\n",
      "Pair{AbstractString,Int64}(\"albeit\", 1)\n",
      "Pair{AbstractString,Int64}(\"not erase\", 1)\n",
      "Pair{AbstractString,Int64}(\"rates in RNA\", 1)\n",
      "Pair{AbstractString,Int64}(\"most KLK\", 1)\n",
      "Pair{AbstractString,Int64}(\"VSV glycoprotein G\", 2)\n",
      "Pair{AbstractString,Int64}(\"purified KLK 13\", 1)\n",
      "Pair{AbstractString,Int64}(\"of the DEDDh\", 1)\n",
      "Pair{AbstractString,Int64}(\"by compound-specific\", 1)\n",
      "Pair{AbstractString,Int64}(\"311 -C 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"Previous exposure to\", 1)\n",
      "Pair{AbstractString,Int64}(\"protein 5\", 1)\n",
      "Pair{AbstractString,Int64}(\"RaTG 13 was\", 2)\n",
      "Pair{AbstractString,Int64}(\"Shim-pack GIST C\", 1)\n",
      "Pair{AbstractString,Int64}(\"and human\", 2)\n",
      "Pair{AbstractString,Int64}(\"changes were\", 2)\n",
      "Pair{AbstractString,Int64}(\"0832 , Sigma-Aldrich\", 1)\n",
      "Pair{AbstractString,Int64}(\"NaHCO 3 (\", 2)\n",
      "Pair{AbstractString,Int64}(\". A\", 7)\n",
      "Pair{AbstractString,Int64}(\"in PBST and\", 1)\n",
      "Pair{AbstractString,Int64}(\"to \\\"\", 1)\n",
      "Pair{AbstractString,Int64}(\"° C. Medium\", 1)\n",
      "Pair{AbstractString,Int64}(\"incubated for 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"132 and His\", 1)\n",
      "Pair{AbstractString,Int64}(\"outbreak of severe\", 1)\n",
      "Pair{AbstractString,Int64}(\"15 , [\", 1)\n",
      "Pair{AbstractString,Int64}(\"experiments. In\", 1)\n",
      "Pair{AbstractString,Int64}(\"roughly antiperiplanar\", 1)\n",
      "Pair{AbstractString,Int64}(\"devised a\", 1)\n",
      "Pair{AbstractString,Int64}(\"five 220\", 1)\n",
      "Pair{AbstractString,Int64}(\"mutations among different\", 1)\n",
      "Pair{AbstractString,Int64}(\"This dimerization is\", 1)\n",
      "Pair{AbstractString,Int64}(\"solutions of the\", 1)\n",
      "Pair{AbstractString,Int64}(\"NK\", 3)\n",
      "Pair{AbstractString,Int64}(\"3.48 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"expected to be\", 1)\n",
      "Pair{AbstractString,Int64}(\"Outliers and\", 1)\n",
      "Pair{AbstractString,Int64}(\"46 described\", 1)\n",
      "Pair{AbstractString,Int64}(\"high false negative\", 1)\n",
      "Pair{AbstractString,Int64}(\") to vary\", 1)\n",
      "Pair{AbstractString,Int64}(\"and CSpresentation was\", 1)\n",
      "Pair{AbstractString,Int64}(\"6 cells were\", 1)\n",
      "Pair{AbstractString,Int64}(\"2015 )\", 8)\n",
      "Pair{AbstractString,Int64}(\"injection. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"replicons were\", 1)\n",
      "Pair{AbstractString,Int64}(\"nucleotide substitution of\", 1)\n",
      "Pair{AbstractString,Int64}(\"to viruses (\", 1)\n",
      "Pair{AbstractString,Int64}(\". A crystal\", 1)\n",
      "Pair{AbstractString,Int64}(\", the spike\", 1)\n",
      "Pair{AbstractString,Int64}(\"with NeuroTrace\", 1)\n",
      "Pair{AbstractString,Int64}(\"chosen because they\", 1)\n",
      "Pair{AbstractString,Int64}(\"recordings from one\", 1)\n",
      "Pair{AbstractString,Int64}(\"are critical\", 1)\n",
      "Pair{AbstractString,Int64}(\"was also\", 8)\n",
      "Pair{AbstractString,Int64}(\"of enzymatic\", 1)\n",
      "Pair{AbstractString,Int64}(\"Table 3\", 12)\n",
      "Pair{AbstractString,Int64}(\"5 µM\", 6)\n",
      "Pair{AbstractString,Int64}(\"reservoir ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"equipotent\", 2)\n",
      "Pair{AbstractString,Int64}(\"qRT-PCR ) technology\", 1)\n",
      "Pair{AbstractString,Int64}(\"between CVB 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"and more than\", 1)\n",
      "Pair{AbstractString,Int64}(\"71.70 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"the Sigma\", 1)\n",
      "Pair{AbstractString,Int64}(\"may question\", 1)\n",
      "Pair{AbstractString,Int64}(\"test the effect\", 1)\n",
      "Pair{AbstractString,Int64}(\"alter exploratory\", 1)\n",
      "Pair{AbstractString,Int64}(\"lithium bis\", 1)\n",
      "Pair{AbstractString,Int64}(\"was developed\", 1)\n",
      "Pair{AbstractString,Int64}(\"encouraging us to\", 1)\n",
      "Pair{AbstractString,Int64}(\"ExoN-AE has 258\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"of select genes\", 1)\n",
      "Pair{AbstractString,Int64}(\"PromKine )\", 1)\n",
      "Pair{AbstractString,Int64}(\"taken into account.\", 1)\n",
      "Pair{AbstractString,Int64}(\"The initial compound\", 1)\n",
      "Pair{AbstractString,Int64}(\"Wuhan\", 4)\n",
      "Pair{AbstractString,Int64}(\"were identified\", 3)\n",
      "Pair{AbstractString,Int64}(\"and converted\", 1)\n",
      "Pair{AbstractString,Int64}(\"split-luciferase protein (\", 1)\n",
      "Pair{AbstractString,Int64}(\"h prior to\", 1)\n",
      "Pair{AbstractString,Int64}(\"period (\", 1)\n",
      "Pair{AbstractString,Int64}(\"difference density (\", 1)\n",
      "Pair{AbstractString,Int64}(\"were suspended and\", 1)\n",
      "Pair{AbstractString,Int64}(\", HAE _\", 3)\n",
      "Pair{AbstractString,Int64}(\"Antiviral membrane\", 1)\n",
      "Pair{AbstractString,Int64}(\"methyl\", 4)\n",
      "Pair{AbstractString,Int64}(\"the low micromolar\", 1)\n",
      "Pair{AbstractString,Int64}(\"even against\", 1)\n",
      "Pair{AbstractString,Int64}(\"of 50 ×\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 with other\", 1)\n",
      "Pair{AbstractString,Int64}(\"' region remained\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells according to\", 1)\n",
      "Pair{AbstractString,Int64}(\"' site\", 2)\n",
      "Pair{AbstractString,Int64}(\"agreement\", 3)\n",
      "Pair{AbstractString,Int64}(\"similarly observed\", 1)\n",
      "Pair{AbstractString,Int64}(\"RT-PCR ( Fig.\", 1)\n",
      "Pair{AbstractString,Int64}(\"boxes.\", 1)\n",
      "Pair{AbstractString,Int64}(\"habituation/dishabituation assay\", 1)\n",
      "Pair{AbstractString,Int64}(\"people blind\", 1)\n",
      "Pair{AbstractString,Int64}(\"Wentworth\", 1)\n",
      "Pair{AbstractString,Int64}(\", nor the\", 1)\n",
      "Pair{AbstractString,Int64}(\"vector. Depicted are\", 1)\n",
      "Pair{AbstractString,Int64}(\"556 sonicated\", 1)\n",
      "Pair{AbstractString,Int64}(\"These data\", 6)\n",
      "Pair{AbstractString,Int64}(\"corticosteroids is contraindicated\", 1)\n",
      "Pair{AbstractString,Int64}(\"separation on\", 1)\n",
      "Pair{AbstractString,Int64}(\"to obtain\", 1)\n",
      "Pair{AbstractString,Int64}(\"that study.\", 1)\n",
      "Pair{AbstractString,Int64}(\"floor were different\", 1)\n",
      "Pair{AbstractString,Int64}(\", the nearly\", 1)\n",
      "Pair{AbstractString,Int64}(\"( ExoN- 134\", 1)\n",
      "Pair{AbstractString,Int64}(\", 2016\", 7)\n",
      "Pair{AbstractString,Int64}(\". The two\", 2)\n",
      "Pair{AbstractString,Int64}(\"resonance energy transfer\", 1)\n",
      "Pair{AbstractString,Int64}(\"encode a 67\", 1)\n",
      "Pair{AbstractString,Int64}(\"complete\", 7)\n",
      "Pair{AbstractString,Int64}(\"empty\", 6)\n",
      "Pair{AbstractString,Int64}(\"force\", 1)\n",
      "Pair{AbstractString,Int64}(\"was essentially\", 1)\n",
      "Pair{AbstractString,Int64}(\"DREADD AAV-tetO-hM 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"patched\", 1)\n",
      "Pair{AbstractString,Int64}(\") . Overall\", 2)\n",
      "Pair{AbstractString,Int64}(\"traceability ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"( n= 11\", 1)\n",
      "Pair{AbstractString,Int64}(\"modified behavioral task\", 1)\n",
      "Pair{AbstractString,Int64}(\"entrain\", 1)\n",
      "Pair{AbstractString,Int64}(\"341 3 '\", 1)\n",
      "Pair{AbstractString,Int64}(\"also deleted\", 1)\n",
      "Pair{AbstractString,Int64}(\"well in non-\", 1)\n",
      "Pair{AbstractString,Int64}(\"shown that spatial\", 1)\n",
      "Pair{AbstractString,Int64}(\"C \\\"\", 1)\n",
      "Pair{AbstractString,Int64}(\"a glutamine\", 1)\n",
      "Pair{AbstractString,Int64}(\"requirements\", 2)\n",
      "Pair{AbstractString,Int64}(\", 11 n\", 3)\n",
      "Pair{AbstractString,Int64}(\"RBV\", 2)\n",
      "Pair{AbstractString,Int64}(\"system to control\", 1)\n",
      "Pair{AbstractString,Int64}(\") , and\", 39)\n",
      "Pair{AbstractString,Int64}(\"were analyzed by\", 2)\n",
      "Pair{AbstractString,Int64}(\"debris ( 4,000\", 1)\n",
      "Pair{AbstractString,Int64}(\"role of\", 3)\n",
      "Pair{AbstractString,Int64}(\"during odor fear\", 1)\n",
      "Pair{AbstractString,Int64}(\"only for aketoamides\", 1)\n",
      "Pair{AbstractString,Int64}(\". Mice thus\", 1)\n",
      "Pair{AbstractString,Int64}(\"poliovirus 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"albeit still weak\", 1)\n",
      "Pair{AbstractString,Int64}(\"43\", 10)\n",
      "Pair{AbstractString,Int64}(\"the hydroxy\", 1)\n",
      "Pair{AbstractString,Int64}(\"for another 6\", 1)\n",
      "Pair{AbstractString,Int64}(\"methanol\", 5)\n",
      "Pair{AbstractString,Int64}(\"0.14 ) cm/s\", 1)\n",
      "Pair{AbstractString,Int64}(\"530\", 1)\n",
      "Pair{AbstractString,Int64}(\"% polyethylene\", 1)\n",
      "Pair{AbstractString,Int64}(\"20 -fold\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells were used\", 1)\n",
      "Pair{AbstractString,Int64}(\"j , ns\", 3)\n",
      "Pair{AbstractString,Int64}(\"proteases had\", 1)\n",
      "Pair{AbstractString,Int64}(\"carried out on\", 1)\n",
      "Pair{AbstractString,Int64}(\"Gene Expression Omnibus\", 1)\n",
      "Pair{AbstractString,Int64}(\"remain poorly\", 1)\n",
      "Pair{AbstractString,Int64}(\"primed specificity\", 1)\n",
      "Pair{AbstractString,Int64}(\"( see above\", 1)\n",
      "Pair{AbstractString,Int64}(\"down-regulated\", 1)\n",
      "Pair{AbstractString,Int64}(\"2010 ; Economo\", 1)\n",
      "Pair{AbstractString,Int64}(\"compound exhibits very\", 1)\n",
      "Pair{AbstractString,Int64}(\"M pro has\", 1)\n",
      "Pair{AbstractString,Int64}(\") . Collectively\", 2)\n",
      "Pair{AbstractString,Int64}(\"olfactory-driven behaviors are\", 1)\n",
      "Pair{AbstractString,Int64}(\"methyl L-phenylalaninate\", 1)\n",
      "Pair{AbstractString,Int64}(\"= 0.8 -\", 1)\n",
      "Pair{AbstractString,Int64}(\"Medium\", 1)\n",
      "Pair{AbstractString,Int64}(\"role of LY\", 1)\n",
      "Pair{AbstractString,Int64}(\") . 33\", 1)\n",
      "Pair{AbstractString,Int64}(\"the ExoN domain\", 1)\n",
      "Pair{AbstractString,Int64}(\"field during\", 1)\n",
      "Pair{AbstractString,Int64}(\"in the fluorescence\", 1)\n",
      "Pair{AbstractString,Int64}(\"products were\", 1)\n",
      "Pair{AbstractString,Int64}(\"Counting Z-stacks (\", 1)\n",
      "Pair{AbstractString,Int64}(\"( eugenol\", 1)\n",
      "Pair{AbstractString,Int64}(\", 2003 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"get a colorless\", 1)\n",
      "Pair{AbstractString,Int64}(\"-digit nanomolar range\", 1)\n",
      "Pair{AbstractString,Int64}(\"related with Bat-SARSr-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"experience , and\", 1)\n",
      "Pair{AbstractString,Int64}(\"-MHz ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"points and titrated\", 1)\n",
      "Pair{AbstractString,Int64}(\"excluded\", 4)\n",
      "Pair{AbstractString,Int64}(\"configuration in\", 2)\n",
      "Pair{AbstractString,Int64}(\"in its\", 1)\n",
      "Pair{AbstractString,Int64}(\") , 2\", 2)\n",
      "Pair{AbstractString,Int64}(\"shock does\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 3.48\", 1)\n",
      "Pair{AbstractString,Int64}(\"594 (\", 1)\n",
      "Pair{AbstractString,Int64}(\", indicating that\", 5)\n",
      "Pair{AbstractString,Int64}(\"8 x\", 1)\n",
      "Pair{AbstractString,Int64}(\"post-infection , cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"follows : The\", 2)\n",
      "Pair{AbstractString,Int64}(\"culture ( OD\", 1)\n",
      "Pair{AbstractString,Int64}(\"behavior was\", 2)\n",
      "Pair{AbstractString,Int64}(\"to bregma. HA-stained\", 1)\n",
      "Pair{AbstractString,Int64}(\"The whole genome\", 1)\n",
      "Pair{AbstractString,Int64}(\"comprehensive comparative analysis\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 /cell surface\", 1)\n",
      "Pair{AbstractString,Int64}(\"B 3\", 3)\n",
      "Pair{AbstractString,Int64}(\"Technology by the\", 1)\n",
      "Pair{AbstractString,Int64}(\"124 in a\", 1)\n",
      "Pair{AbstractString,Int64}(\"methylene\", 2)\n",
      "Pair{AbstractString,Int64}(\"RNA bands in\", 1)\n",
      "Pair{AbstractString,Int64}(\"be used\", 5)\n",
      "Pair{AbstractString,Int64}(\"exclude any inherent\", 1)\n",
      "Pair{AbstractString,Int64}(\"Storage of\", 1)\n",
      "Pair{AbstractString,Int64}(\"cases of accompanying\", 1)\n",
      "Pair{AbstractString,Int64}(\"powerful and non-toxic\", 1)\n",
      "Pair{AbstractString,Int64}(\"Apical\", 1)\n",
      "Pair{AbstractString,Int64}(\"only 298\", 1)\n",
      "Pair{AbstractString,Int64}(\"in 293\", 1)\n",
      "Pair{AbstractString,Int64}(\"number of infected\", 1)\n",
      "Pair{AbstractString,Int64}(\", or TMPRSS\", 1)\n",
      "Pair{AbstractString,Int64}(\"the anterior and\", 1)\n",
      "Pair{AbstractString,Int64}(\"2008 )\", 2)\n",
      "Pair{AbstractString,Int64}(\"main chain around\", 1)\n",
      "Pair{AbstractString,Int64}(\"283 proteases ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"13 during infection.\", 1)\n",
      "Pair{AbstractString,Int64}(\"RD 318 cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"twelve-week-old\", 1)\n",
      "Pair{AbstractString,Int64}(\"response to the\", 3)\n",
      "Pair{AbstractString,Int64}(\"0.33\", 1)\n",
      "Pair{AbstractString,Int64}(\"spike gene during\", 1)\n",
      "Pair{AbstractString,Int64}(\"performed using \\\"\", 1)\n",
      "Pair{AbstractString,Int64}(\"assay on\", 1)\n",
      "Pair{AbstractString,Int64}(\"walls and floor\", 1)\n",
      "Pair{AbstractString,Int64}(\"assessment\", 3)\n",
      "Pair{AbstractString,Int64}(\"the slope.\", 1)\n",
      "Pair{AbstractString,Int64}(\"are presented as\", 2)\n",
      "Pair{AbstractString,Int64}(\"proteases of SARS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"rarely do ExoN-AA\", 1)\n",
      "Pair{AbstractString,Int64}(\"the excitability\", 1)\n",
      "Pair{AbstractString,Int64}(\"representations somewhat\", 1)\n",
      "Pair{AbstractString,Int64}(\"splenocyte cultures ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"32\", 13)\n",
      "Pair{AbstractString,Int64}(\"antiviral drugs that\", 1)\n",
      "Pair{AbstractString,Int64}(\"4 = >\", 1)\n",
      "Pair{AbstractString,Int64}(\"and thus ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"other proteases\", 1)\n",
      "Pair{AbstractString,Int64}(\"4,000 ,\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"RBV ) were\", 1)\n",
      "Pair{AbstractString,Int64}(\"complex mixed with\", 1)\n",
      "Pair{AbstractString,Int64}(\"241 pro-forms\", 1)\n",
      "Pair{AbstractString,Int64}(\"was 3.8 µM.\", 1)\n",
      "Pair{AbstractString,Int64}(\"= 1.2 µM\", 1)\n",
      "Pair{AbstractString,Int64}(\"against the M\", 1)\n",
      "Pair{AbstractString,Int64}(\"was performed on\", 1)\n",
      "Pair{AbstractString,Int64}(\"tetO and\", 1)\n",
      "Pair{AbstractString,Int64}(\"4 Diexpressing cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"chain is too\", 1)\n",
      "Pair{AbstractString,Int64}(\"; Zhu et\", 1)\n",
      "Pair{AbstractString,Int64}(\"3.36 %\", 1)\n",
      "Pair{AbstractString,Int64}(\"Phusion Hot\", 1)\n",
      "Pair{AbstractString,Int64}(\"values one order\", 1)\n",
      "Pair{AbstractString,Int64}(\"All primary\", 2)\n",
      "Pair{AbstractString,Int64}(\"the following test\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibitor rupintrivir\", 1)\n",
      "Pair{AbstractString,Int64}(\"and Huh\", 2)\n",
      "Pair{AbstractString,Int64}(\"with 231 increased\", 1)\n",
      "Pair{AbstractString,Int64}(\"PBS-injected mice.\", 1)\n",
      "Pair{AbstractString,Int64}(\"fully enter\", 2)\n",
      "Pair{AbstractString,Int64}(\"- 0.2\", 1)\n",
      "Pair{AbstractString,Int64}(\"some other\", 1)\n",
      "Pair{AbstractString,Int64}(\"this genotype.\", 1)\n",
      "Pair{AbstractString,Int64}(\"DNA (\", 2)\n",
      "Pair{AbstractString,Int64}(\"0.1407 ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"neuronal excitability ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"and nucleocapsid (\", 2)\n",
      "Pair{AbstractString,Int64}(\"the final step\", 1)\n",
      "Pair{AbstractString,Int64}(\"15 , 64\", 1)\n",
      "Pair{AbstractString,Int64}(\"e is important\", 1)\n",
      "Pair{AbstractString,Int64}(\"and for Coxsackievirusinduced\", 1)\n",
      "Pair{AbstractString,Int64}(\"more than 20\", 2)\n",
      "Pair{AbstractString,Int64}(\"M\", 140)\n",
      "Pair{AbstractString,Int64}(\"they allow easy\", 1)\n",
      "Pair{AbstractString,Int64}(\"D 272 E\", 2)\n",
      "Pair{AbstractString,Int64}(\"in TRIzol Reagent\", 1)\n",
      "Pair{AbstractString,Int64}(\"of the LY\", 1)\n",
      "Pair{AbstractString,Int64}(\"employing\", 2)\n",
      "Pair{AbstractString,Int64}(\"isolated from\", 2)\n",
      "Pair{AbstractString,Int64}(\"whole-cell recordings\", 1)\n",
      "Pair{AbstractString,Int64}(\"C pro pocket\", 1)\n",
      "Pair{AbstractString,Int64}(\", prior to\", 1)\n",
      "Pair{AbstractString,Int64}(\"As a\", 6)\n",
      "Pair{AbstractString,Int64}(\"7 )\", 4)\n",
      "Pair{AbstractString,Int64}(\"the first ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"it. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"BstEII\", 1)\n",
      "Pair{AbstractString,Int64}(\"structures of other\", 1)\n",
      "Pair{AbstractString,Int64}(\"283\", 1)\n",
      "Pair{AbstractString,Int64}(\"resistance were filled\", 1)\n",
      "Pair{AbstractString,Int64}(\"by LY 6\", 1)\n",
      "Pair{AbstractString,Int64}(\"Index TM ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"have a relatively\", 1)\n",
      "Pair{AbstractString,Int64}(\"human rhabdomyosarcoma cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"161 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"7 cells ,\", 5)\n",
      "Pair{AbstractString,Int64}(\"oxyanion hole as\", 1)\n",
      "Pair{AbstractString,Int64}(\", respectively.\", 8)\n",
      "Pair{AbstractString,Int64}(\"Leiden. It is\", 1)\n",
      "Pair{AbstractString,Int64}(\"the odors alone\", 1)\n",
      "Pair{AbstractString,Int64}(\"drugs and\", 1)\n",
      "Pair{AbstractString,Int64}(\"modeled zinc finger\", 1)\n",
      "Pair{AbstractString,Int64}(\"damaged the\", 1)\n",
      "Pair{AbstractString,Int64}(\"addition to the\", 2)\n",
      "Pair{AbstractString,Int64}(\"authors mainly\", 1)\n",
      "Pair{AbstractString,Int64}(\"the Cb of\", 1)\n",
      "Pair{AbstractString,Int64}(\"-well plates and\", 1)\n",
      "Pair{AbstractString,Int64}(\"Nonetheless\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Chen et\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 , Vazyme\", 1)\n",
      "Pair{AbstractString,Int64}(\"the type I\", 1)\n",
      "Pair{AbstractString,Int64}(\"nervous system (\", 1)\n",
      "Pair{AbstractString,Int64}(\"Gomez et\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 infection\", 6)\n",
      "Pair{AbstractString,Int64}(\"120 µM.\", 1)\n",
      "Pair{AbstractString,Int64}(\"not suppress\", 1)\n",
      "Pair{AbstractString,Int64}(\"employed porcine\", 1)\n",
      "Pair{AbstractString,Int64}(\"circuit components remain\", 1)\n",
      "Pair{AbstractString,Int64}(\": YP\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.5 mL )\", 1)\n",
      "Pair{AbstractString,Int64}(\"blot for\", 1)\n",
      "Pair{AbstractString,Int64}(\") . A.\", 1)\n",
      "Pair{AbstractString,Int64}(\"117 navigate\", 1)\n",
      "Pair{AbstractString,Int64}(\"important but not\", 1)\n",
      "Pair{AbstractString,Int64}(\"predicted with\", 1)\n",
      "Pair{AbstractString,Int64}(\"\\\" ) in\", 1)\n",
      "Pair{AbstractString,Int64}(\"flash\", 1)\n",
      "Pair{AbstractString,Int64}(\"samples harvested\", 1)\n",
      "Pair{AbstractString,Int64}(\"the box at\", 2)\n",
      "Pair{AbstractString,Int64}(\"must be placed\", 1)\n",
      "Pair{AbstractString,Int64}(\"later by presenting\", 1)\n",
      "Pair{AbstractString,Int64}(\"of odor-evoked\", 1)\n",
      "Pair{AbstractString,Int64}(\"is computed from\", 1)\n",
      "Pair{AbstractString,Int64}(\"reported a\", 1)\n",
      "Pair{AbstractString,Int64}(\"spike-P 250 relative\", 1)\n",
      "Pair{AbstractString,Int64}(\"other odors\", 1)\n",
      "Pair{AbstractString,Int64}(\"nucleocapsid\", 2)\n",
      "Pair{AbstractString,Int64}(\"mRNAs , which\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 mL\", 2)\n",
      "Pair{AbstractString,Int64}(\"solution ,\", 4)\n",
      "Pair{AbstractString,Int64}(\"one-way ANOVA. For\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.007 , Wilcoxon\", 1)\n",
      "Pair{AbstractString,Int64}(\"was observed independently\", 1)\n",
      "Pair{AbstractString,Int64}(\") and TEA\", 1)\n",
      "Pair{AbstractString,Int64}(\"Tables\", 2)\n",
      "Pair{AbstractString,Int64}(\"Twenty-one closely related\", 1)\n",
      "Pair{AbstractString,Int64}(\"mutations may\", 1)\n",
      "Pair{AbstractString,Int64}(\"subsite of the\", 2)\n",
      "Pair{AbstractString,Int64}(\"inhibitors , where\", 1)\n",
      "Pair{AbstractString,Int64}(\"with a transduction\", 1)\n",
      "Pair{AbstractString,Int64}(\"with standard settings\", 1)\n",
      "Pair{AbstractString,Int64}(\"abolished virus replication\", 1)\n",
      "Pair{AbstractString,Int64}(\"E inhibits\", 2)\n",
      "Pair{AbstractString,Int64}(\"deletions ( indels\", 1)\n",
      "Pair{AbstractString,Int64}(\"complexes of three\", 1)\n",
      "Pair{AbstractString,Int64}(\"Musculus EBI Gene\", 1)\n",
      "Pair{AbstractString,Int64}(\"I.\", 1)\n",
      "Pair{AbstractString,Int64}(\"experience\", 1)\n",
      "Pair{AbstractString,Int64}(\"triggered by 60\", 1)\n",
      "Pair{AbstractString,Int64}(\"the 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"RC 311 -C\", 1)\n",
      "Pair{AbstractString,Int64}(\"NSP 15 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"Thermo Fisher Scientific\", 6)\n",
      "Pair{AbstractString,Int64}(\": cFos-tTA transgenic\", 1)\n",
      "Pair{AbstractString,Int64}(\"In parallel\", 2)\n",
      "Pair{AbstractString,Int64}(\"37 resulting in\", 1)\n",
      "Pair{AbstractString,Int64}(\"is prone\", 1)\n",
      "Pair{AbstractString,Int64}(\"velocity (\", 2)\n",
      "Pair{AbstractString,Int64}(\"biological data (\", 1)\n",
      "Pair{AbstractString,Int64}(\"calculated as PFU\", 2)\n",
      "Pair{AbstractString,Int64}(\"together resemble\", 1)\n",
      "Pair{AbstractString,Int64}(\"dichloromethane\", 1)\n",
      "Pair{AbstractString,Int64}(\"for analytical\", 1)\n",
      "Pair{AbstractString,Int64}(\"the extremity\", 1)\n",
      "Pair{AbstractString,Int64}(\"To determine whether\", 5)\n",
      "Pair{AbstractString,Int64}(\"Coronavirus\", 2)\n",
      "Pair{AbstractString,Int64}(\"Dynamics\", 1)\n",
      "Pair{AbstractString,Int64}(\"which Fos-tagged neurons\", 1)\n",
      "Pair{AbstractString,Int64}(\"throughout the\", 3)\n",
      "Pair{AbstractString,Int64}(\"mice from\", 1)\n",
      "Pair{AbstractString,Int64}(\"µM , WT-MHV\", 1)\n",
      "Pair{AbstractString,Int64}(\"for change in\", 1)\n",
      "Pair{AbstractString,Int64}(\"author/funder. . https://doi.org/\", 25)\n",
      "Pair{AbstractString,Int64}(\"127\", 2)\n",
      "Pair{AbstractString,Int64}(\"small.\", 1)\n",
      "Pair{AbstractString,Int64}(\"XLS\", 2)\n",
      "Pair{AbstractString,Int64}(\"( HAE\", 1)\n",
      "Pair{AbstractString,Int64}(\"response , inflammation\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.9 µM against\", 1)\n",
      "Pair{AbstractString,Int64}(\"onto\", 6)\n",
      "Pair{AbstractString,Int64}(\"j , l\", 1)\n",
      "Pair{AbstractString,Int64}(\"be one\", 1)\n",
      "Pair{AbstractString,Int64}(\"929 cells.\", 1)\n",
      "Pair{AbstractString,Int64}(\"13 14 15\", 4)\n",
      "Pair{AbstractString,Int64}(\"x 50\", 1)\n",
      "Pair{AbstractString,Int64}(\"excellent\", 7)\n",
      "Pair{AbstractString,Int64}(\"virus in HAE\", 1)\n",
      "Pair{AbstractString,Int64}(\"a D 272\", 2)\n",
      "Pair{AbstractString,Int64}(\"pocket has a\", 1)\n",
      "Pair{AbstractString,Int64}(\"r performed\", 1)\n",
      "Pair{AbstractString,Int64}(\"preprint maps\", 1)\n",
      "Pair{AbstractString,Int64}(\"MΩ , and\", 1)\n",
      "Pair{AbstractString,Int64}(\"contraindicated for the\", 1)\n",
      "Pair{AbstractString,Int64}(\"248 using 6\", 1)\n",
      "Pair{AbstractString,Int64}(\"the product 5\", 1)\n",
      "Pair{AbstractString,Int64}(\"future work.\", 1)\n",
      "Pair{AbstractString,Int64}(\"matching the\", 1)\n",
      "Pair{AbstractString,Int64}(\"Phenol Red-free\", 1)\n",
      "Pair{AbstractString,Int64}(\"amino acids. Among\", 1)\n",
      "Pair{AbstractString,Int64}(\", 62\", 1)\n",
      "Pair{AbstractString,Int64}(\"by KLK\", 2)\n",
      "Pair{AbstractString,Int64}(\"both the CS+\", 1)\n",
      "Pair{AbstractString,Int64}(\"infection treatment.\", 1)\n",
      "Pair{AbstractString,Int64}(\"innate immune system\", 1)\n",
      "Pair{AbstractString,Int64}(\"was completed ,\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"similarities between olfactory\", 1)\n",
      "Pair{AbstractString,Int64}(\"next step.\", 1)\n",
      "Pair{AbstractString,Int64}(\"the split-luciferase protein\", 1)\n",
      "Pair{AbstractString,Int64}(\"Corporation )\", 1)\n",
      "Pair{AbstractString,Int64}(\"503\", 18)\n",
      "Pair{AbstractString,Int64}(\"inhibitor dec-RVKR-CMK interferes\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 hour. Cells\", 1)\n",
      "Pair{AbstractString,Int64}(\". L 36\", 1)\n",
      "Pair{AbstractString,Int64}(\"% in maintenance\", 2)\n",
      "Pair{AbstractString,Int64}(\"the control\", 14)\n",
      "Pair{AbstractString,Int64}(\"lid \\\" by\", 1)\n",
      "Pair{AbstractString,Int64}(\"severe pneumonia\", 1)\n",
      "Pair{AbstractString,Int64}(\"the sequential presentation\", 1)\n",
      "Pair{AbstractString,Int64}(\"piriform network mechanisms\", 1)\n",
      "Pair{AbstractString,Int64}(\"on a diet\", 1)\n",
      "Pair{AbstractString,Int64}(\"propanoate ( 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"o C for\", 27)\n",
      "Pair{AbstractString,Int64}(\"mice ,\", 8)\n",
      "Pair{AbstractString,Int64}(\"macrophages\", 6)\n",
      "Pair{AbstractString,Int64}(\"replicase promotes\", 1)\n",
      "Pair{AbstractString,Int64}(\"corona-and enteroviruses ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.4688 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"255 author/funder.\", 1)\n",
      "Pair{AbstractString,Int64}(\"and infrastructure which\", 1)\n",
      "Pair{AbstractString,Int64}(\"that at least\", 1)\n",
      "Pair{AbstractString,Int64}(\"product was\", 1)\n",
      "Pair{AbstractString,Int64}(\"piriform neurons\", 25)\n",
      "Pair{AbstractString,Int64}(\"replicon-transfected Huh-T 7\", 1)\n",
      "Pair{AbstractString,Int64}(\"Gene Expression\", 1)\n",
      "Pair{AbstractString,Int64}(\"activity lasts\", 1)\n",
      "Pair{AbstractString,Int64}(\"F 6627 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"tightly. This\", 1)\n",
      "Pair{AbstractString,Int64}(\"MERS-CoV/EMC strain (\", 1)\n",
      "Pair{AbstractString,Int64}(\"different protein\", 1)\n",
      "Pair{AbstractString,Int64}(\"80\", 3)\n",
      "Pair{AbstractString,Int64}(\"a heterologous syncytia\", 1)\n",
      "Pair{AbstractString,Int64}(\"the fluorophor\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 2019\", 1)\n",
      "Pair{AbstractString,Int64}(\"can adapt for\", 1)\n",
      "Pair{AbstractString,Int64}(\"http://www.datamonkey.org/ )\", 1)\n",
      "Pair{AbstractString,Int64}(\"side of odor\", 2)\n",
      "Pair{AbstractString,Int64}(\"C. 11 a\", 1)\n",
      "Pair{AbstractString,Int64}(\", it has\", 2)\n",
      "Pair{AbstractString,Int64}(\"2 % ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells , to\", 1)\n",
      "Pair{AbstractString,Int64}(\"119 ExoN-AA reversion\", 1)\n",
      "Pair{AbstractString,Int64}(\"each protease.\", 1)\n",
      "Pair{AbstractString,Int64}(\"group. Mice\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 mL water.\", 1)\n",
      "Pair{AbstractString,Int64}(\"been described as\", 2)\n",
      "Pair{AbstractString,Int64}(\"C- 150 terminal\", 1)\n",
      "Pair{AbstractString,Int64}(\"odor ( odor\", 1)\n",
      "Pair{AbstractString,Int64}(\"045512\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 min and\", 1)\n",
      "Pair{AbstractString,Int64}(\"originally described\", 1)\n",
      "Pair{AbstractString,Int64}(\"and serine\", 1)\n",
      "Pair{AbstractString,Int64}(\"effect was reversed\", 1)\n",
      "Pair{AbstractString,Int64}(\"replicons was poor\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 61 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"extraction\", 2)\n",
      "Pair{AbstractString,Int64}(\"expression confirmed by\", 1)\n",
      "Pair{AbstractString,Int64}(\"the following criteria\", 1)\n",
      "Pair{AbstractString,Int64}(\"nsp 12 ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"large parts are\", 1)\n",
      "Pair{AbstractString,Int64}(\"37 38\", 4)\n",
      "Pair{AbstractString,Int64}(\"area smaller\", 1)\n",
      "Pair{AbstractString,Int64}(\"RNA as described\", 1)\n",
      "Pair{AbstractString,Int64}(\"indicated that\", 1)\n",
      "Pair{AbstractString,Int64}(\"potent against\", 1)\n",
      "Pair{AbstractString,Int64}(\"the priming\", 1)\n",
      "Pair{AbstractString,Int64}(\"\\\" PCR (\", 1)\n",
      "Pair{AbstractString,Int64}(\"day post seeding\", 1)\n",
      "Pair{AbstractString,Int64}(\"[ 7\", 1)\n",
      "Pair{AbstractString,Int64}(\", p= 0.7209\", 1)\n",
      "Pair{AbstractString,Int64}(\"magnitude lower\", 1)\n",
      "Pair{AbstractString,Int64}(\"hours at\", 7)\n",
      "Pair{AbstractString,Int64}(\"alpha-or betacoronaviruses as\", 1)\n",
      "Pair{AbstractString,Int64}(\"refer to 5\", 1)\n",
      "Pair{AbstractString,Int64}(\"termini\", 1)\n",
      "Pair{AbstractString,Int64}(\"nucleotides and amino\", 1)\n",
      "Pair{AbstractString,Int64}(\"by mometasone ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 1:500 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"immune system\", 2)\n",
      "Pair{AbstractString,Int64}(\"All rights reserved.\", 102)\n",
      "Pair{AbstractString,Int64}(\"to ( -\", 1)\n",
      "Pair{AbstractString,Int64}(\"experimental manipulation. Using\", 1)\n",
      "Pair{AbstractString,Int64}(\"effective concentration\", 1)\n",
      "Pair{AbstractString,Int64}(\"between 11 a\", 1)\n",
      "Pair{AbstractString,Int64}(\"conditioning ( Wilcoxon\", 1)\n",
      "Pair{AbstractString,Int64}(\"observed at 25\", 1)\n",
      "Pair{AbstractString,Int64}(\"request.\", 2)\n",
      "Pair{AbstractString,Int64}(\"was added under\", 1)\n",
      "Pair{AbstractString,Int64}(\", indicated\", 1)\n",
      "Pair{AbstractString,Int64}(\"uniquely also accepts\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 0.72\", 1)\n",
      "Pair{AbstractString,Int64}(\"for murine\", 1)\n",
      "Pair{AbstractString,Int64}(\"cell culture.\", 1)\n",
      "Pair{AbstractString,Int64}(\"points for the\", 1)\n",
      "Pair{AbstractString,Int64}(\"( g-lactam )\", 2)\n",
      "Pair{AbstractString,Int64}(\"s instruction.\", 1)\n",
      "Pair{AbstractString,Int64}(\"defects in ExoN\", 1)\n",
      "Pair{AbstractString,Int64}(\"although the\", 2)\n",
      "Pair{AbstractString,Int64}(\"stock was incubated\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 complex 218\", 1)\n",
      "Pair{AbstractString,Int64}(\"two structures involved\", 1)\n",
      "Pair{AbstractString,Int64}(\"Quasar 670\", 1)\n",
      "Pair{AbstractString,Int64}(\"its possible\", 1)\n",
      "Pair{AbstractString,Int64}(\"rate is\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.1708\", 1)\n",
      "Pair{AbstractString,Int64}(\", transduced\", 1)\n",
      "Pair{AbstractString,Int64}(\"Assays with\", 1)\n",
      "Pair{AbstractString,Int64}(\"compounds towards the\", 1)\n",
      "Pair{AbstractString,Int64}(\"Mice are i.p.\", 1)\n",
      "Pair{AbstractString,Int64}(\"against HRV 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"were mirrored by\", 1)\n",
      "Pair{AbstractString,Int64}(\"genotype ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"deactivated compared\", 1)\n",
      "Pair{AbstractString,Int64}(\"stirring for\", 1)\n",
      "Pair{AbstractString,Int64}(\"l ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\"cinnamoyl group\", 1)\n",
      "Pair{AbstractString,Int64}(\"strain EMC/ 2012\", 1)\n",
      "Pair{AbstractString,Int64}(\"Another constraint of\", 1)\n",
      "Pair{AbstractString,Int64}(\"7 8 9\", 4)\n",
      "Pair{AbstractString,Int64}(\"substrate as catalyzed\", 1)\n",
      "Pair{AbstractString,Int64}(\"bioRxiv preprint between\", 1)\n",
      "Pair{AbstractString,Int64}(\"extracellular viral RNA\", 1)\n",
      "Pair{AbstractString,Int64}(\"points , female\", 1)\n",
      "Pair{AbstractString,Int64}(\"by logarithmic interpolation\", 1)\n",
      "Pair{AbstractString,Int64}(\"Spike\", 1)\n",
      "Pair{AbstractString,Int64}(\"presentation. ( E\", 1)\n",
      "Pair{AbstractString,Int64}(\"silencing experiments. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"sensory\", 3)\n",
      "Pair{AbstractString,Int64}(\"when\", 14)\n",
      "Pair{AbstractString,Int64}(\"Dendrograms are normalized\", 1)\n",
      "Pair{AbstractString,Int64}(\"limitations. To\", 1)\n",
      "Pair{AbstractString,Int64}(\"cause human\", 1)\n",
      "Pair{AbstractString,Int64}(\"odor was delivered\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 151\", 1)\n",
      "Pair{AbstractString,Int64}(\"then subjected\", 1)\n",
      "Pair{AbstractString,Int64}(\"Among them ,\", 3)\n",
      "Pair{AbstractString,Int64}(\"separate Fos-tagging\", 1)\n",
      "Pair{AbstractString,Int64}(\"specificity of\", 5)\n",
      "Pair{AbstractString,Int64}(\": 1.5\", 1)\n",
      "Pair{AbstractString,Int64}(\"both sexes rapidly\", 1)\n",
      "Pair{AbstractString,Int64}(\"were evaluated against\", 1)\n",
      "Pair{AbstractString,Int64}(\"least five\", 1)\n",
      "Pair{AbstractString,Int64}(\"ExoN-AA background\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 ) (\", 1)\n",
      "Pair{AbstractString,Int64}(\"truly three-dimensional\", 1)\n",
      "Pair{AbstractString,Int64}(\"from subsequent\", 1)\n",
      "Pair{AbstractString,Int64}(\"an amino acid\", 1)\n",
      "Pair{AbstractString,Int64}(\"ExoN-ED.\", 2)\n",
      "Pair{AbstractString,Int64}(\"alcohol products 7\", 1)\n",
      "Pair{AbstractString,Int64}(\"memory recall. Even\", 1)\n",
      "Pair{AbstractString,Int64}(\"( D.L.\", 1)\n",
      "Pair{AbstractString,Int64}(\"systematic studies\", 1)\n",
      "Pair{AbstractString,Int64}(\"contents\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 mL\", 2)\n",
      "Pair{AbstractString,Int64}(\"components remain to\", 1)\n",
      "Pair{AbstractString,Int64}(\", respectively. The\", 3)\n",
      "Pair{AbstractString,Int64}(\"until the consumption\", 1)\n",
      "Pair{AbstractString,Int64}(\") , quartet\", 1)\n",
      "Pair{AbstractString,Int64}(\"this high\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 1.8\", 1)\n",
      "Pair{AbstractString,Int64}(\"were 31.43 ±\", 1)\n",
      "Pair{AbstractString,Int64}(\"accept a 2.54\", 1)\n",
      "Pair{AbstractString,Int64}(\"c , Pathway\", 1)\n",
      "Pair{AbstractString,Int64}(\"weak or very\", 1)\n",
      "Pair{AbstractString,Int64}(\"( (\", 2)\n",
      "Pair{AbstractString,Int64}(\"Acute\", 1)\n",
      "Pair{AbstractString,Int64}(\"a hallmark of\", 1)\n",
      "Pair{AbstractString,Int64}(\"the carboxylate of\", 1)\n",
      "Pair{AbstractString,Int64}(\"249 KLK 13\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"preprint GAT GAA\", 1)\n",
      "Pair{AbstractString,Int64}(\"somewhat attenuated compared\", 1)\n",
      "Pair{AbstractString,Int64}(\"intervention against alphacoronaviruses\", 1)\n",
      "Pair{AbstractString,Int64}(\"side ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"sonicated\", 1)\n",
      "Pair{AbstractString,Int64}(\"mechanism of\", 3)\n",
      "Pair{AbstractString,Int64}(\"/nar/gkn 180\", 1)\n",
      "Pair{AbstractString,Int64}(\"inactivating\", 1)\n",
      "Pair{AbstractString,Int64}(\"the CoVs lacking\", 1)\n",
      "Pair{AbstractString,Int64}(\"picornavirus family.\", 1)\n",
      "Pair{AbstractString,Int64}(\"20\", 61)\n",
      "Pair{AbstractString,Int64}(\"regions ( NTR\", 1)\n",
      "Pair{AbstractString,Int64}(\"The recording\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.0002\", 1)\n",
      "Pair{AbstractString,Int64}(\"exhibited low cytotoxicity\", 1)\n",
      "Pair{AbstractString,Int64}(\"tuberculosis\", 1)\n",
      "Pair{AbstractString,Int64}(\"reversion , we\", 1)\n",
      "Pair{AbstractString,Int64}(\"beta-coronavirus lineage\", 1)\n",
      "Pair{AbstractString,Int64}(\"15 ( Table\", 1)\n",
      "Pair{AbstractString,Int64}(\"vs 31.95\", 1)\n",
      "Pair{AbstractString,Int64}(\"the odor-foot shock\", 2)\n",
      "Pair{AbstractString,Int64}(\"necessary\", 6)\n",
      "Pair{AbstractString,Int64}(\"% O\", 1)\n",
      "Pair{AbstractString,Int64}(\"299 important to\", 1)\n",
      "Pair{AbstractString,Int64}(\"above or\", 1)\n",
      "Pair{AbstractString,Int64}(\"conditioning is performed\", 1)\n",
      "Pair{AbstractString,Int64}(\"Elektronik ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\"chain and\", 1)\n",
      "Pair{AbstractString,Int64}(\"the target\", 3)\n",
      "Pair{AbstractString,Int64}(\"recovered variants\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 ×\", 1)\n",
      "Pair{AbstractString,Int64}(\"bottle of\", 1)\n",
      "Pair{AbstractString,Int64}(\"was blinded\", 1)\n",
      "Pair{AbstractString,Int64}(\"silences hM 4\", 1)\n",
      "Pair{AbstractString,Int64}(\"treatment with 500\", 1)\n",
      "Pair{AbstractString,Int64}(\"DSP 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"makes hydrophobic interactions\", 1)\n",
      "Pair{AbstractString,Int64}(\"suggests\", 2)\n",
      "Pair{AbstractString,Int64}(\"9.0 Hz\", 1)\n",
      "Pair{AbstractString,Int64}(\"detection in some\", 1)\n",
      "Pair{AbstractString,Int64}(\"stack is\", 1)\n",
      "Pair{AbstractString,Int64}(\"The benzyl\", 1)\n",
      "Pair{AbstractString,Int64}(\"and the assessment\", 1)\n",
      "Pair{AbstractString,Int64}(\"CNO injection ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"rate ( dS\", 1)\n",
      "Pair{AbstractString,Int64}(\"we devised\", 1)\n",
      "Pair{AbstractString,Int64}(\"suggesting a\", 1)\n",
      "Pair{AbstractString,Int64}(\"directly test these\", 1)\n",
      "Pair{AbstractString,Int64}(\"one click \\\"\", 1)\n",
      "Pair{AbstractString,Int64}(\"increased mutation frequencies\", 1)\n",
      "Pair{AbstractString,Int64}(\"its bisulfite adduct\", 1)\n",
      "Pair{AbstractString,Int64}(\". These inhibitors\", 1)\n",
      "Pair{AbstractString,Int64}(\"viral replication with\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Chen\", 1)\n",
      "Pair{AbstractString,Int64}(\"Variant C ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"224\", 3)\n",
      "Pair{AbstractString,Int64}(\"was also significantly\", 2)\n",
      "Pair{AbstractString,Int64}(\"preprint this rapid\", 1)\n",
      "Pair{AbstractString,Int64}(\"nucleic acid\", 12)\n",
      "Pair{AbstractString,Int64}(\"per cubic millimeter\", 1)\n",
      "Pair{AbstractString,Int64}(\"b\", 12)\n",
      "Pair{AbstractString,Int64}(\"Depletion of immune\", 1)\n",
      "Pair{AbstractString,Int64}(\"h post-infection\", 1)\n",
      "Pair{AbstractString,Int64}(\"ring system\", 1)\n",
      "Pair{AbstractString,Int64}(\". The small\", 1)\n",
      "Pair{AbstractString,Int64}(\"detected via RT-qPCR\", 1)\n",
      "Pair{AbstractString,Int64}(\"ExoN-AA did not\", 1)\n",
      "Pair{AbstractString,Int64}(\"compound 4\", 1)\n",
      "Pair{AbstractString,Int64}(\"M.\", 1)\n",
      "Pair{AbstractString,Int64}(\"and 188\", 4)\n",
      "Pair{AbstractString,Int64}(\"protease inhibitors. To\", 1)\n",
      "Pair{AbstractString,Int64}(\"50 ×\", 1)\n",
      "Pair{AbstractString,Int64}(\"indicated titers or\", 1)\n",
      "Pair{AbstractString,Int64}(\"completely fill\", 1)\n",
      "Pair{AbstractString,Int64}(\"Usage\", 1)\n",
      "Pair{AbstractString,Int64}(\"NMR (\", 2)\n",
      "Pair{AbstractString,Int64}(\"lifetime of\", 1)\n",
      "Pair{AbstractString,Int64}(\"before ,\", 4)\n",
      "Pair{AbstractString,Int64}(\"( Avicel\", 1)\n",
      "Pair{AbstractString,Int64}(\"the significant\", 1)\n",
      "Pair{AbstractString,Int64}(\"in cases of\", 1)\n",
      "Pair{AbstractString,Int64}(\"sequencing. However\", 1)\n",
      "Pair{AbstractString,Int64}(\"the choice between\", 1)\n",
      "Pair{AbstractString,Int64}(\"p= 0.0428 ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"comparisons of\", 1)\n",
      "Pair{AbstractString,Int64}(\"amplified with primers\", 1)\n",
      "Pair{AbstractString,Int64}(\"269\", 1)\n",
      "Pair{AbstractString,Int64}(\"His 40\", 3)\n",
      "Pair{AbstractString,Int64}(\"for 11\", 3)\n",
      "Pair{AbstractString,Int64}(\", SP\", 1)\n",
      "Pair{AbstractString,Int64}(\"genome was \\\"\", 1)\n",
      "Pair{AbstractString,Int64}(\"SARS-CoV (\", 6)\n",
      "Pair{AbstractString,Int64}(\"% ammonia in\", 1)\n",
      "Pair{AbstractString,Int64}(\"drive primary reversion\", 1)\n",
      "Pair{AbstractString,Int64}(\"poured into brine\", 1)\n",
      "Pair{AbstractString,Int64}(\"Nucleoside and base\", 1)\n",
      "Pair{AbstractString,Int64}(\"surprised\", 1)\n",
      "Pair{AbstractString,Int64}(\"Multiple sequence\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 , Tecan\", 1)\n",
      "Pair{AbstractString,Int64}(\"Yamada\", 1)\n",
      "Pair{AbstractString,Int64}(\"mice in which\", 3)\n",
      "Pair{AbstractString,Int64}(\"and D )\", 1)\n",
      "Pair{AbstractString,Int64}(\"human MERS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"and posterior piriform\", 1)\n",
      "Pair{AbstractString,Int64}(\"Experiments were carried\", 1)\n",
      "Pair{AbstractString,Int64}(\"2017 ;\", 6)\n",
      "Pair{AbstractString,Int64}(\"sgRNA were introduced\", 1)\n",
      "Pair{AbstractString,Int64}(\"advantage over\", 2)\n",
      "Pair{AbstractString,Int64}(\"feature the\", 1)\n",
      "Pair{AbstractString,Int64}(\"experiences , and\", 1)\n",
      "Pair{AbstractString,Int64}(\"and competitive fitness\", 1)\n",
      "Pair{AbstractString,Int64}(\"\\\" ( Met\", 1)\n",
      "Pair{AbstractString,Int64}(\", expanding\", 1)\n",
      "Pair{AbstractString,Int64}(\") protein\", 2)\n",
      "Pair{AbstractString,Int64}(\"Whereas the inhibitor\", 1)\n",
      "Pair{AbstractString,Int64}(\"F 6627 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"controls and\", 1)\n",
      "Pair{AbstractString,Int64}(\"subsites beyond S\", 1)\n",
      "Pair{AbstractString,Int64}(\"protein N. Subgenomic\", 1)\n",
      "Pair{AbstractString,Int64}(\"less than 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"For each condition\", 1)\n",
      "Pair{AbstractString,Int64}(\"strategy for liver\", 1)\n",
      "Pair{AbstractString,Int64}(\"2020 , patients\", 1)\n",
      "Pair{AbstractString,Int64}(\"compound in mice\", 1)\n",
      "Pair{AbstractString,Int64}(\"valine residue.\", 1)\n",
      "Pair{AbstractString,Int64}(\"decode\", 1)\n",
      "Pair{AbstractString,Int64}(\"One of the\", 2)\n",
      "Pair{AbstractString,Int64}(\"the vapor diffusion\", 2)\n",
      "Pair{AbstractString,Int64}(\"harboring the\", 3)\n",
      "Pair{AbstractString,Int64}(\"according to a\", 1)\n",
      "Pair{AbstractString,Int64}(\"cage before\", 1)\n",
      "Pair{AbstractString,Int64}(\"( from\", 3)\n",
      "Pair{AbstractString,Int64}(\"for another\", 1)\n",
      "Pair{AbstractString,Int64}(\"cRPMI 2 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"been difficult to\", 1)\n",
      "Pair{AbstractString,Int64}(\"RaTG 13 than\", 2)\n",
      "Pair{AbstractString,Int64}(\"GISAID )\", 1)\n",
      "Pair{AbstractString,Int64}(\"beta-citronellol\", 3)\n",
      "Pair{AbstractString,Int64}(\", p= 0.8438\", 1)\n",
      "Pair{AbstractString,Int64}(\"for significance using\", 1)\n",
      "Pair{AbstractString,Int64}(\"B. Another\", 1)\n",
      "Pair{AbstractString,Int64}(\"bond following a\", 1)\n",
      "Pair{AbstractString,Int64}(\"with respect to\", 2)\n",
      "Pair{AbstractString,Int64}(\") CoVs have\", 1)\n",
      "Pair{AbstractString,Int64}(\"Sciences\", 1)\n",
      "Pair{AbstractString,Int64}(\"research\", 2)\n",
      "Pair{AbstractString,Int64}(\"0.0002 ; c\", 1)\n",
      "Pair{AbstractString,Int64}(\"separately against the\", 1)\n",
      "Pair{AbstractString,Int64}(\"virus has\", 1)\n",
      "Pair{AbstractString,Int64}(\"hours as\", 1)\n",
      "Pair{AbstractString,Int64}(\"( MD\", 1)\n",
      "Pair{AbstractString,Int64}(\"( CS-\", 7)\n",
      "Pair{AbstractString,Int64}(\"recorded by LC\", 1)\n",
      "Pair{AbstractString,Int64}(\"by serine protease\", 1)\n",
      "Pair{AbstractString,Int64}(\"recorded and\", 1)\n",
      "Pair{AbstractString,Int64}(\"( c-e\", 1)\n",
      "Pair{AbstractString,Int64}(\", 0.3 Na-GTP\", 1)\n",
      "Pair{AbstractString,Int64}(\"scattered distribution of\", 1)\n",
      "Pair{AbstractString,Int64}(\"contoured\", 1)\n",
      "Pair{AbstractString,Int64}(\"side ) ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"HRV 2 and\", 1)\n",
      "Pair{AbstractString,Int64}(\"13 is essential\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 e-i\", 1)\n",
      "Pair{AbstractString,Int64}(\"was observed (\", 1)\n",
      "Pair{AbstractString,Int64}(\"pro ( 45\", 1)\n",
      "Pair{AbstractString,Int64}(\"the cDNA\", 1)\n",
      "Pair{AbstractString,Int64}(\"acid dimethyl\", 2)\n",
      "Pair{AbstractString,Int64}(\"mOsm osmolarity\", 1)\n",
      "Pair{AbstractString,Int64}(\", IRF 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"including proper controls\", 1)\n",
      "Pair{AbstractString,Int64}(\") was extracted\", 1)\n",
      "Pair{AbstractString,Int64}(\"To evaluate the\", 1)\n",
      "Pair{AbstractString,Int64}(\"times on each\", 4)\n",
      "Pair{AbstractString,Int64}(\"of this\", 6)\n",
      "Pair{AbstractString,Int64}(\"and has less\", 1)\n",
      "Pair{AbstractString,Int64}(\"0 = none\", 2)\n",
      "Pair{AbstractString,Int64}(\"preprint containing\", 1)\n",
      "Pair{AbstractString,Int64}(\"7.3 , was\", 1)\n",
      "Pair{AbstractString,Int64}(\"1000 replicates\", 1)\n",
      "Pair{AbstractString,Int64}(\"177 gene (\", 1)\n",
      "Pair{AbstractString,Int64}(\"R.\", 1)\n",
      "Pair{AbstractString,Int64}(\"( ppm ,\", 1)\n",
      "Pair{AbstractString,Int64}(\") G (\", 1)\n",
      "Pair{AbstractString,Int64}(\"previously. 61 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"periodinane generated\", 1)\n",
      "Pair{AbstractString,Int64}(\"< 0.05 was\", 1)\n",
      "Pair{AbstractString,Int64}(\"Enrichment\", 1)\n",
      "Pair{AbstractString,Int64}(\", so\", 6)\n",
      "Pair{AbstractString,Int64}(\"probability of ExoN-AE\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"( 0.72 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"of virus\", 6)\n",
      "Pair{AbstractString,Int64}(\"though cFos-tTA transgenic\", 1)\n",
      "Pair{AbstractString,Int64}(\"or 2 ×\", 1)\n",
      "Pair{AbstractString,Int64}(\"U= 13\", 1)\n",
      "Pair{AbstractString,Int64}(\"database ( Supplementary\", 1)\n",
      "Pair{AbstractString,Int64}(\"mimic of\", 1)\n",
      "Pair{AbstractString,Int64}(\"early entry \\\"\", 1)\n",
      "Pair{AbstractString,Int64}(\"bioRxiv preprint. HCoV-HKU\", 1)\n",
      "Pair{AbstractString,Int64}(\"increases in the\", 1)\n",
      "Pair{AbstractString,Int64}(\"S 2 pocket.\", 2)\n",
      "Pair{AbstractString,Int64}(\"transcripts and proteins\", 1)\n",
      "Pair{AbstractString,Int64}(\"surprising observation\", 1)\n",
      "Pair{AbstractString,Int64}(\"to design compounds\", 1)\n",
      "Pair{AbstractString,Int64}(\"their cytotoxicity\", 1)\n",
      "Pair{AbstractString,Int64}(\"important for controlling\", 2)\n",
      "Pair{AbstractString,Int64}(\"CS+ ) and\", 2)\n",
      "Pair{AbstractString,Int64}(\"on specific\", 1)\n",
      "Pair{AbstractString,Int64}(\"Company ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"minutes had\", 2)\n",
      "Pair{AbstractString,Int64}(\"cells. After\", 1)\n",
      "Pair{AbstractString,Int64}(\"each ,\", 1)\n",
      "Pair{AbstractString,Int64}(\". We first\", 1)\n",
      "Pair{AbstractString,Int64}(\"( CVA 16\", 1)\n",
      "Pair{AbstractString,Int64}(\"for aketoamides ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"is related to\", 2)\n",
      "Pair{AbstractString,Int64}(\"humans\", 2)\n",
      "Pair{AbstractString,Int64}(\"precise mapping of\", 1)\n",
      "Pair{AbstractString,Int64}(\"E 64\", 2)\n",
      "Pair{AbstractString,Int64}(\"the 3 -digit\", 1)\n",
      "Pair{AbstractString,Int64}(\"additionally\", 1)\n",
      "Pair{AbstractString,Int64}(\"1.69\", 2)\n",
      "Pair{AbstractString,Int64}(\"day of\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 + -sensitive\", 1)\n",
      "Pair{AbstractString,Int64}(\"CNO ( Gomez\", 1)\n",
      "Pair{AbstractString,Int64}(\"as a control\", 1)\n",
      "Pair{AbstractString,Int64}(\"from the treatment\", 1)\n",
      "Pair{AbstractString,Int64}(\"180 ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"58 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"A therapeutic approach\", 1)\n",
      "Pair{AbstractString,Int64}(\"retrieval of\", 2)\n",
      "Pair{AbstractString,Int64}(\"piriform odor responses\", 1)\n",
      "Pair{AbstractString,Int64}(\") presentation. Data\", 1)\n",
      "Pair{AbstractString,Int64}(\"picomolar ! )\", 1)\n",
      "Pair{AbstractString,Int64}(\"and identification of\", 1)\n",
      "Pair{AbstractString,Int64}(\"In other\", 1)\n",
      "Pair{AbstractString,Int64}(\"pseudoviruses decorated\", 1)\n",
      "Pair{AbstractString,Int64}(\"( M pro\", 2)\n",
      "Pair{AbstractString,Int64}(\"in the replicons\", 1)\n",
      "Pair{AbstractString,Int64}(\") amide\", 1)\n",
      "Pair{AbstractString,Int64}(\"and neutral\", 1)\n",
      "Pair{AbstractString,Int64}(\"490 ) was\", 1)\n",
      "Pair{AbstractString,Int64}(\"and λ phage\", 1)\n",
      "Pair{AbstractString,Int64}(\"expect 11 r\", 1)\n",
      "Pair{AbstractString,Int64}(\"complex between 11\", 1)\n",
      "Pair{AbstractString,Int64}(\"read per sample\", 1)\n",
      "Pair{AbstractString,Int64}(\"conditions : Statistical\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells compared to\", 1)\n",
      "Pair{AbstractString,Int64}(\"performed a quantitative\", 1)\n",
      "Pair{AbstractString,Int64}(\"medium. Ciclesonide and\", 1)\n",
      "Pair{AbstractString,Int64}(\"; Garner et\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.01 PFU/cell\", 1)\n",
      "Pair{AbstractString,Int64}(\"IACUC guidelines.\", 1)\n",
      "Pair{AbstractString,Int64}(\"ANOVA , Figure\", 1)\n",
      "Pair{AbstractString,Int64}(\"may impair T\", 1)\n",
      "Pair{AbstractString,Int64}(\"pH 6.0 ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"red )\", 1)\n",
      "Pair{AbstractString,Int64}(\"( ExoN\", 2)\n",
      "Pair{AbstractString,Int64}(\"conditions. 13\", 1)\n",
      "Pair{AbstractString,Int64}(\"we showed that\", 1)\n",
      "Pair{AbstractString,Int64}(\"of His\", 1)\n",
      "Pair{AbstractString,Int64}(\"1.7 µM\", 2)\n",
      "Pair{AbstractString,Int64}(\", in particular\", 3)\n",
      "Pair{AbstractString,Int64}(\"covered ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"high dose\", 3)\n",
      "Pair{AbstractString,Int64}(\"Metabion ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\"triazolo [\", 1)\n",
      "Pair{AbstractString,Int64}(\"were mirrored\", 2)\n",
      "Pair{AbstractString,Int64}(\"were quantified\", 2)\n",
      "Pair{AbstractString,Int64}(\"7 requires mutations\", 1)\n",
      "Pair{AbstractString,Int64}(\"to HE\", 1)\n",
      "Pair{AbstractString,Int64}(\"stock solution.\", 1)\n",
      "Pair{AbstractString,Int64}(\"control cultures ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"polybrene\", 1)\n",
      "Pair{AbstractString,Int64}(\"passage or\", 1)\n",
      "Pair{AbstractString,Int64}(\"-non-translated regions (\", 1)\n",
      "Pair{AbstractString,Int64}(\"CoV ExoN\", 2)\n",
      "Pair{AbstractString,Int64}(\"3\", 312)\n",
      "Pair{AbstractString,Int64}(\"a primase/processivity\", 1)\n",
      "Pair{AbstractString,Int64}(\"to improve the\", 1)\n",
      "Pair{AbstractString,Int64}(\"500 µL samples\", 1)\n",
      "Pair{AbstractString,Int64}(\"as 9.07 .\", 1)\n",
      "Pair{AbstractString,Int64}(\"described in detail\", 1)\n",
      "Pair{AbstractString,Int64}(\"against\", 113)\n",
      "Pair{AbstractString,Int64}(\"normalize\", 1)\n",
      "Pair{AbstractString,Int64}(\"enterovirus 3\", 12)\n",
      "Pair{AbstractString,Int64}(\"the protease (\", 2)\n",
      "Pair{AbstractString,Int64}(\"we synthesized\", 3)\n",
      "Pair{AbstractString,Int64}(\"in complementation\", 1)\n",
      "Pair{AbstractString,Int64}(\"a classical CysHisGlu/Asp\", 1)\n",
      "Pair{AbstractString,Int64}(\". Thus\", 4)\n",
      "Pair{AbstractString,Int64}(\"morphology. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"nucleic acids from\", 1)\n",
      "Pair{AbstractString,Int64}(\"studies have suggested\", 1)\n",
      "Pair{AbstractString,Int64}(\"method\", 7)\n",
      "Pair{AbstractString,Int64}(\"Nagel ) ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"0.81 ) in\", 1)\n",
      "Pair{AbstractString,Int64}(\"and 96\", 1)\n",
      "Pair{AbstractString,Int64}(\"Society of Laboratory\", 1)\n",
      "Pair{AbstractString,Int64}(\"The therapeutic\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.09 ( 0.81\", 1)\n",
      "Pair{AbstractString,Int64}(\"163 HAE\", 1)\n",
      "Pair{AbstractString,Int64}(\"that its\", 1)\n",
      "Pair{AbstractString,Int64}(\"lysates\", 6)\n",
      "Pair{AbstractString,Int64}(\"µL reservoir\", 1)\n",
      "Pair{AbstractString,Int64}(\"µL mother\", 1)\n",
      "Pair{AbstractString,Int64}(\"Viral titers in\", 1)\n",
      "Pair{AbstractString,Int64}(\"with no brake.\", 1)\n",
      "Pair{AbstractString,Int64}(\"in PCR\", 1)\n",
      "Pair{AbstractString,Int64}(\"only moderately attenuated\", 1)\n",
      "Pair{AbstractString,Int64}(\") luciferase gene\", 1)\n",
      "Pair{AbstractString,Int64}(\"a-ketoamides General\", 1)\n",
      "Pair{AbstractString,Int64}(\"1.4.4 ( http://codonw.sourceforge.net/\", 1)\n",
      "Pair{AbstractString,Int64}(\"310 helix 46\", 1)\n",
      "Pair{AbstractString,Int64}(\"to contract and\", 1)\n",
      "Pair{AbstractString,Int64}(\"constructs\", 1)\n",
      "Pair{AbstractString,Int64}(\"license author/funder.\", 5)\n",
      "Pair{AbstractString,Int64}(\"allow easy\", 1)\n",
      "Pair{AbstractString,Int64}(\"replicon-transfected\", 1)\n",
      "Pair{AbstractString,Int64}(\"investigate whether the\", 2)\n",
      "Pair{AbstractString,Int64}(\"were subject to\", 1)\n",
      "Pair{AbstractString,Int64}(\"All three modeled\", 1)\n",
      "Pair{AbstractString,Int64}(\"428\", 1)\n",
      "Pair{AbstractString,Int64}(\"residue in the\", 1)\n",
      "Pair{AbstractString,Int64}(\"subsite is\", 1)\n",
      "Pair{AbstractString,Int64}(\"possibly due\", 2)\n",
      "Pair{AbstractString,Int64}(\"fact that there\", 1)\n",
      "Pair{AbstractString,Int64}(\"targeted\", 2)\n",
      "Pair{AbstractString,Int64}(\"t\", 3)\n",
      "Pair{AbstractString,Int64}(\"CCL 5 -SYCA\", 1)\n",
      "Pair{AbstractString,Int64}(\"is thresholded using\", 1)\n",
      "Pair{AbstractString,Int64}(\"while EC 50\", 1)\n",
      "Pair{AbstractString,Int64}(\"4 , 5\", 1)\n",
      "Pair{AbstractString,Int64}(\"Emka Technologies )\", 1)\n",
      "Pair{AbstractString,Int64}(\"presented. For velocity\", 1)\n",
      "Pair{AbstractString,Int64}(\"them in\", 1)\n",
      "Pair{AbstractString,Int64}(\"too low a\", 1)\n",
      "Pair{AbstractString,Int64}(\"behaved\", 1)\n",
      "Pair{AbstractString,Int64}(\"induced while\", 1)\n",
      "Pair{AbstractString,Int64}(\"* p< 0.05\", 3)\n",
      "Pair{AbstractString,Int64}(\"whereas in\", 2)\n",
      "Pair{AbstractString,Int64}(\"pro , 4.7\", 1)\n",
      "Pair{AbstractString,Int64}(\"necrosis\", 4)\n",
      "Pair{AbstractString,Int64}(\"pseudoviruses decorated with\", 1)\n",
      "Pair{AbstractString,Int64}(\"SARS-CoV ( Fig.\", 1)\n",
      "Pair{AbstractString,Int64}(\"EMEM )\", 1)\n",
      "Pair{AbstractString,Int64}(\"11 u showed\", 1)\n",
      "Pair{AbstractString,Int64}(\"0201\", 1)\n",
      "Pair{AbstractString,Int64}(\"principles of memory\", 1)\n",
      "Pair{AbstractString,Int64}(\"characterized ( Banerjee\", 1)\n",
      "Pair{AbstractString,Int64}(\"protein 248\", 1)\n",
      "Pair{AbstractString,Int64}(\"mechanisms of learning\", 1)\n",
      "Pair{AbstractString,Int64}(\"Miura\", 3)\n",
      "Pair{AbstractString,Int64}(\"more efficiently on\", 1)\n",
      "Pair{AbstractString,Int64}(\"concentration of 0.2\", 1)\n",
      "Pair{AbstractString,Int64}(\"exhibited a 1.9\", 1)\n",
      "Pair{AbstractString,Int64}(\"ΔG ( MERS\", 2)\n",
      "Pair{AbstractString,Int64}(\"1.2 µM )\", 1)\n",
      "Pair{AbstractString,Int64}(\"replication levels\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 ' benzyl\", 1)\n",
      "Pair{AbstractString,Int64}(\"Dq-mCherry positive\", 1)\n",
      "Pair{AbstractString,Int64}(\"23 24 25\", 4)\n",
      "Pair{AbstractString,Int64}(\"next step\", 4)\n",
      "Pair{AbstractString,Int64}(\"Solution ( Thermo\", 1)\n",
      "Pair{AbstractString,Int64}(\"in 100 %\", 3)\n",
      "Pair{AbstractString,Int64}(\"in 0.15 mL\", 1)\n",
      "Pair{AbstractString,Int64}(\"cyclohexylmethyl (\", 1)\n",
      "Pair{AbstractString,Int64}(\"time spent on\", 2)\n",
      "Pair{AbstractString,Int64}(\"retrieve the perception\", 1)\n",
      "Pair{AbstractString,Int64}(\"I DE is\", 1)\n",
      "Pair{AbstractString,Int64}(\"angle of\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.0312 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"all the virus\", 1)\n",
      "Pair{AbstractString,Int64}(\"pro , IC\", 1)\n",
      "Pair{AbstractString,Int64}(\"pro , totally\", 1)\n",
      "Pair{AbstractString,Int64}(\"H , br\", 1)\n",
      "Pair{AbstractString,Int64}(\"a plate\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.6 mA\", 2)\n",
      "Pair{AbstractString,Int64}(\"Gln 189 side-chain\", 1)\n",
      "Pair{AbstractString,Int64}(\"solution. The\", 2)\n",
      "Pair{AbstractString,Int64}(\"injected with PBS\", 1)\n",
      "Pair{AbstractString,Int64}(\"verified viral\", 1)\n",
      "Pair{AbstractString,Int64}(\", a kind\", 1)\n",
      "Pair{AbstractString,Int64}(\"image\", 2)\n",
      "Pair{AbstractString,Int64}(\"Moreover , as\", 1)\n",
      "Pair{AbstractString,Int64}(\"31.64\", 2)\n",
      "Pair{AbstractString,Int64}(\"the ring\", 1)\n",
      "Pair{AbstractString,Int64}(\"mometasone. A\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"tagged neurons. Previous\", 1)\n",
      "Pair{AbstractString,Int64}(\"to Volker Thiel\", 1)\n",
      "Pair{AbstractString,Int64}(\"formation of functional\", 1)\n",
      "Pair{AbstractString,Int64}(\"125 mM\", 1)\n",
      "Pair{AbstractString,Int64}(\"in the maximum\", 2)\n",
      "Pair{AbstractString,Int64}(\"a clonal\", 1)\n",
      "Pair{AbstractString,Int64}(\"evaluate LY 6\", 1)\n",
      "Pair{AbstractString,Int64}(\"We therefore restricted\", 1)\n",
      "Pair{AbstractString,Int64}(\"the inferred\", 1)\n",
      "Pair{AbstractString,Int64}(\"-MHz instruments. Chemical\", 1)\n",
      "Pair{AbstractString,Int64}(\"was vortexed thoroughly\", 1)\n",
      "Pair{AbstractString,Int64}(\"detected PEDV\", 1)\n",
      "Pair{AbstractString,Int64}(\"algestone\", 1)\n",
      "Pair{AbstractString,Int64}(\", homogenization ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"amides of\", 4)\n",
      "Pair{AbstractString,Int64}(\"viruses 29 .\", 1)\n",
      "Pair{AbstractString,Int64}(\"a new 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Re-EMC/MERS )\", 1)\n",
      "Pair{AbstractString,Int64}(\"mM HEPES\", 1)\n",
      "Pair{AbstractString,Int64}(\"; Quinlan et\", 1)\n",
      "Pair{AbstractString,Int64}(\"Vero )\", 1)\n",
      "Pair{AbstractString,Int64}(\"imaging analysis ImageXpress\", 1)\n",
      "Pair{AbstractString,Int64}(\"the oxidation of\", 1)\n",
      "Pair{AbstractString,Int64}(\"excluded from the\", 2)\n",
      "Pair{AbstractString,Int64}(\"time points and\", 2)\n",
      "Pair{AbstractString,Int64}(\"ΔEnv\", 1)\n",
      "Pair{AbstractString,Int64}(\"fluorophor coumaryl\", 1)\n",
      "Pair{AbstractString,Int64}(\"M 0202 S\", 1)\n",
      "Pair{AbstractString,Int64}(\"5 c )\", 1)\n",
      "Pair{AbstractString,Int64}(\"syncytia , plaques\", 1)\n",
      "Pair{AbstractString,Int64}(\"Transkriptase ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"% DMSO at\", 2)\n",
      "Pair{AbstractString,Int64}(\"Furthermore\", 13)\n",
      "Pair{AbstractString,Int64}(\"aketoamides and\", 1)\n",
      "Pair{AbstractString,Int64}(\"doxycycline was\", 1)\n",
      "Pair{AbstractString,Int64}(\"min.\", 2)\n",
      "Pair{AbstractString,Int64}(\"environmental\", 1)\n",
      "Pair{AbstractString,Int64}(\"sample.\", 1)\n",
      "Pair{AbstractString,Int64}(\"using MUSCLE v\", 1)\n",
      "Pair{AbstractString,Int64}(\", the human\", 1)\n",
      "Pair{AbstractString,Int64}(\"but 149 theoretically\", 1)\n",
      "Pair{AbstractString,Int64}(\"WIV 04 to\", 2)\n",
      "Pair{AbstractString,Int64}(\"ensembles. Nevertheless\", 1)\n",
      "Pair{AbstractString,Int64}(\"olfactory learning\", 2)\n",
      "Pair{AbstractString,Int64}(\"was 167 abolished\", 1)\n",
      "Pair{AbstractString,Int64}(\"process\", 6)\n",
      "Pair{AbstractString,Int64}(\"High Fidelity\", 1)\n",
      "Pair{AbstractString,Int64}(\"were returned to\", 2)\n",
      "Pair{AbstractString,Int64}(\"stirred for another\", 1)\n",
      "Pair{AbstractString,Int64}(\"viral replicons\", 5)\n",
      "Pair{AbstractString,Int64}(\"RNA by degrading\", 1)\n",
      "Pair{AbstractString,Int64}(\"immunocompetent\", 1)\n",
      "Pair{AbstractString,Int64}(\"its performance in\", 1)\n",
      "Pair{AbstractString,Int64}(\"an\", 66)\n",
      "Pair{AbstractString,Int64}(\"CS+ exposure\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 29 AA\", 1)\n",
      "Pair{AbstractString,Int64}(\"checked whether the\", 1)\n",
      "Pair{AbstractString,Int64}(\"as a monomer.\", 1)\n",
      "Pair{AbstractString,Int64}(\"oxidation of the\", 1)\n",
      "Pair{AbstractString,Int64}(\"the CS+ was\", 4)\n",
      "Pair{AbstractString,Int64}(\"US\", 1)\n",
      "Pair{AbstractString,Int64}(\", 38\", 1)\n",
      "Pair{AbstractString,Int64}(\"field. We\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Met\", 3)\n",
      "Pair{AbstractString,Int64}(\"side-chain in the\", 2)\n",
      "Pair{AbstractString,Int64}(\"ExoN-EA ) ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"hand , only\", 1)\n",
      "Pair{AbstractString,Int64}(\"mononuclear immune\", 1)\n",
      "Pair{AbstractString,Int64}(\"area\", 5)\n",
      "Pair{AbstractString,Int64}(\"RNA-launched\", 1)\n",
      "Pair{AbstractString,Int64}(\"beginning of\", 1)\n",
      "Pair{AbstractString,Int64}(\"diagnosis. However ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"Tanaka\", 2)\n",
      "Pair{AbstractString,Int64}(\"primary cell samples\", 1)\n",
      "Pair{AbstractString,Int64}(\", at 37\", 1)\n",
      "Pair{AbstractString,Int64}(\"their excitability\", 1)\n",
      "Pair{AbstractString,Int64}(\", original plasmid\", 1)\n",
      "Pair{AbstractString,Int64}(\"Enteroviruses are\", 1)\n",
      "Pair{AbstractString,Int64}(\"165 ) in\", 1)\n",
      "Pair{AbstractString,Int64}(\"factor ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"RNA in the\", 2)\n",
      "Pair{AbstractString,Int64}(\"NCBI accession\", 1)\n",
      "Pair{AbstractString,Int64}(\"conditions.\", 5)\n",
      "Pair{AbstractString,Int64}(\"toxicity data have\", 1)\n",
      "Pair{AbstractString,Int64}(\"= 9.0 Hz\", 1)\n",
      "Pair{AbstractString,Int64}(\"in electrophysiological recordings\", 1)\n",
      "Pair{AbstractString,Int64}(\"for at\", 3)\n",
      "Pair{AbstractString,Int64}(\"p= 0.0662\", 1)\n",
      "Pair{AbstractString,Int64}(\"read-out was\", 1)\n",
      "Pair{AbstractString,Int64}(\"Ma , Xiao\", 1)\n",
      "Pair{AbstractString,Int64}(\") as recommended\", 1)\n",
      "Pair{AbstractString,Int64}(\"isolated. However ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"otherwise indicated. Primary\", 1)\n",
      "Pair{AbstractString,Int64}(\"fitness , or\", 1)\n",
      "Pair{AbstractString,Int64}(\"MS Excel (\", 1)\n",
      "Pair{AbstractString,Int64}(\"-inhibitor complex mixed\", 1)\n",
      "Pair{AbstractString,Int64}(\"5 minutes to\", 1)\n",
      "Pair{AbstractString,Int64}(\"activity decreased to\", 1)\n",
      "Pair{AbstractString,Int64}(\"L 473 I\", 1)\n",
      "Pair{AbstractString,Int64}(\"ligand CNO\", 2)\n",
      "Pair{AbstractString,Int64}(\"ZV\", 1)\n",
      "Pair{AbstractString,Int64}(\"acid. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"and selected\", 1)\n",
      "Pair{AbstractString,Int64}(\"regression\", 1)\n",
      "Pair{AbstractString,Int64}(\"concentration , were\", 1)\n",
      "Pair{AbstractString,Int64}(\"the stability\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.0088 ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"and homogenized. One-eighth\", 1)\n",
      "Pair{AbstractString,Int64}(\"to Western blot\", 1)\n",
      "Pair{AbstractString,Int64}(\"H NMR\", 3)\n",
      "Pair{AbstractString,Int64}(\"widespread than SARS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"properties of\", 3)\n",
      "Pair{AbstractString,Int64}(\"( Treestar\", 1)\n",
      "Pair{AbstractString,Int64}(\"artificial contextual\", 1)\n",
      "Pair{AbstractString,Int64}(\"genetically 133\", 1)\n",
      "Pair{AbstractString,Int64}(\"to IC 50\", 1)\n",
      "Pair{AbstractString,Int64}(\"suggesting that 92\", 1)\n",
      "Pair{AbstractString,Int64}(\"a low dose\", 1)\n",
      "Pair{AbstractString,Int64}(\"231\", 1)\n",
      "Pair{AbstractString,Int64}(\"amino\", 20)\n",
      "Pair{AbstractString,Int64}(\"( 5 -FU\", 3)\n",
      "Pair{AbstractString,Int64}(\"6 e ΔHSC\", 21)\n",
      "Pair{AbstractString,Int64}(\"Both groups exhibited\", 1)\n",
      "Pair{AbstractString,Int64}(\"key factors\", 1)\n",
      "Pair{AbstractString,Int64}(\"experimental readout for\", 1)\n",
      "Pair{AbstractString,Int64}(\"healthy\", 1)\n",
      "Pair{AbstractString,Int64}(\"11 s can\", 1)\n",
      "Pair{AbstractString,Int64}(\"Metabolic\", 1)\n",
      "Pair{AbstractString,Int64}(\"higher for shorter\", 1)\n",
      "Pair{AbstractString,Int64}(\"Approximately\", 1)\n",
      "Pair{AbstractString,Int64}(\"compared to ΔEnv\", 1)\n",
      "Pair{AbstractString,Int64}(\"normalize cell counts\", 1)\n",
      "Pair{AbstractString,Int64}(\"calcium and\", 2)\n",
      "Pair{AbstractString,Int64}(\"NanoDrop\", 1)\n",
      "Pair{AbstractString,Int64}(\"maintained 409\", 1)\n",
      "Pair{AbstractString,Int64}(\"in DREADDnegative\", 1)\n",
      "Pair{AbstractString,Int64}(\"W= 55\", 1)\n",
      "Pair{AbstractString,Int64}(\"the main\", 4)\n",
      "Pair{AbstractString,Int64}(\", and p\", 1)\n",
      "Pair{AbstractString,Int64}(\"sequence in a\", 1)\n",
      "Pair{AbstractString,Int64}(\"host-cell types ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"presentation and\", 1)\n",
      "Pair{AbstractString,Int64}(\"more\", 25)\n",
      "Pair{AbstractString,Int64}(\"avoiding or\", 1)\n",
      "Pair{AbstractString,Int64}(\"significantly increases\", 2)\n",
      "Pair{AbstractString,Int64}(\"Note the\", 1)\n",
      "Pair{AbstractString,Int64}(\"was 4.9 µM\", 1)\n",
      "Pair{AbstractString,Int64}(\"infection with\", 1)\n",
      "Pair{AbstractString,Int64}(\"is higher than\", 1)\n",
      "Pair{AbstractString,Int64}(\"their retention\", 1)\n",
      "Pair{AbstractString,Int64}(\"determining host tropism\", 1)\n",
      "Pair{AbstractString,Int64}(\"n-butyl in\", 1)\n",
      "Pair{AbstractString,Int64}(\"can lead to\", 2)\n",
      "Pair{AbstractString,Int64}(\", SIGLEC 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"during the 7\", 1)\n",
      "Pair{AbstractString,Int64}(\"added to a\", 2)\n",
      "Pair{AbstractString,Int64}(\"carrying the A\", 1)\n",
      "Pair{AbstractString,Int64}(\"( HSC\", 1)\n",
      "Pair{AbstractString,Int64}(\", HRV 2\", 2)\n",
      "Pair{AbstractString,Int64}(\"weak\", 4)\n",
      "Pair{AbstractString,Int64}(\"MgCl 2 hexahydrate\", 1)\n",
      "Pair{AbstractString,Int64}(\"and values were\", 1)\n",
      "Pair{AbstractString,Int64}(\"against the replicons\", 1)\n",
      "Pair{AbstractString,Int64}(\"preceded odor presentation\", 1)\n",
      "Pair{AbstractString,Int64}(\"at least author/funder.\", 1)\n",
      "Pair{AbstractString,Int64}(\"generate sufficient Free\", 1)\n",
      "Pair{AbstractString,Int64}(\"and \\\"\", 5)\n",
      "Pair{AbstractString,Int64}(\"doi\", 130)\n",
      "Pair{AbstractString,Int64}(\"32 (\", 1)\n",
      "Pair{AbstractString,Int64}(\", the cyclopropylmethyl\", 1)\n",
      "Pair{AbstractString,Int64}(\"the Fos-tagged population\", 1)\n",
      "Pair{AbstractString,Int64}(\"and plates were\", 1)\n",
      "Pair{AbstractString,Int64}(\"as cystic fibrosis\", 1)\n",
      "Pair{AbstractString,Int64}(\"pocket. Having\", 1)\n",
      "Pair{AbstractString,Int64}(\"identity of\", 1)\n",
      "Pair{AbstractString,Int64}(\"H-bonding donor/acceptor\", 1)\n",
      "Pair{AbstractString,Int64}(\"memory. Indeed ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"recall (\", 2)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\". Mice maintained\", 1)\n",
      "Pair{AbstractString,Int64}(\"n= 11 ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"good correlation between\", 1)\n",
      "Pair{AbstractString,Int64}(\"suspensions were analyzed\", 1)\n",
      "Pair{AbstractString,Int64}(\"\\\" released\", 1)\n",
      "Pair{AbstractString,Int64}(\"synonymous mutations increased\", 1)\n",
      "Pair{AbstractString,Int64}(\"odor originating from\", 1)\n",
      "Pair{AbstractString,Int64}(\"Perl\", 1)\n",
      "Pair{AbstractString,Int64}(\"n were\", 1)\n",
      "Pair{AbstractString,Int64}(\"Pro. On the\", 1)\n",
      "Pair{AbstractString,Int64}(\"nucleoside 241\", 1)\n",
      "Pair{AbstractString,Int64}(\"Economo et\", 1)\n",
      "Pair{AbstractString,Int64}(\"and fluticasone ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"infections and prolonged\", 1)\n",
      "Pair{AbstractString,Int64}(\"to Michael acceptors\", 1)\n",
      "Pair{AbstractString,Int64}(\"replication cycles\", 1)\n",
      "Pair{AbstractString,Int64}(\"the largest cortical\", 1)\n",
      "Pair{AbstractString,Int64}(\", U= 22\", 2)\n",
      "Pair{AbstractString,Int64}(\"-CH=O\", 1)\n",
      "Pair{AbstractString,Int64}(\"6 B )\", 3)\n",
      "Pair{AbstractString,Int64}(\"these coronaviruses were\", 1)\n",
      "Pair{AbstractString,Int64}(\"5 ' -\", 2)\n",
      "Pair{AbstractString,Int64}(\"Solution Cell Proliferation\", 1)\n",
      "Pair{AbstractString,Int64}(\"022\", 1)\n",
      "Pair{AbstractString,Int64}(\"temperature higher\", 4)\n",
      "Pair{AbstractString,Int64}(\"418 sulfate\", 1)\n",
      "Pair{AbstractString,Int64}(\"LC 10\", 1)\n",
      "Pair{AbstractString,Int64}(\"of distributed\", 1)\n",
      "Pair{AbstractString,Int64}(\"cases of\", 1)\n",
      "Pair{AbstractString,Int64}(\"coordinates include SARS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"were amplified\", 5)\n",
      "Pair{AbstractString,Int64}(\"4 seconds\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 - 4\", 1)\n",
      "Pair{AbstractString,Int64}(\"luciferase 1 -\", 1)\n",
      "Pair{AbstractString,Int64}(\"was under stronger\", 1)\n",
      "Pair{AbstractString,Int64}(\"Biotrend Chemikalien )\", 1)\n",
      "Pair{AbstractString,Int64}(\"α-hydroxyamides\", 1)\n",
      "Pair{AbstractString,Int64}(\"31 32\", 4)\n",
      "Pair{AbstractString,Int64}(\"total of\", 1)\n",
      "Pair{AbstractString,Int64}(\"higher ( Wilcoxon\", 1)\n",
      "Pair{AbstractString,Int64}(\"but overlapping ensembles\", 1)\n",
      "Pair{AbstractString,Int64}(\"\\\" for\", 3)\n",
      "Pair{AbstractString,Int64}(\"modulates the expression\", 1)\n",
      "Pair{AbstractString,Int64}(\"dS of S\", 1)\n",
      "Pair{AbstractString,Int64}(\"coronavirus species ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"described three peptidyl\", 1)\n",
      "Pair{AbstractString,Int64}(\"183 reduced by\", 1)\n",
      "Pair{AbstractString,Int64}(\". Further\", 2)\n",
      "Pair{AbstractString,Int64}(\"-can accept a\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibits S protein-mediated\", 1)\n",
      "Pair{AbstractString,Int64}(\". However ,\", 12)\n",
      "Pair{AbstractString,Int64}(\"RaTG 13 strains.\", 1)\n",
      "Pair{AbstractString,Int64}(\"more inhibitor appeared\", 1)\n",
      "Pair{AbstractString,Int64}(\"50 -mers )\", 1)\n",
      "Pair{AbstractString,Int64}(\". Silencing\", 1)\n",
      "Pair{AbstractString,Int64}(\"a search\", 1)\n",
      "Pair{AbstractString,Int64}(\"plasmids were\", 1)\n",
      "Pair{AbstractString,Int64}(\"nose. Individual\", 1)\n",
      "Pair{AbstractString,Int64}(\"histological examination. Previous\", 1)\n",
      "Pair{AbstractString,Int64}(\") . Error\", 5)\n",
      "Pair{AbstractString,Int64}(\"2.2 ( 0.5\", 1)\n",
      "Pair{AbstractString,Int64}(\"protein sequence\", 2)\n",
      "Pair{AbstractString,Int64}(\"Stevens ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"https://www.ncbi.nlm.nih.gov/geo (\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Sigma #\", 1)\n",
      "Pair{AbstractString,Int64}(\"quantity and\", 1)\n",
      "Pair{AbstractString,Int64}(\"universal inhibitor should\", 1)\n",
      "Pair{AbstractString,Int64}(\"Subsequently ,\", 6)\n",
      "Pair{AbstractString,Int64}(\"2 h. Then\", 1)\n",
      "Pair{AbstractString,Int64}(\"using Phyre 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"of 529\", 1)\n",
      "Pair{AbstractString,Int64}(\"However , as\", 1)\n",
      "Pair{AbstractString,Int64}(\", and Val\", 1)\n",
      "Pair{AbstractString,Int64}(\"16 into five\", 1)\n",
      "Pair{AbstractString,Int64}(\"cFos-tTA transgene.\", 1)\n",
      "Pair{AbstractString,Int64}(\"recall , the\", 1)\n",
      "Pair{AbstractString,Int64}(\"was degraded. In\", 1)\n",
      "Pair{AbstractString,Int64}(\"if ExoN-AE has\", 1)\n",
      "Pair{AbstractString,Int64}(\") against HCoV-NL\", 2)\n",
      "Pair{AbstractString,Int64}(\"depend on\", 1)\n",
      "Pair{AbstractString,Int64}(\"other strain\", 1)\n",
      "Pair{AbstractString,Int64}(\"the stack\", 2)\n",
      "Pair{AbstractString,Int64}(\"1 -FRAG ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"illnesses 66 ranging\", 1)\n",
      "Pair{AbstractString,Int64}(\"All diffraction\", 1)\n",
      "Pair{AbstractString,Int64}(\"is invariably\", 1)\n",
      "Pair{AbstractString,Int64}(\"72.56 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"Gln in the\", 1)\n",
      "Pair{AbstractString,Int64}(\"9 S\", 1)\n",
      "Pair{AbstractString,Int64}(\"both the enterovirus\", 1)\n",
      "Pair{AbstractString,Int64}(\"Some white\", 1)\n",
      "Pair{AbstractString,Int64}(\"cases and has\", 1)\n",
      "Pair{AbstractString,Int64}(\"the most tolerant\", 1)\n",
      "Pair{AbstractString,Int64}(\"reversion is not\", 1)\n",
      "Pair{AbstractString,Int64}(\"portion of ns\", 1)\n",
      "Pair{AbstractString,Int64}(\"the current\", 1)\n",
      "Pair{AbstractString,Int64}(\"coding region were\", 1)\n",
      "Pair{AbstractString,Int64}(\"this study ,\", 3)\n",
      "Pair{AbstractString,Int64}(\"one-digit\", 2)\n",
      "Pair{AbstractString,Int64}(\"acetate. The organic\", 2)\n",
      "Pair{AbstractString,Int64}(\"( 8,28 nt\", 1)\n",
      "Pair{AbstractString,Int64}(\"was electrified\", 1)\n",
      "Pair{AbstractString,Int64}(\"for each of\", 1)\n",
      "Pair{AbstractString,Int64}(\"period that\", 1)\n",
      "Pair{AbstractString,Int64}(\"Figure 2 D\", 2)\n",
      "Pair{AbstractString,Int64}(\"of non-synonymous to\", 1)\n",
      "Pair{AbstractString,Int64}(\"71 -RLuc )\", 1)\n",
      "Pair{AbstractString,Int64}(\"cleavage site and\", 1)\n",
      "Pair{AbstractString,Int64}(\"tempature for 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 ] and\", 1)\n",
      "Pair{AbstractString,Int64}(\"35\", 8)\n",
      "Pair{AbstractString,Int64}(\"after high temperature\", 1)\n",
      "Pair{AbstractString,Int64}(\"resulting sequence\", 1)\n",
      "Pair{AbstractString,Int64}(\"microscope with a\", 1)\n",
      "Pair{AbstractString,Int64}(\"empty-handed when the\", 1)\n",
      "Pair{AbstractString,Int64}(\"7.4 ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\"and pervasive positive\", 1)\n",
      "Pair{AbstractString,Int64}(\"representative coronaviruses and\", 1)\n",
      "Pair{AbstractString,Int64}(\"55 between MluI\", 1)\n",
      "Pair{AbstractString,Int64}(\"top\", 1)\n",
      "Pair{AbstractString,Int64}(\"-CH=CH-CO-\", 1)\n",
      "Pair{AbstractString,Int64}(\") cells\", 5)\n",
      "Pair{AbstractString,Int64}(\"media as described\", 1)\n",
      "Pair{AbstractString,Int64}(\"from Sigma\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells previously reported\", 1)\n",
      "Pair{AbstractString,Int64}(\"odor-specific\", 1)\n",
      "Pair{AbstractString,Int64}(\"evolution is mainly\", 1)\n",
      "Pair{AbstractString,Int64}(\"41 , whereas\", 1)\n",
      "Pair{AbstractString,Int64}(\") or 48\", 1)\n",
      "Pair{AbstractString,Int64}(\"substituent , the\", 1)\n",
      "Pair{AbstractString,Int64}(\"synonymous substitutions between\", 1)\n",
      "Pair{AbstractString,Int64}(\"Crystals of HCoV-NL\", 2)\n",
      "Pair{AbstractString,Int64}(\"at a specified\", 1)\n",
      "Pair{AbstractString,Int64}(\"u ) .\", 2)\n",
      "Pair{AbstractString,Int64}(\"more flexible\", 1)\n",
      "Pair{AbstractString,Int64}(\"4 d )\", 1)\n",
      "Pair{AbstractString,Int64}(\"side when\", 1)\n",
      "Pair{AbstractString,Int64}(\"- 7 MΩ\", 1)\n",
      "Pair{AbstractString,Int64}(\"were carried out\", 2)\n",
      "Pair{AbstractString,Int64}(\"( with 0\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.08 µM ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"trace reactivation\", 1)\n",
      "Pair{AbstractString,Int64}(\"viral templates would\", 1)\n",
      "Pair{AbstractString,Int64}(\"to MHV infection\", 2)\n",
      "Pair{AbstractString,Int64}(\"1 ) to\", 7)\n",
      "Pair{AbstractString,Int64}(\"overnight (\", 1)\n",
      "Pair{AbstractString,Int64}(\"value :\", 2)\n",
      "Pair{AbstractString,Int64}(\"compared , all\", 1)\n",
      "Pair{AbstractString,Int64}(\"Start\", 1)\n",
      "Pair{AbstractString,Int64}(\"2020.02.20.958785 doi\", 8)\n",
      "Pair{AbstractString,Int64}(\", as steroids\", 1)\n",
      "Pair{AbstractString,Int64}(\"lowest codon usage\", 1)\n",
      "Pair{AbstractString,Int64}(\"doi :\", 130)\n",
      "Pair{AbstractString,Int64}(\"M Li 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"box and their\", 1)\n",
      "Pair{AbstractString,Int64}(\"CoV ) were\", 1)\n",
      "Pair{AbstractString,Int64}(\") increase\", 1)\n",
      "Pair{AbstractString,Int64}(\"group ( Mann-Whitney\", 1)\n",
      "Pair{AbstractString,Int64}(\"paired two-tailed\", 1)\n",
      "Pair{AbstractString,Int64}(\"DMRIE-C was used\", 1)\n",
      "Pair{AbstractString,Int64}(\"CL\", 4)\n",
      "Pair{AbstractString,Int64}(\", habituated\", 1)\n",
      "Pair{AbstractString,Int64}(\"modified the behavioral\", 1)\n",
      "Pair{AbstractString,Int64}(\"with Shim-pack GIST\", 1)\n",
      "Pair{AbstractString,Int64}(\"conditions. The numbers\", 1)\n",
      "Pair{AbstractString,Int64}(\"able to\", 8)\n",
      "Pair{AbstractString,Int64}(\"priming enzyme\", 1)\n",
      "Pair{AbstractString,Int64}(\"promotes\", 2)\n",
      "Pair{AbstractString,Int64}(\"enhance the\", 1)\n",
      "Pair{AbstractString,Int64}(\"able to detect\", 1)\n",
      "Pair{AbstractString,Int64}(\"s , the\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 6.0 g\", 1)\n",
      "Pair{AbstractString,Int64}(\", encouraging\", 1)\n",
      "Pair{AbstractString,Int64}(\"CoV-resistant hamster cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"p= 0.3698 ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"In HCoV-NL\", 1)\n",
      "Pair{AbstractString,Int64}(\"including the prefrontal\", 1)\n",
      "Pair{AbstractString,Int64}(\"mild respiratory symptoms\", 1)\n",
      "Pair{AbstractString,Int64}(\", ExoN-AE reverted\", 1)\n",
      "Pair{AbstractString,Int64}(\"WIV 04 was\", 1)\n",
      "Pair{AbstractString,Int64}(\"its larger\", 1)\n",
      "Pair{AbstractString,Int64}(\": P\", 1)\n",
      "Pair{AbstractString,Int64}(\"bioRxiv preprint ANOVA\", 1)\n",
      "Pair{AbstractString,Int64}(\"copies per ml\", 2)\n",
      "Pair{AbstractString,Int64}(\"investigate whether\", 3)\n",
      "Pair{AbstractString,Int64}(\"day\", 9)\n",
      "Pair{AbstractString,Int64}(\"wheat surfaces\", 1)\n",
      "Pair{AbstractString,Int64}(\"serum ALT while\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 µg/mL puromycin.\", 1)\n",
      "Pair{AbstractString,Int64}(\"mM DTT\", 2)\n",
      "Pair{AbstractString,Int64}(\"extremely potent against\", 1)\n",
      "Pair{AbstractString,Int64}(\"that contribute\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 1 nt\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Fig. 1\", 12)\n",
      "Pair{AbstractString,Int64}(\"M pro ,\", 35)\n",
      "Pair{AbstractString,Int64}(\"inhibitors. To\", 1)\n",
      "Pair{AbstractString,Int64}(\"pQCXIP plasmid\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 8 RNA\", 1)\n",
      "Pair{AbstractString,Int64}(\") were washed\", 1)\n",
      "Pair{AbstractString,Int64}(\"newly identified SARS-CoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"KLK 13 by\", 1)\n",
      "Pair{AbstractString,Int64}(\"replaced by\", 3)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"12,000 ×g. Washing\", 1)\n",
      "Pair{AbstractString,Int64}(\"phenylalanine residue ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"pCAGGS-mCherry )\", 1)\n",
      "Pair{AbstractString,Int64}(\"fibrosis\", 1)\n",
      "Pair{AbstractString,Int64}(\"awake mice\", 1)\n",
      "Pair{AbstractString,Int64}(\"4 +\", 5)\n",
      "Pair{AbstractString,Int64}(\"In our study\", 1)\n",
      "Pair{AbstractString,Int64}(\"and SARS-CoV ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"( 13\", 2)\n",
      "Pair{AbstractString,Int64}(\"in spike protein\", 1)\n",
      "Pair{AbstractString,Int64}(\"drive learned behaviors\", 1)\n",
      "Pair{AbstractString,Int64}(\"at 50\", 1)\n",
      "Pair{AbstractString,Int64}(\"flexibility into the\", 1)\n",
      "Pair{AbstractString,Int64}(\"7000\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 500 µL\", 1)\n",
      "Pair{AbstractString,Int64}(\"4 ) The\", 1)\n",
      "Pair{AbstractString,Int64}(\"similarly close\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.33 µM\", 1)\n",
      "Pair{AbstractString,Int64}(\"subsequently selected\", 1)\n",
      "Pair{AbstractString,Int64}(\"to synonymous\", 1)\n",
      "Pair{AbstractString,Int64}(\"importance of 292\", 1)\n",
      "Pair{AbstractString,Int64}(\"generate CRISPRresistant LY\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 versus bat\", 1)\n",
      "Pair{AbstractString,Int64}(\"analysis as described.\", 1)\n",
      "Pair{AbstractString,Int64}(\"r exhibited IC\", 1)\n",
      "Pair{AbstractString,Int64}(\"permeabilized\", 1)\n",
      "Pair{AbstractString,Int64}(\"of 4 days\", 1)\n",
      "Pair{AbstractString,Int64}(\"finally lead\", 1)\n",
      "Pair{AbstractString,Int64}(\"30 countries ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"to be presumably\", 1)\n",
      "Pair{AbstractString,Int64}(\"geneticin (\", 1)\n",
      "Pair{AbstractString,Int64}(\"in-vitro activity\", 1)\n",
      "Pair{AbstractString,Int64}(\"Banerjee\", 1)\n",
      "Pair{AbstractString,Int64}(\"preprint Nagel )\", 1)\n",
      "Pair{AbstractString,Int64}(\"would not restore\", 1)\n",
      "Pair{AbstractString,Int64}(\". Neutral red\", 1)\n",
      "Pair{AbstractString,Int64}(\"Memory author/funder. All\", 1)\n",
      "Pair{AbstractString,Int64}(\"-Microglobulin (\", 1)\n",
      "Pair{AbstractString,Int64}(\"242 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"the position 7\", 1)\n",
      "Pair{AbstractString,Int64}(\"mobility of\", 1)\n",
      "Pair{AbstractString,Int64}(\"viruses predominantly\", 1)\n",
      "Pair{AbstractString,Int64}(\"292 KLK 13\", 1)\n",
      "Pair{AbstractString,Int64}(\"S 1000 Flow\", 1)\n",
      "Pair{AbstractString,Int64}(\"and higher amount\", 1)\n",
      "Pair{AbstractString,Int64}(\"population between P\", 1)\n",
      "Pair{AbstractString,Int64}(\"at 92 o\", 8)\n",
      "Pair{AbstractString,Int64}(\"optimized with\", 1)\n",
      "Pair{AbstractString,Int64}(\"BMDM )\", 2)\n",
      "Pair{AbstractString,Int64}(\"temperature. The reaction\", 1)\n",
      "Pair{AbstractString,Int64}(\"Nancy. For enterovirus\", 1)\n",
      "Pair{AbstractString,Int64}(\", 1990\", 1)\n",
      "Pair{AbstractString,Int64}(\"virus-induced\", 3)\n",
      "Pair{AbstractString,Int64}(\"by anti-HA\", 1)\n",
      "Pair{AbstractString,Int64}(\"activesite\", 1)\n",
      "Pair{AbstractString,Int64}(\", and hM\", 2)\n",
      "Pair{AbstractString,Int64}(\"robustly escape from\", 1)\n",
      "Pair{AbstractString,Int64}(\"totally inactive (\", 1)\n",
      "Pair{AbstractString,Int64}(\"C protease (\", 1)\n",
      "Pair{AbstractString,Int64}(\"control of viral\", 1)\n",
      "Pair{AbstractString,Int64}(\"particularly well-suited\", 1)\n",
      "Pair{AbstractString,Int64}(\"in PCR instrument\", 1)\n",
      "Pair{AbstractString,Int64}(\"F 216 Y\", 2)\n",
      "Pair{AbstractString,Int64}(\"infected RD _\", 1)\n",
      "Pair{AbstractString,Int64}(\"turned out to\", 1)\n",
      "Pair{AbstractString,Int64}(\"surface expression of\", 1)\n",
      "Pair{AbstractString,Int64}(\"E KO A\", 1)\n",
      "Pair{AbstractString,Int64}(\"however\", 8)\n",
      "Pair{AbstractString,Int64}(\"Feb.\", 1)\n",
      "Pair{AbstractString,Int64}(\"fear conditioning encode\", 1)\n",
      "Pair{AbstractString,Int64}(\"distributed ensembles\", 2)\n",
      "Pair{AbstractString,Int64}(\"[ 52 ]\", 2)\n",
      "Pair{AbstractString,Int64}(\"by cells previously\", 1)\n",
      "Pair{AbstractString,Int64}(\", 2012 ;\", 9)\n",
      "Pair{AbstractString,Int64}(\"under stronger negative\", 1)\n",
      "Pair{AbstractString,Int64}(\"EMC ) or\", 1)\n",
      "Pair{AbstractString,Int64}(\"glutamic\", 1)\n",
      "Pair{AbstractString,Int64}(\"current steps\", 1)\n",
      "Pair{AbstractString,Int64}(\"changes\", 19)\n",
      "Pair{AbstractString,Int64}(\"blocking both coronavirus\", 1)\n",
      "Pair{AbstractString,Int64}(\"-diamidine- 2 '\", 1)\n",
      "Pair{AbstractString,Int64}(\"generate a standard\", 1)\n",
      "Pair{AbstractString,Int64}(\"or CS-presentation.\", 1)\n",
      "Pair{AbstractString,Int64}(\"purity ;\", 1)\n",
      "Pair{AbstractString,Int64}(\", chemogenetic\", 1)\n",
      "Pair{AbstractString,Int64}(\") , however\", 1)\n",
      "Pair{AbstractString,Int64}(\", 1:2\", 1)\n",
      "Pair{AbstractString,Int64}(\"derived 343 from\", 1)\n",
      "Pair{AbstractString,Int64}(\"and P 250\", 2)\n",
      "Pair{AbstractString,Int64}(\"different between the\", 1)\n",
      "Pair{AbstractString,Int64}(\"observed after treatment\", 1)\n",
      "Pair{AbstractString,Int64}(\"50 mL of\", 5)\n",
      "Pair{AbstractString,Int64}(\", used as\", 1)\n",
      "Pair{AbstractString,Int64}(\"Zhang et\", 1)\n",
      "Pair{AbstractString,Int64}(\"MEME ) ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"15 /\", 1)\n",
      "Pair{AbstractString,Int64}(\") . Determination\", 1)\n",
      "Pair{AbstractString,Int64}(\"water or methanol/\", 1)\n",
      "Pair{AbstractString,Int64}(\"permissive for HCoV-HKU\", 1)\n",
      "Pair{AbstractString,Int64}(\"374 in log\", 1)\n",
      "Pair{AbstractString,Int64}(\"conditioning disrupts odor\", 1)\n",
      "Pair{AbstractString,Int64}(\"developed\", 7)\n",
      "Pair{AbstractString,Int64}(\") , The\", 1)\n",
      "Pair{AbstractString,Int64}(\"determine extracellular\", 2)\n",
      "Pair{AbstractString,Int64}(\"examination.\", 1)\n",
      "Pair{AbstractString,Int64}(\"few pictures\", 1)\n",
      "Pair{AbstractString,Int64}(\"pro and\", 12)\n",
      "Pair{AbstractString,Int64}(\"DMEM )\", 1)\n",
      "Pair{AbstractString,Int64}(\"from MERS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibitors of\", 5)\n",
      "Pair{AbstractString,Int64}(\"could 117 navigate\", 1)\n",
      "Pair{AbstractString,Int64}(\"IC 50 values\", 6)\n",
      "Pair{AbstractString,Int64}(\"test ,\", 22)\n",
      "Pair{AbstractString,Int64}(\"; Economo\", 1)\n",
      "Pair{AbstractString,Int64}(\"MERS-CoV.\", 1)\n",
      "Pair{AbstractString,Int64}(\"and dispersed subpopulation\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 -glutamine residue\", 1)\n",
      "Pair{AbstractString,Int64}(\"bioRxiv preprint and\", 1)\n",
      "Pair{AbstractString,Int64}(\"computations and their\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 18 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"mOsm\", 1)\n",
      "Pair{AbstractString,Int64}(\"rate ( FDR\", 1)\n",
      "Pair{AbstractString,Int64}(\"MERS in 2013\", 1)\n",
      "Pair{AbstractString,Int64}(\"carbon atoms.\", 1)\n",
      "Pair{AbstractString,Int64}(\"overnight growth ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"compound showed\", 2)\n",
      "Pair{AbstractString,Int64}(\"TaqManTM\", 1)\n",
      "Pair{AbstractString,Int64}(\"provided ad libitum.\", 1)\n",
      "Pair{AbstractString,Int64}(\"ester with bromoacetonitrile.\", 1)\n",
      "Pair{AbstractString,Int64}(\"r by the\", 1)\n",
      "Pair{AbstractString,Int64}(\"measured in fresh\", 1)\n",
      "Pair{AbstractString,Int64}(\") and 20\", 1)\n",
      "Pair{AbstractString,Int64}(\"measured in current-clamp\", 1)\n",
      "Pair{AbstractString,Int64}(\"DEGs absolute FC\", 1)\n",
      "Pair{AbstractString,Int64}(\"and non-toxic at\", 1)\n",
      "Pair{AbstractString,Int64}(\"% Lglutamine ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"Crystallization\", 1)\n",
      "Pair{AbstractString,Int64}(\"in cell viability\", 1)\n",
      "Pair{AbstractString,Int64}(\"an MX 225\", 1)\n",
      "Pair{AbstractString,Int64}(\"protease is fairly\", 1)\n",
      "Pair{AbstractString,Int64}(\"v 7.2.5\", 1)\n",
      "Pair{AbstractString,Int64}(\"of bat-SARSr-CoV RaTG\", 1)\n",
      "Pair{AbstractString,Int64}(\"115 V ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"the threat\", 1)\n",
      "Pair{AbstractString,Int64}(\"Three groups\", 1)\n",
      "Pair{AbstractString,Int64}(\"should be commercially\", 1)\n",
      "Pair{AbstractString,Int64}(\"cFos-tTA-dependent\", 1)\n",
      "Pair{AbstractString,Int64}(\"from Volker Thiel\", 1)\n",
      "Pair{AbstractString,Int64}(\"for weight and\", 1)\n",
      "Pair{AbstractString,Int64}(\"plate.\", 3)\n",
      "Pair{AbstractString,Int64}(\"substitution is increased\", 1)\n",
      "Pair{AbstractString,Int64}(\"opposite\", 3)\n",
      "Pair{AbstractString,Int64}(\"question\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibitor ( 10\", 2)\n",
      "Pair{AbstractString,Int64}(\"well as by\", 2)\n",
      "Pair{AbstractString,Int64}(\"Amicon YM 10\", 1)\n",
      "Pair{AbstractString,Int64}(\"dispersed throughout\", 1)\n",
      "Pair{AbstractString,Int64}(\", the side-walls\", 1)\n",
      "Pair{AbstractString,Int64}(\"set of free\", 1)\n",
      "Pair{AbstractString,Int64}(\"Dhawale et\", 1)\n",
      "Pair{AbstractString,Int64}(\"resuspended in 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"thresholded\", 1)\n",
      "Pair{AbstractString,Int64}(\"The piriform\", 2)\n",
      "Pair{AbstractString,Int64}(\"and Bat-SL-CoVZC 45\", 4)\n",
      "Pair{AbstractString,Int64}(\"black hole quencher\", 1)\n",
      "Pair{AbstractString,Int64}(\"Real-Time\", 2)\n",
      "Pair{AbstractString,Int64}(\"wells differed 371\", 1)\n",
      "Pair{AbstractString,Int64}(\"tissue saturation\", 1)\n",
      "Pair{AbstractString,Int64}(\"Tonegawa et al.\", 1)\n",
      "Pair{AbstractString,Int64}(\"Fig. 2 ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"of ns\", 2)\n",
      "Pair{AbstractString,Int64}(\"Matsuyama et al.\", 1)\n",
      "Pair{AbstractString,Int64}(\") in THF\", 2)\n",
      "Pair{AbstractString,Int64}(\"1 , and\", 1)\n",
      "Pair{AbstractString,Int64}(\"although the EC\", 1)\n",
      "Pair{AbstractString,Int64}(\"to enter into\", 1)\n",
      "Pair{AbstractString,Int64}(\"Core. Livers were\", 1)\n",
      "Pair{AbstractString,Int64}(\"Four steroid compounds\", 1)\n",
      "Pair{AbstractString,Int64}(\"and stabilization of\", 1)\n",
      "Pair{AbstractString,Int64}(\"SARS-CoV- 2 ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"2018 in\", 1)\n",
      "Pair{AbstractString,Int64}(\"specific mutation pattern\", 1)\n",
      "Pair{AbstractString,Int64}(\"WT.\", 1)\n",
      "Pair{AbstractString,Int64}(\"diverse cellular processes\", 1)\n",
      "Pair{AbstractString,Int64}(\"the non-reinforced\", 1)\n",
      "Pair{AbstractString,Int64}(\"Fig. 5 A\", 1)\n",
      "Pair{AbstractString,Int64}(\"cleaved during virus\", 1)\n",
      "Pair{AbstractString,Int64}(\"5 N 19\", 1)\n",
      "Pair{AbstractString,Int64}(\"antiviral drugs ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"a total of\", 1)\n",
      "Pair{AbstractString,Int64}(\"other cortical structures\", 1)\n",
      "Pair{AbstractString,Int64}(\"that these non-synonymous\", 1)\n",
      "Pair{AbstractString,Int64}(\"player\", 1)\n",
      "Pair{AbstractString,Int64}(\". β-actin was\", 1)\n",
      "Pair{AbstractString,Int64}(\"( see Experimental\", 6)\n",
      "Pair{AbstractString,Int64}(\"AG , Stettlen\", 1)\n",
      "Pair{AbstractString,Int64}(\"669\", 1)\n",
      "Pair{AbstractString,Int64}(\"requiring three\", 1)\n",
      "Pair{AbstractString,Int64}(\"catalytic Cys-His pair\", 1)\n",
      "Pair{AbstractString,Int64}(\"kHz\", 4)\n",
      "Pair{AbstractString,Int64}(\"for betacoronaviruses\", 1)\n",
      "Pair{AbstractString,Int64}(\"for the binding\", 1)\n",
      "Pair{AbstractString,Int64}(\"see Fig. 4\", 1)\n",
      "Pair{AbstractString,Int64}(\"virus. Furthermore\", 1)\n",
      "Pair{AbstractString,Int64}(\"bioRxiv preprint DDX\", 1)\n",
      "Pair{AbstractString,Int64}(\"replication in\", 3)\n",
      "Pair{AbstractString,Int64}(\"1 E )\", 1)\n",
      "Pair{AbstractString,Int64}(\"to MHV\", 4)\n",
      "Pair{AbstractString,Int64}(\"enteroviruses , a\", 1)\n",
      "Pair{AbstractString,Int64}(\"( available\", 1)\n",
      "Pair{AbstractString,Int64}(\") were co-cultured\", 2)\n",
      "Pair{AbstractString,Int64}(\"data ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"RNA templates\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 O. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"DNA in the\", 1)\n",
      "Pair{AbstractString,Int64}(\"however , none\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"However , 2019\", 1)\n",
      "Pair{AbstractString,Int64}(\"S proteins from\", 1)\n",
      "Pair{AbstractString,Int64}(\"Ser in the\", 1)\n",
      "Pair{AbstractString,Int64}(\"830\", 1)\n",
      "Pair{AbstractString,Int64}(\"Thr 130 and\", 1)\n",
      "Pair{AbstractString,Int64}(\"WT-ExoN-DE 130 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"< 50 %\", 1)\n",
      "Pair{AbstractString,Int64}(\"THF\", 3)\n",
      "Pair{AbstractString,Int64}(\"non-reinforced\", 2)\n",
      "Pair{AbstractString,Int64}(\") . Approximately\", 1)\n",
      "Pair{AbstractString,Int64}(\"Re-Nsp 15 -A\", 1)\n",
      "Pair{AbstractString,Int64}(\"probability of\", 2)\n",
      "Pair{AbstractString,Int64}(\"clarity. C.\", 1)\n",
      "Pair{AbstractString,Int64}(\"177 gene\", 1)\n",
      "Pair{AbstractString,Int64}(\"Biosciences ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\"One Step qRT-PCR\", 1)\n",
      "Pair{AbstractString,Int64}(\"NS mutations at\", 1)\n",
      "Pair{AbstractString,Int64}(\"-cinnamamido- 3 -phenylpropanoate\", 1)\n",
      "Pair{AbstractString,Int64}(\"345\", 1)\n",
      "Pair{AbstractString,Int64}(\"reason ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"reservoir containing 6\", 1)\n",
      "Pair{AbstractString,Int64}(\"viruses in Vero\", 1)\n",
      "Pair{AbstractString,Int64}(\"tTA transcription factor\", 1)\n",
      "Pair{AbstractString,Int64}(\"At this\", 1)\n",
      "Pair{AbstractString,Int64}(\"request. The RNAseq\", 1)\n",
      "Pair{AbstractString,Int64}(\"associations include\", 1)\n",
      "Pair{AbstractString,Int64}(\"71 form the\", 1)\n",
      "Pair{AbstractString,Int64}(\"AAVs ) were\", 1)\n",
      "Pair{AbstractString,Int64}(\"MPD ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"process. The\", 1)\n",
      "Pair{AbstractString,Int64}(\", neutrophils\", 1)\n",
      "Pair{AbstractString,Int64}(\", 2012\", 11)\n",
      "Pair{AbstractString,Int64}(\"fold-change in\", 1)\n",
      "Pair{AbstractString,Int64}(\"were additionally supplemented\", 1)\n",
      "Pair{AbstractString,Int64}(\"for ExoN\", 1)\n",
      "Pair{AbstractString,Int64}(\"21 cell suspension\", 1)\n",
      "Pair{AbstractString,Int64}(\"04 to\", 2)\n",
      "Pair{AbstractString,Int64}(\", respectively ,\", 2)\n",
      "Pair{AbstractString,Int64}(\", [ 19\", 1)\n",
      "Pair{AbstractString,Int64}(\"multiple 87 nucleoside\", 1)\n",
      "Pair{AbstractString,Int64}(\"this genetic\", 1)\n",
      "Pair{AbstractString,Int64}(\"in DREADD-negative\", 3)\n",
      "Pair{AbstractString,Int64}(\". The OD\", 1)\n",
      "Pair{AbstractString,Int64}(\"screen\", 4)\n",
      "Pair{AbstractString,Int64}(\", and values\", 1)\n",
      "Pair{AbstractString,Int64}(\"three mutations (\", 1)\n",
      "Pair{AbstractString,Int64}(\"Non-Radioactive\", 1)\n",
      "Pair{AbstractString,Int64}(\"± 5\", 1)\n",
      "Pair{AbstractString,Int64}(\"10.1101 /\", 128)\n",
      "Pair{AbstractString,Int64}(\"( Rluc\", 2)\n",
      "Pair{AbstractString,Int64}(\"and SARS-CoV was\", 1)\n",
      "Pair{AbstractString,Int64}(\"pathways is\", 1)\n",
      "Pair{AbstractString,Int64}(\"improperly reducing\", 1)\n",
      "Pair{AbstractString,Int64}(\"were aligned using\", 1)\n",
      "Pair{AbstractString,Int64}(\"was specific\", 1)\n",
      "Pair{AbstractString,Int64}(\"for residues\", 1)\n",
      "Pair{AbstractString,Int64}(\"to remove\", 2)\n",
      "Pair{AbstractString,Int64}(\"CDS sequences\", 1)\n",
      "Pair{AbstractString,Int64}(\"designated 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"level- 3 facilities\", 1)\n",
      "Pair{AbstractString,Int64}(\"network mechanisms\", 1)\n",
      "Pair{AbstractString,Int64}(\"substrate , but\", 1)\n",
      "Pair{AbstractString,Int64}(\"of their original\", 1)\n",
      "Pair{AbstractString,Int64}(\"fragmented\", 1)\n",
      "Pair{AbstractString,Int64}(\"Then , the\", 2)\n",
      "Pair{AbstractString,Int64}(\"identification of significantly\", 1)\n",
      "Pair{AbstractString,Int64}(\"the olfactory epithelium\", 1)\n",
      "Pair{AbstractString,Int64}(\"rigid Pro. On\", 1)\n",
      "Pair{AbstractString,Int64}(\"compounds 11\", 2)\n",
      "Pair{AbstractString,Int64}(\"cells/well )\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Basu and\", 1)\n",
      "Pair{AbstractString,Int64}(\"and then sliced\", 1)\n",
      "Pair{AbstractString,Int64}(\"hematoxylin and\", 1)\n",
      "Pair{AbstractString,Int64}(\"227 in\", 1)\n",
      "Pair{AbstractString,Int64}(\"CDS )\", 2)\n",
      "Pair{AbstractString,Int64}(\"injection. The solution\", 1)\n",
      "Pair{AbstractString,Int64}(\"experiencing positive selection.\", 1)\n",
      "Pair{AbstractString,Int64}(\"weak toxicity\", 1)\n",
      "Pair{AbstractString,Int64}(\"31.95 ±\", 1)\n",
      "Pair{AbstractString,Int64}(\"342 Petaluma\", 1)\n",
      "Pair{AbstractString,Int64}(\"solid. Synthesis of\", 1)\n",
      "Pair{AbstractString,Int64}(\"viral target ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"When we\", 2)\n",
      "Pair{AbstractString,Int64}(\"assessed by\", 3)\n",
      "Pair{AbstractString,Int64}(\"51 and in\", 1)\n",
      "Pair{AbstractString,Int64}(\". Taken together\", 3)\n",
      "Pair{AbstractString,Int64}(\"( NK\", 1)\n",
      "Pair{AbstractString,Int64}(\"coumaryl and\", 1)\n",
      "Pair{AbstractString,Int64}(\"exhibited\", 12)\n",
      "Pair{AbstractString,Int64}(\"observed no\", 2)\n",
      "Pair{AbstractString,Int64}(\"DNase treatment (\", 1)\n",
      "Pair{AbstractString,Int64}(\"closely related\", 4)\n",
      "Pair{AbstractString,Int64}(\", ThermoScientific\", 1)\n",
      "Pair{AbstractString,Int64}(\"next asked\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Intan\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 337\", 1)\n",
      "Pair{AbstractString,Int64}(\"comparative analysis between\", 1)\n",
      "Pair{AbstractString,Int64}(\") clustered with\", 1)\n",
      "Pair{AbstractString,Int64}(\"ExoN motif\", 2)\n",
      "Pair{AbstractString,Int64}(\"cFos-dependent\", 1)\n",
      "Pair{AbstractString,Int64}(\"freeze/thaw cycles ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"might be improved\", 1)\n",
      "Pair{AbstractString,Int64}(\"NMDA\", 1)\n",
      "Pair{AbstractString,Int64}(\"acceptors that\", 1)\n",
      "Pair{AbstractString,Int64}(\"above. Time course\", 1)\n",
      "Pair{AbstractString,Int64}(\"be designed and\", 1)\n",
      "Pair{AbstractString,Int64}(\"transferred into a\", 1)\n",
      "Pair{AbstractString,Int64}(\"strains now submitted\", 1)\n",
      "Pair{AbstractString,Int64}(\"benzyl therefore\", 1)\n",
      "Pair{AbstractString,Int64}(\"in current-clamp\", 1)\n",
      "Pair{AbstractString,Int64}(\"target of\", 2)\n",
      "Pair{AbstractString,Int64}(\"± 0.26\", 1)\n",
      "Pair{AbstractString,Int64}(\"BetaCoV/Wuhan/ 2019\", 1)\n",
      "Pair{AbstractString,Int64}(\"µm thick\", 1)\n",
      "Pair{AbstractString,Int64}(\"mixed-effects model\", 1)\n",
      "Pair{AbstractString,Int64}(\"in enterovirus-infected cells\", 1)\n",
      "Pair{AbstractString,Int64}(\". For all\", 1)\n",
      "Pair{AbstractString,Int64}(\", compound 11\", 1)\n",
      "Pair{AbstractString,Int64}(\"test whether LY\", 2)\n",
      "Pair{AbstractString,Int64}(\"both coronavirus\", 1)\n",
      "Pair{AbstractString,Int64}(\"( SARS-CoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"800 fluorescence spectrophotometer\", 1)\n",
      "Pair{AbstractString,Int64}(\"dissolved in 10\", 2)\n",
      "Pair{AbstractString,Int64}(\"that five\", 1)\n",
      "Pair{AbstractString,Int64}(\"exhibited robust\", 1)\n",
      "Pair{AbstractString,Int64}(\"sufficient for HCoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"understanding\", 2)\n",
      "Pair{AbstractString,Int64}(\"the proline\", 1)\n",
      "Pair{AbstractString,Int64}(\"monitored at 460\", 1)\n",
      "Pair{AbstractString,Int64}(\"of DREADD receptors\", 1)\n",
      "Pair{AbstractString,Int64}(\"obtained\", 9)\n",
      "Pair{AbstractString,Int64}(\"and its Supplementary\", 1)\n",
      "Pair{AbstractString,Int64}(\"ab )\", 1)\n",
      "Pair{AbstractString,Int64}(\"OptiMEM. Cells were\", 1)\n",
      "Pair{AbstractString,Int64}(\"giving 1 as\", 1)\n",
      "Pair{AbstractString,Int64}(\"stirred for\", 5)\n",
      "Pair{AbstractString,Int64}(\"C terminus\", 1)\n",
      "Pair{AbstractString,Int64}(\"inactive ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"escape behavior upon\", 1)\n",
      "Pair{AbstractString,Int64}(\"that the HCoV-HKU\", 1)\n",
      "Pair{AbstractString,Int64}(\", a 7\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 genome was\", 1)\n",
      "Pair{AbstractString,Int64}(\"becomes undetectable by\", 1)\n",
      "Pair{AbstractString,Int64}(\"diet containing doxycycline.\", 1)\n",
      "Pair{AbstractString,Int64}(\"Fos-tagging from\", 1)\n",
      "Pair{AbstractString,Int64}(\"sterically more\", 1)\n",
      "Pair{AbstractString,Int64}(\"concentration were\", 1)\n",
      "Pair{AbstractString,Int64}(\", 176\", 1)\n",
      "Pair{AbstractString,Int64}(\"the catalytic Cys-His\", 1)\n",
      "Pair{AbstractString,Int64}(\"chloride ( 6.0\", 1)\n",
      "Pair{AbstractString,Int64}(\"lentivirus containing a\", 1)\n",
      "Pair{AbstractString,Int64}(\"doublet (\", 1)\n",
      "Pair{AbstractString,Int64}(\"different tissues may\", 1)\n",
      "Pair{AbstractString,Int64}(\"but is\", 2)\n",
      "Pair{AbstractString,Int64}(\"targeting both\", 1)\n",
      "Pair{AbstractString,Int64}(\"channels olfactometer\", 1)\n",
      "Pair{AbstractString,Int64}(\"in the piriform\", 1)\n",
      "Pair{AbstractString,Int64}(\"changes ( mean\", 1)\n",
      "Pair{AbstractString,Int64}(\"Furthermore , changes\", 1)\n",
      "Pair{AbstractString,Int64}(\"coronaviruses were\", 1)\n",
      "Pair{AbstractString,Int64}(\"in mice 24\", 1)\n",
      "Pair{AbstractString,Int64}(\"nsp\", 36)\n",
      "Pair{AbstractString,Int64}(\"a lid ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"as necessary\", 1)\n",
      "Pair{AbstractString,Int64}(\"Molecular formula\", 1)\n",
      "Pair{AbstractString,Int64}(\"neurons were\", 3)\n",
      "Pair{AbstractString,Int64}(\"were 183\", 1)\n",
      "Pair{AbstractString,Int64}(\"Stock\", 3)\n",
      "Pair{AbstractString,Int64}(\"exoribonuclease within\", 1)\n",
      "Pair{AbstractString,Int64}(\"x 38\", 1)\n",
      "Pair{AbstractString,Int64}(\"contained and\", 1)\n",
      "Pair{AbstractString,Int64}(\"bioRxiv preprint by\", 1)\n",
      "Pair{AbstractString,Int64}(\"left or right\", 2)\n",
      "Pair{AbstractString,Int64}(\"253\", 2)\n",
      "Pair{AbstractString,Int64}(\"( leucine )\", 3)\n",
      "Pair{AbstractString,Int64}(\"( 1.62\", 1)\n",
      "Pair{AbstractString,Int64}(\"cRPMI )\", 1)\n",
      "Pair{AbstractString,Int64}(\"increased replication\", 3)\n",
      "Pair{AbstractString,Int64}(\"SARS-CoV exclusively in\", 1)\n",
      "Pair{AbstractString,Int64}(\"neural circuit mechanisms\", 2)\n",
      "Pair{AbstractString,Int64}(\"of dN\", 1)\n",
      "Pair{AbstractString,Int64}(\", rupintrivir (\", 1)\n",
      "Pair{AbstractString,Int64}(\"Then the\", 4)\n",
      "Pair{AbstractString,Int64}(\"gel electrophoresis\", 2)\n",
      "Pair{AbstractString,Int64}(\"1 infection in\", 2)\n",
      "Pair{AbstractString,Int64}(\"gene during\", 1)\n",
      "Pair{AbstractString,Int64}(\"Mice learned to\", 1)\n",
      "Pair{AbstractString,Int64}(\"temperature.\", 1)\n",
      "Pair{AbstractString,Int64}(\"1990\", 1)\n",
      "Pair{AbstractString,Int64}(\"e , *\", 4)\n",
      "Pair{AbstractString,Int64}(\"or\", 147)\n",
      "Pair{AbstractString,Int64}(\"were silenced\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 10 )\", 2)\n",
      "Pair{AbstractString,Int64}(\"warheads other than\", 1)\n",
      "Pair{AbstractString,Int64}(\"* )\", 1)\n",
      "Pair{AbstractString,Int64}(\"C for 30\", 8)\n",
      "Pair{AbstractString,Int64}(\"CoVs lacking a\", 1)\n",
      "Pair{AbstractString,Int64}(\"or fitness\", 1)\n",
      "Pair{AbstractString,Int64}(\"at UTSW. Furthermore\", 1)\n",
      "Pair{AbstractString,Int64}(\"embraced\", 1)\n",
      "Pair{AbstractString,Int64}(\"walls\", 2)\n",
      "Pair{AbstractString,Int64}(\"November 30 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"minutes also displayed\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.05 ) in\", 1)\n",
      "Pair{AbstractString,Int64}(\"µM against HCoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"we did\", 2)\n",
      "Pair{AbstractString,Int64}(\"DREADD-expressing piriform neurons\", 1)\n",
      "Pair{AbstractString,Int64}(\"of column collected\", 1)\n",
      "Pair{AbstractString,Int64}(\"219 not shown\", 1)\n",
      "Pair{AbstractString,Int64}(\"equilibrated to\", 1)\n",
      "Pair{AbstractString,Int64}(\"EV-A 71 though\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"were significantly\", 2)\n",
      "Pair{AbstractString,Int64}(\"culture the virus\", 1)\n",
      "Pair{AbstractString,Int64}(\"and SNU 01\", 1)\n",
      "Pair{AbstractString,Int64}(\"singlet ( s\", 1)\n",
      "Pair{AbstractString,Int64}(\"entry mediated\", 1)\n",
      "Pair{AbstractString,Int64}(\"control group that\", 2)\n",
      "Pair{AbstractString,Int64}(\"stem cell (\", 1)\n",
      "Pair{AbstractString,Int64}(\"in exploratory behavior\", 4)\n",
      "Pair{AbstractString,Int64}(\"a glutamine residue\", 1)\n",
      "Pair{AbstractString,Int64}(\"spike glycoprotein\", 1)\n",
      "Pair{AbstractString,Int64}(\"25 minutes (\", 1)\n",
      "Pair{AbstractString,Int64}(\"configuration ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"4.36 ( 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"mean RPKM >\", 1)\n",
      "Pair{AbstractString,Int64}(\"in case\", 4)\n",
      "Pair{AbstractString,Int64}(\"higher against most\", 1)\n",
      "Pair{AbstractString,Int64}(\", the mean\", 2)\n",
      "Pair{AbstractString,Int64}(\"0.0004 µM (\", 1)\n",
      "Pair{AbstractString,Int64}(\"GC- 375\", 1)\n",
      "Pair{AbstractString,Int64}(\"a AlexaFluor\", 1)\n",
      "Pair{AbstractString,Int64}(\"IFIH 1 Huh\", 1)\n",
      "Pair{AbstractString,Int64}(\"fundamental\", 1)\n",
      "Pair{AbstractString,Int64}(\"selection with\", 1)\n",
      "Pair{AbstractString,Int64}(\"Enteroviruses are small\", 1)\n",
      "Pair{AbstractString,Int64}(\"promoter drives expression\", 1)\n",
      "Pair{AbstractString,Int64}(\"ment\", 1)\n",
      "Pair{AbstractString,Int64}(\"prediction , we\", 2)\n",
      "Pair{AbstractString,Int64}(\"same compound\", 1)\n",
      "Pair{AbstractString,Int64}(\"nearly equipotent 11\", 1)\n",
      "Pair{AbstractString,Int64}(\"pQCXIP plasmid that\", 1)\n",
      "Pair{AbstractString,Int64}(\"neutral environment.\", 1)\n",
      "Pair{AbstractString,Int64}(\"12 -hour\", 2)\n",
      "Pair{AbstractString,Int64}(\"M pro for\", 2)\n",
      "Pair{AbstractString,Int64}(\"5 days at\", 1)\n",
      "Pair{AbstractString,Int64}(\", resulting in\", 4)\n",
      "Pair{AbstractString,Int64}(\"slope. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"about 35\", 1)\n",
      "Pair{AbstractString,Int64}(\". Three findings\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Tayler\", 1)\n",
      "Pair{AbstractString,Int64}(\"DNA loading buffer\", 1)\n",
      "Pair{AbstractString,Int64}(\"of a white\", 1)\n",
      "Pair{AbstractString,Int64}(\"replicons ,\", 4)\n",
      "Pair{AbstractString,Int64}(\"0:72 µm )\", 1)\n",
      "Pair{AbstractString,Int64}(\"large to\", 1)\n",
      "Pair{AbstractString,Int64}(\"obviously too long\", 1)\n",
      "Pair{AbstractString,Int64}(\"split luciferase (\", 1)\n",
      "Pair{AbstractString,Int64}(\"can retrieve previously\", 1)\n",
      "Pair{AbstractString,Int64}(\"the overall mobility\", 1)\n",
      "Pair{AbstractString,Int64}(\"left undisturbed\", 1)\n",
      "Pair{AbstractString,Int64}(\"were largely\", 1)\n",
      "Pair{AbstractString,Int64}(\"Silencing\", 1)\n",
      "Pair{AbstractString,Int64}(\"prepare the samples\", 1)\n",
      "Pair{AbstractString,Int64}(\"it has\", 4)\n",
      "Pair{AbstractString,Int64}(\"series\", 5)\n",
      "Pair{AbstractString,Int64}(\"= 1.08 µM\", 1)\n",
      "Pair{AbstractString,Int64}(\"Disease\", 2)\n",
      "Pair{AbstractString,Int64}(\"1.1 ( 0.3\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Alexander et\", 1)\n",
      "Pair{AbstractString,Int64}(\"flask was\", 1)\n",
      "Pair{AbstractString,Int64}(\"with lentiviruses\", 1)\n",
      "Pair{AbstractString,Int64}(\"STAT\", 2)\n",
      "Pair{AbstractString,Int64}(\"' = benzyl\", 1)\n",
      "Pair{AbstractString,Int64}(\"tested (\", 1)\n",
      "Pair{AbstractString,Int64}(\"serine protease\", 1)\n",
      "Pair{AbstractString,Int64}(\"Foot , and\", 1)\n",
      "Pair{AbstractString,Int64}(\"with isocyanides\", 1)\n",
      "Pair{AbstractString,Int64}(\"KO cells and\", 1)\n",
      "Pair{AbstractString,Int64}(\"resembles the\", 1)\n",
      "Pair{AbstractString,Int64}(\"by shaking\", 1)\n",
      "Pair{AbstractString,Int64}(\"with an MOI\", 1)\n",
      "Pair{AbstractString,Int64}(\"the cellular and\", 1)\n",
      "Pair{AbstractString,Int64}(\"C , then\", 2)\n",
      "Pair{AbstractString,Int64}(\"primers containing\", 1)\n",
      "Pair{AbstractString,Int64}(\"We next addressed\", 1)\n",
      "Pair{AbstractString,Int64}(\"Leuven and\", 1)\n",
      "Pair{AbstractString,Int64}(\"use specific KLK\", 1)\n",
      "Pair{AbstractString,Int64}(\"constructs 358\", 1)\n",
      "Pair{AbstractString,Int64}(\"with 2 µl\", 1)\n",
      "Pair{AbstractString,Int64}(\"Ciclesonide exhibited\", 1)\n",
      "Pair{AbstractString,Int64}(\"protease is designated\", 1)\n",
      "Pair{AbstractString,Int64}(\"observe likely reflect\", 1)\n",
      "Pair{AbstractString,Int64}(\"of magnitude lower\", 1)\n",
      "Pair{AbstractString,Int64}(\"3.7\", 2)\n",
      "Pair{AbstractString,Int64}(\"speed\", 2)\n",
      "Pair{AbstractString,Int64}(\"Dq-positive cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"player in the\", 1)\n",
      "Pair{AbstractString,Int64}(\"The nucleic acid\", 1)\n",
      "Pair{AbstractString,Int64}(\"its possible animal\", 1)\n",
      "Pair{AbstractString,Int64}(\"gene of HCoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"value ) ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 substitutent\", 1)\n",
      "Pair{AbstractString,Int64}(\"pentanedioate (\", 1)\n",
      "Pair{AbstractString,Int64}(\"that learned\", 1)\n",
      "Pair{AbstractString,Int64}(\"they are\", 1)\n",
      "Pair{AbstractString,Int64}(\"was detected upon\", 1)\n",
      "Pair{AbstractString,Int64}(\"found the\", 3)\n",
      "Pair{AbstractString,Int64}(\"Camostat\", 1)\n",
      "Pair{AbstractString,Int64}(\"been difficult\", 1)\n",
      "Pair{AbstractString,Int64}(\"( cf.\", 1)\n",
      "Pair{AbstractString,Int64}(\"' , although\", 1)\n",
      "Pair{AbstractString,Int64}(\"genus Enterovirus in\", 1)\n",
      "Pair{AbstractString,Int64}(\"expand\", 1)\n",
      "Pair{AbstractString,Int64}(\"nucleoside\", 8)\n",
      "Pair{AbstractString,Int64}(\"these neurons may\", 1)\n",
      "Pair{AbstractString,Int64}(\"1.5 log\", 1)\n",
      "Pair{AbstractString,Int64}(\"consensus\", 2)\n",
      "Pair{AbstractString,Int64}(\"within group\", 1)\n",
      "Pair{AbstractString,Int64}(\", including in\", 1)\n",
      "Pair{AbstractString,Int64}(\"and selective pressure\", 1)\n",
      "Pair{AbstractString,Int64}(\"filled with\", 1)\n",
      "Pair{AbstractString,Int64}(\"different from CVB\", 1)\n",
      "Pair{AbstractString,Int64}(\"peptidomimetic\", 4)\n",
      "Pair{AbstractString,Int64}(\"1 ab )\", 1)\n",
      "Pair{AbstractString,Int64}(\"] -\", 1)\n",
      "Pair{AbstractString,Int64}(\"The quantity\", 1)\n",
      "Pair{AbstractString,Int64}(\": C mutations.\", 1)\n",
      "Pair{AbstractString,Int64}(\"http://www.phylogeny.fr/ )\", 1)\n",
      "Pair{AbstractString,Int64}(\"removed and\", 3)\n",
      "Pair{AbstractString,Int64}(\"CS-as well as\", 1)\n",
      "Pair{AbstractString,Int64}(\"a-e\", 1)\n",
      "Pair{AbstractString,Int64}(\"are faster than\", 1)\n",
      "Pair{AbstractString,Int64}(\", has\", 2)\n",
      "Pair{AbstractString,Int64}(\"( SARS-CoV- 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"indicated by\", 4)\n",
      "Pair{AbstractString,Int64}(\"recommendations ( Invitrogen\", 1)\n",
      "Pair{AbstractString,Int64}(\"explaining\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibitor accepts\", 1)\n",
      "Pair{AbstractString,Int64}(\"stimulus onset (\", 1)\n",
      "Pair{AbstractString,Int64}(\"defects in\", 1)\n",
      "Pair{AbstractString,Int64}(\"cryo-protectant solution\", 1)\n",
      "Pair{AbstractString,Int64}(\"374\", 1)\n",
      "Pair{AbstractString,Int64}(\"to bind\", 1)\n",
      "Pair{AbstractString,Int64}(\", 4\", 5)\n",
      "Pair{AbstractString,Int64}(\"340 results ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"timely and accurately\", 1)\n",
      "Pair{AbstractString,Int64}(\"shows\", 2)\n",
      "Pair{AbstractString,Int64}(\") and in\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 90\", 2)\n",
      "Pair{AbstractString,Int64}(\"during ambient\", 1)\n",
      "Pair{AbstractString,Int64}(\"Gilead Sciences ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"was initiated\", 1)\n",
      "Pair{AbstractString,Int64}(\"also for Michael\", 1)\n",
      "Pair{AbstractString,Int64}(\"impacted by\", 1)\n",
      "Pair{AbstractString,Int64}(\"adducts. In addition\", 1)\n",
      "Pair{AbstractString,Int64}(\"grown in\", 3)\n",
      "Pair{AbstractString,Int64}(\"or Huh-T\", 1)\n",
      "Pair{AbstractString,Int64}(\"E ) .\", 3)\n",
      "Pair{AbstractString,Int64}(\"poor ( EC\", 3)\n",
      "Pair{AbstractString,Int64}(\"detected no\", 1)\n",
      "Pair{AbstractString,Int64}(\"P , 21,291\", 1)\n",
      "Pair{AbstractString,Int64}(\"Receptor recognition\", 1)\n",
      "Pair{AbstractString,Int64}(\"preprint MOI\", 1)\n",
      "Pair{AbstractString,Int64}(\"later , changes\", 1)\n",
      "Pair{AbstractString,Int64}(\"which has satisfactory\", 1)\n",
      "Pair{AbstractString,Int64}(\"all compounds\", 2)\n",
      "Pair{AbstractString,Int64}(\"sensitivity was\", 1)\n",
      "Pair{AbstractString,Int64}(\"Stevens\", 1)\n",
      "Pair{AbstractString,Int64}(\"migration speed\", 1)\n",
      "Pair{AbstractString,Int64}(\"UV absorption\", 1)\n",
      "Pair{AbstractString,Int64}(\"of an olfactory\", 2)\n",
      "Pair{AbstractString,Int64}(\"form the side-walls\", 1)\n",
      "Pair{AbstractString,Int64}(\"strategy for live-attenuated\", 1)\n",
      "Pair{AbstractString,Int64}(\"with CR-LY 6\", 1)\n",
      "Pair{AbstractString,Int64}(\"Purity was\", 1)\n",
      "Pair{AbstractString,Int64}(\"superior to\", 1)\n",
      "Pair{AbstractString,Int64}(\"Exclusively\", 1)\n",
      "Pair{AbstractString,Int64}(\"culture and\", 2)\n",
      "Pair{AbstractString,Int64}(\"inhibitor is\", 1)\n",
      "Pair{AbstractString,Int64}(\"to enter\", 2)\n",
      "Pair{AbstractString,Int64}(\"e-g )\", 1)\n",
      "Pair{AbstractString,Int64}(\"we speculated that\", 1)\n",
      "Pair{AbstractString,Int64}(\"protect mice from\", 1)\n",
      "Pair{AbstractString,Int64}(\"analysis. Normalized data\", 1)\n",
      "Pair{AbstractString,Int64}(\"variants ( Fig.\", 1)\n",
      "Pair{AbstractString,Int64}(\"were valid for\", 1)\n",
      "Pair{AbstractString,Int64}(\"conceivably\", 1)\n",
      "Pair{AbstractString,Int64}(\"( i.e. AA-to-ED\", 1)\n",
      "Pair{AbstractString,Int64}(\"range.\", 1)\n",
      "Pair{AbstractString,Int64}(\"Company , Ltd.\", 1)\n",
      "Pair{AbstractString,Int64}(\"- 2.14\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 infection\", 2)\n",
      "Pair{AbstractString,Int64}(\"bat-derived SARS-like coronaviruses\", 1)\n",
      "Pair{AbstractString,Int64}(\"attempts , 135\", 1)\n",
      "Pair{AbstractString,Int64}(\"luciferase )\", 1)\n",
      "Pair{AbstractString,Int64}(\"µm ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\"cortex , a\", 1)\n",
      "Pair{AbstractString,Int64}(\"activation z-score. Numbers\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibitors for 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"The complex\", 2)\n",
      "Pair{AbstractString,Int64}(\"28 29\", 4)\n",
      "Pair{AbstractString,Int64}(\"still a\", 1)\n",
      "Pair{AbstractString,Int64}(\"natural selection\", 5)\n",
      "Pair{AbstractString,Int64}(\"formed by\", 6)\n",
      "Pair{AbstractString,Int64}(\"each CS- ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"477 per\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"right )\", 1)\n",
      "Pair{AbstractString,Int64}(\"the KingFisher\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Schoenenberger\", 1)\n",
      "Pair{AbstractString,Int64}(\"three peptidyl Michael\", 1)\n",
      "Pair{AbstractString,Int64}(\"and found that\", 1)\n",
      "Pair{AbstractString,Int64}(\"propagation competent\", 1)\n",
      "Pair{AbstractString,Int64}(\"running towards the\", 1)\n",
      "Pair{AbstractString,Int64}(\"protein undergoes several\", 1)\n",
      "Pair{AbstractString,Int64}(\"; thus ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"CNO injection was\", 1)\n",
      "Pair{AbstractString,Int64}(\"The resulting dark\", 1)\n",
      "Pair{AbstractString,Int64}(\"is the most\", 2)\n",
      "Pair{AbstractString,Int64}(\"( HFMD )\", 1)\n",
      "Pair{AbstractString,Int64}(\"tried was cyclopropylmethyl\", 1)\n",
      "Pair{AbstractString,Int64}(\"propanoate ( 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"making hydrophobic\", 1)\n",
      "Pair{AbstractString,Int64}(\"conservative substitutions\", 1)\n",
      "Pair{AbstractString,Int64}(\"; Tan et\", 2)\n",
      "Pair{AbstractString,Int64}(\"was similarly observed\", 1)\n",
      "Pair{AbstractString,Int64}(\"by ( Srinivasan\", 1)\n",
      "Pair{AbstractString,Int64}(\"between Pro 189\", 1)\n",
      "Pair{AbstractString,Int64}(\"conformational change is\", 1)\n",
      "Pair{AbstractString,Int64}(\"n= 13 ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"MHV-nsp\", 1)\n",
      "Pair{AbstractString,Int64}(\"one doubt the\", 1)\n",
      "Pair{AbstractString,Int64}(\"to non-specific inhibition\", 1)\n",
      "Pair{AbstractString,Int64}(\"group was\", 1)\n",
      "Pair{AbstractString,Int64}(\"M MES ,\", 2)\n",
      "Pair{AbstractString,Int64}(\") , macrophages\", 1)\n",
      "Pair{AbstractString,Int64}(\") is much\", 1)\n",
      "Pair{AbstractString,Int64}(\"defects , selection\", 1)\n",
      "Pair{AbstractString,Int64}(\"Figure 2 F\", 1)\n",
      "Pair{AbstractString,Int64}(\"this trend persists\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 substitutent of\", 1)\n",
      "Pair{AbstractString,Int64}(\"63 M\", 39)\n",
      "Pair{AbstractString,Int64}(\"Thr 47\", 1)\n",
      "Pair{AbstractString,Int64}(\"I ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.05 were\", 1)\n",
      "Pair{AbstractString,Int64}(\"were 293 nucleotideswere\", 1)\n",
      "Pair{AbstractString,Int64}(\"shown with carbon\", 1)\n",
      "Pair{AbstractString,Int64}(\"information cannot be\", 1)\n",
      "Pair{AbstractString,Int64}(\", in which\", 4)\n",
      "Pair{AbstractString,Int64}(\"the nucleophilic\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibitor appeared\", 1)\n",
      "Pair{AbstractString,Int64}(\"purifying selection\", 1)\n",
      "Pair{AbstractString,Int64}(\"the host immune\", 1)\n",
      "Pair{AbstractString,Int64}(\"is not compatible\", 1)\n",
      "Pair{AbstractString,Int64}(\") . Alternatively\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 x 10\", 1)\n",
      "Pair{AbstractString,Int64}(\"the protease.\", 1)\n",
      "Pair{AbstractString,Int64}(\"Celite to\", 1)\n",
      "Pair{AbstractString,Int64}(\"previously. 48\", 1)\n",
      "Pair{AbstractString,Int64}(\"Dulbecco '\", 1)\n",
      "Pair{AbstractString,Int64}(\"threat posed by\", 1)\n",
      "Pair{AbstractString,Int64}(\"E substitution was\", 1)\n",
      "Pair{AbstractString,Int64}(\"cleavage site of\", 1)\n",
      "Pair{AbstractString,Int64}(\"classical CysHisGlu/Asp catalytic\", 1)\n",
      "Pair{AbstractString,Int64}(\"partially compensate\", 1)\n",
      "Pair{AbstractString,Int64}(\"crystal with compound\", 1)\n",
      "Pair{AbstractString,Int64}(\"adjustment using\", 1)\n",
      "Pair{AbstractString,Int64}(\"suggests that behaviorally\", 1)\n",
      "Pair{AbstractString,Int64}(\"* p= 0.0234\", 1)\n",
      "Pair{AbstractString,Int64}(\"GAA GCA GCT\", 1)\n",
      "Pair{AbstractString,Int64}(\"Male mice were\", 1)\n",
      "Pair{AbstractString,Int64}(\"myocarditis ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"and spleen HCoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 Dq )\", 1)\n",
      "Pair{AbstractString,Int64}(\"bias of\", 2)\n",
      "Pair{AbstractString,Int64}(\", l-m )\", 2)\n",
      "Pair{AbstractString,Int64}(\"-sensitive dye Fura-\", 1)\n",
      "Pair{AbstractString,Int64}(\"( ORFs )\", 1)\n",
      "Pair{AbstractString,Int64}(\"controlexpressing\", 1)\n",
      "Pair{AbstractString,Int64}(\"14 carried their\", 1)\n",
      "Pair{AbstractString,Int64}(\"carried out.\", 1)\n",
      "Pair{AbstractString,Int64}(\"in hM 3\", 6)\n",
      "Pair{AbstractString,Int64}(\"( MEME\", 1)\n",
      "Pair{AbstractString,Int64}(\"01\", 2)\n",
      "Pair{AbstractString,Int64}(\"¢ -phosphorylated using\", 1)\n",
      "Pair{AbstractString,Int64}(\"nucleophilic addition with\", 1)\n",
      "Pair{AbstractString,Int64}(\"which allows\", 2)\n",
      "Pair{AbstractString,Int64}(\"positive selection at\", 1)\n",
      "Pair{AbstractString,Int64}(\"samples incubated at\", 1)\n",
      "Pair{AbstractString,Int64}(\"significant increase over\", 1)\n",
      "Pair{AbstractString,Int64}(\"74 mg ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"' benzyl\", 1)\n",
      "Pair{AbstractString,Int64}(\"both from the\", 1)\n",
      "Pair{AbstractString,Int64}(\"; Tan\", 2)\n",
      "Pair{AbstractString,Int64}(\"animal (\", 1)\n",
      "Pair{AbstractString,Int64}(\"suspected patients\", 1)\n",
      "Pair{AbstractString,Int64}(\"custom-made\", 2)\n",
      "Pair{AbstractString,Int64}(\"coronavirus.\", 1)\n",
      "Pair{AbstractString,Int64}(\"consistent with previous\", 1)\n",
      "Pair{AbstractString,Int64}(\"next\", 16)\n",
      "Pair{AbstractString,Int64}(\"the hallmark\", 1)\n",
      "Pair{AbstractString,Int64}(\"indicate that restriction\", 1)\n",
      "Pair{AbstractString,Int64}(\"The DNA-launched SARS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"2.5 min\", 1)\n",
      "Pair{AbstractString,Int64}(\"Consistent with\", 5)\n",
      "Pair{AbstractString,Int64}(\". 1 M\", 2)\n",
      "Pair{AbstractString,Int64}(\"1.08 µM\", 1)\n",
      "Pair{AbstractString,Int64}(\"; fold change\", 1)\n",
      "Pair{AbstractString,Int64}(\"2.1 µM\", 2)\n",
      "Pair{AbstractString,Int64}(\"its classification\", 1)\n",
      "Pair{AbstractString,Int64}(\"of DREADDs in\", 1)\n",
      "Pair{AbstractString,Int64}(\"as catalyzed by\", 1)\n",
      "Pair{AbstractString,Int64}(\"learning were\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 366 nt\", 1)\n",
      "Pair{AbstractString,Int64}(\"in water or\", 1)\n",
      "Pair{AbstractString,Int64}(\"Fig. 3 B\", 1)\n",
      "Pair{AbstractString,Int64}(\"; they are\", 1)\n",
      "Pair{AbstractString,Int64}(\", 2 µM\", 1)\n",
      "Pair{AbstractString,Int64}(\"renifor ) luciferase\", 1)\n",
      "Pair{AbstractString,Int64}(\"for which the\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 was generated\", 1)\n",
      "Pair{AbstractString,Int64}(\"/ /doi.org/ 10.1101\", 1)\n",
      "Pair{AbstractString,Int64}(\"us (\", 1)\n",
      "Pair{AbstractString,Int64}(\"minutes was increased\", 1)\n",
      "Pair{AbstractString,Int64}(\"showed that in\", 1)\n",
      "Pair{AbstractString,Int64}(\", 21,291 nt\", 1)\n",
      "Pair{AbstractString,Int64}(\"virus infection. Therefore\", 1)\n",
      "Pair{AbstractString,Int64}(\"pro in complex\", 5)\n",
      "Pair{AbstractString,Int64}(\". 11\", 2)\n",
      "Pair{AbstractString,Int64}(\"removed using\", 1)\n",
      "Pair{AbstractString,Int64}(\"February\", 3)\n",
      "Pair{AbstractString,Int64}(\"and 0.08 µM\", 1)\n",
      "Pair{AbstractString,Int64}(\"to visualize its\", 1)\n",
      "Pair{AbstractString,Int64}(\"sample even\", 1)\n",
      "Pair{AbstractString,Int64}(\"same ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"Cloning\", 1)\n",
      "Pair{AbstractString,Int64}(\"prone\", 3)\n",
      "Pair{AbstractString,Int64}(\"over the entire\", 1)\n",
      "Pair{AbstractString,Int64}(\"Stable\", 2)\n",
      "Pair{AbstractString,Int64}(\"E ( CR\", 2)\n",
      "Pair{AbstractString,Int64}(\"The olfactory\", 1)\n",
      "Pair{AbstractString,Int64}(\". From\", 2)\n",
      "Pair{AbstractString,Int64}(\") T-ZNA-\", 1)\n",
      "Pair{AbstractString,Int64}(\"with lentiviral\", 2)\n",
      "Pair{AbstractString,Int64}(\"to each well.\", 1)\n",
      "Pair{AbstractString,Int64}(\"then confirmed that\", 1)\n",
      "Pair{AbstractString,Int64}(\"protein of HCoV-HKU\", 1)\n",
      "Pair{AbstractString,Int64}(\"under a Scientifica\", 1)\n",
      "Pair{AbstractString,Int64}(\"as negative and\", 1)\n",
      "Pair{AbstractString,Int64}(\"212\", 1)\n",
      "Pair{AbstractString,Int64}(\"amount of the\", 3)\n",
      "Pair{AbstractString,Int64}(\", peptidomimetic inhibitors\", 1)\n",
      "Pair{AbstractString,Int64}(\"pockets\", 2)\n",
      "Pair{AbstractString,Int64}(\"behavioral protocol to\", 1)\n",
      "Pair{AbstractString,Int64}(\"µM in Huh\", 2)\n",
      "Pair{AbstractString,Int64}(\"mL dichloromethane (\", 1)\n",
      "Pair{AbstractString,Int64}(\"Piriform cortex has\", 1)\n",
      "Pair{AbstractString,Int64}(\"503 over common\", 1)\n",
      "Pair{AbstractString,Int64}(\"with antibodies\", 3)\n",
      "Pair{AbstractString,Int64}(\"RD cells.\", 2)\n",
      "Pair{AbstractString,Int64}(\"results of the\", 1)\n",
      "Pair{AbstractString,Int64}(\"box. Materials\", 1)\n",
      "Pair{AbstractString,Int64}(\"Accordingly\", 1)\n",
      "Pair{AbstractString,Int64}(\"To ensure that\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 I\", 1)\n",
      "Pair{AbstractString,Int64}(\"population that\", 1)\n",
      "Pair{AbstractString,Int64}(\"to form\", 1)\n",
      "Pair{AbstractString,Int64}(\"impaired in cFos-tTA\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 ZYD ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"determine whether loss\", 1)\n",
      "Pair{AbstractString,Int64}(\"( NSP 15\", 1)\n",
      "Pair{AbstractString,Int64}(\"memory in the\", 1)\n",
      "Pair{AbstractString,Int64}(\"11 12 13\", 4)\n",
      "Pair{AbstractString,Int64}(\"an Amicon\", 1)\n",
      "Pair{AbstractString,Int64}(\"causes sub-lethal\", 1)\n",
      "Pair{AbstractString,Int64}(\") and washed\", 1)\n",
      "Pair{AbstractString,Int64}(\"the Non-Destructive Cytotoxicity\", 1)\n",
      "Pair{AbstractString,Int64}(\"to GISAID database\", 1)\n",
      "Pair{AbstractString,Int64}(\"transitions. Consequently\", 1)\n",
      "Pair{AbstractString,Int64}(\", relative synonymous\", 1)\n",
      "Pair{AbstractString,Int64}(\"performed with\", 2)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"during recovery ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"separated from\", 1)\n",
      "Pair{AbstractString,Int64}(\"plate luminometer\", 1)\n",
      "Pair{AbstractString,Int64}(\"control and split\", 1)\n",
      "Pair{AbstractString,Int64}(\"x previously\", 1)\n",
      "Pair{AbstractString,Int64}(\"that despite\", 2)\n",
      "Pair{AbstractString,Int64}(\"site was efficiently\", 1)\n",
      "Pair{AbstractString,Int64}(\"comprehensive\", 2)\n",
      "Pair{AbstractString,Int64}(\"group were analyzed\", 1)\n",
      "Pair{AbstractString,Int64}(\". Following this\", 1)\n",
      "Pair{AbstractString,Int64}(\"co-cultured\", 2)\n",
      "Pair{AbstractString,Int64}(\"improved albeit still\", 1)\n",
      "Pair{AbstractString,Int64}(\"KLK-kinin\", 1)\n",
      "Pair{AbstractString,Int64}(\"TriM Scope II\", 1)\n",
      "Pair{AbstractString,Int64}(\". Compound 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"and 0.91 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"frequency of\", 1)\n",
      "Pair{AbstractString,Int64}(\"a larger population\", 1)\n",
      "Pair{AbstractString,Int64}(\"this result\", 2)\n",
      "Pair{AbstractString,Int64}(\") and transfected\", 1)\n",
      "Pair{AbstractString,Int64}(\"matched-pairs signed rank\", 11)\n",
      "Pair{AbstractString,Int64}(\"cyclohexylmethyl derivative\", 1)\n",
      "Pair{AbstractString,Int64}(\"ExoN-DE background.\", 1)\n",
      "Pair{AbstractString,Int64}(\"YNA ) ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"2019 ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"sex (\", 2)\n",
      "Pair{AbstractString,Int64}(\"by a normal\", 1)\n",
      "Pair{AbstractString,Int64}(\"reflect\", 3)\n",
      "Pair{AbstractString,Int64}(\"evolution is an\", 1)\n",
      "Pair{AbstractString,Int64}(\"ExoN- 128 AA\", 1)\n",
      "Pair{AbstractString,Int64}(\"of MERS\", 1)\n",
      "Pair{AbstractString,Int64}(\"20 representative coronaviruses\", 1)\n",
      "Pair{AbstractString,Int64}(\"to control\", 2)\n",
      "Pair{AbstractString,Int64}(\"interface. Apical\", 1)\n",
      "Pair{AbstractString,Int64}(\"maturation ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"generate a\", 2)\n",
      "Pair{AbstractString,Int64}(\"at discovering peptidomimetic\", 1)\n",
      "Pair{AbstractString,Int64}(\"the reason for\", 1)\n",
      "Pair{AbstractString,Int64}(\"surface and endosomal\", 1)\n",
      "Pair{AbstractString,Int64}(\", a metal-coordinating\", 1)\n",
      "Pair{AbstractString,Int64}(\"x water-immersion\", 1)\n",
      "Pair{AbstractString,Int64}(\"cFos-tTA system is\", 1)\n",
      "Pair{AbstractString,Int64}(\"Chinese Society\", 1)\n",
      "Pair{AbstractString,Int64}(\", where 186\", 1)\n",
      "Pair{AbstractString,Int64}(\"genus ( HCoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"effect was to\", 1)\n",
      "Pair{AbstractString,Int64}(\"did not abolish\", 1)\n",
      "Pair{AbstractString,Int64}(\"such as Gly\", 1)\n",
      "Pair{AbstractString,Int64}(\"ExoN mutants suggest\", 1)\n",
      "Pair{AbstractString,Int64}(\"a CS-compromises the\", 1)\n",
      "Pair{AbstractString,Int64}(\"studied extensively\", 1)\n",
      "Pair{AbstractString,Int64}(\"passages. These 147\", 1)\n",
      "Pair{AbstractString,Int64}(\"Li et al.\", 1)\n",
      "Pair{AbstractString,Int64}(\"and memory recall\", 1)\n",
      "Pair{AbstractString,Int64}(\"plate one\", 1)\n",
      "Pair{AbstractString,Int64}(\"as well as\", 21)\n",
      "Pair{AbstractString,Int64}(\"to the manufacturers\", 1)\n",
      "Pair{AbstractString,Int64}(\"infection conditions\", 1)\n",
      "Pair{AbstractString,Int64}(\"by syncytia was\", 1)\n",
      "Pair{AbstractString,Int64}(\"be placed in\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 reference genome\", 1)\n",
      "Pair{AbstractString,Int64}(\"resonance\", 1)\n",
      "Pair{AbstractString,Int64}(\"59\", 10)\n",
      "Pair{AbstractString,Int64}(\"makes the revertant\", 1)\n",
      "Pair{AbstractString,Int64}(\"ethyl-acetate )\", 1)\n",
      "Pair{AbstractString,Int64}(\"in Huh-T\", 3)\n",
      "Pair{AbstractString,Int64}(\"cortex. Ten\", 1)\n",
      "Pair{AbstractString,Int64}(\"after CS+ presentation\", 4)\n",
      "Pair{AbstractString,Int64}(\"90.7 % )\", 1)\n",
      "Pair{AbstractString,Int64}(\"and used to\", 1)\n",
      "Pair{AbstractString,Int64}(\"et al. 282\", 1)\n",
      "Pair{AbstractString,Int64}(\"% [\", 1)\n",
      "Pair{AbstractString,Int64}(\") for reversion\", 1)\n",
      "Pair{AbstractString,Int64}(\"Wilcoxon Signed Ranks\", 6)\n",
      "Pair{AbstractString,Int64}(\"low\", 15)\n",
      "Pair{AbstractString,Int64}(\"free-enzyme structure (\", 1)\n",
      "Pair{AbstractString,Int64}(\"due to low\", 1)\n",
      "Pair{AbstractString,Int64}(\"U= 24\", 1)\n",
      "Pair{AbstractString,Int64}(\"Lipofectamin\", 2)\n",
      "Pair{AbstractString,Int64}(\"both coronavirus replication\", 1)\n",
      "Pair{AbstractString,Int64}(\"have different\", 1)\n",
      "Pair{AbstractString,Int64}(\"contextual memory in\", 1)\n",
      "Pair{AbstractString,Int64}(\"for HCoV-\", 3)\n",
      "Pair{AbstractString,Int64}(\"restricted (\", 1)\n",
      "Pair{AbstractString,Int64}(\"29 In brief\", 1)\n",
      "Pair{AbstractString,Int64}(\"6 cells for\", 1)\n",
      "Pair{AbstractString,Int64}(\"assay , BHK-\", 1)\n",
      "Pair{AbstractString,Int64}(\"C. Then 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"6.02 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"- 15\", 2)\n",
      "Pair{AbstractString,Int64}(\"memory ( Basu\", 1)\n",
      "Pair{AbstractString,Int64}(\"proteins in\", 1)\n",
      "Pair{AbstractString,Int64}(\"E promotes entry\", 1)\n",
      "Pair{AbstractString,Int64}(\"different between\", 1)\n",
      "Pair{AbstractString,Int64}(\"primary reversion is\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.9892 . author/funder.\", 1)\n",
      "Pair{AbstractString,Int64}(\", 3.5 mmol\", 1)\n",
      "Pair{AbstractString,Int64}(\"calculated. HCoV-OC\", 1)\n",
      "Pair{AbstractString,Int64}(\"A fragment\", 1)\n",
      "Pair{AbstractString,Int64}(\"Energy of\", 1)\n",
      "Pair{AbstractString,Int64}(\"U= 37\", 1)\n",
      "Pair{AbstractString,Int64}(\"43 : 5.0\", 1)\n",
      "Pair{AbstractString,Int64}(\"p= 0.3282 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"Calu et\", 2)\n",
      "Pair{AbstractString,Int64}(\"Five days\", 1)\n",
      "Pair{AbstractString,Int64}(\"virus detection.\", 1)\n",
      "Pair{AbstractString,Int64}(\"E translation ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"cellular proteases and\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.03 ) in\", 2)\n",
      "Pair{AbstractString,Int64}(\"or empty\", 2)\n",
      "Pair{AbstractString,Int64}(\"six-membered d-lactam in\", 1)\n",
      "Pair{AbstractString,Int64}(\"/doi.org/ 10\", 1)\n",
      "Pair{AbstractString,Int64}(\"to SARS-CoV M\", 1)\n",
      "Pair{AbstractString,Int64}(\"spike-P 250\", 1)\n",
      "Pair{AbstractString,Int64}(\"Previous\", 3)\n",
      "Pair{AbstractString,Int64}(\"with DAPI. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"induction and the\", 1)\n",
      "Pair{AbstractString,Int64}(\"not tested in\", 1)\n",
      "Pair{AbstractString,Int64}(\"synthesis.\", 2)\n",
      "Pair{AbstractString,Int64}(\"min inter-trial interval.\", 1)\n",
      "Pair{AbstractString,Int64}(\"* p= 3.946\", 1)\n",
      "Pair{AbstractString,Int64}(\"of Ala\", 3)\n",
      "Pair{AbstractString,Int64}(\"of interest were\", 1)\n",
      "Pair{AbstractString,Int64}(\"8.5 , and\", 2)\n",
      "Pair{AbstractString,Int64}(\"l , *\", 1)\n",
      "Pair{AbstractString,Int64}(\"Artificial\", 1)\n",
      "Pair{AbstractString,Int64}(\"enthalpy.\", 1)\n",
      "Pair{AbstractString,Int64}(\", with foot\", 1)\n",
      "Pair{AbstractString,Int64}(\"a series of\", 3)\n",
      "Pair{AbstractString,Int64}(\"e fl/fl\", 8)\n",
      "Pair{AbstractString,Int64}(\"activates\", 1)\n",
      "Pair{AbstractString,Int64}(\"8 , W=-\", 2)\n",
      "Pair{AbstractString,Int64}(\"HEPES ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"slice of\", 1)\n",
      "Pair{AbstractString,Int64}(\"antiviral compounds.\", 1)\n",
      "Pair{AbstractString,Int64}(\"for 3 h\", 2)\n",
      "Pair{AbstractString,Int64}(\"the replace-\", 1)\n",
      "Pair{AbstractString,Int64}(\"296 after HCoV-HKU\", 1)\n",
      "Pair{AbstractString,Int64}(\"189 can\", 1)\n",
      "Pair{AbstractString,Int64}(\"with 1\", 8)\n",
      "Pair{AbstractString,Int64}(\"post-transfection and\", 1)\n",
      "Pair{AbstractString,Int64}(\"learning as necessary\", 1)\n",
      "Pair{AbstractString,Int64}(\"but replication-competent\", 1)\n",
      "Pair{AbstractString,Int64}(\"The order and\", 1)\n",
      "Pair{AbstractString,Int64}(\"virus entry. As\", 1)\n",
      "Pair{AbstractString,Int64}(\"epidemic diarrhea\", 1)\n",
      "Pair{AbstractString,Int64}(\"regions roughly\", 1)\n",
      "Pair{AbstractString,Int64}(\"3.5 mmol\", 1)\n",
      "Pair{AbstractString,Int64}(\") SARS-CoV- 2\", 2)\n",
      "Pair{AbstractString,Int64}(\"cells continued\", 1)\n",
      "Pair{AbstractString,Int64}(\"the cFos-tTA system\", 1)\n",
      "Pair{AbstractString,Int64}(\"interpolation\", 1)\n",
      "Pair{AbstractString,Int64}(\"is induced in\", 1)\n",
      "Pair{AbstractString,Int64}(\"5.4 µM against\", 1)\n",
      "Pair{AbstractString,Int64}(\", loss\", 1)\n",
      "Pair{AbstractString,Int64}(\"S 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"Reagent and frozen.\", 1)\n",
      "Pair{AbstractString,Int64}(\"back. Cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"quantified\", 4)\n",
      "Pair{AbstractString,Int64}(\") . Ciclesonide\", 1)\n",
      "Pair{AbstractString,Int64}(\"the authors\", 1)\n",
      "Pair{AbstractString,Int64}(\"learning in hippocampal\", 1)\n",
      "Pair{AbstractString,Int64}(\"E on\", 1)\n",
      "Pair{AbstractString,Int64}(\"and averaged in\", 1)\n",
      "Pair{AbstractString,Int64}(\"preceded odor\", 1)\n",
      "Pair{AbstractString,Int64}(\"examination. Previous studies\", 1)\n",
      "Pair{AbstractString,Int64}(\"assayed in\", 1)\n",
      "Pair{AbstractString,Int64}(\"latter group\", 1)\n",
      "Pair{AbstractString,Int64}(\"collected from host\", 1)\n",
      "Pair{AbstractString,Int64}(\"intermediate 131\", 1)\n",
      "Pair{AbstractString,Int64}(\"by means of\", 1)\n",
      "Pair{AbstractString,Int64}(\"a grid\", 1)\n",
      "Pair{AbstractString,Int64}(\"1024 x\", 1)\n",
      "Pair{AbstractString,Int64}(\"19647 T in\", 1)\n",
      "Pair{AbstractString,Int64}(\"two cell types\", 1)\n",
      "Pair{AbstractString,Int64}(\"with MERS-CoV\", 4)\n",
      "Pair{AbstractString,Int64}(\"below against\", 1)\n",
      "Pair{AbstractString,Int64}(\"as control\", 1)\n",
      "Pair{AbstractString,Int64}(\"residue independently (\", 1)\n",
      "Pair{AbstractString,Int64}(\"from partial or\", 1)\n",
      "Pair{AbstractString,Int64}(\", and focussed\", 1)\n",
      "Pair{AbstractString,Int64}(\"target for the\", 1)\n",
      "Pair{AbstractString,Int64}(\"less plasticity than\", 1)\n",
      "Pair{AbstractString,Int64}(\"first added into\", 1)\n",
      "Pair{AbstractString,Int64}(\"compared to\", 11)\n",
      "Pair{AbstractString,Int64}(\"no cytopathic\", 1)\n",
      "Pair{AbstractString,Int64}(\"and we stereotaxically\", 1)\n",
      "Pair{AbstractString,Int64}(\"exposure to ethyl-acetate\", 1)\n",
      "Pair{AbstractString,Int64}(\"or through\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 1.62 mL\", 1)\n",
      "Pair{AbstractString,Int64}(\". In virus-infected\", 1)\n",
      "Pair{AbstractString,Int64}(\"21 cell\", 1)\n",
      "Pair{AbstractString,Int64}(\"in parts per\", 1)\n",
      "Pair{AbstractString,Int64}(\"Lglutamine ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"value of CT\", 7)\n",
      "Pair{AbstractString,Int64}(\"to associate a\", 1)\n",
      "Pair{AbstractString,Int64}(\"been proposed\", 1)\n",
      "Pair{AbstractString,Int64}(\"a 12 -well\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"a period of\", 1)\n",
      "Pair{AbstractString,Int64}(\"7.3\", 1)\n",
      "Pair{AbstractString,Int64}(\"nsp 12 and\", 7)\n",
      "Pair{AbstractString,Int64}(\"assay , an\", 1)\n",
      "Pair{AbstractString,Int64}(\"to support auto-associative\", 1)\n",
      "Pair{AbstractString,Int64}(\"washed 2 times\", 1)\n",
      "Pair{AbstractString,Int64}(\"surrounding A\", 1)\n",
      "Pair{AbstractString,Int64}(\"plate. The\", 1)\n",
      "Pair{AbstractString,Int64}(\") , SD\", 2)\n",
      "Pair{AbstractString,Int64}(\"p )\", 1)\n",
      "Pair{AbstractString,Int64}(\"6 cells. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"drives HCoV-HKU\", 1)\n",
      "Pair{AbstractString,Int64}(\"by hand. hM\", 1)\n",
      "Pair{AbstractString,Int64}(\"CC-BY-ND 4.0\", 5)\n",
      "Pair{AbstractString,Int64}(\"however , 317\", 1)\n",
      "Pair{AbstractString,Int64}(\"host cell.\", 1)\n",
      "Pair{AbstractString,Int64}(\"is not only\", 1)\n",
      "Pair{AbstractString,Int64}(\", for each\", 1)\n",
      "Pair{AbstractString,Int64}(\"RNA transcripts ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"μL\", 3)\n",
      "Pair{AbstractString,Int64}(\", the cyano\", 1)\n",
      "Pair{AbstractString,Int64}(\"standard derived from\", 1)\n",
      "Pair{AbstractString,Int64}(\"Different organisms\", 1)\n",
      "Pair{AbstractString,Int64}(\"of inhibitors ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"alpha-coronavirus lineage as\", 1)\n",
      "Pair{AbstractString,Int64}(\"and fused\", 1)\n",
      "Pair{AbstractString,Int64}(\"truncated and fused\", 1)\n",
      "Pair{AbstractString,Int64}(\"CCD detector\", 1)\n",
      "Pair{AbstractString,Int64}(\"sample\", 15)\n",
      "Pair{AbstractString,Int64}(\"A testing box\", 1)\n",
      "Pair{AbstractString,Int64}(\"we observed\", 6)\n",
      "Pair{AbstractString,Int64}(\"VSV * ΔG\", 8)\n",
      "Pair{AbstractString,Int64}(\"active site mutants\", 1)\n",
      "Pair{AbstractString,Int64}(\"\\\" ( main-chain\", 1)\n",
      "Pair{AbstractString,Int64}(\"we evaluated\", 1)\n",
      "Pair{AbstractString,Int64}(\"more inhibitor\", 1)\n",
      "Pair{AbstractString,Int64}(\"216\", 3)\n",
      "Pair{AbstractString,Int64}(\"different ( Mann-Whitney\", 1)\n",
      "Pair{AbstractString,Int64}(\"infectious cycle.\", 1)\n",
      "Pair{AbstractString,Int64}(\"the filled\", 1)\n",
      "Pair{AbstractString,Int64}(\"we also performed\", 2)\n",
      "Pair{AbstractString,Int64}(\"but without evoking\", 1)\n",
      "Pair{AbstractString,Int64}(\"6 E as\", 1)\n",
      "Pair{AbstractString,Int64}(\"RNA genomes in\", 1)\n",
      "Pair{AbstractString,Int64}(\"qubit\", 1)\n",
      "Pair{AbstractString,Int64}(\"for the severe\", 1)\n",
      "Pair{AbstractString,Int64}(\"Fos-tagged neural ensembles.\", 1)\n",
      "Pair{AbstractString,Int64}(\"among these sequences\", 1)\n",
      "Pair{AbstractString,Int64}(\"primary cell\", 1)\n",
      "Pair{AbstractString,Int64}(\"64 , 65\", 1)\n",
      "Pair{AbstractString,Int64}(\"roughly antiperiplanar (\", 1)\n",
      "Pair{AbstractString,Int64}(\"two-tailed unpaired\", 1)\n",
      "Pair{AbstractString,Int64}(\"the binding of\", 1)\n",
      "Pair{AbstractString,Int64}(\"system ( 9\", 1)\n",
      "Pair{AbstractString,Int64}(\"2.3 µM\", 1)\n",
      "Pair{AbstractString,Int64}(\"high (\", 1)\n",
      "Pair{AbstractString,Int64}(\"was delineated on\", 1)\n",
      "Pair{AbstractString,Int64}(\"means of N\", 1)\n",
      "Pair{AbstractString,Int64}(\"defined at\", 1)\n",
      "Pair{AbstractString,Int64}(\"activation , as\", 1)\n",
      "Pair{AbstractString,Int64}(\"largely inactive in\", 1)\n",
      "Pair{AbstractString,Int64}(\"146074\", 1)\n",
      "Pair{AbstractString,Int64}(\"at 50 mM\", 1)\n",
      "Pair{AbstractString,Int64}(\"versus the host\", 1)\n",
      "Pair{AbstractString,Int64}(\"box along\", 1)\n",
      "Pair{AbstractString,Int64}(\"50 values\", 16)\n",
      "Pair{AbstractString,Int64}(\"recovered\", 2)\n",
      "Pair{AbstractString,Int64}(\"was stopped\", 1)\n",
      "Pair{AbstractString,Int64}(\"replaced with growth\", 1)\n",
      "Pair{AbstractString,Int64}(\"as by\", 2)\n",
      "Pair{AbstractString,Int64}(\"Pilatus 6\", 1)\n",
      "Pair{AbstractString,Int64}(\"fear conditioning were\", 1)\n",
      "Pair{AbstractString,Int64}(\"ratio ( 3.91\", 1)\n",
      "Pair{AbstractString,Int64}(\"to CoV. Among\", 1)\n",
      "Pair{AbstractString,Int64}(\"mineral oil (\", 1)\n",
      "Pair{AbstractString,Int64}(\"GEO\", 1)\n",
      "Pair{AbstractString,Int64}(\"* *\", 46)\n",
      "Pair{AbstractString,Int64}(\"Fiji. Counting\", 1)\n",
      "Pair{AbstractString,Int64}(\"compounds 11 a\", 2)\n",
      "Pair{AbstractString,Int64}(\"\\\" of the\", 3)\n",
      "Pair{AbstractString,Int64}(\"pro resembles\", 1)\n",
      "Pair{AbstractString,Int64}(\", 23.3\", 1)\n",
      "Pair{AbstractString,Int64}(\"Renilla luciferase activity\", 2)\n",
      "Pair{AbstractString,Int64}(\"-isobutyl\", 2)\n",
      "Pair{AbstractString,Int64}(\"30 31\", 4)\n",
      "Pair{AbstractString,Int64}(\"unable to generate\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 a ES\", 1)\n",
      "Pair{AbstractString,Int64}(\"presented.\", 3)\n",
      "Pair{AbstractString,Int64}(\"decreased activity\", 1)\n",
      "Pair{AbstractString,Int64}(\"expression of KLKs\", 1)\n",
      "Pair{AbstractString,Int64}(\"tightly\", 1)\n",
      "Pair{AbstractString,Int64}(\". Pharmacological inhibition\", 1)\n",
      "Pair{AbstractString,Int64}(\", and 1.4\", 1)\n",
      "Pair{AbstractString,Int64}(\"frequencies and 175\", 1)\n",
      "Pair{AbstractString,Int64}(\". Standard\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 infection RD\", 1)\n",
      "Pair{AbstractString,Int64}(\"contextual memory\", 2)\n",
      "Pair{AbstractString,Int64}(\"five small inserts\", 1)\n",
      "Pair{AbstractString,Int64}(\"controlling\", 2)\n",
      "Pair{AbstractString,Int64}(\"their performance in\", 1)\n",
      "Pair{AbstractString,Int64}(\"rate is only\", 1)\n",
      "Pair{AbstractString,Int64}(\"networks have suggested\", 1)\n",
      "Pair{AbstractString,Int64}(\"the fusion of\", 1)\n",
      "Pair{AbstractString,Int64}(\"synchrotron beamline BL\", 1)\n",
      "Pair{AbstractString,Int64}(\"control of\", 5)\n",
      "Pair{AbstractString,Int64}(\"constituting such\", 1)\n",
      "Pair{AbstractString,Int64}(\"by either\", 1)\n",
      "Pair{AbstractString,Int64}(\", Expression of\", 1)\n",
      "Pair{AbstractString,Int64}(\"Nei-Gojobori method (\", 1)\n",
      "Pair{AbstractString,Int64}(\"this may\", 1)\n",
      "Pair{AbstractString,Int64}(\"solution. Compared\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 h. The\", 2)\n",
      "Pair{AbstractString,Int64}(\"residue of 11\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 pocket (\", 2)\n",
      "Pair{AbstractString,Int64}(\"t ) ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibitors. At\", 1)\n",
      "Pair{AbstractString,Int64}(\"reports\", 1)\n",
      "Pair{AbstractString,Int64}(\"or MERS-CoV.\", 1)\n",
      "Pair{AbstractString,Int64}(\"of the recombinant\", 1)\n",
      "Pair{AbstractString,Int64}(\"and Thr\", 1)\n",
      "Pair{AbstractString,Int64}(\"into 307\", 1)\n",
      "Pair{AbstractString,Int64}(\", Index\", 1)\n",
      "Pair{AbstractString,Int64}(\"7 - 216\", 1)\n",
      "Pair{AbstractString,Int64}(\"While exposure to\", 1)\n",
      "Pair{AbstractString,Int64}(\"a result ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"drug cytotoxicity\", 1)\n",
      "Pair{AbstractString,Int64}(\"E-specific sgRNA were\", 1)\n",
      "Pair{AbstractString,Int64}(\"60 shaped by\", 1)\n",
      "Pair{AbstractString,Int64}(\"procedure. Reagents\", 1)\n",
      "Pair{AbstractString,Int64}(\"Zhan and\", 1)\n",
      "Pair{AbstractString,Int64}(\"89 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"20 A\", 1)\n",
      "Pair{AbstractString,Int64}(\"13 ( 200\", 1)\n",
      "Pair{AbstractString,Int64}(\", DNAPTP 6\", 1)\n",
      "Pair{AbstractString,Int64}(\"Fukunaga et al.\", 1)\n",
      "Pair{AbstractString,Int64}(\"at optimizing the\", 1)\n",
      "Pair{AbstractString,Int64}(\"variant ( D\", 1)\n",
      "Pair{AbstractString,Int64}(\"efforts therefore\", 1)\n",
      "Pair{AbstractString,Int64}(\"A 549\", 2)\n",
      "Pair{AbstractString,Int64}(\"results obtained\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 )\", 46)\n",
      "Pair{AbstractString,Int64}(\"38 39\", 4)\n",
      "Pair{AbstractString,Int64}(\"efficacy to lopinavir\", 1)\n",
      "Pair{AbstractString,Int64}(\"human Calu 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"showed IC 50\", 1)\n",
      "Pair{AbstractString,Int64}(\"of free enzyme\", 1)\n",
      "Pair{AbstractString,Int64}(\"P 250 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"Fig. 10 c\", 1)\n",
      "Pair{AbstractString,Int64}(\"600 µL washing\", 1)\n",
      "Pair{AbstractString,Int64}(\"pro shows that\", 1)\n",
      "Pair{AbstractString,Int64}(\"odor originating\", 1)\n",
      "Pair{AbstractString,Int64}(\"different odor\", 2)\n",
      "Pair{AbstractString,Int64}(\"70 -\", 1)\n",
      "Pair{AbstractString,Int64}(\"to hM 4\", 1)\n",
      "Pair{AbstractString,Int64}(\"212 N sg\", 1)\n",
      "Pair{AbstractString,Int64}(\"this type of\", 1)\n",
      "Pair{AbstractString,Int64}(\"Volker Thiel (\", 1)\n",
      "Pair{AbstractString,Int64}(\"Assessment\", 1)\n",
      "Pair{AbstractString,Int64}(\"preparing the\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.0001 ; f\", 1)\n",
      "Pair{AbstractString,Int64}(\", resulting\", 4)\n",
      "Pair{AbstractString,Int64}(\"absence 181\", 1)\n",
      "Pair{AbstractString,Int64}(\"13 ( collected\", 1)\n",
      "Pair{AbstractString,Int64}(\"have been carried\", 1)\n",
      "Pair{AbstractString,Int64}(\"medium of\", 1)\n",
      "Pair{AbstractString,Int64}(\"preprint Reactivation of\", 1)\n",
      "Pair{AbstractString,Int64}(\"300 µl OptiMEM\", 1)\n",
      "Pair{AbstractString,Int64}(\"with Shim-pack\", 1)\n",
      "Pair{AbstractString,Int64}(\"well characterized (\", 1)\n",
      "Pair{AbstractString,Int64}(\"the replicons\", 2)\n",
      "Pair{AbstractString,Int64}(\"common features\", 1)\n",
      "Pair{AbstractString,Int64}(\"5 B )\", 2)\n",
      "Pair{AbstractString,Int64}(\"experience. We\", 1)\n",
      "Pair{AbstractString,Int64}(\"λ phage )\", 1)\n",
      "Pair{AbstractString,Int64}(\"models for\", 1)\n",
      "Pair{AbstractString,Int64}(\"codon usage\", 4)\n",
      "Pair{AbstractString,Int64}(\"built into Fo-Fc\", 1)\n",
      "Pair{AbstractString,Int64}(\"far rarely\", 1)\n",
      "Pair{AbstractString,Int64}(\"storage in hippocampusrelated\", 1)\n",
      "Pair{AbstractString,Int64}(\"s recommendations. Transfection.\", 1)\n",
      "Pair{AbstractString,Int64}(\"addressed with careful\", 1)\n",
      "Pair{AbstractString,Int64}(\"coverslipped with\", 1)\n",
      "Pair{AbstractString,Int64}(\"previously ( pBAC-REP-RLuc\", 1)\n",
      "Pair{AbstractString,Int64}(\"482 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"U= 27 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"range of enteroviruses\", 1)\n",
      "Pair{AbstractString,Int64}(\"treating this disease\", 1)\n",
      "Pair{AbstractString,Int64}(\"restrict\", 1)\n",
      "Pair{AbstractString,Int64}(\") . An\", 1)\n",
      "Pair{AbstractString,Int64}(\"427 diluted\", 1)\n",
      "Pair{AbstractString,Int64}(\"555 phosphate ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"blue )\", 2)\n",
      "Pair{AbstractString,Int64}(\"20 x\", 2)\n",
      "Pair{AbstractString,Int64}(\"different SARS-CoV- 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 ) .\", 12)\n",
      "Pair{AbstractString,Int64}(\"acid of\", 1)\n",
      "Pair{AbstractString,Int64}(\"the CS+ causes\", 1)\n",
      "Pair{AbstractString,Int64}(\"of neural\", 3)\n",
      "Pair{AbstractString,Int64}(\"the anti-SARS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\") . Finally\", 5)\n",
      "Pair{AbstractString,Int64}(\"63 ( ref.\", 1)\n",
      "Pair{AbstractString,Int64}(\"detection. In\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\", 0.6\", 1)\n",
      "Pair{AbstractString,Int64}(\"' -C-methyladenosine (\", 1)\n",
      "Pair{AbstractString,Int64}(\"2008\", 5)\n",
      "Pair{AbstractString,Int64}(\"antiviral effects ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"titer of each\", 1)\n",
      "Pair{AbstractString,Int64}(\"we synthesized are\", 1)\n",
      "Pair{AbstractString,Int64}(\"presentation\", 21)\n",
      "Pair{AbstractString,Int64}(\"volumes of cell\", 1)\n",
      "Pair{AbstractString,Int64}(\"expression , detected\", 1)\n",
      "Pair{AbstractString,Int64}(\"are the human\", 1)\n",
      "Pair{AbstractString,Int64}(\"functions\", 3)\n",
      "Pair{AbstractString,Int64}(\"its performance\", 1)\n",
      "Pair{AbstractString,Int64}(\"evolutionary trajectories\", 1)\n",
      "Pair{AbstractString,Int64}(\"as follows.\", 1)\n",
      "Pair{AbstractString,Int64}(\"added. Cells were\", 1)\n",
      "Pair{AbstractString,Int64}(\"p= 0.890 ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"CH 2 Cl\", 15)\n",
      "Pair{AbstractString,Int64}(\"s ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\", the CS+\", 3)\n",
      "Pair{AbstractString,Int64}(\"a reduction in\", 1)\n",
      "Pair{AbstractString,Int64}(\"was degraded (\", 1)\n",
      "Pair{AbstractString,Int64}(\"Vero or\", 1)\n",
      "Pair{AbstractString,Int64}(\"LHMDS )\", 1)\n",
      "Pair{AbstractString,Int64}(\"livers\", 2)\n",
      "Pair{AbstractString,Int64}(\"dropwise to the\", 1)\n",
      "Pair{AbstractString,Int64}(\", 342\", 1)\n",
      "Pair{AbstractString,Int64}(\"4\", 149)\n",
      "Pair{AbstractString,Int64}(\"data upon article\", 1)\n",
      "Pair{AbstractString,Int64}(\"proteases of human\", 1)\n",
      "Pair{AbstractString,Int64}(\": C transition\", 1)\n",
      "Pair{AbstractString,Int64}(\"be embedded\", 1)\n",
      "Pair{AbstractString,Int64}(\"assessment of\", 2)\n",
      "Pair{AbstractString,Int64}(\"bath was removed.\", 1)\n",
      "Pair{AbstractString,Int64}(\"and positive\", 1)\n",
      "Pair{AbstractString,Int64}(\"decode odor\", 1)\n",
      "Pair{AbstractString,Int64}(\"was only 2.35\", 1)\n",
      "Pair{AbstractString,Int64}(\"pH 7.4 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"per non-synonymous\", 1)\n",
      "Pair{AbstractString,Int64}(\"inactivity )\", 1)\n",
      "Pair{AbstractString,Int64}(\"24 hours ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"spun\", 2)\n",
      "Pair{AbstractString,Int64}(\"calculated the distance\", 1)\n",
      "Pair{AbstractString,Int64}(\"two-tailed unpaired student\", 1)\n",
      "Pair{AbstractString,Int64}(\"Glo assay kit\", 1)\n",
      "Pair{AbstractString,Int64}(\"x NEAA/\", 2)\n",
      "Pair{AbstractString,Int64}(\"to be preferential\", 1)\n",
      "Pair{AbstractString,Int64}(\"turn\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Ma ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"fit\", 6)\n",
      "Pair{AbstractString,Int64}(\", Figure 2\", 6)\n",
      "Pair{AbstractString,Int64}(\"threat\", 2)\n",
      "Pair{AbstractString,Int64}(\"with multiple\", 1)\n",
      "Pair{AbstractString,Int64}(\"in the eukaryotic\", 1)\n",
      "Pair{AbstractString,Int64}(\"of the immune\", 1)\n",
      "Pair{AbstractString,Int64}(\"sensitive\", 1)\n",
      "Pair{AbstractString,Int64}(\"the patterns\", 1)\n",
      "Pair{AbstractString,Int64}(\"for \\\" early\", 1)\n",
      "Pair{AbstractString,Int64}(\"medium (\", 5)\n",
      "Pair{AbstractString,Int64}(\"replication ( 19\", 1)\n",
      "Pair{AbstractString,Int64}(\"which this enzyme\", 1)\n",
      "Pair{AbstractString,Int64}(\"pirform cortex and\", 1)\n",
      "Pair{AbstractString,Int64}(\"infection , we\", 2)\n",
      "Pair{AbstractString,Int64}(\"treating this\", 1)\n",
      "Pair{AbstractString,Int64}(\"Leica SP 5\", 2)\n",
      "Pair{AbstractString,Int64}(\"hoc\", 1)\n",
      "Pair{AbstractString,Int64}(\"x 50 cm\", 1)\n",
      "Pair{AbstractString,Int64}(\"data indicate that\", 2)\n",
      "Pair{AbstractString,Int64}(\"pCG 1 expression\", 1)\n",
      "Pair{AbstractString,Int64}(\"that KLK\", 8)\n",
      "Pair{AbstractString,Int64}(\"p= 0.0001 .\", 1)\n",
      "Pair{AbstractString,Int64}(\"that memories are\", 1)\n",
      "Pair{AbstractString,Int64}(\"by 1 nt\", 1)\n",
      "Pair{AbstractString,Int64}(\"Mus Musculus\", 1)\n",
      "Pair{AbstractString,Int64}(\", Tecan\", 1)\n",
      "Pair{AbstractString,Int64}(\"data collection.\", 1)\n",
      "Pair{AbstractString,Int64}(\"we devised a\", 1)\n",
      "Pair{AbstractString,Int64}(\"in ExoN-AA but\", 1)\n",
      "Pair{AbstractString,Int64}(\"the position\", 7)\n",
      "Pair{AbstractString,Int64}(\"separately\", 2)\n",
      "Pair{AbstractString,Int64}(\"precise temporal\", 1)\n",
      "Pair{AbstractString,Int64}(\"2019\", 22)\n",
      "Pair{AbstractString,Int64}(\"session. The median\", 1)\n",
      "Pair{AbstractString,Int64}(\"12 , p<\", 1)\n",
      "Pair{AbstractString,Int64}(\"71 and\", 4)\n",
      "Pair{AbstractString,Int64}(\"trees\", 1)\n",
      "Pair{AbstractString,Int64}(\"proteases Supplemental\", 1)\n",
      "Pair{AbstractString,Int64}(\"virus-induced CPE or\", 1)\n",
      "Pair{AbstractString,Int64}(\"to + 500\", 1)\n",
      "Pair{AbstractString,Int64}(\") synclinal (\", 1)\n",
      "Pair{AbstractString,Int64}(\") . Dendrograms\", 1)\n",
      "Pair{AbstractString,Int64}(\"only 30\", 1)\n",
      "Pair{AbstractString,Int64}(\"was assigned\", 1)\n",
      "Pair{AbstractString,Int64}(\"complex with compounds\", 1)\n",
      "Pair{AbstractString,Int64}(\"pirform cortex\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibitors were used\", 1)\n",
      "Pair{AbstractString,Int64}(\"a-helical domain\", 1)\n",
      "Pair{AbstractString,Int64}(\"compounds , see\", 1)\n",
      "Pair{AbstractString,Int64}(\"PFU ) were\", 1)\n",
      "Pair{AbstractString,Int64}(\"contained WT-revertants by\", 1)\n",
      "Pair{AbstractString,Int64}(\"in-situ cyclization\", 1)\n",
      "Pair{AbstractString,Int64}(\"panel C )\", 1)\n",
      "Pair{AbstractString,Int64}(\"prepared with\", 5)\n",
      "Pair{AbstractString,Int64}(\"ensure the\", 1)\n",
      "Pair{AbstractString,Int64}(\"passage 91\", 1)\n",
      "Pair{AbstractString,Int64}(\") ; and\", 1)\n",
      "Pair{AbstractString,Int64}(\"much more frequently\", 1)\n",
      "Pair{AbstractString,Int64}(\"µM. replication cycles\", 1)\n",
      "Pair{AbstractString,Int64}(\"GraphPad Prism\", 3)\n",
      "Pair{AbstractString,Int64}(\"Systemic\", 2)\n",
      "Pair{AbstractString,Int64}(\"target areas for\", 1)\n",
      "Pair{AbstractString,Int64}(\"CoV ExoN motif\", 1)\n",
      "Pair{AbstractString,Int64}(\"The copyright holder\", 147)\n",
      "Pair{AbstractString,Int64}(\"substituents must reach\", 1)\n",
      "Pair{AbstractString,Int64}(\"proteases was monitored\", 1)\n",
      "Pair{AbstractString,Int64}(\"by centrifugation (\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 ' -to-\", 2)\n",
      "Pair{AbstractString,Int64}(\"sample even prepared\", 1)\n",
      "Pair{AbstractString,Int64}(\"stereochemistry at\", 1)\n",
      "Pair{AbstractString,Int64}(\"loop\", 1)\n",
      "Pair{AbstractString,Int64}(\"M 288\", 1)\n",
      "Pair{AbstractString,Int64}(\"the housekeeping\", 2)\n",
      "Pair{AbstractString,Int64}(\"and LY 6\", 1)\n",
      "Pair{AbstractString,Int64}(\"David\", 1)\n",
      "Pair{AbstractString,Int64}(\"provide the\", 3)\n",
      "Pair{AbstractString,Int64}(\"8155 -\", 2)\n",
      "Pair{AbstractString,Int64}(\"purchased from\", 2)\n",
      "Pair{AbstractString,Int64}(\"303 part\", 1)\n",
      "Pair{AbstractString,Int64}(\", single read\", 1)\n",
      "Pair{AbstractString,Int64}(\") infection experiments\", 1)\n",
      "Pair{AbstractString,Int64}(\"MgCl\", 5)\n",
      "Pair{AbstractString,Int64}(\"host proteases cleave\", 1)\n",
      "Pair{AbstractString,Int64}(\"eradicating RNA viral\", 1)\n",
      "Pair{AbstractString,Int64}(\"308 polynucleotide\", 1)\n",
      "Pair{AbstractString,Int64}(\", which will\", 3)\n",
      "Pair{AbstractString,Int64}(\"was quenched\", 3)\n",
      "Pair{AbstractString,Int64}(\") , doublet\", 1)\n",
      "Pair{AbstractString,Int64}(\"Notably , we\", 1)\n",
      "Pair{AbstractString,Int64}(\"; Illig\", 1)\n",
      "Pair{AbstractString,Int64}(\"r with EC\", 1)\n",
      "Pair{AbstractString,Int64}(\"shock , were\", 1)\n",
      "Pair{AbstractString,Int64}(\"crystals were protected\", 1)\n",
      "Pair{AbstractString,Int64}(\"visible genomic DNA\", 1)\n",
      "Pair{AbstractString,Int64}(\"equiv (\", 1)\n",
      "Pair{AbstractString,Int64}(\"supplemented with 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"323\", 1)\n",
      "Pair{AbstractString,Int64}(\"146 observed\", 1)\n",
      "Pair{AbstractString,Int64}(\"24 -well plate.\", 1)\n",
      "Pair{AbstractString,Int64}(\"S 1 D\", 1)\n",
      "Pair{AbstractString,Int64}(\"II\", 3)\n",
      "Pair{AbstractString,Int64}(\"al. , 2011\", 5)\n",
      "Pair{AbstractString,Int64}(\"should show good\", 1)\n",
      "Pair{AbstractString,Int64}(\"replication , viral\", 1)\n",
      "Pair{AbstractString,Int64}(\"the CS+ increased\", 1)\n",
      "Pair{AbstractString,Int64}(\"as none of\", 1)\n",
      "Pair{AbstractString,Int64}(\"and then subjected\", 1)\n",
      "Pair{AbstractString,Int64}(\"mice , which\", 1)\n",
      "Pair{AbstractString,Int64}(\"binding site.\", 1)\n",
      "Pair{AbstractString,Int64}(\"( DCM )\", 1)\n",
      "Pair{AbstractString,Int64}(\"the conditioning box\", 1)\n",
      "Pair{AbstractString,Int64}(\"( MN\", 1)\n",
      "Pair{AbstractString,Int64}(\"they may\", 1)\n",
      "Pair{AbstractString,Int64}(\"8.5 µM\", 1)\n",
      "Pair{AbstractString,Int64}(\"two days and\", 1)\n",
      "Pair{AbstractString,Int64}(\"- 71.70\", 1)\n",
      "Pair{AbstractString,Int64}(\"virus that developed\", 1)\n",
      "Pair{AbstractString,Int64}(\"-yl ) -\", 1)\n",
      "Pair{AbstractString,Int64}(\"2013 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"100 mg/kg/ 10\", 2)\n",
      "Pair{AbstractString,Int64}(\"of 4\", 1)\n",
      "Pair{AbstractString,Int64}(\"were amplified by\", 3)\n",
      "Pair{AbstractString,Int64}(\"area covered by\", 2)\n",
      "Pair{AbstractString,Int64}(\"only a ~\", 1)\n",
      "Pair{AbstractString,Int64}(\"the existence of\", 1)\n",
      "Pair{AbstractString,Int64}(\"administered through the\", 1)\n",
      "Pair{AbstractString,Int64}(\"CA ) was\", 1)\n",
      "Pair{AbstractString,Int64}(\"attempt\", 1)\n",
      "Pair{AbstractString,Int64}(\"The excitability\", 1)\n",
      "Pair{AbstractString,Int64}(\"at ( Table\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 3.75 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"Parasagittal or coronal\", 1)\n",
      "Pair{AbstractString,Int64}(\"region. Consequently\", 1)\n",
      "Pair{AbstractString,Int64}(\"odor escape\", 1)\n",
      "Pair{AbstractString,Int64}(\"but also improve\", 1)\n",
      "Pair{AbstractString,Int64}(\", 2016 ;\", 7)\n",
      "Pair{AbstractString,Int64}(\"different SARS-CoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"45 ( p\", 1)\n",
      "Pair{AbstractString,Int64}(\"s ( 5\", 1)\n",
      "Pair{AbstractString,Int64}(\"M pro by\", 2)\n",
      "Pair{AbstractString,Int64}(\"whereas in the\", 1)\n",
      "Pair{AbstractString,Int64}(\"of capped dipeptide\", 1)\n",
      "Pair{AbstractString,Int64}(\"Triton-X- 100\", 1)\n",
      "Pair{AbstractString,Int64}(\"DNA-launched SARS-CoV replicon\", 1)\n",
      "Pair{AbstractString,Int64}(\"recent evolution of\", 1)\n",
      "Pair{AbstractString,Int64}(\"6 Emediated enhancement\", 1)\n",
      "Pair{AbstractString,Int64}(\"General procedure. Reagents\", 1)\n",
      "Pair{AbstractString,Int64}(\"the active-site\", 1)\n",
      "Pair{AbstractString,Int64}(\"uncoupled 273 from\", 1)\n",
      "Pair{AbstractString,Int64}(\"Plexiglas\", 1)\n",
      "Pair{AbstractString,Int64}(\"by Arg 39\", 1)\n",
      "Pair{AbstractString,Int64}(\"Atomic\", 1)\n",
      "Pair{AbstractString,Int64}(\"p= 0.1407 ;\", 1)\n",
      "Pair{AbstractString,Int64}(\". The number\", 1)\n",
      "Pair{AbstractString,Int64}(\"insert segment\", 2)\n",
      "Pair{AbstractString,Int64}(\"eventual sites of\", 1)\n",
      "Pair{AbstractString,Int64}(\"Volker Thiel\", 2)\n",
      "Pair{AbstractString,Int64}(\"the main proteases\", 1)\n",
      "Pair{AbstractString,Int64}(\"clinical inspectors from\", 1)\n",
      "Pair{AbstractString,Int64}(\"Crystallization. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"of other proteases\", 1)\n",
      "Pair{AbstractString,Int64}(\"and detected by\", 2)\n",
      "Pair{AbstractString,Int64}(\"studies of E.\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"Commission of China\", 1)\n",
      "Pair{AbstractString,Int64}(\"nt ; ExoN-DD\", 1)\n",
      "Pair{AbstractString,Int64}(\"Mice were given\", 1)\n",
      "Pair{AbstractString,Int64}(\"Animals that\", 1)\n",
      "Pair{AbstractString,Int64}(\"concentrated to\", 1)\n",
      "Pair{AbstractString,Int64}(\"Life Technologies\", 1)\n",
      "Pair{AbstractString,Int64}(\"in control wells\", 1)\n",
      "Pair{AbstractString,Int64}(\"We compared\", 1)\n",
      "Pair{AbstractString,Int64}(\"assume that the\", 1)\n",
      "Pair{AbstractString,Int64}(\"mutational bias of\", 1)\n",
      "Pair{AbstractString,Int64}(\") can\", 1)\n",
      "Pair{AbstractString,Int64}(\", with its\", 1)\n",
      "Pair{AbstractString,Int64}(\": mCherry\", 2)\n",
      "Pair{AbstractString,Int64}(\"71 3 C\", 17)\n",
      "Pair{AbstractString,Int64}(\"were detected by\", 1)\n",
      "Pair{AbstractString,Int64}(\"deeply\", 3)\n",
      "Pair{AbstractString,Int64}(\"this concentration ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"transition mutation was\", 1)\n",
      "Pair{AbstractString,Int64}(\"main-chain amides of\", 3)\n",
      "Pair{AbstractString,Int64}(\"two acceptors for\", 1)\n",
      "Pair{AbstractString,Int64}(\"that 125\", 1)\n",
      "Pair{AbstractString,Int64}(\", are\", 1)\n",
      "Pair{AbstractString,Int64}(\"% DMSO.\", 1)\n",
      "Pair{AbstractString,Int64}(\"what was observed\", 1)\n",
      "Pair{AbstractString,Int64}(\"in Huh-T 7\", 3)\n",
      "Pair{AbstractString,Int64}(\"a pre-dissolved in\", 1)\n",
      "Pair{AbstractString,Int64}(\"ratio of nucleotide\", 1)\n",
      "Pair{AbstractString,Int64}(\"acid dimethyl ester\", 2)\n",
      "Pair{AbstractString,Int64}(\"His-tagged purified\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 0.75 mmol\", 1)\n",
      "Pair{AbstractString,Int64}(\"86 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"229 E infection\", 1)\n",
      "Pair{AbstractString,Int64}(\"HRV 14\", 5)\n",
      "Pair{AbstractString,Int64}(\", long-term\", 1)\n",
      "Pair{AbstractString,Int64}(\"/TMPRSS 2 cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"networks ( Haberly\", 1)\n",
      "Pair{AbstractString,Int64}(\"poured into 10\", 1)\n",
      "Pair{AbstractString,Int64}(\"vs. 4.67 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"box after CS+\", 1)\n",
      "Pair{AbstractString,Int64}(\". Synthesis\", 7)\n",
      "Pair{AbstractString,Int64}(\"the tTA-responsive promoter\", 1)\n",
      "Pair{AbstractString,Int64}(\"phenotypes. 286 All\", 1)\n",
      "Pair{AbstractString,Int64}(\"protein of\", 1)\n",
      "Pair{AbstractString,Int64}(\"acids 71 The\", 1)\n",
      "Pair{AbstractString,Int64}(\"at 5 kHz\", 1)\n",
      "Pair{AbstractString,Int64}(\"silica ,\", 3)\n",
      "Pair{AbstractString,Int64}(\"GraphPad Prism v\", 1)\n",
      "Pair{AbstractString,Int64}(\"Influenza Data (\", 1)\n",
      "Pair{AbstractString,Int64}(\"murine Ly\", 1)\n",
      "Pair{AbstractString,Int64}(\"284 not\", 1)\n",
      "Pair{AbstractString,Int64}(\"resistance to ciclesonide.\", 1)\n",
      "Pair{AbstractString,Int64}(\"polymerase I ExoN\", 1)\n",
      "Pair{AbstractString,Int64}(\"slightly but significantly\", 1)\n",
      "Pair{AbstractString,Int64}(\"to accumulate in\", 1)\n",
      "Pair{AbstractString,Int64}(\", the a-hydroxyamides\", 1)\n",
      "Pair{AbstractString,Int64}(\"rate (\", 5)\n",
      "Pair{AbstractString,Int64}(\"GSE 146074\", 1)\n",
      "Pair{AbstractString,Int64}(\"as that\", 2)\n",
      "Pair{AbstractString,Int64}(\"as 9.07\", 1)\n",
      "Pair{AbstractString,Int64}(\"further away from\", 1)\n",
      "Pair{AbstractString,Int64}(\"Stock solutions\", 3)\n",
      "Pair{AbstractString,Int64}(\"; Kanter and\", 1)\n",
      "Pair{AbstractString,Int64}(\"273 from its\", 1)\n",
      "Pair{AbstractString,Int64}(\"although this has\", 1)\n",
      "Pair{AbstractString,Int64}(\"48 hours post-transduction\", 2)\n",
      "Pair{AbstractString,Int64}(\"bioRxiv preprint ment\", 1)\n",
      "Pair{AbstractString,Int64}(\"be deactivated compared\", 1)\n",
      "Pair{AbstractString,Int64}(\"a probe targeting\", 1)\n",
      "Pair{AbstractString,Int64}(\"38 in case\", 1)\n",
      "Pair{AbstractString,Int64}(\"target the\", 1)\n",
      "Pair{AbstractString,Int64}(\", and 11\", 3)\n",
      "Pair{AbstractString,Int64}(\"171 We\", 1)\n",
      "Pair{AbstractString,Int64}(\"acidic charge followed\", 1)\n",
      "Pair{AbstractString,Int64}(\"each of the\", 1)\n",
      "Pair{AbstractString,Int64}(\"evaluated by\", 1)\n",
      "Pair{AbstractString,Int64}(\"s protocol. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"reactivation , similar\", 1)\n",
      "Pair{AbstractString,Int64}(\"to dS\", 1)\n",
      "Pair{AbstractString,Int64}(\"requires a methylene\", 1)\n",
      "Pair{AbstractString,Int64}(\", neurons\", 1)\n",
      "Pair{AbstractString,Int64}(\"for determining host\", 1)\n",
      "Pair{AbstractString,Int64}(\"the reproductive\", 1)\n",
      "Pair{AbstractString,Int64}(\"olfactory learning.\", 1)\n",
      "Pair{AbstractString,Int64}(\"fresh , unfrozen\", 1)\n",
      "Pair{AbstractString,Int64}(\"poor inhibitory activity\", 1)\n",
      "Pair{AbstractString,Int64}(\"475 A fragment.\", 1)\n",
      "Pair{AbstractString,Int64}(\"develop a\", 1)\n",
      "Pair{AbstractString,Int64}(\"Superimpostion of residues\", 1)\n",
      "Pair{AbstractString,Int64}(\"1.6 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"studies have been\", 1)\n",
      "Pair{AbstractString,Int64}(\"exceeds 81,000 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\". Only the\", 1)\n",
      "Pair{AbstractString,Int64}(\"2012 to\", 1)\n",
      "Pair{AbstractString,Int64}(\"nucleotideswere\", 1)\n",
      "Pair{AbstractString,Int64}(\"on mCitrine\", 1)\n",
      "Pair{AbstractString,Int64}(\") system\", 1)\n",
      "Pair{AbstractString,Int64}(\"and equal cell\", 1)\n",
      "Pair{AbstractString,Int64}(\"genetic model\", 1)\n",
      "Pair{AbstractString,Int64}(\"revealed by X-ray\", 1)\n",
      "Pair{AbstractString,Int64}(\"memory encoding (\", 1)\n",
      "Pair{AbstractString,Int64}(\"Center at\", 1)\n",
      "Pair{AbstractString,Int64}(\"contribution of reagents\", 1)\n",
      "Pair{AbstractString,Int64}(\"in Extended\", 2)\n",
      "Pair{AbstractString,Int64}(\"diseases. High mutation\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibitor versus normalized\", 1)\n",
      "Pair{AbstractString,Int64}(\"suggested that\", 3)\n",
      "Pair{AbstractString,Int64}(\"Site-directed\", 1)\n",
      "Pair{AbstractString,Int64}(\"window\", 3)\n",
      "Pair{AbstractString,Int64}(\", see\", 1)\n",
      "Pair{AbstractString,Int64}(\". 184 WT-MHV\", 1)\n",
      "Pair{AbstractString,Int64}(\"L 36\", 2)\n",
      "Pair{AbstractString,Int64}(\"- ) evolved\", 1)\n",
      "Pair{AbstractString,Int64}(\"usual\", 1)\n",
      "Pair{AbstractString,Int64}(\"against HCoV\", 6)\n",
      "Pair{AbstractString,Int64}(\"for the enterovirus\", 1)\n",
      "Pair{AbstractString,Int64}(\"HeLa Rh cells\", 8)\n",
      "Pair{AbstractString,Int64}(\"injected intraperitoneally\", 1)\n",
      "Pair{AbstractString,Int64}(\"silencing of\", 7)\n",
      "Pair{AbstractString,Int64}(\"effect. Stock\", 1)\n",
      "Pair{AbstractString,Int64}(\"shown )\", 3)\n",
      "Pair{AbstractString,Int64}(\"preprint scale\", 1)\n",
      "Pair{AbstractString,Int64}(\"7 ( 0.75\", 1)\n",
      "Pair{AbstractString,Int64}(\"synonymous substitution\", 5)\n",
      "Pair{AbstractString,Int64}(\"34 % .\", 1)\n",
      "Pair{AbstractString,Int64}(\"harvested 48\", 1)\n",
      "Pair{AbstractString,Int64}(\"lack topographic organization.\", 1)\n",
      "Pair{AbstractString,Int64}(\"the sulfur atoms\", 1)\n",
      "Pair{AbstractString,Int64}(\"experiments also\", 1)\n",
      "Pair{AbstractString,Int64}(\"protocol APAFIS\", 1)\n",
      "Pair{AbstractString,Int64}(\"against which\", 2)\n",
      "Pair{AbstractString,Int64}(\"were observed\", 1)\n",
      "Pair{AbstractString,Int64}(\"form , by\", 1)\n",
      "Pair{AbstractString,Int64}(\"the pattern of\", 1)\n",
      "Pair{AbstractString,Int64}(\"side ) and\", 1)\n",
      "Pair{AbstractString,Int64}(\"transmissible\", 2)\n",
      "Pair{AbstractString,Int64}(\"in n=\", 1)\n",
      "Pair{AbstractString,Int64}(\"neutral odor.\", 1)\n",
      "Pair{AbstractString,Int64}(\"replication detected via\", 1)\n",
      "Pair{AbstractString,Int64}(\"TMPRSS 2 were\", 2)\n",
      "Pair{AbstractString,Int64}(\", 1.45\", 1)\n",
      "Pair{AbstractString,Int64}(\"benzyl substituent\", 1)\n",
      "Pair{AbstractString,Int64}(\"h. After the\", 1)\n",
      "Pair{AbstractString,Int64}(\", 83 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"hypothesis that piriform\", 1)\n",
      "Pair{AbstractString,Int64}(\"washed\", 12)\n",
      "Pair{AbstractString,Int64}(\"203 to\", 1)\n",
      "Pair{AbstractString,Int64}(\". An inverted\", 1)\n",
      "Pair{AbstractString,Int64}(\", LOC 400759\", 1)\n",
      "Pair{AbstractString,Int64}(\"62 of\", 1)\n",
      "Pair{AbstractString,Int64}(\"degradation , among\", 1)\n",
      "Pair{AbstractString,Int64}(\"hemispheres of piriform\", 2)\n",
      "Pair{AbstractString,Int64}(\", John\", 1)\n",
      "Pair{AbstractString,Int64}(\"Data Figure\", 1)\n",
      "Pair{AbstractString,Int64}(\"for the electrophysiology\", 1)\n",
      "Pair{AbstractString,Int64}(\"APAFIS # 2016012909576100\", 1)\n",
      "Pair{AbstractString,Int64}(\"corresponding to odor\", 1)\n",
      "Pair{AbstractString,Int64}(\"Consequently ,\", 3)\n",
      "Pair{AbstractString,Int64}(\"is available to\", 1)\n",
      "Pair{AbstractString,Int64}(\"e )\", 3)\n",
      "Pair{AbstractString,Int64}(\"cell lysates were\", 2)\n",
      "Pair{AbstractString,Int64}(\"with 5 %\", 1)\n",
      "Pair{AbstractString,Int64}(\"to February 2020\", 1)\n",
      "Pair{AbstractString,Int64}(\"other 272\", 1)\n",
      "Pair{AbstractString,Int64}(\"for activation\", 1)\n",
      "Pair{AbstractString,Int64}(\"180 TMPRSS\", 1)\n",
      "Pair{AbstractString,Int64}(\"their half-maximal effective\", 1)\n",
      "Pair{AbstractString,Int64}(\"120 µM ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"335 All rights\", 1)\n",
      "Pair{AbstractString,Int64}(\"the precise molecular\", 1)\n",
      "Pair{AbstractString,Int64}(\"4.92 g ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"2020.03.05.979260 doi :\", 32)\n",
      "Pair{AbstractString,Int64}(\"do so\", 1)\n",
      "Pair{AbstractString,Int64}(\", the data\", 1)\n",
      "Pair{AbstractString,Int64}(\"but without\", 1)\n",
      "Pair{AbstractString,Int64}(\"this compound exhibited\", 1)\n",
      "Pair{AbstractString,Int64}(\"The coronavirus M\", 1)\n",
      "Pair{AbstractString,Int64}(\"is not true\", 1)\n",
      "Pair{AbstractString,Int64}(\"at 72 °\", 2)\n",
      "Pair{AbstractString,Int64}(\"passage\", 6)\n",
      "Pair{AbstractString,Int64}(\"perturbing the activity\", 1)\n",
      "Pair{AbstractString,Int64}(\"genes of\", 2)\n",
      "Pair{AbstractString,Int64}(\"n= 5\", 2)\n",
      "Pair{AbstractString,Int64}(\"http://www.kazusa.or.jp/codon/\", 1)\n",
      "Pair{AbstractString,Int64}(\"293 This\", 1)\n",
      "Pair{AbstractString,Int64}(\"pocket offers\", 1)\n",
      "Pair{AbstractString,Int64}(\"in ( 33\", 1)\n",
      "Pair{AbstractString,Int64}(\"algorithm. On each\", 1)\n",
      "Pair{AbstractString,Int64}(\"different concentrations\", 1)\n",
      "Pair{AbstractString,Int64}(\"immuno-stained\", 1)\n",
      "Pair{AbstractString,Int64}(\"proteases Supplemental Table\", 1)\n",
      "Pair{AbstractString,Int64}(\"dye ( Ghostdye\", 1)\n",
      "Pair{AbstractString,Int64}(\"The open\", 1)\n",
      "Pair{AbstractString,Int64}(\", 2 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"substantially\", 2)\n",
      "Pair{AbstractString,Int64}(\", and their\", 3)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"frequency\", 4)\n",
      "Pair{AbstractString,Int64}(\"= 0.0047\", 1)\n",
      "Pair{AbstractString,Int64}(\"neutral red (\", 1)\n",
      "Pair{AbstractString,Int64}(\"PETRA III\", 1)\n",
      "Pair{AbstractString,Int64}(\"is essential for\", 3)\n",
      "Pair{AbstractString,Int64}(\"site of\", 7)\n",
      "Pair{AbstractString,Int64}(\"228 background. 229\", 1)\n",
      "Pair{AbstractString,Int64}(\"after fear conditioning\", 2)\n",
      "Pair{AbstractString,Int64}(\"spike protein in\", 1)\n",
      "Pair{AbstractString,Int64}(\"well was\", 1)\n",
      "Pair{AbstractString,Int64}(\"= cyclohexylmethyl\", 2)\n",
      "Pair{AbstractString,Int64}(\"its conformation\", 1)\n",
      "Pair{AbstractString,Int64}(\"Slides were\", 1)\n",
      "Pair{AbstractString,Int64}(\"major protease\", 1)\n",
      "Pair{AbstractString,Int64}(\"indicated by spheres\", 1)\n",
      "Pair{AbstractString,Int64}(\"sites of all\", 1)\n",
      "Pair{AbstractString,Int64}(\", both of\", 1)\n",
      "Pair{AbstractString,Int64}(\"-FU (\", 3)\n",
      "Pair{AbstractString,Int64}(\"and specific\", 2)\n",
      "Pair{AbstractString,Int64}(\". Collectively ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"high similarity between\", 1)\n",
      "Pair{AbstractString,Int64}(\"The copyright\", 147)\n",
      "Pair{AbstractString,Int64}(\"RSCU\", 7)\n",
      "Pair{AbstractString,Int64}(\"5 High-fidelity 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"microtome\", 1)\n",
      "Pair{AbstractString,Int64}(\"NaCl and\", 1)\n",
      "Pair{AbstractString,Int64}(\"lithium bis (\", 1)\n",
      "Pair{AbstractString,Int64}(\"pyramidal cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"obvious degradation\", 1)\n",
      "Pair{AbstractString,Int64}(\"respond\", 2)\n",
      "Pair{AbstractString,Int64}(\"Wilcoxon\", 24)\n",
      "Pair{AbstractString,Int64}(\"confirmed using\", 1)\n",
      "Pair{AbstractString,Int64}(\"We were\", 1)\n",
      "Pair{AbstractString,Int64}(\"\\\" large \\\"\", 2)\n",
      "Pair{AbstractString,Int64}(\"on the same\", 1)\n",
      "Pair{AbstractString,Int64}(\"both cell\", 1)\n",
      "Pair{AbstractString,Int64}(\"; red :\", 1)\n",
      "Pair{AbstractString,Int64}(\"ESI-MS\", 2)\n",
      "Pair{AbstractString,Int64}(\"slight deviations in\", 1)\n",
      "Pair{AbstractString,Int64}(\"503 , we\", 1)\n",
      "Pair{AbstractString,Int64}(\"room for\", 1)\n",
      "Pair{AbstractString,Int64}(\"mM glucose 25\", 1)\n",
      "Pair{AbstractString,Int64}(\"and cloned into\", 1)\n",
      "Pair{AbstractString,Int64}(\"depolarizing\", 1)\n",
      "Pair{AbstractString,Int64}(\"various proteases. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"the mixture. Afterwards\", 1)\n",
      "Pair{AbstractString,Int64}(\"data 219 not\", 1)\n",
      "Pair{AbstractString,Int64}(\"conditions ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"nucleic acids ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"viruses relative to\", 1)\n",
      "Pair{AbstractString,Int64}(\"were presented\", 1)\n",
      "Pair{AbstractString,Int64}(\"serine )\", 1)\n",
      "Pair{AbstractString,Int64}(\"various complexes ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"observations ,\", 1)\n",
      "Pair{AbstractString,Int64}(\". Crystals\", 1)\n",
      "Pair{AbstractString,Int64}(\"tool ( doi\", 1)\n",
      "Pair{AbstractString,Int64}(\", NK\", 1)\n",
      "Pair{AbstractString,Int64}(\"2132\", 1)\n",
      "Pair{AbstractString,Int64}(\"LLC-MK\", 2)\n",
      "Pair{AbstractString,Int64}(\"of HCoV-NL\", 6)\n",
      "Pair{AbstractString,Int64}(\"- 51\", 2)\n",
      "Pair{AbstractString,Int64}(\"data suggest\", 10)\n",
      "Pair{AbstractString,Int64}(\"34 and EV-A\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells but\", 1)\n",
      "Pair{AbstractString,Int64}(\"pro , and\", 11)\n",
      "Pair{AbstractString,Int64}(\"the assessment\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 regions\", 1)\n",
      "Pair{AbstractString,Int64}(\"His at the\", 1)\n",
      "Pair{AbstractString,Int64}(\"evaluated by a\", 1)\n",
      "Pair{AbstractString,Int64}(\"Primary bone\", 1)\n",
      "Pair{AbstractString,Int64}(\"SCRPSY LY\", 1)\n",
      "Pair{AbstractString,Int64}(\"weaker while there\", 1)\n",
      "Pair{AbstractString,Int64}(\"keep the\", 2)\n",
      "Pair{AbstractString,Int64}(\"experiments. The\", 2)\n",
      "Pair{AbstractString,Int64}(\"vast majority\", 1)\n",
      "Pair{AbstractString,Int64}(\"1.45 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"this substituent has\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 and 5\", 1)\n",
      "Pair{AbstractString,Int64}(\"behavior in response\", 2)\n",
      "Pair{AbstractString,Int64}(\"recognize surface lineage\", 1)\n",
      "Pair{AbstractString,Int64}(\"reserved.\", 102)\n",
      "Pair{AbstractString,Int64}(\"port after CS+\", 1)\n",
      "Pair{AbstractString,Int64}(\". The IC\", 1)\n",
      "Pair{AbstractString,Int64}(\"39 ( ref.\", 1)\n",
      "Pair{AbstractString,Int64}(\"2009 ) .\", 2)\n",
      "Pair{AbstractString,Int64}(\"13 or\", 3)\n",
      "Pair{AbstractString,Int64}(\"176 TMPRSS 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"peptide\", 8)\n",
      "Pair{AbstractString,Int64}(\"p= 0.9064\", 1)\n",
      "Pair{AbstractString,Int64}(\"before testing ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"transcriptomic analyses were\", 1)\n",
      "Pair{AbstractString,Int64}(\"bands of\", 2)\n",
      "Pair{AbstractString,Int64}(\"effect of Ly\", 1)\n",
      "Pair{AbstractString,Int64}(\"96 RNA\", 2)\n",
      "Pair{AbstractString,Int64}(\": /\", 1)\n",
      "Pair{AbstractString,Int64}(\"injection , assessed\", 1)\n",
      "Pair{AbstractString,Int64}(\", thus\", 2)\n",
      "Pair{AbstractString,Int64}(\"collect the\", 1)\n",
      "Pair{AbstractString,Int64}(\"13 , whereas\", 1)\n",
      "Pair{AbstractString,Int64}(\"out all\", 1)\n",
      "Pair{AbstractString,Int64}(\"samples with\", 1)\n",
      "Pair{AbstractString,Int64}(\"current-clamp mode by\", 1)\n",
      "Pair{AbstractString,Int64}(\"efforts therefore aimed\", 1)\n",
      "Pair{AbstractString,Int64}(\"placed\", 6)\n",
      "Pair{AbstractString,Int64}(\"( 6 hpi\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 95\", 1)\n",
      "Pair{AbstractString,Int64}(\"sulfate ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"the introduction\", 1)\n",
      "Pair{AbstractString,Int64}(\"assays. Therefore ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"of cells per\", 1)\n",
      "Pair{AbstractString,Int64}(\"nucleotide substitutions ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"HRV 14 in\", 2)\n",
      "Pair{AbstractString,Int64}(\"weak toxicity was\", 1)\n",
      "Pair{AbstractString,Int64}(\"learning resulted\", 1)\n",
      "Pair{AbstractString,Int64}(\") . Both\", 2)\n",
      "Pair{AbstractString,Int64}(\"04 genome was\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells supported\", 1)\n",
      "Pair{AbstractString,Int64}(\"a RNA copy\", 1)\n",
      "Pair{AbstractString,Int64}(\"housekeeping\", 2)\n",
      "Pair{AbstractString,Int64}(\"least 45 minutes\", 1)\n",
      "Pair{AbstractString,Int64}(\"prepared by R\", 4)\n",
      "Pair{AbstractString,Int64}(\"found to\", 4)\n",
      "Pair{AbstractString,Int64}(\"Flex robot according\", 1)\n",
      "Pair{AbstractString,Int64}(\"µm step )\", 1)\n",
      "Pair{AbstractString,Int64}(\"splicing variants\", 1)\n",
      "Pair{AbstractString,Int64}(\"their\", 40)\n",
      "Pair{AbstractString,Int64}(\"wheat surfaces )\", 1)\n",
      "Pair{AbstractString,Int64}(\". B\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 ' and\", 2)\n",
      "Pair{AbstractString,Int64}(\"sections ( median\", 1)\n",
      "Pair{AbstractString,Int64}(\"CC-BY-ND\", 5)\n",
      "Pair{AbstractString,Int64}(\"sample prepared by\", 2)\n",
      "Pair{AbstractString,Int64}(\"reduced presence\", 1)\n",
      "Pair{AbstractString,Int64}(\"with the binding\", 1)\n",
      "Pair{AbstractString,Int64}(\"generated by removal\", 1)\n",
      "Pair{AbstractString,Int64}(\"the same high\", 1)\n",
      "Pair{AbstractString,Int64}(\"Time-course experiments\", 1)\n",
      "Pair{AbstractString,Int64}(\"ppm , δ\", 1)\n",
      "Pair{AbstractString,Int64}(\"clinically. Currently ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"Probe (\", 1)\n",
      "Pair{AbstractString,Int64}(\"of different\", 4)\n",
      "Pair{AbstractString,Int64}(\"The 180\", 1)\n",
      "Pair{AbstractString,Int64}(\"pocket of HCoV-NL\", 1)\n",
      "Pair{AbstractString,Int64}(\"High mutation rates\", 1)\n",
      "Pair{AbstractString,Int64}(\"pCAGGS-mCherry\", 1)\n",
      "Pair{AbstractString,Int64}(\"virus have\", 1)\n",
      "Pair{AbstractString,Int64}(\"pathways to\", 1)\n",
      "Pair{AbstractString,Int64}(\"identify potential pathways\", 1)\n",
      "Pair{AbstractString,Int64}(\"Roy et al.\", 1)\n",
      "Pair{AbstractString,Int64}(\"into the crystal\", 1)\n",
      "Pair{AbstractString,Int64}(\"titer of\", 2)\n",
      "Pair{AbstractString,Int64}(\"was performed with\", 2)\n",
      "Pair{AbstractString,Int64}(\"( serotype 8\", 1)\n",
      "Pair{AbstractString,Int64}(\"were assessed.\", 1)\n",
      "Pair{AbstractString,Int64}(\"has been made\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.3282 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"ExoN-DD\", 1)\n",
      "Pair{AbstractString,Int64}(\"and 100 µM\", 2)\n",
      "Pair{AbstractString,Int64}(\"was performed as\", 3)\n",
      "Pair{AbstractString,Int64}(\"< 0.01\", 3)\n",
      "Pair{AbstractString,Int64}(\"was placed onto\", 2)\n",
      "Pair{AbstractString,Int64}(\"in hippocampal neural\", 1)\n",
      "Pair{AbstractString,Int64}(\"for Firefly or\", 1)\n",
      "Pair{AbstractString,Int64}(\"not covered by\", 1)\n",
      "Pair{AbstractString,Int64}(\"The potential\", 2)\n",
      "Pair{AbstractString,Int64}(\"α-ketoamides Molecular\", 1)\n",
      "Pair{AbstractString,Int64}(\"conditioning was\", 1)\n",
      "Pair{AbstractString,Int64}(\"by CNO (\", 1)\n",
      "Pair{AbstractString,Int64}(\"Nucleoside and\", 1)\n",
      "Pair{AbstractString,Int64}(\"282 showed recently\", 1)\n",
      "Pair{AbstractString,Int64}(\"with differential gene\", 1)\n",
      "Pair{AbstractString,Int64}(\"6 hpi Cell\", 1)\n",
      "Pair{AbstractString,Int64}(\"session is video-recorded\", 1)\n",
      "Pair{AbstractString,Int64}(\"center through offering\", 1)\n",
      "Pair{AbstractString,Int64}(\"clinical detection in\", 1)\n",
      "Pair{AbstractString,Int64}(\"process and until\", 1)\n",
      "Pair{AbstractString,Int64}(\", threonine\", 1)\n",
      "Pair{AbstractString,Int64}(\"which have helped\", 1)\n",
      "Pair{AbstractString,Int64}(\"bioRxiv preprint statistical\", 1)\n",
      "Pair{AbstractString,Int64}(\"non- The copyright\", 1)\n",
      "Pair{AbstractString,Int64}(\". Cytotoxic effects\", 1)\n",
      "Pair{AbstractString,Int64}(\"steps )\", 1)\n",
      "Pair{AbstractString,Int64}(\"convenient surrogate\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"genomic DNA became\", 1)\n",
      "Pair{AbstractString,Int64}(\"RNA in\", 3)\n",
      "Pair{AbstractString,Int64}(\"surprisingly high (\", 1)\n",
      "Pair{AbstractString,Int64}(\"lid is\", 1)\n",
      "Pair{AbstractString,Int64}(\"while only 29\", 1)\n",
      "Pair{AbstractString,Int64}(\"approach to\", 2)\n",
      "Pair{AbstractString,Int64}(\"Compound 2 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"; Wilcoxon\", 2)\n",
      "Pair{AbstractString,Int64}(\"Dynamics simulations 26\", 1)\n",
      "Pair{AbstractString,Int64}(\"pvalues were corrected\", 1)\n",
      "Pair{AbstractString,Int64}(\"% FBS/\", 3)\n",
      "Pair{AbstractString,Int64}(\"protein. Interestingly\", 1)\n",
      "Pair{AbstractString,Int64}(\"11 r by\", 1)\n",
      "Pair{AbstractString,Int64}(\"transcripts ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"P 1 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"being tested. We\", 1)\n",
      "Pair{AbstractString,Int64}(\"substituents must\", 1)\n",
      "Pair{AbstractString,Int64}(\"ORF 1 ab\", 2)\n",
      "Pair{AbstractString,Int64}(\"question the\", 1)\n",
      "Pair{AbstractString,Int64}(\"exhibited submicromolar IC\", 1)\n",
      "Pair{AbstractString,Int64}(\"specific KLK 13\", 1)\n",
      "Pair{AbstractString,Int64}(\"* < 0.01\", 1)\n",
      "Pair{AbstractString,Int64}(\"reagents\", 1)\n",
      "Pair{AbstractString,Int64}(\"or CD 8\", 1)\n",
      "Pair{AbstractString,Int64}(\"RBV were\", 1)\n",
      "Pair{AbstractString,Int64}(\"was 9.07\", 1)\n",
      "Pair{AbstractString,Int64}(\"cellular nucleic\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibitors to\", 3)\n",
      "Pair{AbstractString,Int64}(\") to the\", 1)\n",
      "Pair{AbstractString,Int64}(\"identify of\", 1)\n",
      "Pair{AbstractString,Int64}(\"oxygen ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"E at 33\", 1)\n",
      "Pair{AbstractString,Int64}(\"non-\", 1)\n",
      "Pair{AbstractString,Int64}(\"independent of\", 1)\n",
      "Pair{AbstractString,Int64}(\"member\", 1)\n",
      "Pair{AbstractString,Int64}(\"of detected\", 2)\n",
      "Pair{AbstractString,Int64}(\", 1 =\", 2)\n",
      "Pair{AbstractString,Int64}(\"good to mediocre\", 2)\n",
      "Pair{AbstractString,Int64}(\"the nsp 12\", 2)\n",
      "Pair{AbstractString,Int64}(\"manifestations\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 days post-infection\", 1)\n",
      "Pair{AbstractString,Int64}(\"nt ExoN-AE contained\", 1)\n",
      "Pair{AbstractString,Int64}(\"the temperature\", 3)\n",
      "Pair{AbstractString,Int64}(\"the onset\", 1)\n",
      "Pair{AbstractString,Int64}(\"purified SARS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"that adaptive evolution\", 1)\n",
      "Pair{AbstractString,Int64}(\"cm , H\", 1)\n",
      "Pair{AbstractString,Int64}(\"compounds tested biologically.\", 1)\n",
      "Pair{AbstractString,Int64}(\", p= 0.001\", 2)\n",
      "Pair{AbstractString,Int64}(\"RT )\", 2)\n",
      "Pair{AbstractString,Int64}(\". Differentially\", 1)\n",
      "Pair{AbstractString,Int64}(\"them ,\", 3)\n",
      "Pair{AbstractString,Int64}(\"back-wall ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\"the soaked\", 1)\n",
      "Pair{AbstractString,Int64}(\"but becomes undetectable\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 ZV 9\", 1)\n",
      "Pair{AbstractString,Int64}(\"significance for change\", 1)\n",
      "Pair{AbstractString,Int64}(\"presented 4\", 4)\n",
      "Pair{AbstractString,Int64}(\"and 2 nt\", 2)\n",
      "Pair{AbstractString,Int64}(\"43 countries as\", 1)\n",
      "Pair{AbstractString,Int64}(\"after 428 plating\", 1)\n",
      "Pair{AbstractString,Int64}(\"reduced viral RNA\", 1)\n",
      "Pair{AbstractString,Int64}(\") or HCoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"HCoV- 229 E.\", 1)\n",
      "Pair{AbstractString,Int64}(\"20 µM )\", 1)\n",
      "Pair{AbstractString,Int64}(\"13 should\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 mL\", 4)\n",
      "Pair{AbstractString,Int64}(\"and then transferred\", 1)\n",
      "Pair{AbstractString,Int64}(\"then NaBH\", 1)\n",
      "Pair{AbstractString,Int64}(\"carcinoma cells (\", 1)\n",
      "Pair{AbstractString,Int64}(\"overlaid on\", 1)\n",
      "Pair{AbstractString,Int64}(\"40 41 42\", 4)\n",
      "Pair{AbstractString,Int64}(\"treatment by\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 mM 11\", 1)\n",
      "Pair{AbstractString,Int64}(\"the viral replicons\", 2)\n",
      "Pair{AbstractString,Int64}(\"hours ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"M mometasone. A\", 1)\n",
      "Pair{AbstractString,Int64}(\"showed that\", 12)\n",
      "Pair{AbstractString,Int64}(\"by serum ALT\", 1)\n",
      "Pair{AbstractString,Int64}(\"ratio paired student\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.0119 ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"determined. To determine\", 1)\n",
      "Pair{AbstractString,Int64}(\"individual antiviral\", 1)\n",
      "Pair{AbstractString,Int64}(\"concentrated to 21\", 1)\n",
      "Pair{AbstractString,Int64}(\"competing\", 1)\n",
      "Pair{AbstractString,Int64}(\"No reuse\", 102)\n",
      "Pair{AbstractString,Int64}(\"g , 21.8\", 1)\n",
      "Pair{AbstractString,Int64}(\"P 250 directly\", 1)\n",
      "Pair{AbstractString,Int64}(\"preprint proteases was\", 1)\n",
      "Pair{AbstractString,Int64}(\"revertants are\", 1)\n",
      "Pair{AbstractString,Int64}(\"days later by\", 1)\n",
      "Pair{AbstractString,Int64}(\"Figure 2\", 16)\n",
      "Pair{AbstractString,Int64}(\"cells exhibited low\", 1)\n",
      "Pair{AbstractString,Int64}(\"Green RT-PCR\", 1)\n",
      "Pair{AbstractString,Int64}(\"a consequence of\", 3)\n",
      "Pair{AbstractString,Int64}(\"2012 .\", 1)\n",
      "Pair{AbstractString,Int64}(\"had reduced\", 1)\n",
      "Pair{AbstractString,Int64}(\"using 1.5 mL\", 1)\n",
      "Pair{AbstractString,Int64}(\"-the\", 1)\n",
      "Pair{AbstractString,Int64}(\"reversibly with\", 1)\n",
      "Pair{AbstractString,Int64}(\"In short\", 1)\n",
      "Pair{AbstractString,Int64}(\". Geometric restraints\", 1)\n",
      "Pair{AbstractString,Int64}(\"the candidate\", 1)\n",
      "Pair{AbstractString,Int64}(\"Pseudotyping of\", 1)\n",
      "Pair{AbstractString,Int64}(\"odor or\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 ) SARS-CoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\", dexamethasone ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"of human rhinovirus\", 1)\n",
      "Pair{AbstractString,Int64}(\"proofreading exoribonuclease\", 1)\n",
      "Pair{AbstractString,Int64}(\"the start of\", 1)\n",
      "Pair{AbstractString,Int64}(\"CS-presentation. (\", 1)\n",
      "Pair{AbstractString,Int64}(\", did\", 1)\n",
      "Pair{AbstractString,Int64}(\"transduction efficiency of\", 1)\n",
      "Pair{AbstractString,Int64}(\"only one\", 2)\n",
      "Pair{AbstractString,Int64}(\"313 -RNF 114\", 1)\n",
      "Pair{AbstractString,Int64}(\"Sanger sequencing. Silent\", 1)\n",
      "Pair{AbstractString,Int64}(\"Saffire\", 1)\n",
      "Pair{AbstractString,Int64}(\"the CMV\", 2)\n",
      "Pair{AbstractString,Int64}(\"1 -FRAG\", 1)\n",
      "Pair{AbstractString,Int64}(\"habituation/dishabituation assay (\", 1)\n",
      "Pair{AbstractString,Int64}(\"was only marginally\", 1)\n",
      "Pair{AbstractString,Int64}(\", the sample\", 1)\n",
      "Pair{AbstractString,Int64}(\"V amino\", 1)\n",
      "Pair{AbstractString,Int64}(\"on ice\", 3)\n",
      "Pair{AbstractString,Int64}(\"functional relevance has\", 1)\n",
      "Pair{AbstractString,Int64}(\"C-like protease\", 1)\n",
      "Pair{AbstractString,Int64}(\"indicate a partial\", 1)\n",
      "Pair{AbstractString,Int64}(\"limits CoV entry\", 1)\n",
      "Pair{AbstractString,Int64}(\"derivative of\", 1)\n",
      "Pair{AbstractString,Int64}(\", and concentrated\", 1)\n",
      "Pair{AbstractString,Int64}(\"activity ( 16\", 1)\n",
      "Pair{AbstractString,Int64}(\"Even though cFos-tTA\", 1)\n",
      "Pair{AbstractString,Int64}(\"-glutamine\", 1)\n",
      "Pair{AbstractString,Int64}(\"the Nucleospin RNA\", 1)\n",
      "Pair{AbstractString,Int64}(\"an ExoN-AA\", 1)\n",
      "Pair{AbstractString,Int64}(\"sequence , and\", 1)\n",
      "Pair{AbstractString,Int64}(\"performed one\", 1)\n",
      "Pair{AbstractString,Int64}(\"crystal\", 18)\n",
      "Pair{AbstractString,Int64}(\"The antiviral activity\", 1)\n",
      "Pair{AbstractString,Int64}(\"by Western blot\", 1)\n",
      "Pair{AbstractString,Int64}(\"closely related coronavirus\", 1)\n",
      "Pair{AbstractString,Int64}(\") of\", 21)\n",
      "Pair{AbstractString,Int64}(\"on naïve\", 3)\n",
      "Pair{AbstractString,Int64}(\"to ciclesonide.\", 1)\n",
      "Pair{AbstractString,Int64}(\"of primary reversion\", 1)\n",
      "Pair{AbstractString,Int64}(\"9644 , 481\", 1)\n",
      "Pair{AbstractString,Int64}(\"both experimental (\", 1)\n",
      "Pair{AbstractString,Int64}(\"Activated\", 1)\n",
      "Pair{AbstractString,Int64}(\"a Leica SP\", 2)\n",
      "Pair{AbstractString,Int64}(\"and neurons responding\", 1)\n",
      "Pair{AbstractString,Int64}(\"RevertAid\", 1)\n",
      "Pair{AbstractString,Int64}(\"concentrated and purified\", 4)\n",
      "Pair{AbstractString,Int64}(\"r )\", 2)\n",
      "Pair{AbstractString,Int64}(\"RNA between\", 1)\n",
      "Pair{AbstractString,Int64}(\"of many of\", 1)\n",
      "Pair{AbstractString,Int64}(\"either from the\", 1)\n",
      "Pair{AbstractString,Int64}(\"4.67 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"' (\", 3)\n",
      "Pair{AbstractString,Int64}(\"affinis\", 1)\n",
      "Pair{AbstractString,Int64}(\"laboratory-confirmed symptomatic\", 1)\n",
      "Pair{AbstractString,Int64}(\"Li\", 2)\n",
      "Pair{AbstractString,Int64}(\"the a-ketoamides\", 1)\n",
      "Pair{AbstractString,Int64}(\"rapid\", 3)\n",
      "Pair{AbstractString,Int64}(\"232 mutations\", 1)\n",
      "Pair{AbstractString,Int64}(\"setting up crystallization\", 1)\n",
      "Pair{AbstractString,Int64}(\"belonging to beta-coronavirus\", 1)\n",
      "Pair{AbstractString,Int64}(\"Briefly , 5\", 1)\n",
      "Pair{AbstractString,Int64}(\"discussed in their\", 1)\n",
      "Pair{AbstractString,Int64}(\"Compounds that were\", 1)\n",
      "Pair{AbstractString,Int64}(\"neural activation\", 1)\n",
      "Pair{AbstractString,Int64}(\"followed\", 15)\n",
      "Pair{AbstractString,Int64}(\"more general\", 1)\n",
      "Pair{AbstractString,Int64}(\"correlates with increased\", 1)\n",
      "Pair{AbstractString,Int64}(\"a 2.5\", 1)\n",
      "Pair{AbstractString,Int64}(\"nucleotidylation. Within nsp\", 1)\n",
      "Pair{AbstractString,Int64}(\"shRNAs\", 1)\n",
      "Pair{AbstractString,Int64}(\"preprint of B\", 1)\n",
      "Pair{AbstractString,Int64}(\"p= 0.007 for\", 1)\n",
      "Pair{AbstractString,Int64}(\", its large\", 1)\n",
      "Pair{AbstractString,Int64}(\"by VSV glycoprotein\", 1)\n",
      "Pair{AbstractString,Int64}(\"an odor.\", 1)\n",
      "Pair{AbstractString,Int64}(\"severe and lethal\", 1)\n",
      "Pair{AbstractString,Int64}(\"procedure )\", 7)\n",
      "Pair{AbstractString,Int64}(\"in replication\", 2)\n",
      "Pair{AbstractString,Int64}(\"increases\", 6)\n",
      "Pair{AbstractString,Int64}(\"Rank correlation\", 1)\n",
      "Pair{AbstractString,Int64}(\"this trend\", 2)\n",
      "Pair{AbstractString,Int64}(\"buffer containing 0.1\", 1)\n",
      "Pair{AbstractString,Int64}(\"complex mixed\", 1)\n",
      "Pair{AbstractString,Int64}(\"noted\", 2)\n",
      "Pair{AbstractString,Int64}(\"P\", 128)\n",
      "Pair{AbstractString,Int64}(\"we achieved\", 1)\n",
      "Pair{AbstractString,Int64}(\"fold changes in\", 1)\n",
      "Pair{AbstractString,Int64}(\", H\", 1)\n",
      "Pair{AbstractString,Int64}(\"cFos-tTA-dependent transcripts\", 1)\n",
      "Pair{AbstractString,Int64}(\"SARS-CoV , MERS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"region targeted by\", 1)\n",
      "Pair{AbstractString,Int64}(\"described. 29 In\", 1)\n",
      "Pair{AbstractString,Int64}(\"cannot\", 7)\n",
      "Pair{AbstractString,Int64}(\"but are not\", 1)\n",
      "Pair{AbstractString,Int64}(\"recent\", 3)\n",
      "Pair{AbstractString,Int64}(\"or 40 μ\", 1)\n",
      "Pair{AbstractString,Int64}(\"was confirmed\", 3)\n",
      "Pair{AbstractString,Int64}(\"DCM\", 1)\n",
      "Pair{AbstractString,Int64}(\"11 l\", 2)\n",
      "Pair{AbstractString,Int64}(\"we ectopically\", 1)\n",
      "Pair{AbstractString,Int64}(\"were purchased\", 2)\n",
      "Pair{AbstractString,Int64}(\"One\", 18)\n",
      "Pair{AbstractString,Int64}(\"p= 0.0020 and\", 1)\n",
      "Pair{AbstractString,Int64}(\"For all mice\", 1)\n",
      "Pair{AbstractString,Int64}(\"quantified using trypan\", 2)\n",
      "Pair{AbstractString,Int64}(\"56 ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"open to one\", 2)\n",
      "Pair{AbstractString,Int64}(\"were calculated based\", 2)\n",
      "Pair{AbstractString,Int64}(\"cells and DC\", 1)\n",
      "Pair{AbstractString,Int64}(\"that murine Ly\", 1)\n",
      "Pair{AbstractString,Int64}(\"a )\", 12)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"given 10\", 1)\n",
      "Pair{AbstractString,Int64}(\") was added.\", 2)\n",
      "Pair{AbstractString,Int64}(\"piriform ensembles that\", 4)\n",
      "Pair{AbstractString,Int64}(\"in ExoN- 128\", 1)\n",
      "Pair{AbstractString,Int64}(\"Mandairon\", 1)\n",
      "Pair{AbstractString,Int64}(\"2020 ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"vapor-diffusion sitting-drop method\", 1)\n",
      "Pair{AbstractString,Int64}(\"of positive-strand RNA\", 1)\n",
      "Pair{AbstractString,Int64}(\"are still determined\", 1)\n",
      "Pair{AbstractString,Int64}(\"a mock infection\", 1)\n",
      "Pair{AbstractString,Int64}(\"for increased replication\", 1)\n",
      "Pair{AbstractString,Int64}(\"Samples were\", 2)\n",
      "Pair{AbstractString,Int64}(\"position in the\", 1)\n",
      "Pair{AbstractString,Int64}(\"window is defined\", 1)\n",
      "Pair{AbstractString,Int64}(\"The virus replicated\", 1)\n",
      "Pair{AbstractString,Int64}(\"by host proteases\", 1)\n",
      "Pair{AbstractString,Int64}(\"not male Ly\", 2)\n",
      "Pair{AbstractString,Int64}(\"not carry\", 1)\n",
      "Pair{AbstractString,Int64}(\"author/funder. The\", 10)\n",
      "Pair{AbstractString,Int64}(\"traveller returning from\", 1)\n",
      "Pair{AbstractString,Int64}(\"enzyme during HCoV-HKU\", 1)\n",
      "Pair{AbstractString,Int64}(\"we passaged\", 2)\n",
      "Pair{AbstractString,Int64}(\"replicon ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"Gene identifiers for\", 1)\n",
      "Pair{AbstractString,Int64}(\"- 96\", 1)\n",
      "Pair{AbstractString,Int64}(\"active sampling\", 1)\n",
      "Pair{AbstractString,Int64}(\"5 D )\", 1)\n",
      "Pair{AbstractString,Int64}(\"infection. 153\", 1)\n",
      "Pair{AbstractString,Int64}(\"for other\", 1)\n",
      "Pair{AbstractString,Int64}(\"Ctrl\", 2)\n",
      "Pair{AbstractString,Int64}(\"229 E ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"elsewhere\", 1)\n",
      "Pair{AbstractString,Int64}(\"a hydrogen bond\", 6)\n",
      "Pair{AbstractString,Int64}(\"clinical detection\", 1)\n",
      "Pair{AbstractString,Int64}(\"unpublished ; PDB\", 2)\n",
      "Pair{AbstractString,Int64}(\". 5\", 1)\n",
      "Pair{AbstractString,Int64}(\"allowing 339\", 1)\n",
      "Pair{AbstractString,Int64}(\"infectivity analyses (\", 1)\n",
      "Pair{AbstractString,Int64}(\"Only the\", 1)\n",
      "Pair{AbstractString,Int64}(\"pro s.\", 3)\n",
      "Pair{AbstractString,Int64}(\"DD and\", 1)\n",
      "Pair{AbstractString,Int64}(\"the absence\", 7)\n",
      "Pair{AbstractString,Int64}(\"fully accounted\", 1)\n",
      "Pair{AbstractString,Int64}(\"presented by\", 1)\n",
      "Pair{AbstractString,Int64}(\"excitability.\", 1)\n",
      "Pair{AbstractString,Int64}(\"virus stimulates\", 1)\n",
      "Pair{AbstractString,Int64}(\"upon article publication.\", 1)\n",
      "Pair{AbstractString,Int64}(\"250 author/funder. All\", 1)\n",
      "Pair{AbstractString,Int64}(\"PDB entry\", 8)\n",
      "Pair{AbstractString,Int64}(\"step of\", 1)\n",
      "Pair{AbstractString,Int64}(\"and storage in\", 1)\n",
      "Pair{AbstractString,Int64}(\"viral nucleic acids\", 3)\n",
      "Pair{AbstractString,Int64}(\"female mice. Ly\", 1)\n",
      "Pair{AbstractString,Int64}(\"foot shock does\", 1)\n",
      "Pair{AbstractString,Int64}(\"discovery of\", 1)\n",
      "Pair{AbstractString,Int64}(\"the viral nucleic\", 1)\n",
      "Pair{AbstractString,Int64}(\"was rectangular\", 1)\n",
      "Pair{AbstractString,Int64}(\"Reverse (\", 1)\n",
      "Pair{AbstractString,Int64}(\"study.\", 1)\n",
      "Pair{AbstractString,Int64}(\"CNO in\", 1)\n",
      "Pair{AbstractString,Int64}(\"catalyze nucleotidylation. Within\", 1)\n",
      "Pair{AbstractString,Int64}(\"VEGFC -VRP ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"T in the\", 1)\n",
      "Pair{AbstractString,Int64}(\"were mirrored in\", 1)\n",
      "Pair{AbstractString,Int64}(\", DMRIE-C was\", 1)\n",
      "Pair{AbstractString,Int64}(\"cultures (\", 1)\n",
      "Pair{AbstractString,Int64}(\"changes were 1.00\", 2)\n",
      "Pair{AbstractString,Int64}(\"are impaired when\", 1)\n",
      "Pair{AbstractString,Int64}(\"exhibits very\", 1)\n",
      "Pair{AbstractString,Int64}(\"B and C\", 7)\n",
      "Pair{AbstractString,Int64}(\"types of proteases\", 1)\n",
      "Pair{AbstractString,Int64}(\"Fos-tagged piriform cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"C. Infection was\", 1)\n",
      "Pair{AbstractString,Int64}(\"5.4125\", 1)\n",
      "Pair{AbstractString,Int64}(\"In other words\", 1)\n",
      "Pair{AbstractString,Int64}(\"or CS-\", 2)\n",
      "Pair{AbstractString,Int64}(\"well. After\", 1)\n",
      "Pair{AbstractString,Int64}(\"4.4 µM\", 1)\n",
      "Pair{AbstractString,Int64}(\"i-k )\", 1)\n",
      "Pair{AbstractString,Int64}(\"89 120\", 1)\n",
      "Pair{AbstractString,Int64}(\"changes in sniffing\", 1)\n",
      "Pair{AbstractString,Int64}(\"screening\", 2)\n",
      "Pair{AbstractString,Int64}(\"as Gly ,\", 1)\n",
      "Pair{AbstractString,Int64}(\". Huh\", 2)\n",
      "Pair{AbstractString,Int64}(\"consequences\", 1)\n",
      "Pair{AbstractString,Int64}(\"12 -P\", 4)\n",
      "Pair{AbstractString,Int64}(\"Data were\", 1)\n",
      "Pair{AbstractString,Int64}(\"= benzyl (\", 1)\n",
      "Pair{AbstractString,Int64}(\"target\", 19)\n",
      "Pair{AbstractString,Int64}(\"were chosen\", 1)\n",
      "Pair{AbstractString,Int64}(\"or TMPRSS 2\", 2)\n",
      "Pair{AbstractString,Int64}(\"nucleocapsid ( N\", 2)\n",
      "Pair{AbstractString,Int64}(\"infection experiments with\", 1)\n",
      "Pair{AbstractString,Int64}(\"loading buffer (\", 1)\n",
      "Pair{AbstractString,Int64}(\"mapped separately against\", 1)\n",
      "Pair{AbstractString,Int64}(\"most tolerant among\", 1)\n",
      "Pair{AbstractString,Int64}(\"with mixtures of\", 1)\n",
      "Pair{AbstractString,Int64}(\"7.1 e 4\", 1)\n",
      "Pair{AbstractString,Int64}(\"] from\", 1)\n",
      "Pair{AbstractString,Int64}(\", where\", 5)\n",
      "Pair{AbstractString,Int64}(\"infection by cleavage-resistant\", 1)\n",
      "Pair{AbstractString,Int64}(\"cleaved during\", 2)\n",
      "Pair{AbstractString,Int64}(\"to 3 ′\", 1)\n",
      "Pair{AbstractString,Int64}(\"90 MHV-ExoN-AA can\", 1)\n",
      "Pair{AbstractString,Int64}(\"flexible glutamine\", 1)\n",
      "Pair{AbstractString,Int64}(\"the substituent to\", 1)\n",
      "Pair{AbstractString,Int64}(\"195 196 KLK\", 1)\n",
      "Pair{AbstractString,Int64}(\". For details\", 1)\n",
      "Pair{AbstractString,Int64}(\". Three mutations\", 1)\n",
      "Pair{AbstractString,Int64}(\"25 2020 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"Alamar-Blue™\", 1)\n",
      "Pair{AbstractString,Int64}(\"this insert\", 1)\n",
      "Pair{AbstractString,Int64}(\"degree of\", 2)\n",
      "Pair{AbstractString,Int64}(\"position ,\", 2)\n",
      "Pair{AbstractString,Int64}(\") but\", 1)\n",
      "Pair{AbstractString,Int64}(\"by western\", 1)\n",
      "Pair{AbstractString,Int64}(\"pattern\", 6)\n",
      "Pair{AbstractString,Int64}(\"pA step\", 1)\n",
      "Pair{AbstractString,Int64}(\"these protein-\", 1)\n",
      "Pair{AbstractString,Int64}(\"322\", 1)\n",
      "Pair{AbstractString,Int64}(\"and reduces 855\", 1)\n",
      "Pair{AbstractString,Int64}(\"Analyses\", 1)\n",
      "Pair{AbstractString,Int64}(\"improvement in case\", 1)\n",
      "Pair{AbstractString,Int64}(\"possible that the\", 1)\n",
      "Pair{AbstractString,Int64}(\"the side-chain. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"then acetic\", 1)\n",
      "Pair{AbstractString,Int64}(\"for the catalytic\", 1)\n",
      "Pair{AbstractString,Int64}(\"( green\", 1)\n",
      "Pair{AbstractString,Int64}(\"capped tripeptidyl\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Gossen\", 1)\n",
      "Pair{AbstractString,Int64}(\"while no\", 2)\n",
      "Pair{AbstractString,Int64}(\"Biological. Company ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"cagcacccacgtgctgggcctcgggctcctgc tcagcctgctgtccgccctgctgcggcttggcc cctgaacccagctttcttgtacaaagtggtcccc\", 1)\n",
      "Pair{AbstractString,Int64}(\"would be electrostatic\", 1)\n",
      "Pair{AbstractString,Int64}(\". While\", 4)\n",
      "Pair{AbstractString,Int64}(\"CoVs encode a\", 1)\n",
      "Pair{AbstractString,Int64}(\"of mice to\", 2)\n",
      "Pair{AbstractString,Int64}(\"2019 -nCoV nucleic\", 4)\n",
      "Pair{AbstractString,Int64}(\"0.006 , Wilcoxon\", 1)\n",
      "Pair{AbstractString,Int64}(\"E. LY 6\", 1)\n",
      "Pair{AbstractString,Int64}(\"S protein by\", 1)\n",
      "Pair{AbstractString,Int64}(\"with 1 µg/mL\", 1)\n",
      "Pair{AbstractString,Int64}(\"patterns remains to\", 1)\n",
      "Pair{AbstractString,Int64}(\"10.1101 / 618249\", 10)\n",
      "Pair{AbstractString,Int64}(\"a ( 6\", 1)\n",
      "Pair{AbstractString,Int64}(\"containing 0.1 M\", 3)\n",
      "Pair{AbstractString,Int64}(\"of broadspectrum\", 1)\n",
      "Pair{AbstractString,Int64}(\", consistent with\", 2)\n",
      "Pair{AbstractString,Int64}(\"It is possible\", 1)\n",
      "Pair{AbstractString,Int64}(\"-PRIC\", 1)\n",
      "Pair{AbstractString,Int64}(\"EC 50 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"during odor-foot shock\", 4)\n",
      "Pair{AbstractString,Int64}(\"an asterisk\", 1)\n",
      "Pair{AbstractString,Int64}(\"group cannot\", 1)\n",
      "Pair{AbstractString,Int64}(\"g-lactam. At the\", 1)\n",
      "Pair{AbstractString,Int64}(\") , 2.5\", 1)\n",
      "Pair{AbstractString,Int64}(\"ExoN (\", 5)\n",
      "Pair{AbstractString,Int64}(\"nucleotide substitutions\", 5)\n",
      "Pair{AbstractString,Int64}(\"The authors declare\", 2)\n",
      "Pair{AbstractString,Int64}(\"2 +\", 1)\n",
      "Pair{AbstractString,Int64}(\"The former displayed\", 1)\n",
      "Pair{AbstractString,Int64}(\"4 ) ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"for at least\", 3)\n",
      "Pair{AbstractString,Int64}(\"relationships was\", 1)\n",
      "Pair{AbstractString,Int64}(\"~ 40 %\", 1)\n",
      "Pair{AbstractString,Int64}(\"165 ) residues.\", 1)\n",
      "Pair{AbstractString,Int64}(\", Macherey author/funder.\", 1)\n",
      "Pair{AbstractString,Int64}(\"P 2 benzyl\", 3)\n",
      "Pair{AbstractString,Int64}(\"antagonist of\", 1)\n",
      "Pair{AbstractString,Int64}(\"primase/processivity factor (\", 1)\n",
      "Pair{AbstractString,Int64}(\"gene of\", 3)\n",
      "Pair{AbstractString,Int64}(\"above strongly\", 1)\n",
      "Pair{AbstractString,Int64}(\"neutrophils were also\", 1)\n",
      "Pair{AbstractString,Int64}(\"the host cells\", 1)\n",
      "Pair{AbstractString,Int64}(\", f-g\", 2)\n",
      "Pair{AbstractString,Int64}(\"Bat-SL-CoVZC 45 .\", 2)\n",
      "Pair{AbstractString,Int64}(\"EV-D 68 can\", 1)\n",
      "Pair{AbstractString,Int64}(\"hence the\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 . 288\", 1)\n",
      "Pair{AbstractString,Int64}(\"E 6\", 10)\n",
      "Pair{AbstractString,Int64}(\"1.00 ( 0.41\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 4,000 ×\", 1)\n",
      "Pair{AbstractString,Int64}(\"their researches\", 1)\n",
      "Pair{AbstractString,Int64}(\"5.8.0131 ( ref.\", 1)\n",
      "Pair{AbstractString,Int64}(\"varied from 24\", 1)\n",
      "Pair{AbstractString,Int64}(\"when quantifying the\", 1)\n",
      "Pair{AbstractString,Int64}(\"selective advantage\", 1)\n",
      "Pair{AbstractString,Int64}(\"the replication site\", 1)\n",
      "Pair{AbstractString,Int64}(\"this model\", 2)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"= none ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"symptoms in otherwise\", 1)\n",
      "Pair{AbstractString,Int64}(\"hydrolysis. The\", 1)\n",
      "Pair{AbstractString,Int64}(\". Stock solutions\", 1)\n",
      "Pair{AbstractString,Int64}(\"presentation. Data\", 1)\n",
      "Pair{AbstractString,Int64}(\"and eosin (\", 1)\n",
      "Pair{AbstractString,Int64}(\"to a three-day\", 1)\n",
      "Pair{AbstractString,Int64}(\"acquired with a\", 2)\n",
      "Pair{AbstractString,Int64}(\"per well in\", 2)\n",
      "Pair{AbstractString,Int64}(\") was observed\", 1)\n",
      "Pair{AbstractString,Int64}(\"substitution\", 15)\n",
      "Pair{AbstractString,Int64}(\"influenced by sex\", 1)\n",
      "Pair{AbstractString,Int64}(\"either sex (\", 1)\n",
      "Pair{AbstractString,Int64}(\"formed by Arg\", 1)\n",
      "Pair{AbstractString,Int64}(\"been successfully\", 1)\n",
      "Pair{AbstractString,Int64}(\"sequences were analyzed\", 1)\n",
      "Pair{AbstractString,Int64}(\"variants have\", 1)\n",
      "Pair{AbstractString,Int64}(\"natural selection acts\", 1)\n",
      "Pair{AbstractString,Int64}(\"will not show\", 1)\n",
      "Pair{AbstractString,Int64}(\"Western blot for\", 1)\n",
      "Pair{AbstractString,Int64}(\"we expect\", 1)\n",
      "Pair{AbstractString,Int64}(\"lineage markers ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"analysis between\", 1)\n",
      "Pair{AbstractString,Int64}(\"littermates\", 1)\n",
      "Pair{AbstractString,Int64}(\"methyl ester afforded\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 b )\", 1)\n",
      "Pair{AbstractString,Int64}(\"converted to percentage\", 1)\n",
      "Pair{AbstractString,Int64}(\"mutations such\", 1)\n",
      "Pair{AbstractString,Int64}(\"18 and 0.5\", 1)\n",
      "Pair{AbstractString,Int64}(\"structure , is\", 1)\n",
      "Pair{AbstractString,Int64}(\"We observed\", 2)\n",
      "Pair{AbstractString,Int64}(\"10 4 PFU\", 1)\n",
      "Pair{AbstractString,Int64}(\"membranes ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"ensemble ? For\", 1)\n",
      "Pair{AbstractString,Int64}(\"neutral\", 13)\n",
      "Pair{AbstractString,Int64}(\"tested by quantifying\", 1)\n",
      "Pair{AbstractString,Int64}(\"long-term memory\", 1)\n",
      "Pair{AbstractString,Int64}(\"the enzymatic\", 1)\n",
      "Pair{AbstractString,Int64}(\"p= 0.0592\", 1)\n",
      "Pair{AbstractString,Int64}(\", acc. AY\", 1)\n",
      "Pair{AbstractString,Int64}(\"neighbors above the\", 1)\n",
      "Pair{AbstractString,Int64}(\"Genome\", 1)\n",
      "Pair{AbstractString,Int64}(\"25 2020\", 1)\n",
      "Pair{AbstractString,Int64}(\"glutamine (\", 1)\n",
      "Pair{AbstractString,Int64}(\"1.00 g\", 1)\n",
      "Pair{AbstractString,Int64}(\"et al demonstrated\", 1)\n",
      "Pair{AbstractString,Int64}(\"Experimental Procedures and\", 2)\n",
      "Pair{AbstractString,Int64}(\"via RT-qPCR normalized\", 1)\n",
      "Pair{AbstractString,Int64}(\"suite ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\"the virus infection\", 1)\n",
      "Pair{AbstractString,Int64}(\"50 ) were\", 1)\n",
      "Pair{AbstractString,Int64}(\"affinity\", 1)\n",
      "Pair{AbstractString,Int64}(\"S ,\", 3)\n",
      "Pair{AbstractString,Int64}(\"mL ) and\", 3)\n",
      "Pair{AbstractString,Int64}(\". Similar to\", 1)\n",
      "Pair{AbstractString,Int64}(\"This compound showed\", 1)\n",
      "Pair{AbstractString,Int64}(\"To prepare\", 1)\n",
      "Pair{AbstractString,Int64}(\"and Figure\", 3)\n",
      "Pair{AbstractString,Int64}(\"warrant further\", 1)\n",
      "Pair{AbstractString,Int64}(\"of activation is\", 1)\n",
      "Pair{AbstractString,Int64}(\"503 and incubated\", 1)\n",
      "Pair{AbstractString,Int64}(\"p= 0.0430 ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"clinical inspectors\", 1)\n",
      "Pair{AbstractString,Int64}(\"T-ZNA- 3 -BHQ-\", 1)\n",
      "Pair{AbstractString,Int64}(\"preventing , treating\", 1)\n",
      "Pair{AbstractString,Int64}(\"variation of the\", 1)\n",
      "Pair{AbstractString,Int64}(\"described. 60\", 1)\n",
      "Pair{AbstractString,Int64}(\"1:1 Avicel\", 1)\n",
      "Pair{AbstractString,Int64}(\"Therefore , the\", 1)\n",
      "Pair{AbstractString,Int64}(\"ensembles , similar\", 1)\n",
      "Pair{AbstractString,Int64}(\"complex process\", 1)\n",
      "Pair{AbstractString,Int64}(\", different proteases\", 1)\n",
      "Pair{AbstractString,Int64}(\"; Roland\", 3)\n",
      "Pair{AbstractString,Int64}(\". Catalytic\", 1)\n",
      "Pair{AbstractString,Int64}(\"from tetramethylsilane.\", 1)\n",
      "Pair{AbstractString,Int64}(\"Rice.\", 1)\n",
      "Pair{AbstractString,Int64}(\"complementation. Following\", 1)\n",
      "Pair{AbstractString,Int64}(\"\\\" mode for\", 1)\n",
      "Pair{AbstractString,Int64}(\"bound\", 1)\n",
      "Pair{AbstractString,Int64}(\"for MERS-CoV in\", 1)\n",
      "Pair{AbstractString,Int64}(\"selected\", 4)\n",
      "Pair{AbstractString,Int64}(\"that usually\", 1)\n",
      "Pair{AbstractString,Int64}(\"22 23\", 4)\n",
      "Pair{AbstractString,Int64}(\"amino-acid\", 3)\n",
      "Pair{AbstractString,Int64}(\"in a conformational\", 1)\n",
      "Pair{AbstractString,Int64}(\", we\", 99)\n",
      "Pair{AbstractString,Int64}(\"1 - 3\", 3)\n",
      "Pair{AbstractString,Int64}(\"( HKU 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"plasmid pCAGGS- 229\", 1)\n",
      "Pair{AbstractString,Int64}(\"poor values against\", 1)\n",
      "Pair{AbstractString,Int64}(\"which will favor\", 1)\n",
      "Pair{AbstractString,Int64}(\"described previously\", 10)\n",
      "Pair{AbstractString,Int64}(\"50 = 1.2\", 1)\n",
      "Pair{AbstractString,Int64}(\"Differences in behavioral\", 1)\n",
      "Pair{AbstractString,Int64}(\"DNA using\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Bottom\", 1)\n",
      "Pair{AbstractString,Int64}(\"the WT- 270\", 1)\n",
      "Pair{AbstractString,Int64}(\", we developed\", 1)\n",
      "Pair{AbstractString,Int64}(\"133 stable\", 1)\n",
      "Pair{AbstractString,Int64}(\"Minatohara et\", 1)\n",
      "Pair{AbstractString,Int64}(\"s after\", 3)\n",
      "Pair{AbstractString,Int64}(\"variants ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"kilobase per million\", 1)\n",
      "Pair{AbstractString,Int64}(\"contained 30\", 1)\n",
      "Pair{AbstractString,Int64}(\"two CS-odors were\", 1)\n",
      "Pair{AbstractString,Int64}(\"of BESSY (\", 1)\n",
      "Pair{AbstractString,Int64}(\"to the host\", 1)\n",
      "Pair{AbstractString,Int64}(\"between CS+ and\", 1)\n",
      "Pair{AbstractString,Int64}(\"was performed inside\", 1)\n",
      "Pair{AbstractString,Int64}(\"rely on\", 1)\n",
      "Pair{AbstractString,Int64}(\"assessed by direct\", 1)\n",
      "Pair{AbstractString,Int64}(\"the DREADD ligand\", 2)\n",
      "Pair{AbstractString,Int64}(\"by TLC analysis\", 1)\n",
      "Pair{AbstractString,Int64}(\"U : C\", 3)\n",
      "Pair{AbstractString,Int64}(\"protein by KLK\", 1)\n",
      "Pair{AbstractString,Int64}(\"affecting the detection\", 1)\n",
      "Pair{AbstractString,Int64}(\"loss of Ly\", 1)\n",
      "Pair{AbstractString,Int64}(\"Images\", 2)\n",
      "Pair{AbstractString,Int64}(\"naïve or TMPRSS\", 1)\n",
      "Pair{AbstractString,Int64}(\", we further\", 3)\n",
      "Pair{AbstractString,Int64}(\"protein prevents\", 1)\n",
      "Pair{AbstractString,Int64}(\"the activity-dependent\", 1)\n",
      "Pair{AbstractString,Int64}(\"inflammation was\", 1)\n",
      "Pair{AbstractString,Int64}(\"( GO )\", 1)\n",
      "Pair{AbstractString,Int64}(\"passaged until all\", 1)\n",
      "Pair{AbstractString,Int64}(\"; that ratio\", 2)\n",
      "Pair{AbstractString,Int64}(\"10 13 genome\", 1)\n",
      "Pair{AbstractString,Int64}(\"cleavage. Also here\", 1)\n",
      "Pair{AbstractString,Int64}(\") . PCR\", 1)\n",
      "Pair{AbstractString,Int64}(\"titer\", 5)\n",
      "Pair{AbstractString,Int64}(\"purification. ESI-MS (\", 1)\n",
      "Pair{AbstractString,Int64}(\"197\", 1)\n",
      "Pair{AbstractString,Int64}(\"Spearman\", 1)\n",
      "Pair{AbstractString,Int64}(\", i.e.\", 4)\n",
      "Pair{AbstractString,Int64}(\"inserted into the\", 2)\n",
      "Pair{AbstractString,Int64}(\"image A. C.\", 1)\n",
      "Pair{AbstractString,Int64}(\"commercially available kits\", 1)\n",
      "Pair{AbstractString,Int64}(\"Respiratory\", 1)\n",
      "Pair{AbstractString,Int64}(\"Mice maintained\", 1)\n",
      "Pair{AbstractString,Int64}(\"theoretically achievable\", 1)\n",
      "Pair{AbstractString,Int64}(\"in whole genome\", 1)\n",
      "Pair{AbstractString,Int64}(\"Infectious Peritonitis\", 1)\n",
      "Pair{AbstractString,Int64}(\"n represents\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Shimadzu\", 1)\n",
      "Pair{AbstractString,Int64}(\"at Leiden University\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.05 were considered\", 1)\n",
      "Pair{AbstractString,Int64}(\"is good\", 1)\n",
      "Pair{AbstractString,Int64}(\"2001\", 4)\n",
      "Pair{AbstractString,Int64}(\"inhibiting\", 1)\n",
      "Pair{AbstractString,Int64}(\"usage (\", 3)\n",
      "Pair{AbstractString,Int64}(\"( Cai\", 1)\n",
      "Pair{AbstractString,Int64}(\"= 4\", 2)\n",
      "Pair{AbstractString,Int64}(\"4 cells each\", 1)\n",
      "Pair{AbstractString,Int64}(\"375 only through\", 1)\n",
      "Pair{AbstractString,Int64}(\"ensembles that were\", 5)\n",
      "Pair{AbstractString,Int64}(\") . Gene\", 1)\n",
      "Pair{AbstractString,Int64}(\"in pathway.\", 1)\n",
      "Pair{AbstractString,Int64}(\"Sigma )\", 1)\n",
      "Pair{AbstractString,Int64}(\"165 interface. Apical\", 1)\n",
      "Pair{AbstractString,Int64}(\"free\", 6)\n",
      "Pair{AbstractString,Int64}(\"176\", 2)\n",
      "Pair{AbstractString,Int64}(\"showed any\", 1)\n",
      "Pair{AbstractString,Int64}(\"a-keto carbon\", 1)\n",
      "Pair{AbstractString,Int64}(\"from site\", 1)\n",
      "Pair{AbstractString,Int64}(\"sensitive to\", 1)\n",
      "Pair{AbstractString,Int64}(\"the dN/dS\", 1)\n",
      "Pair{AbstractString,Int64}(\"usage within\", 1)\n",
      "Pair{AbstractString,Int64}(\"the calcium indicator\", 1)\n",
      "Pair{AbstractString,Int64}(\"At 6\", 1)\n",
      "Pair{AbstractString,Int64}(\"encoding the MERS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"increased viral replication.\", 1)\n",
      "Pair{AbstractString,Int64}(\". Cellular\", 1)\n",
      "Pair{AbstractString,Int64}(\"control animals were\", 1)\n",
      "Pair{AbstractString,Int64}(\"of virus.\", 1)\n",
      "Pair{AbstractString,Int64}(\"mg ·\", 3)\n",
      "Pair{AbstractString,Int64}(\"marker\", 1)\n",
      "Pair{AbstractString,Int64}(\"r in Vero\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 min )\", 1)\n",
      "Pair{AbstractString,Int64}(\"the center of\", 1)\n",
      "Pair{AbstractString,Int64}(\") and EV-A\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\", cells were\", 8)\n",
      "Pair{AbstractString,Int64}(\"shifts were\", 1)\n",
      "Pair{AbstractString,Int64}(\"immunohistochemistry\", 1)\n",
      "Pair{AbstractString,Int64}(\"if a projected\", 1)\n",
      "Pair{AbstractString,Int64}(\"The coding\", 1)\n",
      "Pair{AbstractString,Int64}(\"2019 -Novel\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 pocket is\", 3)\n",
      "Pair{AbstractString,Int64}(\"at one end\", 1)\n",
      "Pair{AbstractString,Int64}(\"described the capped\", 1)\n",
      "Pair{AbstractString,Int64}(\", pH 7.5\", 1)\n",
      "Pair{AbstractString,Int64}(\"studies have led\", 1)\n",
      "Pair{AbstractString,Int64}(\"ExoN-AA could conceivably\", 1)\n",
      "Pair{AbstractString,Int64}(\"were transduced\", 3)\n",
      "Pair{AbstractString,Int64}(\"the split-luciferase\", 1)\n",
      "Pair{AbstractString,Int64}(\"that mutations within\", 1)\n",
      "Pair{AbstractString,Int64}(\"odor representations\", 2)\n",
      "Pair{AbstractString,Int64}(\"ImageXpress Micro\", 2)\n",
      "Pair{AbstractString,Int64}(\"causes hepatitis and\", 1)\n",
      "Pair{AbstractString,Int64}(\"from its more\", 1)\n",
      "Pair{AbstractString,Int64}(\"P 250 relative\", 1)\n",
      "Pair{AbstractString,Int64}(\"a resolution\", 1)\n",
      "Pair{AbstractString,Int64}(\"E had\", 2)\n",
      "Pair{AbstractString,Int64}(\"respectively. A crystal\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 that made\", 1)\n",
      "Pair{AbstractString,Int64}(\"After injection ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"paleocortical structure ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"identity in\", 1)\n",
      "Pair{AbstractString,Int64}(\"protein is\", 1)\n",
      "Pair{AbstractString,Int64}(\"while keeping\", 1)\n",
      "Pair{AbstractString,Int64}(\"more versatile\", 1)\n",
      "Pair{AbstractString,Int64}(\"coronaviruses and\", 3)\n",
      "Pair{AbstractString,Int64}(\"impairs S\", 1)\n",
      "Pair{AbstractString,Int64}(\"Ala , or\", 1)\n",
      "Pair{AbstractString,Int64}(\", DC\", 1)\n",
      "Pair{AbstractString,Int64}(\"family members and\", 1)\n",
      "Pair{AbstractString,Int64}(\"spectrophotometer (\", 1)\n",
      "Pair{AbstractString,Int64}(\"oligoadenylate ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"IL\", 1)\n",
      "Pair{AbstractString,Int64}(\"associated\", 8)\n",
      "Pair{AbstractString,Int64}(\"181 occurring\", 1)\n",
      "Pair{AbstractString,Int64}(\"the EV-D 68\", 1)\n",
      "Pair{AbstractString,Int64}(\"- 156 DSP\", 2)\n",
      "Pair{AbstractString,Int64}(\"original design with\", 1)\n",
      "Pair{AbstractString,Int64}(\"then viral supernatant\", 1)\n",
      "Pair{AbstractString,Int64}(\"effects , and\", 1)\n",
      "Pair{AbstractString,Int64}(\"coronavirus , as\", 1)\n",
      "Pair{AbstractString,Int64}(\"are genes that\", 1)\n",
      "Pair{AbstractString,Int64}(\"compensate\", 1)\n",
      "Pair{AbstractString,Int64}(\"our conclusions drawn\", 1)\n",
      "Pair{AbstractString,Int64}(\"compound will have\", 1)\n",
      "Pair{AbstractString,Int64}(\"( CoVs\", 2)\n",
      "Pair{AbstractString,Int64}(\"] [ 8\", 1)\n",
      "Pair{AbstractString,Int64}(\"mutant 10 times\", 1)\n",
      "Pair{AbstractString,Int64}(\"in Hank\", 1)\n",
      "Pair{AbstractString,Int64}(\") in SARS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 93.1 %\", 1)\n",
      "Pair{AbstractString,Int64}(\"a better fit\", 1)\n",
      "Pair{AbstractString,Int64}(\"acc. AY 278741\", 1)\n",
      "Pair{AbstractString,Int64}(\"containing 6 -\", 1)\n",
      "Pair{AbstractString,Int64}(\"purity ; Biosyntan\", 1)\n",
      "Pair{AbstractString,Int64}(\"marker (\", 1)\n",
      "Pair{AbstractString,Int64}(\"1.6 fold\", 1)\n",
      "Pair{AbstractString,Int64}(\") ( 6\", 3)\n",
      "Pair{AbstractString,Int64}(\"pH 6.0 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"His 40 in\", 1)\n",
      "Pair{AbstractString,Int64}(\"by dilution\", 1)\n",
      "Pair{AbstractString,Int64}(\"was measured to\", 1)\n",
      "Pair{AbstractString,Int64}(\"a bacterial artificial\", 1)\n",
      "Pair{AbstractString,Int64}(\"21 and\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.30\", 1)\n",
      "Pair{AbstractString,Int64}(\"-FU can be\", 1)\n",
      "Pair{AbstractString,Int64}(\"in evolution\", 1)\n",
      "Pair{AbstractString,Int64}(\"and against MERS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"reagents and\", 1)\n",
      "Pair{AbstractString,Int64}(\"per supernatant genomic\", 2)\n",
      "Pair{AbstractString,Int64}(\"( 3 C\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Grogg-Chemie AG\", 1)\n",
      "Pair{AbstractString,Int64}(\") per\", 4)\n",
      "Pair{AbstractString,Int64}(\"/ 2020\", 1)\n",
      "Pair{AbstractString,Int64}(\"artificial activity patterns\", 1)\n",
      "Pair{AbstractString,Int64}(\"hole as in\", 1)\n",
      "Pair{AbstractString,Int64}(\") and synonymous\", 1)\n",
      "Pair{AbstractString,Int64}(\"224 substitution in\", 1)\n",
      "Pair{AbstractString,Int64}(\"µM or\", 1)\n",
      "Pair{AbstractString,Int64}(\"2.35 % of\", 1)\n",
      "Pair{AbstractString,Int64}(\"recorded using\", 1)\n",
      "Pair{AbstractString,Int64}(\"5 O\", 1)\n",
      "Pair{AbstractString,Int64}(\"enzymatic\", 2)\n",
      "Pair{AbstractString,Int64}(\"430 All rights\", 1)\n",
      "Pair{AbstractString,Int64}(\"as well (\", 1)\n",
      "Pair{AbstractString,Int64}(\", rupintrivir\", 1)\n",
      "Pair{AbstractString,Int64}(\"moiety was probably\", 1)\n",
      "Pair{AbstractString,Int64}(\"Designer\", 2)\n",
      "Pair{AbstractString,Int64}(\"broad-spectrum activity\", 1)\n",
      "Pair{AbstractString,Int64}(\"collection. After\", 1)\n",
      "Pair{AbstractString,Int64}(\"to 4 sections\", 1)\n",
      "Pair{AbstractString,Int64}(\"used RD 311\", 1)\n",
      "Pair{AbstractString,Int64}(\"behaviors are\", 1)\n",
      "Pair{AbstractString,Int64}(\"the individual viral\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 14 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"binding site of\", 1)\n",
      "Pair{AbstractString,Int64}(\", which has\", 2)\n",
      "Pair{AbstractString,Int64}(\"1 . author/funder.\", 1)\n",
      "Pair{AbstractString,Int64}(\"-LOC 26010 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"JLIGAND 52 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"B ;\", 2)\n",
      "Pair{AbstractString,Int64}(\"79\", 2)\n",
      "Pair{AbstractString,Int64}(\"odor responses and\", 1)\n",
      "Pair{AbstractString,Int64}(\"p ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\"shock pairing\", 1)\n",
      "Pair{AbstractString,Int64}(\"group comparison. Values\", 1)\n",
      "Pair{AbstractString,Int64}(\"was first PCR\", 1)\n",
      "Pair{AbstractString,Int64}(\". Interestingly\", 2)\n",
      "Pair{AbstractString,Int64}(\"4 , 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"mice at both\", 1)\n",
      "Pair{AbstractString,Int64}(\"cortex of\", 2)\n",
      "Pair{AbstractString,Int64}(\"Depending on the\", 1)\n",
      "Pair{AbstractString,Int64}(\"account. Mus Musculus\", 1)\n",
      "Pair{AbstractString,Int64}(\"lowest except for\", 1)\n",
      "Pair{AbstractString,Int64}(\"been thought to\", 1)\n",
      "Pair{AbstractString,Int64}(\"versus that of\", 1)\n",
      "Pair{AbstractString,Int64}(\"anaesthetized with ketamine/xylazine\", 1)\n",
      "Pair{AbstractString,Int64}(\"shock was\", 1)\n",
      "Pair{AbstractString,Int64}(\", 1 µL\", 1)\n",
      "Pair{AbstractString,Int64}(\"odor escape behavior\", 1)\n",
      "Pair{AbstractString,Int64}(\"s instructions. Absorbance\", 1)\n",
      "Pair{AbstractString,Int64}(\"due to non-specific\", 1)\n",
      "Pair{AbstractString,Int64}(\"16 17\", 4)\n",
      "Pair{AbstractString,Int64}(\"SARS-CoV-related\", 1)\n",
      "Pair{AbstractString,Int64}(\"substituents to\", 1)\n",
      "Pair{AbstractString,Int64}(\"reduction\", 3)\n",
      "Pair{AbstractString,Int64}(\"using QuantiFast\", 1)\n",
      "Pair{AbstractString,Int64}(\", 2 way\", 1)\n",
      "Pair{AbstractString,Int64}(\"animals\", 2)\n",
      "Pair{AbstractString,Int64}(\"virus pairs used\", 1)\n",
      "Pair{AbstractString,Int64}(\"each CDS\", 1)\n",
      "Pair{AbstractString,Int64}(\"Our results\", 1)\n",
      "Pair{AbstractString,Int64}(\"seconds after CS+\", 1)\n",
      "Pair{AbstractString,Int64}(\"was increased\", 4)\n",
      "Pair{AbstractString,Int64}(\"morphology was\", 1)\n",
      "Pair{AbstractString,Int64}(\"in cell cultures\", 1)\n",
      "Pair{AbstractString,Int64}(\"pro ( ref.\", 3)\n",
      "Pair{AbstractString,Int64}(\"The relative\", 4)\n",
      "Pair{AbstractString,Int64}(\"151 at\", 1)\n",
      "Pair{AbstractString,Int64}(\"conditioned stimulus (\", 2)\n",
      "Pair{AbstractString,Int64}(\"0.0047 µM in\", 1)\n",
      "Pair{AbstractString,Int64}(\"6627\", 2)\n",
      "Pair{AbstractString,Int64}(\"CNO ( 5\", 2)\n",
      "Pair{AbstractString,Int64}(\"( the difference\", 1)\n",
      "Pair{AbstractString,Int64}(\"the GraphPad\", 1)\n",
      "Pair{AbstractString,Int64}(\"Lipofectamine 2000\", 2)\n",
      "Pair{AbstractString,Int64}(\"analogues ( 6\", 1)\n",
      "Pair{AbstractString,Int64}(\"CoV infection\", 3)\n",
      "Pair{AbstractString,Int64}(\"GlnLactam\", 4)\n",
      "Pair{AbstractString,Int64}(\"reactivation\", 14)\n",
      "Pair{AbstractString,Int64}(\"b )\", 8)\n",
      "Pair{AbstractString,Int64}(\"c ) ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"GFP reporter (\", 1)\n",
      "Pair{AbstractString,Int64}(\") , two-tailed\", 3)\n",
      "Pair{AbstractString,Int64}(\"proteins. Strikingly ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"pairs\", 1)\n",
      "Pair{AbstractString,Int64}(\"' -non-translated\", 1)\n",
      "Pair{AbstractString,Int64}(\"free-enzyme\", 2)\n",
      "Pair{AbstractString,Int64}(\"that the oxygen\", 1)\n",
      "Pair{AbstractString,Int64}(\"among\", 9)\n",
      "Pair{AbstractString,Int64}(\") . Mice\", 4)\n",
      "Pair{AbstractString,Int64}(\"SPINK\", 1)\n",
      "Pair{AbstractString,Int64}(\"the EC\", 9)\n",
      "Pair{AbstractString,Int64}(\"and occasionally\", 1)\n",
      "Pair{AbstractString,Int64}(\"dimethylamino )\", 1)\n",
      "Pair{AbstractString,Int64}(\"% sodium\", 1)\n",
      "Pair{AbstractString,Int64}(\"Tan et al.\", 2)\n",
      "Pair{AbstractString,Int64}(\"the neural inputs\", 1)\n",
      "Pair{AbstractString,Int64}(\"pseudoviruses co-expressing CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"wound healing ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"marked. The necrosis\", 1)\n",
      "Pair{AbstractString,Int64}(\"were dramatically\", 1)\n",
      "Pair{AbstractString,Int64}(\"3.76\", 1)\n",
      "Pair{AbstractString,Int64}(\"clozapine N-oxide (\", 1)\n",
      "Pair{AbstractString,Int64}(\"replacement of\", 2)\n",
      "Pair{AbstractString,Int64}(\"to collect\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells. Each\", 1)\n",
      "Pair{AbstractString,Int64}(\"by the sequential\", 1)\n",
      "Pair{AbstractString,Int64}(\"of APs in\", 1)\n",
      "Pair{AbstractString,Int64}(\"all mice\", 2)\n",
      "Pair{AbstractString,Int64}(\"in MHV genome\", 1)\n",
      "Pair{AbstractString,Int64}(\", ExoN-AE )\", 1)\n",
      "Pair{AbstractString,Int64}(\"emerged\", 1)\n",
      "Pair{AbstractString,Int64}(\"solution until pH\", 2)\n",
      "Pair{AbstractString,Int64}(\"throughout the anterior\", 1)\n",
      "Pair{AbstractString,Int64}(\", mice in\", 1)\n",
      "Pair{AbstractString,Int64}(\"infections. Furthermore ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 position led\", 1)\n",
      "Pair{AbstractString,Int64}(\"based\", 11)\n",
      "Pair{AbstractString,Int64}(\"along with subgenomic\", 1)\n",
      "Pair{AbstractString,Int64}(\"containing the spike\", 1)\n",
      "Pair{AbstractString,Int64}(\"we were unable\", 1)\n",
      "Pair{AbstractString,Int64}(\"gel electrophoresis.\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibitor complexes\", 1)\n",
      "Pair{AbstractString,Int64}(\"lid \\\" in\", 1)\n",
      "Pair{AbstractString,Int64}(\"other odors that\", 1)\n",
      "Pair{AbstractString,Int64}(\"K at synchrotron\", 1)\n",
      "Pair{AbstractString,Int64}(\"- 0.2 mm\", 1)\n",
      "Pair{AbstractString,Int64}(\"have helped\", 1)\n",
      "Pair{AbstractString,Int64}(\") -dimethyl 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"PFA , permeabilized\", 1)\n",
      "Pair{AbstractString,Int64}(\"generalized fear\", 1)\n",
      "Pair{AbstractString,Int64}(\"mock-infected\", 2)\n",
      "Pair{AbstractString,Int64}(\", as calculated\", 1)\n",
      "Pair{AbstractString,Int64}(\"able to culture\", 1)\n",
      "Pair{AbstractString,Int64}(\"learning\", 10)\n",
      "Pair{AbstractString,Int64}(\"transgene.\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"stimuli\", 4)\n",
      "Pair{AbstractString,Int64}(\"enterovirus ( except\", 1)\n",
      "Pair{AbstractString,Int64}(\"129 ED ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"OptiMEM.\", 1)\n",
      "Pair{AbstractString,Int64}(\"inter-trial interval. pin\", 1)\n",
      "Pair{AbstractString,Int64}(\"± 0.15\", 1)\n",
      "Pair{AbstractString,Int64}(\"MHV pathogenesis\", 1)\n",
      "Pair{AbstractString,Int64}(\"within the article\", 1)\n",
      "Pair{AbstractString,Int64}(\"The ester group\", 1)\n",
      "Pair{AbstractString,Int64}(\"on SARS-CoV- 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"a measurable\", 1)\n",
      "Pair{AbstractString,Int64}(\"big advantage over\", 1)\n",
      "Pair{AbstractString,Int64}(\"incubation period\", 1)\n",
      "Pair{AbstractString,Int64}(\"NucleoMag- 96\", 2)\n",
      "Pair{AbstractString,Int64}(\"replicons , 11\", 1)\n",
      "Pair{AbstractString,Int64}(\"the National\", 1)\n",
      "Pair{AbstractString,Int64}(\") . Piriform\", 1)\n",
      "Pair{AbstractString,Int64}(\"Mann-Whitney U\", 4)\n",
      "Pair{AbstractString,Int64}(\"experiment in cFos-tTA\", 1)\n",
      "Pair{AbstractString,Int64}(\"Qiagen , Aarhaus\", 1)\n",
      "Pair{AbstractString,Int64}(\", triethylamine\", 2)\n",
      "Pair{AbstractString,Int64}(\", the enteroviral\", 1)\n",
      "Pair{AbstractString,Int64}(\"the impact\", 2)\n",
      "Pair{AbstractString,Int64}(\"not requiring conformational\", 1)\n",
      "Pair{AbstractString,Int64}(\"338 S 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"infection. KLK 13\", 1)\n",
      "Pair{AbstractString,Int64}(\"9 (\", 4)\n",
      "Pair{AbstractString,Int64}(\"After bath\", 1)\n",
      "Pair{AbstractString,Int64}(\"release (\", 1)\n",
      "Pair{AbstractString,Int64}(\"analog , 5\", 1)\n",
      "Pair{AbstractString,Int64}(\"testing , to\", 1)\n",
      "Pair{AbstractString,Int64}(\"mutations.\", 1)\n",
      "Pair{AbstractString,Int64}(\"evaluated against purified\", 1)\n",
      "Pair{AbstractString,Int64}(\"remains\", 2)\n",
      "Pair{AbstractString,Int64}(\"effect.\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 0.43\", 1)\n",
      "Pair{AbstractString,Int64}(\"6 cells\", 9)\n",
      "Pair{AbstractString,Int64}(\"stereoselective manner ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"application of CNO\", 1)\n",
      "Pair{AbstractString,Int64}(\"19 ) ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"inoculated onto fresh\", 1)\n",
      "Pair{AbstractString,Int64}(\"bond from\", 6)\n",
      "Pair{AbstractString,Int64}(\"Theoretically ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"primary cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"monitored daily\", 1)\n",
      "Pair{AbstractString,Int64}(\", including KLK\", 1)\n",
      "Pair{AbstractString,Int64}(\"gene between WIV\", 1)\n",
      "Pair{AbstractString,Int64}(\"normalized\", 7)\n",
      "Pair{AbstractString,Int64}(\"Accordingly ,\", 1)\n",
      "Pair{AbstractString,Int64}(\") . The\", 67)\n",
      "Pair{AbstractString,Int64}(\"and transfected with\", 1)\n",
      "Pair{AbstractString,Int64}(\"other coronavirus strains\", 1)\n",
      "Pair{AbstractString,Int64}(\") was downloaded\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 0.41\", 1)\n",
      "Pair{AbstractString,Int64}(\"above action potential\", 1)\n",
      "Pair{AbstractString,Int64}(\"antiviral compounds. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"YNA )\", 1)\n",
      "Pair{AbstractString,Int64}(\"fed a\", 1)\n",
      "Pair{AbstractString,Int64}(\". Doxycycline was\", 1)\n",
      "Pair{AbstractString,Int64}(\"SARS-CoV replicon with\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 substituent may\", 1)\n",
      "Pair{AbstractString,Int64}(\"refine the experiments\", 1)\n",
      "Pair{AbstractString,Int64}(\"P values <\", 1)\n",
      "Pair{AbstractString,Int64}(\"228 susceptible\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells stably expressing\", 1)\n",
      "Pair{AbstractString,Int64}(\"c 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"not indicate a\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 µM )\", 6)\n",
      "Pair{AbstractString,Int64}(\"10 4 (\", 3)\n",
      "Pair{AbstractString,Int64}(\"concentration of compound\", 1)\n",
      "Pair{AbstractString,Int64}(\"surface )\", 1)\n",
      "Pair{AbstractString,Int64}(\"million (\", 1)\n",
      "Pair{AbstractString,Int64}(\"the box '\", 1)\n",
      "Pair{AbstractString,Int64}(\"Mice\", 11)\n",
      "Pair{AbstractString,Int64}(\", T\", 2)\n",
      "Pair{AbstractString,Int64}(\"Bottom )\", 1)\n",
      "Pair{AbstractString,Int64}(\"Assays\", 1)\n",
      "Pair{AbstractString,Int64}(\"the severe pneumonia\", 1)\n",
      "Pair{AbstractString,Int64}(\"environmental context\", 1)\n",
      "Pair{AbstractString,Int64}(\"biochemical assays\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells stable\", 1)\n",
      "Pair{AbstractString,Int64}(\"large aromatic\", 1)\n",
      "Pair{AbstractString,Int64}(\"during three\", 1)\n",
      "Pair{AbstractString,Int64}(\"is mainly used\", 1)\n",
      "Pair{AbstractString,Int64}(\"ranged from\", 1)\n",
      "Pair{AbstractString,Int64}(\"elements are main-chain\", 1)\n",
      "Pair{AbstractString,Int64}(\"it is possible\", 2)\n",
      "Pair{AbstractString,Int64}(\"inactivation experiments as\", 1)\n",
      "Pair{AbstractString,Int64}(\"Grogg-Chemie\", 1)\n",
      "Pair{AbstractString,Int64}(\"chloroquine\", 1)\n",
      "Pair{AbstractString,Int64}(\"task to assess\", 1)\n",
      "Pair{AbstractString,Int64}(\"4 ] ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"per supernatant\", 2)\n",
      "Pair{AbstractString,Int64}(\"and then single-housed\", 1)\n",
      "Pair{AbstractString,Int64}(\"- ) P\", 9)\n",
      "Pair{AbstractString,Int64}(\"inhibits 168 virus\", 1)\n",
      "Pair{AbstractString,Int64}(\"the enterovirus\", 13)\n",
      "Pair{AbstractString,Int64}(\"as the exclusive\", 1)\n",
      "Pair{AbstractString,Int64}(\"307 susceptible\", 1)\n",
      "Pair{AbstractString,Int64}(\"the A 25\", 1)\n",
      "Pair{AbstractString,Int64}(\"in our silencing\", 1)\n",
      "Pair{AbstractString,Int64}(\"13 inhibitor\", 3)\n",
      "Pair{AbstractString,Int64}(\"was largely\", 2)\n",
      "Pair{AbstractString,Int64}(\"change is\", 2)\n",
      "Pair{AbstractString,Int64}(\"injected with the\", 1)\n",
      "Pair{AbstractString,Int64}(\"rapid evolution is\", 1)\n",
      "Pair{AbstractString,Int64}(\"no difference was\", 1)\n",
      "Pair{AbstractString,Int64}(\"E as\", 2)\n",
      "Pair{AbstractString,Int64}(\"replication fidelity\", 1)\n",
      "Pair{AbstractString,Int64}(\"P value :\", 2)\n",
      "Pair{AbstractString,Int64}(\"involved in interacting\", 1)\n",
      "Pair{AbstractString,Int64}(\"an expression plasmid\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.1 ) for\", 2)\n",
      "Pair{AbstractString,Int64}(\"gene during the\", 1)\n",
      "Pair{AbstractString,Int64}(\"with Vav\", 1)\n",
      "Pair{AbstractString,Int64}(\"alphacoronaviruses is\", 1)\n",
      "Pair{AbstractString,Int64}(\"propagated virus\", 1)\n",
      "Pair{AbstractString,Int64}(\", viral cell\", 1)\n",
      "Pair{AbstractString,Int64}(\"viral replication detected\", 1)\n",
      "Pair{AbstractString,Int64}(\"implemented\", 1)\n",
      "Pair{AbstractString,Int64}(\"pathogen of Corona\", 1)\n",
      "Pair{AbstractString,Int64}(\"CNO )\", 1)\n",
      "Pair{AbstractString,Int64}(\"replication and host\", 1)\n",
      "Pair{AbstractString,Int64}(\"Metabolic Core. Livers\", 1)\n",
      "Pair{AbstractString,Int64}(\"where natural selection\", 1)\n",
      "Pair{AbstractString,Int64}(\"such interaction\", 1)\n",
      "Pair{AbstractString,Int64}(\"for immunofluorescence\", 1)\n",
      "Pair{AbstractString,Int64}(\"d ) .\", 4)\n",
      "Pair{AbstractString,Int64}(\"site ( Fig.\", 1)\n",
      "Pair{AbstractString,Int64}(\"replication cycle. First\", 1)\n",
      "Pair{AbstractString,Int64}(\", D 2438\", 1)\n",
      "Pair{AbstractString,Int64}(\"developing 63\", 1)\n",
      "Pair{AbstractString,Int64}(\"Omega\", 1)\n",
      "Pair{AbstractString,Int64}(\") . Sections\", 1)\n",
      "Pair{AbstractString,Int64}(\"a conserved functional\", 1)\n",
      "Pair{AbstractString,Int64}(\"nsp 14 that\", 1)\n",
      "Pair{AbstractString,Int64}(\"yl )\", 1)\n",
      "Pair{AbstractString,Int64}(\"33 along\", 1)\n",
      "Pair{AbstractString,Int64}(\"Di-IRES-mCitrine\", 3)\n",
      "Pair{AbstractString,Int64}(\"s in\", 2)\n",
      "Pair{AbstractString,Int64}(\"passed since the\", 1)\n",
      "Pair{AbstractString,Int64}(\"diet. Five\", 1)\n",
      "Pair{AbstractString,Int64}(\"and 1 as\", 1)\n",
      "Pair{AbstractString,Int64}(\"typical coronaviruses\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.17\", 1)\n",
      "Pair{AbstractString,Int64}(\"to 15\", 1)\n",
      "Pair{AbstractString,Int64}(\"were exposed to\", 1)\n",
      "Pair{AbstractString,Int64}(\"~ 150 kDa\", 1)\n",
      "Pair{AbstractString,Int64}(\"Y ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"0.5 µM SARS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"48 , 49\", 1)\n",
      "Pair{AbstractString,Int64}(\"Photinus pyralis\", 1)\n",
      "Pair{AbstractString,Int64}(\"concentration of 1\", 2)\n",
      "Pair{AbstractString,Int64}(\"view of\", 3)\n",
      "Pair{AbstractString,Int64}(\"indicate that at\", 1)\n",
      "Pair{AbstractString,Int64}(\"shown with\", 2)\n",
      "Pair{AbstractString,Int64}(\"100 µL and\", 1)\n",
      "Pair{AbstractString,Int64}(\"or 3 CL\", 1)\n",
      "Pair{AbstractString,Int64}(\"test ( j\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Darwinian\", 1)\n",
      "Pair{AbstractString,Int64}(\"18.5 µM )\", 1)\n",
      "Pair{AbstractString,Int64}(\"mA )\", 1)\n",
      "Pair{AbstractString,Int64}(\"Twenty-four hours\", 1)\n",
      "Pair{AbstractString,Int64}(\"phenotype ( 14\", 1)\n",
      "Pair{AbstractString,Int64}(\"24 h. Then\", 1)\n",
      "Pair{AbstractString,Int64}(\"two variants are\", 1)\n",
      "Pair{AbstractString,Int64}(\"2014 ) ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"generously provided by\", 1)\n",
      "Pair{AbstractString,Int64}(\") . Slides\", 1)\n",
      "Pair{AbstractString,Int64}(\"order to keep\", 1)\n",
      "Pair{AbstractString,Int64}(\"A virus 9\", 1)\n",
      "Pair{AbstractString,Int64}(\"tested ( in\", 1)\n",
      "Pair{AbstractString,Int64}(\"that are not\", 2)\n",
      "Pair{AbstractString,Int64}(\"determine extracellular infectivity\", 1)\n",
      "Pair{AbstractString,Int64}(\"High mutation\", 1)\n",
      "Pair{AbstractString,Int64}(\"d-e , i-j\", 1)\n",
      "Pair{AbstractString,Int64}(\") . Catalytic\", 1)\n",
      "Pair{AbstractString,Int64}(\", IC\", 1)\n",
      "Pair{AbstractString,Int64}(\"mixed together using\", 1)\n",
      "Pair{AbstractString,Int64}(\", 0.45 µM\", 1)\n",
      "Pair{AbstractString,Int64}(\"tube , followed\", 1)\n",
      "Pair{AbstractString,Int64}(\"s of SARS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"elution buffer was\", 1)\n",
      "Pair{AbstractString,Int64}(\"HCoV-HKU 1 replication\", 2)\n",
      "Pair{AbstractString,Int64}(\"growth medium of\", 1)\n",
      "Pair{AbstractString,Int64}(\"to 1 .\", 1)\n",
      "Pair{AbstractString,Int64}(\"a much smaller\", 1)\n",
      "Pair{AbstractString,Int64}(\"of cells\", 6)\n",
      "Pair{AbstractString,Int64}(\"6 c ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"49 .\", 1)\n",
      "Pair{AbstractString,Int64}(\"concentrated and\", 5)\n",
      "Pair{AbstractString,Int64}(\"briefly. The mixture\", 1)\n",
      "Pair{AbstractString,Int64}(\"are used\", 1)\n",
      "Pair{AbstractString,Int64}(\"4 = marked.\", 1)\n",
      "Pair{AbstractString,Int64}(\"range of our\", 1)\n",
      "Pair{AbstractString,Int64}(\"bisulfite\", 1)\n",
      "Pair{AbstractString,Int64}(\"reversed\", 1)\n",
      "Pair{AbstractString,Int64}(\"areas for the\", 1)\n",
      "Pair{AbstractString,Int64}(\"Infected Ly\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"and aldehydes (\", 1)\n",
      "Pair{AbstractString,Int64}(\"a GFP\", 1)\n",
      "Pair{AbstractString,Int64}(\"did not exhibit\", 1)\n",
      "Pair{AbstractString,Int64}(\"at various\", 1)\n",
      "Pair{AbstractString,Int64}(\". Error bars\", 5)\n",
      "Pair{AbstractString,Int64}(\"GT (\", 1)\n",
      "Pair{AbstractString,Int64}(\"providing\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 for\", 2)\n",
      "Pair{AbstractString,Int64}(\"delineated on a\", 1)\n",
      "Pair{AbstractString,Int64}(\"for 1 h\", 1)\n",
      "Pair{AbstractString,Int64}(\", were assayed\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 53 -\", 1)\n",
      "Pair{AbstractString,Int64}(\"Viral mutation rates\", 1)\n",
      "Pair{AbstractString,Int64}(\"protein N.\", 1)\n",
      "Pair{AbstractString,Int64}(\"follows\", 5)\n",
      "Pair{AbstractString,Int64}(\"cleavage. 246 Furthermore\", 1)\n",
      "Pair{AbstractString,Int64}(\"by their\", 2)\n",
      "Pair{AbstractString,Int64}(\"4 R )\", 1)\n",
      "Pair{AbstractString,Int64}(\"ISG screen was\", 1)\n",
      "Pair{AbstractString,Int64}(\"was carried\", 3)\n",
      "Pair{AbstractString,Int64}(\"recording pipettes\", 1)\n",
      "Pair{AbstractString,Int64}(\"12.5 µL\", 1)\n",
      "Pair{AbstractString,Int64}(\"we aim to\", 1)\n",
      "Pair{AbstractString,Int64}(\"using Hank '\", 1)\n",
      "Pair{AbstractString,Int64}(\"their modulation\", 1)\n",
      "Pair{AbstractString,Int64}(\"( DAPI\", 1)\n",
      "Pair{AbstractString,Int64}(\"the floor )\", 1)\n",
      "Pair{AbstractString,Int64}(\". /\", 1)\n",
      "Pair{AbstractString,Int64}(\"7 ,\", 4)\n",
      "Pair{AbstractString,Int64}(\"20 fold\", 1)\n",
      "Pair{AbstractString,Int64}(\"the diet ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"before and after\", 4)\n",
      "Pair{AbstractString,Int64}(\"Hepatic CD\", 1)\n",
      "Pair{AbstractString,Int64}(\"Median Filter\", 1)\n",
      "Pair{AbstractString,Int64}(\"evolved to regulate\", 1)\n",
      "Pair{AbstractString,Int64}(\"explaining the poor\", 1)\n",
      "Pair{AbstractString,Int64}(\"sensitivity was not\", 1)\n",
      "Pair{AbstractString,Int64}(\"To identify potential\", 1)\n",
      "Pair{AbstractString,Int64}(\"the CMV promoter\", 2)\n",
      "Pair{AbstractString,Int64}(\"or 1.5 mL\", 1)\n",
      "Pair{AbstractString,Int64}(\"resistance were\", 1)\n",
      "Pair{AbstractString,Int64}(\"required for virus\", 1)\n",
      "Pair{AbstractString,Int64}(\"at 460\", 1)\n",
      "Pair{AbstractString,Int64}(\"cellular nucleic acids\", 1)\n",
      "Pair{AbstractString,Int64}(\", 25 %\", 4)\n",
      "Pair{AbstractString,Int64}(\"to timely\", 1)\n",
      "Pair{AbstractString,Int64}(\"put\", 1)\n",
      "Pair{AbstractString,Int64}(\", positive\", 1)\n",
      "Pair{AbstractString,Int64}(\". All products\", 1)\n",
      "Pair{AbstractString,Int64}(\"of coronaviral\", 2)\n",
      "Pair{AbstractString,Int64}(\"69 strategy\", 1)\n",
      "Pair{AbstractString,Int64}(\"the corresponding pocket\", 1)\n",
      "Pair{AbstractString,Int64}(\"space\", 6)\n",
      "Pair{AbstractString,Int64}(\"( formed\", 1)\n",
      "Pair{AbstractString,Int64}(\"preservation solution\", 1)\n",
      "Pair{AbstractString,Int64}(\"predicted mechanism for\", 1)\n",
      "Pair{AbstractString,Int64}(\"with EC\", 8)\n",
      "Pair{AbstractString,Int64}(\"Urbani\", 1)\n",
      "Pair{AbstractString,Int64}(\"samples were\", 6)\n",
      "Pair{AbstractString,Int64}(\"-substituent. The compound\", 1)\n",
      "Pair{AbstractString,Int64}(\"for long\", 1)\n",
      "Pair{AbstractString,Int64}(\"author/funder. Cell\", 1)\n",
      "Pair{AbstractString,Int64}(\"-substituent. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"environment ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"Voltage-clamp recordings were\", 1)\n",
      "Pair{AbstractString,Int64}(\"to warm\", 1)\n",
      "Pair{AbstractString,Int64}(\"i.p. ) injected\", 1)\n",
      "Pair{AbstractString,Int64}(\"' s instructions.\", 1)\n",
      "Pair{AbstractString,Int64}(\"cooled\", 1)\n",
      "Pair{AbstractString,Int64}(\"ensemble of neurons\", 1)\n",
      "Pair{AbstractString,Int64}(\"to Bat-SL-CoVZC 45\", 1)\n",
      "Pair{AbstractString,Int64}(\"Alternatively , if\", 1)\n",
      "Pair{AbstractString,Int64}(\"Geometric restraints\", 1)\n",
      "Pair{AbstractString,Int64}(\"stabilization\", 1)\n",
      "Pair{AbstractString,Int64}(\"what caused the\", 1)\n",
      "Pair{AbstractString,Int64}(\"> 15 .\", 1)\n",
      "Pair{AbstractString,Int64}(\", 2 x\", 1)\n",
      "Pair{AbstractString,Int64}(\"this alkylation\", 1)\n",
      "Pair{AbstractString,Int64}(\"about\", 5)\n",
      "Pair{AbstractString,Int64}(\"weight were\", 1)\n",
      "Pair{AbstractString,Int64}(\"DNA-launched\", 1)\n",
      "Pair{AbstractString,Int64}(\") were fed\", 1)\n",
      "Pair{AbstractString,Int64}(\"following odors\", 1)\n",
      "Pair{AbstractString,Int64}(\"- 2 oxopyrrolidin-\", 1)\n",
      "Pair{AbstractString,Int64}(\"( filled in\", 1)\n",
      "Pair{AbstractString,Int64}(\"leave small fragments\", 1)\n",
      "Pair{AbstractString,Int64}(\") Experimental design\", 2)\n",
      "Pair{AbstractString,Int64}(\"( 66\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 1.1 mmol\", 1)\n",
      "Pair{AbstractString,Int64}(\"the main protease\", 2)\n",
      "Pair{AbstractString,Int64}(\"olfactometer\", 1)\n",
      "Pair{AbstractString,Int64}(\"al. 43 described\", 1)\n",
      "Pair{AbstractString,Int64}(\"enteroviruses has\", 1)\n",
      "Pair{AbstractString,Int64}(\"synapses away\", 1)\n",
      "Pair{AbstractString,Int64}(\"Zeng et al.\", 1)\n",
      "Pair{AbstractString,Int64}(\"there is probably\", 1)\n",
      "Pair{AbstractString,Int64}(\"229 E-CoV-Rluc (\", 1)\n",
      "Pair{AbstractString,Int64}(\"fever or radiologic\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 -PAI 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"than that between\", 3)\n",
      "Pair{AbstractString,Int64}(\"strain EMC\", 2)\n",
      "Pair{AbstractString,Int64}(\", 16 .\", 1)\n",
      "Pair{AbstractString,Int64}(\"E ( MOI=\", 1)\n",
      "Pair{AbstractString,Int64}(\"Banerjee et al.\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Supplementary Table\", 1)\n",
      "Pair{AbstractString,Int64}(\"this effect was\", 1)\n",
      "Pair{AbstractString,Int64}(\"cortex is\", 2)\n",
      "Pair{AbstractString,Int64}(\"plan to\", 1)\n",
      "Pair{AbstractString,Int64}(\", where the\", 1)\n",
      "Pair{AbstractString,Int64}(\"that ciclesonide\", 1)\n",
      "Pair{AbstractString,Int64}(\"Figure 5\", 9)\n",
      "Pair{AbstractString,Int64}(\"Fostagged during\", 2)\n",
      "Pair{AbstractString,Int64}(\"RSA 2011 was\", 1)\n",
      "Pair{AbstractString,Int64}(\"4 a was\", 1)\n",
      "Pair{AbstractString,Int64}(\"used\", 51)\n",
      "Pair{AbstractString,Int64}(\"puromycin and were\", 1)\n",
      "Pair{AbstractString,Int64}(\"disrupted. The results\", 1)\n",
      "Pair{AbstractString,Int64}(\"the walls\", 1)\n",
      "Pair{AbstractString,Int64}(\"of 40 μ\", 1)\n",
      "Pair{AbstractString,Int64}(\"2009\", 5)\n",
      "Pair{AbstractString,Int64}(\"; data\", 1)\n",
      "Pair{AbstractString,Int64}(\"recall ( Wilcoxon\", 1)\n",
      "Pair{AbstractString,Int64}(\"index. On\", 1)\n",
      "Pair{AbstractString,Int64}(\"mice. The\", 1)\n",
      "Pair{AbstractString,Int64}(\". For example\", 1)\n",
      "Pair{AbstractString,Int64}(\"shaking for 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"kind of coronavirus\", 1)\n",
      "Pair{AbstractString,Int64}(\"the side-chain of\", 1)\n",
      "Pair{AbstractString,Int64}(\"53 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\") from WIV\", 1)\n",
      "Pair{AbstractString,Int64}(\", diffused\", 1)\n",
      "Pair{AbstractString,Int64}(\"nt ) ,\", 8)\n",
      "Pair{AbstractString,Int64}(\"0.006\", 1)\n",
      "Pair{AbstractString,Int64}(\"none of them\", 1)\n",
      "Pair{AbstractString,Int64}(\"efficiency of at\", 1)\n",
      "Pair{AbstractString,Int64}(\"poor antiviral activities\", 1)\n",
      "Pair{AbstractString,Int64}(\"the amide keto\", 1)\n",
      "Pair{AbstractString,Int64}(\"behavioral response to\", 1)\n",
      "Pair{AbstractString,Int64}(\"infection. Higher viral\", 1)\n",
      "Pair{AbstractString,Int64}(\"that liver damage\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells is important\", 1)\n",
      "Pair{AbstractString,Int64}(\"188\", 10)\n",
      "Pair{AbstractString,Int64}(\"restriction\", 7)\n",
      "Pair{AbstractString,Int64}(\"onto a\", 4)\n",
      "Pair{AbstractString,Int64}(\"the cell debris\", 1)\n",
      "Pair{AbstractString,Int64}(\"olfactory conditioning.\", 1)\n",
      "Pair{AbstractString,Int64}(\"doi : bioRxiv\", 127)\n",
      "Pair{AbstractString,Int64}(\"Bottom\", 1)\n",
      "Pair{AbstractString,Int64}(\"mutation per\", 1)\n",
      "Pair{AbstractString,Int64}(\"7 cells grown\", 1)\n",
      "Pair{AbstractString,Int64}(\"determined by\", 13)\n",
      "Pair{AbstractString,Int64}(\"degraded.\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 nt variants\", 1)\n",
      "Pair{AbstractString,Int64}(\"acquired\", 3)\n",
      "Pair{AbstractString,Int64}(\"s ) for\", 3)\n",
      "Pair{AbstractString,Int64}(\"recently demonstrated to\", 1)\n",
      "Pair{AbstractString,Int64}(\"solvent was evaporated\", 6)\n",
      "Pair{AbstractString,Int64}(\"medium of replicon-transfected\", 1)\n",
      "Pair{AbstractString,Int64}(\"made of\", 1)\n",
      "Pair{AbstractString,Int64}(\"30 ) ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"or proKLK\", 1)\n",
      "Pair{AbstractString,Int64}(\"an anti-HCoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"presenting the odors.\", 1)\n",
      "Pair{AbstractString,Int64}(\"Viral replicons. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"the largest RNA\", 1)\n",
      "Pair{AbstractString,Int64}(\"with complete linkage\", 1)\n",
      "Pair{AbstractString,Int64}(\"S\", 136)\n",
      "Pair{AbstractString,Int64}(\"odor exposure during\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells for\", 1)\n",
      "Pair{AbstractString,Int64}(\"Step U+ Enzyme\", 1)\n",
      "Pair{AbstractString,Int64}(\"averaged ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"was obtained\", 3)\n",
      "Pair{AbstractString,Int64}(\"Testing for\", 1)\n",
      "Pair{AbstractString,Int64}(\"viruses is their\", 1)\n",
      "Pair{AbstractString,Int64}(\"fished and\", 1)\n",
      "Pair{AbstractString,Int64}(\"injection.\", 1)\n",
      "Pair{AbstractString,Int64}(\"one , a-ketoamides\", 1)\n",
      "Pair{AbstractString,Int64}(\"nsp 14 arose\", 1)\n",
      "Pair{AbstractString,Int64}(\"about 180\", 1)\n",
      "Pair{AbstractString,Int64}(\"active Fos-expressing piriform\", 1)\n",
      "Pair{AbstractString,Int64}(\"concentration-dependent\", 2)\n",
      "Pair{AbstractString,Int64}(\"E potently inhibited\", 1)\n",
      "Pair{AbstractString,Int64}(\"role in\", 2)\n",
      "Pair{AbstractString,Int64}(\"one odor\", 1)\n",
      "Pair{AbstractString,Int64}(\"In contrast to\", 1)\n",
      "Pair{AbstractString,Int64}(\"vitro model\", 1)\n",
      "Pair{AbstractString,Int64}(\". In the\", 5)\n",
      "Pair{AbstractString,Int64}(\"greatly in\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\") in DREADD-negative\", 1)\n",
      "Pair{AbstractString,Int64}(\"U+ Mix ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"P 2 .\", 2)\n",
      "Pair{AbstractString,Int64}(\"group-housed with littermates\", 1)\n",
      "Pair{AbstractString,Int64}(\"prior\", 6)\n",
      "Pair{AbstractString,Int64}(\"500 µL )\", 1)\n",
      "Pair{AbstractString,Int64}(\"- 4.36 (\", 1)\n",
      "Pair{AbstractString,Int64}(\", while MHV-ExoN\", 1)\n",
      "Pair{AbstractString,Int64}(\"was determined by\", 6)\n",
      "Pair{AbstractString,Int64}(\". Gene mutations\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Q 222\", 1)\n",
      "Pair{AbstractString,Int64}(\"circles )\", 1)\n",
      "Pair{AbstractString,Int64}(\"-FU )\", 4)\n",
      "Pair{AbstractString,Int64}(\"report the\", 1)\n",
      "Pair{AbstractString,Int64}(\"In the virus-infected\", 1)\n",
      "Pair{AbstractString,Int64}(\"G served as\", 1)\n",
      "Pair{AbstractString,Int64}(\"effect caused\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.5 (\", 2)\n",
      "Pair{AbstractString,Int64}(\"KLK expression\", 1)\n",
      "Pair{AbstractString,Int64}(\"nucleotide changes\", 1)\n",
      "Pair{AbstractString,Int64}(\"of phage DNA\", 1)\n",
      "Pair{AbstractString,Int64}(\") with EC\", 1)\n",
      "Pair{AbstractString,Int64}(\"70 o\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 ,\", 24)\n",
      "Pair{AbstractString,Int64}(\"p= 0.0072 ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"flexibility ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"ice-cold ACSF solution\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Berlin ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"1.75 g ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"In the S\", 1)\n",
      "Pair{AbstractString,Int64}(\"red : Glu\", 1)\n",
      "Pair{AbstractString,Int64}(\"extremity\", 2)\n",
      "Pair{AbstractString,Int64}(\"cytopathic effect caused\", 1)\n",
      "Pair{AbstractString,Int64}(\"these pathways is\", 1)\n",
      "Pair{AbstractString,Int64}(\"specifically inhibits\", 1)\n",
      "Pair{AbstractString,Int64}(\"mg · mL\", 3)\n",
      "Pair{AbstractString,Int64}(\"number of 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"41 ( 1996\", 1)\n",
      "Pair{AbstractString,Int64}(\"of contextual fear\", 1)\n",
      "Pair{AbstractString,Int64}(\"by us (\", 1)\n",
      "Pair{AbstractString,Int64}(\"mutations within\", 1)\n",
      "Pair{AbstractString,Int64}(\", Gln\", 1)\n",
      "Pair{AbstractString,Int64}(\"15 in HeLa\", 1)\n",
      "Pair{AbstractString,Int64}(\"human TMPRRS\", 1)\n",
      "Pair{AbstractString,Int64}(\"preserved in Hank\", 1)\n",
      "Pair{AbstractString,Int64}(\"by KLK 13\", 2)\n",
      "Pair{AbstractString,Int64}(\"together resemble the\", 1)\n",
      "Pair{AbstractString,Int64}(\"-FLuc 34\", 1)\n",
      "Pair{AbstractString,Int64}(\"chromatography on silica\", 2)\n",
      "Pair{AbstractString,Int64}(\"statistically very significant.\", 1)\n",
      "Pair{AbstractString,Int64}(\"statistical differences\", 2)\n",
      "Pair{AbstractString,Int64}(\"292\", 1)\n",
      "Pair{AbstractString,Int64}(\"in spike gene\", 3)\n",
      "Pair{AbstractString,Int64}(\"the para position\", 2)\n",
      "Pair{AbstractString,Int64}(\"-nCoV )\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 21.71\", 1)\n",
      "Pair{AbstractString,Int64}(\"DBTs. This process\", 1)\n",
      "Pair{AbstractString,Int64}(\"and decapitated. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"a histidine residue\", 1)\n",
      "Pair{AbstractString,Int64}(\"2020.02.10.936898 doi\", 5)\n",
      "Pair{AbstractString,Int64}(\"; these\", 1)\n",
      "Pair{AbstractString,Int64}(\"and 2,700\", 1)\n",
      "Pair{AbstractString,Int64}(\", can accept\", 1)\n",
      "Pair{AbstractString,Int64}(\"pervasive episodic selection\", 1)\n",
      "Pair{AbstractString,Int64}(\"Kgluconate , 13\", 1)\n",
      "Pair{AbstractString,Int64}(\"prevents activation (\", 1)\n",
      "Pair{AbstractString,Int64}(\"P 3 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"four\", 4)\n",
      "Pair{AbstractString,Int64}(\"improving the\", 1)\n",
      "Pair{AbstractString,Int64}(\"with the lid\", 1)\n",
      "Pair{AbstractString,Int64}(\"; Kato et\", 1)\n",
      "Pair{AbstractString,Int64}(\"( qRT-PCR\", 2)\n",
      "Pair{AbstractString,Int64}(\"Authors\", 1)\n",
      "Pair{AbstractString,Int64}(\". 206\", 1)\n",
      "Pair{AbstractString,Int64}(\"remained\", 4)\n",
      "Pair{AbstractString,Int64}(\"al. 43\", 1)\n",
      "Pair{AbstractString,Int64}(\", before being\", 1)\n",
      "Pair{AbstractString,Int64}(\"lid and the\", 1)\n",
      "Pair{AbstractString,Int64}(\"at 850\", 1)\n",
      "Pair{AbstractString,Int64}(\"!\", 1)\n",
      "Pair{AbstractString,Int64}(\"CC\", 5)\n",
      "Pair{AbstractString,Int64}(\"tribasic dihydrate ,\", 3)\n",
      "Pair{AbstractString,Int64}(\"frozen. Total\", 1)\n",
      "Pair{AbstractString,Int64}(\"CNO treatment on\", 1)\n",
      "Pair{AbstractString,Int64}(\"a-ketoamides. Synthesis\", 1)\n",
      "Pair{AbstractString,Int64}(\"conditioning.\", 3)\n",
      "Pair{AbstractString,Int64}(\"side and\", 1)\n",
      "Pair{AbstractString,Int64}(\"MHV ( Extended\", 1)\n",
      "Pair{AbstractString,Int64}(\"not CD\", 1)\n",
      "Pair{AbstractString,Int64}(\"were performed\", 9)\n",
      "Pair{AbstractString,Int64}(\", the\", 143)\n",
      "Pair{AbstractString,Int64}(\"a-keto carbon ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"stimulator (\", 1)\n",
      "Pair{AbstractString,Int64}(\"CS-compromises the\", 1)\n",
      "Pair{AbstractString,Int64}(\"μM ) 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"pT 7\", 2)\n",
      "Pair{AbstractString,Int64}(\"is their\", 1)\n",
      "Pair{AbstractString,Int64}(\"39 , Asn\", 2)\n",
      "Pair{AbstractString,Int64}(\"at least\", 8)\n",
      "Pair{AbstractString,Int64}(\"genetics system (\", 1)\n",
      "Pair{AbstractString,Int64}(\"sources. Different\", 1)\n",
      "Pair{AbstractString,Int64}(\"software ( GraphPad\", 1)\n",
      "Pair{AbstractString,Int64}(\", without subsequent\", 1)\n",
      "Pair{AbstractString,Int64}(\"were seeded in\", 10)\n",
      "Pair{AbstractString,Int64}(\"key intermediates 6\", 1)\n",
      "Pair{AbstractString,Int64}(\"5 ) in\", 2)\n",
      "Pair{AbstractString,Int64}(\"codon bias\", 1)\n",
      "Pair{AbstractString,Int64}(\"1024 x 1024\", 1)\n",
      "Pair{AbstractString,Int64}(\") and P\", 1)\n",
      "Pair{AbstractString,Int64}(\"FBS and 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"can be efficiently\", 1)\n",
      "Pair{AbstractString,Int64}(\"RT-PCR Kit (\", 1)\n",
      "Pair{AbstractString,Int64}(\"478\", 1)\n",
      "Pair{AbstractString,Int64}(\"were injected\", 3)\n",
      "Pair{AbstractString,Int64}(\"these Fos-tagged\", 1)\n",
      "Pair{AbstractString,Int64}(\"20 ℃ using\", 1)\n",
      "Pair{AbstractString,Int64}(\"for understanding\", 1)\n",
      "Pair{AbstractString,Int64}(\"encode a\", 1)\n",
      "Pair{AbstractString,Int64}(\"behaviorally relevant\", 1)\n",
      "Pair{AbstractString,Int64}(\"TNT ; Zhu\", 1)\n",
      "Pair{AbstractString,Int64}(\"earlier (\", 1)\n",
      "Pair{AbstractString,Int64}(\"networks apply\", 1)\n",
      "Pair{AbstractString,Int64}(\"30 31 32\", 4)\n",
      "Pair{AbstractString,Int64}(\"pro and SARS-CoV\", 2)\n",
      "Pair{AbstractString,Int64}(\"before formaldehyde\", 1)\n",
      "Pair{AbstractString,Int64}(\"is large\", 1)\n",
      "Pair{AbstractString,Int64}(\"that basic\", 1)\n",
      "Pair{AbstractString,Int64}(\"functional relevance\", 1)\n",
      "Pair{AbstractString,Int64}(\"( c\", 1)\n",
      "Pair{AbstractString,Int64}(\"5 minutes was\", 1)\n",
      "Pair{AbstractString,Int64}(\"extremity of\", 1)\n",
      "Pair{AbstractString,Int64}(\"activity during\", 1)\n",
      "Pair{AbstractString,Int64}(\"ΔHSC mice of\", 1)\n",
      "Pair{AbstractString,Int64}(\"issued\", 1)\n",
      "Pair{AbstractString,Int64}(\"preprint Procedures\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 side-chain\", 1)\n",
      "Pair{AbstractString,Int64}(\"be sterically\", 1)\n",
      "Pair{AbstractString,Int64}(\"from one\", 2)\n",
      "Pair{AbstractString,Int64}(\"citrate\", 3)\n",
      "Pair{AbstractString,Int64}(\"( EnSpire 2300\", 1)\n",
      "Pair{AbstractString,Int64}(\"velocity were significantly\", 1)\n",
      "Pair{AbstractString,Int64}(\"38 39 40\", 4)\n",
      "Pair{AbstractString,Int64}(\"Differentially expressed\", 1)\n",
      "Pair{AbstractString,Int64}(\"their researches ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"of registered\", 1)\n",
      "Pair{AbstractString,Int64}(\"re-activation\", 1)\n",
      "Pair{AbstractString,Int64}(\"generate the\", 2)\n",
      "Pair{AbstractString,Int64}(\"Fos-tagged during odor-foot\", 1)\n",
      "Pair{AbstractString,Int64}(\"mg/mL.\", 1)\n",
      "Pair{AbstractString,Int64}(\"one end\", 1)\n",
      "Pair{AbstractString,Int64}(\"these viruses (\", 1)\n",
      "Pair{AbstractString,Int64}(\". Most\", 1)\n",
      "Pair{AbstractString,Int64}(\"means\", 2)\n",
      "Pair{AbstractString,Int64}(\"rapidly succumbed\", 1)\n",
      "Pair{AbstractString,Int64}(\"member of\", 1)\n",
      "Pair{AbstractString,Int64}(\". Figure\", 2)\n",
      "Pair{AbstractString,Int64}(\"time course\", 1)\n",
      "Pair{AbstractString,Int64}(\"HeLa cells.\", 1)\n",
      "Pair{AbstractString,Int64}(\"control group (\", 2)\n",
      "Pair{AbstractString,Int64}(\"dimensions\", 1)\n",
      "Pair{AbstractString,Int64}(\"( f )\", 2)\n",
      "Pair{AbstractString,Int64}(\"preprint Quantitative\", 1)\n",
      "Pair{AbstractString,Int64}(\"C pro residues\", 1)\n",
      "Pair{AbstractString,Int64}(\"TRIzol Reagent\", 2)\n",
      "Pair{AbstractString,Int64}(\"enteroviral replicons ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"N-terminal Edman degradation\", 1)\n",
      "Pair{AbstractString,Int64}(\". Besides\", 1)\n",
      "Pair{AbstractString,Int64}(\"bromoacetonitrile (\", 1)\n",
      "Pair{AbstractString,Int64}(\"to ciclesonide. A\", 1)\n",
      "Pair{AbstractString,Int64}(\"are processed into\", 1)\n",
      "Pair{AbstractString,Int64}(\"μL reservoir\", 1)\n",
      "Pair{AbstractString,Int64}(\"460 nm with\", 1)\n",
      "Pair{AbstractString,Int64}(\"described previously 60\", 1)\n",
      "Pair{AbstractString,Int64}(\"TM ( Hampton\", 1)\n",
      "Pair{AbstractString,Int64}(\", and 229\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells. To determine\", 1)\n",
      "Pair{AbstractString,Int64}(\"found\", 21)\n",
      "Pair{AbstractString,Int64}(\"the results equivocal.\", 1)\n",
      "Pair{AbstractString,Int64}(\"36.84 ±\", 1)\n",
      "Pair{AbstractString,Int64}(\"escape to\", 1)\n",
      "Pair{AbstractString,Int64}(\"SARS-CoV even\", 1)\n",
      "Pair{AbstractString,Int64}(\"and CNO injection\", 1)\n",
      "Pair{AbstractString,Int64}(\"6 e\", 44)\n",
      "Pair{AbstractString,Int64}(\"directly lysed\", 1)\n",
      "Pair{AbstractString,Int64}(\"Three mutations are\", 1)\n",
      "Pair{AbstractString,Int64}(\"passaged viruses with\", 1)\n",
      "Pair{AbstractString,Int64}(\"has fundamentally\", 1)\n",
      "Pair{AbstractString,Int64}(\"bioRxiv preprint separated\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 rounds\", 2)\n",
      "Pair{AbstractString,Int64}(\"more quickly\", 2)\n",
      "Pair{AbstractString,Int64}(\"lentivirus production and\", 1)\n",
      "Pair{AbstractString,Int64}(\"knew\", 1)\n",
      "Pair{AbstractString,Int64}(\"indistinguishable from DREADD-negative\", 1)\n",
      "Pair{AbstractString,Int64}(\"tested 124\", 1)\n",
      "Pair{AbstractString,Int64}(\"in ImageJ. A\", 1)\n",
      "Pair{AbstractString,Int64}(\"( μM\", 1)\n",
      "Pair{AbstractString,Int64}(\"conclusion ,\", 3)\n",
      "Pair{AbstractString,Int64}(\"of the usual\", 1)\n",
      "Pair{AbstractString,Int64}(\"on HCoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"temperature inactivation\", 2)\n",
      "Pair{AbstractString,Int64}(\"hole 518\", 1)\n",
      "Pair{AbstractString,Int64}(\"and a goat\", 1)\n",
      "Pair{AbstractString,Int64}(\"containing 20 mM\", 1)\n",
      "Pair{AbstractString,Int64}(\"pro .\", 20)\n",
      "Pair{AbstractString,Int64}(\"remained unchanged (\", 1)\n",
      "Pair{AbstractString,Int64}(\", 16 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"481\", 1)\n",
      "Pair{AbstractString,Int64}(\"Candidate\", 2)\n",
      "Pair{AbstractString,Int64}(\", then perturbing\", 1)\n",
      "Pair{AbstractString,Int64}(\"One of\", 2)\n",
      "Pair{AbstractString,Int64}(\"for memory retrieval\", 2)\n",
      "Pair{AbstractString,Int64}(\"Cai et\", 1)\n",
      "Pair{AbstractString,Int64}(\"or DC\", 1)\n",
      "Pair{AbstractString,Int64}(\"after CS+ (\", 1)\n",
      "Pair{AbstractString,Int64}(\"µL reservoir composed\", 1)\n",
      "Pair{AbstractString,Int64}(\"air-liquid 165\", 1)\n",
      "Pair{AbstractString,Int64}(\"-b\", 2)\n",
      "Pair{AbstractString,Int64}(\"Cytotoxic effects caused\", 1)\n",
      "Pair{AbstractString,Int64}(\"EV-D\", 10)\n",
      "Pair{AbstractString,Int64}(\"exposure , we\", 1)\n",
      "Pair{AbstractString,Int64}(\"corticosteroids , which\", 1)\n",
      "Pair{AbstractString,Int64}(\"two steps )\", 1)\n",
      "Pair{AbstractString,Int64}(\"the properties\", 2)\n",
      "Pair{AbstractString,Int64}(\"( i.e.\", 2)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"values against\", 4)\n",
      "Pair{AbstractString,Int64}(\"drug targeting MERS-and\", 1)\n",
      "Pair{AbstractString,Int64}(\"Single rod-like crystals\", 1)\n",
      "Pair{AbstractString,Int64}(\"12,000 ×g ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"0.5 h at\", 1)\n",
      "Pair{AbstractString,Int64}(\"12,000 × g\", 1)\n",
      "Pair{AbstractString,Int64}(\"CellCapture\", 1)\n",
      "Pair{AbstractString,Int64}(\"pseudoparticle (\", 1)\n",
      "Pair{AbstractString,Int64}(\"PEG 6,000 ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"abolish the learned\", 1)\n",
      "Pair{AbstractString,Int64}(\"were chemogenetically\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Fleischmann\", 1)\n",
      "Pair{AbstractString,Int64}(\"MHV-nsp 14 .\", 1)\n",
      "Pair{AbstractString,Int64}(\"pink carbon\", 1)\n",
      "Pair{AbstractString,Int64}(\"into cells.\", 1)\n",
      "Pair{AbstractString,Int64}(\": beta-citronellol\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 substituent.\", 1)\n",
      "Pair{AbstractString,Int64}(\"and Franks ,\", 4)\n",
      "Pair{AbstractString,Int64}(\"189 , whereas\", 1)\n",
      "Pair{AbstractString,Int64}(\"ketamine/xylazine\", 1)\n",
      "Pair{AbstractString,Int64}(\"therefore also been\", 1)\n",
      "Pair{AbstractString,Int64}(\"marrow derived\", 1)\n",
      "Pair{AbstractString,Int64}(\"virus infection by\", 1)\n",
      "Pair{AbstractString,Int64}(\"stained with DAPI.\", 1)\n",
      "Pair{AbstractString,Int64}(\"2004\", 3)\n",
      "Pair{AbstractString,Int64}(\"= > 40\", 1)\n",
      "Pair{AbstractString,Int64}(\"coronavirus ( MERS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"-A 25 V\", 1)\n",
      "Pair{AbstractString,Int64}(\"protected by cryo-protectant\", 1)\n",
      "Pair{AbstractString,Int64}(\"two structures\", 1)\n",
      "Pair{AbstractString,Int64}(\"in changes\", 1)\n",
      "Pair{AbstractString,Int64}(\"NK cells\", 3)\n",
      "Pair{AbstractString,Int64}(\"11 r in\", 2)\n",
      "Pair{AbstractString,Int64}(\"virus 9 .\", 1)\n",
      "Pair{AbstractString,Int64}(\"addition , the\", 2)\n",
      "Pair{AbstractString,Int64}(\"compounds , which\", 1)\n",
      "Pair{AbstractString,Int64}(\"the stack is\", 1)\n",
      "Pair{AbstractString,Int64}(\"p= 0.0054\", 3)\n",
      "Pair{AbstractString,Int64}(\"7 a -b\", 1)\n",
      "Pair{AbstractString,Int64}(\"Wentworth ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"cFos production is\", 1)\n",
      "Pair{AbstractString,Int64}(\"transcription factor ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"315 ( M\", 1)\n",
      "Pair{AbstractString,Int64}(\"pathogen\", 2)\n",
      "Pair{AbstractString,Int64}(\"28 RA\", 1)\n",
      "Pair{AbstractString,Int64}(\"determined using one-way\", 1)\n",
      "Pair{AbstractString,Int64}(\"p= 0.6842 ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"by an aversive\", 1)\n",
      "Pair{AbstractString,Int64}(\"( from Korea\", 1)\n",
      "Pair{AbstractString,Int64}(\"amino-acid esters 4\", 1)\n",
      "Pair{AbstractString,Int64}(\", 200\", 1)\n",
      "Pair{AbstractString,Int64}(\"372\", 1)\n",
      "Pair{AbstractString,Int64}(\"virus , the\", 1)\n",
      "Pair{AbstractString,Int64}(\"and base analogs\", 1)\n",
      "Pair{AbstractString,Int64}(\"acid.\", 2)\n",
      "Pair{AbstractString,Int64}(\"Top ) Experimental\", 1)\n",
      "Pair{AbstractString,Int64}(\"DREADDnegative\", 2)\n",
      "Pair{AbstractString,Int64}(\"the Firefly\", 1)\n",
      "Pair{AbstractString,Int64}(\"CleavEx\", 4)\n",
      "Pair{AbstractString,Int64}(\"protein by\", 1)\n",
      "Pair{AbstractString,Int64}(\"groups , Mann-Whitney\", 1)\n",
      "Pair{AbstractString,Int64}(\"220 mutations are\", 1)\n",
      "Pair{AbstractString,Int64}(\"first line of\", 1)\n",
      "Pair{AbstractString,Int64}(\"325 virus entry\", 1)\n",
      "Pair{AbstractString,Int64}(\"blood\", 1)\n",
      "Pair{AbstractString,Int64}(\"or below against\", 1)\n",
      "Pair{AbstractString,Int64}(\"genome ) replication\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 51\", 1)\n",
      "Pair{AbstractString,Int64}(\", Takara\", 1)\n",
      "Pair{AbstractString,Int64}(\"40 µM\", 3)\n",
      "Pair{AbstractString,Int64}(\"a-ketoamides were evaluated\", 1)\n",
      "Pair{AbstractString,Int64}(\"4 Di ;\", 1)\n",
      "Pair{AbstractString,Int64}(\") clustered\", 1)\n",
      "Pair{AbstractString,Int64}(\"two 2 nt\", 1)\n",
      "Pair{AbstractString,Int64}(\"and CVB 3\", 9)\n",
      "Pair{AbstractString,Int64}(\"expressed the HKU\", 1)\n",
      "Pair{AbstractString,Int64}(\"evaporated and the\", 5)\n",
      "Pair{AbstractString,Int64}(\"computed over\", 1)\n",
      "Pair{AbstractString,Int64}(\"performed using\", 7)\n",
      "Pair{AbstractString,Int64}(\"the antiviral target\", 1)\n",
      "Pair{AbstractString,Int64}(\"was used for\", 5)\n",
      "Pair{AbstractString,Int64}(\"( methanol/water ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"revert within just\", 1)\n",
      "Pair{AbstractString,Int64}(\"respectively (\", 1)\n",
      "Pair{AbstractString,Int64}(\"Finally\", 9)\n",
      "Pair{AbstractString,Int64}(\"software.\", 2)\n",
      "Pair{AbstractString,Int64}(\"the C\", 1)\n",
      "Pair{AbstractString,Int64}(\"the cultures\", 1)\n",
      "Pair{AbstractString,Int64}(\"Furthermore , the\", 1)\n",
      "Pair{AbstractString,Int64}(\"-well plate high\", 1)\n",
      "Pair{AbstractString,Int64}(\"D , Table\", 1)\n",
      "Pair{AbstractString,Int64}(\"130 and His\", 1)\n",
      "Pair{AbstractString,Int64}(\"in enterovirus-infected cell\", 1)\n",
      "Pair{AbstractString,Int64}(\"they allow\", 1)\n",
      "Pair{AbstractString,Int64}(\"toll\", 1)\n",
      "Pair{AbstractString,Int64}(\". Multiple cell\", 1)\n",
      "Pair{AbstractString,Int64}(\"Upon ambient exposure\", 1)\n",
      "Pair{AbstractString,Int64}(\"synonymous codon\", 2)\n",
      "Pair{AbstractString,Int64}(\"succumbed\", 1)\n",
      "Pair{AbstractString,Int64}(\"restoration of\", 1)\n",
      "Pair{AbstractString,Int64}(\"; no difference\", 1)\n",
      "Pair{AbstractString,Int64}(\"revealed that the\", 1)\n",
      "Pair{AbstractString,Int64}(\"Predicting the likely\", 1)\n",
      "Pair{AbstractString,Int64}(\"objects detected in\", 1)\n",
      "Pair{AbstractString,Int64}(\"Fluor\", 1)\n",
      "Pair{AbstractString,Int64}(\"evolution might contribute\", 1)\n",
      "Pair{AbstractString,Int64}(\"this purpose\", 2)\n",
      "Pair{AbstractString,Int64}(\"two proteins. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"at 4 o\", 5)\n",
      "Pair{AbstractString,Int64}(\"mM sodium\", 2)\n",
      "Pair{AbstractString,Int64}(\"using CellCapture\", 1)\n",
      "Pair{AbstractString,Int64}(\"and 2.1 µM\", 1)\n",
      "Pair{AbstractString,Int64}(\"complement of\", 1)\n",
      "Pair{AbstractString,Int64}(\"NaCl. After\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells and significantly\", 2)\n",
      "Pair{AbstractString,Int64}(\"Viruses\", 1)\n",
      "Pair{AbstractString,Int64}(\"we i.p.\", 1)\n",
      "Pair{AbstractString,Int64}(\"4 h at\", 1)\n",
      "Pair{AbstractString,Int64}(\"described three\", 1)\n",
      "Pair{AbstractString,Int64}(\"The P 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"and subsequently selected\", 1)\n",
      "Pair{AbstractString,Int64}(\"GlnLactam ) in\", 1)\n",
      "Pair{AbstractString,Int64}(\"MES , 7.5\", 1)\n",
      "Pair{AbstractString,Int64}(\"task\", 1)\n",
      "Pair{AbstractString,Int64}(\"supposed\", 1)\n",
      "Pair{AbstractString,Int64}(\"by the 310\", 1)\n",
      "Pair{AbstractString,Int64}(\"evaporated to dryness\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.03 µM (\", 1)\n",
      "Pair{AbstractString,Int64}(\"PDB entry 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"not\", 222)\n",
      "Pair{AbstractString,Int64}(\"pellet suspension\", 1)\n",
      "Pair{AbstractString,Int64}(\"product 4 was\", 1)\n",
      "Pair{AbstractString,Int64}(\"immediately\", 3)\n",
      "Pair{AbstractString,Int64}(\"3 ( 30\", 1)\n",
      "Pair{AbstractString,Int64}(\"residues from the\", 1)\n",
      "Pair{AbstractString,Int64}(\"E-mediated restriction of\", 1)\n",
      "Pair{AbstractString,Int64}(\"nucleotide substitutions are\", 1)\n",
      "Pair{AbstractString,Int64}(\"min at 4\", 1)\n",
      "Pair{AbstractString,Int64}(\"an IC 50\", 1)\n",
      "Pair{AbstractString,Int64}(\"culture. 148\", 1)\n",
      "Pair{AbstractString,Int64}(\"attenuating\", 1)\n",
      "Pair{AbstractString,Int64}(\"aldehydes ( -CH=O\", 1)\n",
      "Pair{AbstractString,Int64}(\"C. 11\", 1)\n",
      "Pair{AbstractString,Int64}(\"YM\", 1)\n",
      "Pair{AbstractString,Int64}(\"attenuated compared\", 2)\n",
      "Pair{AbstractString,Int64}(\", The olfactory\", 1)\n",
      "Pair{AbstractString,Int64}(\"A numerical score\", 1)\n",
      "Pair{AbstractString,Int64}(\"was generated.\", 1)\n",
      "Pair{AbstractString,Int64}(\"ns= 0.0740 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"11 f bound\", 1)\n",
      "Pair{AbstractString,Int64}(\"released\", 1)\n",
      "Pair{AbstractString,Int64}(\"6.0 g\", 1)\n",
      "Pair{AbstractString,Int64}(\"CNO ,\", 3)\n",
      "Pair{AbstractString,Int64}(\"the 520 corresponding\", 1)\n",
      "Pair{AbstractString,Int64}(\"passage in\", 1)\n",
      "Pair{AbstractString,Int64}(\"Training always started\", 1)\n",
      "Pair{AbstractString,Int64}(\"20 kHz\", 1)\n",
      "Pair{AbstractString,Int64}(\"towards\", 5)\n",
      "Pair{AbstractString,Int64}(\"PDE\", 1)\n",
      "Pair{AbstractString,Int64}(\", from\", 1)\n",
      "Pair{AbstractString,Int64}(\"Python\", 1)\n",
      "Pair{AbstractString,Int64}(\"frequently reported\", 1)\n",
      "Pair{AbstractString,Int64}(\"All inhibitors\", 1)\n",
      "Pair{AbstractString,Int64}(\"14 EV-A 71\", 1)\n",
      "Pair{AbstractString,Int64}(\"' s solutions\", 1)\n",
      "Pair{AbstractString,Int64}(\"their half-maximal\", 1)\n",
      "Pair{AbstractString,Int64}(\"most probably\", 2)\n",
      "Pair{AbstractString,Int64}(\"Middle East.\", 1)\n",
      "Pair{AbstractString,Int64}(\"accompanying\", 1)\n",
      "Pair{AbstractString,Int64}(\"113 proofreading\", 1)\n",
      "Pair{AbstractString,Int64}(\"vesicular stomatitis\", 1)\n",
      "Pair{AbstractString,Int64}(\"One NS mutation\", 1)\n",
      "Pair{AbstractString,Int64}(\"100 µM\", 3)\n",
      "Pair{AbstractString,Int64}(\"8.67 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 -yl\", 1)\n",
      "Pair{AbstractString,Int64}(\"or 1.5\", 1)\n",
      "Pair{AbstractString,Int64}(\"kind gift\", 3)\n",
      "Pair{AbstractString,Int64}(\"were soaked\", 1)\n",
      "Pair{AbstractString,Int64}(\"commercially available\", 3)\n",
      "Pair{AbstractString,Int64}(\"induction of\", 2)\n",
      "Pair{AbstractString,Int64}(\"extracted by\", 1)\n",
      "Pair{AbstractString,Int64}(\"mM MgCl 2\", 2)\n",
      "Pair{AbstractString,Int64}(\"SPINK 6\", 1)\n",
      "Pair{AbstractString,Int64}(\"the amygdala (\", 1)\n",
      "Pair{AbstractString,Int64}(\"see\", 15)\n",
      "Pair{AbstractString,Int64}(\"M pro is\", 5)\n",
      "Pair{AbstractString,Int64}(\"cells 57 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"AAVs ) expressing\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells were infected\", 1)\n",
      "Pair{AbstractString,Int64}(\"cyclohexyl group\", 1)\n",
      "Pair{AbstractString,Int64}(\"suppress the innate\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"Reagents were purchased\", 1)\n",
      "Pair{AbstractString,Int64}(\"the inhibitors to\", 2)\n",
      "Pair{AbstractString,Int64}(\"effects of steroids\", 1)\n",
      "Pair{AbstractString,Int64}(\"diluted with\", 2)\n",
      "Pair{AbstractString,Int64}(\"areas\", 3)\n",
      "Pair{AbstractString,Int64}(\"2 that\", 1)\n",
      "Pair{AbstractString,Int64}(\"Overall ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"mycoplasma\", 1)\n",
      "Pair{AbstractString,Int64}(\"300 µM ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"partially compensate for\", 1)\n",
      "Pair{AbstractString,Int64}(\"and 316 that\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Bolding and\", 4)\n",
      "Pair{AbstractString,Int64}(\"siblings that did\", 1)\n",
      "Pair{AbstractString,Int64}(\"dec-RVKR-CMK\", 1)\n",
      "Pair{AbstractString,Int64}(\"( e.g.\", 2)\n",
      "Pair{AbstractString,Int64}(\"PCR ( qRT-PCR\", 1)\n",
      "Pair{AbstractString,Int64}(\"resistance ciclesonide\", 1)\n",
      "Pair{AbstractString,Int64}(\"previously resistant\", 1)\n",
      "Pair{AbstractString,Int64}(\"per kilobase per\", 1)\n",
      "Pair{AbstractString,Int64}(\"the remaining passages.\", 1)\n",
      "Pair{AbstractString,Int64}(\"The side-chain\", 1)\n",
      "Pair{AbstractString,Int64}(\"GC-\", 1)\n",
      "Pair{AbstractString,Int64}(\"-Ctreference ) and\", 1)\n",
      "Pair{AbstractString,Int64}(\"Bat-SL-CoVZXC\", 1)\n",
      "Pair{AbstractString,Int64}(\"100 % DMSO.\", 1)\n",
      "Pair{AbstractString,Int64}(\"SYBR Green RT-PCR\", 1)\n",
      "Pair{AbstractString,Int64}(\"selection , we\", 1)\n",
      "Pair{AbstractString,Int64}(\"Automated\", 1)\n",
      "Pair{AbstractString,Int64}(\"- 2148\", 1)\n",
      "Pair{AbstractString,Int64}(\"was added. Cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"the patterns of\", 1)\n",
      "Pair{AbstractString,Int64}(\"was evaluated. VeroE\", 1)\n",
      "Pair{AbstractString,Int64}(\"which LY\", 1)\n",
      "Pair{AbstractString,Int64}(\"131 All\", 1)\n",
      "Pair{AbstractString,Int64}(\"training , the\", 1)\n",
      "Pair{AbstractString,Int64}(\"241 pro-forms that\", 1)\n",
      "Pair{AbstractString,Int64}(\"bis ( dimethylamino\", 1)\n",
      "Pair{AbstractString,Int64}(\"The read-out\", 1)\n",
      "Pair{AbstractString,Int64}(\"= 9.8 µM\", 1)\n",
      "Pair{AbstractString,Int64}(\"DECTRIS ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\"inactivated at 92\", 2)\n",
      "Pair{AbstractString,Int64}(\"or as single\", 1)\n",
      "Pair{AbstractString,Int64}(\"nsp 14 ,\", 3)\n",
      "Pair{AbstractString,Int64}(\"Fig. 6 )\", 1)\n",
      "Pair{AbstractString,Int64}(\", treatment\", 1)\n",
      "Pair{AbstractString,Int64}(\"using the MEGA-X\", 1)\n",
      "Pair{AbstractString,Int64}(\"= cyclopentylmethyl\", 2)\n",
      "Pair{AbstractString,Int64}(\") . 254\", 1)\n",
      "Pair{AbstractString,Int64}(\", the medium\", 1)\n",
      "Pair{AbstractString,Int64}(\"furin ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"61 )\", 1)\n",
      "Pair{AbstractString,Int64}(\". Determination\", 1)\n",
      "Pair{AbstractString,Int64}(\"substantially between P\", 1)\n",
      "Pair{AbstractString,Int64}(\"from a spatially\", 1)\n",
      "Pair{AbstractString,Int64}(\"yield\", 7)\n",
      "Pair{AbstractString,Int64}(\"Assay System (\", 1)\n",
      "Pair{AbstractString,Int64}(\"fivemembered g-lactam.\", 1)\n",
      "Pair{AbstractString,Int64}(\"a neural ensemble\", 1)\n",
      "Pair{AbstractString,Int64}(\"429\", 1)\n",
      "Pair{AbstractString,Int64}(\"value for SARS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"355 different proteases\", 1)\n",
      "Pair{AbstractString,Int64}(\"extensive 58\", 1)\n",
      "Pair{AbstractString,Int64}(\"should be .\", 1)\n",
      "Pair{AbstractString,Int64}(\") 1 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"are in the\", 1)\n",
      "Pair{AbstractString,Int64}(\"1989 ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"challenge for preventing\", 1)\n",
      "Pair{AbstractString,Int64}(\"however , we\", 1)\n",
      "Pair{AbstractString,Int64}(\"codons\", 2)\n",
      "Pair{AbstractString,Int64}(\"layers of\", 1)\n",
      "Pair{AbstractString,Int64}(\"lethal respiratory disease\", 1)\n",
      "Pair{AbstractString,Int64}(\"we decided to\", 4)\n",
      "Pair{AbstractString,Int64}(\"and contextual memory\", 1)\n",
      "Pair{AbstractString,Int64}(\"an ExoN-AA mutation\", 1)\n",
      "Pair{AbstractString,Int64}(\"synthesis associated with\", 1)\n",
      "Pair{AbstractString,Int64}(\"exceeds the ability\", 1)\n",
      "Pair{AbstractString,Int64}(\"discovering peptidomimetic\", 1)\n",
      "Pair{AbstractString,Int64}(\"Nterminal\", 1)\n",
      "Pair{AbstractString,Int64}(\"the benzyl\", 2)\n",
      "Pair{AbstractString,Int64}(\"and experimental data\", 1)\n",
      "Pair{AbstractString,Int64}(\"WT-revertants\", 2)\n",
      "Pair{AbstractString,Int64}(\"harmful ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"ΔHSC BMDM\", 1)\n",
      "Pair{AbstractString,Int64}(\"- 0.9\", 1)\n",
      "Pair{AbstractString,Int64}(\"bioRxiv preprint at\", 1)\n",
      "Pair{AbstractString,Int64}(\"516 viruses were\", 1)\n",
      "Pair{AbstractString,Int64}(\"activate\", 2)\n",
      "Pair{AbstractString,Int64}(\"of RNA isolated\", 1)\n",
      "Pair{AbstractString,Int64}(\"6 E expressing\", 7)\n",
      "Pair{AbstractString,Int64}(\"plasticity ( Tayler\", 1)\n",
      "Pair{AbstractString,Int64}(\"into S 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"small inserts and\", 1)\n",
      "Pair{AbstractString,Int64}(\"has been described\", 2)\n",
      "Pair{AbstractString,Int64}(\"in 2003 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"manner to\", 1)\n",
      "Pair{AbstractString,Int64}(\") . Apparently\", 1)\n",
      "Pair{AbstractString,Int64}(\"CNO to reactivate\", 1)\n",
      "Pair{AbstractString,Int64}(\"method. Several\", 1)\n",
      "Pair{AbstractString,Int64}(\"flash column\", 1)\n",
      "Pair{AbstractString,Int64}(\"min together\", 1)\n",
      "Pair{AbstractString,Int64}(\"cRPMI 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"during the recording.\", 1)\n",
      "Pair{AbstractString,Int64}(\"than 0.05\", 1)\n",
      "Pair{AbstractString,Int64}(\"give the product\", 4)\n",
      "Pair{AbstractString,Int64}(\"likely results 62\", 1)\n",
      "Pair{AbstractString,Int64}(\"to bregma.\", 1)\n",
      "Pair{AbstractString,Int64}(\"6 , W=-\", 2)\n",
      "Pair{AbstractString,Int64}(\"specific mutation\", 1)\n",
      "Pair{AbstractString,Int64}(\", 40 µM\", 1)\n",
      "Pair{AbstractString,Int64}(\"for one hour\", 1)\n",
      "Pair{AbstractString,Int64}(\"not significant.\", 2)\n",
      "Pair{AbstractString,Int64}(\"= mild\", 1)\n",
      "Pair{AbstractString,Int64}(\"superior : P\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 /S 2\", 3)\n",
      "Pair{AbstractString,Int64}(\", Cys\", 1)\n",
      "Pair{AbstractString,Int64}(\"compound is\", 2)\n",
      "Pair{AbstractString,Int64}(\"KLK 13 inhibitor.\", 1)\n",
      "Pair{AbstractString,Int64}(\"cell subsets\", 1)\n",
      "Pair{AbstractString,Int64}(\"DNA and\", 3)\n",
      "Pair{AbstractString,Int64}(\"25 , 30\", 1)\n",
      "Pair{AbstractString,Int64}(\"temperature on\", 1)\n",
      "Pair{AbstractString,Int64}(\"collection. After centrifuge\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.0779 . author/funder.\", 1)\n",
      "Pair{AbstractString,Int64}(\"with a lentiviral\", 1)\n",
      "Pair{AbstractString,Int64}(\"samples were resuspended\", 1)\n",
      "Pair{AbstractString,Int64}(\"half\", 2)\n",
      "Pair{AbstractString,Int64}(\"ExoN- 128\", 1)\n",
      "Pair{AbstractString,Int64}(\"Thiel (\", 1)\n",
      "Pair{AbstractString,Int64}(\"cages throughout\", 1)\n",
      "Pair{AbstractString,Int64}(\"serotype\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 12\", 3)\n",
      "Pair{AbstractString,Int64}(\"4 Di-mediated neural\", 1)\n",
      "Pair{AbstractString,Int64}(\"B ) During\", 1)\n",
      "Pair{AbstractString,Int64}(\"is determined\", 1)\n",
      "Pair{AbstractString,Int64}(\"to Volker\", 1)\n",
      "Pair{AbstractString,Int64}(\"cytotoxic for the\", 1)\n",
      "Pair{AbstractString,Int64}(\"fusion and\", 1)\n",
      "Pair{AbstractString,Int64}(\"Here , we\", 2)\n",
      "Pair{AbstractString,Int64}(\"days but\", 1)\n",
      "Pair{AbstractString,Int64}(\"replicons at a\", 1)\n",
      "Pair{AbstractString,Int64}(\"AAV-tetO-hM\", 6)\n",
      "Pair{AbstractString,Int64}(\"cells per cubic\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.6454\", 1)\n",
      "Pair{AbstractString,Int64}(\"significant.\", 4)\n",
      "Pair{AbstractString,Int64}(\"of mice that\", 2)\n",
      "Pair{AbstractString,Int64}(\"we generated transgenic\", 1)\n",
      "Pair{AbstractString,Int64}(\"capped dipeptide\", 1)\n",
      "Pair{AbstractString,Int64}(\"enthalpy. Accordingly\", 1)\n",
      "Pair{AbstractString,Int64}(\"it\", 26)\n",
      "Pair{AbstractString,Int64}(\"observation of the\", 1)\n",
      "Pair{AbstractString,Int64}(\"degree of inflammation\", 1)\n",
      "Pair{AbstractString,Int64}(\"analyses to a\", 1)\n",
      "Pair{AbstractString,Int64}(\"samples , and\", 1)\n",
      "Pair{AbstractString,Int64}(\"PBS ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"RT-PCR Kit\", 1)\n",
      "Pair{AbstractString,Int64}(\"two was less\", 1)\n",
      "Pair{AbstractString,Int64}(\"electrophoresis 5\", 1)\n",
      "Pair{AbstractString,Int64}(\"amounts of viral\", 1)\n",
      "Pair{AbstractString,Int64}(\"The quality of\", 1)\n",
      "Pair{AbstractString,Int64}(\"Considering\", 1)\n",
      "Pair{AbstractString,Int64}(\"o , the\", 1)\n",
      "Pair{AbstractString,Int64}(\"of the genus\", 1)\n",
      "Pair{AbstractString,Int64}(\"Rz )\", 1)\n",
      "Pair{AbstractString,Int64}(\"crystals was soaked\", 1)\n",
      "Pair{AbstractString,Int64}(\"pro :\", 3)\n",
      "Pair{AbstractString,Int64}(\"thereby confirming the\", 1)\n",
      "Pair{AbstractString,Int64}(\"cassette instead of\", 1)\n",
      "Pair{AbstractString,Int64}(\"Cbz-\", 1)\n",
      "Pair{AbstractString,Int64}(\", 3.0 days\", 1)\n",
      "Pair{AbstractString,Int64}(\"We observed that\", 1)\n",
      "Pair{AbstractString,Int64}(\"sequences were translated\", 1)\n",
      "Pair{AbstractString,Int64}(\"between Pro\", 1)\n",
      "Pair{AbstractString,Int64}(\"as the RNA\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 - 9\", 1)\n",
      "Pair{AbstractString,Int64}(\"activity drives HCoV-HKU\", 1)\n",
      "Pair{AbstractString,Int64}(\"the basis\", 2)\n",
      "Pair{AbstractString,Int64}(\"Motif\", 2)\n",
      "Pair{AbstractString,Int64}(\"that piriform neural\", 1)\n",
      "Pair{AbstractString,Int64}(\"Berlin ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"Primer Reverse (\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 within MHV-ExoN\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.0112 , Figure\", 1)\n",
      "Pair{AbstractString,Int64}(\"escape behavior :\", 1)\n",
      "Pair{AbstractString,Int64}(\"to IC\", 1)\n",
      "Pair{AbstractString,Int64}(\"figure )\", 1)\n",
      "Pair{AbstractString,Int64}(\"Supernatant\", 2)\n",
      "Pair{AbstractString,Int64}(\"the capped\", 1)\n",
      "Pair{AbstractString,Int64}(\"enterovirus proteases :\", 1)\n",
      "Pair{AbstractString,Int64}(\"the PEDV RNA\", 1)\n",
      "Pair{AbstractString,Int64}(\"at 252\", 1)\n",
      "Pair{AbstractString,Int64}(\"Dunnett ' s\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"sequenced 2019 -nCoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"with the antibiotic\", 1)\n",
      "Pair{AbstractString,Int64}(\"eluted\", 3)\n",
      "Pair{AbstractString,Int64}(\"50 values one\", 1)\n",
      "Pair{AbstractString,Int64}(\"to get\", 1)\n",
      "Pair{AbstractString,Int64}(\"-well plate. One\", 2)\n",
      "Pair{AbstractString,Int64}(\"we verified its\", 1)\n",
      "Pair{AbstractString,Int64}(\"NTR )\", 1)\n",
      "Pair{AbstractString,Int64}(\"this paper.\", 1)\n",
      "Pair{AbstractString,Int64}(\"intervention against\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 = His\", 1)\n",
      "Pair{AbstractString,Int64}(\"have a significant\", 1)\n",
      "Pair{AbstractString,Int64}(\"RdRps (\", 1)\n",
      "Pair{AbstractString,Int64}(\"pro shows\", 1)\n",
      "Pair{AbstractString,Int64}(\"cases is\", 2)\n",
      "Pair{AbstractString,Int64}(\"coronavirus main protease.\", 1)\n",
      "Pair{AbstractString,Int64}(\", which silences\", 1)\n",
      "Pair{AbstractString,Int64}(\"and CRISPR-resistant\", 1)\n",
      "Pair{AbstractString,Int64}(\"our data clearly\", 1)\n",
      "Pair{AbstractString,Int64}(\"in different\", 2)\n",
      "Pair{AbstractString,Int64}(\"This result shows\", 1)\n",
      "Pair{AbstractString,Int64}(\"suggest a conserved\", 1)\n",
      "Pair{AbstractString,Int64}(\"that they\", 4)\n",
      "Pair{AbstractString,Int64}(\"isolated according\", 1)\n",
      "Pair{AbstractString,Int64}(\"pseudo-randomized : testing\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 1.67\", 1)\n",
      "Pair{AbstractString,Int64}(\"SARS-CoV- 2 RNA\", 1)\n",
      "Pair{AbstractString,Int64}(\"harbouring\", 2)\n",
      "Pair{AbstractString,Int64}(\"stem\", 1)\n",
      "Pair{AbstractString,Int64}(\"known as 2019\", 1)\n",
      "Pair{AbstractString,Int64}(\"( contoured at\", 1)\n",
      "Pair{AbstractString,Int64}(\"protein 14 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"32 While\", 1)\n",
      "Pair{AbstractString,Int64}(\"behaviors characteristic\", 1)\n",
      "Pair{AbstractString,Int64}(\"the Patchmaster\", 1)\n",
      "Pair{AbstractString,Int64}(\"odor presentations\", 2)\n",
      "Pair{AbstractString,Int64}(\"in turn ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"potential fusion 331\", 1)\n",
      "Pair{AbstractString,Int64}(\"only the\", 2)\n",
      "Pair{AbstractString,Int64}(\"Two substrates with\", 1)\n",
      "Pair{AbstractString,Int64}(\"adjacent\", 3)\n",
      "Pair{AbstractString,Int64}(\"glycoprotein\", 4)\n",
      "Pair{AbstractString,Int64}(\"50.11 % of\", 1)\n",
      "Pair{AbstractString,Int64}(\"400 MHz )\", 1)\n",
      "Pair{AbstractString,Int64}(\", indicating a\", 1)\n",
      "Pair{AbstractString,Int64}(\"CD 13 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"model predicts\", 1)\n",
      "Pair{AbstractString,Int64}(\"though (\", 1)\n",
      "Pair{AbstractString,Int64}(\"were averaged for\", 1)\n",
      "Pair{AbstractString,Int64}(\"for 15 min\", 1)\n",
      "Pair{AbstractString,Int64}(\"artificial shortage of\", 1)\n",
      "Pair{AbstractString,Int64}(\"potential relative\", 1)\n",
      "Pair{AbstractString,Int64}(\"boxes ( 40\", 1)\n",
      "Pair{AbstractString,Int64}(\"N-terminal\", 2)\n",
      "Pair{AbstractString,Int64}(\"made possible\", 1)\n",
      "Pair{AbstractString,Int64}(\"virus. .\", 1)\n",
      "Pair{AbstractString,Int64}(\"per well.\", 1)\n",
      "Pair{AbstractString,Int64}(\"the reactivation\", 1)\n",
      "Pair{AbstractString,Int64}(\"realistic undertaking. While\", 1)\n",
      "Pair{AbstractString,Int64}(\"-CH=CH-CO- )\", 1)\n",
      "Pair{AbstractString,Int64}(\"inspectors from\", 2)\n",
      "Pair{AbstractString,Int64}(\"9.4\", 1)\n",
      "Pair{AbstractString,Int64}(\", showed\", 1)\n",
      "Pair{AbstractString,Int64}(\"Extended methods ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"ΔHSC mice were\", 2)\n",
      "Pair{AbstractString,Int64}(\"For HCoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"reported when comparing\", 1)\n",
      "Pair{AbstractString,Int64}(\"picomolar !\", 1)\n",
      "Pair{AbstractString,Int64}(\"drives expression of\", 1)\n",
      "Pair{AbstractString,Int64}(\"RPKM )\", 1)\n",
      "Pair{AbstractString,Int64}(\"for preventing ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"20 μM\", 1)\n",
      "Pair{AbstractString,Int64}(\"lysates of Huh\", 1)\n",
      "Pair{AbstractString,Int64}(\"thiohemiketal accepts a\", 1)\n",
      "Pair{AbstractString,Int64}(\"fluorescence. Figure 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"overlapping PCR.\", 1)\n",
      "Pair{AbstractString,Int64}(\"of RNA viruses\", 2)\n",
      "Pair{AbstractString,Int64}(\"the intermediates 4\", 1)\n",
      "Pair{AbstractString,Int64}(\"beamline BL 14.2\", 1)\n",
      "Pair{AbstractString,Int64}(\"the target range\", 1)\n",
      "Pair{AbstractString,Int64}(\"Dq ) ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"DMRIE-C was\", 1)\n",
      "Pair{AbstractString,Int64}(\"the pocket is\", 1)\n",
      "Pair{AbstractString,Int64}(\"5 -fluorouracil\", 5)\n",
      "Pair{AbstractString,Int64}(\"12.5\", 1)\n",
      "Pair{AbstractString,Int64}(\"One Step\", 3)\n",
      "Pair{AbstractString,Int64}(\"reducing the amount\", 1)\n",
      "Pair{AbstractString,Int64}(\"mean ±\", 1)\n",
      "Pair{AbstractString,Int64}(\"kind of\", 1)\n",
      "Pair{AbstractString,Int64}(\"conferred\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.42\", 1)\n",
      "Pair{AbstractString,Int64}(\"necessary and\", 3)\n",
      "Pair{AbstractString,Int64}(\"2010 )\", 4)\n",
      "Pair{AbstractString,Int64}(\"host tropism\", 1)\n",
      "Pair{AbstractString,Int64}(\"compound to be\", 1)\n",
      "Pair{AbstractString,Int64}(\"areas and\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 6 RNA\", 1)\n",
      "Pair{AbstractString,Int64}(\"the fluorine\", 2)\n",
      "Pair{AbstractString,Int64}(\"then cinnamoyl\", 1)\n",
      "Pair{AbstractString,Int64}(\"mice ( 12\", 1)\n",
      "Pair{AbstractString,Int64}(\"CS- 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"two pooled\", 2)\n",
      "Pair{AbstractString,Int64}(\"have to\", 1)\n",
      "Pair{AbstractString,Int64}(\"substitution at\", 1)\n",
      "Pair{AbstractString,Int64}(\"presented for\", 1)\n",
      "Pair{AbstractString,Int64}(\"prepared by removing\", 1)\n",
      "Pair{AbstractString,Int64}(\"( c-d ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"strongly depend on\", 1)\n",
      "Pair{AbstractString,Int64}(\". Ly 6\", 1)\n",
      "Pair{AbstractString,Int64}(\"37 As a\", 1)\n",
      "Pair{AbstractString,Int64}(\"> 2 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"the HKU\", 2)\n",
      "Pair{AbstractString,Int64}(\"can tightly bind\", 1)\n",
      "Pair{AbstractString,Int64}(\"43 countries\", 1)\n",
      "Pair{AbstractString,Int64}(\"propargyl (\", 1)\n",
      "Pair{AbstractString,Int64}(\"cases so far\", 1)\n",
      "Pair{AbstractString,Int64}(\"in 6\", 1)\n",
      "Pair{AbstractString,Int64}(\"475 A\", 1)\n",
      "Pair{AbstractString,Int64}(\"kind\", 4)\n",
      "Pair{AbstractString,Int64}(\"acetate =\", 2)\n",
      "Pair{AbstractString,Int64}(\"3 a -c\", 1)\n",
      "Pair{AbstractString,Int64}(\"polar interactions\", 1)\n",
      "Pair{AbstractString,Int64}(\") , have\", 1)\n",
      "Pair{AbstractString,Int64}(\", the sidechain\", 1)\n",
      "Pair{AbstractString,Int64}(\"recall ,\", 3)\n",
      "Pair{AbstractString,Int64}(\", 57 and\", 1)\n",
      "Pair{AbstractString,Int64}(\") are\", 3)\n",
      "Pair{AbstractString,Int64}(\"replication ( Fig.\", 1)\n",
      "Pair{AbstractString,Int64}(\"researchers studying KLKs\", 1)\n",
      "Pair{AbstractString,Int64}(\"similar observation has\", 1)\n",
      "Pair{AbstractString,Int64}(\"WT\", 4)\n",
      "Pair{AbstractString,Int64}(\") proteins.\", 1)\n",
      "Pair{AbstractString,Int64}(\"E and\", 7)\n",
      "Pair{AbstractString,Int64}(\"a function of\", 1)\n",
      "Pair{AbstractString,Int64}(\"cap in the\", 1)\n",
      "Pair{AbstractString,Int64}(\"to trigger odor\", 1)\n",
      "Pair{AbstractString,Int64}(\"21.71 ± 0.04\", 1)\n",
      "Pair{AbstractString,Int64}(\"in obtaining a\", 1)\n",
      "Pair{AbstractString,Int64}(\"to a dramatic\", 1)\n",
      "Pair{AbstractString,Int64}(\"' 6\", 1)\n",
      "Pair{AbstractString,Int64}(\"China ) (\", 1)\n",
      "Pair{AbstractString,Int64}(\"Data was\", 1)\n",
      "Pair{AbstractString,Int64}(\"compounds 6\", 2)\n",
      "Pair{AbstractString,Int64}(\"instead\", 6)\n",
      "Pair{AbstractString,Int64}(\"analog sensitivity\", 2)\n",
      "Pair{AbstractString,Int64}(\"wound healing\", 1)\n",
      "Pair{AbstractString,Int64}(\"restoring\", 1)\n",
      "Pair{AbstractString,Int64}(\"263 NT )\", 1)\n",
      "Pair{AbstractString,Int64}(\"SalI\", 1)\n",
      "Pair{AbstractString,Int64}(\"Mice were group-housed\", 1)\n",
      "Pair{AbstractString,Int64}(\"each comparison\", 1)\n",
      "Pair{AbstractString,Int64}(\", the stereochemistry\", 1)\n",
      "Pair{AbstractString,Int64}(\"exonucleases ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"injection , mice\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 hexahydrate ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 WIV\", 4)\n",
      "Pair{AbstractString,Int64}(\"We predicted\", 1)\n",
      "Pair{AbstractString,Int64}(\"further improvement might\", 1)\n",
      "Pair{AbstractString,Int64}(\"multiple nucleoside 241\", 1)\n",
      "Pair{AbstractString,Int64}(\"sequence. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"the latter\", 3)\n",
      "Pair{AbstractString,Int64}(\"and Reijmers ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"attractivity of\", 1)\n",
      "Pair{AbstractString,Int64}(\"Vero and\", 1)\n",
      "Pair{AbstractString,Int64}(\"water-immersion objective.\", 1)\n",
      "Pair{AbstractString,Int64}(\"ends of\", 1)\n",
      "Pair{AbstractString,Int64}(\"smaller than 40\", 1)\n",
      "Pair{AbstractString,Int64}(\"arrow\", 1)\n",
      "Pair{AbstractString,Int64}(\"289\", 2)\n",
      "Pair{AbstractString,Int64}(\"be the hallmark\", 1)\n",
      "Pair{AbstractString,Int64}(\"would be different.\", 1)\n",
      "Pair{AbstractString,Int64}(\"their functional relevance\", 1)\n",
      "Pair{AbstractString,Int64}(\", Pro\", 1)\n",
      "Pair{AbstractString,Int64}(\"across these\", 1)\n",
      "Pair{AbstractString,Int64}(\"modification by\", 1)\n",
      "Pair{AbstractString,Int64}(\"is dramatically\", 2)\n",
      "Pair{AbstractString,Int64}(\"359\", 1)\n",
      "Pair{AbstractString,Int64}(\"unless otherwise stated.\", 1)\n",
      "Pair{AbstractString,Int64}(\"( top )\", 1)\n",
      "Pair{AbstractString,Int64}(\"rounds of freeze/thaw\", 2)\n",
      "Pair{AbstractString,Int64}(\"enzymatic assays.\", 1)\n",
      "Pair{AbstractString,Int64}(\"Ontology\", 1)\n",
      "Pair{AbstractString,Int64}(\"almost absolute\", 1)\n",
      "Pair{AbstractString,Int64}(\"[ 19 ]\", 1)\n",
      "Pair{AbstractString,Int64}(\"are produced\", 1)\n",
      "Pair{AbstractString,Int64}(\"in TRIzol\", 1)\n",
      "Pair{AbstractString,Int64}(\"at individual\", 1)\n",
      "Pair{AbstractString,Int64}(\"virus yield was\", 1)\n",
      "Pair{AbstractString,Int64}(\"the lifetime\", 1)\n",
      "Pair{AbstractString,Int64}(\"/MeOH (\", 3)\n",
      "Pair{AbstractString,Int64}(\"dissolved in 20\", 2)\n",
      "Pair{AbstractString,Int64}(\"1 : Crystal\", 1)\n",
      "Pair{AbstractString,Int64}(\"marginally\", 1)\n",
      "Pair{AbstractString,Int64}(\"a ES\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"up-to 20\", 1)\n",
      "Pair{AbstractString,Int64}(\"317 not able\", 1)\n",
      "Pair{AbstractString,Int64}(\"17 ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"hepatitis\", 3)\n",
      "Pair{AbstractString,Int64}(\"The open field\", 1)\n",
      "Pair{AbstractString,Int64}(\"HCoV NL 63\", 3)\n",
      "Pair{AbstractString,Int64}(\"displayed a concentration-dependent\", 1)\n",
      "Pair{AbstractString,Int64}(\"contained 30 mM\", 1)\n",
      "Pair{AbstractString,Int64}(\", acting\", 1)\n",
      "Pair{AbstractString,Int64}(\"correlates\", 1)\n",
      "Pair{AbstractString,Int64}(\"purified by\", 4)\n",
      "Pair{AbstractString,Int64}(\"optimized solution\", 1)\n",
      "Pair{AbstractString,Int64}(\"chemogenetic\", 5)\n",
      "Pair{AbstractString,Int64}(\"titer but 182\", 1)\n",
      "Pair{AbstractString,Int64}(\"dianionic\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells. ( G\", 1)\n",
      "Pair{AbstractString,Int64}(\"CS- ( Figure\", 1)\n",
      "Pair{AbstractString,Int64}(\"observed after associative\", 1)\n",
      "Pair{AbstractString,Int64}(\"indicated by the\", 2)\n",
      "Pair{AbstractString,Int64}(\"interaction\", 8)\n",
      "Pair{AbstractString,Int64}(\"ISGs which\", 1)\n",
      "Pair{AbstractString,Int64}(\"2494 as\", 1)\n",
      "Pair{AbstractString,Int64}(\"the naturally\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Scheme 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"the inhibitors is\", 1)\n",
      "Pair{AbstractString,Int64}(\"recombinant virus\", 1)\n",
      "Pair{AbstractString,Int64}(\"and spleens\", 1)\n",
      "Pair{AbstractString,Int64}(\"sample preservation solution\", 1)\n",
      "Pair{AbstractString,Int64}(\"at a low\", 1)\n",
      "Pair{AbstractString,Int64}(\"an electric foot\", 1)\n",
      "Pair{AbstractString,Int64}(\"traces of\", 1)\n",
      "Pair{AbstractString,Int64}(\"Being in the\", 1)\n",
      "Pair{AbstractString,Int64}(\"sequence ( CDS\", 1)\n",
      "Pair{AbstractString,Int64}(\"hippocampal neural networks\", 1)\n",
      "Pair{AbstractString,Int64}(\"no powerful\", 1)\n",
      "Pair{AbstractString,Int64}(\"LaVision\", 1)\n",
      "Pair{AbstractString,Int64}(\"to directly test\", 1)\n",
      "Pair{AbstractString,Int64}(\"non-synonymous\", 6)\n",
      "Pair{AbstractString,Int64}(\"87\", 3)\n",
      "Pair{AbstractString,Int64}(\"inappropriate KLK 13\", 1)\n",
      "Pair{AbstractString,Int64}(\"by furin during\", 1)\n",
      "Pair{AbstractString,Int64}(\"128 AA\", 1)\n",
      "Pair{AbstractString,Int64}(\"and insertions\", 1)\n",
      "Pair{AbstractString,Int64}(\"coronavirus RNA from\", 1)\n",
      "Pair{AbstractString,Int64}(\"pro with 11\", 5)\n",
      "Pair{AbstractString,Int64}(\"in ExoN-DA or\", 1)\n",
      "Pair{AbstractString,Int64}(\"Zhang , respectively.\", 1)\n",
      "Pair{AbstractString,Int64}(\"of replication (\", 1)\n",
      "Pair{AbstractString,Int64}(\"based cleavage\", 1)\n",
      "Pair{AbstractString,Int64}(\"Experiments\", 2)\n",
      "Pair{AbstractString,Int64}(\"missing in\", 1)\n",
      "Pair{AbstractString,Int64}(\"was presented 4\", 2)\n",
      "Pair{AbstractString,Int64}(\"this enzyme would\", 1)\n",
      "Pair{AbstractString,Int64}(\"BMDM ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"One Sample\", 6)\n",
      "Pair{AbstractString,Int64}(\"The median\", 2)\n",
      "Pair{AbstractString,Int64}(\"culture assays\", 1)\n",
      "Pair{AbstractString,Int64}(\"The organic layers\", 2)\n",
      "Pair{AbstractString,Int64}(\"diet containing\", 2)\n",
      "Pair{AbstractString,Int64}(\"introduced flexibility into\", 1)\n",
      "Pair{AbstractString,Int64}(\"with ketamine\", 1)\n",
      "Pair{AbstractString,Int64}(\", HCoV 229\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 20 /\", 2)\n",
      "Pair{AbstractString,Int64}(\"housekeeping gene Beta-\", 1)\n",
      "Pair{AbstractString,Int64}(\"LB 940 96\", 1)\n",
      "Pair{AbstractString,Int64}(\"Vav 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"reactivate\", 1)\n",
      "Pair{AbstractString,Int64}(\"lineages at MOI\", 1)\n",
      "Pair{AbstractString,Int64}(\"acid. The nucleic\", 1)\n",
      "Pair{AbstractString,Int64}(\"022 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"at 498 nm\", 1)\n",
      "Pair{AbstractString,Int64}(\"infections. For\", 1)\n",
      "Pair{AbstractString,Int64}(\"cFos expression (\", 1)\n",
      "Pair{AbstractString,Int64}(\"product 10\", 1)\n",
      "Pair{AbstractString,Int64}(\"with 11 a\", 7)\n",
      "Pair{AbstractString,Int64}(\"fixed with\", 2)\n",
      "Pair{AbstractString,Int64}(\"lead to\", 9)\n",
      "Pair{AbstractString,Int64}(\"In nearly all\", 1)\n",
      "Pair{AbstractString,Int64}(\"Measurements\", 1)\n",
      "Pair{AbstractString,Int64}(\"6264 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"16 ) viruses\", 1)\n",
      "Pair{AbstractString,Int64}(\"electrified.\", 1)\n",
      "Pair{AbstractString,Int64}(\"the migration\", 1)\n",
      "Pair{AbstractString,Int64}(\", Foot ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"The foot\", 1)\n",
      "Pair{AbstractString,Int64}(\"to MERS-CoV ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"and compared the\", 1)\n",
      "Pair{AbstractString,Int64}(\"# 2016012909576100 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"full-length HCoV-HKU 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"of the CMV\", 1)\n",
      "Pair{AbstractString,Int64}(\"15 %\", 3)\n",
      "Pair{AbstractString,Int64}(\"a density\", 2)\n",
      "Pair{AbstractString,Int64}(\"copy. 478 Nucleoside\", 1)\n",
      "Pair{AbstractString,Int64}(\"points and doses\", 1)\n",
      "Pair{AbstractString,Int64}(\"antigen-presentation to\", 1)\n",
      "Pair{AbstractString,Int64}(\"contents results\", 1)\n",
      "Pair{AbstractString,Int64}(\"or even inactivity\", 1)\n",
      "Pair{AbstractString,Int64}(\"pCAGGS-\", 1)\n",
      "Pair{AbstractString,Int64}(\"4 d-e ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"Mantel-Cox test\", 1)\n",
      "Pair{AbstractString,Int64}(\"them , 15\", 1)\n",
      "Pair{AbstractString,Int64}(\"fl/fl splenocyte cultures\", 1)\n",
      "Pair{AbstractString,Int64}(\"Codon Usage Database\", 1)\n",
      "Pair{AbstractString,Int64}(\"- 8\", 5)\n",
      "Pair{AbstractString,Int64}(\"absent of\", 1)\n",
      "Pair{AbstractString,Int64}(\"similar for all\", 1)\n",
      "Pair{AbstractString,Int64}(\"structures involved\", 1)\n",
      "Pair{AbstractString,Int64}(\"studying\", 2)\n",
      "Pair{AbstractString,Int64}(\"points for\", 2)\n",
      "Pair{AbstractString,Int64}(\"eliminate DSP post-lysis\", 1)\n",
      "Pair{AbstractString,Int64}(\"changes , the\", 1)\n",
      "Pair{AbstractString,Int64}(\"curve. PCR conditions\", 1)\n",
      "Pair{AbstractString,Int64}(\"does not form\", 1)\n",
      "Pair{AbstractString,Int64}(\"hepatitis in\", 1)\n",
      "Pair{AbstractString,Int64}(\"common isotonic solution\", 1)\n",
      "Pair{AbstractString,Int64}(\"setting (\", 1)\n",
      "Pair{AbstractString,Int64}(\"s. In\", 1)\n",
      "Pair{AbstractString,Int64}(\"storage 2.3\", 1)\n",
      "Pair{AbstractString,Int64}(\"17 ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\". CC-BY-ND\", 5)\n",
      "Pair{AbstractString,Int64}(\"or stability in\", 1)\n",
      "Pair{AbstractString,Int64}(\"inverted fluorescence microscope\", 1)\n",
      "Pair{AbstractString,Int64}(\"neural ensemble representing\", 1)\n",
      "Pair{AbstractString,Int64}(\"by residues\", 1)\n",
      "Pair{AbstractString,Int64}(\"replication. In\", 1)\n",
      "Pair{AbstractString,Int64}(\"Gln 189\", 13)\n",
      "Pair{AbstractString,Int64}(\", MHV\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibitor dec-RVKR-CMK\", 1)\n",
      "Pair{AbstractString,Int64}(\"To test\", 12)\n",
      "Pair{AbstractString,Int64}(\"it is estimated\", 1)\n",
      "Pair{AbstractString,Int64}(\"23 , 24\", 1)\n",
      "Pair{AbstractString,Int64}(\") and Coxsackivirus\", 1)\n",
      "Pair{AbstractString,Int64}(\"sets were\", 1)\n",
      "Pair{AbstractString,Int64}(\"betacoronaviruses as well\", 1)\n",
      "Pair{AbstractString,Int64}(\"/MeOH ( 40\", 1)\n",
      "Pair{AbstractString,Int64}(\"the cyclopropylmethyl\", 2)\n",
      "Pair{AbstractString,Int64}(\"the accumulation\", 1)\n",
      "Pair{AbstractString,Int64}(\"request. Stable\", 1)\n",
      "Pair{AbstractString,Int64}(\"difficult\", 2)\n",
      "Pair{AbstractString,Int64}(\"binding. Superimposition\", 1)\n",
      "Pair{AbstractString,Int64}(\"Detection 2019\", 1)\n",
      "Pair{AbstractString,Int64}(\"ex vivo\", 1)\n",
      "Pair{AbstractString,Int64}(\"EMC/\", 1)\n",
      "Pair{AbstractString,Int64}(\"the acids 5\", 1)\n",
      "Pair{AbstractString,Int64}(\"Representative sniff\", 1)\n",
      "Pair{AbstractString,Int64}(\"shifts.\", 1)\n",
      "Pair{AbstractString,Int64}(\"least 45\", 1)\n",
      "Pair{AbstractString,Int64}(\"our analysis\", 1)\n",
      "Pair{AbstractString,Int64}(\"inserts and deletions\", 1)\n",
      "Pair{AbstractString,Int64}(\"a realistic\", 1)\n",
      "Pair{AbstractString,Int64}(\", with EC\", 5)\n",
      "Pair{AbstractString,Int64}(\"dyad. 22\", 1)\n",
      "Pair{AbstractString,Int64}(\"13 ) ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"for using as\", 2)\n",
      "Pair{AbstractString,Int64}(\"hallmark of coronaviral\", 2)\n",
      "Pair{AbstractString,Int64}(\"encoded by the\", 1)\n",
      "Pair{AbstractString,Int64}(\"and chloroquine\", 1)\n",
      "Pair{AbstractString,Int64}(\"( CD 4\", 1)\n",
      "Pair{AbstractString,Int64}(\"size and the\", 1)\n",
      "Pair{AbstractString,Int64}(\"synthesis of\", 1)\n",
      "Pair{AbstractString,Int64}(\") + .\", 2)\n",
      "Pair{AbstractString,Int64}(\"phage )\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.7209 , Figure\", 1)\n",
      "Pair{AbstractString,Int64}(\"The ISG screen\", 1)\n",
      "Pair{AbstractString,Int64}(\"been reported. Lee\", 1)\n",
      "Pair{AbstractString,Int64}(\"Tests , Figure\", 1)\n",
      "Pair{AbstractString,Int64}(\"exposing\", 2)\n",
      "Pair{AbstractString,Int64}(\"Gene ontology (\", 1)\n",
      "Pair{AbstractString,Int64}(\"infected mice\", 2)\n",
      "Pair{AbstractString,Int64}(\"design\", 7)\n",
      "Pair{AbstractString,Int64}(\"position of mice\", 4)\n",
      "Pair{AbstractString,Int64}(\", ref.\", 2)\n",
      "Pair{AbstractString,Int64}(\"reserved. No\", 102)\n",
      "Pair{AbstractString,Int64}(\"for the virus\", 1)\n",
      "Pair{AbstractString,Int64}(\"Cells were transfected\", 1)\n",
      "Pair{AbstractString,Int64}(\"63 M pro\", 39)\n",
      "Pair{AbstractString,Int64}(\"and olfactory-driven behaviors\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 modifications against\", 1)\n",
      "Pair{AbstractString,Int64}(\"two hydrogen\", 2)\n",
      "Pair{AbstractString,Int64}(\"a sample\", 1)\n",
      "Pair{AbstractString,Int64}(\"mL ) was\", 4)\n",
      "Pair{AbstractString,Int64}(\"opposed to\", 1)\n",
      "Pair{AbstractString,Int64}(\"spike protein were\", 1)\n",
      "Pair{AbstractString,Int64}(\"a methylene group\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 3 carboxymethoxyphenol\", 1)\n",
      "Pair{AbstractString,Int64}(\"thresholded using\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"RNA-launched transfection\", 1)\n",
      "Pair{AbstractString,Int64}(\"next analyzed\", 1)\n",
      "Pair{AbstractString,Int64}(\") . Predicting\", 1)\n",
      "Pair{AbstractString,Int64}(\"codon usage difference\", 1)\n",
      "Pair{AbstractString,Int64}(\"was presented. Furthermore\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibitors of coronavirus\", 1)\n",
      "Pair{AbstractString,Int64}(\"on HCoV- 229\", 1)\n",
      "Pair{AbstractString,Int64}(\"% vol/vol\", 1)\n",
      "Pair{AbstractString,Int64}(\"( of the\", 1)\n",
      "Pair{AbstractString,Int64}(\"regions\", 5)\n",
      "Pair{AbstractString,Int64}(\") CoVs\", 2)\n",
      "Pair{AbstractString,Int64}(\"family.\", 1)\n",
      "Pair{AbstractString,Int64}(\"difference was\", 2)\n",
      "Pair{AbstractString,Int64}(\"perfused with ice-cold\", 1)\n",
      "Pair{AbstractString,Int64}(\"center of the\", 1)\n",
      "Pair{AbstractString,Int64}(\"Apart from\", 1)\n",
      "Pair{AbstractString,Int64}(\"among all typical\", 1)\n",
      "Pair{AbstractString,Int64}(\"probably a consequence\", 1)\n",
      "Pair{AbstractString,Int64}(\"non-synonymous substitution rate\", 1)\n",
      "Pair{AbstractString,Int64}(\"in samples\", 1)\n",
      "Pair{AbstractString,Int64}(\"whether nucleoside\", 1)\n",
      "Pair{AbstractString,Int64}(\"The ester\", 1)\n",
      "Pair{AbstractString,Int64}(\"piriform neurons was\", 2)\n",
      "Pair{AbstractString,Int64}(\"was 2.41 that\", 1)\n",
      "Pair{AbstractString,Int64}(\"CaCo 2\", 2)\n",
      "Pair{AbstractString,Int64}(\"gastroenteritis virus\", 2)\n",
      "Pair{AbstractString,Int64}(\"0.0210 ; e\", 1)\n",
      "Pair{AbstractString,Int64}(\"with other\", 2)\n",
      "Pair{AbstractString,Int64}(\"experiencing\", 1)\n",
      "Pair{AbstractString,Int64}(\"2000 or\", 1)\n",
      "Pair{AbstractString,Int64}(\"http://www.kazusa.or.jp/codon/ )\", 1)\n",
      "Pair{AbstractString,Int64}(\"odor ( Schoenenberger\", 1)\n",
      "Pair{AbstractString,Int64}(\"the target protease.\", 1)\n",
      "Pair{AbstractString,Int64}(\"47 mL\", 1)\n",
      "Pair{AbstractString,Int64}(\"12 and\", 7)\n",
      "Pair{AbstractString,Int64}(\"acetophenide conferred a\", 1)\n",
      "Pair{AbstractString,Int64}(\"all four viruses\", 1)\n",
      "Pair{AbstractString,Int64}(\"were\", 358)\n",
      "Pair{AbstractString,Int64}(\"recovery of\", 2)\n",
      "Pair{AbstractString,Int64}(\"performed.\", 1)\n",
      "Pair{AbstractString,Int64}(\"of inflammation\", 1)\n",
      "Pair{AbstractString,Int64}(\"500 mM NaCl\", 1)\n",
      "Pair{AbstractString,Int64}(\"group ,\", 5)\n",
      "Pair{AbstractString,Int64}(\"gggcacccactgctgccagagcttcctgtgca acatcagtgcagccgacggcgggctgcgggc cagcacccacgtgctgggcctcgggctcctgc\", 1)\n",
      "Pair{AbstractString,Int64}(\"HRV 2\", 7)\n",
      "Pair{AbstractString,Int64}(\"period of 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"BHQ- 2 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"the sample that\", 1)\n",
      "Pair{AbstractString,Int64}(\"cloned into pWPI\", 1)\n",
      "Pair{AbstractString,Int64}(\"minutes ( Figure\", 2)\n",
      "Pair{AbstractString,Int64}(\"cells Cells and\", 1)\n",
      "Pair{AbstractString,Int64}(\"smeared\", 1)\n",
      "Pair{AbstractString,Int64}(\"two was\", 1)\n",
      "Pair{AbstractString,Int64}(\"plates containing mock-infected\", 1)\n",
      "Pair{AbstractString,Int64}(\"main protease\", 3)\n",
      "Pair{AbstractString,Int64}(\"a 4 -fluorobenzyl\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 replication in\", 2)\n",
      "Pair{AbstractString,Int64}(\"time period that\", 1)\n",
      "Pair{AbstractString,Int64}(\"values were excluded\", 1)\n",
      "Pair{AbstractString,Int64}(\"( CVB 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"culture for 7\", 1)\n",
      "Pair{AbstractString,Int64}(\"supposed to\", 1)\n",
      "Pair{AbstractString,Int64}(\"residues 127 -\", 1)\n",
      "Pair{AbstractString,Int64}(\"is very urgent\", 1)\n",
      "Pair{AbstractString,Int64}(\", were generously\", 1)\n",
      "Pair{AbstractString,Int64}(\"was weak\", 1)\n",
      "Pair{AbstractString,Int64}(\"high-content high-throughput imaging\", 1)\n",
      "Pair{AbstractString,Int64}(\"better inhibitory\", 1)\n",
      "Pair{AbstractString,Int64}(\"Stable LY\", 2)\n",
      "Pair{AbstractString,Int64}(\"separation\", 1)\n",
      "Pair{AbstractString,Int64}(\"not form\", 1)\n",
      "Pair{AbstractString,Int64}(\"same scale as\", 1)\n",
      "Pair{AbstractString,Int64}(\"2020 .\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 ε replicase\", 1)\n",
      "Pair{AbstractString,Int64}(\"mock infected or\", 3)\n",
      "Pair{AbstractString,Int64}(\"size\", 14)\n",
      "Pair{AbstractString,Int64}(\"181\", 2)\n",
      "Pair{AbstractString,Int64}(\"activation ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"acid substitution (\", 1)\n",
      "Pair{AbstractString,Int64}(\"the Firefly (\", 1)\n",
      "Pair{AbstractString,Int64}(\"d-lactam\", 1)\n",
      "Pair{AbstractString,Int64}(\"the 304 author/funder.\", 1)\n",
      "Pair{AbstractString,Int64}(\"preprint ( B\", 1)\n",
      "Pair{AbstractString,Int64}(\"-FU ( Fig.\", 2)\n",
      "Pair{AbstractString,Int64}(\"0.5322 ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"aligning the catalytic\", 1)\n",
      "Pair{AbstractString,Int64}(\"after the inactivation\", 2)\n",
      "Pair{AbstractString,Int64}(\"Spike protein of\", 1)\n",
      "Pair{AbstractString,Int64}(\"standard\", 11)\n",
      "Pair{AbstractString,Int64}(\"GAT\", 1)\n",
      "Pair{AbstractString,Int64}(\"pocket ( formed\", 1)\n",
      "Pair{AbstractString,Int64}(\"by removal\", 1)\n",
      "Pair{AbstractString,Int64}(\") or VSV\", 1)\n",
      "Pair{AbstractString,Int64}(\"into the human\", 1)\n",
      "Pair{AbstractString,Int64}(\"/Nsp 14\", 1)\n",
      "Pair{AbstractString,Int64}(\"NS mutations 215\", 1)\n",
      "Pair{AbstractString,Int64}(\"r against HCoV\", 2)\n",
      "Pair{AbstractString,Int64}(\"with MHV-A\", 1)\n",
      "Pair{AbstractString,Int64}(\"delayed brain tumor\", 1)\n",
      "Pair{AbstractString,Int64}(\"of pneumonia\", 1)\n",
      "Pair{AbstractString,Int64}(\"structure of\", 13)\n",
      "Pair{AbstractString,Int64}(\"P <\", 3)\n",
      "Pair{AbstractString,Int64}(\"( pH 6.0\", 1)\n",
      "Pair{AbstractString,Int64}(\"aldehyde\", 3)\n",
      "Pair{AbstractString,Int64}(\"yield ) as\", 1)\n",
      "Pair{AbstractString,Int64}(\"infectious\", 3)\n",
      "Pair{AbstractString,Int64}(\"from frozen\", 1)\n",
      "Pair{AbstractString,Int64}(\"of 512\", 1)\n",
      "Pair{AbstractString,Int64}(\"side-chain. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"P 1 capsid-coding\", 1)\n",
      "Pair{AbstractString,Int64}(\"IL 28\", 1)\n",
      "Pair{AbstractString,Int64}(\"high doses\", 1)\n",
      "Pair{AbstractString,Int64}(\"supernatant\", 10)\n",
      "Pair{AbstractString,Int64}(\"indicating that their\", 1)\n",
      "Pair{AbstractString,Int64}(\"picornavirus family. Among\", 1)\n",
      "Pair{AbstractString,Int64}(\"Prism\", 4)\n",
      "Pair{AbstractString,Int64}(\"that HCoV-HKU 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"counted by\", 1)\n",
      "Pair{AbstractString,Int64}(\"of coronaviral replication.We\", 1)\n",
      "Pair{AbstractString,Int64}(\"individual samples\", 1)\n",
      "Pair{AbstractString,Int64}(\"protein from MHV-ExoN\", 1)\n",
      "Pair{AbstractString,Int64}(\"132 DE along\", 1)\n",
      "Pair{AbstractString,Int64}(\"tissue ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"diet , which\", 1)\n",
      "Pair{AbstractString,Int64}(\"did not differ\", 1)\n",
      "Pair{AbstractString,Int64}(\"270 All\", 1)\n",
      "Pair{AbstractString,Int64}(\"lineage B have\", 1)\n",
      "Pair{AbstractString,Int64}(\"CoV ( Extended\", 1)\n",
      "Pair{AbstractString,Int64}(\"suppressed replication of\", 1)\n",
      "Pair{AbstractString,Int64}(\"alkaline hydrolysis. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"we predicted earlier.\", 1)\n",
      "Pair{AbstractString,Int64}(\"while mice author/funder.\", 1)\n",
      "Pair{AbstractString,Int64}(\"147 data\", 1)\n",
      "Pair{AbstractString,Int64}(\"care guidelines (\", 1)\n",
      "Pair{AbstractString,Int64}(\"pro for betacoronaviruses\", 1)\n",
      "Pair{AbstractString,Int64}(\"other warheads\", 2)\n",
      "Pair{AbstractString,Int64}(\"/CD 32 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"fold change >\", 1)\n",
      "Pair{AbstractString,Int64}(\". Multiple\", 1)\n",
      "Pair{AbstractString,Int64}(\"tropism and\", 1)\n",
      "Pair{AbstractString,Int64}(\"any 226\", 1)\n",
      "Pair{AbstractString,Int64}(\"bond from the\", 3)\n",
      "Pair{AbstractString,Int64}(\"Experts\", 1)\n",
      "Pair{AbstractString,Int64}(\"were unchanged\", 1)\n",
      "Pair{AbstractString,Int64}(\"maximum-likelihood\", 1)\n",
      "Pair{AbstractString,Int64}(\"transforming cells with\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 4.12 )\", 1)\n",
      "Pair{AbstractString,Int64}(\", 12.0\", 2)\n",
      "Pair{AbstractString,Int64}(\"a 1.9\", 1)\n",
      "Pair{AbstractString,Int64}(\"o C on\", 2)\n",
      "Pair{AbstractString,Int64}(\"Preliminary pharmacokinetics tests\", 1)\n",
      "Pair{AbstractString,Int64}(\"for 68\", 1)\n",
      "Pair{AbstractString,Int64}(\"30 , 2019\", 1)\n",
      "Pair{AbstractString,Int64}(\", 21 which\", 1)\n",
      "Pair{AbstractString,Int64}(\"Technologies , Petaluma\", 1)\n",
      "Pair{AbstractString,Int64}(\"animal care\", 1)\n",
      "Pair{AbstractString,Int64}(\"Recently , Fos-tagging\", 1)\n",
      "Pair{AbstractString,Int64}(\"-can accept\", 1)\n",
      "Pair{AbstractString,Int64}(\"solution ( 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"225 We detected\", 1)\n",
      "Pair{AbstractString,Int64}(\"medium supplemented with\", 2)\n",
      "Pair{AbstractString,Int64}(\"assay ( EC\", 1)\n",
      "Pair{AbstractString,Int64}(\"line of\", 1)\n",
      "Pair{AbstractString,Int64}(\"a KLK\", 2)\n",
      "Pair{AbstractString,Int64}(\"are viable\", 1)\n",
      "Pair{AbstractString,Int64}(\"agar. Single plaques\", 1)\n",
      "Pair{AbstractString,Int64}(\"( hence\", 1)\n",
      "Pair{AbstractString,Int64}(\"This latter\", 1)\n",
      "Pair{AbstractString,Int64}(\"identify of SARS-CoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"a learned context-shock\", 1)\n",
      "Pair{AbstractString,Int64}(\"7 µm\", 2)\n",
      "Pair{AbstractString,Int64}(\"optimized solution such\", 1)\n",
      "Pair{AbstractString,Int64}(\"88 - 95\", 1)\n",
      "Pair{AbstractString,Int64}(\"dilutions of\", 3)\n",
      "Pair{AbstractString,Int64}(\", replication\", 2)\n",
      "Pair{AbstractString,Int64}(\"cells 199 permissive\", 1)\n",
      "Pair{AbstractString,Int64}(\"standard generated\", 1)\n",
      "Pair{AbstractString,Int64}(\"level between WIV\", 1)\n",
      "Pair{AbstractString,Int64}(\"\\\" , cannot\", 1)\n",
      "Pair{AbstractString,Int64}(\"p= 0.0779 ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"The first crystal\", 1)\n",
      "Pair{AbstractString,Int64}(\"T 4\", 2)\n",
      "Pair{AbstractString,Int64}(\"Promega\", 11)\n",
      "Pair{AbstractString,Int64}(\"trained to associate\", 1)\n",
      "Pair{AbstractString,Int64}(\"even in the\", 1)\n",
      "Pair{AbstractString,Int64}(\"spatial\", 2)\n",
      "Pair{AbstractString,Int64}(\"action potentials triggered\", 2)\n",
      "Pair{AbstractString,Int64}(\"was poor\", 2)\n",
      "Pair{AbstractString,Int64}(\"inoculum\", 1)\n",
      "Pair{AbstractString,Int64}(\"syncytia. For\", 1)\n",
      "Pair{AbstractString,Int64}(\"6.0 software (\", 1)\n",
      "Pair{AbstractString,Int64}(\"group compared\", 1)\n",
      "Pair{AbstractString,Int64}(\"but only\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\". A. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"Supplementary Table S\", 1)\n",
      "Pair{AbstractString,Int64}(\"recombinant SARS-CoV\", 3)\n",
      "Pair{AbstractString,Int64}(\"at l =\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.01 ) compared\", 1)\n",
      "Pair{AbstractString,Int64}(\"using RHD 2132\", 1)\n",
      "Pair{AbstractString,Int64}(\"activity lasts for\", 1)\n",
      "Pair{AbstractString,Int64}(\"3.81 ) mV\", 1)\n",
      "Pair{AbstractString,Int64}(\", further research\", 1)\n",
      "Pair{AbstractString,Int64}(\") bias of\", 1)\n",
      "Pair{AbstractString,Int64}(\"prior to being\", 1)\n",
      "Pair{AbstractString,Int64}(\"Initially\", 1)\n",
      "Pair{AbstractString,Int64}(\", 95\", 1)\n",
      "Pair{AbstractString,Int64}(\"an Anthos\", 1)\n",
      "Pair{AbstractString,Int64}(\"above.\", 6)\n",
      "Pair{AbstractString,Int64}(\"th , 330\", 1)\n",
      "Pair{AbstractString,Int64}(\"sensitivity ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"micromolar range or\", 1)\n",
      "Pair{AbstractString,Int64}(\"spectrophotometer\", 1)\n",
      "Pair{AbstractString,Int64}(\"altered in SARS-CoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 O\", 1)\n",
      "Pair{AbstractString,Int64}(\"were 2.3 µM\", 1)\n",
      "Pair{AbstractString,Int64}(\"before 9 .\", 1)\n",
      "Pair{AbstractString,Int64}(\"therefore binds\", 1)\n",
      "Pair{AbstractString,Int64}(\"background. In previous\", 1)\n",
      "Pair{AbstractString,Int64}(\"very urgent\", 1)\n",
      "Pair{AbstractString,Int64}(\"proteins of either\", 1)\n",
      "Pair{AbstractString,Int64}(\"to be originated\", 1)\n",
      "Pair{AbstractString,Int64}(\"labeled during\", 1)\n",
      "Pair{AbstractString,Int64}(\") mV\", 7)\n",
      "Pair{AbstractString,Int64}(\"a factor of\", 1)\n",
      "Pair{AbstractString,Int64}(\"7500 Fast Real-Time\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 cells ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"neutral odors\", 1)\n",
      "Pair{AbstractString,Int64}(\"6 was then\", 1)\n",
      "Pair{AbstractString,Int64}(\"2.5 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"analysis. Then ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"( FDR\", 1)\n",
      "Pair{AbstractString,Int64}(\"Technical Guidelines for\", 1)\n",
      "Pair{AbstractString,Int64}(\"saturated brine (\", 3)\n",
      "Pair{AbstractString,Int64}(\"box. A testing\", 1)\n",
      "Pair{AbstractString,Int64}(\"is performed\", 1)\n",
      "Pair{AbstractString,Int64}(\"to controls when\", 1)\n",
      "Pair{AbstractString,Int64}(\"for P\", 1)\n",
      "Pair{AbstractString,Int64}(\"quantitative fusion assay\", 1)\n",
      "Pair{AbstractString,Int64}(\"Immune cell counts\", 1)\n",
      "Pair{AbstractString,Int64}(\"factors\", 3)\n",
      "Pair{AbstractString,Int64}(\"replication ( 7\", 1)\n",
      "Pair{AbstractString,Int64}(\"9.07 . However\", 1)\n",
      "Pair{AbstractString,Int64}(\"\\\" covered by\", 1)\n",
      "Pair{AbstractString,Int64}(\"Doxycycline\", 1)\n",
      "Pair{AbstractString,Int64}(\"QuantiFast SYBR Green\", 1)\n",
      "Pair{AbstractString,Int64}(\"this site\", 1)\n",
      "Pair{AbstractString,Int64}(\"′ , and\", 1)\n",
      "Pair{AbstractString,Int64}(\", intersectional\", 1)\n",
      "Pair{AbstractString,Int64}(\"mmol ) were\", 5)\n",
      "Pair{AbstractString,Int64}(\"yielded a\", 1)\n",
      "Pair{AbstractString,Int64}(\"Free Energy of\", 1)\n",
      "Pair{AbstractString,Int64}(\"mM Tris-HCl\", 2)\n",
      "Pair{AbstractString,Int64}(\"whereas on\", 1)\n",
      "Pair{AbstractString,Int64}(\"K at\", 2)\n",
      "Pair{AbstractString,Int64}(\"passaged in the\", 1)\n",
      "Pair{AbstractString,Int64}(\"supported or\", 1)\n",
      "Pair{AbstractString,Int64}(\"and Glu\", 1)\n",
      "Pair{AbstractString,Int64}(\"that the increased\", 1)\n",
      "Pair{AbstractString,Int64}(\"prolonged passage\", 2)\n",
      "Pair{AbstractString,Int64}(\"Treating HAE cultures\", 1)\n",
      "Pair{AbstractString,Int64}(\"Y=- 0.8 and\", 1)\n",
      "Pair{AbstractString,Int64}(\"lethal 18 .\", 1)\n",
      "Pair{AbstractString,Int64}(\"asked whether artificial\", 1)\n",
      "Pair{AbstractString,Int64}(\"had directly\", 1)\n",
      "Pair{AbstractString,Int64}(\"to 1.2 %\", 1)\n",
      "Pair{AbstractString,Int64}(\", has been\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 CL pro\", 3)\n",
      "Pair{AbstractString,Int64}(\"once on each\", 2)\n",
      "Pair{AbstractString,Int64}(\"three lineages of\", 1)\n",
      "Pair{AbstractString,Int64}(\"I , Table\", 1)\n",
      "Pair{AbstractString,Int64}(\"215 were\", 1)\n",
      "Pair{AbstractString,Int64}(\"s more ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"CleavEx method\", 1)\n",
      "Pair{AbstractString,Int64}(\"efficiently cleaved by\", 1)\n",
      "Pair{AbstractString,Int64}(\"Fig. S\", 7)\n",
      "Pair{AbstractString,Int64}(\", genomic RNA\", 1)\n",
      "Pair{AbstractString,Int64}(\"S 2 ′\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells was confirmed\", 1)\n",
      "Pair{AbstractString,Int64}(\"kDa\", 2)\n",
      "Pair{AbstractString,Int64}(\", 0.2 M\", 2)\n",
      "Pair{AbstractString,Int64}(\"either\", 8)\n",
      "Pair{AbstractString,Int64}(\"experiments or increasing\", 1)\n",
      "Pair{AbstractString,Int64}(\"stored\", 8)\n",
      "Pair{AbstractString,Int64}(\"of injection\", 1)\n",
      "Pair{AbstractString,Int64}(\"values were calculated\", 2)\n",
      "Pair{AbstractString,Int64}(\"TLC ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\"a 2.54 -Å\", 1)\n",
      "Pair{AbstractString,Int64}(\"attenuated behavioral\", 1)\n",
      "Pair{AbstractString,Int64}(\"degradation. To\", 1)\n",
      "Pair{AbstractString,Int64}(\"His 41 ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"the release of\", 1)\n",
      "Pair{AbstractString,Int64}(\"- 216 methyltransferase\", 1)\n",
      "Pair{AbstractString,Int64}(\"14 are\", 1)\n",
      "Pair{AbstractString,Int64}(\"8 - 11\", 2)\n",
      "Pair{AbstractString,Int64}(\"biosafety level- 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"and LY\", 1)\n",
      "Pair{AbstractString,Int64}(\"for each\", 5)\n",
      "Pair{AbstractString,Int64}(\"or Dunnett '\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 in HeLa\", 2)\n",
      "Pair{AbstractString,Int64}(\", 10 mM\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.3698 ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"220 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"fluorine can accept\", 2)\n",
      "Pair{AbstractString,Int64}(\"Quinlan et al.\", 1)\n",
      "Pair{AbstractString,Int64}(\"time , Kim\", 1)\n",
      "Pair{AbstractString,Int64}(\"floor ( Met\", 1)\n",
      "Pair{AbstractString,Int64}(\", which were\", 1)\n",
      "Pair{AbstractString,Int64}(\"solvent was\", 6)\n",
      "Pair{AbstractString,Int64}(\") As a\", 1)\n",
      "Pair{AbstractString,Int64}(\"compared the number\", 2)\n",
      "Pair{AbstractString,Int64}(\"-FU is converted\", 1)\n",
      "Pair{AbstractString,Int64}(\"C under\", 1)\n",
      "Pair{AbstractString,Int64}(\"rigid Pro\", 1)\n",
      "Pair{AbstractString,Int64}(\"parts per\", 1)\n",
      "Pair{AbstractString,Int64}(\"transcriptomic approach to\", 1)\n",
      "Pair{AbstractString,Int64}(\"result in the\", 1)\n",
      "Pair{AbstractString,Int64}(\"follows.\", 1)\n",
      "Pair{AbstractString,Int64}(\"We 166 found\", 1)\n",
      "Pair{AbstractString,Int64}(\"execute Gene\", 1)\n",
      "Pair{AbstractString,Int64}(\"all substrates\", 1)\n",
      "Pair{AbstractString,Int64}(\"the genome (\", 1)\n",
      "Pair{AbstractString,Int64}(\"also required\", 1)\n",
      "Pair{AbstractString,Int64}(\"in HAE\", 4)\n",
      "Pair{AbstractString,Int64}(\", though ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"has less plasticity\", 1)\n",
      "Pair{AbstractString,Int64}(\"protocol. Liver\", 1)\n",
      "Pair{AbstractString,Int64}(\"12 maintained at\", 1)\n",
      "Pair{AbstractString,Int64}(\"a 227 network\", 1)\n",
      "Pair{AbstractString,Int64}(\"resulted in improved\", 1)\n",
      "Pair{AbstractString,Int64}(\"virus and 77\", 1)\n",
      "Pair{AbstractString,Int64}(\"for Coxsackievirusinduced pancreatitis.\", 1)\n",
      "Pair{AbstractString,Int64}(\"dried over Na\", 4)\n",
      "Pair{AbstractString,Int64}(\"aldehyde inhibitors ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"The S protein\", 1)\n",
      "Pair{AbstractString,Int64}(\"hampered HCoV-HKU 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"the isobutyl group\", 2)\n",
      "Pair{AbstractString,Int64}(\"may be subjected\", 1)\n",
      "Pair{AbstractString,Int64}(\"and emotional\", 1)\n",
      "Pair{AbstractString,Int64}(\"nucleoside analogues\", 1)\n",
      "Pair{AbstractString,Int64}(\"polymerase ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"sites. The MERS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"released by Chinese\", 1)\n",
      "Pair{AbstractString,Int64}(\"penetrating deeply into\", 1)\n",
      "Pair{AbstractString,Int64}(\"100 µg\", 1)\n",
      "Pair{AbstractString,Int64}(\"pseudorandomized. Training always\", 1)\n",
      "Pair{AbstractString,Int64}(\"mutations in nsp\", 4)\n",
      "Pair{AbstractString,Int64}(\"one-step\", 1)\n",
      "Pair{AbstractString,Int64}(\"i.e. the best\", 1)\n",
      "Pair{AbstractString,Int64}(\"min to kill\", 1)\n",
      "Pair{AbstractString,Int64}(\"Data\", 39)\n",
      "Pair{AbstractString,Int64}(\"protein was involved\", 1)\n",
      "Pair{AbstractString,Int64}(\"cultured from Ly\", 1)\n",
      "Pair{AbstractString,Int64}(\"learned odor-context\", 1)\n",
      "Pair{AbstractString,Int64}(\"detection efficiency.\", 1)\n",
      "Pair{AbstractString,Int64}(\"percent identity\", 1)\n",
      "Pair{AbstractString,Int64}(\". In accordance\", 1)\n",
      "Pair{AbstractString,Int64}(\"1.04 ( 0.03\", 1)\n",
      "Pair{AbstractString,Int64}(\"16 disrupting\", 1)\n",
      "Pair{AbstractString,Int64}(\"values :\", 3)\n",
      "Pair{AbstractString,Int64}(\"volume 150 µl/well\", 1)\n",
      "Pair{AbstractString,Int64}(\"Hot\", 1)\n",
      "Pair{AbstractString,Int64}(\"quality of nucleic\", 1)\n",
      "Pair{AbstractString,Int64}(\"ion-trap\", 1)\n",
      "Pair{AbstractString,Int64}(\"were further tested\", 1)\n",
      "Pair{AbstractString,Int64}(\"287 significantly\", 1)\n",
      "Pair{AbstractString,Int64}(\"t in HeLa\", 1)\n",
      "Pair{AbstractString,Int64}(\"pyralis ) or\", 1)\n",
      "Pair{AbstractString,Int64}(\"originated from\", 2)\n",
      "Pair{AbstractString,Int64}(\"cell populations\", 2)\n",
      "Pair{AbstractString,Int64}(\"unexpected. The activity\", 1)\n",
      "Pair{AbstractString,Int64}(\"CT ( 23.02\", 1)\n",
      "Pair{AbstractString,Int64}(\"if the viral\", 1)\n",
      "Pair{AbstractString,Int64}(\", The Netherlands.\", 1)\n",
      "Pair{AbstractString,Int64}(\"of either MERS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"coronaviral replication.We\", 1)\n",
      "Pair{AbstractString,Int64}(\"sequence identify of\", 1)\n",
      "Pair{AbstractString,Int64}(\"2.3 µM against\", 1)\n",
      "Pair{AbstractString,Int64}(\"the latter ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"difference of the\", 2)\n",
      "Pair{AbstractString,Int64}(\"Fo -Fc difference\", 1)\n",
      "Pair{AbstractString,Int64}(\". The organic\", 1)\n",
      "Pair{AbstractString,Int64}(\"A diffraction data\", 1)\n",
      "Pair{AbstractString,Int64}(\") . (\", 3)\n",
      "Pair{AbstractString,Int64}(\"memory recall. To\", 3)\n",
      "Pair{AbstractString,Int64}(\"resulted in\", 7)\n",
      "Pair{AbstractString,Int64}(\"0.46 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"232 mutations in\", 1)\n",
      "Pair{AbstractString,Int64}(\"so as the\", 1)\n",
      "Pair{AbstractString,Int64}(\"pairs.\", 1)\n",
      "Pair{AbstractString,Int64}(\"decreased to IC\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Saffire 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"356\", 1)\n",
      "Pair{AbstractString,Int64}(\"six inhibitor\", 1)\n",
      "Pair{AbstractString,Int64}(\"a six-membered d-lactam\", 1)\n",
      "Pair{AbstractString,Int64}(\"228 mutagen\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"and flash-cooled\", 2)\n",
      "Pair{AbstractString,Int64}(\"silica gel (\", 2)\n",
      "Pair{AbstractString,Int64}(\"a single gene\", 1)\n",
      "Pair{AbstractString,Int64}(\"similarity to the\", 1)\n",
      "Pair{AbstractString,Int64}(\"An\", 2)\n",
      "Pair{AbstractString,Int64}(\"to tetO\", 1)\n",
      "Pair{AbstractString,Int64}(\"French national\", 1)\n",
      "Pair{AbstractString,Int64}(\"to false\", 1)\n",
      "Pair{AbstractString,Int64}(\"℃. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"days at 32\", 2)\n",
      "Pair{AbstractString,Int64}(\"poorly understood. Recent\", 1)\n",
      "Pair{AbstractString,Int64}(\"close relative of\", 1)\n",
      "Pair{AbstractString,Int64}(\"mA , 0.6\", 1)\n",
      "Pair{AbstractString,Int64}(\"to CD\", 1)\n",
      "Pair{AbstractString,Int64}(\"inflammation was moderately\", 1)\n",
      "Pair{AbstractString,Int64}(\"six\", 1)\n",
      "Pair{AbstractString,Int64}(\"replication. However ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"or higher as\", 1)\n",
      "Pair{AbstractString,Int64}(\"0 , 1.25\", 1)\n",
      "Pair{AbstractString,Int64}(\"hippocampus. In both\", 1)\n",
      "Pair{AbstractString,Int64}(\"Yet\", 1)\n",
      "Pair{AbstractString,Int64}(\"M HCl was\", 2)\n",
      "Pair{AbstractString,Int64}(\"co-cultured for 16\", 1)\n",
      "Pair{AbstractString,Int64}(\"targeted by the\", 1)\n",
      "Pair{AbstractString,Int64}(\"The key\", 1)\n",
      "Pair{AbstractString,Int64}(\"stock solution. Cultures\", 1)\n",
      "Pair{AbstractString,Int64}(\"The two SARS-CoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"B cells or\", 1)\n",
      "Pair{AbstractString,Int64}(\"library. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"RT-qPCR as\", 2)\n",
      "Pair{AbstractString,Int64}(\"Gly 143 and\", 1)\n",
      "Pair{AbstractString,Int64}(\"off doxycycline\", 1)\n",
      "Pair{AbstractString,Int64}(\"and then codon-based\", 1)\n",
      "Pair{AbstractString,Int64}(\"control. 48 hours\", 1)\n",
      "Pair{AbstractString,Int64}(\"were 348\", 1)\n",
      "Pair{AbstractString,Int64}(\"that chemogenetic silencing\", 1)\n",
      "Pair{AbstractString,Int64}(\"Crystal structures of\", 2)\n",
      "Pair{AbstractString,Int64}(\"diagnostic facility of\", 1)\n",
      "Pair{AbstractString,Int64}(\"also confirms that\", 1)\n",
      "Pair{AbstractString,Int64}(\"negatively\", 1)\n",
      "Pair{AbstractString,Int64}(\"33 Apart from\", 1)\n",
      "Pair{AbstractString,Int64}(\"developed an intersectional\", 1)\n",
      "Pair{AbstractString,Int64}(\"that behaviorally relevant\", 1)\n",
      "Pair{AbstractString,Int64}(\"correlated with the\", 1)\n",
      "Pair{AbstractString,Int64}(\"progress of\", 3)\n",
      "Pair{AbstractString,Int64}(\"aspirated and\", 1)\n",
      "Pair{AbstractString,Int64}(\"min at 12,000\", 3)\n",
      "Pair{AbstractString,Int64}(\"476 All rights\", 1)\n",
      "Pair{AbstractString,Int64}(\"pellet\", 3)\n",
      "Pair{AbstractString,Int64}(\"7 a\", 1)\n",
      "Pair{AbstractString,Int64}(\"were selected\", 1)\n",
      "Pair{AbstractString,Int64}(\"easily\", 2)\n",
      "Pair{AbstractString,Int64}(\"× 10\", 2)\n",
      "Pair{AbstractString,Int64}(\"and found by\", 1)\n",
      "Pair{AbstractString,Int64}(\"coronavirus seriously.\", 1)\n",
      "Pair{AbstractString,Int64}(\"target proteases (\", 1)\n",
      "Pair{AbstractString,Int64}(\"comparatively good\", 1)\n",
      "Pair{AbstractString,Int64}(\"the carboxylate\", 1)\n",
      "Pair{AbstractString,Int64}(\"25 V (\", 1)\n",
      "Pair{AbstractString,Int64}(\"damage the\", 1)\n",
      "Pair{AbstractString,Int64}(\"l =\", 1)\n",
      "Pair{AbstractString,Int64}(\"their maximum\", 3)\n",
      "Pair{AbstractString,Int64}(\"( Tables\", 1)\n",
      "Pair{AbstractString,Int64}(\"well established\", 1)\n",
      "Pair{AbstractString,Int64}(\"DNAs were linearized\", 1)\n",
      "Pair{AbstractString,Int64}(\"that Ly 6\", 1)\n",
      "Pair{AbstractString,Int64}(\") , was\", 2)\n",
      "Pair{AbstractString,Int64}(\"via the\", 2)\n",
      "Pair{AbstractString,Int64}(\"the 287 significantly\", 1)\n",
      "Pair{AbstractString,Int64}(\"and coronaviral\", 1)\n",
      "Pair{AbstractString,Int64}(\"the effective number\", 1)\n",
      "Pair{AbstractString,Int64}(\"the silencing\", 2)\n",
      "Pair{AbstractString,Int64}(\"the central\", 1)\n",
      "Pair{AbstractString,Int64}(\"Plasmid constructs\", 1)\n",
      "Pair{AbstractString,Int64}(\"is regulated in\", 1)\n",
      "Pair{AbstractString,Int64}(\"Alexander\", 1)\n",
      "Pair{AbstractString,Int64}(\"positive neurons were\", 1)\n",
      "Pair{AbstractString,Int64}(\"Figure 1 C\", 1)\n",
      "Pair{AbstractString,Int64}(\"liquid nitrogen.\", 3)\n",
      "Pair{AbstractString,Int64}(\") accumulated 15\", 1)\n",
      "Pair{AbstractString,Int64}(\", IFITM\", 1)\n",
      "Pair{AbstractString,Int64}(\"in Fiji.\", 1)\n",
      "Pair{AbstractString,Int64}(\", 2008 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"s. When\", 1)\n",
      "Pair{AbstractString,Int64}(\"between these\", 2)\n",
      "Pair{AbstractString,Int64}(\"Cowansage\", 1)\n",
      "Pair{AbstractString,Int64}(\"( delayed\", 1)\n",
      "Pair{AbstractString,Int64}(\"neurons that respond\", 1)\n",
      "Pair{AbstractString,Int64}(\"discrimination of\", 1)\n",
      "Pair{AbstractString,Int64}(\"to create\", 2)\n",
      "Pair{AbstractString,Int64}(\"194 entry\", 1)\n",
      "Pair{AbstractString,Int64}(\"RSCU bias will\", 1)\n",
      "Pair{AbstractString,Int64}(\"a seafood and\", 1)\n",
      "Pair{AbstractString,Int64}(\"and RNA )\", 1)\n",
      "Pair{AbstractString,Int64}(\"detector. All\", 1)\n",
      "Pair{AbstractString,Int64}(\"total gene number\", 1)\n",
      "Pair{AbstractString,Int64}(\"will hereafter\", 1)\n",
      "Pair{AbstractString,Int64}(\"by day 6\", 1)\n",
      "Pair{AbstractString,Int64}(\"nucleotide substitution\", 2)\n",
      "Pair{AbstractString,Int64}(\"was dissolved\", 8)\n",
      "Pair{AbstractString,Int64}(\"glutamine side-chain ,\", 1)\n",
      "Pair{AbstractString,Int64}(\", 0.79 mmol\", 2)\n",
      "Pair{AbstractString,Int64}(\"doxycycline five days\", 1)\n",
      "Pair{AbstractString,Int64}(\"outbreak was\", 1)\n",
      "Pair{AbstractString,Int64}(\"the solution. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"unable\", 3)\n",
      "Pair{AbstractString,Int64}(\"BESSY ( Berlin\", 1)\n",
      "Pair{AbstractString,Int64}(\"side-chain ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"Reijmers , 2015\", 1)\n",
      "Pair{AbstractString,Int64}(\"Infection with some\", 1)\n",
      "Pair{AbstractString,Int64}(\"reduce\", 3)\n",
      "Pair{AbstractString,Int64}(\"pro , 12.3\", 1)\n",
      "Pair{AbstractString,Int64}(\"Plates were\", 1)\n",
      "Pair{AbstractString,Int64}(\"long , we\", 1)\n",
      "Pair{AbstractString,Int64}(\"a-ketoamide\", 4)\n",
      "Pair{AbstractString,Int64}(\"were also determined\", 1)\n",
      "Pair{AbstractString,Int64}(\"rely\", 1)\n",
      "Pair{AbstractString,Int64}(\"reverse\", 2)\n",
      "Pair{AbstractString,Int64}(\"degradation of nucleic\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 min together\", 1)\n",
      "Pair{AbstractString,Int64}(\"into Fo-Fc\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Automate\", 1)\n",
      "Pair{AbstractString,Int64}(\"a certain\", 1)\n",
      "Pair{AbstractString,Int64}(\") pervasive episodic\", 1)\n",
      "Pair{AbstractString,Int64}(\"n= 3 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"2.3 Agarose\", 1)\n",
      "Pair{AbstractString,Int64}(\"diet to avoid\", 1)\n",
      "Pair{AbstractString,Int64}(\"_ GFP\", 1)\n",
      "Pair{AbstractString,Int64}(\"All primary cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"Euclidean distance\", 1)\n",
      "Pair{AbstractString,Int64}(\"ToxiLight ( measuring\", 1)\n",
      "Pair{AbstractString,Int64}(\") for HCoV-NL\", 1)\n",
      "Pair{AbstractString,Int64}(\"from author/funder. All\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibition of cathepsin\", 1)\n",
      "Pair{AbstractString,Int64}(\"genome. Gene expression\", 1)\n",
      "Pair{AbstractString,Int64}(\"of 1.7 x\", 1)\n",
      "Pair{AbstractString,Int64}(\"available kits :\", 1)\n",
      "Pair{AbstractString,Int64}(\"Cytofix/Cytoperm )\", 1)\n",
      "Pair{AbstractString,Int64}(\"Tor 2 to\", 1)\n",
      "Pair{AbstractString,Int64}(\"two independent screens\", 1)\n",
      "Pair{AbstractString,Int64}(\"probably unable to\", 1)\n",
      "Pair{AbstractString,Int64}(\"unknown. Interestingly\", 1)\n",
      "Pair{AbstractString,Int64}(\"infections.\", 2)\n",
      "Pair{AbstractString,Int64}(\"the author/funder. All\", 3)\n",
      "Pair{AbstractString,Int64}(\"( n= 6\", 2)\n",
      "Pair{AbstractString,Int64}(\"inhibitors revealed\", 1)\n",
      "Pair{AbstractString,Int64}(\"alphacoronavirus proteases\", 1)\n",
      "Pair{AbstractString,Int64}(\"via RT-qPCR\", 1)\n",
      "Pair{AbstractString,Int64}(\"but leave small\", 1)\n",
      "Pair{AbstractString,Int64}(\"box\", 24)\n",
      "Pair{AbstractString,Int64}(\"using maximum-likelihood\", 1)\n",
      "Pair{AbstractString,Int64}(\"retrieval after CNO\", 1)\n",
      "Pair{AbstractString,Int64}(\"5 B and\", 2)\n",
      "Pair{AbstractString,Int64}(\"neural activation experiments.\", 1)\n",
      "Pair{AbstractString,Int64}(\"anti-HA\", 1)\n",
      "Pair{AbstractString,Int64}(\"induction of author/funder.\", 1)\n",
      "Pair{AbstractString,Int64}(\"resulting in highest-probability\", 1)\n",
      "Pair{AbstractString,Int64}(\"previous studies\", 3)\n",
      "Pair{AbstractString,Int64}(\"serine\", 2)\n",
      "Pair{AbstractString,Int64}(\"codonoptimized (\", 1)\n",
      "Pair{AbstractString,Int64}(\"allows a better\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 , P\", 1)\n",
      "Pair{AbstractString,Int64}(\"detection efficiency\", 1)\n",
      "Pair{AbstractString,Int64}(\"162\", 2)\n",
      "Pair{AbstractString,Int64}(\"\\\" .\", 1)\n",
      "Pair{AbstractString,Int64}(\"from -\", 1)\n",
      "Pair{AbstractString,Int64}(\"was first\", 3)\n",
      "Pair{AbstractString,Int64}(\", to exclude\", 1)\n",
      "Pair{AbstractString,Int64}(\"Corona Virus Disease\", 1)\n",
      "Pair{AbstractString,Int64}(\"between the specific\", 1)\n",
      "Pair{AbstractString,Int64}(\"50 ( μM\", 1)\n",
      "Pair{AbstractString,Int64}(\"with HCoV-\", 7)\n",
      "Pair{AbstractString,Int64}(\"196 KLK 13\", 1)\n",
      "Pair{AbstractString,Int64}(\": Fit\", 1)\n",
      "Pair{AbstractString,Int64}(\"° C. containing\", 1)\n",
      "Pair{AbstractString,Int64}(\"triethylamine\", 2)\n",
      "Pair{AbstractString,Int64}(\", U= 28\", 1)\n",
      "Pair{AbstractString,Int64}(\"cortex Adeno-Associated Viruses\", 1)\n",
      "Pair{AbstractString,Int64}(\"D 913 E\", 1)\n",
      "Pair{AbstractString,Int64}(\"Aarhaus )\", 1)\n",
      "Pair{AbstractString,Int64}(\"CoV. Among 10\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"responsible\", 1)\n",
      "Pair{AbstractString,Int64}(\"the lifetime of\", 1)\n",
      "Pair{AbstractString,Int64}(\"Basu\", 1)\n",
      "Pair{AbstractString,Int64}(\"4 5\", 4)\n",
      "Pair{AbstractString,Int64}(\"cDNA obtained\", 1)\n",
      "Pair{AbstractString,Int64}(\"once. The\", 1)\n",
      "Pair{AbstractString,Int64}(\") technology\", 1)\n",
      "Pair{AbstractString,Int64}(\"viruses with\", 2)\n",
      "Pair{AbstractString,Int64}(\"acids 71\", 1)\n",
      "Pair{AbstractString,Int64}(\"in interacting\", 1)\n",
      "Pair{AbstractString,Int64}(\"of binding.\", 1)\n",
      "Pair{AbstractString,Int64}(\"identity )\", 1)\n",
      "Pair{AbstractString,Int64}(\"median\", 11)\n",
      "Pair{AbstractString,Int64}(\"fit 375\", 1)\n",
      "Pair{AbstractString,Int64}(\"reaction\", 25)\n",
      "Pair{AbstractString,Int64}(\". According to\", 1)\n",
      "Pair{AbstractString,Int64}(\"with 10 silent\", 1)\n",
      "Pair{AbstractString,Int64}(\", and .\", 1)\n",
      "Pair{AbstractString,Int64}(\"and SARS-CoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 capsid-coding sequence\", 1)\n",
      "Pair{AbstractString,Int64}(\"M )\", 2)\n",
      "Pair{AbstractString,Int64}(\"/ 2020.03.11.987016 doi\", 3)\n",
      "Pair{AbstractString,Int64}(\", demonstrating that\", 3)\n",
      "Pair{AbstractString,Int64}(\"in their piriform\", 1)\n",
      "Pair{AbstractString,Int64}(\"towards the opposite\", 1)\n",
      "Pair{AbstractString,Int64}(\"IHIC and splenocytes\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibitors against RNA\", 1)\n",
      "Pair{AbstractString,Int64}(\", 17\", 1)\n",
      "Pair{AbstractString,Int64}(\"e ΔHSC mice\", 16)\n",
      "Pair{AbstractString,Int64}(\"CSpresentation\", 1)\n",
      "Pair{AbstractString,Int64}(\"14.2\", 2)\n",
      "Pair{AbstractString,Int64}(\"binds\", 1)\n",
      "Pair{AbstractString,Int64}(\"General procedure. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"with acceptable\", 1)\n",
      "Pair{AbstractString,Int64}(\"In the SARS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"100 µg ·\", 1)\n",
      "Pair{AbstractString,Int64}(\"Luciferase\", 4)\n",
      "Pair{AbstractString,Int64}(\"were fished from\", 1)\n",
      "Pair{AbstractString,Int64}(\"glucose 25\", 1)\n",
      "Pair{AbstractString,Int64}(\"2.89 - 2.69\", 1)\n",
      "Pair{AbstractString,Int64}(\"might contribute to\", 1)\n",
      "Pair{AbstractString,Int64}(\", we concluded\", 1)\n",
      "Pair{AbstractString,Int64}(\"cryo-buffer\", 1)\n",
      "Pair{AbstractString,Int64}(\"Assay System kit\", 3)\n",
      "Pair{AbstractString,Int64}(\"odor presentation and\", 1)\n",
      "Pair{AbstractString,Int64}(\"mL )\", 17)\n",
      "Pair{AbstractString,Int64}(\"ms\", 2)\n",
      "Pair{AbstractString,Int64}(\"with the replication\", 1)\n",
      "Pair{AbstractString,Int64}(\"with increasing\", 1)\n",
      "Pair{AbstractString,Int64}(\"' -to-\", 2)\n",
      "Pair{AbstractString,Int64}(\"SO\", 5)\n",
      "Pair{AbstractString,Int64}(\"alignment of LY\", 1)\n",
      "Pair{AbstractString,Int64}(\"form\", 7)\n",
      "Pair{AbstractString,Int64}(\"https://www.gisaid.org/ )\", 1)\n",
      "Pair{AbstractString,Int64}(\"such artificial\", 1)\n",
      "Pair{AbstractString,Int64}(\"the single\", 1)\n",
      "Pair{AbstractString,Int64}(\"ketamine and xylazine\", 1)\n",
      "Pair{AbstractString,Int64}(\"e\", 55)\n",
      "Pair{AbstractString,Int64}(\"in liver (\", 2)\n",
      "Pair{AbstractString,Int64}(\"( SARS )\", 1)\n",
      "Pair{AbstractString,Int64}(\"S RNA bands\", 1)\n",
      "Pair{AbstractString,Int64}(\"of each protease.\", 1)\n",
      "Pair{AbstractString,Int64}(\"H ) Olfactory\", 1)\n",
      "Pair{AbstractString,Int64}(\"as R\", 1)\n",
      "Pair{AbstractString,Int64}(\"negative author/funder. All\", 1)\n",
      "Pair{AbstractString,Int64}(\"To this solution\", 1)\n",
      "Pair{AbstractString,Int64}(\"undetectable\", 2)\n",
      "Pair{AbstractString,Int64}(\"° C we\", 1)\n",
      "Pair{AbstractString,Int64}(\"p= 0.007 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"a template\", 1)\n",
      "Pair{AbstractString,Int64}(\"307 susceptible cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"Several free-enzyme crystals\", 1)\n",
      "Pair{AbstractString,Int64}(\"to 321 non-optimal\", 1)\n",
      "Pair{AbstractString,Int64}(\"Being\", 1)\n",
      "Pair{AbstractString,Int64}(\"are small ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"Flx 800 fluorescence\", 1)\n",
      "Pair{AbstractString,Int64}(\"in all splenic\", 1)\n",
      "Pair{AbstractString,Int64}(\"fibrosis 11 or\", 1)\n",
      "Pair{AbstractString,Int64}(\"abolished\", 5)\n",
      "Pair{AbstractString,Int64}(\"bioRxiv preprint baseline\", 1)\n",
      "Pair{AbstractString,Int64}(\"the maximum velocity\", 2)\n",
      "Pair{AbstractString,Int64}(\"had directly 92\", 1)\n",
      "Pair{AbstractString,Int64}(\"cases and 2,700\", 1)\n",
      "Pair{AbstractString,Int64}(\") of these\", 1)\n",
      "Pair{AbstractString,Int64}(\"P 250\", 11)\n",
      "Pair{AbstractString,Int64}(\"the tTA-responsive\", 1)\n",
      "Pair{AbstractString,Int64}(\"certain direction by\", 1)\n",
      "Pair{AbstractString,Int64}(\"concentrations ,\", 1)\n",
      "Pair{AbstractString,Int64}(\", the ratio\", 1)\n",
      "Pair{AbstractString,Int64}(\"Obtained results\", 1)\n",
      "Pair{AbstractString,Int64}(\"treatment for infections\", 1)\n",
      "Pair{AbstractString,Int64}(\"to low affinity.\", 1)\n",
      "Pair{AbstractString,Int64}(\"we infected HAE\", 1)\n",
      "Pair{AbstractString,Int64}(\"a commonly\", 1)\n",
      "Pair{AbstractString,Int64}(\"of virus in\", 3)\n",
      "Pair{AbstractString,Int64}(\", two mutations\", 1)\n",
      "Pair{AbstractString,Int64}(\"blocks fusion\", 1)\n",
      "Pair{AbstractString,Int64}(\"63 ) ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"the Ca 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"data can\", 1)\n",
      "Pair{AbstractString,Int64}(\"complex with\", 10)\n",
      "Pair{AbstractString,Int64}(\"administered intraperitoneally (\", 1)\n",
      "Pair{AbstractString,Int64}(\"exclude potential offtarget\", 1)\n",
      "Pair{AbstractString,Int64}(\"Sections of liver\", 1)\n",
      "Pair{AbstractString,Int64}(\"and then fixed\", 1)\n",
      "Pair{AbstractString,Int64}(\"protection from CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"expected to have\", 1)\n",
      "Pair{AbstractString,Int64}(\", 2015 ;\", 2)\n",
      "Pair{AbstractString,Int64}(\"323 replication of\", 1)\n",
      "Pair{AbstractString,Int64}(\"which silences\", 1)\n",
      "Pair{AbstractString,Int64}(\"replicons\", 20)\n",
      "Pair{AbstractString,Int64}(\"shaking for\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.34 g ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"degraded (\", 1)\n",
      "Pair{AbstractString,Int64}(\"modified version of\", 1)\n",
      "Pair{AbstractString,Int64}(\"generated our target\", 1)\n",
      "Pair{AbstractString,Int64}(\"acids that are\", 1)\n",
      "Pair{AbstractString,Int64}(\"representations\", 2)\n",
      "Pair{AbstractString,Int64}(\"by the 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"= 5 µM\", 1)\n",
      "Pair{AbstractString,Int64}(\"the medium contained\", 1)\n",
      "Pair{AbstractString,Int64}(\"RAxML v 8.1.21\", 1)\n",
      "Pair{AbstractString,Int64}(\"co-cultured with susceptible\", 1)\n",
      "Pair{AbstractString,Int64}(\"individual transcripts as\", 1)\n",
      "Pair{AbstractString,Int64}(\"N-terminal cMYC epitope\", 1)\n",
      "Pair{AbstractString,Int64}(\"whether MHV-ExoN\", 1)\n",
      "Pair{AbstractString,Int64}(\"performed herein show\", 1)\n",
      "Pair{AbstractString,Int64}(\"W= 56 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"AG ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"for enteroviruses\", 1)\n",
      "Pair{AbstractString,Int64}(\"A testing\", 1)\n",
      "Pair{AbstractString,Int64}(\"cassette\", 2)\n",
      "Pair{AbstractString,Int64}(\"became\", 1)\n",
      "Pair{AbstractString,Int64}(\"virus-infected Huh 7\", 2)\n",
      "Pair{AbstractString,Int64}(\"2 oxopyrrolidin-\", 1)\n",
      "Pair{AbstractString,Int64}(\"2008 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"The changes\", 1)\n",
      "Pair{AbstractString,Int64}(\"Then , 1\", 1)\n",
      "Pair{AbstractString,Int64}(\", Remove\", 1)\n",
      "Pair{AbstractString,Int64}(\"confirms that nsp\", 1)\n",
      "Pair{AbstractString,Int64}(\", 1.5\", 1)\n",
      "Pair{AbstractString,Int64}(\"capacity , and\", 1)\n",
      "Pair{AbstractString,Int64}(\"To ensure the\", 1)\n",
      "Pair{AbstractString,Int64}(\"animal\", 7)\n",
      "Pair{AbstractString,Int64}(\"in the ligand-free\", 1)\n",
      "Pair{AbstractString,Int64}(\"proteases tested. However\", 1)\n",
      "Pair{AbstractString,Int64}(\"suggest that silencing\", 1)\n",
      "Pair{AbstractString,Int64}(\"or VSV *\", 1)\n",
      "Pair{AbstractString,Int64}(\"BMDM RNA was\", 1)\n",
      "Pair{AbstractString,Int64}(\"spent on\", 2)\n",
      "Pair{AbstractString,Int64}(\"was similar but\", 1)\n",
      "Pair{AbstractString,Int64}(\"and Ifnar\", 1)\n",
      "Pair{AbstractString,Int64}(\"Bio-Assay\", 1)\n",
      "Pair{AbstractString,Int64}(\"http://virological.org/t/initialgenome-release-of-novel-coronavirus/ 319\", 1)\n",
      "Pair{AbstractString,Int64}(\"largely odor-specific (\", 1)\n",
      "Pair{AbstractString,Int64}(\"reaction buffer at\", 1)\n",
      "Pair{AbstractString,Int64}(\"32 33\", 4)\n",
      "Pair{AbstractString,Int64}(\"2019 in\", 1)\n",
      "Pair{AbstractString,Int64}(\"fitness , and\", 1)\n",
      "Pair{AbstractString,Int64}(\"and Siegelbaum ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"/nar/gkz 268 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.4688\", 1)\n",
      "Pair{AbstractString,Int64}(\"Schoggins\", 1)\n",
      "Pair{AbstractString,Int64}(\"cyclopropyl that showed\", 1)\n",
      "Pair{AbstractString,Int64}(\"to silence\", 1)\n",
      "Pair{AbstractString,Int64}(\"Campden Instruments ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"( bottom )\", 1)\n",
      "Pair{AbstractString,Int64}(\"performed by equilibrating\", 1)\n",
      "Pair{AbstractString,Int64}(\". Intermediate 251\", 1)\n",
      "Pair{AbstractString,Int64}(\"open )\", 1)\n",
      "Pair{AbstractString,Int64}(\"human pathogens.\", 1)\n",
      "Pair{AbstractString,Int64}(\"odors strongly depend\", 1)\n",
      "Pair{AbstractString,Int64}(\"their susceptibility to\", 1)\n",
      "Pair{AbstractString,Int64}(\"ribozyme\", 1)\n",
      "Pair{AbstractString,Int64}(\"the geometry\", 1)\n",
      "Pair{AbstractString,Int64}(\"test for\", 3)\n",
      "Pair{AbstractString,Int64}(\"Note that\", 2)\n",
      "Pair{AbstractString,Int64}(\"containing the three\", 1)\n",
      "Pair{AbstractString,Int64}(\"pathway of\", 1)\n",
      "Pair{AbstractString,Int64}(\"release the atomic\", 1)\n",
      "Pair{AbstractString,Int64}(\"were prepared as\", 1)\n",
      "Pair{AbstractString,Int64}(\", j )\", 1)\n",
      "Pair{AbstractString,Int64}(\"streptomycin sulfate ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"very\", 11)\n",
      "Pair{AbstractString,Int64}(\") in PAML\", 1)\n",
      "Pair{AbstractString,Int64}(\"presented here\", 1)\n",
      "Pair{AbstractString,Int64}(\"a 20 X\", 1)\n",
      "Pair{AbstractString,Int64}(\"Annotation Database\", 1)\n",
      "Pair{AbstractString,Int64}(\"condition of 0.1\", 1)\n",
      "Pair{AbstractString,Int64}(\"likely\", 4)\n",
      "Pair{AbstractString,Int64}(\"As the n-butyl\", 1)\n",
      "Pair{AbstractString,Int64}(\", but are\", 1)\n",
      "Pair{AbstractString,Int64}(\"odor-context\", 1)\n",
      "Pair{AbstractString,Int64}(\"Position 7 s\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"from olfactory sensory\", 1)\n",
      "Pair{AbstractString,Int64}(\"< 0.01 was\", 1)\n",
      "Pair{AbstractString,Int64}(\"association was\", 1)\n",
      "Pair{AbstractString,Int64}(\"bioRxiv preprint The\", 11)\n",
      "Pair{AbstractString,Int64}(\"f g\", 1)\n",
      "Pair{AbstractString,Int64}(\"had\", 19)\n",
      "Pair{AbstractString,Int64}(\"s each.\", 2)\n",
      "Pair{AbstractString,Int64}(\"used for RT-qPCR\", 1)\n",
      "Pair{AbstractString,Int64}(\"% protective\", 1)\n",
      "Pair{AbstractString,Int64}(\", with regard\", 1)\n",
      "Pair{AbstractString,Int64}(\"( DREADDs )\", 1)\n",
      "Pair{AbstractString,Int64}(\"was performed using\", 2)\n",
      "Pair{AbstractString,Int64}(\"of coronaviruses\", 1)\n",
      "Pair{AbstractString,Int64}(\"TaqMan Probe\", 1)\n",
      "Pair{AbstractString,Int64}(\"expected , this\", 1)\n",
      "Pair{AbstractString,Int64}(\"of the homogenate\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 1.69\", 2)\n",
      "Pair{AbstractString,Int64}(\"100 mL\", 1)\n",
      "Pair{AbstractString,Int64}(\"Sullivan , 2010\", 1)\n",
      "Pair{AbstractString,Int64}(\"30 ~\", 1)\n",
      "Pair{AbstractString,Int64}(\"to surface of\", 1)\n",
      "Pair{AbstractString,Int64}(\"specific infectivity analyses\", 1)\n",
      "Pair{AbstractString,Int64}(\"182 differed\", 1)\n",
      "Pair{AbstractString,Int64}(\"( bottom\", 1)\n",
      "Pair{AbstractString,Int64}(\"severe viral disease.\", 1)\n",
      "Pair{AbstractString,Int64}(\"column chromatography on\", 2)\n",
      "Pair{AbstractString,Int64}(\"chromatography on\", 2)\n",
      "Pair{AbstractString,Int64}(\"conditioning included\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 (\", 14)\n",
      "Pair{AbstractString,Int64}(\"view of 11\", 1)\n",
      "Pair{AbstractString,Int64}(\"253 phenotypically ExoN\", 1)\n",
      "Pair{AbstractString,Int64}(\", the enzyme\", 1)\n",
      "Pair{AbstractString,Int64}(\"or 48\", 1)\n",
      "Pair{AbstractString,Int64}(\", and from\", 1)\n",
      "Pair{AbstractString,Int64}(\"the NMDA receptor\", 1)\n",
      "Pair{AbstractString,Int64}(\"2016 ;\", 7)\n",
      "Pair{AbstractString,Int64}(\"suggest that R\", 1)\n",
      "Pair{AbstractString,Int64}(\"the odor port\", 2)\n",
      "Pair{AbstractString,Int64}(\"incrementally. To identify\", 1)\n",
      "Pair{AbstractString,Int64}(\"of the mother\", 1)\n",
      "Pair{AbstractString,Int64}(\"other human coronaviruses\", 2)\n",
      "Pair{AbstractString,Int64}(\"Fig. 3 b\", 1)\n",
      "Pair{AbstractString,Int64}(\"a secondary cleavage\", 1)\n",
      "Pair{AbstractString,Int64}(\"observations of endogenous\", 1)\n",
      "Pair{AbstractString,Int64}(\"replicon with\", 1)\n",
      "Pair{AbstractString,Int64}(\"buffer and then\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.43\", 1)\n",
      "Pair{AbstractString,Int64}(\"2020 ( Fig.\", 1)\n",
      "Pair{AbstractString,Int64}(\"piriform neural ensembles\", 2)\n",
      "Pair{AbstractString,Int64}(\"screening of\", 1)\n",
      "Pair{AbstractString,Int64}(\"proKLK 14\", 1)\n",
      "Pair{AbstractString,Int64}(\"MOI ) were\", 1)\n",
      "Pair{AbstractString,Int64}(\"S cleavage or\", 1)\n",
      "Pair{AbstractString,Int64}(\"stronger negative (\", 1)\n",
      "Pair{AbstractString,Int64}(\"between MluI\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells form a\", 1)\n",
      "Pair{AbstractString,Int64}(\"a partial memory\", 1)\n",
      "Pair{AbstractString,Int64}(\"with Prism\", 1)\n",
      "Pair{AbstractString,Int64}(\"acids from\", 1)\n",
      "Pair{AbstractString,Int64}(\"TMPRSS 2 cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"piriform cortex with\", 1)\n",
      "Pair{AbstractString,Int64}(\"presentation of 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"The quality\", 1)\n",
      "Pair{AbstractString,Int64}(\"dilution with sterile\", 1)\n",
      "Pair{AbstractString,Int64}(\"n by\", 1)\n",
      "Pair{AbstractString,Int64}(\", the replicon\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 25\", 1)\n",
      "Pair{AbstractString,Int64}(\"and in HCoV-NL\", 1)\n",
      "Pair{AbstractString,Int64}(\"goat anti-mouse AlexaFluor\", 1)\n",
      "Pair{AbstractString,Int64}(\"( positive\", 1)\n",
      "Pair{AbstractString,Int64}(\"the flexibility of\", 1)\n",
      "Pair{AbstractString,Int64}(\"1.95\", 1)\n",
      "Pair{AbstractString,Int64}(\"a cFos-dependent\", 1)\n",
      "Pair{AbstractString,Int64}(\"of 355\", 1)\n",
      "Pair{AbstractString,Int64}(\"kb\", 2)\n",
      "Pair{AbstractString,Int64}(\"increased liver damage\", 1)\n",
      "Pair{AbstractString,Int64}(\". Selected\", 1)\n",
      "Pair{AbstractString,Int64}(\"Life Technologies ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"nonreinforced stimuli\", 1)\n",
      "Pair{AbstractString,Int64}(\"p= 0.0269 for\", 1)\n",
      "Pair{AbstractString,Int64}(\"= Ctcompetitor\", 1)\n",
      "Pair{AbstractString,Int64}(\"The evolutionary pathways\", 1)\n",
      "Pair{AbstractString,Int64}(\"high\", 23)\n",
      "Pair{AbstractString,Int64}(\"sub-lethal hepatitis\", 1)\n",
      "Pair{AbstractString,Int64}(\"GPI-anchored\", 1)\n",
      "Pair{AbstractString,Int64}(\"the protein 248\", 1)\n",
      "Pair{AbstractString,Int64}(\"the Nterminal two\", 1)\n",
      "Pair{AbstractString,Int64}(\"at 237 <\", 1)\n",
      "Pair{AbstractString,Int64}(\"MERS-CoV when\", 1)\n",
      "Pair{AbstractString,Int64}(\"W= 34 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"create\", 2)\n",
      "Pair{AbstractString,Int64}(\"scale as image\", 1)\n",
      "Pair{AbstractString,Int64}(\"and both\", 1)\n",
      "Pair{AbstractString,Int64}(\"of precise temporal\", 1)\n",
      "Pair{AbstractString,Int64}(\"the previous\", 1)\n",
      "Pair{AbstractString,Int64}(\"and Cys 145\", 1)\n",
      "Pair{AbstractString,Int64}(\"The conformational\", 1)\n",
      "Pair{AbstractString,Int64}(\"in CH\", 3)\n",
      "Pair{AbstractString,Int64}(\"of replicon-transfected\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells using\", 1)\n",
      "Pair{AbstractString,Int64}(\"ExoN-DE. These\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.672\", 1)\n",
      "Pair{AbstractString,Int64}(\"hours post-transfection ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"using HiScript\", 1)\n",
      "Pair{AbstractString,Int64}(\"; HKU 4\", 1)\n",
      "Pair{AbstractString,Int64}(\"In accordance\", 1)\n",
      "Pair{AbstractString,Int64}(\". Crystallization was\", 1)\n",
      "Pair{AbstractString,Int64}(\"ensure that\", 1)\n",
      "Pair{AbstractString,Int64}(\"change compared with\", 1)\n",
      "Pair{AbstractString,Int64}(\"post-infection , there\", 1)\n",
      "Pair{AbstractString,Int64}(\"sgRNA were\", 1)\n",
      "Pair{AbstractString,Int64}(\", doxycycline was\", 1)\n",
      "Pair{AbstractString,Int64}(\"to serious outcome\", 1)\n",
      "Pair{AbstractString,Int64}(\"fluorophor coumaryl and\", 1)\n",
      "Pair{AbstractString,Int64}(\"21 cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"( BioTek\", 1)\n",
      "Pair{AbstractString,Int64}(\"were done as\", 1)\n",
      "Pair{AbstractString,Int64}(\"Another view\", 1)\n",
      "Pair{AbstractString,Int64}(\"the intermediate\", 1)\n",
      "Pair{AbstractString,Int64}(\"To this\", 1)\n",
      "Pair{AbstractString,Int64}(\", the open\", 1)\n",
      "Pair{AbstractString,Int64}(\"experimental conditions. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"close relative\", 1)\n",
      "Pair{AbstractString,Int64}(\"also from the\", 1)\n",
      "Pair{AbstractString,Int64}(\"s solutions that\", 1)\n",
      "Pair{AbstractString,Int64}(\"the vast\", 1)\n",
      "Pair{AbstractString,Int64}(\"input-output\", 1)\n",
      "Pair{AbstractString,Int64}(\"it might\", 1)\n",
      "Pair{AbstractString,Int64}(\"E again\", 1)\n",
      "Pair{AbstractString,Int64}(\"could induce the\", 1)\n",
      "Pair{AbstractString,Int64}(\"for His-tagged proteins.\", 1)\n",
      "Pair{AbstractString,Int64}(\"-FRAG ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"but not male\", 2)\n",
      "Pair{AbstractString,Int64}(\"in which Fos-tagged\", 1)\n",
      "Pair{AbstractString,Int64}(\"313 -RNF\", 1)\n",
      "Pair{AbstractString,Int64}(\"reservoir composed\", 1)\n",
      "Pair{AbstractString,Int64}(\"families\", 1)\n",
      "Pair{AbstractString,Int64}(\"intermediates 4 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"protocol. Liver and\", 1)\n",
      "Pair{AbstractString,Int64}(\"generate clonal LY\", 1)\n",
      "Pair{AbstractString,Int64}(\"( the\", 5)\n",
      "Pair{AbstractString,Int64}(\"2 site\", 4)\n",
      "Pair{AbstractString,Int64}(\"i.p. injected the\", 1)\n",
      "Pair{AbstractString,Int64}(\"significant decrease in\", 1)\n",
      "Pair{AbstractString,Int64}(\"disease (\", 2)\n",
      "Pair{AbstractString,Int64}(\"tTA-responsive\", 1)\n",
      "Pair{AbstractString,Int64}(\"273 from\", 1)\n",
      "Pair{AbstractString,Int64}(\"G and H\", 2)\n",
      "Pair{AbstractString,Int64}(\"height )\", 1)\n",
      "Pair{AbstractString,Int64}(\"recommended\", 2)\n",
      "Pair{AbstractString,Int64}(\"control ) and\", 1)\n",
      "Pair{AbstractString,Int64}(\"specifically due\", 1)\n",
      "Pair{AbstractString,Int64}(\"time points ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"neurons during contextual\", 1)\n",
      "Pair{AbstractString,Int64}(\"the relative\", 2)\n",
      "Pair{AbstractString,Int64}(\"cell-based assay (\", 1)\n",
      "Pair{AbstractString,Int64}(\"sequence was replaced\", 1)\n",
      "Pair{AbstractString,Int64}(\"The side-chains of\", 3)\n",
      "Pair{AbstractString,Int64}(\"fidelity , fitness\", 1)\n",
      "Pair{AbstractString,Int64}(\"suitable one\", 1)\n",
      "Pair{AbstractString,Int64}(\"coronavirus template\", 1)\n",
      "Pair{AbstractString,Int64}(\"been 334\", 1)\n",
      "Pair{AbstractString,Int64}(\"as we predicted\", 1)\n",
      "Pair{AbstractString,Int64}(\"etiological agents of\", 1)\n",
      "Pair{AbstractString,Int64}(\"solution ) against\", 1)\n",
      "Pair{AbstractString,Int64}(\"identified LY 6\", 1)\n",
      "Pair{AbstractString,Int64}(\"alanine substitution of\", 1)\n",
      "Pair{AbstractString,Int64}(\"sensitivity of the\", 1)\n",
      "Pair{AbstractString,Int64}(\"they described\", 1)\n",
      "Pair{AbstractString,Int64}(\"entry. As we\", 1)\n",
      "Pair{AbstractString,Int64}(\"studies will\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibitor.\", 1)\n",
      "Pair{AbstractString,Int64}(\"Differences in data\", 1)\n",
      "Pair{AbstractString,Int64}(\"a 67\", 1)\n",
      "Pair{AbstractString,Int64}(\"a 7\", 2)\n",
      "Pair{AbstractString,Int64}(\"the MEGA-X\", 1)\n",
      "Pair{AbstractString,Int64}(\"n.s.\", 3)\n",
      "Pair{AbstractString,Int64}(\"last accessed\", 1)\n",
      "Pair{AbstractString,Int64}(\"286 All rights\", 1)\n",
      "Pair{AbstractString,Int64}(\"are normalized read\", 1)\n",
      "Pair{AbstractString,Int64}(\"Tan\", 2)\n",
      "Pair{AbstractString,Int64}(\"and has therefore\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"145 , 144\", 1)\n",
      "Pair{AbstractString,Int64}(\"significantly increased fitness\", 1)\n",
      "Pair{AbstractString,Int64}(\"be the one\", 1)\n",
      "Pair{AbstractString,Int64}(\", Glu 71\", 1)\n",
      "Pair{AbstractString,Int64}(\"as 2019 -nCoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"CPE or by\", 1)\n",
      "Pair{AbstractString,Int64}(\"RNA-Seq analysis. BMDM\", 1)\n",
      "Pair{AbstractString,Int64}(\"of cells.\", 1)\n",
      "Pair{AbstractString,Int64}(\"and neutral odors\", 1)\n",
      "Pair{AbstractString,Int64}(\"otherwise stated.\", 1)\n",
      "Pair{AbstractString,Int64}(\"were fished and\", 1)\n",
      "Pair{AbstractString,Int64}(\"and 40 µL\", 1)\n",
      "Pair{AbstractString,Int64}(\"using ice-cold passive\", 1)\n",
      "Pair{AbstractString,Int64}(\"started with\", 3)\n",
      "Pair{AbstractString,Int64}(\"1 C\", 2)\n",
      "Pair{AbstractString,Int64}(\"331\", 1)\n",
      "Pair{AbstractString,Int64}(\"following step\", 2)\n",
      "Pair{AbstractString,Int64}(\"a homodimer\", 1)\n",
      "Pair{AbstractString,Int64}(\". The expression\", 1)\n",
      "Pair{AbstractString,Int64}(\"database , https://www.ncbi.nlm.nih.gov/geo\", 1)\n",
      "Pair{AbstractString,Int64}(\"coronavirus replication\", 1)\n",
      "Pair{AbstractString,Int64}(\"are prepared\", 1)\n",
      "Pair{AbstractString,Int64}(\"the distance\", 3)\n",
      "Pair{AbstractString,Int64}(\"despite their\", 1)\n",
      "Pair{AbstractString,Int64}(\"What\", 2)\n",
      "Pair{AbstractString,Int64}(\"was apparently\", 1)\n",
      "Pair{AbstractString,Int64}(\"solid\", 5)\n",
      "Pair{AbstractString,Int64}(\"cycle\", 2)\n",
      "Pair{AbstractString,Int64}(\"150 terminal region\", 1)\n",
      "Pair{AbstractString,Int64}(\"identity to that\", 1)\n",
      "Pair{AbstractString,Int64}(\"similar efficacy\", 1)\n",
      "Pair{AbstractString,Int64}(\", patients\", 1)\n",
      "Pair{AbstractString,Int64}(\"have dual\", 1)\n",
      "Pair{AbstractString,Int64}(\"here show\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 -S )\", 1)\n",
      "Pair{AbstractString,Int64}(\"microscope and processed\", 1)\n",
      "Pair{AbstractString,Int64}(\"infection , indicating\", 1)\n",
      "Pair{AbstractString,Int64}(\"were immediately transferred\", 1)\n",
      "Pair{AbstractString,Int64}(\"r against MERS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"at least 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"The recombinant\", 1)\n",
      "Pair{AbstractString,Int64}(\"total presentation of\", 1)\n",
      "Pair{AbstractString,Int64}(\"create a\", 1)\n",
      "Pair{AbstractString,Int64}(\"immunopathology in infected\", 1)\n",
      "Pair{AbstractString,Int64}(\"503 ( Vazyme\", 1)\n",
      "Pair{AbstractString,Int64}(\"not significantly\", 2)\n",
      "Pair{AbstractString,Int64}(\"effect of\", 10)\n",
      "Pair{AbstractString,Int64}(\"pCAGGS-mCherry ) as\", 1)\n",
      "Pair{AbstractString,Int64}(\"lysates (\", 1)\n",
      "Pair{AbstractString,Int64}(\"treatment.\", 1)\n",
      "Pair{AbstractString,Int64}(\"coordinates include\", 1)\n",
      "Pair{AbstractString,Int64}(\"FBS/PBS\", 1)\n",
      "Pair{AbstractString,Int64}(\"( from Leu\", 1)\n",
      "Pair{AbstractString,Int64}(\"account.\", 2)\n",
      "Pair{AbstractString,Int64}(\"sample incubated at\", 2)\n",
      "Pair{AbstractString,Int64}(\", dN/dS of\", 1)\n",
      "Pair{AbstractString,Int64}(\"foot shock. Behavioral\", 1)\n",
      "Pair{AbstractString,Int64}(\"ExoN-AE arising\", 1)\n",
      "Pair{AbstractString,Int64}(\"µg\", 3)\n",
      "Pair{AbstractString,Int64}(\"E KO\", 2)\n",
      "Pair{AbstractString,Int64}(\"; their\", 1)\n",
      "Pair{AbstractString,Int64}(\"mice show a\", 1)\n",
      "Pair{AbstractString,Int64}(\"were silenced using\", 1)\n",
      "Pair{AbstractString,Int64}(\"henceforth called\", 1)\n",
      "Pair{AbstractString,Int64}(\"194 entry into\", 1)\n",
      "Pair{AbstractString,Int64}(\"9 S )\", 1)\n",
      "Pair{AbstractString,Int64}(\"TriM Scope\", 1)\n",
      "Pair{AbstractString,Int64}(\"a plethysmograph.\", 1)\n",
      "Pair{AbstractString,Int64}(\"light-brown filtrate\", 1)\n",
      "Pair{AbstractString,Int64}(\"ciclesonide ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"Sevelinges\", 1)\n",
      "Pair{AbstractString,Int64}(\"Fig. 5\", 7)\n",
      "Pair{AbstractString,Int64}(\"enzyme crystals was\", 1)\n",
      "Pair{AbstractString,Int64}(\"trace ' (\", 1)\n",
      "Pair{AbstractString,Int64}(\"pro-KLK 13 242\", 1)\n",
      "Pair{AbstractString,Int64}(\", and HRV\", 2)\n",
      "Pair{AbstractString,Int64}(\") . Support\", 1)\n",
      "Pair{AbstractString,Int64}(\"yellow\", 3)\n",
      "Pair{AbstractString,Int64}(\"] [ 51\", 1)\n",
      "Pair{AbstractString,Int64}(\"ExoN-AA did\", 1)\n",
      "Pair{AbstractString,Int64}(\"15 o C\", 1)\n",
      "Pair{AbstractString,Int64}(\"ThermoFisher\", 2)\n",
      "Pair{AbstractString,Int64}(\"2016 ; Yamada\", 1)\n",
      "Pair{AbstractString,Int64}(\"are the\", 2)\n",
      "Pair{AbstractString,Int64}(\"chromatography ( TLC\", 1)\n",
      "Pair{AbstractString,Int64}(\"60 Hz\", 1)\n",
      "Pair{AbstractString,Int64}(\"non-toxic at this\", 1)\n",
      "Pair{AbstractString,Int64}(\"complementation of split\", 1)\n",
      "Pair{AbstractString,Int64}(\"we investigate\", 1)\n",
      "Pair{AbstractString,Int64}(\"( CoV )\", 1)\n",
      "Pair{AbstractString,Int64}(\"d\", 16)\n",
      "Pair{AbstractString,Int64}(\"/ 2020.03.05.979260\", 32)\n",
      "Pair{AbstractString,Int64}(\"frequency , reflecting\", 1)\n",
      "Pair{AbstractString,Int64}(\"the fusion\", 1)\n",
      "Pair{AbstractString,Int64}(\"cctgaacccagctttcttgtacaaagtggtcccc\", 1)\n",
      "Pair{AbstractString,Int64}(\"are much more\", 1)\n",
      "Pair{AbstractString,Int64}(\"of enteroviral replicons\", 1)\n",
      "Pair{AbstractString,Int64}(\"dimethyl\", 4)\n",
      "Pair{AbstractString,Int64}(\"% ( cut\", 1)\n",
      "Pair{AbstractString,Int64}(\"demonstrated to lead\", 1)\n",
      "Pair{AbstractString,Int64}(\"week-old\", 1)\n",
      "Pair{AbstractString,Int64}(\"that appear to\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 :\", 3)\n",
      "Pair{AbstractString,Int64}(\"than that\", 9)\n",
      "Pair{AbstractString,Int64}(\") and AAV\", 1)\n",
      "Pair{AbstractString,Int64}(\"( M ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"96 RNA kit\", 2)\n",
      "Pair{AbstractString,Int64}(\"FIPV ) ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"that re-exposure\", 1)\n",
      "Pair{AbstractString,Int64}(\"was calculated as\", 3)\n",
      "Pair{AbstractString,Int64}(\"fifth\", 1)\n",
      "Pair{AbstractString,Int64}(\"Mayford and\", 1)\n",
      "Pair{AbstractString,Int64}(\"explanation for\", 1)\n",
      "Pair{AbstractString,Int64}(\"were downloaded\", 2)\n",
      "Pair{AbstractString,Int64}(\"preference\", 1)\n",
      "Pair{AbstractString,Int64}(\") Representative\", 1)\n",
      "Pair{AbstractString,Int64}(\"for 2 hours\", 4)\n",
      "Pair{AbstractString,Int64}(\"common cold to\", 1)\n",
      "Pair{AbstractString,Int64}(\"< 20\", 1)\n",
      "Pair{AbstractString,Int64}(\"performed against\", 1)\n",
      "Pair{AbstractString,Int64}(\"modifications 7,8\", 1)\n",
      "Pair{AbstractString,Int64}(\"among different\", 1)\n",
      "Pair{AbstractString,Int64}(\", with selectivity\", 1)\n",
      "Pair{AbstractString,Int64}(\"pdU ) G\", 1)\n",
      "Pair{AbstractString,Int64}(\"perturbed in\", 1)\n",
      "Pair{AbstractString,Int64}(\"multiplicity\", 1)\n",
      "Pair{AbstractString,Int64}(\"a ) and\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells according\", 1)\n",
      "Pair{AbstractString,Int64}(\"342 Petaluma ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"% author/funder.\", 2)\n",
      "Pair{AbstractString,Int64}(\"their alterations of\", 1)\n",
      "Pair{AbstractString,Int64}(\"031439745.1\", 1)\n",
      "Pair{AbstractString,Int64}(\"Sigma # N\", 1)\n",
      "Pair{AbstractString,Int64}(\"over common isotonic\", 1)\n",
      "Pair{AbstractString,Int64}(\"tool (\", 1)\n",
      "Pair{AbstractString,Int64}(\"box was used\", 2)\n",
      "Pair{AbstractString,Int64}(\"recommendations. LY 6\", 1)\n",
      "Pair{AbstractString,Int64}(\", an isobutyl\", 1)\n",
      "Pair{AbstractString,Int64}(\", treating ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"a new\", 2)\n",
      "Pair{AbstractString,Int64}(\"the indicated\", 4)\n",
      "Pair{AbstractString,Int64}(\"euthanized 3 -or\", 1)\n",
      "Pair{AbstractString,Int64}(\". One mutation\", 1)\n",
      "Pair{AbstractString,Int64}(\"RBC lysis\", 1)\n",
      "Pair{AbstractString,Int64}(\": ethyl-acetate. (\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.1 M\", 14)\n",
      "Pair{AbstractString,Int64}(\"is consistent\", 2)\n",
      "Pair{AbstractString,Int64}(\"sources. Different organisms\", 1)\n",
      "Pair{AbstractString,Int64}(\"IC 50 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"underlying olfactory learning\", 1)\n",
      "Pair{AbstractString,Int64}(\"n= 8\", 9)\n",
      "Pair{AbstractString,Int64}(\"- 70\", 1)\n",
      "Pair{AbstractString,Int64}(\"11 e and\", 2)\n",
      "Pair{AbstractString,Int64}(\"column chromatography\", 4)\n",
      "Pair{AbstractString,Int64}(\"phenyl ring of\", 1)\n",
      "Pair{AbstractString,Int64}(\"network mechanisms can\", 1)\n",
      "Pair{AbstractString,Int64}(\"cyclopropylmethyl ( 11\", 1)\n",
      "Pair{AbstractString,Int64}(\"for weight\", 1)\n",
      "Pair{AbstractString,Int64}(\"results suggest\", 5)\n",
      "Pair{AbstractString,Int64}(\"deaths\", 1)\n",
      "Pair{AbstractString,Int64}(\"the fits\", 1)\n",
      "Pair{AbstractString,Int64}(\"together using 1.5\", 1)\n",
      "Pair{AbstractString,Int64}(\"were prepared with\", 1)\n",
      "Pair{AbstractString,Int64}(\"in future\", 1)\n",
      "Pair{AbstractString,Int64}(\"use of a\", 1)\n",
      "Pair{AbstractString,Int64}(\"from where\", 1)\n",
      "Pair{AbstractString,Int64}(\"of the experimental\", 1)\n",
      "Pair{AbstractString,Int64}(\"spike genes\", 1)\n",
      "Pair{AbstractString,Int64}(\"CoVs (\", 1)\n",
      "Pair{AbstractString,Int64}(\"commercially available TaqManTM\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.0312 ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\"shock. The CS+\", 1)\n",
      "Pair{AbstractString,Int64}(\"sequencing identified\", 1)\n",
      "Pair{AbstractString,Int64}(\". One NS\", 1)\n",
      "Pair{AbstractString,Int64}(\"its host\", 1)\n",
      "Pair{AbstractString,Int64}(\"and quality\", 1)\n",
      "Pair{AbstractString,Int64}(\"to that of\", 1)\n",
      "Pair{AbstractString,Int64}(\"2004 .\", 1)\n",
      "Pair{AbstractString,Int64}(\"and intracellular Renilla\", 1)\n",
      "Pair{AbstractString,Int64}(\"equilibration of\", 1)\n",
      "Pair{AbstractString,Int64}(\"on a\", 7)\n",
      "Pair{AbstractString,Int64}(\"seeded in 12\", 1)\n",
      "Pair{AbstractString,Int64}(\"leukemia , 12\", 1)\n",
      "Pair{AbstractString,Int64}(\"C All rights\", 1)\n",
      "Pair{AbstractString,Int64}(\"that in addition\", 1)\n",
      "Pair{AbstractString,Int64}(\"change.\", 1)\n",
      "Pair{AbstractString,Int64}(\"F )\", 6)\n",
      "Pair{AbstractString,Int64}(\"infection , the\", 1)\n",
      "Pair{AbstractString,Int64}(\"progress of incubation\", 2)\n",
      "Pair{AbstractString,Int64}(\"accumulate\", 1)\n",
      "Pair{AbstractString,Int64}(\"Automated cell detection\", 1)\n",
      "Pair{AbstractString,Int64}(\"standard DNA marker\", 1)\n",
      "Pair{AbstractString,Int64}(\"' , P\", 1)\n",
      "Pair{AbstractString,Int64}(\"Following this ,\", 1)\n",
      "Pair{AbstractString,Int64}(\") and selective\", 1)\n",
      "Pair{AbstractString,Int64}(\"the concentration\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 infection are\", 1)\n",
      "Pair{AbstractString,Int64}(\"s minimal essential\", 1)\n",
      "Pair{AbstractString,Int64}(\"RD _ ctrl\", 4)\n",
      "Pair{AbstractString,Int64}(\"Tests\", 3)\n",
      "Pair{AbstractString,Int64}(\"of 2019 -nCoV\", 7)\n",
      "Pair{AbstractString,Int64}(\"entry and\", 3)\n",
      "Pair{AbstractString,Int64}(\"can accept\", 4)\n",
      "Pair{AbstractString,Int64}(\"in control mice\", 1)\n",
      "Pair{AbstractString,Int64}(\"\\\" open\", 1)\n",
      "Pair{AbstractString,Int64}(\"suppressed\", 2)\n",
      "Pair{AbstractString,Int64}(\"BMDM ) were\", 1)\n",
      "Pair{AbstractString,Int64}(\"the resulting\", 3)\n",
      "Pair{AbstractString,Int64}(\"for murine CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"p= 0.7209 ,\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"glycerol\", 3)\n",
      "Pair{AbstractString,Int64}(\"from 24\", 1)\n",
      "Pair{AbstractString,Int64}(\"hosts ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"50 % [\", 1)\n",
      "Pair{AbstractString,Int64}(\"[ 4 ]\", 1)\n",
      "Pair{AbstractString,Int64}(\"as a\", 19)\n",
      "Pair{AbstractString,Int64}(\"was recently demonstrated\", 1)\n",
      "Pair{AbstractString,Int64}(\"stimulation of\", 1)\n",
      "Pair{AbstractString,Int64}(\"DNAPTP\", 1)\n",
      "Pair{AbstractString,Int64}(\"the odors\", 1)\n",
      "Pair{AbstractString,Int64}(\"L 36 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"148 The\", 1)\n",
      "Pair{AbstractString,Int64}(\"be the development\", 1)\n",
      "Pair{AbstractString,Int64}(\"that developed\", 1)\n",
      "Pair{AbstractString,Int64}(\"angiotensin-converting\", 1)\n",
      "Pair{AbstractString,Int64}(\"preprint between\", 1)\n",
      "Pair{AbstractString,Int64}(\"moiety has\", 1)\n",
      "Pair{AbstractString,Int64}(\"of its\", 2)\n",
      "Pair{AbstractString,Int64}(\"3 facilities\", 1)\n",
      "Pair{AbstractString,Int64}(\"protecting group of\", 1)\n",
      "Pair{AbstractString,Int64}(\"traveller returning\", 1)\n",
      "Pair{AbstractString,Int64}(\"paper. Correspondence\", 1)\n",
      "Pair{AbstractString,Int64}(\"In a\", 1)\n",
      "Pair{AbstractString,Int64}(\"primers adding\", 1)\n",
      "Pair{AbstractString,Int64}(\"their codon\", 1)\n",
      "Pair{AbstractString,Int64}(\"result ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"Representative\", 1)\n",
      "Pair{AbstractString,Int64}(\"HPLC and was\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 6 PFU/mL\", 1)\n",
      "Pair{AbstractString,Int64}(\"area percentage\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 c\", 1)\n",
      "Pair{AbstractString,Int64}(\"antiviral effect of\", 1)\n",
      "Pair{AbstractString,Int64}(\"light/ 12 -hour\", 1)\n",
      "Pair{AbstractString,Int64}(\"view shown here\", 1)\n",
      "Pair{AbstractString,Int64}(\"of two non-reinforced\", 1)\n",
      "Pair{AbstractString,Int64}(\"to either half\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells , while\", 1)\n",
      "Pair{AbstractString,Int64}(\"characteristics of piriform\", 1)\n",
      "Pair{AbstractString,Int64}(\"in the reference\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 ' ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"also not changed\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 -oxopyrrolidin- 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"steroid , did\", 1)\n",
      "Pair{AbstractString,Int64}(\"studying learning and\", 1)\n",
      "Pair{AbstractString,Int64}(\"non-synonymous to synonymous\", 1)\n",
      "Pair{AbstractString,Int64}(\"kits\", 1)\n",
      "Pair{AbstractString,Int64}(\"was relatively\", 1)\n",
      "Pair{AbstractString,Int64}(\"S , 4\", 1)\n",
      "Pair{AbstractString,Int64}(\"defining\", 1)\n",
      "Pair{AbstractString,Int64}(\"Biosearch Technologies\", 2)\n",
      "Pair{AbstractString,Int64}(\"that nsp\", 1)\n",
      "Pair{AbstractString,Int64}(\"above. and RNA\", 1)\n",
      "Pair{AbstractString,Int64}(\"to proteolytic activation\", 2)\n",
      "Pair{AbstractString,Int64}(\"but significantly\", 1)\n",
      "Pair{AbstractString,Int64}(\"54 . Briefly\", 1)\n",
      "Pair{AbstractString,Int64}(\"yield ) ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"cases are\", 2)\n",
      "Pair{AbstractString,Int64}(\"test that\", 1)\n",
      "Pair{AbstractString,Int64}(\"Scientific ) and\", 2)\n",
      "Pair{AbstractString,Int64}(\"317 not\", 1)\n",
      "Pair{AbstractString,Int64}(\"the proteolytic activation\", 1)\n",
      "Pair{AbstractString,Int64}(\"further research\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 47 mL\", 1)\n",
      "Pair{AbstractString,Int64}(\"229 230 Fixed\", 1)\n",
      "Pair{AbstractString,Int64}(\"N-substituted amino\", 1)\n",
      "Pair{AbstractString,Int64}(\"whether the 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"match of size\", 1)\n",
      "Pair{AbstractString,Int64}(\"RD HCoV-HKU 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"of other strain\", 1)\n",
      "Pair{AbstractString,Int64}(\"did not suppress\", 1)\n",
      "Pair{AbstractString,Int64}(\"is pseudo-randomized.\", 1)\n",
      "Pair{AbstractString,Int64}(\"increased expression\", 1)\n",
      "Pair{AbstractString,Int64}(\"6 f-h )\", 1)\n",
      "Pair{AbstractString,Int64}(\"E for\", 1)\n",
      "Pair{AbstractString,Int64}(\"diffused\", 1)\n",
      "Pair{AbstractString,Int64}(\", d\", 2)\n",
      "Pair{AbstractString,Int64}(\"and formed 162\", 1)\n",
      "Pair{AbstractString,Int64}(\"by far\", 1)\n",
      "Pair{AbstractString,Int64}(\"side-chain. The main-chain\", 1)\n",
      "Pair{AbstractString,Int64}(\"5 High-fidelity\", 1)\n",
      "Pair{AbstractString,Int64}(\"of the alcohol\", 1)\n",
      "Pair{AbstractString,Int64}(\"in virus-infected Huh\", 2)\n",
      "Pair{AbstractString,Int64}(\"spike gene\", 10)\n",
      "Pair{AbstractString,Int64}(\"and CSpresentation. They\", 1)\n",
      "Pair{AbstractString,Int64}(\"entry 2 BX\", 1)\n",
      "Pair{AbstractString,Int64}(\"from the main-chain\", 4)\n",
      "Pair{AbstractString,Int64}(\"or are\", 1)\n",
      "Pair{AbstractString,Int64}(\"than Ly 6\", 1)\n",
      "Pair{AbstractString,Int64}(\"Ifnar -/mice. In\", 1)\n",
      "Pair{AbstractString,Int64}(\". Then the\", 1)\n",
      "Pair{AbstractString,Int64}(\"Passive\", 1)\n",
      "Pair{AbstractString,Int64}(\"Lee\", 1)\n",
      "Pair{AbstractString,Int64}(\"both hemispheres ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"indeed the priming\", 1)\n",
      "Pair{AbstractString,Int64}(\"was added. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"available\", 18)\n",
      "Pair{AbstractString,Int64}(\"Miura et al.\", 3)\n",
      "Pair{AbstractString,Int64}(\"and 341 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"to escape from\", 2)\n",
      "Pair{AbstractString,Int64}(\"species , have\", 1)\n",
      "Pair{AbstractString,Int64}(\"catalytic triad\", 1)\n",
      "Pair{AbstractString,Int64}(\"quality of\", 2)\n",
      "Pair{AbstractString,Int64}(\"the substrate-binding\", 3)\n",
      "Pair{AbstractString,Int64}(\", triethylamine (\", 2)\n",
      "Pair{AbstractString,Int64}(\"accession\", 2)\n",
      "Pair{AbstractString,Int64}(\"SARS-CoV- 2 was\", 4)\n",
      "Pair{AbstractString,Int64}(\"b-unsaturated esters (\", 1)\n",
      "Pair{AbstractString,Int64}(\"Sigma-Aldrich ) .\", 2)\n",
      "Pair{AbstractString,Int64}(\"long been hypothesized\", 1)\n",
      "Pair{AbstractString,Int64}(\"of the crystal\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 E )\", 1)\n",
      "Pair{AbstractString,Int64}(\", VIPERIN\", 1)\n",
      "Pair{AbstractString,Int64}(\"Ciclesonide\", 3)\n",
      "Pair{AbstractString,Int64}(\"membrane fusion can\", 1)\n",
      "Pair{AbstractString,Int64}(\"% protective effect\", 1)\n",
      "Pair{AbstractString,Int64}(\"50 , 72\", 1)\n",
      "Pair{AbstractString,Int64}(\"test ( d-e\", 1)\n",
      "Pair{AbstractString,Int64}(\". Importantly\", 1)\n",
      "Pair{AbstractString,Int64}(\"Fos-tagging resulted\", 1)\n",
      "Pair{AbstractString,Int64}(\"Cytometer with\", 1)\n",
      "Pair{AbstractString,Int64}(\"was 14.2\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 O were\", 1)\n",
      "Pair{AbstractString,Int64}(\"organs was\", 1)\n",
      "Pair{AbstractString,Int64}(\"n\", 17)\n",
      "Pair{AbstractString,Int64}(\"whether any\", 1)\n",
      "Pair{AbstractString,Int64}(\"= cyclopentylmethyl )\", 1)\n",
      "Pair{AbstractString,Int64}(\"mL of CH\", 2)\n",
      "Pair{AbstractString,Int64}(\"2 pockets (\", 1)\n",
      "Pair{AbstractString,Int64}(\". 116 Only\", 1)\n",
      "Pair{AbstractString,Int64}(\"reagent\", 1)\n",
      "Pair{AbstractString,Int64}(\"prepared with an\", 1)\n",
      "Pair{AbstractString,Int64}(\"serious outcome\", 1)\n",
      "Pair{AbstractString,Int64}(\"and 200 mM\", 1)\n",
      "Pair{AbstractString,Int64}(\"threshold ( see\", 1)\n",
      "Pair{AbstractString,Int64}(\"detected with a\", 1)\n",
      "Pair{AbstractString,Int64}(\"that ciclesonide interacted\", 1)\n",
      "Pair{AbstractString,Int64}(\"mice of the\", 1)\n",
      "Pair{AbstractString,Int64}(\"related to problems\", 1)\n",
      "Pair{AbstractString,Int64}(\"Among\", 6)\n",
      "Pair{AbstractString,Int64}(\"and Wilson\", 2)\n",
      "Pair{AbstractString,Int64}(\"much larger\", 1)\n",
      "Pair{AbstractString,Int64}(\"d ) and\", 1)\n",
      "Pair{AbstractString,Int64}(\"SD. P value\", 1)\n",
      "Pair{AbstractString,Int64}(\"cause clinical disease\", 1)\n",
      "Pair{AbstractString,Int64}(\"with crystal structures\", 1)\n",
      "Pair{AbstractString,Int64}(\"Li 2 SO\", 1)\n",
      "Pair{AbstractString,Int64}(\"detectable reporter\", 1)\n",
      "Pair{AbstractString,Int64}(\"compound proved inactive\", 1)\n",
      "Pair{AbstractString,Int64}(\"program XDSAPP\", 1)\n",
      "Pair{AbstractString,Int64}(\"towards the\", 2)\n",
      "Pair{AbstractString,Int64}(\") , 15\", 1)\n",
      "Pair{AbstractString,Int64}(\"MERS-CoV (\", 6)\n",
      "Pair{AbstractString,Int64}(\", T 3327\", 1)\n",
      "Pair{AbstractString,Int64}(\"The inhibitory potencies\", 1)\n",
      "Pair{AbstractString,Int64}(\", murine astrocytoma\", 1)\n",
      "Pair{AbstractString,Int64}(\"pro residues have\", 1)\n",
      "Pair{AbstractString,Int64}(\"′ )\", 1)\n",
      "Pair{AbstractString,Int64}(\"of MHV\", 7)\n",
      "Pair{AbstractString,Int64}(\"Cytotoxicity Bio-Assay\", 1)\n",
      "Pair{AbstractString,Int64}(\"-yl ) propanoate\", 1)\n",
      "Pair{AbstractString,Int64}(\"buffer and\", 1)\n",
      "Pair{AbstractString,Int64}(\"ethyl-acetate or exposed\", 1)\n",
      "Pair{AbstractString,Int64}(\"analysis. Normalized\", 1)\n",
      "Pair{AbstractString,Int64}(\"white solid (\", 3)\n",
      "Pair{AbstractString,Int64}(\"delivered 4\", 1)\n",
      "Pair{AbstractString,Int64}(\"Further , we\", 1)\n",
      "Pair{AbstractString,Int64}(\"protein 5 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"193 mutations\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.05\", 10)\n",
      "Pair{AbstractString,Int64}(\"decrease significantly\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Michael\", 1)\n",
      "Pair{AbstractString,Int64}(\"2.5 mM\", 1)\n",
      "Pair{AbstractString,Int64}(\"respectively ( indicated\", 1)\n",
      "Pair{AbstractString,Int64}(\"( space between\", 1)\n",
      "Pair{AbstractString,Int64}(\"small size ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"back-wall\", 6)\n",
      "Pair{AbstractString,Int64}(\"ms ) to\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"single-stranded RNA genome\", 1)\n",
      "Pair{AbstractString,Int64}(\"HATU ) was\", 1)\n",
      "Pair{AbstractString,Int64}(\"RNA.\", 1)\n",
      "Pair{AbstractString,Int64}(\"during learning\", 3)\n",
      "Pair{AbstractString,Int64}(\"because P 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"such as antiviral\", 1)\n",
      "Pair{AbstractString,Int64}(\"stainless-steel rods\", 1)\n",
      "Pair{AbstractString,Int64}(\"be an excellent\", 1)\n",
      "Pair{AbstractString,Int64}(\"04 and RaTG\", 5)\n",
      "Pair{AbstractString,Int64}(\"transduction with\", 1)\n",
      "Pair{AbstractString,Int64}(\"and Hank '\", 2)\n",
      "Pair{AbstractString,Int64}(\"CPEs )\", 1)\n",
      "Pair{AbstractString,Int64}(\"percent\", 2)\n",
      "Pair{AbstractString,Int64}(\"brief , Huh\", 1)\n",
      "Pair{AbstractString,Int64}(\"ToxiLight (\", 1)\n",
      "Pair{AbstractString,Int64}(\". The complex\", 1)\n",
      "Pair{AbstractString,Int64}(\"luminescence\", 1)\n",
      "Pair{AbstractString,Int64}(\"* p= 0.0210\", 1)\n",
      "Pair{AbstractString,Int64}(\"entropy\", 1)\n",
      "Pair{AbstractString,Int64}(\"during ethyl-acetate\", 1)\n",
      "Pair{AbstractString,Int64}(\", 0.3 EGTA\", 1)\n",
      "Pair{AbstractString,Int64}(\"procedure.\", 2)\n",
      "Pair{AbstractString,Int64}(\"realized that\", 1)\n",
      "Pair{AbstractString,Int64}(\"viable progeny\", 1)\n",
      "Pair{AbstractString,Int64}(\"basic odor detection\", 1)\n",
      "Pair{AbstractString,Int64}(\"µM ) when\", 1)\n",
      "Pair{AbstractString,Int64}(\"vector between\", 1)\n",
      "Pair{AbstractString,Int64}(\"selectivity of the\", 1)\n",
      "Pair{AbstractString,Int64}(\"-P 4\", 1)\n",
      "Pair{AbstractString,Int64}(\"the smaller P\", 1)\n",
      "Pair{AbstractString,Int64}(\"ensembles during\", 1)\n",
      "Pair{AbstractString,Int64}(\"adaptive capacity\", 1)\n",
      "Pair{AbstractString,Int64}(\"points ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"fragment. Samples\", 2)\n",
      "Pair{AbstractString,Int64}(\", respectively )\", 1)\n",
      "Pair{AbstractString,Int64}(\"one mutation\", 1)\n",
      "Pair{AbstractString,Int64}(\"are located close\", 1)\n",
      "Pair{AbstractString,Int64}(\"174\", 1)\n",
      "Pair{AbstractString,Int64}(\"fusion assay has\", 1)\n",
      "Pair{AbstractString,Int64}(\"accctgaacaagggctgctccccgatctgtccc ggcccgagcgtcaatcttggagtggcgtccgt gggcacccactgctgccagagcttcctgtgca\", 1)\n",
      "Pair{AbstractString,Int64}(\"compound against the\", 1)\n",
      "Pair{AbstractString,Int64}(\"from damaged\", 1)\n",
      "Pair{AbstractString,Int64}(\"expression of 294\", 1)\n",
      "Pair{AbstractString,Int64}(\"4.92 g\", 1)\n",
      "Pair{AbstractString,Int64}(\"This observation suggests\", 1)\n",
      "Pair{AbstractString,Int64}(\"antibody and a\", 1)\n",
      "Pair{AbstractString,Int64}(\"the primary\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 infection 30\", 1)\n",
      "Pair{AbstractString,Int64}(\"enteroviruses include\", 1)\n",
      "Pair{AbstractString,Int64}(\"same scale\", 1)\n",
      "Pair{AbstractString,Int64}(\"synonymous site (\", 1)\n",
      "Pair{AbstractString,Int64}(\"518\", 1)\n",
      "Pair{AbstractString,Int64}(\"of stress and\", 1)\n",
      "Pair{AbstractString,Int64}(\"the detection of\", 2)\n",
      "Pair{AbstractString,Int64}(\", we achieved\", 1)\n",
      "Pair{AbstractString,Int64}(\"protocol. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"competent for\", 1)\n",
      "Pair{AbstractString,Int64}(\"with nsp 12\", 1)\n",
      "Pair{AbstractString,Int64}(\"and CaCo\", 1)\n",
      "Pair{AbstractString,Int64}(\"by dideoxy sequencing.\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 :\", 2)\n",
      "Pair{AbstractString,Int64}(\"the virus\", 18)\n",
      "Pair{AbstractString,Int64}(\"of the monomeric\", 1)\n",
      "Pair{AbstractString,Int64}(\"respiratory syndrome\", 2)\n",
      "Pair{AbstractString,Int64}(\"system is the\", 1)\n",
      "Pair{AbstractString,Int64}(\"indicator for\", 1)\n",
      "Pair{AbstractString,Int64}(\"significance was\", 3)\n",
      "Pair{AbstractString,Int64}(\"alphacoronaviruses 28\", 1)\n",
      "Pair{AbstractString,Int64}(\"Animals were anaesthetized\", 1)\n",
      "Pair{AbstractString,Int64}(\": P 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"x ) and\", 2)\n",
      "Pair{AbstractString,Int64}(\"England\", 1)\n",
      "Pair{AbstractString,Int64}(\") and SNU\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 N sg\", 1)\n",
      "Pair{AbstractString,Int64}(\"accepts three H-bonds\", 1)\n",
      "Pair{AbstractString,Int64}(\"mice 25 .\", 1)\n",
      "Pair{AbstractString,Int64}(\"see also Fig.\", 1)\n",
      "Pair{AbstractString,Int64}(\"by the percent\", 1)\n",
      "Pair{AbstractString,Int64}(\"vol/vol\", 2)\n",
      "Pair{AbstractString,Int64}(\"hippocampus.\", 1)\n",
      "Pair{AbstractString,Int64}(\"to the same\", 1)\n",
      "Pair{AbstractString,Int64}(\"showed that the\", 3)\n",
      "Pair{AbstractString,Int64}(\"high temperature will\", 1)\n",
      "Pair{AbstractString,Int64}(\"compound proved extremely\", 1)\n",
      "Pair{AbstractString,Int64}(\"preprint at a\", 1)\n",
      "Pair{AbstractString,Int64}(\"fragmented viral\", 1)\n",
      "Pair{AbstractString,Int64}(\"1.3 and\", 1)\n",
      "Pair{AbstractString,Int64}(\"the lowest except\", 1)\n",
      "Pair{AbstractString,Int64}(\"bind to this\", 1)\n",
      "Pair{AbstractString,Int64}(\"217 software (\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 39.1\", 1)\n",
      "Pair{AbstractString,Int64}(\"2020.02.27.969006 doi :\", 8)\n",
      "Pair{AbstractString,Int64}(\"solid ( 88\", 1)\n",
      "Pair{AbstractString,Int64}(\"the DREADD\", 2)\n",
      "Pair{AbstractString,Int64}(\"1.3 µM against\", 1)\n",
      "Pair{AbstractString,Int64}(\"previously been 223\", 1)\n",
      "Pair{AbstractString,Int64}(\"were active\", 4)\n",
      "Pair{AbstractString,Int64}(\"4 Mg-ATP\", 1)\n",
      "Pair{AbstractString,Int64}(\"TFA\", 1)\n",
      "Pair{AbstractString,Int64}(\"the soaked crystals\", 1)\n",
      "Pair{AbstractString,Int64}(\"al.\", 76)\n",
      "Pair{AbstractString,Int64}(\"directional variation of\", 1)\n",
      "Pair{AbstractString,Int64}(\"between group comparison\", 1)\n",
      "Pair{AbstractString,Int64}(\"While we made\", 1)\n",
      "Pair{AbstractString,Int64}(\"( side-walls )\", 1)\n",
      "Pair{AbstractString,Int64}(\"CS+ was\", 5)\n",
      "Pair{AbstractString,Int64}(\"samples ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"98 point\", 1)\n",
      "Pair{AbstractString,Int64}(\"gene B\", 1)\n",
      "Pair{AbstractString,Int64}(\"IFITM 1\", 1)\n",
      "Pair{AbstractString,Int64}(\". Differences in\", 2)\n",
      "Pair{AbstractString,Int64}(\"cages throughout the\", 1)\n",
      "Pair{AbstractString,Int64}(\"cFos-tTA-dependent transcripts and\", 1)\n",
      "Pair{AbstractString,Int64}(\"( HCoV- 229\", 1)\n",
      "Pair{AbstractString,Int64}(\"setting\", 2)\n",
      "Pair{AbstractString,Int64}(\"between 3.6\", 1)\n",
      "Pair{AbstractString,Int64}(\"C in OptiMEM.\", 1)\n",
      "Pair{AbstractString,Int64}(\"experimental evidences that\", 1)\n",
      "Pair{AbstractString,Int64}(\"LHMDS ) in\", 1)\n",
      "Pair{AbstractString,Int64}(\"be superior to\", 1)\n",
      "Pair{AbstractString,Int64}(\"/uPAR proteins\", 1)\n",
      "Pair{AbstractString,Int64}(\"see Experimental\", 6)\n",
      "Pair{AbstractString,Int64}(\"SIGLEC 1 -FRAG\", 1)\n",
      "Pair{AbstractString,Int64}(\"infection of\", 1)\n",
      "Pair{AbstractString,Int64}(\"are represented as\", 1)\n",
      "Pair{AbstractString,Int64}(\"significant improvement\", 1)\n",
      "Pair{AbstractString,Int64}(\"custom-written MatLab programs\", 1)\n",
      "Pair{AbstractString,Int64}(\"x p-s/RPMI\", 2)\n",
      "Pair{AbstractString,Int64}(\"prepared with R\", 4)\n",
      "Pair{AbstractString,Int64}(\". 52\", 1)\n",
      "Pair{AbstractString,Int64}(\"beta-CoV )\", 1)\n",
      "Pair{AbstractString,Int64}(\"in the C-terminal\", 1)\n",
      "Pair{AbstractString,Int64}(\"is truly three-dimensional\", 1)\n",
      "Pair{AbstractString,Int64}(\"µm ) with\", 1)\n",
      "Pair{AbstractString,Int64}(\"168\", 1)\n",
      "Pair{AbstractString,Int64}(\"SCALA\", 1)\n",
      "Pair{AbstractString,Int64}(\"2.20 - 2.14\", 1)\n",
      "Pair{AbstractString,Int64}(\", according\", 2)\n",
      "Pair{AbstractString,Int64}(\"negative rate of\", 2)\n",
      "Pair{AbstractString,Int64}(\"spectrum protease\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 C-like protease.\", 1)\n",
      "Pair{AbstractString,Int64}(\"transfected with\", 2)\n",
      "Pair{AbstractString,Int64}(\"analyses were performed\", 2)\n",
      "Pair{AbstractString,Int64}(\"in P 1\", 2)\n",
      "Pair{AbstractString,Int64}(\"nucleotide substitutions (\", 1)\n",
      "Pair{AbstractString,Int64}(\", presentation\", 1)\n",
      "Pair{AbstractString,Int64}(\"is higher\", 1)\n",
      "Pair{AbstractString,Int64}(\"All mice were\", 2)\n",
      "Pair{AbstractString,Int64}(\"a measurable behavioral\", 1)\n",
      "Pair{AbstractString,Int64}(\"statistical differences between\", 1)\n",
      "Pair{AbstractString,Int64}(\"compound for\", 1)\n",
      "Pair{AbstractString,Int64}(\"is truly\", 1)\n",
      "Pair{AbstractString,Int64}(\"Na 2 SO\", 4)\n",
      "Pair{AbstractString,Int64}(\"affinity of\", 1)\n",
      "Pair{AbstractString,Int64}(\"inactive Michaelacceptor compound\", 1)\n",
      "Pair{AbstractString,Int64}(\"mechanism for viral\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 b )\", 1)\n",
      "Pair{AbstractString,Int64}(\"hours after 428\", 1)\n",
      "Pair{AbstractString,Int64}(\"synthesized are\", 1)\n",
      "Pair{AbstractString,Int64}(\"hand , the\", 1)\n",
      "Pair{AbstractString,Int64}(\"bioRxiv preprint 7\", 1)\n",
      "Pair{AbstractString,Int64}(\"for lentivirus production\", 1)\n",
      "Pair{AbstractString,Int64}(\"nsp 2 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"delineated\", 1)\n",
      "Pair{AbstractString,Int64}(\"the expression fold\", 1)\n",
      "Pair{AbstractString,Int64}(\"in an attempt\", 1)\n",
      "Pair{AbstractString,Int64}(\"It\", 10)\n",
      "Pair{AbstractString,Int64}(\"C. Meanwhile\", 1)\n",
      "Pair{AbstractString,Int64}(\"compound exhibits\", 1)\n",
      "Pair{AbstractString,Int64}(\"2019 -nCoV ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"SARS-like coronavirus WIV\", 1)\n",
      "Pair{AbstractString,Int64}(\"increasing concentrations\", 1)\n",
      "Pair{AbstractString,Int64}(\"approximately when\", 1)\n",
      "Pair{AbstractString,Int64}(\"majority\", 2)\n",
      "Pair{AbstractString,Int64}(\"speed reported\", 1)\n",
      "Pair{AbstractString,Int64}(\"cyano\", 1)\n",
      "Pair{AbstractString,Int64}(\"with ketamine/xylazine\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells infected\", 5)\n",
      "Pair{AbstractString,Int64}(\"role for Ly\", 1)\n",
      "Pair{AbstractString,Int64}(\"as the mean\", 1)\n",
      "Pair{AbstractString,Int64}(\"work\", 4)\n",
      "Pair{AbstractString,Int64}(\"see Table\", 1)\n",
      "Pair{AbstractString,Int64}(\"for virus research\", 1)\n",
      "Pair{AbstractString,Int64}(\"in 373 ratio\", 1)\n",
      "Pair{AbstractString,Int64}(\"DMSO (\", 2)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"odor stimulus (\", 1)\n",
      "Pair{AbstractString,Int64}(\", pH range\", 1)\n",
      "Pair{AbstractString,Int64}(\"ethyl-acetate. ( Bottom\", 1)\n",
      "Pair{AbstractString,Int64}(\"selected with 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"treatment with 224\", 1)\n",
      "Pair{AbstractString,Int64}(\"11 ) ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"warheads\", 3)\n",
      "Pair{AbstractString,Int64}(\"that were activated\", 4)\n",
      "Pair{AbstractString,Int64}(\"t= 24 h\", 1)\n",
      "Pair{AbstractString,Int64}(\"was presented. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"infection by inactivating\", 1)\n",
      "Pair{AbstractString,Int64}(\"the universal\", 1)\n",
      "Pair{AbstractString,Int64}(\"a restriction factor\", 1)\n",
      "Pair{AbstractString,Int64}(\"the opposite side\", 2)\n",
      "Pair{AbstractString,Int64}(\"by Designer Drugs\", 1)\n",
      "Pair{AbstractString,Int64}(\"all of them\", 1)\n",
      "Pair{AbstractString,Int64}(\"antiviral drugs and\", 1)\n",
      "Pair{AbstractString,Int64}(\"of S protein\", 2)\n",
      "Pair{AbstractString,Int64}(\"by SARS-CoV and\", 1)\n",
      "Pair{AbstractString,Int64}(\"trigger memory recall.\", 1)\n",
      "Pair{AbstractString,Int64}(\", P<\", 1)\n",
      "Pair{AbstractString,Int64}(\"crystallization trials\", 1)\n",
      "Pair{AbstractString,Int64}(\"previously ( 9\", 1)\n",
      "Pair{AbstractString,Int64}(\". After 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"ISG\", 1)\n",
      "Pair{AbstractString,Int64}(\", we relaxed\", 1)\n",
      "Pair{AbstractString,Int64}(\", including one\", 1)\n",
      "Pair{AbstractString,Int64}(\"foot shock\", 11)\n",
      "Pair{AbstractString,Int64}(\"STAT 1 −/−\", 1)\n",
      "Pair{AbstractString,Int64}(\"s multiple comparisons\", 1)\n",
      "Pair{AbstractString,Int64}(\"study , KLKs\", 1)\n",
      "Pair{AbstractString,Int64}(\"light\", 1)\n",
      "Pair{AbstractString,Int64}(\"the patched\", 1)\n",
      "Pair{AbstractString,Int64}(\"Vero cells )\", 2)\n",
      "Pair{AbstractString,Int64}(\") supplemented with\", 2)\n",
      "Pair{AbstractString,Int64}(\"pro . Because\", 1)\n",
      "Pair{AbstractString,Int64}(\"Production System\", 1)\n",
      "Pair{AbstractString,Int64}(\"Pharmacological inhibition\", 1)\n",
      "Pair{AbstractString,Int64}(\", Expression\", 1)\n",
      "Pair{AbstractString,Int64}(\"was scored using\", 1)\n",
      "Pair{AbstractString,Int64}(\"barrier and cause\", 1)\n",
      "Pair{AbstractString,Int64}(\"either (\", 1)\n",
      "Pair{AbstractString,Int64}(\"Database was\", 1)\n",
      "Pair{AbstractString,Int64}(\"the order\", 1)\n",
      "Pair{AbstractString,Int64}(\"clarity. B.\", 1)\n",
      "Pair{AbstractString,Int64}(\"insert\", 2)\n",
      "Pair{AbstractString,Int64}(\"in immunocompetent\", 1)\n",
      "Pair{AbstractString,Int64}(\"in 205\", 1)\n",
      "Pair{AbstractString,Int64}(\"genotype.\", 1)\n",
      "Pair{AbstractString,Int64}(\"-Novel Coronavirus (\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 e-f\", 1)\n",
      "Pair{AbstractString,Int64}(\"one\", 37)\n",
      "Pair{AbstractString,Int64}(\", which can\", 1)\n",
      "Pair{AbstractString,Int64}(\", Vero E\", 1)\n",
      "Pair{AbstractString,Int64}(\"at both\", 2)\n",
      "Pair{AbstractString,Int64}(\"replicons , and\", 2)\n",
      "Pair{AbstractString,Int64}(\"years after\", 1)\n",
      "Pair{AbstractString,Int64}(\"reversion based on\", 1)\n",
      "Pair{AbstractString,Int64}(\"These parameters\", 1)\n",
      "Pair{AbstractString,Int64}(\"E , pCAGGS-MERS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"187 ( the\", 1)\n",
      "Pair{AbstractString,Int64}(\") based\", 1)\n",
      "Pair{AbstractString,Int64}(\"spike amino\", 1)\n",
      "Pair{AbstractString,Int64}(\"gene from\", 1)\n",
      "Pair{AbstractString,Int64}(\"the large\", 3)\n",
      "Pair{AbstractString,Int64}(\"is dispensible\", 1)\n",
      "Pair{AbstractString,Int64}(\"one yet-unknown\", 1)\n",
      "Pair{AbstractString,Int64}(\"upon request\", 1)\n",
      "Pair{AbstractString,Int64}(\"Cas 9 as\", 1)\n",
      "Pair{AbstractString,Int64}(\"esters\", 2)\n",
      "Pair{AbstractString,Int64}(\"59 60\", 4)\n",
      "Pair{AbstractString,Int64}(\"LZ et\", 1)\n",
      "Pair{AbstractString,Int64}(\"11 was\", 1)\n",
      "Pair{AbstractString,Int64}(\"gBlock was\", 1)\n",
      "Pair{AbstractString,Int64}(\"the template structures\", 1)\n",
      "Pair{AbstractString,Int64}(\"13 204 cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"interference with\", 1)\n",
      "Pair{AbstractString,Int64}(\"all the enzymatic\", 1)\n",
      "Pair{AbstractString,Int64}(\"81,000 , and\", 1)\n",
      "Pair{AbstractString,Int64}(\"Fos-tagged ensembles during\", 1)\n",
      "Pair{AbstractString,Int64}(\"could 297 be\", 1)\n",
      "Pair{AbstractString,Int64}(\"( - 80\", 1)\n",
      "Pair{AbstractString,Int64}(\"and relative\", 1)\n",
      "Pair{AbstractString,Int64}(\"cleaved by\", 1)\n",
      "Pair{AbstractString,Int64}(\"PCR using\", 1)\n",
      "Pair{AbstractString,Int64}(\"and inserted into\", 1)\n",
      "Pair{AbstractString,Int64}(\"a complex process\", 1)\n",
      "Pair{AbstractString,Int64}(\"Lambrecht ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"227 network\", 1)\n",
      "Pair{AbstractString,Int64}(\", and neurons\", 1)\n",
      "Pair{AbstractString,Int64}(\"pan CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"that of ciclesonide.\", 1)\n",
      "Pair{AbstractString,Int64}(\"tube containing 200\", 1)\n",
      "Pair{AbstractString,Int64}(\"fluorinated\", 1)\n",
      "Pair{AbstractString,Int64}(\") . C\", 1)\n",
      "Pair{AbstractString,Int64}(\"bioRxiv preprint Then\", 1)\n",
      "Pair{AbstractString,Int64}(\"PEG 4,000 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"was delineated\", 1)\n",
      "Pair{AbstractString,Int64}(\"cooled with\", 1)\n",
      "Pair{AbstractString,Int64}(\"the detectable viral\", 2)\n",
      "Pair{AbstractString,Int64}(\"also suppressed replication\", 1)\n",
      "Pair{AbstractString,Int64}(\"silenced. Their median\", 1)\n",
      "Pair{AbstractString,Int64}(\"a site (\", 1)\n",
      "Pair{AbstractString,Int64}(\"E blocks fusion\", 1)\n",
      "Pair{AbstractString,Int64}(\"that the posterior\", 1)\n",
      "Pair{AbstractString,Int64}(\"RaTG 13 genome\", 1)\n",
      "Pair{AbstractString,Int64}(\"regulators of neural\", 1)\n",
      "Pair{AbstractString,Int64}(\"inflammation suggests\", 1)\n",
      "Pair{AbstractString,Int64}(\"recording solution ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"of exonucleases ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"rate of polyprotein\", 1)\n",
      "Pair{AbstractString,Int64}(\"chosen\", 2)\n",
      "Pair{AbstractString,Int64}(\"dilution factor\", 1)\n",
      "Pair{AbstractString,Int64}(\"344 variation.\", 1)\n",
      "Pair{AbstractString,Int64}(\"memory trace reactivation\", 1)\n",
      "Pair{AbstractString,Int64}(\"days before surgery.\", 1)\n",
      "Pair{AbstractString,Int64}(\") harbouring T\", 1)\n",
      "Pair{AbstractString,Int64}(\"; ExoN-AD\", 1)\n",
      "Pair{AbstractString,Int64}(\"the compound against\", 2)\n",
      "Pair{AbstractString,Int64}(\"13 hampered\", 1)\n",
      "Pair{AbstractString,Int64}(\"reduced the detectable\", 1)\n",
      "Pair{AbstractString,Int64}(\"samples tested and\", 1)\n",
      "Pair{AbstractString,Int64}(\"µL ) was\", 2)\n",
      "Pair{AbstractString,Int64}(\"enables the\", 1)\n",
      "Pair{AbstractString,Int64}(\"well as in\", 1)\n",
      "Pair{AbstractString,Int64}(\"protease \\\" (\", 1)\n",
      "Pair{AbstractString,Int64}(\"modulates proteolytic activation\", 1)\n",
      "Pair{AbstractString,Int64}(\"14 infection\", 1)\n",
      "Pair{AbstractString,Int64}(\", but is\", 2)\n",
      "Pair{AbstractString,Int64}(\"3327 C )\", 1)\n",
      "Pair{AbstractString,Int64}(\"to total\", 1)\n",
      "Pair{AbstractString,Int64}(\"effect that was\", 1)\n",
      "Pair{AbstractString,Int64}(\"of MHV ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"acid substitution at\", 1)\n",
      "Pair{AbstractString,Int64}(\"and the amide\", 2)\n",
      "Pair{AbstractString,Int64}(\"much larger and\", 1)\n",
      "Pair{AbstractString,Int64}(\"in RNAlater\", 1)\n",
      "Pair{AbstractString,Int64}(\"nt mutants\", 3)\n",
      "Pair{AbstractString,Int64}(\"pVSV * plasmid\", 1)\n",
      "Pair{AbstractString,Int64}(\"of regulators\", 1)\n",
      "Pair{AbstractString,Int64}(\"beta-citronellol , CS+\", 1)\n",
      "Pair{AbstractString,Int64}(\"pro , 1.7\", 1)\n",
      "Pair{AbstractString,Int64}(\"linear for all\", 1)\n",
      "Pair{AbstractString,Int64}(\"8.1.21 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"bisulfite adduct ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"hypothesis ,\", 5)\n",
      "Pair{AbstractString,Int64}(\"the spike furin\", 1)\n",
      "Pair{AbstractString,Int64}(\"addition 174 to\", 1)\n",
      "Pair{AbstractString,Int64}(\"be kept\", 1)\n",
      "Pair{AbstractString,Int64}(\". MHV-ExoN (\", 1)\n",
      "Pair{AbstractString,Int64}(\"189 by\", 1)\n",
      "Pair{AbstractString,Int64}(\"the recombinant proteases\", 2)\n",
      "Pair{AbstractString,Int64}(\"174 to RNA\", 1)\n",
      "Pair{AbstractString,Int64}(\"90 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"with S-HKU 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"epitope to\", 1)\n",
      "Pair{AbstractString,Int64}(\"Ile\", 1)\n",
      "Pair{AbstractString,Int64}(\"the samples were\", 4)\n",
      "Pair{AbstractString,Int64}(\"bias of ExoN-AE\", 1)\n",
      "Pair{AbstractString,Int64}(\"The percentage\", 1)\n",
      "Pair{AbstractString,Int64}(\"231 increased\", 1)\n",
      "Pair{AbstractString,Int64}(\"ratio in the\", 1)\n",
      "Pair{AbstractString,Int64}(\"blind\", 1)\n",
      "Pair{AbstractString,Int64}(\". Previous exposure\", 1)\n",
      "Pair{AbstractString,Int64}(\"of the surprisingly\", 1)\n",
      "Pair{AbstractString,Int64}(\"behavior have been\", 1)\n",
      "Pair{AbstractString,Int64}(\"Exclusively Activated by\", 1)\n",
      "Pair{AbstractString,Int64}(\". After bath\", 1)\n",
      "Pair{AbstractString,Int64}(\"NaH 2 PO\", 1)\n",
      "Pair{AbstractString,Int64}(\"restricted size\", 2)\n",
      "Pair{AbstractString,Int64}(\"function of time\", 1)\n",
      "Pair{AbstractString,Int64}(\"these replicons has\", 1)\n",
      "Pair{AbstractString,Int64}(\"network , which\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 6 FV\", 2)\n",
      "Pair{AbstractString,Int64}(\"preprint concentrations\", 1)\n",
      "Pair{AbstractString,Int64}(\". The RSCU\", 2)\n",
      "Pair{AbstractString,Int64}(\"tagging of active\", 1)\n",
      "Pair{AbstractString,Int64}(\"container (\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"inactive against\", 4)\n",
      "Pair{AbstractString,Int64}(\"Seventeen years\", 1)\n",
      "Pair{AbstractString,Int64}(\"engineered and\", 1)\n",
      "Pair{AbstractString,Int64}(\"RNA standard derived\", 2)\n",
      "Pair{AbstractString,Int64}(\"Rockland )\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 -yl\", 1)\n",
      "Pair{AbstractString,Int64}(\"PFU )\", 3)\n",
      "Pair{AbstractString,Int64}(\"12 , and\", 1)\n",
      "Pair{AbstractString,Int64}(\"of viral disease.\", 1)\n",
      "Pair{AbstractString,Int64}(\"HA-tagged LY\", 1)\n",
      "Pair{AbstractString,Int64}(\"% PFA/PBS.\", 1)\n",
      "Pair{AbstractString,Int64}(\"44 45 46\", 4)\n",
      "Pair{AbstractString,Int64}(\"session. Ethylacetate\", 1)\n",
      "Pair{AbstractString,Int64}(\"compound 1 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"substitution rates (\", 1)\n",
      "Pair{AbstractString,Int64}(\"that the cyclopropylmethyl\", 1)\n",
      "Pair{AbstractString,Int64}(\"the two HRV\", 1)\n",
      "Pair{AbstractString,Int64}(\"cyclopentylmethyl ( compound\", 1)\n",
      "Pair{AbstractString,Int64}(\"-membered ring (\", 1)\n",
      "Pair{AbstractString,Int64}(\"suggest that if\", 1)\n",
      "Pair{AbstractString,Int64}(\"formed by residue\", 1)\n",
      "Pair{AbstractString,Int64}(\"' , 6\", 1)\n",
      "Pair{AbstractString,Int64}(\"high throughput\", 1)\n",
      "Pair{AbstractString,Int64}(\"the inflammatory\", 1)\n",
      "Pair{AbstractString,Int64}(\"analog. Incorporation of\", 1)\n",
      "Pair{AbstractString,Int64}(\"specificity of the\", 3)\n",
      "Pair{AbstractString,Int64}(\"for 2 h\", 4)\n",
      "Pair{AbstractString,Int64}(\"-b ]\", 1)\n",
      "Pair{AbstractString,Int64}(\"C-terminal N\", 1)\n",
      "Pair{AbstractString,Int64}(\"on humans\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibitor accepts a\", 1)\n",
      "Pair{AbstractString,Int64}(\"female\", 9)\n",
      "Pair{AbstractString,Int64}(\"added. The reaction\", 1)\n",
      "Pair{AbstractString,Int64}(\"during the recent\", 1)\n",
      "Pair{AbstractString,Int64}(\"olfactometer (\", 1)\n",
      "Pair{AbstractString,Int64}(\"red ) or\", 1)\n",
      "Pair{AbstractString,Int64}(\"Regardless\", 1)\n",
      "Pair{AbstractString,Int64}(\"retrieve the\", 1)\n",
      "Pair{AbstractString,Int64}(\"data demonstrate that\", 1)\n",
      "Pair{AbstractString,Int64}(\"DNA loading\", 1)\n",
      "Pair{AbstractString,Int64}(\"extremity where\", 1)\n",
      "Pair{AbstractString,Int64}(\"p= 0.6191 ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"changed significantly.\", 1)\n",
      "Pair{AbstractString,Int64}(\"20 passages\", 1)\n",
      "Pair{AbstractString,Int64}(\"began\", 1)\n",
      "Pair{AbstractString,Int64}(\"-controlled\", 1)\n",
      "Pair{AbstractString,Int64}(\"quantifying propagated\", 1)\n",
      "Pair{AbstractString,Int64}(\"was described in\", 1)\n",
      "Pair{AbstractString,Int64}(\"ThermoFisher ) according\", 1)\n",
      "Pair{AbstractString,Int64}(\"from two\", 1)\n",
      "Pair{AbstractString,Int64}(\"PBS containing calcium\", 2)\n",
      "Pair{AbstractString,Int64}(\"available on request.\", 1)\n",
      "Pair{AbstractString,Int64}(\"Rz ) ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"dimethylamino ) methylene\", 1)\n",
      "Pair{AbstractString,Int64}(\"that partially compensate\", 1)\n",
      "Pair{AbstractString,Int64}(\"technical\", 2)\n",
      "Pair{AbstractString,Int64}(\") phenotype\", 1)\n",
      "Pair{AbstractString,Int64}(\"was isolated ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"the protein\", 3)\n",
      "Pair{AbstractString,Int64}(\"inactive against HCoV-NL\", 1)\n",
      "Pair{AbstractString,Int64}(\"MERS-CoV and SARS-CoV\", 3)\n",
      "Pair{AbstractString,Int64}(\"are only\", 2)\n",
      "Pair{AbstractString,Int64}(\"Cells and viruses.\", 1)\n",
      "Pair{AbstractString,Int64}(\"from viral\", 1)\n",
      "Pair{AbstractString,Int64}(\"Haberly , 1990\", 1)\n",
      "Pair{AbstractString,Int64}(\"R configuration\", 2)\n",
      "Pair{AbstractString,Int64}(\"dropwise a solution\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 was\", 2)\n",
      "Pair{AbstractString,Int64}(\", Matsuyama et\", 1)\n",
      "Pair{AbstractString,Int64}(\"generated at\", 1)\n",
      "Pair{AbstractString,Int64}(\"preprint the\", 2)\n",
      "Pair{AbstractString,Int64}(\", host proteases\", 1)\n",
      "Pair{AbstractString,Int64}(\"% in\", 5)\n",
      "Pair{AbstractString,Int64}(\"transfection\", 3)\n",
      "Pair{AbstractString,Int64}(\") synclinal\", 1)\n",
      "Pair{AbstractString,Int64}(\", 2008\", 5)\n",
      "Pair{AbstractString,Int64}(\"emerged in\", 1)\n",
      "Pair{AbstractString,Int64}(\"appeared so\", 1)\n",
      "Pair{AbstractString,Int64}(\"Cytotoxic\", 1)\n",
      "Pair{AbstractString,Int64}(\"tagged in\", 1)\n",
      "Pair{AbstractString,Int64}(\"self-activation\", 2)\n",
      "Pair{AbstractString,Int64}(\"and transmission\", 1)\n",
      "Pair{AbstractString,Int64}(\"( PBS\", 2)\n",
      "Pair{AbstractString,Int64}(\"20 ° C\", 1)\n",
      "Pair{AbstractString,Int64}(\"were both necessary\", 1)\n",
      "Pair{AbstractString,Int64}(\"( blue )\", 2)\n",
      "Pair{AbstractString,Int64}(\": 50\", 1)\n",
      "Pair{AbstractString,Int64}(\"failure 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"olfactory bulb ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"RT-qPCR ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"KLK mechanism of\", 1)\n",
      "Pair{AbstractString,Int64}(\"- 2.1 µM\", 1)\n",
      "Pair{AbstractString,Int64}(\"or 2 nt\", 1)\n",
      "Pair{AbstractString,Int64}(\"with susceptible\", 2)\n",
      "Pair{AbstractString,Int64}(\"determined via the\", 1)\n",
      "Pair{AbstractString,Int64}(\"cFos-tTA transgenic\", 12)\n",
      "Pair{AbstractString,Int64}(\"solid (\", 4)\n",
      "Pair{AbstractString,Int64}(\"Previous studies\", 2)\n",
      "Pair{AbstractString,Int64}(\"control. All\", 1)\n",
      "Pair{AbstractString,Int64}(\"to ethyl-acetate (\", 1)\n",
      "Pair{AbstractString,Int64}(\"cyclopropylmethyl moiety\", 1)\n",
      "Pair{AbstractString,Int64}(\"and synonymous mutations\", 1)\n",
      "Pair{AbstractString,Int64}(\"6 a-b\", 1)\n",
      "Pair{AbstractString,Int64}(\"mL - 1\", 6)\n",
      "Pair{AbstractString,Int64}(\"s recommendations. LY\", 1)\n",
      "Pair{AbstractString,Int64}(\"common\", 6)\n",
      "Pair{AbstractString,Int64}(\"greatly improved albeit\", 1)\n",
      "Pair{AbstractString,Int64}(\"plasmid 55 between\", 1)\n",
      "Pair{AbstractString,Int64}(\". Collectively\", 2)\n",
      "Pair{AbstractString,Int64}(\"Ile 165 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"Fourth Edition\", 1)\n",
      "Pair{AbstractString,Int64}(\"the antiviral\", 6)\n",
      "Pair{AbstractString,Int64}(\"aimed\", 4)\n",
      "Pair{AbstractString,Int64}(\"up crystallization using\", 1)\n",
      "Pair{AbstractString,Int64}(\"in the vast\", 1)\n",
      "Pair{AbstractString,Int64}(\"indicating that the\", 1)\n",
      "Pair{AbstractString,Int64}(\"but exhibit\", 1)\n",
      "Pair{AbstractString,Int64}(\"103\", 1)\n",
      "Pair{AbstractString,Int64}(\"Tor\", 2)\n",
      "Pair{AbstractString,Int64}(\"is linked to\", 1)\n",
      "Pair{AbstractString,Int64}(\"infection of Vero\", 1)\n",
      "Pair{AbstractString,Int64}(\"pre-treated with the\", 1)\n",
      "Pair{AbstractString,Int64}(\"whereas the slight\", 1)\n",
      "Pair{AbstractString,Int64}(\"isobutyl side-chain\", 1)\n",
      "Pair{AbstractString,Int64}(\"to a seafood\", 1)\n",
      "Pair{AbstractString,Int64}(\", HKU-S or\", 1)\n",
      "Pair{AbstractString,Int64}(\"Villigen ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"made available under\", 5)\n",
      "Pair{AbstractString,Int64}(\"matching\", 1)\n",
      "Pair{AbstractString,Int64}(\"protein antibody\", 1)\n",
      "Pair{AbstractString,Int64}(\"replicons confirmed this\", 1)\n",
      "Pair{AbstractString,Int64}(\"incorporates into growing\", 1)\n",
      "Pair{AbstractString,Int64}(\"groups ( 0.68\", 1)\n",
      "Pair{AbstractString,Int64}(\"Compound 8 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"( n= 12\", 1)\n",
      "Pair{AbstractString,Int64}(\"= benzyl and\", 1)\n",
      "Pair{AbstractString,Int64}(\"6 RNA copies\", 1)\n",
      "Pair{AbstractString,Int64}(\"torsion angle\", 1)\n",
      "Pair{AbstractString,Int64}(\"49 and its\", 1)\n",
      "Pair{AbstractString,Int64}(\"both necessary and\", 1)\n",
      "Pair{AbstractString,Int64}(\"W. Schoggins ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"positive strand RNA\", 1)\n",
      "Pair{AbstractString,Int64}(\"opportunities for interaction\", 1)\n",
      "Pair{AbstractString,Int64}(\"give the a-ketoamides\", 1)\n",
      "Pair{AbstractString,Int64}(\"in the ExoN-AA\", 1)\n",
      "Pair{AbstractString,Int64}(\"extend\", 1)\n",
      "Pair{AbstractString,Int64}(\"Titers\", 2)\n",
      "Pair{AbstractString,Int64}(\"corresponding halfside\", 1)\n",
      "Pair{AbstractString,Int64}(\"genomes.\", 1)\n",
      "Pair{AbstractString,Int64}(\"that mean\", 1)\n",
      "Pair{AbstractString,Int64}(\"site due\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.30 ( 3.48\", 1)\n",
      "Pair{AbstractString,Int64}(\"buffer formalin\", 1)\n",
      "Pair{AbstractString,Int64}(\"Technologies ,\", 3)\n",
      "Pair{AbstractString,Int64}(\"from the position\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 One of\", 1)\n",
      "Pair{AbstractString,Int64}(\"requires mutations\", 1)\n",
      "Pair{AbstractString,Int64}(\"been noted before\", 1)\n",
      "Pair{AbstractString,Int64}(\"liquor\", 2)\n",
      "Pair{AbstractString,Int64}(\", with a\", 5)\n",
      "Pair{AbstractString,Int64}(\"so it is\", 1)\n",
      "Pair{AbstractString,Int64}(\"These results showed\", 1)\n",
      "Pair{AbstractString,Int64}(\"most suitable\", 1)\n",
      "Pair{AbstractString,Int64}(\", individual\", 1)\n",
      "Pair{AbstractString,Int64}(\"fluorescence microscopy that\", 1)\n",
      "Pair{AbstractString,Int64}(\"maximum-likelihood method and\", 1)\n",
      "Pair{AbstractString,Int64}(\"indicate memory recall\", 1)\n",
      "Pair{AbstractString,Int64}(\"side , for\", 2)\n",
      "Pair{AbstractString,Int64}(\"2 Cl\", 15)\n",
      "Pair{AbstractString,Int64}(\"as the concentration\", 1)\n",
      "Pair{AbstractString,Int64}(\"isolated , and\", 1)\n",
      "Pair{AbstractString,Int64}(\"TFA ) was\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"3 C proteases\", 1)\n",
      "Pair{AbstractString,Int64}(\"acids were\", 3)\n",
      "Pair{AbstractString,Int64}(\"three classes\", 1)\n",
      "Pair{AbstractString,Int64}(\"after/before\", 3)\n",
      "Pair{AbstractString,Int64}(\"hours after habituation\", 1)\n",
      "Pair{AbstractString,Int64}(\"Candidate clones were\", 1)\n",
      "Pair{AbstractString,Int64}(\"chymotrypsin-like fold\", 1)\n",
      "Pair{AbstractString,Int64}(\"2132 amplifier boards\", 1)\n",
      "Pair{AbstractString,Int64}(\") in methanol\", 2)\n",
      "Pair{AbstractString,Int64}(\"contrast ,\", 5)\n",
      "Pair{AbstractString,Int64}(\"Beta- 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"firefly\", 1)\n",
      "Pair{AbstractString,Int64}(\"more rapidly\", 1)\n",
      "Pair{AbstractString,Int64}(\"housekeeping gene B\", 1)\n",
      "Pair{AbstractString,Int64}(\", or competitive\", 1)\n",
      "Pair{AbstractString,Int64}(\"and \\\" large\", 1)\n",
      "Pair{AbstractString,Int64}(\"spleens of infected\", 1)\n",
      "Pair{AbstractString,Int64}(\"during the\", 5)\n",
      "Pair{AbstractString,Int64}(\"fragments of\", 1)\n",
      "Pair{AbstractString,Int64}(\"the publicly\", 1)\n",
      "Pair{AbstractString,Int64}(\", replication ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"preprint. HCoV-HKU\", 1)\n",
      "Pair{AbstractString,Int64}(\"by cleavage-resistant MERS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"represented\", 1)\n",
      "Pair{AbstractString,Int64}(\", Vamzye\", 1)\n",
      "Pair{AbstractString,Int64}(\"278741 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"embedded\", 3)\n",
      "Pair{AbstractString,Int64}(\"especially the\", 1)\n",
      "Pair{AbstractString,Int64}(\"correction for\", 1)\n",
      "Pair{AbstractString,Int64}(\"\\\" CGGCGGGCACGT\", 1)\n",
      "Pair{AbstractString,Int64}(\"infection. We\", 1)\n",
      "Pair{AbstractString,Int64}(\": crystallographic data\", 1)\n",
      "Pair{AbstractString,Int64}(\"genus Alphacoronavirus\", 1)\n",
      "Pair{AbstractString,Int64}(\", 49 Using\", 1)\n",
      "Pair{AbstractString,Int64}(\"spill over into\", 1)\n",
      "Pair{AbstractString,Int64}(\"a certain direction\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 HEPES ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"f , ns\", 3)\n",
      "Pair{AbstractString,Int64}(\"et al. ,\", 70)\n",
      "Pair{AbstractString,Int64}(\"229\", 44)\n",
      "Pair{AbstractString,Int64}(\"because this\", 1)\n",
      "Pair{AbstractString,Int64}(\"229 We tested\", 1)\n",
      "Pair{AbstractString,Int64}(\"pronounced\", 2)\n",
      "Pair{AbstractString,Int64}(\"Thermo Fisher\", 6)\n",
      "Pair{AbstractString,Int64}(\"following odors (\", 1)\n",
      "Pair{AbstractString,Int64}(\"splenocytes\", 2)\n",
      "Pair{AbstractString,Int64}(\"4 was\", 1)\n",
      "Pair{AbstractString,Int64}(\"has some limitations.\", 1)\n",
      "Pair{AbstractString,Int64}(\"was invariably isobutyl\", 1)\n",
      "Pair{AbstractString,Int64}(\"PFU/ml )\", 1)\n",
      "Pair{AbstractString,Int64}(\"and amino\", 1)\n",
      "Pair{AbstractString,Int64}(\". This dimerization\", 1)\n",
      "Pair{AbstractString,Int64}(\"198 HKU 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"liver ( d\", 1)\n",
      "Pair{AbstractString,Int64}(\"metal-coordinating active site\", 1)\n",
      "Pair{AbstractString,Int64}(\"from the oxyanion\", 2)\n",
      "Pair{AbstractString,Int64}(\"collected at 100\", 2)\n",
      "Pair{AbstractString,Int64}(\", 4.7\", 1)\n",
      "Pair{AbstractString,Int64}(\"agarose gel\", 2)\n",
      "Pair{AbstractString,Int64}(\"F. In spite\", 1)\n",
      "Pair{AbstractString,Int64}(\"termed S\", 1)\n",
      "Pair{AbstractString,Int64}(\"infected with HCoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"domains\", 3)\n",
      "Pair{AbstractString,Int64}(\"in SARS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"all splenic\", 1)\n",
      "Pair{AbstractString,Int64}(\"49 Using an\", 1)\n",
      "Pair{AbstractString,Int64}(\"37 Because\", 1)\n",
      "Pair{AbstractString,Int64}(\", cellular RNA\", 2)\n",
      "Pair{AbstractString,Int64}(\"Macherey Nagel\", 3)\n",
      "Pair{AbstractString,Int64}(\"paper.\", 1)\n",
      "Pair{AbstractString,Int64}(\"cytotoxicity were assessed.\", 1)\n",
      "Pair{AbstractString,Int64}(\"enough for using\", 1)\n",
      "Pair{AbstractString,Int64}(\"14 were concentrated\", 1)\n",
      "Pair{AbstractString,Int64}(\"Chimera ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"mode by\", 1)\n",
      "Pair{AbstractString,Int64}(\"Scheme 1 )\", 2)\n",
      "Pair{AbstractString,Int64}(\"MERS-CoV S cleavage\", 2)\n",
      "Pair{AbstractString,Int64}(\"pro is the\", 1)\n",
      "Pair{AbstractString,Int64}(\"All CoVs\", 1)\n",
      "Pair{AbstractString,Int64}(\"and 5.2\", 1)\n",
      "Pair{AbstractString,Int64}(\"TM ( Sigma-Aldrich\", 1)\n",
      "Pair{AbstractString,Int64}(\"boceprivir\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 position ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"Bruker\", 1)\n",
      "Pair{AbstractString,Int64}(\"piriform neural\", 3)\n",
      "Pair{AbstractString,Int64}(\"hold for\", 1)\n",
      "Pair{AbstractString,Int64}(\"potent suppression\", 1)\n",
      "Pair{AbstractString,Int64}(\"quantitative behavioral\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.5230\", 1)\n",
      "Pair{AbstractString,Int64}(\"and 5 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"11 r for\", 1)\n",
      "Pair{AbstractString,Int64}(\"sensitivity to\", 2)\n",
      "Pair{AbstractString,Int64}(\"Rega 3 medium\", 1)\n",
      "Pair{AbstractString,Int64}(\"a CysHis dyad.\", 1)\n",
      "Pair{AbstractString,Int64}(\"1000\", 2)\n",
      "Pair{AbstractString,Int64}(\"cells and CRISPR-resistant\", 1)\n",
      "Pair{AbstractString,Int64}(\"WIV 04 versus\", 2)\n",
      "Pair{AbstractString,Int64}(\"continuously\", 1)\n",
      "Pair{AbstractString,Int64}(\"to WT-ExoN-DE 130\", 1)\n",
      "Pair{AbstractString,Int64}(\"incorporated ( 59\", 1)\n",
      "Pair{AbstractString,Int64}(\"7 / 16\", 1)\n",
      "Pair{AbstractString,Int64}(\"the protease is\", 1)\n",
      "Pair{AbstractString,Int64}(\"receptors\", 3)\n",
      "Pair{AbstractString,Int64}(\"pA. We\", 1)\n",
      "Pair{AbstractString,Int64}(\"glutamine lactam (\", 1)\n",
      "Pair{AbstractString,Int64}(\"KLK 13 during\", 2)\n",
      "Pair{AbstractString,Int64}(\"a close relative\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 ( CVB\", 1)\n",
      "Pair{AbstractString,Int64}(\"level of KLKs\", 1)\n",
      "Pair{AbstractString,Int64}(\"224 and nsp\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Lipofectamin\", 1)\n",
      "Pair{AbstractString,Int64}(\"H-bond from His\", 1)\n",
      "Pair{AbstractString,Int64}(\"thawing ( RT\", 1)\n",
      "Pair{AbstractString,Int64}(\"using Alamar-Blue™ Cell\", 1)\n",
      "Pair{AbstractString,Int64}(\"we found that\", 4)\n",
      "Pair{AbstractString,Int64}(\"reported that\", 3)\n",
      "Pair{AbstractString,Int64}(\"Mouse\", 1)\n",
      "Pair{AbstractString,Int64}(\"373 ratio\", 1)\n",
      "Pair{AbstractString,Int64}(\"_ TMPRSS\", 2)\n",
      "Pair{AbstractString,Int64}(\"= 0.48 µM\", 1)\n",
      "Pair{AbstractString,Int64}(\"manufacturer ' s\", 13)\n",
      "Pair{AbstractString,Int64}(\"protease \\\"\", 1)\n",
      "Pair{AbstractString,Int64}(\"for subsequent experiments.\", 1)\n",
      "Pair{AbstractString,Int64}(\"small and\", 2)\n",
      "Pair{AbstractString,Int64}(\"expressing TMPRSS 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"poor activity in\", 1)\n",
      "Pair{AbstractString,Int64}(\"285 mutations in\", 1)\n",
      "Pair{AbstractString,Int64}(\"were similar to\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 45 mg\", 1)\n",
      "Pair{AbstractString,Int64}(\"activity in virus-infected\", 1)\n",
      "Pair{AbstractString,Int64}(\"odor memories\", 1)\n",
      "Pair{AbstractString,Int64}(\"respectively. The mean\", 1)\n",
      "Pair{AbstractString,Int64}(\"254 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"their excitability. If\", 1)\n",
      "Pair{AbstractString,Int64}(\"the N-cap (\", 1)\n",
      "Pair{AbstractString,Int64}(\"was performed in\", 1)\n",
      "Pair{AbstractString,Int64}(\"so was the\", 1)\n",
      "Pair{AbstractString,Int64}(\"Dq-expressing cells.\", 2)\n",
      "Pair{AbstractString,Int64}(\"function. The SARS-CoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"6 f-h\", 1)\n",
      "Pair{AbstractString,Int64}(\"proteases was\", 2)\n",
      "Pair{AbstractString,Int64}(\"their piriform cortex\", 1)\n",
      "Pair{AbstractString,Int64}(\"also occasionally\", 1)\n",
      "Pair{AbstractString,Int64}(\"custom-made switcher allowed\", 1)\n",
      "Pair{AbstractString,Int64}(\"of each stock\", 1)\n",
      "Pair{AbstractString,Int64}(\"of an almost\", 1)\n",
      "Pair{AbstractString,Int64}(\"Next ,\", 8)\n",
      "Pair{AbstractString,Int64}(\"current steps and\", 1)\n",
      "Pair{AbstractString,Int64}(\"neurons in\", 4)\n",
      "Pair{AbstractString,Int64}(\"anti-mouse secondary\", 2)\n",
      "Pair{AbstractString,Int64}(\"carry the cFos-tTA\", 1)\n",
      "Pair{AbstractString,Int64}(\". Hits were\", 1)\n",
      "Pair{AbstractString,Int64}(\"and then cinnamoyl\", 1)\n",
      "Pair{AbstractString,Int64}(\"shown that TetTag-dependent\", 1)\n",
      "Pair{AbstractString,Int64}(\"acid ( TFA\", 1)\n",
      "Pair{AbstractString,Int64}(\"methanol ( 5\", 2)\n",
      "Pair{AbstractString,Int64}(\"KCl\", 2)\n",
      "Pair{AbstractString,Int64}(\"a neutral environment\", 1)\n",
      "Pair{AbstractString,Int64}(\"( FRET\", 1)\n",
      "Pair{AbstractString,Int64}(\"hydroxy\", 1)\n",
      "Pair{AbstractString,Int64}(\", the compound\", 5)\n",
      "Pair{AbstractString,Int64}(\"result also\", 1)\n",
      "Pair{AbstractString,Int64}(\"eventual\", 1)\n",
      "Pair{AbstractString,Int64}(\"and hM 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 ) were\", 1)\n",
      "Pair{AbstractString,Int64}(\"at the P\", 1)\n",
      "Pair{AbstractString,Int64}(\": Crystal structures\", 1)\n",
      "Pair{AbstractString,Int64}(\"were generated by\", 3)\n",
      "Pair{AbstractString,Int64}(\"Lipofectamine\", 2)\n",
      "Pair{AbstractString,Int64}(\"RNA. For that\", 1)\n",
      "Pair{AbstractString,Int64}(\"outbreak of\", 3)\n",
      "Pair{AbstractString,Int64}(\"0.6 µM\", 1)\n",
      "Pair{AbstractString,Int64}(\"competing interests. Supplementary\", 1)\n",
      "Pair{AbstractString,Int64}(\"memory formation\", 2)\n",
      "Pair{AbstractString,Int64}(\"examined the\", 4)\n",
      "Pair{AbstractString,Int64}(\"18.5\", 1)\n",
      "Pair{AbstractString,Int64}(\"number of recorded\", 1)\n",
      "Pair{AbstractString,Int64}(\"replicons ( Table\", 1)\n",
      "Pair{AbstractString,Int64}(\"addition frequently\", 1)\n",
      "Pair{AbstractString,Int64}(\"available to researchers\", 1)\n",
      "Pair{AbstractString,Int64}(\"application. After\", 1)\n",
      "Pair{AbstractString,Int64}(\"literature as\", 1)\n",
      "Pair{AbstractString,Int64}(\"foot shock. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"( PSI ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"collection tube. 100\", 1)\n",
      "Pair{AbstractString,Int64}(\"and memory (\", 1)\n",
      "Pair{AbstractString,Int64}(\"violet (\", 1)\n",
      "Pair{AbstractString,Int64}(\"stopped 24\", 1)\n",
      "Pair{AbstractString,Int64}(\"· mL -\", 6)\n",
      "Pair{AbstractString,Int64}(\"suppression of\", 1)\n",
      "Pair{AbstractString,Int64}(\"this insertion\", 1)\n",
      "Pair{AbstractString,Int64}(\"furoate , mifepristone\", 1)\n",
      "Pair{AbstractString,Int64}(\", compared to\", 5)\n",
      "Pair{AbstractString,Int64}(\"SARS-CoV- 2 strain\", 3)\n",
      "Pair{AbstractString,Int64}(\", or one\", 1)\n",
      "Pair{AbstractString,Int64}(\"T 3327 C\", 1)\n",
      "Pair{AbstractString,Int64}(\"to problems\", 1)\n",
      "Pair{AbstractString,Int64}(\"recordings in\", 2)\n",
      "Pair{AbstractString,Int64}(\"purified proteases and\", 1)\n",
      "Pair{AbstractString,Int64}(\"years\", 2)\n",
      "Pair{AbstractString,Int64}(\"10 a )\", 1)\n",
      "Pair{AbstractString,Int64}(\"seconds\", 2)\n",
      "Pair{AbstractString,Int64}(\"bound to\", 1)\n",
      "Pair{AbstractString,Int64}(\"group \\\" ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"were Fostagged\", 2)\n",
      "Pair{AbstractString,Int64}(\"includes the CS+-tagged\", 1)\n",
      "Pair{AbstractString,Int64}(\"E 64 d\", 1)\n",
      "Pair{AbstractString,Int64}(\"the second lowest.\", 1)\n",
      "Pair{AbstractString,Int64}(\"g - 11\", 2)\n",
      "Pair{AbstractString,Int64}(\"implicated\", 2)\n",
      "Pair{AbstractString,Int64}(\"in cell culture-based\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"Nucleic acid\", 2)\n",
      "Pair{AbstractString,Int64}(\") for SARS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"Rh cells as\", 1)\n",
      "Pair{AbstractString,Int64}(\"an 8\", 1)\n",
      "Pair{AbstractString,Int64}(\"presentation author/funder.\", 1)\n",
      "Pair{AbstractString,Int64}(\"degradation showing the\", 1)\n",
      "Pair{AbstractString,Int64}(\"repulsion of\", 1)\n",
      "Pair{AbstractString,Int64}(\"34 , p=\", 2)\n",
      "Pair{AbstractString,Int64}(\"Scientific ) .\", 2)\n",
      "Pair{AbstractString,Int64}(\"In a hydrogenation\", 1)\n",
      "Pair{AbstractString,Int64}(\"a revealed a\", 1)\n",
      "Pair{AbstractString,Int64}(\"Bat-SARSr-CoV RaTG\", 5)\n",
      "Pair{AbstractString,Int64}(\". Subsequently\", 4)\n",
      "Pair{AbstractString,Int64}(\"using HCoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"together ,\", 3)\n",
      "Pair{AbstractString,Int64}(\"to HCoV-NL\", 1)\n",
      "Pair{AbstractString,Int64}(\"of 292\", 1)\n",
      "Pair{AbstractString,Int64}(\"= 0.01\", 2)\n",
      "Pair{AbstractString,Int64}(\"employing the\", 1)\n",
      "Pair{AbstractString,Int64}(\"described before. To\", 1)\n",
      "Pair{AbstractString,Int64}(\"recombinant viruses VSV\", 1)\n",
      "Pair{AbstractString,Int64}(\"a model\", 1)\n",
      "Pair{AbstractString,Int64}(\"RSCU ) and\", 1)\n",
      "Pair{AbstractString,Int64}(\"triad , whereas\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.0054 ;\", 2)\n",
      "Pair{AbstractString,Int64}(\"evolution of a\", 1)\n",
      "Pair{AbstractString,Int64}(\"that causes hepatitis\", 1)\n",
      "Pair{AbstractString,Int64}(\"exact interaction\", 1)\n",
      "Pair{AbstractString,Int64}(\", W\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 to bat\", 1)\n",
      "Pair{AbstractString,Int64}(\"nucleotide substitutions between\", 1)\n",
      "Pair{AbstractString,Int64}(\"Fisher Scientific qubit\", 1)\n",
      "Pair{AbstractString,Int64}(\"to accommodate its\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 90 -\", 1)\n",
      "Pair{AbstractString,Int64}(\"Briefly ,\", 6)\n",
      "Pair{AbstractString,Int64}(\"Fig. 6\", 9)\n",
      "Pair{AbstractString,Int64}(\"accepted. In the\", 1)\n",
      "Pair{AbstractString,Int64}(\"/EC 50 )\", 2)\n",
      "Pair{AbstractString,Int64}(\"11 µM )\", 1)\n",
      "Pair{AbstractString,Int64}(\"and self-activation at\", 1)\n",
      "Pair{AbstractString,Int64}(\"Gln 189 is\", 1)\n",
      "Pair{AbstractString,Int64}(\"a ~ 1.5\", 1)\n",
      "Pair{AbstractString,Int64}(\"or maturation ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"treating , 57\", 1)\n",
      "Pair{AbstractString,Int64}(\"bioRxiv preprint action\", 1)\n",
      "Pair{AbstractString,Int64}(\"a bit too\", 1)\n",
      "Pair{AbstractString,Int64}(\"of RBD\", 2)\n",
      "Pair{AbstractString,Int64}(\"velocity upon odor\", 1)\n",
      "Pair{AbstractString,Int64}(\"the TMPRSS\", 2)\n",
      "Pair{AbstractString,Int64}(\"this has\", 1)\n",
      "Pair{AbstractString,Int64}(\"-diamidine- 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"Cb\", 3)\n",
      "Pair{AbstractString,Int64}(\"36 was\", 1)\n",
      "Pair{AbstractString,Int64}(\"198\", 1)\n",
      "Pair{AbstractString,Int64}(\"essential medium (\", 2)\n",
      "Pair{AbstractString,Int64}(\"MEME\", 2)\n",
      "Pair{AbstractString,Int64}(\"; b\", 1)\n",
      "Pair{AbstractString,Int64}(\"then washed with\", 1)\n",
      "Pair{AbstractString,Int64}(\"full spike region.\", 1)\n",
      "Pair{AbstractString,Int64}(\"Axel ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"and its 20\", 1)\n",
      "Pair{AbstractString,Int64}(\"indistinguishable\", 1)\n",
      "Pair{AbstractString,Int64}(\"bath\", 2)\n",
      "Pair{AbstractString,Int64}(\"now used widely\", 1)\n",
      "Pair{AbstractString,Int64}(\"site. Note that\", 1)\n",
      "Pair{AbstractString,Int64}(\", one-tailed\", 1)\n",
      "Pair{AbstractString,Int64}(\"( CDCl 3\", 2)\n",
      "Pair{AbstractString,Int64}(\"catalytic triad ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"KLK 13 in\", 1)\n",
      "Pair{AbstractString,Int64}(\"association\", 5)\n",
      "Pair{AbstractString,Int64}(\"regard to its\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 SO 4\", 5)\n",
      "Pair{AbstractString,Int64}(\"viability assays\", 1)\n",
      "Pair{AbstractString,Int64}(\"ES cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"were quantified using\", 2)\n",
      "Pair{AbstractString,Int64}(\". All three\", 1)\n",
      "Pair{AbstractString,Int64}(\"trypsin/EDTA (\", 1)\n",
      "Pair{AbstractString,Int64}(\"the specific\", 4)\n",
      "Pair{AbstractString,Int64}(\"substitutions per\", 2)\n",
      "Pair{AbstractString,Int64}(\"× 25 mL\", 4)\n",
      "Pair{AbstractString,Int64}(\"whose\", 1)\n",
      "Pair{AbstractString,Int64}(\"Fig 4 D\", 1)\n",
      "Pair{AbstractString,Int64}(\"follows : the\", 1)\n",
      "Pair{AbstractString,Int64}(\"substitution in ExoN\", 1)\n",
      "Pair{AbstractString,Int64}(\"for infection experiments\", 1)\n",
      "Pair{AbstractString,Int64}(\"high yields.\", 1)\n",
      "Pair{AbstractString,Int64}(\"deletions\", 2)\n",
      "Pair{AbstractString,Int64}(\"corresponding MHV A\", 1)\n",
      "Pair{AbstractString,Int64}(\"contextual learning in\", 1)\n",
      "Pair{AbstractString,Int64}(\"was inactive\", 1)\n",
      "Pair{AbstractString,Int64}(\"necrosis. 0\", 1)\n",
      "Pair{AbstractString,Int64}(\"CS- : beta-citronellol\", 1)\n",
      "Pair{AbstractString,Int64}(\"to what was\", 1)\n",
      "Pair{AbstractString,Int64}(\"-IFI 17\", 1)\n",
      "Pair{AbstractString,Int64}(\"and 18\", 1)\n",
      "Pair{AbstractString,Int64}(\"largely unknown. Interestingly\", 1)\n",
      "Pair{AbstractString,Int64}(\"% ) .\", 2)\n",
      "Pair{AbstractString,Int64}(\"on December 2019\", 1)\n",
      "Pair{AbstractString,Int64}(\"13 -expressing RD\", 1)\n",
      "Pair{AbstractString,Int64}(\"reaction was stirred\", 7)\n",
      "Pair{AbstractString,Int64}(\"( 1.6 fold\", 1)\n",
      "Pair{AbstractString,Int64}(\"% ) among\", 1)\n",
      "Pair{AbstractString,Int64}(\"17\", 12)\n",
      "Pair{AbstractString,Int64}(\"1 x penicillin/streptomycin\", 1)\n",
      "Pair{AbstractString,Int64}(\"activation.\", 1)\n",
      "Pair{AbstractString,Int64}(\"blot. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"KLK 13 concentrations\", 1)\n",
      "Pair{AbstractString,Int64}(\"proposed to\", 1)\n",
      "Pair{AbstractString,Int64}(\"projected steady growth\", 1)\n",
      "Pair{AbstractString,Int64}(\"Dratio was\", 1)\n",
      "Pair{AbstractString,Int64}(\"the best activity\", 1)\n",
      "Pair{AbstractString,Int64}(\"While the results\", 1)\n",
      "Pair{AbstractString,Int64}(\"\\\" experimental\", 1)\n",
      "Pair{AbstractString,Int64}(\". 152 The\", 1)\n",
      "Pair{AbstractString,Int64}(\"traveled ratio\", 1)\n",
      "Pair{AbstractString,Int64}(\"genes as percentage\", 1)\n",
      "Pair{AbstractString,Int64}(\"with the temperature\", 2)\n",
      "Pair{AbstractString,Int64}(\"doses of MHV\", 2)\n",
      "Pair{AbstractString,Int64}(\", and neutrophils\", 3)\n",
      "Pair{AbstractString,Int64}(\"spleen immune cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"strain Frankfurt- 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"because\", 13)\n",
      "Pair{AbstractString,Int64}(\", Y\", 1)\n",
      "Pair{AbstractString,Int64}(\"-CO-CO-NH- )\", 1)\n",
      "Pair{AbstractString,Int64}(\"respectively. 23 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"that the inactivation\", 2)\n",
      "Pair{AbstractString,Int64}(\"interest is\", 1)\n",
      "Pair{AbstractString,Int64}(\"13,000 x\", 1)\n",
      "Pair{AbstractString,Int64}(\"not associated with\", 1)\n",
      "Pair{AbstractString,Int64}(\"39.1 %\", 1)\n",
      "Pair{AbstractString,Int64}(\"background\", 1)\n",
      "Pair{AbstractString,Int64}(\"ExoNs hydrolyze nucleotides\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.5 mM\", 1)\n",
      "Pair{AbstractString,Int64}(\"or the empty\", 1)\n",
      "Pair{AbstractString,Int64}(\"preprint. 9\", 1)\n",
      "Pair{AbstractString,Int64}(\"hour. Cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"purified using\", 1)\n",
      "Pair{AbstractString,Int64}(\"preprint supplemented with\", 1)\n",
      "Pair{AbstractString,Int64}(\"memory of the\", 1)\n",
      "Pair{AbstractString,Int64}(\"-SYCA 5\", 1)\n",
      "Pair{AbstractString,Int64}(\"silencing neutral\", 1)\n",
      "Pair{AbstractString,Int64}(\", Wilcoxon\", 6)\n",
      "Pair{AbstractString,Int64}(\"be cleaved during\", 1)\n",
      "Pair{AbstractString,Int64}(\"of each residue\", 1)\n",
      "Pair{AbstractString,Int64}(\"relative changes\", 2)\n",
      "Pair{AbstractString,Int64}(\"that the relative\", 1)\n",
      "Pair{AbstractString,Int64}(\"is able\", 2)\n",
      "Pair{AbstractString,Int64}(\"at (\", 1)\n",
      "Pair{AbstractString,Int64}(\", CS+ )\", 1)\n",
      "Pair{AbstractString,Int64}(\"Among the members\", 1)\n",
      "Pair{AbstractString,Int64}(\"E directly\", 1)\n",
      "Pair{AbstractString,Int64}(\"associated with foot\", 1)\n",
      "Pair{AbstractString,Int64}(\"in DMEM\", 2)\n",
      "Pair{AbstractString,Int64}(\"then performed\", 1)\n",
      "Pair{AbstractString,Int64}(\"analysis , the\", 1)\n",
      "Pair{AbstractString,Int64}(\"selected cells were\", 1)\n",
      "Pair{AbstractString,Int64}(\"of broad-spectrum antiviral\", 2)\n",
      "Pair{AbstractString,Int64}(\"µL of 50\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.1708 ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"non-toxic , with\", 1)\n",
      "Pair{AbstractString,Int64}(\"band of His-tagged\", 1)\n",
      "Pair{AbstractString,Int64}(\"6 mm ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"d-lactam in the\", 1)\n",
      "Pair{AbstractString,Int64}(\"detectable numbers of\", 1)\n",
      "Pair{AbstractString,Int64}(\"11 as\", 1)\n",
      "Pair{AbstractString,Int64}(\"reactivation generates\", 1)\n",
      "Pair{AbstractString,Int64}(\", f-h )\", 1)\n",
      "Pair{AbstractString,Int64}(\"does not erase\", 1)\n",
      "Pair{AbstractString,Int64}(\"Both nsp\", 1)\n",
      "Pair{AbstractString,Int64}(\"to generalized\", 1)\n",
      "Pair{AbstractString,Int64}(\"on improving the\", 1)\n",
      "Pair{AbstractString,Int64}(\"a-amino\", 1)\n",
      "Pair{AbstractString,Int64}(\", the cDNA\", 1)\n",
      "Pair{AbstractString,Int64}(\"selection in\", 1)\n",
      "Pair{AbstractString,Int64}(\"shape :\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.01\", 12)\n",
      "Pair{AbstractString,Int64}(\"growth hormone sequence\", 1)\n",
      "Pair{AbstractString,Int64}(\"restriction of CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\", we 180\", 1)\n",
      "Pair{AbstractString,Int64}(\"Nonetheless , our\", 1)\n",
      "Pair{AbstractString,Int64}(\"trimethylsilyl\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"Whole-cell recording pipettes\", 1)\n",
      "Pair{AbstractString,Int64}(\"viruses are faster\", 1)\n",
      "Pair{AbstractString,Int64}(\"245 that\", 1)\n",
      "Pair{AbstractString,Int64}(\"μM\", 4)\n",
      "Pair{AbstractString,Int64}(\"Supplementary Table\", 2)\n",
      "Pair{AbstractString,Int64}(\"were more\", 1)\n",
      "Pair{AbstractString,Int64}(\") to amino\", 1)\n",
      "Pair{AbstractString,Int64}(\"changes when\", 1)\n",
      "Pair{AbstractString,Int64}(\"a family\", 1)\n",
      "Pair{AbstractString,Int64}(\"delivery was 0.74\", 1)\n",
      "Pair{AbstractString,Int64}(\"Most strikingly ,\", 1)\n",
      "Pair{AbstractString,Int64}(\". Single\", 1)\n",
      "Pair{AbstractString,Int64}(\"this solution\", 1)\n",
      "Pair{AbstractString,Int64}(\"14 EV-A\", 1)\n",
      "Pair{AbstractString,Int64}(\"responses we\", 1)\n",
      "Pair{AbstractString,Int64}(\"therapeutic index.\", 1)\n",
      "Pair{AbstractString,Int64}(\"of one ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"probably telaprivir\", 1)\n",
      "Pair{AbstractString,Int64}(\"-/mice. In\", 1)\n",
      "Pair{AbstractString,Int64}(\"direction by far.\", 1)\n",
      "Pair{AbstractString,Int64}(\"stopped 24 hours\", 1)\n",
      "Pair{AbstractString,Int64}(\"carbon. Both\", 1)\n",
      "Pair{AbstractString,Int64}(\"Haberly , 2003\", 1)\n",
      "Pair{AbstractString,Int64}(\"20 ℃ with\", 1)\n",
      "Pair{AbstractString,Int64}(\"3.6 and 4.9\", 1)\n",
      "Pair{AbstractString,Int64}(\"have been described\", 5)\n",
      "Pair{AbstractString,Int64}(\"polyprotein-coding sequence\", 1)\n",
      "Pair{AbstractString,Int64}(\"replication , assembly\", 1)\n",
      "Pair{AbstractString,Int64}(\"Re-EMC/MERS\", 1)\n",
      "Pair{AbstractString,Int64}(\"non-toxic concentrations (\", 1)\n",
      "Pair{AbstractString,Int64}(\"in terms\", 1)\n",
      "Pair{AbstractString,Int64}(\"increase of\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 ( 300\", 1)\n",
      "Pair{AbstractString,Int64}(\"156 mRNA. We\", 1)\n",
      "Pair{AbstractString,Int64}(\"62 Infection\", 1)\n",
      "Pair{AbstractString,Int64}(\"were maintained\", 4)\n",
      "Pair{AbstractString,Int64}(\"to those labeled\", 1)\n",
      "Pair{AbstractString,Int64}(\"T-to-C ( T\", 1)\n",
      "Pair{AbstractString,Int64}(\"the sequential\", 1)\n",
      "Pair{AbstractString,Int64}(\"Red-free\", 1)\n",
      "Pair{AbstractString,Int64}(\"activity we have\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 2.4\", 1)\n",
      "Pair{AbstractString,Int64}(\"11 f\", 9)\n",
      "Pair{AbstractString,Int64}(\"and not\", 3)\n",
      "Pair{AbstractString,Int64}(\"CoV-resistant\", 1)\n",
      "Pair{AbstractString,Int64}(\"proteolytic\", 5)\n",
      "Pair{AbstractString,Int64}(\"2015 ; Dhawale\", 1)\n",
      "Pair{AbstractString,Int64}(\"GO\", 3)\n",
      "Pair{AbstractString,Int64}(\"minutes (\", 3)\n",
      "Pair{AbstractString,Int64}(\"we infected RD\", 1)\n",
      "Pair{AbstractString,Int64}(\"to permit\", 1)\n",
      "Pair{AbstractString,Int64}(\"after the discovery\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.0072 ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"further test\", 1)\n",
      "Pair{AbstractString,Int64}(\"Scheme\", 2)\n",
      "Pair{AbstractString,Int64}(\"is connected\", 1)\n",
      "Pair{AbstractString,Int64}(\"RNA genome (\", 1)\n",
      "Pair{AbstractString,Int64}(\"Chemikalien\", 1)\n",
      "Pair{AbstractString,Int64}(\"and anxiety\", 1)\n",
      "Pair{AbstractString,Int64}(\"PEG 3,350\", 2)\n",
      "Pair{AbstractString,Int64}(\"check they\", 1)\n",
      "Pair{AbstractString,Int64}(\"and an overlay\", 1)\n",
      "Pair{AbstractString,Int64}(\"based on the\", 4)\n",
      "Pair{AbstractString,Int64}(\"acc. AY\", 1)\n",
      "Pair{AbstractString,Int64}(\"the pQCXIP plasmid\", 1)\n",
      "Pair{AbstractString,Int64}(\"accompanying disease such\", 1)\n",
      "Pair{AbstractString,Int64}(\"left side divided\", 1)\n",
      "Pair{AbstractString,Int64}(\"compound with acceptable\", 1)\n",
      "Pair{AbstractString,Int64}(\"perception ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"distance (\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 = moderate\", 1)\n",
      "Pair{AbstractString,Int64}(\"= moderate\", 1)\n",
      "Pair{AbstractString,Int64}(\"specificity subsites beyond\", 1)\n",
      "Pair{AbstractString,Int64}(\"for 1\", 9)\n",
      "Pair{AbstractString,Int64}(\"to a neutral\", 1)\n",
      "Pair{AbstractString,Int64}(\", or Ser\", 1)\n",
      "Pair{AbstractString,Int64}(\"stack.\", 1)\n",
      "Pair{AbstractString,Int64}(\"a fluorescence\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 , the\", 1)\n",
      "Pair{AbstractString,Int64}(\"treatment , with\", 1)\n",
      "Pair{AbstractString,Int64}(\"that drive piriform\", 1)\n",
      "Pair{AbstractString,Int64}(\"pocket has fundamentally\", 1)\n",
      "Pair{AbstractString,Int64}(\"upon activation\", 1)\n",
      "Pair{AbstractString,Int64}(\"restricted ( Extended\", 1)\n",
      "Pair{AbstractString,Int64}(\") P 250\", 6)\n",
      "Pair{AbstractString,Int64}(\"in paraffin\", 1)\n",
      "Pair{AbstractString,Int64}(\". LY 6\", 4)\n",
      "Pair{AbstractString,Int64}(\"patients infected by\", 1)\n",
      "Pair{AbstractString,Int64}(\"at 0.2\", 1)\n",
      "Pair{AbstractString,Int64}(\"a kind gift\", 1)\n",
      "Pair{AbstractString,Int64}(\"read 50\", 1)\n",
      "Pair{AbstractString,Int64}(\"experiments were carried\", 1)\n",
      "Pair{AbstractString,Int64}(\"include the\", 2)\n",
      "Pair{AbstractString,Int64}(\"is induced\", 1)\n",
      "Pair{AbstractString,Int64}(\"complexes , the\", 1)\n",
      "Pair{AbstractString,Int64}(\"amino acids were\", 1)\n",
      "Pair{AbstractString,Int64}(\"; they\", 1)\n",
      "Pair{AbstractString,Int64}(\"p= 0.0779\", 2)\n",
      "Pair{AbstractString,Int64}(\"the virus ,\", 1)\n",
      "Pair{AbstractString,Int64}(\", culture\", 1)\n",
      "Pair{AbstractString,Int64}(\"( d-e\", 1)\n",
      "Pair{AbstractString,Int64}(\"people for\", 1)\n",
      "Pair{AbstractString,Int64}(\"odor ( Supplementary\", 1)\n",
      "Pair{AbstractString,Int64}(\"other compounds that\", 1)\n",
      "Pair{AbstractString,Int64}(\"However , one\", 1)\n",
      "Pair{AbstractString,Int64}(\"Synthesis (\", 1)\n",
      "Pair{AbstractString,Int64}(\"2011 ; Chen\", 1)\n",
      "Pair{AbstractString,Int64}(\"and selectivity values\", 1)\n",
      "Pair{AbstractString,Int64}(\"well supported clades\", 1)\n",
      "Pair{AbstractString,Int64}(\"declare that the\", 1)\n",
      "Pair{AbstractString,Int64}(\"were detected both\", 1)\n",
      "Pair{AbstractString,Int64}(\"spleen (\", 2)\n",
      "Pair{AbstractString,Int64}(\"higher for the\", 1)\n",
      "Pair{AbstractString,Int64}(\"further passaged\", 1)\n",
      "Pair{AbstractString,Int64}(\"not sufficient\", 1)\n",
      "Pair{AbstractString,Int64}(\"is less flexible\", 1)\n",
      "Pair{AbstractString,Int64}(\"luciferase fusion assay\", 1)\n",
      "Pair{AbstractString,Int64}(\"144 and\", 2)\n",
      "Pair{AbstractString,Int64}(\"0.1402 , Figure\", 1)\n",
      "Pair{AbstractString,Int64}(\"g , *\", 2)\n",
      "Pair{AbstractString,Int64}(\"e-expressing immune cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"the CCP 4\", 2)\n",
      "Pair{AbstractString,Int64}(\"deactivated\", 1)\n",
      "Pair{AbstractString,Int64}(\"a similar incubation\", 1)\n",
      "Pair{AbstractString,Int64}(\"is indeed\", 2)\n",
      "Pair{AbstractString,Int64}(\"2 MERS is\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells , purified\", 1)\n",
      "Pair{AbstractString,Int64}(\"mapping of the\", 1)\n",
      "Pair{AbstractString,Int64}(\"pp 1 ab\", 1)\n",
      "Pair{AbstractString,Int64}(\"arose ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"were killed. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"this insert segment\", 1)\n",
      "Pair{AbstractString,Int64}(\"SG\", 2)\n",
      "Pair{AbstractString,Int64}(\"( 52\", 2)\n",
      "Pair{AbstractString,Int64}(\"C. Medium was\", 1)\n",
      "Pair{AbstractString,Int64}(\"reduction ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\"experimental set-up.\", 1)\n",
      "Pair{AbstractString,Int64}(\"that of SARS-CoV.\", 1)\n",
      "Pair{AbstractString,Int64}(\"hold for LLC-MK\", 1)\n",
      "Pair{AbstractString,Int64}(\"box ' floor\", 1)\n",
      "Pair{AbstractString,Int64}(\"three sites (\", 1)\n",
      "Pair{AbstractString,Int64}(\"protein was subjected\", 1)\n",
      "Pair{AbstractString,Int64}(\"the nucleotide\", 2)\n",
      "Pair{AbstractString,Int64}(\"Frankfurt- 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"p= 0.0072\", 1)\n",
      "Pair{AbstractString,Int64}(\"MOI =\", 3)\n",
      "Pair{AbstractString,Int64}(\"numbers ( 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"Kit ( Qiagen\", 1)\n",
      "Pair{AbstractString,Int64}(\"Berlin\", 1)\n",
      "Pair{AbstractString,Int64}(\"efforts\", 3)\n",
      "Pair{AbstractString,Int64}(\"for shorter\", 1)\n",
      "Pair{AbstractString,Int64}(\"preprint expressed\", 1)\n",
      "Pair{AbstractString,Int64}(\"and CS-as\", 1)\n",
      "Pair{AbstractString,Int64}(\"analog\", 5)\n",
      "Pair{AbstractString,Int64}(\"CoVs LYRa\", 1)\n",
      "Pair{AbstractString,Int64}(\"ethylacetate paired with\", 1)\n",
      "Pair{AbstractString,Int64}(\"associative olfactory learning.\", 1)\n",
      "Pair{AbstractString,Int64}(\"relationships\", 3)\n",
      "Pair{AbstractString,Int64}(\"4.3\", 1)\n",
      "Pair{AbstractString,Int64}(\"the replication ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"protecting group\", 1)\n",
      "Pair{AbstractString,Int64}(\"to test memory\", 1)\n",
      "Pair{AbstractString,Int64}(\"Current was\", 1)\n",
      "Pair{AbstractString,Int64}(\"3.36\", 1)\n",
      "Pair{AbstractString,Int64}(\"with two\", 1)\n",
      "Pair{AbstractString,Int64}(\"was plated at\", 1)\n",
      "Pair{AbstractString,Int64}(\"increase in its\", 1)\n",
      "Pair{AbstractString,Int64}(\"in CVB 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"using DBT-\", 1)\n",
      "Pair{AbstractString,Int64}(\"side-chain. Alternatively ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"the medial\", 1)\n",
      "Pair{AbstractString,Int64}(\"eugenol and betacitronellol\", 1)\n",
      "Pair{AbstractString,Int64}(\"a series\", 3)\n",
      "Pair{AbstractString,Int64}(\"L-glutamine\", 1)\n",
      "Pair{AbstractString,Int64}(\", the order\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"to baseline exploratory\", 1)\n",
      "Pair{AbstractString,Int64}(\"FBS/ 10\", 1)\n",
      "Pair{AbstractString,Int64}(\"and Cas 9\", 1)\n",
      "Pair{AbstractString,Int64}(\"SARS-CoV- 2 to\", 1)\n",
      "Pair{AbstractString,Int64}(\"silenced Fos-tagged ensembles\", 1)\n",
      "Pair{AbstractString,Int64}(\"degradation showing\", 1)\n",
      "Pair{AbstractString,Int64}(\"strains now available\", 1)\n",
      "Pair{AbstractString,Int64}(\"well. Crystallization. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"weak.\", 1)\n",
      "Pair{AbstractString,Int64}(\"difference between the\", 3)\n",
      "Pair{AbstractString,Int64}(\"4 ) or\", 1)\n",
      "Pair{AbstractString,Int64}(\"but only ~\", 1)\n",
      "Pair{AbstractString,Int64}(\"n= 8 ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"8 , 10\", 1)\n",
      "Pair{AbstractString,Int64}(\"( or oxyanion\", 1)\n",
      "Pair{AbstractString,Int64}(\"but antiviral activities\", 1)\n",
      "Pair{AbstractString,Int64}(\"chloroform and 60\", 1)\n",
      "Pair{AbstractString,Int64}(\"at least 14\", 1)\n",
      "Pair{AbstractString,Int64}(\"potential threshold before\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 Dq\", 5)\n",
      "Pair{AbstractString,Int64}(\"hepatic viral burden\", 1)\n",
      "Pair{AbstractString,Int64}(\"( N\", 3)\n",
      "Pair{AbstractString,Int64}(\"significantly different\", 3)\n",
      "Pair{AbstractString,Int64}(\"our aketoamides\", 1)\n",
      "Pair{AbstractString,Int64}(\"near this\", 1)\n",
      "Pair{AbstractString,Int64}(\"viruses in\", 4)\n",
      "Pair{AbstractString,Int64}(\"groups respectively ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"compound performed\", 1)\n",
      "Pair{AbstractString,Int64}(\"GCaMP ( n=\", 1)\n",
      "Pair{AbstractString,Int64}(\"that was inactivated\", 1)\n",
      "Pair{AbstractString,Int64}(\"leading to a\", 1)\n",
      "Pair{AbstractString,Int64}(\"subsequently assembled by\", 1)\n",
      "Pair{AbstractString,Int64}(\"extracted by conventional\", 1)\n",
      "Pair{AbstractString,Int64}(\"preprint Statistical\", 1)\n",
      "Pair{AbstractString,Int64}(\"adenylate kinase from\", 1)\n",
      "Pair{AbstractString,Int64}(\"and space\", 1)\n",
      "Pair{AbstractString,Int64}(\"different requirements\", 1)\n",
      "Pair{AbstractString,Int64}(\"Four steroid\", 1)\n",
      "Pair{AbstractString,Int64}(\"( experimental group\", 3)\n",
      "Pair{AbstractString,Int64}(\"mice throughout the\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells were treated\", 2)\n",
      "Pair{AbstractString,Int64}(\"ZYD ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"PEG 1,000\", 3)\n",
      "Pair{AbstractString,Int64}(\"trypsin/EDTA\", 1)\n",
      "Pair{AbstractString,Int64}(\"the same species\", 1)\n",
      "Pair{AbstractString,Int64}(\"against the major\", 1)\n",
      "Pair{AbstractString,Int64}(\"pro revealed\", 1)\n",
      "Pair{AbstractString,Int64}(\"spatially\", 1)\n",
      "Pair{AbstractString,Int64}(\"Influenza\", 1)\n",
      "Pair{AbstractString,Int64}(\"a-amino acid\", 1)\n",
      "Pair{AbstractString,Int64}(\"structural protein\", 1)\n",
      "Pair{AbstractString,Int64}(\"using as\", 3)\n",
      "Pair{AbstractString,Int64}(\"the SARS-CoV replicon\", 3)\n",
      "Pair{AbstractString,Int64}(\"protected\", 2)\n",
      "Pair{AbstractString,Int64}(\"Scope II microscope\", 1)\n",
      "Pair{AbstractString,Int64}(\"in culture\", 2)\n",
      "Pair{AbstractString,Int64}(\"structures of SARS-CoV\", 2)\n",
      "Pair{AbstractString,Int64}(\"strain Frankfurt-\", 1)\n",
      "Pair{AbstractString,Int64}(\"Multiple\", 2)\n",
      "Pair{AbstractString,Int64}(\", ethyl-acetate (\", 1)\n",
      "Pair{AbstractString,Int64}(\"6 E significantly\", 1)\n",
      "Pair{AbstractString,Int64}(\"and that CNO-mediated\", 1)\n",
      "Pair{AbstractString,Int64}(\"trigger 325\", 1)\n",
      "Pair{AbstractString,Int64}(\"and enterovirus\", 1)\n",
      "Pair{AbstractString,Int64}(\"of His-tagged\", 1)\n",
      "Pair{AbstractString,Int64}(\"efficiently\", 3)\n",
      "Pair{AbstractString,Int64}(\"( transparent surface\", 1)\n",
      "Pair{AbstractString,Int64}(\"hM 4 Di-IRES-mCitrine\", 2)\n",
      "Pair{AbstractString,Int64}(\"0.7\", 3)\n",
      "Pair{AbstractString,Int64}(\"grown\", 3)\n",
      "Pair{AbstractString,Int64}(\"aversive versus\", 1)\n",
      "Pair{AbstractString,Int64}(\"objective\", 1)\n",
      "Pair{AbstractString,Int64}(\"not include the\", 1)\n",
      "Pair{AbstractString,Int64}(\"the host cell.\", 1)\n",
      "Pair{AbstractString,Int64}(\"receptor , host\", 1)\n",
      "Pair{AbstractString,Int64}(\"* p= 0.0026\", 1)\n",
      "Pair{AbstractString,Int64}(\"that such\", 1)\n",
      "Pair{AbstractString,Int64}(\", and SARS-CoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"the substrate as\", 1)\n",
      "Pair{AbstractString,Int64}(\"put back on\", 1)\n",
      "Pair{AbstractString,Int64}(\"analogs relative to\", 1)\n",
      "Pair{AbstractString,Int64}(\"of RSCU\", 1)\n",
      "Pair{AbstractString,Int64}(\"lack 89 of\", 1)\n",
      "Pair{AbstractString,Int64}(\"coronavirus complete\", 1)\n",
      "Pair{AbstractString,Int64}(\"structure-based\", 3)\n",
      "Pair{AbstractString,Int64}(\"subjected to\", 9)\n",
      "Pair{AbstractString,Int64}(\"were lysed\", 2)\n",
      "Pair{AbstractString,Int64}(\"beta-CoV lineage\", 1)\n",
      "Pair{AbstractString,Int64}(\"only three\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 pseudovirus\", 1)\n",
      "Pair{AbstractString,Int64}(\"6 were\", 1)\n",
      "Pair{AbstractString,Int64}(\"As members\", 1)\n",
      "Pair{AbstractString,Int64}(\"sufficient\", 16)\n",
      "Pair{AbstractString,Int64}(\"at 59\", 1)\n",
      "Pair{AbstractString,Int64}(\"cathepsin/endosomal pathway\", 1)\n",
      "Pair{AbstractString,Int64}(\"polynucleotide\", 1)\n",
      "Pair{AbstractString,Int64}(\"At the same\", 1)\n",
      "Pair{AbstractString,Int64}(\"C ) ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"combinatorial\", 1)\n",
      "Pair{AbstractString,Int64}(\", can thus\", 1)\n",
      "Pair{AbstractString,Int64}(\"best compromise compound\", 1)\n",
      "Pair{AbstractString,Int64}(\"Vero cells\", 13)\n",
      "Pair{AbstractString,Int64}(\"interaction between these\", 1)\n",
      "Pair{AbstractString,Int64}(\", odor\", 1)\n",
      "Pair{AbstractString,Int64}(\"shRNAs targeting KLK\", 1)\n",
      "Pair{AbstractString,Int64}(\"FBS\", 2)\n",
      "Pair{AbstractString,Int64}(\"84 % yield\", 1)\n",
      "Pair{AbstractString,Int64}(\"by logarithmic\", 1)\n",
      "Pair{AbstractString,Int64}(\"z-score )\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.0312 and W=\", 2)\n",
      "Pair{AbstractString,Int64}(\"instrument to denature\", 1)\n",
      "Pair{AbstractString,Int64}(\"totally\", 1)\n",
      "Pair{AbstractString,Int64}(\"according to\", 17)\n",
      "Pair{AbstractString,Int64}(\"HCoV-HKU 1 infection.\", 2)\n",
      "Pair{AbstractString,Int64}(\"were collected and\", 1)\n",
      "Pair{AbstractString,Int64}(\"apparent\", 1)\n",
      "Pair{AbstractString,Int64}(\"is 10 -\", 1)\n",
      "Pair{AbstractString,Int64}(\"Information files or\", 1)\n",
      "Pair{AbstractString,Int64}(\"available within the\", 1)\n",
      "Pair{AbstractString,Int64}(\"maximum velocity\", 7)\n",
      "Pair{AbstractString,Int64}(\"and 1 %\", 1)\n",
      "Pair{AbstractString,Int64}(\"infections with the\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"were translated and\", 1)\n",
      "Pair{AbstractString,Int64}(\"C and\", 4)\n",
      "Pair{AbstractString,Int64}(\"concentrations ( Fig.\", 1)\n",
      "Pair{AbstractString,Int64}(\"Sanger\", 1)\n",
      "Pair{AbstractString,Int64}(\"when the CS+\", 1)\n",
      "Pair{AbstractString,Int64}(\"using the soaking\", 1)\n",
      "Pair{AbstractString,Int64}(\", ( Grogg-Chemie\", 1)\n",
      "Pair{AbstractString,Int64}(\"intersectional genetic\", 2)\n",
      "Pair{AbstractString,Int64}(\"in parallel plates\", 1)\n",
      "Pair{AbstractString,Int64}(\", but nafamostat\", 1)\n",
      "Pair{AbstractString,Int64}(\"fusion competent for\", 1)\n",
      "Pair{AbstractString,Int64}(\"E or the\", 1)\n",
      "Pair{AbstractString,Int64}(\"reasons\", 1)\n",
      "Pair{AbstractString,Int64}(\"b-position in\", 1)\n",
      "Pair{AbstractString,Int64}(\"-to- 5\", 2)\n",
      "Pair{AbstractString,Int64}(\"information associated with\", 1)\n",
      "Pair{AbstractString,Int64}(\"followed by nucleophilic\", 1)\n",
      "Pair{AbstractString,Int64}(\"purification. Synthesis of\", 1)\n",
      "Pair{AbstractString,Int64}(\"for codon\", 1)\n",
      "Pair{AbstractString,Int64}(\"× DNA loading\", 1)\n",
      "Pair{AbstractString,Int64}(\"web\", 1)\n",
      "Pair{AbstractString,Int64}(\"the N-capped amino\", 1)\n",
      "Pair{AbstractString,Int64}(\"which is dispensable\", 1)\n",
      "Pair{AbstractString,Int64}(\". This substitution\", 1)\n",
      "Pair{AbstractString,Int64}(\"rounds\", 2)\n",
      "Pair{AbstractString,Int64}(\"improvement might\", 1)\n",
      "Pair{AbstractString,Int64}(\"ExoN of SARS-CoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"% . Crystals\", 1)\n",
      "Pair{AbstractString,Int64}(\"The product\", 1)\n",
      "Pair{AbstractString,Int64}(\", for SARS\", 1)\n",
      "Pair{AbstractString,Int64}(\"50 10 -\", 1)\n",
      "Pair{AbstractString,Int64}(\"compounds ( 11\", 1)\n",
      "Pair{AbstractString,Int64}(\"S 1 /S\", 8)\n",
      "Pair{AbstractString,Int64}(\"S protein activation.\", 1)\n",
      "Pair{AbstractString,Int64}(\", both\", 6)\n",
      "Pair{AbstractString,Int64}(\"saline ( PBS\", 1)\n",
      "Pair{AbstractString,Int64}(\"hours post-transduction ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"of the whole\", 1)\n",
      "Pair{AbstractString,Int64}(\"turn , receives\", 1)\n",
      "Pair{AbstractString,Int64}(\"480 ribavirin\", 1)\n",
      "Pair{AbstractString,Int64}(\"amidation of acyl\", 1)\n",
      "Pair{AbstractString,Int64}(\"Peritonitis Virus (\", 1)\n",
      "Pair{AbstractString,Int64}(\"observe is\", 1)\n",
      "Pair{AbstractString,Int64}(\"than 78,000 confirmed\", 1)\n",
      "Pair{AbstractString,Int64}(\"from 7.5 to\", 1)\n",
      "Pair{AbstractString,Int64}(\"of codons ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"conditions. A numerical\", 1)\n",
      "Pair{AbstractString,Int64}(\"conditioning. Another\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.0001 ; c\", 1)\n",
      "Pair{AbstractString,Int64}(\"synonymous codon usage\", 2)\n",
      "Pair{AbstractString,Int64}(\"increased their sniff\", 1)\n",
      "Pair{AbstractString,Int64}(\") and transcriptome\", 1)\n",
      "Pair{AbstractString,Int64}(\"50 = 0.8\", 1)\n",
      "Pair{AbstractString,Int64}(\"addition with\", 1)\n",
      "Pair{AbstractString,Int64}(\"are not\", 3)\n",
      "Pair{AbstractString,Int64}(\"odor-responsive neurons detected\", 1)\n",
      "Pair{AbstractString,Int64}(\"largely inactive against\", 1)\n",
      "Pair{AbstractString,Int64}(\"defense against novel\", 1)\n",
      "Pair{AbstractString,Int64}(\"human\", 27)\n",
      "Pair{AbstractString,Int64}(\"is included in\", 1)\n",
      "Pair{AbstractString,Int64}(\"7 MΩ\", 1)\n",
      "Pair{AbstractString,Int64}(\"; d ,\", 5)\n",
      "Pair{AbstractString,Int64}(\"f , 11\", 1)\n",
      "Pair{AbstractString,Int64}(\"authors declare\", 2)\n",
      "Pair{AbstractString,Int64}(\"modulation 296 after\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 in\", 3)\n",
      "Pair{AbstractString,Int64}(\"lineage markers\", 1)\n",
      "Pair{AbstractString,Int64}(\"becoming the potential\", 1)\n",
      "Pair{AbstractString,Int64}(\"Indeed , neither\", 1)\n",
      "Pair{AbstractString,Int64}(\"which followed by\", 1)\n",
      "Pair{AbstractString,Int64}(\", only synonymous\", 1)\n",
      "Pair{AbstractString,Int64}(\"also been\", 3)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"medical conditions. 13\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 81\", 1)\n",
      "Pair{AbstractString,Int64}(\"µM against\", 21)\n",
      "Pair{AbstractString,Int64}(\"evolution of SARS-CoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"personal communication\", 1)\n",
      "Pair{AbstractString,Int64}(\"S protein-mediated syncytia\", 1)\n",
      "Pair{AbstractString,Int64}(\", none of\", 1)\n",
      "Pair{AbstractString,Int64}(\"to cells\", 1)\n",
      "Pair{AbstractString,Int64}(\") vs. 0.30\", 1)\n",
      "Pair{AbstractString,Int64}(\"activation of DREADD\", 1)\n",
      "Pair{AbstractString,Int64}(\"two preservation\", 1)\n",
      "Pair{AbstractString,Int64}(\"] Enteroviruses\", 1)\n",
      "Pair{AbstractString,Int64}(\"177 long-term passage\", 1)\n",
      "Pair{AbstractString,Int64}(\"sniff frequency ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"pathways associated with\", 1)\n",
      "Pair{AbstractString,Int64}(\"to GISAID\", 1)\n",
      "Pair{AbstractString,Int64}(\"reduced exploratory behavior\", 1)\n",
      "Pair{AbstractString,Int64}(\"with P 1\", 3)\n",
      "Pair{AbstractString,Int64}(\"Normalized\", 1)\n",
      "Pair{AbstractString,Int64}(\"per cubic\", 2)\n",
      "Pair{AbstractString,Int64}(\"CS+ ( black\", 1)\n",
      "Pair{AbstractString,Int64}(\"( originally\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 ε\", 1)\n",
      "Pair{AbstractString,Int64}(\"quantitative PCR ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"the low\", 1)\n",
      "Pair{AbstractString,Int64}(\"titer at\", 1)\n",
      "Pair{AbstractString,Int64}(\"21 mg ·\", 1)\n",
      "Pair{AbstractString,Int64}(\"included in pathway.\", 1)\n",
      "Pair{AbstractString,Int64}(\"a reduced presence\", 1)\n",
      "Pair{AbstractString,Int64}(\"Thus , we\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 residue in\", 1)\n",
      "Pair{AbstractString,Int64}(\"added , before\", 2)\n",
      "Pair{AbstractString,Int64}(\"the former\", 1)\n",
      "Pair{AbstractString,Int64}(\"6 - 8\", 3)\n",
      "Pair{AbstractString,Int64}(\"and Charles\", 1)\n",
      "Pair{AbstractString,Int64}(\"Step qRT-PCR\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells continued to\", 1)\n",
      "Pair{AbstractString,Int64}(\"three days ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"F ) .\", 4)\n",
      "Pair{AbstractString,Int64}(\"- 132\", 1)\n",
      "Pair{AbstractString,Int64}(\"the horn\", 1)\n",
      "Pair{AbstractString,Int64}(\"S and 18\", 1)\n",
      "Pair{AbstractString,Int64}(\"get\", 1)\n",
      "Pair{AbstractString,Int64}(\"and testing/open\", 1)\n",
      "Pair{AbstractString,Int64}(\"of 251\", 1)\n",
      "Pair{AbstractString,Int64}(\"three-day time window\", 1)\n",
      "Pair{AbstractString,Int64}(\"dipeptidyl aldehyde\", 1)\n",
      "Pair{AbstractString,Int64}(\"infer whether\", 1)\n",
      "Pair{AbstractString,Int64}(\"learning constitute an\", 1)\n",
      "Pair{AbstractString,Int64}(\"One mutation per\", 1)\n",
      "Pair{AbstractString,Int64}(\"50 mL )\", 1)\n",
      "Pair{AbstractString,Int64}(\"a volume of\", 1)\n",
      "Pair{AbstractString,Int64}(\") , again\", 1)\n",
      "Pair{AbstractString,Int64}(\"medium.\", 1)\n",
      "Pair{AbstractString,Int64}(\"neural inputs that\", 1)\n",
      "Pair{AbstractString,Int64}(\"represents a\", 1)\n",
      "Pair{AbstractString,Int64}(\". We tried\", 1)\n",
      "Pair{AbstractString,Int64}(\"demonstrated that KLK-kinin\", 1)\n",
      "Pair{AbstractString,Int64}(\"biological processes. Gene\", 1)\n",
      "Pair{AbstractString,Int64}(\"pathway.\", 1)\n",
      "Pair{AbstractString,Int64}(\"containing the\", 7)\n",
      "Pair{AbstractString,Int64}(\"n ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"moiety\", 3)\n",
      "Pair{AbstractString,Int64}(\"cleavage using a\", 1)\n",
      "Pair{AbstractString,Int64}(\"collected the sample\", 1)\n",
      "Pair{AbstractString,Int64}(\"CDCl 3 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"suggest that Fos-tagged\", 1)\n",
      "Pair{AbstractString,Int64}(\"decreasing sensitivity to\", 1)\n",
      "Pair{AbstractString,Int64}(\"fear conditioning has\", 1)\n",
      "Pair{AbstractString,Int64}(\"have another big\", 1)\n",
      "Pair{AbstractString,Int64}(\"pairing during training\", 1)\n",
      "Pair{AbstractString,Int64}(\", original\", 1)\n",
      "Pair{AbstractString,Int64}(\"and negative\", 1)\n",
      "Pair{AbstractString,Int64}(\"associated with other\", 1)\n",
      "Pair{AbstractString,Int64}(\"statistically significant\", 2)\n",
      "Pair{AbstractString,Int64}(\"the right side\", 2)\n",
      "Pair{AbstractString,Int64}(\"calculated by using\", 1)\n",
      "Pair{AbstractString,Int64}(\"herein\", 1)\n",
      "Pair{AbstractString,Int64}(\"heated\", 1)\n",
      "Pair{AbstractString,Int64}(\"36 37\", 4)\n",
      "Pair{AbstractString,Int64}(\"m ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"are very\", 1)\n",
      "Pair{AbstractString,Int64}(\"methionines. 37 Because\", 1)\n",
      "Pair{AbstractString,Int64}(\"or number of\", 1)\n",
      "Pair{AbstractString,Int64}(\"the size and\", 1)\n",
      "Pair{AbstractString,Int64}(\"that despite the\", 2)\n",
      "Pair{AbstractString,Int64}(\"degrading 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"% (\", 3)\n",
      "Pair{AbstractString,Int64}(\", and in\", 1)\n",
      "Pair{AbstractString,Int64}(\"The UV\", 1)\n",
      "Pair{AbstractString,Int64}(\"sequences of 108\", 1)\n",
      "Pair{AbstractString,Int64}(\"requiring\", 2)\n",
      "Pair{AbstractString,Int64}(\"FBS/PBS and\", 1)\n",
      "Pair{AbstractString,Int64}(\"experiments conducted\", 1)\n",
      "Pair{AbstractString,Int64}(\"and Coxsackievirus A\", 1)\n",
      "Pair{AbstractString,Int64}(\"NCBI Reference Sequence\", 1)\n",
      "Pair{AbstractString,Int64}(\"Chi-squared test\", 2)\n",
      "Pair{AbstractString,Int64}(\"h , *\", 1)\n",
      "Pair{AbstractString,Int64}(\"using RHD\", 1)\n",
      "Pair{AbstractString,Int64}(\"( General\", 1)\n",
      "Pair{AbstractString,Int64}(\"in treating this\", 1)\n",
      "Pair{AbstractString,Int64}(\"the bovine\", 1)\n",
      "Pair{AbstractString,Int64}(\"synonymous mutations\", 6)\n",
      "Pair{AbstractString,Int64}(\"mixture.\", 1)\n",
      "Pair{AbstractString,Int64}(\"-fold reduction )\", 1)\n",
      "Pair{AbstractString,Int64}(\"red ( Sigma\", 1)\n",
      "Pair{AbstractString,Int64}(\"methylene ]\", 1)\n",
      "Pair{AbstractString,Int64}(\"probably a\", 2)\n",
      "Pair{AbstractString,Int64}(\"differences in\", 3)\n",
      "Pair{AbstractString,Int64}(\"3 C protease\", 3)\n",
      "Pair{AbstractString,Int64}(\"different reservoir ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"cabinets in\", 1)\n",
      "Pair{AbstractString,Int64}(\"acid of 2019\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.1544\", 1)\n",
      "Pair{AbstractString,Int64}(\"1.1\", 3)\n",
      "Pair{AbstractString,Int64}(\", the corresponding\", 1)\n",
      "Pair{AbstractString,Int64}(\"] and\", 1)\n",
      "Pair{AbstractString,Int64}(\"general procedure\", 6)\n",
      "Pair{AbstractString,Int64}(\"mainly used\", 1)\n",
      "Pair{AbstractString,Int64}(\"to tetO and\", 1)\n",
      "Pair{AbstractString,Int64}(\"pT 7 -CVB\", 1)\n",
      "Pair{AbstractString,Int64}(\"size , flexibility\", 1)\n",
      "Pair{AbstractString,Int64}(\"20 fold more\", 1)\n",
      "Pair{AbstractString,Int64}(\"500\", 16)\n",
      "Pair{AbstractString,Int64}(\"replaced\", 7)\n",
      "Pair{AbstractString,Int64}(\", giving\", 1)\n",
      "Pair{AbstractString,Int64}(\"251\", 2)\n",
      "Pair{AbstractString,Int64}(\"experimental (\", 1)\n",
      "Pair{AbstractString,Int64}(\"as their cytotoxicity\", 1)\n",
      "Pair{AbstractString,Int64}(\", 2012 )\", 2)\n",
      "Pair{AbstractString,Int64}(\"Real-Time PCR (\", 1)\n",
      "Pair{AbstractString,Int64}(\"3.5 mmol )\", 1)\n",
      "Pair{AbstractString,Int64}(\"as steroids suppress\", 1)\n",
      "Pair{AbstractString,Int64}(\"14 .\", 2)\n",
      "Pair{AbstractString,Int64}(\"discovery rate (\", 1)\n",
      "Pair{AbstractString,Int64}(\"Test ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\"or release (\", 1)\n",
      "Pair{AbstractString,Int64}(\"E expression and\", 1)\n",
      "Pair{AbstractString,Int64}(\"reach the \\\"\", 2)\n",
      "Pair{AbstractString,Int64}(\"2 residue that\", 1)\n",
      "Pair{AbstractString,Int64}(\"same compound into\", 1)\n",
      "Pair{AbstractString,Int64}(\"entry receptor\", 1)\n",
      "Pair{AbstractString,Int64}(\"% to\", 6)\n",
      "Pair{AbstractString,Int64}(\"the 2014 outbreak\", 1)\n",
      "Pair{AbstractString,Int64}(\"priming enzyme during\", 1)\n",
      "Pair{AbstractString,Int64}(\"a blasticidin\", 1)\n",
      "Pair{AbstractString,Int64}(\"30 mM MgCl\", 1)\n",
      "Pair{AbstractString,Int64}(\"hormone\", 1)\n",
      "Pair{AbstractString,Int64}(\"data sets of\", 1)\n",
      "Pair{AbstractString,Int64}(\", nor\", 2)\n",
      "Pair{AbstractString,Int64}(\"14 -ExoN\", 3)\n",
      "Pair{AbstractString,Int64}(\"clone 9\", 1)\n",
      "Pair{AbstractString,Int64}(\"small and covered\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 D )\", 1)\n",
      "Pair{AbstractString,Int64}(\"×g.\", 1)\n",
      "Pair{AbstractString,Int64}(\"\\\" ( 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"this solution ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"1.09 in hM\", 1)\n",
      "Pair{AbstractString,Int64}(\"tools available\", 1)\n",
      "Pair{AbstractString,Int64}(\"was extracted and\", 1)\n",
      "Pair{AbstractString,Int64}(\"condition\", 3)\n",
      "Pair{AbstractString,Int64}(\"% nucleotide\", 1)\n",
      "Pair{AbstractString,Int64}(\"fibroblasts were\", 1)\n",
      "Pair{AbstractString,Int64}(\"C or\", 2)\n",
      "Pair{AbstractString,Int64}(\"clarity.\", 6)\n",
      "Pair{AbstractString,Int64}(\"KLKs in\", 1)\n",
      "Pair{AbstractString,Int64}(\"521\", 1)\n",
      "Pair{AbstractString,Int64}(\"were stored for\", 1)\n",
      "Pair{AbstractString,Int64}(\"purification. ESI-MS\", 1)\n",
      "Pair{AbstractString,Int64}(\"systemic\", 1)\n",
      "Pair{AbstractString,Int64}(\"infected BHK-\", 1)\n",
      "Pair{AbstractString,Int64}(\"a commonly used\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 . /\", 1)\n",
      "Pair{AbstractString,Int64}(\"CPE )\", 1)\n",
      "Pair{AbstractString,Int64}(\"13 177 gene\", 1)\n",
      "Pair{AbstractString,Int64}(\"a cell-based\", 1)\n",
      "Pair{AbstractString,Int64}(\"p= 0.0026\", 1)\n",
      "Pair{AbstractString,Int64}(\"Odors activate\", 1)\n",
      "Pair{AbstractString,Int64}(\"hypothesis , we\", 2)\n",
      "Pair{AbstractString,Int64}(\"contains the M\", 1)\n",
      "Pair{AbstractString,Int64}(\"our study ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"NC\", 2)\n",
      "Pair{AbstractString,Int64}(\"determine whether nucleoside\", 1)\n",
      "Pair{AbstractString,Int64}(\"features :\", 1)\n",
      "Pair{AbstractString,Int64}(\"kit , Macherey\", 4)\n",
      "Pair{AbstractString,Int64}(\"utilized\", 1)\n",
      "Pair{AbstractString,Int64}(\") . Reference\", 1)\n",
      "Pair{AbstractString,Int64}(\"we identified\", 1)\n",
      "Pair{AbstractString,Int64}(\"cytotoxicity and potent\", 1)\n",
      "Pair{AbstractString,Int64}(\", belonging\", 1)\n",
      "Pair{AbstractString,Int64}(\"to probe\", 1)\n",
      "Pair{AbstractString,Int64}(\"30 minutes had\", 2)\n",
      "Pair{AbstractString,Int64}(\"HA-stained neurons (\", 1)\n",
      "Pair{AbstractString,Int64}(\"supporting\", 4)\n",
      "Pair{AbstractString,Int64}(\"Odor perception\", 1)\n",
      "Pair{AbstractString,Int64}(\"16 , 176\", 1)\n",
      "Pair{AbstractString,Int64}(\"that lost more\", 1)\n",
      "Pair{AbstractString,Int64}(\"mixing of cell\", 1)\n",
      "Pair{AbstractString,Int64}(\"buffer at 37\", 1)\n",
      "Pair{AbstractString,Int64}(\"recent studies\", 1)\n",
      "Pair{AbstractString,Int64}(\"also contained\", 1)\n",
      "Pair{AbstractString,Int64}(\"µg/mL puromycin.\", 1)\n",
      "Pair{AbstractString,Int64}(\"protease. 29 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"logarithmic\", 1)\n",
      "Pair{AbstractString,Int64}(\"flip\", 1)\n",
      "Pair{AbstractString,Int64}(\"the tested dN/dS\", 1)\n",
      "Pair{AbstractString,Int64}(\"that they confer\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Dejean\", 1)\n",
      "Pair{AbstractString,Int64}(\"medium ( DMEM\", 1)\n",
      "Pair{AbstractString,Int64}(\"were empty-handed\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 5\", 7)\n",
      "Pair{AbstractString,Int64}(\"fibroblasts were transduced\", 1)\n",
      "Pair{AbstractString,Int64}(\"have been observed\", 1)\n",
      "Pair{AbstractString,Int64}(\"efficient\", 3)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"50 = 4.3\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells expressing human\", 1)\n",
      "Pair{AbstractString,Int64}(\"glycoprotein that is\", 1)\n",
      "Pair{AbstractString,Int64}(\"capsid-coding sequence\", 1)\n",
      "Pair{AbstractString,Int64}(\", 25 mM\", 1)\n",
      "Pair{AbstractString,Int64}(\"navigate this\", 1)\n",
      "Pair{AbstractString,Int64}(\", 65 Chemical\", 1)\n",
      "Pair{AbstractString,Int64}(\"r with\", 1)\n",
      "Pair{AbstractString,Int64}(\"observed near\", 1)\n",
      "Pair{AbstractString,Int64}(\"corticosteroids was associated\", 1)\n",
      "Pair{AbstractString,Int64}(\"methyl amino-acid ester\", 1)\n",
      "Pair{AbstractString,Int64}(\", resp.\", 1)\n",
      "Pair{AbstractString,Int64}(\"Omnibus (\", 1)\n",
      "Pair{AbstractString,Int64}(\"CSis\", 1)\n",
      "Pair{AbstractString,Int64}(\"129\", 1)\n",
      "Pair{AbstractString,Int64}(\"infection are reported\", 1)\n",
      "Pair{AbstractString,Int64}(\"in resistance to\", 1)\n",
      "Pair{AbstractString,Int64}(\"protease was\", 1)\n",
      "Pair{AbstractString,Int64}(\"the cyclopropylmethyl side-chain\", 1)\n",
      "Pair{AbstractString,Int64}(\"HA-stained neurons\", 1)\n",
      "Pair{AbstractString,Int64}(\"drugs , including\", 1)\n",
      "Pair{AbstractString,Int64}(\"database ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"among sites (\", 1)\n",
      "Pair{AbstractString,Int64}(\"were almost\", 1)\n",
      "Pair{AbstractString,Int64}(\"Di expression\", 2)\n",
      "Pair{AbstractString,Int64}(\"telaprivir and boceprivir\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 2 x\", 3)\n",
      "Pair{AbstractString,Int64}(\"shock ( see\", 1)\n",
      "Pair{AbstractString,Int64}(\"recommendations. LY\", 1)\n",
      "Pair{AbstractString,Int64}(\"hypothesis (\", 1)\n",
      "Pair{AbstractString,Int64}(\"and boceprivir\", 1)\n",
      "Pair{AbstractString,Int64}(\"culture plates along\", 1)\n",
      "Pair{AbstractString,Int64}(\"NSP\", 1)\n",
      "Pair{AbstractString,Int64}(\") isolated\", 1)\n",
      "Pair{AbstractString,Int64}(\"ExoN-AA could\", 2)\n",
      "Pair{AbstractString,Int64}(\"oil\", 3)\n",
      "Pair{AbstractString,Int64}(\"In this\", 3)\n",
      "Pair{AbstractString,Int64}(\"intracellular Renilla luciferase\", 1)\n",
      "Pair{AbstractString,Int64}(\"piriform cortex activity\", 1)\n",
      "Pair{AbstractString,Int64}(\"bioRxiv preprint. SARS-CoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 B and\", 1)\n",
      "Pair{AbstractString,Int64}(\"on L\", 1)\n",
      "Pair{AbstractString,Int64}(\"clearly\", 2)\n",
      "Pair{AbstractString,Int64}(\"early lead\", 1)\n",
      "Pair{AbstractString,Int64}(\"susceptible cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"plates at 349\", 1)\n",
      "Pair{AbstractString,Int64}(\"h , activity\", 1)\n",
      "Pair{AbstractString,Int64}(\"similar between the\", 1)\n",
      "Pair{AbstractString,Int64}(\"mutations , D\", 1)\n",
      "Pair{AbstractString,Int64}(\"with antibodies specific\", 2)\n",
      "Pair{AbstractString,Int64}(\"counted using\", 1)\n",
      "Pair{AbstractString,Int64}(\") . Detailed\", 1)\n",
      "Pair{AbstractString,Int64}(\"data (\", 3)\n",
      "Pair{AbstractString,Int64}(\", artificial reactivation\", 1)\n",
      "Pair{AbstractString,Int64}(\"crystal structures of\", 3)\n",
      "Pair{AbstractString,Int64}(\"cells. We have\", 1)\n",
      "Pair{AbstractString,Int64}(\"odor presentations was\", 1)\n",
      "Pair{AbstractString,Int64}(\"a foot shock\", 1)\n",
      "Pair{AbstractString,Int64}(\"truncated\", 2)\n",
      "Pair{AbstractString,Int64}(\"The cellular and\", 1)\n",
      "Pair{AbstractString,Int64}(\"control cultures\", 1)\n",
      "Pair{AbstractString,Int64}(\"were intraperitoneally (\", 1)\n",
      "Pair{AbstractString,Int64}(\"238 with\", 1)\n",
      "Pair{AbstractString,Int64}(\"viruses containing\", 1)\n",
      "Pair{AbstractString,Int64}(\"from the Codon\", 1)\n",
      "Pair{AbstractString,Int64}(\"copy.\", 2)\n",
      "Pair{AbstractString,Int64}(\"C proteases of\", 1)\n",
      "Pair{AbstractString,Int64}(\"to 1\", 1)\n",
      "Pair{AbstractString,Int64}(\", cannot generate\", 1)\n",
      "Pair{AbstractString,Int64}(\"2003 ; Seoane\", 1)\n",
      "Pair{AbstractString,Int64}(\"p= 0.9064 .\", 1)\n",
      "Pair{AbstractString,Int64}(\"p< 0.01 .\", 1)\n",
      "Pair{AbstractString,Int64}(\"follows. In\", 1)\n",
      "Pair{AbstractString,Int64}(\"our early lead\", 1)\n",
      "Pair{AbstractString,Int64}(\"5.0 x\", 1)\n",
      "Pair{AbstractString,Int64}(\"6 mutations per\", 1)\n",
      "Pair{AbstractString,Int64}(\"stranded\", 1)\n",
      "Pair{AbstractString,Int64}(\"E ( EC\", 1)\n",
      "Pair{AbstractString,Int64}(\"Behavioral testing\", 1)\n",
      "Pair{AbstractString,Int64}(\"5 minutes\", 7)\n",
      "Pair{AbstractString,Int64}(\"to check\", 1)\n",
      "Pair{AbstractString,Int64}(\", which this\", 1)\n",
      "Pair{AbstractString,Int64}(\"is crucial for\", 1)\n",
      "Pair{AbstractString,Int64}(\"KLK 13 activity\", 1)\n",
      "Pair{AbstractString,Int64}(\"detectable virus in\", 1)\n",
      "Pair{AbstractString,Int64}(\"C-like proteases\", 1)\n",
      "Pair{AbstractString,Int64}(\"factors affecting the\", 1)\n",
      "Pair{AbstractString,Int64}(\"coronaviruses , HCoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"were dissociated using\", 1)\n",
      "Pair{AbstractString,Int64}(\"libitum. Mice were\", 1)\n",
      "Pair{AbstractString,Int64}(\"500 nM KLK\", 2)\n",
      "Pair{AbstractString,Int64}(\"Soluble proteins were\", 1)\n",
      "Pair{AbstractString,Int64}(\"purification.\", 4)\n",
      "Pair{AbstractString,Int64}(\"odor presentation.\", 1)\n",
      "Pair{AbstractString,Int64}(\"in R\", 1)\n",
      "Pair{AbstractString,Int64}(\"causative\", 2)\n",
      "Pair{AbstractString,Int64}(\"S 2 regions\", 1)\n",
      "Pair{AbstractString,Int64}(\", His ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"again significantly blocked\", 1)\n",
      "Pair{AbstractString,Int64}(\"included presentation of\", 1)\n",
      "Pair{AbstractString,Int64}(\"due to incomplete\", 1)\n",
      "Pair{AbstractString,Int64}(\"were Fos-tagged during\", 2)\n",
      "Pair{AbstractString,Int64}(\") , (\", 1)\n",
      "Pair{AbstractString,Int64}(\"new coronavirus as\", 1)\n",
      "Pair{AbstractString,Int64}(\"enteroviral\", 10)\n",
      "Pair{AbstractString,Int64}(\"229 E-Rluc (\", 3)\n",
      "Pair{AbstractString,Int64}(\"kit , Thermo\", 1)\n",
      "Pair{AbstractString,Int64}(\"or AAV-tetO-hM 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"cleavage mutants have\", 1)\n",
      "Pair{AbstractString,Int64}(\"dissociated using Accumax\", 1)\n",
      "Pair{AbstractString,Int64}(\", innersalt\", 1)\n",
      "Pair{AbstractString,Int64}(\"as above. and\", 1)\n",
      "Pair{AbstractString,Int64}(\"a quick assessment\", 1)\n",
      "Pair{AbstractString,Int64}(\"dS )\", 2)\n",
      "Pair{AbstractString,Int64}(\"and stored\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.28 ) cm/s\", 1)\n",
      "Pair{AbstractString,Int64}(\"in The\", 1)\n",
      "Pair{AbstractString,Int64}(\"also increase\", 1)\n",
      "Pair{AbstractString,Int64}(\"but exhibit good\", 1)\n",
      "Pair{AbstractString,Int64}(\"x 10 5\", 7)\n",
      "Pair{AbstractString,Int64}(\"cysteine proteases\", 1)\n",
      "Pair{AbstractString,Int64}(\"seconds ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"was produced only\", 1)\n",
      "Pair{AbstractString,Int64}(\", p= 0.1071\", 1)\n",
      "Pair{AbstractString,Int64}(\"a significant protective\", 1)\n",
      "Pair{AbstractString,Int64}(\"was isobutyl\", 1)\n",
      "Pair{AbstractString,Int64}(\"and filtered. The\", 2)\n",
      "Pair{AbstractString,Int64}(\"in the National\", 1)\n",
      "Pair{AbstractString,Int64}(\"entry 2\", 2)\n",
      "Pair{AbstractString,Int64}(\", p= 0.0112\", 1)\n",
      "Pair{AbstractString,Int64}(\"of the experiment.\", 2)\n",
      "Pair{AbstractString,Int64}(\"0.0047\", 1)\n",
      "Pair{AbstractString,Int64}(\"a Scientifica TriM\", 1)\n",
      "Pair{AbstractString,Int64}(\", with\", 21)\n",
      "Pair{AbstractString,Int64}(\"C. Binding\", 1)\n",
      "Pair{AbstractString,Int64}(\"nt variants\", 1)\n",
      "Pair{AbstractString,Int64}(\"of different KLKs\", 1)\n",
      "Pair{AbstractString,Int64}(\"microtiter plate\", 1)\n",
      "Pair{AbstractString,Int64}(\"up\", 6)\n",
      "Pair{AbstractString,Int64}(\"commercially available system\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 b\", 2)\n",
      "Pair{AbstractString,Int64}(\"tripeptidyl a-ketoamide Cbz-\", 1)\n",
      "Pair{AbstractString,Int64}(\"37 Because the\", 1)\n",
      "Pair{AbstractString,Int64}(\"DREADD-expressing cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"at 498\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 A\", 4)\n",
      "Pair{AbstractString,Int64}(\"presented during\", 1)\n",
      "Pair{AbstractString,Int64}(\"-isobutyl substituent\", 1)\n",
      "Pair{AbstractString,Int64}(\". P 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"identify piriform\", 1)\n",
      "Pair{AbstractString,Int64}(\"phage DNA using\", 1)\n",
      "Pair{AbstractString,Int64}(\"coordinate\", 1)\n",
      "Pair{AbstractString,Int64}(\"11 n (\", 2)\n",
      "Pair{AbstractString,Int64}(\"reversion has\", 1)\n",
      "Pair{AbstractString,Int64}(\") cm/s and\", 1)\n",
      "Pair{AbstractString,Int64}(\", Petaluma\", 1)\n",
      "Pair{AbstractString,Int64}(\"surgery. Wild-type control\", 1)\n",
      "Pair{AbstractString,Int64}(\"Crystallization.\", 1)\n",
      "Pair{AbstractString,Int64}(\"monitored exploratory\", 1)\n",
      "Pair{AbstractString,Int64}(\"ranged from 10\", 1)\n",
      "Pair{AbstractString,Int64}(\"of odor presentations\", 2)\n",
      "Pair{AbstractString,Int64}(\"19 ] [\", 1)\n",
      "Pair{AbstractString,Int64}(\"were passaged until\", 1)\n",
      "Pair{AbstractString,Int64}(\", or cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"DBTs.\", 1)\n",
      "Pair{AbstractString,Int64}(\"Leuven\", 1)\n",
      "Pair{AbstractString,Int64}(\"permission. The\", 101)\n",
      "Pair{AbstractString,Int64}(\"training (\", 1)\n",
      "Pair{AbstractString,Int64}(\"were fed\", 1)\n",
      "Pair{AbstractString,Int64}(\"samples were incubated\", 1)\n",
      "Pair{AbstractString,Int64}(\"( GraphPad\", 3)\n",
      "Pair{AbstractString,Int64}(\"One day post-seeding\", 1)\n",
      "Pair{AbstractString,Int64}(\"Ala\", 4)\n",
      "Pair{AbstractString,Int64}(\"the contributions of\", 1)\n",
      "Pair{AbstractString,Int64}(\"exposure , caused\", 1)\n",
      "Pair{AbstractString,Int64}(\"been infected\", 1)\n",
      "Pair{AbstractString,Int64}(\"pro revealed that\", 1)\n",
      "Pair{AbstractString,Int64}(\", dexamethasone\", 1)\n",
      "Pair{AbstractString,Int64}(\"upon odor exposure.\", 1)\n",
      "Pair{AbstractString,Int64}(\"in triplicate or\", 1)\n",
      "Pair{AbstractString,Int64}(\"genome , many\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells ) were\", 1)\n",
      "Pair{AbstractString,Int64}(\"ethyl-acetate (\", 3)\n",
      "Pair{AbstractString,Int64}(\"E impairs\", 1)\n",
      "Pair{AbstractString,Int64}(\"7 cells with\", 2)\n",
      "Pair{AbstractString,Int64}(\"missense\", 1)\n",
      "Pair{AbstractString,Int64}(\"medium\", 16)\n",
      "Pair{AbstractString,Int64}(\"ablate\", 1)\n",
      "Pair{AbstractString,Int64}(\"some efforts\", 1)\n",
      "Pair{AbstractString,Int64}(\"observed between SARS-CoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"order to understand\", 1)\n",
      "Pair{AbstractString,Int64}(\"Silencing these Fos-tagged\", 1)\n",
      "Pair{AbstractString,Int64}(\"and aligned\", 1)\n",
      "Pair{AbstractString,Int64}(\"; however ,\", 3)\n",
      "Pair{AbstractString,Int64}(\"fusion can occur\", 1)\n",
      "Pair{AbstractString,Int64}(\"t-test with Welch\", 1)\n",
      "Pair{AbstractString,Int64}(\"on the specificity\", 1)\n",
      "Pair{AbstractString,Int64}(\"was generated from\", 1)\n",
      "Pair{AbstractString,Int64}(\"of subgenomic\", 1)\n",
      "Pair{AbstractString,Int64}(\"th\", 7)\n",
      "Pair{AbstractString,Int64}(\"1 h at\", 1)\n",
      "Pair{AbstractString,Int64}(\"The log 10\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"habituation ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"data have been\", 1)\n",
      "Pair{AbstractString,Int64}(\"; Seoane\", 1)\n",
      "Pair{AbstractString,Int64}(\"in total\", 1)\n",
      "Pair{AbstractString,Int64}(\"smaller space at\", 1)\n",
      "Pair{AbstractString,Int64}(\"carbon atoms )\", 1)\n",
      "Pair{AbstractString,Int64}(\"polyprotein (\", 1)\n",
      "Pair{AbstractString,Int64}(\"as the Cb\", 1)\n",
      "Pair{AbstractString,Int64}(\"Instead of exposure\", 1)\n",
      "Pair{AbstractString,Int64}(\"pulmonary edema ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"C as\", 1)\n",
      "Pair{AbstractString,Int64}(\"chain around residues\", 1)\n",
      "Pair{AbstractString,Int64}(\"piriform cortex is\", 1)\n",
      "Pair{AbstractString,Int64}(\"ml EP\", 2)\n",
      "Pair{AbstractString,Int64}(\"in OptiMEM (\", 2)\n",
      "Pair{AbstractString,Int64}(\"regulate nsp 12\", 1)\n",
      "Pair{AbstractString,Int64}(\"-phenylindole ( DAPI\", 1)\n",
      "Pair{AbstractString,Int64}(\"ectopically\", 2)\n",
      "Pair{AbstractString,Int64}(\"poor\", 7)\n",
      "Pair{AbstractString,Int64}(\"6 E specifically\", 1)\n",
      "Pair{AbstractString,Int64}(\"1.2880\", 1)\n",
      "Pair{AbstractString,Int64}(\"to determine\", 4)\n",
      "Pair{AbstractString,Int64}(\"corticosteroids was\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 % FBS/PBS\", 1)\n",
      "Pair{AbstractString,Int64}(\"would not\", 3)\n",
      "Pair{AbstractString,Int64}(\". Compound 7\", 1)\n",
      "Pair{AbstractString,Int64}(\"n. This resulted\", 1)\n",
      "Pair{AbstractString,Int64}(\"enables the replication\", 1)\n",
      "Pair{AbstractString,Int64}(\"-nCoV ) is\", 1)\n",
      "Pair{AbstractString,Int64}(\"is presented.\", 1)\n",
      "Pair{AbstractString,Int64}(\"causes an increase\", 1)\n",
      "Pair{AbstractString,Int64}(\"same beta-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"incubated with\", 4)\n",
      "Pair{AbstractString,Int64}(\"and females for\", 1)\n",
      "Pair{AbstractString,Int64}(\"undergone author/funder.\", 1)\n",
      "Pair{AbstractString,Int64}(\"To selectively\", 1)\n",
      "Pair{AbstractString,Int64}(\"dipeptidyl a-ketoamides have\", 1)\n",
      "Pair{AbstractString,Int64}(\"bioRxiv preprint A\", 2)\n",
      "Pair{AbstractString,Int64}(\"reducing it. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"was presented.\", 2)\n",
      "Pair{AbstractString,Int64}(\"PCR and inserted\", 1)\n",
      "Pair{AbstractString,Int64}(\"regions roughly every\", 1)\n",
      "Pair{AbstractString,Int64}(\"proven\", 2)\n",
      "Pair{AbstractString,Int64}(\"of His-tagged purified\", 1)\n",
      "Pair{AbstractString,Int64}(\"regard\", 1)\n",
      "Pair{AbstractString,Int64}(\"bacteria were spun\", 1)\n",
      "Pair{AbstractString,Int64}(\"convenient\", 1)\n",
      "Pair{AbstractString,Int64}(\"M as described\", 1)\n",
      "Pair{AbstractString,Int64}(\"2016012909576100 ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\"4 µ and\", 1)\n",
      "Pair{AbstractString,Int64}(\"the cooling bath\", 1)\n",
      "Pair{AbstractString,Int64}(\"330\", 3)\n",
      "Pair{AbstractString,Int64}(\"and 341\", 1)\n",
      "Pair{AbstractString,Int64}(\"( MEME )\", 1)\n",
      "Pair{AbstractString,Int64}(\"CS- )\", 5)\n",
      "Pair{AbstractString,Int64}(\"metabolic genes ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"in 0.9\", 1)\n",
      "Pair{AbstractString,Int64}(\"19 outbreak\", 1)\n",
      "Pair{AbstractString,Int64}(\"/ 4 A\", 1)\n",
      "Pair{AbstractString,Int64}(\"nsp 14 -nsp\", 1)\n",
      "Pair{AbstractString,Int64}(\") for 11\", 1)\n",
      "Pair{AbstractString,Int64}(\"the corresponding halfside\", 1)\n",
      "Pair{AbstractString,Int64}(\". Ciclesonide\", 1)\n",
      "Pair{AbstractString,Int64}(\"slight deviations\", 1)\n",
      "Pair{AbstractString,Int64}(\"a new box\", 1)\n",
      "Pair{AbstractString,Int64}(\"box and\", 2)\n",
      "Pair{AbstractString,Int64}(\"has provided\", 1)\n",
      "Pair{AbstractString,Int64}(\"still be good\", 1)\n",
      "Pair{AbstractString,Int64}(\"highorder\", 1)\n",
      "Pair{AbstractString,Int64}(\"11 p )\", 1)\n",
      "Pair{AbstractString,Int64}(\"DREADD-negative cells remained\", 1)\n",
      "Pair{AbstractString,Int64}(\"6.0 and\", 1)\n",
      "Pair{AbstractString,Int64}(\"with the\", 62)\n",
      "Pair{AbstractString,Int64}(\"residues 45 -\", 1)\n",
      "Pair{AbstractString,Int64}(\"and high doses\", 1)\n",
      "Pair{AbstractString,Int64}(\"second\", 3)\n",
      "Pair{AbstractString,Int64}(\"be noted\", 1)\n",
      "Pair{AbstractString,Int64}(\"respiratory disease\", 2)\n",
      "Pair{AbstractString,Int64}(\"2 D )\", 1)\n",
      "Pair{AbstractString,Int64}(\"bars : SEM\", 2)\n",
      "Pair{AbstractString,Int64}(\"Production System (\", 1)\n",
      "Pair{AbstractString,Int64}(\"data points\", 1)\n",
      "Pair{AbstractString,Int64}(\"( -sc\", 1)\n",
      "Pair{AbstractString,Int64}(\"syncytia\", 8)\n",
      "Pair{AbstractString,Int64}(\"suggest that there\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibitory effect on\", 1)\n",
      "Pair{AbstractString,Int64}(\"further purification. Synthesis\", 1)\n",
      "Pair{AbstractString,Int64}(\"reduced in female\", 1)\n",
      "Pair{AbstractString,Int64}(\") . Notably\", 1)\n",
      "Pair{AbstractString,Int64}(\"Synthesis of\", 12)\n",
      "Pair{AbstractString,Int64}(\"/RaTG 13 was\", 1)\n",
      "Pair{AbstractString,Int64}(\"PBS ) from\", 1)\n",
      "Pair{AbstractString,Int64}(\"was inserted into\", 1)\n",
      "Pair{AbstractString,Int64}(\"4 DNA\", 1)\n",
      "Pair{AbstractString,Int64}(\"3.0 days\", 1)\n",
      "Pair{AbstractString,Int64}(\"= cinnamoyl ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"drops and\", 1)\n",
      "Pair{AbstractString,Int64}(\"and few systematic\", 1)\n",
      "Pair{AbstractString,Int64}(\"at the end\", 2)\n",
      "Pair{AbstractString,Int64}(\"activity were performed\", 1)\n",
      "Pair{AbstractString,Int64}(\"91 of\", 1)\n",
      "Pair{AbstractString,Int64}(\"ExoN-EE )\", 2)\n",
      "Pair{AbstractString,Int64}(\") started with\", 1)\n",
      "Pair{AbstractString,Int64}(\"Bonemarrow\", 1)\n",
      "Pair{AbstractString,Int64}(\"~ 4 µM.\", 1)\n",
      "Pair{AbstractString,Int64}(\"database ( NCBI\", 1)\n",
      "Pair{AbstractString,Int64}(\"- 15 o\", 1)\n",
      "Pair{AbstractString,Int64}(\"and immune\", 1)\n",
      "Pair{AbstractString,Int64}(\"we next plan\", 1)\n",
      "Pair{AbstractString,Int64}(\"predicts that five\", 1)\n",
      "Pair{AbstractString,Int64}(\"significant change. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"55\", 8)\n",
      "Pair{AbstractString,Int64}(\"s protocol.\", 3)\n",
      "Pair{AbstractString,Int64}(\", Lonza\", 1)\n",
      "Pair{AbstractString,Int64}(\"a pronounced plasticity\", 1)\n",
      "Pair{AbstractString,Int64}(\"SARS-CoV replicon ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"the percent necrosis.\", 1)\n",
      "Pair{AbstractString,Int64}(\"negative author/funder.\", 1)\n",
      "Pair{AbstractString,Int64}(\"density (\", 1)\n",
      "Pair{AbstractString,Int64}(\"thiohemiketal is\", 7)\n",
      "Pair{AbstractString,Int64}(\", E\", 2)\n",
      "Pair{AbstractString,Int64}(\"CT ( 25.58\", 1)\n",
      "Pair{AbstractString,Int64}(\"indicated\", 10)\n",
      "Pair{AbstractString,Int64}(\"these experiments\", 2)\n",
      "Pair{AbstractString,Int64}(\"-nCoV nucleic acids\", 2)\n",
      "Pair{AbstractString,Int64}(\"coordinates\", 2)\n",
      "Pair{AbstractString,Int64}(\"interest were\", 1)\n",
      "Pair{AbstractString,Int64}(\"furin cleavage\", 1)\n",
      "Pair{AbstractString,Int64}(\"the slight improvement\", 1)\n",
      "Pair{AbstractString,Int64}(\"to ethyl-acetate\", 4)\n",
      "Pair{AbstractString,Int64}(\"as slight\", 1)\n",
      "Pair{AbstractString,Int64}(\"189 ( see\", 1)\n",
      "Pair{AbstractString,Int64}(\", 30 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"Siegelbaum\", 1)\n",
      "Pair{AbstractString,Int64}(\"RBC lysis buffer\", 1)\n",
      "Pair{AbstractString,Int64}(\"been observed even\", 1)\n",
      "Pair{AbstractString,Int64}(\"MES ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"pocket with about\", 1)\n",
      "Pair{AbstractString,Int64}(\"recent study\", 1)\n",
      "Pair{AbstractString,Int64}(\"achievable ( ~\", 1)\n",
      "Pair{AbstractString,Int64}(\"similarly low\", 1)\n",
      "Pair{AbstractString,Int64}(\"In Ly\", 1)\n",
      "Pair{AbstractString,Int64}(\"Signed Ranks\", 6)\n",
      "Pair{AbstractString,Int64}(\"increased sniff\", 1)\n",
      "Pair{AbstractString,Int64}(\"as numbers of\", 1)\n",
      "Pair{AbstractString,Int64}(\"neural circuit input-output\", 1)\n",
      "Pair{AbstractString,Int64}(\"the two groups\", 2)\n",
      "Pair{AbstractString,Int64}(\"only\", 40)\n",
      "Pair{AbstractString,Int64}(\"once.\", 1)\n",
      "Pair{AbstractString,Int64}(\"A numerical\", 1)\n",
      "Pair{AbstractString,Int64}(\"81\", 1)\n",
      "Pair{AbstractString,Int64}(\"was enriched in\", 1)\n",
      "Pair{AbstractString,Int64}(\"the opposite end\", 1)\n",
      "Pair{AbstractString,Int64}(\"monitored sniffing behavior\", 1)\n",
      "Pair{AbstractString,Int64}(\"one-way\", 1)\n",
      "Pair{AbstractString,Int64}(\"27 , p=\", 1)\n",
      "Pair{AbstractString,Int64}(\"contents results in\", 1)\n",
      "Pair{AbstractString,Int64}(\"ontology\", 1)\n",
      "Pair{AbstractString,Int64}(\"that respond\", 1)\n",
      "Pair{AbstractString,Int64}(\"of MHV that\", 1)\n",
      "Pair{AbstractString,Int64}(\"donor/acceptor properties of\", 1)\n",
      "Pair{AbstractString,Int64}(\"betacoronaviruses as\", 1)\n",
      "Pair{AbstractString,Int64}(\"Re-Nsp\", 1)\n",
      "Pair{AbstractString,Int64}(\") presentation were\", 1)\n",
      "Pair{AbstractString,Int64}(\"request. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"mm )\", 2)\n",
      "Pair{AbstractString,Int64}(\"no resistant virus\", 1)\n",
      "Pair{AbstractString,Int64}(\"reversibly\", 1)\n",
      "Pair{AbstractString,Int64}(\"information cannot\", 1)\n",
      "Pair{AbstractString,Int64}(\"s but\", 1)\n",
      "Pair{AbstractString,Int64}(\"Celite to remove\", 1)\n",
      "Pair{AbstractString,Int64}(\"of proteases ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"briefly.\", 1)\n",
      "Pair{AbstractString,Int64}(\"viral replication\", 3)\n",
      "Pair{AbstractString,Int64}(\"in small-animal\", 1)\n",
      "Pair{AbstractString,Int64}(\"NMDA receptor ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"and causes A\", 1)\n",
      "Pair{AbstractString,Int64}(\"sgRNA\", 2)\n",
      "Pair{AbstractString,Int64}(\"HAE _ ctrl\", 1)\n",
      "Pair{AbstractString,Int64}(\"dissolved in\", 10)\n",
      "Pair{AbstractString,Int64}(\"generated our\", 1)\n",
      "Pair{AbstractString,Int64}(\"presence of the\", 3)\n",
      "Pair{AbstractString,Int64}(\"a-ketoamides that target\", 1)\n",
      "Pair{AbstractString,Int64}(\"protein antibody and\", 1)\n",
      "Pair{AbstractString,Int64}(\"inactivation damage. Table\", 1)\n",
      "Pair{AbstractString,Int64}(\"Michaelacceptor compound SG\", 1)\n",
      "Pair{AbstractString,Int64}(\"that virus inactivation\", 1)\n",
      "Pair{AbstractString,Int64}(\"multiple comparison test\", 1)\n",
      "Pair{AbstractString,Int64}(\"virus in the\", 6)\n",
      "Pair{AbstractString,Int64}(\"RPKM ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\"Storage\", 1)\n",
      "Pair{AbstractString,Int64}(\"144 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"of steroids or\", 1)\n",
      "Pair{AbstractString,Int64}(\"68 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"5 % MTS\", 1)\n",
      "Pair{AbstractString,Int64}(\"site was degraded\", 1)\n",
      "Pair{AbstractString,Int64}(\"and prolonged\", 1)\n",
      "Pair{AbstractString,Int64}(\". We therefore\", 2)\n",
      "Pair{AbstractString,Int64}(\"the CS+ (\", 1)\n",
      "Pair{AbstractString,Int64}(\") replication\", 1)\n",
      "Pair{AbstractString,Int64}(\"done in triplicate\", 1)\n",
      "Pair{AbstractString,Int64}(\"RNA from disrupting\", 1)\n",
      "Pair{AbstractString,Int64}(\"same high\", 1)\n",
      "Pair{AbstractString,Int64}(\"human TMPRSS 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"cell lysates (\", 1)\n",
      "Pair{AbstractString,Int64}(\"and cells\", 2)\n",
      "Pair{AbstractString,Int64}(\"146074 ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\"as against\", 3)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"revealed a\", 1)\n",
      "Pair{AbstractString,Int64}(\"depend on experience\", 1)\n",
      "Pair{AbstractString,Int64}(\"by Mann-Whitney U\", 1)\n",
      "Pair{AbstractString,Int64}(\"determined that KLK\", 1)\n",
      "Pair{AbstractString,Int64}(\"positive rate\", 1)\n",
      "Pair{AbstractString,Int64}(\"expression and\", 1)\n",
      "Pair{AbstractString,Int64}(\"1.9 -fold increase\", 1)\n",
      "Pair{AbstractString,Int64}(\"the cytotoxic effect\", 1)\n",
      "Pair{AbstractString,Int64}(\"has its P\", 1)\n",
      "Pair{AbstractString,Int64}(\"-Fam-AGA ( pdC\", 1)\n",
      "Pair{AbstractString,Int64}(\"Particle\", 1)\n",
      "Pair{AbstractString,Int64}(\"a primase/processivity factor\", 1)\n",
      "Pair{AbstractString,Int64}(\"compensate for the\", 1)\n",
      "Pair{AbstractString,Int64}(\"kDa ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"compounds were\", 3)\n",
      "Pair{AbstractString,Int64}(\"cryoprotectant\", 1)\n",
      "Pair{AbstractString,Int64}(\"E-specific\", 2)\n",
      "Pair{AbstractString,Int64}(\"and other\", 3)\n",
      "Pair{AbstractString,Int64}(\"density ( contoured\", 1)\n",
      "Pair{AbstractString,Int64}(\"Arg 188\", 2)\n",
      "Pair{AbstractString,Int64}(\", Sacco\", 1)\n",
      "Pair{AbstractString,Int64}(\"SARS-CoV infection of\", 1)\n",
      "Pair{AbstractString,Int64}(\"Bolding and\", 4)\n",
      "Pair{AbstractString,Int64}(\"assay kit\", 1)\n",
      "Pair{AbstractString,Int64}(\"30 minutes\", 6)\n",
      "Pair{AbstractString,Int64}(\", 2011 )\", 5)\n",
      "Pair{AbstractString,Int64}(\"Another optimization screen\", 1)\n",
      "Pair{AbstractString,Int64}(\"acetate = 4\", 2)\n",
      "Pair{AbstractString,Int64}(\"with geneticin\", 1)\n",
      "Pair{AbstractString,Int64}(\"results were mirrored\", 1)\n",
      "Pair{AbstractString,Int64}(\"guidelines. Viral\", 1)\n",
      "Pair{AbstractString,Int64}(\"plasmid DNA\", 1)\n",
      "Pair{AbstractString,Int64}(\"of several 283\", 1)\n",
      "Pair{AbstractString,Int64}(\"replicons confirmed\", 1)\n",
      "Pair{AbstractString,Int64}(\"% , 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"as inhibitors\", 4)\n",
      "Pair{AbstractString,Int64}(\"except for\", 3)\n",
      "Pair{AbstractString,Int64}(\"does not seem\", 1)\n",
      "Pair{AbstractString,Int64}(\"neural silencing\", 1)\n",
      "Pair{AbstractString,Int64}(\"enteroviral replicons as\", 1)\n",
      "Pair{AbstractString,Int64}(\"removed and 100\", 1)\n",
      "Pair{AbstractString,Int64}(\"Our data suggest\", 1)\n",
      "Pair{AbstractString,Int64}(\"All cell lines\", 1)\n",
      "Pair{AbstractString,Int64}(\"89 of\", 1)\n",
      "Pair{AbstractString,Int64}(\"nucleoside analogues (\", 1)\n",
      "Pair{AbstractString,Int64}(\", methanol/\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 -and 5\", 1)\n",
      "Pair{AbstractString,Int64}(\"was pseudorandomized.\", 1)\n",
      "Pair{AbstractString,Int64}(\") cells were\", 1)\n",
      "Pair{AbstractString,Int64}(\"recover\", 1)\n",
      "Pair{AbstractString,Int64}(\"A ) During\", 1)\n",
      "Pair{AbstractString,Int64}(\"a ) ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"quantifying propagated virus\", 1)\n",
      "Pair{AbstractString,Int64}(\"maximum velocity (\", 2)\n",
      "Pair{AbstractString,Int64}(\"bias\", 5)\n",
      "Pair{AbstractString,Int64}(\"= benzyl.\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.04 , which\", 1)\n",
      "Pair{AbstractString,Int64}(\"21.61 ±\", 1)\n",
      "Pair{AbstractString,Int64}(\") and WIV\", 1)\n",
      "Pair{AbstractString,Int64}(\"observed even\", 1)\n",
      "Pair{AbstractString,Int64}(\"for each mouse.\", 1)\n",
      "Pair{AbstractString,Int64}(\"common features shared\", 1)\n",
      "Pair{AbstractString,Int64}(\"step , the\", 2)\n",
      "Pair{AbstractString,Int64}(\"or as\", 2)\n",
      "Pair{AbstractString,Int64}(\"was harvested ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"of 21.8\", 1)\n",
      "Pair{AbstractString,Int64}(\"a free space\", 1)\n",
      "Pair{AbstractString,Int64}(\"the inhibitor ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"Gln 189 by\", 1)\n",
      "Pair{AbstractString,Int64}(\"preprint retrieval was\", 1)\n",
      "Pair{AbstractString,Int64}(\"examples to\", 1)\n",
      "Pair{AbstractString,Int64}(\"with an area\", 1)\n",
      "Pair{AbstractString,Int64}(\"their similar animal\", 1)\n",
      "Pair{AbstractString,Int64}(\"atoms.\", 1)\n",
      "Pair{AbstractString,Int64}(\"in vitro model\", 1)\n",
      "Pair{AbstractString,Int64}(\"High-fidelity 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"Hz ) ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"the side-walls\", 3)\n",
      "Pair{AbstractString,Int64}(\"treated cells infected\", 1)\n",
      "Pair{AbstractString,Int64}(\"or proKLK 14\", 1)\n",
      "Pair{AbstractString,Int64}(\"of necrosis for\", 1)\n",
      "Pair{AbstractString,Int64}(\"increased 238\", 1)\n",
      "Pair{AbstractString,Int64}(\"61 based\", 1)\n",
      "Pair{AbstractString,Int64}(\"overlaid\", 1)\n",
      "Pair{AbstractString,Int64}(\"to hold for\", 1)\n",
      "Pair{AbstractString,Int64}(\"periodinane ( 337\", 1)\n",
      "Pair{AbstractString,Int64}(\") in liver\", 1)\n",
      "Pair{AbstractString,Int64}(\"still circulating in\", 1)\n",
      "Pair{AbstractString,Int64}(\"/ 2020.02.10.936898\", 5)\n",
      "Pair{AbstractString,Int64}(\"lid (\", 1)\n",
      "Pair{AbstractString,Int64}(\"odor ( neutral\", 1)\n",
      "Pair{AbstractString,Int64}(\"Prism ( GraphPad\", 1)\n",
      "Pair{AbstractString,Int64}(\"substituent was still\", 1)\n",
      "Pair{AbstractString,Int64}(\"2013 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"infected HAE _\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.0156 for the\", 1)\n",
      "Pair{AbstractString,Int64}(\"that spatial\", 1)\n",
      "Pair{AbstractString,Int64}(\"at 120 µM.\", 1)\n",
      "Pair{AbstractString,Int64}(\"parental and passaged\", 1)\n",
      "Pair{AbstractString,Int64}(\"abundance was calculated\", 1)\n",
      "Pair{AbstractString,Int64}(\", pCAGGS-MERS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"choice and will\", 1)\n",
      "Pair{AbstractString,Int64}(\"a free\", 1)\n",
      "Pair{AbstractString,Int64}(\"multiple CS+ (\", 1)\n",
      "Pair{AbstractString,Int64}(\"Supplementary Information files\", 1)\n",
      "Pair{AbstractString,Int64}(\"relationships among\", 1)\n",
      "Pair{AbstractString,Int64}(\"marker-positive\", 1)\n",
      "Pair{AbstractString,Int64}(\", 2007 )\", 2)\n",
      "Pair{AbstractString,Int64}(\"31 32 33\", 4)\n",
      "Pair{AbstractString,Int64}(\"of odor-evoked activity\", 1)\n",
      "Pair{AbstractString,Int64}(\"at one extremity\", 1)\n",
      "Pair{AbstractString,Int64}(\"Met 165 and\", 1)\n",
      "Pair{AbstractString,Int64}(\"43 cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"W= 34\", 1)\n",
      "Pair{AbstractString,Int64}(\"the 289\", 1)\n",
      "Pair{AbstractString,Int64}(\"per cubic millimeter.\", 1)\n",
      "Pair{AbstractString,Int64}(\"infecting DBT cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"terms\", 1)\n",
      "Pair{AbstractString,Int64}(\"259 away from\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.45\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells , the\", 7)\n",
      "Pair{AbstractString,Int64}(\"et\", 77)\n",
      "Pair{AbstractString,Int64}(\"cages\", 1)\n",
      "Pair{AbstractString,Int64}(\"independent recovery attempts\", 1)\n",
      "Pair{AbstractString,Int64}(\"virus inactivation by\", 1)\n",
      "Pair{AbstractString,Int64}(\"positive-sense RNA viruses\", 1)\n",
      "Pair{AbstractString,Int64}(\"to either\", 1)\n",
      "Pair{AbstractString,Int64}(\"without evoking the\", 1)\n",
      "Pair{AbstractString,Int64}(\"of recorded\", 1)\n",
      "Pair{AbstractString,Int64}(\"of high\", 1)\n",
      "Pair{AbstractString,Int64}(\"bars\", 7)\n",
      "Pair{AbstractString,Int64}(\"-Boc group inserted\", 1)\n",
      "Pair{AbstractString,Int64}(\"only two\", 2)\n",
      "Pair{AbstractString,Int64}(\"for 12 h.\", 2)\n",
      "Pair{AbstractString,Int64}(\"88\", 3)\n",
      "Pair{AbstractString,Int64}(\"Compound\", 6)\n",
      "Pair{AbstractString,Int64}(\"contract\", 1)\n",
      "Pair{AbstractString,Int64}(\"Master Mix\", 1)\n",
      "Pair{AbstractString,Int64}(\"Spleen\", 1)\n",
      "Pair{AbstractString,Int64}(\"of split\", 1)\n",
      "Pair{AbstractString,Int64}(\"attenuated\", 4)\n",
      "Pair{AbstractString,Int64}(\"compounds and virus\", 1)\n",
      "Pair{AbstractString,Int64}(\"5.759 x\", 1)\n",
      "Pair{AbstractString,Int64}(\"72 , 90\", 1)\n",
      "Pair{AbstractString,Int64}(\"results show\", 2)\n",
      "Pair{AbstractString,Int64}(\"( CSV )\", 1)\n",
      "Pair{AbstractString,Int64}(\"subjected to the\", 1)\n",
      "Pair{AbstractString,Int64}(\"a BAC reverse\", 1)\n",
      "Pair{AbstractString,Int64}(\", and virus\", 1)\n",
      "Pair{AbstractString,Int64}(\"in water )\", 1)\n",
      "Pair{AbstractString,Int64}(\"standard deviations\", 2)\n",
      "Pair{AbstractString,Int64}(\"mostly hydrophobic in\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 1996\", 1)\n",
      "Pair{AbstractString,Int64}(\". Supernatant was\", 1)\n",
      "Pair{AbstractString,Int64}(\"copies/mL\", 1)\n",
      "Pair{AbstractString,Int64}(\"is transient. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 cells.\", 1)\n",
      "Pair{AbstractString,Int64}(\"a related form\", 1)\n",
      "Pair{AbstractString,Int64}(\"it includes the\", 1)\n",
      "Pair{AbstractString,Int64}(\"binding. We will\", 1)\n",
      "Pair{AbstractString,Int64}(\"viral cysteine\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 13 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"expression of regulators\", 1)\n",
      "Pair{AbstractString,Int64}(\"and were\", 1)\n",
      "Pair{AbstractString,Int64}(\"- 426\", 1)\n",
      "Pair{AbstractString,Int64}(\"cough , fever\", 1)\n",
      "Pair{AbstractString,Int64}(\"= cyclopropyl\", 1)\n",
      "Pair{AbstractString,Int64}(\"equilibrating\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 expression vector\", 1)\n",
      "Pair{AbstractString,Int64}(\"tested 124 in\", 1)\n",
      "Pair{AbstractString,Int64}(\"decreases\", 4)\n",
      "Pair{AbstractString,Int64}(\"of Ser 144\", 2)\n",
      "Pair{AbstractString,Int64}(\"Experimental Procedures )\", 3)\n",
      "Pair{AbstractString,Int64}(\"CMeA ) was\", 1)\n",
      "Pair{AbstractString,Int64}(\"conditioning were both\", 1)\n",
      "Pair{AbstractString,Int64}(\"For i-m\", 1)\n",
      "Pair{AbstractString,Int64}(\"2014 ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\"and velocity\", 1)\n",
      "Pair{AbstractString,Int64}(\"individual a-ketoamides have\", 1)\n",
      "Pair{AbstractString,Int64}(\"are regulated by\", 1)\n",
      "Pair{AbstractString,Int64}(\"for 8\", 1)\n",
      "Pair{AbstractString,Int64}(\"neurons responsive\", 2)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"4,000 varied from\", 1)\n",
      "Pair{AbstractString,Int64}(\"key intermediates\", 1)\n",
      "Pair{AbstractString,Int64}(\"- 11 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"s (\", 4)\n",
      "Pair{AbstractString,Int64}(\"from SARS-CoV-\", 2)\n",
      "Pair{AbstractString,Int64}(\"inferred relationships was\", 1)\n",
      "Pair{AbstractString,Int64}(\"Among 10 genetically\", 1)\n",
      "Pair{AbstractString,Int64}(\", Vazyme\", 4)\n",
      "Pair{AbstractString,Int64}(\"observed that the\", 1)\n",
      "Pair{AbstractString,Int64}(\"alcohol. Oxidation of\", 1)\n",
      "Pair{AbstractString,Int64}(\"most tolerant\", 1)\n",
      "Pair{AbstractString,Int64}(\"antigen presentation\", 1)\n",
      "Pair{AbstractString,Int64}(\"9 as\", 1)\n",
      "Pair{AbstractString,Int64}(\"Probe Kit\", 1)\n",
      "Pair{AbstractString,Int64}(\"8 hours\", 1)\n",
      "Pair{AbstractString,Int64}(\"is fairly small.\", 1)\n",
      "Pair{AbstractString,Int64}(\") , nor\", 1)\n",
      "Pair{AbstractString,Int64}(\"olfactory cortex\", 1)\n",
      "Pair{AbstractString,Int64}(\"fairly\", 1)\n",
      "Pair{AbstractString,Int64}(\"s reported that\", 1)\n",
      "Pair{AbstractString,Int64}(\"Figure 2 G\", 1)\n",
      "Pair{AbstractString,Int64}(\"KLKs makes one\", 1)\n",
      "Pair{AbstractString,Int64}(\"tube ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"FBS/ 10 mM\", 1)\n",
      "Pair{AbstractString,Int64}(\"f ) and\", 1)\n",
      "Pair{AbstractString,Int64}(\"23.3 mmol )\", 1)\n",
      "Pair{AbstractString,Int64}(\"was indistinguishable\", 1)\n",
      "Pair{AbstractString,Int64}(\"formed four\", 1)\n",
      "Pair{AbstractString,Int64}(\"7 cells. Intracellular\", 1)\n",
      "Pair{AbstractString,Int64}(\"CS+ and CSpresentation.\", 1)\n",
      "Pair{AbstractString,Int64}(\"The HCoV-NL\", 1)\n",
      "Pair{AbstractString,Int64}(\"pro s in\", 1)\n",
      "Pair{AbstractString,Int64}(\"mutational\", 1)\n",
      "Pair{AbstractString,Int64}(\"to robustly escape\", 1)\n",
      "Pair{AbstractString,Int64}(\"good inhibitor\", 1)\n",
      "Pair{AbstractString,Int64}(\") 6,000 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"and stained\", 4)\n",
      "Pair{AbstractString,Int64}(\"was surprisingly\", 3)\n",
      "Pair{AbstractString,Int64}(\"in CVB\", 1)\n",
      "Pair{AbstractString,Int64}(\"originated\", 2)\n",
      "Pair{AbstractString,Int64}(\"samples with the\", 1)\n",
      "Pair{AbstractString,Int64}(\"we stereotaxically injected\", 1)\n",
      "Pair{AbstractString,Int64}(\"and vaccines (\", 1)\n",
      "Pair{AbstractString,Int64}(\"immunosuppressant effects ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"simplicity , we\", 1)\n",
      "Pair{AbstractString,Int64}(\"aromatic\", 1)\n",
      "Pair{AbstractString,Int64}(\"236 consensus\", 1)\n",
      "Pair{AbstractString,Int64}(\"ml collection\", 1)\n",
      "Pair{AbstractString,Int64}(\"behavioral significance\", 1)\n",
      "Pair{AbstractString,Int64}(\"both virus families\", 1)\n",
      "Pair{AbstractString,Int64}(\"* p= 0.0428\", 1)\n",
      "Pair{AbstractString,Int64}(\"of piriform odor\", 2)\n",
      "Pair{AbstractString,Int64}(\"viral replicons ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"indicated. Primary\", 1)\n",
      "Pair{AbstractString,Int64}(\"2300 Multilabel reader\", 1)\n",
      "Pair{AbstractString,Int64}(\"Being in\", 1)\n",
      "Pair{AbstractString,Int64}(\"( GlnLactam\", 1)\n",
      "Pair{AbstractString,Int64}(\"78 °\", 3)\n",
      "Pair{AbstractString,Int64}(\"inactivation experiments\", 1)\n",
      "Pair{AbstractString,Int64}(\"e ΔHSC organs\", 1)\n",
      "Pair{AbstractString,Int64}(\"nonstructural\", 2)\n",
      "Pair{AbstractString,Int64}(\"Overall\", 2)\n",
      "Pair{AbstractString,Int64}(\"of piriform ensembles\", 3)\n",
      "Pair{AbstractString,Int64}(\"for was\", 1)\n",
      "Pair{AbstractString,Int64}(\". Statistical significance\", 1)\n",
      "Pair{AbstractString,Int64}(\"existence of\", 1)\n",
      "Pair{AbstractString,Int64}(\"Viral titers\", 1)\n",
      "Pair{AbstractString,Int64}(\"mutations 215\", 1)\n",
      "Pair{AbstractString,Int64}(\"using the\", 27)\n",
      "Pair{AbstractString,Int64}(\"warheads have\", 1)\n",
      "Pair{AbstractString,Int64}(\", Pro 189\", 1)\n",
      "Pair{AbstractString,Int64}(\"bioRxiv preprint expressed\", 1)\n",
      "Pair{AbstractString,Int64}(\"the nucleic\", 4)\n",
      "Pair{AbstractString,Int64}(\"sites ( Figure\", 1)\n",
      "Pair{AbstractString,Int64}(\"found that learned\", 1)\n",
      "Pair{AbstractString,Int64}(\"H-bonds from the\", 1)\n",
      "Pair{AbstractString,Int64}(\"and flash-cooled in\", 2)\n",
      "Pair{AbstractString,Int64}(\"PBS or\", 2)\n",
      "Pair{AbstractString,Int64}(\"PBS )\", 1)\n",
      "Pair{AbstractString,Int64}(\"their activity is\", 1)\n",
      "Pair{AbstractString,Int64}(\"with MOLREP\", 1)\n",
      "Pair{AbstractString,Int64}(\"28 ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"stability in the\", 1)\n",
      "Pair{AbstractString,Int64}(\"ciclesonide ( Fig.\", 1)\n",
      "Pair{AbstractString,Int64}(\"; Antibodies\", 1)\n",
      "Pair{AbstractString,Int64}(\"of coding sequences\", 1)\n",
      "Pair{AbstractString,Int64}(\"shown as individual\", 1)\n",
      "Pair{AbstractString,Int64}(\"motif III ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"EP tube\", 2)\n",
      "Pair{AbstractString,Int64}(\"phenotypes. Next\", 1)\n",
      "Pair{AbstractString,Int64}(\". Fos-tagging\", 1)\n",
      "Pair{AbstractString,Int64}(\"the latter (\", 1)\n",
      "Pair{AbstractString,Int64}(\"paired\", 8)\n",
      "Pair{AbstractString,Int64}(\"( Phusion\", 1)\n",
      "Pair{AbstractString,Int64}(\"s instruction. CC\", 1)\n",
      "Pair{AbstractString,Int64}(\"140\", 2)\n",
      "Pair{AbstractString,Int64}(\"mM NaCl.\", 1)\n",
      "Pair{AbstractString,Int64}(\"quantitative fusion\", 1)\n",
      "Pair{AbstractString,Int64}(\"assays. Stock solutions\", 1)\n",
      "Pair{AbstractString,Int64}(\"cell culture. No\", 1)\n",
      "Pair{AbstractString,Int64}(\"linkage method employing\", 1)\n",
      "Pair{AbstractString,Int64}(\"to RaTG\", 1)\n",
      "Pair{AbstractString,Int64}(\"with decreasing\", 1)\n",
      "Pair{AbstractString,Int64}(\"position , encouraging\", 1)\n",
      "Pair{AbstractString,Int64}(\"the horn \\\"\", 1)\n",
      "Pair{AbstractString,Int64}(\"far the\", 3)\n",
      "Pair{AbstractString,Int64}(\"buffer (\", 4)\n",
      "Pair{AbstractString,Int64}(\"plasmids were used\", 1)\n",
      "Pair{AbstractString,Int64}(\"can be found\", 1)\n",
      "Pair{AbstractString,Int64}(\"and accounting for\", 1)\n",
      "Pair{AbstractString,Int64}(\"517\", 1)\n",
      "Pair{AbstractString,Int64}(\"Peritonitis\", 1)\n",
      "Pair{AbstractString,Int64}(\"strong immunosuppressant effects\", 1)\n",
      "Pair{AbstractString,Int64}(\"-PRIC 285\", 1)\n",
      "Pair{AbstractString,Int64}(\"generate STAT\", 1)\n",
      "Pair{AbstractString,Int64}(\"inoculated with\", 1)\n",
      "Pair{AbstractString,Int64}(\"x p-s/DMEM\", 1)\n",
      "Pair{AbstractString,Int64}(\"a was completely\", 1)\n",
      "Pair{AbstractString,Int64}(\"z-score\", 1)\n",
      "Pair{AbstractString,Int64}(\"cocrystallization. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"-azacytidine ( AZC\", 1)\n",
      "Pair{AbstractString,Int64}(\", the pattern\", 1)\n",
      "Pair{AbstractString,Int64}(\"side-walls ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"the inactive 11\", 1)\n",
      "Pair{AbstractString,Int64}(\"with a 2.5\", 1)\n",
      "Pair{AbstractString,Int64}(\"RNA levels\", 1)\n",
      "Pair{AbstractString,Int64}(\"more accessible than\", 1)\n",
      "Pair{AbstractString,Int64}(\"N-substituted amino-acid esters\", 1)\n",
      "Pair{AbstractString,Int64}(\"during ambient odor\", 1)\n",
      "Pair{AbstractString,Int64}(\"pyridinium 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"Also ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"though cFos-tTA\", 1)\n",
      "Pair{AbstractString,Int64}(\"during contextual fear\", 2)\n",
      "Pair{AbstractString,Int64}(\"of defense against\", 1)\n",
      "Pair{AbstractString,Int64}(\". The attenuated\", 1)\n",
      "Pair{AbstractString,Int64}(\"( AZC\", 1)\n",
      "Pair{AbstractString,Int64}(\"in HAE modulates\", 1)\n",
      "Pair{AbstractString,Int64}(\"suppressed viral\", 1)\n",
      "Pair{AbstractString,Int64}(\", greatly\", 1)\n",
      "Pair{AbstractString,Int64}(\", SIGLEC\", 1)\n",
      "Pair{AbstractString,Int64}(\"true ,\", 1)\n",
      "Pair{AbstractString,Int64}(\": Mann-Whitney\", 1)\n",
      "Pair{AbstractString,Int64}(\"dose-dependent manner\", 1)\n",
      "Pair{AbstractString,Int64}(\"5.8 µM in\", 1)\n",
      "Pair{AbstractString,Int64}(\"the lack\", 3)\n",
      "Pair{AbstractString,Int64}(\"in MHV-ExoN (\", 1)\n",
      "Pair{AbstractString,Int64}(\"hindered from\", 1)\n",
      "Pair{AbstractString,Int64}(\"the bulk transduced\", 1)\n",
      "Pair{AbstractString,Int64}(\"degraded ; however\", 1)\n",
      "Pair{AbstractString,Int64}(\"-ggTCGTTTAGTTGAGAAAAGT\", 1)\n",
      "Pair{AbstractString,Int64}(\"HIV- 1 infection\", 1)\n",
      "Pair{AbstractString,Int64}(\"induction\", 4)\n",
      "Pair{AbstractString,Int64}(\"mediocre\", 3)\n",
      "Pair{AbstractString,Int64}(\"a pre-dissolved\", 1)\n",
      "Pair{AbstractString,Int64}(\"the author/funder. .\", 25)\n",
      "Pair{AbstractString,Int64}(\"and n=\", 2)\n",
      "Pair{AbstractString,Int64}(\"remdesivir\", 1)\n",
      "Pair{AbstractString,Int64}(\"inactivation was\", 1)\n",
      "Pair{AbstractString,Int64}(\"addition , ciclesonide\", 1)\n",
      "Pair{AbstractString,Int64}(\"betacitronellol ) (\", 1)\n",
      "Pair{AbstractString,Int64}(\"r\", 20)\n",
      "Pair{AbstractString,Int64}(\", UK\", 2)\n",
      "Pair{AbstractString,Int64}(\"processes\", 2)\n",
      "Pair{AbstractString,Int64}(\") , at\", 1)\n",
      "Pair{AbstractString,Int64}(\"mutants in alphacoronaviruses\", 1)\n",
      "Pair{AbstractString,Int64}(\"reduced adaptive capacity\", 1)\n",
      "Pair{AbstractString,Int64}(\"thick\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 4,5 -dimethylthiazol-\", 1)\n",
      "Pair{AbstractString,Int64}(\"complexes. A.\", 1)\n",
      "Pair{AbstractString,Int64}(\"bioRxiv\", 127)\n",
      "Pair{AbstractString,Int64}(\"Silencing these\", 1)\n",
      "Pair{AbstractString,Int64}(\"for storing associative\", 1)\n",
      "Pair{AbstractString,Int64}(\"progress\", 10)\n",
      "Pair{AbstractString,Int64}(\"p= 0.0073 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"al. 46 described\", 1)\n",
      "Pair{AbstractString,Int64}(\"7 8\", 4)\n",
      "Pair{AbstractString,Int64}(\"or release\", 1)\n",
      "Pair{AbstractString,Int64}(\"a-b )\", 3)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"293\", 4)\n",
      "Pair{AbstractString,Int64}(\"d , j\", 2)\n",
      "Pair{AbstractString,Int64}(\"their sniff\", 1)\n",
      "Pair{AbstractString,Int64}(\"screened from\", 1)\n",
      "Pair{AbstractString,Int64}(\"-methyl 2 -amino-\", 1)\n",
      "Pair{AbstractString,Int64}(\"the 2019 -nCoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"h ) or\", 1)\n",
      "Pair{AbstractString,Int64}(\"replication , fitness\", 1)\n",
      "Pair{AbstractString,Int64}(\"the addition of\", 2)\n",
      "Pair{AbstractString,Int64}(\"be cleaved\", 1)\n",
      "Pair{AbstractString,Int64}(\"derived from the\", 1)\n",
      "Pair{AbstractString,Int64}(\"overcame the\", 1)\n",
      "Pair{AbstractString,Int64}(\"If it\", 1)\n",
      "Pair{AbstractString,Int64}(\"totally inactive\", 1)\n",
      "Pair{AbstractString,Int64}(\"CDCl 3 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"polyprotein , the\", 1)\n",
      "Pair{AbstractString,Int64}(\"cytopathic effects\", 1)\n",
      "Pair{AbstractString,Int64}(\"μ M mometasone.\", 1)\n",
      "Pair{AbstractString,Int64}(\"related coronaviruses\", 2)\n",
      "Pair{AbstractString,Int64}(\"and COPD. 17\", 1)\n",
      "Pair{AbstractString,Int64}(\"4 /\", 2)\n",
      "Pair{AbstractString,Int64}(\"learning activates sparse\", 1)\n",
      "Pair{AbstractString,Int64}(\"S ident (\", 1)\n",
      "Pair{AbstractString,Int64}(\"in dimethyl sulfoxide\", 2)\n",
      "Pair{AbstractString,Int64}(\"P 2 side-chain\", 1)\n",
      "Pair{AbstractString,Int64}(\"( time spent\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.05 was\", 1)\n",
      "Pair{AbstractString,Int64}(\"no approved\", 1)\n",
      "Pair{AbstractString,Int64}(\"suggesting it\", 1)\n",
      "Pair{AbstractString,Int64}(\"a cannot\", 1)\n",
      "Pair{AbstractString,Int64}(\"binding domain\", 1)\n",
      "Pair{AbstractString,Int64}(\"Vero cells (\", 2)\n",
      "Pair{AbstractString,Int64}(\"were selected using\", 1)\n",
      "Pair{AbstractString,Int64}(\"freely moving mice\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.6 mA )\", 1)\n",
      "Pair{AbstractString,Int64}(\"DECTRIS )\", 1)\n",
      "Pair{AbstractString,Int64}(\"large fraction of\", 1)\n",
      "Pair{AbstractString,Int64}(\"binds above\", 1)\n",
      "Pair{AbstractString,Int64}(\"velocity after\", 1)\n",
      "Pair{AbstractString,Int64}(\"N-oxide\", 1)\n",
      "Pair{AbstractString,Int64}(\"developed will not\", 1)\n",
      "Pair{AbstractString,Int64}(\"multiplying\", 1)\n",
      "Pair{AbstractString,Int64}(\"absolute\", 3)\n",
      "Pair{AbstractString,Int64}(\"SERPINE\", 1)\n",
      "Pair{AbstractString,Int64}(\"substitution between SARS-CoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"stable during recovery\", 1)\n",
      "Pair{AbstractString,Int64}(\"≤\", 1)\n",
      "Pair{AbstractString,Int64}(\", hippocampal\", 1)\n",
      "Pair{AbstractString,Int64}(\"steady\", 1)\n",
      "Pair{AbstractString,Int64}(\"bioRxiv preprint Procedures\", 1)\n",
      "Pair{AbstractString,Int64}(\", total 98\", 1)\n",
      "Pair{AbstractString,Int64}(\"Met 165\", 4)\n",
      "Pair{AbstractString,Int64}(\"-RSAD 2 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\") can skew\", 1)\n",
      "Pair{AbstractString,Int64}(\"s ( 0.6\", 1)\n",
      "Pair{AbstractString,Int64}(\"nucleophilic attack of\", 1)\n",
      "Pair{AbstractString,Int64}(\"atom of Asp\", 2)\n",
      "Pair{AbstractString,Int64}(\", 6 x\", 1)\n",
      "Pair{AbstractString,Int64}(\"GO categories were\", 1)\n",
      "Pair{AbstractString,Int64}(\"cell-cell fusion\", 2)\n",
      "Pair{AbstractString,Int64}(\"worldwide. Compared\", 1)\n",
      "Pair{AbstractString,Int64}(\"preprint this\", 1)\n",
      "Pair{AbstractString,Int64}(\"Experimental Procedures\", 5)\n",
      "Pair{AbstractString,Int64}(\"in freely\", 1)\n",
      "Pair{AbstractString,Int64}(\"increased T\", 1)\n",
      "Pair{AbstractString,Int64}(\"selection. These\", 1)\n",
      "Pair{AbstractString,Int64}(\"copy\", 4)\n",
      "Pair{AbstractString,Int64}(\"addition with isocyanides\", 1)\n",
      "Pair{AbstractString,Int64}(\"prediction\", 3)\n",
      "Pair{AbstractString,Int64}(\"omitted for\", 6)\n",
      "Pair{AbstractString,Int64}(\"* p< 0.01\", 4)\n",
      "Pair{AbstractString,Int64}(\"2 - 3\", 2)\n",
      "Pair{AbstractString,Int64}(\"entry inhibitors\", 1)\n",
      "Pair{AbstractString,Int64}(\"http://codonw.sourceforge.net/ )\", 1)\n",
      "Pair{AbstractString,Int64}(\"Medicine [ 4\", 1)\n",
      "Pair{AbstractString,Int64}(\"_ 031439745.1 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"TMPRSS\", 16)\n",
      "Pair{AbstractString,Int64}(\"the intermediate revertants\", 1)\n",
      "Pair{AbstractString,Int64}(\"to complete recovery\", 1)\n",
      "Pair{AbstractString,Int64}(\"no effect (\", 1)\n",
      "Pair{AbstractString,Int64}(\"overlapping ensembles\", 1)\n",
      "Pair{AbstractString,Int64}(\"yields. HCoV-HKU\", 1)\n",
      "Pair{AbstractString,Int64}(\"to form clade\", 1)\n",
      "Pair{AbstractString,Int64}(\"determinants\", 1)\n",
      "Pair{AbstractString,Int64}(\"in the envelope.\", 1)\n",
      "Pair{AbstractString,Int64}(\"cats ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"and RTqPCR assessment\", 1)\n",
      "Pair{AbstractString,Int64}(\"and subjected\", 2)\n",
      "Pair{AbstractString,Int64}(\"but no\", 2)\n",
      "Pair{AbstractString,Int64}(\"samples were transferred\", 1)\n",
      "Pair{AbstractString,Int64}(\"determined by subtracting\", 1)\n",
      "Pair{AbstractString,Int64}(\"s centroid in\", 1)\n",
      "Pair{AbstractString,Int64}(\"i.p. injected with\", 2)\n",
      "Pair{AbstractString,Int64}(\"CT ( 31.43\", 1)\n",
      "Pair{AbstractString,Int64}(\"to remove debris\", 1)\n",
      "Pair{AbstractString,Int64}(\"categories were\", 1)\n",
      "Pair{AbstractString,Int64}(\"E expressing or\", 6)\n",
      "Pair{AbstractString,Int64}(\"amides of Gly\", 1)\n",
      "Pair{AbstractString,Int64}(\"gctggcatcgggaacgtggtggactttggctac accctgaacaagggctgctccccgatctgtccc ggcccgagcgtcaatcttggagtggcgtccgt\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.3698 ; g\", 1)\n",
      "Pair{AbstractString,Int64}(\"of viruses containing\", 1)\n",
      "Pair{AbstractString,Int64}(\", while no\", 1)\n",
      "Pair{AbstractString,Int64}(\"S bands of\", 1)\n",
      "Pair{AbstractString,Int64}(\"into the pVSV\", 1)\n",
      "Pair{AbstractString,Int64}(\"enzyme. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.1 M sodium\", 3)\n",
      "Pair{AbstractString,Int64}(\"v 8.1.21\", 1)\n",
      "Pair{AbstractString,Int64}(\"also reported that\", 1)\n",
      "Pair{AbstractString,Int64}(\"17 The split\", 1)\n",
      "Pair{AbstractString,Int64}(\"performed one day\", 1)\n",
      "Pair{AbstractString,Int64}(\", were calculated\", 1)\n",
      "Pair{AbstractString,Int64}(\"male and female\", 1)\n",
      "Pair{AbstractString,Int64}(\"Scientific )\", 7)\n",
      "Pair{AbstractString,Int64}(\". Mice were\", 1)\n",
      "Pair{AbstractString,Int64}(\"transcriptomic approach\", 1)\n",
      "Pair{AbstractString,Int64}(\"-mediated cleavage. 246\", 1)\n",
      "Pair{AbstractString,Int64}(\"p-s/DMEM ( cDMEM\", 1)\n",
      "Pair{AbstractString,Int64}(\"E , G\", 2)\n",
      "Pair{AbstractString,Int64}(\"0.81\", 1)\n",
      "Pair{AbstractString,Int64}(\"Nucleospin RNA\", 1)\n",
      "Pair{AbstractString,Int64}(\". Sections of\", 1)\n",
      "Pair{AbstractString,Int64}(\"to visualize\", 2)\n",
      "Pair{AbstractString,Int64}(\", our data\", 3)\n",
      "Pair{AbstractString,Int64}(\"0.06\", 1)\n",
      "Pair{AbstractString,Int64}(\"an effect that\", 1)\n",
      "Pair{AbstractString,Int64}(\"home cage\", 2)\n",
      "Pair{AbstractString,Int64}(\"for that\", 1)\n",
      "Pair{AbstractString,Int64}(\"sets\", 3)\n",
      "Pair{AbstractString,Int64}(\"Di ( experimental\", 1)\n",
      "Pair{AbstractString,Int64}(\"or LC\", 1)\n",
      "Pair{AbstractString,Int64}(\"centrifuged at 13,000\", 1)\n",
      "Pair{AbstractString,Int64}(\"we replaced the\", 2)\n",
      "Pair{AbstractString,Int64}(\"WIV 04\", 21)\n",
      "Pair{AbstractString,Int64}(\"were characterized\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 15\", 2)\n",
      "Pair{AbstractString,Int64}(\"0.0066 ; d\", 1)\n",
      "Pair{AbstractString,Int64}(\"gave rise to\", 2)\n",
      "Pair{AbstractString,Int64}(\"163 HAE _\", 1)\n",
      "Pair{AbstractString,Int64}(\"pocket and thus\", 1)\n",
      "Pair{AbstractString,Int64}(\"receptor for HCoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"W 17\", 1)\n",
      "Pair{AbstractString,Int64}(\"make it\", 1)\n",
      "Pair{AbstractString,Int64}(\"in 2003\", 2)\n",
      "Pair{AbstractString,Int64}(\"Mandairon et al.\", 1)\n",
      "Pair{AbstractString,Int64}(\"282\", 1)\n",
      "Pair{AbstractString,Int64}(\"follow that\", 1)\n",
      "Pair{AbstractString,Int64}(\"NH of\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 strain Nancy.\", 1)\n",
      "Pair{AbstractString,Int64}(\"number of mice\", 1)\n",
      "Pair{AbstractString,Int64}(\"PBS-injected\", 1)\n",
      "Pair{AbstractString,Int64}(\"Chemical shifts\", 1)\n",
      "Pair{AbstractString,Int64}(\"completely abolished in\", 1)\n",
      "Pair{AbstractString,Int64}(\"49 and\", 1)\n",
      "Pair{AbstractString,Int64}(\", two-tailed unpaired\", 1)\n",
      "Pair{AbstractString,Int64}(\"and HRV\", 3)\n",
      "Pair{AbstractString,Int64}(\"viral sg\", 1)\n",
      "Pair{AbstractString,Int64}(\"it is obvious\", 1)\n",
      "Pair{AbstractString,Int64}(\"box. Two control\", 1)\n",
      "Pair{AbstractString,Int64}(\"are mostly\", 1)\n",
      "Pair{AbstractString,Int64}(\"mice at\", 2)\n",
      "Pair{AbstractString,Int64}(\"increased sniff frequency\", 1)\n",
      "Pair{AbstractString,Int64}(\"will have to\", 1)\n",
      "Pair{AbstractString,Int64}(\"MOI= 1 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"enzyme. As\", 1)\n",
      "Pair{AbstractString,Int64}(\"14 sites caused\", 1)\n",
      "Pair{AbstractString,Int64}(\"decided\", 4)\n",
      "Pair{AbstractString,Int64}(\"although the integrity\", 1)\n",
      "Pair{AbstractString,Int64}(\"* <\", 1)\n",
      "Pair{AbstractString,Int64}(\"cortical area receiving\", 1)\n",
      "Pair{AbstractString,Int64}(\"11.8 µM in\", 1)\n",
      "Pair{AbstractString,Int64}(\"µM. The CC\", 1)\n",
      "Pair{AbstractString,Int64}(\"preprint Data generated\", 1)\n",
      "Pair{AbstractString,Int64}(\"poses to\", 1)\n",
      "Pair{AbstractString,Int64}(\"and 285\", 1)\n",
      "Pair{AbstractString,Int64}(\"generated RD cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"complex structure\", 1)\n",
      "Pair{AbstractString,Int64}(\"temporal limitations\", 1)\n",
      "Pair{AbstractString,Int64}(\"VSV )\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibitors of the\", 3)\n",
      "Pair{AbstractString,Int64}(\"at A 25\", 1)\n",
      "Pair{AbstractString,Int64}(\"diluted in media\", 1)\n",
      "Pair{AbstractString,Int64}(\"clade 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"serum ALT\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"The rapid\", 1)\n",
      "Pair{AbstractString,Int64}(\"the enteroviral 3\", 6)\n",
      "Pair{AbstractString,Int64}(\"- 84\", 2)\n",
      "Pair{AbstractString,Int64}(\"R 503 after\", 1)\n",
      "Pair{AbstractString,Int64}(\"ester group\", 1)\n",
      "Pair{AbstractString,Int64}(\"artificial reactivation of\", 2)\n",
      "Pair{AbstractString,Int64}(\") revealed\", 1)\n",
      "Pair{AbstractString,Int64}(\"the coronavirus main\", 1)\n",
      "Pair{AbstractString,Int64}(\"or beta-citronellol )\", 1)\n",
      "Pair{AbstractString,Int64}(\"and the difference\", 1)\n",
      "Pair{AbstractString,Int64}(\"portion\", 1)\n",
      "Pair{AbstractString,Int64}(\"surface pathway\", 1)\n",
      "Pair{AbstractString,Int64}(\"TMPRSS 2 is\", 1)\n",
      "Pair{AbstractString,Int64}(\"Test\", 9)\n",
      "Pair{AbstractString,Int64}(\"Furthermore , we\", 4)\n",
      "Pair{AbstractString,Int64}(\"was found\", 2)\n",
      "Pair{AbstractString,Int64}(\"in livers and\", 1)\n",
      "Pair{AbstractString,Int64}(\"complete viral genomes\", 1)\n",
      "Pair{AbstractString,Int64}(\"and Coxsackievirus\", 1)\n",
      "Pair{AbstractString,Int64}(\"20 ℃.\", 1)\n",
      "Pair{AbstractString,Int64}(\"- 90 %\", 1)\n",
      "Pair{AbstractString,Int64}(\"' -and 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"2011 )\", 5)\n",
      "Pair{AbstractString,Int64}(\"fit into the\", 1)\n",
      "Pair{AbstractString,Int64}(\"microtiter plate. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"extensively for\", 1)\n",
      "Pair{AbstractString,Int64}(\"out to\", 1)\n",
      "Pair{AbstractString,Int64}(\"infectivity was calculated\", 1)\n",
      "Pair{AbstractString,Int64}(\"( having\", 1)\n",
      "Pair{AbstractString,Int64}(\"ExoN function. Given\", 1)\n",
      "Pair{AbstractString,Int64}(\"CS-presentation. ( F\", 1)\n",
      "Pair{AbstractString,Int64}(\"were regularly\", 1)\n",
      "Pair{AbstractString,Int64}(\", where IC\", 1)\n",
      "Pair{AbstractString,Int64}(\"space that\", 1)\n",
      "Pair{AbstractString,Int64}(\"cyclopropylmethyl\", 6)\n",
      "Pair{AbstractString,Int64}(\"the total inactivity\", 2)\n",
      "Pair{AbstractString,Int64}(\"of Bat-SARr-CoV RaTG\", 1)\n",
      "Pair{AbstractString,Int64}(\"from HCoV- 229\", 1)\n",
      "Pair{AbstractString,Int64}(\"solution and stored\", 1)\n",
      "Pair{AbstractString,Int64}(\"N ) genes\", 1)\n",
      "Pair{AbstractString,Int64}(\"DOX ) removal\", 1)\n",
      "Pair{AbstractString,Int64}(\"and 28 S\", 1)\n",
      "Pair{AbstractString,Int64}(\"13 than that\", 2)\n",
      "Pair{AbstractString,Int64}(\"after high\", 1)\n",
      "Pair{AbstractString,Int64}(\"a bootstrap analysis\", 1)\n",
      "Pair{AbstractString,Int64}(\"as the N-cap\", 1)\n",
      "Pair{AbstractString,Int64}(\", - 72.56\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells and SARS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"helicase/NTPase ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"Luciferase Assay System\", 4)\n",
      "Pair{AbstractString,Int64}(\"In HCoV-NL 63\", 1)\n",
      "Pair{AbstractString,Int64}(\"of the mutation\", 1)\n",
      "Pair{AbstractString,Int64}(\"as Z-stacks\", 1)\n",
      "Pair{AbstractString,Int64}(\"P values\", 4)\n",
      "Pair{AbstractString,Int64}(\"can thus make\", 1)\n",
      "Pair{AbstractString,Int64}(\"treatment will also\", 1)\n",
      "Pair{AbstractString,Int64}(\"554 author/funder.\", 1)\n",
      "Pair{AbstractString,Int64}(\"1.00 g ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"floor. In\", 1)\n",
      "Pair{AbstractString,Int64}(\"protease , rather\", 1)\n",
      "Pair{AbstractString,Int64}(\"while in\", 1)\n",
      "Pair{AbstractString,Int64}(\"solution also\", 1)\n",
      "Pair{AbstractString,Int64}(\"s choice and\", 1)\n",
      "Pair{AbstractString,Int64}(\"published elsewhere )\", 1)\n",
      "Pair{AbstractString,Int64}(\"custom-written ImageJ plugin.\", 1)\n",
      "Pair{AbstractString,Int64}(\"from the template\", 1)\n",
      "Pair{AbstractString,Int64}(\"Sigma-Aldrich ) and\", 1)\n",
      "Pair{AbstractString,Int64}(\"on the\", 24)\n",
      "Pair{AbstractString,Int64}(\"inhibitors in\", 1)\n",
      "Pair{AbstractString,Int64}(\"structure-based design\", 1)\n",
      "Pair{AbstractString,Int64}(\"red\", 5)\n",
      "Pair{AbstractString,Int64}(\"mostly purifying selection\", 1)\n",
      "Pair{AbstractString,Int64}(\"evolve in a\", 1)\n",
      "Pair{AbstractString,Int64}(\") . Such\", 1)\n",
      "Pair{AbstractString,Int64}(\"neurons detected in\", 1)\n",
      "Pair{AbstractString,Int64}(\"Seventeen\", 1)\n",
      "Pair{AbstractString,Int64}(\"6 ( General\", 1)\n",
      "Pair{AbstractString,Int64}(\"detectable virus\", 1)\n",
      "Pair{AbstractString,Int64}(\"th position of\", 1)\n",
      "Pair{AbstractString,Int64}(\"conditioned stimulus\", 3)\n",
      "Pair{AbstractString,Int64}(\"by adding FRET\", 1)\n",
      "Pair{AbstractString,Int64}(\"compounds that we\", 1)\n",
      "Pair{AbstractString,Int64}(\"temperature inactivation damage.\", 1)\n",
      "Pair{AbstractString,Int64}(\"O. The reaction\", 1)\n",
      "Pair{AbstractString,Int64}(\"silica gel\", 3)\n",
      "Pair{AbstractString,Int64}(\"harvested and\", 4)\n",
      "Pair{AbstractString,Int64}(\"be\", 66)\n",
      "Pair{AbstractString,Int64}(\"a treatment\", 1)\n",
      "Pair{AbstractString,Int64}(\". f-g ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"virus ( having\", 1)\n",
      "Pair{AbstractString,Int64}(\"was moderately\", 1)\n",
      "Pair{AbstractString,Int64}(\"( as representative\", 1)\n",
      "Pair{AbstractString,Int64}(\"information associated\", 1)\n",
      "Pair{AbstractString,Int64}(\"mice ( Bolding\", 1)\n",
      "Pair{AbstractString,Int64}(\"E-Rev\", 1)\n",
      "Pair{AbstractString,Int64}(\"at the C\", 1)\n",
      "Pair{AbstractString,Int64}(\"Normalized data can\", 1)\n",
      "Pair{AbstractString,Int64}(\"washed 3 x\", 1)\n",
      "Pair{AbstractString,Int64}(\"site to\", 1)\n",
      "Pair{AbstractString,Int64}(\"neurons alters their\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells was 1.8\", 1)\n",
      "Pair{AbstractString,Int64}(\") Olfactory memory\", 1)\n",
      "Pair{AbstractString,Int64}(\"or 11\", 2)\n",
      "Pair{AbstractString,Int64}(\"human SARS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"34 and\", 1)\n",
      "Pair{AbstractString,Int64}(\"200 and P\", 1)\n",
      "Pair{AbstractString,Int64}(\"a , by\", 1)\n",
      "Pair{AbstractString,Int64}(\"cell counts for\", 1)\n",
      "Pair{AbstractString,Int64}(\"of N\", 1)\n",
      "Pair{AbstractString,Int64}(\"; thus\", 1)\n",
      "Pair{AbstractString,Int64}(\"VSVpp\", 1)\n",
      "Pair{AbstractString,Int64}(\"to act in\", 1)\n",
      "Pair{AbstractString,Int64}(\"of memory formation\", 1)\n",
      "Pair{AbstractString,Int64}(\"viral genomes\", 1)\n",
      "Pair{AbstractString,Int64}(\"All work\", 1)\n",
      "Pair{AbstractString,Int64}(\"At 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"Mutations\", 1)\n",
      "Pair{AbstractString,Int64}(\"proteins. The measured\", 1)\n",
      "Pair{AbstractString,Int64}(\"glutamine. 29 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"using GraphPad\", 2)\n",
      "Pair{AbstractString,Int64}(\". Slides were\", 1)\n",
      "Pair{AbstractString,Int64}(\"after i.p. injection\", 1)\n",
      "Pair{AbstractString,Int64}(\"had similar\", 1)\n",
      "Pair{AbstractString,Int64}(\"small. Although we\", 1)\n",
      "Pair{AbstractString,Int64}(\". Detailed\", 1)\n",
      "Pair{AbstractString,Int64}(\"were determined\", 5)\n",
      "Pair{AbstractString,Int64}(\"Start II High\", 1)\n",
      "Pair{AbstractString,Int64}(\") of our\", 1)\n",
      "Pair{AbstractString,Int64}(\". First ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"to take\", 1)\n",
      "Pair{AbstractString,Int64}(\"IHIC by\", 1)\n",
      "Pair{AbstractString,Int64}(\"its slow temporal\", 1)\n",
      "Pair{AbstractString,Int64}(\"period of\", 1)\n",
      "Pair{AbstractString,Int64}(\"Di vector.\", 1)\n",
      "Pair{AbstractString,Int64}(\"and equilibration\", 1)\n",
      "Pair{AbstractString,Int64}(\"; data 219\", 1)\n",
      "Pair{AbstractString,Int64}(\"encode an\", 1)\n",
      "Pair{AbstractString,Int64}(\"KO A 549\", 1)\n",
      "Pair{AbstractString,Int64}(\"above the action\", 1)\n",
      "Pair{AbstractString,Int64}(\"Urbani strain\", 1)\n",
      "Pair{AbstractString,Int64}(\"8.5\", 4)\n",
      "Pair{AbstractString,Int64}(\"has strong\", 1)\n",
      "Pair{AbstractString,Int64}(\"or control Huh\", 3)\n",
      "Pair{AbstractString,Int64}(\"M detector\", 1)\n",
      "Pair{AbstractString,Int64}(\"bromoacetonitrile.\", 1)\n",
      "Pair{AbstractString,Int64}(\"45 and\", 2)\n",
      "Pair{AbstractString,Int64}(\"tested whether\", 1)\n",
      "Pair{AbstractString,Int64}(\"and found small\", 1)\n",
      "Pair{AbstractString,Int64}(\"NL 63\", 4)\n",
      "Pair{AbstractString,Int64}(\"to report the\", 1)\n",
      "Pair{AbstractString,Int64}(\"maps displaying significant\", 1)\n",
      "Pair{AbstractString,Int64}(\"Coxsackivirus\", 1)\n",
      "Pair{AbstractString,Int64}(\"the resting\", 2)\n",
      "Pair{AbstractString,Int64}(\"passages\", 3)\n",
      "Pair{AbstractString,Int64}(\"single interaction\", 1)\n",
      "Pair{AbstractString,Int64}(\"myocardic inflammation\", 1)\n",
      "Pair{AbstractString,Int64}(\"described 425\", 1)\n",
      "Pair{AbstractString,Int64}(\"The series\", 1)\n",
      "Pair{AbstractString,Int64}(\"GIST C\", 1)\n",
      "Pair{AbstractString,Int64}(\"G ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"strains to form\", 1)\n",
      "Pair{AbstractString,Int64}(\": Glu )\", 1)\n",
      "Pair{AbstractString,Int64}(\"available on\", 2)\n",
      "Pair{AbstractString,Int64}(\"angle\", 1)\n",
      "Pair{AbstractString,Int64}(\"167 abolished\", 1)\n",
      "Pair{AbstractString,Int64}(\"of RS\", 1)\n",
      "Pair{AbstractString,Int64}(\"comparing SARS-CoV- 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"at discovering\", 1)\n",
      "Pair{AbstractString,Int64}(\"plasmid (\", 1)\n",
      "Pair{AbstractString,Int64}(\"constitutively\", 2)\n",
      "Pair{AbstractString,Int64}(\"prepare CVB 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"transfection reagent\", 1)\n",
      "Pair{AbstractString,Int64}(\"as we\", 1)\n",
      "Pair{AbstractString,Int64}(\"dimethyl ester\", 2)\n",
      "Pair{AbstractString,Int64}(\"permissiveness to\", 1)\n",
      "Pair{AbstractString,Int64}(\"of this result\", 1)\n",
      "Pair{AbstractString,Int64}(\"c-e , i-k\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"The MERS-CoV S\", 1)\n",
      "Pair{AbstractString,Int64}(\"of the fluorinated\", 1)\n",
      "Pair{AbstractString,Int64}(\"Ltd. )\", 1)\n",
      "Pair{AbstractString,Int64}(\"fully enter the\", 2)\n",
      "Pair{AbstractString,Int64}(\"floor )\", 2)\n",
      "Pair{AbstractString,Int64}(\"effects were analyzed\", 1)\n",
      "Pair{AbstractString,Int64}(\") , 3.76\", 1)\n",
      "Pair{AbstractString,Int64}(\"leucine ) ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"with the activating\", 1)\n",
      "Pair{AbstractString,Int64}(\"are available within\", 1)\n",
      "Pair{AbstractString,Int64}(\"The side-chain of\", 1)\n",
      "Pair{AbstractString,Int64}(\", CoV ExoNs\", 1)\n",
      "Pair{AbstractString,Int64}(\"incubated\", 29)\n",
      "Pair{AbstractString,Int64}(\": G\", 3)\n",
      "Pair{AbstractString,Int64}(\"were additionally\", 1)\n",
      "Pair{AbstractString,Int64}(\"connections (\", 1)\n",
      "Pair{AbstractString,Int64}(\"aldehyde and its\", 1)\n",
      "Pair{AbstractString,Int64}(\"The excitability of\", 1)\n",
      "Pair{AbstractString,Int64}(\"( http://codonw.sourceforge.net/ )\", 1)\n",
      "Pair{AbstractString,Int64}(\"in the sniff\", 1)\n",
      "Pair{AbstractString,Int64}(\"one side\", 1)\n",
      "Pair{AbstractString,Int64}(\"5 B\", 4)\n",
      "Pair{AbstractString,Int64}(\"assessed\", 6)\n",
      "Pair{AbstractString,Int64}(\"food\", 1)\n",
      "Pair{AbstractString,Int64}(\"the ExoN (\", 1)\n",
      "Pair{AbstractString,Int64}(\"and probably also\", 1)\n",
      "Pair{AbstractString,Int64}(\"g/kg\", 1)\n",
      "Pair{AbstractString,Int64}(\"replication-competent replicon and\", 1)\n",
      "Pair{AbstractString,Int64}(\"for treatment\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.05 .\", 1)\n",
      "Pair{AbstractString,Int64}(\"purified SARS-CoV M\", 1)\n",
      "Pair{AbstractString,Int64}(\". Residues from\", 1)\n",
      "Pair{AbstractString,Int64}(\"DMSO. After\", 1)\n",
      "Pair{AbstractString,Int64}(\"×g\", 2)\n",
      "Pair{AbstractString,Int64}(\"facility\", 2)\n",
      "Pair{AbstractString,Int64}(\"low cytotoxicity and\", 1)\n",
      "Pair{AbstractString,Int64}(\", were observed\", 1)\n",
      "Pair{AbstractString,Int64}(\"and Bat-SARSr-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\", the solvent\", 1)\n",
      "Pair{AbstractString,Int64}(\"xylazine\", 1)\n",
      "Pair{AbstractString,Int64}(\", pH 6.0\", 5)\n",
      "Pair{AbstractString,Int64}(\"C virus was\", 1)\n",
      "Pair{AbstractString,Int64}(\"OD 498 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"WT-ExoN- 132 DE\", 1)\n",
      "Pair{AbstractString,Int64}(\"Reverse ( 10\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Millipore\", 1)\n",
      "Pair{AbstractString,Int64}(\"22 Yet\", 1)\n",
      "Pair{AbstractString,Int64}(\"by 500\", 1)\n",
      "Pair{AbstractString,Int64}(\", 281\", 1)\n",
      "Pair{AbstractString,Int64}(\"CT\", 11)\n",
      "Pair{AbstractString,Int64}(\"by microtiter\", 1)\n",
      "Pair{AbstractString,Int64}(\"sectioned at 5\", 1)\n",
      "Pair{AbstractString,Int64}(\"shared by\", 2)\n",
      "Pair{AbstractString,Int64}(\"61 ,\", 3)\n",
      "Pair{AbstractString,Int64}(\"referred\", 1)\n",
      "Pair{AbstractString,Int64}(\"50 = 0.0048\", 1)\n",
      "Pair{AbstractString,Int64}(\"transduced with a\", 1)\n",
      "Pair{AbstractString,Int64}(\"\\\" summer flu\", 1)\n",
      "Pair{AbstractString,Int64}(\"TMPRSS 2 -expressing\", 2)\n",
      "Pair{AbstractString,Int64}(\"with foot-shock (\", 1)\n",
      "Pair{AbstractString,Int64}(\", IL\", 1)\n",
      "Pair{AbstractString,Int64}(\"show a bias\", 1)\n",
      "Pair{AbstractString,Int64}(\"PFU/cell , as\", 1)\n",
      "Pair{AbstractString,Int64}(\", 4 Mg-ATP\", 1)\n",
      "Pair{AbstractString,Int64}(\"originate from\", 1)\n",
      "Pair{AbstractString,Int64}(\"hour. Cells were\", 1)\n",
      "Pair{AbstractString,Int64}(\"viruses have\", 1)\n",
      "Pair{AbstractString,Int64}(\"injection of an\", 1)\n",
      "Pair{AbstractString,Int64}(\"The recording solution\", 1)\n",
      "Pair{AbstractString,Int64}(\"TMPRSS 2 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"to WT-ExoN- 132\", 1)\n",
      "Pair{AbstractString,Int64}(\"key factors affecting\", 1)\n",
      "Pair{AbstractString,Int64}(\") or inactivation\", 1)\n",
      "Pair{AbstractString,Int64}(\"active during odor-foot\", 1)\n",
      "Pair{AbstractString,Int64}(\"follows : 12.5\", 1)\n",
      "Pair{AbstractString,Int64}(\"CVB 3 and\", 1)\n",
      "Pair{AbstractString,Int64}(\"against passage\", 1)\n",
      "Pair{AbstractString,Int64}(\"c , *\", 5)\n",
      "Pair{AbstractString,Int64}(\"median speed of\", 1)\n",
      "Pair{AbstractString,Int64}(\"as measured\", 1)\n",
      "Pair{AbstractString,Int64}(\"delivered 4 seconds\", 1)\n",
      "Pair{AbstractString,Int64}(\"of virus replication\", 1)\n",
      "Pair{AbstractString,Int64}(\"cysteine residue onto\", 1)\n",
      "Pair{AbstractString,Int64}(\"Germany ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\"cycle of\", 1)\n",
      "Pair{AbstractString,Int64}(\"and did not\", 1)\n",
      "Pair{AbstractString,Int64}(\"MERS-CoV poses to\", 1)\n",
      "Pair{AbstractString,Int64}(\"The analysis showed\", 2)\n",
      "Pair{AbstractString,Int64}(\"Signed Rank Test\", 3)\n",
      "Pair{AbstractString,Int64}(\"µl ) was\", 1)\n",
      "Pair{AbstractString,Int64}(\"only modest\", 1)\n",
      "Pair{AbstractString,Int64}(\"Sidak ' s\", 2)\n",
      "Pair{AbstractString,Int64}(\"box as a\", 1)\n",
      "Pair{AbstractString,Int64}(\"enter TMPRSS 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"in CH 2\", 3)\n",
      "Pair{AbstractString,Int64}(\"3,822 nt )\", 1)\n",
      "Pair{AbstractString,Int64}(\"of viral nucleic\", 7)\n",
      "Pair{AbstractString,Int64}(\"E-CoV-Rluc (\", 1)\n",
      "Pair{AbstractString,Int64}(\"20 pA. We\", 1)\n",
      "Pair{AbstractString,Int64}(\"， these\", 1)\n",
      "Pair{AbstractString,Int64}(\"training ( Supplementary\", 1)\n",
      "Pair{AbstractString,Int64}(\"is in\", 3)\n",
      "Pair{AbstractString,Int64}(\"3 ) cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"insets )\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 hour\", 3)\n",
      "Pair{AbstractString,Int64}(\"labeled\", 4)\n",
      "Pair{AbstractString,Int64}(\"respectively , Mann-Whitney\", 1)\n",
      "Pair{AbstractString,Int64}(\"the replicon encodes\", 1)\n",
      "Pair{AbstractString,Int64}(\"data can be\", 1)\n",
      "Pair{AbstractString,Int64}(\"weak (\", 1)\n",
      "Pair{AbstractString,Int64}(\"factors that may\", 1)\n",
      "Pair{AbstractString,Int64}(\"-Boc group\", 2)\n",
      "Pair{AbstractString,Int64}(\", transduced with\", 1)\n",
      "Pair{AbstractString,Int64}(\"infection before\", 1)\n",
      "Pair{AbstractString,Int64}(\"that arose\", 1)\n",
      "Pair{AbstractString,Int64}(\"and long-term\", 1)\n",
      "Pair{AbstractString,Int64}(\"both DNA and\", 1)\n",
      "Pair{AbstractString,Int64}(\"of immune\", 2)\n",
      "Pair{AbstractString,Int64}(\"give product 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"20 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"people blind to\", 1)\n",
      "Pair{AbstractString,Int64}(\"' position\", 1)\n",
      "Pair{AbstractString,Int64}(\"then transferred into\", 1)\n",
      "Pair{AbstractString,Int64}(\"high-temperature inactivation\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.0234\", 1)\n",
      "Pair{AbstractString,Int64}(\"m/z\", 2)\n",
      "Pair{AbstractString,Int64}(\"market.\", 1)\n",
      "Pair{AbstractString,Int64}(\"at 0.2 M\", 1)\n",
      "Pair{AbstractString,Int64}(\", respectively. Copy\", 1)\n",
      "Pair{AbstractString,Int64}(\"then NaBH 4\", 1)\n",
      "Pair{AbstractString,Int64}(\"that Fos-tagging\", 1)\n",
      "Pair{AbstractString,Int64}(\"Synthesis of alcohols\", 1)\n",
      "Pair{AbstractString,Int64}(\"than 95\", 2)\n",
      "Pair{AbstractString,Int64}(\"agarose gel electrophoresis.\", 1)\n",
      "Pair{AbstractString,Int64}(\"4.3 µM\", 1)\n",
      "Pair{AbstractString,Int64}(\"of LY\", 5)\n",
      "Pair{AbstractString,Int64}(\"for SARS-CoV M\", 3)\n",
      "Pair{AbstractString,Int64}(\"important experimental\", 1)\n",
      "Pair{AbstractString,Int64}(\"CS+.\", 1)\n",
      "Pair{AbstractString,Int64}(\"region are increased\", 1)\n",
      "Pair{AbstractString,Int64}(\"infection (\", 7)\n",
      "Pair{AbstractString,Int64}(\"employing Spearman\", 1)\n",
      "Pair{AbstractString,Int64}(\"473 I )\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 to 5\", 1)\n",
      "Pair{AbstractString,Int64}(\"serious outcome ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"associated with tissue\", 1)\n",
      "Pair{AbstractString,Int64}(\"when comparing SARS-CoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 strain WIV\", 3)\n",
      "Pair{AbstractString,Int64}(\"solutions\", 7)\n",
      "Pair{AbstractString,Int64}(\"points. To determine\", 1)\n",
      "Pair{AbstractString,Int64}(\"for hM 4\", 1)\n",
      "Pair{AbstractString,Int64}(\"poured into\", 2)\n",
      "Pair{AbstractString,Int64}(\"will favor\", 1)\n",
      "Pair{AbstractString,Int64}(\"memory encoding\", 1)\n",
      "Pair{AbstractString,Int64}(\"but is dramatically\", 1)\n",
      "Pair{AbstractString,Int64}(\"observed that mice\", 1)\n",
      "Pair{AbstractString,Int64}(\"transfection. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"readout\", 2)\n",
      "Pair{AbstractString,Int64}(\"single\", 11)\n",
      "Pair{AbstractString,Int64}(\"the relatively\", 1)\n",
      "Pair{AbstractString,Int64}(\"Similarly , the\", 1)\n",
      "Pair{AbstractString,Int64}(\"a by reducing\", 1)\n",
      "Pair{AbstractString,Int64}(\"to error ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"variants in isolated\", 1)\n",
      "Pair{AbstractString,Int64}(\"of active\", 1)\n",
      "Pair{AbstractString,Int64}(\"first trained mice\", 1)\n",
      "Pair{AbstractString,Int64}(\"homodimer , with\", 1)\n",
      "Pair{AbstractString,Int64}(\"11 µM\", 1)\n",
      "Pair{AbstractString,Int64}(\"~ 5\", 1)\n",
      "Pair{AbstractString,Int64}(\"Aqueous One\", 1)\n",
      "Pair{AbstractString,Int64}(\"close to\", 2)\n",
      "Pair{AbstractString,Int64}(\"04 /RaTG 13\", 1)\n",
      "Pair{AbstractString,Int64}(\"submitted to GISAID\", 1)\n",
      "Pair{AbstractString,Int64}(\"current-clamp recordings\", 1)\n",
      "Pair{AbstractString,Int64}(\"was good\", 1)\n",
      "Pair{AbstractString,Int64}(\"quickly 259 away\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 /S\", 3)\n",
      "Pair{AbstractString,Int64}(\") when determining\", 1)\n",
      "Pair{AbstractString,Int64}(\"for reversion\", 1)\n",
      "Pair{AbstractString,Int64}(\"at least reducing\", 1)\n",
      "Pair{AbstractString,Int64}(\"' and P\", 2)\n",
      "Pair{AbstractString,Int64}(\"Center\", 4)\n",
      "Pair{AbstractString,Int64}(\"dN/dS ( ω\", 1)\n",
      "Pair{AbstractString,Int64}(\"pseudostratified cultures (\", 1)\n",
      "Pair{AbstractString,Int64}(\"piriform cortex Adeno-Associated\", 1)\n",
      "Pair{AbstractString,Int64}(\"-ExoN\", 3)\n",
      "Pair{AbstractString,Int64}(\"combining the limitations\", 1)\n",
      "Pair{AbstractString,Int64}(\"according to Chimera.\", 1)\n",
      "Pair{AbstractString,Int64}(\"penetrating\", 1)\n",
      "Pair{AbstractString,Int64}(\"Fig\", 3)\n",
      "Pair{AbstractString,Int64}(\"60 x\", 1)\n",
      "Pair{AbstractString,Int64}(\"these inhibitors revealed\", 1)\n",
      "Pair{AbstractString,Int64}(\"250 236\", 1)\n",
      "Pair{AbstractString,Int64}(\"assess memory\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.0066 ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"S 2 subunits\", 1)\n",
      "Pair{AbstractString,Int64}(\"strikingly , long-term\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"± 0.21 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"in PBST\", 1)\n",
      "Pair{AbstractString,Int64}(\"two\", 44)\n",
      "Pair{AbstractString,Int64}(\"persistent animal\", 1)\n",
      "Pair{AbstractString,Int64}(\"C for long\", 1)\n",
      "Pair{AbstractString,Int64}(\"CS+-tagged ensemble ?\", 1)\n",
      "Pair{AbstractString,Int64}(\"a goat\", 1)\n",
      "Pair{AbstractString,Int64}(\"and refinement. Diffraction\", 1)\n",
      "Pair{AbstractString,Int64}(\"conditioning has provided\", 1)\n",
      "Pair{AbstractString,Int64}(\"in addition to\", 1)\n",
      "Pair{AbstractString,Int64}(\"values > 20\", 1)\n",
      "Pair{AbstractString,Int64}(\"6.0 g ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"DE is 254\", 1)\n",
      "Pair{AbstractString,Int64}(\"Chinese Society of\", 1)\n",
      "Pair{AbstractString,Int64}(\", but poor\", 1)\n",
      "Pair{AbstractString,Int64}(\"following people for\", 1)\n",
      "Pair{AbstractString,Int64}(\"demonstrated to\", 1)\n",
      "Pair{AbstractString,Int64}(\"affected.\", 1)\n",
      "Pair{AbstractString,Int64}(\"millimeter (\", 1)\n",
      "Pair{AbstractString,Int64}(\"suspected\", 1)\n",
      "Pair{AbstractString,Int64}(\") . 5\", 1)\n",
      "Pair{AbstractString,Int64}(\"at 120\", 1)\n",
      "Pair{AbstractString,Int64}(\"and 480 ribavirin\", 1)\n",
      "Pair{AbstractString,Int64}(\"C- 150\", 1)\n",
      "Pair{AbstractString,Int64}(\"a mixture of\", 2)\n",
      "Pair{AbstractString,Int64}(\"( 0.70\", 1)\n",
      "Pair{AbstractString,Int64}(\"the supernatant\", 3)\n",
      "Pair{AbstractString,Int64}(\"The commercially\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 ZYD\", 2)\n",
      "Pair{AbstractString,Int64}(\"subsequently selected with\", 1)\n",
      "Pair{AbstractString,Int64}(\"as , in\", 1)\n",
      "Pair{AbstractString,Int64}(\"these mutants as\", 1)\n",
      "Pair{AbstractString,Int64}(\"bulk transduced population\", 1)\n",
      "Pair{AbstractString,Int64}(\"the literature as\", 1)\n",
      "Pair{AbstractString,Int64}(\"in maintenance\", 2)\n",
      "Pair{AbstractString,Int64}(\"a well-studied model\", 1)\n",
      "Pair{AbstractString,Int64}(\"( MOI= 5\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.72\", 1)\n",
      "Pair{AbstractString,Int64}(\"and 147\", 1)\n",
      "Pair{AbstractString,Int64}(\"and other associative\", 1)\n",
      "Pair{AbstractString,Int64}(\"( P <\", 3)\n",
      "Pair{AbstractString,Int64}(\"Odors were delivered\", 1)\n",
      "Pair{AbstractString,Int64}(\"them had\", 1)\n",
      "Pair{AbstractString,Int64}(\"be the 304\", 1)\n",
      "Pair{AbstractString,Int64}(\"lead to a\", 2)\n",
      "Pair{AbstractString,Int64}(\"in increased viral\", 1)\n",
      "Pair{AbstractString,Int64}(\"Wesson et al.\", 2)\n",
      "Pair{AbstractString,Int64}(\"-Å\", 1)\n",
      "Pair{AbstractString,Int64}(\"marrow derived macrophages\", 1)\n",
      "Pair{AbstractString,Int64}(\"thiohemiacetal -can\", 1)\n",
      "Pair{AbstractString,Int64}(\"MERS-and SARS-CoV even\", 1)\n",
      "Pair{AbstractString,Int64}(\"Given these results\", 1)\n",
      "Pair{AbstractString,Int64}(\"crystals were fished\", 1)\n",
      "Pair{AbstractString,Int64}(\"evolutionary hurdles\", 1)\n",
      "Pair{AbstractString,Int64}(\"if ExoN-AE\", 1)\n",
      "Pair{AbstractString,Int64}(\"shown ) .\", 2)\n",
      "Pair{AbstractString,Int64}(\"Middle-East\", 1)\n",
      "Pair{AbstractString,Int64}(\"checked\", 2)\n",
      "Pair{AbstractString,Int64}(\"detectable viral templates\", 2)\n",
      "Pair{AbstractString,Int64}(\"extract meaningful\", 1)\n",
      "Pair{AbstractString,Int64}(\"acid methyl\", 1)\n",
      "Pair{AbstractString,Int64}(\"the n-butyl\", 2)\n",
      "Pair{AbstractString,Int64}(\"down-regulated ( blue\", 1)\n",
      "Pair{AbstractString,Int64}(\"and labeled with\", 1)\n",
      "Pair{AbstractString,Int64}(\"expression of\", 10)\n",
      "Pair{AbstractString,Int64}(\"could 297\", 1)\n",
      "Pair{AbstractString,Int64}(\"accumulating seven\", 1)\n",
      "Pair{AbstractString,Int64}(\"catalyst. NaOAc\", 1)\n",
      "Pair{AbstractString,Int64}(\"180 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"GEO )\", 1)\n",
      "Pair{AbstractString,Int64}(\"yield was\", 1)\n",
      "Pair{AbstractString,Int64}(\"though\", 6)\n",
      "Pair{AbstractString,Int64}(\"unlikely that\", 2)\n",
      "Pair{AbstractString,Int64}(\"tests with the\", 1)\n",
      "Pair{AbstractString,Int64}(\"rapidly contained\", 1)\n",
      "Pair{AbstractString,Int64}(\"tissue culture and\", 1)\n",
      "Pair{AbstractString,Int64}(\"Calu 3 lung\", 1)\n",
      "Pair{AbstractString,Int64}(\"and . CC-BY-ND\", 1)\n",
      "Pair{AbstractString,Int64}(\"both dN and\", 2)\n",
      "Pair{AbstractString,Int64}(\"flow gating\", 1)\n",
      "Pair{AbstractString,Int64}(\"presentation (\", 4)\n",
      "Pair{AbstractString,Int64}(\"investigation on\", 1)\n",
      "Pair{AbstractString,Int64}(\"DREADD expression\", 4)\n",
      "Pair{AbstractString,Int64}(\"30 ,\", 3)\n",
      "Pair{AbstractString,Int64}(\"M +\", 2)\n",
      "Pair{AbstractString,Int64}(\"6 -carboxyfluorescein\", 1)\n",
      "Pair{AbstractString,Int64}(\"improvement in\", 1)\n",
      "Pair{AbstractString,Int64}(\"with 1 µL\", 2)\n",
      "Pair{AbstractString,Int64}(\"olfactory sensory\", 2)\n",
      "Pair{AbstractString,Int64}(\"from that\", 1)\n",
      "Pair{AbstractString,Int64}(\"synthetic\", 6)\n",
      "Pair{AbstractString,Int64}(\"virus. . CC-BY-ND\", 1)\n",
      "Pair{AbstractString,Int64}(\"samples preserved in\", 2)\n",
      "Pair{AbstractString,Int64}(\"136\", 1)\n",
      "Pair{AbstractString,Int64}(\"Coronaviruses (\", 1)\n",
      "Pair{AbstractString,Int64}(\"a -b ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"KLKs ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"MS spectra.\", 1)\n",
      "Pair{AbstractString,Int64}(\"are permissive\", 2)\n",
      "Pair{AbstractString,Int64}(\"neural networks\", 2)\n",
      "Pair{AbstractString,Int64}(\"are covered\", 1)\n",
      "Pair{AbstractString,Int64}(\"efficient HCoV-HKU 1\", 1)\n",
      "Pair{AbstractString,Int64}(\": mixed-effects\", 1)\n",
      "Pair{AbstractString,Int64}(\"that 288\", 1)\n",
      "Pair{AbstractString,Int64}(\"structure elucidation\", 1)\n",
      "Pair{AbstractString,Int64}(\"as described above\", 2)\n",
      "Pair{AbstractString,Int64}(\"also accepts P\", 1)\n",
      "Pair{AbstractString,Int64}(\"binding. We\", 1)\n",
      "Pair{AbstractString,Int64}(\"aldehydes (\", 1)\n",
      "Pair{AbstractString,Int64}(\"not reveal significant\", 1)\n",
      "Pair{AbstractString,Int64}(\"51 and\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 5 cells/well\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 10 8\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 33 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"III ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"9 10 11\", 4)\n",
      "Pair{AbstractString,Int64}(\"3 4 5\", 4)\n",
      "Pair{AbstractString,Int64}(\"controlling murine\", 1)\n",
      "Pair{AbstractString,Int64}(\"displayed\", 4)\n",
      "Pair{AbstractString,Int64}(\"polymerization\", 1)\n",
      "Pair{AbstractString,Int64}(\"the shorter propargyl\", 1)\n",
      "Pair{AbstractString,Int64}(\", the false\", 1)\n",
      "Pair{AbstractString,Int64}(\"plate reader (\", 3)\n",
      "Pair{AbstractString,Int64}(\"transfer\", 1)\n",
      "Pair{AbstractString,Int64}(\"] . The\", 1)\n",
      "Pair{AbstractString,Int64}(\"substitutions\", 11)\n",
      "Pair{AbstractString,Int64}(\"inhibitors by\", 1)\n",
      "Pair{AbstractString,Int64}(\", against which\", 2)\n",
      "Pair{AbstractString,Int64}(\", 2009\", 5)\n",
      "Pair{AbstractString,Int64}(\"encoding for\", 5)\n",
      "Pair{AbstractString,Int64}(\"such as cystic\", 1)\n",
      "Pair{AbstractString,Int64}(\"lowest among all\", 2)\n",
      "Pair{AbstractString,Int64}(\"analysis if the\", 1)\n",
      "Pair{AbstractString,Int64}(\"min at 72\", 1)\n",
      "Pair{AbstractString,Int64}(\"global gene expression\", 1)\n",
      "Pair{AbstractString,Int64}(\"2017\", 10)\n",
      "Pair{AbstractString,Int64}(\"( 3.48 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"generates a state\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 mM sodium\", 1)\n",
      "Pair{AbstractString,Int64}(\"assays , 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"substantially between\", 1)\n",
      "Pair{AbstractString,Int64}(\"/test 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"72 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"via\", 6)\n",
      "Pair{AbstractString,Int64}(\"the excitability of\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.05 ;\", 3)\n",
      "Pair{AbstractString,Int64}(\"195 All\", 1)\n",
      "Pair{AbstractString,Int64}(\"odor. The box\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 I ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"genomic RNA\", 3)\n",
      "Pair{AbstractString,Int64}(\"41 ( side-walls\", 1)\n",
      "Pair{AbstractString,Int64}(\"as described. author/funder.\", 1)\n",
      "Pair{AbstractString,Int64}(\"MatLab programs\", 1)\n",
      "Pair{AbstractString,Int64}(\"mapped to RdRps\", 1)\n",
      "Pair{AbstractString,Int64}(\"during acute\", 1)\n",
      "Pair{AbstractString,Int64}(\"cyclopropylmethyl substituent was\", 1)\n",
      "Pair{AbstractString,Int64}(\"Disease ( HFMD\", 1)\n",
      "Pair{AbstractString,Int64}(\"virus using\", 1)\n",
      "Pair{AbstractString,Int64}(\"important experimental readout\", 1)\n",
      "Pair{AbstractString,Int64}(\"researchers studying\", 1)\n",
      "Pair{AbstractString,Int64}(\"x-axis\", 1)\n",
      "Pair{AbstractString,Int64}(\"To enable\", 1)\n",
      "Pair{AbstractString,Int64}(\"the figure\", 2)\n",
      "Pair{AbstractString,Int64}(\"RaTG 13 .\", 2)\n",
      "Pair{AbstractString,Int64}(\") and HCoV-NL\", 1)\n",
      "Pair{AbstractString,Int64}(\"and stored at\", 1)\n",
      "Pair{AbstractString,Int64}(\"be sterically more\", 1)\n",
      "Pair{AbstractString,Int64}(\"( pp 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"equipotent activity against\", 1)\n",
      "Pair{AbstractString,Int64}(\"and immunopathology in\", 1)\n",
      "Pair{AbstractString,Int64}(\"[ 8 ]\", 1)\n",
      "Pair{AbstractString,Int64}(\"For plaque assays\", 1)\n",
      "Pair{AbstractString,Int64}(\"to catalyze nucleotidylation.\", 1)\n",
      "Pair{AbstractString,Int64}(\"85 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"h post-infection ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 5 stable\", 2)\n",
      "Pair{AbstractString,Int64}(\"of functional connections\", 1)\n",
      "Pair{AbstractString,Int64}(\"i-j ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\"; 2.2\", 1)\n",
      "Pair{AbstractString,Int64}(\"all SARS-CoV- 2\", 2)\n",
      "Pair{AbstractString,Int64}(\"for between\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"522\", 1)\n",
      "Pair{AbstractString,Int64}(\"on silica ,\", 3)\n",
      "Pair{AbstractString,Int64}(\"of virus-induced expression\", 1)\n",
      "Pair{AbstractString,Int64}(\". Only\", 1)\n",
      "Pair{AbstractString,Int64}(\"read-out\", 1)\n",
      "Pair{AbstractString,Int64}(\"enteroviruses include pathogens\", 1)\n",
      "Pair{AbstractString,Int64}(\"set-up. In\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells each\", 1)\n",
      "Pair{AbstractString,Int64}(\"create sufficient binding\", 1)\n",
      "Pair{AbstractString,Int64}(\"217 software\", 1)\n",
      "Pair{AbstractString,Int64}(\", J\", 1)\n",
      "Pair{AbstractString,Int64}(\"slight improvement\", 1)\n",
      "Pair{AbstractString,Int64}(\"influence\", 1)\n",
      "Pair{AbstractString,Int64}(\"prone to\", 3)\n",
      "Pair{AbstractString,Int64}(\"and immune-compromised\", 1)\n",
      "Pair{AbstractString,Int64}(\"e-expressing\", 1)\n",
      "Pair{AbstractString,Int64}(\"and water\", 1)\n",
      "Pair{AbstractString,Int64}(\", belonging to\", 1)\n",
      "Pair{AbstractString,Int64}(\"after CNO application.\", 1)\n",
      "Pair{AbstractString,Int64}(\"ESI\", 1)\n",
      "Pair{AbstractString,Int64}(\"MgCl 2 ,\", 3)\n",
      "Pair{AbstractString,Int64}(\") bias\", 1)\n",
      "Pair{AbstractString,Int64}(\", final\", 1)\n",
      "Pair{AbstractString,Int64}(\"with a fluorescent\", 1)\n",
      "Pair{AbstractString,Int64}(\"PEDV RNA\", 1)\n",
      "Pair{AbstractString,Int64}(\", 2014 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"two types\", 1)\n",
      "Pair{AbstractString,Int64}(\"silent mutations (\", 1)\n",
      "Pair{AbstractString,Int64}(\"the drug target\", 1)\n",
      "Pair{AbstractString,Int64}(\"end\", 4)\n",
      "Pair{AbstractString,Int64}(\"differently after\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.33 µM )\", 1)\n",
      "Pair{AbstractString,Int64}(\"from Korea patient\", 1)\n",
      "Pair{AbstractString,Int64}(\"9 /\", 1)\n",
      "Pair{AbstractString,Int64}(\"400\", 5)\n",
      "Pair{AbstractString,Int64}(\"However ,\", 23)\n",
      "Pair{AbstractString,Int64}(\"can restore\", 1)\n",
      "Pair{AbstractString,Int64}(\", bromoacetonitrile (\", 1)\n",
      "Pair{AbstractString,Int64}(\"transmission of\", 1)\n",
      "Pair{AbstractString,Int64}(\"well as a\", 1)\n",
      "Pair{AbstractString,Int64}(\"cultures by\", 1)\n",
      "Pair{AbstractString,Int64}(\"8 week-old\", 1)\n",
      "Pair{AbstractString,Int64}(\"active neurons\", 1)\n",
      "Pair{AbstractString,Int64}(\"all tested\", 2)\n",
      "Pair{AbstractString,Int64}(\") , were\", 3)\n",
      "Pair{AbstractString,Int64}(\"two methionines\", 1)\n",
      "Pair{AbstractString,Int64}(\"-NH 2 and\", 1)\n",
      "Pair{AbstractString,Int64}(\"camostat\", 1)\n",
      "Pair{AbstractString,Int64}(\"Switzerland )\", 4)\n",
      "Pair{AbstractString,Int64}(\"×\", 13)\n",
      "Pair{AbstractString,Int64}(\"only weak\", 1)\n",
      "Pair{AbstractString,Int64}(\"residue Gln\", 1)\n",
      "Pair{AbstractString,Int64}(\"into the phenyl\", 1)\n",
      "Pair{AbstractString,Int64}(\"S 1000\", 1)\n",
      "Pair{AbstractString,Int64}(\"in future work.\", 1)\n",
      "Pair{AbstractString,Int64}(\"limitation\", 1)\n",
      "Pair{AbstractString,Int64}(\"expressed as\", 1)\n",
      "Pair{AbstractString,Int64}(\"AlexaFluor 488 -\", 1)\n",
      "Pair{AbstractString,Int64}(\"region are\", 1)\n",
      "Pair{AbstractString,Int64}(\"Three days later\", 2)\n",
      "Pair{AbstractString,Int64}(\"conformational variability of\", 1)\n",
      "Pair{AbstractString,Int64}(\"; S ident\", 1)\n",
      "Pair{AbstractString,Int64}(\"taking false\", 1)\n",
      "Pair{AbstractString,Int64}(\"Glu 71 form\", 1)\n",
      "Pair{AbstractString,Int64}(\"Banerjee et\", 1)\n",
      "Pair{AbstractString,Int64}(\"the plasmid\", 1)\n",
      "Pair{AbstractString,Int64}(\"preprint Mice were\", 2)\n",
      "Pair{AbstractString,Int64}(\"465\", 1)\n",
      "Pair{AbstractString,Int64}(\", and Charles\", 1)\n",
      "Pair{AbstractString,Int64}(\"- 6 mutations\", 1)\n",
      "Pair{AbstractString,Int64}(\"( dimethylamino )\", 1)\n",
      "Pair{AbstractString,Int64}(\"retrieval of information\", 1)\n",
      "Pair{AbstractString,Int64}(\") in nonstructural\", 1)\n",
      "Pair{AbstractString,Int64}(\"23.55 ± 0.20\", 1)\n",
      "Pair{AbstractString,Int64}(\"or OPTIMEM\", 1)\n",
      "Pair{AbstractString,Int64}(\"the furin inhibitor\", 1)\n",
      "Pair{AbstractString,Int64}(\"satisfactory broad-spectrum activity\", 1)\n",
      "Pair{AbstractString,Int64}(\"post-transfection\", 2)\n",
      "Pair{AbstractString,Int64}(\"147 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\", ExoN-AE\", 2)\n",
      "Pair{AbstractString,Int64}(\"was observed in\", 2)\n",
      "Pair{AbstractString,Int64}(\"of killing\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibitors and\", 1)\n",
      "Pair{AbstractString,Int64}(\"least one\", 1)\n",
      "Pair{AbstractString,Int64}(\"is possible\", 4)\n",
      "Pair{AbstractString,Int64}(\"stimulation\", 1)\n",
      "Pair{AbstractString,Int64}(\"scheme :\", 1)\n",
      "Pair{AbstractString,Int64}(\"instruments. Chemical shifts\", 1)\n",
      "Pair{AbstractString,Int64}(\"100\", 20)\n",
      "Pair{AbstractString,Int64}(\"series of\", 4)\n",
      "Pair{AbstractString,Int64}(\"cells can encode\", 1)\n",
      "Pair{AbstractString,Int64}(\"MHV in a\", 1)\n",
      "Pair{AbstractString,Int64}(\"B have\", 1)\n",
      "Pair{AbstractString,Int64}(\"93 nucleotide\", 1)\n",
      "Pair{AbstractString,Int64}(\") was used\", 7)\n",
      "Pair{AbstractString,Int64}(\"226 of the\", 1)\n",
      "Pair{AbstractString,Int64}(\"Macherey\", 4)\n",
      "Pair{AbstractString,Int64}(\"( M\", 6)\n",
      "Pair{AbstractString,Int64}(\"0.2 M MgCl\", 2)\n",
      "Pair{AbstractString,Int64}(\"measurements , a\", 1)\n",
      "Pair{AbstractString,Int64}(\"and EV-D 68\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells were excluded\", 1)\n",
      "Pair{AbstractString,Int64}(\"renders\", 3)\n",
      "Pair{AbstractString,Int64}(\"51\", 9)\n",
      "Pair{AbstractString,Int64}(\"partial\", 3)\n",
      "Pair{AbstractString,Int64}(\"compare any\", 1)\n",
      "Pair{AbstractString,Int64}(\"a few pictures\", 1)\n",
      "Pair{AbstractString,Int64}(\"used to execute\", 1)\n",
      "Pair{AbstractString,Int64}(\"immune-compromised\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 f )\", 1)\n",
      "Pair{AbstractString,Int64}(\", and memory\", 2)\n",
      "Pair{AbstractString,Int64}(\"fluorescence microscope and\", 1)\n",
      "Pair{AbstractString,Int64}(\"215 were identified\", 1)\n",
      "Pair{AbstractString,Int64}(\"between groups ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"147 ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\", crystals were\", 1)\n",
      "Pair{AbstractString,Int64}(\"= 20\", 2)\n",
      "Pair{AbstractString,Int64}(\", DC ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"14 15\", 4)\n",
      "Pair{AbstractString,Int64}(\"and Vero E\", 1)\n",
      "Pair{AbstractString,Int64}(\"50 = 13.8\", 1)\n",
      "Pair{AbstractString,Int64}(\"cubic\", 2)\n",
      "Pair{AbstractString,Int64}(\"to assess memory\", 1)\n",
      "Pair{AbstractString,Int64}(\"Edans )\", 2)\n",
      "Pair{AbstractString,Int64}(\"disrupts\", 1)\n",
      "Pair{AbstractString,Int64}(\"' =\", 3)\n",
      "Pair{AbstractString,Int64}(\"the oxygen atom\", 1)\n",
      "Pair{AbstractString,Int64}(\"of natural\", 1)\n",
      "Pair{AbstractString,Int64}(\"1.1 mmol )\", 1)\n",
      "Pair{AbstractString,Int64}(\") GT (\", 1)\n",
      "Pair{AbstractString,Int64}(\"RT-PCR ( qRT-PCR\", 1)\n",
      "Pair{AbstractString,Int64}(\", 2003\", 2)\n",
      "Pair{AbstractString,Int64}(\"were harvested 48\", 1)\n",
      "Pair{AbstractString,Int64}(\"setting ( A.\", 1)\n",
      "Pair{AbstractString,Int64}(\"ExoN- 134\", 1)\n",
      "Pair{AbstractString,Int64}(\"1.0\", 1)\n",
      "Pair{AbstractString,Int64}(\"from David\", 1)\n",
      "Pair{AbstractString,Int64}(\"fit to\", 1)\n",
      "Pair{AbstractString,Int64}(\"Repeated\", 1)\n",
      "Pair{AbstractString,Int64}(\"483 Sensitivity assays\", 1)\n",
      "Pair{AbstractString,Int64}(\"long-term passage\", 2)\n",
      "Pair{AbstractString,Int64}(\"national institutional\", 1)\n",
      "Pair{AbstractString,Int64}(\"d-e , Extended\", 1)\n",
      "Pair{AbstractString,Int64}(\"detail previously.\", 1)\n",
      "Pair{AbstractString,Int64}(\". Parasagittal\", 1)\n",
      "Pair{AbstractString,Int64}(\"usual puromycin resistance\", 1)\n",
      "Pair{AbstractString,Int64}(\"quenched with\", 2)\n",
      "Pair{AbstractString,Int64}(\"inhaled steroid ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"in cells that\", 1)\n",
      "Pair{AbstractString,Int64}(\"503 ,\", 4)\n",
      "Pair{AbstractString,Int64}(\"piriform ensembles activated\", 1)\n",
      "Pair{AbstractString,Int64}(\"a-ketoamides were\", 1)\n",
      "Pair{AbstractString,Int64}(\"protein expression or\", 2)\n",
      "Pair{AbstractString,Int64}(\"acid , bubbled\", 1)\n",
      "Pair{AbstractString,Int64}(\"publication. Atomic coordinates\", 1)\n",
      "Pair{AbstractString,Int64}(\"Five days later\", 1)\n",
      "Pair{AbstractString,Int64}(\"lentiviruses , and\", 1)\n",
      "Pair{AbstractString,Int64}(\"frequently causes\", 1)\n",
      "Pair{AbstractString,Int64}(\") , Dabcyl-KTSAVLQ¯SGFRKM-E\", 1)\n",
      "Pair{AbstractString,Int64}(\"on RD cells.\", 1)\n",
      "Pair{AbstractString,Int64}(\"of DREADD expression\", 1)\n",
      "Pair{AbstractString,Int64}(\"as search model\", 1)\n",
      "Pair{AbstractString,Int64}(\"them\", 10)\n",
      "Pair{AbstractString,Int64}(\"two domains of\", 1)\n",
      "Pair{AbstractString,Int64}(\"- 2.1\", 1)\n",
      "Pair{AbstractString,Int64}(\"p-s/RPMI ( cRPMI\", 2)\n",
      "Pair{AbstractString,Int64}(\"clone\", 1)\n",
      "Pair{AbstractString,Int64}(\"preprint ExoN-DE\", 1)\n",
      "Pair{AbstractString,Int64}(\"number of synonymous\", 1)\n",
      "Pair{AbstractString,Int64}(\"DCM ) ,\", 1)\n",
      "Pair{AbstractString,Int64}(\". We observed\", 1)\n",
      "Pair{AbstractString,Int64}(\"to the opposite\", 1)\n",
      "Pair{AbstractString,Int64}(\"position of the\", 8)\n",
      "Pair{AbstractString,Int64}(\"RD cells were\", 1)\n",
      "Pair{AbstractString,Int64}(\"P< 0.001 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"escape-resistant antiviral drugs\", 1)\n",
      "Pair{AbstractString,Int64}(\"MES ( pH\", 1)\n",
      "Pair{AbstractString,Int64}(\"( inactive\", 1)\n",
      "Pair{AbstractString,Int64}(\"protease of\", 2)\n",
      "Pair{AbstractString,Int64}(\"58\", 8)\n",
      "Pair{AbstractString,Int64}(\"Against HCoV 229\", 1)\n",
      "Pair{AbstractString,Int64}(\"found that ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"p= 0.0312\", 4)\n",
      "Pair{AbstractString,Int64}(\"- 8 %\", 1)\n",
      "Pair{AbstractString,Int64}(\"derived 343\", 1)\n",
      "Pair{AbstractString,Int64}(\"( ppm\", 1)\n",
      "Pair{AbstractString,Int64}(\"The natural mouse\", 1)\n",
      "Pair{AbstractString,Int64}(\"are robust in\", 1)\n",
      "Pair{AbstractString,Int64}(\"and distributed ensembles\", 1)\n",
      "Pair{AbstractString,Int64}(\". Such\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"Apical washes were\", 1)\n",
      "Pair{AbstractString,Int64}(\"involved in ligand\", 1)\n",
      "Pair{AbstractString,Int64}(\"termed\", 1)\n",
      "Pair{AbstractString,Int64}(\"particular Met 49\", 2)\n",
      "Pair{AbstractString,Int64}(\"adaptive evolution server\", 1)\n",
      "Pair{AbstractString,Int64}(\"( formed by\", 1)\n",
      "Pair{AbstractString,Int64}(\"elevated in\", 1)\n",
      "Pair{AbstractString,Int64}(\"temporal dynamics.\", 1)\n",
      "Pair{AbstractString,Int64}(\"immune responses (\", 1)\n",
      "Pair{AbstractString,Int64}(\"pro at\", 1)\n",
      "Pair{AbstractString,Int64}(\"7.5 to\", 1)\n",
      "Pair{AbstractString,Int64}(\"that five 220\", 1)\n",
      "Pair{AbstractString,Int64}(\"bottle\", 1)\n",
      "Pair{AbstractString,Int64}(\"21 22\", 4)\n",
      "Pair{AbstractString,Int64}(\"mL ,\", 5)\n",
      "Pair{AbstractString,Int64}(\"( RT )\", 2)\n",
      "Pair{AbstractString,Int64}(\"codons (\", 1)\n",
      "Pair{AbstractString,Int64}(\"who was\", 1)\n",
      "Pair{AbstractString,Int64}(\"Broad spectrum protease\", 1)\n",
      "Pair{AbstractString,Int64}(\"Vectorlabs ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\"back-wall \\\"\", 3)\n",
      "Pair{AbstractString,Int64}(\"of HCoV\", 2)\n",
      "Pair{AbstractString,Int64}(\"are used (\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.05 , *\", 1)\n",
      "Pair{AbstractString,Int64}(\"3.7 µM against\", 1)\n",
      "Pair{AbstractString,Int64}(\"can thus\", 1)\n",
      "Pair{AbstractString,Int64}(\"may 301 assume\", 1)\n",
      "Pair{AbstractString,Int64}(\"SARS-CoV- 2 with\", 1)\n",
      "Pair{AbstractString,Int64}(\"a similar difference\", 1)\n",
      "Pair{AbstractString,Int64}(\"plaques were visualized\", 1)\n",
      "Pair{AbstractString,Int64}(\"SARS-CoV and\", 7)\n",
      "Pair{AbstractString,Int64}(\"- 48 .\", 1)\n",
      "Pair{AbstractString,Int64}(\"bone\", 1)\n",
      "Pair{AbstractString,Int64}(\"are striking given\", 1)\n",
      "Pair{AbstractString,Int64}(\"Shimadzu Corporation\", 1)\n",
      "Pair{AbstractString,Int64}(\"GISAID database\", 1)\n",
      "Pair{AbstractString,Int64}(\"was observed at\", 2)\n",
      "Pair{AbstractString,Int64}(\"or 221 absence\", 1)\n",
      "Pair{AbstractString,Int64}(\"reported in parts\", 1)\n",
      "Pair{AbstractString,Int64}(\"enter into\", 1)\n",
      "Pair{AbstractString,Int64}(\"EC 50 of\", 3)\n",
      "Pair{AbstractString,Int64}(\"0.0269\", 1)\n",
      "Pair{AbstractString,Int64}(\"a common target\", 1)\n",
      "Pair{AbstractString,Int64}(\"connections\", 2)\n",
      "Pair{AbstractString,Int64}(\", then the\", 1)\n",
      "Pair{AbstractString,Int64}(\"Six to\", 1)\n",
      "Pair{AbstractString,Int64}(\"s in enterovirus-infected\", 1)\n",
      "Pair{AbstractString,Int64}(\"230 Fixed\", 1)\n",
      "Pair{AbstractString,Int64}(\"crude material\", 3)\n",
      "Pair{AbstractString,Int64}(\"neurons was\", 2)\n",
      "Pair{AbstractString,Int64}(\"a smaller\", 1)\n",
      "Pair{AbstractString,Int64}(\"tested. E 64\", 1)\n",
      "Pair{AbstractString,Int64}(\"as single plane\", 1)\n",
      "Pair{AbstractString,Int64}(\"of HIV- 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"250\", 23)\n",
      "Pair{AbstractString,Int64}(\"box. Materials for\", 1)\n",
      "Pair{AbstractString,Int64}(\"[ 2 ]\", 1)\n",
      "Pair{AbstractString,Int64}(\"Whole-cell recording\", 1)\n",
      "Pair{AbstractString,Int64}(\"in the hope\", 1)\n",
      "Pair{AbstractString,Int64}(\"is involved in\", 1)\n",
      "Pair{AbstractString,Int64}(\"density of\", 3)\n",
      "Pair{AbstractString,Int64}(\"the increased synonymous\", 1)\n",
      "Pair{AbstractString,Int64}(\"the binding site.\", 1)\n",
      "Pair{AbstractString,Int64}(\"by a factor\", 1)\n",
      "Pair{AbstractString,Int64}(\"be prone to\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 7 )\", 2)\n",
      "Pair{AbstractString,Int64}(\"295 quantitative PCR\", 1)\n",
      "Pair{AbstractString,Int64}(\"HRV 14 was\", 1)\n",
      "Pair{AbstractString,Int64}(\"excellent activities\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 - 519\", 1)\n",
      "Pair{AbstractString,Int64}(\"564 The\", 1)\n",
      "Pair{AbstractString,Int64}(\"above the pocket\", 1)\n",
      "Pair{AbstractString,Int64}(\"structure ( PDB\", 1)\n",
      "Pair{AbstractString,Int64}(\"progeny during\", 1)\n",
      "Pair{AbstractString,Int64}(\") vs.\", 3)\n",
      "Pair{AbstractString,Int64}(\", Extended Data\", 13)\n",
      "Pair{AbstractString,Int64}(\"and 11 a\", 1)\n",
      "Pair{AbstractString,Int64}(\"ALT ) was\", 1)\n",
      "Pair{AbstractString,Int64}(\". Another optimization\", 1)\n",
      "Pair{AbstractString,Int64}(\"region of\", 4)\n",
      "Pair{AbstractString,Int64}(\"13 , which\", 1)\n",
      "Pair{AbstractString,Int64}(\"the activating DREADD\", 1)\n",
      "Pair{AbstractString,Int64}(\"replication ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"frequency of codon\", 1)\n",
      "Pair{AbstractString,Int64}(\"and positive control\", 1)\n",
      "Pair{AbstractString,Int64}(\"260\", 1)\n",
      "Pair{AbstractString,Int64}(\"efforts to\", 2)\n",
      "Pair{AbstractString,Int64}(\"of stimulus\", 2)\n",
      "Pair{AbstractString,Int64}(\"trace '\", 1)\n",
      "Pair{AbstractString,Int64}(\"sequence ,\", 4)\n",
      "Pair{AbstractString,Int64}(\"and trees were\", 1)\n",
      "Pair{AbstractString,Int64}(\"particularly\", 1)\n",
      "Pair{AbstractString,Int64}(\"as enterovirus 3\", 1)\n",
      "Pair{AbstractString,Int64}(\"7 -\", 3)\n",
      "Pair{AbstractString,Int64}(\"dN/dS ratio of\", 1)\n",
      "Pair{AbstractString,Int64}(\"in non- The\", 1)\n",
      "Pair{AbstractString,Int64}(\"for increased\", 1)\n",
      "Pair{AbstractString,Int64}(\"Therapeutic\", 1)\n",
      "Pair{AbstractString,Int64}(\"4.5\", 1)\n",
      "Pair{AbstractString,Int64}(\"Because of the\", 2)\n",
      "Pair{AbstractString,Int64}(\"CMV promoter ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"normalized response\", 1)\n",
      "Pair{AbstractString,Int64}(\"( -CH=O )\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells or DC\", 1)\n",
      "Pair{AbstractString,Int64}(\"room\", 6)\n",
      "Pair{AbstractString,Int64}(\"responses we observe\", 1)\n",
      "Pair{AbstractString,Int64}(\"Technologies )\", 2)\n",
      "Pair{AbstractString,Int64}(\"% glycerol ,\", 3)\n",
      "Pair{AbstractString,Int64}(\"increased expression of\", 1)\n",
      "Pair{AbstractString,Int64}(\"expressing human\", 2)\n",
      "Pair{AbstractString,Int64}(\"lead\", 13)\n",
      "Pair{AbstractString,Int64}(\"between Fos-tagged\", 1)\n",
      "Pair{AbstractString,Int64}(\"and 1\", 4)\n",
      "Pair{AbstractString,Int64}(\"p< 0.001\", 1)\n",
      "Pair{AbstractString,Int64}(\"with trans-complemented GFP\", 1)\n",
      "Pair{AbstractString,Int64}(\"°\", 38)\n",
      "Pair{AbstractString,Int64}(\"mutant\", 3)\n",
      "Pair{AbstractString,Int64}(\"cells were subjected\", 1)\n",
      "Pair{AbstractString,Int64}(\"substituent has\", 1)\n",
      "Pair{AbstractString,Int64}(\"sequence Table\", 1)\n",
      "Pair{AbstractString,Int64}(\"structures of our\", 1)\n",
      "Pair{AbstractString,Int64}(\"aldehydes have also\", 1)\n",
      "Pair{AbstractString,Int64}(\"cold\", 3)\n",
      "Pair{AbstractString,Int64}(\"0.0073\", 1)\n",
      "Pair{AbstractString,Int64}(\"water. The aqueous\", 1)\n",
      "Pair{AbstractString,Int64}(\"VSV ) pseudoparticle\", 1)\n",
      "Pair{AbstractString,Int64}(\"extend existing\", 1)\n",
      "Pair{AbstractString,Int64}(\"were maintained in\", 2)\n",
      "Pair{AbstractString,Int64}(\"based on restoration\", 1)\n",
      "Pair{AbstractString,Int64}(\"USA 59 and\", 1)\n",
      "Pair{AbstractString,Int64}(\") . Furthermore\", 4)\n",
      "Pair{AbstractString,Int64}(\"observed a good\", 1)\n",
      "Pair{AbstractString,Int64}(\"78,000\", 1)\n",
      "Pair{AbstractString,Int64}(\"cyan ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"mL Hank\", 1)\n",
      "Pair{AbstractString,Int64}(\"for HCoV- 198\", 1)\n",
      "Pair{AbstractString,Int64}(\"of the patched\", 1)\n",
      "Pair{AbstractString,Int64}(\"only compounds designed\", 1)\n",
      "Pair{AbstractString,Int64}(\"to differentiate and\", 1)\n",
      "Pair{AbstractString,Int64}(\"the extracted\", 2)\n",
      "Pair{AbstractString,Int64}(\"and mock infected\", 2)\n",
      "Pair{AbstractString,Int64}(\"ZNF\", 1)\n",
      "Pair{AbstractString,Int64}(\"enzyme expressed in\", 1)\n",
      "Pair{AbstractString,Int64}(\"After pre-processing the\", 1)\n",
      "Pair{AbstractString,Int64}(\"mice was significantly\", 1)\n",
      "Pair{AbstractString,Int64}(\"activation of DREADDs\", 1)\n",
      "Pair{AbstractString,Int64}(\"Ryan et al.\", 1)\n",
      "Pair{AbstractString,Int64}(\"IFIH\", 1)\n",
      "Pair{AbstractString,Int64}(\"exhibit good to\", 1)\n",
      "Pair{AbstractString,Int64}(\"BHK-\", 2)\n",
      "Pair{AbstractString,Int64}(\"sequences were performed\", 1)\n",
      "Pair{AbstractString,Int64}(\"RD 311\", 1)\n",
      "Pair{AbstractString,Int64}(\"qPCR assays\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 ) labeled\", 1)\n",
      "Pair{AbstractString,Int64}(\"run 1.2 %\", 1)\n",
      "Pair{AbstractString,Int64}(\"7.3 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"post-seeding\", 1)\n",
      "Pair{AbstractString,Int64}(\"CleavEx technique\", 1)\n",
      "Pair{AbstractString,Int64}(\", bubbled\", 1)\n",
      "Pair{AbstractString,Int64}(\"compounds for\", 1)\n",
      "Pair{AbstractString,Int64}(\"resulting sequence was\", 1)\n",
      "Pair{AbstractString,Int64}(\"Wild-type control animals\", 1)\n",
      "Pair{AbstractString,Int64}(\"is contraindicated\", 1)\n",
      "Pair{AbstractString,Int64}(\"Database was used\", 1)\n",
      "Pair{AbstractString,Int64}(\"using Alamar-Blue™\", 1)\n",
      "Pair{AbstractString,Int64}(\", we next\", 2)\n",
      "Pair{AbstractString,Int64}(\"a-ketoamides , in-vitro\", 1)\n",
      "Pair{AbstractString,Int64}(\"research on\", 1)\n",
      "Pair{AbstractString,Int64}(\"infection and establishes\", 1)\n",
      "Pair{AbstractString,Int64}(\". In\", 30)\n",
      "Pair{AbstractString,Int64}(\") , triplet\", 1)\n",
      "Pair{AbstractString,Int64}(\"RNA standard\", 3)\n",
      "Pair{AbstractString,Int64}(\"[ 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"in specific\", 1)\n",
      "Pair{AbstractString,Int64}(\", female Ly\", 2)\n",
      "Pair{AbstractString,Int64}(\"its bisulfite\", 1)\n",
      "Pair{AbstractString,Int64}(\"20 21\", 4)\n",
      "Pair{AbstractString,Int64}(\"learning were chemogenetically\", 1)\n",
      "Pair{AbstractString,Int64}(\"93\", 1)\n",
      "Pair{AbstractString,Int64}(\") to\", 14)\n",
      "Pair{AbstractString,Int64}(\"( no.\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"358 KLK\", 1)\n",
      "Pair{AbstractString,Int64}(\"3.75\", 1)\n",
      "Pair{AbstractString,Int64}(\"305\", 1)\n",
      "Pair{AbstractString,Int64}(\"escape behavior after\", 2)\n",
      "Pair{AbstractString,Int64}(\"a resolution of\", 1)\n",
      "Pair{AbstractString,Int64}(\"is a\", 15)\n",
      "Pair{AbstractString,Int64}(\"t in\", 1)\n",
      "Pair{AbstractString,Int64}(\"al. , 2008\", 5)\n",
      "Pair{AbstractString,Int64}(\"compounds.\", 3)\n",
      "Pair{AbstractString,Int64}(\"expressing or\", 6)\n",
      "Pair{AbstractString,Int64}(\"further exploration.\", 1)\n",
      "Pair{AbstractString,Int64}(\"the hM 4\", 1)\n",
      "Pair{AbstractString,Int64}(\"202 and the\", 1)\n",
      "Pair{AbstractString,Int64}(\"]\", 23)\n",
      "Pair{AbstractString,Int64}(\"bond either from\", 1)\n",
      "Pair{AbstractString,Int64}(\"1144\", 1)\n",
      "Pair{AbstractString,Int64}(\"nucleotides for\", 1)\n",
      "Pair{AbstractString,Int64}(\"disease. In\", 1)\n",
      "Pair{AbstractString,Int64}(\"viruses with high\", 1)\n",
      "Pair{AbstractString,Int64}(\"G ) Maximum\", 1)\n",
      "Pair{AbstractString,Int64}(\"compounds. Initially ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"exhibit\", 3)\n",
      "Pair{AbstractString,Int64}(\"human genes was\", 1)\n",
      "Pair{AbstractString,Int64}(\"reinforced\", 3)\n",
      "Pair{AbstractString,Int64}(\"306 fragments\", 1)\n",
      "Pair{AbstractString,Int64}(\") ) of\", 1)\n",
      "Pair{AbstractString,Int64}(\"lysed using\", 3)\n",
      "Pair{AbstractString,Int64}(\"Fos-tagging could also\", 1)\n",
      "Pair{AbstractString,Int64}(\"fusion of viral\", 2)\n",
      "Pair{AbstractString,Int64}(\"trigger odor\", 1)\n",
      "Pair{AbstractString,Int64}(\"of the inhibitory\", 2)\n",
      "Pair{AbstractString,Int64}(\"nucleic acid when\", 1)\n",
      "Pair{AbstractString,Int64}(\"database.\", 1)\n",
      "Pair{AbstractString,Int64}(\". CC-BY-ND 4.0\", 5)\n",
      "Pair{AbstractString,Int64}(\"few systematic\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Fig. 4\", 10)\n",
      "Pair{AbstractString,Int64}(\"to be\", 28)\n",
      "Pair{AbstractString,Int64}(\"elements\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 mM HEPES/\", 1)\n",
      "Pair{AbstractString,Int64}(\"( PDE )\", 1)\n",
      "Pair{AbstractString,Int64}(\"group incubated at\", 2)\n",
      "Pair{AbstractString,Int64}(\"; Gore\", 1)\n",
      "Pair{AbstractString,Int64}(\"was inserted\", 1)\n",
      "Pair{AbstractString,Int64}(\"' = cyclopropyl\", 1)\n",
      "Pair{AbstractString,Int64}(\"abolished in the\", 1)\n",
      "Pair{AbstractString,Int64}(\"and ExoN-inactivation\", 1)\n",
      "Pair{AbstractString,Int64}(\"ethyl-acetate ( the\", 1)\n",
      "Pair{AbstractString,Int64}(\"maintained 409 as\", 1)\n",
      "Pair{AbstractString,Int64}(\"various complexes\", 1)\n",
      "Pair{AbstractString,Int64}(\", the more\", 1)\n",
      "Pair{AbstractString,Int64}(\"treated with\", 3)\n",
      "Pair{AbstractString,Int64}(\"25 kDa\", 1)\n",
      "Pair{AbstractString,Int64}(\"As a measure\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 4 cells\", 7)\n",
      "Pair{AbstractString,Int64}(\"due to factors\", 1)\n",
      "Pair{AbstractString,Int64}(\"whereas such interaction\", 1)\n",
      "Pair{AbstractString,Int64}(\"appeared in the\", 1)\n",
      "Pair{AbstractString,Int64}(\"live-attenuated vaccine\", 1)\n",
      "Pair{AbstractString,Int64}(\"behavioral response. How\", 1)\n",
      "Pair{AbstractString,Int64}(\"63 in\", 1)\n",
      "Pair{AbstractString,Int64}(\"of endogenous cFos\", 1)\n",
      "Pair{AbstractString,Int64}(\"13 inhibitor (\", 2)\n",
      "Pair{AbstractString,Int64}(\"whole genome sequence\", 1)\n",
      "Pair{AbstractString,Int64}(\"titration ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"could represent a\", 1)\n",
      "Pair{AbstractString,Int64}(\"= 5.6\", 1)\n",
      "Pair{AbstractString,Int64}(\"were generated as\", 1)\n",
      "Pair{AbstractString,Int64}(\"lowest among\", 2)\n",
      "Pair{AbstractString,Int64}(\"hydrogen-bond acceptors instead\", 1)\n",
      "Pair{AbstractString,Int64}(\"339\", 1)\n",
      "Pair{AbstractString,Int64}(\"as the side\", 1)\n",
      "Pair{AbstractString,Int64}(\"is that the\", 4)\n",
      "Pair{AbstractString,Int64}(\"RS\", 1)\n",
      "Pair{AbstractString,Int64}(\"of α-hydroxyamides 10\", 1)\n",
      "Pair{AbstractString,Int64}(\"at 25\", 1)\n",
      "Pair{AbstractString,Int64}(\"proofreading\", 4)\n",
      "Pair{AbstractString,Int64}(\"sequence identify\", 1)\n",
      "Pair{AbstractString,Int64}(\"protein , the\", 1)\n",
      "Pair{AbstractString,Int64}(\"µL of\", 10)\n",
      "Pair{AbstractString,Int64}(\"to be identified.\", 1)\n",
      "Pair{AbstractString,Int64}(\"the 123\", 1)\n",
      "Pair{AbstractString,Int64}(\"lineage as an\", 1)\n",
      "Pair{AbstractString,Int64}(\"random drift hypothesis\", 1)\n",
      "Pair{AbstractString,Int64}(\"not restore\", 1)\n",
      "Pair{AbstractString,Int64}(\"Dq-expressing piriform\", 1)\n",
      "Pair{AbstractString,Int64}(\"Alphacoronavirus\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibitor against SARS-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\"respectively. The fourth\", 1)\n",
      "Pair{AbstractString,Int64}(\"first added\", 1)\n",
      "Pair{AbstractString,Int64}(\"maturation\", 2)\n",
      "Pair{AbstractString,Int64}(\"trifluoroacetic acid\", 1)\n",
      "Pair{AbstractString,Int64}(\"were added ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"not permissive to\", 1)\n",
      "Pair{AbstractString,Int64}(\"the compounds\", 6)\n",
      "Pair{AbstractString,Int64}(\"post-seeding ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"and only\", 1)\n",
      "Pair{AbstractString,Int64}(\"the CS+-tagged ensemble\", 1)\n",
      "Pair{AbstractString,Int64}(\"8 ( Ly\", 1)\n",
      "Pair{AbstractString,Int64}(\"crystal of\", 2)\n",
      "Pair{AbstractString,Int64}(\"current reported SARS-CoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"VSVpp entry mediated\", 1)\n",
      "Pair{AbstractString,Int64}(\"rods\", 1)\n",
      "Pair{AbstractString,Int64}(\"number ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"ThermoScientific ) .\", 1)\n",
      "Pair{AbstractString,Int64}(\"in MHV-ExoN\", 1)\n",
      "Pair{AbstractString,Int64}(\"0 ) can\", 1)\n",
      "Pair{AbstractString,Int64}(\"and infected at\", 1)\n",
      "Pair{AbstractString,Int64}(\"slice from the\", 1)\n",
      "Pair{AbstractString,Int64}(\"LTV\", 1)\n",
      "Pair{AbstractString,Int64}(\"the common cold\", 3)\n",
      "Pair{AbstractString,Int64}(\"30 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"S ) or\", 1)\n",
      "Pair{AbstractString,Int64}(\"co-cultured with\", 1)\n",
      "Pair{AbstractString,Int64}(\"model of\", 3)\n",
      "Pair{AbstractString,Int64}(\"or HCoV-NL\", 1)\n",
      "Pair{AbstractString,Int64}(\"evaluated. VeroE 6\", 1)\n",
      "Pair{AbstractString,Int64}(\"SARS-CoV and the\", 1)\n",
      "Pair{AbstractString,Int64}(\"total RNA of\", 1)\n",
      "Pair{AbstractString,Int64}(\"that the P\", 2)\n",
      "Pair{AbstractString,Int64}(\"centroid\", 1)\n",
      "Pair{AbstractString,Int64}(\"s ( see\", 1)\n",
      "Pair{AbstractString,Int64}(\"thick ) of\", 1)\n",
      "Pair{AbstractString,Int64}(\"a Leica\", 2)\n",
      "Pair{AbstractString,Int64}(\"in male\", 2)\n",
      "Pair{AbstractString,Int64}(\"sample was stored\", 2)\n",
      "Pair{AbstractString,Int64}(\"N 5 O\", 1)\n",
      "Pair{AbstractString,Int64}(\", HCoV-NL\", 5)\n",
      "Pair{AbstractString,Int64}(\"2 substituen. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"are favored\", 1)\n",
      "Pair{AbstractString,Int64}(\"to the odor\", 1)\n",
      "Pair{AbstractString,Int64}(\"rather than the\", 1)\n",
      "Pair{AbstractString,Int64}(\"TMPRSS 2 /cell\", 1)\n",
      "Pair{AbstractString,Int64}(\"c , ns\", 1)\n",
      "Pair{AbstractString,Int64}(\"H , L\", 1)\n",
      "Pair{AbstractString,Int64}(\"expressing cell\", 1)\n",
      "Pair{AbstractString,Int64}(\"developed as\", 1)\n",
      "Pair{AbstractString,Int64}(\"potent\", 4)\n",
      "Pair{AbstractString,Int64}(\"the proportion of\", 2)\n",
      "Pair{AbstractString,Int64}(\"domains of\", 2)\n",
      "Pair{AbstractString,Int64}(\"( http://www.datamonkey.org/\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 μL\", 2)\n",
      "Pair{AbstractString,Int64}(\"229 230\", 1)\n",
      "Pair{AbstractString,Int64}(\"bioRxiv preprint response\", 1)\n",
      "Pair{AbstractString,Int64}(\"however , the\", 3)\n",
      "Pair{AbstractString,Int64}(\"n= 6 and\", 1)\n",
      "Pair{AbstractString,Int64}(\"2012 . 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"as using Hank\", 1)\n",
      "Pair{AbstractString,Int64}(\"and manipulate piriform\", 2)\n",
      "Pair{AbstractString,Int64}(\"mapped to\", 1)\n",
      "Pair{AbstractString,Int64}(\"preprint tuberculosis\", 1)\n",
      "Pair{AbstractString,Int64}(\", 307\", 1)\n",
      "Pair{AbstractString,Int64}(\", we speculated\", 1)\n",
      "Pair{AbstractString,Int64}(\"g , 3.5\", 1)\n",
      "Pair{AbstractString,Int64}(\"Biotrend\", 1)\n",
      "Pair{AbstractString,Int64}(\"quit divergent\", 1)\n",
      "Pair{AbstractString,Int64}(\"Then\", 10)\n",
      "Pair{AbstractString,Int64}(\"189 has\", 1)\n",
      "Pair{AbstractString,Int64}(\"cell-type dependence\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 ( 66\", 1)\n",
      "Pair{AbstractString,Int64}(\"ExoN active\", 1)\n",
      "Pair{AbstractString,Int64}(\"was generated\", 5)\n",
      "Pair{AbstractString,Int64}(\"Next , we\", 4)\n",
      "Pair{AbstractString,Int64}(\"bioRxiv preprint Multiple\", 1)\n",
      "Pair{AbstractString,Int64}(\"added back to\", 1)\n",
      "Pair{AbstractString,Int64}(\"are fear-conditioned ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"open field box\", 1)\n",
      "Pair{AbstractString,Int64}(\"53 The complex\", 1)\n",
      "Pair{AbstractString,Int64}(\"acceptors form\", 1)\n",
      "Pair{AbstractString,Int64}(\"as Michael acceptors\", 1)\n",
      "Pair{AbstractString,Int64}(\"( beta-CoV )\", 1)\n",
      "Pair{AbstractString,Int64}(\"catalyzed by the\", 1)\n",
      "Pair{AbstractString,Int64}(\"determinants of\", 1)\n",
      "Pair{AbstractString,Int64}(\"least reducing\", 1)\n",
      "Pair{AbstractString,Int64}(\"Cbz-Leu-GlnLactam-CO-CO-NH-iPr\", 1)\n",
      "Pair{AbstractString,Int64}(\"click\", 1)\n",
      "Pair{AbstractString,Int64}(\"predominantly containing the\", 1)\n",
      "Pair{AbstractString,Int64}(\"for 30\", 11)\n",
      "Pair{AbstractString,Int64}(\"3 X\", 1)\n",
      "Pair{AbstractString,Int64}(\"receptors does\", 1)\n",
      "Pair{AbstractString,Int64}(\"quantitative behavioral readouts.\", 1)\n",
      "Pair{AbstractString,Int64}(\"G transitions.\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"α-ketoamides 11 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"( Anthos Microsystem\", 1)\n",
      "Pair{AbstractString,Int64}(\"10.1093 /nar/gkz 268\", 1)\n",
      "Pair{AbstractString,Int64}(\", betacitronellol\", 1)\n",
      "Pair{AbstractString,Int64}(\"distinct cellular compartments\", 1)\n",
      "Pair{AbstractString,Int64}(\"energy transfer (\", 1)\n",
      "Pair{AbstractString,Int64}(\"were broadly\", 1)\n",
      "Pair{AbstractString,Int64}(\"significance was determined\", 3)\n",
      "Pair{AbstractString,Int64}(\"identity in piriform\", 1)\n",
      "Pair{AbstractString,Int64}(\"bromoacetonitrile. As\", 1)\n",
      "Pair{AbstractString,Int64}(\"the ability of\", 1)\n",
      "Pair{AbstractString,Int64}(\"the catalyst.\", 1)\n",
      "Pair{AbstractString,Int64}(\"to show\", 1)\n",
      "Pair{AbstractString,Int64}(\"of infected patients\", 1)\n",
      "Pair{AbstractString,Int64}(\") . Knowing\", 1)\n",
      "Pair{AbstractString,Int64}(\"CL pro )\", 1)\n",
      "Pair{AbstractString,Int64}(\"we employed porcine\", 1)\n",
      "Pair{AbstractString,Int64}(\"covered , HCoV-NL\", 1)\n",
      "Pair{AbstractString,Int64}(\", the rates\", 1)\n",
      "Pair{AbstractString,Int64}(\"5 log 10\", 1)\n",
      "Pair{AbstractString,Int64}(\"associative memory encoding\", 1)\n",
      "Pair{AbstractString,Int64}(\"microscope\", 6)\n",
      "Pair{AbstractString,Int64}(\"evoked action\", 2)\n",
      "Pair{AbstractString,Int64}(\"( 1 x\", 1)\n",
      "Pair{AbstractString,Int64}(\"inactivation by high\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 lung cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"distance ( u\", 1)\n",
      "Pair{AbstractString,Int64}(\"For high-content\", 1)\n",
      "Pair{AbstractString,Int64}(\"against the new\", 1)\n",
      "Pair{AbstractString,Int64}(\"broad ( br\", 1)\n",
      "Pair{AbstractString,Int64}(\"or from\", 1)\n",
      "Pair{AbstractString,Int64}(\"damage and\", 1)\n",
      "Pair{AbstractString,Int64}(\"candidate compounds. Initially\", 1)\n",
      "Pair{AbstractString,Int64}(\"as KLKs were\", 1)\n",
      "Pair{AbstractString,Int64}(\"Huh 7 and\", 1)\n",
      "Pair{AbstractString,Int64}(\"not exhibit a\", 1)\n",
      "Pair{AbstractString,Int64}(\"a plethysmograph ,\", 1)\n",
      "Pair{AbstractString,Int64}(\", with only\", 1)\n",
      "Pair{AbstractString,Int64}(\"measured particles.\", 1)\n",
      "Pair{AbstractString,Int64}(\"although their non-specific\", 1)\n",
      "Pair{AbstractString,Int64}(\"the hypothesis\", 1)\n",
      "Pair{AbstractString,Int64}(\"the GraphPad Prism\", 1)\n",
      "Pair{AbstractString,Int64}(\"respectively ; no\", 1)\n",
      "Pair{AbstractString,Int64}(\"that mutations\", 1)\n",
      "Pair{AbstractString,Int64}(\") . 1\", 4)\n",
      "Pair{AbstractString,Int64}(\"observed at 252\", 1)\n",
      "Pair{AbstractString,Int64}(\") , which\", 11)\n",
      "Pair{AbstractString,Int64}(\"TFA )\", 1)\n",
      "Pair{AbstractString,Int64}(\"fraction of cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"e ablation on\", 1)\n",
      "Pair{AbstractString,Int64}(\"40 µm\", 1)\n",
      "Pair{AbstractString,Int64}(\"information , are\", 1)\n",
      "Pair{AbstractString,Int64}(\"/ 2020.02.27.969006 doi\", 8)\n",
      "Pair{AbstractString,Int64}(\"more rigid lactam\", 1)\n",
      "Pair{AbstractString,Int64}(\"50 = 9.0\", 1)\n",
      "Pair{AbstractString,Int64}(\"15 )\", 3)\n",
      "Pair{AbstractString,Int64}(\"or HindIII\", 1)\n",
      "Pair{AbstractString,Int64}(\"increasing concentration\", 1)\n",
      "Pair{AbstractString,Int64}(\"fidelity\", 2)\n",
      "Pair{AbstractString,Int64}(\"PFU/mL , so\", 1)\n",
      "Pair{AbstractString,Int64}(\"For RNA-launched\", 1)\n",
      "Pair{AbstractString,Int64}(\"patients with pneumonia\", 1)\n",
      "Pair{AbstractString,Int64}(\"culture medium. Ciclesonide\", 1)\n",
      "Pair{AbstractString,Int64}(\"traces. Artificial reactivation\", 1)\n",
      "Pair{AbstractString,Int64}(\"necrosis.\", 1)\n",
      "Pair{AbstractString,Int64}(\"make hydrophobic\", 1)\n",
      "Pair{AbstractString,Int64}(\"those labeled\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 mL of\", 1)\n",
      "Pair{AbstractString,Int64}(\"We engineered and\", 1)\n",
      "Pair{AbstractString,Int64}(\"whether\", 23)\n",
      "Pair{AbstractString,Int64}(\"view\", 4)\n",
      "Pair{AbstractString,Int64}(\"Hand\", 1)\n",
      "Pair{AbstractString,Int64}(\"spleen immune\", 1)\n",
      "Pair{AbstractString,Int64}(\"Mouse Metabolic\", 1)\n",
      "Pair{AbstractString,Int64}(\"0.631 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"homologous recombination\", 1)\n",
      "Pair{AbstractString,Int64}(\"175562 doi\", 15)\n",
      "Pair{AbstractString,Int64}(\"\\\" , n=\", 2)\n",
      "Pair{AbstractString,Int64}(\"one in the\", 1)\n",
      "Pair{AbstractString,Int64}(\"46 - 85\", 1)\n",
      "Pair{AbstractString,Int64}(\"2014 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"similar animal\", 1)\n",
      "Pair{AbstractString,Int64}(\"126\", 1)\n",
      "Pair{AbstractString,Int64}(\"somewhat attenuated\", 1)\n",
      "Pair{AbstractString,Int64}(\", the IC\", 1)\n",
      "Pair{AbstractString,Int64}(\"receptors to activate\", 1)\n",
      "Pair{AbstractString,Int64}(\"stabilization of piriform\", 1)\n",
      "Pair{AbstractString,Int64}(\"sparse and dispersed\", 1)\n",
      "Pair{AbstractString,Int64}(\"their potential\", 2)\n",
      "Pair{AbstractString,Int64}(\"eugenol\", 4)\n",
      "Pair{AbstractString,Int64}(\"nitrogen. Freshly\", 1)\n",
      "Pair{AbstractString,Int64}(\"spatial patterns of\", 1)\n",
      "Pair{AbstractString,Int64}(\"and crossed them\", 1)\n",
      "Pair{AbstractString,Int64}(\", and 96\", 1)\n",
      "Pair{AbstractString,Int64}(\"effects ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"-conjugated\", 1)\n",
      "Pair{AbstractString,Int64}(\"i.e. slope of\", 1)\n",
      "Pair{AbstractString,Int64}(\"evaporated to generate\", 1)\n",
      "Pair{AbstractString,Int64}(\"ExoN-AA titers\", 1)\n",
      "Pair{AbstractString,Int64}(\"not paired\", 2)\n",
      "Pair{AbstractString,Int64}(\"surprisingly\", 5)\n",
      "Pair{AbstractString,Int64}(\"use a\", 1)\n",
      "Pair{AbstractString,Int64}(\"( see Scheme\", 1)\n",
      "Pair{AbstractString,Int64}(\", 7 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"or X-tremeGENE\", 1)\n",
      "Pair{AbstractString,Int64}(\"Gateway cloning\", 1)\n",
      "Pair{AbstractString,Int64}(\"4 sections\", 1)\n",
      "Pair{AbstractString,Int64}(\"cause polio-like\", 1)\n",
      "Pair{AbstractString,Int64}(\"bond either\", 1)\n",
      "Pair{AbstractString,Int64}(\"replication or fitness\", 1)\n",
      "Pair{AbstractString,Int64}(\"infection experiments ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"were characterized by\", 1)\n",
      "Pair{AbstractString,Int64}(\", HCoV-\", 1)\n",
      "Pair{AbstractString,Int64}(\"of α-hydroxyamides\", 1)\n",
      "Pair{AbstractString,Int64}(\"DNA templates ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"solution in author/funder.\", 1)\n",
      "Pair{AbstractString,Int64}(\"pH 7.4\", 1)\n",
      "Pair{AbstractString,Int64}(\"Dq-expressing\", 8)\n",
      "Pair{AbstractString,Int64}(\"at the indicated\", 2)\n",
      "Pair{AbstractString,Int64}(\"almost ideal match\", 1)\n",
      "Pair{AbstractString,Int64}(\"~ 5 for\", 1)\n",
      "Pair{AbstractString,Int64}(\"detection. Recently ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"and floor were\", 1)\n",
      "Pair{AbstractString,Int64}(\"( median ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"the protease are\", 1)\n",
      "Pair{AbstractString,Int64}(\"5 - (\", 1)\n",
      "Pair{AbstractString,Int64}(\"by up to\", 1)\n",
      "Pair{AbstractString,Int64}(\"flaccid paralysis. 15\", 1)\n",
      "Pair{AbstractString,Int64}(\"mice that had\", 2)\n",
      "Pair{AbstractString,Int64}(\"is perturbed in\", 1)\n",
      "Pair{AbstractString,Int64}(\"drugs ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"glycoprotein , including\", 1)\n",
      "Pair{AbstractString,Int64}(\"we reconsider the\", 1)\n",
      "Pair{AbstractString,Int64}(\"in the CVB\", 2)\n",
      "Pair{AbstractString,Int64}(\", 2014 )\", 2)\n",
      "Pair{AbstractString,Int64}(\"Medicine [\", 1)\n",
      "Pair{AbstractString,Int64}(\"Five\", 1)\n",
      "Pair{AbstractString,Int64}(\"by Mantel-Cox\", 1)\n",
      "Pair{AbstractString,Int64}(\", doublet\", 1)\n",
      "Pair{AbstractString,Int64}(\"the thiohemiketal accepts\", 1)\n",
      "Pair{AbstractString,Int64}(\"the activating\", 2)\n",
      "Pair{AbstractString,Int64}(\"the region targeted\", 1)\n",
      "Pair{AbstractString,Int64}(\", such as\", 3)\n",
      "Pair{AbstractString,Int64}(\"Most coronaviral S\", 1)\n",
      "Pair{AbstractString,Int64}(\"we examined\", 3)\n",
      "Pair{AbstractString,Int64}(\"the amygdala\", 1)\n",
      "Pair{AbstractString,Int64}(\"> 2 ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"of APs\", 2)\n",
      "Pair{AbstractString,Int64}(\"conformation according to\", 1)\n",
      "Pair{AbstractString,Int64}(\"52 , 56\", 1)\n",
      "Pair{AbstractString,Int64}(\"288 T is\", 1)\n",
      "Pair{AbstractString,Int64}(\"at a concentration\", 3)\n",
      "Pair{AbstractString,Int64}(\"as Gly\", 1)\n",
      "Pair{AbstractString,Int64}(\"libitum.\", 1)\n",
      "Pair{AbstractString,Int64}(\"should take strict\", 1)\n",
      "Pair{AbstractString,Int64}(\"ratio after/before between\", 1)\n",
      "Pair{AbstractString,Int64}(\"we also developed\", 1)\n",
      "Pair{AbstractString,Int64}(\"295 ± 5\", 1)\n",
      "Pair{AbstractString,Int64}(\"newly\", 2)\n",
      "Pair{AbstractString,Int64}(\"target cells that\", 1)\n",
      "Pair{AbstractString,Int64}(\"reported 225\", 1)\n",
      "Pair{AbstractString,Int64}(\") cm/s for\", 1)\n",
      "Pair{AbstractString,Int64}(\"we observe\", 2)\n",
      "Pair{AbstractString,Int64}(\"lysed\", 6)\n",
      "Pair{AbstractString,Int64}(\"5 ,\", 4)\n",
      "Pair{AbstractString,Int64}(\"exception of C.\", 1)\n",
      "Pair{AbstractString,Int64}(\"as colorless oil.\", 1)\n",
      "Pair{AbstractString,Int64}(\"pocket in the\", 3)\n",
      "Pair{AbstractString,Int64}(\"EC 50 against\", 4)\n",
      "Pair{AbstractString,Int64}(\", statistical 587\", 1)\n",
      "Pair{AbstractString,Int64}(\"inferred relationships\", 1)\n",
      "Pair{AbstractString,Int64}(\"503 should have\", 1)\n",
      "Pair{AbstractString,Int64}(\"but in the\", 1)\n",
      "Pair{AbstractString,Int64}(\"204\", 1)\n",
      "Pair{AbstractString,Int64}(\"were generated ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"binds above the\", 1)\n",
      "Pair{AbstractString,Int64}(\"controls and the\", 1)\n",
      "Pair{AbstractString,Int64}(\"though it\", 1)\n",
      "Pair{AbstractString,Int64}(\"form clade\", 1)\n",
      "Pair{AbstractString,Int64}(\"04 versus Bat-SARr-CoV\", 1)\n",
      "Pair{AbstractString,Int64}(\", 91\", 2)\n",
      "Pair{AbstractString,Int64}(\"using BioEdit v\", 1)\n",
      "Pair{AbstractString,Int64}(\"12 h.\", 2)\n",
      "Pair{AbstractString,Int64}(\") - 4\", 1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"proteins\", 14)\n",
      "Pair{AbstractString,Int64}(\"the residues (\", 1)\n",
      "Pair{AbstractString,Int64}(\"different drops\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 53\", 2)\n",
      "Pair{AbstractString,Int64}(\"2019 , we\", 1)\n",
      "Pair{AbstractString,Int64}(\"the conditioned odor\", 3)\n",
      "Pair{AbstractString,Int64}(\"so ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"corresponding MHV\", 1)\n",
      "Pair{AbstractString,Int64}(\"MHV-induced loss\", 1)\n",
      "Pair{AbstractString,Int64}(\"are reported in\", 1)\n",
      "Pair{AbstractString,Int64}(\"competitor genomes\", 1)\n",
      "Pair{AbstractString,Int64}(\"µl OptiMEM (\", 1)\n",
      "Pair{AbstractString,Int64}(\"pro of\", 3)\n",
      "Pair{AbstractString,Int64}(\"of the NMDA\", 1)\n",
      "Pair{AbstractString,Int64}(\"Rh ) cells\", 1)\n",
      "Pair{AbstractString,Int64}(\": 12.5\", 1)\n",
      "Pair{AbstractString,Int64}(\"affect\", 3)\n",
      "Pair{AbstractString,Int64}(\"295 ±\", 1)\n",
      "Pair{AbstractString,Int64}(\"Wilcoxon matched-pairs signed\", 11)\n",
      "Pair{AbstractString,Int64}(\"solution consisting\", 1)\n",
      "Pair{AbstractString,Int64}(\"in Figure\", 1)\n",
      "Pair{AbstractString,Int64}(\"of restoring the\", 1)\n",
      "Pair{AbstractString,Int64}(\"highly recommended to\", 1)\n",
      "Pair{AbstractString,Int64}(\"for individual\", 1)\n",
      "Pair{AbstractString,Int64}(\"stock.\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 have\", 1)\n",
      "Pair{AbstractString,Int64}(\"other words\", 1)\n",
      "Pair{AbstractString,Int64}(\"reveal significant\", 1)\n",
      "Pair{AbstractString,Int64}(\"72.56\", 1)\n",
      "Pair{AbstractString,Int64}(\"Korea\", 2)\n",
      "Pair{AbstractString,Int64}(\"manipulations of\", 1)\n",
      "Pair{AbstractString,Int64}(\"samples. Preserved\", 1)\n",
      "Pair{AbstractString,Int64}(\"17 cm ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"curve.\", 1)\n",
      "Pair{AbstractString,Int64}(\"first ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"the culture for\", 1)\n",
      "Pair{AbstractString,Int64}(\"23 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"Asp\", 3)\n",
      "Pair{AbstractString,Int64}(\"( not shown\", 2)\n",
      "Pair{AbstractString,Int64}(\"ratio was as\", 1)\n",
      "Pair{AbstractString,Int64}(\"contains 309\", 1)\n",
      "Pair{AbstractString,Int64}(\"unrelated to the\", 1)\n",
      "Pair{AbstractString,Int64}(\"stronger\", 1)\n",
      "Pair{AbstractString,Int64}(\"its Supplementary Information\", 1)\n",
      "Pair{AbstractString,Int64}(\"alphacoronavirus proteases )\", 1)\n",
      "Pair{AbstractString,Int64}(\"drug cytotoxicity were\", 1)\n",
      "Pair{AbstractString,Int64}(\"LY 6 Emediated\", 1)\n",
      "Pair{AbstractString,Int64}(\"reduced in livers\", 1)\n",
      "Pair{AbstractString,Int64}(\"are synonymous ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"difference is\", 1)\n",
      "Pair{AbstractString,Int64}(\"box is normalized\", 1)\n",
      "Pair{AbstractString,Int64}(\"affected\", 1)\n",
      "Pair{AbstractString,Int64}(\"5 -fluorouracil sensitivity\", 1)\n",
      "Pair{AbstractString,Int64}(\"283 proteases\", 1)\n",
      "Pair{AbstractString,Int64}(\"After centrifuge for\", 2)\n",
      "Pair{AbstractString,Int64}(\"in KLK\", 1)\n",
      "Pair{AbstractString,Int64}(\". Slices\", 1)\n",
      "Pair{AbstractString,Int64}(\"main proteases\", 2)\n",
      "Pair{AbstractString,Int64}(\"8 ) .\", 4)\n",
      "Pair{AbstractString,Int64}(\"osmolarity )\", 1)\n",
      "Pair{AbstractString,Int64}(\"20 µM\", 1)\n",
      "Pair{AbstractString,Int64}(\"CS+ odor presentation\", 1)\n",
      "Pair{AbstractString,Int64}(\"i.e. in\", 1)\n",
      "Pair{AbstractString,Int64}(\"11 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"infectious 150\", 1)\n",
      "Pair{AbstractString,Int64}(\"8.7 µM\", 1)\n",
      "Pair{AbstractString,Int64}(\"cell 310\", 1)\n",
      "Pair{AbstractString,Int64}(\"indicating a\", 1)\n",
      "Pair{AbstractString,Int64}(\"( firefly or\", 1)\n",
      "Pair{AbstractString,Int64}(\"lysis buffer (\", 3)\n",
      "Pair{AbstractString,Int64}(\"of the IC\", 1)\n",
      "Pair{AbstractString,Int64}(\"patient ) were\", 1)\n",
      "Pair{AbstractString,Int64}(\"plasmid encoding\", 2)\n",
      "Pair{AbstractString,Int64}(\"to create sufficient\", 1)\n",
      "Pair{AbstractString,Int64}(\"-CVB\", 1)\n",
      "Pair{AbstractString,Int64}(\"Current was delivered\", 1)\n",
      "Pair{AbstractString,Int64}(\"24 h )\", 1)\n",
      "Pair{AbstractString,Int64}(\"( BHQ-\", 2)\n",
      "Pair{AbstractString,Int64}(\", W=- 36\", 2)\n",
      "Pair{AbstractString,Int64}(\"resulting in too\", 1)\n",
      "Pair{AbstractString,Int64}(\"observation suggests\", 1)\n",
      "Pair{AbstractString,Int64}(\"along the x-axis\", 1)\n",
      "Pair{AbstractString,Int64}(\"from host\", 1)\n",
      "Pair{AbstractString,Int64}(\"complex. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"mixed with 1\", 3)\n",
      "Pair{AbstractString,Int64}(\") and the\", 3)\n",
      "Pair{AbstractString,Int64}(\"COVID- 19 outbreak\", 1)\n",
      "Pair{AbstractString,Int64}(\"16 ,\", 4)\n",
      "Pair{AbstractString,Int64}(\"side-chain changes from\", 1)\n",
      "Pair{AbstractString,Int64}(\"processes (\", 1)\n",
      "Pair{AbstractString,Int64}(\"by far the\", 1)\n",
      "Pair{AbstractString,Int64}(\"measured one\", 1)\n",
      "Pair{AbstractString,Int64}(\"Dq-expressing cells\", 5)\n",
      "Pair{AbstractString,Int64}(\"be processed\", 1)\n",
      "Pair{AbstractString,Int64}(\"values of their\", 1)\n",
      "Pair{AbstractString,Int64}(\"human coronaviruses with\", 1)\n",
      "Pair{AbstractString,Int64}(\"BHK-G 43 cells\", 1)\n",
      "Pair{AbstractString,Int64}(\"nM ) and\", 1)\n",
      "Pair{AbstractString,Int64}(\"Multiple cell types\", 1)\n",
      "Pair{AbstractString,Int64}(\"centrifuge for 1\", 1)\n",
      "Pair{AbstractString,Int64}(\"show that the\", 1)\n",
      "Pair{AbstractString,Int64}(\"containing calcium\", 2)\n",
      "Pair{AbstractString,Int64}(\"Society of\", 1)\n",
      "Pair{AbstractString,Int64}(\"hence\", 1)\n",
      "Pair{AbstractString,Int64}(\"of antivirals against\", 1)\n",
      "Pair{AbstractString,Int64}(\"( pp )\", 1)\n",
      "Pair{AbstractString,Int64}(\"improvements\", 1)\n",
      "Pair{AbstractString,Int64}(\"only 3.36 %\", 1)\n",
      "Pair{AbstractString,Int64}(\"described the surprising\", 1)\n",
      "Pair{AbstractString,Int64}(\"permissive for\", 1)\n",
      "Pair{AbstractString,Int64}(\"mounted\", 1)\n",
      "Pair{AbstractString,Int64}(\"bioRxiv preprint memory\", 1)\n",
      "Pair{AbstractString,Int64}(\"cagcacccacgtgctgggcctcgggctcctgc\", 1)\n",
      "Pair{AbstractString,Int64}(\"unaffected\", 2)\n",
      "Pair{AbstractString,Int64}(\"8 ( 366\", 1)\n",
      "Pair{AbstractString,Int64}(\"above. Ly\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 3.91 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"reveal significant increases\", 1)\n",
      "Pair{AbstractString,Int64}(\"restriction factor\", 2)\n",
      "Pair{AbstractString,Int64}(\"pH 5.5\", 1)\n",
      "Pair{AbstractString,Int64}(\"E and H\", 1)\n",
      "Pair{AbstractString,Int64}(\"IVI ,\", 1)\n",
      "Pair{AbstractString,Int64}(\". According\", 1)\n",
      "Pair{AbstractString,Int64}(\"copyright holder\", 147)\n",
      "Pair{AbstractString,Int64}(\"( 4 ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"fear\", 34)\n",
      "Pair{AbstractString,Int64}(\"general time-reversible\", 1)\n",
      "Pair{AbstractString,Int64}(\"the necessary\", 1)\n",
      "Pair{AbstractString,Int64}(\"https://www.ncbi.nlm.nih.gov/geo\", 1)\n",
      "Pair{AbstractString,Int64}(\"terminus\", 1)\n",
      "Pair{AbstractString,Int64}(\"additional a-helical\", 1)\n",
      "Pair{AbstractString,Int64}(\"3 -and\", 1)\n",
      "Pair{AbstractString,Int64}(\"membrane (\", 2)\n",
      "Pair{AbstractString,Int64}(\"521 competitor genomes\", 1)\n",
      "Pair{AbstractString,Int64}(\"13 were\", 1)\n",
      "Pair{AbstractString,Int64}(\"group inserted\", 1)\n",
      "Pair{AbstractString,Int64}(\"the paired\", 1)\n",
      "Pair{AbstractString,Int64}(\"with 5 mM\", 2)\n",
      "Pair{AbstractString,Int64}(\"after Bonferroni\", 1)\n",
      "Pair{AbstractString,Int64}(\"1 , 0.5\", 1)\n",
      "Pair{AbstractString,Int64}(\"This result is\", 1)\n",
      "Pair{AbstractString,Int64}(\". https://doi.org/\", 127)\n",
      "Pair{AbstractString,Int64}(\"traces. Artificial\", 1)\n",
      "Pair{AbstractString,Int64}(\"both experimental\", 1)\n",
      "Pair{AbstractString,Int64}(\"Datiche et al.\", 1)\n",
      "Pair{AbstractString,Int64}(\"( http://www.phylogeny.fr/ )\", 1)\n",
      "Pair{AbstractString,Int64}(\"that this may\", 1)\n",
      "Pair{AbstractString,Int64}(\"45 46\", 4)\n",
      "Pair{AbstractString,Int64}(\"DREADD expression ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"after injection\", 1)\n",
      "Pair{AbstractString,Int64}(\"Systemic genetic\", 1)\n",
      "Pair{AbstractString,Int64}(\"complexes ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"largely inactive\", 2)\n",
      "Pair{AbstractString,Int64}(\"by comprehensive comparative\", 1)\n",
      "Pair{AbstractString,Int64}(\"split\", 4)\n",
      "Pair{AbstractString,Int64}(\"infection before the\", 1)\n",
      "Pair{AbstractString,Int64}(\"mice\", 94)\n",
      "Pair{AbstractString,Int64}(\"increases ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"The medium was\", 1)\n",
      "Pair{AbstractString,Int64}(\"mutation rates in\", 2)\n",
      "Pair{AbstractString,Int64}(\"determine\", 16)\n",
      "Pair{AbstractString,Int64}(\"58 Six\", 1)\n",
      "Pair{AbstractString,Int64}(\"SARS-CoV- 2 WIV\", 4)\n",
      "Pair{AbstractString,Int64}(\"P 2 residue\", 4)\n",
      "Pair{AbstractString,Int64}(\"for liver and\", 1)\n",
      "Pair{AbstractString,Int64}(\"adjacent to\", 1)\n",
      "Pair{AbstractString,Int64}(\"portion of\", 1)\n",
      "Pair{AbstractString,Int64}(\"learned\", 12)\n",
      "Pair{AbstractString,Int64}(\"leads to\", 1)\n",
      "Pair{AbstractString,Int64}(\"activated during odor\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 -expressing\", 2)\n",
      "Pair{AbstractString,Int64}(\"pA\", 3)\n",
      "Pair{AbstractString,Int64}(\"individual viral\", 1)\n",
      "Pair{AbstractString,Int64}(\"The testing session\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 1.75 g\", 1)\n",
      "Pair{AbstractString,Int64}(\"other odors.\", 1)\n",
      "Pair{AbstractString,Int64}(\"Fig. 7 c\", 1)\n",
      "Pair{AbstractString,Int64}(\"and open\", 1)\n",
      "Pair{AbstractString,Int64}(\"fluorescent viability\", 1)\n",
      "Pair{AbstractString,Int64}(\"upon cell lysis\", 1)\n",
      "Pair{AbstractString,Int64}(\"thus make\", 1)\n",
      "Pair{AbstractString,Int64}(\"the SARS-CoV- 2\", 2)\n",
      "Pair{AbstractString,Int64}(\"\\uf0a2à\\uf0e0 5 ¢\", 1)\n",
      "Pair{AbstractString,Int64}(\"MEM Rega\", 1)\n",
      "Pair{AbstractString,Int64}(\"PEDV )\", 1)\n",
      "Pair{AbstractString,Int64}(\"elution\", 1)\n",
      "Pair{AbstractString,Int64}(\"and could\", 1)\n",
      "Pair{AbstractString,Int64}(\"did mice show\", 1)\n",
      "Pair{AbstractString,Int64}(\"the death\", 1)\n",
      "Pair{AbstractString,Int64}(\"the input (\", 1)\n",
      "Pair{AbstractString,Int64}(\"or maintenance\", 1)\n",
      "Pair{AbstractString,Int64}(\"sufficient binding energy.\", 1)\n",
      "Pair{AbstractString,Int64}(\"M ciclesonide\", 1)\n",
      "Pair{AbstractString,Int64}(\"competing interests.\", 1)\n",
      "Pair{AbstractString,Int64}(\"protease. A single\", 1)\n",
      "Pair{AbstractString,Int64}(\"( P 022\", 1)\n",
      "Pair{AbstractString,Int64}(\"zinc finger 221\", 1)\n",
      "Pair{AbstractString,Int64}(\"2015 ;\", 2)\n",
      "Pair{AbstractString,Int64}(\"CoV S\", 8)\n",
      "Pair{AbstractString,Int64}(\"leucine\", 3)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"due to\", 14)\n",
      "Pair{AbstractString,Int64}(\"and samples harvested\", 1)\n",
      "Pair{AbstractString,Int64}(\"M + H\", 2)\n",
      "Pair{AbstractString,Int64}(\"made of 72\", 1)\n",
      "Pair{AbstractString,Int64}(\"for protection\", 1)\n",
      "Pair{AbstractString,Int64}(\"than 95 %\", 2)\n",
      "Pair{AbstractString,Int64}(\"region were\", 1)\n",
      "Pair{AbstractString,Int64}(\"in obtaining\", 1)\n",
      "Pair{AbstractString,Int64}(\"with RaTG 13\", 1)\n",
      "Pair{AbstractString,Int64}(\") and CVB\", 1)\n",
      "Pair{AbstractString,Int64}(\"1.67 ) vs.\", 1)\n",
      "Pair{AbstractString,Int64}(\"behavior : the\", 1)\n",
      "Pair{AbstractString,Int64}(\"188 as well\", 2)\n",
      "Pair{AbstractString,Int64}(\"the replicons of\", 1)\n",
      "Pair{AbstractString,Int64}(\"70 ° C.\", 1)\n",
      "Pair{AbstractString,Int64}(\"The pellet suspension\", 1)\n",
      "Pair{AbstractString,Int64}(\"= p-fluorobenzyl\", 1)\n",
      "Pair{AbstractString,Int64}(\"solution respectively\", 1)\n",
      "Pair{AbstractString,Int64}(\"clarity. E.\", 1)\n",
      "Pair{AbstractString,Int64}(\"' s or\", 1)\n",
      "Pair{AbstractString,Int64}(\"titers were 183\", 1)\n",
      "Pair{AbstractString,Int64}(\"one side.\", 1)\n",
      "Pair{AbstractString,Int64}(\"concentrations of a-ketoamides\", 1)\n",
      "Pair{AbstractString,Int64}(\"For simplicity ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"testing. Z-scores ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"( but 9.8\", 1)\n",
      "Pair{AbstractString,Int64}(\"acyl chloride\", 2)\n",
      "Pair{AbstractString,Int64}(\"precipitant\", 2)\n",
      "Pair{AbstractString,Int64}(\"doi : 10.1093\", 2)\n",
      "Pair{AbstractString,Int64}(\"summer\", 1)\n",
      "Pair{AbstractString,Int64}(\"the 304\", 1)\n",
      "Pair{AbstractString,Int64}(\"afferent inputs\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibition of subgenomic\", 1)\n",
      "Pair{AbstractString,Int64}(\"is an important\", 1)\n",
      "Pair{AbstractString,Int64}(\"600 0.5\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 cells but\", 1)\n",
      "Pair{AbstractString,Int64}(\"tagging of\", 1)\n",
      "Pair{AbstractString,Int64}(\"Several free-enzyme\", 1)\n",
      "Pair{AbstractString,Int64}(\"the next step.\", 1)\n",
      "Pair{AbstractString,Int64}(\"a derivative with\", 1)\n",
      "Pair{AbstractString,Int64}(\"host proteases\", 2)\n",
      "Pair{AbstractString,Int64}(\"hydrolysis.\", 1)\n",
      "Pair{AbstractString,Int64}(\"dendritic\", 1)\n",
      "Pair{AbstractString,Int64}(\"The affinity of\", 1)\n",
      "Pair{AbstractString,Int64}(\"al. , 2013\", 2)\n",
      "Pair{AbstractString,Int64}(\"on improving\", 1)\n",
      "Pair{AbstractString,Int64}(\"was first dissolved\", 1)\n",
      "Pair{AbstractString,Int64}(\"RNA and genomic\", 1)\n",
      "Pair{AbstractString,Int64}(\"29,900\", 1)\n",
      "Pair{AbstractString,Int64}(\"cm/s for the\", 1)\n",
      "Pair{AbstractString,Int64}(\"250 236 consensus\", 1)\n",
      "Pair{AbstractString,Int64}(\"1.4.4 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"wild-type controls. Mice\", 1)\n",
      "Pair{AbstractString,Int64}(\"extract\", 1)\n",
      "Pair{AbstractString,Int64}(\"a 60\", 1)\n",
      "Pair{AbstractString,Int64}(\"301 assume\", 1)\n",
      "Pair{AbstractString,Int64}(\"120 , 140\", 1)\n",
      "Pair{AbstractString,Int64}(\"the one-digit micromolar\", 2)\n",
      "Pair{AbstractString,Int64}(\"perception , and\", 1)\n",
      "Pair{AbstractString,Int64}(\"HA-tagged LY 6\", 1)\n",
      "Pair{AbstractString,Int64}(\"carry out\", 1)\n",
      "Pair{AbstractString,Int64}(\"plasmid 55\", 1)\n",
      "Pair{AbstractString,Int64}(\"exposed alcohol\", 1)\n",
      "Pair{AbstractString,Int64}(\"impressively demonstrated by\", 1)\n",
      "Pair{AbstractString,Int64}(\"maximum-likelihood method\", 1)\n",
      "Pair{AbstractString,Int64}(\"Sigma F 6627\", 1)\n",
      "Pair{AbstractString,Int64}(\"preprint The copyright\", 8)\n",
      "Pair{AbstractString,Int64}(\"assembly ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 263\", 1)\n",
      "Pair{AbstractString,Int64}(\") down-field\", 1)\n",
      "Pair{AbstractString,Int64}(\") in RAxML\", 1)\n",
      "Pair{AbstractString,Int64}(\"CoV restriction factor\", 1)\n",
      "Pair{AbstractString,Int64}(\"DNA and 28\", 1)\n",
      "Pair{AbstractString,Int64}(\"beta-CoV ) as\", 1)\n",
      "Pair{AbstractString,Int64}(\"expressed in the\", 1)\n",
      "Pair{AbstractString,Int64}(\"condensation of\", 1)\n",
      "Pair{AbstractString,Int64}(\", although\", 3)\n",
      "Pair{AbstractString,Int64}(\"standard DNA\", 1)\n",
      "Pair{AbstractString,Int64}(\"of adenylate kinase\", 1)\n",
      "Pair{AbstractString,Int64}(\"the mouse\", 2)\n",
      "Pair{AbstractString,Int64}(\"µl )\", 1)\n",
      "Pair{AbstractString,Int64}(\"through evolution\", 1)\n",
      "Pair{AbstractString,Int64}(\"well above 15\", 1)\n",
      "Pair{AbstractString,Int64}(\"forming units (\", 1)\n",
      "Pair{AbstractString,Int64}(\"by reducing it.\", 1)\n",
      "Pair{AbstractString,Int64}(\"morphology\", 1)\n",
      "Pair{AbstractString,Int64}(\"activity drives\", 1)\n",
      "Pair{AbstractString,Int64}(\", 63 All\", 1)\n",
      "Pair{AbstractString,Int64}(\", and piriform\", 1)\n",
      "Pair{AbstractString,Int64}(\"E in\", 7)\n",
      "Pair{AbstractString,Int64}(\"Zoom microscope\", 1)\n",
      "Pair{AbstractString,Int64}(\"and cellular membranes\", 2)\n",
      "Pair{AbstractString,Int64}(\"spared\", 1)\n",
      "Pair{AbstractString,Int64}(\"VITROS\", 1)\n",
      "Pair{AbstractString,Int64}(\"by HRV ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"emergence and for\", 1)\n",
      "Pair{AbstractString,Int64}(\"been proposed as\", 1)\n",
      "Pair{AbstractString,Int64}(\"2.20\", 2)\n",
      "Pair{AbstractString,Int64}(\"isobutyl (\", 5)\n",
      "Pair{AbstractString,Int64}(\"we should take\", 1)\n",
      "Pair{AbstractString,Int64}(\"13 The enteroviruses\", 1)\n",
      "Pair{AbstractString,Int64}(\"statistical\", 6)\n",
      "Pair{AbstractString,Int64}(\"< 0.05 )\", 2)\n",
      "Pair{AbstractString,Int64}(\"( A. Kupke\", 1)\n",
      "Pair{AbstractString,Int64}(\"largest insert segment\", 1)\n",
      "Pair{AbstractString,Int64}(\"discussed in this\", 1)\n",
      "Pair{AbstractString,Int64}(\"mild respiratory\", 1)\n",
      "Pair{AbstractString,Int64}(\"expand the\", 1)\n",
      "Pair{AbstractString,Int64}(\"here ;\", 1)\n",
      "Pair{AbstractString,Int64}(\"from site to\", 1)\n",
      "Pair{AbstractString,Int64}(\"laboratories in\", 1)\n",
      "Pair{AbstractString,Int64}(\"min. After\", 1)\n",
      "Pair{AbstractString,Int64}(\"which the activity-dependent\", 1)\n",
      "Pair{AbstractString,Int64}(\"CNO ( see\", 2)\n",
      "Pair{AbstractString,Int64}(\"( adjusted to\", 1)\n",
      "Pair{AbstractString,Int64}(\"revealed that\", 4)\n",
      "Pair{AbstractString,Int64}(\"follow that this\", 1)\n",
      "Pair{AbstractString,Int64}(\", 86\", 1)\n",
      "Pair{AbstractString,Int64}(\"inhibit KLK\", 1)\n",
      "Pair{AbstractString,Int64}(\"20 times higher\", 1)\n",
      "Pair{AbstractString,Int64}(\"10 b\", 1)\n",
      "Pair{AbstractString,Int64}(\"41 42\", 4)\n",
      "Pair{AbstractString,Int64}(\"is connected with\", 1)\n",
      "Pair{AbstractString,Int64}(\"600 µL\", 1)\n",
      "Pair{AbstractString,Int64}(\"RNA were\", 1)\n",
      "Pair{AbstractString,Int64}(\"be lethal\", 1)\n",
      "Pair{AbstractString,Int64}(\"significantly after injection\", 1)\n",
      "Pair{AbstractString,Int64}(\"using serial dilutions\", 1)\n",
      "Pair{AbstractString,Int64}(\"microtome ( Campden\", 1)\n",
      "Pair{AbstractString,Int64}(\"% MTS\", 1)\n",
      "Pair{AbstractString,Int64}(\"pro . Note\", 1)\n",
      "Pair{AbstractString,Int64}(\"Expression of select\", 1)\n",
      "Pair{AbstractString,Int64}(\"there is an\", 1)\n",
      "Pair{AbstractString,Int64}(\"increased replication of\", 1)\n",
      "Pair{AbstractString,Int64}(\"Values\", 1)\n",
      "Pair{AbstractString,Int64}(\"the U\", 1)\n",
      "Pair{AbstractString,Int64}(\"In conclusion\", 3)\n",
      "Pair{AbstractString,Int64}(\"purified pro-KLK\", 1)\n",
      "Pair{AbstractString,Int64}(\"a The copyright\", 5)\n",
      "Pair{AbstractString,Int64}(\"viruses were\", 4)\n",
      "Pair{AbstractString,Int64}(\"perception of the\", 2)\n",
      "Pair{AbstractString,Int64}(\"half-side\", 1)\n",
      "Pair{AbstractString,Int64}(\", 481 respectively\", 1)\n",
      "Pair{AbstractString,Int64}(\"activating DREADD\", 1)\n",
      "Pair{AbstractString,Int64}(\"on mCitrine (\", 1)\n",
      "Pair{AbstractString,Int64}(\"exposure without\", 1)\n",
      "Pair{AbstractString,Int64}(\"( ω\", 1)\n",
      "Pair{AbstractString,Int64}(\"tm\", 1)\n",
      "Pair{AbstractString,Int64}(\"at 37 °\", 8)\n",
      "Pair{AbstractString,Int64}(\"frozen. Total RNA\", 1)\n",
      "Pair{AbstractString,Int64}(\"protect the\", 1)\n",
      "Pair{AbstractString,Int64}(\"causes sub-lethal hepatitis\", 1)\n",
      "Pair{AbstractString,Int64}(\"34 )\", 1)\n",
      "Pair{AbstractString,Int64}(\"366 nt )\", 1)\n",
      "Pair{AbstractString,Int64}(\". Instead of\", 1)\n",
      "Pair{AbstractString,Int64}(\"alkyl substituents\", 1)\n",
      "Pair{AbstractString,Int64}(\"covered \\\"\", 1)\n",
      "Pair{AbstractString,Int64}(\", 12 or\", 1)\n",
      "Pair{AbstractString,Int64}(\"compounds 6 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"RSCUdist : the\", 1)\n",
      "Pair{AbstractString,Int64}(\"the control and\", 7)\n",
      "Pair{AbstractString,Int64}(\"DREADDnegative cells\", 2)\n",
      "Pair{AbstractString,Int64}(\"= Boc\", 1)\n",
      "Pair{AbstractString,Int64}(\"we performed\", 5)\n",
      "Pair{AbstractString,Int64}(\"; www.who.int\", 1)\n",
      "Pair{AbstractString,Int64}(\"respiratory disease ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"45 - 51\", 1)\n",
      "Pair{AbstractString,Int64}(\"5.0 ; EC\", 1)\n",
      "Pair{AbstractString,Int64}(\"phenotype of\", 1)\n",
      "Pair{AbstractString,Int64}(\"scale\", 3)\n",
      "Pair{AbstractString,Int64}(\"preprint neutral\", 1)\n",
      "Pair{AbstractString,Int64}(\"( doi\", 2)\n",
      "Pair{AbstractString,Int64}(\"13 for virus\", 1)\n",
      "Pair{AbstractString,Int64}(\"replaced by a\", 1)\n",
      "Pair{AbstractString,Int64}(\"µL of the\", 1)\n",
      "Pair{AbstractString,Int64}(\"health may\", 1)\n",
      "Pair{AbstractString,Int64}(\"20 min\", 2)\n",
      "Pair{AbstractString,Int64}(\", P 2\", 5)\n",
      "Pair{AbstractString,Int64}(\"154 KLK\", 1)\n",
      "Pair{AbstractString,Int64}(\"mock infection\", 1)\n",
      "Pair{AbstractString,Int64}(\"failed to robustly\", 1)\n",
      "Pair{AbstractString,Int64}(\"42 published\", 1)\n",
      "Pair{AbstractString,Int64}(\"of distributed ensembles\", 1)\n",
      "Pair{AbstractString,Int64}(\"the capped tripeptidyl\", 1)\n",
      "Pair{AbstractString,Int64}(\"oxygen accepts\", 1)\n",
      "Pair{AbstractString,Int64}(\"nsp 10 and\", 1)\n",
      "Pair{AbstractString,Int64}(\"selection on the\", 1)\n",
      "Pair{AbstractString,Int64}(\"stainless-steel rods (\", 1)\n",
      "Pair{AbstractString,Int64}(\"197 Obtained results\", 1)\n",
      "Pair{AbstractString,Int64}(\"problems with\", 1)\n",
      "Pair{AbstractString,Int64}(\"to excellent\", 1)\n",
      "Pair{AbstractString,Int64}(\"to SARS ,\", 1)\n",
      "Pair{AbstractString,Int64}(\". Hits\", 1)\n",
      "Pair{AbstractString,Int64}(\"RNA viruses that\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 4.0 g\", 1)\n",
      "Pair{AbstractString,Int64}(\"mortality.\", 1)\n",
      "Pair{AbstractString,Int64}(\"inoculated onto\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 0 to\", 1)\n",
      "Pair{AbstractString,Int64}(\"/CD 32\", 1)\n",
      "Pair{AbstractString,Int64}(\"285 TMPRSS 2\", 1)\n",
      "Pair{AbstractString,Int64}(\"× DNA\", 1)\n",
      "Pair{AbstractString,Int64}(\"DBT cells with\", 1)\n",
      "Pair{AbstractString,Int64}(\"kHz , with\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 2 times\", 3)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pair{AbstractString,Int64}(\"has 258 profound\", 1)\n",
      "Pair{AbstractString,Int64}(\"of CoV\", 3)\n",
      "Pair{AbstractString,Int64}(\"39 40 41\", 4)\n",
      "Pair{AbstractString,Int64}(\"µM , whereas\", 1)\n",
      "Pair{AbstractString,Int64}(\", assessed by\", 1)\n",
      "Pair{AbstractString,Int64}(\"recent studies have\", 1)\n",
      "Pair{AbstractString,Int64}(\"been modeled\", 1)\n",
      "Pair{AbstractString,Int64}(\"them showed better\", 1)\n",
      "Pair{AbstractString,Int64}(\"one CS+\", 1)\n",
      "Pair{AbstractString,Int64}(\"against HRV 2\", 4)\n",
      "Pair{AbstractString,Int64}(\"( j ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"RiboMax™ Large-Scale\", 1)\n",
      "Pair{AbstractString,Int64}(\", remdesivir\", 1)\n",
      "Pair{AbstractString,Int64}(\"Testing The\", 1)\n",
      "Pair{AbstractString,Int64}(\"times at (\", 1)\n",
      "Pair{AbstractString,Int64}(\"NaCl and 5\", 1)\n",
      "Pair{AbstractString,Int64}(\"ns p= 0.1544\", 1)\n",
      "Pair{AbstractString,Int64}(\"Shimadzu Corporation )\", 1)\n",
      "Pair{AbstractString,Int64}(\"To explore\", 1)\n",
      "Pair{AbstractString,Int64}(\"rank test\", 12)\n",
      "Pair{AbstractString,Int64}(\"N. Subgenomic\", 1)\n",
      "Pair{AbstractString,Int64}(\"1.8\", 3)\n",
      "Pair{AbstractString,Int64}(\"49 (\", 1)\n",
      "Pair{AbstractString,Int64}(\", demonstrates that\", 1)\n",
      "Pair{AbstractString,Int64}(\"treatment for\", 1)\n",
      "Pair{AbstractString,Int64}(\"high casefatality rate\", 1)\n",
      "Pair{AbstractString,Int64}(\"% PEG\", 7)\n",
      "Pair{AbstractString,Int64}(\"burden and\", 1)\n",
      "Pair{AbstractString,Int64}(\"see Tables\", 1)\n",
      "Pair{AbstractString,Int64}(\"of column\", 1)\n",
      "Pair{AbstractString,Int64}(\"ΔHSC mice exhibited\", 2)\n",
      "Pair{AbstractString,Int64}(\"were delivered using\", 1)\n",
      "Pair{AbstractString,Int64}(\"variants are\", 1)\n",
      "Pair{AbstractString,Int64}(\"triggered\", 2)\n",
      "Pair{AbstractString,Int64}(\"undergone author/funder. All\", 1)\n",
      "Pair{AbstractString,Int64}(\"Fig. S 2\", 2)\n",
      "Pair{AbstractString,Int64}(\"p= 3.946 x\", 1)\n",
      "Pair{AbstractString,Int64}(\"Ctcompetitor -Ctreference\", 1)\n",
      "Pair{AbstractString,Int64}(\"difference is that\", 1)\n",
      "Pair{AbstractString,Int64}(\"protease inhibitor Kazal-type\", 1)\n",
      "Pair{AbstractString,Int64}(\"enzymes\", 5)\n",
      "Pair{AbstractString,Int64}(\"a significant improvement\", 1)\n",
      "Pair{AbstractString,Int64}(\"correlation distance measurement.\", 1)\n",
      "Pair{AbstractString,Int64}(\"spleen were preserved\", 1)\n",
      "Pair{AbstractString,Int64}(\"from HAE 359\", 1)\n",
      "Pair{AbstractString,Int64}(\"cortex ,\", 2)\n",
      "Pair{AbstractString,Int64}(\"in Extended Data\", 2)\n",
      "Pair{AbstractString,Int64}(\"M ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"odor stimulus\", 1)\n",
      "Pair{AbstractString,Int64}(\"pathogens. Antiviral membrane\", 1)\n",
      "Pair{AbstractString,Int64}(\"In nearly\", 1)\n",
      "Pair{AbstractString,Int64}(\"limited\", 1)\n",
      "Pair{AbstractString,Int64}(\"is invariably isobutyl\", 1)\n",
      "Pair{AbstractString,Int64}(\"memory retrieval. Its\", 1)\n",
      "Pair{AbstractString,Int64}(\"did not display\", 1)\n",
      "Pair{AbstractString,Int64}(\"were acquired\", 3)\n",
      "Pair{AbstractString,Int64}(\"cells. To\", 1)\n",
      "Pair{AbstractString,Int64}(\"were soaked in\", 1)\n",
      "Pair{AbstractString,Int64}(\"of ExoN-AE makes\", 1)\n",
      "Pair{AbstractString,Int64}(\"I ExoN\", 1)\n",
      "Pair{AbstractString,Int64}(\"further\", 21)\n",
      "Pair{AbstractString,Int64}(\". Approximately\", 1)\n",
      "Pair{AbstractString,Int64}(\"50 = 2.1\", 1)\n",
      "Pair{AbstractString,Int64}(\"methionines.\", 1)\n",
      "Pair{AbstractString,Int64}(\"the RSCU distance\", 1)\n",
      "Pair{AbstractString,Int64}(\"RNA isolated from\", 1)\n",
      "Pair{AbstractString,Int64}(\"with SalI or\", 1)\n",
      "Pair{AbstractString,Int64}(\"Transfection.\", 1)\n",
      "Pair{AbstractString,Int64}(\"Di-expressing piriform\", 1)\n",
      "Pair{AbstractString,Int64}(\"reference to an\", 3)\n",
      "Pair{AbstractString,Int64}(\"low micromolar\", 1)\n",
      "Pair{AbstractString,Int64}(\"immune cells\", 5)\n",
      "Pair{AbstractString,Int64}(\"5 D\", 1)\n",
      "Pair{AbstractString,Int64}(\", all the\", 1)\n",
      "Pair{AbstractString,Int64}(\"dose-dependent\", 1)\n",
      "Pair{AbstractString,Int64}(\"] In-vitro RNA\", 1)\n",
      "Pair{AbstractString,Int64}(\"18 ( 5\", 1)\n",
      "Pair{AbstractString,Int64}(\"Plasmid constructs 358\", 1)\n",
      "Pair{AbstractString,Int64}(\"as auto-associative networks\", 1)\n",
      "Pair{AbstractString,Int64}(\"on the role\", 1)\n",
      "Pair{AbstractString,Int64}(\"parent compound\", 1)\n",
      "Pair{AbstractString,Int64}(\"analyses to\", 1)\n",
      "Pair{AbstractString,Int64}(\"MERS-CoV pp\", 1)\n",
      "Pair{AbstractString,Int64}(\"of this study\", 1)\n",
      "Pair{AbstractString,Int64}(\"positive selection\", 3)\n",
      "Pair{AbstractString,Int64}(\"and the position\", 1)\n",
      "Pair{AbstractString,Int64}(\"substituent , whereas\", 1)\n",
      "Pair{AbstractString,Int64}(\"Biosciences\", 4)\n",
      "Pair{AbstractString,Int64}(\"ablation on global\", 1)\n",
      "Pair{AbstractString,Int64}(\"a RNA\", 1)\n",
      "Pair{AbstractString,Int64}(\"our data indicate\", 1)\n",
      "Pair{AbstractString,Int64}(\"and inserted\", 1)\n",
      "Pair{AbstractString,Int64}(\"bis\", 2)\n",
      "Pair{AbstractString,Int64}(\"Enrichment Analyses for\", 1)\n",
      "Pair{AbstractString,Int64}(\"templates\", 10)\n",
      "Pair{AbstractString,Int64}(\"catalytic His\", 4)\n",
      "Pair{AbstractString,Int64}(\"the condensation of\", 1)\n",
      "Pair{AbstractString,Int64}(\"the basis of\", 1)\n",
      "Pair{AbstractString,Int64}(\"is true for\", 1)\n",
      "Pair{AbstractString,Int64}(\"Stettlen ,\", 1)\n",
      "Pair{AbstractString,Int64}(\"An estimate of\", 1)\n",
      "Pair{AbstractString,Int64}(\"The benzyl therefore\", 1)\n",
      "Pair{AbstractString,Int64}(\"overlaying 430 All\", 1)\n",
      "Pair{AbstractString,Int64}(\"Renilla Luciferase\", 4)\n",
      "Pair{AbstractString,Int64}(\"the Middle\", 1)\n",
      "Pair{AbstractString,Int64}(\"after HCoV-HKU\", 1)\n",
      "Pair{AbstractString,Int64}(\"functions that\", 1)\n",
      "Pair{AbstractString,Int64}(\"window before and\", 1)\n",
      "Pair{AbstractString,Int64}(\"an ExoN\", 1)\n",
      "Pair{AbstractString,Int64}(\"in the olfactory\", 2)\n",
      "Pair{AbstractString,Int64}(\"4 Di-positive\", 1)\n",
      "Pair{AbstractString,Int64}(\"TT ( pdU\", 1)\n",
      "Pair{AbstractString,Int64}(\"by equivalent depolarization\", 1)\n",
      "Pair{AbstractString,Int64}(\"a projected\", 1)\n",
      "Pair{AbstractString,Int64}(\"cells 197 Obtained\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 -DCt competitor\", 1)\n",
      "Pair{AbstractString,Int64}(\"filled outline of\", 1)\n",
      "Pair{AbstractString,Int64}(\"cyanomethyl )\", 1)\n",
      "Pair{AbstractString,Int64}(\"For details regarding\", 1)\n",
      "Pair{AbstractString,Int64}(\"tested using semi-\", 1)\n",
      "Pair{AbstractString,Int64}(\"structural\", 2)\n",
      "Pair{AbstractString,Int64}(\"previously with slight\", 1)\n",
      "Pair{AbstractString,Int64}(\"( 2\", 20)\n",
      "Pair{AbstractString,Int64}(\"IHIC and\", 1)\n",
      "Pair{AbstractString,Int64}(\"that learned escape\", 1)\n",
      "Pair{AbstractString,Int64}(\"urgent\", 1)\n",
      "Pair{AbstractString,Int64}(\"single capped dipeptidyl\", 1)\n",
      "Pair{AbstractString,Int64}(\"of mice\", 18)\n",
      "Pair{AbstractString,Int64}(\"The methyl amino-acid\", 1)\n",
      "Pair{AbstractString,Int64"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Excessive output truncated after 524289 bytes."
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "}(\"These results\", 2)\n",
      "Pair{AbstractString,Int64}(\"determined by plaque\", 3)\n",
      "Pair{AbstractString,Int64}(\"only ~\", 1)\n",
      "Pair{AbstractString,Int64}(\"of cellular\", 2)\n",
      "Pair{AbstractString,Int64}(\"supernatant genomic\", 2)\n",
      "Pair{AbstractString,Int64}(\"to 18\", 1)\n",
      "Pair{AbstractString,Int64}(\"and 218\", 1)\n",
      "Pair{AbstractString,Int64}(\"members as\", 1)\n",
      "Pair{AbstractString,Int64}(\"alone. The\", 1)\n",
      "Pair{AbstractString,Int64}(\"and native\", 1)\n",
      "Pair{AbstractString,Int64}(\", P\", 8)\n",
      "Pair{AbstractString,Int64}(\"be expected.\", 1)\n",
      "Pair{AbstractString,Int64}(\"and compound 11\", 1)\n",
      "Pair{AbstractString,Int64}(\", 254\", 1)\n",
      "Pair{AbstractString,Int64}(\"is easily explained\", 1)\n",
      "Pair{AbstractString,Int64}(\"the MEME analysis\", 1)\n",
      "Pair{AbstractString,Int64}(\"by incubating\", 1)\n",
      "Pair{AbstractString,Int64}(\"and incubated the\", 1)\n",
      "Pair{AbstractString,Int64}(\"± 0.08 (\", 1)\n",
      "Pair{AbstractString,Int64}(\"cRPMI ) was\", 1)\n",
      "Pair{AbstractString,Int64}(\"in which piriform\", 1)\n",
      "Pair{AbstractString,Int64}(\"expanding the\", 1)\n",
      "Pair{AbstractString,Int64}(\"7 -EV 71\", 1)\n",
      "Pair{AbstractString,Int64}(\"a therapeutic index.\", 1)\n",
      "Pair{AbstractString,Int64}(\"] . False\", 1)\n",
      "Pair{AbstractString,Int64}(\"to KLK\", 1)\n",
      "Pair{AbstractString,Int64}(\"take 303 part\", 1)\n",
      "Pair{AbstractString,Int64}(\"only 30 -\", 1)\n",
      "Pair{AbstractString,Int64}(\"long-term passage 91\", 1)\n",
      "Pair{AbstractString,Int64}(\"of RNA\", 7)\n",
      "Pair{AbstractString,Int64}(\"an important factor\", 1)\n",
      "Pair{AbstractString,Int64}(\"2 -substituents\", 1)\n",
      "Pair{AbstractString,Int64}(\"to \\\" The\", 1)\n",
      "Pair{AbstractString,Int64}(\"substitutions per non-syn"
     ]
    }
   ],
   "source": [
    "# Trigram\n",
    "for trigram in ngrams(sd1,3)\n",
    "    println(trigram)\n",
    "end"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[32m\u001b[1m  Resolving\u001b[22m\u001b[39m package versions...\n",
      "\u001b[32m\u001b[1m   Updating\u001b[22m\u001b[39m `C:\\Users\\Chandrapal Panwar\\.julia\\environments\\v1.4\\Project.toml`\n",
      "\u001b[90m [no changes]\u001b[39m\n",
      "\u001b[32m\u001b[1m   Updating\u001b[22m\u001b[39m `C:\\Users\\Chandrapal Panwar\\.julia\\environments\\v1.4\\Manifest.toml`\n",
      "\u001b[90m [no changes]\u001b[39m\n",
      "┌ Info: Running `conda install -y nltk` in root environment\n",
      "└ @ Conda C:\\Users\\Chandrapal Panwar\\.julia\\packages\\Conda\\3rPhK\\src\\Conda.jl:113\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting package metadata (current_repodata.json): ...working... done\n",
      "Solving environment: ...working... done\n",
      "\n",
      "# All requested packages already installed.\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "┌ Info: Precompiling PyCall [438e738f-606a-5dbb-bf0a-cddfbfd45ab0]\n",
      "└ @ Base loading.jl:1260\n"
     ]
    }
   ],
   "source": [
    "Pkg.add(\"Conda\")\n",
    "using Conda\n",
    "Conda.add(\"nltk\")\n",
    "using PyCall\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "┌ Warning: `@pyimport foo` is deprecated in favor of `foo = pyimport(\"foo\")`.\n",
      "│   caller = _pywrap_pyimport(::PyObject) at PyCall.jl:399\n",
      "└ @ PyCall C:\\Users\\Chandrapal Panwar\\.julia\\packages\\PyCall\\zqDXB\\src\\PyCall.jl:399\n"
     ]
    }
   ],
   "source": [
    "@pyimport nltk.tag as ptag"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "FileDocument(\"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\", TextAnalysis.DocumentMetadata(Languages.English(), \"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\", \"Unknown Author\", \"Unknown Time\"))"
      ]
     },
     "execution_count": 46,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sd1 = FileDocument(\"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "45119-element Array{String,1}:\n",
       " \"Odor\"\n",
       " \"perception\"\n",
       " \",\"\n",
       " \"and\"\n",
       " \"emotional\"\n",
       " \"and\"\n",
       " \"behavioral\"\n",
       " \"responses\"\n",
       " \"to\"\n",
       " \"odors\"\n",
       " \"strongly\"\n",
       " \"depend\"\n",
       " \"on\"\n",
       " ⋮\n",
       " \"the\"\n",
       " \"author/funder.\"\n",
       " \".\"\n",
       " \"https://doi.org/\"\n",
       " \"10.1101\"\n",
       " \"/\"\n",
       " \"175562\"\n",
       " \"doi\"\n",
       " \":\"\n",
       " \"bioRxiv\"\n",
       " \"preprint\"\n",
       " \".\""
      ]
     },
     "execution_count": 47,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# TextAnalysis.tokens()\n",
    "mytokens = tokens(sd1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "45119-element Array{Tuple{String,String},1}:\n",
       " (\"Odor\", \"NNP\")\n",
       " (\"perception\", \"NN\")\n",
       " (\",\", \",\")\n",
       " (\"and\", \"CC\")\n",
       " (\"emotional\", \"JJ\")\n",
       " (\"and\", \"CC\")\n",
       " (\"behavioral\", \"JJ\")\n",
       " (\"responses\", \"NNS\")\n",
       " (\"to\", \"TO\")\n",
       " (\"odors\", \"NNS\")\n",
       " (\"strongly\", \"RB\")\n",
       " (\"depend\", \"VBP\")\n",
       " (\"on\", \"IN\")\n",
       " ⋮\n",
       " (\"the\", \"DT\")\n",
       " (\"author/funder.\", \"NN\")\n",
       " (\".\", \".\")\n",
       " (\"https://doi.org/\", \"VB\")\n",
       " (\"10.1101\", \"CD\")\n",
       " (\"/\", \"JJ\")\n",
       " (\"175562\", \"CD\")\n",
       " (\"doi\", \"NN\")\n",
       " (\":\", \":\")\n",
       " (\"bioRxiv\", \"NN\")\n",
       " (\"preprint\", \"NN\")\n",
       " (\".\", \".\")"
      ]
     },
     "execution_count": 48,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Using NLTK tags for finding the part of speech of our tokens\n",
    "ptag.pos_tag(mytokens)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[32m\u001b[1m  Resolving\u001b[22m\u001b[39m package versions...\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m Clustering ─────── v0.14.0\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m NearestNeighbors ─ v0.4.4\n",
      "\u001b[32m\u001b[1m  Installed\u001b[22m\u001b[39m Distances ──────── v0.8.2\n",
      "\u001b[32m\u001b[1m   Updating\u001b[22m\u001b[39m `C:\\Users\\Chandrapal Panwar\\.julia\\environments\\v1.4\\Project.toml`\n",
      " \u001b[90m [aaaa29a8]\u001b[39m\u001b[92m + Clustering v0.14.0\u001b[39m\n",
      "\u001b[32m\u001b[1m   Updating\u001b[22m\u001b[39m `C:\\Users\\Chandrapal Panwar\\.julia\\environments\\v1.4\\Manifest.toml`\n",
      " \u001b[90m [aaaa29a8]\u001b[39m\u001b[92m + Clustering v0.14.0\u001b[39m\n",
      " \u001b[90m [b4f34e82]\u001b[39m\u001b[92m + Distances v0.8.2\u001b[39m\n",
      " \u001b[90m [b8a86587]\u001b[39m\u001b[92m + NearestNeighbors v0.4.4\u001b[39m\n"
     ]
    }
   ],
   "source": [
    "#Pkg.add(\"Clustering\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [],
   "source": [
    "using TextAnalysis.Languages "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "FileDocument(\"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\", TextAnalysis.DocumentMetadata(Languages.English(), \"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\", \"Unknown Author\", \"Unknown Time\"))"
      ]
     },
     "execution_count": 53,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sd1 = FileDocument(\"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "1952-element Array{SubString{String},1}:\n",
       " \"Odor perception, and emotional and behavioral responses to odors strongly depend on experience, and learned odor-context associations often last for the lifetime of an animal (Mouly and Sullivan, 2010) .\"\n",
       " \"The cellular and neural circuit mechanisms underlying olfactory learning and memory, however, remain poorly understood.\"\n",
       " \"\"\n",
       " \" Recent studies on episodic and contextual learning in hippocampal neural networks have suggested that memories are encoded in the activity of distributed ensembles of neurons, often referred to as a 'memory trace' (Mayford and Reijmers, 2015; Poo et al., 2016; Tonegawa et al., 2015) .\"\n",
       " \"The neurons constituting such a memory trace are thought to encode information about the environmental context and associated emotions of past experiences, and their activity is necessary and sufficient for memory retrieval (Liu et al., 2012; Reijmers et al., 2007; Tanaka et al., 2014) .\"\n",
       " \"\"\n",
       " \" Here, we investigate the organization of odor memory traces in the olfactory (piriform) cortex of mice.\"\n",
       " \"The piriform cortex, a trilaminar paleocortical structure, is the largest cortical area receiving direct afferent inputs from the olfactory bulb, which, in turn, receives topographically organized inputs from olfactory sensory neurons in the nose.\"\n",
       " \"Individual piriform neurons respond to combinatorial inputs from multiple Piriform cortex has long been hypothesized to support auto-associative network functions that can retrieve previously learned information from partial or degraded sensory inputs (Haberly, 2001; Wilson and Sullivan, 2011) .\"\n",
       " \"Piriform pyramidal cells form a large recurrent network, which is reciprocally connected with adjacent highorder associative areas including the prefrontal, entorhinal and perirhinal cortex and the amygdala (Johnson et al., 2000; Sadrian and Wilson, 2015) .\"\n",
       " \"Storage of information is made possible by NMDA-dependent, associative plasticity of connections (Johenning et al., 2009; Kanter and Haberly, 1990; Quinlan et al., 2004) .\"\n",
       " \"\"\n",
       " \" Furthermore, changes in piriform network activity (Chapuis and Wilson, 2011; Chen et al., 2011; Li et al., 2008; Sevelinges et al., 2004) and stabilization of piriform odor representations (Shakhawat et al., 2015) have been observed after associative olfactory learning.\"\n",
       " ⋮\n",
       " \"All rights reserved.\"\n",
       " \"No reuse allowed without permission.\"\n",
       " \"\"\n",
       " \" The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.\"\n",
       " \"Virus titer (log 10 PFU/ml) All rights reserved.\"\n",
       " \"No reuse allowed without permission.\"\n",
       " \"\"\n",
       " \" The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/175562 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/175562 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/175562 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.\"\n",
       " \"The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.\"\n",
       " \"The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.\"\n",
       " \"The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.. https://doi.org/10.1101/175562 doi: bioRxiv preprint.\"\n",
       " \"\""
      ]
     },
     "execution_count": 68,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "t=split_sentences(text(sd1))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "metadata": {},
   "outputs": [
    {
     "ename": "MethodError",
     "evalue": "MethodError: no method matching text(::Array{SubString{String},1})\nClosest candidates are:\n  text(!Matched::FileDocument) at C:\\Users\\Chandrapal Panwar\\.julia\\packages\\TextAnalysis\\pcFQf\\src\\document.jl:105\n  text(!Matched::StringDocument) at C:\\Users\\Chandrapal Panwar\\.julia\\packages\\TextAnalysis\\pcFQf\\src\\document.jl:109\n  text(!Matched::TokenDocument) at C:\\Users\\Chandrapal Panwar\\.julia\\packages\\TextAnalysis\\pcFQf\\src\\document.jl:111\n  ...",
     "output_type": "error",
     "traceback": [
      "MethodError: no method matching text(::Array{SubString{String},1})\nClosest candidates are:\n  text(!Matched::FileDocument) at C:\\Users\\Chandrapal Panwar\\.julia\\packages\\TextAnalysis\\pcFQf\\src\\document.jl:105\n  text(!Matched::StringDocument) at C:\\Users\\Chandrapal Panwar\\.julia\\packages\\TextAnalysis\\pcFQf\\src\\document.jl:109\n  text(!Matched::TokenDocument) at C:\\Users\\Chandrapal Panwar\\.julia\\packages\\TextAnalysis\\pcFQf\\src\\document.jl:111\n  ...",
      "",
      "Stacktrace:",
      " [1] top-level scope at .\\In[62]:1"
     ]
    }
   ],
   "source": [
    "# remove punctuation\n",
    "for sentence in split_sentences(text(t))\n",
    "    t=prepare!(sentence, strip_punctuation)\n",
    "    println(\"Word token=> $t\")\n",
    "end\n",
    "#prepare!(t, strip_punctuation)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 84,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "TokenDocument{String}([\"To\", \"be\", \"or\", \"not\", \"to\", \"be\"], TextAnalysis.DocumentMetadata(Languages.English(), \"Untitled Document\", \"Unknown Author\", \"Unknown Time\"))"
      ]
     },
     "execution_count": 84,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "using TextAnalysis\n",
    "td = TokenDocument(\"To be or not to be...\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 90,
   "metadata": {},
   "outputs": [
    {
     "ename": "LoadError",
     "evalue": "syntax: invalid character literal",
     "output_type": "error",
     "traceback": [
      "syntax: invalid character literal",
      ""
     ]
    }
   ],
   "source": [
    "str = StringDocument(\"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\")\n",
    "#sd1 = FileDocument(\"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\")\n",
    "#t=split_sentences(text(str))\n",
    "text(str)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 92,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "\"Odor perception, and emotional and behavioral responses to odors strongly depend on experience, and learned odor-context associations often last for the lifetime of an animal (Mouly and Sullivan, 2010) . The cellular and neural circuit mechanisms underlying olfactory learning and memory, however, remain poorly understood.\\r\\n Recent studies on episodic and contextual learning in hippocampal neural networks have suggested that memories are encoded in the activity of distributed ensembles of neurons, often referred to as a 'memory trace' (Mayford and Reijmers, 2015; Poo et al., 2016; Tonegawa et al., 2015) . The neurons constituting such a memory trace are thought to encode information about the environmental context and associated emotions of past experiences, and their activity is necessary and sufficient for memory retrieval (Liu et al., 2012; Reijmers et al., 2007; Tanaka et al., 2014) .\\r\\n Here, we investigate the organization of odor memory traces in the olfactory (piriform) cortex of mice. The piriform cortex, a trilaminar paleocortical structure, is the largest cortical area receiving direct afferent inputs from the olfactory bulb, which, in turn, receives topographically organized inputs from olfactory sensory neurons in the nose. Individual piriform neurons respond to combinatorial inputs from multiple Piriform cortex has long been hypothesized to support auto-associative network functions that can retrieve previously learned information from partial or degraded sensory inputs (Haberly, 2001; Wilson and Sullivan, 2011) . Piriform pyramidal cells form a large recurrent network, which is reciprocally connected with adjacent highorder associative areas including the prefrontal, entorhinal and perirhinal cortex and the amygdala (Johnson et al., 2000; Sadrian and Wilson, 2015) . Storage of information is made possible by NMDA-dependent, associative plasticity of connections (Johenning et al., 2009; Kanter and Haberly, 1990; Quinlan et al., 2004) .\\r\\n Furthermore, changes in piriform network activity (Chapuis and Wilson, 2011; Chen et al., 2011; Li et al., 2008; Sevelinges et al., 2004) and stabilization of piriform odor representations (Shakhawat et al., 2015) have been observed after associative olfactory learning. Finally, excitotoxic lesions of the posterior piriform cortex in rats perturb odor fear memories (Sacco and Sacchetti, 2010) , and optogenetic stimulation of artificial piriform ensembles is sufficient to drive learned behaviors (Choi et al., 2011) . Taken together, these studies have led to the hypothesis that piriform neural ensembles encode olfactory memory traces.\\r\\n To test this hypothesis, we have developed an intersectional genetic strategy in mice to target piriform neurons that are activated by olfactory experience. We employed\\r\\n To selectively label and manipulate piriform neurons that were activated during odor exposure, we used cFos-tTA transgenic mice (Reijmers et al., 2007) , in which the activity-dependent cFos promoter drives expression of the tTA transcription factor, and we stereotaxically injected in their piriform cortex Adeno-Associated Viruses (AAVs) expressing Designer Receptors Exclusively Activated by Designer Drugs (DREADDs) under the control of the tTA-responsive promoter tetO ( Figure 1A) .\\r\\n DREADDs (Alexander et al., 2009; Ferguson et al., 2011) increase (hM3Dq:mCherry) or decrease (HA:hM4Di-IRES-mCitrine) the excitability of neurons upon activation by their ligand clozapine N-oxide (CNO). Temporal control of DREADD expression is provided by doxycycline, administered through the diet, which interferes with the binding of tTA to tetO and thus suppresses transgene expression (Gossen et al., 1995) . Mice maintained in their home cage on a doxycycline-containing diet showed low basal DREADD expression, detected by anti-HA immunohistochemistry and native mCherry fluorescence (Figure 1B and C) . To induce DREADD expression, mice were taken off doxycycline five days before odor presentation. Exposure to odor followed by foot shock (see Experimental Procedures) resulted in the Fos-tagging of sparse neural ensembles (median (interquartile range): 1.6 (0.3) % of piriform neurons, n=2 for hM4Di; 2.2 (0.5) % of piriform neurons, n=3 for hM3Dq), which were broadly dispersed throughout the anterior and posterior piriform cortex ( Figure 1B and C), consistent with previous observations of endogenous cFos expression (Datiche et al., 2001; Illig and Haberly, 2003) .\\r\\n To test whether CNO-mediated activation of DREADDs in Fos-tagged piriform neurons alters their excitability, we performed whole-cell recordings in acute brain author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint slices containing piriform cortex. DREADD-expressing neurons were identified based on mCitrine (HA:hM4Di-IRES-mCitrine) or mCherry (hM3Dq:mCherry) fluorescence. Figure 1) . In the absence of the DREADD ligand CNO, the resting membrane potential of DREADD-expressing piriform neurons was indistinguishable from DREADD-negative cells (-71.70 (4.37) mV in n=24, (4.32) mV in hM4Di-expressing cells, n=14, -72.56 (3.92) mV in hM3Dq-expressing cells, n=15, p=0.631, Kruskal-Wallis ANOVA, Figure 1D ). After bath application of CNO (5 µM), the membrane properties selectively changed in DREADD-expressing cells: hM4Di-expressing cells hyperpolarized (-74.51 (5.17 ) mV, n=12, p=0.007, Wilcoxon Signed Ranks Test), and hM3Dq-expressing cells depolarized (-66.95 (4.12) mV, n=13, p=0.001, Wilcoxon Signed Ranks Test), whereas the resting membrane potential of DREADD-negative cells remained unchanged (-70.28 (3.81) mV, n=15, p=0.890, Wilcoxon Signed Ranks Test, Figure 1D ). The relative changes were 1.00 (0.03) in DREADD-negative cells (n=15, p=0.890, One Sample Wilcoxon Signed Rank Test), 1.04 (0.03) in hM4Di-expressing cells, (n=12, p<0.001, One Sample Wilcoxon Signed Rank Test) and 0.91 (0.09) in hM3Dq-expressing cells, (n=13, p=0.001, One Sample Wilcoxon Signed Rank Test), Figure 1E ). To determine the impact of CNO treatment on neuronal excitability, we delivered depolarizing current steps and compared the number of action potentials triggered by 60 pA step current above action potential threshold (see Experimental Procedures). We found that in hM4Di-expressing cells, the number of evoked action potentials significantly decreased after CNO application (3.75 (1.59) vs. 0.30 (3.48), n=10, p=0.010, Wilcoxon Signed Ranks Test), whereas in hM3Dq-expressing cells, the number of author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint action potentials significantly increased after CNO application (8.67 (9.10) vs. 19.0 (27.6) mV, n=11, p=0.006, Wilcoxon Signed Ranks Test). In contrast, DREADDnegative cells did not exhibit a significant change in the number of evoked action potentials upon CNO application (4.33 (1.67) vs. 4.67 (3.00), n=9, p=0.672, Wilcoxon Signed Ranks Test, Figure 1F ). The relative changes were 1.00 (0.41) in DREADDnegative cells, (n=9, p=0.641, One Sample Wilcoxon Signed Rank Tests), 0.09 (0.81) in hM4Di-expressing cells (n=10, p=0.010, One Sample Wilcoxon Signed Rank Tests) and 1.81±1.09 in hM3Dq-expressing cells (n=11, p=0.004, One Sample Wilcoxon Signed Rank Tests, Figure 1G ). Taken together, these experiments show that Fos-tagging during olfactory fear conditioning marks a sparse and dispersed subpopulation of piriform neurons, and that CNO-mediated activation of hM4Di or hM3Dq expressed in Fos-tagged neurons selectively decreases or increases their excitability.\\r\\n If Fos-tagged piriform neurons that were activated during olfactory learning constitute an essential component of an olfactory memory trace, then perturbing the activity of these ensembles should interfere with memory recall. To test this prediction, we Fostagged piriform neurons during olfactory fear conditioning, and then chemogenetically silenced Fos-tagged ensembles during odor fear memory recall. cFos-tTA transgenic mice (\\\"experimental group\\\", n=8) were bilaterally injected in piriform cortex with the AAV-tetO-hM4Di vector. Wild-type mice injected with the same virus served as a control (\\\"control group\\\", n=8). During training, the conditioned odor (CS+) was presented at one of the ends of the conditioning box and paired with a foot shock, applied to the side of the box where the odor was delivered (see Experimental author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint Procedures and Figure 2A ). Mice thus learned to escape from the conditioned stimulus (CS+) by running towards the opposite side of the box. Two control odors, whose presentation was not paired with foot shock (CS-), were used to entrain odorspecific as opposed to generalized fear responses (Chen et al., 2011) . Doxycycline was removed from the diet of mice prior to training, to permit the induction of hM4Di expression in active Fos-expressing piriform neurons of mice of the experimental group. Mice were returned to doxycycline-containing diet immediately after odor fear conditioning, and memory recall of the odor-foot shock association was tested three days later by presenting the odors alone in a new box ( Figure 2B and C). All mice were intraperitoneally (i.p.) injected with CNO before testing, to exclude potential offtarget effects by CNO (Gomez et al., 2017) . For all mice, we verified viral gene expression by post hoc histological examination. Previous studies have suggested that the posterior piriform cortex is important for associative memory encoding (Sacco et al., 2010; Calu et al., 2008; Haberly 2001) . We therefore excluded from our analysis mice in which parts of the posterior piriform cortex were spared from viral infection (Supplementary Figure 2) . We combined two parameters to quantify the learned escape behavior: the position of the mouse in the box, and the increase in its maximum velocity after odor presentation ( Figure 2B ). These parameters were chosen because they described well the behavioral response of mice to the odor-foot shock pairing during training (Supplementary Figure 3) .\\r\\n As expected, mice in the control group exhibited robust escape behavior upon CS+ presentation. The median position of the mice in the box at the end of stimulus delivery was 0.74 (with 0 defined as the site of odor presentation and 1 as the opposite end of the box), and mice exhibited a 1.9-fold increase of their maximum velocity ( Figure 2E, G and H) . In contrast, escape behavior was significantly reduced author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint in mice in which Fos-tagged neurons were silenced. Their median position in the box at the end of stimulus delivery, (0.42), and their 1.1-fold increase in maximum velocity were significantly lower than in the control group (Mann-Whitney test, U=7, p=0.007 for both position and velocity, Figure 2E , G and H). The difference in the position of mice after CS+ and CS-presentation was also significantly different between the control and experimental groups (Mann-Whitney test, U=13, p=0.0499, Figure 2F ). Differences in behavioral responses of cFos-tTA transgenic mice were specifically due to hM4Di activation, as escape behavior after CNO injection was unaffected in cFos-tTA transgenic mice injected with AAVs expressing the calcium indicator GCaMP (n=9, Supplementary Figure 4 ).\\r\\n All mice were similarly close to the odor port at the onset of CS+ odor presentation (Mann-Whitney test, U=28, p=0.7209, Figure 2D ), and had similar maximum velocities during the 7 s time period that preceded odor presentation (Mann-Whitney test, U=24, p=0.4418, Figure 2G ). These observations exclude any inherent differences between groups due to how the CS+ was presented. Furthermore, escape behavior in control mice was specific to the CS+, as none of the mice ran away from the CS-( Figure 2D , E and H). Finally, we did not observe any differences in the overall mobility of mice throughout the entire testing session. The median speed of mice of all groups (0.68 (0.28) cm/s and 0.46 (0.14) cm/s for the control and experimental groups respectively) was not significantly different (Mann-Whitney test, U=22, p=0.3282), nor did mice show a bias for the left or right side of the testing box (time spent on the left side divided by time spent on the right side: 1.36 (0.72) and\\r\\n 1.51 (0.52) for the control and experimental groups respectively, Mann-Whitney test, U=27, p=0.6454). Together, these data suggest that Fos-tagged piriform ensembles that were activated during olfactory fear conditioning are necessary for odor fear author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint memory recall.\\r\\n Even though cFos-tTA transgenic mice expressing hM4Di failed to robustly escape from the CS+, it should be noted that they behaved differently after CS+ and CSpresentation. They were further away from the odor port after CS+ presentation than after CS-presentation (Wilcoxon matched-pairs signed rank test, W=-36, p=0.0078, Figure 2E ), and their maximum velocity ratio was significantly higher (Wilcoxon matched-pairs signed rank test, W=-36, p=0.0078, Figure 2H ). This could indicate a partial memory of the learned association, possibly due to incomplete silencing of Fos-tagged neurons in piriform cortex, or the existence of parallel neural pathways that partially compensate for the loss of piriform functions. Finally, silencing of the piriform cortex in only one hemisphere did not abolish the learned escape behavior to the conditioned odor (Supplementary Figure 5) .\\r\\n It is possible that the chemogenetic silencing of piriform ensembles that were Fostagged during olfactory fear conditioning disrupts odor detection and discrimination rather than selectively affecting odor fear memory recall. To test this possibility, we monitored sniffing behavior of mice during an olfactory habituation/dishabituation assay ( Figure 3A) , a well established test for odor detection and discrimination (Coronas-Samano et al., 2016; Verhagen et al., 2007; Wesson et al., 2008) . Fostagging of piriform neurons during olfactory fear conditioning was performed as described above. Three days later, changes in sniffing behavior in response to odor exposure were tested in a plethysmograph, while hM4Di-expressing piriform neurons author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint in cFos-tTA transgenic mice were silenced using i.p. injection of CNO (see Experimental Procedures). We observed that mice of both experimental (n=8) and control (n=6) groups increased their sniff frequency upon presentation of a novel, neutral odor (1.6 fold increase; Wilcoxon matched-pairs signed rank test, W=21, p=0.0312 and W=36, p=0.0078 for the control and experimental groups, respectively; Figure 3B and C). Repeated exposure to the same odor (neutral odor or CS-) resulted in a decrease in the sniff frequency, reflecting habituation after 4 consecutive exposures (Figure 3C and D) . Finally, presentation of the CS+ increased sniff frequency (1.4 fold increase; Wilcoxon matched-pairs signed rank test, W=21, p=0.0312 and W=34, p=0.0156 for the control and experimental groups, respectively, Figure 3D ). Such changes in odor sampling behavior have been shown to report the detection and discrimination of different odor stimuli (Coronas-Samano et al., 2016; Verhagen et al., 2007; Wesson et al., 2008) . Therefore, these data suggest that basic behaviors characteristic of odor sampling, detection and discrimination were unaffected by the silencing of piriform ensembles that were Fostagged during olfactory fear conditioning.\\r\\n Odors activate unique but overlapping ensembles of piriform neurons (Bolding and Franks, 2017; Poo and Isaacson, 2009; Roland et al., 2017; Stettler and Axel, 2009 ). Therefore, silencing neurons that respond to one odor could partially interfere with the retrieval of information associated with other odors. To test this prediction we modified the behavioral protocol to separate Fos-tagging from olfactory fear conditioning. cFos-tTA transgenic and wild-type control mice were injected with AAV-tetO-hM4Di, and expression of hM4Di in piriform neurons was induced while mice author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint were exposed to odor (eugenol or beta-citronellol) in a neutral environment, without subsequent foot shock. Fos-tagging resulted in the labeling of sparse neural ensembles, similar to those labeled during odor-foot shock association (1.1 (0.3) % of piriform neurons, n=3). Mice were returned to doxycycline-containing diet and trained two days later to associate a different odor (ethyl-acetate, CS+) with foot shock.\\r\\n Behavioral testing of the learned escape behavior in response to the CS+ was performed one day later, after i.p. injection of CNO (see Experimental Procedures and Figure 4A ). We found that learned escape behavior of cFos-tTA transgenic mice expressing hM4Di (experimental group, n=12) was similar but somewhat attenuated compared to wild-type controls (n=10). Both groups exhibited an escape behavior after CS+ presentation, indicated by the significant increase in the maximum velocity (Wilcoxon matched-pairs signed rank test, W=55, p=0.0020 and W=56, p=0.0269 for the control and experimental groups respectively; no difference in the maximum velocity ratio after/before between groups, Mann-Whitney test, U=35, p=0.1071, Figure 4E and F). The difference in the position of mice after CS+ and CSpresentation was also similar between the control and experimental groups (Mann-Whitney test, U=37, p=0.1402, Figure 4D ). However, we observed that silencing neutral odor representations somewhat dampened the behavioral response to the conditioned stimulus, as behavioral responses were less robust compared to controls when quantifying the position of mice in the box after CS+ presentation (Mann-Whitney test, U=22, p=0.0112, Figure 4B and C). The attenuated behavioral responses we observe likely reflect the partial degradation of odor information as a result of the silencing of piriform neurons responsive to both the CS+ and the neutral odor.\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint\\r\\n If piriform neural ensembles that were activated during olfactory fear conditioning encode an odor fear memory trace, then reactivation of these neurons may be sufficient to trigger memory recall. To test this prediction, we devised a modified behavioral task to assess memory recall. We monitored exploratory behavior in an open field during ambient odor exposure or artificial reactivation of Fos-tagged neurons, a test that does not rely on an escape behavior from an odor originating from a spatially defined odor source ( Figure 5A ). We first trained mice to associate ethyl-acetate, the CS+, with foot shock. Upon ambient exposure to the CS+, diffused above the open field, exploratory behavior of mice was significantly attenuated compared to baseline exploratory behavior measured one day earlier (Wilcoxon matched-pairs signed rank test, n=6, W=-21, p=0.0312). Ethyl-acetate exposure did not affect exploratory behavior of mice that had previously been exposed to ethylacetate without foot shock (Wilcoxon matched-pairs signed rank test, n=5, W=1, p>0.9999). These data suggest that re-exposure of mice to the CS+ causes an increase in anxiety-like behavior, expressed as a decrease in exploratory behavior in the open field.\\r\\n We next repeated the experiment in cFos-tTA transgenic mice, in which the piriform cortex had been infected with the activating DREADD AAV-tetO-hM3Dq (Supplementary Figure 6) . Instead of exposure to the CS+, we i.p. injected the DREADD ligand CNO to reactivate hM3Dq-expressing piriform neurons that were Fos-tagged during odor-foot shock association (see Experimental Procedures and Figure 5A ). We found that CNO-mediated reactivation, similar to CS+ exposure, caused a significant decrease in exploratory behavior of mice that had undergone author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint olfactory fear conditioning (Wilcoxon matched-pairs signed rank test, n=8, W=-34, p=0.0156, Figure 5B and C). CNO-mediated reactivation of piriform neurons that were Fos-tagged during ethyl-acetate exposure without foot shock did not affect exploratory behavior of mice (Wilcoxon matched-pairs signed rank test, n=7, W=-10, p=0.4688, Figure 5B and C). Taken together, these data suggest that chemogenetic reactivation of piriform neurons that were active during odor-foot shock exposure is sufficient to trigger fear memory recall.\\r\\n We next asked whether artificial memory recall depends on the specificity of the Fostagged neural ensemble: is piriform reactivation sufficient to trigger fear memory recall, as long as it includes the CS+-tagged ensemble? For this purpose, we generated a synthetic Fos-tagged ensemble, by sequentially exposing mice to ethylacetate paired with foot-shock (CS+) and neutral odors (CS-, eugenol and betacitronellol) ( Figure 5D ). While exposure to the CS+ resulted in anxiety-like behavior similar to what was observed in the previous experiment (Wilcoxon matched-pairs signed rank test, n=6, W=-21, p=0.0312, Figure 5E and F), artificial reactivation of synthetic Fos-tagged ensemble did not result in changes in exploratory behavior that would indicate memory recall (Wilcoxon matched-pairs signed rank test, n=8, W=-4, p=0.8438, Figure 5E and F). These data suggest that during artificial reactivation, piriform cortex cannot extract meaningful information from the synthetic representation generated by the sequential presentation of CS+ and CS-odors. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint\\r\\n The piriform cortex has long been thought to provide the substrate for storing associative olfactory memories. Several studies have shown that piriform cortex cells can encode information that carries behavioral significance (Calu et al., 2008 , Mandairon et al., 2014 , Sacco et al., 2010 ), yet their functional relevance has never been assessed by direct experimental manipulation. Using a cFos-dependent, intersectional genetic approach to visualize and manipulate piriform neurons activated during olfactory fear conditioning (Figure 1 ), we found that chemogenetic silencing of Fos-tagged piriform ensembles abolished a learned odor escape behavior (Figure 2 ). Silencing these Fos-tagged ensembles did not alter basic odor detection and discrimination (Figure 3) . Furthermore, chemogenetic silencing of neurons responsive to a neutral odor only moderately attenuated odor fear memory recall, suggesting that Fos-tagged piriform ensembles are largely odor-specific ( Figure 4) . Finally, specific chemogenetic reactivation of piriform ensembles that were active during learning resulted in reduced exploratory behavior in an open field assay, an effect that was similarly observed when exposing mice to the conditioned odor stimulus (Figure 5) . Together, our experiments identify piriform neurons that were active during learning as necessary and sufficient to trigger odor fear memory recall. This indicates the formation of functional connections between Fos-tagged piriform cells and other associative areas and firmly establish piriform ensembles as an essential component of odor fear memory traces.\\r\\n Artificial reactivation of an olfactory memory trace author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint Reactivation of neurons by CNO-mediated activation of DREADD receptors does not recapitulate the temporal characteristics of piriform odor responses and their modulation by active sampling (Bolding and Franks, 2017; Miura et al., 2012) .\\r\\n Despite this, reactivation of Fos-tagged piriform ensembles activated during odor-foot shock exposure was sufficient to elicit a behavioral response. How piriform neural circuits and downstream target regions process such artificial activity patterns remains to be determined. One possibility is that despite the temporal limitations of memory trace reactivation, piriform network mechanisms can retrieve the perception of the reinforced conditioned stimulus. Consistent with this model, recent studies have shown that spatial patterns of odor-evoked activity, in the absence of precise temporal information, are sufficient to decode odor identity in piriform cortex (Bolding and Franks, 2017; Miura et al., 2012; Roland et al., 2017) . Alternatively, it is possible that reactivation generates a state of stress and anxiety, but without evoking the perception of the odor. Candidate target areas for the processing of odor-fear associations include the basolateral amygdala and the medial prefrontal cortex (Dejean et al., 2016; Gore et al., 2015) , however, the relevant neural circuit components remain to be identified. Finally, reactivating a synthetic ensemble of neurons activated during exposure to the reinforced stimulus (CS+) and the nonreinforced stimuli (CS-) did not elicit a measurable behavioral response. This observation suggests that behaviorally relevant information cannot be extracted from a neural ensemble representing conflicting (aversive versus neutral) information. This result is consistent with the finding that the reactivation of an artificial contextual memory in the hippocampus competes with the retrieval of a learned context-shock association (Ramirez et al., 2013) . author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint\\r\\n A general limitation of the cFos-tTA system is that the tagging of active neurons is transient. The duration of the expression of tTA-dependent proteins is determined by the time course of induction and the stability of cFos-tTA-dependent transcripts and proteins in tagged neurons. Previous studies have shown that TetTag-dependent expression of regulators of neural activity lasts for at least five days but becomes undetectable by thirty days (Cowansage et al., 2014; Liu et al., 2012; Zhang et al., 2015) . We therefore restricted our manipulations of neural activity and behavioral analyses to a three-day time window after Fos-tagging of neurons during olfactory conditioning. Another constraint of the Fos-tagging system is its slow temporal dynamics. As a consequence, neurons responding to the reinforced conditioned stimulus (CS+), the odorant paired with foot shock, and neurons responding to the non-reinforced conditioned stimuli (CS-), the odorants not associated with foot shock, were tagged in our silencing experiments. The use of a CS-compromises the specificity of the Fos-tagged neural ensembles. Nevertheless, the CS-provides an important experimental readout for the odor selectivity of the behavioral response.\\r\\n Interestingly, the number of Fos-tagged piriform neurons we observe is significantly lower than the number of odor-responsive neurons detected in electrophysiological recordings in awake mice (Bolding and Franks, 2017; Miura et al., 2012; Zhan and Luo, 2010) . However, in these experiments, a large fraction of cells exhibited low firing rates. Therefore, the Fos-tagged population could represent a subpopulation of neurons that is strongly activated by odor (Schoenenberger et al., 2009) . Fos-tagging could also mark plastic changes supporting memory formation (Cole et al., 1989; Minatohara et al., 2015) . Indeed, cFos mRNA levels decrease significantly after injection of an antagonist of the NMDA receptor, a key player in the induction of author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint synaptic plasticity (Tayler et al., 2011) , and long-term memory and synaptic plasticity are impaired when cFos production is perturbed in the central nervous system (Fleischmann et al., 2003; Seoane et al., 2012) . Future studies will allow to directly test these hypotheses.\\r\\n The hippocampus has been studied extensively for its role in spatial and contextual memory (Basu and Siegelbaum, 2015) . Recently, Fos-tagging of hippocampal neurons during contextual fear conditioning has provided important insight into the cellular and neural circuit mechanisms of learning and memory (Cai et al., 2016; Garner et al., 2012; Liu et al., 2012; Ramirez et al., 2013; Roy et al., 2017; Ryan et al., 2015) . However, if principles of memory formation and storage in hippocampusrelated neural networks apply to other cortical structures remains largely unknown.\\r\\n Interestingly, pirform cortex and hippocampus share a similar circuit organization and both have been modeled as auto-associative networks (Haberly, 2001) . Consistent with this model, our findings reveal striking similarities between olfactory memory traces in the piriform cortex and memory traces of contextual fear in the hippocampus. In both piriform cortex and hippocampus, learning activates sparse and distributed ensembles of neurons that appear to lack topographic organization. Furthermore, hippocampal neurons tagged during contextual fear conditioning, and piriform neurons tagged during olfactory fear conditioning were both necessary and sufficient for memory retrieval (Liu et al., 2012; Tanaka et al., 2014) , The olfactory system is particularly well-suited for studying learning and memory. Indeed, odor memories and olfactory-driven behaviors are robust in animals, including in mice, providing highly reliable and quantitative behavioral readouts. Furthermore, the author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint olfactory cortex is only two synapses away from the peripheral olfactory sensory neurons in the olfactory epithelium, and the neural inputs that drive piriform cortex activity have been well characterized (Banerjee et al., 2015; Dhawale et al., 2010; Economo et al., 2016; Fukunaga et al., 2012; Kato et al., 2012; Roland et al., 2016; Yamada et al., 2017) . Knowing the properties of neural input patterns will provide the basis for understanding neural circuit input-output computations and their modification by learning and experience.\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint\\r\\n Mice were housed at 24°C with a 12-hour light/12-hour dark cycle with standard food and water provided ad libitum. Mice were group-housed with littermates until the beginning of surgery and then single-housed in ventilated cages throughout the duration of the experiment. cFos-tTA mice (Reijmers et al., 2007) were fed a diet containing 1g/kg doxycycline for a minimum of 4 days before surgery. Wild-type control animals were siblings that did not carry the cFos-tTA transgene. The age of mice at the time of behavioral testing ranged from 10 to 15 weeks. Male mice were used for the hM4Di-mediated neural silencing, and females for the hM3Dq-mediated neural activation experiments. Experiments were carried out according to European and French national institutional animal care guidelines (protocol APAFIS#2016012909576100).\\r\\n The Adeno-associated viruses (AAVs) were generated at Penn Vector Core, University of Pennsylvania (serotype 8, 10 13 genome copies/mL, 1:2 dilution with sterile PBS on the day of injection).\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint\\r\\n Mice were anaesthetized with ketamine/xylazine (100 mg/kg/10 mg/kg, Sigma-Aldrich) and AAV vectors were injected stereotaxically into the piriform cortex Sections were washed 2 times in PBST and 1 time in PBS, mounted on slides and author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint coverslipped with Vectashield mounting medium (Vectorlabs). Images were acquired as Z-stacks (70 to 140 µm in total thickness, step size 7 µm) with a Leica SP5 confocal microscope or as single plane sections with a Zeiss Axio Zoom microscope and processed in Fiji.\\r\\n Counting Z-stacks (7 µm step) were acquired with a Leica SP5 confocal microscope with a 20X objective and a resolution of 512x512 or 1024x1024 pixels (1 pixel: 0:72µm). First, the piriform cortex was delineated on a maximum projection of each stack using a custom-written ImageJ macro. For each mouse, 2 to 4 sections (median volume 9.3e-2 (9.4e-3) mm 3 ) were analyzed, from one or both hemispheres, at Y=-0.8 and Y=-1.6mm relative to bregma.\\r\\n HA-stained neurons (hM4Di) were counted by hand. hM3Dq-mCherry positive neurons were counted using a custom-written ImageJ plugin. After pre-processing the image (Subtract Background, Remove Outliers and Median Filter), the z-stack is thresholded using the RenyiEntropy algorithm. On each slice of the stack, objects with an area smaller than 40µm 2 are removed using the Analyze Particle command in ImageJ. A mask of each slice from the stack is created containing the filled outline of the measured particles. The number of cells is the number of 3D objects detected in the stack. Automated cell detection can be prone to error, thus, the detection efficiency was checked on a few pictures by two people blind to the experimental conditions.\\r\\n The numbers of cells were calculated as numbers of cells per cubic millimeter. An estimate of the total number of neurons (counterstained with NeuroTrace) per cubic millimeter (7.1e4 mm -3 ) was obtained by counting neurons in representative volumes author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint containing the three layers of piriform cortex. Numbers are consistent with those obtained by (Srinivasan and Stevens, 2017) .\\r\\n Parasagittal or coronal slices (300 µm thick) of piriform cortex were prepared from 6-8 week-old cFos-tTA mice injected with AAV-tetO-hM4Di-IRES-mCitrine or AAV-tetO-hM3Dq-mCherry. Animals were anaesthetized with ketamine and xylazine (100 mg/kg/10 mg/kg, Sigma-Aldrich), perfused with ice-cold ACSF (125 mM NaCl, 2.5 mM KCl, 25 mM glucose 25mM NaHCO 3 , 1.25 mM NaH 2 PO 4 , 2 mM CaCl 2 , 1 mM MgCl 2 , 1 mM pyruvic acid, bubbled with 95% O 2 and 5% CO 2 and adjusted to 295±5 mOsm osmolarity), and decapitated. The brain was cooled with ice-cold ACSF solution and then sliced using a 7000SM2 vibrating microtome (Campden Instruments, UK). Slices were incubated in the same solution at 34°C for one hour and then continuously perfused with ACSF solution (2 mL/min) at 34°C in the recording chamber. DREADD expression was detected with two-photon excitation (830nm, ChameleonMRU-X1, Coherent, UK) under a Scientifica TriM Scope II microscope (LaVision, Germany), with a 60x water-immersion objective. Whole-cell recording pipettes with 5-7 MΩ resistance were filled with the following solution (in mM): 122 Kgluconate, 13 KCl, 10 HEPES, 10 phosphocreatine, 4 Mg-ATP, 0.3 Na-GTP, 0.3 EGTA (adjusted to pH 7.35 with KOH). The recording solution also contained the morphological tracer Alexa Fluor 594 (5 µM, red channel) for nonexpressing and hM4Di-positive cells or the Ca 2+ -sensitive dye Fura-2 (300 µM, replacing EGTA in the recording solution, green channel) for hM3Dq-positive cells, to enable identification of the patched cell and to visualize its morphology. The excitability of cells was measured in current-clamp mode by 500 ms steps of current author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint injections from -300 to +500 pA with steps of 20 pA. We compared the number of action potentials triggered by equivalent depolarization at each step 60 pA above the action potential threshold before and after CNO (5 µM) application. The series resistance was usually <20 MΩ, and data were discarded if it changed by more than 20% during the recording. Signals were amplified using EPC10-2 amplifiers (HEKA Elektronik, Lambrecht, Germany). Voltage-clamp recordings were filtered at 5 kHz and sampled at 10 kHz, and current-clamp recordings were filtered at 10 kHz and sampled at 20 kHz, with the Patchmaster v2x32 program (HEKA Elektronik).\\r\\n A training box was used to train mice to escape from an odor. The box was rectangular (L 57 cm, W 17 cm, H 64 cm), with a grid floor made of 72 stainless-steel rods (diameter=6 mm, space between rods=2 mm). Current was delivered by an aversive stimulator (MedAssociates, 115V, 60 Hz). A custom-made switcher allowed an electric foot shock (0.6 mA, 0.6 ms) to be applied independently to either half of the box. A testing box was used to test memory retrieval. Its dimensions were similar to the training box. Materials for the walls and floor were different from the training box to create a different context. The open field box consisted of a white Plexiglas (6 mm thickness) container (50 cm x 50 cm x 38 height).\\r\\n Sniffing behavior was monitored in freely moving mice using a plethysmograph (Emka Technologies). Electrical signals were amplified using RHD 2132 amplifier boards (Intan Technologies) and band-pass filtered (1-30 Hz).\\r\\n Odors were delivered using an 8 channels olfactometer (Automate Scientific).\\r\\n Continuous air flowed through a bottle of mineral oil (Sigma-Aldrich) to avoid pressure changes when presenting the odors. The following odors (Sigma-Aldrich), author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint at a concentration of 1% (vol/vol) in mineral oil were used: ethyl-acetate, betacitronellol, eugenol, limonene, and pinene.\\r\\n AAVs were stereotaxically injected in both hemispheres of piriform cortex of cFos-tTA transgenic mice and littermate wild-type controls. Mice were given 10 days to recover, before being habituated on day 11 to the training and testing/open field/plethysmograph boxes (40 and 10 min each). 24 hours after habituation, doxycycline was removed and replaced by a normal diet. Five days later, mice were fear conditioned to ethyl-acetate or exposed to an odor (odor \\\"C\\\") in a neutral environment. Immediately after fear conditioning or odor presentation, mice were put back on a diet containing doxycycline. During fear conditioning, ethyl-acetate (the CS+) was presented at one extremity of the training box, and the corresponding halfside of the box' floor was electrified for 0.6 s (0.6 mA). The foot shock was delivered 4 seconds after CS+ presentation, and the CS+ lasted for an additional 2 seconds.\\r\\n Mice learned to escape to the opposite side when the CS+ was presented. The CS+ was presented 4 times, 2 times on each side of the box. When fear conditioning included presentation of two non-reinforced conditioned stimuli (CS-1 and CS-2), the choice between CS+ and CS-as well as the side of presentation was pseudorandomized. Training always started with one presentation of each CS-followed by the CS+. The two CS-odors were presented during 7 seconds, 3 times each, with a total presentation of 3 on the right side and 3 on the left side. During testing, the CS+ was presented 4 times (2 times on each side), and each CS-was presented once on each side, for 7 s each. As for the training session, the order and side of odor presentations was pseudo-randomized: testing always started with one presentation author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint of each CS-, followed by one CS+ presentation.\\r\\n Clozapine-N-oxide (C0832, Sigma-Aldrich) was first dissolved in dimethylsulfoxide (DMSO, D2438, Sigma, final concentration of 1% vol/vol), then further diluted in 0.9% sterile saline solution to a final concentration of 0.2 mg/mL. Aliquots were stored for up to two months at -20°C and equilibrated to room temperature before injection. The solution was administered intraperitoneally (3 mg/kg). After injection, mice were left undisturbed for 25 minutes (Figure 2 and 4) or 5 minutes (Figure 3 and 5) in their home cage before the start of the experiment.\\r\\n Electrophysiological data were analyzed with custom-made software in Python 3.0 and averaged in MS Excel (Microsoft, USA). Behavioral sessions were video recorded and analyzed using custom-written MatLab programs (available upon request). Parameters extracted from multiple CS+ (CS-) presentation were averaged for each mouse.\\r\\n Statistical analysis was performed with Prism (GraphPad). Non-parametric tests were used: Mann-Whitney test for between group comparison, and Wilcoxon matchedpairs signed rank test for within group comparison. Values are represented as median (interquartile range) unless otherwise stated. The n represents number of mice, or number of recorded neurons for the electrophysiology experiment. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint before and after CNO application. After CNO application, the number of APs in DREADD-negative cells (Ctrl) does not change, significantly decreases in hM4Diexpressing cells and significantly increases in hM3Dq-expressing cells.\\r\\n (G) The relative change in the number of APs after/before CNO in DREADD-negative cells (Ctrl) does not change, significantly decreases in hM4Di-expressing cells and significantly increases in hM3Dq-expressing cells.\\r\\n Data are shown as individual data points, median and interquartile range, ** p<0.01. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint A) During fear conditioning, mice are trained to associate ethyl-acetate (CS+) with foot shock. The CS+ is presented at one end of the training box, and the corresponding half-side of the box is electrified. Training includes the presentation of two other odors that are not paired with foot shock (CS-, eugenol and beta-citronellol).\\r\\n The CS+ is presented 4 times (2 times on each side) and each CS-three times. The order and side of odor presentations is pseudo-randomized. B) During testing, the CS+ is presented 4 times (2 times on each side), and each CSis presented once on each side, for 7 s each. The testing session is video-recorded and the position of the mouse's centroid in the box along the x-axis is extracted (top).\\r\\n The length of the box is normalized to 1. Since the odor is presented randomly on the left or right side of the testing box, 0 is defined as the extremity where the odor is presented. For velocity measurements, a 7 s time window is defined both before and after stimulus onset (bottom). C) Experimental design: cFos-tTA transgenic mice (experimental group, n=8) or wildtype mice (control group, n=8) are injected with AAV-tetO-hM4Di in both hemispheres of piriform cortex. Ten days later, mice are habituated to the training and testing boxes. Upon doxycycline (DOX) removal, mice are fear-conditioned, and hM4Di expression (green circles) is induced in Fos-tagged neurons (filled in blue and yellow). Mice are returned to DOX-containing diet to avoid further expression of hM4Di. Three days later, all mice are i.p. injected with CNO, which silences hM4Diexpressing neurons (black crosses), and memory retrieval is tested.\\r\\n (D to H) Olfactory memory retrieval after CNO injection, assessed by the position of mice in the testing box and their maximum velocity upon odor exposure. Memory author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint retrieval was significantly impaired in cFos-tTA transgenic mice in which piriform neurons that were active during learning were chemogenetically silenced during memory recall (experimental group).\\r\\n (D) Position of mice in the testing box as a function of time after CS+ (black) and CS-(blue) presentation. (E) Position in the testing box 7 s after CS+ or CS-presentation.\\r\\n (F) Position 7 s after CS-presentation subtracted from the position 7 s after CS+ presentation.\\r\\n (G) Maximum velocity during a 7 s time window before and after CS+ presentation.\\r\\n (H) Maximum velocity ratio after/before CS+ (left) or CS-(right) presentation.\\r\\n Data are shown as median and interquartile range, * p<0.05; ** p<0.01; n.s. not significant.\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint (A) (Top) Experimental design: Fos-tagging during olfactory fear conditioning is performed as described in Figure 2 . Three days after fear conditioning, sniffing behavior is quantified in a plethysmograph. Mice are i.p. injected with CNO, habituated to the plethysmograph for 15 min, and each odor is then presented for 8 s with a 2.5 min inter-trial interval. pin: pinene, CS-: beta-citronellol, CS+: ethyl-acetate.\\r\\n (Bottom) Representative sniff recordings from one mouse when exposed to pinene or the CS+ (ethyl-acetate). Data are shown as median and interquartile range, * p<0.05; ** p<0.01; n.s.: not significant author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint Data are shown as median and interquartile range, * p<0.05; ** p<0.01; n.s. not significant.\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint (B) As a measure for exploratory activity, the distance travelled is computed from minute 15 to 18 (corresponding to odor exposure during test 2). For artificial memory recall, the distance travelled is computed over the entire testing session. Ethylacetate exposure and CNO injection lead to a decrease in exploratory behavior in mice previously fear conditioned to ethyl-acetate (n=6 and n=8 respectively).\\r\\n Previous exposure to ethyl-acetate without foot shock does not alter exploratory behavior upon re-exposure to ethyl-acetate or re-activation of Fos-tagged piriform ensembles (n=5 and n=7 respectively).\\r\\n (C) Distance traveled ratio test 2/test 1. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint \\r\\n Training Testing The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint baseline author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/297226 doi: bioRxiv preprint. SARS-CoV-2, also known as 2019-nCoV, is a novel coronavirus (CoVs) isolated from patients with pneumonia in China 2019. SARS-CoV-2 has a similar incubation period (median, 3.0 days) and a relatively lower fatality rate than SARS-CoV or MERS-CoV (1), but it is estimated that the reproductive number of SARS-CoV-2 is higher than that of SARS-CoV (2) .\\r\\n What's more, some laboratory-confirmed symptomatic cases are absent of apparent cough, fever or radiologic manifestations, making it difficult to timely and accurately find out all infected patients (3, 4) . As of February 28, 2020, patients infected by SARS-CoV-2 have been diagnosed in more than 30 countries, and more than 78,000 confirmed cases and 2,700 deaths associate with SARS-CoV-2 infection are reported in China alone.\\r\\n The genetic information of a virus is essential for not only its classification and traceability, but also its pathogenicity. At the whole genome level, the sequence identify of SARS-CoV-2 was 50% to MERS-CoV, 79% to SARS-CoV, 88% to two bat-derived SARS-like coronaviruses, Bat-SL-CoVZC45 and Bat-SL-CoVZXC21 (collected in 2018 in Zhoushan, China) , and 96% to Bat-SARSr-CoV RaTG13 (collected in 2013 in Yunnan, China) (5) . Each genome of all SARS-CoV-2 strains now submitted online contains nearly 29,900 nucleotides (nt), which are predicted with at least 14 open reading frames (ORFs) (5′ to 3′), such as ORF1ab (P, 21,291 nt), spike (S, 3,822 nt), ORF3a (8,28 nt), envelope (E, 228 nt), membrane (M, 669nt), ORF8 (366 nt), and nucleocapsid (N,1,260 nt) (6) . Among them, the spike gene encoded a glycoprotein that is crucial for determining host tropism and transmission capacity, and highly divergent when compared with that of bat-SARSr-CoV RaTG13 (93.1% nucleotide identity) (6, 7) .\\r\\n Generally, the rates of nucleotide substitution of RNA viruses are faster than their hosts, and . CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint this rapid evolution is mainly shaped by natural selection (mostly purifying selection) (8) . Gene mutations such as nucleotide substitutions, deletions and insertions have been frequently reported when comparing SARS-CoV-2 with other viruses (5) (6) (7) . In this work, we investigated the mutation pattern of SARS-CoV-2 by comprehensive comparative genomic analysis of the nonsynonymous/ synonymous substitution, relative synonymous codon usage (RSCU) and selective pressure in order to explore their potential in evolution and function.\\r\\n The SARS-CoV-2 reference genome Wuhan-Hu-1 (NC_045512) and WIV04 (MN996528) was downloaded from the GenBank database. Other newly sequenced 2019-nCoV genomes were downloaded from the Global Initiative on Sharing Avian Influenza Data (GISAID) database (https://www.gisaid.org/). Twenty-one closely related coronavirus complete genome sequences and their coding sequences (CDS) were downloaded from GenBank database (Supplementary   Table S1 ).\\r\\n Genome sequences were aligned using MUSCLE v3.8.31 (9), followed by manual adjustment using BioEdit v7.2.5. Phylogenetic analyses of complete genome were performed using maximum-likelihood method and general time-reversible model of nucleotide substitution with gamma-distributed rates among sites (GTR+G) in RAxML v8.1.21 (10) . Support for the inferred relationships was evaluated by a bootstrap analysis with 1000 replicates and trees were rooted with the alpha-coronavirus lineage as an outgroup. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint\\r\\n The coding sequences were translated and aligned using the MEGA-X program (11) , and then codon-based sequence alignment was used for further analysis. Phylogenetic analyses of coding sequences were performed using MEGA-X software. The changes of amino acids or nucleotides for each CDS sequences were analyzed using in-house Perl script.\\r\\n The number of synonymous substitutions per synonymous site (dS), and the number of non-synonymous substitutions per non-synonymous site (dN), for each coding region were calculated using the Nei-Gojobori method (Jukes-Cantor) in PAML package (12) .\\r\\n The adaptive evolution server (http://www.datamonkey.org/) was used to identify eventual sites of positive selection. For this purpose, the following test has been used: mixed-effects model of evolution (MEME), which allows the distribution of dN/dS (ω) to vary from site to site and from branch to branch at a site (13) . This test allowed us to infer episodic and pervasive positive selection at individual sites.\\r\\n To investigate the potential relative synonymous codon usage (RSCU) bias of the spike protein from SARS-CoV-2 and its closely related coronaviruses, the coding sequence (CDS) of spike protein in these coronaviruses were calculated with CodonW 1.4.4 (http://codonw.sourceforge.net/). The RSCU of human genes was retrieved from the Codon Usage Database (http://www.kazusa.or.jp/codon/). The potential relationships among these sequences were calculated with a squared Euclidean distance ( u h u ). Besides RSCU, the effective number of codons (ENc) was used as a simple metrics to verify codon bias and to explore the source of virus.\\r\\n . CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint\\r\\n Statistical analyses were performed using the R statistical package (version 3.\\r\\n Chi-squared test was used to compare any two data sets, and data were considered significantly different if the two-tailed p value was less than 0.05.\\r\\n The newly identified SARS-CoV-2 strain WIV04 genome sequence was closely related with Bat-SARSr-CoV RaTG13 and Bat-SL-CoVZC45, which was collected from host Rhinolophus affinis (5) . Compared with RaTG13 genome, many nucleotide substitutions are observed, but there are only five small inserts and deletions (indels) mutations, and the largest insert segment in WIV04 genome was \\\" CGGCGGGCACGT \\\" sequence, which is located near the boundary of S1 and S2 regions of spike protein. Interestingly, only synonymous mutation are observed near this insertion sequence (Fig. 1 , panel C). Compared with Bat-SL-CoVZC45 genome, this insert segment is detected as well, but non-synonymous mutations are also observed around it. Then, we further compared the proportion of synonymous mutations in spike gene between WIV04 and RaTG13 or Bat-SL-CoVZC45. The ratio of nucleotide substitutions (263 NT) to amino acid substitutions (29 AA) was 9.07 from WIV04 to RaTG13, which was significantly higher than the ratio (3.91, 864/221) from WIV04 to Bat-SL-CoVZC45 (p < 0.05) (Fig. 2) . Moreover, we also observed that the synonymous mutations increased in whole genome level between WIV04 and RaTG13 than that between WIV04 and Bat-SL-CoVZC45.\\r\\n Next, the detailed nucleotide changes of each comparison group were analyzed, and we found that the T-to-C (T:C) transition mutation was enriched in the nucleotide substitutions between WIV04 and RaTG13 (Fig. 1B) . It's reported that the CoVs lacking a 3'-to-5' exoribonuclease (ExoN) accumulated 15 to 20 fold more A:G and U:C transition (14) . What's more, the RNA mutagen 5-fluorouracil (5-FU) treatment will also increase the U:C and A:G transitions. Consequently, this increased T:C mutation implies the ExoN of SARS-CoV-2 may be deactivated compared with that of RaTG13. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint Furthermore, we checked whether the ratio of synonymous substitution to missense substitution is increased within SARS-CoV-2 strains. From December 2019 to February 2020, the genome sequences of 108 strains of SARS-CoV-2 virus have been submitted to GISAID database worldwide. Compared with the standard SARS-CoV-2 strain WIV04, total 98 point mutations were detected at 93 nucleotide sites of all SARS-CoV-2 strains with genome sequence available on Feb. 25 2020 (Fig. 1A) . However, only 58 of these nucleotide mutations caused changes in amino acids. Among them, 15 nucleotide substitutions at 14 sites caused changes in 7 amino acids of the spike protein were observed. Consequently, the proportion of synonymous mutations (~40%) among all current reported SARS-CoV-2 strains is similar to that between WIV04 and Bat-SL-CoVZC45 (39.1%), but is dramatically less than that between WIV04 and RaTG13 (90.7%).\\r\\n According to random drift hypothesis (15) ， these nucleotide mutations among different SARS-CoV-2 strains now available are still determined by neutral evolution. In short, there has no powerful factor to force SARS-CoV-2 to evolve in a certain direction by far. However, we should take strict precautions against the strong factors that may cause directional variation of SARS-CoV-2 both in natural environment and infection treatment.\\r\\n To investigate whether the mutation pattern between SARS-CoV-2 and RaTG13 is unique across all coronavirus species, we further compared their alterations of nucleotides and amino acids in comparison to other representative coronaviruses. Phylogenetic analysis of SARS-CoV-2 and its 20 closely related coronaviruses formed four well supported clades (Fig. 2) . The two SARS-CoV-2 strain WIV04 (from Wuhan) and SNU01 (from Korea patient) were clustered with SARS-CoV-related strains to form clade 1, belonging to beta-coronavirus lineage B. The Compared with Bat-SARSr-CoV RaTG13, there were 293 nucleotideswere altered while only 29 amino acids were altered in SARS-CoV-2; their ratio was as high as 9.07. However, this ratio detected in the comparisons of other human coronaviruses with their similar animal coronaviruses was less than 5.0. For example, the ratio of human SARS-CoV Tor2 to bat SARS-like CoVs LYRa11 was 4.81; that ratio of human MERS-CoV EMC2012 to bat SARS-like CoVs RSA2011 was 2.80; that ratio of human coronavirus 229E to bat coronavirus HKU2 was 4.55. These results indicate that the relative level of synonymous substitutions between human SARS-CoV-2 and its possible animal origin (RaTG13) is much higher than that between other human coronaviruses and their potential animal sources.\\r\\n Different organisms, even different protein coding genes of the same species, have different frequency of codon usage (16) . The RSCU bias will reveal the difference of the host source. We calculated the distance of RSCU between spike genes of 20 representative coronaviruses and\\r\\n and Bat-SARSr-CoV RaTG13 was 1.17, which was the lowest except for SNU01 (another strain of SARS-CoV-2), indicating that their codon preference was almost the same; the second lowest codon usage difference was 2.41 that was detected between SARS-CoV-2 WIV04 and Bat-SL-CoVZC45 (Fig. 2) . The coronaviruses in same beta-CoV lineage B have a relatively close \\r\\n To infer whether the retention of synonymous mutations is supported or hindered by natural selection, we further studied non-synonymous substitution rate (dN) and synonymous substitution rate (dS) in spike gene (Table 1) . Generally, positive (Darwinian) selection increases, but negative (purifying) selection decreases the ratios of non-synonymous to synonymous substitution rates (dN/dS). The results showed that both dN and dS of S gene of SARS-CoV-2 WIV04 versus Bat-SARr-CoV RaTG13 were the lowest among all typical coronaviruses, while those of SARS-CoV Tor2 versus bat SARS-like coronavirus WIV1 were the second lowest. When the ratio of dN to dS of spike gene is compared, all the tested dN/dS are less than 1, indicating that these non-synonymous mutations are harmful, and negative selection will reduce their retention speed. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint Among them, dN/dS of spike gene of SARS-CoV-2 WIV04 versus that of Bat-SARr-CoV RaTG13 was 0.04, which was the lowest among all comparisons, reconfirming that the rate of synonymous mutation was increased between WIV04 and RaTG13 strains. Moreover, dN/dS rate of polyprotein (ORF1ab) and nucleocapsid (N) genes were similar with that of spike gene (Table   S2 ).\\r\\n The spike (S) protein undergoes several drastic changes during virus infection. For instance, its large parts are cleaved during infection by cellular proteases and expose the receptors to activate viral attachment to the host (17) . As the receptor binding domain (RBD) of spike protein was involved in interacting with human angiotensin-converting enzyme 2 (ACE2) protein, the RBD region was thought to be preferential targets of natural selection (18) . Consistent with this hypothesis, our results showed that both dN and dS of RBD region are increased in comparison with whole spike gene region across all the virus pairs used in this study (Table 1) . Notably, the dN/dS ratio of RBD region in SARS-CoV-2 WIV04 was dramatically increased in about 3 fold in comparison with full spike region. Consequently, these mutations may be subjected to Darwin's choice and will become evidence of adaptive protein evolution.\\r\\n To detect positive selection on the spike gene, the MEME analysis was performed.\\r\\n Significant (p < 0.05) pervasive episodic selection was detected in three sites (48th, 254th, 330th position using the reference sequence of WIV04). On the 254th position of the spike amino acid sequence, there is a histidine residue instead of a phenylalanine residue, whereas on the 330th amino acidic position in WIV04 sequence, there is a glutamine residue instead of an valine residue.\\r\\n The results described above strongly supporting the action of positive selection in spike gene during the recent evolution of SARS-CoV-2 and RaTG13. However, this result should be . CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint addressed with careful explain, because the RBD region of spike gene from SARS-CoV-2 was quit divergent with that of RaTG13 (Fig. S1) , suggesting it might have originated from homologous recombination between RaTG13 and one yet-unknown coronavirus (19) .\\r\\n Through comprehensive comparative analysis between SARS-CoV-2 and other coronaviruses, we found the synonymous mutations is dramatically elevated between SARS-CoV-2 and RaTG13 than that of other coronavirus strains, and the nucleotide mutations were enriched in T:C transition.\\r\\n The specific mutation pattern may caused by the loss function of RNA 3'-to-5' exoribonuclease (ExoN). Moreover, as the SARS-CoV-2 was supposed to be originated from Bat-SARSr-CoV RaTG13, the increased synonymous substitution between SARS-CoV-2 and RaTG13 strain suggested the SARS-CoV-2 genome was under stronger negative (purifying) selection. We also detected some sites in spike protein was experiencing positive selection. These observations suggest that adaptive evolution might contribute to its host shifts. However, as the RNA mutagen (e.g. 5-FU) could induce the same mutation pattern, the mechanism of the mutation pattern observed between SARS-CoV-2 and RaTG13 should be further investigated in future work. The number of amino acid changes. RSCUdist: the RSCU distance of spike protein; ENc: effective number of codons; S ident(%): the percent identity of S protein sequence between different coronaviruses. The NT/AA ratio between WIV04/RaTG13 was statistically significant higher (Chi-squared test, P<0.01) than that of other strain pairs.\\r\\n . CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint *The whole genome sequence identity between these paired strains was larger than 95%.\\r\\n . CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.969006 doi: bioRxiv preprint. The COVID-19 outbreak began on December 2019 in Wuhan, China (1), and the causative virus, SARS-CoV-2, has rapidly spread into 43 countries as of February 27, 2020. The number of infected patients exceeds 81,000, and the death toll exceeds 2,700. Several drugs, including lopinavir, remdesivir, and chloroquine, have been reported to be presumably effective in treating this disease (2, 3) .\\r\\n Systemic treatment with corticosteroids is contraindicated for the severe pneumonia caused by viruses such as MERS-CoV and SARS-CoV, as steroids suppress the innate immune system, resulting in increased viral replication. In fact, for SARS in 2003 and MERS in 2013, treatment with corticosteroids was associated with increased mortality (4, 5). In the present study, we reconsider the use of inhaled corticosteroids, which have been excluded from the treatment of pneumonia caused by coronavirus.\\r\\n Steroid compounds, which are expected to have dual functions in blocking both coronavirus replication and host inflammation, were screened from a chemical library. The cytopathic effect caused by MERS-CoV infection was measured to evaluate viral replication. Four steroid compounds, ciclesonide, mometasone furoate, mifepristone, and algestone acetophenide conferred a greater than 95% cell survival rate (Fig. S1 ).\\r\\n Next, concentration-dependent viral growth suppression and drug cytotoxicity were assessed. Ciclesonide exhibited low cytotoxicity and potent suppression of viral growth (Fig.   1a ). Cortisone and prednisolone, which are commonly used for systemic steroid treatment, dexamethasone, which has strong immunosuppressant effects, and fluticasone, a commonly used inhaled steroid, did not suppress viral growth (Fig. 1a) .\\r\\n The antiviral effects of steroids against various species of viruses were tested by quantifying propagated virus in culture medium. Ciclesonide and mometasone also suppressed replication of other coronaviruses, HCoV-229E and SARS-CoV, but not replication of RS virus or influenza virus (Fig. S2 ). In addition, ciclesonide slightly but significantly inhibited rubella virus (having positive strand RNA genome) replication (Fig. S2 ), suggesting that ciclesonide interacted with the replication site of positive-strand RNA virus intracellularly.\\r\\n To identify the drug target of virus replication, we conducted 11 consecutive MERS-CoV passages in the presence of 40 μ M ciclesonide or 40 μ M mometasone. A mutant virus that developed resistance ciclesonide was generated (Fig. 1b) , but no resistant virus to author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.11.987016 doi: bioRxiv preprint mometasone was generated. Next-generation sequencing identified that an amino acid substitution at A25V (C19647T in the reference sequence NC_019843.3) in nonstructural protein 15 (NSP15), coronavirus endoribonuclease (6-8), was the predicted mechanism for viral resistance to ciclesonide. A recombinant virus carrying the A25V amino acid substitution (Re-Nsp15-A25V) was generated from the parental MERS-CoV/EMC strain (Re-EMC/MERS) with a BAC reverse genetics system (9), which overcame the antiviral effect of ciclesonide (Fig. 1c) . Interestingly, the mutant virus was inhibited by mometasone, suggesting that the antiviral target of mometasone was different from that of ciclesonide.\\r\\n Next, the effect of ciclesonide on SARS-CoV-2 infection was evaluated. VeroE6/TMPRSS2 cells(10) were infected with authentic SARS-CoV-2 in the presence of steroids or other inhibitors. At 6 h post-infection, cellular RNA was isolated, and real-time PCR was conducted to quantify the amount of viral RNA (11). Ciclesonide and mometasone suppressed viral replication with a similar efficacy to lopinavir (Fig. 2a) . For comparison, viral cell entry inhibitors were tested. E64d substantially reduced viral RNA levels, but nafamostat and camostat had only modest antiviral effects, suggesting that SARS-CoV-2 primarily utilized the cathepsin/endosomal pathway of cell entry rather than the TMPRSS2/cell surface pathway to enter TMPRSS2-expressing cells, consistent with a recent study (12). author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https: //doi.org/10.1101 //doi.org/10. /2020 The copyright holder for this preprint (which was not peer-reviewed) is th . https://doi.org/10.1101/2020.03.11.987016 doi: bioRxiv preprint a RNA copy (log 10 ) SARS-CoV-2 RNA in cell (6 hpi Cell viability (%) author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is th . https://doi.org/10.1101/2020.03.11.987016 doi: bioRxiv preprint. Seventeen years have passed since the outbreak of severe acute respiratory syndrome (SARS) in 2003, but there is yet no approved treatment for infections with the SARS coronavirus (SARS-CoV). 1 One of the reasons is that despite the devastating consequences of SARS for the affected patients, the development of an antiviral drug against this virus would not be commercially viable in view of the fact that the virus has been rapidly contained and did not reappear since 2004. As a result, we were empty-handed when the Middle-East respiratory syndrome coronavirus (MERS-CoV), a close relative of SARS-CoV, emerged in 2012. 2 MERS is characterized by severe respiratory disease, quite similar to SARS, but in addition frequently causes renal failure 3 . Although the number of registered MERS cases is low (2494 as of November 30, 2019; www.who.int), the threat MERS-CoV poses to global public health may be even more serious than that presented by SARS-CoV. This is related to the high casefatality rate (about 35%, compared to 10% for SARS), and to the fact that MERS cases are still accumulating seven years after the discovery of the virus, whereas the SARS outbreak was essentially contained within 6 months. The potential for human-to-human transmission of MERS-CoV has been impressively demonstrated by the 2015 outbreak in South Korea, where 186 cases could be traced back to a single infected traveller returning from the Middle East. 4 SARS-like coronaviruses are still circulating in bats in China, [5] [6] [7] [8] from where they may spill over into the human population; this is probably what caused the current outbreak of atypical pneumonia in Wuhan, which is linked to a seafood and animal market. The RNA genome (Gen-Bank accession code: MN908947.2; http://virological.org/t/initialgenome-release-of-novel-coronavirus/319, last accessed on January 11, 2020) of the new betacoronavirus features around 82% identity to that of SARS-CoV.\\r\\n In spite of the considerable threat posed by SARS-CoV and related viruses, as well as by MERS-CoV, it is obvious that the number of cases so far does not warrant the commercial development of an antiviral drug targeting MERS-and SARS-CoV even if a projected steady growth of the number of MERS cases is taken into account. A possible solution to the problem could be the development of broadspectrum antiviral drugs that are directed against the major viral protease, a target that is shared by all coronavirus genera as well as, in a related form, by members of the large genus Enterovirus in the picornavirus family. Among the members of the genus Alphacoronavirus are the human coronaviruses (HCoV) NL63 (ref. 9) and 229E 10 that usually cause only mild respiratory symptoms in otherwise healthy individuals, but are much more widespread than SARS-CoV or MERS-CoV. Therapeutic intervention against alphacoronaviruses is indicated in cases of accompanying disease such as cystic fibrosis 11 or leukemia, 12 or certain other underlying medical conditions. 13 The enteroviruses include pathogens such as EV-D68, the causative agent of the 2014 outbreak of the \\\"summer flu\\\" in the US, 14 EV-A71 and Coxsackievirus A16 (CVA16), the etiological agents of Hand, Foot, and Mouth Disease (HFMD), 15 Coxsackievirus B3 (CVB3), which can cause myocardic inflammation, 16 and human rhinoviruses (HRV), notoriously known to lead to the common cold but also capable of causing exacerbations of asthma and COPD. 17 Infection with some of these viruses can lead to serious outcome; thus, EV-D68 can cause polio-like disease, 18 and EV-A71 infection can proceed to aseptic meningitis, encephalitis, pulmonary edema, viral myocarditis, and acute flaccid paralysis. 15, [19] [20] Enteroviruses cause clinical disease much more frequently than coronaviruses, so that an antiviral drug targeting both virus families should be commercially viable.\\r\\n However, enteroviruses are very different from coronaviruses. While both of them have a single-stranded RNA genome of positive polarity, that of enteroviruses is very small (just 7 -9 kb) whereas coronaviruses feature the largest RNA genome known to date (27 -34 kb) . Enteroviruses are small, naked particles, whereas coronaviruses are much larger and enveloped. Nevertheless, a related feature shared by these two groups of viruses is their type of major protease, 21 which in the enteroviruses is encoded by the 3C region of the genome (hence the protease is designated 3C pro ). In coronaviruses, non-structural protein 5 (Nsp5) is the main protease (M pro ). Similar to the enteroviral 3C pro , it is a cysteine protease in the vast majority of cases and has therefore also been called \\\"3C-like protease\\\" (3CL pro ). The first crystal structure of a CoV M pro or 3CL pro (ref. 22) revealed that two of the three domains of the enzyme together resemble the chymotrypsin-like fold of the enteroviral 3C pro , but there is an additional a-helical domain that is involved in the dimerization of the protease (Fig. 1A) . This dimerization is essential for the catalytic activity of the CoV M pro , whereas the enteroviral 3C pro (Fig. 1B) functions as a monomer. Further, the enteroviral 3C pro features a classical Cys...His...Glu/Asp catalytic triad, whereas the CoV M pro only has a Cys...His dyad. 22 Yet, there are a number of common features shared between the two types of proteases, in particular their almost absolute requirement for Gln in the P1 position of the substrate and space for only small amino-acid residues such as Gly, Ala, or Ser in the P1' position, encouraging us to explore the coronaviral M pro and the enteroviral 3C pro as a common target for the design of broad-spectrum antiviral compounds. The fact that there is no known human protease with a specificity for Gln at the cleavage site of the substrate increases the attractivity of this viral target, as there is hope that the inhibitors to be developed will not show toxicity versus the host cell. Indeed, neither the enterovirus 3C pro inhibitor rupintrivir, which was developed as a treatment of the common cold caused by HRV, nor the peptide aldehyde inhibitor of the coronavirus M pro that was recently demonstrated to lead to complete recovery of cats from the normally fatal infection with Feline Infectious Peritonitis Virus (FIPV), showed any toxic effects on humans or cats, respectively. 23, 24 Figure 1: Crystal structures of SARS-CoV main protease (M pro , ref. 26 ; PDB entry 2BX4) and Coxsackivirus B3 3C protease (3C pro ; Tan et al., unpublished; PDB entry 3ZYD). Catalytic residues are indicated by spheres (yellow, Cys; blue, His; red: Glu). A. The coronavirus M pro is a homodimer, with each monomer comprising three domains. B. The structure of the monomeric CVB3 3C pro resembles the Nterminal two domains of the SARS-CoV M pro . Structure is on the same scale as image A. C. Superimpostion of residues from the two structures involved in ligand binding. Superimposition was carried out by aligning the catalytic Cys-His pair of each protease. Residues of the SARS-CoV M pro are shown with carbon atoms in cyan, CVB3 3C pro residues have orange carbons and are labeled with an asterisk (*).\\r\\n As the proteases targeted in our study all specifically cleave the peptide bond following a P1-glutamine residue (HCoV-NL63 M pro uniquely also accepts P1 = His at the Nsp13/Nsp14 cleavage site 31 ), we decided to use a 5-membered ring (g-lactam) derivative of glutamine (henceforth called GlnLactam) as the P1 residue in all our aketoamides (see Scheme 1). This moiety has been found to be a good mimic of glutamine and enhance the power of the inhibitors by up to 10-fold, most probably because compared to the flexible glutamine side-chain, the more rigid lactam leads to a reduction of the loss of entropy upon binding to the target protease. 29, 32 Our synthetic efforts therefore aimed at optimizing the substituents at the P1', P2, and P3 positions of the a-ketoamides.\\r\\n Synthesis (Scheme 1) started with the dianionic alkylation of N-Boc glutamic acid dimethyl ester with bromoacetonitrile. As expected, this alkylation occurred in a highly stereoselective manner, giving 1 as the exclusive product. In the following step, the cyano group of 1 was subjected to hydrogenation. The in-situ cyclization of the resulting intermediate afforded the lactam 2. The lactam derivative 3 was generated by removal of the protecting group of 2. On the other hand, the amidation of acyl chloride and a-amino acid methyl ester afforded the intermediates 4, which gave rise to the acids 5 via alkaline hydrolysis. The key intermediates 6 were obtained via the condensation of the lactam derivative 3 and the N-capped amino acids 5. The ester group of compounds 6 was then reduced to the corresponding alcohol. Oxidation of the alcohol products 7 by Dess-Martin periodinane generated the aldehydes 8, which followed by nucleophilic addition with isocyanides gave rise to compounds 9 under acidic conditions. Then, the a-hydroxyamides 10 were prepared by removing the acetyl group of compounds 9. In the final step, the oxidation of the exposed alcohol group in compounds 10 generated our target a-ketoamides 11.\\r\\n The inhibitory potencies of candidate a-ketoamides were evaluated against purified recombinant SARS-CoV M pro , HCoV-NL63 M pro , CVB3 3C pro , and EV-A71 3C pro . The most potent compounds were further tested against viral replicons and against SARS-CoV, MERS-CoV, or a whole range of enteroviruses in cell culture-based assays (Tables 1 -3 and Supplementary Table 1 \\r\\n To enable the rapid and biosafe screening of antivirals against corona-and enteroviruses, a non-infectious, but replication-competent SARS-CoV replicon was used 33 along with subgenomic replicons of CVB3 34 and EV-A71 (kind gift of B. Zhang, Wuhan, China). The easily detectable reporter activity (firefly or Renilla luciferase) of these replicons has previously been shown to reflect viral RNA synthesis. [33] [34] [35] In-vitro RNA transcripts of the enteroviral replicons were also used for transfection. For the SARS-CoV replicon containing the CMV promoter, only the plasmid DNA was used for transfection.\\r\\n The initial compound to be designed and synthesized was 11a, which carries a cinnamoyl N-cap in the P3 position, a benzyl group in P2, the glutamine lactam (GlnLactam) in P1, and benzyl in P1' (Table  1) . This compound showed good to mediocre activities against recombinant SARS-CoV M pro (IC50 = 1.95 µM; for all compounds, see Tables 1 -3 for standard deviations), CVB3 3C pro (IC50 = 6.6 µM), and EV-A71 3C pro (IC50 = 1.2 µM), but was surprisingly completely inactive (IC50 > 50 µM) against HCoV-NL63 M pro . These values were mirrored in the SARS-CoV and in the enterovirus replicons (Table 2) . In virus-infected cell cultures, the results obtained were also good to mediocre (Table 3) : SARS-CoV (EC50 = 5.8 µM in Vero E6 cells), MERS-CoV (EC50 = 0.0047 µM in Huh7 cells), HCoV 229E (EC50 = 11.8 µM in Huh7 cells), or a host of enteroviruses (EC50 = 9.8 µM against EV-A71 in RD cells; EC50 = 0.48 µM against EV-D68 in HeLa Rh cells; EC50 = 5.6 µM against HRV2 in HeLa Rh cells). In all cell types tested, the compound generally proved to be non-toxic, with selectivity indices (CC50/EC50) usually >10 (Table 3) .\\r\\n Crystal structures of compound 11a in complex with SARS-CoV M pro , HCoV-NL63 M pro , and CVB3 3C pro demonstrated that the aketo-carbon is covalently linked to the active-site Cys (no. 145, 144, and 147, resp.) of the protease (Fig. 2, 3a-c) . The resulting thiohemiketal is in the R configuration in the SARS-CoV and HCoV-NL63 M pro but in the S configuration in the CVB3 3C pro complex. The reason for this difference is that the oxygen atom of the thiohemiketal accepts a hydrogen bond from the catalytic His40 in the CVB3 protease, rather than from the main-chain amides of the oxyanion hole as in the SARS-CoV and HCoV-NL63 enzymes (Fig. 3a ,b,c insets). It is remarkable that we succeeded in obtaining a crystal structure of compound 11a in complex with the HCoV-NL63 M pro , even though it has no inhibitory effect on the activity of the enzyme (IC50 > 50 µM), (Fig. 2c) . Apparently, the compound is able to bind to this M pro in the absence of peptide substrate, but cannot compete with substrate for the binding site due to low affinity. A similar observation has been made in one of our previous studies, where we were able to determine the crystal structure of a complex between the inactive Michaelacceptor compound SG74 and the EV-D68 3C pro (ref. 29 ; PDB entry 3ZV9). Figure 2 : Fit of compound 11a (pink carbon atoms) to the target proteases (wheat surfaces) as revealed by X-ray crystallography of the complexes. A. Fo -Fc difference density (contoured at 3s) for 11a in the substrate-binding site of the SARS-CoV M pro (transparent surface). Selected side-chains of the protease are shown with green carbon atoms. B. Another view of 11a in the substrate binding site of the SARS-CoV M pro . Note the \\\"lid\\\" formed by residue Met49 and its neighbors above the S2 pocket. C. 11a in the substrate-binding site of HCoV-NL63 M pro . Because of the restricted size of the S2 pocket, the P2 benzyl group of the compound cannot enter deeply into this site. Note that the S2 pocket is also covered by a \\\"lid\\\" centred around Thr47. D. 11a in the substrate-binding site of the CVB3 3C pro . The S2 site is large and not covered by a \\\"lid\\\".\\r\\n The crystal structures indicated that the fits of the P1' benzyl group of 11a in the S1' pocket and of the P3 cinnamoyl cap in the S3 subsite might be improved (see Fig. 3a -c). Compounds 11b -11e and 11g -11l were synthesized in an attempt to do so; however, none of them showed better inhibitory activity against the majority of the recombinant proteases, compared to the parent compound, 11a (see Supplementary Results). To investigate whether the P3 residue of the inhibitor is dispensible, we synthesized compound 11f, which only comprises P2 = Boc, P1 = GlnLactam, and P1' = benzyl. 11f was inactive against all purified proteases and in all replicons tested, but showed some activity against HRV2 in HeLa Rh cells (EC50 = 9.0 µM). A crystal structure of 11f bound to HCoV-NL63 M pro demonstrated that the P2-Boc group entered the S2 pocket (Fig. 3d ). In conclusion, although there is probably room for further improvement, we decided to maintain the original design with P1' = benzyl and P3 = cinnamoyl, and focussed on improving the P2 substituen.\\r\\n The crystal structures of SARS-CoV M pro , HCoV-NL63 M pro , and CVB3 3C pro in complex with 11a revealed a fundamental difference between the S2 pockets of the coronavirus proteases and the enterovirus proteases: The cavities are covered by a \\\"lid\\\" in the former but are open to one side in the latter (Fig. 2,b-d) . In SARS-CoV M pro , the lid is formed by the 310 helix 46 -51 and in HCoV-NL63 M pro by the loop 43-48. Residues from the lid, in particular Met49 in the case of SARS-CoV M pro , can thus make hydrophobic interactions with the P2 substitutent of the inhibitor, whereas such interaction is missing in the enterovirus 3C pro s. In addition to the lid, the S2 pocket is lined by the \\\"back-wall\\\" (main-chain atoms of residues 186 and 188 and Cb atom of Asp187), the side-walls (Gln189, His41), as well as the \\\"floor\\\" (Met165) in SARS-CoV M pro . In HCoV-NL63 M pro , the corresponding structural elements are main-chain atoms of residues 187 and 188 as well as the Cb atom of Asp187 (back-wall), Pro189 and His41 (side-walls), and Ile165 (floor). Finally, in CVB3 3C pro , Arg39, Asn69, and Glu71 form the back-wall, residues 127-132 and His40 form the side-walls, and Val162 constitutes the floor.\\r\\n In addition, the S2 pocket is of different size in the various proteases. The SARS-CoV enzyme features the largest S2 pocket, with a volume of 252 Å 3 embraced by the residues (Gln189, His41) defining the side-walls of the pocket in the ligand-free enzyme, as calculated by using Chimera, 36 followed by the CVB3 3C pro S2 pocket with about 180 Å 3 (space between Thr130 and His40). The HCoV-NL63 M pro has by far the smallest S2 pocket of the three enzymes, with a free space of only 45 Å 3 between Pro189 and His41, according to Chimera.\\r\\n In agreement with these observations, a good fit is observed between the P2 benzyl group of 11a and the S2 subsite of the SARS-CoV M pro as well as that of the CVB3 3C pro (Fig. 3a,c) . In contrast, the crystal structure of the complex between 11a and HCoV-NL63 M pro , against which the compound is inactive, demonstrates that the P2-benzyl group cannot fully enter the S2 pocket of the enzyme because of the restricted size of this site (Fig. 3b) .\\r\\n Thus, the properties of our target proteases with respect to the S2 pocket were defined at this point as \\\"small\\\" and \\\"covered by a lid\\\" for HCoV-NL63 M pro , \\\"large\\\" and \\\"covered\\\" for SARS-CoV M pro , and \\\"large\\\" and \\\"open\\\" for CVB3 3C pro . Through comparison with crystal structures of other proteases of the same virus genus (HCoV-229E M pro for alphacoronaviruses 28 (PDB entry 1P9S); HKU4 M pro for betacoronaviruses (Ma, Xiao et al., unpublished; PDB entry 2YNA); and EV-A71 3C pro for enteroviruses 30 (PDB entry 3SGK), we ensured that our conclusions drawn from the template structures were valid for other family members as well.\\r\\n To explore the sensitivity of the S2 pocket towards a polar substituent in the para position of the benzyl group, we synthesized compound 11m carrying a 4-fluorobenzyl group in P2. This substitution abolished almost all activity against the SARS-CoV M pro (IC50 > 50 µM), and the compound proved inactive against HCoV-NL63 M pro as well, whereas IC50 values were 2.3 µM against the EV-A71 3C pro and 8.7 µM against CVB3 3C pro . From this, we concluded that the introduction of the polar fluorine atom is not compatible with the geometry of the S2 pocket of SARS-CoV M pro , whereas the fluorine can accept a hydrogen bond from Arg39 in EV-A71 3C pro (ref. 30 ) and probably also CVB3 3C pro . In SARS-CoV M pro , however, the carbonyl groups of residues 186 and 188 might lead to a repulsion of the fluorinated benzyl group. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.10.936898 doi: bioRxiv preprint Table 2 : a-ketoamide-induced inhibition of subgenomic RNA synthesis using replicons in a cell-based assay (EC50, μM) 1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.10.936898 doi: bioRxiv preprint Table 3 . The thiohemiketal is in the R configuration, with its oxygen accepting two hydrogen bonds from the oxyanion-hole amides of Gly143 and Cys145. The amide oxygen accepts an H-bond from His41. The side-chains of Ser144 and Arg188 have been omitted for clarity. B. The P2-benzyl substituent of 11a cannot fully enter the S2 pocket of the HCoV-NL63 M pro , which is much smaller and has less plasticity than the corresponding pocket of SARS-CoV M pro (cf. A). The benzyl therefore binds above the pocket in the view shown here; this is probably the reason for the total inactivity (IC50 > 50 µM) of compound 11a against HCoV-NL63 M pro . The small size of the pocket is due to the replacement of the flexible Gln189 of the SARS-CoV M pro by the more rigid Pro189 in this enzyme. The stereochemistry of the thiohemiketal is R. The sidechains of Ala143 and Gln188 have been omitted for clarity. C. Binding of 11a to the CVB3 3C pro . The stereochemistry of the thiohemiketal is S, as the group accepts a hydrogen bond from His41, whereas the amide keto group accepts three H-bonds from the oxyanion hole (residues 145 -147). The side-chain of Gln146 has been omitted for clarity. D. The crystal structure of 11f in complex with HCoV-NL63 M pro shows that this short (inactive) compound lacking a P3 residue has its P2-Boc group inserted into the S2 pocket of the protease. The stereochemistry of the thiohemiketal is S. The sidechains of Ala143 and Gln188 have been omitted for clarity. E. In contrast to P2 = benzyl in 11a, the isobutyl group of 11n is small and flexible enough to enter into the narrow S2 pocket of the HCoV-NL63 M pro . The thiohemiketal is in the R configuration. The side-chains of Ala143 and Gln188 have been omitted for clarity. F. In spite of its small size, the cyclopropylmethyl side-chain in the P2 position of 11s can tightly bind to the S2 subsite of the SARS-CoV M pro , as this pocket exhibits pronounced plasticity due to the conformational flexibility of Gln189 (see also Fig. 4 ). The stereochemistry of the thiohemiketal is S. The side-chains of Ser144 and Arg188 have been omitted for clarity.\\r\\n As the P2-benzyl group of 11a was apparently too large to fit into the S2 pocket of the HCoV-NL63 M pro , we replaced it by isobutyl in 11n. This resulted in improved activities against SARS-CoV M pro (IC50 = 0.33 µM) and in a very good activity against HCoV-NL63 M pro (IC50 = 1.08 µM, compare with the inactive 11a). For EV-A71 3C pro , however, the activity decreased to IC50 = 13.8 µM, different from CVB3 3C pro , where IC50 was 3.8 µM. Our interpretation of this result is that the smaller P2-isobutyl substituent of 11n can still interact with the \\\"lid\\\" (in particular, Met49) of the SARS-CoV M pro S2 site, but is unable to reach the \\\"back-wall\\\" of the EV-A71 3C pro pocket and thus, in the absence of a \\\"lid\\\", cannot generate sufficient enthalpy of binding. We will see from examples to follow that this trend persists among all inhibitors with a smaller P2 substituent: Even though the SARS-CoV M pro S2 pocket has a larger volume than that of the enterovirus 3C pro , the enzyme can be efficiently inhibited by compounds carrying a small P2 residue that makes hydrophobic interactions with the lid (Met49) and floor (Met165) residues.\\r\\n The EC50 of 11n was >10 µM against the EV-A71 and CVB3 replicons, and even in the SARS-CoV replicon, the activity of 11n was relatively weak (EC50 = 7.0 µM; Table 2 ). In agreement with the replicon data, 11n proved inactive against EV-A71 in RD cells and showed limited activity against HRV2 or HRV14 in HeLa Rh cells (Table 3) . Only the comparatively good activity (EC50 = 4.4 µM) against EV-D68 in HeLa Rh cells was unexpected. The activity of 11n against HCoV 229E in Huh7 cells was good (EC50 = 0.6 µM), and against MERS-CoV in Huh7 cells, it was excellent, with EC50 = 0.0048 µM, while in Vero cells, the EC50 against MERS-CoV was as high as 9.2 µM. Similarly, the EC50 against SARS-CoV in Vero cells was 14.2 µM (Table 3) .\\r\\n We managed to obtain crystals of 11n in complex with the M pro of HCoV NL63 and found the P2 isobutyl group to be well embedded in the S2 pocket (Fig. 3e) . This is not only a consequence of the smaller size of the isobutyl group compared to the benzyl group, but also of its larger conformational flexibility, which allows a better fit to the binding site.\\r\\n When we replaced the P2-isobutyl residue of 11n by n-butyl in 11o, the activities were as follows: IC50 = 8.5 µM for SARS-CoV M pro , totally inactive (IC50 > 50 µM) against HCoV-NL63 M pro , IC50 = 3.2 µM for EV-A71 3C pro , and 5.2 µM for CVB3 3C pro . The decreased activity in case of SARS-CoV M pro and the total inactivity against HCoV-NL63 M pro indicate that the n-butyl chain is too long for the S2 pocket of these proteases, whereas the slight improvement against EV-A71 3C pro and CVB3 3C pro is probably a consequence of the extra space that is available to long and flexible substituents because of the lack of a lid covering the enterovirus 3C pro pocket.\\r\\n As the n-butyl substituent in P2 of 11o was obviously too long, we next synthesized a derivative with the shorter propargyl (ethinylmethyl) as the P2 residue (compound 11p). This led to very mediocre activities against all tested proteases. Using cyclopropyl as the P2 residue (compound 11q), the IC50 values were even higher against most of the proteases tested. Obviously, the P2 side-chain requires a methylene group in the b-position in order to provide the necessary flexibility for the substituent to be embedded in the S2 pocket.\\r\\n Having realized that in addition to size, flexibility of the P2 substituent may be an important factor influencing inhibitory activity, we introduced flexibility into the phenyl ring of 11a by reducing it. The cyclohexylmethyl derivative 11r exhibited IC50 = 0.7 µM against SARS-CoV M pro , 12.3 µM against HCoV-NL63 M pro , 1.7 µM against EV-A71 3C pro , and 0.9 µM against CVB3 3C pro . Thus, the replace- 1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.10.936898 doi: bioRxiv preprint ment of the phenyl group by the cyclohexyl group led to a significant improvement of the inhibitory activity against the recombinant SARS-CoV M pro and to a dramatic improvement in case of CVB3 3C pro . Even for the HCoV-NL63 M pro , against which 11a was completely inactive, greatly improved albeit still weak activity was observed (Table 1 ). In the viral replicons, 11r performed very well, with EC50 = 0.8 -0.9 µM for the EV-A71 replicon, 0.45 µM for CVB3, and 1.4 µM for SARS-CoV (Table 2 ). In the virus-infected cell culture assays (Table 3) , 11r exhibited EC50 = 3.7 µM against EV-A71 in RD cells and EC50 = 0.48 -0.7 µM against EV-D68, HRV2, and HRV14 in HeLa cells. Against HCoV 229E in Huh7 cells, the EC50 was surprisingly low (1.8 µM). Interestingly, the compound proved extremely potent against MERS-CoV in Huh7 cells, with EC50 = 0.0004 µM (400 picomolar!). Even in Vero cells, EC50 against MERS-CoV was 5 µM, and the EC50 against SARS-CoV in Vero E6 cells was 1.8 -2.1 µM, i.e. the best activity we have seen for an M pro inhibitor against SARS-CoV in this type of cells. The therapeutic index (CC50/EC50) of 11r against EV-D68, HRV2, and HRV14 was >15 in HeLa Rh cells as well as against CVB3 in Huh-T7 cells, but only ~5 for EV-A71 in RD cells.\\r\\n At this point, we decided to systematically vary the size of the ring system in P2. The next substituent to be tried was cyclopropylmethyl (compound 11s, which showed good activities against SARS-CoV M pro (IC50 = 0.24 µM) and HCoV-NL63 M pro (1.4 µM), but poor values against EV-A71 3C pro (IC50 = 18.5 µM) and CVB3 3C pro (IC50 = 4.3 µM) (Table 1) . 11s was shown to inhibit the SARS-CoV replicon with an EC50 of about 2 µM, whereas activity against the EV-A71 and CVB3 replicons was poor (EC50 values > 20 µM) ( Table 2 ). The replicon results were mirrored by the antiviral activity of 11s in enterovirus-infected cells (Table 3) , which was weak or very weak. By contrast, the compound inhibited HCoV 229E and MERS-CoV in Huh7 cells with EC50 of 1.3 and 0.08 µM, respectively. The activity against the latter virus in Vero cells was poor (EC50 ~11 µM), and so was the anti-SARS-CoV activity in Vero E6 cells (Table 3) .\\r\\n We next analyzed the crystal structure of the complex between SARS-CoV M pro and compound 11s (Fig. 3f) . The cyclopropylmethyl substituent was found to be incorporated deeply into the S2 pocket, making hydrophobic interactions with Met49 (the lid), Met165 (the floor), and the Cb of Asp187 (the back-wall). In spite of the small size of the P2 substituent, this is possible because the S2 pocket of SARS-CoV M pro is flexible enough to contract and enclose the P2 moiety tightly. This plasticity is expressed in a conformational change of residue Gln189, both in the main chain and in the side-chain. The main-chain conformational change is connected with a flip of the peptide between Gln189 and Thr190. The c1 torsion angle of the Gln189 side-chain changes from roughly antiperiplanar (ap) to (-)synclinal (-sc) (Fig. 4) . The conformational variability of Gln189 has been noted before, both in Molecular Dynamics simulations 26 and in other crystal structures. 37 As a consequence of these changes, the sidechain oxygen of Gln189 can accept a 2.54-Å hydrogen bond from the main-chain NH of the P2 residue in the 11s complex (see Fig. 4 ). The affinity of 11s for the S2 pocket of HCoV-NL63 M pro is good because of an almost ideal match of size and not requiring conformational changes, which this enzyme would not be able to undergo because of the replacement of the flexible Gln189 by the more rigid Pro. On the other hand, docking of the same compound into the crystal structure of the CVB3 3C pro revealed that the cyclopropylmethyl moiety was probably unable to generate sufficient Free Energy of binding because of the missing lid and the large size of the S2 pocket in the enterovirus 3C pro , thereby explaining the poor inhibitory activity of 11s against these targets. (Table 1) . Experiments with the viral replicons confirmed this trend, although the EC50 value for SARS-CoV (6.8 µM) was surprisingly high (Table 2 ). In Huh7 cells infected with MERS-CoV, this compound exhibited EC50 = 0.1 µM (but 9.8 µM in Vero cells), whereas EC50 was 7.0 µM against SARS-CoV in Vero E6 cells. The compound was largely inactive against EV-A71 in RD cells and inhibited the replication of the two HRV subtypes tested (in HeLa Rh cells) with EC50 values of ~4 µM. The CC50 of 11t in HeLa cells was 65 µM, i.e. the therapeutic index was well above 15 (Table 3) .\\r\\n Obviously, this substituent was still a bit too small for the enterovirus proteases, so as the next step, we tested P2 = cyclopentylmethyl (compound 11u). This turned out to be the one compound with acceptable IC50 values against all tested enzymes: 1.3 µM against SARS-CoV M pro , 5.4 µM against HCoV-NL63 M pro , 4.7 µM against EV-A71 3C pro , and 1.9 µM against CVB3 3C pro ( Table 1 ). The activity against the replicons was between 3.6 and 4.9 µM ( Table 2 ). In Huh7 cells infected with HCoV 229E or MERS-CoV, 11u showed EC50 = 2.5 or 0.03 µM (11.1 µM for MERS-CoV in Vero cells), while EC50 was 4.9 µM against SARS-CoV in Vero E6 cells (Table 3) .\\r\\n 11u appeared so far the best compromise compound, yet for each of the individual viral enzymes, the following compounds proved superior: P2 = cyclopropylmethyl (compound 11s) for SARS-CoV M pro , P2 = isobutyl (compound 11n) and P2 = cyclopropylmethyl (11s) for HCoV-NL63 M pro , P2 = benzyl (11a) or cyclohexylmethyl (11r) for EV-A71 3C pro , and 11r for CVB3 3C pro . In other words, the nearly equipotent 11u is indeed a compromise. Therefore, in view of the surprisingly good antiviral activity of 11r against HCoV 229E in Huh7 cells, we relaxed the condition that the universal inhibitor should show good activity against the recombinant HCoV-NL63 M pro , and selected 11r (P2 = cyclohexylmethyl) as the lead compound for further development. This compound exhibited submicromolar IC50 values against CVB3 3C pro and SARS-CoV M pro , and IC50 = 1.7 µM against EV-A71 3C pro (Table 1) , as well as similarly low EC50 values in the replicons of these viruses (Table 2 ). In Huh7 cells infected with MERS-CoV, the performance of this compound was excellent, with EC50 = 0.0004 µM, and even against HCoV 229E in Huh7 cells and SARS-CoV in Vero E6 cells, EC50 values of 1.8 and 2.1 µM, respectively, were observed (Table 3) . Also in enterovirus-infected cell culture, the compound performed well, with EC50 values of 0.7 µM or below against HRV2, HRV14, and EV-D68 in HeLa (Rh) cells and selectivity values >15. The only concern is the activity of the compound against EV-A71 in RD cells, for which the EC50 value was 3.7 µM, resulting in too low a therapeutic index. On the other hand, only weak toxicity was detected for 11r in Vero or Huh-T7 cells. Preliminary pharmacokinetics tests with the compound in mice did not indicate a toxicity problem (to be published elsewhere).\\r\\n We describe here the structure-based design, the synthesis, and the assessment of capped dipeptide a-ketoamides that target the main protease of alpha-or betacoronaviruses as well as the 3C protease of enteroviruses. Through crystallographic analyses of a total of six inhibitor complexes of three different proteases in this study, we found the a-ketoamide warhead (-CO-CO-NH-) to be sterically more versatile than other warheads such as Michael acceptors (-CH=CH-CO-) and aldehydes (-CH=O), because it features two acceptors for hydrogen bonds from the protein, namely the a-keto oxygen and the amide oxygen, whereas the other warheads have only one such acceptor. In the various complexes, the hydroxy group (or oxyanion) of the thiohemiketal that is formed by the nucleophilic attack of the activesite cysteine residue onto the a-keto carbon, can accept one or two hydrogen bonds from the main-chain amides of the oxyanion hole. In addition, the amide oxygen of the inhibitor accepts a hydrogen bond from the catalytic His side-chain. Alternatively, the thiohemiketal can interact with the catalytic His residue and the amide oxygen with the main-chain amides of the oxyanion hole. Depending on the exact interaction, the stereochemistry at the thiohemiketal C atom would be different. We have previously observed a similar difference in case of aldehyde inhibitors, where the single interaction point -the oxyanion of the thiohemiacetal -can accept a hydrogen bond either from the oxyanion hole or from the catalytic His side-chain, 37 resulting in different stereochemistry of the thiohemiacetal carbon. Both aketoamides and aldehydes react reversibly with the catalytic nucleophile of proteases, whereas Michael acceptors form irreversible adducts.\\r\\n In addition to better matching the H-bonding donor/acceptor properties of the catalytic center through offering two hydrogen-bond acceptors instead of one, a-ketoamides have another big advantage over aldehydes and a,b-unsaturated esters (Michael acceptors) in that they allow easy extension of the inhibitors to probe the primed specificity subsites beyond S1', although this has so far rarely been explored (e.g., ref. 38 in case of calpain).\\r\\n The most prominent a-ketoamide drugs are probably telaprivir and boceprivir, peptidomimetic inhibitors of the HCV NS3/4A protease, 39, 40 which have helped revolutionize the treatment of chronic HCV infections. For viral cysteine proteases, a-ketoamides have only occasionally been described as inhibitors and few systematic studies have been carried out.\\r\\n A number of capped dipeptidyl a-ketoamides have been described as inhibitors of the norovirus 3C-like protease. 41 These were optimized with respect to their P1' substituent, whereas P2 was isobutyl in most cases and occasionally benzyl. The former displayed IC50 values one order of magnitude lower than the latter, indicating that the S2 pocket of the norovirus 3CL protease is fairly small. Although we did not include the norovirus 3CL pro in our study, expanding the target range of our inhibitors to norovirus is probably a realistic undertaking. While our study was underway, Zeng et al. 42 published a series of a-ketoamides as inhibitors of the EV-A71 3C pro . These authors mainly studied the structure-activity relationships of the P1' residue and found small alkyl substituents to be superior to larger ones. Interestingly, they also reported that a six-membered d-lactam in the P1 position led to 2 -3 times higher activities, compared to the fivemembered g-lactam. At the same time, Kim et al. 43 described a series of five a-ketoamides with P1' = cyclopropyl that showed submicromolar activity against EV-D68 and two HRV strains.\\r\\n Occasionally, individual a-ketoamides have been reported in the literature as inhibitors of both the enterovirus 3C protease and the coronavirus main protease. A single capped dipeptidyl a-ketoamide, Cbz-Leu-GlnLactam-CO-CO-NH-iPr, was described that inhibited the recombinant transmissible gastroenteritis virus (TGEV) and SARS-CoV M pro s as well as human rhinovirus and poliovirus 3C pro s in the one-digit micromolar range. 44 Coded GC-375, this compound showed poor activity in cell culture against EV-A71 though (EC50 = 15.2 µM), probably because P2 was isobutyl. As we have shown here, an isobutyl side-chain in the P2 position of the inhibitors is too small to completely fill the S2 pocket of the EV-A71 3C pro and the CVB3 3C pro .\\r\\n Among a series of aldehydes, Prior et al. 45 described the capped tripeptidyl a-ketoamide Cbz-1-naphthylalanine-Leu-GlnLactam-CO-CO-NH-iPr, which showed IC50 values in the 3-digit nanomolar range against HRV 3C pro and SARS-CoV M pro , as well as EC50 values of 0.03 µM against HRV18 and 0.5 µM against HCoV 229E in cell culture. No optimization of this compound was performed and no toxicity data have been reported.\\r\\n For compounds with warheads other than a-ketoamides, in-vitro activity against both corona-and enteroviruses has also occasionally been reported. Lee et al. 46 described three peptidyl Michael acceptors that displayed inhibitory activity against the M pro s of SARS-CoV and HCoV 229E as well as against the 3C pro of CVB3. These inhibitors had an IC50 10 -20 times higher for the CVB3 enzyme, compared to SARS-CoV M pro . P2 was invariably isobutyl (leucine) in these compounds, suggesting that further improvement might be possible.\\r\\n In addition to Michael acceptors, peptide aldehydes have also been used to explore the inhibition of coronavirus M pro s as well as enterovirus 3C pro s. Kim et al. 44 reported a dipeptidyl aldehyde and its bisulfite adduct, both of which exhibited good inhibitory activities against the isolated 3C proteases of human rhinovirus and poliovirus as well as against the 3C-like proteases of a number of coronaviruses, but antiviral activities in cell culture against EV-A71 were poor (EC50 >10 µM), again most probably due to P2 being isobutyl (leucine).\\r\\n In our series of compounds, we used P1 = GlnLactam (g-lactam) throughout, because this substituent has proven to be an excellent surrogate for glutamine. 29, 32 While we made some efforts to optimize the P1' residue of the compounds as well as the N-cap (P3), we mainly focussed on optimization of the P2 substituent. In nearly all studies aiming at discovering peptidomimetic inhibitors of coronavirus M pro s, P2 is invariably isobutyl (leucine), and this residue has also been used in the efforts to design compounds that would inhibit enterovirus 3C pro s as well (see above). From crystal structures of our early lead compound, 11a (cinnamoyl-Phe-GlnLactam-CO-CO-NH-Bz), in complex with the M pro s of HCoV NL63 (as representative of the alphacoronavirus proteases) and SARS-CoV (beta-CoV) as well as the 3C pro of Coxsackievirus B3 (enterovirus proteases), we found that the S2 pocket has fundamentally different shapes in these enzymes. In the SARS-CoV M pro , the S2 subsite is a deep hydrophobic pocket that is truly three-dimensional in shape: the \\\"walls\\\" of the groove are formed by the polypeptide main chain around residues 186 -188 as well as by the side-chains of His41 (of the catalytic dyad) and Gln189, whereas the \\\"floor\\\" is formed by Met165 and the \\\"lid\\\" by residues 45 -51, in particular Met49. The two methionines provide important interaction points for the P2 substituents of inhibitors; while these interactions are mostly hydrophobic in character, we have previously described the surprising observation of the carboxylate of an aspartic residue in P2 that made polar interactions with the sulfur atoms of these methionines. 37 Because the pocket offers so many opportunities for interaction and features a pronounced plasticity, P2 substituents such as isobutyl (from Leu), which are too small to fill the pocket entirely, can still generate sufficient binding enthalpy. Accordingly, the S2 pocket of SARS-CoV M pro is the most tolerant among the three enzymes investigated here, in terms of versatility of the P2-substituents accepted.\\r\\n In the S2 pocket of the HCoV-NL63 M pro , Gln189 is replaced by proline and this change is accompanied by a significant loss of flexibility; whereas the side-chain of Gln189 of SARS-CoV M pro is found to accommodate its conformation according to the steric requirements of the P2 substituent, the proline is less flexible, leading to a much smaller space at the entrance to the pocket. As a consequence, a P2benzyl substituent is hindered from penetrating deeply into the pocket, whereas the smaller and more flexible isobutyl group of P2-Leu is not.\\r\\n Finally, in the 3C pro s of EV-A71 and CVB3, the S2 pocket lacks a lid, i.e. it is open to one side. As a consequence, it offers less interaction points for P2 substituents of inhibitors, so that such substituents must reach the \\\"back-wall\\\" of the pocket (formed by Arg39, Asn69, Glu71) in order to create sufficient binding energy. Hence, large aromatic substituents such as benzyl are favored by the enterovirus 3C pro s.\\r\\n When we introduced a fluoro substituent in the para position of the P2-benzyl group of our lead compound, 11a, we observed good activity against the enterovirus 3C pro s but complete inactivity against the coronavirus M pro s (see Table 1 , compound 11m). This is easily explained on the basis of the crystal structures: In the enterovirus 3C pro s, the fluorine can accept a hydrogen bond from Arg39 (ref. 30) , whereas in the coronavirus M pro s, there would be electrostatic repulsion from the main-chain carbonyls of residues 186 and 188. In agreement with this, rupintrivir (which has P2 = p-fluorobenzyl) is a good inhibitor of the enteroviral 3C pro s, 46 but not of the coronaviral main proteases, as we predicted earlier. 28 In this structure-based inhibitor optimization study, we achieved major improvements over our original lead compound, 11a, by systematically varying the size and the flexibility of the P2-substituent. The compound presenting so far the best compromise between the different requirements of the S2 pockets (SARS-CoV M pro : large and covered, HCoV-NL63 M pro : small and covered, and CVB3 3C pro : large and open) is 11u (P2 = cyclopentylmethyl), which has satisfactory broad-spectrum activity against all proteases tested. However, with regard to its antiviral activities in cell cultures, it is inferior to 11r (P2 = cyclohexylmethyl). The latter compound exhibits very good inhibitory activity against the SARS-CoV M pro as well as the enterovirus 3C pro , and its performance in the SARS-CoV and enterovirus replicons is convincing. Being in the low micromolar range (EV-A71, CVB3), the data for the antiviral activity in cell culture for 11r correlate well with the inhibitory power of the compound against the recombinant proteases as well as in the replicon-based assays. This is not true, though, for the surprisingly good in-cellulo activity of 11r against HCoV 229E in Huh7 cells. Also, the correlation does not seem to hold for LLC-MK2 and CaCo2 cells. We tested the antiviral activity of many of our compounds against HCoV NL63 in these two cell types and found that all of them had low-or submicromolar EC50 values against this virus in LLC-MK2 cells but were largely inactive in CaCo2 cells (not shown). Furthermore, 11r and all other compounds that we synthesized are inactive (EC50 > 87 µM) against CVB3 in Vero cells (not shown), but exhibit good to excellent activities against the same virus in Huh-T7 cells. We have previously observed similar poor antiviral activities in Vero cells not only for aketoamides, but also for Michael acceptors (Zhu et al., unpublished work) . A similar cell-type dependence is seen for the antiviral activity of 11r against MERS-CoV and SARS-CoV. Whereas the inhibitor exhibits excellent activity against MERS-CoV when Huh7 cells are the host cells (400 pM), the inhibitory activity is weaker by a factor of up to 12,500 when Vero cells are used (EC50 = 5 µM). On the other hand, 11r exhibits excellent anti-MERS-CoV activity in human Calu3 lung cells, i.e. in the primary target cells where the compound will have to act in a therapeutic setting (A. Kupke, personal communication). As we tested antiviral activity against SARS-CoV exclusively in Vero cells, the EC50 values determined for our compounds against this virus are in the one-digit micromolar range or higher; the best is again compound 11r with EC50 = 2.1 µM. Interestingly, the relatively weaker activity (or even inactivity) of our inhibitors against RNA viruses in Vero cells was observed independently in the virology laboratories in Leuven and in Leiden. It is thus unlikely that the lack of activity in Vero cells is related to problems with the experimental set-up. In preliminary experiments, we replaced the P3 cinnamoyl group of 11r by the fluorophor coumaryl and found by fluorescence microscopy that much more inhibitor appeared to accumulate in Huh7 cells compared to Vero cells (D.L., R.H. & Irina Majoul, unpublished).\\r\\n Regardless of which cell system is the most suitable one for the testing of peptidomimetic antiviral compounds, we next plan to test 11r in small-animal models for MERS and for Coxsackievirusinduced pancreatitis. In parallel, we aim to refine the experiments to quantify the accumulation of peptidomimetic protease inhibitors in different host-cell types, in the hope to find an explanation for the observed cell-type dependencies.\\r\\n This work demonstrates the power of structure-based approaches in the design of broad-spectrum antiviral compounds with roughly equipotent activity against coronaviruses and enteroviruses. We observed a good correlation between the inhibitory activity of the designed compounds against the isolated proteases, in viral replicons, and in virus-infected Huh7 cells. One of the compounds (11r) exhibits excellent anti-MERS-CoV activity in virus-infected Huh7 cells. Because of the high similarity between the main proteases of SARS-CoV and the novel BetaCoV/Wuhan/2019, we expect 11r to exhibit good antiviral activity against the new coronavirus as well.\\r\\n Crystallization. The recombinant production and purification of SARS-CoV M pro with authentic N and C termini was described in detail previously. 48, 49 Using an Amicon YM10 membrane (EMD Millipore), the purified SARS-CoV M pro was concentrated to 21 mg·mL -1 in buffer A (20 mM Tris-HCl, 150 mM NaCl, 1 mM DTT, 1 mM EDTA, pH 7.5). Crystallization was performed by equilibrating 1 µL protein (mixed with 1 µL precipitant solution) against 500 µl reservoir containing 6 -8% polyethylene glycol (PEG) 6,000, 0.1 M MES (pH 6.0), at 20℃ using the vapor diffusion sitting-drop method. Compounds 11a and 11s were dissolved in 100% DMSO at 50 mM and 200 mM stock concentrations, respectively. A crystal of the free enzyme was soaked in cryo-protectant buffer containing 20% MPD, 6% PEG 6,000, 0.1 M MES, 7.5 mM 11a, pH 6.0, for 2 h at 20℃. Another set of free enzyme crystals was soaked in another cryoprotectant buffer with 6% PEG 6,000, 5% MPD, 0.1 M MES, 15% glycerol, 10 mM 11s, pH 6.0, for 2 h. Subsequently, crystals were fished and flash-cooled in liquid nitrogen prior to data collection.\\r\\n Crystals of HCoV-NL63 M pro with 11a were obtained using cocrystallization. The concentrated HCoV-NL63 M pro (45 mg·mL -1 ) was incubated with 5 mM 11a for 4 h at 20℃, followed by setting up crystallization using the vapor diffusion sitting-drop method at 20℃ with equilibration of 1 µL protein (mixed with 1 µL mother liquor) against 500 µL reservoir composed of 0.1 M lithium sulfate monohydrate, 0.1 M sodium citrate tribasic dihydrate, 25% PEG 1,000, pH 6.0. The crystals were protected by a cryo-buffer containing 0.1 M lithium sulfate monohydrate, 0.1 M sodium citrate tribasic dihydrate, 25% PEG 1,000, 15% glycerol, 2 mM 11a, pH 6.0 and flash-cooled in liquid nitrogen.\\r\\n Crystals of HCoV-NL63 M pro with 11n or 11f were generated by using the soaking method. Several free-enzyme crystals were soaked in cryo-protectant buffer containing 0.1 M lithium sulfate monohydrate, 0.1 M sodium citrate tribasic dihydrate, 25% PEG 1,000, 15% glycerol, 5 mM 11n (or 11f), pH 6.0. Subsequently, the soaked crystals were flash-cooled in liquid nitrogen.\\r\\n Freshly prepared CVB3 3C pro at a concentration of 21.8 mg·mL -1 was incubated with 5 mM 11a pre-dissolved in 100% DMSO at room tempature for 1 h. Some white precipitate appeared in the mixture. Afterwards, the sample was centrifuged at 13,000 x g for 20 min at 4 ℃. The supernatant was subjected to crystallization trials using the following, commercially available kits: Sigma TM (Sigma-Aldrich), Index TM , and PEG Rx TM (Hampton Research). Single rod-like crystals were detected both from the Index TM screen, under the condition of 0.1 M MgCl2 hexahydrate, 0.1 M Bis-Tris, 25% PEG 3,350, pH 5.5, and from the Sigma TM screen at 0.2 M Li2SO4, 0.1 M Tris-HCl, 30% PEG 4,000, pH 8.5. Crystal optimization was performed by using the vapor-diffusion sitting-drop method, with 1 μL CVB3 3C pro -inhibitor complex mixed with 1 μL precipitant solution, and equilibration against 500 μL reservoir containing 0.1 M Tris-HCl, 0.2 M MgCl2, pH 8.5, and PEG 3,350 varied from 22% to 27%. Another optimization screen was also performed against a different reservoir, 0.1 M Tris-HCl, 0.2 M MgCl2, pH range from 7.5 to 8.5, and PEG 4,000 varied from 24% to 34%. Crystals were fished from different drops and protected by cryo-protectant solution consisting of the mother liquor and 10% glycerol. Subsequently, the crystals were flash-cooled with liquid nitrogen.\\r\\n Diffraction data collection, structure elucidation and refinement. Diffraction data from the crystal of the SARS-CoV M pro in complex with 11a were collected at 100 K at synchrotron beamline PXI-X06SA (PSI, Villigen, Switzerland) using a Pilatus 6M detector (DECTRIS). A diffraction data set from the SARS-CoV M pro crystal with compound 11s was collected at 100 K at beamline P11 of PETRA III (DESY, Hamburg, Germany), using the same type of detector. All diffraction data sets of HCoV-NL63 M pro complex structures and of the complex of CVB3 3C pro with 11a were collected at synchrotron beamline BL14.2 of BESSY (Berlin, Germany), using an MX225 CCD detector (Rayonics). All data sets were processed by the program XDSAPP and scaled by SCALA from the CCP4 suite. [50] [51] [52] The structure of SARS-CoV M pro with 11a was determined by molecular replacement with the structure of the complex between SARS-CoV M pro and SG85 (PDB entry 3TNT; Zhu et al., unpublished) as search model, employing the MOLREP program (also from the CCP4 suite). 52, 53 The complex structures of HCoV-NL63 M pro with 11a, 11f, and 11n were also determined with MOLREP, using as a search model the structure of the free enzyme determined by us (LZ et al., unpublished) . The complex structure between CVB3 3C pro and 11a was determined based on the search model of the free-enzyme structure (PDB entry 3ZYD; Tan et al., unpublished) . Geometric restraints for the compounds 11a, 11f, 11n, and 11s were generated by using JLIGAND 52, 54 and built into Fo-Fc difference density using the COOT software. 55 Refinement of the structures was performed with REFMAC version 5.8.0131 (ref. 52, 56, 57) .\\r\\n A buffer containing 20 mM Tris-HCl, 100 mM NaCl, 1 mM EDTA, 1 mM DTT, pH 7.3, was used for all the enzymatic assays. Two substrates with the cleavage sites of M pro and 3C pro , respectively (indicated by the arrow, ↓), Dabcyl-KTSAVLQ¯SGFRKM-E(Edans)-NH2 and Dabcyl-KEALFQ¯GPPQF-E(Edans)-NH2 (95% purity; Biosyntan), were employed in the fluorescence resonance energy transfer (FRET)based cleavage assay, using a 96-well microtiter plate. The dequenching of the Edans fluorescence due to the cleavage of the substrate as catalyzed by the proteases was monitored at 460 nm with excitation at 360 nm, using a Flx800 fluorescence spectrophotometer (BioTek). Curves of relative fluorescence units (RFU) against substrate concentration were linear for all substrates up to beyond 50 µM, indicating a minimal influence of the inner-filter effect. Stock solutions of the compounds were prepared by dissolving them in 100% DMSO. The UV absorption of 11a was found to be negligible at l = 360 nm, so that no interference with the FRET signal through the inner-filter effect was to be expected. For the determination of the IC50, different proteases at a specified final concentration (0.5 µM SARS-CoV or HCoV-NL63 M pro , 2 µM CVB3 3C pro , 3 µM EV-A71 3C pro ) were separately incubated with the inhibitor at various concentrations (0 to 100 μM) in reaction buffer at 37℃ for 10 min. Afterwards, the reaction was initiated by adding FRET peptide substrate at 20 μM final concentration (final volume: 50 μl). The IC50 value was determined by using the GraphPad Prism 6.0 software (GraphPad). Measurements of enzymatic activity were performed in triplicate and are presented as the mean ± standard deviations (SD).\\r\\n Assessment of inhibitory activity of a-ketoamides using viral replicons and virus-infected cells Cells and viruses. Hepatocellular carcinoma cells (Huh7; ref. 58) and their derivative constitutively expressing T7 RNA polymerase (Huh-T7; ref. 59) were grown in Dulbecco's modified minimal essential medium (DMEM) supplemented with 2 mM glutamine, 100 U·mL -1 penicillin, 100 µg·mL -1 streptomycin sulfate, and fetal calf serum (10% in growth medium and 2% in maintenance medium). Huh-T7 cells were additionally supplemented with geneticin (G-418 sulfate, 400 µg·mL -1 ). Huh-T7 cells were used for the enteroviral replicons as well as for infection experiments with CVB3 strain Nancy.\\r\\n For enterovirus (except CVB3) infection experiments, human rhabdomyosarcoma cells (RD; for EV-A71; BRCR strain) and HeLa Rh cells (for EV-D68 and human rhinoviruses) were grown in MEM Rega 3 medium supplemented with 1% sodium bicarbonate, 1% Lglutamine, and fetal calf serum (10% in growth medium and 2% in maintenance medium). For HCoV-229E (a kind gift from Volker Thiel (Bern, Switzerland)), culture and infection experiments were carried out as described. 60 For MERS-CoV or SARS-CoV infection experiments, Vero, Vero E6, and Huh7 cells were cultured as described previously. 61, 62 Infection of Vero and Huh7 cells with MERS-CoV (strain EMC/2012) and SARS-CoV infection of Vero E6 cells (strain Frankfurt-1) at low multiplicity of infection (MOI) were done as described before. 61, 63 All work with live MERS-CoV and SARS-CoV was performed inside biosafety cabinets in biosafety level-3 facilities at Leiden University Medical Center, The Netherlands. Viral replicons. The DNA-launched SARS-CoV replicon harbouring Renilla luciferase as reporter directly downstream of the SARS-CoV replicase polyprotein-coding sequence (pp1a, pp1ab, Urbani strain, acc. AY278741), in the context of a bacterial artificial chromosome (BAC) under the control of the CMV promoter, has been described previously (pBAC-REP-RLuc). 33 Apart from the replicase polyprotein, the replicon encodes the following features: the 5'-and 3'-non-translated regions (NTR), a ribozyme (Rz), the bovine growth hormone sequence, and structural protein N.\\r\\n Subgenomic replicons of CVB3 (pT7-CVB3-FLuc 34 ) and EV-A71 (pT7-EV71-RLuc) harbouring T7-controlled complete viral genomes, in which the P1 capsid-coding sequence was replaced by the Firefly (Photinus pyralis) or Renilla (Renilla renifor) luciferase gene, were generously provided by F. van Kuppeveld and B. Zhang, respectively. To prepare CVB3 and EV-A71 replicon RNA transcripts, plasmid DNAs were linearized by digestion with SalI or HindIII (New England Biolabs), respectively. Copy RNA transcripts were synthesized in vitro using linearized DNA templates, T7 RNA polymerase, and the T7 RiboMax™ Large-Scale RNA Production System (Promega) according to the manufacturer's recommendations.\\r\\n Transfection. Huh-T7 cells grown in 12-well plates to a confluency of 80% -90% (2 -3 x 10 5 cells/well) were washed with 1 mL OptiMEM (Invitrogen) and transfected with 0.25 µg of the replication-competent replicon and Lipofectamin2000 or X-tremeGENE9 in 300 µl OptiMEM (final volume) as recommended by the manufacturer (Invitrogen or Roche, respectively). The transfection mixtures were incubated at 37°C for 4 to 5 h (Lipofectamin2000) or overnight (X-tremeGENE9), prior to being replaced with growth medium containing the compound under investigation. For RNA-launched transfection of enteroviral replicons, DMRIE-C was used as transfection reagent according to the manufacturer's recommendations (Invitrogen). All experiments were done in triplicate or quadruplicate and the results are presented as mean values ± SD.\\r\\n Testing for inhibitory activity of candidate compounds. Initially, we performed a quick assessment of the inhibitory activity of the candidate compounds towards the enteroviral and coronaviral replicons at a concentration of 40 µM in Huh-T7 cells. Compounds that were relatively powerful and non-toxic at this concentration, were assayed in a dose-dependent manner to estimate their half-maximal effective concentration (EC50) as well as their cytotoxicity (CC50), as described. 29 In brief, different concentrations of a-ketoamides (40 µM in screening experiments or increasing concentration (0, 1.25, 2.5, 5, 10, 20, 40 µM) when determining the EC50) were added to growth medium of replicon-transfected Huh-T7 cells. Twenty-four hours later, the cells were washed with 1 mL phosphate-buffered saline (PBS or OPTIMEM, Invitrogen) and lysed in 0.15 mL Passive lysis buffer (Promega) at room temperature (RT) for 10 min. After freezing (-80 o C) and thawing (RT), the cell debris was removed by centrifugation (16,000 x g, 1 min) and the supernatant (10 or 20 µl) was assayed for Firefly or Renilla luciferase activity (Promega or Biotrend Chemikalien) using an Anthos Lucy-3 luminescence plate reader (Anthos Microsystem).\\r\\n The antiviral activity of the compounds was evaluated in a cytopathic effect (CPE) readout assay using MTS [3-(4,5-dimethylthiazol-2-yl)-5-(3carboxymethoxyphenol)-2-(4-sulfophenyl)-2H-tetrazolium, innersalt]-based assay. Briefly, 24 h prior to infection, cells were seeded in 96-well plates at a density of 2.5 x 10 4 (RD cells) or of 1.7 x 10 4 (HeLa Rh) per well in medium supplemented with 2% FCS. For HRV2 and HRV14 infection, the medium contained 30 mM MgCl2. The next day, serial dilutions of the compounds and virus inoculum were added. The read-out was performed 3 days post infection as follows: The medium was removed and 100 μl of 5% MTS in Phenol Red-free MEM was added to each well. Plates were incubated for 1 h at 37°C, then the optical density at 498 nm (OD498) of each well was measured by microtiter plate reader (Saffire 2 , Tecan). The OD values were converted to percentage of controls and the EC50 was calculated by logarithmic interpolation as the concentration of compound that results in a 50% protective effect against virus-induced CPE. For each condition, cell morphology was also evaluated microscopically.\\r\\n Assays with MERS-CoV and SARS-CoV were performed as previously described. 61, 63 In brief, Huh7, Vero, or Vero E6 cells were seeded in 96-well plates at a density of 1 × 10 4 (Huh7 and Vero E6) or 2 × 10 4 cells (Vero) per well. After overnight growth, cells were treated with the indicated compound concentrations or DMSO (solvent control) and infected with an MOI of 0.005 (final volume 150 µl/well in Eagle's minimal essential medium (EMEM) containing 2% FCS, 2 mM L-glutamine, and antibiotics). Huh7 cells were incubated for two days and Vero/VeroE6 cells for three days, and differences in cell viability caused by virus-induced CPE or by compound-specific side effects were analyzed using the CellTiter 96 AQueous Non-Radioactive Cell Proliferation Assay (Promega), according to the manufacturer's instructions. Absorbance at 490 nm (A490) was measured using a Berthold Mithras LB 940 96-well plate reader (Berthold). Cytotoxic effects caused by compound treatment alone were monitored in parallel plates containing mock-infected cells. HCoV-229E-Rev: 5′-ggTCGTTTAGTTGAGAAAAGT -3′, and 229E-ZNA probe: 5'-6-Fam-AGA (pdC)TT(pdU)G(pdU)GT(pdC)TA(pdC)T-ZNA-3-BHQ-1 -3' (Metabion). Standard curves were prepared using serial dilutions of RNA isolated from virus stock. Data were analysed using GraphPad Prism 5.0; EC50 values were calculated based on a 4parameter logistic statistics equation. In parallel to the qPCR assays with inhibitors, cell viability assays were performed using Alamar-Blue™ Cell Viability Reagent (ThermoFisher) according to the manufacturer's instruction. CC50 values were calculated using inhibitor versus normalized response statistics equation by including proper controls (no inhibitor and 1% Triton-X-100-treated cells).\\r\\n Determination of the cell toxicity of candidate compounds. The CellTiter 96Aqueous One Solution Cell Proliferation Assay (MTS test, Promega), the CellTiter Glo assay kit (Promega), the Non-Destructive Cytotoxicity Bio-Assay (ToxiLight (measuring the release of adenylate kinase from damaged cells), Lonza Rockland), or the AlamarBlue™ Cell Viability Reagent (ThermoFisher) were used to determine the cytotoxic effect of compounds towards host cells according to the manufacturers' recommendations. 29, 65 Chemical synthesis of a-ketoamides General procedure. Reagents were purchased from commercial sources and used without purification. HSGF 254 (0.15 -0.2 mm thickness) was used for analytical thin-layer chromatography (TLC). All products were characterized by their NMR and MS spectra. 1 H NMR spectra were recorded on 300-MHz, 400-MHz, or 500-MHz instruments. Chemical shifts were reported in parts per million (ppm, δ) down-field from tetramethylsilane. Proton coupling patterns were described as singlet (s), doublet (d), triplet (t), quartet (q), multiplet (m), and broad (br). Mass spectra were recorded using a Bruker ESI ion-trap HCT Ultra. HPLC spectra were recorded by LC20A or LC10A (Shimadzu Corporation) with Shim-pack GIST C18 (5 µm, 4.6x150mm) with three solvent systems (methanol/water, methanol/ 0.1% HCOOH in water or methanol/0.1% ammonia in water). Purity was determined by reversed-phase HPLC and was ≥95% for all compounds tested biologically.\\r\\n Synthesis of (2S,4R)-dimethyl 2-(tert-butoxycarbonylamino)-4-(cyanomethyl) pentanedioate (1).\\r\\n To a solution of N-Boc-L-glutamic acid dimethyl ester (6.0 g, 21.8 mmol) in THF (60 mL) was added dropwise a solution of lithium bis(trimethylsilyl)amide (LHMDS) in THF (47 mL, 1 M) at −78°C under nitrogen. The resulting dark mixture was stirred at −78°C. Meanwhile, bromoacetonitrile (1.62 mL, 23.3 mmol) was added dropwise to the dianion solution over a period of 1 h while keeping the temperature below −70°C. The reaction mixture was stirred at −78°C for additional 2 h. After the consumption of the reactant was confirmed by TLC analysis, the reaction was quenched by methanol (3 mL), and acetic acid (3 mL) in precooled THF (20 mL) was added.\\r\\n After stirring for 30 min, the cooling bath was removed. The reaction mixture was allowed to warm up to room temperature and then poured into brine (40 mL). The organic layer was concentrated and purified by flash column chromatography (petroleum ether/ethyl acetate = 4/1) to give product 1 (4.92 g, 72%) as colorless oil. 1 H NMR (CDCl3, 400 MHz) δ 5.23 (1H, d, J = 9.0 Hz), 4.43-4.36 (1H, m), 3.77(1H, s), 3.76 (1H, s), 2.89-2.69 (3H, m), 2.20-2.14 (2H, m), 1.45 (9H, s). ESI-MS (m/z): 315 (M + H) + .\\r\\n Synthesis of (S)-methyl 2-(tert-butoxycarbonylamino)-3-((S)-2oxopyrrolidin-3-yl)propanoate (2) .\\r\\n In a hydrogenation flask was placed compound 1 (4.0 g, 12.7 mmol), 5 mL of chloroform and 60 mL of methanol before the addition of PtO2. The resulting mixture was stirred under hydrogen at 20 o C for 12 h. Then the mixture was filtered over Celite to remove the catalyst. NaOAc (6.77 g, 25.5 mmol) was added to the filtrate before the resulting mixture was stirred at 60 o C for 12 h. The reaction was quenched with water (30 mL). The suspension was extracted with ethyl acetate. The organic layers were combined, dried (MgSO4), and filtered. The light-brown filtrate was concentrated and purified by silica gel column chromatography (petroleum ether/ethyl acetate = 4/1) to give the product 2 (2.20 g, 61%) as white solid. 1 H NMR (CDCl3) δ 6.02 (1H, br), 5. Synthesis of (S)-methyl 2-amino-3-((S)-2-oxopyrrolidin-3yl)propanoate (3).\\r\\n Compound 2 (1.0 g, 3.5 mmol) was dissolved in 10 mL dichloromethane (DCM), then 10 mL trifluoroacetic acid (TFA) was added. The reaction mixture was stirred at 20 o C for 0.5 h, and concentrated in vacuo to get a colorless oil, which could be used for the following step without purification.\\r\\n ESI-MS (m/z): 187 (M + H) + .\\r\\n Synthesis of methyl N-substituted amino-acid esters 4 General procedure. The methyl amino-acid ester hydrochloride (6.0 mmol) was dissolved in 20 mL CH2Cl2, and then acyl chloride (6.0 mmol), triethylamine (1.69 mL, 12.0 mmol) were added, before the reaction was stirred for 2 h at 20 o C. The reaction mixture was diluted with 20 mL CH2Cl2, washed with 50 mL of saturated brine (2 × 25 mL), and dried over Na2SO4. The solvent was evaporated and the product 4 was obtained as white solid (70-95% yield), which could be used for the next step without further purification.\\r\\n (S)-methyl 2-cinnamamido-3-phenylpropanoate (4a). The methyl L-phenylalaninate hydrochloride (1.30 g, 6.0 mmol) was dissolved in 20 mL CH2Cl2, and then cinnamoyl chloride (1.00 g, 6.0 mmol), triethylamine (1.69 mL, 12.0 mmol) were added, before the reaction was stirred for 2 h at room temperature. The reaction mixture was diluted with 20 mL CH2Cl2, washed with 50 mL of saturated brine (2×25 mL), and dried over Na2SO4. The solvent was evaporated and the product 4a was obtained as white solid (1.75 g, 95%), which could be used for the next step without further purification.\\r\\n Synthesis of N-substituted amino acids 5 (general procedure). 1 M NaOH (5 mL) was added to a solution of compound 4 (3.0 mmol) in methanol (5 mL). The reaction was stirred for 20 min at 20 o C. Then 1 M HCl was added to the reaction solution until pH = 1. Then the reaction mixture was extracted with 100 mL of CH2Cl2 (2 × 50 mL) and the organic layer was washed with 50 mL of brine and dried over Na2SO4. The solvent was evaporated and the crude material purified on silica, eluted with mixtures of CH2Cl2/MeOH (20/1) to afford the product 5 (90-96% yield) as a white solid.\\r\\n Synthesis of compounds 6 (General procedure). Compound 5 (2.7 mmol) was dissolved in 10 mL of dry CH2Cl2. To this solution, 1.5 equiv (1.54 g) of 1-[bis(dimethylamino)methylene]-1H-1,2,3triazolo[4,5-b]pyridinium 3-oxide hexafluorophosphate (HATU) was added, and the reaction was stirred for 0.5 h at 20 o C. Then compound 3 (500 mg, 2.7 mmol) and TEA (0.70 mL, 5.42 mmol) was added to the reaction. The reaction was stirred for another 6 h. The reaction mixture was poured into 10 mL water. The aqueous solution was extracted with 50 mL of CH2Cl2 (2 × 25 mL) and washed with 50 mL of saturated brine (2 × 25 mL) and dried over Na2SO4. The solvent was evaporated and the crude material purified on silica, eluted with a mixture of CH2Cl2/MeOH (40/1) to give the product 6 (62-84% yield).\\r\\n Synthesis of alcohols 7 (general procedure). Compound 6 (1.1 mmol) was dissolved in methanol (40 mL), then NaBH4 (0.34 g, 8.8 mmol) was added under ambient conditions. The reaction mixture was stirred at 20 o C for 2 h. Then the reaction was quenched with water (30 mL). The suspension was extracted with ethyl acetate. The organic layers were combined, dried, and filtered. The filtrate was evaporated to dryness and could be used for the next step without further purification (46-85% yield).\\r\\n Synthesis of aldehydes 8 (general procedure). Compound 7 (0.75 mmol) was dissolved in CH2Cl2, then Dess-Martin periodinane (337 mg, 0.79 mmol) and NaHCO3 (66 mg, 0.79 mmol) were added. The resulting mixture was stirred at 20 o C for 1 h. The mixture was concentrated and purified by column chromatography on silica gel (CH2Cl2/MeOH = 20/1) to give the product 8 as white solid (88-95% yield).\\r\\n Synthesis of compounds 9 (general procedure). Compound 8 (0.40 mmol) was dissolved in CH2Cl2, and then acetic acid (0.028 g, 0.47 mmol) and isocyanide (0.43 mmol) were added successively to the solution. The reaction was stirred at 20 o C for 24 h. Then the solvent was evaporated and the crude material purified on silica, eluted with a mixture of CH2Cl2/MeOH (20/1) to give the product 9 (46-84%).\\r\\n Synthesis of α-hydroxyamides 10 (general procedure). 1 M NaOH (0.5 mL) was added to a solution of compound 9 (0.164 mmol) in methanol (5 mL). The reaction was stirred at 20 o C for 0.5 h until the consumption of compound 9 was confirmed by TLC analysis. Then, 1 M HCl was added to the reaction solution until pH = 7. Following this, the solvent was evaporated to generate the product 10 as white solid, which could be used directly in the next step.\\r\\n Synthesis of α-ketoamides 11 (general procedure). Compound 10 was dissolved in CH2Cl2, then Dess-Martin periodinane (74 mg, 0.176 mmol) and NaHCO3 (30 mg, 0.176 mmol) were added. The resulting mixture was stirred at 20 o C for 1 h. The mixture was concentrated and purified by column chromatography on silica gel (CH2Cl2/MeOH = 20/1) to give the a-ketoamides 11 as light yellow solid (52-79% in two steps). \\r\\n Detailed results of the variation of the P1' and P3 substituents Synthesis of a-ketoamides 11b -11e and 11g -11l Supplemental Table 1 : inhibitory activities (IC50 (μM)) of α-ketoamides with P1' and P3 modifications against viral proteases Supplemental Table 2 : crystallographic data for complexes between viral proteases and α-ketoamides Molecular formula strings and biological data (CSV)\\r\\n Authors will release the atomic coordinates and experimental data upon article publication. Atomic coordinates include SARS-CoV M pro in complex with compounds 11a (5N19), 11s (5N5O), HCoV-NL63 M pro in complex with 11a (6FV2), 11n (6FV1), 11f (5NH0), and CVB3 3C pro in complex with (5NFS). 1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.10.936898 doi: bioRxiv preprint  1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.10.936898 doi: bioRxiv preprint. Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) or 2019-Novel Coronavirus (2019-nCoV) is the pathogen of Corona Virus Disease 2019 [1] . Nucleic acid detection of 2019-nCoV is one of the key indicators for clinical diagnosis. However, the false negative rate of 2019-nCoV nucleic acid in clinical practice is very high, and the positive rate is only 30-50% [2] . False negative detection means missed detection, which will lead to not only the delayed diagnosis of suspected patients, but also the 2019-nCoV carriers becoming the potential sources of virus infection. Therefore, it is very urgent to improve the detection rate of 2019-nCoV nucleic acid clinically.\\r\\n Currently, fluorescent quantitative RT-PCR (qRT-PCR) technology is mainly used to detect the nucleic acid of 2019-nCoV. The quality of nucleic acid template is admittedly one of the key factors affecting the detection efficiency. According to \\\"The Laboratory Technical Guidelines for Detection 2019-nCoV (Fourth Edition)\\\" issued by National Health and Commission of China [3] and \\\"The Experts' Consensus on Nucleic Acid Detection of 2019-nCoV\\\" released by Chinese Society of Laboratory Medicine [4] , the human samples must be placed in 56 o C for at least 45 minutes or higher for shorter time to inactivate the viruses in order to keep the inspectors from virus infection before the nucleic acids were isolated.\\r\\n However, 2019-nCoV is a single stranded RNA virus. Theoretically, the progress of killing the viruses with high temperature will damage the integrity of their RNA genomes in the samples, thus improperly reducing the amount of target viral templates, and eventually leading to high false negative rate of the virus detection. In this study, we employed porcine epidemic diarrhea virus (PEDV), a kind of coronavirus, as a model virus to test the above hypothesis and our results demonstrated that the virus inactivation progress by high temperature damaged the detectable template of the coronavirus seriously. Biological. Company, Ltd.) or 1 ng λDNA (3010, Takara, Japan) were suspended and mixed together using 1.5 mL of sample preservation medium (R503, Vamzye, China) or 1.5 mL Hank's solution.\\r\\n Nucleic acid (DNA/RNA) was extracted using a commercial kit (RC311-C1, Vazyme) according to the manufacturer's protocol. Briefly, 500 µL samples preserved in R503 was first added into a 1.5 ml EP tube containing 200 µL of absolute ethanol and 200 µL samples preserved in Hank's solution was added into a 1.5 ml EP tube containing 500 µL lysis buffer.\\r\\n The mixture was vortexed thoroughly and then transferred into a column placed onto a 2 mL EP collection tube. After centrifuge for 1 min at 12,000×g, the column was placed onto a fresh 2 mL EP collection tube, followed by adding 600 µL washing buffer and then centrifuge for 30 sec at 12,000×g. Washing step was repeated once. The column was placed onto a new 2 mL collection. After centrifuge for 2 min at 12,000×g, the column was then placed onto a 1.5 ml collection tube. 100 µL and 40 µL elution buffer was added into the center of the membrane of column collected the sample prepared by R503 and Hank's solution respectively and then centrifuged for 1 min at 12,000 × g to collect the nucleic acid. The nucleic acid was stored at -30 ~ -15 o C for usage within short time and at -70 o C for long storage 2.3 Agarose gel electrophoresis 5 µL of the extracted nucleic acids, 1 µL of 10 × DNA loading buffer (P022, Vazyme) and 4 µL of nuclease-free H 2 O were mixed and then vortexed briefly. The mixture was heated in PCR instrument to denature the nucleic acid for 2 min and then subjected to 1.2% agarose gel electrophoresis separation on ice for 7-10 min together with the standard DNA marker (MD103, Vazyme).\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint\\r\\n Quantitative Real-Time PCR (qRT-PCR) was performed to detect the PEDV RNA and λ phage DNA using HiScript II One Step qRT-PCR Probe Kit (Q222-CN, Vazyme) according to the manufacturer's protocol. The reaction (25 µL) was assembled as follows: 12.5 µL of 2 × One Step U+ Mix, 1.25 µL of One Step U+ Enzyme Mix, 0.5 µL of 50 × ROX Reference Dye 1, 0.5 µL of Gene Specific Primer Forward (10 µM), 0.5 µL of Gene Specific Primer Reverse (10 µM), 0.25 µL of TaqMan Probe (10 µM), and 4.5 µL nuclease free H2O.\\r\\n The reaction was carried out on ABI StepOnePlusTM using following conditions: \\r\\n Statistical significance was determined using the paired two-tailed t test. A value of p < 0.05 was considered statistically significant, and p < 0.01 was considered statistically very significant. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint 6 / 16 into five groups. Three groups were incubated at 56 o C for 30, 45 and 60 minutes, respectively. The fourth was incubated at 92 o C for 5 minutes, and the fifth group was stored at 4 o C as no inactivation control. After the high-temperature inactivation was completed, nucleic acids (including DNA and RNA) were extracted by conventional methods, and subjected to run 1.2% agarose gel electrophoresis. \\r\\n To understand the effect of the virus inactivation progress with the temperature higher than 56 o C on the integrity of viral nucleic acid, we quantitated the amount of Table S1 ). The results suggest that only 50.11% of the detectable viral templates left after the inactivation progress of incubation at 56 o C for 30 minutes and only 3.36% of the detectable viral templates left after the inactivation progress of incubation at 92 o C for 5 minutes.\\r\\n To investigate whether any solutions are able to protect viral RNA integrity from author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint 7 / 16 disrupting during the inactivation progress with high temperature, R503 (Vazyme, China) was used to prepare the samples instead of Hank's solution. We then performed the same viral inactivation experiments as using Hank's solution.\\r\\n Compared with the control group that the sample was stored at 4 o C, both DNA and RNA in the samples incubated at 56 o C for 30 -60 minutes also displayed obvious degradation, among which the 28 S and 18 S bands of total RNA of human cells were obviously smeared, and the large bands of genomic DNA became weaker while there were almost no visible genomic DNA and 28 S RNA bands in the sample incubated at 92 o C for 5 minutes (Fig 1D) . However, the degradation of nucleic acid during the inactivation progress was not as serious as that of the samples prepared by Hank's solution. The results suggest that R503 should have a significant protective effect of both total RNA and genomic DNA from degradation.\\r\\n To determine the effect of the inactivation progress with temperature higher than 56 o C on the integrity of viral nucleic acid in the samples prepared with R503, we examined the amount of porcine PEDV coronavirus by qRT-PCR as above.\\r\\n Compared with the CT (31.64 ± 0.10) of the control group that the sample was stored at 4 o C, the mean value of CT (31.31 ± 0.17) of the group incubated at 56 o C for 30 minutes had no significant change. The difference of CTs between the two groups was less than 0.5 ( Fig. 1E ; Figure S1B ; Table S1 ). Moreover, the mean value of CT (31.43 ± 0.08) in the group incubated at 92 o C for 5 minutes was also not changed significantly. The difference of CTs between the two groups was less than 0.5 either (Fig. 1F; Fig. S1B ; Table S1 ). These results showed that the inactivation progress with the temperature higher than 56 o C did not significantly affect the detectable template of viral nucleic acid when it was prepared with R503, suggesting that the fragmented viral RNA could still be good for using as a template for qRT-PCR even if the viral nucleic acid was degraded.\\r\\n In order to understand the advantage of R503 over common isotonic solution in author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint\\r\\n protecting the detectable amount of coronavirus RNA from disrupting, we first compared the difference of the detectable amount of coronavirus RNA between the two samples that are prepared with R503 and Hank's solution and stored at 4 o C. The results showed that the detectable amount of the viral RNA in the two prepared solutions was similar (Fig. 1G, Table S1 ), and the difference between the mean values of their CT was less than 0.5 (31.64 ± 0.10 vs 31.95 ± 0.10). The results suggest that there is no significant difference in the amount of detectable viral RNA templates between the samples prepared by either R503 or Hank's solutions that were stored at Table S1 ). The results suggest that the detectable templates of viruses in the samples prepared by Hank's solution was only 32.30% of R503 solution after the samples were inactivated at 56 o C for 30 minutes to kill the virus. Furthermore, after inactivation at 92 o C for 5 minutes, the CT of detected PEDV virus in the two preservation solutions were 31.43 ± 0.08 (R503) and 36.84 ± 0.37 (Hank's), respectively. The mean value of CT of detected virus in the samples prepared by Hank's solution was 5.4125 higher than that prepared by R503 (P < 0.0001; Figure 1I , Table S1 ). The results reveal the detectable amount of virus in the samples prepared by Hank's solution was only 2.35% of R503 solution after the samples were inactivated at 92 o C for 5 minutes to kill the virus.\\r\\n In addition to the disruptive effect of the virus inactivation with high temperature on the detection of RNA virus, the detectable amount of the viral DNA (λ phage) in the samples was also significantly affected. qPCR results (Fig. S1C, Table S1) showed that mean value of CT (23.55 ± 0.20) of detectable virus in the samples author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint 9 / 16 incubated at 56 o C for 30 minutes was increased by 0.5230 (P < 0.01) compared with the CT (23.02 ± 0.15) of the samples stored at 4 o C. The mean value of CT (25.58 ± 0.21) was increased by 2.5572 (P < 0.0001) in the sample incubated at 92 o C for 5 minutes.\\r\\n Furthermore, we examined the amount of phage DNA in the sample prepared with R503 after high temperature treatment by qPCR. The results (Fig. S1D, Table S1) showed that the mean value of CT (21.71 ± 0.04) in the sample prepared by R503 and incubated at 56 o C for 30 minutes had no significant change compared with that in the control group (21.61 ± 0.06) (the difference between the two was less than 0.5). The mean value of CT (22.90 ± 0.26) was increased by 1.2880 in the sample that was inactivated at 92 o C for 5 minutes, suggesting that 92 o C treatment significantly reduced the detectable amount of DNA viruses in sample even prepared by R503.\\r\\n In conclusion, our results show that virus inactivation with temperature higher than 56 o C result in the degradation of viral nucleic acids seriously, which will lead to the artificial shortage of detectable templates of viral nucleic acids in samples, and finally lead to false negative of clinical detection in some samples. Preserved with the optimized solution such as R503, the detectable templates of viral nucleic acids can be kept unchanged after the samples were incubated at 56 o C or higher for killing the viruses although the integrity of cellular nucleic acids were obviously disrupted. The results suggest that if the samples were prepared with an optimal solution, the detectable numbers of viral templates would be unchanged because virus inactivation progress with high temperature does not erase all the RNA but leave small fragments of nucleic acids that are good enough for using as the template for qRT-PCR detection.\\r\\n If it is true for 2019-nCoV, preparing the human samples with the optimal solution can not only keep the clinical inspectors from virus infection by inactivating the virus using 56 o C or higher as normal, but also improve the detection sensitivity of viral nucleic acid in the sample, leading to avoiding or at least reducing false negative author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint 10 / 16 results of detection.\\r\\n Recently, Chen reported that the inactivation by incubating the sample with 56 o C for 30 min to kill the virus had no significant effect on the detection of 2019-nCoV by qRT-PCR [5] . However, combining the limitations such as only two samples tested and higher amount of virus in the two samples as the authors discussed in their researches, and our experimental evidences that the virus inactivation by high temperature did reduce the detectable amount of the porcine coronavirus template seriously, it is highly recommended to carry out systematic investigation on the impact of high temperature inactivation on the integrity of 2019-nCoV nucleic acids and develop a sample preservation solution to protect the detectable templates of 2019-nCoV nucleic acids from high temperature inactivation damage. Table S1 . Figure S1 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint 15 / 16 The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.20.958785 doi: bioRxiv preprint. HCoV-HKU1 replication in the presence of both inhibitors (Fig. 2B) . All inhibitors were used 151 at non-toxic concentrations (Fig. 2C) . 152 The experiments conducted so far suggested that KLK13 is required for virus infection. 153 However, one may question the specificity of the KLK13 protease inhibitors. To ensure that 154 KLK13 is indeed the priming enzyme during HCoV-HKU1 infection, we developed HAE 155 cultures by transforming cells with lentiviral vectors encoding shRNAs targeting KLK13 156 mRNA. We then confirmed that the expression of the protease was silenced (HAE_shKLK13). (Fig. 3A) . Importantly, HAE_shKLK13 cells continued to differentiate and formed 162 pseudostratified cultures (Fig. 3B) . Next, we infected HAE_ctrl, HAE_GFP, HAE_vector, and 163 HAE_shKLK13 with HCoV-HKU1 (10 6 RNA copies per ml) and incubated them for 2 h at 164 32°C with the viral stock solution. Cultures were maintained at 32°C for 5 days at an air-liquid 165 interface. Apical washes were collected, and virus yield was determined by RT-qPCR. We 166 found that, in contrast to that in control cultures, replication of virus in HAE_shKLK13 was 167 abolished (Fig. 3C) . Overall, these data suggest that silencing the KLK13 gene in HAE inhibits 168 virus infection, indicating that KLK13 is necessary for HCoV-HKU1 infection. 171 We determined that KLK13 is essential for efficient HCoV-HKU1 infection in HAE The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint proteases my render RD cells permissive, we generated RD cells expressing human KLK13 or 176 TMPRSS2 proteases. RD cells were transduced with lentiviral vectors harboring the KLK13 177 gene (RD_KLK13), control vector (RD_ctrl), or TMPRSS2 (RD_TMPRSS2). Due to the lack 178 of KLK13 specific antibodies, we verified its presence based on RT-PCR (Fig. 4A) . The 179 presence of TMPRSS2 in RD_TMPRSS2 cells was confirmed using Western blot. The 180 TMPRSS2 band in RD cells was observed at 25 kDa, which corresponds to one of the naturally 181 occurring splicing variants (Fig. 4B) . Subsequently, we transduced RD_ctrl, RD_KLK13 and pseudoviruses, compared to ΔEnv, which was completely abolished in the presence of KLK13 193 inhibitor (Fig 4D) . Overall, these data demonstrated that KLK13 activity drives HCoV-HKU1 194 entry into cells. 195 196 KLK13 enables the replication of HCoV-HKU1 in RD cells 197 Obtained results showed that KLK13 expression on RD cells was sufficient for HCoV-198 HKU1 pseudovirus entry. Here, we aimed to test whether KLK13 presence renders RD cells 199 permissive for HCoV-HKU1 infection. For this, we infected RD_ctrl and RD_KLK13 cells 200 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint with HCoV-HKU1 (10 8 RNA copies per ml) and incubated the culture for 7 days at 32°C in the 201 presence or absence of a KLK13 inhibitor (10 µM) or DMSO. Next, cellular RNA was isolated 202 and the presence of HCoV-HKU1 N subgenomic mRNA (N sg mRNA), which is considered 203 to be a hallmark of coronaviral infection, was assessed (14). sg mRNA appeared in RD_KLK13 204 cells, while no signal was detected in cultures supplemented with the KLK13 inhibitor nor in 205 RD_ctrl cells (Fig. 5A) . 206 To further confirm the role of KLK13 during HCoV-HKU1 infection RD HCoV-HKU1 infection was analyzed by means of N sg mRNA detection. Again, we found that 212 N sg mRNA was produced only in the presence of KLK13 (Fig. 5B) . Further, we passaged in the presence of KLK13 (Fig. 5B) . However, we observed no cytopathic effects (CPEs), and 218 replication levels were very low (no significant increase over control levels on RT-qPCR; data 219 not shown). To further test the effect of KLK13 on replication of HCoV-HKU1 in RD cells, the 220 virus stock was incubated with purified KLK13 (200 nM) and incubated in the presence or 221 absence of a KLK13 inhibitor (10 µM) or DMSO. After 2 h at 32°C pre-treated virus stock was 222 diluted in media as described above and overlaid on RD cells. After 7 days at 32°C we evaluated 223 the presence of the HCoV-HKU1 N sg mRNA. The virus replicated only after treatment with 224 KLK13, and supplementation with the inhibitor blocked this effect infection (Fig. 5C) . 225 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint KLK13 primes the HCoV-HKU1 S protein 227 Expression of KLK13 by cells previously resistant to HCoV-HKU1 renders them 228 susceptible; therefore, we asked whether this is due to proteolytic activation of the S protein. 229 We tested this using the CleavEx method, in which a peptide of interest is exposed in the KLK13. Subsequently, proteins were resolved by SDS-PAGE and detected by western blotting 238 with antibodies specific for His-tagged proteins. The analysis showed that, in the presence of 239 500 nM KLK13, the CleavEx protein harboring the S1/S2 cleavage site was degraded; however, 240 the CleavEx protein harboring the S2/S2' site remained intact. Because KLKs are produced as 241 pro-forms that undergo self-activation, an additional band of His-tagged purified pro-KLK13 242 (HisTag-pro-KLK13) was observed after treatment with 500 nM KLK13 (Fig. 6A) . The product 243 of the S1/S2 cleavage was further sequenced by N-terminal Edman degradation showing the 244 following sequence: R↓SISA, which corresponds to the S1/S2 site. This result shows clearly 245 that the S1/S2 region of the HCoV-HKU1 S protein is prone to KLK13-mediated cleavage. 246 Furthermore, we aimed to confirm the cleavage using a full-length Spike protein of HCoV-HKU1 (HKU1-S). For this, we expressed the HKU1-S in 293T cells, purified the protein 248 using 6 × His tag, and incubated for 3 h at 37°C with increasing concentrations of purified 249 KLK13. Subsequently, HKU-S or mock proteins were resolved by SDS-PAGE and detected by 250 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint western blotting with antibodies specific to the tag. The analysis showed that in the presence of 251 1 µM KLK13 the HKU1-S was degraded ( Figure 6B ). The S protein was observed at 252 ~150 kDa, which is consistent with the migration speed reported for these highly glycosylated 253 proteins (1). 254 255 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint\\r\\n Receptor recognition is the first, essential step of the virus infection process. The The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint concentrations(63). Broad spectrum protease inhibitors are now used widely for virus research, 281 although their non-specific activity makes the results equivocal. For example, Matsuyama et al. 282 showed recently that the furin inhibitor dec-RVKR-CMK interferes with the activity of several 283 proteases, and that its previously described inhibitory activity during MERS-CoV infection is 284 not specific to furin; instead, its activity is due to non-specific inhibition of cathepsin L and 285 TMPRSS2 (81). We tried to use specific KLK inhibitors developed in our laboratory (61).\\r\\n Considering the small arsenal of tools available to researchers studying KLKs, only three 287 compounds were readily available. Treating HAE cultures with these inhibitors revealed that 288 only compounds designed to inhibit KLK13 hampered HCoV-HKU1 replication. However, the 289 great similarity between different KLKs makes one doubt the specificity of these inhibitors, 290 despite their performance in biochemical assays. Therefore, we decided to silence KLK13 in 291 HAE cultures. This abolished virus replication ex vivo, thereby confirming the importance of 292 KLK13 during infection. KLK13 is thought to be secreted and membrane-bound(82, 83). 293 This study showed that HCoV-HKU1 infection in HAE modulates the expression of 294 different KLKs, including KLK13. In our study, KLKs expression was tested using semi-295 quantitative PCR, and for that reason, we were unable to show the level of KLKs modulation 296 after HCoV-HKU1. However, the pattern of virus-induced expression of several KLKs could 297 be observed. KLK mechanism of activation is a complex process and until now it has only 298 been proven that most KLK genes are regulated by steroids and other hormones (84). It is also 299 important to remember, that KLK expression is regulated in a similar manner, and the induction 300 of a single gene usually results in overexpression of the whole cluster (85, 86). While one may 301 assume that the virus stimulates KLK13 production to promote the infection, this up-regulation 302 is likely a natural response of the damaged tissue, as KLKs were previously reported to take 303 part also in tissue regeneration (87-89). Further, increased expression of KLKs may be the 304 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint response to the inflammatory process, as Seliga et al demonstrated that KLK-kinin system is a 305 potent modulator of innate immune responses (90).\\r\\n The experiments performed herein show the importance of KLK13 for virus entry into 307 susceptible cells; therefore, we speculated that scattered distribution of different KLKs in 308 different tissues may be one of the determinants of the HCoV-HKU1 tropism (53, 91). We 309 tested the purified enzyme expressed in the eukaryotic cells; however, we also developed a cell 310 line constitutively expressing the enzyme. As an in vitro model for our studies, we used RD 311 cells previously reported to carry attachment receptors for the virus (26). Here, using 312 pseudoviruses decorated with S-HKU1 proteins we showed that KLK13 presence on RD cells 313 is sufficient for virus entry and renders these cells permissive. Our experiments also showed 314 that in contrast to previous reports, TMPRSS2 is not involved in this process(51). Furthermore, 315 we observed that RD cells supported the replication of the virus in the presence of KLK13 and 316 that this effect was reversed in the presence of the specific KLK13 inhibitor. We were, however, 317 not able to culture the virus to high yields. HCoV-HKU1 replication in KLK13-expressing RD 318 cells remained inefficient and RTqPCR assessment did not reveal significant increases in the 319 amounts of viral RNA. For that reason, we are only able to detect viral sg mRNAs, which are 320 considered to be the hallmark of coronaviral replication.We believe that this may be due to 321 non-optimal infection conditions, which may include inappropriate KLK13 concentrations or 322 low density of the entry receptor. Also, it is possible that RD cells may not support efficient 323 replication of the virus due to factors unrelated to the entry process. Nonetheless, our results\\r\\n show that the HCoV-HKU1 entry receptor is present on RD cells, and we were able to trigger 325 virus entry and replication; these findings warrant further exploration.\\r\\n Most coronaviral S proteins are processed into S1 and S2 subunits by host proteases, The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint HCoV-HKU1 S protein has two regions that are prone to proteolytic activation: the S1/S2 furin 330 cleavage site and a secondary cleavage site termed S2', which is adjacent to a potential fusion 331 peptide (38). While the S1/S2 site is believed to be processed by furin during protein 332 biosynthesis, the S2/S2' site is expected to be cleaved during virus entry. As we already knew being tested. We found that the S1/S2 site was efficiently cleaved by KLK13, whereas the 338 S2/S2' region remained intact. As CleavEx technique is a convenient surrogate system allowing 339 for precise mapping of the cleavage site, it has some limitations. To ensure the reliability of 340 results, purified full-length HCoV-HKU1 S protein was subjected to the proteolytic cleavage.\\r\\n Also here we observed efficient cleavage of the HCoV-HKU1 S protein by KLK13.\\r\\n While the results presented here show that KLK13 is able to process the HCoV-HKU1 The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint to cross the species barrier and cause severe diseases in humans, further research on the role of 355 different proteases in coronaviral infections is necessary. 356 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint\\r\\n Plasmid constructs 358 KLK13 and TMPRSS2 genes were amplified by PCR using cDNA obtained from HAE 359 cells. Each PCR product was cloned into pWPI plasmid for lentivirus production and sequence Table 2 . (Table 3 ; 500 nM each). β-actin was used as a household gene reference.\\r\\n The reaction was carried out according to the scheme: 5 min at 95°C, followed by 35 cycles of 465 30 s at 95°C, 20 s at 59°C and 20 s at 72°C, followed by 10 min at 72°C. containing proKLK13 or proKLK14 were concentrated and the buffer was changed to 50mM 528 Tris pH 7.5, 150 mM NaCl. After purification and self-activation at 37°C for 24 h, activity of 529 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint proteases was assessed by serine protease inhibitor Kazal-type 6 (SPINK6) titration, as 530 described previously (63). Protein expression was performed in E. coli BL21 and was induced by the addition of 553 0.5 mM IPTG to the bacterial culture (OD600 0.5-0.6), followed by shaking for 3h at 37°C. 554 author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint Then, the bacteria were spun down and the pellet was suspended in buffer A (10 mM sodium 555 phosphate, 500 mM NaCl and 5 mM imidazole, pH 7.4). The pellet suspension was then 556 sonicated and spun down. Soluble proteins were purified using HisTrap TM Excel (GE The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.01.971499 doi: bioRxiv preprint. 4 hepatoma cells was largely specific to CoV. Among 10 genetically divergent viruses tested, only hepatitis C virus was also restricted (Extended Data Fig. 1f ).\\r\\n LY6E is a member of the LY6/uPAR family of GPI-anchored proteins, and is implicated in diverse cellular processes 14 . To evaluate the specificity of LY6E for inhibiting CoV, we ectopically expressed select LY6/uPAR proteins 9 and infected with HCoV-229E. Only LY6E inhibited CoV infection (Extended Data Fig. 1g) . We previously showed that a conserved residue (L36) is required for LY6Emediated enhancement of influenza A virus 9 . L36 was also required to restrict HCoV-229E infection (Extended Data Fig. 1h) . These data suggest a conserved functional domain that is responsible for both the virus-enhancing and -restricting phenotypes.\\r\\n Next, we investigated the effect of LY6E on specific steps of the virus replication cycle. First, we tested whether LY6E restricts HCoV-229E attachment to the cell surface and observed no effect (Extended Data Fig. 2a) . In accordance, LY6E had no effect on surface expression of CD13, the specific receptor for HCoV-229E (Extended Data Fig. 2b) . To test whether LY6E restricts viral entry, we used a vesicular stomatitis virus (VSV) pseudoparticle (pp) system bearing CoV spike (S) proteins.\\r\\n Strikingly, LY6E significantly inhibited VSVpp entry mediated by S proteins from HCoV-229E (Fig.   2a) , MERS-CoV (Fig. 2b) , SARS-CoV (Fig. 2c) , and SARS-CoV-2 (Fig. 2d) . Entry mediated by VSV glycoprotein G was only marginally impacted by LY6E (Extended Data Fig. 2c, d) . To test whether LY6E interferes with the fusion of viral and cellular membranes, we performed a syncytia formation assay using propagation competent VSV pseudoviruses co-expressing CoV S protein and a GFP reporter (VSV*ΔG(CoV S)) (Extended Data Fig. 3a) . LY6E potently inhibited CoV S protein-mediated syncytia formation (Fig. 2e-f ). To determine whether LY6E impairs S protein expression or maturation, we also performed a heterologous syncytia formation assay. CoV-resistant hamster cells infected with trans-complemented GFP reporter VSV*ΔG(CoV) were co-cultured with susceptible target cells that ectopically expressed LY6E. LY6E again significantly blocked syncytia formation, demonstrating that LY6E inhibits S protein-mediated fusion and not S protein expression or maturation (Extended Data Fig. 3b-c) . As membrane fusion can occur without syncytia formation, we also performed a quantitative fusion assay in which mixing of cell contents results in complementation of split luciferase (Fig. 2g) .\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint Cells co-transfected with plasmids encoding CoV S proteins and split luciferase 1-7 were mixed with susceptible target cells co-expressing LY6E or vector control and split luciferase 8-11. LY6E reduced CoV S-mediated cell-cell fusion, indicating that LY6E blocks fusion of viral and cellular membranes ( Fig. 2h) . Time-course experiments revealed that LY6E had no effect on HCoV-229E translation, replication, assembly, or release (Extended Data Fig. 2e-i) . Collectively, our results demonstrate that LY6E specifically inhibits CoV S protein-mediated membrane fusion, which is required for viral entry.\\r\\n We next addressed whether LY6E modulates proteolytic activation of the S protein. Upon S proteinmediated binding to the respective receptor, host proteases cleave the S protein and make it fusion competent for \\\"early entry\\\" at the cell surface or \\\"late entry\\\" in endosomes 15, 16 . Pharmacological inhibition of cell surface and endosomal proteases had no effect on LY6E-mediated restriction of CoV infection (Extended Data Fig. 4a-c) . Next, we addressed whether LY6E directly interferes with the proteolytic activation of S protein. Mutation of S1/S2 and S2′ in the S protein prevents activation (Extended Data Fig. 4d ) 16, 17 . Ectopic LY6E expression had no effect on MERS-CoV S cleavage or infection by cleavage-resistant MERS-CoV pp (Extended Data Fig. 4e-g) . Collectively, our data indicate that restriction of viral fusion is independent of S protein activation.\\r\\n Next, we aimed to evaluate LY6E-mediated CoV restriction in vivo. Systemic genetic ablation of Ly6e in mice is embryonic lethal 18 . Therefore, we generated transgenic Ly6e fl/fl mice, which are functionally 'wild-type (WT)' for Ly6e expression, and crossed them with Vav1-iCre mice to ablate Ly6e in cells that originate from a hematopoietic stem cell (HSC) progenitor (Extended Data Fig. 5a-b ). This genetic model was chosen since immune cells are critical for protection from CoV in vivo 5,19-23 . Bonemarrow derived macrophages (BMDM) from Ly6e ΔHSC mice had reduced Ly6e mRNA levels (Extended Data Fig. 5c ). The natural mouse pathogen MHV is a well-studied model CoV that causes hepatitis and encephalomyelitis in mice 24 . Ly6e ΔHSC BMDM were more susceptible to MHV infection than Ly6e fl/fl BMDM, demonstrating that murine Ly6e is a restriction factor for murine CoV (Extended Data Fig.   5d ).\\r\\n To determine whether Ly6e is important for controlling CoV infection in vivo, Ly6e fl/fl and Ly6e ΔHSC mice were infected with a high dose (5 x 10 4 PFU) of MHV that causes sub-lethal hepatitis in WT author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint C57BL/6J mice 25 . Since MHV pathogenesis is influenced by sex 26 , we performed independent experiments in male and female mice. Ly6e ΔHSC mice of both sexes rapidly succumbed to MHV infection by day 6 ( Fig. 3a-b, Extended Data Fig. 6a-b) . To examine viral pathogenesis and immunopathology in infected mice, Ly6e fl/fl and Ly6e ΔHSC mice were infected with the same high dose of MHV and euthanized 3-or 5-days post-infection. At both time points, female Ly6e ΔHSC mice exhibited significantly higher levels of liver damage as measured by serum ALT while no difference was observed in male mice (Fig. 3c,i, Extended Data Fig. 6c,i) . However, hepatic viral burden did not differ in either sex ( Fig. 3d,j, Extended Data Fig. 6d,j) , possibly due to tissue saturation at this high dose. Indeed, female but not male Ly6e ΔHSC mice exhibited increased liver damage and viral burden at a low dose (5 PFU) of MHV (Extended Data Fig. 7a -b, f-g). Spleen viral burden was elevated in female and male Ly6e ΔHSC mice at both time points and doses of MHV (Fig. 3e,k, Extended Data Fig. 6e ,k, Extended Data Fig.   7c ,h). At 3 days post-infection, there was no difference in liver necrosis, but inflammation was moderately reduced in female but not male Ly6e ΔHSC mice ( Fig. 3f-h, Extended Data Fig. 6f-h) . By 5 days post-infection and at both low and high doses of MHV, female Ly6e ΔHSC mice had significantly higher levels of liver necrosis and a reduced presence of mononuclear immune cells ( Fig. 3l- Fig. 7k-n) . These data demonstrate that Ly6e in hematopoietic cells is important for controlling murine CoV infection.\\r\\n We next used a transcriptomic approach to evaluate the effect of Ly6e ablation on global gene expression in liver and spleen of female Ly6e fl/fl and Ly6e ΔHSC mice injected with PBS or the high dose of MHV at 3-and 5-days post-infection. In infected tissues, loss of Ly6e in hematopoietic cells correlated with differential gene expression in pathways associated with tissue damage, such as liverspecific metabolic genes, angiogenesis, wound healing, and immune response to viruses (Fig. 4a-c) . We also observed a striking loss of genes associated with the type I IFN response, inflammation, antigen presentation, and B cells in infected Ly6e ΔHSC mice ( Fig. 4d-e, Extended Data Fig. 8) . 7 The histopathology (Fig. 3h,n) and transcriptome data (Fig 4a-e, Extended Data Fig. 8 Fig. 7k-n, Extended Data Fig. 9 ). Multiple cell types were depleted in livers and spleens of infected Ly6e ΔHSC mice, irrespective of sex (Figure 4f g, Extended Data Fig. 10a-b) . B cells were dramatically reduced in livers from infected Ly6e ΔHSC mice.\\r\\n Hepatic CD4 + T cells, NK cells, dendritic cells (DC), macrophages, and neutrophils were also depleted in infected Ly6e ΔHSC mice, whereas CD8 + T cells were unchanged (Fig. 4f, Extended Data Fig. 10a ).\\r\\n Infected Ly6e ΔHSC mice had a reduction in all splenic immune cell subsets except for CD4 + and CD8 + T cells (Fig. 4g, Extended Data Fig. 10b) . Depletion of immune cell populations in Ly6e ΔHSC organs was MHV-dependent, as cell numbers were not altered in PBS-injected mice.\\r\\n To determine whether loss of immune cells in MHV-infected Ly6e ΔHSC mice correlates with increased permissiveness to infection, we assessed MHV infection in splenocytes cultured from Ly6e ΔHSC , Ly6e fl/fl , and Ifnar -/mice. In Ly6e fl/fl splenocyte cultures, MHV infected macrophages, neutrophils, DC, and B cells, but not CD4 + or CD8 + T cells, as previously published (Fig. 4h, Extended Data Fig. 10c Together, our data demonstrate an important role for Ly6e in immune cell-mediated control of CoV infection. Higher viral burden and lower liver inflammation suggests that liver damage in MHV-infected female Ly6e ΔHSC mice may be due to viral pathogenesis that exceeds the ability of the immune system to control the infection. Notably, we observed MHV-induced loss of three classes of cells: 1) cells that are not permissive to the virus (CD4 + T cells), 2) cells that are permissive to MHV but do not demonstrate Ly6e KO-dependent infection (NK cells, macrophages, and neutrophils), and 3) cells that are permissive to MHV in a Ly6e KO-dependent manner (B cells and DC). This latter group is capable of antigen-presentation to CD4 + T cells 27,28 . Thus, in the absence of Ly6e, unrestricted MHV infection author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint of B cells or DC may impair T cell priming, thereby negatively impacting recruitment and activation of NK cells, macrophages, and neutrophils that contribute to control of viral disease.\\r\\n In conclusion, we identified LY6E as a pan CoV restriction factor that limits CoV entry and protects the host from severe viral disease. Our findings are striking given that LY6E has been primarily associated with an enhancing phenotype in which LY6E promotes entry of multiple viruses 29 . However, our data clearly expand the role of LY6E during coronaviral infection and establishes a novel function in protecting the host immune cell compartment. Determining the precise molecular mechanism underlying how LY6E inhibits CoV S protein-mediated membrane fusion will advance our understanding of cellular antiviral defenses against these important human pathogens. Antiviral membrane fusion inhibitors have been successfully implemented for treatment of HIV-1 infection 30 . A therapeutic approach mimicking the mechanism of action of LY6E could provide a first line of defense against novel emerging CoV infections. Furthermore, delineating which Ly6e-expressing immune cells protect mice from MHV will provide new insight into how individual antiviral effectors in distinct cellular compartments modulate viral pathogenesis. author/funder. All rights reserved. No reuse allowed without permission. 34-41 (1996) . Welch's correction (c-d, f-g), one-tailed Mann-Whitney U test (e), ratio paired student's t-test (g), 2way\\r\\n ANOVA followed by Sidak's or Dunnett's multiple comparison test (j,l). Error bars: SD. P values: c, ** p=0.0088; d, *** p=0.0001; e, * p=0.0143; f, * p=0.0286; g, * p=0.0101; h, *** p=0.0010; j, ** p=0.0073, p=0.0033; l, ns=0.0740, ** p=0.0013, *** p=0.0001. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint p=<0.0001; c, **p=0.0066; d, **** p=<0.0001; f, **** p=<0.0001; h, ns p=0.9975, ** p=0.0054, ns p=0.1520, 0.9892. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint scale bars are 500 µM (4x) and 100 µM (20x). Statistical significance was determined by Mantel-Cox test (a), two-tailed unpaired student's t-test with Welch's correction (c-e, i-k), two-tailed Mann-Whitney U test (f-g, l-m). Error bars: SEM (b), SD (c-g, i-m). P values: a, *** p=0.0002; c, ** p=0.0031; d, ns p=0.2136; e, **** p=5.759 x 10 -7 ; f, ns p=0.3698; g, ** p=0.0054; i, * p=0.0430; j, ns p=0.1321; k, ** p=0.0094; l, ** p=0.0072; m, **** p=8.741 x 10 -5 . author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint maps displaying significant changes (mean RPKM > 0.5; fold change > 2; FDR ≤ 0.05) in liver (a) and spleen (b). Dendrograms are normalized read data (row z-score) clustered with complete linkage method employing Spearman Rank correlation distance measurement. c, Pathway analysis of KO versus WT.\\r\\n Up-(red) or down-regulated (blue) pathways indicated by activation z-score. Numbers show significantly dysregulated genes as percentage of total gene number included in pathway. d-e, Expression of select genes in liver (d) and spleen (e). f-g, Immune cell counts from liver (f) and spleen \\r\\n HCoV-229E: 1.5 x 10 5 stable LY6E expressing or control cells were seeded in a 12-well plate and infected with HCoV-229E in a serial dilution. Cells were incubated for 2 hours at 33°C, then viral supernatant was removed, and an overlay of 1:1 Avicel (2.4%) (Avicel RC-581NF) and 2x DMEM, author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint supplemented with 10% FBS and 1x penicillin/streptomycin (p-s), was added. Cells were incubated for 3 days at 33°C before medium was removed, cells were washed, and stained with crystal violet (Sigma-Aldrich). Plaque forming units (PFU) were calculated.\\r\\n HCoV-OC43: 5.0 x 10 4 stable LY6E expressing or control cells were seeded in a 24-well plate and infected at 33°C for 1 hour with HCoV-OC43 (MOI=1). Virus was aspirated and 10% FBS/1x NEAA/1x p-s/RPMI (cRPMI) was added back to cells. 24 hours post-infection, cells were dissociated using Accumax (Sigma-Aldrich), fixed in 1% PFA, permeabilized per manufacturer's protocol (BD Cytofix/Cytoperm), and stained for nucleoprotein (1:500) and a goat anti-mouse AlexaFluor488- The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint anti-mouse secondary antibody. Images were acquired with a fluorescence microscope and analyzed using ImageXpress Micro XLS (Molecular Devices, Sunnyvale, CA).\\r\\n For infection assays, 1 x 10 5 target cells were seeded in a 24-well plate one day prior infection. Cells were infected with HCoV-229E-Rluc (MOI 0.1) in OptiMEM (Gibco) for 2 hours at 33°C. Cells were washed 1x with PBS and 10% FBS/1x NEAA/1x p-s/DMEM (cDMEM) was added back. Cells were incubated at 33°C for 24 hours, then washed with PBS and lysed using the Renilla Luciferase Assay System kit (Promega). Rluc activity was measured using a plate luminometer (EnSpire 2300 Multilabel reader by Perkin Elmer). For the reconstitution with CRISPR resistant LY6E (CR LY6E), 2.5 x 10 4 cells were seeded in a 24-well plate. One day post-seeding, cells were either left untransduced or transduced with a lentiviral vector encoding for CR LY6E or the empty control. 48 hours post-transduction, cell lysates were either infected with HCoV-229E-Rluc (MOI=0.1) for 24 hours as described above or harvested for Western blot.\\r\\n Pseudotyping of VSV*ΔG(Fluc) was performed as previously described 50 with two silent mutation (T2568G, T3327C)), have been described previously [51] [52] [53] . The expression plasmid for SARS-CoV-2 was generated as follows: the coding sequence of a synthetic, codonoptimized (for human cells) SARS-CoV-2 DNA (GeneArt Gene Synthesis, Thermo Fisher Scientific) author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint based on the publicly available protein sequence in the National Center for Biotechnology Information database (NCBI Reference Sequence: YP_009724390.1) was PCR-amplified and cloned into the pCG1 expression vector between BamHI and XbaI restriction sites. The MERS-CoV S cleavage mutants have been described before 17 \\r\\n The split luciferase fusion assay has been described before, with slight modifications 54 . Briefly, 6 x 10 5 293LTV cells were transfected with plasmids encoding CoV S proteins (pCAGGS-HCoV-229E, pCAGGS-MERS-CoV) together with a split-luciferase construct (Rluc8155-156 DSP1-7). The empty expression plasmid (pCAGGS-mCherry) as well as a construct encoding for VSV G served as negative and positive control, respectively. 2 x 10 5 stable LY6E expressing or empty control Huh7 cells were transfected using Lipofectamine 2000 (Thermo Fisher Scientific) with a plasmid encoding the second half of the split-luciferase protein (Rluc8155-156 DSP8-11). Approximately 30 hours post-transfection, both cell populations were dissociated (TryPLE Express, Gibco), counted, and equal cell numbers (2x10 4 cells each) were co-cultured for 16 -20 hours at 37°C. Supernatant was harvested and cells lysed using ice-cold passive lysis buffer (Promega). Cells were immediately transferred to -80°C for at least author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint 1 hour, before Rluc activity was determined using the Renilla Luciferase Assay System (Promega). Cells were kept on ice during all times post-cell lysis to eliminate DSP post-lysis complementation.\\r\\n Following extraction of total RNA from MERS-CoV (strain EMC) infected VeroE6 cells, the cDNA encoding the MERS-CoV S protein was generated by reverse transcription (RevertAid Premium Reverse Transkriptase, ThermoScientific). Three overlapping cDNA fragments were amplified by PCR (Phusion Hot Start II High Fidelity Polymerase, Thermo Scientific) and subsequently assembled by overlapping PCR. The cDNAs encoding the S proteins of either MERS-CoV (strain EMC) or HCoV-229E (truncated variant lacking the 10 amino acids at the C terminus, original plasmid pCAGGS-229E S) were amplified by PCR and inserted into the pVSV* plasmid 55 between MluI and BstEII restriction sites. The resulting plasmids were used to generate the recombinant viruses VSV*ΔG(MERS S) and VSV*ΔG(229E S) according to a published procedure 56 . All viruses were propagated on BHK-G43 cells 57 , resulting in viruses predominantly containing the homotypic VSV glycoprotein G in the envelope.\\r\\n Cells expressing LY6E or empty control were infected with VSV*ΔG(MERS S) or VSV*ΔG(229E S) at a MOI of 0.01. At 20 hours post-infection, the cells were fixed with 3% PFA/PBS and the nuclei stained with 4',6-diamidine-2'-phenylindole (DAPI, Sigma). An inverted fluorescence microscope (Zeiss) was used to determine area percentage area covered by syncytia.\\r\\n For heterologous cell-cell fusion assay, BHK-21 cells were infected with VSV*ΔG(CoV S) at a MOI of 1. After 2 hours, the cells were treated with trypsin/EDTA (Life Technologies, Zug, Switzerland) and resuspended in DMEM supplemented with 5% FBS (2x10 4 cells/mL). The infected BHK-21 cell suspension (500 µL) was seeded into 24-well cell culture plates along with LY6E-or empty controlexpressing Huh7 (1 x 10 4 cells). Cell co-cultures were incubated for 7 hours at 37°C, fixed with 3.6% author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint formaldehyde diluted in PBS, (Grogg-Chemie AG, Stettlen, Switzerland) and stained with DAPI. The percentage area covered by syncytia was calculated as above.\\r\\n Ly6e tm1a ES cells were obtained from the EUCOMM consortium 58 \\r\\n Six to twelve-week-old female and male mice were injected intraperitoneally with MHV-A59 diluted in PBS to the indicated titers or PBS for a mock infection control. All infected mice were monitored daily for weight and mortality. Animals that lost more than 20% of their original body weight were euthanized per IACUC guidelines.\\r\\n Viral titers in liver and spleen were determined from frozen organs after weighing, homogenization, and plaque assay on L929 cells. Alanine aminotransferase (ALT) was measured in fresh, unfrozen serum using VITROS MicroSlide Technology by the UTSW Mouse Metabolic Core. Livers were fixed in 10% author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint neutral buffer formalin, embedded in paraffin, sectioned at 5 µM, and stained with hematoxylin and eosin (H&E). Slides were analyzed by an independent pathologist (UTSW Animal Resource Center) who was blinded to experimental conditions. A numerical score was assigned for degree of inflammation and degree of necrosis for each liver. Inflammatory cell infiltration was scored using the following criteria: 0 = none, 1 = minimal, 2 = mild, 3 = moderate, and 4 = marked. The necrosis score was defined by the percent necrosis. 0 = none, 1 = <2%, 2 = 2-20%, 3 = 21-40%, and 4 = >40%.\\r\\n All primary cells were maintained in 10% FBS/10 mM HEPES/1 mM sodium pyruvate/2 mM Lglutamine/1x p-s/RPMI (cRPMI2) unless otherwise indicated.\\r\\n Primary bone marrow derived macrophages (BMDM) were prepared as described previously 60 . Viable cells were quantified using trypan blue exclusion. BMDM were plated at 2.5 x 10 4 cells per well in non- The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint separated from IHIC by centrifugation at 850 x g for 30 minutes, 22°C, with no brake. Red blood cells were removed from the resulting pellet with RBC lysis buffer (Tonbo Biosciences). Viable cells were quantified using trypan blue exclusion. IHIC and splenocytes (4 x 10 5 ) from infected or mock-infected mice were stained with a fluorescent viability dye (Ghostdye Violet 450, Tonbo Biosciences), treated with anti-CD16/CD32 (Tonbo Biosciences), stained with antibodies that recognize surface lineage markers, and then fixed in 1% PFA/PBS. Fixed volumes of cell suspensions were analyzed by flow cytometry the next day. Absolute cell counts were determined by multiplying final number of lineage marker-positive cells by dilution factor used to normalize cell counts for antibody staining.\\r\\n All primary cell samples were resuspended in 3% FBS/PBS and analyzed using a S1000 Flow Cytometer with a A600 96-well plate high throughput extension and compensated using CellCapture software (Stratedigm). Data was analyzed with FlowJo Software (Treestar). The flow gating strategy for liver and spleen immune cells is included in Extended Data Figure 9 .\\r\\n Differences in data were tested for significance using GraphPad Prism v8.3.1 for Windows (GraphPad).\\r\\n For details regarding the statistical tests applied, please refer to the figure legends. P values < 0.05 were considered significant.\\r\\n The authors declare that the data supporting the findings of this study are available within the article and its Supplementary Information files or are available on request. The RNAseq data discussed in this publication have been deposited in the Gene Expression Omnibus (GEO) database, https://www.ncbi.nlm.nih.gov/geo (GSE146074).\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint\\r\\n We would like to thank the following people for their generous contribution of reagents and infrastructure which greatly enhanced this study, especially the sequencing facility in Bern, the diagnostic facility of IVI, and the Animal Resource Center at UTSW. Furthermore, we are grateful to \\r\\n The authors declare no competing interests.\\r\\n Supplementary Information is available for this paper.\\r\\n Correspondence and requests for materials should be addressed to Volker Thiel, John W. Schoggins, and Charles M. Rice.\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint was harvested and extracellular viral RNA was extracted and viral replication detected via qRT-PCR (f). Cell lysates were harvested and intracellular Renilla luciferase activity was detected upon cell lysis (g). Cells were subjected to 3 rounds of freeze/thaw cycles. Cell debris was removed and the supernatant titrated on naïve Huh7 cells. Intracellular infectivity was determined (h). Supernatant was harvested and The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint ANOVA followed by Sidak's multiple comparisons test (c). Error bars: SD. P values: c, **** p=<0.0001 <0.0001. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint two pooled experiments. For i-m, n=8 (Ly6e fl/fl ,), n=6 (Ly6e ΔHSC ), or n=3 (PBS) from two pooled experiments. In h,n, scale bars are 500 µM (4x) and 100 µM (20x). Significance for was tested by\\r\\n Mann-Whitney U test (f-g, l-m). Error bars: SEM (b), SD (c-g, i-m). P value: a, **** p=3.946 x 10 -6 ; c, ns p=1144; d, ns p=0.1407; e, ** p=0.0026; f, ns p=0.1708; g, ns p=0.0779; i, ns p=0.6191; j, ns p=0.6842; k, * p=0.0119; l, ns p>0.9999; m, ns p=0.0779. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint -c, f-h) , two-tailed Mann-Whitney U test (d-e, i-j). Error bars: SD. P value: a, * p=0.0234; b, * p=0.0428; c, *** p=0.0004; d, * p=0.0210; e, ** p=0.0054; f, ns p=0.1544; g, ns p=0.5322; h, ns p=0.0662; i, ns p=0.0592; j, ns p=0.9064. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint DDX3X -DDX3, VIPERIN -RSAD2, SERPINE1 -PAI 1, IL28RA -CRF2, VEGFC -VRP, SP110 -IFI41, IRF2 -PRIC285, ZNF313 -RNF114, DNAPTP6 -LOC26010, LOC400759 -PSEUDOGENE,   PSCD1 -CYTH1, IFITM1 -IFI17, SIGLEC1 -FRAG, CCL5 -SYCA5, CXCL9 -MIG, IFIH1 Huh7 cells expressing TMPRSS2 were generated as follows. In order to generate TMPRSS2-encoding retroviral vectors, the open reading frame of human TMPRRS2 was first PCR amplified with primers adding an N-terminal cMYC epitope to the TMPRSS2 coding sequence. The resulting sequence was inserted into a modified version of the pQCXIP plasmid that contains a blasticidin resistance cassette instead of the usual puromycin resistance cassette 17 . Huh7 cells stably expressing human TMPRSS2 were generated by retroviral transduction and selection with the antibiotic blasticidin (50 µg/ml).\\r\\n Following selection, cells were maintained in culture medium (cDMEM) supplemented with 10 µg/ml blasticidin. To generate STAT1 -/-_CEACAM1 cells, human STAT1 −/− fibroblasts were transduced with lentiviruses encoding for the murine CoV receptor CEACAM1 (kind gift from David Wentworth, CDC, Atlanta, USA 59 and subsequently selected with 1 µg/mL puromycin. Stable LY6E expressing cell lines were generated upon lentiviral transduction with SCRPSY LY6E or as control SCRPSY Empty in DMEM containing 4 µg/ml polybrene (Millipore) and 20 mM HEPES buffer solution (Gibco). Cells author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint were selected using 2-4 µg/mL puromycin and were passaged until all cells in control wells without lentivirus were killed. The puromycin selected cells were further passaged and frozen down for subsequent experiments. The stable cell lines harboring LY6E orthologues have been described before, with the exception of C. dromedarius 9 . For this, a gBlock was ordered (XM_031439745.1) and Gateway cloning performed as described previously to generate pENTR and pSCRPSY plasmids 9 . Constructs encoding for Ly6/uPAR family members and LY6E ASM mutants have been described before 9 .\\r\\n To generate clonal LY6E KO cells and CRISPR-resistant LY6E, A549 cells were transduced with lentivirus containing a LY6E-specific sgRNA and Cas9 as described previously 9 . To generate a clonal cell line, the bulk transduced population was plated at single cell dilutions. Candidate clones were screened by Western blot for LY6E expression and Sanger sequencing. Silent mutations in the region targeted by the LY6E-specific sgRNA were introduced into HA-tagged LY6E to generate CRISPRresistant LY6E (CR LY6E). LY6E KO A549 cells were reconstituted with CR-LY6E by lentiviral transduction and expression confirmed by Western blot.\\r\\n All cell lines were regularly tested to check they were free of mycoplasma contamination using a commercially available system (PCR Mycoplasma test kit I/RT Variant C, PromKine).\\r\\n The ISG screen was performed as described previously with slight modifications 7,8 . Briefly, 5 x 10 3 Huh7 cells were seeded, transduced with individual lentiviruses, and 48 hours post-transduction infected with HCoV-229E at 33°C. Infection was stopped 24 hours (MOI=0.1) or 48 hours (MOI=0.01) postinfection and plates were immuno-stained as described previously 61 with an anti-HCoV-229E N protein antibody and a AlexaFluor488-conjugated donkey anti-mouse secondary antibody (Extended methods;\\r\\n Antibodies for immunofluorescence and flow cytometry). For high-content high-throughput imaging analysis ImageXpress Micro XLS (Molecular Devices, Sunnyvale, CA) was used as previously described 8 . Hits were normalized to cells expressing the empty vector. Depicted are genes that were author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint expressed in two independent screens with a transduction efficiency of at least 1 % (cut off). ISGs which were cytotoxic for the cells were excluded from the analysis. Normalized data can be found in Extended Data \\r\\n Viral RNA was extracted from cell lysates (NucleoMag-96RNA kit, Macherey Nagel) or supernatant (NucleoMag-Vet kit, Macherey Nagel) using the KingFisher Flex robot according to the manufacturer's recommendations. The commercially available TaqManTM Fast Virus 1-Step Master Mix (Applied Biosystems) was used for RT-qPCR with 2 µl of RNA input added to 8 µl of prepared mastermix per author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint sample. Viral RNA was detected using HCoV-229E specific primers normalized to a qRT-PCR standard for HCoV-229E (contains the M gene of HCoV-229E 63 ). Intracellular viral RNA was normalized to total RNA (determined via the housekeeping gene Beta-2-Microglobulin (B2M)). Cellular RNA was extracted from cell lysates of Huh7 cells using the Nucleospin RNA kit (Machery Nagel) according to the manufacturer's recommendations, quantified via NanoDrop, and used to generate a standard curve.\\r\\n PCR conditions are available upon request. \\r\\n Stable LY6E expressing or control Huh7 cells were seeded in a 24-well plate (4 x 10 4 cells) and mock infected or inoculated with HCoV-229E (MOI=5) in OptiMEM (Gibco) on ice for 1 hour. Cells were washed at least 3x with PBS and harvested immediately (t=0 h) or incubated at 33°C for 24 hours (t=24 h), before cell lysis and extraction of viral RNA as described above. Viral RNA was detected via RT-qPCR normalized to the housekeeping gene B2M as described above.\\r\\n Time course experiment 8 x 10 4 cells stable LY6E expressing or control Huh7 cells were seeded in 12-well plates and mock infected or infected with 229E-CoV-Rluc (MOI=0.1) for 2 hours at 33°C in OptiMEM. Cells were washed 3x with PBS and samples harvested at the indicated time points. To determine intracellular replication, cell lysates were collected and viral RNA extracted (NucleoMag-96RNA kit, Macherey author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint Nagel), followed by RT-qPCR as described before, or cells were lysed using the Renilla Luciferase Assay System kit (Promega) according to the manufacturer's recommendations and Rluc activity determined. To determine extracellular viral replication, viral supernatant was harvested, and viral RNA extracted (NucleoMag-Vet kit, Macherey Nagel), followed by RT-qPCR as described before. To determine intracellular infectivity, cells were subjected to 3 rounds of freeze/thaw cycles, centrifuged to remove debris (4,000 × g for 10 min), and the supernatant titrated on naïve Huh7 cells. To determine extracellular infectivity, supernatant was harvested at the indicated time points and titrated on naïve \\r\\n To test various protease inhibitors, 2 x 10 4 naïve or TMPRSS2-expressing, control or LY6E cells were seeded in a 96-well plate. One day post seeding, cells were pre-treated with the following compounds for 1 hour in OptiMEM, at 37°C: DMSO (1:500, Sigma-Aldrich), E64 D (10 µM, Sigma-Aldrich), and/or Camostat (100 µM, Sigma-Aldrich). Cells were mock infected or infected with HCoV-229E-Rluc (MOI 0.1) in the presence of the inhibitors for 2 hours at 33°C. Medium was changed to DMEM and cells incubated for 24 hours at 33°C. Cells were lysed and Rluc activity detected using the Renilla Luciferase Assay System kit (Promega) according to the manufacturer's recommendations.\\r\\n LY6E expressing or control Huh7 (2 x 10 5 cells) were seeded in 6-well cell culture plates. Cells were transfected using Lipofectamine 2000 (Invitrogen) with an expression plasmid encoding for MERS-CoV S (pCAGGS-MERS S). Cell lysates were harvested 48 hours post-transfection and subjected to Western blot analysis as described.\\r\\n author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint\\r\\n Multiple sequence alignment of LY6E amino acid sequences of different species (NCBI accession numbers provided in the figure) was performed using the EBI web based Clustal Omega tool (doi:\\r\\n 10.1093/nar/gkz268). The phylogenetic tree was built with the \\\"one click\\\" mode for phylogenetic analysis (http://www.phylogeny.fr/) with standard settings (doi: 10.1093/nar/gkn180; doi:\\r\\n 10.1186/1471-2148-10-8).\\r\\n Sections of liver and spleen were preserved in RNAlater Stabilization Solution (Thermo Fisher Scientific) and frozen. Thawed samples were transferred to PBS and homogenized. One-eighth of the homogenate was mixed with TRIzol Reagent (Thermo Fisher Scientific). BMDM were directly lysed in TRIzol Reagent and frozen. Total RNA was isolated according to the manufacturer's protocol. Liver and spleen RNA were subject to DNase treatment (TURBO DNA-Free kit, Thermo Fisher Scientific) per the manufacturer's protocol prior to RNA-Seq analysis. BMDM RNA was analyzed by one-step qRT-PCR using QuantiFast SYBR Green RT-PCR Kit (Qiagen) using Applied Biosciences 7500 Fast Real-Time PCR System.\\r\\n The quantity and quality of the extracted RNA was assessed using a Thermo Fisher Scientific qubit 2. author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint Data generated from individual samples (>30 million read per sample, single read 50-mers) were mapped separately against the GRCm38 murine reference genome. Gene expression was calculated for individual transcripts as reads per kilobase per million bases mapped (RPKM). All transcriptomic analyses were performed using CLC Genomics Workbench 20 (Qiagen, Aarhaus).\\r\\n Differentially expressed genes (DEGs) were identified by calculating fold changes in expression, pvalues were corrected by taking false discovery rate (FDR) for multiple comparison into account.\\r\\n Mus Musculus EBI Gene Ontology Annotation Database was used to execute Gene ontology (GO) Enrichment Analyses for biological processes. Gene identifiers for DEGs absolute FC > 5, RPKM > 2) were used as input and identification of significantly enriched GO categories. P-values for specific GO categories were generated after Bonferroni correction for multiple testing. Z-scores, used as indicator for activation of biological processes (positive value) or inactivation (negative value), were calculated based on the expression fold change as follows: The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint gctggcatcgggaacgtggtggactttggctac accctgaacaagggctgctccccgatctgtccc ggcccgagcgtcaatcttggagtggcgtccgt gggcacccactgctgccagagcttcctgtgca acatcagtgcagccgacggcgggctgcgggc cagcacccacgtgctgggcctcgggctcctgc tcagcctgctgtccgccctgctgcggcttggcc cctgaacccagctttcttgtacaaagtggtcccc author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.979260 doi: bioRxiv preprint. The rapid evolution of RNA viruses represents a significant challenge for preventing, treating, 57 and eradicating RNA viral diseases. High mutation rates in RNA viruses generate extensive 58 opportunities to overcome evolutionary hurdles, such as antiviral drugs, host immunity, or 59 engineered attenuating changes (1). The evolutionary pathways traversed by RNA viruses are 60 shaped by natural selection, which will favor some evolutionary trajectories more than others 61 based on whether mutations are beneficial, deleterious, or neutral (2). Predicting the likely results 62 of RNA virus evolution is an important step for anticipating viral emergence and for developing 63 escape-resistant antiviral drugs and vaccines (3, 4). 64 65 Coronaviruses (CoVs) are a family of positive-sense RNA viruses that cause human illnesses 66 ranging from the common cold to severe and lethal respiratory disease (5). All CoVs encode a 67 proofreading exoribonuclease within nonstructural protein 14 (nsp14-ExoN) that is critical for 68 replication, fidelity, fitness, and virulence, and ExoN-inactivation has been proposed as a 69 strategy for live-attenuated vaccine development (6-15). As members of the DEDDh superfamily 70 of exonucleases, CoV ExoNs hydrolyze nucleotides using four metal-coordinating amino acids 71 The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint ExoN active site mutants in alphacoronaviruses, including transmissible gastroenteritis virus and 77 hCoV-229E, have yet to be recovered and are proposed to be lethal for replication (19, 20) . infections and prolonged passage in tissue culture and following treatment with multiple 87 nucleoside analogues (6-11, 13, 14). SARS-CoV-ExoN-AA also is stable during acute and 88 persistent animal infections in immunocompetent and immune-compromised mice (12) . The lack 89 of primary reversion is not due simply to reduced adaptive capacity, as both SARS-CoV-and 90 MHV-ExoN-AA can adapt for increased replication (7, 14). Most strikingly, long-term passage 91 of MHV-ExoN-AA (250 passages, P250) yielded a highly fit population that had directly 92 compensated for defective proofreading through evolution of a likely high-fidelity RdRp (7 requires mutations to all four sites ( Figure 1A ). Viral mutation rates in the absence of 113 proofreading range from 10 -4 to 10 -6 mutations per nucleotide per round of replication (µ) (1). 114\\r\\n Assuming an ExoN-AA mutation rate of 10 -4 µ and accounting for codon degeneracy, the 115 probability of restoring the native amino acid sequence in a single round of replication is 10 -18 . 116\\r\\n Only rarely do ExoN-AA titers exceed 10 6 PFU/mL, so it is unlikely that ExoN-AA could 117 navigate this genetic barrier in a single infectious cycle. Thus, we hypothesized that ExoN-AA 118 reversion, if possible, would proceed incrementally. To identify potential pathways towards 119\\r\\n ExoN-AA reversion, we examined the possible single-nucleotide substitutions surrounding A89 120 and A91 ( Figure 1B ). Three mutations are synonymous, and five mutations yield amino acids 121 unlikely to coordinate with the positively-charged metals required for ExoN catalysis (glycine, 122 valine, proline, threonine, and serine) (16, 19, 21, 22). One mutation per site can restore the 123 acidic charge (i.e. AA-to-ED) but not the native amino acid. These variants have not been tested 124 in a CoV ExoN, but biochemical studies of E. coli DNA polymerase I ExoN mutants suggest that 125 these conservative substitutions would not restore WT-like ExoN activity (23) . We predicted 126 stepwise pathways to ExoN-AAàDE reversion based on restoration of acidic charge followed 127 by reversion to native amino acids ( Figure 1C ). We engineered and recovered variants in ExoN-128 AA requiring three mutations (3nt; ExoN-AD, ExoN-EA), two mutations (2nt; ExoN-DA, ExoN-129 ED, ExoN-AE), or one mutation (1nt; ExoN-DD, ExoN-EE) for reversion to WT-ExoN-DE 130 (Table 1) . We will hereafter refer to these mutants as intermediate revertants. All intermediate 131 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint revertants generated viable progeny during recovery, demonstrating that reversion to WT-ExoN-132 DE along these pathways is theoretically possible. The 3nt and 2nt mutants were genetically 133 stable during recovery, as confirmed by dideoxy sequencing. However, both 1nt mutants (ExoN-134 DD and ExoN-EE) reverted to WT-ExoN-DE during three independent recovery attempts, 135\\r\\n suggesting that these two variants are less fit than WT-ExoN-DE and demonstrating that 136 reversion by 1nt mutation is readily accessible. To test whether the 3nt or 2nt mutants would 137 revert more rapidly than ExoN-AA (4nt), we passaged three lineages of each mutant 10 times at (Table 1 ). In contrast, the 2nt ExoN-AE contained WT-revertants by P2 145 in all lineages at MOI = 0.5 PFU/cell and by P8 in one lineage at MOI = 0.01 PFU/cell. Once 146 observed, WT-revertants dominated the ExoN-AE population for the remaining passages. These 147 data indicate that at least one 2nt mutation pathway can lead to full reversion in tissue culture. 148\\r\\n The probability of ExoN-AE arising during a single infectious cycle of ExoN-AA is low but 149 theoretically achievable (~10 -9 ), so ExoN-AA could conceivably revert within just two infectious 150 cycles. However, complete reversion has never been observed even during prolonged passage or 151 persistent infections, suggesting that additional barriers to the replication, fitness, or maintenance 152 of intermediate revertants exist. 153 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint \\r\\n GAT GAA\\r\\n GCA GCT\\r\\n ExoN Motif I C All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. \\r\\n All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint\\r\\n the intermediate revertants are viable as recombinants but are not found in ExoN-AA 159 populations, we hypothesized that they confer no selective advantage over ExoN-AA (8, 9, 13). 160\\r\\n To test this hypothesis, we first analyzed replication of the 3nt and 2nt intermediate revertants 161 ( The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. nsp12/14-P250) were significantly more sensitive to 5-FU ( Figure 4B ). Finally, the nsp12-P250 224 and nsp14-P250 mutations significantly decreased fitness relative to WT-ExoN-DE ( Figure 4C ). 225\\r\\n We detected no statistical differences between the specific infectivity of WT-ExoN-DE and any 226 of the nsp12-P250 and nsp14-P250 variants in isolated infections ( Figure 4D ). Thus, mutations 227 in nsp12 and nsp14 that arose in the ExoN-AA background were detrimental to replication, 228 mutagen sensitivity, and competitive fitness in the presence of a fully-reverted ExoN-DE. These 229 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint Our results also extend existing studies of CoV ExoN motif I. Motif I AAàDE mutations in the 249 SARS-CoV nsp14-ExoN dramatically reduce nuclease activity in biochemical assays, but no 250 study has examined the contributions of each residue independently (16, 18) . Intermediate 251 revertants of ExoN-AA did not display consistent or statistical differences in replication, 5-FU 252 sensitivity, or competitive fitness relative to ExoN-AA, suggesting that they remain 253 phenotypically ExoN(-) during infection and supporting previous studies that motif I DE is 254 essential for WT ExoN function. Given these results, we were surprised to observe repeated 255 reversion of the ExoN-AE but not the other two 2nt variants, ExoN-DA and ExoN-ED. One 256 potential explanation is that the specific mutational bias of ExoN-AE makes the revertant 257 mutations more accessible than in ExoN-DA or ExoN-ED. Alternatively, if ExoN-AE has 258 profound replication or fitness defects, selection could drive primary reversion more quickly 259 away from this genotype. Consistent with this hypothesis, ExoN-AE reverted more quickly at a 260 higher MOI, where natural selection acts more efficiently on a larger population size (Table 1) Consistent with this hypothesis, the fitness effects of mutations in nsp12-P250 and nsp14-P250 269 differ based on the motif I genotype; they are beneficial in ExoN-AA but detrimental in the WT-270 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint ExoN-DE background. In previous studies, it has been difficult to determine whether the fitness 271 defects in ExoN(-) CoVs are directly linked to low-fidelity replication or through some other 272 mechanism. Our data suggest that the proofreading function of nsp14-ExoN can be uncoupled 273 from its more general role in replication (Figure 4) The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint Cloning and recovery of recombinant viruses. Site-directed mutagenesis in MHV genome 306 fragments was performed using \\\"round the horn\\\" PCR (originally described in (33)). Briefly, 307 adjacent primers containing the mutation of interest were 5¢-phosphorylated using T4 308 polynucleotide kinase (NEB, M0201S) using the buffer from the T4 DNA ligase, which contains 309 ATP (M0202S). PCR was performed on a plasmid template using the Q5 High-fidelity 2x previously (9). Briefly, genomic RNA was detected with a 5' 6-carboxyfluorescein (FAM) and 341 3' black hole quencher 1 (BHQ-1) labeled probe targeting nsp2 (Biosearch Technologies,  342 Petaluma, CA), and RNA copy number was calculated by reference to an RNA standard derived 343 from the MHV A fragment. Samples were plated in technical duplicate to minimize well-to-well 344 variation. Titers were determined by plaque assay in DBT-9 cells, and specific infectivity was 345 calculated as PFU per supernatant genomic RNA copy. 346 347 5-fluorouracil sensitivity assays. Stock solutions of 5-fluorouracil (Sigma F6627) were 348 prepared in dimethyl sulfoxide (DMSO). Sensitivity assays were performed in 24-well plates at 349 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint MOI = 0.01 PFU/cell, as previously described (7) were averaged, and values were excluded from subsequent analysis if the duplicate wells differed 371 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint by > 0.5 Ct. The relative abundance of competitor and reference were determined by subtracting 372\\r\\n Ct thresholds (DCtcompetitor = Ctcompetitor -Ctreference) and converted to reflect the fold-change in 373 ratio (Dratio = 2 -DCt competitor ). The log10Dratio was plotted against passage number, and the change 374 in log10Dratio (i.e. slope of linear regression) is the relative fitness. Note that regressions were fit 375 only through P1-P4, as slight deviations in 1:1 ratio in the input (P0) can skew the slope. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/618249 doi: bioRxiv preprint. 9 also deleted ORF 4a, which is dispensable for MHV replication in cell culture (52). The C-150 terminal region of ns2 within MHV-ExoN(-) P250 was truncated and fused to HE with a -1 151 frameshift. Ns2 is a phosphodiesterase (PDE) that protects viral RNA by degrading 2¢\\uf0a2à\\uf0e05¢\\uf0a2 152 oligoadenylate, the activating factor for cellular RNase L (53-55). The portion of ns2 deleted in The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint similar for all four viruses (Fig. 4B) . These data suggest that the increased replication of P250 173 viruses relative to WT-MHV is not fully accounted for by increased RNA synthesis. In addition 174 to RNA synthesis defects, ExoN(-) CoVs have up-to 20-fold increased mutation frequencies and 175 profoundly increased sensitivity to nucleoside and base analogs relative to WT CoVs (13, 14, 16, 176 17, 38). To determine whether nucleoside analog sensitivity of MHV-ExoN(-) was altered by 177 long-term passage, we treated cells infected with parental and passaged viruses with the base 178 analog, 5-fluorouracil (5-FU). 5-FU is converted intracellularly into a nucleoside analog that 179 incorporates into growing RNA strands and causes A:G and U:C mutations. For simplicity, we 180 hereafter refer to 5-FU as a nucleoside analog. Incorporation of 5-FU is increased in the absence 181 of ExoN activity (16). All viruses displayed a concentration-dependent decrease in viral titer but 182 differed greatly in their susceptibility to 5-FU (Fig. 4C ). At 120µM, WT-MHV P3 titers were 183 reduced by ~1 log 10 , while MHV-ExoN(-) P3 titers were undetectable (> 5 log 10 -fold reduction). 184\\r\\n WT-MHV 5-FU sensitivity was not altered by passage. MHV-ExoN(-) P250 was less susceptible 185 than MHV-ExoN(-) P3 to 5-FU treatment, with only a ~1.5 log 10 decrease in titer at 120 µM. replication cycles in which 5-FU can be incorporated (59) . MHV-ExoN(-) P250 had multiple 193 mutations in the spike glycoprotein, including one in the spike furin cleavage site that reduced 194 syncytia formation. To test whether the spike mutations manifested in resistance to 5-FU, we 195 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. However, M288T is not predicted to catalyze nucleotidylation. Within nsp14, 4 NS mutations 215\\r\\n were identified in the ExoN domain, and 2 NS mutations were in the C-terminal N7-216 methyltransferase domain (Fig. 6B ). We next modeled the structure of MHV nsp14 using Phyre 2 217 software (63) , resulting in highest-probability similarity to the SARS-CoV nsp14-nsp10 complex 218 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint between the structures) of 100% for residues 3-519 of MHV-nsp14. The model predicts that five 220 mutations are located close to surface of the protein (Fig. 6B ). All three modeled zinc finger 221 domains contain one NS mutation (F216Y, Y248H, L473I). Two mutations, D128E and F216Y, 222 are located near the interface between nsp10 and nsp14, though neither site has previously been 223 implicated in nsp10-nsp14 interaction (15, 64, 65). One NS mutation resulted in a D272E 224 substitution in ExoN motif III, a metal-coordinating active site residue. We previously reported 225 that alanine substitution of D272 results in an ExoN(-) phenotype (14), but the viability or 226 phenotype of a D272E substitution was not tested in that study. These data suggest that a 227 network of residues evolved to regulate nsp12 and nsp14 activity or stability in the ExoN(-) 228 background. 229 230 Fixed mutations in nsp12 and nsp14 in MHV-ExoN(-) P250 directly correlate with 231 increased resistance to multiple nucleoside analogs. To determine approximately when the 232 mutations in nsp12 and nsp14 arose, we performed di-deoxy sequencing across these protein-233 coding regions roughly every 20 passages (P10, 31, 50, 72, 90, 100, 120, 140, 160, 180, 200, 220, 234 240) . By this method, we detected consensus NS mutations at P10, P50, and P160 for nsp12, and 235 at P50 and P160 for nsp14 (Fig. 6) . Both nsp12 and nsp14 carried their full complement of P250 236 consensus mutations by P160, except for a minority variant (D913E) in nsp12 maintained at 237 <50% of the population between P200 and P250. These passage levels correlated with increased 238 replication of MHV-ExoN(-) (Fig. 2B ) and with decreasing sensitivity to 5-FU (Fig. 7A ). Neither 239 replication nor 5-FU sensitivity of MHV-ExoN(-) changed substantially between P160 and P250. 240\\r\\n To determine whether MHV-ExoN(-) evolved increased resistance to multiple nucleoside 241 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. implicating a complex evolutionary interaction between these two proteins. The measured fitness 284 correlated with the patterns of nucleoside analog resistance and RNA synthesis associated with 285 mutations in nsp12 and nsp14, suggesting a link between the evolution of these virus phenotypes. 286 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint\\r\\n The result also confirms that nsp12 and nsp14 are important but not sufficient to account for the 287 significantly increased fitness of MHV-ExoN(-) P250 relative to MHV-ExoN(-) P3. 288 289 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. Mutations conferring increased replication fidelity to RNA viruses have most frequently been 334 mapped to RdRps (24, 25, 30, 72) . Three findings suggest that mutations within nsp12-P250 335 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. with nsp12, acting as a primase/processivity factor (74, 75) and a helicase/NTPase, respectively 358 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. \\r\\n Cell culture. DBT-9 (delayed brain tumor, murine astrocytoma clone 9) cells were maintained 409 as described previously (91) The virus titer of each stock was determined by plaque assay using DBT-9 cells as described 425 previously (14, 91). For plaque assays of viruses containing the spike protein from MHV-ExoN(-426 ) P250, which does not form syncytia, plaques were visualized with neutral red (Sigma #N6264, 427 diluted 1:10 in PBS containing calcium and magnesium). Neutral red was added 24 hours after 428 plating and incubated for an additional 3-8 hours before formaldehyde fixation. Plaque 429 purification was performed by infecting DBT cells with serial dilutions of virus and overlaying 430 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint the cultures with agar. Single plaques were isolated, resuspended in PBS containing calcium and 431 magnesium, and inoculated onto fresh DBTs. This process was completed 3 times before 432 experimental stocks were generated, as above. and RNA copy number was calculated by reference to an RNA standard derived from the MHV 475 A fragment. Samples were plated in technical duplicate to minimize well-to-well variation. Titers 476 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint were determined by plaque assay in DBT-9 cells, and specific infectivity was calculated as PFU 477 per supernatant genomic RNA copy. 478\\r\\n Nucleoside and base analog sensitivity assays. 5-azacytidine (AZC), 5-fluorouracil (5-FU), and 480 ribavirin (RBV) were purchased from Sigma (product numbers A2385, F6627, and R9644, 481 respectively). Stock solutions of 5-FU and RBV were prepared in dimethyl sulfoxide (DMSO). 482\\r\\n 2'-C-methyladenosine (CMeA) was received from Gilead Sciences, Inc (Foster City, CA). 483\\r\\n Sensitivity assays were performed as described previously (16) viruses were detected with the same probe used in specific infectivity analyses (14). Reference 516 viruses were detected by a probe targeting the same region but with 10 silent mutations (5'-517 TCCGAACTACTGCAACCCCAAGTG-3') and labeled with 5' Quasar 670 and 3' black hole 518 quencher 2 (BHQ-2) (Biosearch Technologies, Petaluma, CA). RNA copy number was 519 calculated by reference to an RNA standard generated by in vitro transcription of the 520 corresponding MHV A fragment, and relative RNA abundance was calculated as the ratio of 521 competitor genomes to reference genomes. 522 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint and synonymous mutations (purple) are below. 564 The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint statistical significance was determined using one-way ANOVA. For Panel C, statistical 587 significance for change in titer of MHV-ExoN(-) spike-P250 relative to MHV-ExoN(-) P3 was 588 determined using Mann-Whitney test (*P<0.05, **<0.01, ***P<0.001, ns = not significant). 589 The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint tuberculosis DNA polymerase III αβ2ε replicase promotes polymerization and reduces 855 All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. Virus titer (log 10 PFU/ml) All rights reserved. No reuse allowed without permission.\\r\\n The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/175562 doi: bioRxiv preprint. \""
      ]
     },
     "execution_count": 92,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "str=read(\"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/treat_Cluster_1.txt\", String)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 93,
   "metadata": {},
   "outputs": [
    {
     "ename": "MethodError",
     "evalue": "MethodError: no method matching prepare!(::String, ::UInt32)\nClosest candidates are:\n  prepare!(!Matched::Corpus, ::UInt32; skip_patterns, skip_words) at C:\\Users\\Chandrapal Panwar\\.julia\\packages\\TextAnalysis\\pcFQf\\src\\preprocessing.jl:224\n  prepare!(!Matched::AbstractDocument, ::UInt32; skip_patterns, skip_words) at C:\\Users\\Chandrapal Panwar\\.julia\\packages\\TextAnalysis\\pcFQf\\src\\preprocessing.jl:241",
     "output_type": "error",
     "traceback": [
      "MethodError: no method matching prepare!(::String, ::UInt32)\nClosest candidates are:\n  prepare!(!Matched::Corpus, ::UInt32; skip_patterns, skip_words) at C:\\Users\\Chandrapal Panwar\\.julia\\packages\\TextAnalysis\\pcFQf\\src\\preprocessing.jl:224\n  prepare!(!Matched::AbstractDocument, ::UInt32; skip_patterns, skip_words) at C:\\Users\\Chandrapal Panwar\\.julia\\packages\\TextAnalysis\\pcFQf\\src\\preprocessing.jl:241",
      "",
      "Stacktrace:",
      " [1] top-level scope at In[93]:1"
     ]
    }
   ],
   "source": [
    "prepare!(str, strip_punctuation)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 94,
   "metadata": {},
   "outputs": [],
   "source": [
    "using TextAnalysis\n",
    "using Random"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 103,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Corpus{Union{FileDocument, NGramDocument, StringDocument, TokenDocument}}(Union{FileDocument, NGramDocument, StringDocument, TokenDocument}[FileDocument(\"C:\\\\Users\\\\Chandrapal Panwar\\\\Desktop\\\\python\\\\covid-19\\\\covid_data\\\\treat_Cluster_1.txt\", TextAnalysis.DocumentMetadata(Languages.English(), \"C:\\\\Users\\\\Chandrapal Panwar\\\\Desktop\\\\python\\\\covid-19\\\\covid_data\\\\treat_Cluster_1.txt\", \"Unknown Author\", \"Unknown Time\"))], 0, Dict{String,Int64}(), Dict{String,Array{Int64,1}}(), TextHashFunction(hash, 100))"
      ]
     },
     "execution_count": 103,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "str=DirectoryCorpus( \"C:/Users/Chandrapal Panwar/Desktop/python/covid-19/covid_data\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 104,
   "metadata": {},
   "outputs": [],
   "source": [
    "standardize!(str, StringDocument)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 106,
   "metadata": {},
   "outputs": [],
   "source": [
    "    prepare!(str, strip_case)\n",
    "    prepare!(str, strip_case)\n",
    "    prepare!(str, strip_punctuation)\n",
    "    prepare!(str, strip_punctuation)\n",
    "    prepare!(str, strip_stopwords)\n",
    "    prepare!(str, strip_stopwords)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 107,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Corpus{Union{FileDocument, NGramDocument, StringDocument, TokenDocument}}(Union{FileDocument, NGramDocument, StringDocument, TokenDocument}[StringDocument{String}(\"odor perception    emotional   behavioral responses   odors strongly depend   experience    learned odor context associations         lifetime     animal  mouly   sullivan  2010      cellular   neural circuit mechanisms underlying olfactory learning   memory     remain poorly understood \\r\\n recent studies   episodic   contextual learning   hippocampal neural networks   suggested   memories   encoded     activity   distributed ensembles   neurons    referred        memory trace   mayford   reijmers  2015  poo et al  2016  tonegawa et al  2015      neurons constituting     memory trace       encode information     environmental context   associated emotions   past experiences      activity       sufficient   memory retrieval  liu et al  2012  reijmers et al  2007  tanaka et al  2014   \\r\\n      investigate   organization   odor memory traces     olfactory  piriform  cortex   mice    piriform cortex    trilaminar paleocortical structure      largest cortical   receiving direct afferent inputs     olfactory bulb          receives topographically organized inputs   olfactory sensory neurons     nose  individual piriform neurons respond   combinatorial inputs   multiple piriform cortex       hypothesized   support auto associative network functions     retrieve previously learned information   partial   degraded sensory inputs  haberly  2001  wilson   sullivan  2011    piriform pyramidal cells form     recurrent network      reciprocally connected   adjacent highorder associative   including   prefrontal  entorhinal   perirhinal cortex     amygdala  johnson et al  2000  sadrian   wilson  2015    storage   information         nmda dependent  associative plasticity   connections  johenning et al  2009  kanter   haberly  1990  quinlan et al  2004   \\r\\n furthermore  changes   piriform network activity  chapuis   wilson  2011  chen et al  2011  li et al  2008  sevelinges et al  2004    stabilization   piriform odor representations  shakhawat et al  2015      observed   associative olfactory learning  finally  excitotoxic lesions     posterior piriform cortex   rats perturb odor fear memories  sacco   sacchetti  2010      optogenetic stimulation   artificial piriform ensembles   sufficient   drive learned behaviors  choi et al  2011           studies   led     hypothesis   piriform neural ensembles encode olfactory memory traces \\r\\n   test   hypothesis      developed   intersectional genetic strategy   mice   target piriform neurons     activated   olfactory experience    employed\\r\\n   selectively label   manipulate piriform neurons     activated   odor exposure      cfos tta transgenic mice  reijmers et al  2007          activity dependent cfos promoter drives expression     tta transcription factor      stereotaxically injected     piriform cortex adeno associated viruses  aavs  expressing designer receptors exclusively activated   designer drugs  dreadds      control     tta responsive promoter teto   figure 1a   \\r\\n dreadds  alexander et al  2009  ferguson et al  2011  increase  hm3dq mcherry    decrease  ha hm4di ires mcitrine    excitability   neurons   activation     ligand clozapine   oxide  cno  temporal control   dreadd expression   provided   doxycycline  administered     diet    interferes     binding   tta   teto     suppresses transgene expression  gossen et al  1995    mice maintained     home cage     doxycycline containing diet   low basal dreadd expression  detected   anti ha immunohistochemistry   native mcherry fluorescence  figure 1b          induce dreadd expression  mice       doxycycline five days   odor presentation  exposure   odor followed   foot shock    experimental procedures  resulted     fos tagging   sparse neural ensembles  median  interquartile range  1 6  0 3      piriform neurons   =2   hm4di  2 2  0 5      piriform neurons   =3   hm3dq      broadly dispersed throughout   anterior   posterior piriform cortex   figure 1b      consistent   previous observations   endogenous cfos expression  datiche et al  2001  illig   haberly  2003   \\r\\n   test   cno mediated activation   dreadds   fos tagged piriform neurons alters   excitability    performed   cell recordings   acute brain author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint slices containing piriform cortex  dreadd expressing neurons   identified based   mcitrine  ha hm4di ires mcitrine    mcherry  hm3dq mcherry  fluorescence  figure 1        absence     dreadd ligand cno    resting membrane potential   dreadd expressing piriform neurons   indistinguishable   dreadd negative cells  71 70  4 37  mv    =24   4 32  mv   hm4di expressing cells   =14   72 56  3 92  mv   hm3dq expressing cells   =15   =0 631  kruskal wallis anova  figure 1d     bath application   cno  5 µm    membrane properties selectively changed   dreadd expressing cells  hm4di expressing cells hyperpolarized  74 51  5 17   mv   =12   =0 007  wilcoxon signed ranks test    hm3dq expressing cells depolarized  66 95  4 12  mv   =13   =0 001  wilcoxon signed ranks test  whereas   resting membrane potential   dreadd negative cells remained unchanged  70 28  3 81  mv   =15   =0 890  wilcoxon signed ranks test  figure 1d     relative changes   1 00  0 03    dreadd negative cells   =15   =0 890    sample wilcoxon signed rank test  1 04  0 03    hm4di expressing cells    =12    0 001    sample wilcoxon signed rank test    0 91  0 09    hm3dq expressing cells    =13   =0 001    sample wilcoxon signed rank test  figure 1e     determine   impact   cno treatment   neuronal excitability    delivered depolarizing current steps   compared       action potentials triggered   60 pa step current   action potential threshold    experimental procedures    found     hm4di expressing cells        evoked action potentials significantly decreased   cno application  3 75  1 59  vs  0 30  3 48   =10   =0 010  wilcoxon signed ranks test  whereas   hm3dq expressing cells        author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint action potentials significantly increased   cno application  8 67  9 10  vs  19 0  27 6  mv   =11   =0 006  wilcoxon signed ranks test    contrast  dreaddnegative cells     exhibit   significant change         evoked action potentials   cno application  4 33  1 67  vs  4 67  3 00   =9   =0 672  wilcoxon signed ranks test  figure 1f     relative changes   1 00  0 41    dreaddnegative cells    =9   =0 641    sample wilcoxon signed rank tests  0 09  0 81    hm4di expressing cells   =10   =0 010    sample wilcoxon signed rank tests    1 81±1 09   hm3dq expressing cells   =11   =0 004    sample wilcoxon signed rank tests  figure 1g          experiments     fos tagging   olfactory fear conditioning marks   sparse   dispersed subpopulation   piriform neurons      cno mediated activation   hm4di   hm3dq expressed   fos tagged neurons selectively decreases   increases   excitability \\r\\n   fos tagged piriform neurons     activated   olfactory learning constitute   essential component     olfactory memory trace    perturbing   activity     ensembles   interfere   memory recall    test   prediction    fostagged piriform neurons   olfactory fear conditioning      chemogenetically silenced fos tagged ensembles   odor fear memory recall  cfos tta transgenic mice  experimental     =8    bilaterally injected   piriform cortex     aav teto hm4di vector  wild type mice injected       virus served     control  control     =8    training    conditioned odor  cs                    conditioning box   paired     foot shock  applied           box     odor   delivered    experimental author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint procedures   figure 2a   mice   learned   escape     conditioned stimulus  cs    running towards   opposite       box    control odors    presentation     paired   foot shock  cs        entrain odorspecific   opposed   generalized fear responses  chen et al  2011    doxycycline   removed     diet   mice prior   training    permit   induction   hm4di expression   active fos expressing piriform neurons   mice     experimental    mice   returned   doxycycline containing diet immediately   odor fear conditioning    memory recall     odor foot shock association   tested   days         odors         box   figure 2b        mice   intraperitoneally       injected   cno   testing    exclude potential offtarget effects   cno  gomez et al  2017        mice    verified viral gene expression   post hoc histological examination  previous studies   suggested     posterior piriform cortex       associative memory encoding  sacco et al  2010  calu et al  2008  haberly 2001        excluded     analysis mice           posterior piriform cortex   spared   viral infection  supplementary figure 2      combined   parameters   quantify   learned escape behavior    position     mouse     box      increase     maximum velocity   odor presentation   figure 2b     parameters   chosen     described     behavioral response   mice     odor foot shock pairing   training  supplementary figure 3   \\r\\n   expected  mice     control   exhibited robust escape behavior   cs  presentation    median position     mice     box         stimulus delivery   0 74    0 defined     site   odor presentation   1     opposite       box    mice exhibited   1 9 fold increase     maximum velocity   figure 2e             contrast  escape behavior   significantly reduced author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint   mice     fos tagged neurons   silenced    median position     box         stimulus delivery   0 42      1 1 fold increase   maximum velocity   significantly lower       control    mann whitney test   =7   =0 007     position   velocity  figure 2e            difference     position   mice   cs    cs presentation     significantly       control   experimental    mann whitney test   =13   =0 0499  figure 2f   differences   behavioral responses   cfos tta transgenic mice   specifically due   hm4di activation    escape behavior   cno injection   unaffected   cfos tta transgenic mice injected   aavs expressing   calcium indicator gcamp   =9  supplementary figure 4  \\r\\n   mice   similarly close     odor port     onset   cs  odor presentation  mann whitney test   =28   =0 7209  figure 2d       similar maximum velocities     7   time period   preceded odor presentation  mann whitney test   =24   =0 4418  figure 2g     observations exclude   inherent differences     due       cs       furthermore  escape behavior   control mice   specific     cs    none     mice ran       cs  figure 2d          finally        observe   differences     overall mobility   mice throughout   entire testing session    median speed   mice        0 68  0 28  cm/    0 46  0 14  cm/      control   experimental   respectively      significantly    mann whitney test   =22   =0 3282      mice     bias     left           testing box  time spent     left   divided   time spent          1 36  0 72   \\r\\n 1 51  0 52      control   experimental   respectively  mann whitney test   =27   =0 6454       data suggest   fos tagged piriform ensembles     activated   olfactory fear conditioning       odor fear author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint memory recall \\r\\n     cfos tta transgenic mice expressing hm4di failed   robustly escape     cs        noted     behaved     cs    cspresentation              odor port   cs  presentation     cs presentation  wilcoxon matched pairs signed rank test   = 36   =0 0078  figure 2e       maximum velocity ratio   significantly    wilcoxon matched pairs signed rank test   = 36   =0 0078  figure 2h       indicate   partial memory     learned association  possibly due   incomplete silencing   fos tagged neurons   piriform cortex      existence   parallel neural pathways   partially compensate     loss   piriform functions  finally  silencing     piriform cortex       hemisphere     abolish   learned escape behavior     conditioned odor  supplementary figure 5   \\r\\n           chemogenetic silencing   piriform ensembles     fostagged   olfactory fear conditioning disrupts odor detection   discrimination     selectively affecting odor fear memory recall    test   possibility    monitored sniffing behavior   mice     olfactory habituation/dishabituation assay   figure 3a        established test   odor detection   discrimination  coronas samano et al  2016  verhagen et al  2007  wesson et al  2008    fostagging   piriform neurons   olfactory fear conditioning   performed   described      days    changes   sniffing behavior   response   odor exposure   tested     plethysmograph    hm4di expressing piriform neurons author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint   cfos tta transgenic mice   silenced using      injection   cno    experimental procedures    observed   mice     experimental   =8    control   =6    increased   sniff frequency   presentation     novel  neutral odor  1 6 fold increase  wilcoxon matched pairs signed rank test   =21   =0 0312    =36   =0 0078     control   experimental    respectively  figure 3b      repeated exposure       odor  neutral odor   cs  resulted     decrease     sniff frequency  reflecting habituation   4 consecutive exposures  figure 3c        finally  presentation     cs  increased sniff frequency  1 4 fold increase  wilcoxon matched pairs signed rank test   =21   =0 0312    =34   =0 0156     control   experimental    respectively  figure 3d     changes   odor sampling behavior     shown   report   detection   discrimination     odor stimuli  coronas samano et al  2016  verhagen et al  2007  wesson et al  2008         data suggest   basic behaviors characteristic   odor sampling  detection   discrimination   unaffected     silencing   piriform ensembles     fostagged   olfactory fear conditioning \\r\\n odors activate unique   overlapping ensembles   piriform neurons  bolding   franks  2017  poo   isaacson  2009  roland et al  2017  stettler   axel  2009      silencing neurons   respond     odor   partially interfere     retrieval   information associated     odors    test   prediction   modified   behavioral protocol   separate fos tagging   olfactory fear conditioning  cfos tta transgenic   wild type control mice   injected   aav teto hm4di    expression   hm4di   piriform neurons   induced   mice author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint   exposed   odor  eugenol   beta citronellol      neutral environment    subsequent foot shock  fos tagging resulted     labeling   sparse neural ensembles  similar     labeled   odor foot shock association  1 1  0 3      piriform neurons   =3  mice   returned   doxycycline containing diet   trained   days     associate     odor  ethyl acetate  cs    foot shock \\r\\n behavioral testing     learned escape behavior   response     cs    performed   day           injection   cno    experimental procedures   figure 4a     found   learned escape behavior   cfos tta transgenic mice expressing hm4di  experimental     =12    similar   somewhat attenuated compared   wild type controls   =10      exhibited   escape behavior   cs  presentation  indicated     significant increase     maximum velocity  wilcoxon matched pairs signed rank test   =55   =0 0020    =56   =0 0269     control   experimental   respectively    difference     maximum velocity ratio  /       mann whitney test   =35   =0 1071  figure 4e        difference     position   mice   cs    cspresentation     similar     control   experimental    mann whitney test   =37   =0 1402  figure 4d        observed   silencing neutral odor representations somewhat dampened   behavioral response     conditioned stimulus    behavioral responses     robust compared   controls   quantifying   position   mice     box   cs  presentation  mann whitney test   =22   =0 0112  figure 4b        attenuated behavioral responses   observe   reflect   partial degradation   odor information     result     silencing   piriform neurons responsive       cs      neutral odor \\r\\n author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint\\r\\n   piriform neural ensembles     activated   olfactory fear conditioning encode   odor fear memory trace    reactivation     neurons     sufficient   trigger memory recall    test   prediction    devised   modified behavioral task   assess memory recall    monitored exploratory behavior       field   ambient odor exposure   artificial reactivation   fos tagged neurons    test       rely     escape behavior     odor originating     spatially defined odor source   figure 5a       trained mice   associate ethyl acetate    cs    foot shock    ambient exposure     cs  diffused       field  exploratory behavior   mice   significantly attenuated compared   baseline exploratory behavior measured   day earlier  wilcoxon matched pairs signed rank test   =6   = 21   =0 0312  ethyl acetate exposure     affect exploratory behavior   mice     previously   exposed   ethylacetate   foot shock  wilcoxon matched pairs signed rank test   =5   =1    0 9999    data suggest   re exposure   mice     cs  causes   increase   anxiety   behavior  expressed     decrease   exploratory behavior       field \\r\\n     repeated   experiment   cfos tta transgenic mice        piriform cortex     infected     activating dreadd aav teto hm3dq  supplementary figure 6    instead   exposure     cs         injected   dreadd ligand cno   reactivate hm3dq expressing piriform neurons     fos tagged   odor foot shock association    experimental procedures   figure 5a     found   cno mediated reactivation  similar   cs  exposure  caused   significant decrease   exploratory behavior   mice     undergone author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint olfactory fear conditioning  wilcoxon matched pairs signed rank test   =8   = 34   =0 0156  figure 5b      cno mediated reactivation   piriform neurons     fos tagged   ethyl acetate exposure   foot shock     affect exploratory behavior   mice  wilcoxon matched pairs signed rank test   =7   = 10   =0 4688  figure 5b             data suggest   chemogenetic reactivation   piriform neurons     active   odor foot shock exposure   sufficient   trigger fear memory recall \\r\\n         artificial memory recall depends     specificity     fostagged neural ensemble    piriform reactivation sufficient   trigger fear memory recall          includes   cs tagged ensemble      purpose    generated   synthetic fos tagged ensemble    sequentially exposing mice   ethylacetate paired   foot shock  cs    neutral odors  cs  eugenol   betacitronellol    figure 5d     exposure     cs  resulted   anxiety   behavior similar       observed     previous experiment  wilcoxon matched pairs signed rank test   =6   = 21   =0 0312  figure 5e      artificial reactivation   synthetic fos tagged ensemble     result   changes   exploratory behavior     indicate memory recall  wilcoxon matched pairs signed rank test   =8   = 4   =0 8438  figure 5e        data suggest     artificial reactivation  piriform cortex   extract meaningful information     synthetic representation generated     sequential presentation   cs    cs odors  author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint\\r\\n   piriform cortex           provide   substrate   storing associative olfactory memories    studies   shown   piriform cortex cells   encode information   carries behavioral significance  calu et al  2008   mandairon et al  2014   sacco et al  2010       functional relevance       assessed   direct experimental manipulation  using   cfos dependent  intersectional genetic approach   visualize   manipulate piriform neurons activated   olfactory fear conditioning  figure 1     found   chemogenetic silencing   fos tagged piriform ensembles abolished   learned odor escape behavior  figure 2   silencing   fos tagged ensembles     alter basic odor detection   discrimination  figure 3    furthermore  chemogenetic silencing   neurons responsive     neutral odor   moderately attenuated odor fear memory recall  suggesting   fos tagged piriform ensembles     odor specific   figure 4    finally  specific chemogenetic reactivation   piriform ensembles     active   learning resulted   reduced exploratory behavior       field assay    effect     similarly observed   exposing mice     conditioned odor stimulus  figure 5         experiments identify piriform neurons     active   learning       sufficient   trigger odor fear memory recall    indicates   formation   functional connections   fos tagged piriform cells     associative     firmly establish piriform ensembles     essential component   odor fear memory traces \\r\\n artificial reactivation     olfactory memory trace author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint reactivation   neurons   cno mediated activation   dreadd receptors     recapitulate   temporal characteristics   piriform odor responses     modulation   active sampling  bolding   franks  2017  miura et al  2012   \\r\\n despite    reactivation   fos tagged piriform ensembles activated   odor foot shock exposure   sufficient   elicit   behavioral response    piriform neural circuits   downstream target regions process   artificial activity patterns remains     determined    possibility     despite   temporal limitations   memory trace reactivation  piriform network mechanisms   retrieve   perception     reinforced conditioned stimulus  consistent     model  recent studies   shown   spatial patterns   odor evoked activity      absence   precise temporal information    sufficient   decode odor identity   piriform cortex  bolding   franks  2017  miura et al  2012  roland et al  2017    alternatively          reactivation generates       stress   anxiety      evoking   perception     odor  candidate target       processing   odor fear associations include   basolateral amygdala     medial prefrontal cortex  dejean et al  2016  gore et al  2015         relevant neural circuit components remain     identified  finally  reactivating   synthetic ensemble   neurons activated   exposure     reinforced stimulus  cs      nonreinforced stimuli  cs      elicit   measurable behavioral response    observation suggests   behaviorally relevant information     extracted     neural ensemble representing conflicting  aversive versus neutral  information    result   consistent     finding     reactivation     artificial contextual memory     hippocampus competes     retrieval     learned context shock association  ramirez et al  2013    author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint\\r\\n     limitation     cfos tta system       tagging   active neurons   transient    duration     expression   tta dependent proteins   determined     time course   induction     stability   cfos tta dependent transcripts   proteins   tagged neurons  previous studies   shown   tettag dependent expression   regulators   neural activity lasts       five days     undetectable   thirty days  cowansage et al  2014  liu et al  2012  zhang et al  2015        restricted   manipulations   neural activity   behavioral analyses       day time window   fos tagging   neurons   olfactory conditioning    constraint     fos tagging system     slow temporal dynamics      consequence  neurons responding     reinforced conditioned stimulus  cs    odorant paired   foot shock    neurons responding       reinforced conditioned stimuli  cs    odorants   associated   foot shock    tagged     silencing experiments          cs compromises   specificity     fos tagged neural ensembles  nevertheless    cs provides     experimental readout     odor selectivity     behavioral response \\r\\n interestingly        fos tagged piriform neurons   observe   significantly lower         odor responsive neurons detected   electrophysiological recordings   awake mice  bolding   franks  2017  miura et al  2012  zhan   luo  2010           experiments      fraction   cells exhibited low firing rates       fos tagged population   represent   subpopulation   neurons     strongly activated   odor  schoenenberger et al  2009    fos tagging     mark plastic changes supporting memory formation  cole et al  1989  minatohara et al  2015    indeed  cfos mrna levels decrease significantly   injection     antagonist     nmda receptor    key player     induction   author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint synaptic plasticity  tayler et al  2011        term memory   synaptic plasticity   impaired   cfos production   perturbed     central nervous system  fleischmann et al  2003  seoane et al  2012    future studies   allow   directly test   hypotheses \\r\\n   hippocampus     studied extensively     role   spatial   contextual memory  basu   siegelbaum  2015    recently  fos tagging   hippocampal neurons   contextual fear conditioning   provided   insight     cellular   neural circuit mechanisms   learning   memory  cai et al  2016  garner et al  2012  liu et al  2012  ramirez et al  2013  roy et al  2017  ryan et al  2015         principles   memory formation   storage   hippocampusrelated neural networks apply     cortical structures remains   unknown \\r\\n interestingly  pirform cortex   hippocampus share   similar circuit organization         modeled   auto associative networks  haberly  2001    consistent     model    findings reveal striking similarities   olfactory memory traces     piriform cortex   memory traces   contextual fear     hippocampus      piriform cortex   hippocampus  learning activates sparse   distributed ensembles   neurons   appear   lack topographic organization  furthermore  hippocampal neurons tagged   contextual fear conditioning    piriform neurons tagged   olfactory fear conditioning         sufficient   memory retrieval  liu et al  2012  tanaka et al  2014      olfactory system   particularly   suited   studying learning   memory  indeed  odor memories   olfactory driven behaviors   robust   animals  including   mice  providing highly reliable   quantitative behavioral readouts  furthermore    author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint olfactory cortex       synapses       peripheral olfactory sensory neurons     olfactory epithelium      neural inputs   drive piriform cortex activity       characterized  banerjee et al  2015  dhawale et al  2010  economo et al  2016  fukunaga et al  2012  kato et al  2012  roland et al  2016  yamada et al  2017    knowing   properties   neural input patterns   provide   basis   understanding neural circuit input output computations     modification   learning   experience \\r\\n author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint\\r\\n mice   housed   24°      12 hour light/12 hour dark cycle   standard food   water provided ad libitum  mice     housed   littermates     beginning   surgery     single housed   ventilated cages throughout   duration     experiment  cfos tta mice  reijmers et al  2007    fed   diet containing 1g/kg doxycycline     minimum   4 days   surgery  wild type control animals   siblings       carry   cfos tta transgene    age   mice     time   behavioral testing ranged   10   15 weeks  male mice         hm4di mediated neural silencing    females     hm3dq mediated neural activation experiments  experiments   carried   according   european   french national institutional animal care guidelines  protocol apafis 2016012909576100 \\r\\n   adeno associated viruses  aavs    generated   penn vector core  university   pennsylvania  serotype 8  10 13 genome copies/ml  1 2 dilution   sterile pbs     day   injection \\r\\n author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint\\r\\n mice   anaesthetized   ketamine/xylazine  100 mg/kg/10 mg/kg  sigma aldrich    aav vectors   injected stereotaxically     piriform cortex sections   washed 2 times   pbst   1 time   pbs  mounted   slides   author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint coverslipped   vectashield mounting medium  vectorlabs  images   acquired     stacks  70   140 µm   total thickness  step size 7 µm      leica sp5 confocal microscope     single plane sections     zeiss axio zoom microscope   processed   fiji \\r\\n counting   stacks  7 µm step    acquired     leica sp5 confocal microscope     20x objective     resolution   512x512   1024x1024 pixels  1 pixel  0 72µm       piriform cortex   delineated     maximum projection     stack using   custom written imagej macro      mouse  2   4 sections  median volume 9 3e 2  9 4e 3  mm 3     analyzed          hemispheres     = 0 8    = 1 6mm relative   bregma \\r\\n ha stained neurons  hm4di    counted   hand  hm3dq mcherry positive neurons   counted using   custom written imagej plugin    pre processing   image  subtract background  remove outliers   median filter      stack   thresholded using   renyientropy algorithm      slice     stack  objects           40µm 2   removed using   analyze particle command   imagej    mask     slice     stack   created containing   filled outline     measured particles        cells         3d objects detected     stack  automated cell detection     prone   error       detection efficiency   checked       pictures     people blind     experimental conditions \\r\\n       cells   calculated       cells   cubic millimeter    estimate     total     neurons  counterstained   neurotrace    cubic millimeter  7 1e4 mm  3     obtained   counting neurons   representative volumes author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint containing     layers   piriform cortex      consistent     obtained    srinivasan   stevens  2017   \\r\\n parasagittal   coronal slices  300 µm thick    piriform cortex   prepared   6 8 week   cfos tta mice injected   aav teto hm4di ires mcitrine   aav teto hm3dq mcherry  animals   anaesthetized   ketamine   xylazine  100 mg/kg/10 mg/kg  sigma aldrich  perfused   ice cold acsf  125 mm nacl  2 5 mm kcl  25 mm glucose 25mm nahco 3   1 25 mm nah 2 po 4   2 mm cacl 2   1 mm mgcl 2   1 mm pyruvic acid  bubbled   95    2   5  co 2   adjusted   295±5 mosm osmolarity    decapitated    brain   cooled   ice cold acsf solution     sliced using   7000sm2 vibrating microtome  campden instruments  uk  slices   incubated       solution   34°      hour     continuously perfused   acsf solution  2 ml/min    34°      recording chamber  dreadd expression   detected     photon excitation  830nm  chameleonmru x1  coherent  uk      scientifica trim scope ii microscope  lavision  germany      60x water immersion objective    cell recording pipettes   5 7 mω resistance   filled     following solution    mm  122 kgluconate  13 kcl  10 hepes  10 phosphocreatine  4 mg atp  0 3 na gtp  0 3 egta  adjusted   ph 7 35   koh    recording solution   contained   morphological tracer alexa fluor 594  5 µm  red channel    nonexpressing   hm4di positive cells     ca 2   sensitive dye fura 2  300 µm  replacing egta     recording solution  green channel    hm3dq positive cells    enable identification     patched cell     visualize   morphology    excitability   cells   measured   current clamp mode   500 ms steps   current author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint injections    300    500 pa   steps   20 pa    compared       action potentials triggered   equivalent depolarization     step 60 pa     action potential threshold       cno  5 µm  application    series resistance   usually  20 mω    data   discarded     changed       20      recording  signals   amplified using epc10 2 amplifiers  heka elektronik  lambrecht  germany  voltage clamp recordings   filtered   5 khz   sampled   10 khz    current clamp recordings   filtered   10 khz   sampled   20 khz      patchmaster v2x32 program  heka elektronik \\r\\n   training box       train mice   escape     odor    box   rectangular    57 cm    17 cm    64 cm      grid floor     72 stainless steel rods  diameter=6 mm  space   rods=2 mm  current   delivered     aversive stimulator  medassociates  115v  60 hz    custom   switcher allowed   electric foot shock  0 6 ma  0 6 ms      applied independently     half     box    testing box       test memory retrieval    dimensions   similar     training box  materials     walls   floor         training box   create     context      field box consisted     white plexiglas  6 mm thickness  container  50 cm   50 cm   38 height \\r\\n sniffing behavior   monitored   freely moving mice using   plethysmograph  emka technologies  electrical signals   amplified using rhd 2132 amplifier boards  intan technologies    band pass filtered  1 30 hz \\r\\n odors   delivered using   8 channels olfactometer  automate scientific \\r\\n continuous air flowed     bottle   mineral oil  sigma aldrich    avoid pressure changes       odors    following odors  sigma aldrich  author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint     concentration   1   vol/vol    mineral oil      ethyl acetate  betacitronellol  eugenol  limonene    pinene \\r\\n aavs   stereotaxically injected     hemispheres   piriform cortex   cfos tta transgenic mice   littermate wild type controls  mice     10 days   recover      habituated   day 11     training   testing/  field/plethysmograph boxes  40   10 min    24 hours   habituation  doxycycline   removed   replaced     normal diet  five days    mice   fear conditioned   ethyl acetate   exposed     odor  odor         neutral environment  immediately   fear conditioning   odor presentation  mice           diet containing doxycycline    fear conditioning  ethyl acetate    cs          extremity     training box      corresponding halfside     box  floor   electrified   0 6    0 6 ma    foot shock   delivered 4     cs  presentation      cs  lasted     additional 2   \\r\\n mice learned   escape     opposite       cs         cs      4 times  2 times           box    fear conditioning included presentation       reinforced conditioned stimuli  cs 1   cs 2    choice   cs    cs             presentation   pseudorandomized  training   started     presentation     cs followed     cs      cs odors       7    3 times        total presentation   3           3     left      testing    cs      4 times  2 times            cs                7            training session            odor presentations   pseudo randomized  testing   started     presentation author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint     cs  followed     cs  presentation \\r\\n clozapine   oxide  c0832  sigma aldrich      dissolved   dimethylsulfoxide  dmso  d2438  sigma  final concentration   1  vol/vol      diluted   0 9  sterile saline solution     final concentration   0 2 mg/ml  aliquots   stored         months    20°    equilibrated     temperature   injection    solution   administered intraperitoneally  3 mg/kg    injection  mice   left undisturbed   25 minutes  figure 2   4    5 minutes  figure 3   5      home cage     start     experiment \\r\\n electrophysiological data   analyzed   custom   software   python 3 0   averaged   ms excel  microsoft  usa  behavioral sessions   video recorded   analyzed using custom written matlab programs  available   request  parameters extracted   multiple cs   cs  presentation   averaged     mouse \\r\\n statistical analysis   performed   prism  graphpad    parametric tests      mann whitney test       comparison    wilcoxon matchedpairs signed rank test       comparison  values   represented   median  interquartile range  unless otherwise stated      represents     mice        recorded neurons     electrophysiology experiment  author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint       cno application    cno application        aps   dreadd negative cells  ctrl      change  significantly decreases   hm4diexpressing cells   significantly increases   hm3dq expressing cells \\r\\n       relative change         aps  /  cno   dreadd negative cells  ctrl      change  significantly decreases   hm4di expressing cells   significantly increases   hm3dq expressing cells \\r\\n data   shown   individual data    median   interquartile range      0 01  author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint      fear conditioning  mice   trained   associate ethyl acetate  cs    foot shock    cs                training box      corresponding half       box   electrified  training includes   presentation       odors       paired   foot shock  cs  eugenol   beta citronellol \\r\\n   cs      4 times  2 times            cs   times            odor presentations   pseudo randomized       testing    cs      4 times  2 times            csis              7        testing session   video recorded     position     mouse   centroid     box       axis   extracted  top \\r\\n   length     box   normalized   1      odor     randomly     left           testing box  0   defined     extremity     odor        velocity measurements    7   time window   defined         stimulus onset  bottom     experimental design  cfos tta transgenic mice  experimental     =8    wildtype mice  control     =8    injected   aav teto hm4di     hemispheres   piriform cortex  ten days    mice   habituated     training   testing boxes    doxycycline  dox  removal  mice   fear conditioned    hm4di expression  green circles    induced   fos tagged neurons  filled   blue   yellow  mice   returned   dox containing diet   avoid   expression   hm4di    days      mice        injected   cno    silences hm4diexpressing neurons  black crosses    memory retrieval   tested \\r\\n         olfactory memory retrieval   cno injection  assessed     position   mice     testing box     maximum velocity   odor exposure  memory author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint retrieval   significantly impaired   cfos tta transgenic mice     piriform neurons     active   learning   chemogenetically silenced   memory recall  experimental   \\r\\n     position   mice     testing box     function   time   cs   black    cs blue  presentation      position     testing box 7     cs    cs presentation \\r\\n     position 7     cs presentation subtracted     position 7     cs  presentation \\r\\n     maximum velocity     7   time window       cs  presentation \\r\\n     maximum velocity ratio  /  cs   left    cs    presentation \\r\\n data   shown   median   interquartile range      0 05      0 01         significant \\r\\n author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint      top  experimental design  fos tagging   olfactory fear conditioning   performed   described   figure 2     days   fear conditioning  sniffing behavior   quantified     plethysmograph  mice        injected   cno  habituated     plethysmograph   15 min      odor         8       2 5 min inter trial interval  pin  pinene  cs  beta citronellol  cs  ethyl acetate \\r\\n  bottom  representative sniff recordings     mouse   exposed   pinene     cs   ethyl acetate  data   shown   median   interquartile range      0 05      0 01         significant author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint data   shown   median   interquartile range      0 05      0 01         significant \\r\\n author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint         measure   exploratory activity    distance travelled   computed   minute 15   18  corresponding   odor exposure   test 2    artificial memory recall    distance travelled   computed     entire testing session  ethylacetate exposure   cno injection lead     decrease   exploratory behavior   mice previously fear conditioned   ethyl acetate   =6    =8 respectively \\r\\n previous exposure   ethyl acetate   foot shock     alter exploratory behavior   re exposure   ethyl acetate   re activation   fos tagged piriform ensembles   =5    =7 respectively \\r\\n     distance traveled ratio test 2/test 1  author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint \\r\\n training testing   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint baseline author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/297226 doi  biorxiv preprint  sars cov 2        2019 ncov      novel coronavirus  covs  isolated   patients   pneumonia   china 2019  sars cov 2     similar incubation period  median  3 0 days      relatively lower fatality rate   sars cov   mers cov  1        estimated     reproductive     sars cov 2           sars cov  2   \\r\\n          laboratory confirmed symptomatic     absent   apparent cough  fever   radiologic manifestations      difficult   timely   accurately       infected patients  3  4        february 28  2020  patients infected   sars cov 2     diagnosed       30 countries        78 000 confirmed     2 700 deaths associate   sars cov 2 infection   reported   china   \\r\\n   genetic information     virus   essential         classification   traceability        pathogenicity        genome level    sequence identify   sars cov 2   50    mers cov  79    sars cov  88      bat derived sars   coronaviruses  bat sl covzc45   bat sl covzxc21  collected   2018   zhoushan  china      96    bat sarsr cov ratg13  collected   2013   yunnan  china   5      genome     sars cov 2 strains   submitted online contains nearly 29 900 nucleotides  nt      predicted       14   reading frames  orfs   5′   3′      orf1ab     21 291 nt  spike     3 822 nt  orf3a  8 28 nt  envelope     228 nt  membrane     669nt  orf8  366 nt    nucleocapsid    1 260 nt   6           spike gene encoded   glycoprotein     crucial   determining host tropism   transmission capacity    highly divergent   compared       bat sarsr cov ratg13  93 1  nucleotide identity   6  7   \\r\\n      rates   nucleotide substitution   rna viruses   faster     hosts      cc   nd 4 0 international license author/funder        available       copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 02 27 969006 doi  biorxiv preprint   rapid evolution   mainly shaped   natural selection    purifying selection   8    gene mutations     nucleotide substitutions  deletions   insertions     frequently reported   comparing sars cov 2     viruses  5   6   7             investigated   mutation pattern   sars cov 2   comprehensive comparative genomic analysis     nonsynonymous/ synonymous substitution  relative synonymous codon usage  rscu    selective pressure       explore   potential   evolution   function \\r\\n   sars cov 2 reference genome wuhan hu 1  nc 045512    wiv04  mn996528    downloaded     genbank database    newly sequenced 2019 ncov genomes   downloaded     global initiative   sharing avian influenza data  gisaid  database  https //www gisaid org/  twenty   closely related coronavirus complete genome sequences     coding sequences  cds    downloaded   genbank database  supplementary   table s1  \\r\\n genome sequences   aligned using muscle v3 8 31  9  followed   manual adjustment using bioedit v7 2 5  phylogenetic analyses   complete genome   performed using maximum likelihood method     time reversible model   nucleotide substitution   gamma distributed rates   sites  gtr      raxml v8 1 21  10    support     inferred relationships   evaluated     bootstrap analysis   1000 replicates   trees   rooted     alpha coronavirus lineage     outgroup    copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 02 27 969006 doi  biorxiv preprint\\r\\n   coding sequences   translated   aligned using   mega   program  11        codon based sequence alignment         analysis  phylogenetic analyses   coding sequences   performed using mega   software    changes   amino acids   nucleotides     cds sequences   analyzed using   house perl script \\r\\n       synonymous substitutions   synonymous site  ds            synonymous substitutions     synonymous site  dn      coding region   calculated using   nei gojobori method  jukes cantor    paml package  12   \\r\\n   adaptive evolution server  http //www datamonkey org/        identify eventual sites   positive selection      purpose    following test        mixed effects model   evolution  meme    allows   distribution   dn/ds  ω    vary   site   site     branch   branch     site  13      test allowed     infer episodic   pervasive positive selection   individual sites \\r\\n   investigate   potential relative synonymous codon usage  rscu  bias     spike protein   sars cov 2     closely related coronaviruses    coding sequence  cds    spike protein     coronaviruses   calculated   codonw 1 4 4  http //codonw sourceforge net/    rscu   human genes   retrieved     codon usage database  http //www kazusa   jp/codon/    potential relationships     sequences   calculated     squared euclidean distance           besides rscu    effective     codons  enc          simple metrics   verify codon bias     explore   source   virus \\r\\n   cc   nd 4 0 international license author/funder        available       copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 02 27 969006 doi  biorxiv preprint\\r\\n statistical analyses   performed using     statistical package  version 3 \\r\\n chi squared test       compare     data sets    data   considered significantly         tailed   value       0 05 \\r\\n   newly identified sars cov 2 strain wiv04 genome sequence   closely related   bat sarsr cov ratg13   bat sl covzc45      collected   host rhinolophus affinis  5    compared   ratg13 genome    nucleotide substitutions   observed          five   inserts   deletions  indels  mutations      largest insert segment   wiv04 genome     cggcgggcacgt   sequence      located near   boundary   s1   s2 regions   spike protein  interestingly    synonymous mutation   observed near   insertion sequence  fig  1   panel    compared   bat sl covzc45 genome    insert segment   detected          synonymous mutations     observed             compared   proportion   synonymous mutations   spike gene   wiv04   ratg13   bat sl covzc45    ratio   nucleotide substitutions  263 nt    amino acid substitutions  29 aa    9 07   wiv04   ratg13      significantly       ratio  3 91  864/221    wiv04   bat sl covzc45      0 05   fig  2    moreover      observed     synonymous mutations increased     genome level   wiv04   ratg13       wiv04   bat sl covzc45 \\r\\n      detailed nucleotide changes     comparison     analyzed      found                 transition mutation   enriched     nucleotide substitutions   wiv04   ratg13  fig  1b        reported     covs lacking   3   5  exoribonuclease  exon  accumulated 15   20 fold             transition  14             rna mutagen 5 fluorouracil  5 fu  treatment     increase             transitions  consequently    increased     mutation implies   exon   sars cov 2     deactivated compared       ratg13    copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 02 27 969006 doi  biorxiv preprint furthermore    checked     ratio   synonymous substitution   missense substitution   increased   sars cov 2 strains    december 2019   february 2020    genome sequences   108 strains   sars cov 2 virus     submitted   gisaid database worldwide  compared     standard sars cov 2 strain wiv04  total 98   mutations   detected   93 nucleotide sites     sars cov 2 strains   genome sequence available   feb  25 2020  fig  1a         58     nucleotide mutations caused changes   amino acids       15 nucleotide substitutions   14 sites caused changes   7 amino acids     spike protein   observed  consequently    proportion   synonymous mutations  ~40      current reported sars cov 2 strains   similar       wiv04   bat sl covzc45  39 1      dramatically         wiv04   ratg13  90 7 \\r\\n according   random drift hypothesis  15  ，   nucleotide mutations     sars cov 2 strains   available     determined   neutral evolution    short        powerful factor   force sars cov 2   evolve       direction               strict precautions     strong factors     cause directional variation   sars cov 2     natural environment   infection treatment \\r\\n   investigate     mutation pattern   sars cov 2   ratg13   unique     coronavirus species      compared   alterations   nucleotides   amino acids   comparison     representative coronaviruses  phylogenetic analysis   sars cov 2     20 closely related coronaviruses formed     supported clades  fig  2        sars cov 2 strain wiv04    wuhan    snu01    korea patient    clustered   sars cov related strains   form clade 1  belonging   beta coronavirus lineage      compared   bat sarsr cov ratg13      293 nucleotideswere altered     29 amino acids   altered   sars cov 2    ratio         9 07       ratio detected     comparisons     human coronaviruses     similar animal coronaviruses       5 0    example    ratio   human sars cov tor2   bat sars   covs lyra11   4 81    ratio   human mers cov emc2012   bat sars   covs rsa2011   2 80    ratio   human coronavirus 229e   bat coronavirus hku2   4 55    results indicate     relative level   synonymous substitutions   human sars cov 2       animal origin  ratg13                human coronaviruses     potential animal sources \\r\\n   organisms      protein coding genes       species      frequency   codon usage  16      rscu bias   reveal   difference     host source    calculated   distance   rscu   spike genes   20 representative coronaviruses  \\r\\n   bat sarsr cov ratg13   1 17        lowest except   snu01    strain   sars cov 2  indicating     codon preference              lowest codon usage difference   2 41     detected   sars cov 2 wiv04   bat sl covzc45  fig  2      coronaviruses     beta cov lineage       relatively close \\r\\n   infer     retention   synonymous mutations   supported   hindered   natural selection      studied   synonymous substitution rate  dn    synonymous substitution rate  ds    spike gene  table 1       positive  darwinian  selection increases    negative  purifying  selection decreases   ratios     synonymous   synonymous substitution rates  dn/ds    results       dn   ds     gene   sars cov 2 wiv04 versus bat sarr cov ratg13     lowest     typical coronaviruses        sars cov tor2 versus bat sars   coronavirus wiv1       lowest      ratio   dn   ds   spike gene   compared      tested dn/ds       1  indicating       synonymous mutations   harmful    negative selection   reduce   retention speed    copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 02 27 969006 doi  biorxiv preprint      dn/ds   spike gene   sars cov 2 wiv04 versus     bat sarr cov ratg13   0 04        lowest     comparisons  reconfirming     rate   synonymous mutation   increased   wiv04   ratg13 strains  moreover  dn/ds rate   polyprotein  orf1ab    nucleocapsid     genes   similar       spike gene  table   s2  \\r\\n   spike     protein undergoes   drastic changes   virus infection    instance          cleaved   infection   cellular proteases   expose   receptors   activate viral attachment     host  17        receptor binding domain  rbd    spike protein   involved   interacting   human angiotensin converting enzyme 2  ace2  protein    rbd region         preferential targets   natural selection  18    consistent     hypothesis    results       dn   ds   rbd region   increased   comparison     spike gene region       virus pairs       study  table 1    notably    dn/ds ratio   rbd region   sars cov 2 wiv04   dramatically increased     3 fold   comparison     spike region  consequently    mutations     subjected   darwin   choice       evidence   adaptive protein evolution \\r\\n   detect positive selection     spike gene    meme analysis   performed \\r\\n significant      0 05  pervasive episodic selection   detected     sites  48th  254th  330th position using   reference sequence   wiv04      254th position     spike amino acid sequence        histidine residue instead     phenylalanine residue  whereas     330th amino acidic position   wiv04 sequence        glutamine residue instead     valine residue \\r\\n   results described   strongly supporting   action   positive selection   spike gene     recent evolution   sars cov 2   ratg13       result       cc   nd 4 0 international license author/funder        available       copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 02 27 969006 doi  biorxiv preprint addressed   careful explain      rbd region   spike gene   sars cov 2   quit divergent       ratg13  fig  s1    suggesting       originated   homologous recombination   ratg13       unknown coronavirus  19   \\r\\n   comprehensive comparative analysis   sars cov 2     coronaviruses    found   synonymous mutations   dramatically elevated   sars cov 2   ratg13         coronavirus strains      nucleotide mutations   enriched       transition \\r\\n   specific mutation pattern   caused     loss function   rna 3   5  exoribonuclease  exon  moreover      sars cov 2   supposed     originated   bat sarsr cov ratg13    increased synonymous substitution   sars cov 2   ratg13 strain suggested   sars cov 2 genome     stronger negative  purifying  selection      detected   sites   spike protein   experiencing positive selection    observations suggest   adaptive evolution   contribute     host shifts         rna mutagen       5 fu    induce     mutation pattern    mechanism     mutation pattern observed   sars cov 2   ratg13       investigated   future          amino acid changes  rscudist    rscu distance   spike protein  enc  effective     codons    ident    percent identity     protein sequence     coronaviruses    nt/aa ratio   wiv04/ratg13   statistically significant    chi squared test    0 01          strain pairs \\r\\n   cc   nd 4 0 international license author/funder        available       copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 02 27 969006 doi  biorxiv preprint      genome sequence identity     paired strains   larger   95 \\r\\n   cc   nd 4 0 international license author/funder        available       copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 02 27 969006 doi  biorxiv preprint    covid 19 outbreak     december 2019   wuhan  china  1      causative virus  sars cov 2    rapidly spread   43 countries     february 27  2020        infected patients exceeds 81 000      death toll exceeds 2 700    drugs  including lopinavir  remdesivir    chloroquine      reported     presumably effective   treating   disease  2  3   \\r\\n systemic treatment   corticosteroids   contraindicated     severe pneumonia caused   viruses     mers cov   sars cov    steroids suppress   innate immune system  resulting   increased viral replication         sars   2003   mers   2013  treatment   corticosteroids   associated   increased mortality  4  5        study    reconsider       inhaled corticosteroids        excluded     treatment   pneumonia caused   coronavirus \\r\\n steroid compounds      expected     dual functions   blocking   coronavirus replication   host inflammation    screened     chemical library    cytopathic effect caused   mers cov infection   measured   evaluate viral replication    steroid compounds  ciclesonide  mometasone furoate  mifepristone    algestone acetophenide conferred       95  cell survival rate  fig  s1  \\r\\n    concentration dependent viral growth suppression   drug cytotoxicity   assessed  ciclesonide exhibited low cytotoxicity   potent suppression   viral growth  fig    1a   cortisone   prednisolone      commonly     systemic steroid treatment  dexamethasone      strong immunosuppressant effects    fluticasone    commonly   inhaled steroid      suppress viral growth  fig  1a   \\r\\n   antiviral effects   steroids   various species   viruses   tested   quantifying propagated virus   culture medium  ciclesonide   mometasone   suppressed replication     coronaviruses  hcov 229e   sars cov      replication   rs virus   influenza virus  fig  s2     addition  ciclesonide slightly   significantly inhibited rubella virus    positive strand rna genome  replication  fig  s2   suggesting   ciclesonide interacted     replication site   positive strand rna virus intracellularly \\r\\n   identify   drug target   virus replication    conducted 11 consecutive mers cov passages     presence   40 μ   ciclesonide   40 μ   mometasone    mutant virus   developed resistance ciclesonide   generated  fig  1b        resistant virus   author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 11 987016 doi  biorxiv preprint mometasone   generated    generation sequencing identified     amino acid substitution   a25v  c19647t     reference sequence nc 019843 3    nonstructural protein 15  nsp15  coronavirus endoribonuclease  6 8      predicted mechanism   viral resistance   ciclesonide    recombinant virus carrying   a25v amino acid substitution  re nsp15 a25v    generated     parental mers cov/emc strain  re emc/mers      bac reverse genetics system  9    overcame   antiviral effect   ciclesonide  fig  1c    interestingly    mutant virus   inhibited   mometasone  suggesting     antiviral target   mometasone           ciclesonide \\r\\n      effect   ciclesonide   sars cov 2 infection   evaluated  veroe6/tmprss2 cells 10    infected   authentic sars cov 2     presence   steroids     inhibitors    6   post infection  cellular rna   isolated    real time pcr   conducted   quantify   amount   viral rna  11  ciclesonide   mometasone suppressed viral replication     similar efficacy   lopinavir  fig  2a      comparison  viral cell entry inhibitors   tested  e64d substantially reduced viral rna levels    nafamostat   camostat     modest antiviral effects  suggesting   sars cov 2 primarily utilized   cathepsin/endosomal pathway   cell entry       tmprss2/cell surface pathway   enter tmprss2 expressing cells  consistent     recent study  12  author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https  //doi org/10 1101 //doi org/10  /2020   copyright holder     preprint        peer reviewed    th   https //doi org/10 1101/2020 03 11 987016 doi  biorxiv preprint   rna copy  log 10   sars cov 2 rna   cell  6 hpi cell viability   author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed    th   https //doi org/10 1101/2020 03 11 987016 doi  biorxiv preprint  seventeen     passed     outbreak   severe acute respiratory syndrome  sars    2003            approved treatment   infections     sars coronavirus  sars cov  1       reasons     despite   devastating consequences   sars     affected patients    development     antiviral drug     virus       commercially viable   view           virus     rapidly contained       reappear   2004      result      empty handed     middle east respiratory syndrome coronavirus  mers cov    close relative   sars cov  emerged   2012  2 mers   characterized   severe respiratory disease    similar   sars      addition frequently causes renal failure 3           registered mers     low  2494     november 30  2019  www   int    threat mers cov poses   global public health         serious         sars cov      related       casefatality rate    35  compared   10    sars            mers       accumulating seven       discovery     virus  whereas   sars outbreak   essentially contained   6 months    potential   human   human transmission   mers cov     impressively demonstrated     2015 outbreak   south korea    186       traced       single infected traveller returning     middle east  4 sars   coronaviruses     circulating   bats   china   5   6   7   8          spill       human population      probably   caused   current outbreak   atypical pneumonia   wuhan      linked     seafood   animal market    rna genome  gen bank accession code  mn908947 2  http //virological org/ /initialgenome release   novel coronavirus/319    accessed   january 11  2020        betacoronavirus features   82  identity       sars cov \\r\\n   spite     considerable threat posed   sars cov   related viruses          mers cov      obvious                   warrant   commercial development     antiviral drug targeting mers   sars cov       projected steady growth         mers         account      solution             development   broadspectrum antiviral drugs     directed     major viral protease    target     shared     coronavirus genera            related form            genus enterovirus     picornavirus family            genus alphacoronavirus     human coronaviruses  hcov  nl63  ref  9    229e 10   usually cause   mild respiratory symptoms   otherwise healthy individuals          widespread   sars cov   mers cov  therapeutic intervention   alphacoronaviruses   indicated       accompanying disease     cystic fibrosis 11   leukemia  12       underlying medical conditions  13   enteroviruses include pathogens     ev d68    causative agent     2014 outbreak      summer flu         14 ev a71   coxsackievirus a16  cva16    etiological agents   hand  foot    mouth disease  hfmd  15 coxsackievirus b3  cvb3      cause myocardic inflammation  16   human rhinoviruses  hrv  notoriously     lead     common cold     capable   causing exacerbations   asthma   copd  17 infection         viruses   lead   serious outcome     ev d68   cause polio   disease  18   ev a71 infection   proceed   aseptic meningitis  encephalitis  pulmonary edema  viral myocarditis    acute flaccid paralysis  15   19   20  enteroviruses cause clinical disease     frequently   coronaviruses        antiviral drug targeting   virus families     commercially viable \\r\\n    enteroviruses         coronaviruses              single stranded rna genome   positive polarity      enteroviruses          7  9 kb  whereas coronaviruses feature   largest rna genome     date  27  34 kb    enteroviruses      naked particles  whereas coronaviruses     larger   enveloped  nevertheless    related feature shared           viruses     type   major protease  21       enteroviruses   encoded     3c region     genome  hence   protease   designated 3c pro     coronaviruses    structural protein 5  nsp5      main protease    pro   similar     enteroviral 3c pro         cysteine protease     vast majority               called  3c   protease   3cl pro       crystal structure     cov   pro   3cl pro  ref  22  revealed           domains     enzyme   resemble   chymotrypsin   fold     enteroviral 3c pro           additional   helical domain     involved     dimerization     protease  fig  1a      dimerization   essential     catalytic activity     cov   pro   whereas   enteroviral 3c pro  fig  1b  functions     monomer       enteroviral 3c pro features   classical cys   glu/asp catalytic triad  whereas   cov   pro       cys   dyad  22              common features shared       types   proteases    particular     absolute requirement   gln     p1 position     substrate   space       amino acid residues     gly  ala    ser     p1  position  encouraging     explore   coronaviral   pro     enteroviral 3c pro     common target     design   broad spectrum antiviral compounds                human protease     specificity   gln     cleavage site     substrate increases   attractivity     viral target        hope     inhibitors     developed       toxicity versus   host cell  indeed  neither   enterovirus 3c pro inhibitor rupintrivir      developed     treatment     common cold caused   hrv      peptide aldehyde inhibitor     coronavirus   pro     recently demonstrated   lead   complete recovery   cats     normally fatal infection   feline infectious peritonitis virus  fipv      toxic effects   humans   cats  respectively  23  24 figure 1  crystal structures   sars cov main protease    pro   ref  26   pdb entry 2bx4    coxsackivirus b3 3c protease  3c pro   tan et al  unpublished  pdb entry 3zyd  catalytic residues   indicated   spheres  yellow  cys  blue     red  glu       coronavirus   pro     homodimer      monomer comprising   domains       structure     monomeric cvb3 3c pro resembles   nterminal   domains     sars cov   pro   structure         scale   image       superimpostion   residues       structures involved   ligand binding  superimposition   carried     aligning   catalytic cys   pair     protease  residues     sars cov   pro   shown   carbon atoms   cyan  cvb3 3c pro residues   orange carbons     labeled     asterisk  \\r\\n     proteases targeted     study   specifically cleave   peptide bond following   p1 glutamine residue  hcov nl63   pro uniquely   accepts p1 =       nsp13/nsp14 cleavage site 31     decided       5 membered ring    lactam  derivative   glutamine  henceforth called glnlactam      p1 residue       aketoamides    scheme 1    moiety     found         mimic   glutamine   enhance   power     inhibitors       10 fold    probably   compared     flexible glutamine   chain      rigid lactam leads     reduction     loss   entropy   binding     target protease  29  32   synthetic efforts   aimed   optimizing   substituents     p1  p2    p3 positions       ketoamides \\r\\n synthesis  scheme 1  started     dianionic alkylation     boc glutamic acid dimethyl ester   bromoacetonitrile    expected    alkylation occurred     highly stereoselective manner  giving 1     exclusive product      following step    cyano     1   subjected   hydrogenation      situ cyclization     resulting intermediate afforded   lactam 2    lactam derivative 3   generated   removal     protecting     2        hand    amidation   acyl chloride     amino acid methyl ester afforded   intermediates 4      rise     acids 5 via alkaline hydrolysis    key intermediates 6   obtained via   condensation     lactam derivative 3       capped amino acids 5    ester     compounds 6     reduced     corresponding alcohol  oxidation     alcohol products 7   dess martin periodinane generated   aldehydes 8    followed   nucleophilic addition   isocyanides   rise   compounds 9   acidic conditions         hydroxyamides 10   prepared   removing   acetyl     compounds 9      final step    oxidation     exposed alcohol     compounds 10 generated   target   ketoamides 11 \\r\\n   inhibitory potencies   candidate   ketoamides   evaluated   purified recombinant sars cov   pro   hcov nl63   pro   cvb3 3c pro     ev a71 3c pro       potent compounds     tested   viral replicons     sars cov  mers cov        range   enteroviruses   cell culture based assays  tables 1  3   supplementary table 1 \\r\\n   enable   rapid   biosafe screening   antivirals   corona   enteroviruses      infectious    replication competent sars cov replicon     33     subgenomic replicons   cvb3 34   ev a71    gift      zhang  wuhan  china    easily detectable reporter activity  firefly   renilla luciferase      replicons   previously   shown   reflect viral rna synthesis   33   34   35    vitro rna transcripts     enteroviral replicons         transfection      sars cov replicon containing   cmv promoter      plasmid dna       transfection \\r\\n   initial compound     designed   synthesized   11a    carries   cinnamoyl   cap     p3 position    benzyl     p2    glutamine lactam  glnlactam    p1    benzyl   p1   table  1      compound       mediocre activities   recombinant sars cov   pro  ic50 = 1 95 µm      compounds    tables 1  3   standard deviations  cvb3 3c pro  ic50 = 6 6 µm    ev a71 3c pro  ic50 = 1 2 µm      surprisingly completely inactive  ic50   50 µm    hcov nl63   pro     values   mirrored     sars cov       enterovirus replicons  table 2      virus infected cell cultures    results obtained         mediocre  table 3    sars cov  ec50 = 5 8 µm   vero e6 cells  mers cov  ec50 = 0 0047 µm   huh7 cells  hcov 229e  ec50 = 11 8 µm   huh7 cells      host   enteroviruses  ec50 = 9 8 µm   ev a71   rd cells  ec50 = 0 48 µm   ev d68   hela rh cells  ec50 = 5 6 µm   hrv2   hela rh cells      cell types tested    compound   proved       toxic    selectivity indices  cc50/ec50  usually  10  table 3   \\r\\n crystal structures   compound 11a   complex   sars cov   pro   hcov nl63   pro     cvb3 3c pro demonstrated     aketo carbon   covalently linked     active site cys     145  144    147  resp      protease  fig  2  3a        resulting thiohemiketal         configuration     sars cov   hcov nl63   pro         configuration     cvb3 3c pro complex    reason     difference       oxygen atom     thiohemiketal accepts   hydrogen bond     catalytic his40     cvb3 protease          main chain amides     oxyanion hole       sars cov   hcov nl63 enzymes  fig  3a      insets      remarkable     succeeded   obtaining   crystal structure   compound 11a   complex     hcov nl63   pro             inhibitory effect     activity     enzyme  ic50   50 µm   fig  2c    apparently    compound   able   bind       pro     absence   peptide substrate      compete   substrate     binding site due   low affinity    similar observation               previous studies        able   determine   crystal structure     complex     inactive michaelacceptor compound sg74     ev d68 3c pro  ref  29   pdb entry 3zv9  figure 2   fit   compound 11a  pink carbon atoms      target proteases  wheat surfaces    revealed     ray crystallography     complexes     fo  fc difference density  contoured   3s    11a     substrate binding site     sars cov   pro  transparent surface  selected   chains     protease   shown   green carbon atoms       view   11a     substrate binding site     sars cov   pro   note    lid  formed   residue met49     neighbors     s2 pocket     11a     substrate binding site   hcov nl63   pro         restricted size     s2 pocket    p2 benzyl       compound   enter deeply     site  note     s2 pocket     covered      lid  centred   thr47     11a     substrate binding site     cvb3 3c pro     s2 site         covered      lid \\r\\n   crystal structures indicated     fits     p1  benzyl     11a     s1  pocket       p3 cinnamoyl cap     s3 subsite     improved    fig  3a     compounds 11b  11e   11g  11l   synthesized     attempt           none         inhibitory activity     majority     recombinant proteases  compared     parent compound  11a    supplementary results    investigate     p3 residue     inhibitor   dispensible    synthesized compound 11f      comprises p2 = boc  p1 = glnlactam    p1  = benzyl  11f   inactive     purified proteases       replicons tested        activity   hrv2   hela rh cells  ec50 = 9 0 µm    crystal structure   11f bound   hcov nl63   pro demonstrated     p2 boc   entered   s2 pocket  fig  3d     conclusion        probably       improvement    decided   maintain   original design   p1  = benzyl   p3 = cinnamoyl    focussed   improving   p2 substituen \\r\\n   crystal structures   sars cov   pro   hcov nl63   pro     cvb3 3c pro   complex   11a revealed   fundamental difference     s2 pockets     coronavirus proteases     enterovirus proteases    cavities   covered      lid      former                 latter  fig  2          sars cov   pro     lid   formed     310 helix 46  51     hcov nl63   pro     loop 43 48  residues     lid    particular met49         sars cov   pro         hydrophobic interactions     p2 substitutent     inhibitor  whereas   interaction   missing     enterovirus 3c pro      addition     lid    s2 pocket   lined        wall   main chain atoms   residues 186   188   cb atom   asp187      walls  gln189  his41           floor   met165    sars cov   pro     hcov nl63   pro     corresponding structural elements   main chain atoms   residues 187   188         cb atom   asp187    wall  pro189   his41    walls    ile165  floor  finally    cvb3 3c pro   arg39  asn69    glu71 form     wall  residues 127 132   his40 form     walls    val162 constitutes   floor \\r\\n   addition    s2 pocket       size     various proteases    sars cov enzyme features   largest s2 pocket      volume   252 å 3 embraced     residues  gln189  his41  defining     walls     pocket     ligand free enzyme    calculated   using chimera  36 followed     cvb3 3c pro s2 pocket     180 å 3  space   thr130   his40    hcov nl63   pro           s2 pocket       enzymes      free space     45 å 3   pro189   his41  according   chimera \\r\\n   agreement     observations      fit   observed     p2 benzyl     11a     s2 subsite     sars cov   pro             cvb3 3c pro  fig  3a        contrast    crystal structure     complex   11a   hcov nl63   pro         compound   inactive  demonstrates     p2 benzyl       enter   s2 pocket     enzyme       restricted size     site  fig  3b   \\r\\n      properties     target proteases   respect     s2 pocket   defined                covered     lid    hcov nl63   pro          covered    sars cov   pro                 cvb3 3c pro     comparison   crystal structures     proteases       virus genus  hcov 229e   pro   alphacoronaviruses 28  pdb entry 1p9s  hku4   pro   betacoronaviruses  ma  xiao et al  unpublished  pdb entry 2yna    ev a71 3c pro   enteroviruses 30  pdb entry 3sgk    ensured     conclusions drawn     template structures   valid     family       \\r\\n   explore   sensitivity     s2 pocket towards   polar substituent     para position     benzyl      synthesized compound 11m carrying   4 fluorobenzyl     p2    substitution abolished     activity     sars cov   pro  ic50   50 µm      compound proved inactive   hcov nl63   pro      whereas ic50 values   2 3 µm     ev a71 3c pro   8 7 µm   cvb3 3c pro          concluded     introduction     polar fluorine atom     compatible     geometry     s2 pocket   sars cov   pro   whereas   fluorine   accept   hydrogen bond   arg39   ev a71 3c pro  ref  30     probably   cvb3 3c pro     sars cov   pro        carbonyl     residues 186   188   lead     repulsion     fluorinated benzyl    author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 02 10 936898 doi  biorxiv preprint table 2     ketoamide induced inhibition   subgenomic rna synthesis using replicons     cell based assay  ec50  μm  1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60 author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 02 10 936898 doi  biorxiv preprint table 3     thiohemiketal         configuration      oxygen accepting   hydrogen bonds     oxyanion hole amides   gly143   cys145    amide oxygen accepts     bond   his41      chains   ser144   arg188     omitted   clarity       p2 benzyl substituent   11a     enter   s2 pocket     hcov nl63   pro                 plasticity     corresponding pocket   sars cov   pro  cf       benzyl   binds     pocket     view shown        probably   reason     total inactivity  ic50   50 µm    compound 11a   hcov nl63   pro       size     pocket   due     replacement     flexible gln189     sars cov   pro       rigid pro189     enzyme    stereochemistry     thiohemiketal        sidechains   ala143   gln188     omitted   clarity     binding   11a     cvb3 3c pro     stereochemistry     thiohemiketal            accepts   hydrogen bond   his41  whereas   amide keto   accepts     bonds     oxyanion hole  residues 145  147      chain   gln146     omitted   clarity       crystal structure   11f   complex   hcov nl63   pro       short  inactive  compound lacking   p3 residue     p2 boc   inserted     s2 pocket     protease    stereochemistry     thiohemiketal        sidechains   ala143   gln188     omitted   clarity       contrast   p2 = benzyl   11a    isobutyl     11n       flexible     enter     narrow s2 pocket     hcov nl63   pro     thiohemiketal         configuration      chains   ala143   gln188     omitted   clarity       spite       size    cyclopropylmethyl   chain     p2 position   11s   tightly bind     s2 subsite     sars cov   pro       pocket exhibits pronounced plasticity due     conformational flexibility   gln189      fig  4     stereochemistry     thiohemiketal          chains   ser144   arg188     omitted   clarity \\r\\n     p2 benzyl     11a   apparently       fit     s2 pocket     hcov nl63   pro     replaced     isobutyl   11n    resulted   improved activities   sars cov   pro  ic50 = 0 33 µm            activity   hcov nl63   pro  ic50 = 1 08 µm  compare     inactive 11a    ev a71 3c pro        activity decreased   ic50 = 13 8 µm      cvb3 3c pro     ic50   3 8 µm    interpretation     result         p2 isobutyl substituent   11n     interact      lid     particular  met49      sars cov   pro s2 site      unable   reach      wall      ev a71 3c pro pocket          absence      lid    generate sufficient enthalpy   binding          examples   follow     trend persists     inhibitors       p2 substituent        sars cov   pro s2 pocket     larger volume         enterovirus 3c pro     enzyme     efficiently inhibited   compounds carrying     p2 residue   makes hydrophobic interactions     lid  met49    floor  met165  residues \\r\\n   ec50   11n    10 µm     ev a71   cvb3 replicons          sars cov replicon    activity   11n   relatively weak  ec50 = 7 0 µm  table 2     agreement     replicon data  11n proved inactive   ev a71   rd cells     limited activity   hrv2   hrv14   hela rh cells  table 3        comparatively   activity  ec50 = 4 4 µm    ev d68   hela rh cells   unexpected    activity   11n   hcov 229e   huh7 cells      ec50 = 0 6 µm      mers cov   huh7 cells      excellent    ec50 = 0 0048 µm      vero cells    ec50   mers cov         9 2 µm  similarly    ec50   sars cov   vero cells   14 2 µm  table 3   \\r\\n   managed   obtain crystals   11n   complex       pro   hcov nl63   found   p2 isobutyl         embedded     s2 pocket  fig  3e              consequence       size     isobutyl   compared     benzyl            larger conformational flexibility    allows     fit     binding site \\r\\n     replaced   p2 isobutyl residue   11n     butyl   11o    activities     follows  ic50 = 8 5 µm   sars cov   pro   totally inactive  ic50   50 µm    hcov nl63   pro   ic50 = 3 2 µm   ev a71 3c pro     5 2 µm   cvb3 3c pro     decreased activity       sars cov   pro     total inactivity   hcov nl63   pro indicate       butyl chain           s2 pocket     proteases  whereas   slight improvement   ev a71 3c pro   cvb3 3c pro   probably   consequence     extra space     available       flexible substituents       lack     lid covering   enterovirus 3c pro pocket \\r\\n       butyl substituent   p2   11o   obviously          synthesized   derivative     shorter propargyl  ethinylmethyl      p2 residue  compound 11p    led     mediocre activities     tested proteases  using cyclopropyl     p2 residue  compound 11q    ic50 values               proteases tested  obviously    p2   chain requires   methylene         position       provide     flexibility     substituent     embedded     s2 pocket \\r\\n   realized     addition   size  flexibility     p2 substituent         factor influencing inhibitory activity    introduced flexibility     phenyl ring   11a   reducing      cyclohexylmethyl derivative 11r exhibited ic50 = 0 7 µm   sars cov   pro   12 3 µm   hcov nl63   pro   1 7 µm   ev a71 3c pro     0 9 µm   cvb3 3c pro        replace  1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60 author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 02 10 936898 doi  biorxiv preprint ment     phenyl       cyclohexyl   led     significant improvement     inhibitory activity     recombinant sars cov   pro       dramatic improvement       cvb3 3c pro         hcov nl63   pro       11a   completely inactive  greatly improved albeit   weak activity   observed  table 1       viral replicons  11r performed        ec50 = 0 8  0 9 µm     ev a71 replicon  0 45 µm   cvb3    1 4 µm   sars cov  table 2       virus infected cell culture assays  table 3    11r exhibited ec50 = 3 7 µm   ev a71   rd cells   ec50 = 0 48  0 7 µm   ev d68  hrv2    hrv14   hela cells    hcov 229e   huh7 cells    ec50   surprisingly low  1 8 µm  interestingly    compound proved extremely potent   mers cov   huh7 cells    ec50 = 0 0004 µm  400 picomolar      vero cells  ec50   mers cov   5 µm      ec50   sars cov   vero e6 cells   1 8  2 1 µm           activity     seen       pro inhibitor   sars cov     type   cells    therapeutic index  cc50/ec50    11r   ev d68  hrv2    hrv14    15   hela rh cells         cvb3   huh t7 cells      ~5   ev a71   rd cells \\r\\n          decided   systematically vary   size     ring system   p2      substituent     tried   cyclopropylmethyl  compound 11s        activities   sars cov   pro  ic50 = 0 24 µm    hcov nl63   pro  1 4 µm    poor values   ev a71 3c pro  ic50 = 18 5 µm    cvb3 3c pro  ic50 = 4 3 µm   table 1    11s   shown   inhibit   sars cov replicon     ec50     2 µm  whereas activity     ev a71   cvb3 replicons   poor  ec50 values   20 µm    table 2     replicon results   mirrored     antiviral activity   11s   enterovirus infected cells  table 3        weak     weak    contrast    compound inhibited hcov 229e   mers cov   huh7 cells   ec50   1 3   0 08 µm  respectively    activity     latter virus   vero cells   poor  ec50 ~11 µm          anti sars cov activity   vero e6 cells  table 3   \\r\\n     analyzed   crystal structure     complex   sars cov   pro   compound 11s  fig  3f      cyclopropylmethyl substituent   found     incorporated deeply     s2 pocket    hydrophobic interactions   met49    lid  met165    floor      cb   asp187      wall    spite       size     p2 substituent            s2 pocket   sars cov   pro   flexible     contract   enclose   p2 moiety tightly    plasticity   expressed     conformational change   residue gln189        main chain         chain    main chain conformational change   connected     flip     peptide   gln189   thr190    c1 torsion angle     gln189   chain changes   roughly antiperiplanar  ap     synclinal  sc   fig  4      conformational variability   gln189     noted        molecular dynamics simulations 26       crystal structures  37     consequence     changes    sidechain oxygen   gln189   accept   2 54 å hydrogen bond     main chain nh     p2 residue     11s complex    fig  4     affinity   11s     s2 pocket   hcov nl63   pro             ideal match   size     requiring conformational changes      enzyme       able   undergo       replacement     flexible gln189       rigid pro        hand  docking       compound     crystal structure     cvb3 3c pro revealed     cyclopropylmethyl moiety   probably unable   generate sufficient free energy   binding       missing lid       size     s2 pocket     enterovirus 3c pro   thereby explaining   poor inhibitory activity   11s     targets   table 1    experiments     viral replicons confirmed   trend      ec50 value   sars cov  6 8 µm    surprisingly    table 2     huh7 cells infected   mers cov    compound exhibited ec50 = 0 1 µm    9 8 µm   vero cells  whereas ec50   7 0 µm   sars cov   vero e6 cells    compound     inactive   ev a71   rd cells   inhibited   replication       hrv subtypes tested    hela rh cells    ec50 values   ~4 µm    cc50   11t   hela cells   65 µm         therapeutic index       15  table 3   \\r\\n obviously    substituent       bit         enterovirus proteases          step    tested p2 = cyclopentylmethyl  compound 11u                compound   acceptable ic50 values     tested enzymes  1 3 µm   sars cov   pro   5 4 µm   hcov nl63   pro   4 7 µm   ev a71 3c pro     1 9 µm   cvb3 3c pro   table 1     activity     replicons     3 6   4 9 µm   table 2     huh7 cells infected   hcov 229e   mers cov  11u   ec50 = 2 5   0 03 µm  11 1 µm   mers cov   vero cells    ec50   4 9 µm   sars cov   vero e6 cells  table 3   \\r\\n 11u appeared         compromise compound            individual viral enzymes    following compounds proved superior  p2 = cyclopropylmethyl  compound 11s    sars cov   pro   p2 = isobutyl  compound 11n    p2 = cyclopropylmethyl  11s    hcov nl63   pro   p2 = benzyl  11a    cyclohexylmethyl  11r    ev a71 3c pro     11r   cvb3 3c pro       words    nearly equipotent 11u   indeed   compromise       view     surprisingly   antiviral activity   11r   hcov 229e   huh7 cells    relaxed   condition     universal inhibitor       activity     recombinant hcov nl63   pro     selected 11r  p2 = cyclohexylmethyl      lead compound     development    compound exhibited submicromolar ic50 values   cvb3 3c pro   sars cov   pro     ic50 = 1 7 µm   ev a71 3c pro  table 1          similarly low ec50 values     replicons     viruses  table 2     huh7 cells infected   mers cov    performance     compound   excellent    ec50 = 0 0004 µm        hcov 229e   huh7 cells   sars cov   vero e6 cells  ec50 values   1 8   2 1 µm  respectively    observed  table 3        enterovirus infected cell culture    compound performed      ec50 values   0 7 µm       hrv2  hrv14    ev d68   hela  rh  cells   selectivity values  15      concern     activity     compound   ev a71   rd cells        ec50 value   3 7 µm  resulting     low   therapeutic index        hand    weak toxicity   detected   11r   vero   huh t7 cells  preliminary pharmacokinetics tests     compound   mice     indicate   toxicity        published elsewhere \\r\\n   describe     structure based design    synthesis      assessment   capped dipeptide   ketoamides   target   main protease   alpha   betacoronaviruses         3c protease   enteroviruses    crystallographic analyses     total   six inhibitor complexes       proteases     study    found     ketoamide warhead  co co nh      sterically   versatile     warheads     michael acceptors  ch=ch co    aldehydes  ch=       features   acceptors   hydrogen bonds     protein  namely     keto oxygen     amide oxygen  whereas     warheads         acceptor      various complexes    hydroxy      oxyanion      thiohemiketal     formed     nucleophilic attack     activesite cysteine residue onto     keto carbon    accept       hydrogen bonds     main chain amides     oxyanion hole    addition    amide oxygen     inhibitor accepts   hydrogen bond     catalytic     chain  alternatively    thiohemiketal   interact     catalytic   residue     amide oxygen     main chain amides     oxyanion hole  depending     exact interaction    stereochemistry     thiohemiketal   atom            previously observed   similar difference       aldehyde inhibitors      single interaction      oxyanion     thiohemiacetal    accept   hydrogen bond       oxyanion hole       catalytic     chain  37 resulting     stereochemistry     thiohemiacetal carbon    aketoamides   aldehydes react reversibly     catalytic nucleophile   proteases  whereas michael acceptors form irreversible adducts \\r\\n   addition     matching     bonding donor/acceptor properties     catalytic center   offering   hydrogen bond acceptors instead        ketoamides       advantage   aldehydes       unsaturated esters  michael acceptors        allow easy extension     inhibitors   probe   primed specificity subsites beyond s1            rarely   explored       ref  38       calpain \\r\\n     prominent   ketoamide drugs   probably telaprivir   boceprivir  peptidomimetic inhibitors     hcv ns3/4a protease  39  40     helped revolutionize   treatment   chronic hcv infections    viral cysteine proteases    ketoamides     occasionally   described   inhibitors     systematic studies     carried   \\r\\n       capped dipeptidyl   ketoamides     described   inhibitors     norovirus 3c   protease  41     optimized   respect     p1  substituent  whereas p2   isobutyl         occasionally benzyl    former displayed ic50 values       magnitude lower     latter  indicating     s2 pocket     norovirus 3cl protease   fairly            include   norovirus 3cl pro     study  expanding   target range     inhibitors   norovirus   probably   realistic undertaking      study   underway  zeng et al  42 published   series     ketoamides   inhibitors     ev a71 3c pro     authors mainly studied   structure activity relationships     p1  residue   found   alkyl substituents     superior   larger ones  interestingly      reported     six membered   lactam     p1 position led   2  3 times   activities  compared     fivemembered   lactam        time  kim et al  43 described   series   five   ketoamides   p1  = cyclopropyl     submicromolar activity   ev d68     hrv strains \\r\\n occasionally  individual   ketoamides     reported     literature   inhibitors       enterovirus 3c protease     coronavirus main protease    single capped dipeptidyl   ketoamide  cbz leu glnlactam co co nh ipr    described   inhibited   recombinant transmissible gastroenteritis virus  tgev    sars cov   pro         human rhinovirus   poliovirus 3c pro         digit micromolar range  44 coded gc 375    compound   poor activity   cell culture   ev a71    ec50 = 15 2 µm  probably   p2   isobutyl        shown      isobutyl   chain     p2 position     inhibitors         completely fill   s2 pocket     ev a71 3c pro     cvb3 3c pro  \\r\\n     series   aldehydes  prior et al  45 described   capped tripeptidyl   ketoamide cbz 1 naphthylalanine leu glnlactam co co nh ipr      ic50 values     3 digit nanomolar range   hrv 3c pro   sars cov   pro         ec50 values   0 03 µm   hrv18   0 5 µm   hcov 229e   cell culture    optimization     compound   performed     toxicity data     reported \\r\\n   compounds   warheads       ketoamides    vitro activity     corona   enteroviruses     occasionally   reported  lee et al  46 described   peptidyl michael acceptors   displayed inhibitory activity       pro     sars cov   hcov 229e           3c pro   cvb3    inhibitors     ic50 10  20 times       cvb3 enzyme  compared   sars cov   pro   p2   invariably isobutyl  leucine      compounds  suggesting     improvement       \\r\\n   addition   michael acceptors  peptide aldehydes           explore   inhibition   coronavirus   pro         enterovirus 3c pro    kim et al  44 reported   dipeptidyl aldehyde     bisulfite adduct        exhibited   inhibitory activities     isolated 3c proteases   human rhinovirus   poliovirus           3c   proteases         coronaviruses    antiviral activities   cell culture   ev a71   poor  ec50  10 µm      probably due   p2   isobutyl  leucine \\r\\n     series   compounds      p1 = glnlactam    lactam  throughout      substituent   proven       excellent surrogate   glutamine  29  32         efforts   optimize   p1  residue     compounds           cap  p3    mainly focussed   optimization     p2 substituent    nearly   studies aiming   discovering peptidomimetic inhibitors   coronavirus   pro    p2   invariably isobutyl  leucine      residue             efforts   design compounds     inhibit enterovirus 3c pro               crystal structures       lead compound  11a  cinnamoyl phe glnlactam co co nh bz    complex       pro     hcov nl63    representative     alphacoronavirus proteases    sars cov  beta cov          3c pro   coxsackievirus b3  enterovirus proteases    found     s2 pocket   fundamentally   shapes     enzymes      sars cov   pro     s2 subsite     deep hydrophobic pocket     truly   dimensional   shape     walls      groove   formed     polypeptide main chain   residues 186  188             chains   his41      catalytic dyad    gln189  whereas    floor    formed   met165      lid    residues 45  51    particular met49      methionines provide   interaction       p2 substituents   inhibitors      interactions     hydrophobic   character      previously described   surprising observation     carboxylate     aspartic residue   p2     polar interactions     sulfur atoms     methionines  37     pocket offers     opportunities   interaction   features   pronounced plasticity  p2 substituents     isobutyl    leu            fill   pocket entirely      generate sufficient binding enthalpy  accordingly    s2 pocket   sars cov   pro       tolerant       enzymes investigated      terms   versatility     p2 substituents accepted \\r\\n     s2 pocket     hcov nl63   pro   gln189   replaced   proline     change   accompanied     significant loss   flexibility  whereas     chain   gln189   sars cov   pro   found   accommodate   conformation according     steric requirements     p2 substituent    proline     flexible  leading         space     entrance     pocket      consequence    p2benzyl substituent   hindered   penetrating deeply     pocket  whereas         flexible isobutyl     p2 leu     \\r\\n finally      3c pro     ev a71   cvb3    s2 pocket lacks   lid                        consequence    offers   interaction     p2 substituents   inhibitors        substituents   reach      wall      pocket  formed   arg39  asn69  glu71        create sufficient binding energy  hence    aromatic substituents     benzyl   favored     enterovirus 3c pro   \\r\\n     introduced   fluoro substituent     para position     p2 benzyl       lead compound  11a    observed   activity     enterovirus 3c pro     complete inactivity     coronavirus   pro      table 1   compound 11m      easily explained     basis     crystal structures      enterovirus 3c pro      fluorine   accept   hydrogen bond   arg39  ref  30    whereas     coronavirus   pro          electrostatic repulsion     main chain carbonyls   residues 186   188    agreement      rupintrivir      p2 =   fluorobenzyl        inhibitor     enteroviral 3c pro    46         coronaviral main proteases      predicted earlier  28     structure based inhibitor optimization study    achieved major improvements     original lead compound  11a    systematically varying   size     flexibility     p2 substituent    compound           compromise       requirements     s2 pockets  sars cov   pro       covered  hcov nl63   pro       covered    cvb3 3c pro            11u  p2 = cyclopentylmethyl      satisfactory broad spectrum activity     proteases tested       regard     antiviral activities   cell cultures      inferior   11r  p2 = cyclohexylmethyl    latter compound exhibits     inhibitory activity     sars cov   pro         enterovirus 3c pro       performance     sars cov   enterovirus replicons   convincing        low micromolar range  ev a71  cvb3    data     antiviral activity   cell culture   11r correlate       inhibitory power     compound     recombinant proteases           replicon based assays        true         surprisingly     cellulo activity   11r   hcov 229e   huh7 cells       correlation         hold   llc mk2   caco2 cells    tested   antiviral activity         compounds   hcov nl63       cell types   found           low   submicromolar ec50 values     virus   llc mk2 cells       inactive   caco2 cells    shown  furthermore  11r       compounds     synthesized   inactive  ec50   87 µm    cvb3   vero cells    shown    exhibit     excellent activities       virus   huh t7 cells      previously observed similar poor antiviral activities   vero cells       aketoamides        michael acceptors  zhu et al  unpublished        similar cell type dependence   seen     antiviral activity   11r   mers cov   sars cov  whereas   inhibitor exhibits excellent activity   mers cov   huh7 cells     host cells  400 pm    inhibitory activity   weaker     factor       12 500   vero cells      ec50 = 5 µm        hand  11r exhibits excellent anti mers cov activity   human calu3 lung cells           primary target cells     compound       act     therapeutic setting     kupke  personal communication      tested antiviral activity   sars cov exclusively   vero cells    ec50 values determined     compounds     virus         digit micromolar range              compound 11r   ec50 = 2 1 µm  interestingly    relatively weaker activity      inactivity      inhibitors   rna viruses   vero cells   observed independently     virology laboratories   leuven     leiden        unlikely     lack   activity   vero cells   related         experimental set      preliminary experiments    replaced   p3 cinnamoyl     11r     fluorophor coumaryl   found   fluorescence microscopy       inhibitor appeared   accumulate   huh7 cells compared   vero cells              irina majoul  unpublished \\r\\n regardless     cell system       suitable       testing   peptidomimetic antiviral compounds      plan   test 11r     animal models   mers     coxsackievirusinduced pancreatitis    parallel    aim   refine   experiments   quantify   accumulation   peptidomimetic protease inhibitors     host cell types      hope       explanation     observed cell type dependencies \\r\\n     demonstrates   power   structure based approaches     design   broad spectrum antiviral compounds   roughly equipotent activity   coronaviruses   enteroviruses    observed     correlation     inhibitory activity     designed compounds     isolated proteases    viral replicons      virus infected huh7 cells        compounds  11r  exhibits excellent anti mers cov activity   virus infected huh7 cells          similarity     main proteases   sars cov     novel betacov/wuhan/2019    expect 11r   exhibit   antiviral activity       coronavirus     \\r\\n crystallization    recombinant production   purification   sars cov   pro   authentic       termini   described   detail previously  48  49 using   amicon ym10 membrane  emd millipore    purified sars cov   pro   concentrated   21 mg·ml  1   buffer    20 mm tris hcl  150 mm nacl  1 mm dtt  1 mm edta  ph 7 5  crystallization   performed   equilibrating 1 µl protein  mixed   1 µl precipitant solution    500 µl reservoir containing 6  8  polyethylene glycol  peg  6 000  0 1   mes  ph 6 0    20℃ using   vapor diffusion sitting drop method  compounds 11a   11s   dissolved   100  dmso   50 mm   200 mm stock concentrations  respectively    crystal     free enzyme   soaked   cryo protectant buffer containing 20  mpd  6  peg 6 000  0 1   mes  7 5 mm 11a  ph 6 0    2     20℃    set   free enzyme crystals   soaked     cryoprotectant buffer   6  peg 6 000  5  mpd  0 1   mes  15  glycerol  10 mm 11s  ph 6 0    2    subsequently  crystals   fished   flash cooled   liquid nitrogen prior   data collection \\r\\n crystals   hcov nl63   pro   11a   obtained using cocrystallization    concentrated hcov nl63   pro  45 mg·ml  1     incubated   5 mm 11a   4     20℃  followed   setting   crystallization using   vapor diffusion sitting drop method   20℃   equilibration   1 µl protein  mixed   1 µl mother liquor    500 µl reservoir composed   0 1   lithium sulfate monohydrate  0 1   sodium citrate tribasic dihydrate  25  peg 1 000  ph 6 0    crystals   protected     cryo buffer containing 0 1   lithium sulfate monohydrate  0 1   sodium citrate tribasic dihydrate  25  peg 1 000  15  glycerol  2 mm 11a  ph 6 0   flash cooled   liquid nitrogen \\r\\n crystals   hcov nl63   pro   11n   11f   generated   using   soaking method    free enzyme crystals   soaked   cryo protectant buffer containing 0 1   lithium sulfate monohydrate  0 1   sodium citrate tribasic dihydrate  25  peg 1 000  15  glycerol  5 mm 11n    11f  ph 6 0  subsequently    soaked crystals   flash cooled   liquid nitrogen \\r\\n freshly prepared cvb3 3c pro     concentration   21 8 mg·ml  1   incubated   5 mm 11a pre dissolved   100  dmso     tempature   1      white precipitate appeared     mixture  afterwards    sample   centrifuged   13 000       20 min   4 ℃    supernatant   subjected   crystallization trials using   following  commercially available kits  sigma tm  sigma aldrich  index tm     peg rx tm  hampton research  single rod   crystals   detected       index tm screen      condition   0 1   mgcl2 hexahydrate  0 1   bis tris  25  peg 3 350  ph 5 5        sigma tm screen   0 2   li2so4  0 1   tris hcl  30  peg 4 000  ph 8 5  crystal optimization   performed   using   vapor diffusion sitting drop method    1 μl cvb3 3c pro  inhibitor complex mixed   1 μl precipitant solution    equilibration   500 μl reservoir containing 0 1   tris hcl  0 2   mgcl2  ph 8 5    peg 3 350 varied   22    27    optimization screen     performed       reservoir  0 1   tris hcl  0 2   mgcl2  ph range   7 5   8 5    peg 4 000 varied   24    34  crystals   fished     drops   protected   cryo protectant solution consisting     mother liquor   10  glycerol  subsequently    crystals   flash cooled   liquid nitrogen \\r\\n diffraction data collection  structure elucidation   refinement  diffraction data     crystal     sars cov   pro   complex   11a   collected   100     synchrotron beamline pxi x06sa  psi  villigen  switzerland  using   pilatus 6m detector  dectris    diffraction data set     sars cov   pro crystal   compound 11s   collected   100     beamline p11   petra iii  desy  hamburg  germany  using     type   detector    diffraction data sets   hcov nl63   pro complex structures       complex   cvb3 3c pro   11a   collected   synchrotron beamline bl14 2   bessy  berlin  germany  using   mx225 ccd detector  rayonics    data sets   processed     program xdsapp   scaled   scala     ccp4 suite   50   51   52    structure   sars cov   pro   11a   determined   molecular replacement     structure     complex   sars cov   pro   sg85  pdb entry 3tnt  zhu et al  unpublished    search model  employing   molrep program        ccp4 suite  52  53   complex structures   hcov nl63   pro   11a  11f    11n     determined   molrep  using     search model   structure     free enzyme determined      lz et al  unpublished      complex structure   cvb3 3c pro   11a   determined based     search model     free enzyme structure  pdb entry 3zyd  tan et al  unpublished    geometric restraints     compounds 11a  11f  11n    11s   generated   using jligand 52  54   built   fo fc difference density using   coot software  55 refinement     structures   performed   refmac version 5 8 0131  ref  52  56  57   \\r\\n   buffer containing 20 mm tris hcl  100 mm nacl  1 mm edta  1 mm dtt  ph 7 3            enzymatic assays    substrates     cleavage sites     pro   3c pro   respectively  indicated     arrow  ↓  dabcyl ktsavlq¯sgfrkm   edans nh2   dabcyl kealfq¯gppqf   edans nh2  95  purity  biosyntan    employed     fluorescence resonance energy transfer  fret based cleavage assay  using   96   microtiter plate    dequenching     edans fluorescence due     cleavage     substrate   catalyzed     proteases   monitored   460 nm   excitation   360 nm  using   flx800 fluorescence spectrophotometer  biotek  curves   relative fluorescence units  rfu    substrate concentration   linear     substrates     beyond 50 µm  indicating   minimal influence     inner filter effect  stock solutions     compounds   prepared   dissolving     100  dmso    uv absorption   11a   found     negligible     = 360 nm        interference     fret signal     inner filter effect       expected      determination     ic50    proteases     specified final concentration  0 5 µm sars cov   hcov nl63   pro   2 µm cvb3 3c pro   3 µm ev a71 3c pro     separately incubated     inhibitor   various concentrations  0   100 μm    reaction buffer   37℃   10 min  afterwards    reaction   initiated   adding fret peptide substrate   20 μm final concentration  final volume  50 μl    ic50 value   determined   using   graphpad prism 6 0 software  graphpad  measurements   enzymatic activity   performed   triplicate           mean ± standard deviations  sd \\r\\n assessment   inhibitory activity     ketoamides using viral replicons   virus infected cells cells   viruses  hepatocellular carcinoma cells  huh7  ref  58      derivative constitutively expressing t7 rna polymerase  huh t7  ref  59    grown   dulbecco   modified minimal essential medium  dmem  supplemented   2 mm glutamine  100  ·ml  1 penicillin  100 µg·ml  1 streptomycin sulfate    fetal calf serum  10    growth medium   2    maintenance medium  huh t7 cells   additionally supplemented   geneticin    418 sulfate  400 µg·ml  1   huh t7 cells         enteroviral replicons         infection experiments   cvb3 strain nancy \\r\\n   enterovirus  except cvb3  infection experiments  human rhabdomyosarcoma cells  rd    ev a71  brcr strain    hela rh cells    ev d68   human rhinoviruses    grown   mem rega 3 medium supplemented   1  sodium bicarbonate  1  lglutamine    fetal calf serum  10    growth medium   2    maintenance medium    hcov 229e      gift   volker thiel  bern  switzerland  culture   infection experiments   carried     described  60   mers cov   sars cov infection experiments  vero  vero e6    huh7 cells   cultured   described previously  61  62 infection   vero   huh7 cells   mers cov  strain emc/2012    sars cov infection   vero e6 cells  strain frankfurt 1    low multiplicity   infection  moi        described    61  63       live mers cov   sars cov   performed inside biosafety cabinets   biosafety level 3 facilities   leiden university medical center    netherlands  viral replicons    dna launched sars cov replicon harbouring renilla luciferase   reporter directly downstream     sars cov replicase polyprotein coding sequence  pp1a  pp1ab  urbani strain  acc  ay278741      context     bacterial artificial chromosome  bac      control     cmv promoter      described previously  pbac rep rluc  33 apart     replicase polyprotein    replicon encodes   following features    5   3   translated regions  ntr    ribozyme  rz    bovine growth hormone sequence    structural protein   \\r\\n subgenomic replicons   cvb3  pt7 cvb3 fluc 34     ev a71  pt7 ev71 rluc  harbouring t7 controlled complete viral genomes        p1 capsid coding sequence   replaced     firefly  photinus pyralis    renilla  renilla renifor  luciferase gene    generously provided      van kuppeveld      zhang  respectively    prepare cvb3   ev a71 replicon rna transcripts  plasmid dnas   linearized   digestion   sali   hindiii    england biolabs  respectively  copy rna transcripts   synthesized   vitro using linearized dna templates  t7 rna polymerase      t7 ribomax™   scale rna production system  promega  according     manufacturer   recommendations \\r\\n transfection  huh t7 cells grown   12   plates     confluency   80   90   2  3   10 5 cells/     washed   1 ml optimem  invitrogen    transfected   0 25 µg     replication competent replicon   lipofectamin2000     tremegene9   300 µl optimem  final volume    recommended     manufacturer  invitrogen   roche  respectively    transfection mixtures   incubated   37°    4   5    lipofectamin2000    overnight    tremegene9  prior     replaced   growth medium containing   compound   investigation    rna launched transfection   enteroviral replicons  dmrie         transfection reagent according     manufacturer   recommendations  invitrogen    experiments       triplicate   quadruplicate     results       mean values ± sd \\r\\n testing   inhibitory activity   candidate compounds  initially    performed   quick assessment     inhibitory activity     candidate compounds towards   enteroviral   coronaviral replicons     concentration   40 µm   huh t7 cells  compounds     relatively powerful     toxic     concentration    assayed     dose dependent manner   estimate   half maximal effective concentration  ec50          cytotoxicity  cc50    described  29   brief    concentrations     ketoamides  40 µm   screening experiments   increasing concentration  0  1 25  2 5  5  10  20  40 µm    determining   ec50    added   growth medium   replicon transfected huh t7 cells  twenty   hours      cells   washed   1 ml phosphate buffered saline  pbs   optimem  invitrogen    lysed   0 15 ml passive lysis buffer  promega      temperature  rt    10 min    freezing  80        thawing  rt    cell debris   removed   centrifugation  16 000      1 min      supernatant  10   20 µl    assayed   firefly   renilla luciferase activity  promega   biotrend chemikalien  using   anthos lucy 3 luminescence plate reader  anthos microsystem \\r\\n   antiviral activity     compounds   evaluated     cytopathic effect  cpe  readout assay using mts  3 4 5 dimethylthiazol 2 yl 5 3carboxymethoxyphenol 2 4 sulfophenyl 2h tetrazolium  innersalt based assay  briefly  24   prior   infection  cells   seeded   96   plates     density   2 5   10 4  rd cells      1 7   10 4  hela rh        medium supplemented   2  fcs    hrv2   hrv14 infection    medium contained 30 mm mgcl2      day  serial dilutions     compounds   virus inoculum   added    read     performed 3 days post infection   follows    medium   removed   100 μl   5  mts   phenol red free mem   added        plates   incubated   1     37°       optical density   498 nm  od498          measured   microtiter plate reader  saffire 2   tecan    od values   converted   percentage   controls     ec50   calculated   logarithmic interpolation     concentration   compound   results     50  protective effect   virus induced cpe      condition  cell morphology     evaluated microscopically \\r\\n assays   mers cov   sars cov   performed   previously described  61  63   brief  huh7  vero    vero e6 cells   seeded   96   plates     density   1 × 10 4  huh7   vero e6    2 × 10 4 cells  vero         overnight growth  cells   treated     indicated compound concentrations   dmso  solvent control    infected     moi   0 005  final volume 150 µl/    eagle   minimal essential medium  emem  containing 2  fcs  2 mm   glutamine    antibiotics  huh7 cells   incubated     days   vero/veroe6 cells     days    differences   cell viability caused   virus induced cpe     compound specific   effects   analyzed using   celltiter 96 aqueous   radioactive cell proliferation assay  promega  according     manufacturer   instructions  absorbance   490 nm  a490    measured using   berthold mithras lb 940 96   plate reader  berthold  cytotoxic effects caused   compound treatment     monitored   parallel plates containing mock infected cells  hcov 229e rev  5′ ggtcgtttagttgagaaaagt  3′    229e zna probe  5 6 fam aga  pdc tt pdu   pdu gt pdc ta pdc   zna 3 bhq 1  3   metabion  standard curves   prepared using serial dilutions   rna isolated   virus stock  data   analysed using graphpad prism 5 0  ec50 values   calculated based     4parameter logistic statistics equation    parallel     qpcr assays   inhibitors  cell viability assays   performed using alamar blue™ cell viability reagent  thermofisher  according     manufacturer   instruction  cc50 values   calculated using inhibitor versus normalized response statistics equation   including proper controls    inhibitor   1  triton   100 treated cells \\r\\n determination     cell toxicity   candidate compounds    celltiter 96aqueous   solution cell proliferation assay  mts test  promega    celltiter glo assay kit  promega      destructive cytotoxicity bio assay  toxilight  measuring   release   adenylate kinase   damaged cells  lonza rockland      alamarblue™ cell viability reagent  thermofisher        determine   cytotoxic effect   compounds towards host cells according     manufacturers  recommendations  29  65 chemical synthesis     ketoamides   procedure  reagents   purchased   commercial sources       purification  hsgf 254  0 15  0 2 mm thickness        analytical thin layer chromatography  tlc    products   characterized     nmr   ms spectra  1   nmr spectra   recorded   300 mhz  400 mhz    500 mhz instruments  chemical shifts   reported       million  ppm  δ    field   tetramethylsilane  proton coupling patterns   described   singlet     doublet     triplet     quartet     multiplet       broad  br  mass spectra   recorded using   bruker esi ion trap hct ultra  hplc spectra   recorded   lc20a   lc10a  shimadzu corporation    shim pack gist c18  5 µm  4 6x150mm      solvent systems  methanol/water  methanol/ 0 1  hcooh   water   methanol/0 1  ammonia   water  purity   determined   reversed phase hplc     ≥95      compounds tested biologically \\r\\n synthesis    2s 4r dimethyl 2 tert butoxycarbonylamino 4 cyanomethyl  pentanedioate  1 \\r\\n     solution     boc   glutamic acid dimethyl ester  6 0    21 8 mmol    thf  60 ml    added dropwise   solution   lithium bis trimethylsilyl amide  lhmds    thf  47 ml  1      −78°    nitrogen    resulting dark mixture   stirred   −78°   meanwhile  bromoacetonitrile  1 62 ml  23 3 mmol    added dropwise     dianion solution     period   1     keeping   temperature   −70°     reaction mixture   stirred   −78°    additional 2        consumption     reactant   confirmed   tlc analysis    reaction   quenched   methanol  3 ml    acetic acid  3 ml    precooled thf  20 ml    added \\r\\n   stirring   30 min    cooling bath   removed    reaction mixture   allowed   warm       temperature     poured   brine  40 ml    organic layer   concentrated   purified   flash column chromatography  petroleum ether/ethyl acetate = 4/1      product 1  4 92    72    colorless oil  1   nmr  cdcl3  400 mhz  δ 5 23  1h       = 9 0 hz  4 43 4 36  1h     3 77 1h     3 76  1h     2 89 2 69  3h     2 20 2 14  2h     1 45  9h     esi ms   /   315            \\r\\n synthesis      methyl 2 tert butoxycarbonylamino 3   2oxopyrrolidin 3 yl propanoate  2   \\r\\n     hydrogenation flask   placed compound 1  4 0    12 7 mmol  5 ml   chloroform   60 ml   methanol     addition   pto2    resulting mixture   stirred   hydrogen   20       12        mixture   filtered   celite   remove   catalyst  naoac  6 77    25 5 mmol    added     filtrate     resulting mixture   stirred   60       12      reaction   quenched   water  30 ml    suspension   extracted   ethyl acetate    organic layers   combined  dried  mgso4    filtered    light brown filtrate   concentrated   purified   silica gel column chromatography  petroleum ether/ethyl acetate = 4/1        product 2  2 20    61    white solid  1   nmr  cdcl3  δ 6 02  1h  br  5  synthesis      methyl 2 amino 3   2 oxopyrrolidin 3yl propanoate  3 \\r\\n compound 2  1 0    3 5 mmol    dissolved   10 ml dichloromethane  dcm    10 ml trifluoroacetic acid  tfa    added    reaction mixture   stirred   20       0 5      concentrated   vacuo       colorless oil              following step   purification \\r\\n esi ms   /   187            \\r\\n synthesis   methyl   substituted amino acid esters 4   procedure    methyl amino acid ester hydrochloride  6 0 mmol    dissolved   20 ml ch2cl2      acyl chloride  6 0 mmol  triethylamine  1 69 ml  12 0 mmol    added      reaction   stirred   2     20        reaction mixture   diluted   20 ml ch2cl2  washed   50 ml   saturated brine  2 × 25 ml    dried   na2so4    solvent   evaporated     product 4   obtained   white solid  70 95  yield                step     purification \\r\\n    methyl 2 cinnamamido 3 phenylpropanoate  4a    methyl   phenylalaninate hydrochloride  1 30    6 0 mmol    dissolved   20 ml ch2cl2      cinnamoyl chloride  1 00    6 0 mmol  triethylamine  1 69 ml  12 0 mmol    added      reaction   stirred   2       temperature    reaction mixture   diluted   20 ml ch2cl2  washed   50 ml   saturated brine  2×25 ml    dried   na2so4    solvent   evaporated     product 4a   obtained   white solid  1 75    95                step     purification \\r\\n synthesis     substituted amino acids 5    procedure  1   naoh  5 ml    added     solution   compound 4  3 0 mmol    methanol  5 ml    reaction   stirred   20 min   20        1   hcl   added     reaction solution   ph = 1      reaction mixture   extracted   100 ml   ch2cl2  2 × 50 ml      organic layer   washed   50 ml   brine   dried   na2so4    solvent   evaporated     crude material purified   silica  eluted   mixtures   ch2cl2/meoh  20/1    afford   product 5  90 96  yield      white solid \\r\\n synthesis   compounds 6    procedure  compound 5  2 7 mmol    dissolved   10 ml   dry ch2cl2      solution  1 5 equiv  1 54      1 bis dimethylamino methylene 1h 1 2 3triazolo 4 5   pyridinium 3 oxide hexafluorophosphate  hatu    added      reaction   stirred   0 5     20        compound 3  500 mg  2 7 mmol    tea  0 70 ml  5 42 mmol    added     reaction    reaction   stirred     6      reaction mixture   poured   10 ml water    aqueous solution   extracted   50 ml   ch2cl2  2 × 25 ml    washed   50 ml   saturated brine  2 × 25 ml    dried   na2so4    solvent   evaporated     crude material purified   silica  eluted     mixture   ch2cl2/meoh  40/1        product 6  62 84  yield \\r\\n synthesis   alcohols 7    procedure  compound 6  1 1 mmol    dissolved   methanol  40 ml    nabh4  0 34    8 8 mmol    added   ambient conditions    reaction mixture   stirred   20       2        reaction   quenched   water  30 ml    suspension   extracted   ethyl acetate    organic layers   combined  dried    filtered    filtrate   evaporated   dryness               step     purification  46 85  yield \\r\\n synthesis   aldehydes 8    procedure  compound 7  0 75 mmol    dissolved   ch2cl2    dess martin periodinane  337 mg  0 79 mmol    nahco3  66 mg  0 79 mmol    added    resulting mixture   stirred   20       1      mixture   concentrated   purified   column chromatography   silica gel  ch2cl2/meoh = 20/1        product 8   white solid  88 95  yield \\r\\n synthesis   compounds 9    procedure  compound 8  0 40 mmol    dissolved   ch2cl2      acetic acid  0 028    0 47 mmol    isocyanide  0 43 mmol    added successively     solution    reaction   stirred   20       24        solvent   evaporated     crude material purified   silica  eluted     mixture   ch2cl2/meoh  20/1        product 9  46 84 \\r\\n synthesis   α hydroxyamides 10    procedure  1   naoh  0 5 ml    added     solution   compound 9  0 164 mmol    methanol  5 ml    reaction   stirred   20       0 5       consumption   compound 9   confirmed   tlc analysis     1   hcl   added     reaction solution   ph = 7  following      solvent   evaporated   generate   product 10   white solid          directly       step \\r\\n synthesis   α ketoamides 11    procedure  compound 10   dissolved   ch2cl2    dess martin periodinane  74 mg  0 176 mmol    nahco3  30 mg  0 176 mmol    added    resulting mixture   stirred   20       1      mixture   concentrated   purified   column chromatography   silica gel  ch2cl2/meoh = 20/1          ketoamides 11   light yellow solid  52 79      steps  \\r\\n detailed results     variation     p1    p3 substituents synthesis     ketoamides 11b  11e   11g  11l supplemental table 1   inhibitory activities  ic50  μm    α ketoamides   p1    p3 modifications   viral proteases supplemental table 2   crystallographic data   complexes   viral proteases   α ketoamides molecular formula strings   biological data  csv \\r\\n authors   release   atomic coordinates   experimental data   article publication  atomic coordinates include sars cov   pro   complex   compounds 11a  5n19  11s  5n5o  hcov nl63   pro   complex   11a  6fv2  11n  6fv1  11f  5nh0    cvb3 3c pro   complex    5nfs  1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60 author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 02 10 936898 doi  biorxiv preprint  1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60 author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 02 10 936898 doi  biorxiv preprint  severe acute respiratory syndrome coronavirus 2  sars cov 2    2019 novel coronavirus  2019 ncov      pathogen   corona virus disease 2019  1    nucleic acid detection   2019 ncov         key indicators   clinical diagnosis       false negative rate   2019 ncov nucleic acid   clinical practice            positive rate     30 50   2    false negative detection means missed detection      lead         delayed diagnosis   suspected patients        2019 ncov carriers becoming   potential sources   virus infection           urgent   improve   detection rate   2019 ncov nucleic acid clinically \\r\\n currently  fluorescent quantitative rt pcr  qrt pcr  technology   mainly     detect   nucleic acid   2019 ncov    quality   nucleic acid template   admittedly       key factors affecting   detection efficiency  according      laboratory technical guidelines   detection 2019 ncov  fourth edition  issued   national health   commission   china  3       experts  consensus   nucleic acid detection   2019 ncov  released   chinese society   laboratory medicine  4      human samples     placed   56           45 minutes       shorter time   inactivate   viruses           inspectors   virus infection     nucleic acids   isolated \\r\\n    2019 ncov     single stranded rna virus  theoretically    progress   killing   viruses     temperature   damage   integrity     rna genomes     samples    improperly reducing   amount   target viral templates    eventually leading     false negative rate     virus detection      study    employed porcine epidemic diarrhea virus  pedv        coronavirus      model virus   test     hypothesis     results demonstrated     virus inactivation progress     temperature damaged   detectable template     coronavirus seriously  biological  company  ltd    1 ng λdna  3010  takara  japan    suspended   mixed   using 1 5 ml   sample preservation medium  r503  vamzye  china    1 5 ml hank   solution \\r\\n nucleic acid  dna/rna    extracted using   commercial kit  rc311 c1  vazyme  according     manufacturer   protocol  briefly  500 µl samples preserved   r503     added     1 5 ml ep tube containing 200 µl   absolute ethanol   200 µl samples preserved   hank   solution   added     1 5 ml ep tube containing 500 µl lysis buffer \\r\\n   mixture   vortexed thoroughly     transferred     column placed onto   2 ml ep collection tube    centrifuge   1 min   12 000×     column   placed onto   fresh 2 ml ep collection tube  followed   adding 600 µl washing buffer     centrifuge   30 sec   12 000×   washing step   repeated      column   placed onto     2 ml collection    centrifuge   2 min   12 000×     column     placed onto   1 5 ml collection tube  100 µl   40 µl elution buffer   added     center     membrane   column collected   sample prepared   r503   hank   solution respectively     centrifuged   1 min   12 000 ×     collect   nucleic acid    nucleic acid   stored    30 ~  15       usage   short time      70         storage 2 3 agarose gel electrophoresis 5 µl     extracted nucleic acids  1 µl   10 × dna loading buffer  p022  vazyme    4 µl   nuclease free   2     mixed     vortexed briefly    mixture   heated   pcr instrument   denature   nucleic acid   2 min     subjected   1 2  agarose gel electrophoresis separation   ice   7 10 min       standard dna marker  md103  vazyme \\r\\n author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 02 20 958785 doi  biorxiv preprint\\r\\n quantitative real time pcr  qrt pcr    performed   detect   pedv rna   λ phage dna using hiscript ii   step qrt pcr probe kit  q222 cn  vazyme  according     manufacturer   protocol    reaction  25 µl    assembled   follows  12 5 µl   2 ×   step    mix  1 25 µl     step    enzyme mix  0 5 µl   50 × rox reference dye 1  0 5 µl   gene specific primer forward  10 µm  0 5 µl   gene specific primer reverse  10 µm  0 25 µl   taqman probe  10 µm    4 5 µl nuclease free h2o \\r\\n   reaction   carried     abi steponeplustm using following conditions  \\r\\n statistical significance   determined using   paired   tailed   test    value       0 05   considered statistically significant        0 01   considered statistically   significant    copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 02 20 958785 doi  biorxiv preprint 6 / 16   five          incubated   56       30  45   60 minutes  respectively    fourth   incubated   92       5 minutes      fifth     stored   4         inactivation control        temperature inactivation   completed  nucleic acids  including dna   rna    extracted   conventional methods    subjected   run 1 2  agarose gel electrophoresis  \\r\\n   understand   effect     virus inactivation progress     temperature     56         integrity   viral nucleic acid    quantitated   amount   table s1     results suggest     50 11      detectable viral templates left     inactivation progress   incubation   56       30 minutes     3 36      detectable viral templates left     inactivation progress   incubation   92       5 minutes \\r\\n   investigate     solutions   able   protect viral rna integrity   author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 02 20 958785 doi  biorxiv preprint 7 / 16 disrupting     inactivation progress     temperature  r503  vazyme  china        prepare   samples instead   hank   solution      performed     viral inactivation experiments   using hank   solution \\r\\n compared     control       sample   stored   4        dna   rna     samples incubated   56       30  60 minutes   displayed obvious degradation        28     18   bands   total rna   human cells   obviously smeared        bands   genomic dna   weaker           visible genomic dna   28   rna bands     sample incubated   92       5 minutes  fig 1d         degradation   nucleic acid     inactivation progress       serious         samples prepared   hank   solution    results suggest   r503       significant protective effect     total rna   genomic dna   degradation \\r\\n   determine   effect     inactivation progress   temperature     56         integrity   viral nucleic acid     samples prepared   r503    examined   amount   porcine pedv coronavirus   qrt pcr     \\r\\n compared     ct  31 64 ± 0 10      control       sample   stored   4        mean value   ct  31 31 ± 0 17        incubated   56       30 minutes     significant change    difference   cts               0 5   fig  1e   figure s1b   table s1   moreover    mean value   ct  31 43 ± 0 08        incubated   92       5 minutes       changed significantly    difference   cts               0 5    fig  1f  fig  s1b   table s1     results       inactivation progress     temperature     56         significantly affect   detectable template   viral nucleic acid       prepared   r503  suggesting     fragmented viral rna           using     template   qrt pcr       viral nucleic acid   degraded \\r\\n       understand   advantage   r503   common isotonic solution   author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 02 20 958785 doi  biorxiv preprint\\r\\n protecting   detectable amount   coronavirus rna   disrupting      compared   difference     detectable amount   coronavirus rna       samples     prepared   r503   hank   solution   stored   4        results       detectable amount     viral rna       prepared solutions   similar  fig  1g  table s1       difference     mean values     ct       0 5  31 64 ± 0 10 vs 31 95 ± 0 10    results suggest         significant difference     amount   detectable viral rna templates     samples prepared     r503   hank   solutions     stored   table s1     results suggest     detectable templates   viruses     samples prepared   hank   solution     32 30    r503 solution     samples   inactivated   56       30 minutes   kill   virus  furthermore    inactivation   92       5 minutes    ct   detected pedv virus       preservation solutions   31 43 ± 0 08  r503    36 84 ± 0 37  hank    respectively    mean value   ct   detected virus     samples prepared   hank   solution   5 4125       prepared   r503      0 0001  figure 1i   table s1     results reveal   detectable amount   virus     samples prepared   hank   solution     2 35    r503 solution     samples   inactivated   92       5 minutes   kill   virus \\r\\n   addition     disruptive effect     virus inactivation     temperature     detection   rna virus    detectable amount     viral dna  λ phage      samples     significantly affected  qpcr results  fig  s1c  table s1      mean value   ct  23 55 ± 0 20    detectable virus     samples author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 02 20 958785 doi  biorxiv preprint 9 / 16 incubated   56       30 minutes   increased   0 5230      0 01  compared     ct  23 02 ± 0 15      samples stored   4        mean value   ct  25 58 ± 0 21    increased   2 5572      0 0001      sample incubated   92       5 minutes \\r\\n furthermore    examined   amount   phage dna     sample prepared   r503     temperature treatment   qpcr    results  fig  s1d  table s1        mean value   ct  21 71 ± 0 04      sample prepared   r503   incubated   56       30 minutes     significant change compared         control    21 61 ± 0 06     difference             0 5    mean value   ct  22 90 ± 0 26    increased   1 2880     sample     inactivated   92       5 minutes  suggesting   92     treatment significantly reduced   detectable amount   dna viruses   sample   prepared   r503 \\r\\n   conclusion    results     virus inactivation   temperature     56     result     degradation   viral nucleic acids seriously      lead     artificial shortage   detectable templates   viral nucleic acids   samples    finally lead   false negative   clinical detection     samples  preserved     optimized solution     r503    detectable templates   viral nucleic acids     kept unchanged     samples   incubated   56           killing   viruses     integrity   cellular nucleic acids   obviously disrupted    results suggest       samples   prepared     optimal solution    detectable     viral templates     unchanged   virus inactivation progress     temperature     erase     rna   leave   fragments   nucleic acids           using     template   qrt pcr detection \\r\\n       true   2019 ncov  preparing   human samples     optimal solution           clinical inspectors   virus infection   inactivating   virus using 56           normal      improve   detection sensitivity   viral nucleic acid     sample  leading   avoiding       reducing false negative author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 02 20 958785 doi  biorxiv preprint 10 / 16 results   detection \\r\\n recently  chen reported     inactivation   incubating   sample   56       30 min   kill   virus     significant effect     detection   2019 ncov   qrt pcr  5       combining   limitations         samples tested     amount   virus       samples     authors discussed     researches      experimental evidences     virus inactivation     temperature   reduce   detectable amount     porcine coronavirus template seriously      highly recommended   carry   systematic investigation     impact     temperature inactivation     integrity   2019 ncov nucleic acids   develop   sample preservation solution   protect   detectable templates   2019 ncov nucleic acids     temperature inactivation damage  table s1   figure s1 author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 02 20 958785 doi  biorxiv preprint 15 / 16   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 02 20 958785 doi  biorxiv preprint  hcov hku1 replication     presence     inhibitors  fig  2b      inhibitors     151     toxic concentrations  fig  2c    152   experiments conducted     suggested   klk13   required   virus infection  153        question   specificity     klk13 protease inhibitors    ensure   154 klk13   indeed   priming enzyme   hcov hku1 infection    developed hae 155 cultures   transforming cells   lentiviral vectors encoding shrnas targeting klk13 156 mrna      confirmed     expression     protease   silenced  hae shklk13   fig  3a    importantly  hae shklk13 cells continued   differentiate   formed 162 pseudostratified cultures  fig  3b         infected hae ctrl  hae gfp  hae vector    163 hae shklk13   hcov hku1  10 6 rna copies   ml    incubated     2     164 32°      viral stock solution  cultures   maintained   32°    5 days     air liquid 165 interface  apical washes   collected    virus yield   determined   rt qpcr    166 found      contrast       control cultures  replication   virus   hae shklk13   167 abolished  fig  3c    overall    data suggest   silencing   klk13 gene   hae inhibits 168 virus infection  indicating   klk13       hcov hku1 infection  171   determined   klk13   essential   efficient hcov hku1 infection   hae   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 01 971499 doi  biorxiv preprint proteases   render rd cells permissive    generated rd cells expressing human klk13   176 tmprss2 proteases  rd cells   transduced   lentiviral vectors harboring   klk13 177 gene  rd klk13  control vector  rd ctrl    tmprss2  rd tmprss2  due     lack 178   klk13 specific antibodies    verified   presence based   rt pcr  fig  4a      179 presence   tmprss2   rd tmprss2 cells   confirmed using western blot    180 tmprss2 band   rd cells   observed   25 kda    corresponds         naturally 181 occurring splicing variants  fig  4b    subsequently    transduced rd ctrl  rd klk13   pseudoviruses  compared   δenv      completely abolished     presence   klk13 193 inhibitor  fig 4d    overall    data demonstrated   klk13 activity drives hcov hku1 194 entry   cells  195 196 klk13 enables   replication   hcov hku1   rd cells 197 obtained results     klk13 expression   rd cells   sufficient   hcov 198 hku1 pseudovirus entry       aimed   test   klk13 presence renders rd cells 199 permissive   hcov hku1 infection         infected rd ctrl   rd klk13 cells 200 author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 01 971499 doi  biorxiv preprint   hcov hku1  10 8 rna copies   ml    incubated   culture   7 days   32°      201 presence   absence     klk13 inhibitor  10 µm    dmso     cellular rna   isolated 202     presence   hcov hku1   subgenomic mrna    sg mrna      considered 203       hallmark   coronaviral infection    assessed  14  sg mrna appeared   rd klk13 204 cells      signal   detected   cultures supplemented     klk13 inhibitor     205 rd ctrl cells  fig  5a    206     confirm   role   klk13   hcov hku1 infection rd hcov hku1 infection   analyzed   means     sg mrna detection       found   212   sg mrna   produced       presence   klk13  fig  5b         passaged     presence   klk13  fig  5b         observed   cytopathic effects  cpes    218 replication levels     low    significant increase   control levels   rt qpcr  data 219   shown      test   effect   klk13   replication   hcov hku1   rd cells    220 virus stock   incubated   purified klk13  200 nm    incubated     presence   221 absence     klk13 inhibitor  10 µm    dmso    2     32°  pre treated virus stock   222 diluted   media   described     overlaid   rd cells    7 days   32°    evaluated 223   presence     hcov hku1   sg mrna    virus replicated     treatment   224 klk13    supplementation     inhibitor blocked   effect infection  fig  5c    225 author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 01 971499 doi  biorxiv preprint klk13 primes   hcov hku1   protein 227 expression   klk13   cells previously resistant   hcov hku1 renders   228 susceptible               due   proteolytic activation       protein  229   tested   using   cleavex method        peptide       exposed     klk13  subsequently  proteins   resolved   sds page   detected   western blotting 238   antibodies specific     tagged proteins    analysis          presence   239 500 nm klk13    cleavex protein harboring   s1/s2 cleavage site   degraded     240   cleavex protein harboring   s2/s2  site remained intact    klks   produced   241 pro forms   undergo self activation    additional band     tagged purified pro klk13 242  histag pro klk13    observed   treatment   500 nm klk13  fig  6a      product 243     s1/s2 cleavage     sequenced     terminal edman degradation     244 following sequence   ↓sisa    corresponds     s1/s2 site    result     245     s1/s2 region     hcov hku1   protein   prone   klk13 mediated cleavage  246 furthermore    aimed   confirm   cleavage using     length spike protein   hcov hku1  hku1           expressed   hku1     293t cells  purified   protein 248 using 6 ×   tag    incubated   3     37°    increasing concentrations   purified 249 klk13  subsequently  hku     mock proteins   resolved   sds page   detected   250 author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 01 971499 doi  biorxiv preprint western blotting   antibodies specific     tag    analysis         presence   251 1 µm klk13   hku1     degraded   figure 6b       protein   observed   252 ~150 kda      consistent     migration speed reported     highly glycosylated 253 proteins  1  254 255 author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 01 971499 doi  biorxiv preprint\\r\\n receptor recognition        essential step     virus infection process      copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 01 971499 doi  biorxiv preprint concentrations 63  broad spectrum protease inhibitors       widely   virus research  281       specific activity makes   results equivocal    example  matsuyama et al  282   recently     furin inhibitor dec rvkr cmk interferes     activity     283 proteases        previously described inhibitory activity   mers cov infection   284   specific   furin  instead    activity   due     specific inhibition   cathepsin     285 tmprss2  81    tried     specific klk inhibitors developed     laboratory  61 \\r\\n considering     arsenal   tools available   researchers studying klks      287 compounds   readily available  treating hae cultures     inhibitors revealed   288   compounds designed   inhibit klk13 hampered hcov hku1 replication       289   similarity     klks makes   doubt   specificity     inhibitors  290 despite   performance   biochemical assays       decided   silence klk13   291 hae cultures    abolished virus replication ex vivo  thereby confirming   importance   292 klk13   infection  klk13         secreted   membrane bound 82  83  293   study     hcov hku1 infection   hae modulates   expression   294   klks  including klk13      study  klks expression   tested using semi 295 quantitative pcr        reason      unable       level   klks modulation 296   hcov hku1       pattern   virus induced expression     klks   297   observed  klk mechanism   activation     complex process             298   proven     klk genes   regulated   steroids     hormones  84        299     remember    klk expression   regulated     similar manner      induction 300     single gene usually results   overexpression       cluster  85  86        301 assume     virus stimulates klk13 production   promote   infection      regulation 302       natural response     damaged tissue    klks   previously reported     303       tissue regeneration  87 89     increased expression   klks       304 author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 01 971499 doi  biorxiv preprint response     inflammatory process    seliga et al demonstrated   klk kinin system     305 potent modulator   innate immune responses  90 \\r\\n   experiments performed herein     importance   klk13   virus entry   307 susceptible cells       speculated   scattered distribution     klks   308   tissues           determinants     hcov hku1 tropism  53  91    309 tested   purified enzyme expressed     eukaryotic cells         developed   cell 310 line constitutively expressing   enzyme        vitro model     studies      rd 311 cells previously reported   carry attachment receptors     virus  26     using 312 pseudoviruses decorated     hku1 proteins       klk13 presence   rd cells 313   sufficient   virus entry   renders   cells permissive    experiments     314     contrast   previous reports  tmprss2     involved     process 51  furthermore  315   observed   rd cells supported   replication     virus     presence   klk13   316     effect   reversed     presence     specific klk13 inhibitor          317   able   culture   virus     yields  hcov hku1 replication   klk13 expressing rd 318 cells remained inefficient   rtqpcr assessment     reveal significant increases     319 amounts   viral rna      reason        able   detect viral sg mrnas      320 considered       hallmark   coronaviral replication   believe         due   321   optimal infection conditions      include inappropriate klk13 concentrations   322 low density     entry receptor             rd cells     support efficient 323 replication     virus due   factors unrelated     entry process  nonetheless    results\\r\\n       hcov hku1 entry receptor       rd cells        able   trigger 325 virus entry   replication    findings warrant   exploration \\r\\n   coronaviral   proteins   processed   s1   s2 subunits   host proteases    copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 01 971499 doi  biorxiv preprint hcov hku1   protein     regions     prone   proteolytic activation    s1/s2 furin 330 cleavage site     secondary cleavage site termed s2      adjacent     potential fusion 331 peptide  38      s1/s2 site   believed     processed   furin   protein 332 biosynthesis    s2/s2  site   expected     cleaved   virus entry            tested    found     s1/s2 site   efficiently cleaved   klk13  whereas   338 s2/s2  region remained intact    cleavex technique     convenient surrogate system allowing 339   precise mapping     cleavage site        limitations    ensure   reliability   340 results  purified   length hcov hku1   protein   subjected     proteolytic cleavage \\r\\n       observed efficient cleavage     hcov hku1   protein   klk13 \\r\\n     results         klk13   able   process   hcov hku1   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 01 971499 doi  biorxiv preprint   cross   species barrier   cause severe diseases   humans    research     role   355   proteases   coronaviral infections      356 author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 01 971499 doi  biorxiv preprint\\r\\n plasmid constructs 358 klk13   tmprss2 genes   amplified   pcr using cdna obtained   hae 359 cells    pcr product   cloned   pwpi plasmid   lentivirus production   sequence table 2    table 3   500 nm    β actin         household gene reference \\r\\n   reaction   carried   according     scheme  5 min   95°   followed   35 cycles   465 30     95°   20     59°    20     72°   followed   10 min   72°   containing proklk13   proklk14   concentrated     buffer   changed   50mm 528 tris ph 7 5  150 mm nacl    purification   self activation   37°    24    activity   529 author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 01 971499 doi  biorxiv preprint proteases   assessed   serine protease inhibitor kazal type 6  spink6  titration    530 described previously  63  protein expression   performed      coli bl21     induced     addition   553 0 5 mm iptg     bacterial culture  od600 0 5 0 6  followed   shaking   3h   37°   554 author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 01 971499 doi  biorxiv preprint      bacteria   spun       pellet   suspended   buffer    10 mm sodium 555 phosphate  500 mm nacl   5 mm imidazole  ph 7 4    pellet suspension     556 sonicated   spun    soluble proteins   purified using histrap tm excel  ge   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 01 971499 doi  biorxiv preprint  4 hepatoma cells     specific   cov    10 genetically divergent viruses tested    hepatitis   virus     restricted  extended data fig  1f  \\r\\n ly6e           ly6/upar family   gpi anchored proteins      implicated   diverse cellular processes 14     evaluate   specificity   ly6e   inhibiting cov    ectopically expressed select ly6/upar proteins 9   infected   hcov 229e    ly6e inhibited cov infection  extended data fig  1g      previously       conserved residue  l36    required   ly6emediated enhancement   influenza   virus 9   l36     required   restrict hcov 229e infection  extended data fig  1h      data suggest   conserved functional domain     responsible       virus enhancing    restricting phenotypes \\r\\n      investigated   effect   ly6e   specific steps     virus replication cycle       tested   ly6e restricts hcov 229e attachment     cell surface   observed   effect  extended data fig  2a      accordance  ly6e     effect   surface expression   cd13    specific receptor   hcov 229e  extended data fig  2b      test   ly6e restricts viral entry        vesicular stomatitis virus  vsv  pseudoparticle  pp  system bearing cov spike     proteins \\r\\n strikingly  ly6e significantly inhibited vsvpp entry mediated     proteins   hcov 229e  fig    2a    mers cov  fig  2b    sars cov  fig  2c      sars cov 2  fig  2d    entry mediated   vsv glycoprotein       marginally impacted   ly6e  extended data fig  2c         test   ly6e interferes     fusion   viral   cellular membranes    performed   syncytia formation assay using propagation competent vsv pseudoviruses co expressing cov   protein     gfp reporter  vsv δg cov     extended data fig  3a    ly6e potently inhibited cov   protein mediated syncytia formation  fig  2e       determine   ly6e impairs   protein expression   maturation      performed   heterologous syncytia formation assay  cov resistant hamster cells infected   trans complemented gfp reporter vsv δg cov    co cultured   susceptible target cells   ectopically expressed ly6e  ly6e   significantly blocked syncytia formation  demonstrating   ly6e inhibits   protein mediated fusion       protein expression   maturation  extended data fig  3b        membrane fusion   occur   syncytia formation      performed   quantitative fusion assay     mixing   cell contents results   complementation   split luciferase  fig  2g   \\r\\n author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint cells co transfected   plasmids encoding cov   proteins   split luciferase 1 7   mixed   susceptible target cells co expressing ly6e   vector control   split luciferase 8 11  ly6e reduced cov   mediated cell cell fusion  indicating   ly6e blocks fusion   viral   cellular membranes   fig  2h    time course experiments revealed   ly6e     effect   hcov 229e translation  replication  assembly    release  extended data fig  2e      collectively    results demonstrate   ly6e specifically inhibits cov   protein mediated membrane fusion      required   viral entry \\r\\n     addressed   ly6e modulates proteolytic activation       protein      proteinmediated binding     respective receptor  host proteases cleave     protein       fusion competent      entry      cell surface    late entry    endosomes 15  16   pharmacological inhibition   cell surface   endosomal proteases     effect   ly6e mediated restriction   cov infection  extended data fig  4a           addressed   ly6e directly interferes     proteolytic activation     protein  mutation   s1/s2   s2′       protein prevents activation  extended data fig  4d   16  17   ectopic ly6e expression     effect   mers cov   cleavage   infection   cleavage resistant mers cov pp  extended data fig  4e      collectively    data indicate   restriction   viral fusion   independent     protein activation \\r\\n      aimed   evaluate ly6e mediated cov restriction   vivo  systemic genetic ablation   ly6e   mice   embryonic lethal 18        generated transgenic ly6e fl/fl mice      functionally  wild type  wt    ly6e expression    crossed     vav1 icre mice   ablate ly6e   cells   originate     hematopoietic stem cell  hsc  progenitor  extended data fig  5a       genetic model   chosen   immune cells   critical   protection   cov   vivo 5 19 23   bonemarrow derived macrophages  bmdm    ly6e δhsc mice   reduced ly6e mrna levels  extended data fig  5c     natural mouse pathogen mhv       studied model cov   causes hepatitis   encephalomyelitis   mice 24   ly6e δhsc bmdm     susceptible   mhv infection   ly6e fl/fl bmdm  demonstrating   murine ly6e     restriction factor   murine cov  extended data fig    5d  \\r\\n   determine   ly6e       controlling cov infection   vivo  ly6e fl/fl   ly6e δhsc mice   infected       dose  5   10 4 pfu    mhv   causes sub lethal hepatitis   wt author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint c57bl/6j mice 25     mhv pathogenesis   influenced   sex 26     performed independent experiments   male   female mice  ly6e δhsc mice     sexes rapidly succumbed   mhv infection   day 6   fig  3a    extended data fig  6a        examine viral pathogenesis   immunopathology   infected mice  ly6e fl/fl   ly6e δhsc mice   infected         dose   mhv   euthanized 3   5 days post infection      time    female ly6e δhsc mice exhibited significantly   levels   liver damage   measured   serum alt     difference   observed   male mice  fig  3c    extended data fig  6c         hepatic viral burden           sex   fig  3d    extended data fig  6d      possibly due   tissue saturation       dose  indeed  female     male ly6e δhsc mice exhibited increased liver damage   viral burden     low dose  5 pfu    mhv  extended data fig  7a          spleen viral burden   elevated   female   male ly6e δhsc mice     time     doses   mhv  fig  3e    extended data fig  6e     extended data fig    7c       3 days post infection        difference   liver necrosis    inflammation   moderately reduced   female     male ly6e δhsc mice   fig  3f    extended data fig  6f        5 days post infection       low     doses   mhv  female ly6e δhsc mice   significantly   levels   liver necrosis     reduced presence   mononuclear immune cells   fig  3l  fig  7k        data demonstrate   ly6e   hematopoietic cells       controlling murine cov infection \\r\\n         transcriptomic approach   evaluate   effect   ly6e ablation   global gene expression   liver   spleen   female ly6e fl/fl   ly6e δhsc mice injected   pbs       dose   mhv   3   5 days post infection    infected tissues  loss   ly6e   hematopoietic cells correlated   differential gene expression   pathways associated   tissue damage      liverspecific metabolic genes  angiogenesis  wound healing    immune response   viruses  fig  4a          observed   striking loss   genes associated     type   ifn response  inflammation  antigen presentation      cells   infected ly6e δhsc mice   fig  4d    extended data fig  8    7   histopathology  fig  3h      transcriptome data  fig 4a    extended data fig  8 fig  7k    extended data fig  9   multiple cell types   depleted   livers   spleens   infected ly6e δhsc mice  irrespective   sex  figure 4f    extended data fig  10a        cells   dramatically reduced   livers   infected ly6e δhsc mice \\r\\n hepatic cd4     cells  nk cells  dendritic cells  dc  macrophages    neutrophils     depleted   infected ly6e δhsc mice  whereas cd8     cells   unchanged  fig  4f  extended data fig  10a  \\r\\n infected ly6e δhsc mice     reduction     splenic immune cell subsets except   cd4     cd8     cells  fig  4g  extended data fig  10b    depletion   immune cell populations   ly6e δhsc organs   mhv dependent    cell       altered   pbs injected mice \\r\\n   determine   loss   immune cells   mhv infected ly6e δhsc mice correlates   increased permissiveness   infection    assessed mhv infection   splenocytes cultured   ly6e δhsc   ly6e fl/fl     ifnar  /mice    ly6e fl/fl splenocyte cultures  mhv infected macrophages  neutrophils  dc      cells      cd4     cd8     cells    previously published  fig  4h  extended data fig  10c      data demonstrate     role   ly6e   immune cell mediated control   cov infection    viral burden   lower liver inflammation suggests   liver damage   mhv infected female ly6e δhsc mice     due   viral pathogenesis   exceeds   ability     immune system   control   infection  notably    observed mhv induced loss     classes   cells  1  cells       permissive     virus  cd4     cells  2  cells     permissive   mhv       demonstrate ly6e ko dependent infection  nk cells  macrophages    neutrophils    3  cells     permissive   mhv     ly6e ko dependent manner    cells   dc    latter     capable   antigen presentation   cd4     cells 27 28          absence   ly6e  unrestricted mhv infection author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint     cells   dc   impair   cell priming  thereby negatively impacting recruitment   activation   nk cells  macrophages    neutrophils   contribute   control   viral disease \\r\\n   conclusion    identified ly6e     pan cov restriction factor   limits cov entry   protects   host   severe viral disease    findings   striking     ly6e     primarily associated     enhancing phenotype     ly6e promotes entry   multiple viruses 29        data   expand   role   ly6e   coronaviral infection   establishes   novel function   protecting   host immune cell compartment  determining   precise molecular mechanism underlying   ly6e inhibits cov   protein mediated membrane fusion   advance   understanding   cellular antiviral defenses       human pathogens  antiviral membrane fusion inhibitors     successfully implemented   treatment   hiv 1 infection 30     therapeutic approach mimicking   mechanism   action   ly6e   provide     line   defense   novel emerging cov infections  furthermore  delineating   ly6e expressing immune cells protect mice   mhv   provide   insight     individual antiviral effectors   distinct cellular compartments modulate viral pathogenesis  author/funder    rights reserved    reuse allowed   permission  34 41  1996    welch   correction              tailed mann whitney   test     ratio paired student     test     2way\\r\\n anova followed   sidak     dunnett   multiple comparison test       error bars  sd    values        =0 0088        =0 0001        =0 0143        =0 0286        =0 0101        =0 0010        =0 0073   =0 0033     ns=0 0740     =0 0013     =0 0001  author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint  = 0 0001       =0 0066        = 0 0001        = 0 0001     ns  =0 9975     =0 0054  ns  =0 1520  0 9892  author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint scale bars   500 µm  4x    100 µm  20x  statistical significance   determined   mantel cox test       tailed unpaired student     test   welch   correction              tailed mann whitney   test            error bars  sem     sd              values        =0 0002        =0 0031     ns  =0 2136        =5 759   10  7      ns  =0 3698        =0 0054        =0 0430     ns  =0 1321        =0 0094        =0 0072        =8 741   10  5   author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint maps displaying significant changes  mean rpkm   0 5  fold change   2  fdr ≤ 0 05    liver       spleen     dendrograms   normalized read data  row   score  clustered   complete linkage method employing spearman rank correlation distance measurement     pathway analysis   ko versus wt \\r\\n   red      regulated  blue  pathways indicated   activation   score      significantly dysregulated genes   percentage   total gene   included   pathway       expression   select genes   liver       spleen          immune cell counts   liver       spleen \\r\\n hcov 229e  1 5   10 5 stable ly6e expressing   control cells   seeded     12   plate   infected   hcov 229e     serial dilution  cells   incubated   2 hours   33°     viral supernatant   removed      overlay   1 1 avicel  2 4   avicel rc 581nf    2x dmem  author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint supplemented   10  fbs   1x penicillin/streptomycin         added  cells   incubated   3 days   33°    medium   removed  cells   washed    stained   crystal violet  sigma aldrich  plaque forming units  pfu    calculated \\r\\n hcov oc43  5 0   10 4 stable ly6e expressing   control cells   seeded     24   plate   infected   33°    1 hour   hcov oc43  moi=1  virus   aspirated   10  fbs/1x neaa/1x    /rpmi  crpmi    added     cells  24 hours post infection  cells   dissociated using accumax  sigma aldrich  fixed   1  pfa  permeabilized   manufacturer   protocol  bd cytofix/cytoperm    stained   nucleoprotein  1 500      goat anti mouse alexafluor488    copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint anti mouse secondary antibody  images   acquired     fluorescence microscope   analyzed using imagexpress micro xls  molecular devices  sunnyvale  ca \\r\\n   infection assays  1   10 5 target cells   seeded     24   plate   day prior infection  cells   infected   hcov 229e rluc  moi 0 1    optimem  gibco    2 hours   33°   cells   washed 1x   pbs   10  fbs/1x neaa/1x    /dmem  cdmem    added    cells   incubated   33°    24 hours    washed   pbs   lysed using   renilla luciferase assay system kit  promega  rluc activity   measured using   plate luminometer  enspire 2300 multilabel reader   perkin elmer      reconstitution   crispr resistant ly6e  cr ly6e  2 5   10 4 cells   seeded     24   plate    day post seeding  cells     left untransduced   transduced     lentiviral vector encoding   cr ly6e     empty control  48 hours post transduction  cell lysates     infected   hcov 229e rluc  moi=0 1    24 hours   described     harvested   western blot \\r\\n pseudotyping   vsv δg fluc    performed   previously described 50     silent mutation  t2568g  t3327c      described previously  51   52   53      expression plasmid   sars cov 2   generated   follows    coding sequence     synthetic  codonoptimized    human cells  sars cov 2 dna  geneart gene synthesis  thermo fisher scientific  author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint based     publicly available protein sequence     national center   biotechnology information database  ncbi reference sequence  yp 009724390 1    pcr amplified   cloned     pcg1 expression vector   bamhi   xbai restriction sites    mers cov   cleavage mutants     described   17 \\r\\n   split luciferase fusion assay     described      slight modifications 54   briefly  6   10 5 293ltv cells   transfected   plasmids encoding cov   proteins  pcaggs hcov 229e  pcaggs mers cov        split luciferase construct  rluc8155 156 dsp1 7    empty expression plasmid  pcaggs mcherry          construct encoding   vsv   served   negative   positive control  respectively  2   10 5 stable ly6e expressing   empty control huh7 cells   transfected using lipofectamine 2000  thermo fisher scientific      plasmid encoding     half     split luciferase protein  rluc8155 156 dsp8 11  approximately 30 hours post transfection    cell populations   dissociated  tryple express  gibco  counted    equal cell    2x10 4 cells      co cultured   16  20 hours   37°   supernatant   harvested   cells lysed using ice cold passive lysis buffer  promega  cells   immediately transferred    80°        author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint 1 hour    rluc activity   determined using   renilla luciferase assay system  promega  cells   kept   ice     times post cell lysis   eliminate dsp post lysis complementation \\r\\n following extraction   total rna   mers cov  strain emc  infected veroe6 cells    cdna encoding   mers cov   protein   generated   reverse transcription  revertaid premium reverse transkriptase  thermoscientific    overlapping cdna fragments   amplified   pcr  phusion hot start ii   fidelity polymerase  thermo scientific    subsequently assembled   overlapping pcr    cdnas encoding     proteins     mers cov  strain emc    hcov 229e  truncated variant lacking   10 amino acids       terminus  original plasmid pcaggs 229e      amplified   pcr   inserted     pvsv  plasmid 55   mlui   bsteii restriction sites    resulting plasmids       generate   recombinant viruses vsv δg mers      vsv δg 229e    according     published procedure 56     viruses   propagated   bhk g43 cells 57   resulting   viruses predominantly containing   homotypic vsv glycoprotein       envelope \\r\\n cells expressing ly6e   empty control   infected   vsv δg mers      vsv δg 229e        moi   0 01    20 hours post infection    cells   fixed   3  pfa/pbs     nuclei stained   4 6 diamidine 2 phenylindole  dapi  sigma    inverted fluorescence microscope  zeiss        determine   percentage   covered   syncytia \\r\\n   heterologous cell cell fusion assay  bhk 21 cells   infected   vsv δg cov        moi   1    2 hours    cells   treated   trypsin/edta  life technologies  zug  switzerland    resuspended   dmem supplemented   5  fbs  2x10 4 cells/ml    infected bhk 21 cell suspension  500 µl    seeded   24   cell culture plates     ly6e   empty controlexpressing huh7  1   10 4 cells  cell co cultures   incubated   7 hours   37°   fixed   3 6  author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint formaldehyde diluted   pbs   grogg chemie ag  stettlen  switzerland    stained   dapi    percentage   covered   syncytia   calculated     \\r\\n ly6e tm1a es cells   obtained     eucomm consortium 58 \\r\\n six   twelve week   female   male mice   injected intraperitoneally   mhv a59 diluted   pbs     indicated titers   pbs     mock infection control    infected mice   monitored daily   weight   mortality  animals   lost     20      original body weight   euthanized   iacuc guidelines \\r\\n viral titers   liver   spleen   determined   frozen organs   weighing  homogenization    plaque assay   l929 cells  alanine aminotransferase  alt    measured   fresh  unfrozen serum using vitros microslide technology     utsw mouse metabolic core  livers   fixed   10  author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint neutral buffer formalin  embedded   paraffin  sectioned   5 µm    stained   hematoxylin   eosin       slides   analyzed     independent pathologist  utsw animal resource center      blinded   experimental conditions    numerical score   assigned   degree   inflammation   degree   necrosis     liver  inflammatory cell infiltration   scored using   following criteria  0 = none  1 = minimal  2 = mild  3 = moderate    4 = marked    necrosis score   defined     percent necrosis  0 = none  1 =  2  2 = 2 20  3 = 21 40    4 =  40 \\r\\n   primary cells   maintained   10  fbs/10 mm hepes/1 mm sodium pyruvate/2 mm lglutamine/1x    /rpmi  crpmi2  unless otherwise indicated \\r\\n primary bone marrow derived macrophages  bmdm    prepared   described previously 60   viable cells   quantified using trypan blue exclusion  bmdm   plated   2 5   10 4 cells            copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint separated   ihic   centrifugation   850       30 minutes  22°       brake  red blood cells   removed     resulting pellet   rbc lysis buffer  tonbo biosciences  viable cells   quantified using trypan blue exclusion  ihic   splenocytes  4   10 5     infected   mock infected mice   stained     fluorescent viability dye  ghostdye violet 450  tonbo biosciences  treated   anti cd16/cd32  tonbo biosciences  stained   antibodies   recognize surface lineage markers      fixed   1  pfa/pbs  fixed volumes   cell suspensions   analyzed   flow cytometry     day  absolute cell counts   determined   multiplying final     lineage marker positive cells   dilution factor     normalize cell counts   antibody staining \\r\\n   primary cell samples   resuspended   3  fbs/pbs   analyzed using   s1000 flow cytometer     a600 96   plate   throughput extension   compensated using cellcapture software  stratedigm  data   analyzed   flowjo software  treestar    flow gating strategy   liver   spleen immune cells   included   extended data figure 9  \\r\\n differences   data   tested   significance using graphpad prism v8 3 1   windows  graphpad \\r\\n   details regarding   statistical tests applied  please refer     figure legends    values   0 05   considered significant \\r\\n   authors declare     data supporting   findings     study   available     article     supplementary information files     available   request    rnaseq data discussed     publication     deposited     gene expression omnibus  geo  database  https //www ncbi nlm nih gov/geo  gse146074 \\r\\n author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint\\r\\n         thank   following people     generous contribution   reagents   infrastructure   greatly enhanced   study  especially   sequencing facility   bern    diagnostic facility   ivi      animal resource center   utsw  furthermore      grateful   \\r\\n   authors declare   competing   \\r\\n supplementary information   available     paper \\r\\n correspondence   requests   materials     addressed   volker thiel  john    schoggins    charles    rice \\r\\n author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint   harvested   extracellular viral rna   extracted   viral replication detected via qrt pcr     cell lysates   harvested   intracellular renilla luciferase activity   detected   cell lysis     cells   subjected   3 rounds   freeze/thaw cycles  cell debris   removed     supernatant titrated   naïve huh7 cells  intracellular infectivity   determined     supernatant   harvested     copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint anova followed   sidak   multiple comparisons test     error bars  sd    values        = 0 0001  0 0001  author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint   pooled experiments          =8  ly6e fl/fl    =6  ly6e δhsc      =3  pbs      pooled experiments         scale bars   500 µm  4x    100 µm  20x  significance     tested  \\r\\n mann whitney   test            error bars  sem     sd              value        =3 946   10  6      ns  =1144     ns  =0 1407        =0 0026     ns  =0 1708     ns  =0 0779     ns  =0 6191     ns  =0 6842        =0 0119     ns   0 9999     ns  =0 0779  author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint              tailed mann whitney   test            error bars  sd    value        =0 0234        =0 0428        =0 0004        =0 0210        =0 0054     ns  =0 1544     ns  =0 5322     ns  =0 0662     ns  =0 0592     ns  =0 9064  author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint ddx3x  ddx3  viperin  rsad2  serpine1  pai 1  il28ra  crf2  vegfc  vrp  sp110  ifi41  irf2  pric285  znf313  rnf114  dnaptp6  loc26010  loc400759  pseudogene    pscd1  cyth1  ifitm1  ifi17  siglec1  frag  ccl5  syca5  cxcl9  mig  ifih1 huh7 cells expressing tmprss2   generated   follows        generate tmprss2 encoding retroviral vectors      reading frame   human tmprrs2     pcr amplified   primers adding     terminal cmyc epitope     tmprss2 coding sequence    resulting sequence   inserted     modified version     pqcxip plasmid   contains   blasticidin resistance cassette instead     usual puromycin resistance cassette 17   huh7 cells stably expressing human tmprss2   generated   retroviral transduction   selection     antibiotic blasticidin  50 µg/ml \\r\\n following selection  cells   maintained   culture medium  cdmem  supplemented   10 µg/ml blasticidin    generate stat1  / ceacam1 cells  human stat1 −/− fibroblasts   transduced   lentiviruses encoding     murine cov receptor ceacam1    gift   david wentworth  cdc  atlanta  usa 59   subsequently selected   1 µg/ml puromycin  stable ly6e expressing cell lines   generated   lentiviral transduction   scrpsy ly6e     control scrpsy empty   dmem containing 4 µg/ml polybrene  millipore    20 mm hepes buffer solution  gibco  cells author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint   selected using 2 4 µg/ml puromycin     passaged     cells   control     lentivirus   killed    puromycin selected cells     passaged   frozen     subsequent experiments    stable cell lines harboring ly6e orthologues     described        exception      dromedarius 9          gblock      xm 031439745 1    gateway cloning performed   described previously   generate pentr   pscrpsy plasmids 9   constructs encoding   ly6/upar family     ly6e asm mutants     described   9  \\r\\n   generate clonal ly6e ko cells   crispr resistant ly6e  a549 cells   transduced   lentivirus containing   ly6e specific sgrna   cas9   described previously 9     generate   clonal cell line    bulk transduced population   plated   single cell dilutions  candidate clones   screened   western blot   ly6e expression   sanger sequencing  silent mutations     region targeted     ly6e specific sgrna   introduced   ha tagged ly6e   generate crisprresistant ly6e  cr ly6e  ly6e ko a549 cells   reconstituted   cr ly6e   lentiviral transduction   expression confirmed   western blot \\r\\n   cell lines   regularly tested   check     free   mycoplasma contamination using   commercially available system  pcr mycoplasma test kit  /rt variant    promkine \\r\\n   isg screen   performed   described previously   slight modifications 7 8   briefly  5   10 3 huh7 cells   seeded  transduced   individual lentiviruses    48 hours post transduction infected   hcov 229e   33°   infection   stopped 24 hours  moi=0 1    48 hours  moi=0 01  postinfection   plates   immuno stained   described previously 61     anti hcov 229e   protein antibody     alexafluor488 conjugated donkey anti mouse secondary antibody  extended methods \\r\\n antibodies   immunofluorescence   flow cytometry      content   throughput imaging analysis imagexpress micro xls  molecular devices  sunnyvale  ca        previously described 8   hits   normalized   cells expressing   empty vector  depicted   genes     author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint expressed     independent screens     transduction efficiency       1    cut    isgs     cytotoxic     cells   excluded     analysis  normalized data     found   extended data \\r\\n viral rna   extracted   cell lysates  nucleomag 96rna kit  macherey nagel    supernatant  nucleomag vet kit  macherey nagel  using   kingfisher flex robot according     manufacturer   recommendations    commercially available taqmantm fast virus 1 step master mix  applied biosystems        rt qpcr   2 µl   rna input added   8 µl   prepared mastermix   author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint sample  viral rna   detected using hcov 229e specific primers normalized     qrt pcr standard   hcov 229e  contains     gene   hcov 229e 63   intracellular viral rna   normalized   total rna  determined via   housekeeping gene beta 2 microglobulin  b2m  cellular rna   extracted   cell lysates   huh7 cells using   nucleospin rna kit  machery nagel  according     manufacturer   recommendations  quantified via nanodrop        generate   standard curve \\r\\n pcr conditions   available   request  \\r\\n stable ly6e expressing   control huh7 cells   seeded     24   plate  4   10 4 cells    mock infected   inoculated   hcov 229e  moi=5    optimem  gibco    ice   1 hour  cells   washed     3x   pbs   harvested immediately   =0      incubated   33°    24 hours   =24      cell lysis   extraction   viral rna   described    viral rna   detected via rt qpcr normalized     housekeeping gene b2m   described   \\r\\n time course experiment 8   10 4 cells stable ly6e expressing   control huh7 cells   seeded   12   plates   mock infected   infected   229e cov rluc  moi=0 1    2 hours   33°    optimem  cells   washed 3x   pbs   samples harvested     indicated time      determine intracellular replication  cell lysates   collected   viral rna extracted  nucleomag 96rna kit  macherey author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint nagel  followed   rt qpcr   described      cells   lysed using   renilla luciferase assay system kit  promega  according     manufacturer   recommendations   rluc activity determined    determine extracellular viral replication  viral supernatant   harvested    viral rna extracted  nucleomag vet kit  macherey nagel  followed   rt qpcr   described      determine intracellular infectivity  cells   subjected   3 rounds   freeze/thaw cycles  centrifuged   remove debris  4 000 ×     10 min      supernatant titrated   naïve huh7 cells    determine extracellular infectivity  supernatant   harvested     indicated time     titrated   naïve \\r\\n   test various protease inhibitors  2   10 4 naïve   tmprss2 expressing  control   ly6e cells   seeded     96   plate    day post seeding  cells   pre treated     following compounds   1 hour   optimem    37°   dmso  1 500  sigma aldrich  e64    10 µm  sigma aldrich   /  camostat  100 µm  sigma aldrich  cells   mock infected   infected   hcov 229e rluc  moi 0 1      presence     inhibitors   2 hours   33°   medium   changed   dmem   cells incubated   24 hours   33°   cells   lysed   rluc activity detected using   renilla luciferase assay system kit  promega  according     manufacturer   recommendations \\r\\n ly6e expressing   control huh7  2   10 5 cells    seeded   6   cell culture plates  cells   transfected using lipofectamine 2000  invitrogen      expression plasmid encoding   mers cov    pcaggs mers    cell lysates   harvested 48 hours post transfection   subjected   western blot analysis   described \\r\\n author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint\\r\\n multiple sequence alignment   ly6e amino acid sequences     species  ncbi accession   provided     figure    performed using   ebi web based clustal omega tool  doi \\r\\n 10 1093/nar/gkz268    phylogenetic tree   built        click  mode   phylogenetic analysis  http //www phylogeny fr/    standard settings  doi  10 1093/nar/gkn180  doi \\r\\n 10 1186/1471 2148 10 8 \\r\\n sections   liver   spleen   preserved   rnalater stabilization solution  thermo fisher scientific    frozen  thawed samples   transferred   pbs   homogenized    eighth     homogenate   mixed   trizol reagent  thermo fisher scientific  bmdm   directly lysed   trizol reagent   frozen  total rna   isolated according     manufacturer   protocol  liver   spleen rna   subject   dnase treatment  turbo dna free kit  thermo fisher scientific      manufacturer   protocol prior   rna seq analysis  bmdm rna   analyzed     step qrt pcr using quantifast sybr green rt pcr kit  qiagen  using applied biosciences 7500 fast real time pcr system \\r\\n   quantity   quality     extracted rna   assessed using   thermo fisher scientific qubit 2  author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint data generated   individual samples  30 million read   sample  single read 50 mers    mapped separately     grcm38 murine reference genome  gene expression   calculated   individual transcripts   reads   kilobase   million bases mapped  rpkm    transcriptomic analyses   performed using clc genomics workbench 20  qiagen  aarhaus \\r\\n differentially expressed genes  degs    identified   calculating fold changes   expression  pvalues   corrected   taking false discovery rate  fdr    multiple comparison   account \\r\\n mus musculus ebi gene ontology annotation database       execute gene ontology     enrichment analyses   biological processes  gene identifiers   degs absolute fc   5  rpkm   2        input   identification   significantly enriched   categories    values   specific   categories   generated   bonferroni correction   multiple testing    scores      indicator   activation   biological processes  positive value    inactivation  negative value    calculated based     expression fold change   follows    copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint gctggcatcgggaacgtggtggactttggctac accctgaacaagggctgctccccgatctgtccc ggcccgagcgtcaatcttggagtggcgtccgt gggcacccactgctgccagagcttcctgtgca acatcagtgcagccgacggcgggctgcgggc cagcacccacgtgctgggcctcgggctcctgc tcagcctgctgtccgccctgctgcggcttggcc cctgaacccagctttcttgtacaaagtggtcccc author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed        https //doi org/10 1101/2020 03 05 979260 doi  biorxiv preprint    rapid evolution   rna viruses represents   significant challenge   preventing  treating  57   eradicating rna viral diseases    mutation rates   rna viruses generate extensive 58 opportunities   overcome evolutionary hurdles      antiviral drugs  host immunity    59 engineered attenuating changes  1    evolutionary pathways traversed   rna viruses   60 shaped   natural selection      favor   evolutionary trajectories       61 based     mutations   beneficial  deleterious    neutral  2  predicting     results 62   rna virus evolution       step   anticipating viral emergence     developing 63 escape resistant antiviral drugs   vaccines  3  4  64 65 coronaviruses  covs      family   positive sense rna viruses   cause human illnesses 66 ranging     common cold   severe   lethal respiratory disease  5    covs encode   67 proofreading exoribonuclease   nonstructural protein 14  nsp14 exon      critical   68 replication  fidelity  fitness    virulence    exon inactivation     proposed     69 strategy   live attenuated vaccine development  6 15          deddh superfamily 70   exonucleases  cov exons hydrolyze nucleotides using   metal coordinating amino acids 71   copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/618249 doi  biorxiv preprint exon active site mutants   alphacoronaviruses  including transmissible gastroenteritis virus   77 hcov 229e          recovered     proposed     lethal   replication  19  20    infections   prolonged passage   tissue culture   following treatment   multiple 87 nucleoside analogues  6 11  13  14  sars cov exon aa     stable   acute   88 persistent animal infections   immunocompetent   immune compromised mice  12      lack 89   primary reversion     due simply   reduced adaptive capacity      sars cov   90 mhv exon aa   adapt   increased replication  7  14    strikingly    term passage 91   mhv exon aa  250 passages  p250  yielded   highly fit population     directly 92 compensated   defective proofreading   evolution         fidelity rdrp  7 requires mutations       sites   figure 1a   viral mutation rates     absence   113 proofreading range   10  4   10  6 mutations   nucleotide   round   replication  µ   1  114\\r\\n assuming   exon aa mutation rate   10  4 µ   accounting   codon degeneracy    115 probability   restoring   native amino acid sequence     single round   replication   10  18   116\\r\\n   rarely   exon aa titers exceed 10 6 pfu/ml        unlikely   exon aa   117 navigate   genetic barrier     single infectious cycle       hypothesized   exon aa 118 reversion         proceed incrementally    identify potential pathways towards 119\\r\\n exon aa reversion    examined     single nucleotide substitutions surrounding a89 120   a91   figure 1b     mutations   synonymous    five mutations yield amino acids 121 unlikely   coordinate     positively charged metals required   exon catalysis  glycine  122 valine  proline  threonine    serine   16  19  21  22    mutation   site   restore   123 acidic charge       aa   ed        native amino acid    variants       tested 124     cov exon    biochemical studies      coli dna polymerase   exon mutants suggest   125   conservative substitutions     restore wt   exon activity  23      predicted 126 stepwise pathways   exon aaàde reversion based   restoration   acidic charge followed 127   reversion   native amino acids   figure 1c     engineered   recovered variants   exon 128 aa requiring   mutations  3nt  exon ad  exon ea    mutations  2nt  exon da  exon 129 ed  exon ae      mutation  1nt  exon dd  exon ee    reversion   wt exon de 130  table 1        hereafter refer     mutants   intermediate revertants    intermediate 131   rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/618249 doi  biorxiv preprint revertants generated viable progeny   recovery  demonstrating   reversion   wt exon 132 de     pathways   theoretically      3nt   2nt mutants   genetically 133 stable   recovery    confirmed   dideoxy sequencing       1nt mutants  exon 134 dd   exon ee  reverted   wt exon de     independent recovery attempts  135\\r\\n suggesting       variants     fit   wt exon de   demonstrating   136 reversion   1nt mutation   readily accessible    test     3nt   2nt mutants   137 revert   rapidly   exon aa  4nt    passaged   lineages     mutant 10 times    table 1     contrast    2nt exon ae contained wt revertants   p2 145     lineages   moi = 0 5 pfu/cell     p8     lineage   moi = 0 01 pfu/cell    146 observed  wt revertants dominated   exon ae population     remaining passages    147 data indicate         2nt mutation pathway   lead     reversion   tissue culture  148\\r\\n   probability   exon ae arising     single infectious cycle   exon aa   low   149 theoretically achievable  ~10  9     exon aa   conceivably revert       infectious 150 cycles     complete reversion       observed     prolonged passage   151 persistent infections  suggesting   additional barriers     replication  fitness    maintenance 152   intermediate revertants exist  153   rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/618249 doi  biorxiv preprint \\r\\n gat gaa\\r\\n gca gct\\r\\n exon motif       rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed      author/funder  \\r\\n   rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/618249 doi  biorxiv preprint\\r\\n   intermediate revertants   viable   recombinants       found   exon aa 159 populations    hypothesized     confer   selective advantage   exon aa  8  9  13  160\\r\\n   test   hypothesis      analyzed replication     3nt   2nt intermediate revertants 161     copyright holder     preprint        peer reviewed      author/funder    copyright holder     preprint        peer reviewed      author/funder    copyright holder     preprint        peer reviewed      author/funder  nsp12/14 p250    significantly   sensitive   5 fu   figure 4b   finally    nsp12 p250 224   nsp14 p250 mutations significantly decreased fitness relative   wt exon de   figure 4c   225\\r\\n   detected   statistical differences     specific infectivity   wt exon de     226     nsp12 p250   nsp14 p250 variants   isolated infections   figure 4d      mutations 227   nsp12   nsp14   arose     exon aa background   detrimental   replication  228 mutagen sensitivity    competitive fitness     presence       reverted exon de    229   rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed      author/funder    copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/618249 doi  biorxiv preprint   results   extend existing studies   cov exon motif    motif   aaàde mutations     249 sars cov nsp14 exon dramatically reduce nuclease activity   biochemical assays      250 study   examined   contributions     residue independently  16  18    intermediate 251 revertants   exon aa     display consistent   statistical differences   replication  5 fu 252 sensitivity    competitive fitness relative   exon aa  suggesting     remain 253 phenotypically exon    infection   supporting previous studies   motif   de   254 essential   wt exon function      results      surprised   observe repeated 255 reversion     exon ae           2nt variants  exon da   exon ed    256 potential explanation       specific mutational bias   exon ae makes   revertant 257 mutations   accessible     exon da   exon ed  alternatively    exon ae   258 profound replication   fitness defects  selection   drive primary reversion   quickly 259       genotype  consistent     hypothesis  exon ae reverted   quickly     260   moi    natural selection acts   efficiently     larger population size  table 1  consistent     hypothesis    fitness effects   mutations   nsp12 p250   nsp14 p250 269   based     motif   genotype      beneficial   exon aa   detrimental     wt 270   rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/618249 doi  biorxiv preprint exon de background    previous studies        difficult   determine     fitness 271 defects   exon  covs   directly linked   low fidelity replication         272 mechanism    data suggest     proofreading function   nsp14 exon     uncoupled 273         role   replication  figure 4    copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/618249 doi  biorxiv preprint cloning   recovery   recombinant viruses  site directed mutagenesis   mhv genome 306 fragments   performed using  round   horn  pcr  originally described    33  briefly  307 adjacent primers containing   mutation       5¢ phosphorylated using t4 308 polynucleotide kinase  neb  m0201s  using   buffer     t4 dna ligase    contains 309 atp  m0202s  pcr   performed     plasmid template using   q5   fidelity 2x previously  9  briefly  genomic rna   detected     5  6 carboxyfluorescein  fam    341 3  black hole quencher 1  bhq 1  labeled probe targeting nsp2  biosearch technologies   342 petaluma  ca    rna copy     calculated   reference     rna standard derived 343     mhv   fragment  samples   plated   technical duplicate   minimize       344 variation  titers   determined   plaque assay   dbt 9 cells    specific infectivity   345 calculated   pfu   supernatant genomic rna copy  346 347 5 fluorouracil sensitivity assays  stock solutions   5 fluorouracil  sigma f6627    348 prepared   dimethyl sulfoxide  dmso  sensitivity assays   performed   24   plates   349   rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/618249 doi  biorxiv preprint moi = 0 01 pfu/cell    previously described  7    averaged    values   excluded   subsequent analysis     duplicate   differed 371   rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/618249 doi  biorxiv preprint     0 5 ct    relative abundance   competitor   reference   determined   subtracting 372\\r\\n ct thresholds  dctcompetitor = ctcompetitor  ctreference    converted   reflect   fold change   373 ratio  dratio = 2  dct competitor     log10dratio   plotted   passage        change 374   log10dratio       slope   linear regression      relative fitness  note   regressions   fit 375     p1 p4    slight deviations   1 1 ratio     input  p0    skew   slope    copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/618249 doi  biorxiv preprint  9   deleted orf 4a      dispensable   mhv replication   cell culture  52      150 terminal region   ns2   mhv exon  p250   truncated   fused          1 151 frameshift  ns2     phosphodiesterase  pde    protects viral rna   degrading 2¢\\uf0a2à\\uf0e05¢\\uf0a2 152 oligoadenylate    activating factor   cellular rnase    53 55    portion   ns2 deleted     copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/175562 doi  biorxiv preprint similar       viruses  fig  4b      data suggest     increased replication   p250 173 viruses relative   wt mhv       accounted     increased rna synthesis    addition 174   rna synthesis defects  exon  covs       20 fold increased mutation frequencies   175 profoundly increased sensitivity   nucleoside   base analogs relative   wt covs  13  14  16  176 17  38    determine   nucleoside analog sensitivity   mhv exon    altered   177   term passage    treated cells infected   parental   passaged viruses     base 178 analog  5 fluorouracil  5 fu  5 fu   converted intracellularly     nucleoside analog   179 incorporates   growing rna strands   causes           mutations    simplicity    180 hereafter refer   5 fu     nucleoside analog  incorporation   5 fu   increased     absence 181   exon activity  16    viruses displayed   concentration dependent decrease   viral titer   182 differed greatly     susceptibility   5 fu  fig  4c     120µm  wt mhv p3 titers   183 reduced   ~1 log 10     mhv exon  p3 titers   undetectable   5 log 10  fold reduction  184\\r\\n wt mhv 5 fu sensitivity     altered   passage  mhv exon  p250     susceptible 185   mhv exon  p3   5 fu treatment        ~1 5 log 10 decrease   titer   120 µm  replication cycles     5 fu     incorporated  59    mhv exon  p250   multiple 193 mutations     spike glycoprotein  including       spike furin cleavage site   reduced 194 syncytia formation    test     spike mutations manifested   resistance   5 fu    195   rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed      author/funder     m288t     predicted   catalyze nucleotidylation    nsp14  4 ns mutations 215\\r\\n   identified     exon domain    2 ns mutations         terminal n7 216 methyltransferase domain  fig  6b       modeled   structure   mhv nsp14 using phyre 2 217 software  63    resulting     probability similarity     sars cov nsp14 nsp10 complex 218   rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/175562 doi  biorxiv preprint     structures    100    residues 3 519   mhv nsp14    model predicts   five 220 mutations   located close   surface     protein  fig  6b       modeled zinc finger 221 domains contain   ns mutation  f216y  y248h  l473i    mutations  d128e   f216y  222   located near   interface   nsp10   nsp14    neither site   previously   223 implicated   nsp10 nsp14 interaction  15  64  65    ns mutation resulted     d272e 224 substitution   exon motif iii    metal coordinating active site residue    previously reported 225   alanine substitution   d272 results     exon  phenotype  14      viability   226 phenotype     d272e substitution     tested     study    data suggest     227 network   residues evolved   regulate nsp12   nsp14 activity   stability     exon  228 background  229 230 fixed mutations   nsp12   nsp14   mhv exon  p250 directly correlate   231 increased resistance   multiple nucleoside analogs    determine approximately     232 mutations   nsp12   nsp14 arose    performed di deoxy sequencing     protein 233 coding regions roughly   20 passages  p10  31  50  72  90  100  120  140  160  180  200  220  234 240        method    detected consensus ns mutations   p10  p50    p160   nsp12    235   p50   p160   nsp14  fig  6      nsp12   nsp14 carried     complement   p250 236 consensus mutations   p160  except     minority variant  d913e    nsp12 maintained   237  50      population   p200   p250    passage levels correlated   increased 238 replication   mhv exon   fig  2b       decreasing sensitivity   5 fu  fig  7a   neither 239 replication   5 fu sensitivity   mhv exon  changed substantially   p160   p250  240\\r\\n   determine   mhv exon  evolved increased resistance   multiple nucleoside 241   rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed      author/funder    copyright holder     preprint        peer reviewed      author/funder  implicating   complex evolutionary interaction       proteins    measured fitness 284 correlated     patterns   nucleoside analog resistance   rna synthesis associated   285 mutations   nsp12   nsp14  suggesting   link     evolution     virus phenotypes  286   rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/175562 doi  biorxiv preprint\\r\\n   result   confirms   nsp12   nsp14         sufficient   account     287 significantly increased fitness   mhv exon  p250 relative   mhv exon  p3  288 289   rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed      author/funder    copyright holder     preprint        peer reviewed      author/funder  mutations conferring increased replication fidelity   rna viruses     frequently   334 mapped   rdrps  24  25  30  72      findings suggest   mutations   nsp12 p250 335   rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed      author/funder    nsp12  acting     primase/processivity factor  74  75      helicase/ntpase  respectively 358   rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed      author/funder    copyright holder     preprint        peer reviewed      author/funder  \\r\\n cell culture  dbt 9  delayed brain tumor  murine astrocytoma clone 9  cells   maintained 409   described previously  91    virus titer     stock   determined   plaque assay using dbt 9 cells   described 425 previously  14  91    plaque assays   viruses containing   spike protein   mhv exon 426   p250        form syncytia  plaques   visualized   neutral red  sigma  n6264  427 diluted 1 10   pbs containing calcium   magnesium  neutral red   added 24 hours   428 plating   incubated     additional 3 8 hours   formaldehyde fixation  plaque 429 purification   performed   infecting dbt cells   serial dilutions   virus   overlaying 430   rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/175562 doi  biorxiv preprint   cultures   agar  single plaques   isolated  resuspended   pbs containing calcium   431 magnesium    inoculated onto fresh dbts    process   completed 3 times   432 experimental stocks   generated         rna copy     calculated   reference     rna standard derived     mhv 475   fragment  samples   plated   technical duplicate   minimize       variation  titers 476   rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/175562 doi  biorxiv preprint   determined   plaque assay   dbt 9 cells    specific infectivity   calculated   pfu 477   supernatant genomic rna copy  478\\r\\n nucleoside   base analog sensitivity assays  5 azacytidine  azc  5 fluorouracil  5 fu    480 ribavirin  rbv    purchased   sigma  product   a2385  f6627    r9644  481 respectively  stock solutions   5 fu   rbv   prepared   dimethyl sulfoxide  dmso  482\\r\\n 2   methyladenosine  cmea    received   gilead sciences  inc  foster city  ca  483\\r\\n sensitivity assays   performed   described previously  16  viruses   detected       probe     specific infectivity analyses  14  reference 516 viruses   detected     probe targeting     region     10 silent mutations  5 517 tccgaactactgcaaccccaagtg 3    labeled   5  quasar 670   3  black hole 518 quencher 2  bhq 2   biosearch technologies  petaluma  ca  rna copy     519 calculated   reference     rna standard generated     vitro transcription     520 corresponding mhv   fragment    relative rna abundance   calculated     ratio   521 competitor genomes   reference genomes  522   rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed      author/funder    copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/175562 doi  biorxiv preprint   synonymous mutations  purple       564   copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/175562 doi  biorxiv preprint statistical significance   determined using     anova    panel    statistical 587 significance   change   titer   mhv exon  spike p250 relative   mhv exon  p3   588 determined using mann whitney test    0 05   0 01     0 001  ns =   significant  589   copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/175562 doi  biorxiv preprint   rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/175562 doi  biorxiv preprint   rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/175562 doi  biorxiv preprint tuberculosis dna polymerase iii αβ2ε replicase promotes polymerization   reduces 855   rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/175562 doi  biorxiv preprint   rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed      author/funder    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed      author/funder  virus titer  log 10 pfu/ml    rights reserved    reuse allowed   permission \\r\\n   copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/175562 doi  biorxiv preprint   copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/175562 doi  biorxiv preprint   copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/175562 doi  biorxiv preprint   copyright holder     preprint        peer reviewed      author/funder    copyright holder     preprint        peer reviewed      author/funder    copyright holder     preprint        peer reviewed      author/funder    copyright holder     preprint        peer reviewed      author/funder    https //doi org/10 1101/175562 doi  biorxiv preprint  \", TextAnalysis.DocumentMetadata(Languages.English(), \"C:\\\\Users\\\\Chandrapal Panwar\\\\Desktop\\\\python\\\\covid-19\\\\covid_data\\\\treat_Cluster_1.txt\", \"Unknown Author\", \"Unknown Time\"))], 0, Dict{String,Int64}(), Dict{String,Array{Int64,1}}(), TextHashFunction(hash, 100))"
      ]
     },
     "execution_count": 107,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "str"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Julia 1.4.1",
   "language": "julia",
   "name": "julia-1.4"
  },
  "language_info": {
   "file_extension": ".jl",
   "mimetype": "application/julia",
   "name": "julia",
   "version": "1.4.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
